<!DOCTYPE html> <html lang=en class=pb-page data-request-id=95cc4c41-b9a0-4cea-9b41-59bb967999b4 style><!--
 Page saved with SingleFileZ 
 url: https://onlinelibrary.wiley.com/doi/full/10.1111/ecog.04271?fbclid=IwAR3QXjPbqOqDjncGo_X-mq6UY-CRzYlwoPfGtJw-e5vvK6SLEQl8Zo_SxkE 
 saved date: Tue Oct 20 2020 18:48:45 GMT+0200 (Central European Summer Time)
--><head data-pb-dropzone=head><meta charset=utf-8><meta http-equiv=X-UA-Compatible content="IE=edge">
<meta name=pbContext content=";issue:issue:doi\:10.1111/ecog.2019.v42.i8;page:string:Article/Chapter View;ctype:string:Journal Content;website:website:pericles;journal:journal:16000587;wgroup:string:Publication Websites;requestedJournal:journal:16000587;pageGroup:string:Publication Pages;subPage:string:Full Text;article:article:doi\:10.1111/ecog.04271">
<meta name=robots content=noarchive><meta name=pb-robots-disabled><meta property=og:title content="sars: an R package for fitting, evaluating and comparing species–area relationship models"><meta property=og:type content=Article><meta property=og:url content=https://onlinelibrary.wiley.com/doi/10.1111/ecog.04271><meta property=og:image content=https://onlinelibrary.wiley.com/cms/asset/a94c2575-9625-4bbc-8ac6-0fa1327312c9/ecog.2019.v42.i8.cover.gif><meta property=og:site_name content="Wiley Online Library"><meta property=og:description content="The species–area relationship (SAR) constitutes one of the most general ecological patterns globally. A number of different SAR models have been proposed. Recent work has shown that no single model u..."><meta name=viewport content="width=device-width,initial-scale=1"><meta name=apple-itunes-app content="app-id=782998940"><meta name=citation_doi content=10.1111/ecog.04271><meta name=dc.identifier content=10.1111/ecog.04271><meta name=citation_journal_title content=Ecography><meta name=citation_title content="sars: an R package for fitting, evaluating and comparing species–area relationship models"><meta name=citation_keywords content="diversity–area relationship"><meta name=citation_keywords content="island biogeography"><meta name=citation_keywords content="species–area relationship"><meta name=citation_volume content=42><meta name=citation_issue content=8><meta name=citation_firstpage content=1446><meta name=citation_lastpage content=1455><meta name=citation_publication_date content=2019/08/01><meta name=citation_online_date content=2019/04/15><meta name=citation_issn content=1600-0587><meta name=citation_language content=en><meta name=citation_author content="Thomas J. Matthews"><meta name=citation_author_institution content="GEES (School of Geography, Earth and Environmental Sciences) Birmingham Inst. of Forest Research, The Univ. of Birmingham Birmingham UK"><meta name=citation_author_institution content="CE3C – Centre for Ecology, Evolution and Environmental Changes/Azorean Biodiversity Group Univ. dos Açores – Depto de Ciências e Engenharia do Ambiente Angra do Heroísmo Açores Portugal"><meta name=citation_author content="Kostas A. Triantis"><meta name=citation_author_institution content="Dept of Ecology and Taxonomy, Faculty of Biology National and Kapodistrian Univ. of Athens Athens Greece"><meta name=citation_author content="Robert J. Whittaker"><meta name=citation_author_institution content="School of Geography and the Environment Univ. of Oxford Oxford UK"><meta name=citation_author_institution content="Center for Macroecology, Evolution and Climate Natural History Museum of Denmark, Univ. of Copenhagen Copenhagen Ø Denmark"><meta name=citation_author content="François Guilhaumon"><meta name=citation_author_institution content="MARBEC, IRD, Ifremer Univ. Montpellier, CNRS Montpellier France"><meta name=citation_abstract_html_url content=https://onlinelibrary.wiley.com/doi/abs/10.1111/ecog.04271><meta name=citation_fulltext_html_url content=https://onlinelibrary.wiley.com/doi/full/10.1111/ecog.04271><meta name=citation_pdf_url content=https://onlinelibrary.wiley.com/doi/pdf/10.1111/ecog.04271><meta name=citation_publisher content="John Wiley &amp; Sons, Ltd"><meta name=citation_fulltext_world_readable content><meta name=article_references content="Matthews, T.J., Triantis, K.A., Whittaker, R.J. and Guilhaumon, F. (2019), sars: an R package for fitting, evaluating and comparing species–area relationship models. Ecography, 42: 1446-1455. doi:10.1111/ecog.04271"><meta name=epdf_available content=false><meta name=Description content="The species–area relationship (SAR) constitutes one of the most general ecological patterns globally. A number of different SAR models have been proposed. Recent work has shown that no single model..."><title>sars: an R package for fitting, evaluating and comparing species–area relationship models - Matthews - 2019 - Ecography - Wiley Online Library</title><meta name=referrer content=origin-when-cross-origin><meta name=google-site-verification content=ffz4i4DxIJCdGpRXe0XN6KG6o4FYI-nnFW9I_dg8piM><link rel="shortcut icon" href=images/9.ico><style>.sf-hidden{display:none!important}</style><link rel=canonical href="https://onlinelibrary.wiley.com/doi/full/10.1111/ecog.04271?fbclid=IwAR3QXjPbqOqDjncGo_X-mq6UY-CRzYlwoPfGtJw-e5vvK6SLEQl8Zo_SxkE"><body class=pb-ui><noscript>
				<a href="/action/clickThrough?id=1011&url=https%3A%2F%2Fpubads.g.doubleclick.net%2Fgampad%2Fjump%3Fiu%3D%24googlePublisherCategory%26amp%3Bsz%3D160x600%7C160x320%7C160x160%7C120x600%26amp%3Btile%3D2%26amp%3Bc%3D%24random&loc=%2Fdoi%2Ffull%2F10.1111%2Fecog.04271%3Ffbclid%3DIwAR3QXjPbqOqDjncGo_X-mq6UY-CRzYlwoPfGtJw-e5vvK6SLEQl8Zo_SxkE&pubId=69657925&placeholderId=1014&productId=101018" target="_blank">
					<img src="https://pubads.g.doubleclick.net/gampad/ad?iu=$googlePublisherCategory&amp;sz=160x600|160x320|160x160|120x600&amp;tile=2&amp;c=$random">
				</a>
			</noscript>
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
<link rel=stylesheet type=text/css href=stylesheet_0.css>
 <link rel=stylesheet href=stylesheet_1.css>
<link rel=stylesheet type=text/css href=stylesheet_2.css>
<link rel=stylesheet type=text/css href=stylesheet_3.css>
<link rel=stylesheet type=text/css href=stylesheet_4.css>
<link rel=stylesheet type=text/css href=stylesheet_5.css>
<style>.publication-cover-info{margin-bottom:20px}.admin-message{background-color:#005274;color:white}.banner-wrapper{box-shadow:none}.cover-image__image img{max-width:150px;max-height:150px}h1.citation__title{font-size:1.375rem}@media (max-width:991px){h1.citation__title{font-size:1.125rem}h1.citation__title:after{content:""}}.cart-indicator__wrapper{padding:0}</style>
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
<link href=stylesheet_6.css type=text/css rel=stylesheet><style>#gs-casa-r,#gs-casa-c{background:transparent;border:0;box-sizing:content-box;font-family:Arial,sans-serif;font-weight:normal;letter-spacing:normal;line-height:normal;margin:0;opacity:1;outline:none;overflow:visible;padding:0;pointer-events:auto;text-decoration:none;text-transform:none;transition:none;vertical-align:baseline;z-index:0}#gs-casa-r{height:0;position:absolute;right:0;width:0}#gs-casa-c{bottom:100px;height:110px;overflow:hidden;pointer-events:none;position:fixed;right:0;width:116px}@keyframes gs-casa-a-l-ent{0%{transform:translate3d(96px,0,0)}100%{transform:translate3d(0,0,0)}}@media (max-width:599px),(max-height:599px){#gs-casa-c{border-radius:6px 0 0 6px;bottom:30vh;height:96px;position:fixed;transform:translate(0,48px);width:92px}@keyframes gs-casa-a-l-ent{0%{transform:translate3d(72px,0,0)}100%{transform:translate3d(0,0,0)}}}</style><style>#_atssh{width:1px!important;height:1px!important;border:0!important}</style><link id=altmetric-embed-css rel=stylesheet type=text/css href=stylesheet_7.css>
<div class=skip-links>
 <ul class=rlist>
 
 <li><a href=#article__content tabindex=1>Skip to Article Content</a></li>
 <li><a href=#pane-pcw-detailscon tabindex=1>Skip to Article Information</a></li>
 
 </ul>
</div>
 <div id=pb-page-content data-ng-non-bindable>
 <div data-pb-dropzone=main data-pb-dropzone-name=Main>
 <div data-widget-def=pageHeader data-widget-id=62175478-b98a-43d5-90f2-1f2045ac1a7e class=pageHeader>
 
 
 <header class=page-header>
 <div data-pb-dropzone=main>
 
 <div data-widget-def=ux3-layout-widget data-widget-id=bb5b5e2c-d3d6-4015-b586-eeade38f59d4 class=admin-message>
 
 
 <div class=container><div class=row><div>
 
 
 <div class="pb-ad sf-hidden">
 <div style="background=blue;">
 
</div>
 </div>
 
</div></div></div>
 </div>
 
 
 <div class=container><div class=row><div class="contents col-sm-12 col-md-12">
 
 <div class=pull-left>
 <div data-widget-def=ux3-general-image data-widget-id=ff262197-0fb8-4973-aaa2-2fceba4b5a61 class="logo-container hidden-xs hidden-sm hidden-md">
 
 
 <a href=https://onlinelibrary.wiley.com/ title="Wiley Online Library" tabindex=2><img id=mainLogo alt="Wiley Online Library" src=images/10.png></a>
 </div>
 
 <div data-widget-def=ux3-general-image data-widget-id=9a75b789-224c-4745-833a-930cf6c36f35 class="logo-container-mobile hidden-lg hidden-md hidden-sm sf-hidden">
 
 
 <a href=https://onlinelibrary.wiley.com/ title="Wiley Online Library" tabindex=2></a>
 </div>
 
</div><div class=pull-right>
 <div data-widget-def=UX3QuickSearchWidget data-widget-id=48175bb0-0f70-46e6-bb89-7e2781fdb8e4 class=header-quicksearch>
 
 
 <div class=quick-search-wrapper><a href=# title=search aria-label=Search class="quick-search__item sf-hidden"><i aria-hidden=true class=icon-search></i><span class=quick-search__item__arrow></span></a><div data-linkto-page=true class="quick-search quickSearchFormContainer"><div class=main-search-field-wrapper><div class="tab clearfix"><input type=search placeholder=Search value tabindex=13 class="fakeQSInput quickSearchFilter"><label for=search-within class=sr-only>Search within</label><select name=search-within id=search-within tabindex=10 class=search-within><option value=#pane-67065c09-4a88-49cd-934c-ac707951d35c0 selected>This Journal<option value=#pane-67065c09-4a88-49cd-934c-ac707951d35c1>Anywhere</select><ul role=tablist class="rlist tab__content"><li id=pane-67065c09-4a88-49cd-934c-ac707951d35c0 aria-labelledby=pane-67065c09-4a88-49cd-934c-ac707951d35c0-con role=tab class="tab__pane quick-search-pane__anywhere active"><form action=/action/doSearch name=thisJournalQuickSearch title="Quick Search"><div class="input-group option-0"><label for=searchField0 class=hiddenLabel>Search term</label><input type=search name=AllField id=searchField0 placeholder=Search data-auto-complete-max-words=7 data-auto-complete-max-chars=32 data-contributors-conf=3 data-topics-conf=3 data-publication-titles-conf=3 data-history-items-conf=3 value tabindex=9 class="autocomplete actualQSInput quickSearchFilter ui-autocomplete-input" autocomplete=off><span role=status aria-live=polite class=ui-helper-hidden-accessible></span><div class="search-options-wrapper quickSearchFilter sf-hidden"><a href="https://onlinelibrary.wiley.com/search/advanced?publication=16000587" title tabindex=12 class=advanced> Advanced Search</a><a href="https://onlinelibrary.wiley.com/search/advanced?publication=16000587#citation" title tabindex=12 class=citation> Citation Search</a></div></div><button id=quick-search-btn type=submit title=Search tabindex=11 aria-label="Submit Search" class="btn quick-search__button icon-search"></button></form><li id=pane-67065c09-4a88-49cd-934c-ac707951d35c1 aria-labelledby=pane-67065c09-4a88-49cd-934c-ac707951d35c1-con role=tab class="tab__pane quick-search-pane__anywhere sf-hidden"><form action=https://onlinelibrary.wiley.com/action/doSearch name=defaultQuickSearch title="Quick Search"></form></ul></div></div></div></div>
 </div>
 
 <div data-widget-def=ux3-layout-widget data-widget-id=5ef13ed1-f4ac-4b62-a426-07ec9576a586 class=cart-indicator__wrapper>
 
 
 <div class=pull-left>
 <div data-widget-def=eCommerceCartIndicatorWidget data-widget-id=4a9f7c04-a273-4a25-91fb-8e7bd9970288 class=cart-indicator__wrapper>
 
 
 
 
 
 </div>
 
</div><div class=pull-right>
 <div data-widget-def=literatumNavigationLoginBar data-widget-id=d384ca92-7137-45d6-b3c2-78c9c0a98c2b class=login-bar-container>
 
 
 
 
 
 
 <div class="loginBar login-open">
 <a href="https://onlinelibrary.wiley.com/action/showLogin?uri=%2Fdoi%2Ffull%2F10.1111%2Fecog.04271%3Ffbclid%3DIwAR3QXjPbqOqDjncGo_X-mq6UY-CRzYlwoPfGtJw-e5vvK6SLEQl8Zo_SxkE&amp;aria-label=Log+in+or+Register" class=show-login role=button uri="/doi/full/10.1111/ecog.04271?fbclid=IwAR3QXjPbqOqDjncGo_X-mq6UY-CRzYlwoPfGtJw-e5vvK6SLEQl8Zo_SxkE" aria-label="Log in or Register">
 <span class="icon-profile_empty sf-hidden"></span>
 <span class=sign-in-label>Login / Register</span>
 </a>
 
 </div>
 
 </div>
 
</div>
 </div>
 
</div>
</div></div></div>
 </div>
</header>
 </div>
 
 <div data-widget-def=pageBody data-widget-id=72100436-7a82-49fc-933b-c6c9d8c42914 class="pageBody article-page-tablet body-text">
 
 
 <main id=main-content tabindex=-1>
 
 <div class="page-body pagefulltext">
 <div data-pb-dropzone=main>
 
 
 <div>
 <div data-widget-def=literatumAd data-widget-id=0220a30b-90c0-4b51-8637-141edf12f86a class=banner-wrapper>
 
 
 
 <div class="pb-ad sf-hidden">
 <aside role=complementary>
 
 </aside>
 </div>
 
 </div>
 
 
 
</div>
 <section data-widget-def=ux3-layout-widget data-widget-id=0f7bccba-8328-4dbd-a1fa-e84c526dd9ab>
 
 
 <div class=container><div class=row><div class=col-md-12>
 
 <div>
 
 <article data-figures="/action/ajaxShowFigures?widgetId=5cf4c79f-0ae9-4dc5-96ce-77f62de7ada9&amp;ajax=true&amp;doi=10.1111%2Fecog.04271&amp;pbContext=%3Bissue%3Aissue%3Adoi%5C%3A10.1111%2Fecog.2019.v42.i8%3Bpage%3Astring%3AArticle%2FChapter+View%3Bctype%3Astring%3AJournal+Content%3Bwebsite%3Awebsite%3Apericles%3Bjournal%3Ajournal%3A16000587%3Bwgroup%3Astring%3APublication+Websites%3BrequestedJournal%3Ajournal%3A16000587%3BpageGroup%3Astring%3APublication+Pages%3BsubPage%3Astring%3AFull+Text%3Barticle%3Aarticle%3Adoi%5C%3A10.1111%2Fecog.04271&amp;accordionHeadingWrapper=h2" data-references="/action/ajaxShowEnhancedAbstract?widgetId=5cf4c79f-0ae9-4dc5-96ce-77f62de7ada9&amp;ajax=true&amp;doi=10.1111%2Fecog.04271&amp;pbContext=%3Bissue%3Aissue%3Adoi%5C%3A10.1111%2Fecog.2019.v42.i8%3Bpage%3Astring%3AArticle%2FChapter+View%3Bctype%3Astring%3AJournal+Content%3Bwebsite%3Awebsite%3Apericles%3Bjournal%3Ajournal%3A16000587%3Bwgroup%3Astring%3APublication+Websites%3BrequestedJournal%3Ajournal%3A16000587%3BpageGroup%3Astring%3APublication+Pages%3BsubPage%3Astring%3AFull+Text%3Barticle%3Aarticle%3Adoi%5C%3A10.1111%2Fecog.04271&amp;accordionHeadingWrapper=h2" data-related="/action/ajaxShowRecommended?widgetId=5cf4c79f-0ae9-4dc5-96ce-77f62de7ada9&amp;ajax=true&amp;doi=10.1111%2Fecog.04271&amp;pbContext=%3Bissue%3Aissue%3Adoi%5C%3A10.1111%2Fecog.2019.v42.i8%3Bpage%3Astring%3AArticle%2FChapter+View%3Bctype%3Astring%3AJournal+Content%3Bwebsite%3Awebsite%3Apericles%3Bjournal%3Ajournal%3A16000587%3Bwgroup%3Astring%3APublication+Websites%3BrequestedJournal%3Ajournal%3A16000587%3BpageGroup%3Astring%3APublication+Pages%3BsubPage%3Astring%3AFull+Text%3Barticle%3Aarticle%3Adoi%5C%3A10.1111%2Fecog.04271&amp;accordionHeadingWrapper=h2&amp;showSubjects=true&amp;taxonomiesCodes=enter+your+taxonomy+codes+here&amp;displayCitedByLink=true&amp;displayAlmetricDropzone=true&amp;taxonomyUri=publication-features&amp;topicUri=hide-issue-metadata" data-details="/action/ajaxShowPubInfo?widgetId=5cf4c79f-0ae9-4dc5-96ce-77f62de7ada9&amp;ajax=true&amp;doi=10.1111%2Fecog.04271&amp;pbContext=%3Bissue%3Aissue%3Adoi%5C%3A10.1111%2Fecog.2019.v42.i8%3Bpage%3Astring%3AArticle%2FChapter+View%3Bctype%3Astring%3AJournal+Content%3Bwebsite%3Awebsite%3Apericles%3Bjournal%3Ajournal%3A16000587%3Bwgroup%3Astring%3APublication+Websites%3BrequestedJournal%3Ajournal%3A16000587%3BpageGroup%3Astring%3APublication+Pages%3BsubPage%3Astring%3AFull+Text%3Barticle%3Aarticle%3Adoi%5C%3A10.1111%2Fecog.04271&amp;accordionHeadingWrapper=h2&amp;showSubjects=true&amp;taxonomiesCodes=enter+your+taxonomy+codes+here&amp;displayCitedByLink=true&amp;displayAlmetricDropzone=true&amp;taxonomyUri=publication-features&amp;topicUri=hide-issue-metadata" data-mathjax="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.5/MathJax.js?config=TeX-AMS-MML_HTMLorMML" class=hide_CASA><div class="row article-row"><div id=article__content tabindex=-1 class="col-sm-12 col-md-8 col-lg-8 article__content article-row-left"><div data-pb-dropzone=mrwBannerDropZone class=mrw-banner-dropzone>
 
 
</div><div class=article-citation><div class=citation>
 <div class=pb-dropzone data-pb-dropzone=publicaitonContent-series-title>
 <div id=journal-banner-text class=journal-banner-text><a href=https://onlinelibrary.wiley.com/journal/16000587 title="Ecography homepage">Ecography</a></div><a href=https://onlinelibrary.wiley.com/toc/16000587/2019/42/8 class="volume-issue sf-hidden">Volume 42, Issue 8</a><a href=https://onlinelibrary.wiley.com/journal/16000587 title="Ecography homepage" class=citation--logo><img id=journal-banner-image test=test-value alt=Ecography src=images/15.jpg class=journal-banner-image></a>
 </div>
 <div class="doi-access-container clearfix"><span class=primary-heading><span class=primary-heading>Software note</span></span>
 <div class=doi-access-wrapper><div class="free-access access-type"> <i aria-hidden=true class=icon-icon-lock_open></i> <div class=doi-access> <div class=doi-access tabindex=0>Free Access</div></div></div></div></div>
 <h1 class=citation__title>sars: an R package for fitting, evaluating and comparing species–area relationship models</h1>
 <div class="rlist--inline loa comma visible-xs mobile-authors loa-authors-trunc sf-hidden">
 <div class="accordion-tabbed__tab-mobile accordion__closed">
 </div>
 <div class="accordion-tabbed__tab-mobile accordion__closed">
 </div>
 <div class="accordion-tabbed__tab-mobile accordion__closed">
 </div>
 <div class="accordion-tabbed__tab-mobile accordion__closed">
 </div>
 </div>
 <div class="loa-wrapper loa-authors hidden-xs">
 <div id=sb-1 class=accordion>
 <div class=accordion-tabbed>
 <div class="accordion-tabbed__tab-mobile accordion__closed"><a href="https://onlinelibrary.wiley.com/action/doSearch?ContribAuthorStored=Matthews%2C+Thomas+J" class="author-name accordion-tabbed__control" data-id=a1 data-db-target-for=a1 aria-controls=a1 aria-haspopup=true id=a1_Ctrl role=button><span>Thomas J. Matthews<i aria-hidden=true class=icon-mail_outline></i></span><i aria-hidden=true class="icon-section_arrow_d sf-hidden"></i></a><div class="author-info accordion-tabbed__content sf-hidden" data-db-target-of=a1 aria-labelledby=a1_Ctrl role=region id=a1>
 <p>
 <p></p><a target=_blank title="Link to external resource" href=https://orcid.org/0000-0002-7624-244X>https://orcid.org/0000‐0002‐7624‐244X</a><p></p>
 <p>GEES (School of Geography, Earth and Environmental Sciences), Birmingham Inst. of Forest Research, The Univ. of Birmingham, Birmingham, UK</p>
 <p>CE3C – Centre for Ecology, Evolution and Environmental Changes/Azorean Biodiversity Group, Univ. dos Açores – Depto de Ciências e Engenharia do Ambiente, Angra do Heroísmo, Açores, Portugal</p><a class=moreInfoLink href="https://onlinelibrary.wiley.com/action/doSearch?ContribAuthorStored=Matthews%2C+Thomas+J">Search for more papers by this author</a></div>
 </div>
 <div class="accordion-tabbed__tab-mobile accordion__closed"><a href="https://onlinelibrary.wiley.com/action/doSearch?ContribAuthorStored=Triantis%2C+Kostas+A" class="author-name accordion-tabbed__control" data-id=a2 data-db-target-for=a2 aria-controls=a2 aria-haspopup=true id=a2_Ctrl role=button><span>Kostas A. Triantis</span><i aria-hidden=true class="icon-section_arrow_d sf-hidden"></i></a><div class="author-info accordion-tabbed__content sf-hidden" data-db-target-of=a2 aria-labelledby=a2_Ctrl role=region id=a2>
 <p></p>
 <p>Dept of Ecology and Taxonomy, Faculty of Biology, National and Kapodistrian Univ. of Athens, Athens, Greece</p><a class=moreInfoLink href="https://onlinelibrary.wiley.com/action/doSearch?ContribAuthorStored=Triantis%2C+Kostas+A">Search for more papers by this author</a></div>
 </div>
 <div class="accordion-tabbed__tab-mobile accordion__closed"><a href="https://onlinelibrary.wiley.com/action/doSearch?ContribAuthorStored=Whittaker%2C+Robert+J" class="author-name accordion-tabbed__control" data-id=a3 data-db-target-for=a3 aria-controls=a3 aria-haspopup=true id=a3_Ctrl role=button><span>Robert J. Whittaker</span><i aria-hidden=true class="icon-section_arrow_d sf-hidden"></i></a><div class="author-info accordion-tabbed__content sf-hidden" data-db-target-of=a3 aria-labelledby=a3_Ctrl role=region id=a3>
 <p></p>
 <p>School of Geography and the Environment, Univ. of Oxford, Oxford, UK</p>
 <p>Center for Macroecology, Evolution and Climate, Natural History Museum of Denmark, Univ. of Copenhagen, Copenhagen Ø, Denmark</p><a class=moreInfoLink href="https://onlinelibrary.wiley.com/action/doSearch?ContribAuthorStored=Whittaker%2C+Robert+J">Search for more papers by this author</a></div>
 </div>
 <div class="accordion-tabbed__tab-mobile accordion__closed"><a href="https://onlinelibrary.wiley.com/action/doSearch?ContribAuthorStored=Guilhaumon%2C+Fran%C3%A7ois" class="author-name accordion-tabbed__control" data-id=a4 data-db-target-for=a4 aria-controls=a4 aria-haspopup=true id=a4_Ctrl role=button><span>François Guilhaumon</span><i aria-hidden=true class="icon-section_arrow_d sf-hidden"></i></a><div class="author-info accordion-tabbed__content sf-hidden" data-db-target-of=a4 aria-labelledby=a4_Ctrl role=region id=a4>
 <p></p>
 <p>MARBEC, IRD, Ifremer, Univ. Montpellier, CNRS, Montpellier, France</p><a class=moreInfoLink href="https://onlinelibrary.wiley.com/action/doSearch?ContribAuthorStored=Guilhaumon%2C+Fran%C3%A7ois">Search for more papers by this author</a></div>
 </div>
 </div>
 </div>
 </div>
 <div class=epub-sections><div class=epub-section><span class=epub-state>First published: </span><span class=epub-date>15 March 2019</span></div>
 <div class=epub-section><a class=epub-doi aria-label="Digital Object Identifier" href=https://doi.org/10.1111/ecog.04271>https://doi.org/10.1111/ecog.04271</a></div><div class="epub-section cited-by-count"><span>Citations: <a href=#citedby-section>11</a></span></div>
 </div>
</div></div><div class=fixedCoolBar style=padding-top:0px><nav aria-label="Article navigation and tools" class="coolBar stickybar" style=top:auto;width:auto;left:auto><div class="stickybar__wrapper coolBar__wrapper clearfix"><div class="rlist coolBar__zone"><div class=coolBar__section><a href=# data-db-target-for=article data-db-switch=icon-close_thin aria-haspopup=true aria-controls=article_Pop role=button id=article_Ctrl class="coolBar__ctrl hidden-lg sf-hidden"><i class=icon-abstract></i><span>About</span></a><div data-db-target-of=article aria-labelledby=article_Ctrl role=menu id=article_Pop class="coolBar__drop fixed rlist dropBlock__holder sf-hidden"><div data-target=".article-row-right .tab__nav, .coolBar--download" data-remove=false data-targetclass="hidden-xs hidden-sm" data-toggle=transplant data-direction=from data-transplant=self class="transplant showit"></div></div></div><div class="coolBar__section coolBar--sections"><a href=# data-db-target-for=sections data-db-switch=icon-close_thin aria-haspopup=true aria-controls=sections_Pop role=button id=sections_Ctrl class=coolBar__ctrl><i class=icon-list-unordered></i><span>Sections</span></a><div data-db-target-of=sections aria-labelledby=sections_Ctrl role=menu id=sections_Pop class="coolBar__drop dropBlock__holder rlist fixed sf-hidden"><ul class="rlist w-slide--list"></ul></div></div></div><div class="coolBar__second rlist"><div class="coolBar__section coolBar--download PdfLink scroll-to-target cloned"><a href=https://onlinelibrary.wiley.com/doi/epdf/10.1111/ecog.04271 title=ePDF class="coolBar__ctrl pdf-download"><img src=images/19.png alt=PDF>PDF</a></div>
 
 <div class=relative><a href=# data-db-target-for=0e406ce9-5c43-4e2d-b741-e0b43d1f1865 data-db-switch=icon-close_thin aria-haspopup=true aria-controls=0e406ce9-5c43-4e2d-b741-e0b43d1f1865_Pop role=button id=0e406ce9-5c43-4e2d-b741-e0b43d1f1865_Ctrl class=article-tools__ctrl><i aria-hidden=true class=icon-Icon_Tools></i><span>Tools</span></a><div data-db-target-of=0e406ce9-5c43-4e2d-b741-e0b43d1f1865 aria-labelledby=0e406ce9-5c43-4e2d-b741-e0b43d1f1865_Ctrl role=menu id=0e406ce9-5c43-4e2d-b741-e0b43d1f1865_Pop class="article-tools__block fixed dropBlock__holder sf-hidden"><ul class="rlist w-slide--list"></ul></div></div>
 
 <div class=relative><a id=share-access__ctrl href=# data-db-target-for=dff42abe-4b17-4fe9-a9c5-625405cd1007 data-db-switch=icon-close_thin aria-haspopup=true aria-controls=share_Pop role=button class=share-access__ctrl><i aria-hidden=true class=icon-Icon_Share></i><span>Share</span></a><a id=share__ctrl href=# data-db-target-for=dff42abe-4b17-4fe9-a9c5-625405cd1007 data-db-switch=icon-close_thin aria-haspopup=true aria-controls=share_Pop role=button class="share__ctrl hidden sf-hidden"><i aria-hidden=true class=icon-Icon_Share></i><span>Share</span></a><div data-db-target-of=dff42abe-4b17-4fe9-a9c5-625405cd1007 aria-labelledby=share-access__ctrl role=menu id=share_Pop class="share__block dropBlock__holder fixed sf-hidden">
 
 <div data-share-access="/action/authorizeSharing?doi=10.1111/ecog.04271&amp;shareType=P2P" class="share-access hidden"></div><div class="ux-modal-container share-access__modal"></div>
<p>Share a link</p><span class=sr-only>Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"></ul></div></div>
</div></div></nav></div><div id=accessDenialslot class=accessDenialslot></div><div class=article__body><article>
 
 <div class=abstract-group>
 <section class="article-section article-section__abstract" lang=en data-lang=en id=section-1-en>
 <h2 id=d37977029 class="article-section__header section__title main abstractlang_en">Abstract</h2>
 <div class="article-section__content en main">
 <p>The species–area relationship (SAR) constitutes one of the most general ecological patterns globally. A number of different SAR models have been proposed. Recent work has shown that no single model universally provides the best fit to empirical SAR datasets: multiple models may be of practical and theoretical interest. However, there are no software packages available that a) allow users to fit the full range of published SAR models, or b) provide functions to undertake a range of additional SAR‐related analyses. To address these needs, we have developed the R package ‘sars’ that provides a wide variety of SAR‐related functionality. The package provides functions to: a) fit 20 SAR models using non‐linear and linear regression, b) calculate multi‐model averaged curves using various information criteria, and c) generate confidence intervals using bootstrapping. Plotting functions allow users to depict and scrutinize the fits of individual models and multi‐model averaged curves. The package also provides additional SAR functionality, including functions to fit, plot and evaluate the random placement model using a species–sites abundance matrix, and to fit the general dynamic model of oceanic island biogeography. The ‘sars’ R package will aid future SAR research by providing a comprehensive set of simple to use tools that enable in‐depth exploration of SARs and SAR‐related patterns. The package has been designed to allow other researchers to add new functions and models in the future and thus the package represents a resource for future SAR work that can be built on and expanded by workers in the field.</p>
 
 </div>
 </section>
 </div>
 <div class=pb-dropzone data-pb-dropzone=below-abstract-group>
 
 </div>
 
 <section class="article-section article-section__full">
 <section class=article-section__content id=sec-1>
 
 <h2 class="article-section__title section__title section1" id=sec-1-title> Background</h2>
 
 <p>The species–area relationship (SAR) describes the near universally observed pattern whereby the number of species increases with the area sampled, and it has been described as one of ecology's few laws (Rosenzweig <span><a href=#bib-0027 class="bibLink tab-link" id=bib-0027R data-tab=pane-pcw-references>1995</a></span>). The SAR is a fundamental component of numerous ecological and biogeographical theories, such as the equilibrium theory of island biogeography (MacArthur and Wilson <span><a href=#bib-0018 class="bibLink tab-link" id=bib-0018R data-tab=pane-pcw-references>1967</a></span>). In addition, SAR models have been widely used in applied ecology and conservation biogeography: for example, to predict the number of extinctions due to habitat loss (May et al. <span><a href=#bib-0027 class="bibLink tab-link" id=bib-0027R data-tab=pane-pcw-references>1995</a></span>). Numerous types of SAR have been described, and one primary dichotomy employed is the split of SARs into island SARs (ISARs), whereby each data point is an individual island or isolated sample, and species accumulation curves (SACs) that represent cumulative counts of increased species number with sampling area (Gray et al. 2004; see Scheiner <span><a href=#bib-0028 class="bibLink tab-link" id=bib-0028R data-tab=pane-pcw-references>2003</a></span> for an alternative SAR typology). Whilst the remainder of the paper and the described R package are focused on ISARs, the models and the model fitting procedure can equally be applied to SACs (Matthews et al. 2016a), although it should be noted that in SACs the data points are not independent of one another.
 </p>
 
 <p>Over 20 SAR models have been described in the literature (Dengler 2009, Tjørve 2009, Triantis et al. 2012). However, despite this wide range of models, the majority of SAR studies are still based exclusively on the power model (Arrhenius 1921), which if fitted in its non‐linear (untransformed) form generally takes a convex form. Often, the log–log representation of the power model is used as it can be fitted using standard linear regression, and its parameters are more easily interpretable (Rosenzweig <span><a href=#bib-0027 class="bibLink tab-link" id=bib-0027R data-tab=pane-pcw-references>1995</a></span>). However, whilst the power model has been found to provide a reasonable fit to a wide range of datasets (Dengler 2009, Triantis et al. 2012, Matthews et al. 2016b), it is not universally the best model, and a number of studies have reported other models to provide better fits to empirical data (Triantis et al. 2012, Benchimol and Peres <span><a href=#bib-0002 class="bibLink tab-link" id=bib-0002R data-tab=pane-pcw-references>2013</a></span>, Matthews et al. 2016b). The possibility of scale dependency of the form of the SAR has long been of interest, with, for example, a theoretical case being made for SARs at intermediate spatial scales being approximated by a power model, whilst at larger spatial scales the form of the SAR has been theorised to be sigmoidal (He and Legendre <span><a href=#bib-0016 class="bibLink tab-link" id=bib-0016R data-tab=pane-pcw-references>1996</a></span>, Lomolino 2000). Additionally, it is only recently that the SAR for archipelagos as units of analysis and not just islands has started to be studied (Triantis et al. 2015), and thus we know little about the form of archipelago SARs.
 </p>
 
 <p>Due to the increased recognition of model uncertainty in SAR research, a number of recent studies have employed a multi‐model inference approach (Burnham and Anderson <span><a href=#bib-0004 class="bibLink tab-link" id=bib-0004R data-tab=pane-pcw-references>2002</a></span>) in the analysis of SARs, whereby either 1) multiple SAR models are compared using various criteria (e.g. AIC) and a best model is chosen (Dengler 2009, Benchimol and Peres <span><a href=#bib-0002 class="bibLink tab-link" id=bib-0002R data-tab=pane-pcw-references>2013</a></span>), or 2) multiple SAR models are fitted and a multi‐model averaged curve is calculated using, for example, AIC weights (Guilhaumon et al. 2008). We are not aware of any published software package that enables users to fit, and create multi‐model averaged curves using more than eight SAR models. Considering currently available software, the BAT R package (Cardoso et al. 2014) provides functions to fit three SAR models (linear, power and logarithmic); however, this package is focused on general biodiversity assessment and thus does not provide any additional SAR functionality. The mmSAR R package (Guilhaumon et al. <span><a href=#bib-0010 class="bibLink tab-link" id=bib-0010R data-tab=pane-pcw-references>2010</a></span>) is focussed on SARs and while it allows users to fit eight SAR models using an information theoretic framework, it does not include several models that have been found to provide the best fits to several empirical datasets (Triantis et al. 2012, Matthews et al. 2016b).
 </p>
 
 <p>To provide a set of tools to fill these gaps, we have developed the R package ‘sars’. The package provides functions to fit 20 SAR models using non‐linear and linear regression, calculate multi‐model averaged curves using various information criteria, and generate confidence intervals using bootstrapping. Novel features compared with mmSAR include 1) user‐friendly functions for plotting (the user can now plot weighted multimodel SAR curves along with the individual SAR model curves) and 2) determining the observed shape of the model fit (i.e. linear, convex up, convex down or sigmoidal) and 3) presence or not of an asymptote, and 4) functions to fit, plot and evaluate Coleman's (<span><a href=#bib-0007 class="bibLink tab-link" id=bib-0007R data-tab=pane-pcw-references>1981</a></span>) random placement model using a species‐site abundance matrix, and 5) to fit the general dynamic model (GDM) of island biogeography (Whittaker et al. 2008). In addition, the mmSAR package (which has been deprecated) no longer complies with recognised programming good practice (Wickham 2015), is not on CRAN (the main repository of R packages), and is not user friendly (e.g. it requires the user to load individual models prior to fitting). There was therefore a need to design a new package from scratch.
 </p>
 </section>
 <section class=article-section__content id=sec-2>
 
 <h2 class="article-section__title section__title section1" id=sec-2-title> Methods and features</h2>
 
 <p>The ‘sars’ (species–area relationships) package has been programmed using standard S3 methods and is available on CRAN (ver. 1.1.1; R Core Team), with the development version on GitHub (txm676/sars), meaning researchers can easily add in their own models and functions and integrate these into the multi‐model inference framework. Thus, the package represents a resource for future SAR work that can be built on and expanded by workers in the field.</p>
 
 <section class=article-section__sub-content id=sec-3>
 
 <h3 class="article-section__sub-title section2" id=sec-3-title> Fitting individual SAR models and a set of SAR models</h3>
 
 <p>The package provides functions to fit each of the 20 SAR models listed in Table 1 (see also Triantis et al. 2012). The ‘sar_models’ function can be used to bring up a list of the 20 model names and Table 1 can be generated in the package using the ‘display_sars_models’ function. With the exception of the linear model (which is fitted using standard linear regression), all models are fitted using non‐linear regression and the model parameters are estimated by minimizing the residual sum of squares with an unconstrained Nelder–Mead optimization algorithm and the ‘optim’ R function (R Core Team). The starting values for the parameter estimates are carefully chosen to avoid numerical problems and to speed up the convergence process. However, custom starting values can be provided for any of the 20 models using the ‘start’ argument in the model fit functions.</p>
 
 <div class=article-table-content id=ecog12417-tbl-0001>
 <header class=article-table-caption><span class=table-caption__label>Table Table 1..
 </span>The 20 SAR models available in the ‘sars’ R package. The name of the function to fit each of the models in the package is shown, as is the number of parameters of each model, the model equation, and the model shape. For the model equation, A = sample area, and d, c, z and f are free parameters. Each equation is calculating the number of species. The model shape is the general model shape, as in Triantis et al. (2012). As outlined in the main text, the observed shape can deviate from the general model shape in cases when fitting certain models. This table can be generated in the package using the ‘display_sars_models’ function. *In Triantis et al. (2012), this model was called the exponential model
 </header>
 
 
 
 
 <div class=article-table-content-wrapper>
 <table class="table article-section__table">
 
 
 
 
 
 
 <thead>
 <tr>
 
 <th class=left-aligned>Model</th>
 
 <th class=center-aligned>Function name</th>
 
 <th class=center-aligned>No. parameters</th>
 
 <th class=center-aligned>Equation</th>
 
 <th class=center-aligned>Model shape</th>
 </tr>
 </thead>
 <tbody>
 
 <tr>
 
 <td class=left-aligned>Asymptotic</td>
 
 <td class=left-aligned>sar_asymp()</td>
 
 <td class=center-aligned>3</td>
 
 <td class=left-aligned>d − c × z^A</td>
 
 <td class=left-aligned>Convex</td>
 </tr>
 
 <tr>
 
 <td class=left-aligned>Beta‐P</td>
 
 <td class=left-aligned>sar_betap()</td>
 
 <td class=center-aligned>4</td>
 
 <td class=left-aligned>d × (1 − (1 + (A/c)^z)^(−f))</td>
 
 <td class=left-aligned>Sigmoid</td>
 </tr>
 
 <tr>
 
 <td class=left-aligned>Chapman–Richards</td>
 
 <td class=left-aligned>sar_chapman()</td>
 
 <td class=center-aligned>3</td>
 
 <td class=left-aligned>d × (1 − exp(−z × A)^c)</td>
 
 <td class=left-aligned>Sigmoid</td>
 </tr>
 
 <tr>
 
 <td class=left-aligned>Logarithmic*</td>
 
 <td class=left-aligned>sar_loga()</td>
 
 <td class=center-aligned>2</td>
 
 <td class=left-aligned>c + z × log(A)</td>
 
 <td class=left-aligned>Convex</td>
 </tr>
 
 <tr>
 
 <td class=left-aligned>Extended Power 1</td>
 
 <td class=left-aligned>sar_epm1()</td>
 
 <td class=center-aligned>3</td>
 
 <td class=left-aligned>c × A^(z × A^(−d))</td>
 
 <td class=left-aligned>Convex/Sigmoid</td>
 </tr>
 
 <tr>
 
 <td class=left-aligned>Extended Power 2</td>
 
 <td class=left-aligned>sar_epm2()</td>
 
 <td class=center-aligned>3</td>
 
 <td class=left-aligned>c × A^(z − (d/A))</td>
 
 <td class=left-aligned>Sigmoid</td>
 </tr>
 
 <tr>
 
 <td class=left-aligned>Gompertz</td>
 
 <td class=left-aligned>sar_gompertz()</td>
 
 <td class=center-aligned>3</td>
 
 <td class=left-aligned>d × exp(−exp(−z × (A − c)))</td>
 
 <td class=left-aligned>Sigmoid</td>
 </tr>
 
 <tr>
 
 <td class=left-aligned>Kobayashi</td>
 
 <td class=left-aligned>sar_koba()</td>
 
 <td class=center-aligned>2</td>
 
 <td class=left-aligned>c × log(1 + A/z)</td>
 
 <td class=left-aligned>Convex</td>
 </tr>
 
 <tr>
 
 <td class=left-aligned>Linear</td>
 
 <td class=left-aligned>sar_linear()</td>
 
 <td class=center-aligned>2</td>
 
 <td class=left-aligned>c + z × A</td>
 
 <td class=left-aligned>Linear</td>
 </tr>
 
 <tr>
 
 <td class=left-aligned>Logistic</td>
 
 <td class=left-aligned>sar_heleg()</td>
 
 <td class=center-aligned>3</td>
 
 <td class=left-aligned>c/(f + A^(−z))</td>
 
 <td class=left-aligned>Sigmoid</td>
 </tr>
 
 <tr>
 
 <td class=left-aligned>Monod</td>
 
 <td class=left-aligned>sar_monod()</td>
 
 <td class=center-aligned>2</td>
 
 <td class=left-aligned>d/(1 + c × A^(−1))</td>
 
 <td class=left-aligned>Convex</td>
 </tr>
 
 <tr>
 
 <td class=left-aligned>Morgan–Mercer–Flodin</td>
 
 <td class=left-aligned>sar_mmf()</td>
 
 <td class=center-aligned>3</td>
 
 <td class=left-aligned>d/(1 + c × A^(−z))</td>
 
 <td class=left-aligned>Sigmoid</td>
 </tr>
 
 <tr>
 
 <td class=left-aligned>Negative Exponential</td>
 
 <td class=left-aligned>sar_negexpo()</td>
 
 <td class=center-aligned>2</td>
 
 <td class=left-aligned>d × (1 − exp(−z × A))</td>
 
 <td class=left-aligned>Convex</td>
 </tr>
 
 <tr>
 
 <td class=left-aligned>Persistence Function 1</td>
 
 <td class=left-aligned>sar_p1()</td>
 
 <td class=center-aligned>3</td>
 
 <td class=left-aligned>c × A^z × exp(−d × A)</td>
 
 <td class=left-aligned>Convex</td>
 </tr>
 
 <tr>
 
 <td class=left-aligned>Persistence Function 2</td>
 
 <td class=left-aligned>sar_p2()</td>
 
 <td class=center-aligned>3</td>
 
 <td class=left-aligned>c × A^z × exp(−d/A)</td>
 
 <td class=left-aligned>Sigmoid</td>
 </tr>
 
 <tr>
 
 <td class=left-aligned>Power</td>
 
 <td class=left-aligned>sar_power()</td>
 
 <td class=center-aligned>2</td>
 
 <td class=left-aligned>c × A^z</td>
 
 <td class=left-aligned>Convex</td>
 </tr>
 
 <tr>
 
 <td class=left-aligned>Power Rosenzweig</td>
 
 <td class=left-aligned>sar_powerR()</td>
 
 <td class=center-aligned>3</td>
 
 <td class=left-aligned>f + c × A^z</td>
 
 <td class=left-aligned>Convex</td>
 </tr>
 
 <tr>
 
 <td class=left-aligned>Rational</td>
 
 <td class=left-aligned>sar_ratio()</td>
 
 <td class=center-aligned>3</td>
 
 <td class=left-aligned>(c + z × A)/(1 + d × A)</td>
 
 <td class=left-aligned>Convex</td>
 </tr>
 
 <tr>
 
 <td class=left-aligned>Weibull‐3</td>
 
 <td class=left-aligned>sar_weibull3()</td>
 
 <td class=center-aligned>3</td>
 
 <td class=left-aligned>d × (1 − exp(−c × A^z))</td>
 
 <td class=left-aligned>Sigmoid</td>
 </tr>
 
 <tr>
 
 <td class=left-aligned>Weibull‐4</td>
 
 <td class=left-aligned>sar_weibull4()</td>
 
 <td class=center-aligned>4</td>
 
 <td class=left-aligned>d × (1 − exp(−c × A^z))^f</td>
 
 <td class=left-aligned>Sigmoid</td>
 </tr>
 </tbody>
 </table>
 </div>
 <div class=article-section__table-source></div>
 </div>
 
 <p>Each individual model fit returns an object of class ‘sars’, which is a list of 22 elements containing relevant model fit information, such as the model parameter estimates, the fitted values, the residuals, model fit statistics (e.g. AIC, R<sup>2</sup>), the observed model shape (linear, convex or sigmoidal), whether or not the fit is asymptotic, and convergence information. The returned object can easily be plotted using the ‘plot.sars’ generic function; as this function is based on the base R plotting framework, the plot aesthetics can be edited using standard plotting arguments (see the ‘plot.sars’ documentation in the package). Summary and print generic functions are also provided for class ‘sars’; these functions follow the output of the standard ‘lm’ function in the ‘stats’ R package (R Core Team).
 </p>
 
 <p>Multiple SAR models can be fitted to the same dataset using the ‘sar_multi’ function and the resultant n model fit objects stored together as a ‘fit_collection’ object. This object is a list of class ‘sars’, where each of the n elements contains an individual SAR model fit. Using the ‘plot.sars’ generic function on a ‘fit_collection’ object generates a grid of the n individual model fit plots (Fig. 1). The output from all of the examples given in the paper can be found in the package vignette (available from CRAN and GitHub).</p>
 
 <section class=article-section__inline-figure>
 <figure class=figure id=ecog12417-fig-0001><a target=_blank href=https://onlinelibrary.wiley.com/cms/asset/59ad0266-1086-4a85-bf1b-a227505d7737/ecog12417-fig-0001-m.jpg>
 <picture>
 <img class=figure__image src=images/11.jpg data-lg-src=/cms/asset/59ad0266-1086-4a85-bf1b-a227505d7737/ecog12417-fig-0001-m.jpg alt=image title=image srcset sizes></picture></a><figcaption class=figure__caption>
 <div class=figure__caption__header><strong class=figure__title>Figure 1</strong><div class=figure-extra><a href=# class=open-figure-link>Open in figure viewer</a><a href="https://onlinelibrary.wiley.com/action/downloadFigures?id=ecog12417-fig-0001&amp;doi=10.1111%2Fecog.04271" class=ppt-figure-link><i aria-hidden=true class=icon-Icon_Download></i><span>PowerPoint</span></a></div>
 </div>
 <div class="figure__caption figure__caption-text">
 <p>The fit (solid lines) of the linear and power SAR models to data of plant species richness (solid circles) on 16 Galapagos islands (Preston 1962; the largest island provided in the source paper has been excluded as it is almost six times larger than the second largest). The code to produce this type of plot is given as an example in the ‘Fitting individual SAR models and a set of SAR models’ section. The R<sup>2</sup> values for the linear and power models are 0.32 and 0.49.</p>
 </div>
 </figcaption>
 </figure>
 </section>
 
 <p><i>#load an example dataset (</i>Preston, #1962<i>), fit the logarithmic SAR model, #return a model fit summary and plot the #model fit</i>.
 </p>
 
 <p>library(sars)</p>
 
 <p>data(galap)</p>
 
 <p>fit &lt;‐ sar_loga(data = galap)</p>
 
 <p>summary(fit)</p>
 
 <p>##</p>
 
 <p>## Model:</p>
 
 <p>## Logarithmic</p>
 
 <p>##</p>
 
 <p>## Call:</p>
 
 <p>## S == c + z * log(A)</p>
 
 <p>##</p>
 
 <p>## Did the model converge: TRUE</p>
 
 <p>##</p>
 
 <p>## Residuals:</p>
 
 <p>## 0% 25% 50% 75% 100%</p>
 
 <p>## ‐164.000 ‐25.675 11.350 38.725 115.700</p>
 
 <p>##</p>
 
 <p>## Parameters:</p>
 
 <p>## Estimate Std. Error t value Pr(&gt;|t|) ## 2.5% 97.5%</p>
 
 <p>## c 0.2915341 34.5985767 0.0084262 ## 0.9933958 ‐68.9056193 69.489</p>
 
 <p>## z 30.2802630 8.3203931 3.6392827 ## 0.0026812 13.6394768 46.921</p>
 
 <p>##</p>
 
 <p>## R‐squared: 0.49, Adjusted R‐squared: ## 0.41</p>
 
 <p>## AIC: 141.78, AICc: 143.78, BIC: 144.1</p>
 
 <p>## Observed shape: convex up, Asymptote: ## FALSE</p>
 
 <p>##</p>
 
 <p>##</p>
 
 <p>## Warning: The fitted values of the model ## contain negative values (i.e. negative ## species richness values)</p>
 
 <p>plot(fit)</p>
 
 <p><i>#Create a fit_collection object containing #multiple SAR model fits, and plot all #fits</i>.
 </p>
 
 <p>fitC &lt;‐ sar_multi(data = galap, obj = c("power", "loga", "monod"))</p>
 
 <p>##</p>
 
 <p>## Now attempting to fit the 3 SAR models:</p>
 
 <p>##</p>
 
 <p>## ‐‐ multi_sars ‐‐‐‐‐‐‐‐‐‐‐‐‐‐‐‐‐‐‐‐‐‐‐‐## ‐‐‐‐‐‐‐‐‐‐‐‐‐‐‐‐‐‐‐‐‐‐‐‐ multi‐model ## SAR ‐‐</p>
 
 <p>## &gt; power : v</p>
 
 <p>## &gt; loga : v</p>
 
 <p>## &gt; monod : v</p>
 
 <p>plot(fitC) <i>#see Fig.1</i></p>
 </section>
 
 <section class=article-section__sub-content id=sec-4>
 
 <h3 class="article-section__sub-title section2" id=sec-4-title> Model fit validation</h3>
 
 <p>Model fits can be evaluated through tests of the normality and homoscedasticity of the residuals. Any of three tests can be selected to test the normality of the residuals: 1) the Lilliefors extension of the Kolmogorov normality test (the default; normaTest = ‘lillie’), 2) the Shapiro–Wilk test of normality (to be preferred when sample size is small; ‘shapiro’), and 3) the Kolmogorov–Smirnov test (‘kolmo’). Alternatively, an option to omit a residuals normality test is provided (‘none’). Three options are provided to check for the homogeneity of the residuals: 1) a correlation of the residuals with the model fitted values (the default; ‘cor.fitted’), 2) a correlation of the residuals with the area values (‘cor.area’), or 3) no homogeneity test (‘none’). If a test is selected and is significant at the 5% level a warning is provided in the model summary; alternatively, the full results of the three tests can be accessed in the model fit output. A third model validation check for negative predicted richness values (i.e. when at least one of the fitted values is negative) is automatically undertaken and a warning is provided in the model summary if negative values are predicted.</p>
 
 <p><i>#load an example dataset, fit the linear #SAR model whilst running residual #normality and homogeneity tests, and #return the results of the residual #normality test</i></p>
 
 <p>data(galap)</p>
 
 <p>fit &lt;‐ sar_linear(data = galap, normaTest = "lillie", homoTest = "cor.fitted")</p>
 
 <p>summary(fit) #a warning is provided #indicating the normality test failed</p>
 
 <p>##</p>
 
 <p>## Model:</p>
 
 <p>## Linear model</p>
 
 <p>##</p>
 
 <p>## Call:</p>
 
 <p>## S == c + m*A</p>
 
 <p>##</p>
 
 <p>## Did the model converge: TRUE</p>
 
 <p>##</p>
 
 <p>## Residuals:</p>
 
 <p>## 0% 25% 50% 75% 100%</p>
 
 <p>## ‐107.90 ‐52.10 ‐24.15 31.75 218.00</p>
 
 <p>##</p>
 
 <p>## Parameters:</p>
 
 <p>## Estimate Std. Error t value Pr(&gt;|t|) ## 2.5 % 97.5 %</p>
 
 <p>##  c 70.314003 25.743130 2.731370 ## 0.016228 15.100481 125.5275</p>
 
 <p>##   m 0.185382 0.072884 2.543504 0.023410 ## 0.029060 0.3417</p>
 
 <p>##</p>
 
 <p>## R‐squared: 0.32, Adjusted R‐squared: ## 0.21</p>
 
 <p>##  AIC: 146.36, AICc: 148.36, BIC: 148.68</p>
 
 <p>## Observed shape: linear, Asymptote: ## FALSE</p>
 
 <p>##</p>
 
 <p>##</p>
 
 <p>## Warning: The normality test selected ## indicated the model residuals are not ## normally distributed (i.e. P &lt; 0.05)</p>
 
 <p>fit$normaTest</p>
 
 <p>## $test</p>
 
 <p>## [1] "lillie"</p>
 
 <p>##</p>
 
 <p>## [[2]]</p>
 
 <p>##</p>
 
 <p>##   Lilliefors (Kolmogorov‐Smirnov) ##  normality test</p>
 
 <p>##</p>
 
 <p>## data: res</p>
 
 <p>## D = 0.2146, p‐value = 0.04725</p>
 </section>
 
 <section class=article-section__sub-content id=sec-5>
 
 <h3 class="article-section__sub-title section2" id=sec-5-title> Observed model shape and identifying an asymptote</h3>
 
 <p>Whilst each of the 20 models has a general shape (Tjørve 2009, Triantis et al. 2012; Table 1), the actual observed shape of the model fit can be different, for some models, depending on the parameter estimates. This is important as the shape of the curve has significant implications for conservation applications and the testing of macroecological theory (Rosenzweig <span><a href=#bib-0027 class="bibLink tab-link" id=bib-0027R data-tab=pane-pcw-references>1995</a></span>, Guilhaumon et al. 2008, Triantis et al. 2012). In ‘sars’, the observed shape of a model fit is determined using the sequential algorithm outlined in Triantis et al. (2012). The shape is calculated using the model fit within the observed range of area values. Briefly, the algorithm works by first determining whether the fit is a straight line. Then, if the fit is classified as not being linear, the observed shape is classified as either convex or sigmoidal by analysis of the second derivative (with respect to area) of the model fit (the full algorithm is detailed in Triantis et al. 2012, p. 220). There has also been considerable debate in the SAR literature as to whether or not the SAR is asymptotic (Williamson et al. 2001). In the ‘sars’ package, to determine whether a fit is asymptotic, for the relevant models the fitted model parameters are analysed to check whether the estimated asymptote is within the range of the sample data (Triantis et al. 2012).
 </p>
 </section>
 
 <section class=article-section__sub-content id=sec-6>
 
 <h3 class="article-section__sub-title section2" id=sec-6-title> Multimodel SAR curve</h3>
 
 <p>As well as fitting individual models, the package provides a function (‘sar_average’) to fit up to 20 models, compare the resultant fits using information criteria, and construct a multimodel‐averaged SAR curve based on information criteria weight (Burnham and Anderson <span><a href=#bib-0004 class="bibLink tab-link" id=bib-0004R data-tab=pane-pcw-references>2002</a></span>, Guilhaumon et al. 2008, <span><a href=#bib-0010 class="bibLink tab-link" id=bib-0010R data-tab=pane-pcw-references>2010</a></span>). The multimodel average curve is constructed as a linear combination of individual model fits by multiplying the predicted richness values of each of the successfully fitted models by the model's information criterion weight, and then summing the resultant values across all models (Burnham and Anderson <span><a href=#bib-0004 class="bibLink tab-link" id=bib-0004R data-tab=pane-pcw-references>2002</a></span>). Three information criteria are available in the package: AIC, AIC<sub>c</sub> and BIC (Burnham and Anderson <span><a href=#bib-0004 class="bibLink tab-link" id=bib-0004R data-tab=pane-pcw-references>2002</a></span>). Confidence intervals around the multimodel averaged curve can be calculated using a non‐parametric bootstrap algorithm described in Guilhaumon et al. (<span><a href=#bib-0010 class="bibLink tab-link" id=bib-0010R data-tab=pane-pcw-references>2010</a></span>). Briefly, each of the SAR models used in the ‘sar_average’ function is fitted to the data, and the fitted values and residuals stored. The residuals are then transformed using the approach in Davison and Hinkley (<span><a href=#bib-0008 class="bibLink tab-link" id=bib-0008R data-tab=pane-pcw-references>1997</a></span>, (<span><a href=#bib-0008 class="bibLink tab-link" id=bib-0008R data-tab=pane-pcw-references>1997</a></span>p. 259). For each bootstrap sample, an individual model fit is selected with the probability of selection being equal to that model's information criterion weight. The transformed residuals from this fit are then sampled with replacement and added to the model's fitted richness values. The ‘sar_average’ function is then used to fit all candidate SAR models to this bootstrapped set of response values, and the multimodel averaged fitted values stored. Percentile confidence intervals are then calculated using all bootstrapped fitted values, following Buckland et al. (<span><a href=#bib-0008 class="bibLink tab-link" id=bib-0008R data-tab=pane-pcw-references>1997</a></span>).
 </p>
 
 <p>The ‘sar_average’ function can be used without specifying any models, in which case the package attempts to fit each of the 20 models in Table 1; alternatively, a vector of model names or a ‘fit_collection’ object (generated using the ‘sar_multi’ function) can be provided using the ‘obj’ argument. The three model validation tests listed above (normality and homogeneity of residuals, and negative predicted values) can be selected; if any model fails one or more of the tests during the fitting process it is removed from the resultant multimodel SAR curve. The output of the ‘sar_average’ function is a list of class ‘multi’ and class ‘sars’, with two elements. The first element (‘mmi’) contains the multi model inference (fitted values of the multimodel SAR curve), and the second element (‘details’) contains a range of information regarding the fitting process, including the successfully fitted models, the models removed due to failing any of the validation tests, and the information criterion values, delta values and weights for the successfully fitted models. The returned object can easily be plotted using the ‘plot.multi’ generic function, and multiple plot options are available (using the ‘type’ argument; see Fig. 2). The fits of all the successfully fitted models and the multimodel SAR curve (with or without confidence intervals) can be plotted together (‘type’ = ‘multi’), and a barplot of the information criterion weights of each model can also be produced (‘type’ = ‘bar’).</p>
 
 <section class=article-section__inline-figure>
 <figure class=figure id=ecog12417-fig-0002><a target=_blank href=https://onlinelibrary.wiley.com/cms/asset/f99418e0-d01c-4300-8ea2-a91471909102/ecog12417-fig-0002-m.jpg>
 <picture>
 <img class=figure__image src=images/12.jpg data-lg-src=/cms/asset/f99418e0-d01c-4300-8ea2-a91471909102/ecog12417-fig-0002-m.jpg alt=image title=image srcset sizes></picture></a><figcaption class=figure__caption>
 <div class=figure__caption__header><strong class=figure__title>Figure 2</strong><div class=figure-extra><a href=# class=open-figure-link>Open in figure viewer</a><a href="https://onlinelibrary.wiley.com/action/downloadFigures?id=ecog12417-fig-0002&amp;doi=10.1111%2Fecog.04271" class=ppt-figure-link><i aria-hidden=true class=icon-Icon_Download></i><span>PowerPoint</span></a></div>
 </div>
 <div class="figure__caption figure__caption-text">
 <p>The different plots that can be generated from an object of class ‘multi’. (a) The fit of eight SAR models alongside a multimodel SAR curve generated using AIC<sub>c</sub> weights; (b) the multimodel SAR curve plotted on its own with confidence intervals (the confidence intervals are generated using bootstrapping – see details in the main text); (c) a barplot of the AIC<sub>c</sub> weights of the different models (the model names on the x‐axis match up with legend in (a)). The data are the number of plants species recorded on 32 islands in the Kapingamarangi Atoll (Niering <span class="figureLink bibLink tab-link"><a href=#bib-0024 class="bibLink tab-link" id=bib-0024R data-tab=pane-pcw-references>1963</a></span>; the shape of the SAR curves differs from those presented in the source paper as Niering used a semi‐log plot). The code to produce the plots is given as an example in the ‘Multimodel SAR curve’ section.</p>
 </div>
 </figcaption>
 </figure>
 </section>
 
 <p><i>#load an example dataset (</i>Niering, #1963<i>), run the ‘sar_average’ function #using a vector of model names and with #no model validation tests, and produce #the plots in Figure 2 of the paper</i></p>
 
 <p>data(niering)</p>
 
 <p>#run the ‘sar_average’ function using a #vector of model names</p>
 
 <p>fit &lt;‐ sar_average(data = niering, obj = c("power","loga","koba","mmf","monod","negexpo","chapman","weibull3","asymp"), normaTest = "none", homoTest = "none", neg_check = FALSE, confInt = TRUE, ciN = 50)</p>
 
 <p>##</p>
 
 <p>## Now attempting to fit the 9 SAR models:</p>
 
 <p>##</p>
 
 <p>## ‐‐ multi_sars ‐‐‐‐‐‐‐‐‐‐‐‐‐‐‐‐‐‐‐‐‐‐‐‐## ‐‐‐‐‐‐‐‐‐‐‐‐‐‐‐‐‐‐‐‐‐‐‐‐ multi‐model ## SAR ‐‐</p>
 
 <p>## &gt; power : v</p>
 
 <p>## &gt; loga : v</p>
 
 <p>## &gt; koba : v</p>
 
 <p>## &gt; mmf : v</p>
 
 <p>## &gt; monod : v</p>
 
 <p>## &gt; negexpo : v</p>
 
 <p>##   &gt; chapman : Warning: could not compute ## parameters statistics</p>
 
 <p>## &gt; weibull3 : v</p>
 
 <p>## &gt; asymp  : x</p>
 
 <p>##</p>
 
 <p>## 1 models could not be fitted and have ## been excluded from the multi SAR</p>
 
 <p>##</p>
 
 <p>## All models passed the model validation ## checks</p>
 
 <p>##</p>
 
 <p>## 8 remaining models used to construct ## the multi SAR:</p>
 
 <p>## Power, Logarithmic, Kobayashi, MMF, ## Monod, Negative exponential, Chapman ## Richards, Cumulative Weibull 3 par.</p>
 
 <p>## ‐‐‐‐‐‐‐‐‐‐‐‐‐‐‐‐‐‐‐‐‐‐‐‐‐‐‐‐‐‐‐‐‐‐‐‐## ‐‐‐‐‐‐‐‐‐‐‐‐‐‐‐‐‐‐‐‐‐‐‐‐‐‐‐‐‐‐‐‐‐‐‐‐## ‐‐‐‐‐‐‐‐‐‐‐‐‐‐</p>
 
 <p>##</p>
 
 <p>##  Calculating sar_multi confidence ## intervals ‐ this may take some time:</p>
 
 <p>par(mfrow = c(3,1))</p>
 
 <p>#plot all model fits with the multimodel #SAR curve</p>
 
 <p>plot(fit, ModTitle = "a) Multimodel SAR")</p>
 
 <p>#plot the multimodel SAR curve (with #confidence intervals; see explanation #in the main text, above) on its own</p>
 
 <p>plot(fit, allCurves = FALSE, ModTitle = "b) Multimodel SAR with confidence intervals", confInt = TRUE)</p>
 
 <p>#Barplot of the information criterion #weights of each model</p>
 
 <p>plot(fit, type = "bar", ModTitle = "c) Model weights", cex.lab = 1.3)</p>
 </section>
 
 <section class=article-section__sub-content id=sec-7>
 
 <h3 class="article-section__sub-title section2" id=sec-7-title> Additional functions</h3>
 
 <p>In addition to the main functions used to fit and compare the 20 SAR models, the ‘sars’ package provides additional functions for specific SAR‐based analyses. First, a function is provided to fit the log–log version of the power model (a function that is often fitted in SAR studies; Rosenzweig <span><a href=#bib-0027 class="bibLink tab-link" id=bib-0027R data-tab=pane-pcw-references>1995</a></span>) and compare parameter values with those generated using the non‐linear power model. The log–log version of the power model is not equivalent to its non‐linear counterpart because of non‐equivalence in the study of the variation in a variable and in its transformation (Gitay et al. 1991, He and Legendre <span><a href=#bib-0016 class="bibLink tab-link" id=bib-0016R data-tab=pane-pcw-references>1996</a></span>), and bias of back‐transformed results obtained on a logarithmic scale (Fattorini <span><a href=#bib-0011 class="bibLink tab-link" id=bib-0011R data-tab=pane-pcw-references>2007</a></span>). Second, a function has been added that enables the fitting of Coleman's (<span><a href=#bib-0007 class="bibLink tab-link" id=bib-0007R data-tab=pane-pcw-references>1981</a></span>) random placement model to a species/sites abundance matrix. According to this model, the number of species occurring on an island depends on the relative area of the island and the regional relative species abundances. The fit of the random placement model can be determined through use of a diagnostic plot (which can be generated using the ‘plot.coleman’ function) of island area (log transformed) against species richness, alongside the model's predicted values (Wang et al. <span><a href=#bib-0010 class="bibLink tab-link" id=bib-0010R data-tab=pane-pcw-references>2010</a></span>, Matthews et al. 2015). Following Wang et al. (<span><a href=#bib-0010 class="bibLink tab-link" id=bib-0010R data-tab=pane-pcw-references>2010</a></span>), the model is rejected if more than a third of the observed data points fall beyond one standard deviation from the expected curve. Finally, a function is provided to fit the general dynamic model of island biogeography (Whittaker et al. 2008) using three different SAR models (linear, logarithmic and power).
 </p>
 
 <p><i>#load an example dataset, fit the log‐log #power model, return a model fit summary #and plot the model fit. When ‘compare’ #== TRUE, the non‐linear power model is #also fitted and the resultant parameter #values compared. If any islands have #zero species, a constant (‘con’) is #added to all species richness values</i>.
 </p>
 
 <p>data(galap)</p>
 
 <p>fit &lt;‐ lin_pow(dat = galap, compare = TRUE, con = 1)</p>
 
 <p>summary(fit)</p>
 
 <p>## Model = Log‐log power</p>
 
 <p>##</p>
 
 <p>## Call:</p>
 
 <p>## lm(formula = S ~ A, data = log.data)</p>
 
 <p>##</p>
 
 <p>## Residuals:</p>
 
 <p>## Min 1Q Median 3Q Max</p>
 
 <p>## ‐1.3591 ‐0.7584 0.1177 0.6009 1.0739</p>
 
 <p>##</p>
 
 <p>## Coefficients:</p>
 
 <p>## Estimate Std. Error t value Pr(&gt;|t|)</p>
 
 <p>## LogC​ 3.01865 0.35442​ 8.517 ​6.56e‐07 ## ***</p>
 
 <p>## z 0.33854 0.08523 3.972 0.00139 **</p>
 
 <p>## ‐‐‐</p>
 
 <p>## Signif. codes: 0 ’***’ 0.001 ’**’ ## 0.01 ’*’ 0.05 ’.’ 0.1 ’ ’ 1</p>
 
 <p>##</p>
 
 <p>## Residual standard error: 0.7626 on 14 ## degrees of freedom</p>
 
 <p>## Multiple R‐squared: 0.5298, Adjusted ## R‐squared: 0.4962</p>
 
 <p>##  F‐statistic: 15.78 on 1 and 14 DF, ## p‐value: 0.001391</p>
 
 <p>##</p>
 
 <p>## Power (non‐linear) parameters:</p>
 
 <p>## c = 33.18</p>
 
 <p>## z = 0.28</p>
 
 <p>plot(fit)</p>
 
 <p><i>#load an example dataset, fit the random #placement model and plot the model fit #and standard deviation. The ‘data’ #argument requires a species‐site #abundance matrix: rows are species and #columns are sites. The area argument #requires a vector of site (island) area #values</i>.
 </p>
 
 <p>data(cole_sim)</p>
 
 <p>fit &lt;‐ coleman(data = cole_sim[[1]], area = cole_sim[[2]])</p>
 
 <p>plot(fit, ModTitle = "Hetfield")</p>
 
 <p><i>#load an example dataset, fit the GDM #using the logarithmic SAR model, and #compare the GDM with three alternative #(nested) models: area and time (age of #each island), area only, and intercept #only</i>.
 </p>
 
 <p>data(galap)</p>
 
 <p>galap$t &lt;‐ rgamma(16, 5, scale = 2)#add #a random time variable</p>
 
 <p>gdm(data = galap, model = "loga", mod_sel = TRUE)</p>
 
 <p>##</p>
 
 <p>## GDM fit using the logarithmic SAR model</p>
 
 <p>##</p>
 
 <p>## GDM model summary:</p>
 
 <p>##</p>
 
 <p>## Nonlinear regression model</p>
 
 <p>## model: SR ~ Int + A * log(Area) + Ti * ## Time + Ti2 * Time^2</p>
 
 <p>## data: data</p>
 
 <p>## Int A Ti Ti2</p>
 
 <p>## ‐195.933 35.336 38.536 ‐1.876</p>
 
 <p>## residual sum‐of‐squares: 73750</p>
 
 <p>##</p>
 
 <p>## Number of iterations to convergence: 1</p>
 
 <p>## Achieved convergence tolerance: ## 3.146e‐08</p>
 
 <p>##</p>
 
 <p>## All model summaries:</p>
 
 <p>##</p>
 
 <p>## RSE AIC Delta.AIC</p>
 
 <p>## A 74.44350 187.1908 0.000000</p>
 
 <p>## A + T 77.22723 189.1799 1.989046</p>
 
 <p>## GDM 78.39534 190.3796 3.188759</p>
 
 <p>## Intercept 100.32744 195.8435 8.652648</p>
 </section>
 </section>
 <section class=article-section__content id=sec-8>
 
 <h2 class="article-section__title section__title section1" id=sec-8-title> Conclusions</h2>
 
 <p>The SAR has been a cornerstone of ecological and biogeographical science for almost a century (Arrhenius 1921, MacArthur and Wilson <span><a href=#bib-0018 class="bibLink tab-link" id=bib-0018R data-tab=pane-pcw-references>1967</a></span>, Rosenzweig <span><a href=#bib-0027 class="bibLink tab-link" id=bib-0027R data-tab=pane-pcw-references>1995</a></span>) and its form and fit are still of great significance in both theoretical (Storch 2016) and applied (Chisholm et al. 2018) contexts. The development of the ‘sars’ R package should aid future SAR research by providing a comprehensive set of tools that enable in‐depth exploration of SARs and SAR‐related patterns. In addition, the package has been designed in such a way as to allow other SAR researchers to add (e.g. via GitHub) new functions and models in the future to ensure the package is of lasting value. For example, future additions to the package could include the suite of countryside biogeography SAR models that have recently been published (Pereira et al. 2014), standard SAR functions not so far incorporated (Dengler 2009, <span><a href=#bib-0010 class="bibLink tab-link" id=bib-0010R data-tab=pane-pcw-references>2010</a></span>, Tjørve 2009), or functions specifically intended for analysis of species accumulation curves (Matthews et al. 2016a) or endemics–area relationships. Finally, whilst the focus of this paper has been on classic SARs, there is no reason that the functionality in the ‘sars’ package cannot be used to analyse other diversity–area relationships (e.g. functional or phylogenetic diversity–area relationships, Whittaker et al. 2014, Mazel et al. 2015). Application of the full set of 20 models, in addition to the multimodel SAR framework, included in the ‘sars’ package to a wider range and type of data (e.g. trait and phylogenetic data) will likely be revealing and will help in improving our understanding of SARs, and diversity–area relationships more generally.
 </p>
 
 <p>To cite sars or acknowledge its use, cite this Software note as follows, substituting the version of the application that you used for ’version 0’:</p>
 
 <p>Matthews, T. J., Triantis, K. A., Whittaker, R. J. and Guilhaumon, F. 2019. sars: an R package for fitting, evaluating and comparing species–area relationship models. – Ecography 42: 1446–1455 (ver. 0).</p>
 
 <section class=article-section__sub-content id=sec-9>
 
 <h3 class="article-section__sub-title section2" id=sec-9-title> Data accessibility</h3>
 
 <p>The most recent stable version of the package is freely available from CRAN (&lt;<a href="https://cran.r-project.org/package=sars" class=linkBehavior>https://CRAN.R‐project.org/package=sars</a>&gt;), whilst the development version is hosted on GitHub (&lt;<a href=https://github.com/txm676/sars class=linkBehavior>https://github.com/txm676/sars</a>&gt;). Version 1.1.1 of the package, presented in this article, has been archived on the Zenodo research data repository (doi: 10.5281/zenodo.2573067).
 </p>
 
 <p><i>Acknowledgements</i> – Kevin Cazelles, Jürgen Dengler and Michael Borregaard provided comments which greatly improved the package and paper. Jon Sadler kindly tested the R package. The College of Life and Environmental Sciences (Univ. of Birmingham) kindly provided funding for TJM to visit the National and Kapodistrian Univ. of Athens to work on the paper.
 </p>
 
 <p><i>Author contributions</i> – FG and TJM led the design of the study with input on functions required from KAT and RJW. TJM and FG wrote the R package. KAT contributed datasets. TJM wrote the first draft of the paper. All authors contributed to the submitted manuscript.
 </p>
 </section>
 </section>
 <section class="article-section article-section__references" id=references-section>
 <div class="accordion article-accordion">
 <h2>
 <div tabindex=0 role=button class=accordion__control aria-controls=idsec-10 aria-expanded=false data-references="/action/ajaxShowEnhancedAbstract?widgetId=5cf4c79f-0ae9-4dc5-96ce-77f62de7ada9&amp;ajax=true&amp;doi=10.1111%2Fecog.04271&amp;pbContext=%3Bissue%3Aissue%3Adoi%5C%3A10.1111%2Fecog.2019.v42.i8%3Bpage%3Astring%3AArticle%2FChapter+View%3Bctype%3Astring%3AJournal+Content%3Bwebsite%3Awebsite%3Apericles%3Bjournal%3Ajournal%3A16000587%3Bwgroup%3Astring%3APublication+Websites%3BrequestedJournal%3Ajournal%3A16000587%3BpageGroup%3Astring%3APublication+Pages%3BsubPage%3Astring%3AFull+Text%3Barticle%3Aarticle%3Adoi%5C%3A10.1111%2Fecog.04271&amp;accordionHeadingWrapper=h2&amp;onlyLog=true"><span class=section__title id=references-section-1>References</span><i aria-hidden=true class=icon-section_arrow_d></i></div>
 </h2>
 <div class=accordion__content style=display:none role=region aria-labelledby=references-section-1 id=sec-10>
 <ul class="rlist separator">
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 </ul>
 </div>
 </div>
 </section>
 </section>
 <div style=display:none>
 <ul class="custom-style-list noteList"></ul>
 </div>
 <div class=pb-dropzone data-pb-dropzone=citedBy-dropzone>
 <section id=cited-by class="article-section article-section__citedBy cited-by"><div class="accordion article-accordion"><h2><div tabindex=0 role=button aria-controls=cited-by__content aria-expanded=false class=accordion__control><span id=citedby-section class=section__title>Citing Literature</span><i aria-hidden=true class=icon-section_arrow_d></i></div></h2><div id=cited-by__content class="accordion__content cited-by__content" style=display:none><h3> </h3><ul class="rlist separator cited-by__list"></ul></div></div></section>
 </div>
 
</article></div></div><div class="col-lg-4 col-md-4 col-sm-4 hidden-xs hidden-sm hidden-md article-row-right fixed-element" style=top:-174px><div class="publication-cover-info clearfix">
 
 <div class="cover-image-wrapper cover-image__image hasDetails"><a href=https://onlinelibrary.wiley.com/toc/16000587/2019/42/8 title="View Volume 42, Issue 8"><img src=images/13.jpg alt></a></div><div class="extra-info-wrapper cover-image__details"><p class=volume-issue><a href=https://onlinelibrary.wiley.com/toc/16000587/2019/42/8 title="View Volume 42, Issue 8"><span>Volume</span><span class=val>42</span>, <span>Issue</span><span class=val>8</span></a><p>August 2019<p class=page-range><span>Pages</span><span> 1446-1455</span></p>
 
 
</div>
</div>
 
 
 <div class="pb-ad sf-hidden">
 <div id=advert-rail-1 class="advert advert-rail">
 
 
 
 </div>
 </div>
 
<div class="sticko__parent scrollThenFix" style="height:calc(100vh - 0px)"><div class="tab tab--slide tab--flex sticko__md tabs--xs dynamic-sticko"><ul data-mobile-toggle=slide role=tablist class="rlist tab__nav w-slide--list cloned"><li role=presentation><a data-slide-clone=self href=#pane-pcw-figures aria-controls=#pane-pcw-figures role=tab data-toggle=tab id=pane-pcw-figurescon data-slide-target=#pane-pcw-figures class="article-row-right__tabLinks w-slide__btn figures-tab empty" tabindex=-1 aria-selected=false><i aria-hidden=true class=icon-multi-figure></i><h2>Figures</h2></a><li role=presentation><a data-slide-clone=self href=#pane-pcw-references aria-controls=#pane-pcw-references role=tab data-toggle=tab id=pane-pcw-referencescon data-slide-target=#pane-pcw-references class="article-row-right__tabLinks w-slide__btn references-tab empty" tabindex=-1 aria-selected=false><i aria-hidden=true class=icon-chain></i><h2>References</h2></a><li role=presentation><a data-slide-clone=self href=#pane-pcw-related aria-controls=#pane-pcw-related role=tab data-toggle=tab id=pane-pcw-relatedcon data-slide-target=#pane-pcw-related class="article-row-right__tabLinks w-slide__btn related-tab empty" tabindex=-1 aria-selected=false><i aria-hidden=true class=icon-related-circles-extend-button></i><h2>Related</h2></a><li role=presentation class=active><a data-slide-clone=self href=#pane-pcw-details aria-controls=#pane-pcw-details role=tab data-toggle=tab id=pane-pcw-detailscon data-slide-target=#pane-pcw-details class="article-row-right__tabLinks w-slide__btn details-tab" tabindex=-1 aria-selected=false><i aria-hidden=true class=icon-Icon_Information></i><h2>Information</h2></a></ul><ul role=presentation class="rlist tab__content sticko__child"><li id=pane-pcw-figures aria-labelledby=pane-pcw-figurescon role=tabpanel class="tab__pane article-row-right__panes empty sf-hidden" aria-hidden=true><li id=pane-pcw-references aria-labelledby=pane-pcw-referencescon role=tabpanel class="tab__pane article-row-right__panes empty sf-hidden" aria-hidden=true><li id=pane-pcw-related aria-labelledby=pane-pcw-relatedcon role=tabpanel class="tab__pane article-row-right__panes empty sf-hidden" aria-hidden=true><li id=pane-pcw-details aria-labelledby=pane-pcw-detailscon role=tabpanel class="tab__pane article-row-right__panes active" aria-hidden=true>
<section class=info-tab-wrapper>
<section class=metrics>
<h3 class=info-header>Metrics</h3>
<div class=section__body><div class=cited-by-count><label>Citations: </label><span>11</span></div>
<div class=pb-dropzone data-pb-dropzone=publicaitonContent-info-tab-metrics>
<div data-badge-popover=left data-badge-type=1 data-doi=10.1111/ecog.04271 data-hide-no-mentions=false class=altmetric-embed data-uuid=b34ca3b4-e5bc-7930-7d21-7ff37481835d><a target=_self href="https://www.altmetric.com/details.php?domain=onlinelibrary.wiley.com&amp;citation_id=57215872" rel=popover data-content="<div>    <div style='padding-left: 10px; line-height:18px; border-left: 16px solid #74CFED;'>
      <a class='link-to-altmetric-details-tab' target='_self' href='https://www.altmetric.com/details.php?domain=onlinelibrary.wiley.com&amp;citation_id=57215872&amp;tab=twitter'>
          Tweeted by <b>37</b>
      </a>
    </div>
    <div style='padding-left: 10px; line-height:18px; border-left: 16px solid #2445bd;'>
      <a class='link-to-altmetric-details-tab' target='_self' href='https://www.altmetric.com/details.php?domain=onlinelibrary.wiley.com&amp;citation_id=57215872&amp;tab=facebook'>
          On <b>1</b> Facebook pages
      </a>
    </div>

    <div class='altmetric-embed readers' style='margin-top: 10px;'>
          <div class='altmetric-embed tip_mendeley'
               style='padding-left: 10px; line-height:18px; border-left: 16px solid #A60000;'>
              <b>67</b> readers on Mendeley
          </div>
    </div>
    <div style='margin-top: 10px; text-align: center;'>
        <a class='altmetric_details' target='_self' href='https://www.altmetric.com/details.php?domain=onlinelibrary.wiley.com&amp;citation_id=57215872'>
            See more details
        </a>
    </div>
</div>" style=display:inline-block data-badge-popover=left>
 <img alt="Article has an altmetric score of 22" src=images/17.png width=110 height=20 style=border:0;margin:0;max-width:none>
</a></div>
</div>
</div>
</section>
<section class=details>
<h3 class=info-header>Details</h3>
<p class=copyright>© 2019 The Authors. Ecography © 2019 Nordic Society Oikos</p>
<p></p>
<ul class="cross-mark rlist">
</ul>
</section>
<section class=keywords>
<h3 class=info-header>Keywords</h3>
<ul class="rlist rlist--inline">
<li><a class=badge-type href="https://onlinelibrary.wiley.com/action/doSearch?field1=Keyword&amp;text1=diversity%E2%80%93area%20relationship">diversity–area relationship</a></li>
<li><a class=badge-type href="https://onlinelibrary.wiley.com/action/doSearch?field1=Keyword&amp;text1=island%20biogeography">island biogeography</a></li>
<li><a class=badge-type href="https://onlinelibrary.wiley.com/action/doSearch?field1=Keyword&amp;text1=species%E2%80%93area%20relationship">species–area relationship</a></li>
</ul>
</section>
<section class=publication-history>
<h3 class=info-header>Publication History</h3>
<ul class=rlist>
<li><label>Issue Online: </label>31 July 2019
</li>
<li><label>Version of Record online: </label>15 April 2019
</li>
<li><label>Accepted manuscript online: </label>15 March 2019
</li>
<li><label>Manuscript accepted: </label>11 March 2019
</li>
</ul>
</section>
</section>
<li class=tab__spinner style=display:none><img src="data:image/gif;base64,R0lGODlhAQABAAAAACH5BAEKAAEALAAAAAABAAEAAAICTAEAOw==" id=spinner style=width:100% alt data-single-file-origin-u-r-l=/ux3/widgets/publication-content/images/spinner.gif></ul></div></div></div></div></article><div id=figure-viewer data-ux3-wrapper=figure-viewer data-ux3-transformed-by=figureInit data-ux3-role=parent role=dialog class="figure-viewer sf-hidden"><div class="figure-viewer__reg__top clearfix"></div><div class=figure-viewer__reg__center style="height:calc(100vh - 56px)"></div></div>
 
 <div style=text-align:right!important><a style=font-weight:300!important;font-size:90% href=https://onlinelibrary.wiley.com/doi/pdf/10.1111/ecog.04271>Download PDF</a></div>
</div>
</div></div></div>
 </section>
 
 </div>
 </div>
<div class="w-slide padding-top"><div class=w-slide_head><a href=# class=w-slide__back><i class=icon-arrow_l aria-hidden=true></i>back</a><span class=w-slide__title></span></div><div class=w-slide__content></div></div></main>
 </div>
 
 <div data-widget-def=pageFooter data-widget-id=053994f3-7d80-4bb1-97a4-c67fbc4768cd class=pageFooter>
 
 
 <footer class=page-footer>
 <div data-pb-dropzone=main>
 
 
 <div class=main-footer>
 
 <style>.footer-links{text-align:center}.footer-links h3{font-weight:300}.footer-links a{font-weight:600}@media (min-width:992px){.footer-links{text-align:left}}</style>
 <div data-widget-def=ux3-layout-widget data-widget-id=2753bec1-a79d-4403-827b-6c788ddd6db3 class=footer-sections>
 
 
 <h2 class=sr-only>
 Additional links
 </h2>
 
 <div class=container><div class=row><div class=col-md-3>
 
 <div class="footer-links mb-3">
 <h3>About Wiley Online Library</h3>
 <ul class=rlist>
 <li><a href=https://www.wiley.com/privacy>Privacy Policy</a></li>
 <li><a href=https://onlinelibrary.wiley.com/terms-and-conditions>Terms of Use</a></li>
 <li><a href=https://onlinelibrary.wiley.com/cookies>Cookies</a></li>
 <li><a href=https://onlinelibrary.wiley.com/accessibility>Accessibility</a></li>
 </ul>
</div>
</div><div class=col-md-3>
 
 <div class="footer-links mb-3">
 <h3>Help &amp; Support</h3>
 <ul class=rlist>
 <li><a href=https://hub.wiley.com/community/support/onlinelibrary>Contact Us</a></li>
 </ul>
 </div>
</div><div class=col-md-3>
 
 <div class="footer-links mb-3">
 <h3>Opportunities</h3>
 <ul class=rlist>
 <li><a href=https://onlinelibrary.wiley.com/agents>Subscription Agents</a></li>
 <li><a href=https://onlinelibrary.wiley.com/advertisers>Advertisers &amp; Corporate Partners</a></li>
 </ul>
 </div>
</div><div class=col-md-3>
 
 <div class="footer-links mb-3">
 <h3>Connect with Wiley</h3>
 <ul class=rlist>
 <li><a href=https://www.wiley.com/network>The Wiley Network</a></li>
 <li><a href=https://newsroom.wiley.com/>Wiley Press Room</a></li>
 </ul>
 </div>
</div></div></div>
 </div>
 
 <div data-widget-def=ux3-layout-widget data-widget-id=86584e25-e4db-4118-9550-9b4f5341a4ac class=copyright-container>
 
 
 <div class=container><div class=row><div class=col-md-12>
 <div data-widget-def=UX3HTMLWidget data-widget-id=bfbf6c5d-a650-4662-828f-5275fada9120 class=copyright>
 
 
 <div class="copyright pull-left">Copyright © 1999-2020 <a href=https://www.wiley.com/>John Wiley &amp; Sons, Inc</a>. All rights reserved</div>
 </div>
 
 
 <div class="footer-logo pull-right">
 
<a href=https://www.wiley.com/ title="Wiley Home Page" aria-label=Wiley><img alt="Wiley Home Page" src=images/14.png></a>
</div>
</div></div></div>
 </div>
 
</div>
 </div>
</footer>
 </div>
 
 
 <div id=iucr-failure class=sf-hidden> <p>The full text of this article hosted at iucr.org is unavailable due to technical difficulties. </p> </div> 
 
 
<div class="popup login-popup hidden sf-hidden" role=dialog tabindex=-1 aria-labelledby=loginPopupHead>
 <div class=content>
 
 
 
 </div>
</div>
 
 
<div class="popup change-password-drawer hidden sf-hidden" role=dialog tabindex=-1 aria-labelledby=changePasswordPopupHead>
 <div class=content>
 
 
 
 </div>
</div>
 
 
<div class="popup registration-popup hidden sf-hidden" role=dialog tabindex=-1 aria-labelledby=createAccountPopupHead>
 <div class=content>
 
 
 
 </div>
</div>
 
 
<div class="popup top-drawer request-reset-password-drawer hidden sf-hidden" role=dialog tabindex=-1 aria-labelledby=resetPasswordPopupHead>
 <div class=content>
 
 
 </div>
</div>
 
 <div><div class=pb-dropzone data-pb-dropzone=col-0 title=col-0></div></div>
 
 
<div class="popup top-drawer request-username-drawer username-popup hidden sf-hidden" role=dialog tabindex=-1 aria-labelledby=requestUsernamePopupHead>
 <a href=# class=close role=button aria-label="close dialog"></a>
 <div class=content>
 
 
 
 </div>
</div>
 
 
 </div>
 </div><div class="interesting-jade-owl-XgYroB93js4kI9jotxtT above-blush-bear-qsDNbutaWID8AOEJ-iframe-container" id=gleaming-azure-cat-gRcBok1gi0L6SkGfS8></div>
 
<div id=crossmark-widget style=display:none><div class="crossmark-reset crossmark-overlay"></div><div class="crossmark-reset crossmark-popup"></div></div><ul class="ui-autocomplete ui-front ui-menu ui-widget ui-widget-content ui-corner-all quickSearchAutocomplete" id=ui-id-1 tabindex=0 style=display:none></ul><ul class="ui-autocomplete ui-front ui-menu ui-widget ui-widget-content ui-corner-all quickSearchAutocomplete" id=ui-id-2 tabindex=0 style=display:none></ul><div id=gs-casa-r style=display:none;z-index:10;top:0px><div id=gs-casa-c></div></div><div id=_atssh style=visibility:hidden;height:1px;width:1px;position:absolute;top:-9999px;z-index:100000></div><img src=images/20.04271><div class="altmetric-embed altmetric-popover altmetric-left" data-uuid=b34ca3b4-e5bc-7930-7d21-7ff37481835d id=_altmetric_popover_el_b34ca3b4-e5bc-7930-7d21-7ff37481835d style=margin:0px;display:none><div class=altmetric_container></div></div><span style='border-radius:10px!important;text-indent:12px!important;width:auto!important;height:20px!important;padding:0px 8px!important;text-align:center!important;vertical-align:middle!important;font:bold 11px / 20px "Helvetica Neue",Helvetica,sans-serif!important;color:rgb(255,255,255)!important;background:url("data:image/svg+xml;base64,PHN2ZyBpZD0ic291cmNlIiB3aWR0aD0iMTIiIGhlaWdodD0iMTIiIHZpZXdCb3g9IjAgMCAxMiAxMiIgZmlsbD0ibm9uZSIgeG1sbnM9Imh0dHA6Ly93d3cudzMub3JnLzIwMDAvc3ZnIj4KPGNpcmNsZSBjeD0iNiIgY3k9IjYiIHI9IjYiIGZpbGw9IiNFNjAwMjMiPjwvY2lyY2xlPgo8cGF0aCBmaWxsLXJ1bGU9ImV2ZW5vZGQiIGNsaXAtcnVsZT0iZXZlbm9kZCIgZD0iTTAgNkMwIDguNTYxNSAxLjYwNTUgMTAuNzQ4NSAzLjg2NSAxMS42MDlDMy44MSAxMS4xNDA1IDMuNzUxNSAxMC4zNjggMy44Nzc1IDkuODI2QzMuOTg2IDkuMzYgNC41NzggNi44NTcgNC41NzggNi44NTdDNC41NzggNi44NTcgNC4zOTk1IDYuNDk5NSA0LjM5OTUgNS45N0M0LjM5OTUgNS4xNCA0Ljg4MDUgNC41MiA1LjQ4IDQuNTJDNS45OSA0LjUyIDYuMjM2IDQuOTAyNSA2LjIzNiA1LjM2MUM2LjIzNiA1Ljg3MzUgNS45MDk1IDYuNjM5NSA1Ljc0MSA3LjM1QzUuNjAwNSA3Ljk0NDUgNi4wMzk1IDguNDI5NSA2LjYyNTUgOC40Mjk1QzcuNjg3IDguNDI5NSA4LjUwMzUgNy4zMSA4LjUwMzUgNS42OTRDOC41MDM1IDQuMjYzNSA3LjQ3NTUgMy4yNjQgNi4wMDggMy4yNjRDNC4zMDkgMy4yNjQgMy4zMTE1IDQuNTM4NSAzLjMxMTUgNS44NTZDMy4zMTE1IDYuMzY5NSAzLjUwOSA2LjkxOTUgMy43NTYgNy4yMTlDMy44MDQ1IDcuMjc4NSAzLjgxMiA3LjMzIDMuNzk3NSA3LjM5MDVDMy43NTIgNy41Nzk1IDMuNjUxIDcuOTg1IDMuNjMxNSA4LjA2OEMzLjYwNSA4LjE3NyAzLjU0NSA4LjIwMDUgMy40MzE1IDguMTQ3NUMyLjY4NTUgNy44MDA1IDIuMjE5NSA2LjcxIDIuMjE5NSA1LjgzNEMyLjIxOTUgMy45NDk1IDMuNTg4IDIuMjE5NSA2LjE2NTUgMi4yMTk1QzguMjM3NSAyLjIxOTUgOS44NDggMy42OTYgOS44NDggNS42NjlDOS44NDggNy43Mjc1IDguNTUwNSA5LjM4NDUgNi43NDg1IDkuMzg0NUM2LjE0MyA5LjM4NDUgNS41NzQ1IDkuMDY5NSA1LjM3OTUgOC42OThDNS4zNzk1IDguNjk4IDUuMDggOS44MzkgNS4wMDc1IDEwLjExOEM0Ljg2NjUgMTAuNjYgNC40NzU1IDExLjM0NiA0LjIzMyAxMS43MzU1QzQuNzkyIDExLjkwNzUgNS4zODUgMTIgNiAxMkM5LjMxMzUgMTIgMTIgOS4zMTM1IDEyIDZDMTIgMi42ODY1IDkuMzEzNSAwIDYgMEMyLjY4NjUgMCAwIDIuNjg2NSAwIDZaIiBmaWxsPSJ3aGl0ZSI+PC9wYXRoPgo8L3N2Zz4=") 4px 50% / 12px 12px no-repeat rgb(230,0,35)!important;position:absolute!important;opacity:1!important;z-index:8675309!important;display:none;cursor:pointer!important;border:none!important;-webkit-font-smoothing:antialiased!important;top:106px;left:1053.44px'>Save</span><span style='border-radius:12px;width:24px!important;height:24px!important;background:url("data:image/svg+xml;base64,PHN2ZyBpZD0ic291cmNlIiB3aWR0aD0iMjIiIGhlaWdodD0iMjIiIHZpZXdCb3g9IjAgMCAyMiAyMiIgZmlsbD0ibm9uZSIgeG1sbnM9Imh0dHA6Ly93d3cudzMub3JnLzIwMDAvc3ZnIj4KPGNpcmNsZSBjeD0iMTEiIGN5PSIxMSIgcj0iMTEiIGZpbGw9ImJsYWNrIiBmaWxsLW9wYWNpdHk9IjAuOCI+PC9jaXJjbGU+CjxwYXRoIGZpbGwtcnVsZT0iZXZlbm9kZCIgY2xpcC1ydWxlPSJldmVub2RkIiBkPSJNMTUuMDgzNCA0LjU4MzMzSDEzLjMzMzRWNS43NDk5OUgxNS4wODM0QzE1LjcyNjggNS43NDk5OSAxNi4yNSA2LjI3MzI0IDE2LjI1IDYuOTE2NjZWOC42NjY2NkgxNy40MTY3VjYuOTE2NjZDMTcuNDE2NyA1LjYyOTgzIDE2LjM3MDIgNC41ODMzMyAxNS4wODM0IDQuNTgzMzNaTTE2LjI1IDE1LjA4MzNDMTYuMjUgMTUuNzI2NyAxNS43MjY4IDE2LjI1IDE1LjA4MzQgMTYuMjVIMTMuMzMzNFYxNy40MTY3SDE1LjA4MzRDMTYuMzcwMiAxNy40MTY3IDE3LjQxNjcgMTYuMzcwMiAxNy40MTY3IDE1LjA4MzNWMTMuMzMzM0gxNi4yNVYxNS4wODMzWk01Ljc1MDA0IDE1LjA4MzNWMTMuMzMzM0g0LjU4MzM3VjE1LjA4MzNDNC41ODMzNyAxNi4zNzAyIDUuNjI5ODcgMTcuNDE2NyA2LjkxNjcxIDE3LjQxNjdIOC42NjY3MVYxNi4yNUg2LjkxNjcxQzYuMjczMjkgMTYuMjUgNS43NTAwNCAxNS43MjY3IDUuNzUwMDQgMTUuMDgzM1pNNS43NTAwNCA2LjkxNjY2QzUuNzUwMDQgNi4yNzMyNCA2LjI3MzI5IDUuNzQ5OTkgNi45MTY3MSA1Ljc0OTk5SDguNjY2NzFWNC41ODMzM0g2LjkxNjcxQzUuNjI5ODcgNC41ODMzMyA0LjU4MzM3IDUuNjI5ODMgNC41ODMzNyA2LjkxNjY2VjguNjY2NjZINS43NTAwNFY2LjkxNjY2Wk05LjI1MDA0IDEwLjcwODNDOS4yNTAwNCA5LjkwNDQ5IDkuOTA0NTQgOS4yNDk5OSAxMC43MDg0IDkuMjQ5OTlDMTEuNTEyMiA5LjI0OTk5IDEyLjE2NjcgOS45MDQ0OSAxMi4xNjY3IDEwLjcwODNDMTIuMTY2NyAxMS41MTIyIDExLjUxMjIgMTIuMTY2NyAxMC43MDg0IDEyLjE2NjdDOS45MDQ1NCAxMi4xNjY3IDkuMjUwMDQgMTEuNTEyMiA5LjI1MDA0IDEwLjcwODNaTTEzLjYyNSAxNC41QzEzLjg0OSAxNC41IDE0LjA3MyAxNC40MTQ4IDE0LjI0NCAxNC4yNDM5QzE0LjU4NTIgMTMuOTAyMSAxNC41ODUyIDEzLjM0NzkgMTQuMjQ0IDEzLjAwNjFMMTMuMDcwMyAxMS44MzNDMTMuMjM0MiAxMS40OTA2IDEzLjMzMzQgMTEuMTEyNiAxMy4zMzM0IDEwLjcwODNDMTMuMzMzNCA5LjI2MTA4IDEyLjE1NTYgOC4wODMzMyAxMC43MDg0IDguMDgzMzNDOS4yNjExMiA4LjA4MzMzIDguMDgzMzcgOS4yNjEwOCA4LjA4MzM3IDEwLjcwODNDOC4wODMzNyAxMi4xNTU2IDkuMjYxMTIgMTMuMzMzMyAxMC43MDg0IDEzLjMzMzNDMTEuMTEyNiAxMy4zMzMzIDExLjQ5MDYgMTMuMjM0MiAxMS44MzMgMTMuMDcwMkwxMy4wMDYxIDE0LjI0MzlDMTMuMTc3IDE0LjQxNDggMTMuNDAxIDE0LjUgMTMuNjI1IDE0LjVaIiBmaWxsPSJ3aGl0ZSI+PC9wYXRoPgo8L3N2Zz4=") 50% 50% / 24px 24px no-repeat rgba(0,0,0,0.4)!important;position:absolute!important;opacity:1!important;z-index:8675309!important;display:none;cursor:pointer!important;border:none!important;top:106px;left:1125.58px'></span>