<html style><!--
 Page saved with SingleFile 
 url: https://www.nature.com/articles/s41598-022-14915-2 
 saved date: Wed Jun 29 2022 11:36:52 GMT+1000 (Papua New Guinea Time)
--><meta charset=utf-8>
<style id=brave_speedreader_style>h1,h3{font-weight:bold}h1{font-size:2.2em;line-height:1.3em}h3{font-size:1.2em;line-height:1em}p{font-size:1.2rem;line-height:1.5rem}.subhead{font-size:1.5em;line-height:1em;font-weight:normal;color:#495057}.metadata{margin-top:-0.75em;margin-bottom:1.45em}.title+.metadata{margin-top:-0.75em}a{text-decoration:underline;text-decoration-color:inherit}a[href]{color:#4C54D2}#article{text-rendering:optimizeLegibility;font-family:"Georgia","Times New Roman",Times,serif;user-select:unset}#article *{max-width:680}#article img{margin-bottom:0.5em;display:block;height:auto}figure{margin-bottom:1.15em}#article figure{font-size:0.85em;line-height:1.4em}#article figcaption>*{margin-top:0.25em;margin-bottom:0.25em}#article figure{font-family:Arial,Helvetica,sans-serif}#article figure{color:rgba(0,0,0,0.65)}figure{margin:0}body:not(.watch) figure:not(.float){margin-top:1.4em;margin-bottom:1.4em}hr{position:relative;display:block;width:100%;margin:2.5em 0 3.5em;padding:0;background:rgba(0,0,0,0.2);height:1px;border:0}@media screen{body{margin:0;padding:0;overflow-x:hidden;-webkit-text-size-adjust:none}#article{pointer-events:auto;overflow:visible}#article:focus{outline:none}#article{margin-left:auto;margin-right:auto;padding-top:4.5em;padding-bottom:35px;position:relative}#article{margin-top:0;border-top:none}#article:last-of-type{padding-bottom:45px}}@media only screen and (min-width:780px){#article{max-width:800px;margin:0 auto}}@media screen and (max-height:700px){#article{padding-top:32px;padding-bottom:32px}}</style><title>Comparison of traditional and DNA metabarcoding samples for monitoring tropical soil arthropods (Formicidae, Collembola and Isoptera)</title><link type=image/x-icon rel="shortcut icon" href="data:image/gif;base64,R0lGODlhAQABAAAAACH5BAEKAAEALAAAAAABAAEAAAICTAEAOw=="><link rel=canonical href=https://www.nature.com/articles/s41598-022-14915-2><meta http-equiv=content-security-policy content="default-src 'none'; font-src 'self' data:; img-src 'self' data:; style-src 'unsafe-inline'; media-src 'self' data:; script-src 'unsafe-inline' data:;"><body><div class=c-article-section__content id=article><h1 class="title metadata">Comparison of traditional and DNA metabarcoding samples for monitoring tropical soil arthropods (Formicidae, Collembola and Isoptera)</h1><p class="subhead metadata">Scientific Reports - Comparison of traditional and DNA metabarcoding samples for monitoring tropical soil arthropods (Formicidae, Collembola and Isoptera)<hr><p>Additional details for all method sections are provided in Appendix <a data-track=click data-track-action="supplementary material anchor" data-track-label=link href=#MOESM1>S1</a>.<h3 class=c-article__sub-heading id=Sec3>Study site and field sampling</h3><p>Assemblages of ants, springtails and termites were surveyed on Barro Colorado Island (9.15° N, 79.85° W; 120–160&nbsp;masl) in Panama. The 1542&nbsp;ha Barro Colorado Island is covered with lowland tropical forest and was created around 1910, when the Chagres River was dammed to fill the Panama Canal. Samples were obtained from the 50&nbsp;ha ForestGEO vegetation dynamics plot, which is described in Ref.<sup><a aria-label="Reference 31" data-test=citation-ref data-track=click data-track-action="reference anchor" data-track-label=link href=#ref-CR31 id=ref-link-section-d26459365e1286 title="Anderson-Teixeira, K. J. et al. CTFS-Forest GEO: A worldwide network monitoring forests in an era of global change. Glob. Change Biol. 21, 528–549 (2015).">31</a></sup>. We considered ten locations (500&nbsp;m sections of trails inside or near the plot) that are used for long-term arthropod monitoring and described in Refs.<sup><a aria-label="Reference 33" data-test=citation-ref data-track=click data-track-action="reference anchor" data-track-label=link href=#ref-CR33 id=ref-link-section-d26459365e1290 title="Basset, Y. et al. Enemy-free space and the distribution of ants, springtails and termites in the soil of one tropical rainforest. Eur. J. Soil Biol. 99, 103193. 
                  https://doi.org/10.1016/j.ejsobi.2020.103193
                  
                 (2020).">33</a>,<a aria-label="Reference 44" data-test=citation-ref data-track=click data-track-action="reference anchor" data-track-label=link href=#ref-CR44 id=ref-link-section-d26459365e1293 title="Basset, Y. et al. Cross-continental comparisons of butterfly assemblages in tropical rainforests: Implications for biological monitoring. Insect. Conserv. Divers 6, 223–233 (2013).">44</a></sup> (Fig.&nbsp;<a data-track=click data-track-action="figure anchor" data-track-label=link href=#Fig1>1</a>, Table <a data-track=click data-track-action="table anchor" data-track-label=link href=#Tab1>1</a>).<div class="c-article-section__figure js-c-reading-companion-figures-item" data-container-section=figure data-test=figure data-title="Figure 1" id=figure-1><figure><figcaption><b class=c-article-section__figure-caption data-test=figure-caption-text id=Fig1>Figure 1</b></figcaption><div class=c-article-section__figure-content><div class=c-article-section__figure-item><a class=c-article-section__figure-link data-test=img-link data-track=click data-track-action="view figure" data-track-label=image href=https://www.nature.com/articles/s41598-022-14915-2/figures/1 rel=nofollow><picture><img alt="figure 1" aria-describedby=Fig1 height=482 loading=lazy src=data:null;base64, width=685 srcset sizes></picture></a></div><div class=c-article-section__figure-description data-test=bottom-caption id=figure-1-desc><p>Summary diagram illustrating the workflow of samples for springtails. The black square in the BCI map indicates the location of the 50&nbsp;ha plot. Traditional samples of ants and termites were processed at STRI in Panama. The Panama and BCI maps are freely available at <a href="https://nsf.gov/news/mmg/mmg_disp.jsp?med_id=74874">https://nsf.gov/news/mmg/mmg_disp.jsp?med_id=74874</a> &amp;from. Map from the 50&nbsp;ha plot is also freely available at <a href="https://stridata-si.opendata.arcgis.com/maps/SI::barro-colorado-island-topographic-webmap/explore?location=9.157000%2C-79.848900%2C14.68">https://stridata-si.opendata.arcgis.com/maps/SI::barro-colorado-island-topographic-webmap/explore?location=9.157000%2C-79.848900%2C14.68</a>.</p></div></div></figure></div><div class=c-article-table data-container-section=table data-test=inline-table id=table-1><figure><figcaption class=c-article-table__figcaption><b data-test=table-caption id=Tab1>Table 1 Number of individuals and species of ants, springtails and termites (with BINs; traditional samples) and number of reads, reads in BINs, reads in OTUs and species (metabarcoding samples) recorded in samples grouped by location.</b></figcaption></figure></div><p>Each location was divided into ten sub-locations (Fig.&nbsp;<a data-track=click data-track-action="figure anchor" data-track-label=link href=#Fig1>1</a>; see statistical methods below and Appendix <a data-track=click data-track-action="supplementary material anchor" data-track-label=link href=#MOESM1>S2</a> for explanation of this approach). For each of the ten locations, we randomly selected five sub-locations. At each of these selected sub-locations, we took two paired samples 10&nbsp;cm distant from each other. We obtained 50 paired soil samples in March 2017, during the dry season. We repeated this sampling protocol in December 2017, during the wet season, with different random samples and obtained 100 paired soil samples for the two seasons. We used a 20 × 20&nbsp;cm frame (400&nbsp;cm<sup>2</sup>) to delineate one sample and scoop all litter and soil up to a depth of ca. 5&nbsp;cm or slightly deeper into a container, calibrating each sample to a final volume of 2&nbsp;L. We restricted our sampling to the first 5&nbsp;cm of the litter/soil as tropical soils tend to be shallow, with most of the faunal diversity concentrated in the top few centimeters<sup><a aria-label="Reference 45" data-test=citation-ref data-track=click data-track-action="reference anchor" data-track-label=link href=#ref-CR45 id=ref-link-section-d26459365e1845 title="Ryder Wilkie, K. T., Mertl, A. L. &amp; Traniello, J. F. A. Biodiversity below ground: Probing the subterranean ant fauna of Amazonia. Naturwissenschaften 94, 725–731 (2007).">45</a></sup>. To avoid contamination between samples in the field, we used laboratory gloves and disinfected all tools and receptacles with commercial bleach (Clorox de Centroamérica; hypoclorite of sodium 3.5%, hydroxide of sodium 0.3%), after which sampling gear was rinsed with distilled water, to clean bleach residues.<p>Each pair of samples consisted of two categories: “traditional samples”, from which the soil fauna was extracted and sorted manually according to morphology and molecular data (see below), and “metabarcoding samples” which were analyzed using DNA metabarcoding. Traditional samples were set 4–5&nbsp;h after collecting in Berlese-Tullgren funnels<sup><a aria-label="Reference 46" data-test=citation-ref data-track=click data-track-action="reference anchor" data-track-label=link href=#ref-CR46 id=ref-link-section-d26459365e1852 title="André, H. M., Ducarme, X. &amp; Lebrun, P. Soil biodiversity: Myth, reality or conning?. Oikos 96, 3–24 (2002).">46</a></sup>. Arthropods were extracted during the first 24&nbsp;h at ambient external temperature (i.e., without heat), and subsequently during the next 48&nbsp;h with additional heat provided by 25&nbsp;W bulbs. Berlese-Tullgren extracts were washed, arthropods were separated manually from soil debris and stored in 95% ethanol in a freezer at –&nbsp;20&nbsp;°C until further analysis.<p>Metabarcoding samples were extracted similarly with Berlese-Tullgren with the difference that equipment was thoroughly disinfected with bleach and rinsed with distilled water before and after each sample extraction. Metabarcoding samples were stored at −&nbsp;20&nbsp;°C until being analyzed at the Hajibabaei laboratory at the University of Guelph, Canada, which occurred approximately three months after collection. Geographic coordinates of the samples are indicated in Ref.<sup><a aria-label="Reference 33" data-test=citation-ref data-track=click data-track-action="reference anchor" data-track-label=link href=#ref-CR33 id=ref-link-section-d26459365e1859 title="Basset, Y. et al. Enemy-free space and the distribution of ants, springtails and termites in the soil of one tropical rainforest. Eur. J. Soil Biol. 99, 103193. 
                  https://doi.org/10.1016/j.ejsobi.2020.103193
                  
                 (2020).">33</a></sup> and Fig.&nbsp;<a data-track=click data-track-action="figure anchor" data-track-label=link href=#Fig1>1</a> summarizes the workflow of samples.<h3 class=c-article__sub-heading id=Sec4>Processing of traditional samples</h3><p>Ants, springtails and termites were all identified via morphological and molecular data (individual-based DNA barcoding). For morphological identification, ants were pinned and assigned to species using the ant reference collection of the ForestGEO Arthropod Initiative at the Smithsonian Tropical Research Institute. Springtails were first cleared in 10% KOH, then in lactophenol, and mounted on microscopic slides in Hoyer’s solution. The slides obtained were identified at the Laboratorio de Ecología y Sistemática de Microartrópodos at the Universidad Nacional Autónoma de México (UNAM) using UNAMs extensive reference collections of springtails and specialized literature. Termites were stored in ethanol and soldiers identified using the termite reference collection of the ForestGEO Arthropod Initiative.<p>The standard DNA barcode region of the gene cytochrome c oxidase subunit I (COI) was sequenced for a subset of the specimens collected. DNA barcoding using Sanger sequencing was conducted at the Centre for Biodiversity Genomics, University of Guelph, using methods described in Wilson<sup><a aria-label="Reference 47" data-test=citation-ref data-track=click data-track-action="reference anchor" data-track-label=link href=#ref-CR47 id=ref-link-section-d26459365e1877 title="Wilson, J. J. DNA barcodes for insects. In DNA Barcodes: Methods and Protocols (eds Kress, W. J. &amp; Erickson, D. L.) 17–46 (Springer, 2012).">47</a></sup>. When possible, we sequenced a maximum of five individuals per species. In total, we generated 324 new COI sequences, corresponding to 328 specimens of 171 ants, 114 springtails and 43 termites. These sequences were added to BCI projects BCIFO, BCICL and BCIIS of the Barcode of Life Data System (BOLD; <a href=http://www.barcodinglife.org/index.php>http://www.barcodinglife.org/index.php</a>), which summed 2,892 sequences (Appendix <a data-track=click data-track-action="supplementary material anchor" data-track-label=link href=#MOESM1>S1</a>, Supplementary Table <a data-track=click data-track-action="supplementary material anchor" data-track-label=link href=#MOESM1>S1</a>). These molecular data were used to confirm identifications based on morphology. Each species was attributed a Barcode Index Number (BIN) according to BOLD, which can be used as a proxy taxonomic unit in absence of binomial identification<sup><a aria-label="Reference 14" data-test=citation-ref data-track=click data-track-action="reference anchor" data-track-label=link href=#ref-CR14 id=ref-link-section-d26459365e1894 title="Ratnasingham, S. &amp; Hebert, P. D. N. A DNA-based registry for all animal species: The Barcode Index Number (BIN) system. PLoS ONE 8, e66213. 
                  https://doi.org/10.1371/journal.pone.0066213
                  
                 (2013).">14</a></sup> (e.g., “<i>Ectatomma</i> ACH3273 ruidum”).<h3 class=c-article__sub-heading id=Sec5>DNA metabarcoding</h3><p>Metabarcoding samples were transferred into separate 50&nbsp;mL tubes and centrifuged to collect soil and tissue at the bottom of the tube. Excess preservative ethanol was removed, and samples were left in an incubator at 65&nbsp;°C for 3–6&nbsp;h to evaporate any residual ethanol. Samples were briefly vortexed to dislodge and coarsely homogenize the dried pellet before transferring ~ 0.25&nbsp;g to a PowerSoil bead tube with sterile forceps. Whole DNA was extracted using DNeasy PowerSoil Kit (Qiagen: Toronto, ON, Canada) according to protocol, eluting with 30&nbsp;µL, buffer C5. Isolated DNA was amplified through a two-stage Polymerase Chain Reaction (PCR) for two overlapping amplicons from the DNA barcode region (Helix I to Helix V) of the COI gene. These amplicons were: (1) BR5: size ca. 310&nbsp;bp; forward primer B: 5′CCIGAYATRGCITTYCCICG<sup><a aria-label="Reference 23" data-test=citation-ref data-track=click data-track-action="reference anchor" data-track-label=link href=#ref-CR23 id=ref-link-section-d26459365e1910 title="Hajibabaei, M., Spall, J. L., Shokralla, S. &amp; van Konynenburg, S. Assessing biodiversity of a freshwater benthic macroinvertebrate community through non-destructive environmental barcoding of DNA from preservative ethanol. BMC Ecol. 12, 28. 
                  https://doi.org/10.1186/1472-6785-12-28
                  
                 (2012).">23</a></sup>; reverse primer ArR5: 5′GTRATIGCICCIGCIARIACIGG<sup><a aria-label="Reference 24" data-test=citation-ref data-track=click data-track-action="reference anchor" data-track-label=link href=#ref-CR24 id=ref-link-section-d26459365e1914 title="Gibson, J. et al. Simultaneous assessment of the macrobiome and microbiome in a bulk sample of tropical arthropods through DNA metasystematics. PNAS 111, 8007–8012 (2014).">24</a></sup>. (2) F230R: size ca. 230&nbsp;bp: forward primer LCO1490: 5′-GGTCAACAAATCATAAAGATATTGG<sup><a aria-label="Reference 48" data-test=citation-ref data-track=click data-track-action="reference anchor" data-track-label=link href=#ref-CR48 id=ref-link-section-d26459365e1918 title="Folmer, O., Black, M., Hoeh, W., Lutz, R. &amp; Vrijenhoek, R. DNA primers for amplification of mitochondrial cytochrome c oxidase subunit I from diverse metazoan invertebrates. Mol. Mar. Biol. Biotechnol. 3, 294–299 (1994).">48</a></sup>; reverse primer 230_R: 5′ CTTATRTTRTTTATICGIGGRAAIGC<sup><a aria-label="Reference 49" data-test=citation-ref data-track=click data-track-action="reference anchor" data-track-label=link href=#ref-CR49 id=ref-link-section-d26459365e1922 title="Gibson, J. F. et al. Large-scale biomonitoring of remote and threatened ecosystems via high-throughput sequencing. PLoS ONE 10, e0138432. 
                  https://doi.org/10.1371/journal.pone.0138432
                  
                 (2015).">49</a></sup>. For the map of primers and amplicons see Ref.<sup><a aria-label="Reference 50" data-test=citation-ref data-track=click data-track-action="reference anchor" data-track-label=link href=#ref-CR50 id=ref-link-section-d26459365e1926 title="Hajibabaei, M., Porter, T. M., Wright, M. &amp; Rudar, J. COI metabarcoding primer choice affects richness and recovery of indicator taxa in freshwater systems. PLoS&nbsp;One 14, e0220953. 
                  https://doi.org/10.1371/journal.pone.0220953
                  
                 (2019).">50</a></sup>. These were selected as proven to be effective to optimize recovery of species and genus richness in tropical arthropods<sup><a aria-label="Reference 22" data-test=citation-ref data-track=click data-track-action="reference anchor" data-track-label=link href=#ref-CR22 id=ref-link-section-d26459365e1931 title="McGee, K. M., Porter, T. M., Wright, M. &amp; Hajibabaei, M. Drivers of tropical soil invertebrate community composition and richness across tropical secondary forests using DNA metasystematics. Sci. Rep. 10, 18429. 
                  https://doi.org/10.1038/s41598-020-75452-4
                  
                 (2020).">22</a>,<a aria-label="Reference 50" data-test=citation-ref data-track=click data-track-action="reference anchor" data-track-label=link href=#ref-CR50 id=ref-link-section-d26459365e1934 title="Hajibabaei, M., Porter, T. M., Wright, M. &amp; Rudar, J. COI metabarcoding primer choice affects richness and recovery of indicator taxa in freshwater systems. PLoS&nbsp;One 14, e0220953. 
                  https://doi.org/10.1371/journal.pone.0220953
                  
                 (2019).">50</a></sup>. Reactions had a standard mix as indicated in Appendix <a data-track=click data-track-action="supplementary material anchor" data-track-label=link href=#MOESM1>S1</a>, for a total of 25&nbsp;μL per reaction. A negative control (i.e., reaction with 2&nbsp;μL water instead of DNA) was included to ensure PCR reagents were free of contamination. Reactions underwent 35 cycles of 94&nbsp;°C for 40&nbsp;s, 46&nbsp;°C for 60&nbsp;s, 72&nbsp;°C for 30&nbsp;s using an Eppendorf Mastercycler ep gradient S thermocycler. PCR amplification was visually confirmed through gel electrophoresis using a 1.5% agarose gel. PCR products were purified following the MinElute PCR Purification kit (Qiagen; Toronto, Ontario, Canada) standard protocol, eluting with 15&nbsp;μL molecular biology grade water. A second round of PCR with primers that included the Illumina adapter sequence was run under the same conditions, using the purified product from the first round of PCR as template. Two PCRs with 35 cycles are sufficient for providing consistent and robust results for many samples across different sample types. This protocol has been used routinely with consistent results for biodiversity, ecological and biomonitoring studies (e.g., Refs.<sup><a aria-label="Reference 22" data-test=citation-ref data-track=click data-track-action="reference anchor" data-track-label=link href=#ref-CR22 id=ref-link-section-d26459365e1941 title="McGee, K. M., Porter, T. M., Wright, M. &amp; Hajibabaei, M. Drivers of tropical soil invertebrate community composition and richness across tropical secondary forests using DNA metasystematics. Sci. Rep. 10, 18429. 
                  https://doi.org/10.1038/s41598-020-75452-4
                  
                 (2020).">22</a>,<a aria-label="Reference 49" data-test=citation-ref data-track=click data-track-action="reference anchor" data-track-label=link href=#ref-CR49 id=ref-link-section-d26459365e1944 title="Gibson, J. F. et al. Large-scale biomonitoring of remote and threatened ecosystems via high-throughput sequencing. PLoS ONE 10, e0138432. 
                  https://doi.org/10.1371/journal.pone.0138432
                  
                 (2015).">49</a>,<a aria-label="Reference 51" data-test=citation-ref data-track=click data-track-action="reference anchor" data-track-label=link href=#ref-CR51 id=ref-link-section-d26459365e1947 title="Bush, A. et al. DNA metabarcoding reveals metacommunity dynamics in a threatened boreal wetland wilderness. PNAS 117, 8539–8545 (2020).">51</a></sup>). Additionally, the high number of PCR cycles in the second round was practical because amplification was inadequate at lower cycles, and some samples failed. Purified second round PCR product was sequenced on an Illumina MiSeq using the v3 MiSeq sequencing kit (300&nbsp;bp × 2).<h3 class=c-article__sub-heading id=Sec6>Bioinformatics</h3><p>The bioinformatics of metabarcoding remains in flux<sup><a aria-label="Reference 27" data-test=citation-ref data-track=click data-track-action="reference anchor" data-track-label=link href=#ref-CR27 id=ref-link-section-d26459365e1960 title="Creedy, T. J., Ng, W. S. &amp; Vogler, A. P. Toward accurate species-level metabarcoding of arthropod communities from the tropical forest canopy. Ecol. Evol. 9, 3105–3116 (2019).">27</a></sup>. A wide array of metabarcoding tools exists, and their choice (and relevant parameters) is essential for meaningful results<sup><a aria-label="Reference 52" data-test=citation-ref data-track=click data-track-action="reference anchor" data-track-label=link href=#ref-CR52 id=ref-link-section-d26459365e1964 title="Calderón-Sanou, I. et al. From environmental DNA sequences to ecological conclusions: How strong is the influence of methodological choices?. J. Biogeogr. 47, 193–206 (2020).">52</a></sup>. Available tools include, for example, Mothur<sup><a aria-label="Reference 53" data-test=citation-ref data-track=click data-track-action="reference anchor" data-track-label=link href=#ref-CR53 id=ref-link-section-d26459365e1968 title="Schloss, P. D. Reintroducing mothur: 10 years later. Appl. Env. Microbiol. 86, e02343-19. 
                  https://doi.org/10.1128/AEM.02343-19
                  
                 (2020).">53</a></sup>, Qiime<sup><a aria-label="Reference 54" data-test=citation-ref data-track=click data-track-action="reference anchor" data-track-label=link href=#ref-CR54 id=ref-link-section-d26459365e1972 title="Caporaso, J. G. et al. QIIME allows analysis of high-throughput community sequencing data. Nat. Methods 7, 335–336 (2010).">54</a></sup> or Obitools<sup><a aria-label="Reference 55" data-test=citation-ref data-track=click data-track-action="reference anchor" data-track-label=link href=#ref-CR55 id=ref-link-section-d26459365e1976 title="Boyer, F. et al. Obitools: A unix-inspired software package for DNA metabarcoding. Mol. Ecol. Res. 16, 176–182 (2016).">55</a></sup>. However, an important restriction that pertains to all these tools is the use of customized databases for taxonomy classification. We relied on the mBRAVE cloud-based platform (“Multiplex Barcode Research and Visualization Environment”; <a href=http://www.mbrave.net/>http://www.mbrave.net/</a><sup><a aria-label="Reference 56" data-test=citation-ref data-track=click data-track-action="reference anchor" data-track-label=link href=#ref-CR56 id=ref-link-section-d26459365e1987 title="Ratnasingham, S. mBRAVE: The multiplex barcode research and visualization environment. Biodivers. Inf. Sci. Stand. 3, e37986. 
                  https://doi.org/10.3897/biss.3.37986
                  
                 (2019).">56</a></sup>) since it seamlessly integrates BOLD data as reference datasets, which for BCI represent the best curated COI sequence datasets available. mBRAVE employs a typical workflow based on mature bioinformatic methods and is comparable to other available bioinformatics tools. The sequence reads resulting from metabarcoding were uploaded into the project MBR-BCISOIL of mBRAVE. mBRAVE parameters were optimized for classifying sequences with BOLD datasets which have already been curated and the various sequence errors removed. Hence, the stringent read processing was a bit more relaxed. All mBRAVE parameters are indicated in Supplementary Table <a data-track=click data-track-action="supplementary material anchor" data-track-label=link href=#MOESM1>S1</a>, but the workflow can be summarized as follows.<p>The paired ends of the two overlapping amplicons were pooled (i.e., treated as one fastq file without any merging) as opposed to merging, since (a) we were expecting very high matches to mBRAVE datasets (see below) with variation within a BIN being small; and (b) a sensitivity analysis, where we varied most mBRAVE parameters one by one to optimize the greatest number of BINs recovered in mBRAVE datasets, indicated an optimal recovery of BINs with pooled sequences (Supplementary Appendix <a data-track=click data-track-action="supplementary material anchor" data-track-label=link href=#MOESM1>S1</a> and Supplementary Table <a data-track=click data-track-action="supplementary material anchor" data-track-label=link href=#MOESM1>S2</a>). Reads were trimmed at each end (25&nbsp;bp) to remove primer and adapter sequence, then filtered for a minimum quality score of QV20 and for a minimum and maximum size of 100&nbsp;bp and 500&nbsp;bp, respectively (Appendix <a data-track=click data-track-action="supplementary material anchor" data-track-label=link href=#MOESM1>S1</a>). Denoising, dereplication and removal of chimeric reads, were performed with VSEARCH<sup><a aria-label="Reference 57" data-test=citation-ref data-track=click data-track-action="reference anchor" data-track-label=link href=#ref-CR57 id=ref-link-section-d26459365e2006 title="Rognes, T., Flouri, T., Nichols, B., Quince, C. &amp; Mahé, F. VSEARCH: A versatile open source tool for metagenomics. PeerJ 4, e2584. 
                  https://doi.org/10.7717/peerj.2584
                  
                 (2016).">57</a></sup>, integrated into the mBRAVE pipeline (Appendix <a data-track=click data-track-action="supplementary material anchor" data-track-label=link href=#MOESM1>S1</a>). Erroneous sequences were detected by a divergence range from reference sequences, with parameters as indicated in Supplementary Table <a data-track=click data-track-action="supplementary material anchor" data-track-label=link href=#MOESM1>S1</a>. Sequences were identified to BIN level by matching them to reference BINs based on BOLD library datasets tailored for this study, using VSEARCH. These datasets included sequences obtained specifically for this study (2571 BINs for BCI arthropods) and publicly available datasets within BOLD (Appendix <a data-track=click data-track-action="supplementary material anchor" data-track-label=link href=#MOESM1>S1</a>, Supplementary Table <a data-track=click data-track-action="supplementary material anchor" data-track-label=link href=#MOESM1>S1</a>). BINs assignments were realized at 3%, a commonly employed threshold<sup><a aria-label="Reference 58" data-test=citation-ref data-track=click data-track-action="reference anchor" data-track-label=link href=#ref-CR58 id=ref-link-section-d26459365e2023 title="Edgar, R. C. UPARSE: Highly accurate OTU sequences from microbial amplicon reads. Nat. Methods 10, 996–998 (2013).">58</a></sup>. Reads without a match to reference BOLD datasets were clustered into operational taxonomic units (OTU) based on (1) the minimum OTU size (n = 1 read); (2) OTU threshold (maximum distance inside a generated OTU, a conservative 2%); and (3) exclusion from the OTU threshold when sequencing error introduces spurious haplotypes (chimeras and/or sequence errors, 3%). The BOLD datasets that we used for analyses are currently the best ones available for ants, springtails and termites from BCI (Supplementary Appendix <a data-track=click data-track-action="supplementary material anchor" data-track-label=link href=#MOESM1>S1</a> and Supplementary Table <a data-track=click data-track-action="supplementary material anchor" data-track-label=link href=#MOESM1>S1</a>). It is important to note that, although sequence merging is often recommended to remove potentially erroneous sequences, we opted to pool our reads without merging as this retrieved a higher number of BINs. This strategy is possible because (a) we count with an extensive barcode reference library which has been continuously updated and curated by co-authors of this manuscript and represents the most complete insect barcode reference library available for Barro Colorado Island; and (b) this represents a first step towards a standardized and automated arthropod monitoring protocol where our primary goal is to detect reads matching known species barcodes.<p>Depending on the data and aims of the metabarcoding study, this approach may or may not be relevant.<h3 class=c-article__sub-heading id=Sec7>Statistical analyses</h3><p>To ensure sound comparisons between traditional and metabarcoding samples, we removed five pairs of samples for which DNA metabarcoding samples failed (all from the dry season; PCR failed), leaving 95 pairs of samples for analysis. For monitoring, it is necessary to identify unequivocally species by their binomials or BINs, so that they can be cross-referenced among years. Hence, for species sorted manually, we only considered them if they had been sequenced and had a BIN, given that some species failed to be sequenced (see “<a data-track=click data-track-action="section anchor" data-track-label=link href=#Sec13>Results</a>”). Common (as opposed to rare) species were defined as species belonging to the first quartile of the species-rank prevalence distribution<sup><a aria-label="Reference 59" data-test=citation-ref data-track=click data-track-action="reference anchor" data-track-label=link href=#ref-CR59 id=ref-link-section-d26459365e2047 title="Gaston, K. J. Rarity (Springer, 1994).">59</a></sup>, for species sorted in traditional samples. The 95 metabarcoding samples resulted in a total of 122 sequencing runs (i.e., single fastq files produced by the sequencer through demultiplexing or from a single sample) and hence sampling effort was not strictly identical among locations sampled. However, unequal (but similar) sampling effort among locations did not affect any of our questions, and small differences in sample size among locations were ignored.<p>Despite harvesting traditional and metabarcoding samples only 10&nbsp;cm apart, we can still expect discrepancies in species abundance and faunal comparison between these samples. This results from soil organisms, such as ants, to be very patchily distributed when studied at the 1&nbsp;m<sup>2</sup> scale<sup><a aria-label="Reference 60" data-test=citation-ref data-track=click data-track-action="reference anchor" data-track-label=link href=#ref-CR60 id=ref-link-section-d26459365e2056 title="Kaspari, M. Litter ant patchiness at the 1–m2 scale: Disturbance dynamics in three Neotropical forests. Oecologia 107, 265–273 (1996).">60</a></sup>. To reduce this effect, our analyses emphasized data in which samples were grouped per location, as to evaluate differences at a higher scale. This issue is further discussed in Appendix <a data-track=click data-track-action="supplementary material anchor" data-track-label=link href=#MOESM1>S2</a>.<h3 class=c-article__sub-heading id=Sec8>Question1: similarity between traditional and metabarcoding samples</h3><p>To compare species richness between sample types (traditional and metabarcoding) and taxa (prevalence data), we computed rarefaction curves and estimates of total species richness with the R package iNEXT<sup><a aria-label="Reference 61" data-test=citation-ref data-track=click data-track-action="reference anchor" data-track-label=link href=#ref-CR61 id=ref-link-section-d26459365e2071 title="Hsieh, T. C., Ma, K. H. &amp; Chao, A. iNEXT: An R package for rarefaction and extrapolation of species diversity (Hill numbers). Methods Ecol. Evol. 7, 1451–1456 (2016).">61</a></sup>. To visualize the differences in faunal composition between traditional and metabarcoding samples, we generated differential heat trees using the Metacoder R package<sup><a aria-label="Reference 62" data-test=citation-ref data-track=click data-track-action="reference anchor" data-track-label=link href=#ref-CR62 id=ref-link-section-d26459365e2075 title="Foster, Z. S. L., Sharpton, T. J. &amp; Grünwald, N. J. Metacoder: An R package for visualization and manipulation of community taxonomic diversity data. PLoS Comput. Biol. 13, e1005404. 
                  https://doi.org/10.1371/journal.pcbi.1005404
                  
                 (2017).">62</a></sup>. These trees help identifying the taxonomic hierarchy with terminal nodes representing either BINs or binomial species. With a prevalence matrix, we used the function ‘compare_groups’ of Metacoder and the resulting mean difference between treatments to plot the differential heat tree. The differences, however, were formally tested using a Procrustes rotation, which rotates a configuration (for example ordination results) to maximum similarity with another configuration. First, for each dataset (traditional and metabarcoding), we pooled samples by the 10 locations at our study site (Table <a data-track=click data-track-action="table anchor" data-track-label=link href=#Tab1>1</a>; Fig.&nbsp;<a data-track=click data-track-action="figure anchor" data-track-label=link href=#Fig1>1</a>) and considered matrices of species × locations, filled with the number of samples in which the species was recorded for each location (prevalence; maximum possible of 10 samples for each location). We then performed a non-metric multidimensional scaling (NMDS) for each of the traditional and metabarcoding matrices with the function ‘metaMDS’ of the R package vegan, followed by a Procrustes rotation of the two ordinations with the function ‘procrustes’ of the same package<sup><a aria-label="Reference 63" data-test=citation-ref data-track=click data-track-action="reference anchor" data-track-label=link href=#ref-CR63 id=ref-link-section-d26459365e2085 title="Oksanen, J. et al. vegan: Community Ecology Package. R package version 2.5-3 (2018).">63</a></sup>. The function ‘protest’ of vegan was used to tests for non-randomness (‘significance’) between the traditional and metabarcoding NMDS (999 permutations). We performed Procrustes rotations separately for ants, springtails and termite data, and either for all species surveyed or restricted to common species.<h3 class=c-article__sub-heading id=Sec9>Question 2: regressions between abundance and prevalence</h3><p>We considered the relationships between the following variables, for each species: (a) total abundance vs. prevalence in traditional samples; (b) total abundance in traditional samples vs. prevalence in metabarcoding samples; and (c) prevalence in traditional samples vs. prevalence in metabarcoding samples. We tested these three relationships for the three taxonomic groups together, and then for each group separately, considering either all species or only common species. We first checked linear and non-linear relationships with CurveExpert Professional<sup><a aria-label="Reference 64" data-test=citation-ref data-track=click data-track-action="reference anchor" data-track-label=link href=#ref-CR64 id=ref-link-section-d26459365e2097 title="Hyams, D. G. CurveExpert Professional. A Comprehensive Data Analysis Software System for Windows, Mac, and Linux. Version 1.2.2. 
                  www.curveexpert.net
                  
                 (2011). Accessed 1 Jan 2022.">64</a></sup>, considering the highest coefficient of determination (R<sup>2</sup>) and the lowest Akaike information criterion corrected (AICc) of all models. Given that linear relationships provided the best fit for all models tested, we fitted ordinary least squares regressions to all tested relationships.<h3 class=c-article__sub-heading id=Sec10>Question 3: seasonal shifts</h3><p>We first calculated, for each species collected both in the dry and wet seasons (n = 62), the difference in the prevalence in the wet season samples minus the prevalence in the dry season samples, separately for traditional and metabarcoding samples. We then fitted ordinary least squares regressions between the seasonal difference in traditional samples and that in metabarcoding samples. We performed these regressions for all species, ants, springtails and termites, but not separately for common species, because of low sample size. Eventually, to evaluate the similarity between the traditional and metabarcoding samples regarding shifts in faunal composition between the dry and wet season, we considered separately for traditional and metabarcoding samples, a prevalence matrix of species (n = 62) x season (dry or wet). We then calculated the correlation between the two matrices using a Mantel test, computed with the function ‘mantel’ of the R package vegan (10,000 permutations<sup><a aria-label="Reference 63" data-test=citation-ref data-track=click data-track-action="reference anchor" data-track-label=link href=#ref-CR63 id=ref-link-section-d26459365e2111 title="Oksanen, J. et al. vegan: Community Ecology Package. R package version 2.5-3 (2018).">63</a></sup>).<h3 class=c-article__sub-heading id=Sec11>Question 4: regressions between biomass and read frequency</h3><p>We considered only ants, for which we had reliable measurements of different variables accounting for body size, as well as an allometric correlation of body length to body weight. Analyses were restricted to ant species present in metabarcoding samples for which we had good measurements (1–5 individuals measured per species, n = 28 species). We tested linear and non-linear relationships with CurveExpert Professional<sup><a aria-label="Reference 64" data-test=citation-ref data-track=click data-track-action="reference anchor" data-track-label=link href=#ref-CR64 id=ref-link-section-d26459365e2123 title="Hyams, D. G. CurveExpert Professional. A Comprehensive Data Analysis Software System for Windows, Mac, and Linux. Version 1.2.2. 
                  www.curveexpert.net
                  
                 (2011). Accessed 1 Jan 2022.">64</a></sup>, including, for each species, as dependent variables the total number of reads, the relative read abundance<sup><a aria-label="Reference 65" data-test=citation-ref data-track=click data-track-action="reference anchor" data-track-label=link href=#ref-CR65 id=ref-link-section-d26459365e2127 title="Deagle, B. E. et al. Counting with DNA in metabarcoding studies: How should we convert sequence reads to dietary data?. Mol. Ecol. 28, 391–406 (2019).">65</a></sup>, the total number of sequences and prevalence in metabarcoding samples, and as independent variables either body length (mm), pronotum width (mm), Weber’s length (mm) or body weight (g), measured from ant workers in collections of the ForestGEO Arthropod Initiative.<h3 class=c-article__sub-heading id=Sec12>Question 5: false positive and negative records</h3><p>Assuming no taxonomic errors, we considered all species occurring in metabarcoding samples but not in traditional samples as “possible false positive”, those occurring in traditional samples but not in metabarcoding samples as “possible false negative” and the rest of species as “well surveyed”. Furthermore, we considered species as “probable false positive” and “probable false negative” when their prevalence in samples was ≥ 10, in metabarcoding samples and traditional samples, respectively (i.e., when at least one specimen was collected potentially at each of the locations). We performed a discriminant analysis to test whether morphological traits such as body length, pronotum width and Weber’s length (Question 4) delineated some of the categories defined above. This analysis concerned only Formicidae and the species in categories “probable false negative” and “well surveyed”, as sample size was too low for species in other categories (morphological measurements lacked for most of false positives). Discriminant analysis was performed with the function ‘discrim’ of Systat version 13.1 (Systat Software, Inc., Chicago, IL). Given that the barcodes for our false negative samples exist, we used the ecoPCR<sup><a aria-label="Reference 66" data-test=citation-ref data-track=click data-track-action="reference anchor" data-track-label=link href=#ref-CR66 id=ref-link-section-d26459365e2140 title="Ficetola, G. F. et al. An In Silico approach for the evaluation of DNA barcodes. BMC Genom. 11, 434. 
                  https://doi.org/10.1186/1471-2164-11-434
                  
                 (2010).">66</a></sup> program to run an in silico PCR to test whether or not our primers could amplify these regions. Barcodes were downloaded from the NCBI database to create a reference dataset to match with our PCR primer pairs allowing up to 15 mismatches between primer and target sequences. To test for differences between taxa (Question 6), whenever possible we kept analyses separate among taxa when investigating Questions 1–5.</p></div>