<!DOCTYPE html>
<html class="js" style="--font-family-sans: Europa; --font-family-serif: Noto Serif;" lang="en"><head>
<meta http-equiv="content-type" content="text/html; charset=UTF-8">
        <title>Hybridization in East African swarm-raiding army ants | Frontiers in Zoology | Full Text</title>

        

    <meta name="citation_abstract" content="Hybridization can have complex effects on evolutionary dynamics in ants because of the combination of haplodiploid sex-determination and eusociality. While hybrid non-reproductive workers have been found in a range of species, examples of gene-flow via hybrid queens and males are rare. We studied hybridization in East African army ants (Dorylus subgenus Anomma) using morphology, mitochondrial DNA sequences, and nuclear microsatellites. While the mitochondrial phylogeny had a strong geographic signal, different species were not recovered as monophyletic. At our main study site at Kakamega Forest, a mitochondrial haplotype was shared between a " dorylus="" molestus-like"="" and="" a="" "dorylus="" wilverthi-like"="" form.="" this="" pattern="" is="" best="" explained="" by="" introgression="" following="" hybridization="" between="" d.="" molestus="" wilverthi.="" microsatellite="" data="" from="" workers="" showed="" that="" the="" two="" morphological="" forms="" correspond="" to="" distinct="" genetic="" clusters,="" with="" significant="" proportion="" of="" individuals="" being="" classified="" as="" hybrids.="" we="" conclude="" gene-flow="" army="" ant="" species="" wilverthi="" has="" occurred,="" mating="" continues="" regularly="" produce="" hybrid="" workers.="" particularly="" surprising="" in="" ants="" because="" have="" control="" over="" which="" males="" are="" allowed="" mate="" young="" virgin="" queen="" inside="" colony."="">

    <meta name="description" content="Hybridization can have complex effects on evolutionary dynamics in ants because of the combination of haplodiploid sex-determination and eusociality. While hybrid non-reproductive workers have been found in a range of species, examples of gene-flow via hybrid queens and males are rare. We studied hybridization in East African army ants (Dorylus subgenus Anomma) using morphology, mitochondrial DNA sequences, and nuclear microsatellites. While the mitochondrial phylogeny had a strong geographic signal, different species were not recovered as monophyletic. At our main study site at Kakamega Forest, a mitochondrial haplotype was shared between a " dorylus="" molestus-like"="" and="" a="" "dorylus="" wilverthi-like"="" form.="" this="" pattern="" is="" best="" explained="" by="" introgression="" following="" hybridization="" between="" d.="" molestus="" wilverthi.="" microsatellite="" data="" from="" workers="" showed="" that="" the="" two="" morphological="" forms="" correspond="" to="" distinct="" genetic="" clusters,="" with="" significant="" proportion="" of="" individuals="" being="" classified="" as="" hybrids.="" we="" conclude="" gene-flow="" army="" ant="" species="" wilverthi="" has="" occurred,="" mating="" continues="" regularly="" produce="" hybrid="" workers.="" particularly="" surprising="" in="" ants="" because="" have="" control="" over="" which="" males="" are="" allowed="" mate="" young="" virgin="" queen="" inside="" colony."="">

    <meta name="journal_id" content="12983">

    <meta name="dc.title" content="Hybridization in East African swarm-raiding army ants">

    <meta name="dc.source" content="Frontiers in Zoology 2011 8:1">

    <meta name="dc.format" content="text/html">

    <meta name="dc.publisher" content="BioMed Central">

    <meta name="dc.date" content="2011-08-22">

    <meta name="dc.type" content="OriginalPaper">

    <meta name="dc.language" content="En">

    <meta name="dc.copyright" content="2011 Kronauer et al; licensee BioMed Central Ltd.">

    <meta name="dc.rights" content="http://creativecommons.org/licenses/by/2.0">

    <meta name="dc.rightsAgent" content="reprints@biomedcentral.com">

    <meta name="dc.description" content="Hybridization can have complex effects on evolutionary dynamics in ants because of the combination of haplodiploid sex-determination and eusociality. While hybrid non-reproductive workers have been found in a range of species, examples of gene-flow via hybrid queens and males are rare. We studied hybridization in East African army ants (Dorylus subgenus Anomma) using morphology, mitochondrial DNA sequences, and nuclear microsatellites. While the mitochondrial phylogeny had a strong geographic signal, different species were not recovered as monophyletic. At our main study site at Kakamega Forest, a mitochondrial haplotype was shared between a &quot;Dorylus molestus-like&quot; and a &quot;Dorylus wilverthi-like&quot; form. This pattern is best explained by introgression following hybridization between D. molestus and D. wilverthi. Microsatellite data from workers showed that the two morphological forms correspond to two distinct genetic clusters, with a significant proportion of individuals being classified as hybrids. We conclude that hybridization and gene-flow between the two army ant species D. molestus and D. wilverthi has occurred, and that mating between the two forms continues to regularly produce hybrid workers. Hybridization is particularly surprising in army ants because workers have control over which males are allowed to mate with a young virgin queen inside the colony.">

    <meta name="prism.issn" content="1742-9994">

    <meta name="prism.publicationName" content="Frontiers in Zoology">

    <meta name="prism.publicationDate" content="2011-08-22">

    <meta name="prism.volume" content="8">

    <meta name="prism.number" content="1">

    <meta name="prism.section" content="OriginalPaper">

    <meta name="prism.startingPage" content="20">

    <meta name="prism.copyright" content="2011 Kronauer et al; licensee BioMed Central Ltd.">

    <meta name="prism.rightsAgent" content="reprints@biomedcentral.com">

    <meta name="prism.url" content="https://frontiersinzoology.biomedcentral.com/articles/10.1186/1742-9994-8-20">

    <meta name="prism.doi" content="doi:10.1186/1742-9994-8-20">

    <meta name="citation_pdf_url" content="https://frontiersinzoology.biomedcentral.com/track/pdf/10.1186/1742-9994-8-20">

    <meta name="citation_fulltext_html_url" content="https://frontiersinzoology.biomedcentral.com/articles/10.1186/1742-9994-8-20">

    <meta name="citation_journal_title" content="Frontiers in Zoology">

    <meta name="citation_publisher" content="BioMed Central">

    <meta name="citation_issn" content="1742-9994">

    <meta name="citation_title" content="Hybridization in East African swarm-raiding army ants">

    <meta name="citation_volume" content="8">

    <meta name="citation_issue" content="1">

    <meta name="citation_publication_date" content="2011/12">

    <meta name="citation_online_date" content="2011/08/22">

    <meta name="citation_firstpage" content="20">

    <meta name="citation_article_type" content="Research">

    <meta name="citation_fulltext_world_readable" content="">

    <meta name="dc.identifier" content="doi:10.1186/1742-9994-8-20">

    <meta name="DOI" content="10.1186/1742-9994-8-20">

    <meta name="citation_doi" content="10.1186/1742-9994-8-20">

    <meta name="dc.creator" content="Daniel JC Kronauer">

    <meta name="dc.creator" content="Marcell K Peters">

    <meta name="dc.creator" content="Caspar Schöning">

    <meta name="dc.creator" content="Jacobus J Boomsma">

    <meta name="dc.subject" content="Zoology">

    <meta name="citation_reference" content="citation_journal_title=Annu Rev Ecol Syst; citation_title=Hybrid origins of plant species; citation_author=LH Rieseberg; citation_volume=28; citation_publication_date=1997; citation_pages=359-389; citation_doi=10.1146/annurev.ecolsys.28.1.359; citation_id=CR1">

    <meta name="citation_reference" content="citation_journal_title=Annu Rev Eco Syst; citation_title=The role of hybridization and introgression in the diversification of animals; citation_author=TE Dowling, CL Secor; citation_volume=28; citation_publication_date=1997; citation_pages=593-619; citation_doi=10.1146/annurev.ecolsys.28.1.593; citation_id=CR2">

    <meta name="citation_reference" content="citation_journal_title=Trends Ecol Evol; citation_title=Hybridization and adaptive radiation; citation_author=O Seehausen; citation_volume=19; citation_publication_date=2004; citation_pages=198-207; citation_doi=10.1016/j.tree.2004.01.003; citation_id=CR3">

    <meta name="citation_reference" content="citation_journal_title=Nature; citation_title=Hybrid speciation; citation_author=J Mallet; citation_volume=446; citation_publication_date=2007; citation_pages=279-283; citation_doi=10.1038/nature05706; citation_id=CR4">

    <meta name="citation_reference" content="citation_journal_title=Mol Ecol; citation_title=Homoploid hybrid speciation in animals; citation_author=J Mavárez, M Linares; citation_volume=17; citation_publication_date=2008; citation_pages=4181-4185; citation_doi=10.1111/j.1365-294X.2008.03898.x; citation_id=CR5">

    <meta name="citation_reference" content="citation_journal_title=Annu Rev Genet; citation_title=Genetics and the fitness of hybrids; citation_author=JM Burke, ML Arnold; citation_volume=35; citation_publication_date=2001; citation_pages=31-52; citation_doi=10.1146/annurev.genet.35.102401.085719; citation_id=CR6">

    <meta name="citation_reference" content="citation_journal_title=Mol Ecol; citation_title=Speciation in reverse: morphological and genetic evidence of the collapse of a three-spined stickleback (Gasterosteus aculeatus) species pair; citation_author=EB Taylor, JW Boughman, M Groenenboom, M Sniatynski, D Schluter, JL Gow; citation_volume=15; citation_publication_date=2006; citation_pages=343-355; citation_doi=10.1111/j.1365-294X.2005.02794.x; citation_id=CR7">

    <meta name="citation_reference" content="citation_journal_title=Myrmecol News; citation_title=The causes and consequences of genetic caste determination in ants (Hymenoptera: Formicidae); citation_author=KE Anderson, TA Linksvayer, CR Smith; citation_volume=11; citation_publication_date=2008; citation_pages=119-132; citation_id=CR8">

    <meta name="citation_reference" content="citation_journal_title=Trends Ecol Evol; citation_title=Nature versus nurture in social insect caste differentiation; citation_author=T Schwander, N Lo, M Beekman, BP Oldroyd, L Keller; citation_volume=25; citation_publication_date=2010; citation_pages=275-282; citation_doi=10.1016/j.tree.2009.12.001; citation_id=CR9">

    <meta name="citation_reference" content="citation_journal_title=Ecology; citation_title=Sperm parasitism in ants: selection for interspecific mating and hybridization; citation_author=GJ Umphrey; citation_volume=87; citation_publication_date=2006; citation_pages=2148-2159; citation_doi=10.1890/0012-9658(2006)87[2148:SPIASF]2.0.CO;2; citation_id=CR10">

    <meta name="citation_reference" content="citation_journal_title=Philos T R Soc B; citation_title=Lifelong commitment to the wrong partner: hybridization in ants; citation_author=H Feldhaar, S Foitzik, J Heinze; citation_volume=363; citation_publication_date=2008; citation_pages=2891-2899; citation_doi=10.1098/rstb.2008.0022; citation_id=CR11">

    <meta name="citation_reference" content="citation_journal_title=Myrmecological News; citation_title=Mating, hybridisation and introgression in Lasius ants (Hymenoptera, Formicidae); citation_author=TM Van der Have, JS Pedersen, JJ Boomsma; citation_volume=15; citation_publication_date=2010; citation_pages=109-115; citation_id=CR12">

    <meta name="citation_reference" content="citation_journal_title=Proc Natl Acad Sci USA; citation_title=Genetic basis for queen-worker dimorphism in a social insect; citation_author=VP Volny, DM Gordon; citation_volume=99; citation_publication_date=2002; citation_pages=6108-6111; citation_doi=10.1073/pnas.092066699; citation_id=CR13">

    <meta name="citation_reference" content="citation_journal_title=Nature; citation_title=Complex hybrid origin of genetic caste determination in harvester ants; citation_author=SH Helms Cahan, L Keller; citation_volume=424; citation_publication_date=2003; citation_pages=306-309; citation_doi=10.1038/nature01744; citation_id=CR14">

    <meta name="citation_reference" content="citation_journal_title=Proc Natl Acad Sci USA; citation_title=Segregation distortion causes large-scale differences between male and female genomes in hybrid ants; citation_author=J Kulmuni, B Seifert, P Pamilo; citation_volume=107; citation_publication_date=2010; citation_pages=7371-7376; citation_doi=10.1073/pnas.0912409107; citation_id=CR15">

    <meta name="citation_reference" content="citation_journal_title=BMC Evol Biol; citation_title=A molecular phylogeny of Dorylus army ants provides evidence for multiple evolutionary transitions in foraging niche; citation_author=DJC Kronauer, C Schöning, L Vilhelmsen, JJ Boomsma; citation_volume=7; citation_publication_date=2007; citation_pages=56; citation_doi=10.1186/1471-2148-7-56; citation_id=CR16">

    <meta name="citation_reference" content="citation_journal_title=Mol Ecol; citation_title=Extreme queen-mating frequency and colony fission in African army ants; citation_author=DJC Kronauer, C Schöning, JS Pedersen, JJ Boomsma, J Gadau; citation_volume=13; citation_publication_date=2004; citation_pages=2381-2388; citation_doi=10.1111/j.1365-294X.2004.02262.x; citation_id=CR17">

    <meta name="citation_reference" content="citation_journal_title=J Evolution Biol; citation_title=Dispersal and population structure of a New World predator, the army ant Eciton burchellii; citation_author=SM Berghoff, DJC Kronauer, KJ Edwards, NR Franks; citation_volume=21; citation_publication_date=2008; citation_pages=1125-1132; citation_doi=10.1111/j.1420-9101.2008.01531.x; citation_id=CR18">

    <meta name="citation_reference" content="citation_journal_title=Proc Roy Soc B; citation_title=Colony fusion and worker reproduction after queen loss in army ants; citation_author=DJC Kronauer, C Schöning, P d'Ettorre, JJ Boomsma; citation_volume=277; citation_publication_date=2010; citation_pages=755-763; citation_doi=10.1098/rspb.2009.1591; citation_id=CR19">

    <meta name="citation_reference" content="citation_journal_title=Biol J Linn Soc; citation_title=Sexual competition during colony reproduction in army ants; citation_author=NR Franks, B Hölldobler; citation_volume=30; citation_publication_date=1987; citation_pages=229-243; citation_doi=10.1111/j.1095-8312.1987.tb00298.x; citation_id=CR20">

    <meta name="citation_reference" content="citation_journal_title=J East Afr Nat Hist; citation_title=Does the afrotropical army ant Dorylus (Anomma) molestus go extinct in fragmented forests?; citation_author=C Schöning, W Kinuthia, JJ Boomsma; citation_volume=95; citation_publication_date=2006; citation_pages=163-179; citation_doi=10.2982/0012-8317(2006)95[163:DTAAAD]2.0.CO;2; citation_id=CR21">

    <meta name="citation_reference" content="Peters MK, Okalo B: Severe declines of ant-following birds in African rainforest fragments are facilitated by a subtle change in army ant communities. Biol Conserv. 142: 2050-2058.">

    <meta name="citation_reference" content="citation_journal_title=Biol Conserv; citation_title=Species compensation maintains abundance and raid rates of African swarm-raiding army ants in rainforest fragments; citation_author=MK Peters, G Fischer, G Schaab, M Kraemer; citation_volume=142; citation_publication_date=2009; citation_pages=668-675; citation_doi=10.1016/j.biocon.2008.11.021; citation_id=CR23">

    <meta name="citation_reference" content="citation_title=Der Polymorphismus der afrikanischen Wanderameisen unter biometrischen und biologischen Gesichtspunkten; citation_inbook_title=Sozialpolymorphismus bei Insekten. Probleme der Kastenbildung im Tierreich; citation_publication_date=1974; citation_pages=668-693; citation_id=CR24; citation_author=A Raignier; citation_author=JKA Van Boven; citation_author=R Ceusters; citation_publisher=Wissenschaftliche Verlagsgesellschaft">

    <meta name="citation_reference" content="citation_journal_title=Zeitschr Wissenschaftl Zool; citation_title=Neue Anpassungstypen bei Dorylinengästen Afrikas (Col. Staphylinidae). (218. Beitrag zur Kenntnis der Myrmekophilen.); citation_author=E Wasmann; citation_volume=17; citation_publication_date=1917; citation_pages=257-360; citation_id=CR25">

    <meta name="citation_reference" content="citation_journal_title=Mol Ecol; citation_title=The incomplete natural history of mitochondria; citation_author=JWO Ballard, MC Whitlock; citation_volume=13; citation_publication_date=2004; citation_pages=729-744; citation_doi=10.1046/j.1365-294X.2003.02063.x; citation_id=CR26">

    <meta name="citation_reference" content="citation_journal_title=Science; citation_title=Population size does not influence mitochondrial genetic diversity in animals; citation_author=E Bazin, S Glemin, N Galtier; citation_volume=312; citation_publication_date=2006; citation_pages=570-572; citation_doi=10.1126/science.1122033; citation_id=CR27">

    <meta name="citation_reference" content="citation_journal_title=Nature; citation_title=Modelled atmospheric temperatures and global sea levels over the past million years; citation_author=R Bintanja, RSW Van de Wal, J Oerlemans; citation_volume=437; citation_publication_date=2005; citation_pages=125-128; citation_doi=10.1038/nature03975; citation_id=CR28">

    <meta name="citation_reference" content="citation_journal_title=Global Planet Change; citation_title=Sea-level curves: perplexities of an end-user in morphotectonic applications; citation_author=R Caputo; citation_volume=57; citation_publication_date=2007; citation_pages=417-423; citation_doi=10.1016/j.gloplacha.2007.03.003; citation_id=CR29">

    <meta name="citation_reference" content="citation_journal_title=J Biogeogr; citation_title=The biogeographical assignment of a west Kenyan rain forest remnant: further evidence from analysis of its reptile fauna; citation_author=P Wagner, J Köhler, A Schmitz, W Böhme; citation_volume=35; citation_publication_date=2008; citation_pages=1349-1361; citation_doi=10.1111/j.1365-2699.2008.01883.x; citation_id=CR30">

    <meta name="citation_reference" content="citation_journal_title=J Ecol; citation_title=Pleistocene climatic changes and the distribution of life in East Africa; citation_author=RE Moreau; citation_volume=21; citation_publication_date=1933; citation_pages=415-435; citation_doi=10.2307/2256590; citation_id=CR31">

    <meta name="citation_reference" content="citation_title=Quaternary and environmental research on East African mountains; citation_publication_date=1989; citation_id=CR32; citation_author=WC Mahaney; citation_publisher=AA. Balkema">

    <meta name="citation_reference" content="citation_journal_title=FEMS Microbiol Let; citation_title=How many species are infected with Wolbachia? - a statistical analysis of current data; citation_author=K Hilgenboecker, P Hammerstein, P Schlattmann, A Telschow, JH Werren; citation_volume=281; citation_publication_date=2008; citation_pages=215-220; citation_doi=10.1111/j.1574-6968.2008.01110.x; citation_id=CR33">

    <meta name="citation_reference" content="citation_journal_title=Evolution; citation_title=Extensive introgression of mitochondrial DNA relative to nuclear genes in the Drosophila yakuba species group; citation_author=D Bachtrog, K Thornton, A Clark, P Andolfatto; citation_volume=60; citation_publication_date=2006; citation_pages=292-302; citation_doi=10.1111/j.0014-3820.2006.tb01107.x; citation_id=CR34">

    <meta name="citation_reference" content="citation_journal_title=Proc Natl Acad Sci USA; citation_title=Interspecific cytoplasmic gene flow in the absence of nuclear gene flow: evidence from Drosophila; citation_author=J Powell; citation_volume=80; citation_publication_date=1983; citation_pages=492-495; citation_doi=10.1073/pnas.80.2.492; citation_id=CR35">

    <meta name="citation_reference" content="citation_journal_title=Nat Genet; citation_title=Cytonuclear genomic dissociation in African elephant species; citation_author=A Roca, N Georgiadis, S O'Brien; citation_volume=37; citation_publication_date=2005; citation_pages=96-100; citation_id=CR36">

    <meta name="citation_reference" content="citation_journal_title=Evolution; citation_title=Mitonuclear discordance is caused by rampant mitochondrial introgression in Neodiprion (Hymenoptera: Diprionidae) sawflies; citation_author=CR Linnen, BD Farrell; citation_volume=61; citation_publication_date=2007; citation_pages=1417-1438; citation_doi=10.1111/j.1558-5646.2007.00114.x; citation_id=CR37">

    <meta name="citation_reference" content="citation_journal_title=Heredity; citation_title=Genetics of postzygotic isolation and Haldane's rule in haplodiploids; citation_author=T Koevoets, LW Beukeboom; citation_volume=102; citation_publication_date=2009; citation_pages=16-23; citation_doi=10.1038/hdy.2008.44; citation_id=CR38">

    <meta name="citation_reference" content="citation_journal_title=Proc Roy Soc B; citation_title=Recurrent replacement of mtDNA and cryptic hybridization between two sibling bat species Myotis myotis and Myotis blythii; citation_author=P Berthier, L Excoffier, M Ruedi; citation_volume=273; citation_publication_date=2006; citation_pages=3101-3109; citation_doi=10.1098/rspb.2006.3680; citation_id=CR39">

    <meta name="citation_reference" content="citation_journal_title=Am Nat; citation_title=A statistical approach for distinguishing hybridization and incomplete lineage sorting; citation_author=S Joly, PA McLenachan, PJ Lockhart; citation_volume=174; citation_publication_date=2009; citation_pages=E54-E70; citation_doi=10.1086/600082; citation_id=CR40">

    <meta name="citation_reference" content="citation_journal_title=BMC Evol Biol; citation_title=Evolutionary diversification of cryophilic Grylloblatta species (Grylloblattodea: Grylloblattidae) in alpine habitats of California; citation_author=SD Schoville, GK Roderick; citation_volume=10; citation_publication_date=2010; citation_pages=163; citation_doi=10.1186/1471-2148-10-163; citation_id=CR41">

    <meta name="citation_reference" content="citation_journal_title=Syst Biol; citation_title=Species delimitation: a case study in a problematic ant taxon; citation_author=KG Ross, D Gotzek, MS Ascunce, DD Shoemaker; citation_volume=59; citation_publication_date=2010; citation_pages=162-184; citation_doi=10.1093/sysbio/syp089; citation_id=CR42">

    <meta name="citation_reference" content="citation_journal_title=Evol Ecol; citation_title=Independent hybrid populations of Formica polyctena X rufa wood ants (Hymenoptera: Formicidae) abound under conditions of forest fragmentation; citation_author=B Seifert, J Kulmuni, P Pamilo; citation_volume=24; citation_publication_date=2010; citation_pages=1219-1237; citation_doi=10.1007/s10682-010-9371-8; citation_id=CR43">

    <meta name="citation_reference" content="citation_journal_title=Mol Ecol; citation_title=Male parentage in army ants; citation_author=DJC Kronauer, C Schöning, JJ Boomsma; citation_volume=15; citation_publication_date=2006; citation_pages=1147-1151; citation_doi=10.1111/j.1365-294X.2005.02850.x; citation_id=CR44">

    <meta name="citation_reference" content="citation_journal_title=Sociobiology; citation_title=Taxonomic implications of doryline worker ant morphology: Dorylus subgenus Anomma (Hymenoptera: Formicidae); citation_author=WH Gotwald, RF Schaefer; citation_volume=7; citation_publication_date=1982; citation_pages=187-204; citation_id=CR45">

    <meta name="citation_reference" content="citation_journal_title=Ann Entomol Soc Am; citation_title=Predatory behavior and food preferences of driver ants in selected African habitats; citation_author=WH Gotwald; citation_volume=67; citation_publication_date=1974; citation_pages=877-886; citation_doi=10.1093/aesa/67.6.877; citation_id=CR46">

    <meta name="citation_reference" content="citation_journal_title=Oikos; citation_title=Evolution of allometries in the worker caste of Dorylus army ants; citation_author=C Schöning, W Kinuthia, NR Franks; citation_volume=110; citation_publication_date=2005; citation_pages=231-240; citation_doi=10.1111/j.0030-1299.2005.13672.x; citation_id=CR47">

    <meta name="citation_reference" content="citation_journal_title=J Hum Evol; citation_title=The nature of culture: technological variation in chimpanzee predation on army ants revisited; citation_author=C Schöning, T Humle, Y Möbius, WC McGrew; citation_volume=55; citation_publication_date=2008; citation_pages=48-59; citation_doi=10.1016/j.jhevol.2007.12.002; citation_id=CR48">

    <meta name="citation_reference" content="citation_journal_title=Ann Entomol Soc Am; citation_title=Evolution, weighting, and phylogenetic utility of mitochondrial gene sequences and a compilation of conserved polymerase chain reaction primers; citation_author=C Simon, F Frati, A Beckenbach, B Crespi, H Liu, P Flook; citation_volume=87; citation_publication_date=1994; citation_pages=651-701; citation_doi=10.1093/aesa/87.6.651; citation_id=CR49">

    <meta name="citation_reference" content="citation_journal_title=Mol Ecol Notes; citation_title=Microsatellite markers for the driver ant Dorylus (Anomma) molestus; citation_author=DJC Kronauer, JJ Boomsma, J Gadau; citation_volume=4; citation_publication_date=2004; citation_pages=289-290; citation_doi=10.1111/j.1471-8286.2004.00645.x; citation_id=CR50">

    <meta name="citation_reference" content="citation_journal_title=Bioinformatics; citation_title=MRBAYES: Bayesian inference of phylogeny; citation_author=JP Huelsenbeck, F Ronquist; citation_volume=17; citation_publication_date=2001; citation_pages=754-755; citation_doi=10.1093/bioinformatics/17.8.754; citation_id=CR51">

    <meta name="citation_reference" content="citation_journal_title=Bioinformatics; citation_title=MRBAYES 3: Bayesian phylogenetic inference under mixed models; citation_author=F Ronquist, JP Huelsenbeck; citation_volume=19; citation_publication_date=2003; citation_pages=1572-1574; citation_doi=10.1093/bioinformatics/btg180; citation_id=CR52">

    <meta name="citation_reference" content="citation_title=Genetic algorithm approaches for the phylogenetic analysis of large biological sequence datasets under the maximum likelihood criterion; citation_inbook_title=PhD thesis; citation_publication_date=2006; citation_id=CR53; citation_author=DJ Zwickl">

    <meta name="citation_reference" content="citation_title=MrModeltest v2; citation_inbook_title=Program distributed by the author; citation_publication_date=2004; citation_id=CR54; citation_author=JAA Nylander; citation_publisher=Evolutionary Biology Centre">

    <meta name="citation_reference" content="citation_journal_title=Genetics; citation_title=A cladistics analysis of phenotypic associations with haplotypes inferred from restriction endonuclease mapping and DNA sequence data III. Cladogram estimation; citation_author=AR Templeton, KA Crandall, CF Sing; citation_volume=132; citation_publication_date=1992; citation_pages=619-633; citation_id=CR55">

    <meta name="citation_reference" content="citation_journal_title=Molecular Ecology; citation_title=TCS: a computer program to estimate gene genealogies; citation_author=M Clement, D Posada, KA Crandall; citation_volume=9; citation_publication_date=2000; citation_pages=, 1657-1659; citation_doi=10.1046/j.1365-294x.2000.01020.x; citation_id=CR56">

    <meta name="citation_reference" content="Goudet J: FSTAT, a program to estimate and test gene diversities and fixation indices (version 2.9.3.2). [
                    http://www2.unil.ch/popgen/softwares/fstat.htm
                    
                  ]">

    <meta name="citation_reference" content="citation_journal_title=Genetics; citation_title=Inference of population structure using multilocus genotype data; citation_author=JK Pritchard, M Stephens, P Donnelly; citation_volume=155; citation_publication_date=2000; citation_pages=945-959; citation_id=CR58">

    <meta name="citation_reference" content="Falush D, Stephens M, Pritchard JK: Inference of population structure: Extensions to linked loci and correlated allele frequencies. Genetics. 164: 1567-1587.">

    <meta name="citation_reference" content="citation_journal_title=Mol Ecol Notes; citation_title=DISTRUCT: a program for the graphical display of population structure; citation_author=NA Rosenberg; citation_volume=4; citation_publication_date=2004; citation_pages=137-138; citation_doi=10.1046/j.1471-8286.2003.00566.x; citation_id=CR60">

    <meta name="citation_reference" content="citation_journal_title=J Hered; citation_title=GENECLASS2: A software for genetic assignment and first-generation migrant detection; citation_author=S Piry, A Alapetite, JM Cornuet, D Paetkau, L Baudouin, A Estoup; citation_volume=95; citation_publication_date=2004; citation_pages=536-539; citation_doi=10.1093/jhered/esh074; citation_id=CR61">

    <meta name="citation_reference" content="citation_journal_title=Proc Natl Acad Sci USA; citation_title=Detecting immigration by using multilocus genotypes; citation_author=B Rannala, JL Mountain; citation_volume=94; citation_publication_date=1997; citation_pages=9197-9201; citation_doi=10.1073/pnas.94.17.9197; citation_id=CR62">

    <meta name="citation_reference" content="citation_journal_title=Mol Ecol; citation_title=Genetic assignment methods for the direct, real-time estimation of migration rate: a simulation-based exploration of accuracy and power; citation_author=D Paetkau, R Slade, M Burden, A Estoup; citation_volume=13; citation_publication_date=2004; citation_pages=55-65; citation_doi=10.1046/j.1365-294X.2004.02008.x; citation_id=CR63">

    <meta name="citation_author" content="Daniel JC Kronauer">

    <meta name="citation_author_institution" content="Centre for Social Evolution, Department of Biology, University of Copenhagen, Copenhagen, Denmark">

    <meta name="citation_author_institution" content="Laboratory of Insect Social Evolution, The Rockefeller University, New York, USA">

    <meta name="citation_author" content="Marcell K Peters">

    <meta name="citation_author_institution" content="Zoological Research Museum Alexander Koenig, Bonn, Germany">

    <meta name="citation_author_institution" content="Department of Animal Ecology and Tropical Biology, Biocenter, University of Würzburg, Am Hubland, Würzburg, Germany">

    <meta name="citation_author" content="Caspar Schöning">

    <meta name="citation_author_institution" content="Centre for Social Evolution, Department of Biology, University of Copenhagen, Copenhagen, Denmark">

    <meta name="citation_author_institution" content="Länderinstitut für Bienenkunde, Hohen Neuendorf, Germany">

    <meta name="citation_author" content="Jacobus J Boomsma">

    <meta name="citation_author_institution" content="Centre for Social Evolution, Department of Biology, University of Copenhagen, Copenhagen, Denmark">



        <meta charset="UTF-8">
        <meta http-equiv="X-UA-Compatible" content="IE=edge">
        <meta name="viewport" content="width=device-width, initial-scale=1">

        <!-- For IE 9 and below -->
<!--[if IE]><link rel="shortcut icon" href=/static/images/favicons//favicon.ico><![endif]-->
<!-- Touch Icons - iOS and Android 2.1+ -->
<link rel="apple-touch-icon" href="https://frontiersinzoology.biomedcentral.com/static/images/favicons/bmc/favicon-3f5c56eaaa.png">
<!-- Firefox, Chrome, Safari, IE 11+ and Opera -->
<link rel="icon" href="https://frontiersinzoology.biomedcentral.com/static/images/favicons/bmc/favicon-3f5c56eaaa.png">


        <script type="text/javascript" src="https://cdn.krxd.net/userdata/get?pub=bd339c69-af54-4a21-b4f1-654bcfcd83ca&amp;kxfp=a4a585b5facc3eae4b0c54e654a779152d5ea2e4&amp;technographics=1&amp;callback=Krux.ns.macmillan.kxjsonp_userdata"></script><script type="text/javascript" src="https://beacon.krxd.net/optout_check?callback=Krux.ns.macmillan.kxjsonp_optOutCheck"></script><script type="text/javascript" src="bd339c69-af54-4a21-b4f1-654bcfcd83ca_002"></script><script type="text/javascript" src="bd339c69-af54-4a21-b4f1-654bcfcd83ca"></script><script async="" src="controltag.js"></script><script src="aC6.es"></script><script type="text/javascript" async="" src="analytics.js"></script><script type="text/javascript" async="" src="KDqylSLE.js"></script><script type="text/javascript" async="" src="MathJax.js"></script><script async="" src="gtm.js"></script><script>(function(H){H.className=H.className.replace(/\bno-js\b/,'js')})(document.documentElement)</script>
        <link rel="stylesheet" media="screen" href="basic-0600efa430.css">
<link rel="stylesheet" media="print" href="print-7680b1a80b.css">

<link rel="stylesheet" id="js-mustard" href="main-b2b35c6efd.css" media="only screen and (-webkit-min-device-pixel-ratio:0) and (min-color-index:0), (-ms-high-contrast: none), only all and (min--moz-device-pixel-ratio:0) and (min-resolution: 3e1dpcm)">

        

        <script>
            window.airbrakeErrors = [];
            window.onerror = function(error) {window.airbrakeErrors.push(error);}
        </script>

        
    <script type="text/javascript">
        config = {
            env: 'live',
            site: 'frontiersinzoology.biomedcentral.com',
            siteWithPath: 'frontiersinzoology.biomedcentral.com' + window.location.pathname,
            twitterHashtag: '',
            cmsPrefix: 'https://studio-cms.springernature.com/studio/',
            
            doi: '10.1186/1742-9994-8-20',
            figshareScriptUrl: 'https://widgets.figshare.com/static/figshare.js',
            publisherBrand: 'BioMed Central',
            mustardcut: false
        };
    </script>

        <!-- SpringerLink's event tracker, as per: https://github.com/springernature/springerlink-event-tracker -->

<script type="text/javascript">
document.addEventListener('dataLayerCreated', function() {
    var script = document.createElement('script');
    script.id = 'springerlink-event-tracker';
    script.src = 'https://event-tracker.springernature.com/dist/eventTracker.js';
    script.onload = function () {
        var dL;
        var doi;
        var imprint;
        var bpids;
        var publisher;
        var pageType;
        var eventObject;

        for (var i = 0; i < window.dataLayer.length; i++) {
            if (window.dataLayer[i].event === "dataLayerCreated") {
                dL = window.dataLayer[i];
            }
        }

        if (!dL || !dL.content.article || !dL.content.article.doi) { return false;}

        doi = dL.content.article.doi;
        imprint = dL.content.contentInfo.imprint;
        bpids = dL.session.authentication.authenticationID;
        pageType = dL.page.category.pageType;

        if (pageType) {
            pageType = pageType.charAt(0).toUpperCase() + pageType.substring(1);
        }

        eventObject = {'content_type': pageType, 'doi': doi};

        if (imprint === 'BioMed Central') {
            publisher = 'BMC';
        }
        else if (imprint === 'SpringerOpen') {
            publisher = 'SpringerOpen';
        }
        else {
            throw new Error('Event Tracker Error: The publisher has an unexpected value');
        }

        if (bpids.length > 0) {
            eventObject['business_partner_ids'] = bpids;
        }

        new EventTracker({'platform': publisher}).sendEvent('display', eventObject);
    };
    document.head.appendChild(script);
});
</script>

        <!--Polyfills CustomEvent constructor in IE. Allows us to use events to manage race conditions in client side js-->
<script>
    (function () {
        if (typeof window.CustomEvent === "function") { return false; } // If IE polyfill not needed just return

        function CustomEvent ( event, params ) {
            params = params || { bubbles: false, cancelable: false, detail: undefined };
            var evt = document.createEvent( 'CustomEvent' );
            evt.initCustomEvent( event, params.bubbles, params.cancelable, params.detail );
            return evt;
        }

        CustomEvent.prototype = window.Event.prototype;

        window.CustomEvent = CustomEvent;
    })();
</script>
        
    <script type="text/javascript">
        window.dataLayer = [{"content":{"article":{"doi":"10.1186/1742-9994-8-20","articleType":"Research","peerReviewType":"Closed","supplement":null,"keywords":"Dorylinae;Formicidae;introgression;microsatellites;mtDNA;gene flow"},"contentInfo":{"imprint":"BioMed Central","title":"Hybridization in East African swarm-raiding army ants","publishedAt":1313971200000,"publishedAtDate":"2011-08-22","author":["Daniel JC Kronauer","Marcell K Peters","Caspar Schöning","Jacobus J Boomsma"],"collection":[]},"attributes":{"deliveryPlatform":"oscar","template":"rebrand","cms":null,"copyright":{"creativeCommonsType":null,"openAccess":false},"environment":"live"},"journal":{"siteKey":"frontiersinzoology.biomedcentral.com","volume":"8","issue":"1","title":"Frontiers in Zoology","type":"AJs Life","journalID":12983,"gaCode":"UA-62062676-16","section":[]},"category":{"pmc":{"primarySubject":"Life Sciences"},"contentType":"Research","publishingSegment":"Zoology"}},"session":{"authentication":{"authenticationID":[]}},"version":"1.0.0","page":{"category":{"pageType":"article"}},"event":"dataLayerCreated"}];
        var event = new CustomEvent('dataLayerCreated');
        document.dispatchEvent(event);
    </script><script id="springerlink-event-tracker" src="eventTracker.asc"></script>


        <script>
            
            var mustardcutlink = document.getElementById('js-mustard');
            if (mustardcutlink && window.matchMedia && window.matchMedia(mustardcutlink.media).matches) {
                window.config.mustardcut = true;
            }
        </script>
        <script>
            (function(d){
                var config = window.config;
                if (config && config.mustardcut && config.publisherBrand && sessionStorage.fontsLoaded === 'true') {
                    if (window.config.publisherBrand === 'BioMed Central') {
                        d.style.setProperty('--font-family-sans', 'Europa');
                        d.style.setProperty('--font-family-serif', 'Noto Serif')
                    } else if (window.config.publisherBrand === 'SpringerOpen' || window.config.publisherBrand === 'Springer') {
                        d.style.setProperty('--font-family-sans', 'Source Sans Pro');
                    }
                }
            })(document.documentElement);
        </script>
        <script src="7d6f4b4f-77fb-43b9-b429-eb1e1eeac6aa.js" id="onetrust-bmc-control" charset="UTF-8"></script><script type="text/javascript" src="jquery_002.js"></script><link type="text/css" href="optanon.css" rel="stylesheet"><style>#optanon ul#optanon-menu li { background-color: #F8F8F8 !important }#optanon ul#optanon-menu li.menu-item-selected { background-color: #C5F6F9 !important }#optanon #optanon-popup-wrapper .optanon-white-button-middle { background-color: #1B3051 !important }.optanon-alert-box-wrapper .optanon-alert-box-button-middle { background-color: #1B3051 !important; border-color: #1B3051 !important; }#optanon #optanon-popup-wrapper .optanon-white-button-middle a { color: #ffffff !important }.optanon-alert-box-wrapper .optanon-alert-box-button-middle a { color: #ffffff !important }#optanon #optanon-popup-bottom { background-color: #FFFFFF !important }#optanon.modern #optanon-popup-top, #optanon.modern #optanon-popup-body-left-shading { background-color: #FFFFFF !important }.optanon-alert-box-wrapper { background-color:#EFF6FC !important }.optanon-alert-box-wrapper .optanon-alert-box-bg p { color:#333333 !important }</style>
        
    
        
    

        
    
        
            
                <!-- Google Tag Manager -->
                <script>
                    (function (w, d, s, l, i) {
                    w[l] = w[l] || [];
                    w[l].push({'gtm.start': new Date().getTime(), event: 'gtm.js'});
                    var f = d.getElementsByTagName(s)[0],
                            j = d.createElement(s),
                            dl = l != 'dataLayer' ? '&l=' + l : '';
                    j.async = true;
                    j.src = 'https://www.googletagmanager.com/gtm.js?id=' + i + dl;
                    f.parentNode.insertBefore(j, f);
                })(window, document, 'script', 'dataLayer', 'GTM-TDGJHK');
                </script>
                <!-- End Google Tag Manager -->
            
        
    


        
    
    <link rel="canonical" href="https://frontiersinzoology.biomedcentral.com/articles/10.1186/1742-9994-8-20">
    

        
        
        <meta property="og:url" content="https://frontiersinzoology.biomedcentral.com/articles/10.1186/1742-9994-8-20">
        <meta property="og:type" content="article">
        <meta property="og:site_name" content="Frontiers in Zoology">
        <meta property="og:title" content="Hybridization in East African swarm-raiding army ants">
        <meta property="og:description" content="Hybridization can have complex effects on evolutionary dynamics in ants because of the combination of haplodiploid sex-determination and eusociality. While hybrid non-reproductive workers have been found in a range of species, examples of gene-flow via hybrid queens and males are rare. We studied hybridization in East African army ants (Dorylus subgenus Anomma) using morphology, mitochondrial DNA sequences, and nuclear microsatellites. While the mitochondrial phylogeny had a strong geographic signal, different species were not recovered as monophyletic. At our main study site at Kakamega Forest, a mitochondrial haplotype was shared between a &quot;Dorylus molestus-like&quot; and a &quot;Dorylus wilverthi-like&quot; form. This pattern is best explained by introgression following hybridization between D. molestus and D. wilverthi. Microsatellite data from workers showed that the two morphological forms correspond to two distinct genetic clusters, with a significant proportion of individuals being classified as hybrids. We conclude that hybridization and gene-flow between the two army ant species D. molestus and D. wilverthi has occurred, and that mating between the two forms continues to regularly produce hybrid workers. Hybridization is particularly surprising in army ants because workers have control over which males are allowed to mate with a young virgin queen inside the colony.">
        <meta property="og:image" content="https://images.springer.com/sgw/journals/medium/12983.jpg">
    
    <script type="text/javascript" src="jquery.js"></script><script src="figshare.js"></script><script src="embed.js" data-timestamp="1551088251540"></script><link rel="preload" href="integrator.js"><script type="text/javascript" src="integrator.js"></script><link rel="preload" href="integrator_002.js"><script type="text/javascript" src="integrator_002.js"></script><script src="pubads_impl_313.js" async=""></script><style type="text/css">.MathJax_Hover_Frame {border-radius: .25em; -webkit-border-radius: .25em; -moz-border-radius: .25em; -khtml-border-radius: .25em; box-shadow: 0px 0px 15px #83A; -webkit-box-shadow: 0px 0px 15px #83A; -moz-box-shadow: 0px 0px 15px #83A; -khtml-box-shadow: 0px 0px 15px #83A; border: 1px solid #A6D ! important; display: inline-block; position: absolute}
.MathJax_Menu_Button .MathJax_Hover_Arrow {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 4px; -webkit-border-radius: 4px; -moz-border-radius: 4px; -khtml-border-radius: 4px; font-family: 'Courier New',Courier; font-size: 9px; color: #F0F0F0}
.MathJax_Menu_Button .MathJax_Hover_Arrow span {display: block; background-color: #AAA; border: 1px solid; border-radius: 3px; line-height: 0; padding: 4px}
.MathJax_Hover_Arrow:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_Hover_Arrow:hover span {background-color: #CCC!important}
</style><style type="text/css">#MathJax_About {position: fixed; left: 50%; width: auto; text-align: center; border: 3px outset; padding: 1em 2em; background-color: #DDDDDD; color: black; cursor: default; font-family: message-box; font-size: 120%; font-style: normal; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 15px; -webkit-border-radius: 15px; -moz-border-radius: 15px; -khtml-border-radius: 15px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_About.MathJax_MousePost {outline: none}
.MathJax_Menu {position: absolute; background-color: white; color: black; width: auto; padding: 5px 0px; border: 1px solid #CCCCCC; margin: 0; cursor: default; font: menu; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 5px; -webkit-border-radius: 5px; -moz-border-radius: 5px; -khtml-border-radius: 5px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
.MathJax_MenuItem {padding: 1px 2em; background: transparent}
.MathJax_MenuArrow {position: absolute; right: .5em; padding-top: .25em; color: #666666; font-size: .75em}
.MathJax_MenuActive .MathJax_MenuArrow {color: white}
.MathJax_MenuArrow.RTL {left: .5em; right: auto}
.MathJax_MenuCheck {position: absolute; left: .7em}
.MathJax_MenuCheck.RTL {right: .7em; left: auto}
.MathJax_MenuRadioCheck {position: absolute; left: .7em}
.MathJax_MenuRadioCheck.RTL {right: .7em; left: auto}
.MathJax_MenuLabel {padding: 1px 2em 3px 1.33em; font-style: italic}
.MathJax_MenuRule {border-top: 1px solid #DDDDDD; margin: 4px 3px}
.MathJax_MenuDisabled {color: GrayText}
.MathJax_MenuActive {background-color: #606872; color: white}
.MathJax_MenuDisabled:focus, .MathJax_MenuLabel:focus {background-color: #E8E8E8}
.MathJax_ContextMenu:focus {outline: none}
.MathJax_ContextMenu .MathJax_MenuItem:focus {outline: none}
#MathJax_AboutClose {top: .2em; right: .2em}
.MathJax_Menu .MathJax_MenuClose {top: -10px; left: -10px}
.MathJax_MenuClose {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; font-family: 'Courier New',Courier; font-size: 24px; color: #F0F0F0}
.MathJax_MenuClose span {display: block; background-color: #AAA; border: 1.5px solid; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; line-height: 0; padding: 8px 0 6px}
.MathJax_MenuClose:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_MenuClose:hover span {background-color: #CCC!important}
.MathJax_MenuClose:hover:focus {outline: none}
</style><style type="text/css">.MJX_Assistive_MathML {position: absolute!important; top: 0; left: 0; clip: rect(1px, 1px, 1px, 1px); padding: 1px 0 0 0!important; border: 0!important; height: 1px!important; width: 1px!important; overflow: hidden!important; display: block!important; -webkit-touch-callout: none; -webkit-user-select: none; -khtml-user-select: none; -moz-user-select: none; -ms-user-select: none; user-select: none}
.MJX_Assistive_MathML.MJX_Assistive_MathML_Block {width: 100%!important}
</style><style type="text/css">#MathJax_Zoom {position: absolute; background-color: #F0F0F0; overflow: auto; display: block; z-index: 301; padding: .5em; border: 1px solid black; margin: 0; font-weight: normal; font-style: normal; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; -webkit-box-sizing: content-box; -moz-box-sizing: content-box; box-sizing: content-box; box-shadow: 5px 5px 15px #AAAAAA; -webkit-box-shadow: 5px 5px 15px #AAAAAA; -moz-box-shadow: 5px 5px 15px #AAAAAA; -khtml-box-shadow: 5px 5px 15px #AAAAAA; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_ZoomOverlay {position: absolute; left: 0; top: 0; z-index: 300; display: inline-block; width: 100%; height: 100%; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
#MathJax_ZoomFrame {position: relative; display: inline-block; height: 0; width: 0}
#MathJax_ZoomEventTrap {position: absolute; left: 0; top: 0; z-index: 302; display: inline-block; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
</style><style type="text/css">.MathJax_Preview {color: #888}
#MathJax_Message {position: fixed; left: 1px; bottom: 2px; background-color: #E6E6E6; border: 1px solid #959595; margin: 0px; padding: 2px 8px; z-index: 102; color: black; font-size: 80%; width: auto; white-space: nowrap}
#MathJax_MSIE_Frame {position: absolute; top: 0; left: 0; width: 0px; z-index: 101; border: 0px; margin: 0px; padding: 0px}
.MathJax_Error {color: #CC0000; font-style: italic}
</style><style type="text/css">.MJXp-script {font-size: .8em}
.MJXp-right {-webkit-transform-origin: right; -moz-transform-origin: right; -ms-transform-origin: right; -o-transform-origin: right; transform-origin: right}
.MJXp-bold {font-weight: bold}
.MJXp-italic {font-style: italic}
.MJXp-scr {font-family: MathJax_Script,'Times New Roman',Times,STIXGeneral,serif}
.MJXp-frak {font-family: MathJax_Fraktur,'Times New Roman',Times,STIXGeneral,serif}
.MJXp-sf {font-family: MathJax_SansSerif,'Times New Roman',Times,STIXGeneral,serif}
.MJXp-cal {font-family: MathJax_Caligraphic,'Times New Roman',Times,STIXGeneral,serif}
.MJXp-mono {font-family: MathJax_Typewriter,'Times New Roman',Times,STIXGeneral,serif}
.MJXp-largeop {font-size: 150%}
.MJXp-largeop.MJXp-int {vertical-align: -.2em}
.MJXp-math {display: inline-block; line-height: 1.2; text-indent: 0; font-family: 'Times New Roman',Times,STIXGeneral,serif; white-space: nowrap; border-collapse: collapse}
.MJXp-display {display: block; text-align: center; margin: 1em 0}
.MJXp-math span {display: inline-block}
.MJXp-box {display: block!important; text-align: center}
.MJXp-box:after {content: " "}
.MJXp-rule {display: block!important; margin-top: .1em}
.MJXp-char {display: block!important}
.MJXp-mo {margin: 0 .15em}
.MJXp-mfrac {margin: 0 .125em; vertical-align: .25em}
.MJXp-denom {display: inline-table!important; width: 100%}
.MJXp-denom > * {display: table-row!important}
.MJXp-surd {vertical-align: top}
.MJXp-surd > * {display: block!important}
.MJXp-script-box > *  {display: table!important; height: 50%}
.MJXp-script-box > * > * {display: table-cell!important; vertical-align: top}
.MJXp-script-box > *:last-child > * {vertical-align: bottom}
.MJXp-script-box > * > * > * {display: block!important}
.MJXp-mphantom {visibility: hidden}
.MJXp-munderover, .MJXp-munder {display: inline-table!important}
.MJXp-over {display: inline-block!important; text-align: center}
.MJXp-over > * {display: block!important}
.MJXp-munderover > *, .MJXp-munder > * {display: table-row!important}
.MJXp-mtable {vertical-align: .25em; margin: 0 .125em}
.MJXp-mtable > * {display: inline-table!important; vertical-align: middle}
.MJXp-mtr {display: table-row!important}
.MJXp-mtd {display: table-cell!important; text-align: center; padding: .5em 0 0 .5em}
.MJXp-mtr > .MJXp-mtd:first-child {padding-left: 0}
.MJXp-mtr:first-child > .MJXp-mtd {padding-top: 0}
.MJXp-mlabeledtr {display: table-row!important}
.MJXp-mlabeledtr > .MJXp-mtd:first-child {padding-left: 0}
.MJXp-mlabeledtr:first-child > .MJXp-mtd {padding-top: 0}
.MJXp-merror {background-color: #FFFF88; color: #CC0000; border: 1px solid #CC0000; padding: 1px 3px; font-style: normal; font-size: 90%}
.MJXp-scale0 {-webkit-transform: scaleX(.0); -moz-transform: scaleX(.0); -ms-transform: scaleX(.0); -o-transform: scaleX(.0); transform: scaleX(.0)}
.MJXp-scale1 {-webkit-transform: scaleX(.1); -moz-transform: scaleX(.1); -ms-transform: scaleX(.1); -o-transform: scaleX(.1); transform: scaleX(.1)}
.MJXp-scale2 {-webkit-transform: scaleX(.2); -moz-transform: scaleX(.2); -ms-transform: scaleX(.2); -o-transform: scaleX(.2); transform: scaleX(.2)}
.MJXp-scale3 {-webkit-transform: scaleX(.3); -moz-transform: scaleX(.3); -ms-transform: scaleX(.3); -o-transform: scaleX(.3); transform: scaleX(.3)}
.MJXp-scale4 {-webkit-transform: scaleX(.4); -moz-transform: scaleX(.4); -ms-transform: scaleX(.4); -o-transform: scaleX(.4); transform: scaleX(.4)}
.MJXp-scale5 {-webkit-transform: scaleX(.5); -moz-transform: scaleX(.5); -ms-transform: scaleX(.5); -o-transform: scaleX(.5); transform: scaleX(.5)}
.MJXp-scale6 {-webkit-transform: scaleX(.6); -moz-transform: scaleX(.6); -ms-transform: scaleX(.6); -o-transform: scaleX(.6); transform: scaleX(.6)}
.MJXp-scale7 {-webkit-transform: scaleX(.7); -moz-transform: scaleX(.7); -ms-transform: scaleX(.7); -o-transform: scaleX(.7); transform: scaleX(.7)}
.MJXp-scale8 {-webkit-transform: scaleX(.8); -moz-transform: scaleX(.8); -ms-transform: scaleX(.8); -o-transform: scaleX(.8); transform: scaleX(.8)}
.MJXp-scale9 {-webkit-transform: scaleX(.9); -moz-transform: scaleX(.9); -ms-transform: scaleX(.9); -o-transform: scaleX(.9); transform: scaleX(.9)}
.MathJax_PHTML .noError {vertical-align: ; font-size: 90%; text-align: left; color: black; padding: 1px 3px; border: 1px solid}
</style><link rel="prefetch" href="https://c.disquscdn.com/next/embed/styles/lounge.7881ba4704e5c647ac74c94714fe89c0.css"><link rel="prefetch" href="https://c.disquscdn.com/next/embed/common.bundle.505b628fe4a369d7faa766dd8c23b076.js"><link rel="prefetch" href="https://c.disquscdn.com/next/embed/lounge.bundle.c9237ca4eec89ddb1320c66204dab595.js"><link rel="prefetch" href="https://disqus.com/next/config.js"><style type="text/css" id="qual_style-lzd"></style><style type="text/css" id="qual_style-lve"></style></head>

    <body class="journal journal-fulltext"><div id="MathJax_Message" style="display: none;"></div>
        
    
        
            
                <!-- Google Tag Manager (noscript) -->
                <noscript>
                    <iframe src="https://www.googletagmanager.com/ns.html?id=GTM-TDGJHK"
                            height="0" width="0" style="display:none;visibility:hidden"></iframe>
                </noscript>
                <!-- End Google Tag Manager (noscript) -->
            
        
    


        

        

    <div class="u-vh-full">
        <div class="u-visually-hidden" aria-hidden="true">
    
    <!--?xml version="1.0" encoding="UTF-8"?--><svg xmlns="http://www.w3.org/2000/svg"><defs><filter id="a" filterUnits="userSpaceOnUse" x="0" y="0" width="60.643" height="31.01"><feColorMatrix values="1 0 0 0 0 0 1 0 0 0 0 0 1 0 0 0 0 0 1 0"></feColorMatrix></filter></defs><symbol id="icon-arrow-down" viewBox="0 0 16 8"><path d="M8 6l6-6h2L8 8 0 0h2l6 6"></path></symbol><symbol id="icon-arrow-left-bullet" viewBox="0 0 8 16"><path d="M3 8l5 5v3L0 8l8-8v3L3 8z"></path></symbol><symbol id="icon-arrow-right-bullet" viewBox="0 0 8 16"><path d="M5 8L0 3V0l8 8-8 8v-3l5-5z"></path></symbol><symbol id="icon-arrow-right" viewBox="0 0 32 64"><path d="M0 0v64l32-32z"></path></symbol><symbol id="icon-close" viewBox="0 0 23.998 23.999"><path d="M20.998 0l-8.999 9L3 0 0 3l8.999 9L0 20.999l3 3L11.999 15l8.999 8.999 3-3L14.999 12l8.999-9z"></path></symbol><symbol id="icon-community" viewBox="0 0 60.643 31.01"><mask maskUnits="userSpaceOnUse" x="0" y="0" width="60.643" height="31.01" id="b"><path fill="#FFF" d="M0 31.01h60.643V0H0z" filter="url(#a)"></path></mask><path mask="url(#b)" d="M60.643 25.497L50.005 14.86a7.925 7.925 0 1 0-7.669 0l-6.847 6.847-1.334-1.334a7.925 7.925 0 0 0-3.834-14.861 7.925 7.925 0 0 0-3.834 14.861l-1.334 1.334-6.847-6.847a7.925 7.925 0 1 0-7.669 0L0 25.497h21.362L15.85 31.01h28.943l-5.514-5.513h21.364z"></path></symbol><symbol id="icon-download" viewBox="-301 390 9 14"><path d="M-301 395.6l4.5 5.1 4.5-5.1h-3V390h-3v5.6h-3zm0 6.5h9v1.9h-9z"></path></symbol><symbol id="icon-ext-link" viewBox="0 0 16 16"><path fill="#676767" d="M12.9 16H3.1C1.4 16 0 14.6 0 12.9V3.2C0 1.4 1.4 0 3.1 0h3.7v1H3.1C2 1 1 2 1 3.2v9.7C1 14 2 15 3.1 15h9.7c1.2 0 2.1-1 2.1-2.1V8.7h1v4.2c.1 1.7-1.3 3.1-3 3.1z"></path><path fill="#676767" d="M12.8 2.5l.7.7-9 8.9-.7-.7 9-8.9z"></path><path fill="#676767" d="M9.7 0L16 6.2V0z"></path></symbol><symbol id="icon-info-bordered" viewBox="470.812 270.868 18 18"><path d="M479.812 270.868c-4.972 0-9 4.029-9 9s4.028 9 9 9c4.971 0 9-4.029 9-9s-4.029-9-9-9zm0 16.875a7.875 7.875 0 0 1-7.875-7.875 7.875 7.875 0 1 1 15.75 0 7.875 7.875 0 0 1-7.875 7.875z"></path><path d="M479.284 279.586c.089-.238-.024-.361-.13-.361-.489 0-1.123 1.16-1.359 1.16-.093 0-.173-.092-.173-.174 0-.238.581-.801.751-.971.526-.506 1.217-.895 1.979-.895.567 0 1.174.346.703 1.639l-.952 2.604c-.079.199-.224.531-.224.746 0 .092.054.186.159.186.395 0 1.121-1.135 1.304-1.135.067 0 .158.086.158.199 0 .385-1.542 2.043-2.874 2.043-.477 0-.804-.225-.804-.732 0-.641.447-1.734.538-1.963l.924-2.346zm.727-3.41c0-.586.498-1.066 1.082-1.066.527 0 .91.357.91.906 0 .615-.501 1.068-1.098 1.068-.541-.002-.894-.361-.894-.908z"></path></symbol><symbol id="icon-opr" viewBox="0 0 18 18"><path d="M.9 2.5c-.2-.2-.4-.3-.6-.2-.3.2-.3.4-.3.6 0 .8 0 1.7.1 2.5 0 .5.3.7.7.6.9 0 1.7-.2 2.5-.3h.2c.5-.2.6-.5.3-.8-.2-.2-.4-.4-.6-.5-.1-.1-.2-.2-.4-.3.5-.6 1-1.1 1.7-1.5 4.6-3 10.9-.5 12 4.9.9 4.4-2.2 8.6-6.7 9.2-3.7.5-7.2-1.6-8.4-5.1 0-.1-.1-.2-.1-.3-.1-.3-.5-.5-.8-.4-.3.1-.5.4-.4.8.4 1.3 1.1 2.4 2 3.4 1.4 1.5 3.1 2.4 5.1 2.8 3 .5 5.6-.2 7.8-2.2 2.1-1.9 3-4.2 3-6.9-.1-4-3.1-7.6-7.1-8.4-3.3-.9-6.2 0-8.6 2.3-.2.1-.3.3-.5.5-.3-.2-.6-.5-.9-.7z"></path><path d="M13 4.7c-2.6 1.6-4.5 3.6-5.3 4.6L5.6 7.7l-.9.7L8.3 12c.6-1.6 2.6-4.6 4.9-6.8l-.2-.5z"></path></symbol><symbol id="icon-quotation" viewBox="0 0.333 55.932 35.619"><path d="M43.738.333L31.545 12.207v23.745h24.387V12.207H43.738zm-31.545 0L0 12.207v23.745h24.386V12.207H12.193z"></path></symbol><symbol id="icon-remove" viewBox="-296 388 18 18"><path d="M-291.7 396.1h9v2h-9z"></path><path d="M-287 405.5c-4.7 0-8.5-3.8-8.5-8.5s3.8-8.5 8.5-8.5 8.5 3.8 8.5 8.5-3.8 8.5-8.5 8.5zm0-16c-4.1 0-7.5 3.4-7.5 7.5s3.4 7.5 7.5 7.5 7.5-3.4 7.5-7.5-3.4-7.5-7.5-7.5z"></path></symbol><symbol id="icon-ribbon" viewBox="0 0 36 72"><path d="M0 0v72h.053L18 54.054 35.945 72H36V0H0zm25.8 11.4l3.001 3L15 28.2l-7.801-7.8 3-3L15 22.201 25.8 11.4z"></path></symbol><symbol id="icon-rss" viewBox="0 0 24 24"><g fill="#F26822"><ellipse cx="3.305" cy="20.702" rx="3.306" ry="3.298"></ellipse><path d="M15.978 24h-4.684c0-6.224-5.057-11.27-11.294-11.27V8.058c8.824 0 15.978 7.137 15.978 15.942z"></path><path d="M19.2 23.95C19.2 13.366 10.604 4.79 0 4.79V0c13.255 0 24 10.723 24 23.95h-4.8z"></path></g></symbol><symbol id="icon-search" viewBox="0 0 16 16"><path d="M6.276 12.257C2.815 12.257 0 9.508 0 6.128 0 2.749 2.815 0 6.276 0c3.462 0 6.277 2.749 6.277 6.128.002 3.38-2.815 6.129-6.277 6.129zm0-10.772c-2.622 0-4.755 2.083-4.755 4.643 0 2.562 2.133 4.644 4.755 4.644 2.623 0 4.756-2.082 4.756-4.644 0-2.56-2.134-4.643-4.756-4.643z"></path><path d="M16 13.751L13.641 16l-4.875-4.82 2.36-2.248L16 13.751z"></path></symbol><symbol id="icon-speech-bubble" viewBox="0 0 40 54"><path d="M0 0v40h6.667v13.334L20 40h20V0z"></path></symbol><symbol id="icon-error" viewBox="2.002 0 14 14"><path d="M16.002 11.949L13.951 14 9.002 9.051 4.053 14l-2.051-2.051L6.951 7 2.002 2.05 4.053 0l4.949 4.95L13.951 0l2.051 2.049L11.053 7l4.949 4.949z"></path></symbol><symbol id="icon-info" viewBox="5.502 0 7 14"><path d="M8.165 5.923c.146-.384-.04-.577-.209-.577-.783 0-1.798 1.856-2.177 1.856-.15 0-.277-.147-.277-.279 0-.383.933-1.282 1.203-1.557.846-.811 1.949-1.434 3.174-1.434.908 0 1.879.555 1.125 2.63l-1.526 4.167c-.126.32-.358.854-.358 1.198 0 .148.088.298.255.298.632 0 1.796-1.818 2.091-1.818.105 0 .252.134.252.321C11.717 11.345 9.245 14 7.11 14c-.763 0-1.289-.361-1.289-1.176 0-1.027.717-2.778.865-3.145l1.479-3.756zm1.144-4.211C9.309.771 10.111 0 11.045 0c.845 0 1.457.577 1.457 1.456 0 .983-.804 1.709-1.757 1.709-.866-.001-1.436-.578-1.436-1.453z"></path></symbol><symbol id="icon-success" viewBox="0 0 16 14"><path d="M15.592 0c-4.867 2.984-8.4 6.751-9.987 8.641L1.717 5.595 0 6.979l6.717 6.832c1.157-2.961 4.817-8.748 9.287-12.86L15.592 0z"></path></symbol><symbol id="icon-warning" viewBox="7.002 0 2.789 14"><path d="M7.002 11.211h2.789V14H7.002v-2.789zM7.002 0v3.436l.741 5.326h1.326l.723-5.326V0h-2.79z"></path></symbol><symbol id="icon-facebook-bordered" viewBox="463.812 263.868 32 32"><path d="M479.812 263.868c-8.837 0-16 7.163-16 16s7.163 16 16 16 16-7.163 16-16-7.163-16-16-16zm0 30c-7.732 0-14-6.269-14-14s6.268-14 14-14 14 6.269 14 14-6.267 14-14 14z"></path><path d="M483.025 280.48l.32-2.477h-2.453v-1.582c0-.715.199-1.207 1.227-1.207h1.311v-2.213a17.753 17.753 0 0 0-1.907-.098c-1.894 0-3.186 1.154-3.186 3.271V278h-2.142v2.477h2.142v6.354h2.557v-6.354l2.131.003z"></path></symbol><symbol id="icon-facebook" viewBox="0 0 24 24"><circle fill="#4D67A4" cx="12" cy="12" r="12"></circle><path fill="#FFF" d="M8.396 10.143h2.137V8.165s-.092-1.292.892-2.274c.979-.979 2.298-.887 4.177-.724v2.21h-1.387s-.586-.013-.861.303c-.271.315-.24.764-.24.875v1.588h2.41l-.311 2.458h-2.116v6.336h-2.56v-6.345H8.396v-2.449z"></path></symbol><symbol id="icon-gplus-bordered" viewBox="463.812 263.868 32 32"><path d="M479.812 263.868c-8.838 0-16 7.163-16 16s7.162 16 16 16c8.837 0 16-7.163 16-16s-7.163-16-16-16zm0 30c-7.732 0-14-6.269-14-14s6.268-14 14-14c7.731 0 14 6.269 14 14s-6.267 14-14 14z"></path><path d="M480.5 273.117c.123.076.26.185.416.334.146.152.289.343.432.562.135.211.258.459.359.746.086.286.129.619.129 1.004-.012.7-.164 1.262-.465 1.683a5.374 5.374 0 0 1-.461.562c-.186.176-.375.35-.58.527a2.65 2.65 0 0 0-.322.41.908.908 0 0 0-.176.553c0 .205.059.375.18.508.102.129.199.238.299.336l.662.545a6.48 6.48 0 0 1 1.086 1.119c.293.415.445.955.459 1.625 0 .949-.42 1.789-1.256 2.521-.869.76-2.123 1.15-3.762 1.177-1.373-.015-2.4-.308-3.074-.879-.682-.534-1.023-1.172-1.023-1.917 0-.359.111-.766.336-1.211.215-.443.602-.837 1.172-1.174a6.126 6.126 0 0 1 1.998-.725 15.415 15.415 0 0 1 1.723-.173 3.996 3.996 0 0 1-.377-.596 1.347 1.347 0 0 1-.193-.74c0-.174.023-.317.074-.433a8.83 8.83 0 0 1 .121-.338 5.973 5.973 0 0 1-.633.035c-1.043-.012-1.838-.339-2.381-.979-.574-.603-.861-1.3-.861-2.097 0-.961.404-1.832 1.217-2.616.559-.46 1.135-.757 1.736-.898a8.494 8.494 0 0 1 1.67-.182h3.916l-1.209.704h-1.207v.005l-.005.002zm.754 11.276c0-.498-.162-.932-.484-1.302-.346-.348-.881-.778-1.609-1.29a4.045 4.045 0 0 0-.439-.02c-.102-.016-.354 0-.768.039a8.141 8.141 0 0 0-1.244.275c-.1.036-.24.096-.42.17a2.713 2.713 0 0 0-.553.355 1.985 1.985 0 0 0-.459.592c-.145.25-.215.551-.215.901 0 .687.311 1.253.932 1.698.596.444 1.404.68 2.432.688.928-.014 1.629-.217 2.117-.612.476-.385.71-.886.71-1.494zm-2.729-5.743a1.843 1.843 0 0 0 1.289-.556c.164-.249.271-.506.318-.767a7.32 7.32 0 0 0 .041-.659c0-.772-.195-1.552-.594-2.336a2.611 2.611 0 0 0-.73-.921 1.894 1.894 0 0 0-1.064-.352 1.723 1.723 0 0 0-1.324.64c-.299.437-.438.92-.428 1.457 0 .709.209 1.444.623 2.211.201.356.459.659.775.912.323.248.686.371 1.094.371zm9.698.077h-2.055v-2.056h-.996v2.056h-2.053v.998h2.053v2.052h.996v-2.052h2.055z"></path></symbol><symbol id="icon-gplus" viewBox="0 0 24 24"><circle fill="#D84A36" cx="12" cy="12" r="12"></circle><g fill="#FFF"><path d="M12.012 12.111c-.304-.33-.07-.842.155-1.078.226-.241.639-.56.829-.74.193-.184.66-.605.674-1.868 0 0 .112-1.139-1.127-2.06h.977l1.038-.593h-3.507s-2.083-.067-3.085 1.207c-1 1.276-.966 2.871.114 3.77 1.08.898 2.371.648 2.371.648s-.197.422-.163.775c.036.353.421.849.504.965 0 0-2.386-.011-3.551 1.033-1.165 1.053-1.29 3.097 1.021 3.803 2.312.707 5.303-.055 5.906-2.277 0 0 .262-.908-.189-1.683-.459-.775-1.663-1.571-1.967-1.902zm-2.555-1.83c-.744-1-1.308-2.953 0-3.783 0 0 .955-.579 1.801.266.846.846 1.082 2.277.928 3.003-.15.727-.563.897-.739 1.008-.185.111-1.249.504-1.99-.494zm3.668 5.481c-.035.393-.083 1.254-1.491 1.664-1.409.408-3.826-.264-3.757-1.914 0 0 .006-.698.449-1.089.445-.396 1.094-.794 3.029-.819 0 0 .643.473 1.072.809.427.332.733.952.698 1.349z"></path><path d="M16.323 11.055V9.803h-.752v1.252h-1.255v.748h1.255v1.256h.752v-1.256h1.248v-.748z"></path></g></symbol><symbol id="icon-linkedin" viewBox="0 0 24 24"><circle fill="#0178B5" cx="12" cy="12" r="12"></circle><g fill="#FFF"><circle cx="8.022" cy="8.043" r="1.256"></circle><path d="M6.929 10.246h2.17v6.967h-2.17zm3.533 6.967h2.157v-3.594s-.078-1.627 1.219-1.627c1.301 0 1.211 1.231 1.211 1.635v3.586h2.183v-3.775s.129-2.745-1.472-3.191c-1.605-.445-2.661.071-3.219.985v-.985h-2.079v6.966z"></path></g></symbol><symbol id="icon-reddit" viewBox="0 0 24 24"><circle fill="#BCBCBC" cx="12" cy="12" r="12"></circle><path fill="#FFF" d="M4.661 9.741c.941 0 1.703.761 1.703 1.704 0 .938-.762 1.705-1.703 1.705s-1.704-.767-1.704-1.705c0-.943.763-1.704 1.704-1.704zm13.844 0a1.704 1.704 0 1 1 .001 3.409 1.704 1.704 0 0 1-.001-3.409z"></path><path fill="#FFF" d="M11.736 8.732c4.285 0 7.762 2.283 7.762 5.104 0 2.812-3.477 5.1-7.762 5.1-4.288 0-7.762-2.285-7.762-5.1.001-2.82 3.474-5.104 7.762-5.104z"></path><path fill="#010101" d="M11.736 19.262c-4.461 0-8.088-2.437-8.088-5.426 0-2.994 3.626-5.431 8.088-5.431 4.457 0 8.087 2.437 8.087 5.431 0 2.989-3.63 5.426-8.087 5.426zm0-10.205c-4.104 0-7.438 2.145-7.438 4.779 0 2.633 3.334 4.773 7.438 4.773 4.098 0 7.437-2.142 7.437-4.773 0-2.636-3.339-4.779-7.437-4.779z"></path><circle fill="#FFF" cx="18.014" cy="5.765" r="1.385"></circle><path fill="#010101" d="M18.014 7.456a1.694 1.694 0 0 1-1.695-1.691c0-.934.764-1.694 1.695-1.694s1.693.762 1.693 1.694c0 .934-.762 1.691-1.693 1.691zm0-2.734c-.574 0-1.043.469-1.043 1.043s.469 1.041 1.043 1.041a1.042 1.042 0 0 0 0-2.084zM3.818 13.275a2.068 2.068 0 0 1-1.068-1.811 2.074 2.074 0 0 1 3.538-1.466l-.458.462a1.412 1.412 0 0 0-1.007-.42 1.424 1.424 0 0 0-.688 2.669l-.317.566zm15.866 0l-.316-.566a1.43 1.43 0 0 0 .73-1.245c0-.784-.635-1.424-1.42-1.424-.381 0-.736.149-1.008.42l-.459-.462a2.074 2.074 0 1 1 2.473 3.277z"></path><path fill="#F04B23" d="M9.165 11.604c.704 0 1.272.564 1.272 1.27a1.273 1.273 0 1 1-1.272-1.27zm5.321 0a1.274 1.274 0 1 1-1.275 1.27c0-.704.57-1.27 1.275-1.27z"></path><path fill="#010101" d="M11.79 8.915a.327.327 0 0 1-.307-.44l1.472-4.181 3.616.862c.177.044.283.22.238.395a.32.32 0 0 1-.391.239l-3.045-.729-1.275 3.632a.334.334 0 0 1-.308.222zm-.069 8.376c-2.076 0-2.908-.939-2.943-.982a.332.332 0 0 1 .035-.461.334.334 0 0 1 .459.033c.019.021.713.758 2.449.758 1.766 0 2.539-.763 2.549-.767a.324.324 0 1 1 .469.446c-.04.043-.954.973-3.018.973z"></path></symbol><symbol id="icon-twitter-bordered" viewBox="463.812 263.868 32 32"><path d="M486.416 276.191a5.622 5.622 0 0 1-1.554.429 2.718 2.718 0 0 0 1.19-1.502 5.456 5.456 0 0 1-1.72.657 2.71 2.71 0 0 0-1.979-.854 2.711 2.711 0 0 0-2.642 3.326 7.681 7.681 0 0 1-5.586-2.831 2.714 2.714 0 0 0 .839 3.618 2.748 2.748 0 0 1-1.227-.339v.031a2.71 2.71 0 0 0 2.174 2.656 2.735 2.735 0 0 1-1.229.049 2.726 2.726 0 0 0 2.531 1.883 5.442 5.442 0 0 1-4.01 1.123 7.672 7.672 0 0 0 4.155 1.215c4.983 0 7.71-4.129 7.71-7.711 0-.115-.004-.232-.006-.351a5.41 5.41 0 0 0 1.354-1.399z"></path><path d="M479.812 263.868c-8.837 0-16 7.163-16 16s7.163 16 16 16 16-7.163 16-16-7.163-16-16-16zm0 30c-7.732 0-14-6.269-14-14s6.268-14 14-14 14 6.269 14 14-6.267 14-14 14z"></path></symbol><symbol id="icon-twitter" viewBox="0 0 24 24"><circle fill="#26A7DF" cx="12" cy="12" r="12"></circle><path fill="#FFF" d="M5.903 6.768s2.176 2.883 5.953 3.082c0 0-.51-1.702 1.058-3.004 1.568-1.305 3.508-.2 3.879.292 0 0 1.104-.172 1.875-.691 0 0-.252.995-1.18 1.594 0 0 1.086-.146 1.596-.439 0 0-.533.853-1.447 1.503 0 0 .381 3.595-2.709 6.57-3.09 2.971-8.176 2.352-10.012.797 0 0 2.495.277 4.249-1.219 0 0-2.074-.037-2.671-1.973 0 0 1.036.08 1.215-.119 0 0-2.232-.539-2.254-2.873 0 0 .68.339 1.297.359 0-.001-2.153-1.557-.849-3.879z"></path></symbol><symbol id="icon-weibo-bordered" viewBox="463.812 263.868 32 32"><path d="M479.812 263.868c-8.838 0-16 7.163-16 16s7.162 16 16 16c8.837 0 16-7.163 16-16s-7.163-16-16-16zm0 30c-7.732 0-14-6.269-14-14s6.268-14 14-14c7.731 0 14 6.269 14 14s-6.267 14-14 14z"></path><path d="M478.552 285.348c-2.616.261-4.876-.926-5.044-2.649-.167-1.722 1.814-3.33 4.433-3.588 2.609-.263 4.871.926 5.041 2.647.165 1.721-1.818 3.331-4.43 3.59m5.23-5.718c-.226-.065-.374-.109-.259-.403.25-.639.276-1.188.005-1.581-.515-.734-1.915-.693-3.521-.021 0 0-.508.224-.378-.181.247-.798.209-1.468-.178-1.852-.87-.878-3.194.032-5.183 2.027-1.489 1.494-2.357 3.082-2.357 4.453 0 2.619 3.354 4.213 6.631 4.213 4.297 0 7.154-2.504 7.154-4.493.001-1.198-1.007-1.881-1.914-2.162"></path><path fill="#020202" d="M486.637 274.833a4.176 4.176 0 0 0-3.982-1.291.608.608 0 0 0-.465.72.604.604 0 0 0 .72.466 2.968 2.968 0 0 1 2.827.92 3 3 0 0 1 .625 2.918.602.602 0 0 0 .39.762.603.603 0 0 0 .763-.391v-.001a4.218 4.218 0 0 0-.878-4.103"></path><path fill="#020202" d="M485.041 276.276a2.037 2.037 0 0 0-1.938-.63.518.518 0 0 0-.396.621.517.517 0 0 0 .617.398c.336-.071.702.03.947.307s.312.649.207.979a.52.52 0 0 0 .336.654.523.523 0 0 0 .657-.336 2.038 2.038 0 0 0-.43-1.993m-6.347 5.951c-.09.156-.293.233-.451.166-.151-.062-.204-.235-.115-.389.093-.155.284-.229.44-.168.157.056.214.235.126.391m-.832 1.074c-.253.405-.795.58-1.202.396-.403-.186-.521-.655-.27-1.051.248-.39.771-.566 1.176-.393.413.17.543.636.296 1.048m.95-2.864c-1.244-.326-2.65.294-3.19 1.396-.553 1.119-.021 2.369 1.236 2.775 1.303.42 2.84-.225 3.374-1.436.526-1.183-.132-2.402-1.42-2.735"></path></symbol><symbol id="icon-weibo" viewBox="4 4 24 24"><circle fill="#C8E8F9" cx="16" cy="16" r="12"></circle><path fill="#FFF" d="M9.098 18.194c0 1.981 2.574 3.593 5.757 3.593 3.178 0 5.756-1.611 5.756-3.593 0-1.989-2.578-3.601-5.756-3.601-3.183.001-5.757 1.612-5.757 3.601"></path><path fill="#DF0A21" d="M14.991 21.496c-2.817.278-5.244-.996-5.428-2.85-.183-1.855 1.95-3.586 4.767-3.862 2.813-.278 5.243.996 5.428 2.849.18 1.855-1.957 3.584-4.767 3.863m5.628-6.155c-.24-.067-.404-.116-.279-.432.27-.688.299-1.278.004-1.699-.551-.791-2.062-.748-3.789-.022 0-.001-.548.241-.406-.191.266-.859.224-1.577-.191-1.993-.937-.942-3.437.037-5.577 2.18-1.602 1.61-2.533 3.317-2.533 4.789 0 2.817 3.606 4.533 7.13 4.533 4.626 0 7.701-2.696 7.701-4.834.001-1.292-1.085-2.027-2.06-2.331"></path><path fill="#F4992C" d="M23.689 10.182a4.492 4.492 0 0 0-4.283-1.39.651.651 0 1 0 .269 1.275 3.2 3.2 0 0 1 3.045.99 3.224 3.224 0 0 1 .672 3.138.647.647 0 0 0 .418.823.647.647 0 0 0 .816-.421v-.003a4.51 4.51 0 0 0-.937-4.412"></path><path fill="#F4992C" d="M21.973 11.736a2.183 2.183 0 0 0-2.086-.679.56.56 0 0 0-.428.667.554.554 0 0 0 .662.43v.001c.361-.075.754.034 1.018.33.266.299.34.698.227 1.054a.563.563 0 0 0 .359.706.563.563 0 0 0 .709-.362 2.2 2.2 0 0 0-.461-2.147"></path><path fill="#13110C" d="M15.143 18.139c-.1.17-.315.251-.483.179-.168-.07-.222-.255-.128-.42.097-.167.311-.248.479-.182.165.062.228.252.132.423m-.896 1.155c-.271.434-.854.625-1.295.425-.433-.195-.561-.707-.285-1.128.269-.426.832-.609 1.265-.427.443.183.58.688.315 1.13m1.024-3.084c-1.34-.351-2.852.32-3.434 1.506-.597 1.206-.021 2.546 1.33 2.984 1.402.453 3.055-.243 3.629-1.543.569-1.275-.142-2.587-1.525-2.947"></path></symbol></svg>
</div>

        <a class="u-visually-hidden u-visually-hidden-focus" href="#main-content">
    <span class="c-banner c-banner--info">Skip to content</span>
</a>

        
            
    <div class="adsbox c-ad c-ad--LB1">
        <div class="c-ad__inner">
            <p class="c-ad__label">Advertisement</p>
            <div id="div-gpt-ad-LB1" data-gpt-unitpath="/270604982/bmc/frontiersinzoology/articles" data-gpt-sizes="728x90,970x90" data-gpt-targeting="pos=LB1;doi=10.1186/1742-9994-8-20;kwrd=Dorylinae,Formicidae,introgression,microsatellites,mtDNA,gene flow;pmc=L25007,L19023,L19031,L33030;">
                <noscript>
                    <a href="//pubads.g.doubleclick.net/gampad/jump?iu=/270604982/bmc/frontiersinzoology/articles&amp;sz=728x90,970x90&amp;pos=LB1&amp;doi=10.1186/1742-9994-8-20&amp;kwrd=Dorylinae,Formicidae,introgression,microsatellites,mtDNA,gene flow&amp;pmc=L25007,L19023,L19031,L33030&amp;">
                        <img data-test="gpt-advert-fallback-img"
                             src="//pubads.g.doubleclick.net/gampad/ad?iu=/270604982/bmc/frontiersinzoology/articles&amp;sz=728x90,970x90&amp;pos=LB1&amp;doi=10.1186/1742-9994-8-20&amp;kwrd=Dorylinae,Formicidae,introgression,microsatellites,mtDNA,gene flow&amp;pmc=L25007,L19023,L19031,L33030&amp;"
                             alt="Advertisement"
                             width="728"
                             height="90">
                    </a>
                </noscript>
            <div id="google_ads_iframe_/270604982/bmc/frontiersinzoology/articles_0__container__" style="border: 0pt none; width: 728px; height: 90px;"></div></div>
        </div>
    </div>

        
         
    <div id="membership-message-loader-desktop" class="placeholder" data-placeholder="/placeholder/v1/membership/message"></div>


<header class="c-popup-search" id="top" data-test="publisher-header">
    <nav class="c-navbar">
        <div class="u-container">
            
            <div class="basic-white-logo c-navbar__logo u-margin-right-xxl" itemscope="" itemtype="http://schema.org/Organization" data-test="navbar-logo-header">
                <link itemprop="parentOrganization" href="#parentOrganization">
                <div class="c-logo">
    <a href="https://www.biomedcentral.com/" itemprop="url">
        <img src="logo-bmc-white-aj-e3a70ae848.svg" alt="BMC" itemprop="logo">
        <div class="c-logo__strapline">
            <img src="logo-bmc-white-strapline-sn-e7b1d6e67f.svg" alt="Part of Springer Nature">
        </div>
    </a>
</div>
            </div>
            <button type="button" class="c-navbar__link c-dropdown js-publisher-menu-button u-hidden-at-lg" id="publisher-header-menu-button" data-toggle="collapse" data-test="publisher-header-menu-toggle" data-toggle-linked="publisher-header" data-target="publisher-header-search publisher-header-menu" aria-controls="publisher-header-menu publisher-header-search" aria-expanded="false">
                <span class="c-dropdown__toggle">Menu</span>
            </button>
            <div class="c-navbar__content c-collapse" id="publisher-header-menu">
                
    <ul class="c-navbar__nav" data-test="publisher-navigation">
        
        <li class="c-navbar__item">
            <a class="c-navbar__link" href="https://www.biomedcentral.com/journals">
                Explore journals
            </a>
        </li>
        
        <li class="c-navbar__item">
            <a class="c-navbar__link" href="https://www.biomedcentral.com/getpublished">
                Get published
            </a>
        </li>
        
        <li class="c-navbar__item">
            <a class="c-navbar__link" href="https://www.biomedcentral.com/about">
                About BMC
            </a>
        </li>
        
    </ul>


                <ul class="c-navbar__nav">
                    <li class="c-navbar__item">
                        <button type="button" class="c-navbar__link js-publisher-search-button u-hidden-at-lt-lg" data-toggle="collapse" data-test="header-search-button" data-target="publisher-header-search publisher-header-menu publisher-header-menu-button" aria-controls="publisher-header-search publisher-header-menu" aria-expanded="false">
                            <span class="c-flex c-flex--align-center">
                                Search
                                <svg class="c-icon u-margin-left-xs" width="14" height="14" aria-hidden="true">
                                    <use xlink:href="#icon-search"></use>
                                </svg>
                            </span>
                        </button>
                    </li>
                    <li class="c-navbar__item">
                        <a data-replace="true" data-placeholder="/placeholder/v1/username" class="c-navbar__link" href="https://www.biomedcentral.com/login" data-test="login-link">
                            <span class="u-js-shown">Login</span>
                            <span class="u-js-hidden">My Account</span>
                        </a>
                    </li>
                </ul>
            </div>
        </div>
    </nav>
    <div class="c-popup-search__content c-collapse js-publisher-search-bar" id="publisher-header-search">
        <div class="u-container">
            <div class="c-popup-search__container">
                <div class="ctx-search">
    <form role="search" class="c-form-field" method="GET" action="//www.biomedcentral.com/search" data-track="submit" data-track-category="Search and Results" data-track-action="Submit search" data-dynamic-track-label="" data-track-label="" data-test="global-search" novalidate="">
        <label for="publisherSearch" class="c-form-field__label">Search all BMC articles</label>
        <div class="c-flex">
            <input id="publisherSearch" class="c-form-field__input js-publisher-search-input" autocomplete="off" role="textbox" data-test="search-input" name="query" type="text">
            <div>
                <button class="c-button" type="submit" data-test="search-submit-button">
    <span class="u-visually-hidden">Search</span>
    <svg class="c-icon" width="16" height="16" aria-hidden="true">
        <use xlink:href="#icon-search"></use>
    </svg>
</button>


            </div>
        </div>
        <input name="searchType" value="publisherSearch" type="hidden">
    </form>
</div>

            </div>
        </div>
    </div>
</header>


        
            <header class="c-journal-header c-journal-header--frontiers-in-zoology ctx-journal-header">
                <div class="u-container">
                    <div class="c-journal-header__inner ">
                        
                        <div class="c-journal-title" id="journalTitle">
                            <a href="https://frontiersinzoology.biomedcentral.com/">
    
        
            <img class="c-journal-title__logo" src="logo.svg" alt="Frontiers in Zoology logo">
        
    
    <span class="c-journal-title__text ">Frontiers in Zoology</span>
    
</a>
                        </div>
                        
                    </div>
                </div>
                <div class="c-navbar">
                    <div class="u-container">
                        
                            <button type="button" class="c-navbar__link c-dropdown u-hidden-at-lg" data-toggle="collapse" data-test="journal-header-menu" data-toggle-linked="journal-header" data-target="journal-menu" aria-controls="journal-menu" aria-expanded="false">
                                    <span class="c-dropdown__toggle">Menu</span>
                            </button>
                            <div class="c-navbar__content c-collapse" id="journal-menu">
                                <nav class="c-navbar__nav">
                                    <ul class="c-navbar__nav c-navbar__nav--journal" role="menu" data-test="site-navigation">
                                        
                                            <li class="c-navbar__item" role="menuitem">
                                                <a class="c-navbar__link" href="https://frontiersinzoology.biomedcentral.com/">Home</a>
                                            </li>
                                        
                                            <li class="c-navbar__item" role="menuitem">
                                                <a class="c-navbar__link" href="https://frontiersinzoology.biomedcentral.com/about">About</a>
                                            </li>
                                        
                                            <li class="c-navbar__item" role="menuitem">
                                                <a class="c-navbar__link c-navbar__link--is-shown" href="https://frontiersinzoology.biomedcentral.com/articles">Articles</a>
                                            </li>
                                        
                                            <li class="c-navbar__item" role="menuitem">
                                                <a class="c-navbar__link" href="https://frontiersinzoology.biomedcentral.com/submission-guidelines">Submission Guidelines</a>
                                            </li>
                                        
                                    </ul>
                                </nav>
                            </div>
                        
                    </div>
                </div>
                <div class="c-journal-header__identity c-journal-header__identity--wine-red">
                    
                </div>
            </header>
            
        

        <div class="u-container u-margin-top-xl u-margin-bottom-xl" id="main-content">
            <div class="c-content-layout c-content-layout--fulltext" data-component="sticky-container">
                <div class="c-content-layout__nav u-hidden-at-lt-xl">
                    <nav xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" id="ArticleNavigation" aria-labelledby="toc-menu-title" data-component="sticky"><h3 id="toc-menu-title" class="u-visually-hidden" data-test="toc-title">Table of Contents</h3><ol class="c-nav c-nav--stacked" role="menu" id="tocMenu" aria-expanded="true" data-article-toc=""><li role="menuitem" class="c-nav__item"><a class="c-nav__link" href="#Abs1" data-track-category="Table of Contents" data-track="click" data-track-action="Link Clicked" data-track-label="Abstract" data-test="abstract-link">Abstract</a></li><li role="menuitem" class="c-nav__item"><a class="c-nav__link" href="#Sec1" data-track-category="Table of Contents" data-track="click" data-track-action="Link Clicked" data-track-label="Background">Background</a></li><li role="menuitem" class="c-nav__item"><a class="c-nav__link" href="#Sec2" data-track-category="Table of Contents" data-track="click" data-track-action="Link Clicked" data-track-label="Results">Results</a></li><li role="menuitem" class="c-nav__item"><a class="c-nav__link" href="#Sec6" data-track-category="Table of Contents" data-track="click" data-track-action="Link Clicked" data-track-label="Discussion">Discussion</a></li><li role="menuitem" class="c-nav__item"><a class="c-nav__link" href="#Sec10" data-track-category="Table of Contents" data-track="click" data-track-action="Link Clicked" data-track-label="Conclusions">Conclusions</a></li><li role="menuitem" class="c-nav__item"><a class="c-nav__link" href="#Sec11" data-track-category="Table of Contents" data-track="click" data-track-action="Link Clicked" data-track-label="Materials and methods">Materials and methods</a></li><li role="menuitem" class="c-nav__item"><a class="c-nav__link" href="#Sec16" data-track-category="Table of Contents" data-track="click" data-track-action="Link Clicked" data-track-label="Authors' information">Authors' information</a></li><li role="menuitem" class="c-nav__item"><a class="c-nav__link" href="#Decs" data-test="declarations-link" data-track-category="Table of Contents" data-track="click" data-track-action="Link Clicked" data-track-label="Declarations">
                  Declarations
                </a></li><li role="menuitem" class="c-nav__item"><a class="c-nav__link" href="#Bib1" data-track-category="Table of Contents" data-track="click" data-track-action="Link Clicked" data-track-label="References">References</a></li><li class="c-nav__item"><a href="#Comments" class="c-nav__link">Comments</a></li></ol></nav>
                </div>

                <main class="c-content-layout__main">
                    <article itemscope="" itemtype="http://schema.org/ScholarlyArticle">
                        
                        
    <div class="FulltextWrapper">
        <ul class="c-article-identifiers u-reset-list">
            <li class="c-article-identifiers__item" data-test="article-category">Research</li>
            <li class="c-article-identifiers__item" data-test="open-access">
                <span class="u-text-open-access">Open Access</span>
            </li>
            
            
        </ul>
    </div>

                        <div class="FulltextWrapper"><div xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" class="MainTitleSection"><h1 xmlns="http://www.w3.org/1999/xhtml" class="ArticleTitle" itemprop="headline" lang="en">Hybridization in East African swarm-raiding army ants</h1></div><div xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" class="AuthorNames" data-author-affiliations="true"><ul class="u-listReset"><li class="Author hasAffil" data-jumpto="Aff" data-test="author-name"><span xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="AuthorName" itemprop="author" itemscope="" itemtype="http://schema.org/Person">Daniel&nbsp;JC&nbsp;Kronauer</span><span class="EqualContributionSymbol">†</span><sup><a href="#Aff1" class="AffiliationID">1</a>, <a href="#Aff2" class="AffiliationID">2</a></sup><a href="mailto:dkronauer@rockefeller.edu" class="EmailAuthor" title="Email author"><span class="ContactIcon" aria-hidden="true"></span><span class="u-srOnly">Email author</span></a>, </li><li class="Author hasAffil" data-jumpto="Aff" data-test="author-name"><span xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="AuthorName" itemprop="author" itemscope="" itemtype="http://schema.org/Person">Marcell&nbsp;K&nbsp;Peters</span><span class="EqualContributionSymbol">†</span><sup><a href="#Aff3" class="AffiliationID">3</a>, <a href="#Aff4" class="AffiliationID">4</a></sup>, </li><li class="Author hasAffil" data-jumpto="Aff" data-test="author-name"><span xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="AuthorName" itemprop="author" itemscope="" itemtype="http://schema.org/Person">Caspar&nbsp;Schöning</span><sup><a href="#Aff1" class="AffiliationID">1</a>, <a href="#Aff5" class="AffiliationID">5</a></sup> and </li><li class="Author hasAffil" data-jumpto="Aff" data-test="author-name"><span xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="AuthorName" itemprop="author" itemscope="" itemtype="http://schema.org/Person">Jacobus&nbsp;J&nbsp;Boomsma</span><sup><a href="#Aff1" class="AffiliationID">1</a></sup></li></ul><div class="AffiliationLegend"><div class="AffiliationLegendLine"><span class="EqualContributionSymbol">†</span>Contributed equally</div></div></div><div xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" class="ArticleContextInformation"><div class="ContextInformation"><div class="ContextInformationJournalTitles"><span class="ArticleCitation"><span itemscope="" itemtype="http://schema.org/Periodical" itemid="#periodical"><span xmlns="http://www.w3.org/1999/xhtml" class="JournalTitle">Frontiers in Zoology</span></span><span class="ArticleCitation_Year">2011</span><span class="ArticleCitation_Volume"><strong itemprop="isPartOf" itemtype="http://schema.org/PublicationVolume"><link itemprop="isPartOf" href="#periodical">8</strong>:20</span></span></div><p class="ArticleDOI"><a itemprop="sameAs" class="u-text-inherit" href="https://doi.org/10.1186/1742-9994-8-20">https://doi.org/10.1186/1742-9994-8-20</a></p><p class="Copyright">© &nbsp;Kronauer et al; licensee BioMed Central Ltd.&nbsp;2011</p></div></div><ul class="ArticleHistory u-reset-list"><li class="History HistoryReceived"><strong>Received: </strong><span>16&nbsp;May&nbsp;2011</span></li><li class="History HistoryAccepted"><strong>Accepted: </strong><span>22&nbsp;August&nbsp;2011</span></li><li class="History HistoryOnlineDate"><strong>Published: </strong><span itemprop="datePublished">22&nbsp;August&nbsp;2011</span></li></ul></div>
                        
                        
                        
    

                        <div class="FulltextWrapper"><section xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" class="Abstract Section1 RenderAsSection1 c-section" id="Abs1" lang="en"><h2 class="c-section__heading"><a href="#Abstract" data-test="abstract-heading" data-disable-click-event-default="">Abstract</a></h2><div class="c-section__content" id="Abstract"><div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="AbstractSection" id="ASec1">
                <h3 xmlns="" class="Heading">Background</h3>
                <p class="Para">Hybridization can have complex effects 
on evolutionary dynamics in ants because of the combination of 
haplodiploid sex-determination and eusociality. While hybrid 
non-reproductive workers have been found in a range of species, examples
 of gene-flow via hybrid queens and males are rare. We studied 
hybridization in East African army ants (<em xmlns="" class="EmphasisTypeItalic">Dorylus</em> subgenus <em xmlns="" class="EmphasisTypeItalic">Anomma</em>) using morphology, mitochondrial DNA sequences, and nuclear microsatellites.</p>
              </div><div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="AbstractSection" id="ASec2">
                <h3 xmlns="" class="Heading">Results</h3>
                <p class="Para">While the mitochondrial phylogeny had a 
strong geographic signal, different species were not recovered as 
monophyletic. At our main study site at Kakamega Forest, a mitochondrial
 haplotype was shared between a "<em xmlns="" class="EmphasisTypeItalic">Dorylus molestus</em>-like" and a "<em xmlns="" class="EmphasisTypeItalic">Dorylus wilverthi</em>-like" form. This pattern is best explained by introgression following hybridization between <em xmlns="" class="EmphasisTypeItalic">D. molestus</em> and <em xmlns="" class="EmphasisTypeItalic">D. wilverthi</em>.
 Microsatellite data from workers showed that the two morphological 
forms correspond to two distinct genetic clusters, with a significant 
proportion of individuals being classified as hybrids.</p>
              </div><div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="AbstractSection" id="ASec3">
                <h3 xmlns="" class="Heading">Conclusions</h3>
                <p class="Para">We conclude that hybridization and gene-flow between the two army ant species <em xmlns="" class="EmphasisTypeItalic">D. molestus</em> and <em xmlns="" class="EmphasisTypeItalic">D. wilverthi</em>
 has occurred, and that mating between the two forms continues to 
regularly produce hybrid workers. Hybridization is particularly 
surprising in army ants because workers have control over which males 
are allowed to mate with a young virgin queen inside the colony.</p>
              </div></div></section><section xmlns:func="http://oscar.fig.bmc.com" class="KeywordGroup Section1 RenderAsSection1 c-section" lang="en"><h2 xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" class="Heading js-ToggleCollapseSection c-section__heading" data-sticky-update="true"><a href="#Keywords" data-toggle="collapse" data-toggle-disable-clickoff="" data-target="Keywords">Keywords</a></h2><div class="js-CollapseSection CollapseSection c-section__content" data-test="collapse-section" id="Keywords"><ul class="c-keywords"><li class="c-keywords__item" itemprop="about">Dorylinae</li><li class="c-keywords__item" itemprop="about">Formicidae</li><li class="c-keywords__item" itemprop="about">introgression</li><li class="c-keywords__item" itemprop="about">microsatellites</li><li class="c-keywords__item" itemprop="about">mtDNA</li><li class="c-keywords__item" itemprop="about">gene flow</li></ul></div></section>
            <section xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" id="Sec1" class="Section1 RenderAsSection1 c-section"><h2 class="Heading js-ToggleCollapseSection c-section__heading" data-sticky-update="true"><a href="#Background" data-toggle="collapse" data-toggle-disable-clickoff="" data-target="Background">Background</a></h2><div class="js-CollapseSection CollapseSection c-section__content" id="Background" data-test="collapse-section"><p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Para">While
 botanists have long accepted that hybridization plays an important role
 in plant evolution and regularly leads to the emergence of new species [<span class="CitationRef"><a href="#CR1" data-toggle="citation">1</a></span>], zoologists have traditionally regarded hybridization and interspecific gene flow as rare exceptions [<span class="CitationRef"><a href="#CR2" data-toggle="citation">2</a></span>].
 However, the advent of molecular genetic markers has changed this view 
during the last decade and it is now widely accepted that hybridization 
between closely related species is also common in animals [<span class="CitationRef"><a href="#CR3" data-toggle="citation">3</a></span><span>, <span class="CitationRef"><a href="#CR4" data-toggle="citation">4</a></span>, </span><span class="CitationRef"><a href="#CR5" data-toggle="citation">5</a></span>].
 Hybrids are often non-viable or sterile due to negative epistasis and 
therefore tend to be rather efficiently removed from the population by 
natural selection [<span class="CitationRef"><a href="#CR6" data-toggle="citation">6</a></span>].
 However, in situations where this is not the case, hybridization can 
lead to the collapse of closely related species into a single panmictic 
population [e.g. [<span class="CitationRef"><a href="#CR7" data-toggle="citation">7</a></span>]], or to speciation events when some form of reproductive isolation between parental species and hybrids emerges [<span class="CitationRef"><a href="#CR3" data-toggle="citation">3</a></span><span>, <span class="CitationRef"><a href="#CR4" data-toggle="citation">4</a></span>, </span><span class="CitationRef"><a href="#CR5" data-toggle="citation">5</a></span>].</p><p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Para">In
 haplodiploid eusocial animals such as ants, the consequences of 
hybridization can be strikingly different from those in other organisms.
 First, a diploid queen that has mated with a heterospecific haploid 
male will still produce purebred sons via arrhenotokous parthenogenesis,
 so that hybrid males will only be produced in the F2 generation. 
Second, potentially negative consequences of hybrid sterility can be 
mitigated or even avoided when hybrid individuals mostly or exclusively 
become non-reproductive workers [<span class="CitationRef"><a href="#CR8" data-toggle="citation">8</a></span>, <span class="CitationRef"><a href="#CR9" data-toggle="citation">9</a></span>].
 Hybrid workers do indeed regularly occur in a variety of ant species, 
whereas hybrid queens and males are normally not observed [<span class="CitationRef"><a href="#CR10" data-toggle="citation">10</a></span><span>, <span class="CitationRef"><a href="#CR11" data-toggle="citation">11</a></span>, </span><span class="CitationRef"><a href="#CR12" data-toggle="citation">12</a></span>].
 Hybridization in ants can therefore lead to interesting evolutionary 
novelties, such as genetic caste determination in interdependent 
lineages of <em xmlns="" class="EmphasisTypeItalic">Pogonomyrmex</em> harvester ants, where purebred females become queens and interlineage hybrids become workers [<span class="CitationRef"><a href="#CR13" data-toggle="citation">13</a></span>, <span class="CitationRef"><a href="#CR14" data-toggle="citation">14</a></span>]. Another recent example is a population of <em xmlns="" class="EmphasisTypeItalic">Formica</em> wood ants with two clearly distinct male genepools and hybrid queens [<span class="CitationRef"><a href="#CR15" data-toggle="citation">15</a></span>].
 This stable system is maintained by strong transmission ratio 
distortion with respect to the sex of the offspring. In both cases there
 is no gene flow between the parental populations.</p><p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Para">In
 the present study we investigated possible hybridization between 
different species of African swarm-raiding army ants, with particular 
focus on Kakamega Forest, Kenya, which is one of the few sites where the
 two closely related species, <em xmlns="" class="EmphasisTypeItalic">Dorylus (Anomma) molestus</em> and <em xmlns="" class="EmphasisTypeItalic">Dorylus (Anomma) wilverthi</em>,
 occur in sympatry. African swarm-raiding army ants (the "driver ants") 
are a prominent feature in afrotropical forests, where they are prime 
invertebrate predators with colony sizes of over ten million 
individuals. They form a well-supported clade within the <em xmlns="" class="EmphasisTypeItalic">Dorylus</em> subgenus <em xmlns="" class="EmphasisTypeItalic">Anomma</em>[<span class="CitationRef"><a href="#CR16" data-toggle="citation">16</a></span>].
 Like in other Hymenoptera, females (queens and workers) are diploid, 
while males are haploid and are produced by arrhenotokous 
parthenogenesis. Army ant queens, unlike the queens of most other ants, 
are permanently wingless, never go on a mating flight, and do not found 
new colonies independently. Instead, large colonies produce a 
reproductive brood of several thousand winged males and just a few 
virgin queens. Colony fission occurs when the mother queen and a large 
fraction of the worker-force emigrate from the nest and leave the 
developing reproductive brood and the remaining workers behind. All but 
one of the virgin queens are eliminated in yet unknown ways. After the 
males have eclosed, they disperse on the wing to mate with a young queen
 in another colony. Virgin queens, on the other hand, never leave their 
natal colony and mate with typically 10 - 30 foreign males that enter 
the colony from outside [<span class="CitationRef"><a href="#CR17" data-toggle="citation">17</a></span>]. Because colonies and their queens only disperse on foot, gene flow is highly male-biased [<span class="CitationRef"><a href="#CR18" data-toggle="citation">18</a></span>, <span class="CitationRef"><a href="#CR19" data-toggle="citation">19</a></span>].</p><p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Para">An
 interesting consequence of this idiosyncratic reproductive system is 
that males have to "run the gauntlet of the workers" before they can 
mate, so that workers have ample opportunities to choose the mates of 
their queen-sister [<span class="CitationRef"><a href="#CR20" data-toggle="citation">20</a></span>].
 If her later breeding success is affected by some of her many matings 
being heterospecific, the workers are under selection to influence the 
outcome of this mating process because the result affects their 
inclusive fitness. This implies that documentation of hybridization and 
introgression in army ants provides direct working hypotheses about the 
fitness of mixed genotypes. <em xmlns="" class="EmphasisTypeItalic">Dorylus</em>
 colonies are headed by a single queen (monogynous), but due to the high
 queen-mating frequency, most workers in a colony are half-sisters, i.e.
 they have the same mother queen but different fathers [<span class="CitationRef"><a href="#CR17" data-toggle="citation">17</a></span>].</p><p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Para">Previous work showed that the two species at Kakamega occupy clearly distinct ecological niches, with <em xmlns="" class="EmphasisTypeItalic">D. wilverthi</em> being essentially restricted to intact rainforest, while <em xmlns="" class="EmphasisTypeItalic">D. molestus</em> also occurs in savannah habitats and open agricultural landscapes [<span class="CitationRef"><a href="#CR21" data-toggle="citation">21</a></span><span>, <span class="CitationRef"><a href="#CR22" data-toggle="citation">22</a></span>, </span><span class="CitationRef"><a href="#CR23" data-toggle="citation">23</a></span>].
 Our preliminary morphological assessments suggested that both 
populations at Kakamega are aberrant when compared to allopatric 
populations of the same species, indicating that hybridization and 
introgression may have taken place. That hybridization between closely 
related army ants may occur is also consistent with the observation that
 heterospecific males are occasionally found in <em xmlns="" class="EmphasisTypeItalic">Dorylus</em> (<em xmlns="" class="EmphasisTypeItalic">Anomma</em>) colonies [<span class="CitationRef"><a href="#CR24" data-toggle="citation">24</a></span>], and with the description of the "hybrid variety" <em xmlns="" class="EmphasisTypeItalic">D. sjoestedti var. sjoestedti-wilverthi</em>[<span class="CitationRef"><a href="#CR25" data-toggle="citation">25</a></span>].</p><p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Para">Here
 we use a combination of morphological, mitochondrial and nuclear 
genetic data to show that hybridization and introgression have occurred 
historically between different species of swarm-raiding army ants at a 
large geographic scale in East Africa, and that ongoing hybridization 
can be detected in <em xmlns="" class="EmphasisTypeItalic">Dorylus</em> workers from Kakamega. Our results allow novel inferences about the evolutionary relevance of hybridization in social insects.</p></div></section>
            <section xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" id="Sec2" class="Section1 RenderAsSection1 c-section"><h2 class="Heading js-ToggleCollapseSection c-section__heading" data-sticky-update="true"><a href="#Results" data-toggle="collapse" data-toggle-disable-clickoff="" data-target="Results">Results</a></h2><div class="js-CollapseSection CollapseSection c-section__content" id="Results" data-test="collapse-section"><section id="Sec3" class="Section2 RenderAsSection2">
                <h3 class="Heading">(a) Morphological Analysis</h3>
                <div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Para">The <em xmlns="" class="EmphasisTypeItalic">D. wilverthi</em>-like workers from Kakamega (see Figure <span class="InternalRef"><a href="#Fig1">1</a></span>
 for geographic positions of study sites) had the posterior angles of 
the head prolonged into a raised point, but not recurved outwards as in 
specimens from more western populations (Figure <span class="InternalRef"><a href="#Fig2">2a</a></span>). More than half of the examined <em xmlns="" class="EmphasisTypeItalic">D. molestus</em>-like
 workers lacked the characteristic petiolar tubercles. Workers of both 
forms at Kakamega had relatively shorter antennal scapes than the 
workers from the respective "pure" allopatric populations (Figure <span class="InternalRef"><a href="#Fig2">2b</a></span>).
 The best common regression model for the morphometric data combined was
 SL = 0.3039 * HW + 0.6994 (SL = antennal scape length, HW = maximum 
head width). Differences between the relative residuals between all four
 groups were highly significant (Kruskal-Wallis statistic = 280.7, p 
&lt; 0.0001, Dunn's Multiple Comparison Test in all comparisons p &lt; 
0.01).<figure xmlns="" class="Figure" id="Fig1"><div xmlns="http://www.w3.org/1999/xhtml" class="MediaObject">
                      <a target="_blank" rel="noopener" href="https://media.springernature.com/full/springer-static/image/art%3A10.1186%2F1742-9994-8-20/MediaObjects/12983_2011_Article_166_Fig1_HTML.jpg"><img class="c-lazy-image u-js-shown" src="12983_2011_Article_166_Fig1_HTML.jpg" alt="Figure 1" data-lazy-load="true" data-src="https://media.springernature.com/lw785/springer-static/image/art%3A10.1186%2F1742-9994-8-20/MediaObjects/12983_2011_Article_166_Fig1_HTML.jpg"><noscript><img src="https://media.springernature.com/lw785/springer-static/image/art%3A10.1186%2F1742-9994-8-20/MediaObjects/12983_2011_Article_166_Fig1_HTML.jpg" alt="Figure 1"></img></noscript></a>
                    </div><figcaption class="Caption" lang="en"><div xmlns="http://www.w3.org/1999/xhtml" class="CaptionContent"><span class="CaptionNumber">Figure 1</span>
                        <p xmlns="" class="SimplePara"><strong class="EmphasisTypeBold">Collection sites for</strong> <span class="EmphasisTypeBoldItalic">Dorylus</span> <strong class="EmphasisTypeBold">(</strong> <span class="EmphasisTypeBoldItalic">Anomma</span> <strong class="EmphasisTypeBold">) samples in East Africa</strong>. White = <em class="EmphasisTypeItalic">D. molestus</em>, black = <em class="EmphasisTypeItalic">D. wilverthi</em>, grey = <em class="EmphasisTypeItalic">D. terrificus</em>.
 The dots for three main study sites are plotted in larger size. The 
three West and Central African sites (see Table 1) where samples of <em class="EmphasisTypeItalic">D. sjoestedti</em>, <em class="EmphasisTypeItalic">D. nigricans</em>, and <em class="EmphasisTypeItalic">D. emeryi</em> were collected are given on the inset map.</p>
                      </div></figcaption></figure><figure xmlns="" class="Figure" id="Fig2"><div xmlns="http://www.w3.org/1999/xhtml" class="MediaObject">
                      <a target="_blank" rel="noopener" href="https://media.springernature.com/full/springer-static/image/art%3A10.1186%2F1742-9994-8-20/MediaObjects/12983_2011_Article_166_Fig2_HTML.jpg"><img class="c-lazy-image u-js-shown" src="12983_2011_Article_166_Fig2_HTML.jpg" alt="Figure 2" data-lazy-load="true" data-src="https://media.springernature.com/lw785/springer-static/image/art%3A10.1186%2F1742-9994-8-20/MediaObjects/12983_2011_Article_166_Fig2_HTML.jpg"><noscript><img src="https://media.springernature.com/lw785/springer-static/image/art%3A10.1186%2F1742-9994-8-20/MediaObjects/12983_2011_Article_166_Fig2_HTML.jpg" alt="Figure 2"></img></noscript></a>
                    </div><figcaption class="Caption" lang="en"><div xmlns="http://www.w3.org/1999/xhtml" class="CaptionContent"><span class="CaptionNumber">Figure 2</span>
                        <p xmlns="" class="SimplePara"><strong class="EmphasisTypeBold">Morphological variation among army ant populations</strong>. (A) Photographs of <em class="EmphasisTypeItalic">D. wilverthi</em> (left) and <em class="EmphasisTypeItalic">D. molestus</em>. Arrows indicate diagnostic "horns" in <em class="EmphasisTypeItalic">D. wilverthi</em> and petiolar tubercles in <em class="EmphasisTypeItalic">D. molestus</em>.
 Photographs are courtesy of April Nobile and AntWeb at 
http://www.antweb.org. (B) The relationship between antennal scape 
length (SL) and maximum head width (HW) in workers of "pure" allopatric <em class="EmphasisTypeItalic">D. molestus</em>, "pure" allopatric <em class="EmphasisTypeItalic">D. wilverthi</em>, sympatric <em class="EmphasisTypeItalic">D. molestus</em>-like individuals from Kakamega, and sympatric <em class="EmphasisTypeItalic">D. wilverthi</em>-like individuals from Kakamega.</p>
                      </div></figcaption></figure></div>
              </section><section id="Sec4" class="Section2 RenderAsSection2">
                <h3 class="Heading">(b) Phylogeographic Analysis</h3>
                <div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Para">The phylogenetic analysis of mitochondrial haplotypes resolved several well supported clades (Figures <span class="InternalRef"><a href="#Fig3">3</a></span> and <span class="InternalRef"><a href="#Fig4">4</a></span>, Table <span class="InternalRef"><a href="#Tab1">1</a></span>). Most strikingly, neither <em xmlns="" class="EmphasisTypeItalic">D. molestus</em> nor <em xmlns="" class="EmphasisTypeItalic">D. wilverthi</em>
 were resolved as monophyletic. Instead, the mitochondrial phylogeny 
reflects geographical patterns rather than species affiliations: all 
samples from the coastal area of Kenya and Tanzania south of the Lower 
Tana River form a well-supported clade, extending inland to the southern
 slope of Mt. Kenya (clade 3 in Figures <span class="InternalRef"><a href="#Fig3">3</a></span> and <span class="InternalRef"><a href="#Fig4">4</a></span>).
 The same is true for samples from Ethiopia and two of the more northern
 samples from Kenya and Uganda (clade 4). Samples north of and around 
Lake Victoria form another big clade (clade 2), which includes 
haplotypes from all three species that occur in the area. All common 
haplotypes from Kakamega form a monophyletic group (clade 1), which is 
nested within the larger Lake Victoria group (clade 2).<figure xmlns="" class="Figure" id="Fig3"><div xmlns="http://www.w3.org/1999/xhtml" class="MediaObject">
                      <a target="_blank" rel="noopener" href="https://media.springernature.com/full/springer-static/image/art%3A10.1186%2F1742-9994-8-20/MediaObjects/12983_2011_Article_166_Fig3_HTML.jpg"><img class="c-lazy-image u-js-shown" src="12983_2011_Article_166_Fig3_HTML.jpg" alt="Figure 3" data-lazy-load="true" data-src="https://media.springernature.com/lw785/springer-static/image/art%3A10.1186%2F1742-9994-8-20/MediaObjects/12983_2011_Article_166_Fig3_HTML.jpg"><noscript><img src="https://media.springernature.com/lw785/springer-static/image/art%3A10.1186%2F1742-9994-8-20/MediaObjects/12983_2011_Article_166_Fig3_HTML.jpg" alt="Figure 3"></img></noscript></a>
                    </div><figcaption class="Caption" lang="en"><div xmlns="http://www.w3.org/1999/xhtml" class="CaptionContent"><span class="CaptionNumber">Figure 3</span>
                        <p xmlns="" class="SimplePara"><strong class="EmphasisTypeBold">Maximum likelihood phylogram showing relationships between mitochondrial haplotypes of East African</strong> <span class="EmphasisTypeBoldItalic">Dorylus</span> <strong class="EmphasisTypeBold">(</strong> <span class="EmphasisTypeBoldItalic">Anomma</span> <strong class="EmphasisTypeBold">) swarm-raiding army ants</strong>. Each label gives the haplotype number (see Table 1), the abbreviated species name (Dem: <em class="EmphasisTypeItalic">D. emeryi</em>; Dmo: <em class="EmphasisTypeItalic">D. molestus</em> (green); Dni: <em class="EmphasisTypeItalic">D. nigricans</em>; Dsj: <em class="EmphasisTypeItalic">D. sjoestedti</em> (red); Dte: <em class="EmphasisTypeItalic">D. terrificus</em> (grey); Dwi: <em class="EmphasisTypeItalic">D. wilverthi</em>
 (blue)), the sample locality, and the sample size in parentheses for 
haplotypes that were sampled more than once. Numbered bars on the right 
refer to clades that are discussed in the text. Node labels represent 
posterior probabilities from Bayesian analyses (*: 0.95 - 0.99; **: 
1.0)/maximum likelihood bootstraps (*: 0.75 - 0.9; **: &gt; 0.9). No 
internal labels or a hyphen indicate support values &lt; 0.95 and &lt; 
0.75 for the Bayesian and maximum likelihood analysis, respectively. The
 haplotype that is shared between the <em class="EmphasisTypeItalic">D. molestus</em>-like and the <em class="EmphasisTypeItalic">D. wilverthi</em>-like forms at Kakamega Forest is highlighted in a red box.</p>
                      </div></figcaption></figure><figure xmlns="" class="Figure" id="Fig4"><div xmlns="http://www.w3.org/1999/xhtml" class="MediaObject">
                      <a target="_blank" rel="noopener" href="https://media.springernature.com/full/springer-static/image/art%3A10.1186%2F1742-9994-8-20/MediaObjects/12983_2011_Article_166_Fig4_HTML.jpg"><img class="c-lazy-image u-js-shown" src="12983_2011_Article_166_Fig4_HTML.jpg" alt="Figure 4" data-lazy-load="true" data-src="https://media.springernature.com/lw785/springer-static/image/art%3A10.1186%2F1742-9994-8-20/MediaObjects/12983_2011_Article_166_Fig4_HTML.jpg"><noscript><img src="https://media.springernature.com/lw785/springer-static/image/art%3A10.1186%2F1742-9994-8-20/MediaObjects/12983_2011_Article_166_Fig4_HTML.jpg" alt="Figure 4"></img></noscript></a>
                    </div><figcaption class="Caption" lang="en"><div xmlns="http://www.w3.org/1999/xhtml" class="CaptionContent"><span class="CaptionNumber">Figure 4</span>
                        <p xmlns="" class="SimplePara"><strong class="EmphasisTypeBold">Parsimony network of all East African haplotypes of</strong> <span class="EmphasisTypeBoldItalic">D. molestus</span> <strong class="EmphasisTypeBold">,</strong> <span class="EmphasisTypeBoldItalic">D. wilverthi</span> <strong class="EmphasisTypeBold">and</strong> <span class="EmphasisTypeBoldItalic">D. terrificus</span>.
 A connecting line between haplotypes represents one mutation and small 
open circles represent missing (inferred) haplotypes. Green haploytypes 
are from <em class="EmphasisTypeItalic">D. molestus</em>, blue haplotypes from <em class="EmphasisTypeItalic">D. wilverthi</em>, and grey haplotypes from <em class="EmphasisTypeItalic">D. terrificus</em>.
 Size of haplotype disks is proportional to their frequencies. Boxes 
with numbers indicate the same haplotype clusters as shown in Figure 3.</p>
                      </div></figcaption></figure></div>
                <figure id="Tab1" class="FigureTable"><figcaption class="Caption" lang="en"><div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="CaptionContent"><span class="CaptionNumber">Table 1</span>
                      <p xmlns="" class="SimplePara">List of unique mitochondrial <em class="EmphasisTypeItalic">COII</em> haplotypes included in this study</p>
                    </div></figcaption><div class="Table"><table xmlns="http://www.w3.org/1999/xhtml" border="1"><colgroup><col><col><col><col><col></colgroup><thead><tr class="header"><th><p xmlns="" class="SimplePara">Haplotype no.</p></th><th><p xmlns="" class="SimplePara">GenBank acc. no.</p></th><th><p xmlns="" class="SimplePara">Species</p></th><th><p xmlns="" class="SimplePara">Locality</p></th><th><p xmlns="" class="SimplePara"><span class="EmphasisTypeBoldItalic">N</span> colonies</p></th></tr></thead><tbody><tr class="noclass"><td><p xmlns="" class="SimplePara">1</p></td><td><p xmlns="" class="SimplePara">GQ999016</p></td><td><p xmlns="" class="SimplePara">
                            <em class="EmphasisTypeItalic">D. wilverthi</em>
                          </p></td><td><p xmlns="" class="SimplePara">Kakamega Forest, Kenya</p></td><td><p xmlns="" class="SimplePara">33</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">1</p></td><td><p xmlns="" class="SimplePara">GQ999023</p></td><td><p xmlns="" class="SimplePara">
                            <em class="EmphasisTypeItalic">D. molestus</em>
                          </p></td><td><p xmlns="" class="SimplePara">Kakamega Forest, Kenya</p></td><td><p xmlns="" class="SimplePara">8</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">2</p></td><td><p xmlns="" class="SimplePara">GQ999019</p></td><td><p xmlns="" class="SimplePara">
                            <em class="EmphasisTypeItalic">D. wilverthi</em>
                          </p></td><td><p xmlns="" class="SimplePara">Kakamega Forest, Kenya</p></td><td><p xmlns="" class="SimplePara">1</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">3</p></td><td><p xmlns="" class="SimplePara">GQ999017</p></td><td><p xmlns="" class="SimplePara">
                            <em class="EmphasisTypeItalic">D. molestus</em>
                          </p></td><td><p xmlns="" class="SimplePara">Kakamega Forest, Kenya</p></td><td><p xmlns="" class="SimplePara">6</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">4</p></td><td><p xmlns="" class="SimplePara">GQ999020</p></td><td><p xmlns="" class="SimplePara">
                            <em class="EmphasisTypeItalic">D. molestus</em>
                          </p></td><td><p xmlns="" class="SimplePara">Kakamega Forest, Kenya</p></td><td><p xmlns="" class="SimplePara">5</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">5</p></td><td><p xmlns="" class="SimplePara">GQ999022</p></td><td><p xmlns="" class="SimplePara">
                            <em class="EmphasisTypeItalic">D. molestus</em>
                          </p></td><td><p xmlns="" class="SimplePara">Kakamega Forest, Kenya</p></td><td><p xmlns="" class="SimplePara">7</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">6</p></td><td><p xmlns="" class="SimplePara">GQ999018</p></td><td><p xmlns="" class="SimplePara">
                            <em class="EmphasisTypeItalic">D. molestus</em>
                          </p></td><td><p xmlns="" class="SimplePara">Kakamega Forest, Kenya</p></td><td><p xmlns="" class="SimplePara">12</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">7</p></td><td><p xmlns="" class="SimplePara">GQ999021</p></td><td><p xmlns="" class="SimplePara">
                            <em class="EmphasisTypeItalic">D. molestus</em>
                          </p></td><td><p xmlns="" class="SimplePara">Kakamega Forest, Kenya</p></td><td><p xmlns="" class="SimplePara">3</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">8</p></td><td><p xmlns="" class="SimplePara">GQ999024</p></td><td><p xmlns="" class="SimplePara">
                            <em class="EmphasisTypeItalic">D. molestus</em>
                          </p></td><td><p xmlns="" class="SimplePara">Kakamega Forest, Kenya</p></td><td><p xmlns="" class="SimplePara">1</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">8</p></td><td><p xmlns="" class="SimplePara">GQ999043</p></td><td><p xmlns="" class="SimplePara">
                            <em class="EmphasisTypeItalic">D. molestus</em>
                          </p></td><td><p xmlns="" class="SimplePara">Mt. Elgon, Kenya</p></td><td><p xmlns="" class="SimplePara">1</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">9</p></td><td><p xmlns="" class="SimplePara">GQ999042</p></td><td><p xmlns="" class="SimplePara">
                            <em class="EmphasisTypeItalic">D. molestus</em>
                          </p></td><td><p xmlns="" class="SimplePara">Mt. Kenya (West), Kenya</p></td><td><p xmlns="" class="SimplePara">1</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">10</p></td><td><p xmlns="" class="SimplePara">EF413797</p></td><td><p xmlns="" class="SimplePara">
                            <em class="EmphasisTypeItalic">D. molestus</em>
                          </p></td><td><p xmlns="" class="SimplePara">Mt. Kenya (East), Kenya</p></td><td><p xmlns="" class="SimplePara">1</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">11</p></td><td><p xmlns="" class="SimplePara">GU065701</p></td><td><p xmlns="" class="SimplePara">
                            <em class="EmphasisTypeItalic">D. molestus</em>
                          </p></td><td><p xmlns="" class="SimplePara">Mt. Kenya (East), Kenya</p></td><td><p xmlns="" class="SimplePara">2</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">12</p></td><td><p xmlns="" class="SimplePara">GU065703</p></td><td><p xmlns="" class="SimplePara">
                            <em class="EmphasisTypeItalic">D. molestus</em>
                          </p></td><td><p xmlns="" class="SimplePara">Mt. Kenya (East), Kenya</p></td><td><p xmlns="" class="SimplePara">4</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">13</p></td><td><p xmlns="" class="SimplePara">GU065698</p></td><td><p xmlns="" class="SimplePara">
                            <em class="EmphasisTypeItalic">D. molestus</em>
                          </p></td><td><p xmlns="" class="SimplePara">Mt. Kenya (East), Kenya</p></td><td><p xmlns="" class="SimplePara">5</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">14</p></td><td><p xmlns="" class="SimplePara">GU065699</p></td><td><p xmlns="" class="SimplePara">
                            <em class="EmphasisTypeItalic">D. molestus</em>
                          </p></td><td><p xmlns="" class="SimplePara">Mt. Kenya (East), Kenya</p></td><td><p xmlns="" class="SimplePara">7</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">15</p></td><td><p xmlns="" class="SimplePara">GU065702</p></td><td><p xmlns="" class="SimplePara">
                            <em class="EmphasisTypeItalic">D. molestus</em>
                          </p></td><td><p xmlns="" class="SimplePara">Mt. Kenya (East), Kenya</p></td><td><p xmlns="" class="SimplePara">2</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">16</p></td><td><p xmlns="" class="SimplePara">GU065704</p></td><td><p xmlns="" class="SimplePara">
                            <em class="EmphasisTypeItalic">D. molestus</em>
                          </p></td><td><p xmlns="" class="SimplePara">Mt. Kenya (East), Kenya</p></td><td><p xmlns="" class="SimplePara">1</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">17</p></td><td><p xmlns="" class="SimplePara">GU065700</p></td><td><p xmlns="" class="SimplePara">
                            <em class="EmphasisTypeItalic">D. molestus</em>
                          </p></td><td><p xmlns="" class="SimplePara">Mt. Kenya (East), Kenya</p></td><td><p xmlns="" class="SimplePara">7</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">18</p></td><td><p xmlns="" class="SimplePara">GQ999037</p></td><td><p xmlns="" class="SimplePara">
                            <em class="EmphasisTypeItalic">D. molestus</em>
                          </p></td><td><p xmlns="" class="SimplePara">Mt. Kenya (South), Kenya</p></td><td><p xmlns="" class="SimplePara">1</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">19</p></td><td><p xmlns="" class="SimplePara">GQ999050</p></td><td><p xmlns="" class="SimplePara">
                            <em class="EmphasisTypeItalic">D. molestus</em>
                          </p></td><td><p xmlns="" class="SimplePara">Tana River (East), Kenya</p></td><td><p xmlns="" class="SimplePara">4</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">20</p></td><td><p xmlns="" class="SimplePara">GQ999052</p></td><td><p xmlns="" class="SimplePara">
                            <em class="EmphasisTypeItalic">D. molestus</em>
                          </p></td><td><p xmlns="" class="SimplePara">Tana River (East), Kenya</p></td><td><p xmlns="" class="SimplePara">1</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">21</p></td><td><p xmlns="" class="SimplePara">GQ999049</p></td><td><p xmlns="" class="SimplePara">
                            <em class="EmphasisTypeItalic">D. molestus</em>
                          </p></td><td><p xmlns="" class="SimplePara">Tana River (West), Kenya</p></td><td><p xmlns="" class="SimplePara">1</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">22</p></td><td><p xmlns="" class="SimplePara">GQ999032</p></td><td><p xmlns="" class="SimplePara">
                            <em class="EmphasisTypeItalic">D. molestus</em>
                          </p></td><td><p xmlns="" class="SimplePara">Tana River (West), Kenya</p></td><td><p xmlns="" class="SimplePara">4</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">23</p></td><td><p xmlns="" class="SimplePara">GQ999051</p></td><td><p xmlns="" class="SimplePara">
                            <em class="EmphasisTypeItalic">D. molestus</em>
                          </p></td><td><p xmlns="" class="SimplePara">Tana River (West), Kenya</p></td><td><p xmlns="" class="SimplePara">1</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">24</p></td><td><p xmlns="" class="SimplePara">GQ999027</p></td><td><p xmlns="" class="SimplePara">
                            <em class="EmphasisTypeItalic">D. molestus</em>
                          </p></td><td><p xmlns="" class="SimplePara">Addis Abeba, Ethiopia</p></td><td><p xmlns="" class="SimplePara">1</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">25</p></td><td><p xmlns="" class="SimplePara">GQ999028</p></td><td><p xmlns="" class="SimplePara">
                            <em class="EmphasisTypeItalic">D. molestus</em>
                          </p></td><td><p xmlns="" class="SimplePara">Bale Mts., Ethiopia</p></td><td><p xmlns="" class="SimplePara">1</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">26</p></td><td><p xmlns="" class="SimplePara">GQ999040</p></td><td><p xmlns="" class="SimplePara">
                            <em class="EmphasisTypeItalic">D. wilverthi</em>
                          </p></td><td><p xmlns="" class="SimplePara">Budongo Forest, Uganda</p></td><td><p xmlns="" class="SimplePara">1</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">27</p></td><td><p xmlns="" class="SimplePara">GQ999035</p></td><td><p xmlns="" class="SimplePara">
                            <em class="EmphasisTypeItalic">D. wilverthi</em>
                          </p></td><td><p xmlns="" class="SimplePara">Budongo Forest, Uganda</p></td><td><p xmlns="" class="SimplePara">2</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">28</p></td><td><p xmlns="" class="SimplePara">GQ999031</p></td><td><p xmlns="" class="SimplePara">
                            <em class="EmphasisTypeItalic">D. molestus</em>
                          </p></td><td><p xmlns="" class="SimplePara">Semliki, Uganda</p></td><td><p xmlns="" class="SimplePara">1</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">29</p></td><td><p xmlns="" class="SimplePara">GQ999053</p></td><td><p xmlns="" class="SimplePara">
                            <em class="EmphasisTypeItalic">D. molestus</em>
                          </p></td><td><p xmlns="" class="SimplePara">Mt. Lole, Kenya</p></td><td><p xmlns="" class="SimplePara">1</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">30</p></td><td><p xmlns="" class="SimplePara">GQ999045</p></td><td><p xmlns="" class="SimplePara">
                            <em class="EmphasisTypeItalic">D. molestus</em>
                          </p></td><td><p xmlns="" class="SimplePara">Mt. Warges, Kenya</p></td><td><p xmlns="" class="SimplePara">1</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">31</p></td><td><p xmlns="" class="SimplePara">EF413804</p></td><td><p xmlns="" class="SimplePara">
                            <em class="EmphasisTypeItalic">D. terrificus</em>
                          </p></td><td><p xmlns="" class="SimplePara">Kibale Forest, Uganda</p></td><td><p xmlns="" class="SimplePara">5</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">32</p></td><td><p xmlns="" class="SimplePara">EF413798</p></td><td><p xmlns="" class="SimplePara">
                            <em class="EmphasisTypeItalic">D. wilverthi</em>
                          </p></td><td><p xmlns="" class="SimplePara">Kibale Forest, Uganda</p></td><td><p xmlns="" class="SimplePara">2</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">33</p></td><td><p xmlns="" class="SimplePara">GQ999034</p></td><td><p xmlns="" class="SimplePara">
                            <em class="EmphasisTypeItalic">D. wilverthi</em>
                          </p></td><td><p xmlns="" class="SimplePara">Mabira Forest, Uganda</p></td><td><p xmlns="" class="SimplePara">2</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">34</p></td><td><p xmlns="" class="SimplePara">GQ999030</p></td><td><p xmlns="" class="SimplePara">
                            <em class="EmphasisTypeItalic">D. molestus</em>
                          </p></td><td><p xmlns="" class="SimplePara">Kalinzu, Uganda</p></td><td><p xmlns="" class="SimplePara">1</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">35</p></td><td><p xmlns="" class="SimplePara">GQ999033</p></td><td><p xmlns="" class="SimplePara">
                            <em class="EmphasisTypeItalic">D. molestus</em>
                          </p></td><td><p xmlns="" class="SimplePara">Nakuru, Kenya</p></td><td><p xmlns="" class="SimplePara">1</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">36</p></td><td><p xmlns="" class="SimplePara">GQ999039</p></td><td><p xmlns="" class="SimplePara">
                            <em class="EmphasisTypeItalic">D. terrificus</em>
                          </p></td><td><p xmlns="" class="SimplePara">Mgahinga, Uganda</p></td><td><p xmlns="" class="SimplePara">1</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">37</p></td><td><p xmlns="" class="SimplePara">GQ999038</p></td><td><p xmlns="" class="SimplePara">
                            <em class="EmphasisTypeItalic">D. molestus</em>
                          </p></td><td><p xmlns="" class="SimplePara">Ngong Hills, Kenya</p></td><td><p xmlns="" class="SimplePara">1</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">38</p></td><td><p xmlns="" class="SimplePara">GQ999046</p></td><td><p xmlns="" class="SimplePara">
                            <em class="EmphasisTypeItalic">D. molestus</em>
                          </p></td><td><p xmlns="" class="SimplePara">Kilimanjaro, Tanzania</p></td><td><p xmlns="" class="SimplePara">1</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">39</p></td><td><p xmlns="" class="SimplePara">GQ999047</p></td><td><p xmlns="" class="SimplePara">
                            <em class="EmphasisTypeItalic">D. molestus</em>
                          </p></td><td><p xmlns="" class="SimplePara">Kilimanjaro, Tanzania</p></td><td><p xmlns="" class="SimplePara">1</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">40</p></td><td><p xmlns="" class="SimplePara">GQ999048</p></td><td><p xmlns="" class="SimplePara">
                            <em class="EmphasisTypeItalic">D. molestus</em>
                          </p></td><td><p xmlns="" class="SimplePara">North Pare, Tanzania</p></td><td><p xmlns="" class="SimplePara">1</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">41</p></td><td><p xmlns="" class="SimplePara">GQ999029</p></td><td><p xmlns="" class="SimplePara">
                            <em class="EmphasisTypeItalic">D. molestus</em>
                          </p></td><td><p xmlns="" class="SimplePara">Taita Hills, Kenya</p></td><td><p xmlns="" class="SimplePara">1</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">42</p></td><td><p xmlns="" class="SimplePara">GQ999044</p></td><td><p xmlns="" class="SimplePara">
                            <em class="EmphasisTypeItalic">D. molestus</em>
                          </p></td><td><p xmlns="" class="SimplePara">Taita Hills, Kenya</p></td><td><p xmlns="" class="SimplePara">1</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">43</p></td><td><p xmlns="" class="SimplePara">GQ999041</p></td><td><p xmlns="" class="SimplePara">
                            <em class="EmphasisTypeItalic">D. molestus</em>
                          </p></td><td><p xmlns="" class="SimplePara">Arabuko Sokoke, Kenya</p></td><td><p xmlns="" class="SimplePara">1</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">44</p></td><td><p xmlns="" class="SimplePara">GQ999025</p></td><td><p xmlns="" class="SimplePara">
                            <em class="EmphasisTypeItalic">D. molestus</em>
                          </p></td><td><p xmlns="" class="SimplePara">Shimba Hills, Kenya</p></td><td><p xmlns="" class="SimplePara">1</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">45</p></td><td><p xmlns="" class="SimplePara">GQ999026</p></td><td><p xmlns="" class="SimplePara">
                            <em class="EmphasisTypeItalic">D. molestus</em>
                          </p></td><td><p xmlns="" class="SimplePara">Gombe, Tanzania</p></td><td><p xmlns="" class="SimplePara">1</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">46</p></td><td><p xmlns="" class="SimplePara">EF413795</p></td><td><p xmlns="" class="SimplePara">
                            <em class="EmphasisTypeItalic">D. sjoestedti</em>
                          </p></td><td><p xmlns="" class="SimplePara">Ndoki, Republic of Congo</p></td><td><p xmlns="" class="SimplePara">1</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">47</p></td><td><p xmlns="" class="SimplePara">GQ999054</p></td><td><p xmlns="" class="SimplePara">
                            <em class="EmphasisTypeItalic">D. sjoestedti</em>
                          </p></td><td><p xmlns="" class="SimplePara">Salonga, DR Congo</p></td><td><p xmlns="" class="SimplePara">1</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">48</p></td><td><p xmlns="" class="SimplePara">EF413803</p></td><td><p xmlns="" class="SimplePara">
                            <em class="EmphasisTypeItalic">D. nigricans</em>
                          </p></td><td><p xmlns="" class="SimplePara">Tai, Ivory Coast</p></td><td><p xmlns="" class="SimplePara">1</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">49</p></td><td><p xmlns="" class="SimplePara">EF413773</p></td><td><p xmlns="" class="SimplePara">
                            <em class="EmphasisTypeItalic">D. emeryi</em>
                          </p></td><td><p xmlns="" class="SimplePara">Tai, Ivory Coast</p></td><td><p xmlns="" class="SimplePara">1</p></td></tr></tbody></table></div><div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="TableFooter">
                    <p xmlns="" class="SimplePara">Haplotypes that were 
sampled in different species (haplotype 1) or in different populations 
of the same species (haplotype 8) are listed twice.</p>
                  </div></figure>
              </section><section id="Sec5" class="Section2 RenderAsSection2">
                <h3 class="Heading">(c) Population Structure and Divergence</h3>
                <div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Para">We assessed the population structure and divergence among three populations of <em xmlns="" class="EmphasisTypeItalic">D. molestus</em> (Tana River, Mt. Kenya, Kakamega Forest) and one <em xmlns="" class="EmphasisTypeItalic">D. wilverthi</em>
 population (Kakamega Forest). None of the pair-wise tests for genotypic
 disequilibrium between microsatellite markers was significant (all p 
&gt; 0.05). Pair-wise estimates of genetic differentiation between four 
analysed populations at nuclear microsatellites (<em xmlns="" class="EmphasisTypeItalic">F</em><sub>
                    <em xmlns="" class="EmphasisTypeItalic">ST</em>
                  </sub> ) are given in Table <span class="InternalRef"><a href="#Tab2">2</a></span>. Estimates of <em xmlns="" class="EmphasisTypeItalic">F</em><sub>
                    <em xmlns="" class="EmphasisTypeItalic">ST</em>
                  </sub> between the <em xmlns="" class="EmphasisTypeItalic">D. wilverthi</em>-like and the <em xmlns="" class="EmphasisTypeItalic">D. molestus</em> populations were slightly but consistently higher than estimates between the different <em xmlns="" class="EmphasisTypeItalic">D. molestus</em> populations (Table <span class="InternalRef"><a href="#Tab2">2</a></span>; one-tailed t-test: t = -2.33, df = 4, p = 0.04).<figure xmlns="" id="Tab2" class="FigureTable"><figcaption class="Caption" lang="en"><div xmlns="http://www.w3.org/1999/xhtml" class="CaptionContent"><span class="CaptionNumber">Table 2</span>
                        <p xmlns="" class="SimplePara">Genetic differentiation between three populations of <em class="EmphasisTypeItalic">D.molestus</em> and one sympatric population of <em class="EmphasisTypeItalic">D. wilverthi</em></p>
                      </div></figcaption><div class="Table"><table xmlns="http://www.w3.org/1999/xhtml" border="1"><colgroup><col><col><col><col><col></colgroup><thead><tr class="header"><th>&nbsp;</th><th colspan="3"><p xmlns="" class="SimplePara">
                              <em class="EmphasisTypeItalic">D. molestus</em>
                            </p></th><th><p xmlns="" class="SimplePara">
                              <em class="EmphasisTypeItalic">D. wilverthi</em>
                            </p></th></tr><tr class="header"><th>&nbsp;</th><th><p xmlns="" class="SimplePara">1. Mt. Kenya</p></th><th><p xmlns="" class="SimplePara">2. Tana River</p></th><th><p xmlns="" class="SimplePara">3. Kakamega</p></th><th><p xmlns="" class="SimplePara">4. Kakamega</p></th></tr></thead><tbody><tr class="noclass"><td><p xmlns="" class="SimplePara">1.</p></td><td><p xmlns="" class="SimplePara">-</p></td><td><p xmlns="" class="SimplePara">0.05</p></td><td><p xmlns="" class="SimplePara">0.11</p></td><td><p xmlns="" class="SimplePara">0.14</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">2.</p></td><td><p xmlns="" class="SimplePara">48.9</p></td><td><p xmlns="" class="SimplePara">-</p></td><td><p xmlns="" class="SimplePara">0.12</p></td><td><p xmlns="" class="SimplePara">0.16</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">3.</p></td><td><p xmlns="" class="SimplePara">1.9</p></td><td><p xmlns="" class="SimplePara">4.1</p></td><td><p xmlns="" class="SimplePara">-</p></td><td><p xmlns="" class="SimplePara">0.14</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">4.</p></td><td><p xmlns="" class="SimplePara">3.8</p></td><td><p xmlns="" class="SimplePara">1.0</p></td><td><p xmlns="" class="SimplePara">18.2</p></td><td><p xmlns="" class="SimplePara">-</p></td></tr></tbody></table></div><div xmlns="http://www.w3.org/1999/xhtml" class="TableFooter">
                      <p xmlns="" class="SimplePara">Above diagonal: Pair-wise genetic differentiation measured as <em class="EmphasisTypeItalic">F</em><sub xmlns="http://www.w3.org/1999/xhtml">
                          <em xmlns="" class="EmphasisTypeItalic">ST</em>
                        </sub>. All estimates are significantly 
different from zero (p &lt; 0.05) after standard Bonferroni correction 
for multiple comparisons. Below diagonal: The percentage of individuals 
that were classified as hybrids (individuals with an inferred proportion
 of ancestry &lt; 0.9 in the correct source population) and "shared" 
between each pair of populations according to the STRUCTURE analysis 
(see main text for details).</p>
                    </div></figure></div>
                <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Para">The mitochondrial haplotype diversity was much lower in the <em xmlns="" class="EmphasisTypeItalic">D. wilverthi</em>-like population (<em xmlns="" class="EmphasisTypeItalic">N</em><sub>
                    <em xmlns="" class="EmphasisTypeItalic">a</em>
                  </sub> = 2; <em xmlns="" class="EmphasisTypeItalic">R</em><sub>
                    <em xmlns="" class="EmphasisTypeItalic">s</em>
                  </sub> = 1.55; N = 34) than in the <em xmlns="" class="EmphasisTypeItalic">D. molestus</em> populations (Kakamega: <em xmlns="" class="EmphasisTypeItalic">N</em><sub>
                    <em xmlns="" class="EmphasisTypeItalic">a</em>
                  </sub> = 7; <em xmlns="" class="EmphasisTypeItalic">R</em><sub>
                    <em xmlns="" class="EmphasisTypeItalic">s</em>
                  </sub> = 6.24; N = 42; Mt. Kenya: <em xmlns="" class="EmphasisTypeItalic">N</em><sub>
                    <em xmlns="" class="EmphasisTypeItalic">a</em>
                  </sub> = 8; <em xmlns="" class="EmphasisTypeItalic">R</em><sub>
                    <em xmlns="" class="EmphasisTypeItalic">s</em>
                  </sub> = 6.94; N = 29; Tana River: <em xmlns="" class="EmphasisTypeItalic">N</em><sub>
                    <em xmlns="" class="EmphasisTypeItalic">a</em>
                  </sub> = 5; <em xmlns="" class="EmphasisTypeItalic">R</em><sub>
                    <em xmlns="" class="EmphasisTypeItalic">s</em>
                  </sub> = 5.00; N = 11). Interestingly, the <em xmlns="" class="EmphasisTypeItalic">D. wilverthi</em>-like population was close to fixation (allele frequency = 0.97) for a haplotype that was also common in the sympatric <em xmlns="" class="EmphasisTypeItalic">D. molestus</em>-like population at Kakamega (allele frequency = 0.19) and was closely related to the other local <em xmlns="" class="EmphasisTypeItalic">D. molestus</em> haplotypes (Figures <span class="InternalRef"><a href="#Fig3">3</a></span> and <span class="InternalRef"><a href="#Fig4">4</a></span>).
 At the same time this is the only haplotype that is shared between any 
of the four populations. The second rare haplotype found in <em xmlns="" class="EmphasisTypeItalic">D. wilverthi</em> from Kakamega is closely related to the common haplotype and differs only by a single base-pair (Figure <span class="InternalRef"><a href="#Fig4">4</a></span>).</p>
                <div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Para">Replicate
 runs in STRUCTURE were highly consistent in every analysis. The highest
 likelihood scores were associated with a model with four subdivisions (<em xmlns="" class="EmphasisTypeItalic">k</em> = 4, mean ln <em xmlns="" class="EmphasisTypeItalic">L</em> over five replicate runs = -3185.36; 0.23 SD; for comparison: <em xmlns="" class="EmphasisTypeItalic">k</em> = 2, -3409.86; 0.11 SD; <em xmlns="" class="EmphasisTypeItalic">k</em> = 3, -3245.90; 0.28 SD; all <em xmlns="" class="EmphasisTypeItalic">k</em> &gt; 4 had lower average likelihoods than <em xmlns="" class="EmphasisTypeItalic">k</em> = 4), although a model with five subdivisions had very similar likelihood scores (<em xmlns="" class="EmphasisTypeItalic">k</em>
 = 5, -3185.56; 0.81 SD). The four clusters corresponded well to the 
four populations, although a significant proportion of individuals had 
reasonably high likelihoods of ancestry in a population other than their
 source population (Figure <span class="InternalRef"><a href="#Fig5">5</a></span>; Table <span class="InternalRef"><a href="#Tab2">2</a></span>). By far the strongest uncertainty over individual assignments was found between the <em xmlns="" class="EmphasisTypeItalic">D. wilverthi</em>-like and <em xmlns="" class="EmphasisTypeItalic">D. molestus</em>-like populations at Kakamega Forest, as well as between the Mt. Kenya and Tana River populations of <em xmlns="" class="EmphasisTypeItalic">D. molestus</em>. When set to divide the samples in two groups (<em xmlns="" class="EmphasisTypeItalic">k</em>
 = 2), STRUCTURE consistently recovered one cluster consisting of the 
two Kakamega populations and another cluster consisting of <em xmlns="" class="EmphasisTypeItalic">D. molestus</em> samples from Tana River and Mt. Kenya, rather than combining the three <em xmlns="" class="EmphasisTypeItalic">D. molestus</em> populations. At <em xmlns="" class="EmphasisTypeItalic">k</em> = 3, STRUCTURE split the two Kakamega populations, and at <em xmlns="" class="EmphasisTypeItalic">k</em>
 = 5 the Tana River + Mt. Kenya samples generally formed three clusters,
 while the two Kakamega clusters were retained (results not shown).<figure xmlns="" class="Figure" id="Fig5"><div xmlns="http://www.w3.org/1999/xhtml" class="MediaObject">
                      <a target="_blank" rel="noopener" href="https://media.springernature.com/full/springer-static/image/art%3A10.1186%2F1742-9994-8-20/MediaObjects/12983_2011_Article_166_Fig5_HTML.jpg"><img class="c-lazy-image u-js-shown" src="12983_2011_Article_166_Fig5_HTML.jpg" alt="Figure 5" data-lazy-load="true" data-src="https://media.springernature.com/lw785/springer-static/image/art%3A10.1186%2F1742-9994-8-20/MediaObjects/12983_2011_Article_166_Fig5_HTML.jpg"><noscript><img src="https://media.springernature.com/lw785/springer-static/image/art%3A10.1186%2F1742-9994-8-20/MediaObjects/12983_2011_Article_166_Fig5_HTML.jpg" alt="Figure 5"></img></noscript></a>
                    </div><figcaption class="Caption" lang="en"><div xmlns="http://www.w3.org/1999/xhtml" class="CaptionContent"><span class="CaptionNumber">Figure 5</span>
                        <p xmlns="" class="SimplePara"><strong class="EmphasisTypeBold">Assignment of individuals to four populations using the program STRUCTURE without</strong> <span class="EmphasisTypeBoldItalic">a priori</span> <strong class="EmphasisTypeBold">assumptions</strong>. Shown are assignments of individuals from the <em class="EmphasisTypeItalic">D. wilverthi</em>-like population at Kakamega Forest, the <em class="EmphasisTypeItalic">D. molestus-like</em> population at Kakamega Forest, and the pure <em class="EmphasisTypeItalic">D. molestus</em>
 populations at Tana River and Mt. Kenya. The y-axis represents the 
proportion of each multilocus genotype that is attributable to each of 
the four populations. Samples are grouped along the x-axis according to 
their population of origin.</p>
                      </div></figcaption></figure></div>
                <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Para">To
 quantify the proportion of hybrid workers at Kakamega, we arbitrarily 
classified individuals with an inferred proportion of ancestry &lt; 0.9 
in the correct source population as hybrids. Each hybrid individual was 
then determined to be "shared" between its source population and the 
population in which it had the highest proportion of ancestry among the 
non-source populations. The proportions of individuals that are "shared"
 between each pair of populations are given in Table <span class="InternalRef"><a href="#Tab2">2</a></span>.
 According to this estimate, 18.2% of the genotyped workers from 
Kakamega Forest were hybrids. If we lower the cut-off value for the 
inferred proportion of ancestry from 0.9 to 0.75, still 7.3% of the 
Kakamega workers were hybrids (pairwise comparisons between all 
populations not shown).</p>
                <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Para">Results from a GENECLASS2 analysis were fully consistent with the results obtained from STRUCTURE. Five <em xmlns="" class="EmphasisTypeItalic">D. wilverthi</em>-like individuals were assigned to the <em xmlns="" class="EmphasisTypeItalic">D. molestus</em>-like population from Kakamega (the <em xmlns="" class="EmphasisTypeItalic">D. molestus</em>-like
 population from Kakamega had the highest probability of being the 
source population among the four potential source populations), and one <em xmlns="" class="EmphasisTypeItalic">D. molestus</em>-like worker from Kakamega was assigned to the <em xmlns="" class="EmphasisTypeItalic">D. wilverthi</em>-like population. Similarly, two <em xmlns="" class="EmphasisTypeItalic">D. molestus</em>
 workers from Mt. Kenya were assigned to the Tana River population, and 
six workers from Tana River were assigned to the Mt. Kenya population. 
No other cross-assignments were found.</p>
              </section></div></section>
            <section xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" id="Sec6" class="Section1 RenderAsSection1 c-section"><h2 class="Heading js-ToggleCollapseSection c-section__heading" data-sticky-update="true"><a href="#Discussion" data-toggle="collapse" data-toggle-disable-clickoff="" data-target="Discussion">Discussion</a></h2><div class="js-CollapseSection CollapseSection c-section__content" id="Discussion" data-test="collapse-section"><section id="Sec7" class="Section2 RenderAsSection2">
                <h3 class="Heading">(a) Morphological analysis</h3>
                <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Para">The
 morphological and morphometric data show that two forms of 
swarm-raiding army ants can be distinguished at Kakamega Forest. These 
two forms strongly resemble <em xmlns="" class="EmphasisTypeItalic">D. molestus</em> and <em xmlns="" class="EmphasisTypeItalic">D. wilverthi</em> from allopatric populations, but are also significantly different from "pure" <em xmlns="" class="EmphasisTypeItalic">D. molestus</em> and <em xmlns="" class="EmphasisTypeItalic">D. wilverthi</em> with intermediate phenotypes for several key characters (Figure <span class="InternalRef"><a href="#Fig2">2</a></span>). This suggests that both forms have arisen from hybridization between <em xmlns="" class="EmphasisTypeItalic">D. molestus</em> and <em xmlns="" class="EmphasisTypeItalic">D. wilverthi</em>
 at Kakamega Forest. At the same time, gene-flow has not been sufficient
 to completely collapse the two species into a single homogeneous hybrid
 population.</p>
              </section><section id="Sec8" class="Section2 RenderAsSection2">
                <h3 class="Heading">(b) Mitochondrial phylogeny and introgression</h3>
                <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Para">While mitochondrial haplotypes are clearly structured according to geographic origin (Figures <span class="InternalRef"><a href="#Fig3">3</a></span> and <span class="InternalRef"><a href="#Fig4">4</a></span>), different <em xmlns="" class="EmphasisTypeItalic">Dorylus</em> (<em xmlns="" class="EmphasisTypeItalic">Anomma</em>)
 species are not generally recovered as monophyletic. Below we argue 
that this finding is best explained by the combined effects of 
unrecognized cryptic diversity within <em xmlns="" class="EmphasisTypeItalic">D. molestus</em> and a complex history of mitochondrial introgression in the Lake Victoria region. <em xmlns="" class="EmphasisTypeItalic">D. molestus</em>
 haplotypes tend to cluster into several well supported shallow clades, 
which are connected to other such clades by deeper branches (Figure <span class="InternalRef"><a href="#Fig3">3</a></span>). Furthermore, the Central African <em xmlns="" class="EmphasisTypeItalic">D. sjoestedti</em> samples cluster with <em xmlns="" class="EmphasisTypeItalic">D. molestus</em>
 samples from coastal Kenya and Tanzania, a finding that cannot be 
explained by introgression due to the large geographic distances and the
 fact that more western samples of <em xmlns="" class="EmphasisTypeItalic">D. molestus</em> do not have haplotypes similar to those of <em xmlns="" class="EmphasisTypeItalic">D. sjoestedti</em>.
 Incomplete lineage sorting seems similarly unlikely as an explanation 
(see below), so that this overall pattern is suggestive of unrecognized 
cryptic diversity within <em xmlns="" class="EmphasisTypeItalic">D. molestus</em>.</p>
                <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Para">The
 fact that identical or very similar haplotypes are shared between 
different species in spatially close populations around Lake Victoria, 
on the other hand, points towards a complex history of mitochondrial 
introgression in this narrower geographic area. The most striking 
example is found at Kakamega Forest, where the <em xmlns="" class="EmphasisTypeItalic">D. wilverthi</em>-like form is close to fixation for a haplotype that is also frequent in the <em xmlns="" class="EmphasisTypeItalic">D. molestus</em>-like form. This haplotype is nested within the clade of Kakamega <em xmlns="" class="EmphasisTypeItalic">D. molestus</em> haplotypes (clade 1 in Figure <span class="InternalRef"><a href="#Fig3">3</a></span>).
 The clade of mitochondrial haplotypes from Kakamega Forest is in turn 
nested within a larger clade of haplotypes from the Lake Victoria region
 (clade 2 in Figure <span class="InternalRef"><a href="#Fig3">3</a></span>), which contains additional haplotypes from <em xmlns="" class="EmphasisTypeItalic">D. molestus</em>, <em xmlns="" class="EmphasisTypeItalic">D. terrificus</em>, and <em xmlns="" class="EmphasisTypeItalic">D. wilverthi</em>. This pattern indicates a complex history of repeated mitochondrial introgression between different pairs of species.</p>
                <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Para">The
 combination of two main factors makes it difficult to determine the 
original source species of the Lake Victoria haplotypes: 1. We presently
 lack haplotype information for populations of <em xmlns="" class="EmphasisTypeItalic">D. wilverthi</em> and <em xmlns="" class="EmphasisTypeItalic">D. terrificus</em> that have clearly not undergone mitochondrial introgression, i.e. we are missing "true" <em xmlns="" class="EmphasisTypeItalic">D. wilverthi</em> and <em xmlns="" class="EmphasisTypeItalic">D. terrificus</em>
 haplotypes in our analysis. 2. Mitochondrial DNA undergoes frequent 
selective sweeps, and recurrent sweeps are likely to mask previous 
introgression events [<span class="CitationRef"><a href="#CR26" data-toggle="citation">26</a></span>, <span class="CitationRef"><a href="#CR27" data-toggle="citation">27</a></span>].</p>
                <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Para">During Pleistocene and Holocene glacial cycles [<span class="CitationRef"><a href="#CR28" data-toggle="citation">28</a></span>, <span class="CitationRef"><a href="#CR29" data-toggle="citation">29</a></span>], the Lake Victoria region has seen recurrent expansions and retractions of forest and savannah habitat [<span class="CitationRef"><a href="#CR30" data-toggle="citation">30</a></span><span>, <span class="CitationRef"><a href="#CR31" data-toggle="citation">31</a></span>, </span><span class="CitationRef"><a href="#CR32" data-toggle="citation">32</a></span>], so it is safe to assume that the distributional ranges of rainforest specialists like <em xmlns="" class="EmphasisTypeItalic">D. wilverthi</em> and savannah/forest generalists like <em xmlns="" class="EmphasisTypeItalic">D. molestus</em>
 have changed accordingly. The two species probably broadly overlapped 
during glacial periods of maximum forest expansion, when Kakamega Forest
 was connected to Ugandan rainforests [<span class="CitationRef"><a href="#CR30" data-toggle="citation">30</a></span>], while <em xmlns="" class="EmphasisTypeItalic">D. wilverthi</em>
 has been restricted to forest islands during interglacial periods as is
 the case at present. Given this complex recent biogeographic history, 
it will also be challenging to associate the observed cases of 
introgressive hybridization with discrete historic events.</p>
                <div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Para">It
 is interesting to note that only a single mitochondrial haplotype is 
shared between the two forms at Kakamega, while essentially all other 
haplotypes are restricted to the <em xmlns="" class="EmphasisTypeItalic">D. molestus</em>-like
 form. One possible explanation is that mitochondrial gene-flow at 
Kakamega goes back to a single or very few events, which by chance 
involved only that particular haplotype. Alternatively, the pattern 
could be explained by selection. For example, all other <em xmlns="" class="EmphasisTypeItalic">D. molestus</em>-like haplotypes, or potential cytoplasmic factors coupled to those haplotypes (like <em xmlns="" class="EmphasisTypeItalic">Wolbachia</em> infections [<span class="CitationRef"><a href="#CR33" data-toggle="citation">33</a></span>]), might have epistatic incompatibilities with parts of the <em xmlns="" class="EmphasisTypeItalic">D. wilverthi</em> nuclear genome. That the reduced mitochondrial diversity in the <em xmlns="" class="EmphasisTypeItalic">D. wilverthi</em>-like
 form at Kakamega is simply due to a recent genetic bottleneck seems 
unlikely because diversity at nuclear microsatellite loci is not reduced
 compared to other populations (Table <span class="InternalRef"><a href="#Tab3">3</a></span>).<figure xmlns="" id="Tab3" class="FigureTable"><figcaption class="Caption" lang="en"><div xmlns="http://www.w3.org/1999/xhtml" class="CaptionContent"><span class="CaptionNumber">Table 3</span>
                        <p xmlns="" class="SimplePara">Number of alleles (<em class="EmphasisTypeItalic">N</em><sub xmlns="http://www.w3.org/1999/xhtml">
                            <em xmlns="" class="EmphasisTypeItalic">a</em>
                          </sub>), expected heterozygosity (<em class="EmphasisTypeItalic">H</em><sub xmlns="http://www.w3.org/1999/xhtml">
                            <em xmlns="" class="EmphasisTypeItalic">s</em>
                          </sub>), allelic richness (<em class="EmphasisTypeItalic">R</em><sub xmlns="http://www.w3.org/1999/xhtml">
                            <em xmlns="" class="EmphasisTypeItalic">s</em>
                          </sub>), and inbreeding coefficient (<em class="EmphasisTypeItalic">F</em><sub xmlns="http://www.w3.org/1999/xhtml">
                            <em xmlns="" class="EmphasisTypeItalic">IS</em>
                          </sub>) estimated for five microsatellite loci and three populations of <em class="EmphasisTypeItalic">D.molestus</em> and one sympatric population of <em class="EmphasisTypeItalic">D. wilverthi</em></p>
                      </div></figcaption><div class="Table"><table xmlns="http://www.w3.org/1999/xhtml" border="1"><colgroup><col><col><col><col><col><col><col><col><col><col><col><col><col><col><col><col><col></colgroup><thead><tr class="header"><th>&nbsp;</th><th colspan="12"><p xmlns="" class="SimplePara">
                              <em class="EmphasisTypeItalic">D. molestus</em>
                            </p></th><th colspan="4"><p xmlns="" class="SimplePara">
                              <em class="EmphasisTypeItalic">D. wilverthi</em>
                            </p></th></tr><tr class="header"><th>&nbsp;</th><th colspan="4"><p xmlns="" class="SimplePara">Mt. Kenya</p><p xmlns="" class="SimplePara">(N = 50)</p></th><th colspan="4"><p xmlns="" class="SimplePara">Tana River</p><p xmlns="" class="SimplePara">(N = 42)</p></th><th colspan="4"><p xmlns="" class="SimplePara">Kakamega</p><p xmlns="" class="SimplePara">(N = 55)&gt;</p></th><th colspan="4"><p xmlns="" class="SimplePara">Kakamega</p><p xmlns="" class="SimplePara">(N = 55)</p></th></tr><tr class="header"><th><p xmlns="" class="SimplePara">Locus</p></th><th><p xmlns="" class="SimplePara">
                              <em class="EmphasisTypeItalic">N</em>
                              <sub xmlns="http://www.w3.org/1999/xhtml">
                                <em xmlns="" class="EmphasisTypeItalic">a</em>
                              </sub>
                            </p></th><th><p xmlns="" class="SimplePara">
                              <em class="EmphasisTypeItalic">H</em>
                              <sub xmlns="http://www.w3.org/1999/xhtml">
                                <em xmlns="" class="EmphasisTypeItalic">s</em>
                              </sub>
                            </p></th><th><p xmlns="" class="SimplePara">
                              <em class="EmphasisTypeItalic">R</em>
                              <sub xmlns="http://www.w3.org/1999/xhtml">
                                <em xmlns="" class="EmphasisTypeItalic">s</em>
                              </sub>
                            </p></th><th><p xmlns="" class="SimplePara">
                              <em class="EmphasisTypeItalic">F</em>
                              <sub xmlns="http://www.w3.org/1999/xhtml">
                                <em xmlns="" class="EmphasisTypeItalic">IS</em>
                              </sub>
                            </p></th><th><p xmlns="" class="SimplePara">
                              <em class="EmphasisTypeItalic">N</em>
                              <sub xmlns="http://www.w3.org/1999/xhtml">
                                <em xmlns="" class="EmphasisTypeItalic">a</em>
                              </sub>
                            </p></th><th><p xmlns="" class="SimplePara">
                              <em class="EmphasisTypeItalic">H</em>
                              <sub xmlns="http://www.w3.org/1999/xhtml">
                                <em xmlns="" class="EmphasisTypeItalic">s</em>
                              </sub>
                            </p></th><th><p xmlns="" class="SimplePara">
                              <em class="EmphasisTypeItalic">R</em>
                              <sub xmlns="http://www.w3.org/1999/xhtml">
                                <em xmlns="" class="EmphasisTypeItalic">s</em>
                              </sub>
                            </p></th><th><p xmlns="" class="SimplePara">
                              <em class="EmphasisTypeItalic">F</em>
                              <sub xmlns="http://www.w3.org/1999/xhtml">
                                <em xmlns="" class="EmphasisTypeItalic">IS</em>
                              </sub>
                            </p></th><th><p xmlns="" class="SimplePara">
                              <em class="EmphasisTypeItalic">N</em>
                              <sub xmlns="http://www.w3.org/1999/xhtml">
                                <em xmlns="" class="EmphasisTypeItalic">a</em>
                              </sub>
                            </p></th><th><p xmlns="" class="SimplePara">
                              <em class="EmphasisTypeItalic">H</em>
                              <sub xmlns="http://www.w3.org/1999/xhtml">
                                <em xmlns="" class="EmphasisTypeItalic">s</em>
                              </sub>
                            </p></th><th><p xmlns="" class="SimplePara">
                              <em class="EmphasisTypeItalic">R</em>
                              <sub xmlns="http://www.w3.org/1999/xhtml">
                                <em xmlns="" class="EmphasisTypeItalic">s</em>
                              </sub>
                            </p></th><th><p xmlns="" class="SimplePara">
                              <em class="EmphasisTypeItalic">F</em>
                              <sub xmlns="http://www.w3.org/1999/xhtml">
                                <em xmlns="" class="EmphasisTypeItalic">IS</em>
                              </sub>
                            </p></th><th><p xmlns="" class="SimplePara">
                              <em class="EmphasisTypeItalic">N</em>
                              <sub xmlns="http://www.w3.org/1999/xhtml">
                                <em xmlns="" class="EmphasisTypeItalic">a</em>
                              </sub>
                            </p></th><th><p xmlns="" class="SimplePara">
                              <em class="EmphasisTypeItalic">H</em>
                              <sub xmlns="http://www.w3.org/1999/xhtml">
                                <em xmlns="" class="EmphasisTypeItalic">s</em>
                              </sub>
                            </p></th><th><p xmlns="" class="SimplePara">
                              <em class="EmphasisTypeItalic">R</em>
                              <sub xmlns="http://www.w3.org/1999/xhtml">
                                <em xmlns="" class="EmphasisTypeItalic">s</em>
                              </sub>
                            </p></th><th><p xmlns="" class="SimplePara">
                              <em class="EmphasisTypeItalic">F</em>
                              <sub xmlns="http://www.w3.org/1999/xhtml">
                                <em xmlns="" class="EmphasisTypeItalic">IS</em>
                              </sub>
                            </p></th></tr></thead><tbody><tr class="noclass"><td><p xmlns="" class="SimplePara">DmoB</p></td><td><p xmlns="" class="SimplePara">4</p></td><td><p xmlns="" class="SimplePara">0.56</p></td><td><p xmlns="" class="SimplePara">3.82</p></td><td><p xmlns="" class="SimplePara">-0.18</p></td><td><p xmlns="" class="SimplePara">3</p></td><td><p xmlns="" class="SimplePara">0.55</p></td><td><p xmlns="" class="SimplePara">3.00</p></td><td><p xmlns="" class="SimplePara">0.09</p></td><td><p xmlns="" class="SimplePara">5</p></td><td><p xmlns="" class="SimplePara">0.65</p></td><td><p xmlns="" class="SimplePara">4.97</p></td><td><p xmlns="" class="SimplePara">0.05</p></td><td><p xmlns="" class="SimplePara">7</p></td><td><p xmlns="" class="SimplePara">0.55</p></td><td><p xmlns="" class="SimplePara">6.84</p></td><td><p xmlns="" class="SimplePara">0.04</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">DmoC</p></td><td><p xmlns="" class="SimplePara">4</p></td><td><p xmlns="" class="SimplePara">0.57</p></td><td><p xmlns="" class="SimplePara">4.00</p></td><td><p xmlns="" class="SimplePara">0.07</p></td><td><p xmlns="" class="SimplePara">5</p></td><td><p xmlns="" class="SimplePara">0.68</p></td><td><p xmlns="" class="SimplePara">5.00</p></td><td><p xmlns="" class="SimplePara">0.28</p></td><td><p xmlns="" class="SimplePara">5</p></td><td><p xmlns="" class="SimplePara">0.69</p></td><td><p xmlns="" class="SimplePara">5.00</p></td><td><p xmlns="" class="SimplePara">0.11</p></td><td><p xmlns="" class="SimplePara">4</p></td><td><p xmlns="" class="SimplePara">0.52</p></td><td><p xmlns="" class="SimplePara">3.75</p></td><td><p xmlns="" class="SimplePara">0.23</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">DmoD</p></td><td><p xmlns="" class="SimplePara">5</p></td><td><p xmlns="" class="SimplePara">0.77</p></td><td><p xmlns="" class="SimplePara">5.00</p></td><td><p xmlns="" class="SimplePara">-0.05</p></td><td><p xmlns="" class="SimplePara">8</p></td><td><p xmlns="" class="SimplePara">0.85</p></td><td><p xmlns="" class="SimplePara">8.00</p></td><td><p xmlns="" class="SimplePara">-0.04</p></td><td><p xmlns="" class="SimplePara">8</p></td><td><p xmlns="" class="SimplePara">0.81</p></td><td><p xmlns="" class="SimplePara">7.49</p></td><td><p xmlns="" class="SimplePara">0.02</p></td><td><p xmlns="" class="SimplePara">7</p></td><td><p xmlns="" class="SimplePara">0.74</p></td><td><p xmlns="" class="SimplePara">6.67</p></td><td><p xmlns="" class="SimplePara">0.14</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">DmoG</p></td><td><p xmlns="" class="SimplePara">11</p></td><td><p xmlns="" class="SimplePara">0.86</p></td><td><p xmlns="" class="SimplePara">10.46</p></td><td><p xmlns="" class="SimplePara">0.19*</p></td><td><p xmlns="" class="SimplePara">12</p></td><td><p xmlns="" class="SimplePara">0.86</p></td><td><p xmlns="" class="SimplePara">11.98</p></td><td><p xmlns="" class="SimplePara">0.11</p></td><td><p xmlns="" class="SimplePara">9</p></td><td><p xmlns="" class="SimplePara">0.82</p></td><td><p xmlns="" class="SimplePara">8.49</p></td><td><p xmlns="" class="SimplePara">-0.02</p></td><td><p xmlns="" class="SimplePara">6</p></td><td><p xmlns="" class="SimplePara">0.72</p></td><td><p xmlns="" class="SimplePara">6.00</p></td><td><p xmlns="" class="SimplePara">-0.04</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">DmoO</p></td><td><p xmlns="" class="SimplePara">9</p></td><td><p xmlns="" class="SimplePara">0.86</p></td><td><p xmlns="" class="SimplePara">8.97</p></td><td><p xmlns="" class="SimplePara">0.26*</p></td><td><p xmlns="" class="SimplePara">8</p></td><td><p xmlns="" class="SimplePara">0.71</p></td><td><p xmlns="" class="SimplePara">7.95</p></td><td><p xmlns="" class="SimplePara">0.07</p></td><td><p xmlns="" class="SimplePara">9</p></td><td><p xmlns="" class="SimplePara">0.77</p></td><td><p xmlns="" class="SimplePara">8.99</p></td><td><p xmlns="" class="SimplePara">-0.04</p></td><td><p xmlns="" class="SimplePara">9</p></td><td><p xmlns="" class="SimplePara">0.79</p></td><td><p xmlns="" class="SimplePara">8.67</p></td><td><p xmlns="" class="SimplePara">0.10</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">mean</p></td><td><p xmlns="" class="SimplePara">6.6</p></td><td><p xmlns="" class="SimplePara">0.72</p></td><td><p xmlns="" class="SimplePara">6.45</p></td><td><p xmlns="" class="SimplePara">0.06</p></td><td><p xmlns="" class="SimplePara">7.2</p></td><td><p xmlns="" class="SimplePara">0.73</p></td><td><p xmlns="" class="SimplePara">7.19</p></td><td><p xmlns="" class="SimplePara">0.10</p></td><td><p xmlns="" class="SimplePara">7.2</p></td><td><p xmlns="" class="SimplePara">0.75</p></td><td><p xmlns="" class="SimplePara">6.99</p></td><td><p xmlns="" class="SimplePara">0.02</p></td><td><p xmlns="" class="SimplePara">6.6</p></td><td><p xmlns="" class="SimplePara">0.66</p></td><td><p xmlns="" class="SimplePara">6.39</p></td><td><p xmlns="" class="SimplePara">0.09</p></td></tr></tbody></table></div><div xmlns="http://www.w3.org/1999/xhtml" class="TableFooter">
                      <p xmlns="" class="SimplePara"><em class="EmphasisTypeItalic">F</em><sub xmlns="http://www.w3.org/1999/xhtml">
                          <em xmlns="" class="EmphasisTypeItalic">IS</em>
                        </sub> estimates significantly different from 
zero (p &lt; 0.05 after standard Bonferroni correction for multiple 
comparisons) are marked with an asterisk.</p>
                    </div></figure></div>
                <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Para">Another
 noteworthy case of apparent mitochondrial introgression was found in 
the population from Tana River, where haplotypes from both sides of the 
river tend to cluster in clearly distinct clades (Figures <span class="InternalRef"><a href="#Fig3">3</a></span> and <span class="InternalRef"><a href="#Fig4">4</a></span>)
 although, according to microsatellite data (see below), all samples 
belong to a single population. This suggests that at least one side of 
the river has captured haplotypes from a more distantly related <em xmlns="" class="EmphasisTypeItalic">D. molestus</em>
 clade. This strong genetic structure at the maternally inherited 
mitochondrial locus, and a lack thereof at biparentally inherited 
nuclear loci, stems from the fact that the permanently wingless army ant
 queens cannot cross water barriers, while the winged males readily do 
so [<span class="CitationRef"><a href="#CR18" data-toggle="citation">18</a></span>].</p>
                <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Para">Alternative
 explanations for a lack of monophyly between closely related species 
usually involve the retention of ancestral polymorphisms or incomplete 
lineage sorting. However, these mechanisms are less likely to apply to 
mtDNA as compared to nuclear DNA because ancestral polymorphisms will be
 lost more quickly due to genetic drift given the smaller effective 
population size of mtDNA and because mtDNA in insects evolves 
significantly faster than nuclear DNA (e.g. [<span class="CitationRef"><a href="#CR34" data-toggle="citation">34</a></span>]).
 On the other hand, mtDNA seems to be particularly prone to 
introgression, at least in species where males are the heterogametic or 
haploid sex (e.g. [<span class="CitationRef"><a href="#CR34" data-toggle="citation">34</a></span><span>, <span class="CitationRef"><a href="#CR35" data-toggle="citation">35</a></span>, <span class="CitationRef"><a href="#CR36" data-toggle="citation">36</a></span>, </span><span class="CitationRef"><a href="#CR37" data-toggle="citation">37</a></span>]). One reason for this is that in such species males tend to suffer more from hybridization than females (Haldane's rule; [<span class="CitationRef"><a href="#CR38" data-toggle="citation">38</a></span>]),
 so that gene-flow will be more restricted at nuclear loci relative to 
maternally inherited mitochondrial loci. Furthermore, if incomplete 
lineage sorting occurs at random it should not produce an obvious 
correlation with geographic distribution (e.g. [<span class="CitationRef"><a href="#CR39" data-toggle="citation">39</a></span>]), as we observed in East African <em xmlns="" class="EmphasisTypeItalic">Dorylus</em> (<em xmlns="" class="EmphasisTypeItalic">Anomma</em>)
 species. Under incomplete lineage sorting, the minimum divergence time 
between sequences is given by the time elapsed since speciation [<span class="CitationRef"><a href="#CR40" data-toggle="citation">40</a></span>]. In the absence of gene flow, mutations should therefore accumulate quickly in <em xmlns="" class="EmphasisTypeItalic">cytochrome oxidase II</em> haplotypes, which have high mutation rates of ca. 1.5% per million years in insects (e.g. [<span class="CitationRef"><a href="#CR41" data-toggle="citation">41</a></span>]). Given the strong geographic signal in our mtDNA phylogeny (Figures <span class="InternalRef"><a href="#Fig3">3</a></span> and <span class="InternalRef"><a href="#Fig4">4</a></span>),
 the general lack of overlapping haplotypes between neighbouring 
conspecific populations, and the fact that sympatric species at Kakamega
 share identical haplotypes for fast evolving mitochondrial genes, we 
conclude that introgressive hybridization between army ant species is 
clearly the best explanation for the observed haplotype distribution in 
the Lake Victoria area.</p>
              </section><section id="Sec9" class="Section2 RenderAsSection2">
                <h3 class="Heading">(c) Differentiation and hybridization at nuclear DNA</h3>
                <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Para">The
 two morphological forms at Kakamega Forest can be clearly distinguished
 based on the nuclear microsatellite loci. This again shows that species
 boundaries have not become completely blurred, which would be expected 
if gene flow had been significant over extended periods of time. On the 
other hand, despite genetic differentiation among species being 
consistently higher than between conspecific populations (Table <span class="InternalRef"><a href="#Tab2">2</a></span>),
 a large proportion of workers at Kakamega could not be unequivocally 
assigned to one of the two forms by either STRUCTURE or GENECLASS2 
(Figure <span class="InternalRef"><a href="#Fig5">5</a></span>). At the 
same time, pair-wise assignments between populations were far less 
problematic, even in conspecific comparisons. The only exception was the
 Tana River/Mt. Kenya comparison, which showed very weak differentiation
 making assignments difficult. The reason for this genetic similarity 
may well be that these two populations were connected via contiguous 
gallery forests along the Tana River until the recent past. Overall, 
this implies that a considerable proportion of workers at Kakamega 
Forest are hybrids between <em xmlns="" class="EmphasisTypeItalic">D. molestus</em> and <em xmlns="" class="EmphasisTypeItalic">D. wilverthi</em>.</p>
              </section></div></section>
            <section xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" id="Sec10" class="Section1 RenderAsSection1 c-section"><h2 class="Heading js-ToggleCollapseSection c-section__heading" data-sticky-update="true"><a href="#Conclusions" data-toggle="collapse" data-toggle-disable-clickoff="" data-target="Conclusions">Conclusions</a></h2><div class="js-CollapseSection CollapseSection c-section__content" id="Conclusions" data-test="collapse-section"><p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Para">This study provides conclusive evidence that introgressive hybridization has occurred between different <em xmlns="" class="EmphasisTypeItalic">Dorylus (Anomma)</em> species. This is one of the few cases of documented gene flow between species of eusocial Hymentoptera ([<span class="CitationRef"><a href="#CR11" data-toggle="citation">11</a></span>]; but see [<span class="CitationRef"><a href="#CR42" data-toggle="citation">42</a></span>, <span class="CitationRef"><a href="#CR43" data-toggle="citation">43</a></span>]
 for other recent examples) and shows that reproductive isolation 
between species of East African swarm-raiding army ants is incomplete. 
The frequent occurrence of hybrid workers at Kakamega means that the two
 local forms still mate regularly today. However, it is important to 
keep in mind that hybrid workers do not necessarily translate into gene 
flow, because social insect workers normally do not reproduce. This also
 applies to <em xmlns="" class="EmphasisTypeItalic">Dorylus</em> (<em xmlns="" class="EmphasisTypeItalic">Anomma</em>) [<span class="CitationRef"><a href="#CR19" data-toggle="citation">19</a></span>, <span class="CitationRef"><a href="#CR44" data-toggle="citation">44</a></span>].
 The general notion that hybrid workers are common between several pairs
 of closely related ant species, while hybrid queens and males are 
extremely uncommon [<span class="CitationRef"><a href="#CR11" data-toggle="citation">11</a></span>],
 might explain why the two forms at Kakamega have retained their 
distinct morphology and ecological niche space, instead of having 
collapsed into a single homogeneous hybrid population. Hybridization in 
army ants is particularly surprising because, unlike in most social 
insects where mating occurs outside the nest, workers have control over 
which males obtain access to a virgin queen. Therefore, if hybridization
 would imply reduced colony fitness, workers should be under strong 
selection to recognize heterospecific males and prevent them from 
mating. This suggests that hybridization in army ants entails little or 
no fitness costs or might even be adaptive under certain circumstances. 
Future work will have to investigate the stability of the hybrid zone at
 Kakamega over time, and the effects of hybridization on the actual 
genotypes of <em xmlns="" class="EmphasisTypeItalic">Dorylus</em> queens
 and males, to clarify the precise pattern of hybridization and the 
amount of ongoing gene-flow between the two sympatric forms.</p></div></section>
            <section xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" id="Sec11" class="Section1 RenderAsSection1 c-section"><h2 class="Heading js-ToggleCollapseSection c-section__heading" data-sticky-update="true"><a href="#Materialsandmethods" data-toggle="collapse" data-toggle-disable-clickoff="" data-target="Materialsandmethods">Materials and methods</a></h2><div class="js-CollapseSection CollapseSection c-section__content" id="Materialsandmethods" data-test="collapse-section"><section id="Sec12" class="Section2 RenderAsSection2">
                <h3 class="Heading">(a) Sampling and morphological analysis</h3>
                <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Para">To study patterns of mitochondrial differentiation between species and populations of East African <em xmlns="" class="EmphasisTypeItalic">Dorylus</em> (<em xmlns="" class="EmphasisTypeItalic">Anomma</em>)
 army ants, we collected 151 colony samples (one individual per colony) 
for mitochondrial DNA sequencing from 24 localities, representing four 
morphologically distinguishable species, <em xmlns="" class="EmphasisTypeItalic">D. molestus</em>, <em xmlns="" class="EmphasisTypeItalic">Dorylus sjoestedti</em>, <em xmlns="" class="EmphasisTypeItalic">Dorylus terrificus</em>, and <em xmlns="" class="EmphasisTypeItalic">D. wilverthi</em> (Figure <span class="InternalRef"><a href="#Fig1">1</a></span>; Table <span class="InternalRef"><a href="#Tab1">1</a></span>). A single West African sample of <em xmlns="" class="EmphasisTypeItalic">Dorylus nigricans</em> was also included. A haplotype of the West African species <em xmlns="" class="EmphasisTypeItalic">Dorylus emeryi</em> served as outgroup, as this species belongs to the sister clade of the swarm-hunting <em xmlns="" class="EmphasisTypeItalic">Anomma</em> "driver ants" [<span class="CitationRef"><a href="#CR16" data-toggle="citation">16</a></span>].</p>
                <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Para">Samples
 for microsatellite genotyping (one worker per colony) were collected 
from four populations at three different sites in Kenya: Kakamega Forest
 (where <em xmlns="" class="EmphasisTypeItalic">D. molestus</em>-like and <em xmlns="" class="EmphasisTypeItalic">D. wilverthi</em>-like forms are sympatric; N = 55 each), the eastern slope of Mt. Kenya (<em xmlns="" class="EmphasisTypeItalic">D. molestus</em>; N = 50), and Lower Tana River (<em xmlns="" class="EmphasisTypeItalic">D. molestus</em>; N = 42). Subsets of these samples were also sequenced for mitochondrial DNA (see Table <span class="InternalRef"><a href="#Tab1">1</a></span>). The Mt. Kenya population is geographically situated halfway in between the two other populations (Figure <span class="InternalRef"><a href="#Fig1">1</a></span>). All samples were collected and stored in ethanol.</p>
                <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Para">We identified samples morphologically using the diagnostic criteria given in the original species descriptions and in [<span class="CitationRef"><a href="#CR45" data-toggle="citation">45</a></span>]. Furthermore, samples were directly compared with type specimens. <em xmlns="" class="EmphasisTypeItalic">D. molestus</em>, <em xmlns="" class="EmphasisTypeItalic">D. terrificus</em>, and <em xmlns="" class="EmphasisTypeItalic">D. sjoestedti</em>, which currently have the status of subspecies of <em xmlns="" class="EmphasisTypeItalic">D. nigricans</em>, were treated as species according to [<span class="CitationRef"><a href="#CR45" data-toggle="citation">45</a></span>, <span class="CitationRef"><a href="#CR46" data-toggle="citation">46</a></span>], and the results of an ongoing taxonomic revision of the <em xmlns="" class="EmphasisTypeItalic">Dorylus</em> (<em xmlns="" class="EmphasisTypeItalic">Anomma</em>) swarm-raiding army ants (C. Schöning <em xmlns="" class="EmphasisTypeItalic">et al.</em> unpublished). Importantly, <em xmlns="" class="EmphasisTypeItalic">D. wilverthi</em> and <em xmlns="" class="EmphasisTypeItalic">D. molestus</em> workers are easily distinguished: <em xmlns="" class="EmphasisTypeItalic">D. wilverthi</em>
 workers have the posterior angles of the head uniquely prolonged into a
 raised, slightly outwardly recurved point ("horns"; Figure <span class="InternalRef"><a href="#Fig2">2a</a></span>), while <em xmlns="" class="EmphasisTypeItalic">D. molestus</em> workers have posteroventral extensions ("tubercles") on the petiole (Figure <span class="InternalRef"><a href="#Fig2">2a</a></span>) and significantly shorter appendages than <em xmlns="" class="EmphasisTypeItalic">D. wilverthi</em>[<span class="CitationRef"><a href="#CR47" data-toggle="citation">47</a></span>, <span class="CitationRef"><a href="#CR48" data-toggle="citation">48</a></span>].
 To quantify morphological variation we measured antennal scape length 
(SL) in relation to maximum head width (HW) for large workers (HW &gt; 
1.5 mm) in four groups: <em xmlns="" class="EmphasisTypeItalic">D. molestus</em>-like individuals from Kakamega (N = 54 workers from 10 colonies), <em xmlns="" class="EmphasisTypeItalic">D. wilverthi</em>-like individuals from Kakamega (N = 45 workers from nine colonies), "pure" allopatric <em xmlns="" class="EmphasisTypeItalic">D. molestus</em> (N = 157 workers from 31 colonies belonging to 20 populations across the species' range), and "pure" allopatric <em xmlns="" class="EmphasisTypeItalic">D. wilverthi</em>
 (N = 85 workers from 16 colonies belonging to 10 populations across the
 species' range). Measurements were taken with a MS5 Leica 
stereomicroscope fitted with an ocular micrometer. Data were analysed 
following Schöning <em xmlns="" class="EmphasisTypeItalic">et al.</em>[<span class="CitationRef"><a href="#CR47" data-toggle="citation">47</a></span>].</p>
              </section><section id="Sec13" class="Section2 RenderAsSection2">
                <h3 class="Heading">(b) DNA extraction, amplification, sequencing, and microsatellite genotyping</h3>
                <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Para">DNA for sequencing was extracted from 1-2 worker legs using standard QIAGEN<sup>®</sup> (DNeasy<sup>®</sup>) and MACHEREY-NAGEL (Nucleo Spin<sup>®</sup>Tissue) kits. A mitochondrial fragment of the <em xmlns="" class="EmphasisTypeItalic">cytochrome oxidase II</em> (<em xmlns="" class="EmphasisTypeItalic">COII</em>) gene was amplified and sequenced using primers tRNALeu [<span class="CitationRef"><a href="#CR16" data-toggle="citation">16</a></span>] and Barbara [<span class="CitationRef"><a href="#CR49" data-toggle="citation">49</a></span>] as has been described earlier [<span class="CitationRef"><a href="#CR16" data-toggle="citation">16</a></span>].
 Sequencing reactions were either performed in house or purified PCR 
products were sent to a commercial sequencing facility (Macrogen, 
Korea). The final alignment used in this study consisted of 548 bp of <em xmlns="" class="EmphasisTypeItalic">COII</em> sequence. GenBank accession numbers and details for sequenced samples are given in Table <span class="InternalRef"><a href="#Tab1">1</a></span>.</p>
                <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Para">DNA
 for microsatellite genotyping was extracted by boiling 1-2 worker legs 
in 100 μl of 5% Chelex 100 (Bio-Rad). A total of 202 individuals were 
genotyped at five microsatellite loci (DmoB, DmoC, DmoD, DmoG, DmoO) as 
has been described previously [<span class="CitationRef"><a href="#CR50" data-toggle="citation">50</a></span>].</p>
              </section><section id="Sec14" class="Section2 RenderAsSection2">
                <h3 class="Heading">(c) Phylogeographic Analyses</h3>
                <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Para">Phylogenetic analyses of mitochondrial sequences were conducted in a Bayesian framework using the program MrBayes 3.1.2 [<span class="CitationRef"><a href="#CR51" data-toggle="citation">51</a></span>, <span class="CitationRef"><a href="#CR52" data-toggle="citation">52</a></span>], and in a maximum likelihood framework using the program GARLI 0.96 [<span class="CitationRef"><a href="#CR53" data-toggle="citation">53</a></span>].
 We implemented a general time reversible model with gamma-distributed 
rate heterogeneity (GTR+G), which was selected as the most appropriate 
model for our data by both hierarchical Likelihood Ratio Tests and the 
Akaike Information Criterion in MrModeltest 2.3 [<span class="CitationRef"><a href="#CR54" data-toggle="citation">54</a></span>].</p>
                <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Para">To
 assure convergence of Markov Chain Monte Carlo runs in MrBayes we 
repeated the analysis from two independent starting trees. One cold and 
three heated chains were run in parallel for 20*10<sup>6</sup> 
generations and trees were sampled every 100 generations. Average 
standard deviations of split frequencies were consistently &lt; 0.01 
after 13*10<sup>6</sup> generations, indicating that the independent 
runs had converged. Accordingly, the first 130,000 trees from each run 
were discarded as burn-in. Based on this sample of 140,002 trees from 
both runs combined, all potential scale reduction factors for model 
parameters were &lt; 1.02, indicating effective sampling from the 
posterior probability distribution. A consensus phylogram with posterior
 probabilities based on both runs was calculated in MrBayes.</p>
                <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Para">We
 initially performed three independent runs in GARLI setting the number 
of generations after which the run is terminated if no new significantly
 better scoring topology has been found to 30,000, and otherwise using 
default parameter settings. We then performed a run with 1,000 bootstrap
 replicates setting the above parameter at 10,000.</p>
                <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Para">A mitochondrial haplotype network for all East African haplotypes of <em xmlns="" class="EmphasisTypeItalic">D. molestus</em>, <em xmlns="" class="EmphasisTypeItalic">D. wilverthi</em> and <em xmlns="" class="EmphasisTypeItalic">D. terrificus</em> was constructed using a statistical parsimony approach [<span class="CitationRef"><a href="#CR55" data-toggle="citation">55</a></span>] implemented in TCS version 1.21 [<span class="CitationRef"><a href="#CR56" data-toggle="citation">56</a></span>]. The connection limit in the TCS program was set at unlimited.</p>
              </section><section id="Sec15" class="Section2 RenderAsSection2">
                <h3 class="Heading">(d) Population Structure and Divergence</h3>
                <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Para">The number of alleles per microsatellite locus (<em xmlns="" class="EmphasisTypeItalic">N</em><sub>
                    <em xmlns="" class="EmphasisTypeItalic">a</em>
                  </sub> ), expected heterozygosity (<em xmlns="" class="EmphasisTypeItalic">H</em><sub>
                    <em xmlns="" class="EmphasisTypeItalic">s</em>
                  </sub> ), allelic richness corrected for sample size (<em xmlns="" class="EmphasisTypeItalic">R</em><sub>
                    <em xmlns="" class="EmphasisTypeItalic">s</em>
                  </sub> ), and the inbreeding coefficient <em xmlns="" class="EmphasisTypeItalic">F</em><sub>
                    <em xmlns="" class="EmphasisTypeItalic">IS</em>
                  </sub> were estimated for each population using the program FSTAT 2.9.3.2 [<span class="CitationRef"><a href="#CR57" data-toggle="citation">57</a></span>]. <em xmlns="" class="EmphasisTypeItalic">F</em><sub>
                    <em xmlns="" class="EmphasisTypeItalic">IS</em>
                  </sub> was tested for significant deviation from zero 
using 400 randomisations to assess Hardy Weinberg equilibrium. The same 
program was used to test for genotypic disequilibrium between all pairs 
of microsatellite loci within each population (Table <span class="InternalRef"><a href="#Tab3">3</a></span>). Furthermore, we used FSTAT to calculate pair-wise <em xmlns="" class="EmphasisTypeItalic">F</em><sub>
                    <em xmlns="" class="EmphasisTypeItalic">ST</em>
                  </sub> as an estimate of genetic differentiation at 
microsatellite loci between populations and species. FSTAT was also used
 to calculate <em xmlns="" class="EmphasisTypeItalic">N</em><sub>
                    <em xmlns="" class="EmphasisTypeItalic">a</em>
                  </sub> and <em xmlns="" class="EmphasisTypeItalic">R</em><sub>
                    <em xmlns="" class="EmphasisTypeItalic">s</em>
                  </sub> for mitochondrial haplotypes.</p>
                <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Para">We used two different model-based Bayesian methods to study hybridization between <em xmlns="" class="EmphasisTypeItalic">D. molestus</em> and <em xmlns="" class="EmphasisTypeItalic">D. wilverthi</em> at Kakamega. First, the clustering method of the program STRUCTURE 2.2 [<span class="CitationRef"><a href="#CR58" data-toggle="citation">58</a></span>]
 was used to infer population structure at microsatellite loci without 
prior information on species assignments based on sample locality and 
morphology. The data from all four populations were combined in this 
analysis. To assure convergence and consistency of results, we conducted
 five replicate runs with different random seeds for each <em xmlns="" class="EmphasisTypeItalic">k</em>, the number of subdivisions, ranging from one to 10, under an admixture model with correlated allele frequencies [<span class="CitationRef"><a href="#CR59" data-toggle="citation">59</a></span>]. Markov chains were run for 10<sup>6</sup> generations and the first 10<sup>5</sup>
 generations were discarded as burn-in. Estimates of hybrid frequencies 
were based on individual probabilities of having ancestry in populations
 other than the source population. The comparisons between conspecific 
populations were used as a control to assure that the analyses had 
sufficient power in assigning individuals to different clusters given 
the observed levels of genetic differentiation between populations. More
 recent and potentially ongoing gene-flow between conspecific 
populations should generally result in more ambiguous assignments of 
individuals to populations as compared to genetically isolated 
heterospecific populations, irrespective of whether these occur in 
sympatry or not. We used the program DISTRUCT 1.1 [<span class="CitationRef"><a href="#CR60" data-toggle="citation">60</a></span>] to graphically display STRUCTURE results (Figure <span class="InternalRef"><a href="#Fig4">4</a></span>).</p>
                <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Para">Second, we used the assignment methods in the program GENECLASS2 [<span class="CitationRef"><a href="#CR61" data-toggle="citation">61</a></span>]
 to estimate the likelihood of origin for each individual in each 
potential source population. All four populations were analyzed in a 
single dataset. To calculate likelihoods, we chose a Bayesian criterion 
with a Dirichlet prior distribution for allele frequencies [<span class="CitationRef"><a href="#CR62" data-toggle="citation">62</a></span>] using the resampling method of Paetkau <em xmlns="" class="EmphasisTypeItalic">et al.</em>[<span class="CitationRef"><a href="#CR63" data-toggle="citation">63</a></span>]. The number of simulated individuals for probability computation was set at 10,000.</p>
              </section></div></section>
            <section xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" id="Sec16" class="Section1 RenderAsSection1 c-section"><h2 class="Heading js-ToggleCollapseSection c-section__heading" data-sticky-update="true"><a href="#Authorsinformation" data-toggle="collapse" data-toggle-disable-clickoff="" data-target="Authorsinformation">Authors' information</a></h2><div class="js-CollapseSection CollapseSection c-section__content" id="Authorsinformation" data-test="collapse-section"><p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Para">DJCK
 is the head of the Laboratory of Insect Social Evolution at The 
Rockefeller University. He is broadly interested in the evolution of 
social insects, and army ants are his main study systems. MKP is a 
postdoctoral ecologist at the Department of Animal Ecology and Tropical 
Biology, University of Würzburg, Germany. He studies the ecology and 
conservation of tropical rainforest animals. CS is a postdoctoral fellow
 at the Länderinstitut für Bienenkunde in Hohen Neuendorf, Germany, 
where he presently works on honeybees. He has a long-standing interest 
in the ecology and evolution of African army ants. JJB is the director 
of the Centre for Social Evolution at the University of Copenhagen.</p></div></section>
          <section xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" class="Section1 RenderAsSection1 c-section ArticleNotes" id="ArticleNotes"><h2 class="Heading js-ToggleCollapseSection c-section__heading" data-sticky-update="true"><a href="#Notes" data-toggle="collapse" data-toggle-disable-clickoff="" data-target="Notes">Notes</a></h2><div class="js-CollapseSection CollapseSection c-section__content c-section__content" data-test="collapse-section" id="Notes"><aside xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="ArticleNote ArticleNoteMisc"><p xmlns="" class="SimplePara">Daniel JC Kronauer, Marcell K Peters contributed equally to this work.</p></aside></div></section><section xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" class="Section1 RenderAsSection1 c-section" data-test="declarations-section" id="Decs"><h2 class="Heading js-ToggleCollapseSection c-section__heading" data-sticky-update="true"><a href="#Declarations" data-toggle="collapse" data-toggle-disable-clickoff="" data-target="Declarations">Declarations</a></h2><div class="js-CollapseSection CollapseSection c-section__content" data-test="collapse-section" id="Declarations"><div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Acknowledgments"><h3 xmlns="" class="Heading">Acknowledgements</h3><p xmlns="" class="SimplePara">We
 thank Washington Njagi, Abio Gafo, and Jairus Analo for assistance in 
the field. We are grateful to the Kenya Wildlife Service, the Kenya 
Forest Service, and the National Museums of Kenya for their support and 
for granting us access to the forest reserves. We are grateful to Frank 
E. Rheindt for helpful comments on the manuscript. Funding was provided 
by the Danish National Research Foundation, the Alexander von Humboldt 
Foundation, the Critical Ecosystem Partnership Fund, the Cusanuswerk, 
the International School of Biodiversity Sciences, Denmark, and the 
Harvard Society of Fellows. This publication was funded by the German 
Research Foundation (DFG) and the University of Wuerzburg in the funding
 programme Open Access Publishing.</p></div><aside class="Appendix" id="App1">
              
                <h3 class="Heading">Authors' original submitted files for images</h3><div id="widgetContainer" data-test="figshare-container"></div>
                <div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Para">Below are the links to the authors’ original submitted files for images.<div class="MediaObject" id="MOESM1">
                    <div xmlns="" class="esm-item "><div xmlns="http://www.w3.org/1999/xhtml" class="caption-container"><a class="filename" href="https://static-content.springer.com/esm/art%3A10.1186%2F1742-9994-8-20/MediaObjects/12983_2011_166_MOESM1_ESM.tiff">12983_2011_166_MOESM1_ESM.tiff</a>
                          <span xmlns="" class="SimplePara">Authors’ original file for figure 1</span>
                        </div></div>
                  </div><div class="MediaObject" id="MOESM2">
                    <div xmlns="" class="esm-item "><div xmlns="http://www.w3.org/1999/xhtml" class="caption-container"><a class="filename" href="https://static-content.springer.com/esm/art%3A10.1186%2F1742-9994-8-20/MediaObjects/12983_2011_166_MOESM2_ESM.tiff">12983_2011_166_MOESM2_ESM.tiff</a>
                          <span xmlns="" class="SimplePara">Authors’ original file for figure 2</span>
                        </div></div>
                  </div><div class="MediaObject" id="MOESM3">
                    <div xmlns="" class="esm-item "><div xmlns="http://www.w3.org/1999/xhtml" class="caption-container"><a class="filename" href="https://static-content.springer.com/esm/art%3A10.1186%2F1742-9994-8-20/MediaObjects/12983_2011_166_MOESM3_ESM.tiff">12983_2011_166_MOESM3_ESM.tiff</a>
                          <span xmlns="" class="SimplePara">Authors’ original file for figure 3</span>
                        </div></div>
                  </div><div class="MediaObject" id="MOESM4">
                    <div xmlns="" class="esm-item "><div xmlns="http://www.w3.org/1999/xhtml" class="caption-container"><a class="filename" href="https://static-content.springer.com/esm/art%3A10.1186%2F1742-9994-8-20/MediaObjects/12983_2011_166_MOESM4_ESM.pdf">12983_2011_166_MOESM4_ESM.pdf</a>
                          <span xmlns="" class="SimplePara">Authors’ original file for figure 4</span>
                        </div></div>
                  </div><div class="MediaObject" id="MOESM5">
                    <div xmlns="" class="esm-item "><div xmlns="http://www.w3.org/1999/xhtml" class="caption-container"><a class="filename" href="https://static-content.springer.com/esm/art%3A10.1186%2F1742-9994-8-20/MediaObjects/12983_2011_166_MOESM5_ESM.tiff">12983_2011_166_MOESM5_ESM.tiff</a>
                          <span xmlns="" class="SimplePara">Authors’ original file for figure 5</span>
                        </div></div>
                  </div></div>
              
            </aside><aside xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="ArticleNote ArticleNoteMisc"><h3 xmlns="" class="Heading">Competing interests</h3><p xmlns="" class="SimplePara">The authors declare that they have no competing interests.</p></aside><aside xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="ArticleNote ArticleNoteMisc"><h3 xmlns="" class="Heading">Authors' contributions</h3><p xmlns="" class="SimplePara">DJCK,
 MKP, CS, JJB designed the study an acquired funding. MKP and CS 
collected all army ant samples and identified them. DJCK and MKP 
conducted the molecular genetic studies and analysed the data. CS 
conducted the morphometric analyses. DJCK wrote the first version of the
 paper. All authors contributed to and approved the final manuscript.</p></aside></div></section></div>
                        <div class="FulltextWrapper"><section xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" class="Section1 RenderAsSection1 c-section Affiliations" id="Aff"><h2 class="Heading js-ToggleCollapseSection c-section__heading" data-sticky-update="true"><a href="#Author-Affiliations" data-toggle="collapse" data-toggle-disable-clickoff="" data-target="Author-Affiliations">Authors’ Affiliations</a></h2><div class="js-CollapseSection CollapseSection c-section__content c-section__content" data-test="collapse-section" id="Author-Affiliations"><div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Affiliation" id="Aff1"><span class="AffiliationNumber">(1)</span><div class="AffiliationText">Centre
 for Social Evolution, Department of Biology, University of Copenhagen, 
Universitetsparken 15, 2100&nbsp;Copenhagen, Denmark</div></div><div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Affiliation" id="Aff2"><span class="AffiliationNumber">(2)</span><div class="AffiliationText">Laboratory of Insect Social Evolution, The Rockefeller University, 1230 York Avenue, New York, NY&nbsp;10065, USA</div></div><div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Affiliation" id="Aff3"><span class="AffiliationNumber">(3)</span><div class="AffiliationText">Zoological Research Museum Alexander Koenig, Adenauerallee 160, 53113&nbsp;Bonn, Germany</div></div><div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Affiliation" id="Aff4"><span class="AffiliationNumber">(4)</span><div class="AffiliationText">Department of Animal Ecology and Tropical Biology, Biocenter, University of Würzburg, Am Hubland, 97074&nbsp;Würzburg, Germany</div></div><div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Affiliation" id="Aff5"><span class="AffiliationNumber">(5)</span><div class="AffiliationText">Länderinstitut für Bienenkunde, Friedrich-Engels-Strasse 32, 16540&nbsp;Hohen Neuendorf, Germany</div></div></div></section></div>
                        <div class="FulltextWrapper"><section xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" class="Section1 RenderAsSection1 c-section SectionTypeMaterialsAndMethods" id="Bib1"><h2 class="Heading js-ToggleCollapseSection c-section__heading" data-sticky-update="true"><a href="#References" data-toggle="collapse" data-toggle-disable-clickoff="" data-target="References">References</a></h2><div class="js-CollapseSection CollapseSection c-section__content" data-test="collapse-section" id="References"><ol class="BibliographyWrapper"><li class="Citation"><cite class="CitationContent" id="CR1" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Rieseberg LH: Hybrid origins of plant species. Annu Rev Ecol Syst. 1997, 28: 359-389. 10.1146/annurev.ecolsys.28.1.359.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1146/annurev.ecolsys.28.1.359"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Hybrid%20origins%20of%20plant%20species&amp;author=LH.%20Rieseberg&amp;journal=Annu%20Rev%20Ecol%20Syst&amp;volume=28&amp;pages=359-389&amp;publication_year=1997&amp;doi=10.1146%2Fannurev.ecolsys.28.1.359"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR2" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Dowling
 TE, Secor CL: The role of hybridization and introgression in the 
diversification of animals. Annu Rev Eco Syst. 1997, 28: 593-619. 
10.1146/annurev.ecolsys.28.1.593.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1146/annurev.ecolsys.28.1.593"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=The%20role%20of%20hybridization%20and%20introgression%20in%20the%20diversification%20of%20animals&amp;author=TE.%20Dowling&amp;author=CL.%20Secor&amp;journal=Annu%20Rev%20Eco%20Syst&amp;volume=28&amp;pages=593-619&amp;publication_year=1997&amp;doi=10.1146%2Fannurev.ecolsys.28.1.593"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR3" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Seehausen O: Hybridization and adaptive radiation. Trends Ecol Evol. 2004, 19: 198-207. 10.1016/j.tree.2004.01.003.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1016/j.tree.2004.01.003"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=16701254"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Hybridization%20and%20adaptive%20radiation&amp;author=O.%20Seehausen&amp;journal=Trends%20Ecol%20Evol&amp;volume=19&amp;pages=198-207&amp;publication_year=2004&amp;doi=10.1016%2Fj.tree.2004.01.003"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR4" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Mallet J: Hybrid speciation. Nature. 2007, 446: 279-283. 10.1038/nature05706.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1038/nature05706"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=17361174"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Hybrid%20speciation&amp;author=J.%20Mallet&amp;journal=Nature&amp;volume=446&amp;pages=279-283&amp;publication_year=2007&amp;doi=10.1038%2Fnature05706"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR5" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Mavárez J, Linares M: Homoploid hybrid speciation in animals. Mol Ecol. 2008, 17: 4181-4185. 10.1111/j.1365-294X.2008.03898.x.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1111/j.1365-294X.2008.03898.x"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=19378399"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Homoploid%20hybrid%20speciation%20in%20animals&amp;author=J.%20Mav%C3%A1rez&amp;author=M.%20Linares&amp;journal=Mol%20Ecol&amp;volume=17&amp;pages=4181-4185&amp;publication_year=2008&amp;doi=10.1111%2Fj.1365-294X.2008.03898.x"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR6" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Burke
 JM, Arnold ML: Genetics and the fitness of hybrids. Annu Rev Genet. 
2001, 35: 31-52. 10.1146/annurev.genet.35.102401.085719.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1146/annurev.genet.35.102401.085719"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=11700276"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Genetics%20and%20the%20fitness%20of%20hybrids&amp;author=JM.%20Burke&amp;author=ML.%20Arnold&amp;journal=Annu%20Rev%20Genet&amp;volume=35&amp;pages=31-52&amp;publication_year=2001&amp;doi=10.1146%2Fannurev.genet.35.102401.085719"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR7" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Taylor
 EB, Boughman JW, Groenenboom M, Sniatynski M, Schluter D, Gow JL: 
Speciation in reverse: morphological and genetic evidence of the 
collapse of a three-spined stickleback (<em class="EmphasisTypeItalic">Gasterosteus aculeatus</em>) species pair. Mol Ecol. 2006, 15: 343-355.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1111/j.1365-294X.2005.02794.x"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=16448405"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Speciation%20in%20reverse%3A%20morphological%20and%20genetic%20evidence%20of%20the%20collapse%20of%20a%20three-spined%20stickleback%20%28Gasterosteus%20aculeatus%29%20species%20pair&amp;author=EB.%20Taylor&amp;author=JW.%20Boughman&amp;author=M.%20Groenenboom&amp;author=M.%20Sniatynski&amp;author=D.%20Schluter&amp;author=JL.%20Gow&amp;journal=Mol%20Ecol&amp;volume=15&amp;pages=343-355&amp;publication_year=2006"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR8" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Anderson
 KE, Linksvayer TA, Smith CR: The causes and consequences of genetic 
caste determination in ants (Hymenoptera: Formicidae). Myrmecol News. 
2008, 11: 119-132.</span><span class="Occurrences"><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=The%20causes%20and%20consequences%20of%20genetic%20caste%20determination%20in%20ants%20%28Hymenoptera%3A%20Formicidae%29&amp;author=KE.%20Anderson&amp;author=TA.%20Linksvayer&amp;author=CR.%20Smith&amp;journal=Myrmecol%20News&amp;volume=11&amp;pages=119-132&amp;publication_year=2008"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR9" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Schwander
 T, Lo N, Beekman M, Oldroyd BP, Keller L: Nature versus nurture in 
social insect caste differentiation. Trends Ecol Evol. 2010, 25: 
275-282. 10.1016/j.tree.2009.12.001.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1016/j.tree.2009.12.001"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=20106547"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Nature%20versus%20nurture%20in%20social%20insect%20caste%20differentiation&amp;author=T.%20Schwander&amp;author=N.%20Lo&amp;author=M.%20Beekman&amp;author=BP.%20Oldroyd&amp;author=L.%20Keller&amp;journal=Trends%20Ecol%20Evol&amp;volume=25&amp;pages=275-282&amp;publication_year=2010&amp;doi=10.1016%2Fj.tree.2009.12.001"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR10" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Umphrey
 GJ: Sperm parasitism in ants: selection for interspecific mating and 
hybridization. Ecology. 2006, 87: 2148-2159. 
10.1890/0012-9658(2006)87[2148:SPIASF]2.0.CO;2.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1890/0012-9658%282006%2987%5B2148%3ASPIASF%5D2.0.CO%3B2"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=16995614"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Sperm%20parasitism%20in%20ants%3A%20selection%20for%20interspecific%20mating%20and%20hybridization&amp;author=GJ.%20Umphrey&amp;journal=Ecology&amp;volume=87&amp;pages=2148-2159&amp;publication_year=2006&amp;doi=10.1890%2F0012-9658%282006%2987%5B2148%3ASPIASF%5D2.0.CO%3B2"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR11" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Feldhaar
 H, Foitzik S, Heinze J: Lifelong commitment to the wrong partner: 
hybridization in ants. Philos T R Soc B. 2008, 363: 2891-2899. 
10.1098/rstb.2008.0022.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1098/rstb.2008.0022"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Lifelong%20commitment%20to%20the%20wrong%20partner%3A%20hybridization%20in%20ants&amp;author=H.%20Feldhaar&amp;author=S.%20Foitzik&amp;author=J.%20Heinze&amp;journal=Philos%20T%20R%20Soc%20B&amp;volume=363&amp;pages=2891-2899&amp;publication_year=2008&amp;doi=10.1098%2Frstb.2008.0022"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR12" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Van der Have TM, Pedersen JS, Boomsma JJ: Mating, hybridisation and introgression in <em class="EmphasisTypeItalic">Lasius</em> ants (Hymenoptera, Formicidae). Myrmecological News. 2010, 15: 109-115.</span><span class="Occurrences"><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Mating%2C%20hybridisation%20and%20introgression%20in%20Lasius%20ants%20%28Hymenoptera%2C%20Formicidae%29&amp;author=TM.%20Van%20der%20Have&amp;author=JS.%20Pedersen&amp;author=JJ.%20Boomsma&amp;journal=Myrmecological%20News&amp;volume=15&amp;pages=109-115&amp;publication_year=2010"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR13" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Volny
 VP, Gordon DM: Genetic basis for queen-worker dimorphism in a social 
insect. Proc Natl Acad Sci USA. 2002, 99: 6108-6111. 
10.1073/pnas.092066699.</span><span class="Occurrences"><span class="Occurrence OccurrencePMCID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC122910"><span><span>PubMed Central</span></span></a></span><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1073/pnas.092066699"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=11972049"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Genetic%20basis%20for%20queen-worker%20dimorphism%20in%20a%20social%20insect&amp;author=VP.%20Volny&amp;author=DM.%20Gordon&amp;journal=Proc%20Natl%20Acad%20Sci%20USA&amp;volume=99&amp;pages=6108-6111&amp;publication_year=2002&amp;doi=10.1073%2Fpnas.092066699"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR14" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Helms
 Cahan SH, Keller L: Complex hybrid origin of genetic caste 
determination in harvester ants. Nature. 2003, 424: 306-309. 
10.1038/nature01744.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1038/nature01744"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=12867980"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Complex%20hybrid%20origin%20of%20genetic%20caste%20determination%20in%20harvester%20ants&amp;author=SH.%20Helms%20Cahan&amp;author=L.%20Keller&amp;journal=Nature&amp;volume=424&amp;pages=306-309&amp;publication_year=2003&amp;doi=10.1038%2Fnature01744"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR15" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Kulmuni
 J, Seifert B, Pamilo P: Segregation distortion causes large-scale 
differences between male and female genomes in hybrid ants. Proc Natl 
Acad Sci USA. 2010, 107: 7371-7376. 10.1073/pnas.0912409107.</span><span class="Occurrences"><span class="Occurrence OccurrencePMCID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2867678"><span><span>PubMed Central</span></span></a></span><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1073/pnas.0912409107"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=20368452"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Segregation%20distortion%20causes%20large-scale%20differences%20between%20male%20and%20female%20genomes%20in%20hybrid%20ants&amp;author=J.%20Kulmuni&amp;author=B.%20Seifert&amp;author=P.%20Pamilo&amp;journal=Proc%20Natl%20Acad%20Sci%20USA&amp;volume=107&amp;pages=7371-7376&amp;publication_year=2010&amp;doi=10.1073%2Fpnas.0912409107"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR16" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Kronauer DJC, Schöning C, Vilhelmsen L, Boomsma JJ: A molecular phylogeny of <em class="EmphasisTypeItalic">Dorylus</em>
 army ants provides evidence for multiple evolutionary transitions in 
foraging niche. BMC Evol Biol. 2007, 7: 56-10.1186/1471-2148-7-56.</span><span class="Occurrences"><span class="Occurrence OccurrencePMCID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC1852301"><span><span>PubMed Central</span></span></a></span><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1186/1471-2148-7-56"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=17408491"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=A%20molecular%20phylogeny%20of%20Dorylus%20army%20ants%20provides%20evidence%20for%20multiple%20evolutionary%20transitions%20in%20foraging%20niche&amp;author=DJC.%20Kronauer&amp;author=C.%20Sch%C3%B6ning&amp;author=L.%20Vilhelmsen&amp;author=JJ.%20Boomsma&amp;journal=BMC%20Evol%20Biol&amp;volume=7&amp;pages=56&amp;publication_year=2007&amp;doi=10.1186%2F1471-2148-7-56"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR17" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Kronauer
 DJC, Schöning C, Pedersen JS, Boomsma JJ, Gadau J: Extreme queen-mating
 frequency and colony fission in African army ants. Mol Ecol. 2004, 13: 
2381-2388. 10.1111/j.1365-294X.2004.02262.x.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1111/j.1365-294X.2004.02262.x"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=15245410"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Extreme%20queen-mating%20frequency%20and%20colony%20fission%20in%20African%20army%20ants&amp;author=DJC.%20Kronauer&amp;author=C.%20Sch%C3%B6ning&amp;author=JS.%20Pedersen&amp;author=JJ.%20Boomsma&amp;author=J.%20Gadau&amp;journal=Mol%20Ecol&amp;volume=13&amp;pages=2381-2388&amp;publication_year=2004&amp;doi=10.1111%2Fj.1365-294X.2004.02262.x"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR18" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Berghoff SM, Kronauer DJC, Edwards KJ, Franks NR: Dispersal and population structure of a New World predator, the army ant <em class="EmphasisTypeItalic">Eciton burchellii</em>. J Evolution Biol. 2008, 21: 1125-1132. 10.1111/j.1420-9101.2008.01531.x.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1111/j.1420-9101.2008.01531.x"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Dispersal%20and%20population%20structure%20of%20a%20New%20World%20predator%2C%20the%20army%20ant%20Eciton%20burchellii&amp;author=SM.%20Berghoff&amp;author=DJC.%20Kronauer&amp;author=KJ.%20Edwards&amp;author=NR.%20Franks&amp;journal=J%20Evolution%20Biol&amp;volume=21&amp;pages=1125-1132&amp;publication_year=2008&amp;doi=10.1111%2Fj.1420-9101.2008.01531.x"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR19" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Kronauer
 DJC, Schöning C, d'Ettorre P, Boomsma JJ: Colony fusion and worker 
reproduction after queen loss in army ants. Proc Roy Soc B. 2010, 277: 
755-763. 10.1098/rspb.2009.1591.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1098/rspb.2009.1591"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Colony%20fusion%20and%20worker%20reproduction%20after%20queen%20loss%20in%20army%20ants&amp;author=DJC.%20Kronauer&amp;author=C.%20Sch%C3%B6ning&amp;author=P.%20d%27Ettorre&amp;author=JJ.%20Boomsma&amp;journal=Proc%20Roy%20Soc%20B&amp;volume=277&amp;pages=755-763&amp;publication_year=2010&amp;doi=10.1098%2Frspb.2009.1591"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR20" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Franks
 NR, Hölldobler B: Sexual competition during colony reproduction in army
 ants. Biol J Linn Soc. 1987, 30: 229-243. 
10.1111/j.1095-8312.1987.tb00298.x.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1111/j.1095-8312.1987.tb00298.x"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Sexual%20competition%20during%20colony%20reproduction%20in%20army%20ants&amp;author=NR.%20Franks&amp;author=B.%20H%C3%B6lldobler&amp;journal=Biol%20J%20Linn%20Soc&amp;volume=30&amp;pages=229-243&amp;publication_year=1987&amp;doi=10.1111%2Fj.1095-8312.1987.tb00298.x"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR21" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Schöning C, Kinuthia W, Boomsma JJ: Does the afrotropical army ant <em class="EmphasisTypeItalic">Dorylus (Anomma) molestus</em> go extinct in fragmented forests?. J East Afr Nat Hist. 2006, 95: 163-179. 10.2982/0012-8317(2006)95[163:DTAAAD]2.0.CO;2.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.2982/0012-8317%282006%2995%5B163%3ADTAAAD%5D2.0.CO%3B2"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Does%20the%20afrotropical%20army%20ant%20Dorylus%20%28Anomma%29%20molestus%20go%20extinct%20in%20fragmented%20forests%3F&amp;author=C.%20Sch%C3%B6ning&amp;author=W.%20Kinuthia&amp;author=JJ.%20Boomsma&amp;journal=J%20East%20Afr%20Nat%20Hist&amp;volume=95&amp;pages=163-179&amp;publication_year=2006&amp;doi=10.2982%2F0012-8317%282006%2995%5B163%3ADTAAAD%5D2.0.CO%3B2"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR22" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Peters
 MK, Okalo B: Severe declines of ant-following birds in African 
rainforest fragments are facilitated by a subtle change in army ant 
communities. Biol Conserv. 142: 2050-2058.<span xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Occurrences"><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="https://scholar.google.com/scholar?hl=en&amp;q=Peters%20MK%2C%20Okalo%20B%3A%20Severe%20declines%20of%20ant-following%20birds%20in%20African%20rainforest%20fragments%20are%20facilitated%20by%20a%20subtle%20change%20in%20army%20ant%20communities.%20Biol%20Conserv.%20142%3A%202050-2058."><span><span>Google Scholar</span></span></a></span></span></span><span class="Occurrences"></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR23" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Peters
 MK, Fischer G, Schaab G, Kraemer M: Species compensation maintains 
abundance and raid rates of African swarm-raiding army ants in 
rainforest fragments. Biol Conserv. 2009, 142: 668-675. 
10.1016/j.biocon.2008.11.021.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1016/j.biocon.2008.11.021"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Species%20compensation%20maintains%20abundance%20and%20raid%20rates%20of%20African%20swarm-raiding%20army%20ants%20in%20rainforest%20fragments&amp;author=MK.%20Peters&amp;author=G.%20Fischer&amp;author=G.%20Schaab&amp;author=M.%20Kraemer&amp;journal=Biol%20Conserv&amp;volume=142&amp;pages=668-675&amp;publication_year=2009&amp;doi=10.1016%2Fj.biocon.2008.11.021"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR24" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Raignier
 A, Van Boven JKA, Ceusters R: Der Polymorphismus der afrikanischen 
Wanderameisen unter biometrischen und biologischen Gesichtspunkten. 
Sozialpolymorphismus bei Insekten. Probleme der Kastenbildung im 
Tierreich. Edited by: Schmidt GH. 1974, Stuttgart: Wissenschaftliche 
Verlagsgesellschaft, 668-693.</span><span class="Occurrences"><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Der%20Polymorphismus%20der%20afrikanischen%20Wanderameisen%20unter%20biometrischen%20und%20biologischen%20Gesichtspunkten&amp;author=A.%20Raignier&amp;author=JKA.%20Van%20Boven&amp;author=R.%20Ceusters&amp;pages=668-693&amp;publication_year=1974"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR25" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Wasmann
 E: Neue Anpassungstypen bei Dorylinengästen Afrikas (Col. 
Staphylinidae). (218. Beitrag zur Kenntnis der Myrmekophilen.). Zeitschr
 Wissenschaftl Zool. 1917, 17: 257-360.</span><span class="Occurrences"><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Neue%20Anpassungstypen%20bei%20Dorylineng%C3%A4sten%20Afrikas%20%28Col.%20Staphylinidae%29.%20%28218.%20Beitrag%20zur%20Kenntnis%20der%20Myrmekophilen.%29&amp;author=E.%20Wasmann&amp;journal=Zeitschr%20Wissenschaftl%20Zool&amp;volume=17&amp;pages=257-360&amp;publication_year=1917"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR26" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Ballard
 JWO, Whitlock MC: The incomplete natural history of mitochondria. Mol 
Ecol. 2004, 13: 729-744. 10.1046/j.1365-294X.2003.02063.x.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1046/j.1365-294X.2003.02063.x"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=15012752"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=The%20incomplete%20natural%20history%20of%20mitochondria&amp;author=JWO.%20Ballard&amp;author=MC.%20Whitlock&amp;journal=Mol%20Ecol&amp;volume=13&amp;pages=729-744&amp;publication_year=2004&amp;doi=10.1046%2Fj.1365-294X.2003.02063.x"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR27" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Bazin
 E, Glemin S, Galtier N: Population size does not influence 
mitochondrial genetic diversity in animals. Science. 2006, 312: 570-572.
 10.1126/science.1122033.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1126/science.1122033"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=16645093"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Population%20size%20does%20not%20influence%20mitochondrial%20genetic%20diversity%20in%20animals&amp;author=E.%20Bazin&amp;author=S.%20Glemin&amp;author=N.%20Galtier&amp;journal=Science&amp;volume=312&amp;pages=570-572&amp;publication_year=2006&amp;doi=10.1126%2Fscience.1122033"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR28" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Bintanja
 R, Van de Wal RSW, Oerlemans J: Modelled atmospheric temperatures and 
global sea levels over the past million years. Nature. 2005, 437: 
125-128. 10.1038/nature03975.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1038/nature03975"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=16136140"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Modelled%20atmospheric%20temperatures%20and%20global%20sea%20levels%20over%20the%20past%20million%20years&amp;author=R.%20Bintanja&amp;author=RSW.%20Van%20de%20Wal&amp;author=J.%20Oerlemans&amp;journal=Nature&amp;volume=437&amp;pages=125-128&amp;publication_year=2005&amp;doi=10.1038%2Fnature03975"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR29" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Caputo
 R: Sea-level curves: perplexities of an end-user in morphotectonic 
applications. Global Planet Change. 2007, 57: 417-423. 
10.1016/j.gloplacha.2007.03.003.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1016/j.gloplacha.2007.03.003"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Sea-level%20curves%3A%20perplexities%20of%20an%20end-user%20in%20morphotectonic%20applications&amp;author=R.%20Caputo&amp;journal=Global%20Planet%20Change&amp;volume=57&amp;pages=417-423&amp;publication_year=2007&amp;doi=10.1016%2Fj.gloplacha.2007.03.003"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR30" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Wagner
 P, Köhler J, Schmitz A, Böhme W: The biogeographical assignment of a 
west Kenyan rain forest remnant: further evidence from analysis of its 
reptile fauna. J Biogeogr. 2008, 35: 1349-1361. 
10.1111/j.1365-2699.2008.01883.x.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1111/j.1365-2699.2008.01883.x"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=The%20biogeographical%20assignment%20of%20a%20west%20Kenyan%20rain%20forest%20remnant%3A%20further%20evidence%20from%20analysis%20of%20its%20reptile%20fauna&amp;author=P.%20Wagner&amp;author=J.%20K%C3%B6hler&amp;author=A.%20Schmitz&amp;author=W.%20B%C3%B6hme&amp;journal=J%20Biogeogr&amp;volume=35&amp;pages=1349-1361&amp;publication_year=2008&amp;doi=10.1111%2Fj.1365-2699.2008.01883.x"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR31" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Moreau RE: Pleistocene climatic changes and the distribution of life in East Africa. J Ecol. 1933, 21: 415-435. 10.2307/2256590.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.2307/2256590"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Pleistocene%20climatic%20changes%20and%20the%20distribution%20of%20life%20in%20East%20Africa&amp;author=RE.%20Moreau&amp;journal=J%20Ecol&amp;volume=21&amp;pages=415-435&amp;publication_year=1933&amp;doi=10.2307%2F2256590"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR32" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Mahaney WC: Quaternary and environmental research on East African mountains. 1989, Rotterdam: AA. Balkema</span><span class="Occurrences"><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Quaternary%20and%20environmental%20research%20on%20East%20African%20mountains&amp;author=WC.%20Mahaney&amp;publication_year=1989"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR33" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Hilgenboecker K, Hammerstein P, Schlattmann P, Telschow A, Werren JH: How many species are infected with <em class="EmphasisTypeItalic">Wolbachia</em>? - a statistical analysis of current data. FEMS Microbiol Let. 2008, 281: 215-220. 10.1111/j.1574-6968.2008.01110.x.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1111/j.1574-6968.2008.01110.x"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=How%20many%20species%20are%20infected%20with%20Wolbachia%3F%20-%20a%20statistical%20analysis%20of%20current%20data&amp;author=K.%20Hilgenboecker&amp;author=P.%20Hammerstein&amp;author=P.%20Schlattmann&amp;author=A.%20Telschow&amp;author=JH.%20Werren&amp;journal=FEMS%20Microbiol%20Let&amp;volume=281&amp;pages=215-220&amp;publication_year=2008&amp;doi=10.1111%2Fj.1574-6968.2008.01110.x"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR34" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Bachtrog D, Thornton K, Clark A, Andolfatto P: Extensive introgression of mitochondrial DNA relative to nuclear genes in the <em class="EmphasisTypeItalic">Drosophila yakuba</em> species group. Evolution. 2006, 60: 292-302.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1111/j.0014-3820.2006.tb01107.x"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=16610321"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Extensive%20introgression%20of%20mitochondrial%20DNA%20relative%20to%20nuclear%20genes%20in%20the%20Drosophila%20yakuba%20species%20group&amp;author=D.%20Bachtrog&amp;author=K.%20Thornton&amp;author=A.%20Clark&amp;author=P.%20Andolfatto&amp;journal=Evolution&amp;volume=60&amp;pages=292-302&amp;publication_year=2006"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR35" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Powell J: Interspecific cytoplasmic gene flow in the absence of nuclear gene flow: evidence from <em class="EmphasisTypeItalic">Drosophila</em>. Proc Natl Acad Sci USA. 1983, 80: 492-495. 10.1073/pnas.80.2.492.</span><span class="Occurrences"><span class="Occurrence OccurrencePMCID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC393404"><span><span>PubMed Central</span></span></a></span><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1073/pnas.80.2.492"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=6300849"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Interspecific%20cytoplasmic%20gene%20flow%20in%20the%20absence%20of%20nuclear%20gene%20flow%3A%20evidence%20from%20Drosophila&amp;author=J.%20Powell&amp;journal=Proc%20Natl%20Acad%20Sci%20USA&amp;volume=80&amp;pages=492-495&amp;publication_year=1983&amp;doi=10.1073%2Fpnas.80.2.492"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR36" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Roca A, Georgiadis N, O'Brien S: Cytonuclear genomic dissociation in African elephant species. Nat Genet. 2005, 37: 96-100.</span><span class="Occurrences"><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=15592471"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Cytonuclear%20genomic%20dissociation%20in%20African%20elephant%20species&amp;author=A.%20Roca&amp;author=N.%20Georgiadis&amp;author=S.%20O%27Brien&amp;journal=Nat%20Genet&amp;volume=37&amp;pages=96-100&amp;publication_year=2005"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR37" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Linnen CR, Farrell BD: Mitonuclear discordance is caused by rampant mitochondrial introgression in <em class="EmphasisTypeItalic">Neodiprion</em> (Hymenoptera: Diprionidae) sawflies. Evolution. 2007, 61: 1417-1438. 10.1111/j.1558-5646.2007.00114.x.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1111/j.1558-5646.2007.00114.x"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=17542850"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Mitonuclear%20discordance%20is%20caused%20by%20rampant%20mitochondrial%20introgression%20in%20Neodiprion%20%28Hymenoptera%3A%20Diprionidae%29%20sawflies&amp;author=CR.%20Linnen&amp;author=BD.%20Farrell&amp;journal=Evolution&amp;volume=61&amp;pages=1417-1438&amp;publication_year=2007&amp;doi=10.1111%2Fj.1558-5646.2007.00114.x"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR38" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Koevoets
 T, Beukeboom LW: Genetics of postzygotic isolation and Haldane's rule 
in haplodiploids. Heredity. 2009, 102: 16-23. 10.1038/hdy.2008.44.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1038/hdy.2008.44"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=18523445"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Genetics%20of%20postzygotic%20isolation%20and%20Haldane%27s%20rule%20in%20haplodiploids&amp;author=T.%20Koevoets&amp;author=LW.%20Beukeboom&amp;journal=Heredity&amp;volume=102&amp;pages=16-23&amp;publication_year=2009&amp;doi=10.1038%2Fhdy.2008.44"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR39" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Berthier P, Excoffier L, Ruedi M: Recurrent replacement of mtDNA and cryptic hybridization between two sibling bat species <em class="EmphasisTypeItalic">Myotis myotis</em> and <em class="EmphasisTypeItalic">Myotis blythii</em>. Proc Roy Soc B. 2006, 273: 3101-3109. 10.1098/rspb.2006.3680.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1098/rspb.2006.3680"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Recurrent%20replacement%20of%20mtDNA%20and%20cryptic%20hybridization%20between%20two%20sibling%20bat%20species%20Myotis%20myotis%20and%20Myotis%20blythii&amp;author=P.%20Berthier&amp;author=L.%20Excoffier&amp;author=M.%20Ruedi&amp;journal=Proc%20Roy%20Soc%20B&amp;volume=273&amp;pages=3101-3109&amp;publication_year=2006&amp;doi=10.1098%2Frspb.2006.3680"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR40" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Joly
 S, McLenachan PA, Lockhart PJ: A statistical approach for 
distinguishing hybridization and incomplete lineage sorting. Am Nat. 
2009, 174: E54-E70. 10.1086/600082.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1086/600082"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=19519219"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=A%20statistical%20approach%20for%20distinguishing%20hybridization%20and%20incomplete%20lineage%20sorting&amp;author=S.%20Joly&amp;author=PA.%20McLenachan&amp;author=PJ.%20Lockhart&amp;journal=Am%20Nat&amp;volume=174&amp;pages=E54-E70&amp;publication_year=2009&amp;doi=10.1086%2F600082"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR41" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Schoville SD, Roderick GK: Evolutionary diversification of cryophilic <em class="EmphasisTypeItalic">Grylloblatta</em>
 species (Grylloblattodea: Grylloblattidae) in alpine habitats of 
California. BMC Evol Biol. 2010, 10: 163-10.1186/1471-2148-10-163.</span><span class="Occurrences"><span class="Occurrence OccurrencePMCID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2898686"><span><span>PubMed Central</span></span></a></span><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1186/1471-2148-10-163"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=20525203"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Evolutionary%20diversification%20of%20cryophilic%20Grylloblatta%20species%20%28Grylloblattodea%3A%20Grylloblattidae%29%20in%20alpine%20habitats%20of%20California&amp;author=SD.%20Schoville&amp;author=GK.%20Roderick&amp;journal=BMC%20Evol%20Biol&amp;volume=10&amp;pages=163&amp;publication_year=2010&amp;doi=10.1186%2F1471-2148-10-163"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR42" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Ross
 KG, Gotzek D, Ascunce MS, Shoemaker DD: Species delimitation: a case 
study in a problematic ant taxon. Syst Biol. 2010, 59: 162-184. 
10.1093/sysbio/syp089.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1093/sysbio/syp089"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=20525628"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Species%20delimitation%3A%20a%20case%20study%20in%20a%20problematic%20ant%20taxon&amp;author=KG.%20Ross&amp;author=D.%20Gotzek&amp;author=MS.%20Ascunce&amp;author=DD.%20Shoemaker&amp;journal=Syst%20Biol&amp;volume=59&amp;pages=162-184&amp;publication_year=2010&amp;doi=10.1093%2Fsysbio%2Fsyp089"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR43" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Seifert B, Kulmuni J, Pamilo P: Independent hybrid populations of <em class="EmphasisTypeItalic">Formica polyctena</em> X <em class="EmphasisTypeItalic">rufa</em>
 wood ants (Hymenoptera: Formicidae) abound under conditions of forest 
fragmentation. Evol Ecol. 2010, 24: 1219-1237. 
10.1007/s10682-010-9371-8.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1007/s10682-010-9371-8"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Independent%20hybrid%20populations%20of%20Formica%20polyctena%20X%20rufa%20wood%20ants%20%28Hymenoptera%3A%20Formicidae%29%20abound%20under%20conditions%20of%20forest%20fragmentation&amp;author=B.%20Seifert&amp;author=J.%20Kulmuni&amp;author=P.%20Pamilo&amp;journal=Evol%20Ecol&amp;volume=24&amp;pages=1219-1237&amp;publication_year=2010&amp;doi=10.1007%2Fs10682-010-9371-8"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR44" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Kronauer
 DJC, Schöning C, Boomsma JJ: Male parentage in army ants. Mol Ecol. 
2006, 15: 1147-1151. 10.1111/j.1365-294X.2005.02850.x.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1111/j.1365-294X.2005.02850.x"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=16599973"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Male%20parentage%20in%20army%20ants&amp;author=DJC.%20Kronauer&amp;author=C.%20Sch%C3%B6ning&amp;author=JJ.%20Boomsma&amp;journal=Mol%20Ecol&amp;volume=15&amp;pages=1147-1151&amp;publication_year=2006&amp;doi=10.1111%2Fj.1365-294X.2005.02850.x"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR45" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Gotwald WH, Schaefer RF: Taxonomic implications of doryline worker ant morphology: Dorylus subgenus <em class="EmphasisTypeItalic">Anomma</em> (Hymenoptera: Formicidae). Sociobiology. 1982, 7: 187-204.</span><span class="Occurrences"><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Taxonomic%20implications%20of%20doryline%20worker%20ant%20morphology%3A%20Dorylus%20subgenus%20Anomma%20%28Hymenoptera%3A%20Formicidae%29&amp;author=WH.%20Gotwald&amp;author=RF.%20Schaefer&amp;journal=Sociobiology&amp;volume=7&amp;pages=187-204&amp;publication_year=1982"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR46" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Gotwald
 WH: Predatory behavior and food preferences of driver ants in selected 
African habitats. Ann Entomol Soc Am. 1974, 67: 877-886.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1093/aesa/67.6.877"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Predatory%20behavior%20and%20food%20preferences%20of%20driver%20ants%20in%20selected%20African%20habitats&amp;author=WH.%20Gotwald&amp;journal=Ann%20Entomol%20Soc%20Am&amp;volume=67&amp;pages=877-886&amp;publication_year=1974"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR47" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Schöning C, Kinuthia W, Franks NR: Evolution of allometries in the worker caste of <em class="EmphasisTypeItalic">Dorylus</em> army ants. Oikos. 2005, 110: 231-240. 10.1111/j.0030-1299.2005.13672.x.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1111/j.0030-1299.2005.13672.x"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Evolution%20of%20allometries%20in%20the%20worker%20caste%20of%20Dorylus%20army%20ants&amp;author=C.%20Sch%C3%B6ning&amp;author=W.%20Kinuthia&amp;author=NR.%20Franks&amp;journal=Oikos&amp;volume=110&amp;pages=231-240&amp;publication_year=2005&amp;doi=10.1111%2Fj.0030-1299.2005.13672.x"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR48" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Schöning
 C, Humle T, Möbius Y, McGrew WC: The nature of culture: technological 
variation in chimpanzee predation on army ants revisited. J Hum Evol. 
2008, 55: 48-59. 10.1016/j.jhevol.2007.12.002.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1016/j.jhevol.2007.12.002"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=18275983"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=The%20nature%20of%20culture%3A%20technological%20variation%20in%20chimpanzee%20predation%20on%20army%20ants%20revisited&amp;author=C.%20Sch%C3%B6ning&amp;author=T.%20Humle&amp;author=Y.%20M%C3%B6bius&amp;author=WC.%20McGrew&amp;journal=J%20Hum%20Evol&amp;volume=55&amp;pages=48-59&amp;publication_year=2008&amp;doi=10.1016%2Fj.jhevol.2007.12.002"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR49" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Simon
 C, Frati F, Beckenbach A, Crespi B, Liu H, Flook P: Evolution, 
weighting, and phylogenetic utility of mitochondrial gene sequences and a
 compilation of conserved polymerase chain reaction primers. Ann Entomol
 Soc Am. 1994, 87: 651-701.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1093/aesa/87.6.651"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Evolution%2C%20weighting%2C%20and%20phylogenetic%20utility%20of%20mitochondrial%20gene%20sequences%20and%20a%20compilation%20of%20conserved%20polymerase%20chain%20reaction%20primers&amp;author=C.%20Simon&amp;author=F.%20Frati&amp;author=A.%20Beckenbach&amp;author=B.%20Crespi&amp;author=H.%20Liu&amp;author=P.%20Flook&amp;journal=Ann%20Entomol%20Soc%20Am&amp;volume=87&amp;pages=651-701&amp;publication_year=1994"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR50" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Kronauer DJC, Boomsma JJ, Gadau J: Microsatellite markers for the driver ant <em class="EmphasisTypeItalic">Dorylus (Anomma) molestus</em>. Mol Ecol Notes. 2004, 4: 289-290. 10.1111/j.1471-8286.2004.00645.x.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1111/j.1471-8286.2004.00645.x"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Microsatellite%20markers%20for%20the%20driver%20ant%20Dorylus%20%28Anomma%29%20molestus&amp;author=DJC.%20Kronauer&amp;author=JJ.%20Boomsma&amp;author=J.%20Gadau&amp;journal=Mol%20Ecol%20Notes&amp;volume=4&amp;pages=289-290&amp;publication_year=2004&amp;doi=10.1111%2Fj.1471-8286.2004.00645.x"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR51" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Huelsenbeck
 JP, Ronquist F: MRBAYES: Bayesian inference of phylogeny. 
Bioinformatics. 2001, 17: 754-755. 10.1093/bioinformatics/17.8.754.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1093/bioinformatics/17.8.754"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=11524383"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=MRBAYES%3A%20Bayesian%20inference%20of%20phylogeny&amp;author=JP.%20Huelsenbeck&amp;author=F.%20Ronquist&amp;journal=Bioinformatics&amp;volume=17&amp;pages=754-755&amp;publication_year=2001&amp;doi=10.1093%2Fbioinformatics%2F17.8.754"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR52" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Ronquist
 F, Huelsenbeck JP: MRBAYES 3: Bayesian phylogenetic inference under 
mixed models. Bioinformatics. 2003, 19: 1572-1574. 
10.1093/bioinformatics/btg180.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1093/bioinformatics/btg180"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=12912839"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=MRBAYES%203%3A%20Bayesian%20phylogenetic%20inference%20under%20mixed%20models&amp;author=F.%20Ronquist&amp;author=JP.%20Huelsenbeck&amp;journal=Bioinformatics&amp;volume=19&amp;pages=1572-1574&amp;publication_year=2003&amp;doi=10.1093%2Fbioinformatics%2Fbtg180"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR53" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Zwickl
 DJ: Genetic algorithm approaches for the phylogenetic analysis of large
 biological sequence datasets under the maximum likelihood criterion. 
PhD thesis. 2006, The University of Texas at Austin</span><span class="Occurrences"><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Genetic%20algorithm%20approaches%20for%20the%20phylogenetic%20analysis%20of%20large%20biological%20sequence%20datasets%20under%20the%20maximum%20likelihood%20criterion&amp;author=DJ.%20Zwickl&amp;publication_year=2006"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR54" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Nylander JAA: MrModeltest v2. Program distributed by the author. 2004, Evolutionary Biology Centre, Uppsala University</span><span class="Occurrences"><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=MrModeltest%20v2&amp;author=JAA.%20Nylander&amp;publication_year=2004"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR55" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Templeton
 AR, Crandall KA, Sing CF: A cladistics analysis of phenotypic 
associations with haplotypes inferred from restriction endonuclease 
mapping and DNA sequence data III. Cladogram estimation. Genetics. 1992,
 132: 619-633.</span><span class="Occurrences"><span class="Occurrence OccurrencePMCID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC1205162"><span><span>PubMed Central</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=1385266"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=A%20cladistics%20analysis%20of%20phenotypic%20associations%20with%20haplotypes%20inferred%20from%20restriction%20endonuclease%20mapping%20and%20DNA%20sequence%20data%20III.%20Cladogram%20estimation&amp;author=AR.%20Templeton&amp;author=KA.%20Crandall&amp;author=CF.%20Sing&amp;journal=Genetics&amp;volume=132&amp;pages=619-633&amp;publication_year=1992"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR56" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Clement M, Posada D, Crandall KA: TCS: a computer program to estimate gene genealogies. Molecular Ecology. 2000, 9: , 1657-1659.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1046/j.1365-294x.2000.01020.x"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=11050560"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=TCS%3A%20a%20computer%20program%20to%20estimate%20gene%20genealogies&amp;author=M.%20Clement&amp;author=D.%20Posada&amp;author=KA.%20Crandall&amp;journal=Molecular%20Ecology&amp;volume=9&amp;pages=%2C%201657-1659&amp;publication_year=2000"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR57" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Goudet J: FSTAT, a program to estimate and test gene diversities and fixation indices (version 2.9.3.2). [<span xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="ExternalRef"><a href="http://www2.unil.ch/popgen/softwares/fstat.htm" target="_blank" class="ext-link"><span class="RefSource">http://www2.unil.ch/popgen/softwares/fstat.htm</span></a></span>]</span><span class="Occurrences"></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR58" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Pritchard
 JK, Stephens M, Donnelly P: Inference of population structure using 
multilocus genotype data. Genetics. 2000, 155: 945-959.</span><span class="Occurrences"><span class="Occurrence OccurrencePMCID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC1461096"><span><span>PubMed Central</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=10835412"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Inference%20of%20population%20structure%20using%20multilocus%20genotype%20data&amp;author=JK.%20Pritchard&amp;author=M.%20Stephens&amp;author=P.%20Donnelly&amp;journal=Genetics&amp;volume=155&amp;pages=945-959&amp;publication_year=2000"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR59" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Falush
 D, Stephens M, Pritchard JK: Inference of population structure: 
Extensions to linked loci and correlated allele frequencies. Genetics. 
164: 1567-1587.<span xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Occurrences"><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="https://scholar.google.com/scholar?hl=en&amp;q=Falush%20D%2C%20Stephens%20M%2C%20Pritchard%20JK%3A%20Inference%20of%20population%20structure%3A%20Extensions%20to%20linked%20loci%20and%20correlated%20allele%20frequencies.%20Genetics.%20164%3A%201567-1587."><span><span>Google Scholar</span></span></a></span></span></span><span class="Occurrences"></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR60" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Rosenberg NA: DISTRUCT: a program for the graphical display of population structure. Mol Ecol Notes. 2004, 4: 137-138.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1046/j.1471-8286.2003.00566.x"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=DISTRUCT%3A%20a%20program%20for%20the%20graphical%20display%20of%20population%20structure&amp;author=NA.%20Rosenberg&amp;journal=Mol%20Ecol%20Notes&amp;volume=4&amp;pages=137-138&amp;publication_year=2004"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR61" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Piry
 S, Alapetite A, Cornuet JM, Paetkau D, Baudouin L, Estoup A: 
GENECLASS2: A software for genetic assignment and first-generation 
migrant detection. J Hered. 2004, 95: 536-539. 10.1093/jhered/esh074.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1093/jhered/esh074"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=15475402"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=GENECLASS2%3A%20A%20software%20for%20genetic%20assignment%20and%20first-generation%20migrant%20detection&amp;author=S.%20Piry&amp;author=A.%20Alapetite&amp;author=JM.%20Cornuet&amp;author=D.%20Paetkau&amp;author=L.%20Baudouin&amp;author=A.%20Estoup&amp;journal=J%20Hered&amp;volume=95&amp;pages=536-539&amp;publication_year=2004&amp;doi=10.1093%2Fjhered%2Fesh074"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR62" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Rannala
 B, Mountain JL: Detecting immigration by using multilocus genotypes. 
Proc Natl Acad Sci USA. 1997, 94: 9197-9201. 10.1073/pnas.94.17.9197.</span><span class="Occurrences"><span class="Occurrence OccurrencePMCID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC23111"><span><span>PubMed Central</span></span></a></span><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1073/pnas.94.17.9197"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=9256459"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Detecting%20immigration%20by%20using%20multilocus%20genotypes&amp;author=B.%20Rannala&amp;author=JL.%20Mountain&amp;journal=Proc%20Natl%20Acad%20Sci%20USA&amp;volume=94&amp;pages=9197-9201&amp;publication_year=1997&amp;doi=10.1073%2Fpnas.94.17.9197"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR63" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Paetkau
 D, Slade R, Burden M, Estoup A: Genetic assignment methods for the 
direct, real-time estimation of migration rate: a simulation-based 
exploration of accuracy and power. Mol Ecol. 2004, 13: 55-65. 
10.1046/j.1365-294X.2004.02008.x.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1046/j.1365-294X.2004.02008.x"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=14653788"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Genetic%20assignment%20methods%20for%20the%20direct%2C%20real-time%20estimation%20of%20migration%20rate%3A%20a%20simulation-based%20exploration%20of%20accuracy%20and%20power&amp;author=D.%20Paetkau&amp;author=R.%20Slade&amp;author=M.%20Burden&amp;author=A.%20Estoup&amp;journal=Mol%20Ecol&amp;volume=13&amp;pages=55-65&amp;publication_year=2004&amp;doi=10.1046%2Fj.1365-294X.2004.02008.x"><span><span>Google Scholar</span></span></a></span></span></cite></li></ol></div></section><section xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" class="Section1 RenderAsSection1 c-section" id="CopyrightMessage"><h2 class="Heading js-ToggleCollapseSection c-section__heading" data-sticky-update="true"><a href="#KronaueretallicenseeBioMedCentralLtd" data-toggle="collapse" data-toggle-disable-clickoff="" data-target="KronaueretallicenseeBioMedCentralLtd">Copyright</a></h2><div class="js-CollapseSection CollapseSection c-section__content" data-test="collapse-section" id="KronaueretallicenseeBioMedCentralLtd"><div xmlns="http://www.w3.org/1999/xhtml" class="ArticleCopyright">©&nbsp;Kronauer et al; licensee BioMed Central Ltd.&nbsp;2011<div class="CopyrightComment">
                <p xmlns="" class="SimplePara">This article is published
 under license to BioMed Central Ltd. This is an Open Access article 
distributed under the terms of the Creative Commons Attribution License (<span xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="ExternalRef"><a href="http://creativecommons.org/licenses/by/2.0" target="_blank" class="ext-link"><span class="RefSource">http://creativecommons.org/licenses/by/2.0</span></a></span>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p>
              </div></div></div></section></div>
                        

    <div class="FulltextWrapper">
        <section id="Comments" class="Section1 RenderAsSection1 c-section inline-comments">
            <h2 class="Heading c-section__heading" data-toggle="collapse" data-target="Comments"><a href="#Comments">Comments</a></h2>
            <div class="CollapseSection" data-test="collapse-section" id="Comments">
                
                    <p>By submitting a comment you agree to abide by our <a href="https://www.biomedcentral.com/terms-and-conditions">Terms</a> and <a href="https://www.biomedcentral.com/community-guidelines">Community Guidelines</a>.
 If you find something abusive or that does not comply with our terms or
 guidelines please flag it as inappropriate. Please note that comments 
may be removed without notice if they are flagged by another user or do 
not comply with our community guidelines.</p>
                    <div id="disqus_thread"><iframe id="dsq-app935" name="dsq-app935" allowtransparency="true" scrolling="no" tabindex="0" title="Disqus" style="width: 1px ! important; min-width: 100% ! important; border: medium none ! important; overflow: hidden ! important; height: 0px ! important;" src="https://disqus.com/embed/comments/?base=default&amp;f=frontiersinzoology-biomedcentral-com&amp;t_i=10.1186%2F1742-9994-8-20&amp;t_u=https%3A%2F%2Ffrontiersinzoology.biomedcentral.com%2Farticles%2F10.1186%2F1742-9994-8-20&amp;t_d=Hybridization%20in%20East%20African%20swarm-raiding%20army%20ants&amp;t_t=Hybridization%20in%20East%20African%20swarm-raiding%20army%20ants&amp;s_o=default&amp;d_m=0#version=d136ff79a544be2a7646da439609db5a" frameborder="0" width="100%"></iframe></div>
                

                
            </div>
        </section>
    </div>


                    </article>
                </main>

                <div class="c-content-layout__side u-text-sm">
                    <aside>
                        <div class="p-separator">
                            
        <p class="u-margin-bottom-sm">
            <a id="articlePdf" data-test="download-pdf" class="c-button c-button--primary c-button--full-width" target="_blank" rel="noopener" href="https://frontiersinzoology.biomedcentral.com/track/pdf/10.1186/1742-9994-8-20" data-track="click" data-track-category="PDF Download" data-track-action="Click download article PDF" data-track-label="10.1186/1742-9994-8-20">Download PDF</a>
        </p>
    
                            
                            
    <button type="button" class="c-button c-button--expandable" data-toggle="collapse" data-target="export-citations" data-test="export-citations-button" data-sticky-update="true" aria-controls="export-citations" aria-expanded="false">
        <span class="c-dropdown__toggle">Export citations</span>
    </button>
    <div id="export-citations" class="c-export-citations" data-test="export-citations-content">
        <div class="c-export-citations__section">
            <h4 class="c-export-citations__title">Papers, Zotero, Reference Manager, RefWorks (<strong>.RIS</strong>)</h4>
            <ul class="c-export-citations__list" id="articleReferences">
                <li><a data-track="click" data-track-category="Article Page" data-track-action="Download Papers, Zotero, Reference Manager, RefWorks article citations" data-track-label="10.1186/1742-9994-8-20" href="http://citation-needed.springer.com/v2/references/10.1186/1742-9994-8-20?format=refman&amp;flavour=citation">
                    <svg class="c-export-citations__icon" width="12" height="12" aria-hidden="true">
                        <use xlink:href="#icon-download"></use>
                    </svg>
                    <span class="u-visually-hidden">Download </span>Citations
                </a></li>
                <li><a data-track="click" data-track-category="Article Page" data-track-action="Download Papers, Zotero, Reference Manager, RefWorks references" data-track-label="10.1186/1742-9994-8-20" href="http://citation-needed.springer.com/v2/references/10.1186/1742-9994-8-20?format=refman&amp;flavour=references">
                    <svg class="c-export-citations__icon" width="12" height="12" aria-hidden="true">
                        <use xlink:href="#icon-download"></use>
                    </svg>
                    <span class="u-visually-hidden">Download </span>References
                </a></li>
                <li><a data-track="click" data-track-category="Article Page" data-track-action="Download Papers, Zotero, Reference Manager, RefWorks citations and references" data-track-label="10.1186/1742-9994-8-20" href="http://citation-needed.springer.com/v2/references/10.1186/1742-9994-8-20?format=refman&amp;flavour=full">
                    <svg class="c-export-citations__icon" width="12" height="12" aria-hidden="true">
                        <use xlink:href="#icon-download"></use>
                    </svg>
                    <span class="u-visually-hidden">Download </span>Both
                </a></li>
            </ul>
        </div>
        <div class="c-export-citations__section">
            <h4 class="c-export-citations__title">EndNote (<strong>.ENW</strong>)</h4>
            <ul class="c-export-citations__list">
                <li><a data-track="click" data-track-category="Article Page" data-track-action="Download EndNote article citations" data-track-label="10.1186/1742-9994-8-20" href="http://citation-needed.springer.com/v2/references/10.1186/1742-9994-8-20?format=endnote&amp;flavour=citation">
                    <svg class="c-export-citations__icon" width="12" height="12" aria-hidden="true">
                        <use xlink:href="#icon-download"></use>
                    </svg>
                    <span class="u-visually-hidden">Download </span>Citations
                </a></li>
                <li><a data-track="click" data-track-category="Article Page" data-track-action="Download EndNote references" data-track-label="10.1186/1742-9994-8-20" href="http://citation-needed.springer.com/v2/references/10.1186/1742-9994-8-20?format=endnote&amp;flavour=references">
                    <svg class="c-export-citations__icon" width="12" height="12" aria-hidden="true">
                        <use xlink:href="#icon-download"></use>
                    </svg>
                    <span class="u-visually-hidden">Download </span>References
                </a></li>
                <li><a data-track="click" data-track-category="Article Page" data-track-action="Download EndNote citations and references" data-track-label="10.1186/1742-9994-8-20" href="http://citation-needed.springer.com/v2/references/10.1186/1742-9994-8-20?format=endnote&amp;flavour=full">
                    <svg class="c-export-citations__icon" width="12" height="12" aria-hidden="true">
                        <use xlink:href="#icon-download"></use>
                    </svg>
                    <span class="u-visually-hidden">Download </span>Both
                </a></li>
            </ul>
        </div>
        <div class="c-export-citations__section">
            <h4 class="c-export-citations__title">Mendeley, JabRef (<strong>.BIB</strong>)</h4>
            <ul class="c-export-citations__list">
                <li><a data-track="click" data-track-category="Article Page" data-track-action="Download Mendeley, JabRef article citations" data-track-label="10.1186/1742-9994-8-20" href="http://citation-needed.springer.com/v2/references/10.1186/1742-9994-8-20?format=bibtex&amp;flavour=citation">
                    <svg class="c-export-citations__icon" width="12" height="12" aria-hidden="true">
                        <use xlink:href="#icon-download"></use>
                    </svg>
                    <span class="u-visually-hidden">Download </span>Citations
                </a></li>
                <li><a data-track="click" data-track-category="Article Page" data-track-action="Download Mendeley, JabRef references" data-track-label="10.1186/1742-9994-8-20" href="http://citation-needed.springer.com/v2/references/10.1186/1742-9994-8-20?format=bibtex&amp;flavour=references">
                    <svg class="c-export-citations__icon" width="12" height="12" aria-hidden="true">
                        <use xlink:href="#icon-download"></use>
                    </svg>
                    <span class="u-visually-hidden">Download </span>References
                </a></li>
                <li><a data-track="click" data-track-category="Article Page" data-track-action="Download Mendeley, JabRef citations and references" data-track-label="10.1186/1742-9994-8-20" href="http://citation-needed.springer.com/v2/references/10.1186/1742-9994-8-20?format=bibtex&amp;flavour=full">
                    <svg class="c-export-citations__icon" width="12" height="12" aria-hidden="true">
                        <use xlink:href="#icon-download"></use>
                    </svg>
                    <span class="u-visually-hidden">Download </span>Both
                </a></li>
            </ul>
        </div>
    </div>

                        </div>
                        
    
 
                        
    
 
                        <div class="p-separator">
                            <h3 class="p-section-title">Metrics</h3>
                            <ul class="c-list-group c-list-group--md">
                                
            <li class="c-list-group__item">
                Article accesses:
                
                    4914
                
            </li>
        
                                
            <li class="c-list-group__item">
                Citations: 7 <a href="http://citations.springer.com/item?doi=10.1186/1742-9994-8-20" rel="noopener" target="_blank">more information</a>
            </li>
        
                                <div id="altmetric-loader" class="placeholder" data-replace="true" data-placeholder="/placeholder/v1/article/altmetric/10.1186/1742-9994-8-20 #altmetric-container"></div>
                            </ul>
                        </div>
                        
    <div class="p-separator">
        <div class="js-shown">
            <h3 class="p-section-title">Share this article</h3>
            <ul data-test="social-media-share-buttons" class="c-social-links">
                <li class="c-social-links__item">
                    <button id="shareOnTwitter" class="js-btnShareOnTwitter" data-track="click-nonlink" data-track-category="Article Page" data-track-action="Share Twitter" data-track-label="10.1186/1742-9994-8-20">
                        <span class="u-visually-hidden" data-track="click-nonlink" data-track-category="Article Page" data-track-action="Share Twitter" data-track-label="10.1186/1742-9994-8-20">Share on Twitter</span>
                        <svg class="c-icon c-social-links__icon" width="24" height="24" aria-hidden="true" data-track="click-nonlink" data-track-category="Article Page" data-track-action="Share Twitter" data-track-label="10.1186/1742-9994-8-20">
                            <use xlink:href="#icon-twitter"></use>
                        </svg>
                    </button>
                </li>
                <li class="c-social-links__item">
                    <button id="shareOnFacebook" class="js-btnShareOnFacebook" data-track="click-nonlink" data-track-category="Article Page" data-track-action="Share Facebook" data-track-label="10.1186/1742-9994-8-20">
                        <span class="u-visually-hidden" data-track="click-nonlink" data-track-category="Article Page" data-track-action="Share Facebook" data-track-label="10.1186/1742-9994-8-20">Share on Facebook</span>
                        <svg class="c-icon c-social-links__icon" width="24" height="24" aria-hidden="true" data-track="click-nonlink" data-track-category="Article Page" data-track-action="Share Facebook" data-track-label="10.1186/1742-9994-8-20">
                            <use xlink:href="#icon-facebook"></use>
                        </svg>
                    </button>
                </li>
                <li class="c-social-links__item">
                    <button id="shareOnLinkedIn" class="js-btnShareOnLinkedIn" data-track="click-nonlink" data-track-category="Article Page" data-track-action="Share LinkedIn" data-track-label="10.1186/1742-9994-8-20">
                        <span class="u-visually-hidden" data-track="click-nonlink" data-track-category="Article Page" data-track-action="Share LinkedIn" data-track-label="10.1186/1742-9994-8-20">Share on LinkedIn</span>
                        <svg class="c-icon c-social-links__icon" width="24" height="24" aria-hidden="true" data-track="click-nonlink" data-track-category="Article Page" data-track-action="Share LinkedIn" data-track-label="10.1186/1742-9994-8-20">
                            <use xlink:href="#icon-linkedin"></use>
                        </svg>
                    </button>
                </li>
                <li class="c-social-links__item">
                    <button id="shareOnWeibo" class="js-btnShareOnWeibo" data-track="click-nonlink" data-track-category="Article Page" data-track-action="Share Weibo" data-track-label="10.1186/1742-9994-8-20">
                        <span class="u-visually-hidden" data-track="click-nonlink" data-track-category="Article Page" data-track-action="Share Weibo" data-track-label="10.1186/1742-9994-8-20">Share on Weibo</span>
                        <svg class="c-icon c-social-links__icon" width="24" height="24" aria-hidden="true" data-track="click-nonlink" data-track-category="Article Page" data-track-action="Share Weibo" data-track-label="10.1186/1742-9994-8-20">
                            <use xlink:href="#icon-weibo"></use>
                        </svg>
                    </button>
                </li>
                <li class="c-social-links__item">
                    <button id="shareOnGooglePlus" class="js-btnShareOnGplus" data-track="click-nonlink" data-track-category="Article Page" data-track-action="Share Google+" data-track-label="10.1186/1742-9994-8-20">
                        <span class="u-visually-hidden" data-track="click-nonlink" data-track-category="Article Page" data-track-action="Share Google+" data-track-label="10.1186/1742-9994-8-20">Share on Google Plus</span>
                        <svg class="c-icon c-social-links__icon" width="24" height="24" aria-hidden="true" data-track="click-nonlink" data-track-category="Article Page" data-track-action="Share Google+" data-track-label="10.1186/1742-9994-8-20">
                            <use xlink:href="#icon-gplus"></use>
                        </svg>
                    </button>
                </li>
                <li class="c-social-links__item">
                    <button id="shareOnReddit" class="js-btnShareOnReddit" data-track="click-nonlink" data-track-category="Article Page" data-track-action="Share Reddit" data-track-label="10.1186/1742-9994-8-20">
                        <span class="u-visually-hidden" data-track="click-nonlink" data-track-category="Article Page" data-track-action="Share Reddit" data-track-label="10.1186/1742-9994-8-20">Share on Reddit</span>
                        <svg class="c-icon c-social-links__icon" width="24" height="24" aria-hidden="true" data-track="click-nonlink" data-track-category="Article Page" data-track-action="Share Reddit" data-track-label="10.1186/1742-9994-8-20">
                            <use xlink:href="#icon-reddit"></use>
                        </svg>
                    </button>
                </li>
            </ul>
        </div>
        
            <div data-sharedit="container">
                <a class="c-button u-margin-top-sm" href="https://frontiersinzoology.biomedcentral.com/articles/10.1186/1742-9994-8-20/sharedit" rel="nofollow" data-sharedit="button" data-track="click" data-track-category="sharedit" data-track-action="click article sharedit button" data-track-label="10.1186/1742-9994-8-20">Get shareable link</a>
            </div>
        
    </div>
 
                        
    
 
                        <div class="p-separator">
                            <h3 class="p-section-title">Other Actions</h3>
                            <ul class="c-list-group c-list-group--bordered c-list-group c-list-group--md">
                                <li class="c-list-group__item">
                                    
    <a data-track="click" data-track-category="Article Page" data-track-action="Order article reprints" data-track-label="10.1186/1742-9994-8-20" href="https://www.odysseypress.com/onlinehost/reprint_order.php?type=A&amp;page=0&amp;journal=738&amp;doi=10.1186%2F1742-9994-8-20&amp;volume=8&amp;issue=1&amp;title=Hybridization+in+East+African+swarm-raiding+army+ants&amp;author_name=Daniel+JC+Kronauer&amp;start_page=1&amp;end_page=14">Order reprint</a>

                                </li>
                            </ul>
                        </div>
                    </aside>
                    
    <div id="FulltextAd" class="adsbox c-ad c-ad--skyscraper">
        <div class="c-ad__inner" data-component="sticky" data-test="fulltext-ad">
            <p class="c-ad__label">Advertisement</p>
            <div class="c-ad__content">
                <span id="pbgrd-43415139"></span>
            </div>
        </div>
    </div>

                </div>
            </div>
        </div>


        
            <div class="c-journal-footer">
                <div class="c-journal-footer__inner">
                    <div class="c-journal-footer__summary">
                        <h4 class="c-journal-title c-journal-title--footer">
                            <img class="c-journal-title__logo" src="logo.svg">
                            <span class="c-journal-title__text">Frontiers in Zoology</span>
                        </h4>
                         <p class="c-journal-footer__issn">ISSN: 1742-9994</p>
                    </div>
                    
                        <div class="c-journal-footer__contact">
                            <h4 class="c-journal-footer__contact-title c-journal-footer__contact-title--adjust-for-logo">Contact us</h4>
                            <ul class="c-journal-footer__contact-list">
                                
                                    <li class="c-journal-footer__contact-item">Submission enquiries: <a href="https://www.editorialmanager.com/finz" target="_blank">Access here and click Contact Us</a></li>
                                
                                
                                    <li class="c-journal-footer__contact-item">General enquiries: <a href="mailto:info@biomedcentral.com">info@biomedcentral.com</a></li>
                                
                            </ul>
                        </div>
                    
                </div>
            </div>
        
        
    <img rel="nofollow" class="tracker" style="display:none" src="1742-9994-8-20.png" alt="">

         
    <footer>
        <div class="c-publisher-footer" data-test="publisher-footer">
    <div class="u-container">
        
        <div class="c-flex c-flex--wrap c-flex--space-between" role="navigation" data-test="publisher-footer-menu">
            <div class="c-flex" aria-label="Publisher footer links">
                
                    
                        <ul class="c-list-group c-list-group--sm u-margin-right-lg u-margin-bottom-md">
                            
                                <li class="c-list-group__item">
                                    <a class="u-gray-link" href="http://blogs.biomedcentral.com/">Read more on our blogs</a>
                                </li>
                            
                                <li class="c-list-group__item">
                                    <a class="u-gray-link" href="https://www.biomedcentral.com/login">Receive BMC newsletters</a>
                                </li>
                            
                                <li class="c-list-group__item">
                                    <a class="u-gray-link" href="https://www.biomedcentral.com/account">Manage article alerts</a>
                                </li>
                            
                                <li class="c-list-group__item">
                                    <a class="u-gray-link" href="https://authorservices.springernature.com/go/10BMC">Language editing for authors</a>
                                </li>
                            
                                <li class="c-list-group__item">
                                    <a class="u-gray-link" href="http://authorservices.springernature.com/scientific-editing/">Scientific editing for authors</a>
                                </li>
                            
                        </ul>
                    
                        <ul class="c-list-group c-list-group--sm u-margin-right-lg u-margin-bottom-md">
                            
                                <li class="c-list-group__item">
                                    <a class="u-gray-link" href="https://www.biomedcentral.com/about/policies">Policies</a>
                                </li>
                            
                                <li class="c-list-group__item">
                                    <a class="u-gray-link" href="https://www.biomedcentral.com/accessibility">Accessibility</a>
                                </li>
                            
                                <li class="c-list-group__item">
                                    <a class="u-gray-link" href="https://www.biomedcentral.com/about/press-centre">Press center</a>
                                </li>
                            
                        </ul>
                    
                        <ul class="c-list-group c-list-group--sm u-margin-right-lg u-margin-bottom-md">
                            
                                <li class="c-list-group__item">
                                    <a class="u-gray-link" href="https://www.biomedcentral.com/about/contact-us">Support and Contact</a>
                                </li>
                            
                                <li class="c-list-group__item">
                                    <a class="u-gray-link" href="https://biomedcentral.typeform.com/to/VLXboo">Leave feedback</a>
                                </li>
                            
                                <li class="c-list-group__item">
                                    <a class="u-gray-link" href="https://www.biomedcentral.com/about/jobs">Careers</a>
                                </li>
                            
                        </ul>
                    
                
            </div>
            <div class="u-margin-bottom-lg">
                <h3 id="social-menu" class="u-text-sm u-reset-margin u-text-normal">Follow BMC</h3>
                <ul class="c-flex u-reset-list" data-test="footer-social-links">
                    
                        <li class="u-margin-top-xs u-margin-right-xs">
                            <a href="https://twitter.com/biomedcentral" class="u-gray-link">
                                <span class="u-visually-hidden">BMC Twitter page</span>
                                <svg class="c-icon" width="24" height="24" aria-hidden="true">
                                    <use xlink:href="#icon-twitter-bordered"></use>
                                </svg>
                            </a>
                        </li>
                    
                        <li class="u-margin-top-xs u-margin-right-xs">
                            <a href="https://www.facebook.com/BioMedCentral" class="u-gray-link">
                                <span class="u-visually-hidden">BMC Facebook page</span>
                                <svg class="c-icon" width="24" height="24" aria-hidden="true">
                                    <use xlink:href="#icon-facebook-bordered"></use>
                                </svg>
                            </a>
                        </li>
                    
                        <li class="u-margin-top-xs u-margin-right-xs">
                            <a href="https://plus.google.com/106226337984152901734/" class="u-gray-link">
                                <span class="u-visually-hidden">BMC Google Plus page</span>
                                <svg class="c-icon" width="24" height="24" aria-hidden="true">
                                    <use xlink:href="#icon-gplus-bordered"></use>
                                </svg>
                            </a>
                        </li>
                    
                        <li class="u-margin-top-xs u-margin-right-xs">
                            <a href="http://www.weibo.com/biomedcentral" class="u-gray-link">
                                <span class="u-visually-hidden">BMC Weibo page</span>
                                <svg class="c-icon" width="24" height="24" aria-hidden="true">
                                    <use xlink:href="#icon-weibo-bordered"></use>
                                </svg>
                            </a>
                        </li>
                    
                </ul>
            </div>
        </div>
        <p class="u-reset-margin">
            By using this website, you agree to our
            <a class="u-gray-link" href="https://www.biomedcentral.com/terms-and-conditions">Terms and Conditions</a>,
            <a class="u-gray-link" href="https://www.biomedcentral.com/privacy-statement">Privacy
                statement</a> and
            <a class="u-gray-link" href="https://www.biomedcentral.com/cookies" data-test="cookie-link">Cookies</a> policy.
            
                <a class="optanon-toggle-display u-gray-link" href="javascript:void(0);">Manage the cookies</a> we use in the preference centre.
            
        </p>
    </div>
</div>

        <div class="c-sn-footer c-sn-footer--neutral">
    <div class="u-container">
        <img src="logo-springernature-8bd4e8d82f.svg" class="c-sn-footer__logo" alt="Springer Nature Logo" itemprop="logo">
        <p data-test="copyright">© 2019 BioMed Central Ltd unless otherwise stated. Part of
            <a href="http://www.springernature.com/" itemscope="" itemtype="http://schema.org/Organization" itemid="#parentOrganization">Springer Nature</a>.
        </p>
    </div>
</div>

    </footer>

    </div>

    
    <script type="text/javascript" src="bmc.js" async="async"></script>
    <noscript>
	<img hidden src="https://verify.nature.com/verify/nature.png" border="0" width="0" height="0" style="display: none">
</noscript>

        
<script>
    (function() {
        if (window.config && window.config.mustardcut) {
            var appScript = document.createElement('script');
            appScript.src = '/static/js/app-bundle-06a60a9601.js';
            appScript.async = false;
            document.body.appendChild(appScript);
        }
    })();
</script><script src="app-bundle-06a60a9601.js"></script>

    


<script type="text/javascript" id="polyfill-matches">Element.prototype.matches||(Element.prototype.matches=Element.prototype.matchesSelector||Element.prototype.mozMatchesSelector||Element.prototype.msMatchesSelector||Element.prototype.oMatchesSelector||Element.prototype.webkitMatchesSelector||function(a){a=(this.document||this.ownerDocument).querySelectorAll(a);for(var b=a.length;0<=--b&&a.item(b)!==this;);return-1<b});</script><script type="text/javascript" id="gpt-control" src="gpt.js"></script><script type="text/javascript" id="krux-control-bmc">window.Krux||((Krux=function(){Krux.q.push(arguments)}).q=[]);(function(){var a=document.createElement("script");a.type="text/javascript";a.async=!0;a.src=("https:"===location.protocol?"https:":"http:")+"//cdn.krxd.net/controltag/KDqylSLE.js";var b=document.getElementsByTagName("script")[0];b.parentNode.insertBefore(a,b)})();</script><script type="text/javascript" id="krux-interchange-bmc">window.Krux||((Krux=function(){Krux.q.push(arguments)}).q=[]);(function(){function c(a){a="kxmacmillan_"+a;try{var b=window.localStorage}catch(e){b=null}return b?b[a]||"":navigator.cookieEnabled?(b=document.cookie.match(a+"\x3d([^;]*)"))&&unescape(b[1])||"":""}Krux.user=c("user");Krux.segments=c("segs")?c("segs").split(","):[];var a=[];Krux.user&&a.push("kuid\x3d"+Krux.user);for(var d=0;d<Krux.segments.length;d++)a.push("ksg\x3d"+Krux.segments[d]);Krux.dfppKeyValues=a.length?a.join(";")+";":""})();</script><script type="text/javascript" id="krux-consent">var allowed=google_tag_manager["GTM-TDGJHK"].macro(39);allowed=!0===allowed?1:0;Krux("consent:set",{dc:allowed,al:allowed,tg:allowed,cd:!1,sh:!1,re:!1},function(a,b){});</script><script type="text/javascript" id="gpt-retrieve-ads">function splitKeys(a){var d=[],e="",g=[],h=a.split(";");for(a=0;a<h.length;++a){void 0!==k&&(e=k);var b=h[a].split("\x3d");var k=b[0];k!==e&&(0<e.length&&0<d.length&&g.push([e,d]),d=[]);if(2===b.length&&""!==b[0]&&""!==b[1]){var c=b[1].split(",");for(b=0;b<c.length;++b)d.push(c[b])}}0<k.length&&0<d.length&&g.push([k,d]);return g}
function splitSizes(a){var d=[];if(null!==a){var e=0<=a.indexOf("|")?a.split("|"):a.split(",");for(a=0;a<e.length;++a){var g=e[a].split("x");var h=parseInt(g[0],10);var b=parseInt(g[1],10);2===g.length&&!isNaN(h)&&!isNaN(b)&&0<=h&&0<=b&&d.push([h,b])}}return d}function debounce(a,d){var e=null,g=null;return function(){var h=this,b=Number(new Date),k=arguments;e&&b<e+d?(clearTimeout(g),g=setTimeout(function(){e=b;a.apply(h,k)},d)):(e=b,a.apply(h,k))}}
function addScrollEvent(a){window.addEventListener?window.addEventListener("scroll",a,!1):window.attachEvent("onscroll",a)}function removeScrollEvent(a){window.removeEventListener?window.removeEventListener("scroll",a,!1):window.detachEvent("scroll",a)}function getAdContainers(){return document.querySelectorAll("div[data-gpt-unitpath]")}function isLoggedIn(){return 0<=document.cookie.indexOf("OSCAR_SESSION_COOKIE")?"logged\x3dy;":"logged\x3dn;"}
(function(a,d){function e(f){var a="test";a=a.replace(/[\[\]]/g,"\\$\x26");a=new RegExp("[?\x26]"+a+"(\x3d([^\x26#]*)|\x26|#|$)");a=(a=a.exec(window.location.href))?a[2]?decodeURIComponent(a[2].replace(/\+/g," ")):"":null;a=a&&"ads"===a?"adtype\x3dtest;":"";f=f.getAttribute("data-gpt-targeting");var c=document.querySelector("[data-ad-targeting-search-terms]")?"search\x3d"+document.querySelector("[data-ad-targeting-search-terms]").innerText+";":!1;var d=isLoggedIn();a&&-1===f.indexOf(a)&&(f+=a);c&&
(f+=c);return f+d+b}function g(a,c){for(var f=a.length,b=[];f--;)c(a[f],f)&&(b.push(a[f].slot),a.splice(f,1));b.length&&googletag.pubads().refresh(b);return a}function h(f){var c=Math.max(document.documentElement.clientHeight,a.innerHeight||0);return g(f,function(a){a=document.getElementById(a.divId);a=a.getBoundingClientRect();a=a.top-300;return c>a})}var b,k=getAdContainers(),c=[];d&&(b=d.dfppKeyValues?d.dfppKeyValues:"");a.googletag=a.googletag||{};a.googletag.cmd=a.googletag.cmd||[];for(var m=
0;k[m];++m){var l=k[m];splitKeys(e(l));c.push({divId:l.getAttribute("id"),adUnitPath:l.getAttribute("data-gpt-unitpath"),sizeArray:splitSizes(l.getAttribute("data-gpt-sizes")),targeting:splitKeys(e(l))})}googletag.cmd.push(function(){googletag.pubads().setRequestNonPersonalizedAds(google_tag_manager["GTM-TDGJHK"].macro(43)?0:1);googletag.pubads().disableInitialLoad();googletag.enableServices()});googletag.cmd.push(function(){for(var a=0;c[a];++a)try{c[a].slot=googletag.defineSlot(c[a].adUnitPath,c[a].sizeArray,c[a].divId).addService(googletag.pubads());
for(var b=0,d=c[a].targeting.length;b<d;++b)2===c[a].targeting[b].length&&""!==c[a].targeting[b][0]&&""!==c[a].targeting[b][1]&&c[a].slot.setTargeting(c[a].targeting[b][0],c[a].targeting[b][1])}catch(p){console.log("failed to create slot for",c[a])}});googletag.cmd.push(function(){for(var a=0;c[a];++a)googletag.display(c[a].divId)});googletag.cmd.push(function(){c=h(c)});var n=debounce(function(){googletag.cmd.push(function(){c=h(c);c.length||removeScrollEvent(n)})},250);addScrollEvent(n)})(window,
window.Krux);</script><script type="text/javascript" id="crossmark-script" src="widget.js"></script><script type="text/javascript" id="recommended-script" src="entry-point"></script><script type="text/javascript" id="disqus-script">window.disqus_config=function(){this.page.url="https://frontiersinzoology.biomedcentral.com/articles/10.1186\/1742-9994-8-20";this.page.identifier="10.1186\/1742-9994-8-20"};(function(){var a=document,b=a.createElement("script");b.src="//frontiersinzoology-biomedcentral-com.disqus.com/embed.js";b.setAttribute("data-timestamp",+new Date);(a.head||a.body).appendChild(b)})();</script>
<noscript>Please enable JavaScript to view the <a href="https://disqus.com/?ref_noscript">comments powered by Disqus.</a></noscript><script type="text/javascript" id="">(function(){var a=document,b=a.createElement("script");f=a.getElementsByTagName("script")[0];b.src="//s3.amazonaws.com/ki.js/47412/aC6.js";f.parentNode.insertBefore(b,f)})();</script><iframe style="width: 0px; height: 0px; bottom: 0px; right: 0px; z-index: 100000; position: fixed;" scrolling="no" src="https://recommended.springernature.com/v2211/generated/client.html?source=https://frontiersinzoology.biomedcentral.com" name="uptodate-client" id="uptodate-client" title="Personalised recommendations" allowtransparency="true" frameborder="0"></iframe><iframe style="display: none;"></iframe><iframe id="qualaroo_dnt_frame" style="width: 1px; height: 1px; display: none; opacity: 0;" src="//dnt.qualaroo.com/frame.html"></iframe><div id="optanon" class="modern"><div id="optanon-popup-bg"></div><div id="optanon-popup-wrapper" role="dialog" aria-modal="true" tabindex="-1" lang="en-GB"><div id="optanon-popup-top"><a href="#" onclick="Optanon.TriggerGoogleAnalyticsEvent('OneTrust Cookie Consent', 'Preferences Close Button');" class="optanon-close-link optanon-close optanon-close-ui" title="Close Preference Centre"><div id="optanon-close" style="background: url(https://optanon.blob.core.windows.net/skins/default_flat_bottom_two_button_white/v2/images/optanon-pop-up-close.png);width:34px;height:34px;"></div></a></div><div id="optanon-popup-body"><div id="optanon-popup-body-left"><div id="optanon-popup-body-left-shading"></div><div id="optanon-branding-top-logo" style="background-image: url(https://optanon.blob.core.windows.net/logos/5138/5138:biomedcentral.com/bmc-logo.png) !important;"></div><ul id="optanon-menu"><li class="menu-item-on menu-item-about" title="Your Privacy"><p><a href="#">Your Privacy</a></p></li><li class="menu-item-necessary menu-item-on" title="Strictly Necessary Cookies"><p><a href="#">Strictly Necessary Cookies</a></p></li><li class="menu-item-on menu-item-performance" title="Performance &amp; Analytics Cookies"><p><a href="#">Performance &amp; Analytics Cookies</a></p></li><li class="menu-item-on menu-item-functional" title="Functional Cookies"><p><a href="#">Functional Cookies</a></p></li><li class="menu-item-on menu-item-advertising" title="Targeting Cookies"><p><a href="#">Targeting Cookies</a></p></li><li class="menu-item-moreinfo menu-item-off" title="More Information"><p><a target="_blank" href="https://www.biomedcentral.com/cookies" onclick="Optanon.TriggerGoogleAnalyticsEvent('OneTrust Cookie Consent', 'Preferences Cookie Policy');">More Information</a></p></li></ul></div><div id="optanon-popup-body-right"><h2 aria-label="true">Privacy Preference Centre</h2><h3></h3><div id="optanon-popup-more-info-bar"><div class="optanon-status"><div class="optanon-status-editable"><form><fieldset><p><input value="check" id="chkMain" checked="checked" class="optanon-status-checkbox" type="checkbox"><label for="chkMain">Active</label></p></fieldset></form></div><div class="optanon-status-always-active optanon-status-on"><p>Always Active</p></div></div></div><div id="optanon-main-info-text"></div></div><div class="optanon-bottom-spacer"></div></div><div id="optanon-popup-bottom"> <a href="https://onetrust.com/poweredbyonetrust" target="_blank"><div id="optanon-popup-bottom-logo" style="background: url(https://optanon.blob.core.windows.net/skins/default_flat_bottom_two_button_white/v2/images/cookie-collective-top-bottom.png);width:155px;height:35px;" title="powered by OneTrust"></div></a><div class="optanon-button-wrapper optanon-save-settings-button optanon-close optanon-close-consent"><div class="optanon-white-button-left"></div><div class="optanon-white-button-middle"><a href="#" onclick="Optanon.TriggerGoogleAnalyticsEvent('OneTrust Cookie Consent', 'Preferences Save Settings');">Save Settings</a></div><div class="optanon-white-button-right"></div></div><div class="optanon-button-wrapper optanon-allow-all-button optanon-allow-all" style="display: none;"><div class="optanon-white-button-left"></div><div class="optanon-white-button-middle"><a href="#" onclick="Optanon.TriggerGoogleAnalyticsEvent('OneTrust Cookie Consent', 'Preferences Allow All');">Allow All</a></div><div class="optanon-white-button-right"></div></div></div></div></div><div class="optanon-alert-box-wrapper  " style="bottom: -145px;"><div class="optanon-alert-box-bottom-top"><div class="optanon-alert-box-corner-close"><a class="optanon-alert-box-close" href="#" title="Close Banner" onclick="Optanon.TriggerGoogleAnalyticsEvent('OneTrust Cookie Consent', 'Banner Close Button');"></a></div></div><div class="optanon-alert-box-bg"><div class="optanon-alert-box-logo"> </div><div class="optanon-alert-box-body"><p>We
 use cookies to personalise content and ads, to provide social media 
features and to analyse our traffic. We also share information about 
your use of our site with our social media, advertising and analytics 
partners in accordance with our <a href="https://www.biomedcentral.com/privacy-statement">Privacy Statement</a>. You can manage your preferences in 'Manage Cookies'.</p></div><div class="optanon-clearfix"></div><div class="optanon-alert-box-button-container"><div class="optanon-alert-box-button optanon-button-close"><div class="optanon-alert-box-button-middle"><a class="optanon-alert-box-close" href="#">Close</a></div></div><div class="optanon-alert-box-button optanon-button-allow"><div class="optanon-alert-box-button-middle"><a class="optanon-allow-all" href="#" onclick="Optanon.TriggerGoogleAnalyticsEvent('OneTrust Cookie Consent', 'Banner Accept Cookies');">OK</a></div></div><div class="optanon-alert-box-button optanon-button-more"><div class="optanon-alert-box-button-middle"><a class="optanon-toggle-display" href="#" onclick="Optanon.TriggerGoogleAnalyticsEvent('OneTrust Cookie Consent', 'Banner Open Preferences');">Manage Cookies</a></div></div></div><div class="optanon-clearfix optanon-alert-box-bottom-padding"></div></div></div><img class="pub_300x250 pub_300x250m pub_728x90 text-ad textAd text_ad text_ads text-ads text-ad-links" style="width: 1px !important; height: 1px !important; position: absolute !important; left: -10000px !important; top: -1000px !important;"><div class="kxhead" data-id="KDqylSLE" style="display:none !important;"></div></body><iframe id="kx-proxy-KDqylSLE" src="https://cdn.krxd.net/partnerjs/xdi/proxy.3d2100fd7107262ecb55ce6847f01fa5.html#%21kxcid=KDqylSLE&amp;kxt=https%3A%2F%2Ffrontiersinzoology.biomedcentral.com&amp;kxcl=cdn&amp;kxp=" style="display: none; visibility: hidden; height: 0; width: 0;"></iframe></html>