<!DOCTYPE html> <html lang=en class=pb-page data-request-id=c5cfe72a-da0f-4918-8f96-2e1a7169c311 style><!--
 Page saved with SingleFileZ 
 url: https://besjournals.onlinelibrary.wiley.com/doi/10.1111/2041-210X.13430 
 saved date: Tue Oct 20 2020 18:54:07 GMT+0200 (Central European Summer Time)
--><head data-pb-dropzone=head><meta charset=utf-8><meta http-equiv=X-UA-Compatible content="IE=edge">
<meta name=pbContext content=";issue:issue:doi\:10.1111/mee3.v11.9;page:string:Article/Chapter View;ctype:string:Journal Content;requestedJournal:journal:2041210x;website:website:besjournals;wgroup:string:Publication Websites;pageGroup:string:Publication Pages;article:article:doi\:10.1111/2041-210X.13430;subPage:string:Full Text;journal:journal:2041210x">
<meta name=robots content=noarchive><meta name=pb-robots-disabled><meta property=og:title content="adiv: An r package to analyse biodiversity in ecology"><meta property=og:type content=Article><meta property=og:url content=https://besjournals.onlinelibrary.wiley.com/doi/10.1111/2041-210X.13430><meta property=og:image content=https://besjournals.onlinelibrary.wiley.com/cms/asset/20f39e73-b0d2-49ea-b5ff-2521fb872003/mee3.v11.9.cover.gif><meta property=og:site_name content=besjournals><meta property=og:description content="


R is an open‐source programming environment for statistical computing and graphics structured by numerous contributed packages. The current packages used for biodiversity research focus on limited..."><meta name=viewport content="width=device-width,initial-scale=1"><meta name=citation_doi content=10.1111/2041-210X.13430><meta name=dc.identifier content=10.1111/2041-210X.13430><meta name=citation_journal_title content="Methods in Ecology and Evolution"><meta name=citation_title content="adiv: An r package to analyse biodiversity in ecology"><meta name=citation_keywords content=biodiversity><meta name=citation_keywords content="community ecology"><meta name=citation_keywords content=conservation><meta name=citation_keywords content="diversity apportionment"><meta name=citation_keywords content="diversity measurement"><meta name=citation_keywords content="functional trait"><meta name=citation_keywords content=phylogeny><meta name=citation_keywords content=software><meta name=citation_volume content=11><meta name=citation_issue content=9><meta name=citation_firstpage content=1106><meta name=citation_lastpage content=1112><meta name=citation_publication_date content=2020/09/01><meta name=citation_online_date content=2020/06/26><meta name=citation_issn content=2041-210X><meta name=citation_language content=en><meta name=citation_author content="Sandrine Pavoine"><meta name=citation_author_institution content="Centre d'Ecologie et des Sciences de la Conservation (CESCO) Muséum, national d'Histoire naturelle (MNHN) Centre National de la Recherche Scientifique (CNRS) Sorbonne Université Paris France"><meta name=citation_abstract_html_url content=https://besjournals.onlinelibrary.wiley.com/doi/abs/10.1111/2041-210X.13430><meta name=citation_fulltext_html_url content=https://besjournals.onlinelibrary.wiley.com/doi/full/10.1111/2041-210X.13430><meta name=citation_pdf_url content=https://besjournals.onlinelibrary.wiley.com/doi/pdf/10.1111/2041-210X.13430><meta name=citation_publisher content="John Wiley &amp; Sons, Ltd"><meta name=citation_fulltext_world_readable content><meta name=article_references content="
 Pavoine, S.  adiv: An r package to analyse biodiversity in ecology. Methods Ecol Evol.  2020; 11: 1106– 1112. https://doi.org/10.1111/2041-210X.13430"><meta name=epdf_available content=false><meta name=Description content="Abstract R is an open‐source programming environment for statistical computing and graphics structured by numerous contributed packages. The current packages used for biodiversity research focus on..."><title>adiv: An r package to analyse biodiversity in ecology - Pavoine - 2020 - Methods in Ecology and Evolution - Wiley Online Library</title><meta name=referrer content=origin-when-cross-origin><meta name=google-site-verification content=ffz4i4DxIJCdGpRXe0XN6KG6o4FYI-nnFW9I_dg8piM><link rel="shortcut icon" href=images/19.ico><style>.sf-hidden{display:none!important}</style><link rel=canonical href=https://besjournals.onlinelibrary.wiley.com/doi/10.1111/2041-210X.13430><body class=pb-ui><noscript>
				<a href="/action/clickThrough?id=1011&url=https%3A%2F%2Fpubads.g.doubleclick.net%2Fgampad%2Fjump%3Fiu%3D%24googlePublisherCategory%26amp%3Bsz%3D160x600%7C160x320%7C160x160%7C120x600%26amp%3Btile%3D2%26amp%3Bc%3D%24random&loc=%2Fdoi%2F10.1111%2F2041-210X.13430&pubId=72342593&placeholderId=1014&productId=101018" target="_blank">
					<img src="https://pubads.g.doubleclick.net/gampad/ad?iu=$googlePublisherCategory&amp;sz=160x600|160x320|160x160|120x600&amp;tile=2&amp;c=$random">
				</a>
			</noscript>
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
<link rel=stylesheet type=text/css href=stylesheet_0.css>
 <link rel=stylesheet href=stylesheet_1.css>
<link rel=stylesheet type=text/css href=stylesheet_2.css>
<link rel=stylesheet type=text/css href=stylesheet_3.css>
<link rel=stylesheet type=text/css href=stylesheet_4.css>
<link rel=stylesheet type=text/css href=stylesheet_5.css>
<style>.publication-cover-info{margin-bottom:20px}.hubpage-menu nav{padding-bottom:0;position:relative}.cover-image__image img{max-width:150px;max-height:150px}h1.citation__title{font-size:1.375rem}@media (max-width:991px){h1.citation__title{font-size:1.125rem}h1.citation__title:after{content:""}}.cart-indicator__wrapper{padding:0}</style>
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
<link href=stylesheet_6.css type=text/css rel=stylesheet><style>#gs-casa-r,#gs-casa-c{background:transparent;border:0;box-sizing:content-box;font-family:Arial,sans-serif;font-weight:normal;letter-spacing:normal;line-height:normal;margin:0;opacity:1;outline:none;overflow:visible;padding:0;pointer-events:auto;text-decoration:none;text-transform:none;transition:none;vertical-align:baseline;z-index:0}#gs-casa-r{height:0;position:absolute;right:0;width:0}#gs-casa-c{bottom:100px;height:110px;overflow:hidden;pointer-events:none;position:fixed;right:0;width:116px}@keyframes gs-casa-a-l-ent{0%{transform:translate3d(96px,0,0)}100%{transform:translate3d(0,0,0)}}@media (max-width:599px),(max-height:599px){#gs-casa-c{border-radius:6px 0 0 6px;bottom:30vh;height:96px;position:fixed;transform:translate(0,48px);width:92px}@keyframes gs-casa-a-l-ent{0%{transform:translate3d(72px,0,0)}100%{transform:translate3d(0,0,0)}}}</style><style>#_atssh{width:1px!important;height:1px!important;border:0!important}</style><link id=altmetric-embed-css rel=stylesheet type=text/css href=stylesheet_7.css>
<div class=skip-links>
 <ul class=rlist>
 
 <li><a href=#article__content tabindex=1>Skip to Article Content</a></li>
 <li><a href=#pane-pcw-detailscon tabindex=1>Skip to Article Information</a></li>
 
 </ul>
</div>
 <div id=pb-page-content data-ng-non-bindable>
 <div data-pb-dropzone=main data-pb-dropzone-name=Main>
 <div data-widget-def=pageHeader data-widget-id=62175478-b98a-43d5-90f2-1f2045ac1a7e class=pageHeader>
 
 
 <header class=page-header>
 <div data-pb-dropzone=main>
 
 
 <div class=container><div class=row><div class="contents col-md-12">
 <div data-widget-def=UX3InstitutionBanner data-widget-id=4d1f2e23-571a-4b27-8ab7-e7cd8ca15964 class="institution-wrapper-mobile hidden-lg padded-content sf-hidden" aria-hidden=true>
 
 
 <div class=institution></div>
 </div>
 
 
 <div class=pull-left>
 <div data-widget-def=ux3-general-image data-widget-id=ff262197-0fb8-4973-aaa2-2fceba4b5a61 class="logo-container hidden-xs hidden-sm hidden-md">
 
 
 <a href=https://besjournals.onlinelibrary.wiley.com/ title=besjournals tabindex=2><img id=mainLogo alt=besjournals src=images/20.png></a>
 </div>
 
 <div data-widget-def=ux3-general-image data-widget-id=6b601b23-7301-47c8-8889-242d041241c4 class="logo-container-mobile hidden-lg hidden-md hidden-sm sf-hidden">
 
 
 <a href=https://besjournals.onlinelibrary.wiley.com/ title=besjournals tabindex=2></a>
 </div>
 
 <div data-widget-def=UX3InstitutionBanner data-widget-id=14d0e2f8-27d3-42d0-a477-72015e14e818 class="institution-wrapper hidden-xs hidden-sm hidden-md institution--block">
 
 
 <div class=institution><div class=institution-info-wrapper><span class=institution__intro>Access by</span><span class=institution__name>Czechs- Biological Institution Of Cas</span></div></div>
 </div>
 
</div><div class=pull-right>
 <div data-widget-def=UX3QuickSearchWidget data-widget-id=48175bb0-0f70-46e6-bb89-7e2781fdb8e4 class=header-quicksearch>
 
 
 <div class=quick-search-wrapper><a href=# title=search aria-label=Search class="quick-search__item sf-hidden"><i aria-hidden=true class=icon-search></i><span class=quick-search__item__arrow></span></a><div data-linkto-page=true class="quick-search quickSearchFormContainer"><div class=main-search-field-wrapper><div class="tab clearfix"><input type=search placeholder=Search value tabindex=13 class="fakeQSInput quickSearchFilter"><label for=search-within class=sr-only>Search within</label><select name=search-within id=search-within tabindex=10 class=search-within><option value=#pane-67065c09-4a88-49cd-934c-ac707951d35c0 selected>This Journal<option value=#pane-67065c09-4a88-49cd-934c-ac707951d35c1>Anywhere</select><ul role=tablist class="rlist tab__content"><li id=pane-67065c09-4a88-49cd-934c-ac707951d35c0 aria-labelledby=pane-67065c09-4a88-49cd-934c-ac707951d35c0-con role=tab class="tab__pane quick-search-pane__anywhere active"><form action=/action/doSearch name=thisJournalQuickSearch title="Quick Search"><div class="input-group option-0"><label for=searchField0 class=hiddenLabel>Search term</label><input type=search name=AllField id=searchField0 placeholder=Search data-auto-complete-max-words=7 data-auto-complete-max-chars=32 data-contributors-conf=3 data-topics-conf=3 data-publication-titles-conf=3 data-history-items-conf=3 value tabindex=9 class="autocomplete actualQSInput quickSearchFilter ui-autocomplete-input" autocomplete=off><span role=status aria-live=polite class=ui-helper-hidden-accessible></span><div class="search-options-wrapper quickSearchFilter sf-hidden"><a href="https://besjournals.onlinelibrary.wiley.com/search/advanced?publication=2041210x" title tabindex=12 class=advanced> Advanced Search</a><a href="https://besjournals.onlinelibrary.wiley.com/search/advanced?publication=2041210x#citation" title tabindex=12 class=citation> Citation Search</a></div></div><button id=quick-search-btn type=submit title=Search tabindex=11 aria-label="Submit Search" class="btn quick-search__button icon-search"></button></form><li id=pane-67065c09-4a88-49cd-934c-ac707951d35c1 aria-labelledby=pane-67065c09-4a88-49cd-934c-ac707951d35c1-con role=tab class="tab__pane quick-search-pane__anywhere sf-hidden"><form action=https://besjournals.onlinelibrary.wiley.com/action/doSearch name=defaultQuickSearch title="Quick Search"></form></ul></div></div></div></div>
 </div>
 
 <div data-widget-def=ux3-layout-widget data-widget-id=5ef13ed1-f4ac-4b62-a426-07ec9576a586 class=cart-indicator__wrapper>
 
 
 <div class=pull-left>
 <div data-widget-def=eCommerceCartIndicatorWidget data-widget-id=4a9f7c04-a273-4a25-91fb-8e7bd9970288 class=cart-indicator__wrapper>
 
 
 
 
 
 </div>
 
</div><div class=pull-right>
 <div data-widget-def=literatumNavigationLoginBar data-widget-id=d384ca92-7137-45d6-b3c2-78c9c0a98c2b class=login-bar-container>
 
 
 
 
 
 
 <div class="loginBar login-open">
 <a href="https://besjournals.onlinelibrary.wiley.com/action/showLogin?uri=%2Fdoi%2F10.1111%2F2041-210X.13430&amp;aria-label=Log+in+or+Register" class=show-login role=button uri=/doi/10.1111/2041-210X.13430 aria-label="Log in or Register">
 <span class="icon-profile_empty sf-hidden"></span>
 <span class=sign-in-label>Login / Register</span>
 </a>
 
 </div>
 
 </div>
 
</div>
 </div>
 
 
 <a href=#main1 title="menu drawer" data-db-switch=icon-close_thin data-db-target-for=hub-menu class="hubpage-menu__burger hidden-lg sf-hidden"><i aria-hidden=true class=icon-menu2></i><span></span></a>
</div>
</div></div></div>
 </div>
</header>
 </div>
 
 <div data-widget-def=pageBody data-widget-id=72100436-7a82-49fc-933b-c6c9d8c42914 class="pageBody hub-page-body body-text">
 
 
 <main id=main-content tabindex=-1>
 
 <div class="page-body pagefulltext">
 <div data-pb-dropzone=main>
 
 
 <div>
 
 <div class=hubpage-menu>
 <nav role=navigation data-db-target-of=hub-menu class="container hubpage-menu__nav visible-lg dropBlock__holder">
 <ul class="pull-left rlist--inline">
 <li role=menuitem tabindex=-1 aria-label=Journals class="dropdown menu-parent"><a id=main1-1 href=# title=Journals data-toggle=dropdown data-slide-target=#hubMain-sub-1 class="dropdown__toggle w-slide__btn"><span>Journals</span><i aria-hidden=true class="icon-section_arrow_d pull-right visible-lg"></i></a>
 <div id=hubMain-sub-1 aria-labelledby=main1-1 aria-hidden=true role=menu class="dropdown__menu sf-hidden">
 <div class=menu-items-container>
 
 </div>
 </div>
 </li>
 <li role=menuitem tabindex=-1 aria-label=Topics class="dropdown menu-parent"><a id=main1-2 href=# title=Topics data-toggle=dropdown data-slide-target=#hubMain-sub-2 class="dropdown__toggle w-slide__btn"><span>Topics</span><i aria-hidden=true class="icon-section_arrow_d pull-right visible-lg"></i></a>
 <div id=hubMain-sub-2 aria-labelledby=main1-1 aria-hidden=true role=menu class="dropdown__menu sf-hidden">
 <div class=menu-items-container>
 
 
 </div>
 </div>
 </li>
 </ul>
 <div class=pull-right>
 <ul class="rlist--inline pull-right">
 <li><a href=http://www.britishecologicalsociety.org/membership-community/join-us/ target=_blank>Join the BES</a></li>
 <li><a href=http://www.britishecologicalsociety.org/ target=_blank>BES.org</a></li>
 </ul>
 </div>
 </nav>
</div>
</div>
 <section data-widget-def=ux3-layout-widget data-widget-id=0f7bccba-8328-4dbd-a1fa-e84c526dd9ab>
 
 
 <div class=container><div class=row><div class=col-md-12>
 
 <div>
 
 <article data-figures="/action/ajaxShowFigures?widgetId=5cf4c79f-0ae9-4dc5-96ce-77f62de7ada9&amp;ajax=true&amp;doi=10.1111%2F2041-210X.13430&amp;pbContext=%3Bissue%3Aissue%3Adoi%5C%3A10.1111%2Fmee3.v11.9%3Bpage%3Astring%3AArticle%2FChapter+View%3Bctype%3Astring%3AJournal+Content%3BrequestedJournal%3Ajournal%3A2041210x%3Bwebsite%3Awebsite%3Abesjournals%3Bwgroup%3Astring%3APublication+Websites%3BpageGroup%3Astring%3APublication+Pages%3Barticle%3Aarticle%3Adoi%5C%3A10.1111%2F2041-210X.13430%3BsubPage%3Astring%3AFull+Text%3Bjournal%3Ajournal%3A2041210x&amp;accordionHeadingWrapper=h2" data-references="/action/ajaxShowEnhancedAbstract?widgetId=5cf4c79f-0ae9-4dc5-96ce-77f62de7ada9&amp;ajax=true&amp;doi=10.1111%2F2041-210X.13430&amp;pbContext=%3Bissue%3Aissue%3Adoi%5C%3A10.1111%2Fmee3.v11.9%3Bpage%3Astring%3AArticle%2FChapter+View%3Bctype%3Astring%3AJournal+Content%3BrequestedJournal%3Ajournal%3A2041210x%3Bwebsite%3Awebsite%3Abesjournals%3Bwgroup%3Astring%3APublication+Websites%3BpageGroup%3Astring%3APublication+Pages%3Barticle%3Aarticle%3Adoi%5C%3A10.1111%2F2041-210X.13430%3BsubPage%3Astring%3AFull+Text%3Bjournal%3Ajournal%3A2041210x&amp;accordionHeadingWrapper=h2" data-related="/action/ajaxShowRecommended?widgetId=5cf4c79f-0ae9-4dc5-96ce-77f62de7ada9&amp;ajax=true&amp;doi=10.1111%2F2041-210X.13430&amp;pbContext=%3Bissue%3Aissue%3Adoi%5C%3A10.1111%2Fmee3.v11.9%3Bpage%3Astring%3AArticle%2FChapter+View%3Bctype%3Astring%3AJournal+Content%3BrequestedJournal%3Ajournal%3A2041210x%3Bwebsite%3Awebsite%3Abesjournals%3Bwgroup%3Astring%3APublication+Websites%3BpageGroup%3Astring%3APublication+Pages%3Barticle%3Aarticle%3Adoi%5C%3A10.1111%2F2041-210X.13430%3BsubPage%3Astring%3AFull+Text%3Bjournal%3Ajournal%3A2041210x&amp;accordionHeadingWrapper=h2&amp;showSubjects=true&amp;taxonomiesCodes=enter+your+taxonomy+codes+here&amp;displayCitedByLink=true&amp;displayAlmetricDropzone=true&amp;taxonomyUri=publication-features&amp;topicUri=hide-issue-metadata" data-details="/action/ajaxShowPubInfo?widgetId=5cf4c79f-0ae9-4dc5-96ce-77f62de7ada9&amp;ajax=true&amp;doi=10.1111%2F2041-210X.13430&amp;pbContext=%3Bissue%3Aissue%3Adoi%5C%3A10.1111%2Fmee3.v11.9%3Bpage%3Astring%3AArticle%2FChapter+View%3Bctype%3Astring%3AJournal+Content%3BrequestedJournal%3Ajournal%3A2041210x%3Bwebsite%3Awebsite%3Abesjournals%3Bwgroup%3Astring%3APublication+Websites%3BpageGroup%3Astring%3APublication+Pages%3Barticle%3Aarticle%3Adoi%5C%3A10.1111%2F2041-210X.13430%3BsubPage%3Astring%3AFull+Text%3Bjournal%3Ajournal%3A2041210x&amp;accordionHeadingWrapper=h2&amp;showSubjects=true&amp;taxonomiesCodes=enter+your+taxonomy+codes+here&amp;displayCitedByLink=true&amp;displayAlmetricDropzone=true&amp;taxonomyUri=publication-features&amp;topicUri=hide-issue-metadata" data-mathjax="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.5/MathJax.js?config=TeX-AMS-MML_HTMLorMML" class=hide_CASA><div class="row article-row"><div id=article__content tabindex=-1 class="col-sm-12 col-md-8 col-lg-8 article__content article-row-left"><div data-pb-dropzone=mrwBannerDropZone class=mrw-banner-dropzone>
 
 
</div><div class=article-citation><div class=citation>
 <div class=pb-dropzone data-pb-dropzone=publicaitonContent-series-title>
 <div id=journal-banner-text class=journal-banner-text><a href=https://besjournals.onlinelibrary.wiley.com/journal/2041210x title="Methods in Ecology and Evolution homepage">Methods in Ecology and Evolution</a></div><a href=https://besjournals.onlinelibrary.wiley.com/toc/2041210x/2020/11/9 class="volume-issue sf-hidden">Volume 11, Issue 9</a><a href=https://besjournals.onlinelibrary.wiley.com/journal/2041210x title="Methods in Ecology and Evolution homepage" class=citation--logo><img id=journal-banner-image test=test-value alt="Methods in Ecology and Evolution" src=images/21.jpg class=journal-banner-image></a>
 </div>
 <div class="doi-access-container clearfix"><span class=primary-heading><span class=primary-heading>APPLICATION</span></span>
 <div class=doi-access-wrapper><div class="free-access access-type"> <i aria-hidden=true class=icon-icon-lock_open></i> <div class=doi-access> <div class=doi-access tabindex=0>Free Access</div></div></div></div></div>
 <h1 class=citation__title><span style=font-family:monospace>adiv</span>: An <span class=smallCaps>r</span> package to analyse biodiversity in ecology
 </h1>
 <div class="rlist--inline loa comma visible-xs mobile-authors loa-authors-trunc sf-hidden">
 <div class="accordion-tabbed__tab-mobile accordion__closed">
 </div>
 </div>
 <div class="loa-wrapper loa-authors hidden-xs">
 <div id=sb-1 class=accordion>
 <div class=accordion-tabbed>
 <div class="accordion-tabbed__tab-mobile accordion__closed"><a href="https://besjournals.onlinelibrary.wiley.com/action/doSearch?ContribAuthorStored=Pavoine%2C+Sandrine" class="author-name accordion-tabbed__control" data-id=a1 data-db-target-for=a1 aria-controls=a1 aria-haspopup=true id=a1_Ctrl role=button><span>Sandrine Pavoine<i aria-hidden=true class=icon-mail_outline></i></span><i aria-hidden=true class="icon-section_arrow_d sf-hidden"></i></a><div class="author-info accordion-tabbed__content sf-hidden" data-db-target-of=a1 aria-labelledby=a1_Ctrl role=region id=a1>
 <p class=author-type>Corresponding Author</p>
 <p>
 <p></p>
 <p></p>
 <p class=orcid-account></p>
 <p>Centre d'Ecologie et des Sciences de la Conservation (CESCO), Muséum, national d'Histoire naturelle (MNHN), Centre National de la Recherche Scientifique (CNRS), Sorbonne Université, Paris, France</p>
 <p></p>
 <p>Sandrine Pavoine</p>
 <p>Email: </p><a class=moreInfoLink href="https://besjournals.onlinelibrary.wiley.com/action/doSearch?ContribAuthorStored=Pavoine%2C+Sandrine">Search for more papers by this author</a></div>
 </div>
 </div>
 </div>
 </div>
 <div class=epub-sections><div class=epub-section><span class=epub-state>First published: </span><span class=epub-date>03 June 2020</span></div>
 <div class=epub-section><a class=epub-doi aria-label="Digital Object Identifier" href=https://doi.org/10.1111/2041-210X.13430>https://doi.org/10.1111/2041-210X.13430</a></div>
 </div>
 <div class=epub-sections>
 <div class="article-header__references-container no-truncate" id=mee313430-note-0001 tabindex=0>
 <div class=header-note-content aria-labelledby=header-note>
 <p>The peer review history for this article is available at <a target=_blank title="Link to external resource" href=https://publons.com/publon/10.1111/2041-210X.13430>https://publons.com/publon/10.1111/2041-210X.13430</a></p>
 </div>
 </div>
 </div>
</div></div><div class=fixedCoolBar style=padding-top:0px><nav aria-label="Article navigation and tools" class="coolBar stickybar" style=top:auto;width:auto;left:auto><div class="stickybar__wrapper coolBar__wrapper clearfix"><div class="rlist coolBar__zone"><div class=coolBar__section><a href=# data-db-target-for=article data-db-switch=icon-close_thin aria-haspopup=true aria-controls=article_Pop role=button id=article_Ctrl class="coolBar__ctrl hidden-lg sf-hidden"><i class=icon-abstract></i><span>About</span></a><div data-db-target-of=article aria-labelledby=article_Ctrl role=menu id=article_Pop class="coolBar__drop fixed rlist dropBlock__holder sf-hidden"><div data-target=".article-row-right .tab__nav, .coolBar--download" data-remove=false data-targetclass="hidden-xs hidden-sm" data-toggle=transplant data-direction=from data-transplant=self class="transplant showit"></div></div></div><div class="coolBar__section coolBar--sections"><a href=# data-db-target-for=sections data-db-switch=icon-close_thin aria-haspopup=true aria-controls=sections_Pop role=button id=sections_Ctrl class=coolBar__ctrl><i class=icon-list-unordered></i><span>Sections</span></a><div data-db-target-of=sections aria-labelledby=sections_Ctrl role=menu id=sections_Pop class="coolBar__drop dropBlock__holder rlist fixed sf-hidden"><ul class="rlist w-slide--list"></ul></div></div></div><div class="coolBar__second rlist"><div class="coolBar__section coolBar--download PdfLink scroll-to-target cloned"><a href=https://besjournals.onlinelibrary.wiley.com/doi/epdf/10.1111/2041-210X.13430 title=ePDF class="coolBar__ctrl pdf-download"><img src=images/22.png alt=PDF>PDF</a></div>
 
 <div class=relative><a href=# data-db-target-for=0e406ce9-5c43-4e2d-b741-e0b43d1f1865 data-db-switch=icon-close_thin aria-haspopup=true aria-controls=0e406ce9-5c43-4e2d-b741-e0b43d1f1865_Pop role=button id=0e406ce9-5c43-4e2d-b741-e0b43d1f1865_Ctrl class=article-tools__ctrl><i aria-hidden=true class=icon-Icon_Tools></i><span>Tools</span></a><div data-db-target-of=0e406ce9-5c43-4e2d-b741-e0b43d1f1865 aria-labelledby=0e406ce9-5c43-4e2d-b741-e0b43d1f1865_Ctrl role=menu id=0e406ce9-5c43-4e2d-b741-e0b43d1f1865_Pop class="article-tools__block fixed dropBlock__holder sf-hidden"><ul class="rlist w-slide--list"></ul></div></div>
 
 <div class=relative><a id=share-access__ctrl href=# data-db-target-for=dff42abe-4b17-4fe9-a9c5-625405cd1007 data-db-switch=icon-close_thin aria-haspopup=true aria-controls=share_Pop role=button class=share-access__ctrl><i aria-hidden=true class=icon-Icon_Share></i><span>Share</span></a><a id=share__ctrl href=# data-db-target-for=dff42abe-4b17-4fe9-a9c5-625405cd1007 data-db-switch=icon-close_thin aria-haspopup=true aria-controls=share_Pop role=button class="share__ctrl hidden sf-hidden"><i aria-hidden=true class=icon-Icon_Share></i><span>Share</span></a><div data-db-target-of=dff42abe-4b17-4fe9-a9c5-625405cd1007 aria-labelledby=share-access__ctrl role=menu id=share_Pop class="share__block dropBlock__holder fixed sf-hidden">
 
 <div data-share-access="/action/authorizeSharing?doi=10.1111/2041-210X.13430&amp;shareType=P2P" class="share-access hidden"></div><div class="ux-modal-container share-access__modal"></div>
<p>Share a link</p><span class=sr-only>Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"></ul></div></div>
</div></div></nav></div><div id=accessDenialslot class=accessDenialslot></div><div class=article__body><article>
 
 <div class=abstract-group>
 <section class="article-section article-section__abstract" lang=en data-lang=en id=section-1-en>
 <h2 id=d4615623 class="article-section__header section__title main abstractlang_en">Abstract</h2>
 <div class="article-section__content en main">
 
 <p>
 
 <ol start=1>
 
 
 <li>R is an open‐source programming environment for statistical computing and graphics structured by numerous contributed packages. The current packages used for biodiversity research focus on limited, particular aspects of biodiversity. Most packages focus on the number and abundance of species.</li>
 
 
 <li>I present an <span class=smallCaps>r</span> package named <span style=font-family:monospace>adiv</span> that provides additional methods to measure and analyse biodiversity. <span style=font-family:monospace>adiv</span> contains approaches to quantify species‐based, trait‐based (functional) and phylogenetic diversity (a) within communities (<i>α</i> diversity), (b) between communities (<i>β</i> diversity) and (c) to partition it over space and time (<i>α</i>, <i>β</i> and <i>γ</i> levels of diversity). Partitioning approaches allow evaluating whether the levels of <i>α</i> and <i>β</i> diversity could have been obtained by chance. Moreover, groups of biological entities (e.g. species of the same clade or with similar biological characteristics) that drive each level of diversity (<i>α</i>, <i>β</i> and <i>γ</i>) can be identified via ordination analyses.
 </li>
 
 
 <li>Although the package focuses on interspecific diversity in its current state, the developed approaches can also be applied to analyse intraspecific diversity or, at another level, ecosystem diversity. More generally, the functions can be applied in any discipline interested in the concept of diversity, such as economics or linguistics. Indeed, all available approaches can be easily applied at other scales and to other disciplines provided that the data have the required format: a matrix of abundance or presence/absence data of some entities in some collections and information on the differences between the entities.</li>
 
 
 <li><span style=font-family:monospace>adiv</span> aims to complement existing <span class=smallCaps>r</span> packages to provide scientists with a wide variety of diversity indices, as each index reflects a very specific facet of biodiversity. <span style=font-family:monospace>adiv</span> will grow in the future to integrate as many validated approaches for biodiversity analysis as possible, not yet available in <span class=smallCaps>r</span>. As it includes both traditional and recent viewpoints on how biodiversity should be evaluated, <span style=font-family:monospace>adiv</span> offers a promising platform where methods to analyse biodiversity can be developed and compared in terms of their statistical behaviour and biological relevance. Applications of the most relevant tools for a given study aim will eventually improve research on human‐driven variations in biodiversity.
 </li>
 
 </ol>
 <p></p>
 
 </div>
 </section>
 </div>
 <div class=pb-dropzone data-pb-dropzone=below-abstract-group>
 
 </div>
 
 <section class="article-section article-section__full">
 <section class=article-section__content id=mee313430-sec-0001>
 
 <h2 class="article-section__title section__title section1" id=mee313430-sec-0001-title>1 INTRODUCTION</h2>
 
 <p>The package named <span style=font-family:monospace>adiv</span> (for Analysis of bioDIVersity) focuses on the measurement of biological diversity (or biodiversity, the variability in life from genes to ecosystems) and on the analysis of its organization in space and/or time. I developed <span style=font-family:monospace>adiv</span> in the open‐source R environment for statistical computing and graphics (R Core Team,&nbsp;<span><a href=#mee313430-bib-0017 id=#mee313430-bib-0017R class="bibLink tab-link" data-tab=pane-pcw-references>2019</a></span>). Although this environment is rich in contributed packages, most of those that focus on biodiversity, which include approximately 40 packages (among the 15,300 current packages, R Core Team,&nbsp;<span><a href=#mee313430-bib-0017 id=#mee313430-bib-0017R class="bibLink tab-link" data-tab=pane-pcw-references>2019</a></span>), actually concentrate on <i>species diversity</i> (see Table&nbsp;<a class="tableLink scrollableLink" title="Link to table" href=#mee313430-tbl-0001>1</a> for a glossary with the definitions of the expressions in italics). A few other packages (e.g. <span style=font-family:monospace>FD</span> Laliberté, Legendre, &amp; Shipley, <span><a href=#mee313430-bib-0007 id=#mee313430-bib-0007R class="bibLink tab-link" data-tab=pane-pcw-references>2014</a></span>; <span style=font-family:monospace>TPD</span> Carmona,&nbsp;<span><a href=#mee313430-bib-0001 id=#mee313430-bib-0001R class="bibLink tab-link" data-tab=pane-pcw-references>2018</a></span>) focus on <i>functional diversity</i>, while others (e.g. <span style=font-family:monospace>PhyloMeasures</span> Tsirogiannis &amp; Sandel,&nbsp;<span><a href=#mee313430-bib-0022 id=#mee313430-bib-0022R class="bibLink tab-link" data-tab=pane-pcw-references>2017</a></span>) are dedicated to <i>phylogenetic diversity</i>. More generally, some packages, for example, <span style=font-family:monospace>entropart</span> (Marcon &amp; Herault,&nbsp;<span><a href=#mee313430-bib-0008 id=#mee313430-bib-0008R class="bibLink tab-link" data-tab=pane-pcw-references>2015</a></span>), focus on species, phylogenetic and functional diversity and consider a limited number of diversity indices. Only a few packages focus on other aspects of biodiversity; for example, <span style=font-family:monospace>diveRsity</span> (Keenan, McGinnity, Cross, Crozier, &amp; Prodöhl,&nbsp;<span><a href=#mee313430-bib-0005 id=#mee313430-bib-0005R class="bibLink tab-link" data-tab=pane-pcw-references>2013</a></span>) evaluates genetic diversity in the context of population genetic analyses.
 </p>
 
 <div class=article-table-content id=mee313430-tbl-0001>
 
 
 
 
 <header class=article-table-caption><span class=table-caption__label>TABLE 1.
 </span>Glossary
 </header>
 
 
 
 <div class=article-table-content-wrapper>
 <table class="table article-section__table">
 
 
 
 <thead>
 
 <tr>
 
 <th class="bottom-bordered-cell right-bordered-cell left-aligned">Concept</th>
 
 <th class="bottom-bordered-cell left-aligned">Definitions used in this paper</th>
 
 </tr>
 
 </thead>
 
 <tbody>
 
 <tr>
 
 <td class="right-bordered-cell left-aligned">Community</td>
 
 <td class=left-aligned>Set of species observed at a given time in a given place</td>
 
 </tr>
 
 <tr>
 
 <td class="right-bordered-cell left-aligned">Functional diversity</td>
 
 <td class=left-aligned>Trait‐based diversity applied to functional traits (see the definition of trait‐based diversity below). A functional trait is a trait that influences the way species (or organisms) respond to environmental conditions or the way they contribute to ecosystem properties</td>
 
 </tr>
 
 <tr>
 
 <td class="right-bordered-cell left-aligned">Phylogenetic diversity</td>
 
 <td class=left-aligned>Diversity in the positions of species (or of organisms, for abundance‐weighted diversity) on a phylogenetic tree. A community with many closely related species may have less phylogenetic diversity than a community with few distantly related species</td>
 
 </tr>
 
 <tr>
 
 <td class="right-bordered-cell left-aligned">Phylogenetic correspondence analysis</td>
 
 <td class=left-aligned>Extension, proposed by Pavoine (<span><a href=#mee313430-bib-0010 id=#mee313430-bib-0010R class="bibLink tab-link" data-tab=pane-pcw-references>2016</a></span>), of the well‐known ordination approach named correspondence analysis to include phylogenetic data
 </td>
 
 </tr>
 
 <tr>
 
 <td class="right-bordered-cell left-aligned">Phylogenetic signal</td>
 
 <td class=left-aligned>Positive correlation between the differences in species trait values and the distances between species on a phylogenetic tree</td>
 
 </tr>
 
 <tr>
 
 <td class="right-bordered-cell left-aligned">Species diversity</td>
 
 <td class=left-aligned>Number and abundance of species. Indices of species diversity increase with the number of species in a community and with the evenness in species abundances</td>
 
 </tr>
 
 <tr>
 
 <td class="right-bordered-cell left-aligned">Species evenness</td>
 
 <td class=left-aligned>Evenness in species abundances</td>
 
 </tr>
 
 <tr>
 
 <td class="right-bordered-cell left-aligned">Species richness</td>
 
 <td class=left-aligned>Number of species</td>
 
 </tr>
 
 <tr>
 
 <td class="right-bordered-cell left-aligned">Taxonomic diversity</td>
 
 <td class=left-aligned>Diversity in the taxonomic relationships between species. For example, a community with many species of the same genus may have less taxonomic diversity than a community with few species that belong to different orders</td>
 
 </tr>
 
 <tr>
 
 <td class="right-bordered-cell left-aligned">Trait‐based diversity</td>
 
 <td class=left-aligned>Diversity in trait values. A community with many organisms that share similar trait values may have lower trait‐based diversity than a community with few organisms characterized by distinct trait values</td>
 
 </tr>
 
 </tbody>
 
 </table>
 </div>
 
 
 <div class=article-section__table-source></div>
 </div>
 
 <p><span style=font-family:monospace>adiv</span> enriches the total library of biodiversity packages by providing both traditional statistical approaches to diversity (e.g. Hill,&nbsp;<span><a href=#mee313430-bib-0004 id=#mee313430-bib-0004R class="bibLink tab-link" data-tab=pane-pcw-references>1973</a></span>) and recent approaches that measure <i>trait‐based</i> and <i>phylogenetic diversity</i> (e.g. Chao, Chiu, &amp; Jost,&nbsp;<span><a href=#mee313430-bib-0002 id=#mee313430-bib-0002R class="bibLink tab-link" data-tab=pane-pcw-references>2010</a></span>; Faith,&nbsp;<span><a href=#mee313430-bib-0003 id=#mee313430-bib-0003R class="bibLink tab-link" data-tab=pane-pcw-references>1992</a></span>; Kondratyeva, Grandcolas, &amp; Pavoine,&nbsp;<span><a href=#mee313430-bib-0006 id=#mee313430-bib-0006R class="bibLink tab-link" data-tab=pane-pcw-references>2019</a></span>; Pavoine, Baguette, &amp; Bonsall,&nbsp;<span><a href=#mee313430-bib-0012 id=#mee313430-bib-0012R class="bibLink tab-link" data-tab=pane-pcw-references>2010</a></span>; Pavoine, Love, &amp; Bonsall,&nbsp;<span><a href=#mee313430-bib-0014 id=#mee313430-bib-0014R class="bibLink tab-link" data-tab=pane-pcw-references>2009</a></span>; Pavoine &amp; Ricotta,&nbsp;<span><a href=#mee313430-bib-0016 id=#mee313430-bib-0016R class="bibLink tab-link" data-tab=pane-pcw-references>2019</a></span>). In addition, all approaches can be easily applied to other aspects of diversity. Indeed, <span style=font-family:monospace>adiv</span> relies on the following principle: biodiversity emerges from the differences between entities, regardless of the entities selected (e.g. species, individuals, genera or assemblages) and the criterion used to evaluate the differences between the selected entities (e.g. morphology, behaviour, evolution). The main functions of <span style=font-family:monospace>adiv</span> permit the quantification of species‐based, trait‐based and phylogenetic diversity within communities (<i>α</i> diversity) and between communities (<i>β</i> diversity). They complement these measurements that partition the diversity over all communities (<i>γ</i> diversity) with ordination methods to identify groups of biological entities (e.g. species of the same clade or with similar biological characteristics) that drive levels of diversity (Figure&nbsp;<a href=#mee313430-fig-0001>1</a>). The package is available on CRAN (<a href=https://cran.r-project.org/web/packages/adiv/index.html class=linkBehavior>https://cran.r‐project.org/web/packages/adiv/index.html</a>). As an <span class=smallCaps>r</span> package, it is open‐source to encourage transparency in science, as all codes can be freely read and checked.
 </p>
 
 <section class=article-section__inline-figure>
 <figure class=figure id=mee313430-fig-0001><a target=_blank href=https://besjournals.onlinelibrary.wiley.com/cms/asset/69a8d4e5-6e58-41cc-9abd-1b59b4134867/mee313430-fig-0001-m.jpg>
 <picture>
 <img class=figure__image src=images/24.jpg data-lg-src=/cms/asset/69a8d4e5-6e58-41cc-9abd-1b59b4134867/mee313430-fig-0001-m.jpg alt=image title=image srcset sizes></picture></a><figcaption class=figure__caption>
 <div class=figure__caption__header><strong class=figure__title>FIGURE 1</strong><div class=figure-extra><a href=# class=open-figure-link>Open in figure viewer</a><a href="https://besjournals.onlinelibrary.wiley.com/action/downloadFigures?id=mee313430-fig-0001&amp;doi=10.1111%2F2041-210X.13430" class=ppt-figure-link><i aria-hidden=true class=icon-Icon_Download></i><span>PowerPoint</span></a></div>
 </div>
 <div class="figure__caption figure__caption-text">Schematic organization of <span style=font-family:monospace>adiv</span></div>
 </figcaption>
 </figure>
 </section>
 
 </section>
 <section class=article-section__content id=mee313430-sec-0002>
 
 <h2 class="article-section__title section__title section1" id=mee313430-sec-0002-title>2 DATASETS</h2>
 
 <p>The functions in <span style=font-family:monospace>adiv</span> typically work on a matrix of species presence–absence or abundance data in <i>communities</i> and on species trait data and/or phylogenetic (or taxonomic) data (Figure&nbsp;<a href=#mee313430-fig-0001>1</a>). Data that characterize communities can also be included, such as environmental, spatial or temporal data. As trait‐based and phylogenetic aspects of diversity are often compared in ecological studies, the <span style=font-family:monospace>adiv</span> package implements a few functions to test if closely related species share similar traits (<i>phylogenetic signal</i>, functions <span style=font-family:monospace>rtestdecdiv</span>, <span style=font-family:monospace>K</span>, <span style=font-family:monospace>Kstar</span> and <span style=font-family:monospace>Kw</span>).
 </p>
 
 <p>For illustrative purposes, <span style=font-family:monospace>adiv</span> currently contains eight datasets. One of these datasets, named <span style=font-family:monospace>batcomm</span>, is used below to illustrate some of functions of <span style=font-family:monospace>adiv</span>. <span style=font-family:monospace>batcomm</span> is a list of two components: <span style=font-family:monospace>batcomm$ab</span> contains the abundance of 34 bat species in four habitats (rainforests, cacao plantations, old fields and cornfields) in the Selva Lacandona of Chiapas, Mexico (data collected by Medellín, Equihua, &amp; Amin,&nbsp;<span><a href=#mee313430-bib-0009 id=#mee313430-bib-0009R class="bibLink tab-link" data-tab=pane-pcw-references>2000</a></span>); <span style=font-family:monospace>batcomm$tre</span> contains a phylogenetic tree. The tree is ultrametric, meaning that the total branch length from any tip (species) to the root remains constant.
 </p>
 
 </section>
 <section class=article-section__content id=mee313430-sec-0003>
 
 <h2 class="article-section__title section__title section1" id=mee313430-sec-0003-title>3 DIVERSITY INDICES</h2>
 
 <section class=article-section__sub-content id=mee313430-sec-0004>
 
 <h3 class="article-section__sub-title section2" id=mee313430-sec-0004-title>3.1 Species diversity</h3>
 
 <p><span style=font-family:monospace>adiv</span> contains two main functions for <i>species diversity</i> indices: <span style=font-family:monospace>speciesdiv</span>, which includes widely used indices such as <i>species richness</i> and the Shannon (<span><a href=#mee313430-bib-0019%20#mee313430-bib-0020 id="#mee313430-bib-0019 #mee313430-bib-0020R" class="bibLink tab-link" data-tab=pane-pcw-references>1948a,b</a></span>) index, and <span style=font-family:monospace>divparam</span>, which includes indices that have a parameter to control the importance given to rare versus abundant species in diversity measurements. As an illustration, using <span style=font-family:monospace>divparam</span>, I applied Hill (<span><a href=#mee313430-bib-0004 id=#mee313430-bib-0004R class="bibLink tab-link" data-tab=pane-pcw-references>1973</a></span>) numbers to the bat dataset. Note that all scripts used in this paper are available in the <span style=font-family:monospace>adiv</span> vignette (Pavoine,&nbsp;<span><a href=#mee313430-bib-0011 id=#mee313430-bib-0011R class="bibLink tab-link" data-tab=pane-pcw-references>2020</a></span>), named ‘adiv Package User Guide’, that can be accessed from the R console using the following script: <span style=font-family:monospace>browseVignettes<span style=font-family:monospace>('adiv')</span></span>. As a traditional diversity index, the Hill index increases with species richness and evenness in species abundances. Its parameter, denoted <i>q</i>, increases with the importance given to abundant species compared to rare species in diversity measurement (<i>q</i>&nbsp;=&nbsp;0 means species richness, i.e. equal importance to all species). According to the Hill index, the rainforest dominates in terms of species diversity; however, variations among the diversity levels of the other three habitats indicate how the consideration of abundance data may influence our interpretation of the impact of environmental disturbance on biodiversity (Figure&nbsp;<a href=#mee313430-fig-0002>2a</a>).
 </p>
 
 <section class=article-section__inline-figure>
 <figure class=figure id=mee313430-fig-0002><a target=_blank href=https://besjournals.onlinelibrary.wiley.com/cms/asset/e9f274dc-9183-44a8-b62b-b2d6716df0ad/mee313430-fig-0002-m.jpg>
 <picture>
 <img class=figure__image src=images/25.jpg data-lg-src=/cms/asset/e9f274dc-9183-44a8-b62b-b2d6716df0ad/mee313430-fig-0002-m.jpg alt=image title=image srcset sizes></picture></a><figcaption class=figure__caption>
 <div class=figure__caption__header><strong class=figure__title>FIGURE 2</strong><div class=figure-extra><a href=# class=open-figure-link>Open in figure viewer</a><a href="https://besjournals.onlinelibrary.wiley.com/action/downloadFigures?id=mee313430-fig-0002&amp;doi=10.1111%2F2041-210X.13430" class=ppt-figure-link><i aria-hidden=true class=icon-Icon_Download></i><span>PowerPoint</span></a></div>
 </div>
 <div class="figure__caption figure__caption-text">Bat species diversity (a), phylogenetic diversity (b and c) in four habitats (F&nbsp;=&nbsp;rainforests; P&nbsp;=&nbsp;cacao plantations; O&nbsp;=&nbsp;old fields; C&nbsp;=&nbsp;cornfields) and phylogenetic <i>β</i> diversity between the habitats (d). In panels (a), (b) and (d), the Hill index is given as a function of its parameter <i>q</i>. The higher <i>q</i> is, the less importance is given to rare species (in a) or features (in b and d) in the evaluation of diversity
 </div>
 </figcaption>
 </figure>
 </section>
 
 <p>With species diversity indices, species are implicitly considered interchangeable because only the number and/or abundance of species is important, not their identity. An assemblage of three bird species, say, a blackbird, sparrow and pigeon, would be considered as rich as an assemblage with three species of different groups, say, a blackbird, domestic cat and common wall lizard. In contrast, by considering phylogenetic analysis rather than strictly presence‐ or abundance‐based analyses, a higher value of diversity could be attributed to the second assemblage than to the first as the species in the second are more phylogenetically distinct and have more divergent biological characteristics or traits.</p>
 
 </section>
 
 <section class=article-section__sub-content id=mee313430-sec-0005>
 
 <h3 class="article-section__sub-title section2" id=mee313430-sec-0005-title>3.2 From species diversity to phylogenetic diversity</h3>
 
 <p>In the <span style=font-family:monospace>adiv</span> package, four main functions consider the phylogeny of species when measuring their diversity. Among them, <span style=font-family:monospace>evodiv</span> and <span style=font-family:monospace>evodivparam</span> consist of replacing species in traditional diversity indices (for <span style=font-family:monospace>evodiv</span>) and traditional parametric diversity indices (for <span style=font-family:monospace>evodivparam</span>) with branch units or ‘features’ on phylogenetic trees where species are the tips (Faith,&nbsp;<span><a href=#mee313430-bib-0003 id=#mee313430-bib-0003R class="bibLink tab-link" data-tab=pane-pcw-references>1992</a></span>; Pavoine &amp; Ricotta,&nbsp;<span><a href=#mee313430-bib-0016 id=#mee313430-bib-0016R class="bibLink tab-link" data-tab=pane-pcw-references>2019</a></span>). It is assumed that the number of features supported by a given branch of a phylogenetic tree is equal to the length of that branch such that the richness of features in a community is Faith's widely used phylogenetic diversity index (Faith,&nbsp;<span><a href=#mee313430-bib-0003 id=#mee313430-bib-0003R class="bibLink tab-link" data-tab=pane-pcw-references>1992</a></span>). The abundance of any feature on a given branch of the phylogenetic tree is measured as the summed abundance of all tips (species) descending from that branch. This approach of replacing species by features is simple and can be applied to all diversity indices developed so far. For example, phylogenetic data can be simply added to the analysis of the bat communities using the <span style=font-family:monospace>evodivparam</span> function. The results, which are displayed in Figure&nbsp;<a href=#mee313430-fig-0002>2b</a>, show that when abundant species are given high weight (<i>q</i>&nbsp;&gt;&nbsp;2), all habitats reach similarly low levels of phylogenetic diversity, indicating that, within each habitat, abundant species tend to be closely related.
 </p>
 
 </section>
 
 <section class=article-section__sub-content id=mee313430-sec-0006>
 
 <h3 class="article-section__sub-title section2" id=mee313430-sec-0006-title>3.3 Trait‐based diversity</h3>
 
 <p>The indices dedicated to phylogenetic diversity can be used with trait‐based data if a dendrogram is established, for example, by applying a clustering approach to a matrix of trait‐based differences between species. Three other functions in <span style=font-family:monospace>adiv</span> allow the measurement of diversity using direct trait‐based differences or similarities between species: <span style=font-family:monospace>QE</span>, <span style=font-family:monospace>qHdiv</span> and <span style=font-family:monospace>Rentropy</span>. <span style=font-family:monospace>QE</span> implements Rao's quadratic entropy (<span><a href=#mee313430-bib-0018 id=#mee313430-bib-0018R class="bibLink tab-link" data-tab=pane-pcw-references>1982</a></span>), which is the average abundance‐weighted trait‐based difference between any two species in a community. The more different the traits of any two individuals in a community are, the higher the quadratic entropy is. <span style=font-family:monospace>Rentropy</span> is equivalent to <span style=font-family:monospace>QE</span> but species' relative abundances are squared‐root transformed before calculation. Both <span style=font-family:monospace>QE</span> and <span style=font-family:monospace>Rentropy</span> generalize well‐known species diversity indices to include trait‐based data: <span style=font-family:monospace>QE</span> generalizes the Simpson (<span><a href=#mee313430-bib-0021 id=#mee313430-bib-0021R class="bibLink tab-link" data-tab=pane-pcw-references>1949</a></span>) index, which is a simple function of the Hill index if its parameter <i>q</i>&nbsp;=&nbsp;2, and <span style=font-family:monospace>Rentropy</span> is a direct generalization of the Hill index when <i>q</i>&nbsp;=&nbsp;0.5. <span style=font-family:monospace>qHdiv</span> allows, when required, intraspecific variation in biological trait values to be considered.
 </p>
 
 <p>These (dis)similarity‐based indices can inversely also be used to evaluate phylogenetic diversity if phylogenetic (dis)similarities are used instead of trait‐based (dis)similarities. For example, with the bat dataset, I calculated cophenetic distances (sum of branch lengths along the shortest path) between species on the phylogenetic tree and used them in functions <span style=font-family:monospace>QE</span> and <span style=font-family:monospace>Rentropy</span> (Figure&nbsp;<a href=#mee313430-fig-0002>2c</a>). With <span style=font-family:monospace>QE</span>, all habitats had similar levels of phylogenetic diversity, which is in accordance with the results obtained above with function <span style=font-family:monospace>evodivparam</span> if parameter <i>q</i>&nbsp;=&nbsp;2 (Figure&nbsp;<a href=#mee313430-fig-0002>2b</a>). With <span style=font-family:monospace>Rentropy</span>, the rainforest dominates in terms of phylogenetic diversity, with cacao plantations and old fields having intermediate values and cornfields having the lowest value, which is in accordance with the results of function <span style=font-family:monospace>evodivparam</span> if <i>q</i> approaches 0.5 (Figure&nbsp;<a href=#mee313430-fig-0002>2b</a>).
 </p>
 
 </section>
 
 </section>
 <section class=article-section__content id=mee313430-sec-0007>
 
 <h2 class="article-section__title section__title section1" id=mee313430-sec-0007-title>4 DISSIMILARITY INDICES</h2>
 
 <p>The concepts of diversity and (dis)similarity are linked: the diversity of an assemblage is null if all its components are identical. The biodiversity of a region increases with the increase in the dissimilarities between species and also between communities. Indices of dissimilarity thus complement those of diversity in <span style=font-family:monospace>adiv</span>.
 </p>
 
 <section class=article-section__sub-content id=mee313430-sec-0008>
 
 <h3 class="article-section__sub-title section2" id=mee313430-sec-0008-title>4.1 From species‐to‐species to community‐to‐community dissimilarity indices</h3>
 
 <p>A few functions are dedicated to the calculation of dissimilarities or similarities between species using trait, taxonomic or phylogenetic data (e.g. <span style=font-family:monospace>CFprop</span>, <span style=font-family:monospace>CFbinary</span>, <span style=font-family:monospace>dsimFun</span>, <span style=font-family:monospace>dsimTax</span> and <span style=font-family:monospace>dsimTree</span>). These functions lead to particular mathematical properties for the (dis)similarities between species. These properties named ‘positive semidefinite’ for similarity matrices and ‘Euclidean’ for dissimilarity matrices are exploited, for example, in the <span style=font-family:monospace>dsimcom</span> function of <span style=font-family:monospace>adiv</span> (see details in the <span style=font-family:monospace>adiv</span> vignette named ‘adiv Package User Guide’). <span style=font-family:monospace>dsimcom</span> implements the Pavoine and Ricotta&nbsp;(<span><a href=#mee313430-bib-0015 id=#mee313430-bib-0015R class="bibLink tab-link" data-tab=pane-pcw-references>2014</a></span>) index of similarity between two communities by comparing their species lists, the abundances of each species and the functional or phylogenetic similarities between the species. This function is restricted to matrices of similarities between species that are said positive semidefinite. Other indices of the (dis)similarity between two communities can integrate any matrix of (dis)similarities between species without any restrictions on their mathematical properties apart from having non‐negative values and sometimes being bounded between 0 and 1. This is the case, for example, for the indices available in the <span style=font-family:monospace>dissABC</span> and <span style=font-family:monospace>dissRicotta</span> functions of <span style=font-family:monospace>adiv</span>.
 </p>
 
 </section>
 
 <section class=article-section__sub-content id=mee313430-sec-0009>
 
 <h3 class="article-section__sub-title section2" id=mee313430-sec-0009-title>4.2 From compositional dissimilarity to tree‐based dissimilarity</h3>
 
 <p><span style=font-family:monospace>adiv</span> contains two functions dedicated to phylogeny‐based indices of the dissimilarity between two communities: <span style=font-family:monospace>evodiss</span> and <span style=font-family:monospace>evodiss</span>_<span style=font-family:monospace>family</span>. Although these functions are dedicated to use with phylogenetic trees, they can be more generally applied to other tree‐shaped data, such as trait‐based dendrograms. These functions use the feature perspective described in Section <a class="sectionLink scrollableLink" href=#mee313430-sec-0005>3.2</a>, where a feature is a branch unit on a phylogenetic tree. From this perspective, traditional dissimilarity indices are not applied to species presence/absence or abundance data but to the presence/absence or abundance of each feature. This perspective grants access to a family of dissimilarity indices, and the <span style=font-family:monospace>evodiss</span> and <span style=font-family:monospace>evodiss</span>_<span style=font-family:monospace>family</span> functions contain 30 key indices from this family, including six parametric indices where the importance given to rare versus abundant features can be controlled. For example, applying function <span style=font-family:monospace>evodiss</span> with the chord distance to the bat communities reveals that the two habitats with the most divergent phylogenetic compositions are the old field and cornfield habitats (phylogenetic dissimilarity&nbsp;=&nbsp;0.30), while the most similar habitats are the rainforest and cacao plantation habitats (0.12). Given the large number of species‐based dissimilarity indices developed so far, more indices from this family could be easily added in the future depending on the needs identified in the ecological and conservation literature.
 </p>
 
 </section>
 
 </section>
 <section class=article-section__content id=mee313430-sec-0010>
 
 <h2 class="article-section__title section__title section1" id=mee313430-sec-0010-title>5 APPORTIONMENT AND ORDINATION OF DIVERSITY</h2>
 
 <section class=article-section__sub-content id=mee313430-sec-0011>
 
 <h3 class="article-section__sub-title section2" id=mee313430-sec-0011-title>5.1 Alpha, beta and gamma diversity across space</h3>
 
 <p>When more than two communities are compared, the dissimilarity among them, named <i>β</i> diversity, complements the diversity within each community, named <i>α</i> diversity. The diversity of the whole set of communities (<i>γ</i> diversity) emerges from the combination of <i>α</i> and <i>β</i> diversity. <span style=font-family:monospace>adiv</span> contains functions to partition species‐based, trait‐based or phylogenetic diversity in a nested hierarchy (with <i>α</i>, <i>β</i> and <i>γ</i> levels; <span style=font-family:monospace>abgdivparam</span>, <span style=font-family:monospace>abgevodivparam</span>, <span style=font-family:monospace>eqRao</span>, <span style=font-family:monospace>eqRS</span>, <span style=font-family:monospace>eqRSintra</span> and <span style=font-family:monospace>wapqe</span> functions). For example, <span style=font-family:monospace>abgevodivparam</span> implements partitioning of Hill numbers. Applied to the bat dataset, this function shows that phylogenetic differences between the habitats mostly concern rare species, as phylogenetic <i>β</i> diversity decreases with parameter <i>q</i> (Figure&nbsp;<a href=#mee313430-fig-0002>2d</a>).
 </p>
 
 <p>Some partitioning functions are associated with simple permutation tests to evaluate whether each level of diversity could have been obtained by chance. For example, the application of one of these tests (function <span style=font-family:monospace>rtestEqRS</span>, permutation test) to the bat dataset shows that the differences in the phylogenetic compositions of the four habitats are not significant when species abundances are considered (statistic of the test, <i>β</i> diversity standardized between 0 and 1&nbsp;=&nbsp;2.42&nbsp;×&nbsp;10<sup>−2</sup>, <i>p</i>‐value&nbsp;=&nbsp;0.071), although the differences are significant when only presence/absence data are evaluated (<i>β</i>&nbsp;=&nbsp;2.37&nbsp;×&nbsp;10<sup>−2</sup>, <i>p</i>‐value&nbsp;=&nbsp;0.049; nominal <i>α</i>&nbsp;=&nbsp;0.050). This result confirms that the most abundant species in each habitat tend to be phylogenetically similar, while phylogenetic differences occur for rare species.
 </p>
 
 <p><span style=font-family:monospace>adiv</span> also implements a range of ordination analyses to visualize species and communities as points in a space that reveals which species contribute to the differences between communities according to their traits or taxonomic or phylogenetic positions. For example, the application of <i>phylogenetic correspondence analysis</i> (function <span style=font-family:monospace>evoCA</span>) to the bat dataset with presence/absence data (Figure&nbsp;<a href=#mee313430-fig-0003>3</a>) shows that cornfields were characterized by the absence of many species and many clades observed in other habitats and the presence of two closely related species: the long‐tongued bats <i>Hylonycteris underwoodi</i> and <i>Lichonycteris obscura</i> (Figure&nbsp;<a href=#mee313430-fig-0003>3a–c</a>). However, the representation of the phylogenetic tree on the map of the phylogenetic correspondence analysis highlights that the phylogenetic differences between habitats are small, although significant (see entangled phylogenetic branches on Figure&nbsp;<a href=#mee313430-fig-0003>3d</a>).
 </p>
 
 <section class=article-section__inline-figure>
 <figure class=figure id=mee313430-fig-0003><a target=_blank href=https://besjournals.onlinelibrary.wiley.com/cms/asset/fa313260-1b0e-4c1b-b708-0b6d4f617281/mee313430-fig-0003-m.jpg>
 <picture>
 <img class=figure__image src=images/26.jpg data-lg-src=/cms/asset/fa313260-1b0e-4c1b-b708-0b6d4f617281/mee313430-fig-0003-m.jpg alt=image title=image srcset sizes></picture></a><figcaption class=figure__caption>
 <div class=figure__caption__header><strong class=figure__title>FIGURE 3</strong><div class=figure-extra><a href=# class=open-figure-link>Open in figure viewer</a><a href="https://besjournals.onlinelibrary.wiley.com/action/downloadFigures?id=mee313430-fig-0003&amp;doi=10.1111%2F2041-210X.13430" class=ppt-figure-link><i aria-hidden=true class=icon-Icon_Download></i><span>PowerPoint</span></a></div>
 </div>
 <div class="figure__caption figure__caption-text">Analysis of the bat phylogenetic composition of four habitats (F&nbsp;=&nbsp;rainforests; P&nbsp;=&nbsp;cacao plantations; O&nbsp;=&nbsp;old fields; C&nbsp;=&nbsp;cornfields). (a–c) Phylogenetic tree and factorial maps obtained with phylogenetic correspondence analysis (evoCA; the first axis represents 42% of the phylogenetic dissimilarities among habitats, and the second represents 37%). (a) Coordinates for the nodes of the phylogenetic tree; the positions of the ‘top‐contributing’ nodes/species (explaining more than 5% of the variance on one of the axes) are highlighted in red and labelled. (b) Phylogenetic tree (in red, top‐contributing species; in blue, species descending from the top‐contributing nodes; in violet, the top‐contributing species that descend from a top‐contributing node). (c) Coordinates for the habitats on evoCA axes. (d) 3D map of the phylogenetic tree on evoCA axes (in <span style=font-family:monospace>adiv</span>, the plot is interactive)
 </div>
 </figcaption>
 </figure>
 </section>
 
 </section>
 
 <section class=article-section__sub-content id=mee313430-sec-0012>
 
 <h3 class="article-section__sub-title section2" id=mee313430-sec-0012-title>5.2 Temporal variations and crossed partitioning</h3>
 
 <p>Although they have thus far been mostly used in the context of spatial hierarchy, all partitioning and ordination approaches cited above could be explored to analyse the temporal variations in species, <i>taxonomic</i>, trait‐based and phylogenetic <i>diversity</i>. Additionally, crossed double principal coordinate analysis (crossed‐DPCoA, <span style=font-family:monospace>crossdpcoa</span>_<span style=font-family:monospace>maineffect</span>, <span style=font-family:monospace>crossdpcoa</span>_<span style=font-family:monospace>version1</span> and <span style=font-family:monospace>crossdpcoa</span>_<span style=font-family:monospace>version2</span> functions) allows for the partitioning of species‐based, taxonomic, trait‐based or phylogenetic diversity between two crossed factors according to the methods in Pavoine, Blondel, Dufour, Gasc, and Bonsall&nbsp;(<span><a href=#mee313430-bib-0013 id=#mee313430-bib-0013R class="bibLink tab-link" data-tab=pane-pcw-references>2013</a></span>). For example, 10 regions could be each followed for 5&nbsp;years, resulting in space (regions) and time (years) being two crossed factors affecting diversity. Crossed‐DPCoA allows the effects of space to be disentangled from the effects of time with regard to variations in diversity level. <span style=font-family:monospace>adiv</span> also allows studying spatial or temporal variations in trait‐based diversity in a phylogenetic context thanks to the <span style=font-family:monospace>decdiv</span> function. <span style=font-family:monospace>decdiv</span> indeed implements the approach by Pavoine et al. (<span><a href=#mee313430-bib-0012 id=#mee313430-bib-0012R class="bibLink tab-link" data-tab=pane-pcw-references>2010</a></span>) to partition the <i>α</i>, <i>β</i> and <i>γ</i> trait‐based diversity across the nodes of a phylogenetic tree (see also <span style=font-family:monospace>rtest.decdiv</span> function for associated permutation tests).
 </p>
 
 </section>
 
 </section>
 <section class=article-section__content id=mee313430-sec-0013>
 
 <h2 class="article-section__title section__title section1" id=mee313430-sec-0013-title>6 CONCLUSIONS AND PERSPECTIVES</h2>
 
 <div class=paragraph-element><span style=font-family:monospace>adiv</span> focuses on ecological data to analyse species, trait‐based and phylogenetic diversity. By extension, it can be used by any discipline interested in the general concept of diversity provided that the formats of the datasets are similar (at least presence/absence or abundance of any entity in any collection of entities and potentially information to characterize the differences and similarities between the entities). <span style=font-family:monospace>adiv</span> will grow in the future as a state‐of‐the‐art package for the measurement and statistical analysis of biodiversity. In particular, new developments may concern the following key points:
 <ul class=unordered-list>
 
 <li><i>Connections with other packages on diversity</i>. I have written the <span style=font-family:monospace>adiv</span> package to complement existing packages on diversity. Some functions contained in other packages (e.g. TPD, Carmona,&nbsp;<span><a href=#mee313430-bib-0001 id=#mee313430-bib-0001R class="bibLink tab-link" data-tab=pane-pcw-references>2018</a></span>) are thus not available in <span style=font-family:monospace>adiv</span>. Future developments could include the importation of functions from available packages to ease their calculation with the data format used by <span style=font-family:monospace>adiv</span>.
 </li>
 
 <li><i>Additions of new methodologies to measure diversity and partition it across spatial and temporal units</i>. Each current <span class=smallCaps>r</span> package that tackles the concept of biodiversity contains only a few of the myriad of diversity indices developed so far in the literature. The consequence is that the indices are being used in scientific studies based on their accessibility in popular packages rather than based on their true scientific relevance and interest. Indeed, having these indices in a package renders them more accessible for researchers for whom programming is a constraint and a challenge. <span style=font-family:monospace>adiv</span> aims to complement these packages to offer ecologists a wider variety of diversity indices. In the interest of open science in biodiversity research, researchers around the globe and from any discipline are invited to suggest critical methodologies for analysing biodiversity that could be implemented in <span style=font-family:monospace>adiv</span>.
 </li>
 
 <li><i>Originality, uniqueness and redundancy</i>. <span style=font-family:monospace>adiv</span> also complements current packages by offering ways to evaluate how trait and phylogenetic information impacts evaluations of biodiversity. In particular, <span style=font-family:monospace>adiv</span> contains functions (<span style=font-family:monospace>uniqueness</span> and <span style=font-family:monospace>treeUniqueness</span>) to quantify how redundant or unique a community is compared to a scenario where all species would be maximally dissimilar (e.g. having the most distinct values possible for biological traits). It also contains functions (e.g. <span style=font-family:monospace>distinctDis</span>, <span style=font-family:monospace>distinctTree</span>, <span style=font-family:monospace>distinctTopo</span> and <span style=font-family:monospace>distinctUltra</span>) to identify species that are original in a community because they have unique states of biological traits or no close relatives in the phylogeny. The presence of original species increases the diversity of the community. Further versions of <span style=font-family:monospace>adiv</span> will include more originality indices, especially those that account for species abundances and species extinction risks (see Kondratyeva et&nbsp;al.,&nbsp;<span><a href=#mee313430-bib-0006 id=#mee313430-bib-0006R class="bibLink tab-link" data-tab=pane-pcw-references>2019</a></span> and references therein).
 </li>
 
 </ul>
 </div>
 
 <p>Having all these methodologies in the same package will ease the diffusion of statistical methods among all researchers and data analysts in environmental organizations interested in the analysis of biodiversity. It will also facilitate comparison among methods in terms of their statistical behaviour and biological relevance. All these developments in <span style=font-family:monospace>adiv</span> will contribute, over the years, to open science and advances in (bio)diversity research.
 </p>
 
 </section>
 <div class=article-section__content>
 
 <h2 class="article-section__title section__title section1" id=mee313430-sec-0014-title> ACKNOWLEDGEMENTS</h2>
 
 <p>The author thanks the reviewers for their useful comments on this article. She also thanks Stéphane Dray for his help in transferring certain functions from the now deprecated class <span style=font-family:monospace>‘phylog'</span> of ade4 to the class <span style=font-family:monospace>‘phylo'</span> developed in the <span style=font-family:monospace>ape</span> package and used in the <span style=font-family:monospace>adephylo</span> package, and Giovanni Bacaro for co‐writing the <span style=font-family:monospace>dissRicotta</span> and <span style=font-family:monospace>rare</span>_<span style=font-family:monospace>Rao</span> functions and allowing their inclusion in <span style=font-family:monospace>adiv</span>. The help files of the corresponding functions mention their contributions.
 </p>
 
 <ol></ol>
 </div>
 <section id=openResearch class="article-section article-section__open-research">
 <div class="accordion article-accordion">
 <h2><a href=# class=accordion__control role=button aria-controls=open-research-content aria-expanded=false><span class=section__title id=open-research-section>Open Research</span><i aria-hidden=true class=icon-section_arrow_d></i></a></h2>
 <div class=accordion__content role=region aria-labelledby=open-research-section id=open-research-content style=display:none>
 <h1></h1>
 <section class=article-section__sub-content id=mee313430-sec-0016>
 
 
 
 
 
 </section>
 </div>
 </div>
 </section>
 <section class="article-section article-section__references" id=references-section>
 <div class="accordion article-accordion">
 <h2>
 <div tabindex=0 role=button class=accordion__control aria-controls=idmee313430-bibl-0001 aria-expanded=false data-references="/action/ajaxShowEnhancedAbstract?widgetId=5cf4c79f-0ae9-4dc5-96ce-77f62de7ada9&amp;ajax=true&amp;doi=10.1111%2F2041-210X.13430&amp;pbContext=%3Bissue%3Aissue%3Adoi%5C%3A10.1111%2Fmee3.v11.9%3Bpage%3Astring%3AArticle%2FChapter+View%3Bctype%3Astring%3AJournal+Content%3BrequestedJournal%3Ajournal%3A2041210x%3Bwebsite%3Awebsite%3Abesjournals%3Bwgroup%3Astring%3APublication+Websites%3BpageGroup%3Astring%3APublication+Pages%3Barticle%3Aarticle%3Adoi%5C%3A10.1111%2F2041-210X.13430%3BsubPage%3Astring%3AFull+Text%3Bjournal%3Ajournal%3A2041210x&amp;accordionHeadingWrapper=h2&amp;onlyLog=true"><span class=section__title id=references-section-1>REFERENCES</span><i aria-hidden=true class=icon-section_arrow_d></i></div>
 </h2>
 <div class=accordion__content style=display:none role=region aria-labelledby=references-section-1 id=mee313430-bibl-0001>
 <ul class="rlist separator">
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 </ul>
 </div>
 </div>
 </section>
 </section>
 <div style=display:none>
 <ul class="custom-style-list noteList"></ul>
 </div>
 
</article></div></div><div class="col-lg-4 col-md-4 col-sm-4 hidden-xs hidden-sm hidden-md article-row-right fixed-element" style=top:-174px><div class="publication-cover-info clearfix">
 
 <div class="cover-image-wrapper cover-image__image hasDetails"><a href=https://besjournals.onlinelibrary.wiley.com/toc/2041210x/2020/11/9 title="View Volume 11, Issue 9"><img src=images/27.jpg alt></a></div><div class="extra-info-wrapper cover-image__details"><p class=volume-issue><a href=https://besjournals.onlinelibrary.wiley.com/toc/2041210x/2020/11/9 title="View Volume 11, Issue 9"><span>Volume</span><span class=val>11</span>, <span>Issue</span><span class=val>9</span></a><p>September 2020<p class=page-range><span>Pages</span><span> 1106-1112</span></p>
 
 
</div>
</div>
 
 
 <div class="pb-ad sf-hidden">
 <div id=advert-rail-1 class="advert advert-rail">
 
 
 
 </div>
 </div>
 
<div class="sticko__parent scrollThenFix" style="height:calc(100vh - 0px)"><div class="tab tab--slide tab--flex sticko__md tabs--xs dynamic-sticko"><ul data-mobile-toggle=slide role=tablist class="rlist tab__nav w-slide--list cloned"><li role=presentation><a data-slide-clone=self href=#pane-pcw-figures aria-controls=#pane-pcw-figures role=tab data-toggle=tab id=pane-pcw-figurescon data-slide-target=#pane-pcw-figures class="article-row-right__tabLinks w-slide__btn figures-tab empty" tabindex=-1 aria-selected=false><i aria-hidden=true class=icon-multi-figure></i><h2>Figures</h2></a><li role=presentation><a data-slide-clone=self href=#pane-pcw-references aria-controls=#pane-pcw-references role=tab data-toggle=tab id=pane-pcw-referencescon data-slide-target=#pane-pcw-references class="article-row-right__tabLinks w-slide__btn references-tab empty" tabindex=-1 aria-selected=false><i aria-hidden=true class=icon-chain></i><h2>References</h2></a><li role=presentation><a data-slide-clone=self href=#pane-pcw-related aria-controls=#pane-pcw-related role=tab data-toggle=tab id=pane-pcw-relatedcon data-slide-target=#pane-pcw-related class="article-row-right__tabLinks w-slide__btn related-tab empty" tabindex=-1 aria-selected=false><i aria-hidden=true class=icon-related-circles-extend-button></i><h2>Related</h2></a><li role=presentation class=active><a data-slide-clone=self href=#pane-pcw-details aria-controls=#pane-pcw-details role=tab data-toggle=tab id=pane-pcw-detailscon data-slide-target=#pane-pcw-details class="article-row-right__tabLinks w-slide__btn details-tab" tabindex=-1 aria-selected=false><i aria-hidden=true class=icon-Icon_Information></i><h2>Information</h2></a></ul><ul role=presentation class="rlist tab__content sticko__child"><li id=pane-pcw-figures aria-labelledby=pane-pcw-figurescon role=tabpanel class="tab__pane article-row-right__panes empty sf-hidden" aria-hidden=true><li id=pane-pcw-references aria-labelledby=pane-pcw-referencescon role=tabpanel class="tab__pane article-row-right__panes empty sf-hidden" aria-hidden=true><li id=pane-pcw-related aria-labelledby=pane-pcw-relatedcon role=tabpanel class="tab__pane article-row-right__panes empty sf-hidden" aria-hidden=true><li id=pane-pcw-details aria-labelledby=pane-pcw-detailscon role=tabpanel class="tab__pane article-row-right__panes active" aria-hidden=true>
<section class=info-tab-wrapper>
<section class=metrics>
<h3 class=info-header>Metrics</h3>
<div class=section__body>
<div class=pb-dropzone data-pb-dropzone=publicaitonContent-info-tab-metrics>
<div data-badge-popover=left data-badge-type=1 data-doi=10.1111/2041-210X.13430 data-hide-no-mentions=false class=altmetric-embed data-uuid=67f45dfd-80d4-9356-7ba5-8c74b48cf181><a target=_self href="https://www.altmetric.com/details.php?domain=besjournals.onlinelibrary.wiley.com&amp;citation_id=83441331" rel=popover data-content="<div>    <div style='padding-left: 10px; line-height:18px; border-left: 16px solid #ffd140;'>
      <a class='link-to-altmetric-details-tab' target='_self' href='https://www.altmetric.com/details.php?domain=besjournals.onlinelibrary.wiley.com&amp;citation_id=83441331&amp;tab=blogs'>
          Blogged by <b>1</b>
      </a>
    </div>
    <div style='padding-left: 10px; line-height:18px; border-left: 16px solid #74CFED;'>
      <a class='link-to-altmetric-details-tab' target='_self' href='https://www.altmetric.com/details.php?domain=besjournals.onlinelibrary.wiley.com&amp;citation_id=83441331&amp;tab=twitter'>
          Tweeted by <b>123</b>
      </a>
    </div>
    <div style='padding-left: 10px; line-height:18px; border-left: 16px solid #2445bd;'>
      <a class='link-to-altmetric-details-tab' target='_self' href='https://www.altmetric.com/details.php?domain=besjournals.onlinelibrary.wiley.com&amp;citation_id=83441331&amp;tab=facebook'>
          On <b>2</b> Facebook pages
      </a>
    </div>

    <div class='altmetric-embed readers' style='margin-top: 10px;'>
          <div class='altmetric-embed tip_mendeley'
               style='padding-left: 10px; line-height:18px; border-left: 16px solid #A60000;'>
              <b>53</b> readers on Mendeley
          </div>
    </div>
    <div style='margin-top: 10px; text-align: center;'>
        <a class='altmetric_details' target='_self' href='https://www.altmetric.com/details.php?domain=besjournals.onlinelibrary.wiley.com&amp;citation_id=83441331'>
            See more details
        </a>
    </div>
</div>" style=display:inline-block data-badge-popover=left>
 <img alt="Article has an altmetric score of 86" src=images/17.png width=110 height=20 style=border:0;margin:0;max-width:none>
</a></div>
</div>
</div>
</section>
<section class=details>
<h3 class=info-header>Details</h3>
<p class=copyright>© 2020 British Ecological Society</p>
<p></p>
<ul class="cross-mark rlist">
<li><a data-target=crossmark style=cursor:pointer></a><a data-target=crossmark style=cursor:pointer><img src=images/23.svg width=150></a></li>
</ul>
</section>
<section class=keywords>
<h3 class=info-header>Keywords</h3>
<ul class="rlist rlist--inline">
<li><a class=badge-type href="https://besjournals.onlinelibrary.wiley.com/action/doSearch?field1=Keyword&amp;text1=biodiversity">biodiversity</a></li>
<li><a class=badge-type href="https://besjournals.onlinelibrary.wiley.com/action/doSearch?field1=Keyword&amp;text1=community%20ecology">community ecology</a></li>
<li><a class=badge-type href="https://besjournals.onlinelibrary.wiley.com/action/doSearch?field1=Keyword&amp;text1=conservation">conservation</a></li>
<li><a class=badge-type href="https://besjournals.onlinelibrary.wiley.com/action/doSearch?field1=Keyword&amp;text1=diversity%20apportionment">diversity apportionment</a></li>
<li><a class=badge-type href="https://besjournals.onlinelibrary.wiley.com/action/doSearch?field1=Keyword&amp;text1=diversity%20measurement">diversity measurement</a></li>
<li><a class=badge-type href="https://besjournals.onlinelibrary.wiley.com/action/doSearch?field1=Keyword&amp;text1=functional%20trait">functional trait</a></li>
<li><a class=badge-type href="https://besjournals.onlinelibrary.wiley.com/action/doSearch?field1=Keyword&amp;text1=phylogeny">phylogeny</a></li>
<li><a class=badge-type href="https://besjournals.onlinelibrary.wiley.com/action/doSearch?field1=Keyword&amp;text1=software">software</a></li>
</ul>
</section>
<section class=publication-history>
<h3 class=info-header>Publication History</h3>
<ul class=rlist>
<li><label>Issue Online: </label>02 September 2020
</li>
<li><label>Version of Record online: </label>26 June 2020
</li>
<li><label>Accepted manuscript online: </label>03 June 2020
</li>
<li><label>Manuscript accepted: </label>20 May 2020
</li>
<li><label>Manuscript received: </label>26 February 2020
</li>
</ul>
</section>
</section>
<li class=tab__spinner style=display:none><img src="data:image/gif;base64,R0lGODlhAQABAAAAACH5BAEKAAEALAAAAAABAAEAAAICTAEAOw==" id=spinner style=width:100% alt data-single-file-origin-u-r-l=/ux3/widgets/publication-content/images/spinner.gif></ul></div></div></div></div></article><div id=figure-viewer data-ux3-wrapper=figure-viewer data-ux3-transformed-by=figureInit data-ux3-role=parent role=dialog class="figure-viewer sf-hidden"><div class="figure-viewer__reg__top clearfix"></div><div class=figure-viewer__reg__center style="height:calc(100vh - 56px)"></div></div>
 
 <div style=text-align:right!important><a style=font-weight:300!important;font-size:90% href=https://besjournals.onlinelibrary.wiley.com/doi/pdf/10.1111/2041-210X.13430>Download PDF</a></div>
</div>
</div></div></div>
 </section>
 
 </div>
 </div>
<div class="w-slide padding-top"><div class=w-slide_head><a href=# class=w-slide__back><i class=icon-arrow_l aria-hidden=true></i>back</a><span class=w-slide__title></span></div><div class=w-slide__content></div></div></main>
 </div>
 
 <div data-widget-def=pageFooter data-widget-id=053994f3-7d80-4bb1-97a4-c67fbc4768cd class=pageFooter>
 
 
 <footer class=page-footer>
 <div data-pb-dropzone=main>
 
 <section data-widget-def=ux3-layout-widget data-widget-id=efb69c87-9ba0-4904-93cf-43e770aee16e class="footer-sections padded-content hub-footer hub-first-footer">
 
 
 <div class=container><div class=row><div>
 <div data-widget-def=ux3-layout-widget data-widget-id=cae47753-902f-4371-98d0-422ea9741270>
 
 
 <div class=container><div class=row><div class=col-md-3>
 <div data-widget-def=ux3-general-image data-widget-id=cdd2cb52-dd6b-403e-bac5-424c44c43d67 class=hub-footer-logo>
 
 
 <a href=https://besjournals.onlinelibrary.wiley.com/ title="besjournals Logo"><img alt="besjournals Logo" src=images/28.png></a>
 </div>
 
 
 <div class="hub-copyright hidden-xs hidden-sm hidden-md">
 © 2020 British Ecological Society
</div>
</div><div class=col-md-3>
 <div data-widget-def=UX3HTMLWidget data-widget-id=2c559261-a434-444d-8836-b1967ce4a89e class=hub-links>
 
 
 <div class=collapsible-menu>
 
 <ul class=rlist>
 <li><a href=http://www.britishecologicalsociety.org/ target=_blank>Home</a></li>
 <li><a href=http://www.britishecologicalsociety.org/policy/ target=_blank>Policy</a></li>
 <li><a href=http://www.britishecologicalsociety.org/publications/ target=_blank>Publications</a></li>
 </ul>
</div>
 </div>
 
</div><div class=col-md-3>
 <div data-widget-def=UX3HTMLWidget data-widget-id=4f2c6ddf-29e8-448d-9659-ff1d3c01effd class=hub-links>
 
 
 <div class=collapsible-menu>
 
 <ul class=rlist>
 <li><a href=http://www.britishecologicalsociety.org/membership-community/ target=_blank>Membership &amp; Community</a></li>
 <li><a href=http://www.britishecologicalsociety.org/learning-and-resources/ target=_blank>Learning &amp; Resources</a></li>
 <li><a href=http://www.britishecologicalsociety.org/funding/ target=_blank>Funding</a></li>
 </ul>
</div>
 </div>
 
</div><div class=col-md-3>
 <div data-widget-def=UX3HTMLWidget data-widget-id=f4adb924-08a3-40ee-bbe5-87be44649926 class=hub-links>
 
 
 <div class=collapsible-menu>
 
 <ul class=rlist>
 <li><a href=http://www.britishecologicalsociety.org/news-and-opinion/ target=_blank>News &amp; Opinions</a></li>
 <li><a href=http://www.britishecologicalsociety.org/events/ target=_blank>Events</a></li>
 </ul>
</div>
 </div>
 
</div></div></div>
 </div>
 
 
 <p style=text-align:right><span style="margin:0px;padding:0px;border:0px;outline:0px;font-size:13px;vertical-align:baseline;background:0px 0px transparent;color:#424242;font-family:'Open Sans',Arial,Helvetica,'Lucida Sans Unicode',sans-serif;text-align:right">British Ecological Society, 42 Wharf Road, London, N1 7GS</span><span style="color:#424242;font-family:'Open Sans',Arial,Helvetica,'Lucida Sans Unicode',sans-serif;font-size:13px;text-align:right;background-color:transparent">&nbsp;|&nbsp;</span><span style="margin:0px;padding:0px;border:0px;outline:0px;font-size:13px;vertical-align:baseline;background:0px 0px transparent;color:#424242;font-family:'Open Sans',Arial,Helvetica,'Lucida Sans Unicode',sans-serif;text-align:right">T: +44 20 3994 8282&nbsp;<br>E: hello@britishecologicalsociety.org</span><span style="color:#424242;font-family:'Open Sans',Arial,Helvetica,'Lucida Sans Unicode',sans-serif;font-size:13px;text-align:right;background-color:transparent">&nbsp;|&nbsp;</span><span style="margin:0px;padding:0px;border:0px;outline:0px;font-size:13px;vertical-align:baseline;background:0px 0px transparent;color:#424242;font-family:'Open Sans',Arial,Helvetica,'Lucida Sans Unicode',sans-serif;text-align:right">Charity Registration Number: 281213</span></p>
 
 <div class=col-md-12>
 <div class="hub-copyright visible-xs visible-sm visible-md sf-hidden">
 © 2020 British Ecological Society 
 </div>
</div>
</div></div></div>
 </section>
 
 
 <div class=main-footer>
 <div data-widget-def=ux3-layout-widget data-widget-id=2753bec1-a79d-4403-827b-6c788ddd6db3 class=footer-sections>
 
 
 <h2 class=sr-only>
 Additional links
 </h2>
 
 <div class=container><div class=row><div class=col-md-3>
 
 <div class="footer-links mb-3">
 <h3>About Wiley Online Library</h3>
 <ul class=rlist>
 <li><a href=https://www.wiley.com/privacy>Privacy Policy</a></li>
 <li><a href=https://onlinelibrary.wiley.com/terms-and-conditions>Terms of Use</a></li>
 <li><a href=https://onlinelibrary.wiley.com/cookies>Cookies</a></li>
 <li><a href=https://onlinelibrary.wiley.com/accessibility>Accessibility</a></li>
 </ul>
</div>
</div><div class=col-md-3>
 
 <div class="footer-links mb-3">
 <h3>Help &amp; Support</h3>
 <ul class=rlist>
 <li><a href=https://hub.wiley.com/community/support/onlinelibrary>Contact Us</a></li>
 </ul>
 </div>
</div><div class=col-md-3>
 
 <div class="footer-links mb-3">
 <h3>Opportunities</h3>
 <ul class=rlist>
 <li><a href=https://onlinelibrary.wiley.com/agents>Subscription Agents</a></li>
 <li><a href=https://onlinelibrary.wiley.com/advertisers>Advertisers &amp; Corporate Partners</a></li>
 </ul>
 </div>
</div><div class=col-md-3>
 
 <div class="footer-links mb-3">
 <h3>Connect with Wiley</h3>
 <ul class=rlist>
 <li><a href=https://www.wiley.com/network>The Wiley Network</a></li>
 <li><a href=https://newsroom.wiley.com/>Wiley Press Room</a></li>
 </ul>
 </div>
</div></div></div>
 </div>
 
 
 <style>.footer-links{text-align:center}.footer-links h3{font-weight:300}.footer-links a{font-weight:600}@media (min-width:992px){.footer-links{text-align:left}}</style>
 <div data-widget-def=ux3-layout-widget data-widget-id=86584e25-e4db-4118-9550-9b4f5341a4ac class=copyright-container>
 
 
 <div class=container><div class=row><div class=col-md-12>
 <div data-widget-def=UX3HTMLWidget data-widget-id=bfbf6c5d-a650-4662-828f-5275fada9120 class=copyright>
 
 
 <div class="copyright pull-left">Copyright © 1999-2020 <a href=https://www.wiley.com/>John Wiley &amp; Sons, Inc</a>. All rights reserved</div>
 </div>
 
 
 <div class="footer-logo pull-right">
 
<a href=https://www.wiley.com/ title="Wiley Home Page" aria-label=Wiley><img alt="Wiley Home Page" src=images/29.png></a>
</div>
</div></div></div>
 </div>
 
</div>
 </div>
</footer>
 </div>
 
 
 
<div class="popup login-popup hidden sf-hidden" role=dialog tabindex=-1 aria-labelledby=loginPopupHead>
 <div class=content>
 
 
 
 </div>
</div>
 
 
<div class="popup change-password-drawer hidden sf-hidden" role=dialog tabindex=-1 aria-labelledby=changePasswordPopupHead>
 <div class=content>
 
 
 
 </div>
</div>
 
 
<div class="popup registration-popup hidden sf-hidden" role=dialog tabindex=-1 aria-labelledby=createAccountPopupHead>
 <div class=content>
 
 
 
 </div>
</div>
 
 
<div class="popup top-drawer request-reset-password-drawer hidden sf-hidden" role=dialog tabindex=-1 aria-labelledby=resetPasswordPopupHead>
 <div class=content>
 
 
 </div>
</div>
 
 
<div class="popup top-drawer request-username-drawer username-popup hidden sf-hidden" role=dialog tabindex=-1 aria-labelledby=requestUsernamePopupHead>
 <a href=# class=close role=button aria-label="close dialog"></a>
 <div class=content>
 
 
 
 </div>
</div>
 
 
 </div>
 </div><div class="interesting-jade-owl-XgYroB93js4kI9jotxtT above-blush-bear-qsDNbutaWID8AOEJ-iframe-container" id=gleaming-azure-cat-gRcBok1gi0L6SkGfS8></div>
 
<div id=crossmark-widget style=display:none><div class="crossmark-reset crossmark-overlay"></div><div class="crossmark-reset crossmark-popup"></div></div><ul class="ui-autocomplete ui-front ui-menu ui-widget ui-widget-content ui-corner-all quickSearchAutocomplete" id=ui-id-1 tabindex=0 style=display:none></ul><ul class="ui-autocomplete ui-front ui-menu ui-widget ui-widget-content ui-corner-all quickSearchAutocomplete" id=ui-id-2 tabindex=0 style=display:none></ul><div id=gs-casa-r style=display:none;z-index:10;top:0px><div id=gs-casa-c></div></div><div id=_atssh style=visibility:hidden;height:1px;width:1px;position:absolute;top:-9999px;z-index:100000></div><img src=images/30.13430><div class="altmetric-embed altmetric-popover altmetric-left" data-uuid=67f45dfd-80d4-9356-7ba5-8c74b48cf181 id=_altmetric_popover_el_67f45dfd-80d4-9356-7ba5-8c74b48cf181 style=margin:0px;display:none><div class=altmetric_container></div></div>