<!DOCTYPE html>
<html lang="en"><head><script src="https://js-agent.newrelic.com/nr-spa-1071.min.js"></script><script type="text/javascript" async="" src="recaptcha__en.js"></script><script type="text/javascript" src="300lo.js"></script><script type="text/javascript" src="_ate.js"></script><meta class="foundation-mq-small"><meta class="foundation-mq-small-only"><meta class="foundation-mq-medium"><meta class="foundation-mq-medium-only"><meta class="foundation-mq-large"><meta class="foundation-mq-large-only"><meta class="foundation-mq-xlarge"><meta class="foundation-mq-xlarge-only"><meta class="foundation-mq-xxlarge"><meta class="foundation-data-attribute-namespace">
    
    <!-- Responsive View Port -->
    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1">

    <!-- Meta -->
    <meta charset="utf-8"><meta http-equiv="Content-Type" content="text/html; charset=UTF-8"><meta http-equiv="X-UA-Compatible" content="IE=Edge"><script async="" src="analytics.js"></script><script type="text/javascript">window.NREUM||(NREUM={});NREUM.info = {"beacon":"bam.nr-data.net","errorBeacon":"bam.nr-data.net","licenseKey":"8d76ca6f26","applicationID":"29565620","transactionName":"ZQBRMkVUCEtRVkINXlxKcjVnGgdKRFxVCFQcBEAWTw==","queueTime":1,"applicationTime":3779,"agent":"","atts":""}</script><script type="text/javascript">(window.NREUM||(NREUM={})).loader_config={xpid:"XAYBV15WGwEJUVdXAQMH"};window.NREUM||(NREUM={}),__nr_require=function(t,e,n){function r(n){if(!e[n]){var o=e[n]={exports:{}};t[n][0].call(o.exports,function(e){var o=t[n][1][e];return r(o||e)},o,o.exports)}return e[n].exports}if("function"==typeof __nr_require)return __nr_require;for(var o=0;o<n.length;o++)r(n[o]);return r}({1:[function(t,e,n){function r(t){try{c.console&&console.log(t)}catch(e){}}var o,i=t("ee"),a=t(20),c={};try{o=localStorage.getItem("__nr_flags").split(","),console&&"function"==typeof console.log&&(c.console=!0,o.indexOf("dev")!==-1&&(c.dev=!0),o.indexOf("nr_dev")!==-1&&(c.nrDev=!0))}catch(s){}c.nrDev&&i.on("internal-error",function(t){r(t.stack)}),c.dev&&i.on("fn-err",function(t,e,n){r(n.stack)}),c.dev&&(r("NR AGENT IN DEVELOPMENT MODE"),r("flags: "+a(c,function(t,e){return t}).join(", ")))},{}],2:[function(t,e,n){function r(t,e,n,r,c){try{h?h-=1:o(c||new UncaughtException(t,e,n),!0)}catch(f){try{i("ierr",[f,s.now(),!0])}catch(d){}}return"function"==typeof u&&u.apply(this,a(arguments))}function UncaughtException(t,e,n){this.message=t||"Uncaught error with no additional information",this.sourceURL=e,this.line=n}function o(t,e){var n=e?null:s.now();i("err",[t,n])}var i=t("handle"),a=t(21),c=t("ee"),s=t("loader"),f=t("gos"),u=window.onerror,d=!1,p="nr@seenError",h=0;s.features.err=!0,t(1),window.onerror=r;try{throw new Error}catch(l){"stack"in l&&(t(13),t(12),"addEventListener"in window&&t(6),s.xhrWrappable&&t(14),d=!0)}c.on("fn-start",function(t,e,n){d&&(h+=1)}),c.on("fn-err",function(t,e,n){d&&!n[p]&&(f(n,p,function(){return!0}),this.thrown=!0,o(n))}),c.on("fn-end",function(){d&&!this.thrown&&h>0&&(h-=1)}),c.on("internal-error",function(t){i("ierr",[t,s.now(),!0])})},{}],3:[function(t,e,n){t("loader").features.ins=!0},{}],4:[function(t,e,n){function r(){M++,S=y.hash,this[u]=b.now()}function o(){M--,y.hash!==S&&i(0,!0);var t=b.now();this[l]=~~this[l]+t-this[u],this[d]=t}function i(t,e){E.emit("newURL",[""+y,e])}function a(t,e){t.on(e,function(){this[e]=b.now()})}var c="-start",s="-end",f="-body",u="fn"+c,d="fn"+s,p="cb"+c,h="cb"+s,l="jsTime",m="fetch",v="addEventListener",w=window,y=w.location,b=t("loader");if(w[v]&&b.xhrWrappable){var g=t(10),x=t(11),E=t(8),P=t(6),O=t(13),R=t(7),T=t(14),L=t(9),j=t("ee"),N=j.get("tracer");t(15),b.features.spa=!0;var S,M=0;j.on(u,r),j.on(p,r),j.on(d,o),j.on(h,o),j.buffer([u,d,"xhr-done","xhr-resolved"]),P.buffer([u]),O.buffer(["setTimeout"+s,"clearTimeout"+c,u]),T.buffer([u,"new-xhr","send-xhr"+c]),R.buffer([m+c,m+"-done",m+f+c,m+f+s]),E.buffer(["newURL"]),g.buffer([u]),x.buffer(["propagate",p,h,"executor-err","resolve"+c]),N.buffer([u,"no-"+u]),L.buffer(["new-jsonp","cb-start","jsonp-error","jsonp-end"]),a(T,"send-xhr"+c),a(j,"xhr-resolved"),a(j,"xhr-done"),a(R,m+c),a(R,m+"-done"),a(L,"new-jsonp"),a(L,"jsonp-end"),a(L,"cb-start"),E.on("pushState-end",i),E.on("replaceState-end",i),w[v]("hashchange",i,!0),w[v]("load",i,!0),w[v]("popstate",function(){i(0,M>1)},!0)}},{}],5:[function(t,e,n){function r(t){}if(window.performance&&window.performance.timing&&window.performance.getEntriesByType){var o=t("ee"),i=t("handle"),a=t(13),c=t(12),s="learResourceTimings",f="addEventListener",u="resourcetimingbufferfull",d="bstResource",p="resource",h="-start",l="-end",m="fn"+h,v="fn"+l,w="bstTimer",y="pushState",b=t("loader");b.features.stn=!0,t(8);var g=NREUM.o.EV;o.on(m,function(t,e){var n=t[0];n instanceof g&&(this.bstStart=b.now())}),o.on(v,function(t,e){var n=t[0];n instanceof g&&i("bst",[n,e,this.bstStart,b.now()])}),a.on(m,function(t,e,n){this.bstStart=b.now(),this.bstType=n}),a.on(v,function(t,e){i(w,[e,this.bstStart,b.now(),this.bstType])}),c.on(m,function(){this.bstStart=b.now()}),c.on(v,function(t,e){i(w,[e,this.bstStart,b.now(),"requestAnimationFrame"])}),o.on(y+h,function(t){this.time=b.now(),this.startPath=location.pathname+location.hash}),o.on(y+l,function(t){i("bstHist",[location.pathname+location.hash,this.startPath,this.time])}),f in window.performance&&(window.performance["c"+s]?window.performance[f](u,function(t){i(d,[window.performance.getEntriesByType(p)]),window.performance["c"+s]()},!1):window.performance[f]("webkit"+u,function(t){i(d,[window.performance.getEntriesByType(p)]),window.performance["webkitC"+s]()},!1)),document[f]("scroll",r,{passive:!0}),document[f]("keypress",r,!1),document[f]("click",r,!1)}},{}],6:[function(t,e,n){function r(t){for(var e=t;e&&!e.hasOwnProperty(u);)e=Object.getPrototypeOf(e);e&&o(e)}function o(t){c.inPlace(t,[u,d],"-",i)}function i(t,e){return t[1]}var a=t("ee").get("events"),c=t(23)(a,!0),s=t("gos"),f=XMLHttpRequest,u="addEventListener",d="removeEventListener";e.exports=a,"getPrototypeOf"in Object?(r(document),r(window),r(f.prototype)):f.prototype.hasOwnProperty(u)&&(o(window),o(f.prototype)),a.on(u+"-start",function(t,e){var n=t[1],r=s(n,"nr@wrapped",function(){function t(){if("function"==typeof n.handleEvent)return n.handleEvent.apply(n,arguments)}var e={object:t,"function":n}[typeof n];return e?c(e,"fn-",null,e.name||"anonymous"):n});this.wrapped=t[1]=r}),a.on(d+"-start",function(t){t[1]=this.wrapped||t[1]})},{}],7:[function(t,e,n){function r(t,e,n){var r=t[e];"function"==typeof r&&(t[e]=function(){var t=r.apply(this,arguments);return o.emit(n+"start",arguments,t),t.then(function(e){return o.emit(n+"end",[null,e],t),e},function(e){throw o.emit(n+"end",[e],t),e})})}var o=t("ee").get("fetch"),i=t(20);e.exports=o;var a=window,c="fetch-",s=c+"body-",f=["arrayBuffer","blob","json","text","formData"],u=a.Request,d=a.Response,p=a.fetch,h="prototype";u&&d&&p&&(i(f,function(t,e){r(u[h],e,s),r(d[h],e,s)}),r(a,"fetch",c),o.on(c+"end",function(t,e){var n=this;e?e.clone().arrayBuffer().then(function(t){n.rxSize=t.byteLength,o.emit(c+"done",[null,e],n)}):o.emit(c+"done",[t],n)}))},{}],8:[function(t,e,n){var r=t("ee").get("history"),o=t(23)(r);e.exports=r,o.inPlace(window.history,["pushState","replaceState"],"-")},{}],9:[function(t,e,n){function r(t){function e(){s.emit("jsonp-end",[],p),t.removeEventListener("load",e,!1),t.removeEventListener("error",n,!1)}function n(){s.emit("jsonp-error",[],p),s.emit("jsonp-end",[],p),t.removeEventListener("load",e,!1),t.removeEventListener("error",n,!1)}var r=t&&"string"==typeof t.nodeName&&"script"===t.nodeName.toLowerCase();if(r){var o="function"==typeof t.addEventListener;if(o){var a=i(t.src);if(a){var u=c(a),d="function"==typeof u.parent[u.key];if(d){var p={};f.inPlace(u.parent,[u.key],"cb-",p),t.addEventListener("load",e,!1),t.addEventListener("error",n,!1),s.emit("new-jsonp",[t.src],p)}}}}}function o(){return"addEventListener"in window}function i(t){var e=t.match(u);return e?e[1]:null}function a(t,e){var n=t.match(p),r=n[1],o=n[3];return o?a(o,e[r]):e[r]}function c(t){var e=t.match(d);return e&&e.length>=3?{key:e[2],parent:a(e[1],window)}:{key:t,parent:window}}var s=t("ee").get("jsonp"),f=t(23)(s);if(e.exports=s,o()){var u=/[?&](?:callback|cb)=([^&#]+)/,d=/(.*)\.([^.]+)/,p=/^(\w+)(\.|$)(.*)$/,h=["appendChild","insertBefore","replaceChild"];f.inPlace(HTMLElement.prototype,h,"dom-"),f.inPlace(HTMLHeadElement.prototype,h,"dom-"),f.inPlace(HTMLBodyElement.prototype,h,"dom-"),s.on("dom-start",function(t){r(t[0])})}},{}],10:[function(t,e,n){var r=t("ee").get("mutation"),o=t(23)(r),i=NREUM.o.MO;e.exports=r,i&&(window.MutationObserver=function(t){return this instanceof i?new i(o(t,"fn-")):i.apply(this,arguments)},MutationObserver.prototype=i.prototype)},{}],11:[function(t,e,n){function r(t){var e=a.context(),n=c(t,"executor-",e),r=new f(n);return a.context(r).getCtx=function(){return e},a.emit("new-promise",[r,e],e),r}function o(t,e){return e}var i=t(23),a=t("ee").get("promise"),c=i(a),s=t(20),f=NREUM.o.PR;e.exports=a,f&&(window.Promise=r,["all","race"].forEach(function(t){var e=f[t];f[t]=function(n){function r(t){return function(){a.emit("propagate",[null,!o],i),o=o||!t}}var o=!1;s(n,function(e,n){Promise.resolve(n).then(r("all"===t),r(!1))});var i=e.apply(f,arguments),c=f.resolve(i);return c}}),["resolve","reject"].forEach(function(t){var e=f[t];f[t]=function(t){var n=e.apply(f,arguments);return t!==n&&a.emit("propagate",[t,!0],n),n}}),f.prototype["catch"]=function(t){return this.then(null,t)},f.prototype=Object.create(f.prototype,{constructor:{value:r}}),s(Object.getOwnPropertyNames(f),function(t,e){try{r[e]=f[e]}catch(n){}}),a.on("executor-start",function(t){t[0]=c(t[0],"resolve-",this),t[1]=c(t[1],"resolve-",this)}),a.on("executor-err",function(t,e,n){t[1](n)}),c.inPlace(f.prototype,["then"],"then-",o),a.on("then-start",function(t,e){this.promise=e,t[0]=c(t[0],"cb-",this),t[1]=c(t[1],"cb-",this)}),a.on("then-end",function(t,e,n){this.nextPromise=n;var r=this.promise;a.emit("propagate",[r,!0],n)}),a.on("cb-end",function(t,e,n){a.emit("propagate",[n,!0],this.nextPromise)}),a.on("propagate",function(t,e,n){this.getCtx&&!e||(this.getCtx=function(){if(t instanceof Promise)var e=a.context(t);return e&&e.getCtx?e.getCtx():this})}),r.toString=function(){return""+f})},{}],12:[function(t,e,n){var r=t("ee").get("raf"),o=t(23)(r),i="equestAnimationFrame";e.exports=r,o.inPlace(window,["r"+i,"mozR"+i,"webkitR"+i,"msR"+i],"raf-"),r.on("raf-start",function(t){t[0]=o(t[0],"fn-")})},{}],13:[function(t,e,n){function r(t,e,n){t[0]=a(t[0],"fn-",null,n)}function o(t,e,n){this.method=n,this.timerDuration=isNaN(t[1])?0:+t[1],t[0]=a(t[0],"fn-",this,n)}var i=t("ee").get("timer"),a=t(23)(i),c="setTimeout",s="setInterval",f="clearTimeout",u="-start",d="-";e.exports=i,a.inPlace(window,[c,"setImmediate"],c+d),a.inPlace(window,[s],s+d),a.inPlace(window,[f,"clearImmediate"],f+d),i.on(s+u,r),i.on(c+u,o)},{}],14:[function(t,e,n){function r(t,e){d.inPlace(e,["onreadystatechange"],"fn-",c)}function o(){var t=this,e=u.context(t);t.readyState>3&&!e.resolved&&(e.resolved=!0,u.emit("xhr-resolved",[],t)),d.inPlace(t,y,"fn-",c)}function i(t){b.push(t),l&&(x?x.then(a):v?v(a):(E=-E,P.data=E))}function a(){for(var t=0;t<b.length;t++)r([],b[t]);b.length&&(b=[])}function c(t,e){return e}function s(t,e){for(var n in t)e[n]=t[n];return e}t(6);var f=t("ee"),u=f.get("xhr"),d=t(23)(u),p=NREUM.o,h=p.XHR,l=p.MO,m=p.PR,v=p.SI,w="readystatechange",y=["onload","onerror","onabort","onloadstart","onloadend","onprogress","ontimeout"],b=[];e.exports=u;var g=window.XMLHttpRequest=function(t){var e=new h(t);try{u.emit("new-xhr",[e],e),e.addEventListener(w,o,!1)}catch(n){try{u.emit("internal-error",[n])}catch(r){}}return e};if(s(h,g),g.prototype=h.prototype,d.inPlace(g.prototype,["open","send"],"-xhr-",c),u.on("send-xhr-start",function(t,e){r(t,e),i(e)}),u.on("open-xhr-start",r),l){var x=m&&m.resolve();if(!v&&!m){var E=1,P=document.createTextNode(E);new l(a).observe(P,{characterData:!0})}}else f.on("fn-end",function(t){t[0]&&t[0].type===w||a()})},{}],15:[function(t,e,n){function r(t){var e=this.params,n=this.metrics;if(!this.ended){this.ended=!0;for(var r=0;r<d;r++)t.removeEventListener(u[r],this.listener,!1);if(!e.aborted){if(n.duration=a.now()-this.startTime,4===t.readyState){e.status=t.status;var i=o(t,this.lastSize);if(i&&(n.rxSize=i),this.sameOrigin){var s=t.getResponseHeader("X-NewRelic-App-Data");s&&(e.cat=s.split(", ").pop())}}else e.status=0;n.cbTime=this.cbTime,f.emit("xhr-done",[t],t),c("xhr",[e,n,this.startTime])}}}function o(t,e){var n=t.responseType;if("json"===n&&null!==e)return e;var r="arraybuffer"===n||"blob"===n||"json"===n?t.response:t.responseText;return l(r)}function i(t,e){var n=s(e),r=t.params;r.host=n.hostname+":"+n.port,r.pathname=n.pathname,t.sameOrigin=n.sameOrigin}var a=t("loader");if(a.xhrWrappable){var c=t("handle"),s=t(16),f=t("ee"),u=["load","error","abort","timeout"],d=u.length,p=t("id"),h=t(19),l=t(18),m=window.XMLHttpRequest;a.features.xhr=!0,t(14),f.on("new-xhr",function(t){var e=this;e.totalCbs=0,e.called=0,e.cbTime=0,e.end=r,e.ended=!1,e.xhrGuids={},e.lastSize=null,h&&(h>34||h<10)||window.opera||t.addEventListener("progress",function(t){e.lastSize=t.loaded},!1)}),f.on("open-xhr-start",function(t){this.params={method:t[0]},i(this,t[1]),this.metrics={}}),f.on("open-xhr-end",function(t,e){"loader_config"in NREUM&&"xpid"in NREUM.loader_config&&this.sameOrigin&&e.setRequestHeader("X-NewRelic-ID",NREUM.loader_config.xpid)}),f.on("send-xhr-start",function(t,e){var n=this.metrics,r=t[0],o=this;if(n&&r){var i=l(r);i&&(n.txSize=i)}this.startTime=a.now(),this.listener=function(t){try{"abort"===t.type&&(o.params.aborted=!0),("load"!==t.type||o.called===o.totalCbs&&(o.onloadCalled||"function"!=typeof e.onload))&&o.end(e)}catch(n){try{f.emit("internal-error",[n])}catch(r){}}};for(var c=0;c<d;c++)e.addEventListener(u[c],this.listener,!1)}),f.on("xhr-cb-time",function(t,e,n){this.cbTime+=t,e?this.onloadCalled=!0:this.called+=1,this.called!==this.totalCbs||!this.onloadCalled&&"function"==typeof n.onload||this.end(n)}),f.on("xhr-load-added",function(t,e){var n=""+p(t)+!!e;this.xhrGuids&&!this.xhrGuids[n]&&(this.xhrGuids[n]=!0,this.totalCbs+=1)}),f.on("xhr-load-removed",function(t,e){var n=""+p(t)+!!e;this.xhrGuids&&this.xhrGuids[n]&&(delete this.xhrGuids[n],this.totalCbs-=1)}),f.on("addEventListener-end",function(t,e){e instanceof m&&"load"===t[0]&&f.emit("xhr-load-added",[t[1],t[2]],e)}),f.on("removeEventListener-end",function(t,e){e instanceof m&&"load"===t[0]&&f.emit("xhr-load-removed",[t[1],t[2]],e)}),f.on("fn-start",function(t,e,n){e instanceof m&&("onload"===n&&(this.onload=!0),("load"===(t[0]&&t[0].type)||this.onload)&&(this.xhrCbStart=a.now()))}),f.on("fn-end",function(t,e){this.xhrCbStart&&f.emit("xhr-cb-time",[a.now()-this.xhrCbStart,this.onload,e],e)})}},{}],16:[function(t,e,n){e.exports=function(t){var e=document.createElement("a"),n=window.location,r={};e.href=t,r.port=e.port;var o=e.href.split("://");!r.port&&o[1]&&(r.port=o[1].split("/")[0].split("@").pop().split(":")[1]),r.port&&"0"!==r.port||(r.port="https"===o[0]?"443":"80"),r.hostname=e.hostname||n.hostname,r.pathname=e.pathname,r.protocol=o[0],"/"!==r.pathname.charAt(0)&&(r.pathname="/"+r.pathname);var i=!e.protocol||":"===e.protocol||e.protocol===n.protocol,a=e.hostname===document.domain&&e.port===n.port;return r.sameOrigin=i&&(!e.hostname||a),r}},{}],17:[function(t,e,n){function r(){}function o(t,e,n){return function(){return i(t,[f.now()].concat(c(arguments)),e?null:this,n),e?void 0:this}}var i=t("handle"),a=t(20),c=t(21),s=t("ee").get("tracer"),f=t("loader"),u=NREUM;"undefined"==typeof window.newrelic&&(newrelic=u);var d=["setPageViewName","setCustomAttribute","setErrorHandler","finished","addToTrace","inlineHit","addRelease"],p="api-",h=p+"ixn-";a(d,function(t,e){u[e]=o(p+e,!0,"api")}),u.addPageAction=o(p+"addPageAction",!0),u.setCurrentRouteName=o(p+"routeName",!0),e.exports=newrelic,u.interaction=function(){return(new r).get()};var l=r.prototype={createTracer:function(t,e){var n={},r=this,o="function"==typeof e;return i(h+"tracer",[f.now(),t,n],r),function(){if(s.emit((o?"":"no-")+"fn-start",[f.now(),r,o],n),o)try{return e.apply(this,arguments)}catch(t){throw s.emit("fn-err",[arguments,this,t],n),t}finally{s.emit("fn-end",[f.now()],n)}}}};a("setName,setAttribute,save,ignore,onEnd,getContext,end,get".split(","),function(t,e){l[e]=o(h+e)}),newrelic.noticeError=function(t){"string"==typeof t&&(t=new Error(t)),i("err",[t,f.now()])}},{}],18:[function(t,e,n){e.exports=function(t){if("string"==typeof t&&t.length)return t.length;if("object"==typeof t){if("undefined"!=typeof ArrayBuffer&&t instanceof ArrayBuffer&&t.byteLength)return t.byteLength;if("undefined"!=typeof Blob&&t instanceof Blob&&t.size)return t.size;if(!("undefined"!=typeof FormData&&t instanceof FormData))try{return JSON.stringify(t).length}catch(e){return}}}},{}],19:[function(t,e,n){var r=0,o=navigator.userAgent.match(/Firefox[\/\s](\d+\.\d+)/);o&&(r=+o[1]),e.exports=r},{}],20:[function(t,e,n){function r(t,e){var n=[],r="",i=0;for(r in t)o.call(t,r)&&(n[i]=e(r,t[r]),i+=1);return n}var o=Object.prototype.hasOwnProperty;e.exports=r},{}],21:[function(t,e,n){function r(t,e,n){e||(e=0),"undefined"==typeof n&&(n=t?t.length:0);for(var r=-1,o=n-e||0,i=Array(o<0?0:o);++r<o;)i[r]=t[e+r];return i}e.exports=r},{}],22:[function(t,e,n){e.exports={exists:"undefined"!=typeof window.performance&&window.performance.timing&&"undefined"!=typeof window.performance.timing.navigationStart}},{}],23:[function(t,e,n){function r(t){return!(t&&t instanceof Function&&t.apply&&!t[a])}var o=t("ee"),i=t(21),a="nr@original",c=Object.prototype.hasOwnProperty,s=!1;e.exports=function(t,e){function n(t,e,n,o){function nrWrapper(){var r,a,c,s;try{a=this,r=i(arguments),c="function"==typeof n?n(r,a):n||{}}catch(f){p([f,"",[r,a,o],c])}u(e+"start",[r,a,o],c);try{return s=t.apply(a,r)}catch(d){throw u(e+"err",[r,a,d],c),d}finally{u(e+"end",[r,a,s],c)}}return r(t)?t:(e||(e=""),nrWrapper[a]=t,d(t,nrWrapper),nrWrapper)}function f(t,e,o,i){o||(o="");var a,c,s,f="-"===o.charAt(0);for(s=0;s<e.length;s++)c=e[s],a=t[c],r(a)||(t[c]=n(a,f?c+o:o,i,c))}function u(n,r,o){if(!s||e){var i=s;s=!0;try{t.emit(n,r,o,e)}catch(a){p([a,n,r,o])}s=i}}function d(t,e){if(Object.defineProperty&&Object.keys)try{var n=Object.keys(t);return n.forEach(function(n){Object.defineProperty(e,n,{get:function(){return t[n]},set:function(e){return t[n]=e,e}})}),e}catch(r){p([r])}for(var o in t)c.call(t,o)&&(e[o]=t[o]);return e}function p(e){try{t.emit("internal-error",e)}catch(n){}}return t||(t=o),n.inPlace=f,n.flag=a,n}},{}],ee:[function(t,e,n){function r(){}function o(t){function e(t){return t&&t instanceof r?t:t?s(t,c,i):i()}function n(n,r,o,i){if(!p.aborted||i){t&&t(n,r,o);for(var a=e(o),c=l(n),s=c.length,f=0;f<s;f++)c[f].apply(a,r);var d=u[y[n]];return d&&d.push([b,n,r,a]),a}}function h(t,e){w[t]=l(t).concat(e)}function l(t){return w[t]||[]}function m(t){return d[t]=d[t]||o(n)}function v(t,e){f(t,function(t,n){e=e||"feature",y[n]=e,e in u||(u[e]=[])})}var w={},y={},b={on:h,emit:n,get:m,listeners:l,context:e,buffer:v,abort:a,aborted:!1};return b}function i(){return new r}function a(){(u.api||u.feature)&&(p.aborted=!0,u=p.backlog={})}var c="nr@context",s=t("gos"),f=t(20),u={},d={},p=e.exports=o();p.backlog=u},{}],gos:[function(t,e,n){function r(t,e,n){if(o.call(t,e))return t[e];var r=n();if(Object.defineProperty&&Object.keys)try{return Object.defineProperty(t,e,{value:r,writable:!0,enumerable:!1}),r}catch(i){}return t[e]=r,r}var o=Object.prototype.hasOwnProperty;e.exports=r},{}],handle:[function(t,e,n){function r(t,e,n,r){o.buffer([t],r),o.emit(t,e,n)}var o=t("ee").get("handle");e.exports=r,r.ee=o},{}],id:[function(t,e,n){function r(t){var e=typeof t;return!t||"object"!==e&&"function"!==e?-1:t===window?0:a(t,i,function(){return o++})}var o=1,i="nr@id",a=t("gos");e.exports=r},{}],loader:[function(t,e,n){function r(){if(!x++){var t=g.info=NREUM.info,e=p.getElementsByTagName("script")[0];if(setTimeout(u.abort,3e4),!(t&&t.licenseKey&&t.applicationID&&e))return u.abort();f(y,function(e,n){t[e]||(t[e]=n)}),s("mark",["onload",a()+g.offset],null,"api");var n=p.createElement("script");n.src="https://"+t.agent,e.parentNode.insertBefore(n,e)}}function o(){"complete"===p.readyState&&i()}function i(){s("mark",["domContent",a()+g.offset],null,"api")}function a(){return E.exists&&performance.now?Math.round(performance.now()):(c=Math.max((new Date).getTime(),c))-g.offset}var c=(new Date).getTime(),s=t("handle"),f=t(20),u=t("ee"),d=window,p=d.document,h="addEventListener",l="attachEvent",m=d.XMLHttpRequest,v=m&&m.prototype;NREUM.o={ST:setTimeout,SI:d.setImmediate,CT:clearTimeout,XHR:m,REQ:d.Request,EV:d.Event,PR:d.Promise,MO:d.MutationObserver};var w=""+location,y={beacon:"bam.nr-data.net",errorBeacon:"bam.nr-data.net",agent:"js-agent.newrelic.com/nr-spa-1071.min.js"},b=m&&v&&v[h]&&!/CriOS/.test(navigator.userAgent),g=e.exports={offset:c,now:a,origin:w,features:{},xhrWrappable:b};t(17),p[h]?(p[h]("DOMContentLoaded",i,!1),d[h]("load",r,!1)):(p[l]("onreadystatechange",o),d[l]("onload",r)),s("mark",["firstbyte",c],null,"api");var x=0,E=t(22)},{}]},{},["loader",2,15,5,3,4]);</script>
    <!-- Bookmark Icons -->
    <link rel="apple-touch-icon" sizes="180x180" href="https://academic.oup.com/UI/app/img/apple-touch-icon.png"><link rel="icon" type="image/png" href="https://academic.oup.com/UI/app/img/favicon-32x32.png" sizes="32x32"><link rel="icon" type="image/png" href="https://academic.oup.com/UI/app/img/favicon-16x16.png" sizes="16x16"><link rel="manifest" href="https://academic.oup.com/UI/app/img/manifest.json"><link rel="mask-icon" href="https://academic.oup.com/UI/app/img/safari-pinned-tab.svg" color="#5bbad5"><link rel="shortcut icon" href="https://academic.oup.com/UI/app/img/favicon.ico"><meta name="msapplication-config" content="/UI/app/img/browserconfig.xml"><meta name="theme-color" content="#002f65">

    <!-- Stylesheets -->
    
<link href="css.css" type="text/css" rel="stylesheet">
<link href="globalCSS.css" type="text/css" rel="stylesheet">
<link href="cookiePolicyCSS.css" type="text/css" rel="stylesheet">
<link href="chartist.css" type="text/css" rel="stylesheet">

        <link href="global.css" rel="stylesheet" type="text/css">
    
        <link href="Site-1071638357.css" rel="stylesheet" type="text/css">
    

    <!-- Scripts -->
    
<script src="jquery-1.js" type="text/javascript"></script>

<!--     

 -->

    <script type="text/javascript">
        var App = App || {};
        App.LoginUserInfo = { 
            isInstLoggedIn: [1] 
        };

        App.CurrentSubdomain = 'sysbio';
    </script>

    <!-- Extra Head Content -->
    
    
        <script type="application/ld+json">
        {"@context":"http://schema.org","@id":"https://academic.oup.com/sysbio/article/65/2/228/2427164","@type":"ScholarlyArticle","name":"Total-Evidence Dating under the Fossilized Birth–Death Process","datePublished":"2015-10-22","isPartOf":{"@id":"https://academic.oup.com/sysbio/issue/65/2","@type":"PublicationIssue","issueNumber":"2","datePublished":"2016-03-01","isPartOf":{"@id":"https://academic.oup.com/sysbio","@type":"Periodical","name":"Systematic Biology","issn":["1076-836X"]}},"url":"http://dx.doi.org/10.1093/sysbio/syv080","keywords":["Bayesian phylogenetic inference","birth–death process","MCMC","relaxed clock","total-evidence dating","tree prior"],"inLanguage":"en","copyrightHolder":"Society of Systematic Biologists","copyrightYear":"2018","publisher":"Oxford University Press","sameAs":"https://academic.oup.com/sysbio/article/65/2/228/2427164","author":[{"name":"Zhang, Chi","affiliation":"Department of Bioinformatics and Genetics, Swedish Museum of Natural History, SE-104 05 Stockholm, Sweden;","@type":"Person"},{"name":"Stadler, Tanja","affiliation":"Department of Biosystems Science and Engineering, Eidgenössische Technische Hochschule Zürich, 4053 Basel, Switzerland;    Swiss Institute of Bioinformatics (SIB), Switzerland;","@type":"Person"},{"name":"Klopfstein, Seraina","affiliation":"Department of Bioinformatics and Genetics, Swedish Museum of Natural History, SE-104 05 Stockholm, Sweden;    Department of Invertebrates, Natural History Museum Bern, CH-3005 Bern, Switzerland;","@type":"Person"},{"name":"Heath, Tracy A.","affiliation":"Department of Integrative Biology, University of California, Berkeley, CA 94720 USA;    Department of Ecology and Evolutionary Biology, University of Kansas, Lawrence, KS 66045, USA;    Department of Ecology, Evolution &amp; Organismal Biology, Iowa State University, Ames, IA 50011, USA","@type":"Person"},{"name":"Ronquist, Fredrik","affiliation":"Department of Bioinformatics and Genetics, Swedish Museum of Natural History, SE-104 05 Stockholm, Sweden;","@type":"Person"}],"description":"Abstract.  Bayesian total-evidence dating involves the simultaneous analysis of morphological data from the fossil record and morphological and sequence data fr","pageStart":"228","pageEnd":"249","thumbnailURL":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/sysbio/Issue/65/2/0/m_cover.gif?Expires=2147483647&Signature=Q4RDX4xonogUJ~xW~tqQIm4BGPMlkZChyVuj8JgbehUMsszA1hnaBA3HlZ-aNJsEii5YsWSt9meYdQxMHZWB6uq78Ef9YcfNOc-Q7B-1lmQo2IgDf53GKgsR5Kwh~9u7sT-MZ71K1~-F6y3B~43Ef84~JAWQeMh8RVw8r2ZurILLoiS~9Cwi9lEw4EnTYY6~MEJfBq79nJpJEJ0tesGs4t2M8Bm7PwexDZBE998l5W3IGitbFEbrwIppO5WB3zPaFSe~HuoI5f0NbiAohjwbD8n9sRa4UWJtiZgXUVqX-3iRdnmMggPRvzs9Sv0QO~wIfXmwPSjyB2D917NKH7pcgA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA","headline":"Total-Evidence Dating under the Fossilized Birth–Death Process","image":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/sysbio/Issue/65/2/0/m_cover.gif?Expires=2147483647&Signature=Q4RDX4xonogUJ~xW~tqQIm4BGPMlkZChyVuj8JgbehUMsszA1hnaBA3HlZ-aNJsEii5YsWSt9meYdQxMHZWB6uq78Ef9YcfNOc-Q7B-1lmQo2IgDf53GKgsR5Kwh~9u7sT-MZ71K1~-F6y3B~43Ef84~JAWQeMh8RVw8r2ZurILLoiS~9Cwi9lEw4EnTYY6~MEJfBq79nJpJEJ0tesGs4t2M8Bm7PwexDZBE998l5W3IGitbFEbrwIppO5WB3zPaFSe~HuoI5f0NbiAohjwbD8n9sRa4UWJtiZgXUVqX-3iRdnmMggPRvzs9Sv0QO~wIfXmwPSjyB2D917NKH7pcgA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"}
    </script>
    <meta property="og:type" content="article">
    <meta property="og:title" content="Total-Evidence Dating under the Fossilized Birth–Death Process | Systematic Biology | Oxford Academic">
    <meta property="og:description" content="Abstract.  Bayesian total-evidence dating involves the simultaneous analysis of morphological data from the fossil record and morphological and sequence data fr">
    <meta property="og:url" content="https://academic.oup.com/sysbio/article/65/2/228/2427164">
    <meta property="og:image" content="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/sysbio/Issue/67/5/2/m_sysbio_67_5cover.png?Expires=2147483647&amp;Signature=iMgOj~ozkXgiMv5ypBZDzosyeLzAPALO2wmmKLje0Ck54FjJm347WeTb3PKZWVrxYH9T2vN47duSgGc4dLVp5tBbPeDOQV99~fC0QoiuNqiJsKVtDjUdYShshHjSJhQalWrhtfVODxOV3i~okx0xHvcYKoh0XaVEgY8howRgKsjKDokUzxP8A-IlCyuRe7~RCRWcLvUokOnZjMW6dIfPEHfg4NKlHFUTrgzwYmZcyCm1Idbn11KibNLt9Vs51khpe8c8~eRqAFQvD8TZZR2syjseNNvaC62SMaVI7aravRbjdk14RleGCLdGvpguCHY1RYMY5C~wSqHtyxn-d0GGQA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">
    <meta property="og:updated_time" content="2015-10-22">
    <meta property="og:site_name" content="OUP Academic">
<meta name="citation_author" content="Zhang, Chi"><meta name="citation_author_institution" content=" Department of Bioinformatics and Genetics, Swedish Museum of Natural History, SE-104 05 Stockholm, Sweden; "><meta name="citation_author" content="Stadler, Tanja"><meta name="citation_author_institution" content=" Department of Biosystems Science and Engineering, Eidgenössische Technische Hochschule Zürich, 4053 Basel, Switzerland; "><meta name="citation_author_institution" content=" Swiss Institute of Bioinformatics (SIB), Switzerland; "><meta name="citation_author" content="Klopfstein, Seraina"><meta name="citation_author_institution" content=" Department of Bioinformatics and Genetics, Swedish Museum of Natural History, SE-104 05 Stockholm, Sweden; "><meta name="citation_author_institution" content=" Department of Invertebrates, Natural History Museum Bern, CH-3005 Bern, Switzerland; "><meta name="citation_author" content="Heath, Tracy A."><meta name="citation_author_institution" content=" Department of Integrative Biology, University of California, Berkeley, CA 94720 USA; "><meta name="citation_author_institution" content=" Department of Ecology and Evolutionary Biology, University of Kansas, Lawrence, KS 66045, USA; "><meta name="citation_author_institution" content=" Department of Ecology, Evolution &amp; Organismal Biology, Iowa State University, Ames, IA 50011, USA "><meta name="citation_author" content="Ronquist, Fredrik"><meta name="citation_author_institution" content=" Department of Bioinformatics and Genetics, Swedish Museum of Natural History, SE-104 05 Stockholm, Sweden; "><meta name="citation_title" content="Total-Evidence Dating under the Fossilized Birth–Death Process"><meta name="citation_firstpage" content="228"><meta name="citation_lastpage" content="249"><meta name="citation_doi" content="10.1093/sysbio/syv080"><meta name="citation_journal_title" content="Systematic Biology"><meta name="citation_journal_abbrev" content="Syst Biol"><meta name="citation_volume" content="65"><meta name="citation_issue" content="2"><meta name="citation_publication_date" content="2016/03/01"><meta name="citation_issn" content="1063-5157"><meta name="citation_publisher" content="Oxford University Press"><meta name="citation_reference" content=" Aguiar A., Deans A., Engel M., Forshage M., Huber J., Jennings J., Johnson N., Lelej A., Longino J., Miko I., Ohl M., Rasmussen C., Taeger A., Yu D. 2013. Order Hymenoptera. In: Zhang, Z.-Q. (Ed.) Animal biodiversity: an outline of higher-level classification and survey of taxonomic richness (Addenda 2013). Zootaxa 3703:51–62.  "><meta name="citation_reference" content="citation_title=An evaluation of fossil tip-dating versus node-age calibrations in tetraodontiform fishes (Teleostei: Percomorphaceae); citation_author=Arcila  D.; citation_author=Alexander Pyron  R.; citation_author=Tyler  J.C.; citation_author=Ortí  G.; citation_author=Betancur-R  R.;  citation_journal_title=Mol. Phylogenet. Evol.;  citation_year=2015;  citation_volume=82;  citation_pages=Pt A:131-145; "><meta name="citation_reference" content="citation_title=Effects of models of rate evolution on estimation of divergence dates with special reference to the metazoan 18S ribosomal RNA phylogeny; citation_author=Aris-Brosou  S.; citation_author=Yang  Z.;  citation_journal_title=Syst. Biol.;  citation_year=2002;  citation_volume=51;  citation_pages=703-714; "><meta name="citation_reference" content="citation_title=Bayesian tests of topology hypotheses with an example from diving beetles; citation_author=Bergsten  J.; citation_author=Nilsson  A.N.; citation_author=Ronquist  F.;  citation_journal_title=Syst. Biol.;  citation_year=2013;  citation_volume=62;  citation_pages=660-673; "><meta name="citation_reference" content="citation_title=BEAST 2: a software platform for Bayesian evolutionary analysis; citation_author=Bouckaert  R.; citation_author=Heled  J.; citation_author=Kühnert  D.; citation_author=Vaughan  T.; citation_author=Wu  C.-H.; citation_author=Xie  D.; citation_author=Suchard  M. A.; citation_author=Rambaut  A.; citation_author=Drummond  A. J.;  citation_journal_title=PLoS Comput. Biol.;  citation_year=2014;  citation_volume=10;  citation_pages=e1003537"><meta name="citation_reference" content="citation_title=The reconstructed evolutionary process with the fossil record; citation_author=Didier  G.; citation_author=Royer-Carenzi  M.; citation_author=Laurin  M.;  citation_journal_title=J. Theoret. Biol.;  citation_year=2012;  citation_volume=315;  citation_pages=26-37; "><meta name="citation_reference" content="citation_title=Phylogenetic analysis of molecular and morphological data highlights uncertainty in the relationships of fossil and living species of elopomorpha (Actinopterygii: Teleostei); citation_author=Dornburg  A.; citation_author=Friedman  M.; citation_author=Near  T.J.;  citation_journal_title=Mol. Phylogenet. Evol.;  citation_year=2015;  citation_volume=89;  citation_pages=205-218; "><meta name="citation_reference" content="citation_title=Bayesian random local clocks, or one rate to rule them all; citation_author=Drummond  A.J.; citation_author=Suchard  M.A.;  citation_journal_title=BMC Biol.;  citation_year=2010;  citation_volume=8;  citation_pages=114"><meta name="citation_reference" content="citation_title=Relaxed phylogenetics and dating with confidence; citation_author=Drummond  A.J.; citation_author=Ho  S.Y.; citation_author=Phillips  M.J.; citation_author=Rambaut  A.;  citation_journal_title=PLoS Biol.;  citation_year=2006;  citation_volume=4;  citation_pages=e88"><meta name="citation_reference" content="citation_title=A method for investigating relative timing information on phylogenetic trees; citation_author=Ford  D.; citation_author=Gernhard  T.; citation_author=Matsen  E.;  citation_journal_title=Syst. Biol.;  citation_year=2009;  citation_volume=58;  citation_pages=167"><meta name="citation_reference" content="citation_title=Bayesian inference of sampled ancestor trees for epidemiology and fossil calibration; citation_author=Gavryushkina  A.; citation_author=Welch  D.; citation_author=Stadler  T.; citation_author=Drummond  A.;  citation_journal_title=PLoS Comput. Biol.;  citation_year=2014;  citation_volume=10;  citation_pages=e1003919"><meta name="citation_reference" content="citation_title=The conditioned reconstructed process; citation_author=Gernhard  T.;  citation_journal_title=J. Theoret. Biol.;  citation_year=2008;  citation_volume=253;  citation_pages=769-778; "><meta name="citation_reference" content="citation_title=Reading the entrails of chickens: Molecular timescales of evolution and the illusion of precision; citation_author=Graur  D.; citation_author=Martin  W.;  citation_journal_title=Trends Genet.;  citation_year=2004;  citation_volume=20;  citation_pages=80-86; "><meta name="citation_reference" content="citation_title=Using more than the oldest fossils: Dating osmundaceae with three Bayesian clock approaches; citation_author=Grimm  G.W.; citation_author=Kapli  P.; citation_author=Bomfleur  B.; citation_author=McLoughlin  S.; citation_author=Renner  S.S.;  citation_journal_title=Syst. Biol.;  citation_year=2015;  citation_volume=64;  citation_pages=396-405; "><meta name="citation_reference" content="citation_title=Estimation of branching dates among primates by molecular clocks of nuclear DNA which slowed down in Hominoidea; citation_author=Hasegawa  M.; citation_author=Kishino  H.; citation_author=Yano  T.;  citation_journal_title=J. Human Evol.;  citation_year=1989;  citation_volume=18;  citation_pages=461-476; "><meta name="citation_reference" content="citation_title=A hierarchical Bayesian model for calibrating estimates of species divergence times; citation_author=Heath  T.A.;  citation_journal_title=Syst. Biol.;  citation_year=2012;  citation_volume=61;  citation_pages=793-809; "><meta name="citation_reference" content="citation_title=A Dirichlet process prior for estimating lineage-specific substitution rates; citation_author=Heath  T.A.; citation_author=Holder  M.T.; citation_author=Huelsenbeck  J.P.;  citation_journal_title=Mole. Biol. Evol.;  citation_year=2012;  citation_volume=29;  citation_pages=939-255; "><meta name="citation_reference" content="citation_title=The fossilized birth–death process for coherent calibration of divergence-time estimates; citation_author=Heath  T.A.; citation_author=Huelsenbeck  J.P.; citation_author=Stadler  T.;  citation_journal_title=Proc. Natl Acad. Sci., USA;  citation_year=2014;  citation_volume=111;  citation_pages=E2957-E2966; "><meta name="citation_reference" content="citation_title=Bayesian inference of species divergence times; citation_author=Heath  T.A.; citation_author=Moore  B.R.; citation_author=Chen  M.-H.; citation_author=Kuo  L.; citation_author=Lewis  P.O.; citation_publisher=Chapman &amp; Hall/CRC, Florida, US;  citation_title=Bayesian phylogenetics: methods, algorithms, and applications;  citation_year=2014; "><meta name="citation_reference" content="citation_title=Precision of molecular time estimates; citation_author=Hedges  S.B.; citation_author=Kumar  S.;  citation_journal_title=Trends Genet.;  citation_year=2004;  citation_volume=20;  citation_pages=242-247; "><meta name="citation_reference" content="citation_title=Accounting for calibration uncertainty in phylogenetic estimation of evolutionary divergence times; citation_author=Ho  S.Y.W.; citation_author=Phillips  M.J.;  citation_journal_title=Syst. Biol.;  citation_year=2009;  citation_volume=58;  citation_pages=367-380; "><meta name="citation_reference" content="citation_title=Inferring speciation and extinction rates under different sampling schemes; citation_author=Höhna  S.; citation_author=Stadler  T.; citation_author=Ronquist  F.; citation_author=Britton  T.;  citation_journal_title=Mole. Biol. Evol.;  citation_year=2011;  citation_volume=28;  citation_pages=2577-2589; "><meta name="citation_reference" content="citation_title=A compound Poisson process for relaxing the molecular clock; citation_author=Huelsenbeck  J.P.; citation_author=Larget  B.; citation_author=Swofford  D.L.;  citation_journal_title=Genetics;  citation_year=2000;  citation_volume=154;  citation_pages=1879-1892; "><meta name="citation_reference" content="citation_title=Evolution of protein molecules; citation_author=Jukes  T.H.; citation_author=Cantor  C.R.; citation_author=Munro  H.N.; citation_publisher=Academic Press, New York;  citation_title=Mammalian protein metabolism;  citation_year=1969;  citation_pages=21-123; "><meta name="citation_reference" content="citation_title=On the generalized “birth-and-death” process; citation_author=Kendall  D.G.;  citation_journal_title=Annals Math. Stat.;  citation_year=1948;  citation_volume=19;  citation_pages=1-15; "><meta name="citation_reference" content="citation_title=Maximum likelihood inference of protein phylogeny and th origin of chloroplasts; citation_author=Kishino  H.; citation_author=Miyata  T.; citation_author=Hasegawa  M.;  citation_journal_title=J. Mol. Evol.;  citation_year=1990;  citation_volume=31;  citation_pages=151-160; "><meta name="citation_reference" content="citation_title=Performance of a divergence time estimation method under a probabilistic model of rate evolution; citation_author=Kishino  H.; citation_author=Thorne  J.L.; citation_author=Bruno  W.;  citation_journal_title=Mol. Biol. Evol.;  citation_year=2001;  citation_volume=18;  citation_pages=352-361; "><meta name="citation_reference" content="citation_title=Convergent intron gains in hymenopteran elongation factor-1α; citation_author=Klopfstein  S.; citation_author=Ronquist  F.;  citation_journal_title=Mol. Phylogenet. Evol.;  citation_year=2013;  citation_volume=67;  citation_pages=266-276; "><meta name="citation_reference" content="citation_title=Birth–death models and coalescent point processes: The shape and probability of reconstructed phylogenies; citation_author=Lambert  A.; citation_author=Stadler  T.;  citation_journal_title=Theoret. Popul. Biol.;  citation_year=2013;  citation_volume=90;  citation_pages=113-128; "><meta name="citation_reference" content="citation_title=Phylogenetic uncertainty and molecular clock calibrations: a case study of legless lizards (Pygopodidae, Gekkota); citation_author=Lee  M.S.Y.; citation_author=Oliver  P.M.; citation_author=Hutchinson  M.N.;  citation_journal_title=Mol. Phylogenet. Evol.;  citation_year=2009;  citation_volume=50;  citation_pages=661-666; "><meta name="citation_reference" content="citation_title=Continuous and tractable models for the variation of evolutionary rates; citation_author=Lepage  T.; citation_author=Lawi  S.; citation_author=Tupper  P.; citation_author=Bryant  D.;  citation_journal_title=Math. Biosci.;  citation_year=2006;  citation_volume=199;  citation_pages=216-233; "><meta name="citation_reference" content="citation_title=A general comparison of relaxed molecular clock models; citation_author=Lepage  T.; citation_author=Bryant  D.; citation_author=Philippe  H.; citation_author=Lartillot  N.;  citation_journal_title=Mol. Biol. Evol.;  citation_year=2007;  citation_volume=24;  citation_pages=2669-2680; "><meta name="citation_reference" content="citation_title=A likelihood approach to estimating phylogeny from discrete morphological character data; citation_author=Lewis  P.O.;  citation_journal_title=Syst. Biol.;  citation_year=2001;  citation_volume=50;  citation_pages=913-925; "><meta name="citation_reference" content="citation_title=Polytomies and Bayesian phylogenetic inference; citation_author=Lewis  P.; citation_author=Holder  M.; citation_author=Holsinger  K.;  citation_journal_title=Syst. Biol.;  citation_year=2005;  citation_volume=54;  citation_pages=241-253; "><meta name="citation_reference" content="citation_title=Phylogenomics resolves the timing and pattern of insect evolution; citation_author=Misof  B.; citation_author=Liu  S.; citation_author=Meusemann  K.; citation_author=Peters  R.S.; citation_author=Donath  A.; citation_author=Mayer  C.; citation_author=Frandsen  P.B.; citation_author=Ware  J.; citation_author=Flouri  T.; citation_author=Beutel  R.G.; citation_author=Niehuis  O.; citation_author=Petersen  M.; citation_author=Izquierdo-Carrasco  F.; citation_author=Wappler  T.; citation_author=Rust  J.; citation_author=Aberer  A.J.; citation_author=Aspöck  U.; citation_author=Aspöck  H.; citation_author=Bartel  D.; citation_author=Blanke  A.; citation_author=Berger  S.; citation_author=Böhm  A.; citation_author=Buckley  T.R.; citation_author=Calcott  B.; citation_author=Chen  J.; citation_author=Friedrich  F.; citation_author=Fukui  M.; citation_author=Fujita  M.; citation_author=Greve  C.; citation_author=Grobe  P.; citation_author=Gu  S.; citation_author=Huang  Y.; citation_author=Jermiin  L.S.; citation_author=Kawahara  A.Y.; citation_author=Krogmann  L.; citation_author=Kubiak  M.; citation_author=Lanfear  R.; citation_author=Letsch  H.; citation_author=Li  Y.; citation_author=Li  Z.; citation_author=Li  J.; citation_author=Lu  H.; citation_author=Machida  R.; citation_author=Mashimo  Y.; citation_author=Kapli  P.; citation_author=McKenna  D.D.; citation_author=Meng  G.; citation_author=Nakagaki  Y.; citation_author=Navarrete-Heredia  J.L.; citation_author=Ott  M.; citation_author=Ou  Y.; citation_author=Pass  G.; citation_author=Podsiadlowski  L.; citation_author=Pohl  H.; citation_author=Reumont von  B.M.; citation_author=Schütte  K.; citation_author=Sekiya  K.; citation_author=Shimizu  S.; citation_author=Slipinski  A.; citation_author=Stamatakis  A.; citation_author=Song  W.; citation_author=Su  X.; citation_author=Szucsich  N.U.; citation_author=Tan  M.; citation_author=Tan  X.; citation_author=Tang  M.; citation_author=Tang  J.; citation_author=Timelthaler  G.; citation_author=Tomizuka  S.; citation_author=Trautwein  M.; citation_author=Tong  X.; citation_author=Uchifune  T.; citation_author=Walzl  M.G.; citation_author=Wiegmann  B.M.; citation_author=Wilbrandt  J.; citation_author=Wipfler  B.; citation_author=Wong  T.K.F.; citation_author=Wu  Q.; citation_author=Wu  G.; citation_author=Xie  Y.; citation_author=Yang  S.; citation_author=Yang  Q.; citation_author=Yeates  D.K.; citation_author=Yoshizawa  K.; citation_author=Zhang  Q.; citation_author=Zhang  R.; citation_author=Zhang  W.; citation_author=Zhang  Y.; citation_author=Zhao  J.; citation_author=Zhou  C.; citation_author=Zhou  L.; citation_author=Ziesmann  T.; citation_author=Zou  S.; citation_author=Li  Y.; citation_author=Xu  X.; citation_author=Zhang  Y.; citation_author=Yang  H.; citation_author=Wang  J.; citation_author=Wang  J.; citation_author=Kjer  K.M.; citation_author=Zhou  X.;  citation_journal_title=Science;  citation_year=2014;  citation_volume=346;  citation_pages=763-767; "><meta name="citation_reference" content="citation_title=The reconstructed evolutionary process; citation_author=Nee  S.; citation_author=May  R.M.; citation_author=Harvey  P.H.;  citation_journal_title=Philo. Trans. R. Soc. B;  citation_year=1994;  citation_volume=344;  citation_pages=305-311; "><meta name="citation_reference" content="citation_title=Divergence time estimation using fossils as terminal taxa and the origins of Lissamphibia; citation_author=Pyron  R.A.;  citation_journal_title=Syst. Biol.;  citation_year=2011;  citation_volume=60;  citation_pages=466-481; "><meta name="citation_reference" content="citation_author=R Development Core Team; citation_publisher=R Foundation for Statistical Computing, Vienna, Austria;  citation_title=R: a language and environment for statistical computing;  citation_year=2009; "><meta name="citation_reference" content="citation_title=Seq-Gen: an application for the Monte Carlo simulation of DNA sequence evolution along phylogenetic trees; citation_author=Rambaut  A.; citation_author=Grass  N.C.;  citation_journal_title=Comput. Appl. Biosci.;  citation_year=1997;  citation_volume=13;  citation_pages=235-238; "><meta name="citation_reference" content="citation_title=Inferring speciation times under an episodic molecular clock; citation_author=Rannala  B.; citation_author=Yang  Z.;  citation_journal_title=Syst. Biol.;  citation_year=2007;  citation_volume=56;  citation_pages=453-466; "><meta name="citation_reference" content="citation_title=An outline of evolution of the hymenopterous insects (order vespida); citation_author=Rasnitsyn  A.P.;  citation_journal_title=Orient. Insects;  citation_year=1988;  citation_volume=22;  citation_pages=115-145; "><meta name="citation_reference" content="citation_title=Molecular phylogenetics, morphological cladistics, and fossil record; citation_author=Rasnitsyn  A.P.;  citation_journal_title=Entomol. Rev.;  citation_year=2010;  citation_volume=90;  citation_pages=263-298; "><meta name="citation_reference" content="citation_title=Mrbayes 3.2: Efficient Bayesian phylogenetic inference and model choice across a large model space; citation_author=Ronquist  F.; citation_author=Teslenko  M.; citation_author=van der Mark  P.; citation_author=Ayres  D.L.; citation_author=Darling  A.; citation_author=Höhna  S.; citation_author=Larget  B.; citation_author=Liu  L.; citation_author=Suchard  M.A.; citation_author=Huelsenbeck  J.P.;  citation_journal_title=Syst. Biol.;  citation_year=2012a;  citation_volume=61;  citation_pages=539-542; "><meta name="citation_reference" content="citation_title=A total-evidence approach to dating with fossils, applied to the early radiation of the Hymenoptera; citation_author=Ronquist  F.; citation_author=Klopfstein  S.; citation_author=Vilhelmsen  L.; citation_author=Schulmeister  S.; citation_author=Murray  D.L.; citation_author=Rasnitsyn  A.P.;  citation_journal_title=Syst. Biol.;  citation_year=2012b;  citation_volume=61;  citation_pages=973-999; "><meta name="citation_reference" content="citation_title=On incomplete sampling under birth–death models and connections to the sampling-based coalescent; citation_author=Stadler  T.;  citation_journal_title=J. Theoret. Biol.;  citation_year=2009;  citation_volume=261;  citation_pages=58-66; "><meta name="citation_reference" content="citation_title=Sampling-through-time in birth–death trees; citation_author=Stadler  T.;  citation_journal_title=J. Theoret. Biol.;  citation_year=2010a;  citation_volume=267;  citation_pages=396-404; "><meta name="citation_reference" content=" Stadler T. 2010b. TreeSim: Simulating trees under the birth–death model. R package version 1.0.  "><meta name="citation_reference" content="citation_title=Mammalian phylogeny reveals recent diversification rate shifts; citation_author=Stadler  T.;  citation_journal_title=Proc. Natl. Acad. Sci. USA;  citation_year=2011;  citation_volume=108;  citation_pages=6187-6192; "><meta name="citation_reference" content="citation_title=How can we improve accuracy of macroevolutionary rate estimates? Syst; citation_author=Stadler  T.;  citation_journal_title=Biol.;  citation_year=2012;  citation_volume=62;  citation_pages=321"><meta name="citation_reference" content="citation_title=Estimating speciation and extinction rates for phylogenies of higher taxa; citation_author=Stadler  T.; citation_author=Bokma  F.;  citation_journal_title=Syst. Biol.;  citation_year=2013;  citation_volume=62;  citation_pages=220-230; "><meta name="citation_reference" content="citation_title=Birth–death skyline plot reveals temporal changes of epidemic spread in HIV and hepatitis C virus (HCV); citation_author=Stadler  T.; citation_author=Kühnert  D.; citation_author=Bonhoeffer  S.; citation_author=Drummond  A.J.;  citation_journal_title=Proc. Natl Acad. Sci. USA;  citation_year=2013;  citation_volume=110;  citation_pages=228-233; "><meta name="citation_reference" content="citation_title=Using the fossil record to estimate the age of the last common ancestor of extant primates; citation_author=Tavaré  S.; citation_author=Marshall  C.R.; citation_author=Will  O.; citation_author=Soligo  C.; citation_author=Martin  R.D.;  citation_journal_title=Nature;  citation_year=2002;  citation_volume=416;  citation_pages=726-729; "><meta name="citation_reference" content="citation_title=Divergence time and evolutionary rate estimation with multilocus data; citation_author=Thorne  J.; citation_author=Kishino  H.;  citation_journal_title=Syst. Biol.;  citation_year=2002;  citation_volume=51;  citation_pages=689-702; "><meta name="citation_reference" content="citation_title=Estimating the rate of evolution of the rate of molecular evolution; citation_author=Thorne  J.; citation_author=Kishino  H.; citation_author=Painter  I.S.;  citation_journal_title=Mol. Biol. Evol.;  citation_year=1998;  citation_volume=15;  citation_pages=1647-1657; "><meta name="citation_reference" content="citation_title=Dating primate divergences through an integrated analysis of palaeontological and molecular data; citation_author=Wilkinson  R.D.; citation_author=Steiper  M.E.; citation_author=Soligo  C.; citation_author=Martin  R.D.; citation_author=Yang  Z.; citation_author=Tavaré  S.;  citation_journal_title=Syst. Biol.;  citation_year=2011;  citation_volume=60;  citation_pages=16-31; "><meta name="citation_reference" content="citation_title=Treating fossils as terminal taxa in divergence time estimation reveals ancient vicariance patterns in the palpimanoid spiders; citation_author=Wood  H.M.; citation_author=Matzke  N.J.; citation_author=Gillespie  R.G.; citation_author=Griswold  C.E.;  citation_journal_title=Syst. Biol.;  citation_year=2013;  citation_volume=62;  citation_pages=264-284; "><meta name="citation_reference" content="citation_title=Improving marginal likelihood estimation for Bayesian phylogenetic model selection; citation_author=Xie  W.; citation_author=Lewis  P.O.; citation_author=Fan  Y.; citation_author=Kuo  L.; citation_author=Chen  M.-H.;  citation_journal_title=Syst. Biol.;  citation_year=2011;  citation_volume=60;  citation_pages=150-160; "><meta name="citation_reference" content="citation_title=Bayesian phylogenetic inference using DNA sequences: a Markov chain Monte Carlo method; citation_author=Yang  Z.; citation_author=Rannala  B.;  citation_journal_title=Mol. Biol. Evol.;  citation_year=1997;  citation_volume=14;  citation_pages=717-724; "><meta name="citation_reference" content="citation_title=Bayesian estimation of species divergence times under a molecular clock using multiple fossil calibrations with soft bounds; citation_author=Yang  Z.; citation_author=Rannala  B.;  citation_journal_title=Mol. Biol. Evol.;  citation_year=2006;  citation_volume=23;  citation_pages=212-226; "><meta name="citation_reference" content="citation_title=Comparison of likelihood and Bayesian methods for estimating divergence times using multiple gene loci and calibration points, with application to a radiation of cute-looking mouse lemur species; citation_author=Yang  Z.; citation_author=Yoder  A.D.;  citation_journal_title=Syst. Biol.;  citation_year=2003;  citation_volume=52;  citation_pages=705-716; "><meta name="citation_reference" content="citation_title=Estimation of primate speciation dates using local molecular clocks; citation_author=Yoder  A.D.; citation_author=Yang  Z.;  citation_journal_title=Mol. Biol. Evol.;  citation_year=2000;  citation_volume=17;  citation_pages=1081-1090; "><meta name="citation_reference" content=" Zhang C. 2015. A tutorial on molecular clock dating using MrBayes https://sites.google.com/site/zhangchicool/ , last accessed October 30, 2015.  "><meta name="citation_reference" content="citation_title=Evolutionary divergence and convergence in proteins; citation_author=Zuckerkandl  E.; citation_author=Pauling  L.; citation_author=Bryson  V.; citation_author=Vogel  H.J.; citation_publisher=Academic Press, New York;  citation_title=Evol. Genes Proteins;  citation_year=1965;  citation_pages=97-166; "><meta name="citation_fulltext_world_readable" content=""><meta name="citation_pdf_url" content="https://academic.oup.com/sysbio/article-pdf/65/2/228/24589028/syv080.pdf"><meta name="description" content="Abstract.  Bayesian total-evidence dating involves the simultaneous analysis of morphological data from the fossil record and morphological and sequence data fr">
        <meta name="citation_xml_url" content="https://academic.oup.com/sysbio/article-xml/65/2/228/2427164">



    <link rel="canonical" href="https://academic.oup.com/sysbio/article/65/2/228/2427164">


        <!-- Start: GPT Async -->

<script async="async" src="gpt.js"></script>
<script>
  var googletag = googletag || {};
  googletag.cmd = googletag.cmd || [];
</script>



<script type="text/javascript">
  var gptAdSlots = [];
  googletag.cmd.push(function() {
    var pathChar = '/';
    
    var mapping_ad1 = googletag.sizeMapping().
    addSize([1024, 0], [[970, 90], [728, 90]]).
    addSize([768, 0], [728, 90]).
    addSize([0, 0], [320, 50]).build();
    gptAdSlots["ad1"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'sysbio_Behind_Ad1', [[970, 90], [728, 90], [320, 50]], 'adBlockHeader').
    defineSizeMapping(mapping_ad1).
    addService(googletag.pubads());
    
    var mapping_ad2 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad2"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'sysbio_Behind_Ad2', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyTop').
    defineSizeMapping(mapping_ad2).
    addService(googletag.pubads());
    
    var mapping_ad3 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad3"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'sysbio_Behind_Ad3', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyBottom').
    defineSizeMapping(mapping_ad3).
    addService(googletag.pubads());
    
    var mapping_ad4 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [728, 90]).build();
    gptAdSlots["ad4"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'sysbio_Behind_Ad4', [728, 90], 'adBlockFooter').
    defineSizeMapping(mapping_ad4).
    addService(googletag.pubads());
    
    googletag.pubads().setTargeting("jnlspage", "article");
    googletag.pubads().setTargeting("jnlsurl", "sysbio/article/65/2/228/2427164");
    googletag.pubads().enableSingleRequest();
    googletag.pubads().collapseEmptyDivs();
    
  });
</script>


    <script>
        var oupcookiepolicy_siteid = "journals";;	  	// the website id	
        var oupcookiepolicy_messagetype = 'implied';			            // type of alert message e.g, implied / explicit
        var oupcookiepolicy_preferredlanguage = "en";		// preferred language of the website
        var oupcookiepolicy_impliedmessageclass = 'cookiepolicyimplied';	// the css class for implied alert message
        var oupcookiepolicy_documentroot = '/';	  			                // the document root the cookie is set to
    </script>
    <noscript>We use cookies to enhance your experience on our website. By continuing to use our website, you are agreeing to our use of cookies. You can change your cookie settings at any time. <a href="https://global.oup.com/cookiepolicy/">Find out more</a></noscript>

    <script type="text/javascript">
        var NTPT_PGEXTRA='event_type=full-text&supplier_tag=SC_Journals&siteid=sysbio&authentication_method=IP&authzrequired=false&account_id=2786682&authnips=147.231.250.202&doi=10.1093/sysbio/syv080';
    </script>
<title>
	Total-Evidence Dating under the Fossilized Birth–Death Process | Systematic Biology | Oxford Academic
</title><link rel="preload" href="integrator_002.js"><script type="text/javascript" src="integrator_002.js"></script><link rel="preload" href="integrator.js"><script type="text/javascript" src="integrator.js"></script><script src="pubads_impl_249.js" async=""></script><script id="altmetric-embed-js" src="altmetric_badges-6b74dfff5c7f701b334f3e44e411f88f.js"></script><style></style><script type="text/javascript" src="TeX-AMS-MML_HTMLorMML.js"></script><style type="text/css">.at-icon{fill:#fff;border:0}.at-icon-wrapper{display:inline-block;overflow:hidden}a .at-icon-wrapper{cursor:pointer}.at-rounded,.at-rounded-element .at-icon-wrapper{border-radius:12%}.at-circular,.at-circular-element .at-icon-wrapper{border-radius:50%}.addthis_32x32_style .at-icon{width:2pc;height:2pc}.addthis_24x24_style .at-icon{width:24px;height:24px}.addthis_20x20_style .at-icon{width:20px;height:20px}.addthis_16x16_style .at-icon{width:1pc;height:1pc}#at16lb{display:none;position:absolute;top:0;left:0;width:100%;height:100%;z-index:1001;background-color:#000;opacity:.001}#at_complete,#at_error,#at_share,#at_success{position:static!important}.at15dn{display:none}#at15s,#at16p,#at16p form input,#at16p label,#at16p textarea,#at_share .at_item{font-family:arial,helvetica,tahoma,verdana,sans-serif!important;font-size:9pt!important;outline-style:none;outline-width:0;line-height:1em}* html #at15s.mmborder{position:absolute!important}#at15s.mmborder{position:fixed!important;width:250px!important}#at15s{background:url(data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAoAAAAKCAYAAACNMs+9AAAAGXRFWHRTb2Z0d2FyZQBBZG9iZSBJbWFnZVJlYWR5ccllPAAAABtJREFUeNpiZGBgaGAgAjAxEAlGFVJHIUCAAQDcngCUgqGMqwAAAABJRU5ErkJggg==);float:none;line-height:1em;margin:0;overflow:visible;padding:5px;text-align:left;position:absolute}#at15s a,#at15s span{outline:0;direction:ltr;text-transform:none}#at15s .at-label{margin-left:5px}#at15s .at-icon-wrapper{width:1pc;height:1pc;vertical-align:middle}#at15s .at-icon{width:1pc;height:1pc}.at4-icon{display:inline-block;background-repeat:no-repeat;background-position:top left;margin:0;overflow:hidden;cursor:pointer}.addthis_16x16_style .at4-icon,.addthis_default_style .at4-icon,.at4-icon,.at-16x16{width:1pc;height:1pc;line-height:1pc;background-size:1pc!important}.addthis_32x32_style .at4-icon,.at-32x32{width:2pc;height:2pc;line-height:2pc;background-size:2pc!important}.addthis_24x24_style .at4-icon,.at-24x24{width:24px;height:24px;line-height:24px;background-size:24px!important}.addthis_20x20_style .at4-icon,.at-20x20{width:20px;height:20px;line-height:20px;background-size:20px!important}.at4-icon.circular,.circular .at4-icon,.circular.aticon{border-radius:50%}.at4-icon.rounded,.rounded .at4-icon{border-radius:4px}.at4-icon-left{float:left}#at15s .at4-icon{text-indent:20px;padding:0;overflow:visible;white-space:nowrap;background-size:1pc;width:1pc;height:1pc;background-position:top left;display:inline-block;line-height:1pc}.addthis_vertical_style .at4-icon,.at4-follow-container .at4-icon{margin-right:5px}html>body #at15s{width:250px!important}#at15s.atm{background:none!important;padding:0!important;width:10pc!important}#at15s_inner{background:#fff;border:1px solid #fff;margin:0}#at15s_head{position:relative;background:#f2f2f2;padding:4px;cursor:default;border-bottom:1px solid #e5e5e5}.at15s_head_success{background:#cafd99!important;border-bottom:1px solid #a9d582!important}.at15s_head_success a,.at15s_head_success span{color:#000!important;text-decoration:none}#at15s_brand,#at15sptx,#at16_brand{position:absolute}#at15s_brand{top:4px;right:4px}.at15s_brandx{right:20px!important}a#at15sptx{top:4px;right:4px;text-decoration:none;color:#4c4c4c;font-weight:700}#at15sptx:hover{text-decoration:underline}#at16_brand{top:5px;right:30px;cursor:default}#at_hover{padding:4px}#at_hover .at_item,#at_share .at_item{background:#fff!important;float:left!important;color:#4c4c4c!important}#at_share .at_item .at-icon-wrapper{margin-right:5px}#at_hover .at_bold{font-weight:700;color:#000!important}#at_hover .at_item{width:7pc!important;padding:2px 3px!important;margin:1px;text-decoration:none!important}#at_hover .at_item.athov,#at_hover .at_item:focus,#at_hover .at_item:hover{margin:0!important}#at_hover .at_item.athov,#at_hover .at_item:focus,#at_hover .at_item:hover,#at_share .at_item.athov,#at_share .at_item:hover{background:#f2f2f2!important;border:1px solid #e5e5e5;color:#000!important;text-decoration:none}.ipad #at_hover .at_item:focus{background:#fff!important;border:1px solid #fff}.at15t{display:block!important;height:1pc!important;line-height:1pc!important;padding-left:20px!important;background-position:0 0;text-align:left}.addthis_button,.at15t{cursor:pointer}.addthis_toolbox a.at300b,.addthis_toolbox a.at300m{width:auto}.addthis_toolbox a{margin-bottom:5px;line-height:initial}.addthis_toolbox.addthis_vertical_style{width:200px}.addthis_button_facebook_like .fb_iframe_widget{line-height:100%}.addthis_button_facebook_like iframe.fb_iframe_widget_lift{max-width:none}.addthis_toolbox a.addthis_button_counter,.addthis_toolbox a.addthis_button_facebook_like,.addthis_toolbox a.addthis_button_facebook_send,.addthis_toolbox a.addthis_button_facebook_share,.addthis_toolbox a.addthis_button_foursquare,.addthis_toolbox a.addthis_button_google_plusone,.addthis_toolbox a.addthis_button_linkedin_counter,.addthis_toolbox a.addthis_button_pinterest_pinit,.addthis_toolbox a.addthis_button_tweet{display:inline-block}.at-share-tbx-element .google_plusone_iframe_widget>span>div{vertical-align:top!important}.addthis_toolbox span.addthis_follow_label{display:none}.addthis_toolbox.addthis_vertical_style span.addthis_follow_label{display:block;white-space:nowrap}.addthis_toolbox.addthis_vertical_style a{display:block}.addthis_toolbox.addthis_vertical_style.addthis_32x32_style a{line-height:2pc;height:2pc}.addthis_toolbox.addthis_vertical_style .at300bs{margin-right:4px;float:left}.addthis_toolbox.addthis_20x20_style span{line-height:20px}.addthis_toolbox.addthis_32x32_style span{line-height:2pc}.addthis_toolbox.addthis_pill_combo_style .addthis_button_compact .at15t_compact,.addthis_toolbox.addthis_pill_combo_style a{float:left}.addthis_toolbox.addthis_pill_combo_style a.addthis_button_tweet{margin-top:-2px}.addthis_toolbox.addthis_pill_combo_style .addthis_button_compact .at15t_compact{margin-right:4px}.addthis_default_style .addthis_separator{margin:0 5px;display:inline}div.atclear{clear:both}.addthis_default_style .addthis_separator,.addthis_default_style .at4-icon,.addthis_default_style .at300b,.addthis_default_style .at300bo,.addthis_default_style .at300bs,.addthis_default_style .at300m{float:left}.at300b img,.at300bo img{border:0}a.at300b .at4-icon,a.at300m .at4-icon{display:block}.addthis_default_style .at300b,.addthis_default_style .at300bo,.addthis_default_style .at300m{padding:0 2px}.at300b,.at300bo,.at300bs,.at300m{cursor:pointer}.addthis_button_facebook_like.at300b:hover,.addthis_button_facebook_like.at300bs:hover,.addthis_button_facebook_send.at300b:hover,.addthis_button_facebook_send.at300bs:hover{opacity:1}.addthis_20x20_style .at15t,.addthis_20x20_style .at300bs{overflow:hidden;display:block;height:20px!important;width:20px!important;line-height:20px!important}.addthis_32x32_style .at15t,.addthis_32x32_style .at300bs{overflow:hidden;display:block;height:2pc!important;width:2pc!important;line-height:2pc!important}.at300bs{overflow:hidden;display:block;background-position:0 0;height:1pc;width:1pc;line-height:1pc!important}.addthis_default_style .at15t_compact,.addthis_default_style .at15t_expanded{margin-right:4px}#at_share .at_item{width:123px!important;padding:4px;margin-right:2px;border:1px solid #fff}#at16p{background:url(data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAoAAAAKCAYAAACNMs+9AAAAGXRFWHRTb2Z0d2FyZQBBZG9iZSBJbWFnZVJlYWR5ccllPAAAABtJREFUeNpiZGBgaGAgAjAxEAlGFVJHIUCAAQDcngCUgqGMqwAAAABJRU5ErkJggg==);z-index:10000001;position:absolute;top:50%;left:50%;width:300px;padding:10px;margin:0 auto;margin-top:-185px;margin-left:-155px;font-family:arial,helvetica,tahoma,verdana,sans-serif;font-size:9pt;color:#5e5e5e}#at_share{margin:0;padding:0}#at16pt{position:relative;background:#f2f2f2;height:13px;padding:5px 10px}#at16pt a,#at16pt h4{font-weight:700}#at16pt h4{display:inline;margin:0;padding:0;font-size:9pt;color:#4c4c4c;cursor:default}#at16pt a{position:absolute;top:5px;right:10px;color:#4c4c4c;text-decoration:none;padding:2px}#at15sptx:focus,#at16pt a:focus{outline:thin dotted}#at15s #at16pf a{top:1px}#_atssh{width:1px!important;height:1px!important;border:0!important}.atm{width:10pc!important;padding:0;margin:0;line-height:9pt;letter-spacing:normal;font-family:arial,helvetica,tahoma,verdana,sans-serif;font-size:9pt;color:#444;background:url(data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAoAAAAKCAYAAACNMs+9AAAAGXRFWHRTb2Z0d2FyZQBBZG9iZSBJbWFnZVJlYWR5ccllPAAAABtJREFUeNpiZGBgaGAgAjAxEAlGFVJHIUCAAQDcngCUgqGMqwAAAABJRU5ErkJggg==);padding:4px}.atm-f{text-align:right;border-top:1px solid #ddd;padding:5px 8px}.atm-i{background:#fff;border:1px solid #d5d6d6;padding:0;margin:0;box-shadow:1px 1px 5px rgba(0,0,0,.15)}.atm-s{margin:0!important;padding:0!important}.atm-s a:focus{border:transparent;outline:0;transition:none}#at_hover.atm-s a,.atm-s a{display:block;text-decoration:none;padding:4px 10px;color:#235dab!important;font-weight:400;font-style:normal;transition:none}#at_hover.atm-s .at_bold{color:#235dab!important}#at_hover.atm-s a:hover,.atm-s a:hover{background:#2095f0;text-decoration:none;color:#fff!important}#at_hover.atm-s .at_bold{font-weight:700}#at_hover.atm-s a:hover .at_bold{color:#fff!important}.atm-s a .at-label{vertical-align:middle;margin-left:5px;direction:ltr}.at_PinItButton{display:block;width:40px;height:20px;padding:0;margin:0;background-image:url(//s7.addthis.com/static/t00/pinit00.png);background-repeat:no-repeat}.at_PinItButton:hover{background-position:0 -20px}.addthis_toolbox .addthis_button_pinterest_pinit{position:relative}.at-share-tbx-element .fb_iframe_widget span{vertical-align:baseline!important}#at16pf{height:auto;text-align:right;padding:4px 8px}.at-privacy-info{position:absolute;left:7px;bottom:7px;cursor:pointer;text-decoration:none;font-family:helvetica,arial,sans-serif;font-size:10px;line-height:9pt;letter-spacing:.2px;color:#666}.at-privacy-info:hover{color:#000}.body .wsb-social-share .wsb-social-share-button-vert{padding-top:0;padding-bottom:0}.body .wsb-social-share.addthis_counter_style .addthis_button_tweet.wsb-social-share-button{padding-top:40px}.body .wsb-social-share.addthis_counter_style .addthis_button_google_plusone.wsb-social-share-button{padding-top:0}.body .wsb-social-share.addthis_counter_style .addthis_button_facebook_like.wsb-social-share-button{padding-top:21px}@media print{#at4-follow,#at4-share,#at4-thankyou,#at4-whatsnext,#at4m-mobile,#at15s,.at4,.at4-recommended{display:none!important}}@media screen and (max-width:400px){.at4win{width:100%}}@media screen and (max-height:700px) and (max-width:400px){.at4-thankyou-inner .at4-recommended-container{height:122px;overflow:hidden}.at4-thankyou-inner .at4-recommended .at4-recommended-item:first-child{border-bottom:1px solid #c5c5c5}}</style><style type="text/css">.at-branding-logo{font-family:helvetica,arial,sans-serif;text-decoration:none;font-size:10px;display:inline-block;margin:2px 0;letter-spacing:.2px}.at-branding-logo .at-branding-icon{background-image:url("data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAoAAAAKCAMAAAC67D+PAAAAGXRFWHRTb2Z0d2FyZQBBZG9iZSBJbWFnZVJlYWR5ccllPAAAAAZQTFRF////+GlNUkcc1QAAAB1JREFUeNpiYIQDBjQmAwMmkwEM0JnY1WIxFyDAABGeAFEudiZsAAAAAElFTkSuQmCC")}.at-branding-logo .at-branding-icon,.at-branding-logo .at-privacy-icon{display:inline-block;height:10px;width:10px;margin-left:4px;margin-right:3px;margin-bottom:-1px;background-repeat:no-repeat}.at-branding-logo .at-privacy-icon{background-image:url("data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAkAAAAKCAMAAABR24SMAAAAGXRFWHRTb2Z0d2FyZQBBZG9iZSBJbWFnZVJlYWR5ccllPAAAABhQTFRF8fr9ot/xXcfn2/P5AKva////////AKTWodjhjAAAAAd0Uk5T////////ABpLA0YAAAA6SURBVHjaJMzBDQAwCAJAQaj7b9xifV0kUKJ9ciWxlzWEWI5gMF65KUTv0VKkjVeTerqE/x7+9BVgAEXbAWI8QDcfAAAAAElFTkSuQmCC")}.at-branding-logo span{text-decoration:none}.at-branding-logo .at-branding-addthis,.at-branding-logo .at-branding-powered-by{color:#666}.at-branding-logo .at-branding-addthis:hover{color:#333}.at-cv-with-image .at-branding-addthis,.at-cv-with-image .at-branding-addthis:hover{color:#fff}a.at-branding-logo:visited{color:initial}.at-branding-info{display:inline-block;padding:0 5px;color:#666;border:1px solid #666;border-radius:50%;font-size:10px;line-height:9pt;opacity:.7;transition:all .3s ease;text-decoration:none}.at-branding-info span{border:0;clip:rect(0 0 0 0);height:1px;margin:-1px;overflow:hidden;padding:0;position:absolute;width:1px}.at-branding-info:before{content:'i';font-family:Times New Roman}.at-branding-info:hover{color:#0780df;border-color:#0780df}</style><link id="altmetric-embed-css" rel="stylesheet" type="text/css" href="embed-2c47105b6381604898bbf8ae8a680350.css"><style type="text/css">.MathJax_Hover_Frame {border-radius: .25em; -webkit-border-radius: .25em; -moz-border-radius: .25em; -khtml-border-radius: .25em; box-shadow: 0px 0px 15px #83A; -webkit-box-shadow: 0px 0px 15px #83A; -moz-box-shadow: 0px 0px 15px #83A; -khtml-box-shadow: 0px 0px 15px #83A; border: 1px solid #A6D ! important; display: inline-block; position: absolute}
.MathJax_Menu_Button .MathJax_Hover_Arrow {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 4px; -webkit-border-radius: 4px; -moz-border-radius: 4px; -khtml-border-radius: 4px; font-family: 'Courier New',Courier; font-size: 9px; color: #F0F0F0}
.MathJax_Menu_Button .MathJax_Hover_Arrow span {display: block; background-color: #AAA; border: 1px solid; border-radius: 3px; line-height: 0; padding: 4px}
.MathJax_Hover_Arrow:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_Hover_Arrow:hover span {background-color: #CCC!important}
</style><style type="text/css">#MathJax_About {position: fixed; left: 50%; width: auto; text-align: center; border: 3px outset; padding: 1em 2em; background-color: #DDDDDD; color: black; cursor: default; font-family: message-box; font-size: 120%; font-style: normal; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 15px; -webkit-border-radius: 15px; -moz-border-radius: 15px; -khtml-border-radius: 15px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_About.MathJax_MousePost {outline: none}
.MathJax_Menu {position: absolute; background-color: white; color: black; width: auto; padding: 5px 0px; border: 1px solid #CCCCCC; margin: 0; cursor: default; font: menu; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 5px; -webkit-border-radius: 5px; -moz-border-radius: 5px; -khtml-border-radius: 5px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
.MathJax_MenuItem {padding: 1px 2em; background: transparent}
.MathJax_MenuArrow {position: absolute; right: .5em; padding-top: .25em; color: #666666; font-size: .75em}
.MathJax_MenuActive .MathJax_MenuArrow {color: white}
.MathJax_MenuArrow.RTL {left: .5em; right: auto}
.MathJax_MenuCheck {position: absolute; left: .7em}
.MathJax_MenuCheck.RTL {right: .7em; left: auto}
.MathJax_MenuRadioCheck {position: absolute; left: .7em}
.MathJax_MenuRadioCheck.RTL {right: .7em; left: auto}
.MathJax_MenuLabel {padding: 1px 2em 3px 1.33em; font-style: italic}
.MathJax_MenuRule {border-top: 1px solid #DDDDDD; margin: 4px 3px}
.MathJax_MenuDisabled {color: GrayText}
.MathJax_MenuActive {background-color: #606872; color: white}
.MathJax_MenuDisabled:focus, .MathJax_MenuLabel:focus {background-color: #E8E8E8}
.MathJax_ContextMenu:focus {outline: none}
.MathJax_ContextMenu .MathJax_MenuItem:focus {outline: none}
#MathJax_AboutClose {top: .2em; right: .2em}
.MathJax_Menu .MathJax_MenuClose {top: -10px; left: -10px}
.MathJax_MenuClose {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; font-family: 'Courier New',Courier; font-size: 24px; color: #F0F0F0}
.MathJax_MenuClose span {display: block; background-color: #AAA; border: 1.5px solid; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; line-height: 0; padding: 8px 0 6px}
.MathJax_MenuClose:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_MenuClose:hover span {background-color: #CCC!important}
.MathJax_MenuClose:hover:focus {outline: none}
</style><style type="text/css">.MathJax_Preview .MJXf-math {color: inherit!important}
</style><style type="text/css">.MJX_Assistive_MathML {position: absolute!important; top: 0; left: 0; clip: rect(1px, 1px, 1px, 1px); padding: 1px 0 0 0!important; border: 0!important; height: 1px!important; width: 1px!important; overflow: hidden!important; display: block!important; -webkit-touch-callout: none; -webkit-user-select: none; -khtml-user-select: none; -moz-user-select: none; -ms-user-select: none; user-select: none}
.MJX_Assistive_MathML.MJX_Assistive_MathML_Block {width: 100%!important}
</style><style type="text/css">#MathJax_Zoom {position: absolute; background-color: #F0F0F0; overflow: auto; display: block; z-index: 301; padding: .5em; border: 1px solid black; margin: 0; font-weight: normal; font-style: normal; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; -webkit-box-sizing: content-box; -moz-box-sizing: content-box; box-sizing: content-box; box-shadow: 5px 5px 15px #AAAAAA; -webkit-box-shadow: 5px 5px 15px #AAAAAA; -moz-box-shadow: 5px 5px 15px #AAAAAA; -khtml-box-shadow: 5px 5px 15px #AAAAAA; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_ZoomOverlay {position: absolute; left: 0; top: 0; z-index: 300; display: inline-block; width: 100%; height: 100%; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
#MathJax_ZoomFrame {position: relative; display: inline-block; height: 0; width: 0}
#MathJax_ZoomEventTrap {position: absolute; left: 0; top: 0; z-index: 302; display: inline-block; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
</style><style type="text/css">.MathJax_Preview {color: #888}
#MathJax_Message {position: fixed; left: 1px; bottom: 2px; background-color: #E6E6E6; border: 1px solid #959595; margin: 0px; padding: 2px 8px; z-index: 102; color: black; font-size: 80%; width: auto; white-space: nowrap}
#MathJax_MSIE_Frame {position: absolute; top: 0; left: 0; width: 0px; z-index: 101; border: 0px; margin: 0px; padding: 0px}
.MathJax_Error {color: #CC0000; font-style: italic}
</style></head>

<body data-sitename="systematicbiology" class="off-canvas pg_article" theme-sysbio="" style="cursor: default;">
    <script>(function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,'script','https://www.google-analytics.com/analytics.js','ga');ga('create', 'UA-78288099-3', {'sampleRate': 100});ga('send', 'pageview');</script>
    <a href="#main" class="skipnav">Skip to Main Content</a>
    <form method="post" action="./2427164" id="webform" novalidate="novalidate">
<div class="aspNetHidden">
<input name="__VIEWSTATE" id="__VIEWSTATE" value="/wEPDwUKMTAxNjEyOTA4OQ9kFgJmD2QWAmYPZBYEZg9kFgICDg9kFgICAQ9kFgJmDxYCHgRUZXh0BejOATxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iWmhhbmcsIENoaSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IiBEZXBhcnRtZW50IG9mIEJpb2luZm9ybWF0aWNzIGFuZCBHZW5ldGljcywgU3dlZGlzaCBNdXNldW0gb2YgTmF0dXJhbCBIaXN0b3J5LCBTRS0xMDQgMDUgU3RvY2tob2xtLCBTd2VkZW47ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IlN0YWRsZXIsIFRhbmphIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iIERlcGFydG1lbnQgb2YgQmlvc3lzdGVtcyBTY2llbmNlIGFuZCBFbmdpbmVlcmluZywgRWlkZ2Vuw7Zzc2lzY2hlIFRlY2huaXNjaGUgSG9jaHNjaHVsZSBaw7xyaWNoLCA0MDUzIEJhc2VsLCBTd2l0emVybGFuZDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iIFN3aXNzIEluc3RpdHV0ZSBvZiBCaW9pbmZvcm1hdGljcyAoU0lCKSwgU3dpdHplcmxhbmQ7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9Iktsb3Bmc3RlaW4sIFNlcmFpbmEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSIgRGVwYXJ0bWVudCBvZiBCaW9pbmZvcm1hdGljcyBhbmQgR2VuZXRpY3MsIFN3ZWRpc2ggTXVzZXVtIG9mIE5hdHVyYWwgSGlzdG9yeSwgU0UtMTA0IDA1IFN0b2NraG9sbSwgU3dlZGVuOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSIgRGVwYXJ0bWVudCBvZiBJbnZlcnRlYnJhdGVzLCBOYXR1cmFsIEhpc3RvcnkgTXVzZXVtIEJlcm4sIENILTMwMDUgQmVybiwgU3dpdHplcmxhbmQ7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IkhlYXRoLCBUcmFjeSBBLiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IiBEZXBhcnRtZW50IG9mIEludGVncmF0aXZlIEJpb2xvZ3ksIFVuaXZlcnNpdHkgb2YgQ2FsaWZvcm5pYSwgQmVya2VsZXksIENBIDk0NzIwIFVTQTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iIERlcGFydG1lbnQgb2YgRWNvbG9neSBhbmQgRXZvbHV0aW9uYXJ5IEJpb2xvZ3ksIFVuaXZlcnNpdHkgb2YgS2Fuc2FzLCBMYXdyZW5jZSwgS1MgNjYwNDUsIFVTQTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iIERlcGFydG1lbnQgb2YgRWNvbG9neSwgRXZvbHV0aW9uICZhbXA7IE9yZ2FuaXNtYWwgQmlvbG9neSwgSW93YSBTdGF0ZSBVbml2ZXJzaXR5LCBBbWVzLCBJQSA1MDAxMSwgVVNBICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IlJvbnF1aXN0LCBGcmVkcmlrIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iIERlcGFydG1lbnQgb2YgQmlvaW5mb3JtYXRpY3MgYW5kIEdlbmV0aWNzLCBTd2VkaXNoIE11c2V1bSBvZiBOYXR1cmFsIEhpc3RvcnksIFNFLTEwNCAwNSBTdG9ja2hvbG0sIFN3ZWRlbjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3RpdGxlIiBjb250ZW50PSJUb3RhbC1FdmlkZW5jZSBEYXRpbmcgdW5kZXIgdGhlIEZvc3NpbGl6ZWQgQmlydGjigJNEZWF0aCBQcm9jZXNzIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2ZpcnN0cGFnZSIgY29udGVudD0iMjI4IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2xhc3RwYWdlIiBjb250ZW50PSIyNDkiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fZG9pIiBjb250ZW50PSIxMC4xMDkzL3N5c2Jpby9zeXYwODAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fam91cm5hbF90aXRsZSIgY29udGVudD0iU3lzdGVtYXRpYyBCaW9sb2d5IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2pvdXJuYWxfYWJicmV2IiBjb250ZW50PSJTeXN0IEJpb2wiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fdm9sdW1lIiBjb250ZW50PSI2NSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9pc3N1ZSIgY29udGVudD0iMiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9wdWJsaWNhdGlvbl9kYXRlIiBjb250ZW50PSIyMDE2LzAzLzAxIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2lzc24iIGNvbnRlbnQ9IjEwNjMtNTE1NyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9wdWJsaXNoZXIiIGNvbnRlbnQ9Ik94Zm9yZCBVbml2ZXJzaXR5IFByZXNzIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iIEFndWlhciBBLiwgRGVhbnMgQS4sIEVuZ2VsIE0uLCBGb3JzaGFnZSBNLiwgSHViZXIgSi4sIEplbm5pbmdzIEouLCBKb2huc29uIE4uLCBMZWxlaiBBLiwgTG9uZ2lubyBKLiwgTWlrbyBJLiwgT2hsIE0uLCBSYXNtdXNzZW4gQy4sIFRhZWdlciBBLiwgWXUgRC4gMjAxMy4gT3JkZXIgSHltZW5vcHRlcmEuIEluOiBaaGFuZywgWi4tUS4gKEVkLikgQW5pbWFsIGJpb2RpdmVyc2l0eTogYW4gb3V0bGluZSBvZiBoaWdoZXItbGV2ZWwgY2xhc3NpZmljYXRpb24gYW5kIHN1cnZleSBvZiB0YXhvbm9taWMgcmljaG5lc3MgKEFkZGVuZGEgMjAxMykuIFpvb3RheGEgMzcwMzo1MeKAkzYyLiAgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QW4gZXZhbHVhdGlvbiBvZiBmb3NzaWwgdGlwLWRhdGluZyB2ZXJzdXMgbm9kZS1hZ2UgY2FsaWJyYXRpb25zIGluIHRldHJhb2RvbnRpZm9ybSBmaXNoZXMgKFRlbGVvc3RlaTogUGVyY29tb3JwaGFjZWFlKTsgY2l0YXRpb25fYXV0aG9yPUFyY2lsYSAgRC47IGNpdGF0aW9uX2F1dGhvcj1BbGV4YW5kZXIgUHlyb24gIFIuOyBjaXRhdGlvbl9hdXRob3I9VHlsZXIgIEouQy47IGNpdGF0aW9uX2F1dGhvcj1PcnTDrSAgRy47IGNpdGF0aW9uX2F1dGhvcj1CZXRhbmN1ci1SICBSLjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TW9sLiBQaHlsb2dlbmV0LiBFdm9sLjsgIGNpdGF0aW9uX3llYXI9MjAxNTsgIGNpdGF0aW9uX3ZvbHVtZT04MjsgIGNpdGF0aW9uX3BhZ2VzPVB0IEE6MTMxLTE0NTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RWZmZWN0cyBvZiBtb2RlbHMgb2YgcmF0ZSBldm9sdXRpb24gb24gZXN0aW1hdGlvbiBvZiBkaXZlcmdlbmNlIGRhdGVzIHdpdGggc3BlY2lhbCByZWZlcmVuY2UgdG8gdGhlIG1ldGF6b2FuIDE4UyByaWJvc29tYWwgUk5BIHBoeWxvZ2VueTsgY2l0YXRpb25fYXV0aG9yPUFyaXMtQnJvc291ICBTLjsgY2l0YXRpb25fYXV0aG9yPVlhbmcgIFouOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1TeXN0LiBCaW9sLjsgIGNpdGF0aW9uX3llYXI9MjAwMjsgIGNpdGF0aW9uX3ZvbHVtZT01MTsgIGNpdGF0aW9uX3BhZ2VzPTcwMy03MTQ7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUJheWVzaWFuIHRlc3RzIG9mIHRvcG9sb2d5IGh5cG90aGVzZXMgd2l0aCBhbiBleGFtcGxlIGZyb20gZGl2aW5nIGJlZXRsZXM7IGNpdGF0aW9uX2F1dGhvcj1CZXJnc3RlbiAgSi47IGNpdGF0aW9uX2F1dGhvcj1OaWxzc29uICBBLk4uOyBjaXRhdGlvbl9hdXRob3I9Um9ucXVpc3QgIEYuOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1TeXN0LiBCaW9sLjsgIGNpdGF0aW9uX3llYXI9MjAxMzsgIGNpdGF0aW9uX3ZvbHVtZT02MjsgIGNpdGF0aW9uX3BhZ2VzPTY2MC02NzM7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUJFQVNUIDI6IGEgc29mdHdhcmUgcGxhdGZvcm0gZm9yIEJheWVzaWFuIGV2b2x1dGlvbmFyeSBhbmFseXNpczsgY2l0YXRpb25fYXV0aG9yPUJvdWNrYWVydCAgUi47IGNpdGF0aW9uX2F1dGhvcj1IZWxlZCAgSi47IGNpdGF0aW9uX2F1dGhvcj1Lw7xobmVydCAgRC47IGNpdGF0aW9uX2F1dGhvcj1WYXVnaGFuICBULjsgY2l0YXRpb25fYXV0aG9yPVd1ICBDLi1ILjsgY2l0YXRpb25fYXV0aG9yPVhpZSAgRC47IGNpdGF0aW9uX2F1dGhvcj1TdWNoYXJkICBNLiBBLjsgY2l0YXRpb25fYXV0aG9yPVJhbWJhdXQgIEEuOyBjaXRhdGlvbl9hdXRob3I9RHJ1bW1vbmQgIEEuIEouOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1QTG9TIENvbXB1dC4gQmlvbC47ICBjaXRhdGlvbl95ZWFyPTIwMTQ7ICBjaXRhdGlvbl92b2x1bWU9MTA7ICBjaXRhdGlvbl9wYWdlcz1lMTAwMzUzNyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRoZSByZWNvbnN0cnVjdGVkIGV2b2x1dGlvbmFyeSBwcm9jZXNzIHdpdGggdGhlIGZvc3NpbCByZWNvcmQ7IGNpdGF0aW9uX2F1dGhvcj1EaWRpZXIgIEcuOyBjaXRhdGlvbl9hdXRob3I9Um95ZXItQ2FyZW56aSAgTS47IGNpdGF0aW9uX2F1dGhvcj1MYXVyaW4gIE0uOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1KLiBUaGVvcmV0LiBCaW9sLjsgIGNpdGF0aW9uX3llYXI9MjAxMjsgIGNpdGF0aW9uX3ZvbHVtZT0zMTU7ICBjaXRhdGlvbl9wYWdlcz0yNi0zNzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9UGh5bG9nZW5ldGljIGFuYWx5c2lzIG9mIG1vbGVjdWxhciBhbmQgbW9ycGhvbG9naWNhbCBkYXRhIGhpZ2hsaWdodHMgdW5jZXJ0YWludHkgaW4gdGhlIHJlbGF0aW9uc2hpcHMgb2YgZm9zc2lsIGFuZCBsaXZpbmcgc3BlY2llcyBvZiBlbG9wb21vcnBoYSAoQWN0aW5vcHRlcnlnaWk6IFRlbGVvc3RlaSk7IGNpdGF0aW9uX2F1dGhvcj1Eb3JuYnVyZyAgQS47IGNpdGF0aW9uX2F1dGhvcj1GcmllZG1hbiAgTS47IGNpdGF0aW9uX2F1dGhvcj1OZWFyICBULkouOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1Nb2wuIFBoeWxvZ2VuZXQuIEV2b2wuOyAgY2l0YXRpb25feWVhcj0yMDE1OyAgY2l0YXRpb25fdm9sdW1lPTg5OyAgY2l0YXRpb25fcGFnZXM9MjA1LTIxODsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QmF5ZXNpYW4gcmFuZG9tIGxvY2FsIGNsb2Nrcywgb3Igb25lIHJhdGUgdG8gcnVsZSB0aGVtIGFsbDsgY2l0YXRpb25fYXV0aG9yPURydW1tb25kICBBLkouOyBjaXRhdGlvbl9hdXRob3I9U3VjaGFyZCAgTS5BLjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9Qk1DIEJpb2wuOyAgY2l0YXRpb25feWVhcj0yMDEwOyAgY2l0YXRpb25fdm9sdW1lPTg7ICBjaXRhdGlvbl9wYWdlcz0xMTQiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1SZWxheGVkIHBoeWxvZ2VuZXRpY3MgYW5kIGRhdGluZyB3aXRoIGNvbmZpZGVuY2U7IGNpdGF0aW9uX2F1dGhvcj1EcnVtbW9uZCAgQS5KLjsgY2l0YXRpb25fYXV0aG9yPUhvICBTLlkuOyBjaXRhdGlvbl9hdXRob3I9UGhpbGxpcHMgIE0uSi47IGNpdGF0aW9uX2F1dGhvcj1SYW1iYXV0ICBBLjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9UExvUyBCaW9sLjsgIGNpdGF0aW9uX3llYXI9MjAwNjsgIGNpdGF0aW9uX3ZvbHVtZT00OyAgY2l0YXRpb25fcGFnZXM9ZTg4IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QSBtZXRob2QgZm9yIGludmVzdGlnYXRpbmcgcmVsYXRpdmUgdGltaW5nIGluZm9ybWF0aW9uIG9uIHBoeWxvZ2VuZXRpYyB0cmVlczsgY2l0YXRpb25fYXV0aG9yPUZvcmQgIEQuOyBjaXRhdGlvbl9hdXRob3I9R2VybmhhcmQgIFQuOyBjaXRhdGlvbl9hdXRob3I9TWF0c2VuICBFLjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9U3lzdC4gQmlvbC47ICBjaXRhdGlvbl95ZWFyPTIwMDk7ICBjaXRhdGlvbl92b2x1bWU9NTg7ICBjaXRhdGlvbl9wYWdlcz0xNjciIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1CYXllc2lhbiBpbmZlcmVuY2Ugb2Ygc2FtcGxlZCBhbmNlc3RvciB0cmVlcyBmb3IgZXBpZGVtaW9sb2d5IGFuZCBmb3NzaWwgY2FsaWJyYXRpb247IGNpdGF0aW9uX2F1dGhvcj1HYXZyeXVzaGtpbmEgIEEuOyBjaXRhdGlvbl9hdXRob3I9V2VsY2ggIEQuOyBjaXRhdGlvbl9hdXRob3I9U3RhZGxlciAgVC47IGNpdGF0aW9uX2F1dGhvcj1EcnVtbW9uZCAgQS47ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVBMb1MgQ29tcHV0LiBCaW9sLjsgIGNpdGF0aW9uX3llYXI9MjAxNDsgIGNpdGF0aW9uX3ZvbHVtZT0xMDsgIGNpdGF0aW9uX3BhZ2VzPWUxMDAzOTE5IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VGhlIGNvbmRpdGlvbmVkIHJlY29uc3RydWN0ZWQgcHJvY2VzczsgY2l0YXRpb25fYXV0aG9yPUdlcm5oYXJkICBULjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9Si4gVGhlb3JldC4gQmlvbC47ICBjaXRhdGlvbl95ZWFyPTIwMDg7ICBjaXRhdGlvbl92b2x1bWU9MjUzOyAgY2l0YXRpb25fcGFnZXM9NzY5LTc3ODsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9UmVhZGluZyB0aGUgZW50cmFpbHMgb2YgY2hpY2tlbnM6IE1vbGVjdWxhciB0aW1lc2NhbGVzIG9mIGV2b2x1dGlvbiBhbmQgdGhlIGlsbHVzaW9uIG9mIHByZWNpc2lvbjsgY2l0YXRpb25fYXV0aG9yPUdyYXVyICBELjsgY2l0YXRpb25fYXV0aG9yPU1hcnRpbiAgVy47ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVRyZW5kcyBHZW5ldC47ICBjaXRhdGlvbl95ZWFyPTIwMDQ7ICBjaXRhdGlvbl92b2x1bWU9MjA7ICBjaXRhdGlvbl9wYWdlcz04MC04NjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VXNpbmcgbW9yZSB0aGFuIHRoZSBvbGRlc3QgZm9zc2lsczogRGF0aW5nIG9zbXVuZGFjZWFlIHdpdGggdGhyZWUgQmF5ZXNpYW4gY2xvY2sgYXBwcm9hY2hlczsgY2l0YXRpb25fYXV0aG9yPUdyaW1tICBHLlcuOyBjaXRhdGlvbl9hdXRob3I9S2FwbGkgIFAuOyBjaXRhdGlvbl9hdXRob3I9Qm9tZmxldXIgIEIuOyBjaXRhdGlvbl9hdXRob3I9TWNMb3VnaGxpbiAgUy47IGNpdGF0aW9uX2F1dGhvcj1SZW5uZXIgIFMuUy47ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVN5c3QuIEJpb2wuOyAgY2l0YXRpb25feWVhcj0yMDE1OyAgY2l0YXRpb25fdm9sdW1lPTY0OyAgY2l0YXRpb25fcGFnZXM9Mzk2LTQwNTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RXN0aW1hdGlvbiBvZiBicmFuY2hpbmcgZGF0ZXMgYW1vbmcgcHJpbWF0ZXMgYnkgbW9sZWN1bGFyIGNsb2NrcyBvZiBudWNsZWFyIEROQSB3aGljaCBzbG93ZWQgZG93biBpbiBIb21pbm9pZGVhOyBjaXRhdGlvbl9hdXRob3I9SGFzZWdhd2EgIE0uOyBjaXRhdGlvbl9hdXRob3I9S2lzaGlubyAgSC47IGNpdGF0aW9uX2F1dGhvcj1ZYW5vICBULjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9Si4gSHVtYW4gRXZvbC47ICBjaXRhdGlvbl95ZWFyPTE5ODk7ICBjaXRhdGlvbl92b2x1bWU9MTg7ICBjaXRhdGlvbl9wYWdlcz00NjEtNDc2OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BIGhpZXJhcmNoaWNhbCBCYXllc2lhbiBtb2RlbCBmb3IgY2FsaWJyYXRpbmcgZXN0aW1hdGVzIG9mIHNwZWNpZXMgZGl2ZXJnZW5jZSB0aW1lczsgY2l0YXRpb25fYXV0aG9yPUhlYXRoICBULkEuOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1TeXN0LiBCaW9sLjsgIGNpdGF0aW9uX3llYXI9MjAxMjsgIGNpdGF0aW9uX3ZvbHVtZT02MTsgIGNpdGF0aW9uX3BhZ2VzPTc5My04MDk7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUEgRGlyaWNobGV0IHByb2Nlc3MgcHJpb3IgZm9yIGVzdGltYXRpbmcgbGluZWFnZS1zcGVjaWZpYyBzdWJzdGl0dXRpb24gcmF0ZXM7IGNpdGF0aW9uX2F1dGhvcj1IZWF0aCAgVC5BLjsgY2l0YXRpb25fYXV0aG9yPUhvbGRlciAgTS5ULjsgY2l0YXRpb25fYXV0aG9yPUh1ZWxzZW5iZWNrICBKLlAuOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1Nb2xlLiBCaW9sLiBFdm9sLjsgIGNpdGF0aW9uX3llYXI9MjAxMjsgIGNpdGF0aW9uX3ZvbHVtZT0yOTsgIGNpdGF0aW9uX3BhZ2VzPTkzOS0yNTU7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRoZSBmb3NzaWxpemVkIGJpcnRo4oCTZGVhdGggcHJvY2VzcyBmb3IgY29oZXJlbnQgY2FsaWJyYXRpb24gb2YgZGl2ZXJnZW5jZS10aW1lIGVzdGltYXRlczsgY2l0YXRpb25fYXV0aG9yPUhlYXRoICBULkEuOyBjaXRhdGlvbl9hdXRob3I9SHVlbHNlbmJlY2sgIEouUC47IGNpdGF0aW9uX2F1dGhvcj1TdGFkbGVyICBULjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9UHJvYy4gTmF0bCBBY2FkLiBTY2kuLCBVU0E7ICBjaXRhdGlvbl95ZWFyPTIwMTQ7ICBjaXRhdGlvbl92b2x1bWU9MTExOyAgY2l0YXRpb25fcGFnZXM9RTI5NTctRTI5NjY7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUJheWVzaWFuIGluZmVyZW5jZSBvZiBzcGVjaWVzIGRpdmVyZ2VuY2UgdGltZXM7IGNpdGF0aW9uX2F1dGhvcj1IZWF0aCAgVC5BLjsgY2l0YXRpb25fYXV0aG9yPU1vb3JlICBCLlIuOyBjaXRhdGlvbl9hdXRob3I9Q2hlbiAgTS4tSC47IGNpdGF0aW9uX2F1dGhvcj1LdW8gIEwuOyBjaXRhdGlvbl9hdXRob3I9TGV3aXMgIFAuTy47IGNpdGF0aW9uX3B1Ymxpc2hlcj1DaGFwbWFuICZhbXA7IEhhbGwvQ1JDLCBGbG9yaWRhLCBVUzsgIGNpdGF0aW9uX3RpdGxlPUJheWVzaWFuIHBoeWxvZ2VuZXRpY3M6IG1ldGhvZHMsIGFsZ29yaXRobXMsIGFuZCBhcHBsaWNhdGlvbnM7ICBjaXRhdGlvbl95ZWFyPTIwMTQ7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVByZWNpc2lvbiBvZiBtb2xlY3VsYXIgdGltZSBlc3RpbWF0ZXM7IGNpdGF0aW9uX2F1dGhvcj1IZWRnZXMgIFMuQi47IGNpdGF0aW9uX2F1dGhvcj1LdW1hciAgUy47ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVRyZW5kcyBHZW5ldC47ICBjaXRhdGlvbl95ZWFyPTIwMDQ7ICBjaXRhdGlvbl92b2x1bWU9MjA7ICBjaXRhdGlvbl9wYWdlcz0yNDItMjQ3OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BY2NvdW50aW5nIGZvciBjYWxpYnJhdGlvbiB1bmNlcnRhaW50eSBpbiBwaHlsb2dlbmV0aWMgZXN0aW1hdGlvbiBvZiBldm9sdXRpb25hcnkgZGl2ZXJnZW5jZSB0aW1lczsgY2l0YXRpb25fYXV0aG9yPUhvICBTLlkuVy47IGNpdGF0aW9uX2F1dGhvcj1QaGlsbGlwcyAgTS5KLjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9U3lzdC4gQmlvbC47ICBjaXRhdGlvbl95ZWFyPTIwMDk7ICBjaXRhdGlvbl92b2x1bWU9NTg7ICBjaXRhdGlvbl9wYWdlcz0zNjctMzgwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1JbmZlcnJpbmcgc3BlY2lhdGlvbiBhbmQgZXh0aW5jdGlvbiByYXRlcyB1bmRlciBkaWZmZXJlbnQgc2FtcGxpbmcgc2NoZW1lczsgY2l0YXRpb25fYXV0aG9yPUjDtmhuYSAgUy47IGNpdGF0aW9uX2F1dGhvcj1TdGFkbGVyICBULjsgY2l0YXRpb25fYXV0aG9yPVJvbnF1aXN0ICBGLjsgY2l0YXRpb25fYXV0aG9yPUJyaXR0b24gIFQuOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1Nb2xlLiBCaW9sLiBFdm9sLjsgIGNpdGF0aW9uX3llYXI9MjAxMTsgIGNpdGF0aW9uX3ZvbHVtZT0yODsgIGNpdGF0aW9uX3BhZ2VzPTI1NzctMjU4OTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QSBjb21wb3VuZCBQb2lzc29uIHByb2Nlc3MgZm9yIHJlbGF4aW5nIHRoZSBtb2xlY3VsYXIgY2xvY2s7IGNpdGF0aW9uX2F1dGhvcj1IdWVsc2VuYmVjayAgSi5QLjsgY2l0YXRpb25fYXV0aG9yPUxhcmdldCAgQi47IGNpdGF0aW9uX2F1dGhvcj1Td29mZm9yZCAgRC5MLjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9R2VuZXRpY3M7ICBjaXRhdGlvbl95ZWFyPTIwMDA7ICBjaXRhdGlvbl92b2x1bWU9MTU0OyAgY2l0YXRpb25fcGFnZXM9MTg3OS0xODkyOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Fdm9sdXRpb24gb2YgcHJvdGVpbiBtb2xlY3VsZXM7IGNpdGF0aW9uX2F1dGhvcj1KdWtlcyAgVC5ILjsgY2l0YXRpb25fYXV0aG9yPUNhbnRvciAgQy5SLjsgY2l0YXRpb25fYXV0aG9yPU11bnJvICBILk4uOyBjaXRhdGlvbl9wdWJsaXNoZXI9QWNhZGVtaWMgUHJlc3MsIE5ldyBZb3JrOyAgY2l0YXRpb25fdGl0bGU9TWFtbWFsaWFuIHByb3RlaW4gbWV0YWJvbGlzbTsgIGNpdGF0aW9uX3llYXI9MTk2OTsgIGNpdGF0aW9uX3BhZ2VzPTIxLTEyMzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9T24gdGhlIGdlbmVyYWxpemVkIOKAnGJpcnRoLWFuZC1kZWF0aOKAnSBwcm9jZXNzOyBjaXRhdGlvbl9hdXRob3I9S2VuZGFsbCAgRC5HLjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9QW5uYWxzIE1hdGguIFN0YXQuOyAgY2l0YXRpb25feWVhcj0xOTQ4OyAgY2l0YXRpb25fdm9sdW1lPTE5OyAgY2l0YXRpb25fcGFnZXM9MS0xNTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9TWF4aW11bSBsaWtlbGlob29kIGluZmVyZW5jZSBvZiBwcm90ZWluIHBoeWxvZ2VueSBhbmQgdGggb3JpZ2luIG9mIGNobG9yb3BsYXN0czsgY2l0YXRpb25fYXV0aG9yPUtpc2hpbm8gIEguOyBjaXRhdGlvbl9hdXRob3I9TWl5YXRhICBULjsgY2l0YXRpb25fYXV0aG9yPUhhc2VnYXdhICBNLjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9Si4gTW9sLiBFdm9sLjsgIGNpdGF0aW9uX3llYXI9MTk5MDsgIGNpdGF0aW9uX3ZvbHVtZT0zMTsgIGNpdGF0aW9uX3BhZ2VzPTE1MS0xNjA7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVBlcmZvcm1hbmNlIG9mIGEgZGl2ZXJnZW5jZSB0aW1lIGVzdGltYXRpb24gbWV0aG9kIHVuZGVyIGEgcHJvYmFiaWxpc3RpYyBtb2RlbCBvZiByYXRlIGV2b2x1dGlvbjsgY2l0YXRpb25fYXV0aG9yPUtpc2hpbm8gIEguOyBjaXRhdGlvbl9hdXRob3I9VGhvcm5lICBKLkwuOyBjaXRhdGlvbl9hdXRob3I9QnJ1bm8gIFcuOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1Nb2wuIEJpb2wuIEV2b2wuOyAgY2l0YXRpb25feWVhcj0yMDAxOyAgY2l0YXRpb25fdm9sdW1lPTE4OyAgY2l0YXRpb25fcGFnZXM9MzUyLTM2MTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Q29udmVyZ2VudCBpbnRyb24gZ2FpbnMgaW4gaHltZW5vcHRlcmFuIGVsb25nYXRpb24gZmFjdG9yLTHOsTsgY2l0YXRpb25fYXV0aG9yPUtsb3Bmc3RlaW4gIFMuOyBjaXRhdGlvbl9hdXRob3I9Um9ucXVpc3QgIEYuOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1Nb2wuIFBoeWxvZ2VuZXQuIEV2b2wuOyAgY2l0YXRpb25feWVhcj0yMDEzOyAgY2l0YXRpb25fdm9sdW1lPTY3OyAgY2l0YXRpb25fcGFnZXM9MjY2LTI3NjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QmlydGjigJNkZWF0aCBtb2RlbHMgYW5kIGNvYWxlc2NlbnQgcG9pbnQgcHJvY2Vzc2VzOiBUaGUgc2hhcGUgYW5kIHByb2JhYmlsaXR5IG9mIHJlY29uc3RydWN0ZWQgcGh5bG9nZW5pZXM7IGNpdGF0aW9uX2F1dGhvcj1MYW1iZXJ0ICBBLjsgY2l0YXRpb25fYXV0aG9yPVN0YWRsZXIgIFQuOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1UaGVvcmV0LiBQb3B1bC4gQmlvbC47ICBjaXRhdGlvbl95ZWFyPTIwMTM7ICBjaXRhdGlvbl92b2x1bWU9OTA7ICBjaXRhdGlvbl9wYWdlcz0xMTMtMTI4OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1QaHlsb2dlbmV0aWMgdW5jZXJ0YWludHkgYW5kIG1vbGVjdWxhciBjbG9jayBjYWxpYnJhdGlvbnM6IGEgY2FzZSBzdHVkeSBvZiBsZWdsZXNzIGxpemFyZHMgKFB5Z29wb2RpZGFlLCBHZWtrb3RhKTsgY2l0YXRpb25fYXV0aG9yPUxlZSAgTS5TLlkuOyBjaXRhdGlvbl9hdXRob3I9T2xpdmVyICBQLk0uOyBjaXRhdGlvbl9hdXRob3I9SHV0Y2hpbnNvbiAgTS5OLjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TW9sLiBQaHlsb2dlbmV0LiBFdm9sLjsgIGNpdGF0aW9uX3llYXI9MjAwOTsgIGNpdGF0aW9uX3ZvbHVtZT01MDsgIGNpdGF0aW9uX3BhZ2VzPTY2MS02NjY7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUNvbnRpbnVvdXMgYW5kIHRyYWN0YWJsZSBtb2RlbHMgZm9yIHRoZSB2YXJpYXRpb24gb2YgZXZvbHV0aW9uYXJ5IHJhdGVzOyBjaXRhdGlvbl9hdXRob3I9TGVwYWdlICBULjsgY2l0YXRpb25fYXV0aG9yPUxhd2kgIFMuOyBjaXRhdGlvbl9hdXRob3I9VHVwcGVyICBQLjsgY2l0YXRpb25fYXV0aG9yPUJyeWFudCAgRC47ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU1hdGguIEJpb3NjaS47ICBjaXRhdGlvbl95ZWFyPTIwMDY7ICBjaXRhdGlvbl92b2x1bWU9MTk5OyAgY2l0YXRpb25fcGFnZXM9MjE2LTIzMzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QSBnZW5lcmFsIGNvbXBhcmlzb24gb2YgcmVsYXhlZCBtb2xlY3VsYXIgY2xvY2sgbW9kZWxzOyBjaXRhdGlvbl9hdXRob3I9TGVwYWdlICBULjsgY2l0YXRpb25fYXV0aG9yPUJyeWFudCAgRC47IGNpdGF0aW9uX2F1dGhvcj1QaGlsaXBwZSAgSC47IGNpdGF0aW9uX2F1dGhvcj1MYXJ0aWxsb3QgIE4uOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1Nb2wuIEJpb2wuIEV2b2wuOyAgY2l0YXRpb25feWVhcj0yMDA3OyAgY2l0YXRpb25fdm9sdW1lPTI0OyAgY2l0YXRpb25fcGFnZXM9MjY2OS0yNjgwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BIGxpa2VsaWhvb2QgYXBwcm9hY2ggdG8gZXN0aW1hdGluZyBwaHlsb2dlbnkgZnJvbSBkaXNjcmV0ZSBtb3JwaG9sb2dpY2FsIGNoYXJhY3RlciBkYXRhOyBjaXRhdGlvbl9hdXRob3I9TGV3aXMgIFAuTy47ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVN5c3QuIEJpb2wuOyAgY2l0YXRpb25feWVhcj0yMDAxOyAgY2l0YXRpb25fdm9sdW1lPTUwOyAgY2l0YXRpb25fcGFnZXM9OTEzLTkyNTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9UG9seXRvbWllcyBhbmQgQmF5ZXNpYW4gcGh5bG9nZW5ldGljIGluZmVyZW5jZTsgY2l0YXRpb25fYXV0aG9yPUxld2lzICBQLjsgY2l0YXRpb25fYXV0aG9yPUhvbGRlciAgTS47IGNpdGF0aW9uX2F1dGhvcj1Ib2xzaW5nZXIgIEsuOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1TeXN0LiBCaW9sLjsgIGNpdGF0aW9uX3llYXI9MjAwNTsgIGNpdGF0aW9uX3ZvbHVtZT01NDsgIGNpdGF0aW9uX3BhZ2VzPTI0MS0yNTM7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVBoeWxvZ2Vub21pY3MgcmVzb2x2ZXMgdGhlIHRpbWluZyBhbmQgcGF0dGVybiBvZiBpbnNlY3QgZXZvbHV0aW9uOyBjaXRhdGlvbl9hdXRob3I9TWlzb2YgIEIuOyBjaXRhdGlvbl9hdXRob3I9TGl1ICBTLjsgY2l0YXRpb25fYXV0aG9yPU1ldXNlbWFubiAgSy47IGNpdGF0aW9uX2F1dGhvcj1QZXRlcnMgIFIuUy47IGNpdGF0aW9uX2F1dGhvcj1Eb25hdGggIEEuOyBjaXRhdGlvbl9hdXRob3I9TWF5ZXIgIEMuOyBjaXRhdGlvbl9hdXRob3I9RnJhbmRzZW4gIFAuQi47IGNpdGF0aW9uX2F1dGhvcj1XYXJlICBKLjsgY2l0YXRpb25fYXV0aG9yPUZsb3VyaSAgVC47IGNpdGF0aW9uX2F1dGhvcj1CZXV0ZWwgIFIuRy47IGNpdGF0aW9uX2F1dGhvcj1OaWVodWlzICBPLjsgY2l0YXRpb25fYXV0aG9yPVBldGVyc2VuICBNLjsgY2l0YXRpb25fYXV0aG9yPUl6cXVpZXJkby1DYXJyYXNjbyAgRi47IGNpdGF0aW9uX2F1dGhvcj1XYXBwbGVyICBULjsgY2l0YXRpb25fYXV0aG9yPVJ1c3QgIEouOyBjaXRhdGlvbl9hdXRob3I9QWJlcmVyICBBLkouOyBjaXRhdGlvbl9hdXRob3I9QXNww7ZjayAgVS47IGNpdGF0aW9uX2F1dGhvcj1Bc3DDtmNrICBILjsgY2l0YXRpb25fYXV0aG9yPUJhcnRlbCAgRC47IGNpdGF0aW9uX2F1dGhvcj1CbGFua2UgIEEuOyBjaXRhdGlvbl9hdXRob3I9QmVyZ2VyICBTLjsgY2l0YXRpb25fYXV0aG9yPULDtmhtICBBLjsgY2l0YXRpb25fYXV0aG9yPUJ1Y2tsZXkgIFQuUi47IGNpdGF0aW9uX2F1dGhvcj1DYWxjb3R0ICBCLjsgY2l0YXRpb25fYXV0aG9yPUNoZW4gIEouOyBjaXRhdGlvbl9hdXRob3I9RnJpZWRyaWNoICBGLjsgY2l0YXRpb25fYXV0aG9yPUZ1a3VpICBNLjsgY2l0YXRpb25fYXV0aG9yPUZ1aml0YSAgTS47IGNpdGF0aW9uX2F1dGhvcj1HcmV2ZSAgQy47IGNpdGF0aW9uX2F1dGhvcj1Hcm9iZSAgUC47IGNpdGF0aW9uX2F1dGhvcj1HdSAgUy47IGNpdGF0aW9uX2F1dGhvcj1IdWFuZyAgWS47IGNpdGF0aW9uX2F1dGhvcj1KZXJtaWluICBMLlMuOyBjaXRhdGlvbl9hdXRob3I9S2F3YWhhcmEgIEEuWS47IGNpdGF0aW9uX2F1dGhvcj1Lcm9nbWFubiAgTC47IGNpdGF0aW9uX2F1dGhvcj1LdWJpYWsgIE0uOyBjaXRhdGlvbl9hdXRob3I9TGFuZmVhciAgUi47IGNpdGF0aW9uX2F1dGhvcj1MZXRzY2ggIEguOyBjaXRhdGlvbl9hdXRob3I9TGkgIFkuOyBjaXRhdGlvbl9hdXRob3I9TGkgIFouOyBjaXRhdGlvbl9hdXRob3I9TGkgIEouOyBjaXRhdGlvbl9hdXRob3I9THUgIEguOyBjaXRhdGlvbl9hdXRob3I9TWFjaGlkYSAgUi47IGNpdGF0aW9uX2F1dGhvcj1NYXNoaW1vICBZLjsgY2l0YXRpb25fYXV0aG9yPUthcGxpICBQLjsgY2l0YXRpb25fYXV0aG9yPU1jS2VubmEgIEQuRC47IGNpdGF0aW9uX2F1dGhvcj1NZW5nICBHLjsgY2l0YXRpb25fYXV0aG9yPU5ha2FnYWtpICBZLjsgY2l0YXRpb25fYXV0aG9yPU5hdmFycmV0ZS1IZXJlZGlhICBKLkwuOyBjaXRhdGlvbl9hdXRob3I9T3R0ICBNLjsgY2l0YXRpb25fYXV0aG9yPU91ICBZLjsgY2l0YXRpb25fYXV0aG9yPVBhc3MgIEcuOyBjaXRhdGlvbl9hdXRob3I9UG9kc2lhZGxvd3NraSAgTC47IGNpdGF0aW9uX2F1dGhvcj1Qb2hsICBILjsgY2l0YXRpb25fYXV0aG9yPVJldW1vbnQgdm9uICBCLk0uOyBjaXRhdGlvbl9hdXRob3I9U2Now7x0dGUgIEsuOyBjaXRhdGlvbl9hdXRob3I9U2VraXlhICBLLjsgY2l0YXRpb25fYXV0aG9yPVNoaW1penUgIFMuOyBjaXRhdGlvbl9hdXRob3I9U2xpcGluc2tpICBBLjsgY2l0YXRpb25fYXV0aG9yPVN0YW1hdGFraXMgIEEuOyBjaXRhdGlvbl9hdXRob3I9U29uZyAgVy47IGNpdGF0aW9uX2F1dGhvcj1TdSAgWC47IGNpdGF0aW9uX2F1dGhvcj1TenVjc2ljaCAgTi5VLjsgY2l0YXRpb25fYXV0aG9yPVRhbiAgTS47IGNpdGF0aW9uX2F1dGhvcj1UYW4gIFguOyBjaXRhdGlvbl9hdXRob3I9VGFuZyAgTS47IGNpdGF0aW9uX2F1dGhvcj1UYW5nICBKLjsgY2l0YXRpb25fYXV0aG9yPVRpbWVsdGhhbGVyICBHLjsgY2l0YXRpb25fYXV0aG9yPVRvbWl6dWthICBTLjsgY2l0YXRpb25fYXV0aG9yPVRyYXV0d2VpbiAgTS47IGNpdGF0aW9uX2F1dGhvcj1Ub25nICBYLjsgY2l0YXRpb25fYXV0aG9yPVVjaGlmdW5lICBULjsgY2l0YXRpb25fYXV0aG9yPVdhbHpsICBNLkcuOyBjaXRhdGlvbl9hdXRob3I9V2llZ21hbm4gIEIuTS47IGNpdGF0aW9uX2F1dGhvcj1XaWxicmFuZHQgIEouOyBjaXRhdGlvbl9hdXRob3I9V2lwZmxlciAgQi47IGNpdGF0aW9uX2F1dGhvcj1Xb25nICBULksuRi47IGNpdGF0aW9uX2F1dGhvcj1XdSAgUS47IGNpdGF0aW9uX2F1dGhvcj1XdSAgRy47IGNpdGF0aW9uX2F1dGhvcj1YaWUgIFkuOyBjaXRhdGlvbl9hdXRob3I9WWFuZyAgUy47IGNpdGF0aW9uX2F1dGhvcj1ZYW5nICBRLjsgY2l0YXRpb25fYXV0aG9yPVllYXRlcyAgRC5LLjsgY2l0YXRpb25fYXV0aG9yPVlvc2hpemF3YSAgSy47IGNpdGF0aW9uX2F1dGhvcj1aaGFuZyAgUS47IGNpdGF0aW9uX2F1dGhvcj1aaGFuZyAgUi47IGNpdGF0aW9uX2F1dGhvcj1aaGFuZyAgVy47IGNpdGF0aW9uX2F1dGhvcj1aaGFuZyAgWS47IGNpdGF0aW9uX2F1dGhvcj1aaGFvICBKLjsgY2l0YXRpb25fYXV0aG9yPVpob3UgIEMuOyBjaXRhdGlvbl9hdXRob3I9WmhvdSAgTC47IGNpdGF0aW9uX2F1dGhvcj1aaWVzbWFubiAgVC47IGNpdGF0aW9uX2F1dGhvcj1ab3UgIFMuOyBjaXRhdGlvbl9hdXRob3I9TGkgIFkuOyBjaXRhdGlvbl9hdXRob3I9WHUgIFguOyBjaXRhdGlvbl9hdXRob3I9WmhhbmcgIFkuOyBjaXRhdGlvbl9hdXRob3I9WWFuZyAgSC47IGNpdGF0aW9uX2F1dGhvcj1XYW5nICBKLjsgY2l0YXRpb25fYXV0aG9yPVdhbmcgIEouOyBjaXRhdGlvbl9hdXRob3I9S2plciAgSy5NLjsgY2l0YXRpb25fYXV0aG9yPVpob3UgIFguOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1TY2llbmNlOyAgY2l0YXRpb25feWVhcj0yMDE0OyAgY2l0YXRpb25fdm9sdW1lPTM0NjsgIGNpdGF0aW9uX3BhZ2VzPTc2My03Njc7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRoZSByZWNvbnN0cnVjdGVkIGV2b2x1dGlvbmFyeSBwcm9jZXNzOyBjaXRhdGlvbl9hdXRob3I9TmVlICBTLjsgY2l0YXRpb25fYXV0aG9yPU1heSAgUi5NLjsgY2l0YXRpb25fYXV0aG9yPUhhcnZleSAgUC5ILjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9UGhpbG8uIFRyYW5zLiBSLiBTb2MuIEI7ICBjaXRhdGlvbl95ZWFyPTE5OTQ7ICBjaXRhdGlvbl92b2x1bWU9MzQ0OyAgY2l0YXRpb25fcGFnZXM9MzA1LTMxMTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RGl2ZXJnZW5jZSB0aW1lIGVzdGltYXRpb24gdXNpbmcgZm9zc2lscyBhcyB0ZXJtaW5hbCB0YXhhIGFuZCB0aGUgb3JpZ2lucyBvZiBMaXNzYW1waGliaWE7IGNpdGF0aW9uX2F1dGhvcj1QeXJvbiAgUi5BLjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9U3lzdC4gQmlvbC47ICBjaXRhdGlvbl95ZWFyPTIwMTE7ICBjaXRhdGlvbl92b2x1bWU9NjA7ICBjaXRhdGlvbl9wYWdlcz00NjYtNDgxOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl9hdXRob3I9UiBEZXZlbG9wbWVudCBDb3JlIFRlYW07IGNpdGF0aW9uX3B1Ymxpc2hlcj1SIEZvdW5kYXRpb24gZm9yIFN0YXRpc3RpY2FsIENvbXB1dGluZywgVmllbm5hLCBBdXN0cmlhOyAgY2l0YXRpb25fdGl0bGU9UjogYSBsYW5ndWFnZSBhbmQgZW52aXJvbm1lbnQgZm9yIHN0YXRpc3RpY2FsIGNvbXB1dGluZzsgIGNpdGF0aW9uX3llYXI9MjAwOTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9U2VxLUdlbjogYW4gYXBwbGljYXRpb24gZm9yIHRoZSBNb250ZSBDYXJsbyBzaW11bGF0aW9uIG9mIEROQSBzZXF1ZW5jZSBldm9sdXRpb24gYWxvbmcgcGh5bG9nZW5ldGljIHRyZWVzOyBjaXRhdGlvbl9hdXRob3I9UmFtYmF1dCAgQS47IGNpdGF0aW9uX2F1dGhvcj1HcmFzcyAgTi5DLjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9Q29tcHV0LiBBcHBsLiBCaW9zY2kuOyAgY2l0YXRpb25feWVhcj0xOTk3OyAgY2l0YXRpb25fdm9sdW1lPTEzOyAgY2l0YXRpb25fcGFnZXM9MjM1LTIzODsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9SW5mZXJyaW5nIHNwZWNpYXRpb24gdGltZXMgdW5kZXIgYW4gZXBpc29kaWMgbW9sZWN1bGFyIGNsb2NrOyBjaXRhdGlvbl9hdXRob3I9UmFubmFsYSAgQi47IGNpdGF0aW9uX2F1dGhvcj1ZYW5nICBaLjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9U3lzdC4gQmlvbC47ICBjaXRhdGlvbl95ZWFyPTIwMDc7ICBjaXRhdGlvbl92b2x1bWU9NTY7ICBjaXRhdGlvbl9wYWdlcz00NTMtNDY2OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BbiBvdXRsaW5lIG9mIGV2b2x1dGlvbiBvZiB0aGUgaHltZW5vcHRlcm91cyBpbnNlY3RzIChvcmRlciB2ZXNwaWRhKTsgY2l0YXRpb25fYXV0aG9yPVJhc25pdHN5biAgQS5QLjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9T3JpZW50LiBJbnNlY3RzOyAgY2l0YXRpb25feWVhcj0xOTg4OyAgY2l0YXRpb25fdm9sdW1lPTIyOyAgY2l0YXRpb25fcGFnZXM9MTE1LTE0NTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9TW9sZWN1bGFyIHBoeWxvZ2VuZXRpY3MsIG1vcnBob2xvZ2ljYWwgY2xhZGlzdGljcywgYW5kIGZvc3NpbCByZWNvcmQ7IGNpdGF0aW9uX2F1dGhvcj1SYXNuaXRzeW4gIEEuUC47ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUVudG9tb2wuIFJldi47ICBjaXRhdGlvbl95ZWFyPTIwMTA7ICBjaXRhdGlvbl92b2x1bWU9OTA7ICBjaXRhdGlvbl9wYWdlcz0yNjMtMjk4OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1NcmJheWVzIDMuMjogRWZmaWNpZW50IEJheWVzaWFuIHBoeWxvZ2VuZXRpYyBpbmZlcmVuY2UgYW5kIG1vZGVsIGNob2ljZSBhY3Jvc3MgYSBsYXJnZSBtb2RlbCBzcGFjZTsgY2l0YXRpb25fYXV0aG9yPVJvbnF1aXN0ICBGLjsgY2l0YXRpb25fYXV0aG9yPVRlc2xlbmtvICBNLjsgY2l0YXRpb25fYXV0aG9yPXZhbiBkZXIgTWFyayAgUC47IGNpdGF0aW9uX2F1dGhvcj1BeXJlcyAgRC5MLjsgY2l0YXRpb25fYXV0aG9yPURhcmxpbmcgIEEuOyBjaXRhdGlvbl9hdXRob3I9SMO2aG5hICBTLjsgY2l0YXRpb25fYXV0aG9yPUxhcmdldCAgQi47IGNpdGF0aW9uX2F1dGhvcj1MaXUgIEwuOyBjaXRhdGlvbl9hdXRob3I9U3VjaGFyZCAgTS5BLjsgY2l0YXRpb25fYXV0aG9yPUh1ZWxzZW5iZWNrICBKLlAuOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1TeXN0LiBCaW9sLjsgIGNpdGF0aW9uX3llYXI9MjAxMmE7ICBjaXRhdGlvbl92b2x1bWU9NjE7ICBjaXRhdGlvbl9wYWdlcz01MzktNTQyOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BIHRvdGFsLWV2aWRlbmNlIGFwcHJvYWNoIHRvIGRhdGluZyB3aXRoIGZvc3NpbHMsIGFwcGxpZWQgdG8gdGhlIGVhcmx5IHJhZGlhdGlvbiBvZiB0aGUgSHltZW5vcHRlcmE7IGNpdGF0aW9uX2F1dGhvcj1Sb25xdWlzdCAgRi47IGNpdGF0aW9uX2F1dGhvcj1LbG9wZnN0ZWluICBTLjsgY2l0YXRpb25fYXV0aG9yPVZpbGhlbG1zZW4gIEwuOyBjaXRhdGlvbl9hdXRob3I9U2NodWxtZWlzdGVyICBTLjsgY2l0YXRpb25fYXV0aG9yPU11cnJheSAgRC5MLjsgY2l0YXRpb25fYXV0aG9yPVJhc25pdHN5biAgQS5QLjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9U3lzdC4gQmlvbC47ICBjaXRhdGlvbl95ZWFyPTIwMTJiOyAgY2l0YXRpb25fdm9sdW1lPTYxOyAgY2l0YXRpb25fcGFnZXM9OTczLTk5OTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9T24gaW5jb21wbGV0ZSBzYW1wbGluZyB1bmRlciBiaXJ0aOKAk2RlYXRoIG1vZGVscyBhbmQgY29ubmVjdGlvbnMgdG8gdGhlIHNhbXBsaW5nLWJhc2VkIGNvYWxlc2NlbnQ7IGNpdGF0aW9uX2F1dGhvcj1TdGFkbGVyICBULjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9Si4gVGhlb3JldC4gQmlvbC47ICBjaXRhdGlvbl95ZWFyPTIwMDk7ICBjaXRhdGlvbl92b2x1bWU9MjYxOyAgY2l0YXRpb25fcGFnZXM9NTgtNjY7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVNhbXBsaW5nLXRocm91Z2gtdGltZSBpbiBiaXJ0aOKAk2RlYXRoIHRyZWVzOyBjaXRhdGlvbl9hdXRob3I9U3RhZGxlciAgVC47ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUouIFRoZW9yZXQuIEJpb2wuOyAgY2l0YXRpb25feWVhcj0yMDEwYTsgIGNpdGF0aW9uX3ZvbHVtZT0yNjc7ICBjaXRhdGlvbl9wYWdlcz0zOTYtNDA0OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSIgU3RhZGxlciBULiAyMDEwYi4gVHJlZVNpbTogU2ltdWxhdGluZyB0cmVlcyB1bmRlciB0aGUgYmlydGjigJNkZWF0aCBtb2RlbC4gUiBwYWNrYWdlIHZlcnNpb24gMS4wLiAgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9TWFtbWFsaWFuIHBoeWxvZ2VueSByZXZlYWxzIHJlY2VudCBkaXZlcnNpZmljYXRpb24gcmF0ZSBzaGlmdHM7IGNpdGF0aW9uX2F1dGhvcj1TdGFkbGVyICBULjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9UHJvYy4gTmF0bC4gQWNhZC4gU2NpLiBVU0E7ICBjaXRhdGlvbl95ZWFyPTIwMTE7ICBjaXRhdGlvbl92b2x1bWU9MTA4OyAgY2l0YXRpb25fcGFnZXM9NjE4Ny02MTkyOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Ib3cgY2FuIHdlIGltcHJvdmUgYWNjdXJhY3kgb2YgbWFjcm9ldm9sdXRpb25hcnkgcmF0ZSBlc3RpbWF0ZXM/IFN5c3Q7IGNpdGF0aW9uX2F1dGhvcj1TdGFkbGVyICBULjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9QmlvbC47ICBjaXRhdGlvbl95ZWFyPTIwMTI7ICBjaXRhdGlvbl92b2x1bWU9NjI7ICBjaXRhdGlvbl9wYWdlcz0zMjEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Fc3RpbWF0aW5nIHNwZWNpYXRpb24gYW5kIGV4dGluY3Rpb24gcmF0ZXMgZm9yIHBoeWxvZ2VuaWVzIG9mIGhpZ2hlciB0YXhhOyBjaXRhdGlvbl9hdXRob3I9U3RhZGxlciAgVC47IGNpdGF0aW9uX2F1dGhvcj1Cb2ttYSAgRi47ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVN5c3QuIEJpb2wuOyAgY2l0YXRpb25feWVhcj0yMDEzOyAgY2l0YXRpb25fdm9sdW1lPTYyOyAgY2l0YXRpb25fcGFnZXM9MjIwLTIzMDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QmlydGjigJNkZWF0aCBza3lsaW5lIHBsb3QgcmV2ZWFscyB0ZW1wb3JhbCBjaGFuZ2VzIG9mIGVwaWRlbWljIHNwcmVhZCBpbiBISVYgYW5kIGhlcGF0aXRpcyBDIHZpcnVzIChIQ1YpOyBjaXRhdGlvbl9hdXRob3I9U3RhZGxlciAgVC47IGNpdGF0aW9uX2F1dGhvcj1Lw7xobmVydCAgRC47IGNpdGF0aW9uX2F1dGhvcj1Cb25ob2VmZmVyICBTLjsgY2l0YXRpb25fYXV0aG9yPURydW1tb25kICBBLkouOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1Qcm9jLiBOYXRsIEFjYWQuIFNjaS4gVVNBOyAgY2l0YXRpb25feWVhcj0yMDEzOyAgY2l0YXRpb25fdm9sdW1lPTExMDsgIGNpdGF0aW9uX3BhZ2VzPTIyOC0yMzM7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVVzaW5nIHRoZSBmb3NzaWwgcmVjb3JkIHRvIGVzdGltYXRlIHRoZSBhZ2Ugb2YgdGhlIGxhc3QgY29tbW9uIGFuY2VzdG9yIG9mIGV4dGFudCBwcmltYXRlczsgY2l0YXRpb25fYXV0aG9yPVRhdmFyw6kgIFMuOyBjaXRhdGlvbl9hdXRob3I9TWFyc2hhbGwgIEMuUi47IGNpdGF0aW9uX2F1dGhvcj1XaWxsICBPLjsgY2l0YXRpb25fYXV0aG9yPVNvbGlnbyAgQy47IGNpdGF0aW9uX2F1dGhvcj1NYXJ0aW4gIFIuRC47ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU5hdHVyZTsgIGNpdGF0aW9uX3llYXI9MjAwMjsgIGNpdGF0aW9uX3ZvbHVtZT00MTY7ICBjaXRhdGlvbl9wYWdlcz03MjYtNzI5OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1EaXZlcmdlbmNlIHRpbWUgYW5kIGV2b2x1dGlvbmFyeSByYXRlIGVzdGltYXRpb24gd2l0aCBtdWx0aWxvY3VzIGRhdGE7IGNpdGF0aW9uX2F1dGhvcj1UaG9ybmUgIEouOyBjaXRhdGlvbl9hdXRob3I9S2lzaGlubyAgSC47ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVN5c3QuIEJpb2wuOyAgY2l0YXRpb25feWVhcj0yMDAyOyAgY2l0YXRpb25fdm9sdW1lPTUxOyAgY2l0YXRpb25fcGFnZXM9Njg5LTcwMjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RXN0aW1hdGluZyB0aGUgcmF0ZSBvZiBldm9sdXRpb24gb2YgdGhlIHJhdGUgb2YgbW9sZWN1bGFyIGV2b2x1dGlvbjsgY2l0YXRpb25fYXV0aG9yPVRob3JuZSAgSi47IGNpdGF0aW9uX2F1dGhvcj1LaXNoaW5vICBILjsgY2l0YXRpb25fYXV0aG9yPVBhaW50ZXIgIEkuUy47ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU1vbC4gQmlvbC4gRXZvbC47ICBjaXRhdGlvbl95ZWFyPTE5OTg7ICBjaXRhdGlvbl92b2x1bWU9MTU7ICBjaXRhdGlvbl9wYWdlcz0xNjQ3LTE2NTc7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPURhdGluZyBwcmltYXRlIGRpdmVyZ2VuY2VzIHRocm91Z2ggYW4gaW50ZWdyYXRlZCBhbmFseXNpcyBvZiBwYWxhZW9udG9sb2dpY2FsIGFuZCBtb2xlY3VsYXIgZGF0YTsgY2l0YXRpb25fYXV0aG9yPVdpbGtpbnNvbiAgUi5ELjsgY2l0YXRpb25fYXV0aG9yPVN0ZWlwZXIgIE0uRS47IGNpdGF0aW9uX2F1dGhvcj1Tb2xpZ28gIEMuOyBjaXRhdGlvbl9hdXRob3I9TWFydGluICBSLkQuOyBjaXRhdGlvbl9hdXRob3I9WWFuZyAgWi47IGNpdGF0aW9uX2F1dGhvcj1UYXZhcsOpICBTLjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9U3lzdC4gQmlvbC47ICBjaXRhdGlvbl95ZWFyPTIwMTE7ICBjaXRhdGlvbl92b2x1bWU9NjA7ICBjaXRhdGlvbl9wYWdlcz0xNi0zMTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VHJlYXRpbmcgZm9zc2lscyBhcyB0ZXJtaW5hbCB0YXhhIGluIGRpdmVyZ2VuY2UgdGltZSBlc3RpbWF0aW9uIHJldmVhbHMgYW5jaWVudCB2aWNhcmlhbmNlIHBhdHRlcm5zIGluIHRoZSBwYWxwaW1hbm9pZCBzcGlkZXJzOyBjaXRhdGlvbl9hdXRob3I9V29vZCAgSC5NLjsgY2l0YXRpb25fYXV0aG9yPU1hdHprZSAgTi5KLjsgY2l0YXRpb25fYXV0aG9yPUdpbGxlc3BpZSAgUi5HLjsgY2l0YXRpb25fYXV0aG9yPUdyaXN3b2xkICBDLkUuOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1TeXN0LiBCaW9sLjsgIGNpdGF0aW9uX3llYXI9MjAxMzsgIGNpdGF0aW9uX3ZvbHVtZT02MjsgIGNpdGF0aW9uX3BhZ2VzPTI2NC0yODQ7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUltcHJvdmluZyBtYXJnaW5hbCBsaWtlbGlob29kIGVzdGltYXRpb24gZm9yIEJheWVzaWFuIHBoeWxvZ2VuZXRpYyBtb2RlbCBzZWxlY3Rpb247IGNpdGF0aW9uX2F1dGhvcj1YaWUgIFcuOyBjaXRhdGlvbl9hdXRob3I9TGV3aXMgIFAuTy47IGNpdGF0aW9uX2F1dGhvcj1GYW4gIFkuOyBjaXRhdGlvbl9hdXRob3I9S3VvICBMLjsgY2l0YXRpb25fYXV0aG9yPUNoZW4gIE0uLUguOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1TeXN0LiBCaW9sLjsgIGNpdGF0aW9uX3llYXI9MjAxMTsgIGNpdGF0aW9uX3ZvbHVtZT02MDsgIGNpdGF0aW9uX3BhZ2VzPTE1MC0xNjA7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUJheWVzaWFuIHBoeWxvZ2VuZXRpYyBpbmZlcmVuY2UgdXNpbmcgRE5BIHNlcXVlbmNlczogYSBNYXJrb3YgY2hhaW4gTW9udGUgQ2FybG8gbWV0aG9kOyBjaXRhdGlvbl9hdXRob3I9WWFuZyAgWi47IGNpdGF0aW9uX2F1dGhvcj1SYW5uYWxhICBCLjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TW9sLiBCaW9sLiBFdm9sLjsgIGNpdGF0aW9uX3llYXI9MTk5NzsgIGNpdGF0aW9uX3ZvbHVtZT0xNDsgIGNpdGF0aW9uX3BhZ2VzPTcxNy03MjQ7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUJheWVzaWFuIGVzdGltYXRpb24gb2Ygc3BlY2llcyBkaXZlcmdlbmNlIHRpbWVzIHVuZGVyIGEgbW9sZWN1bGFyIGNsb2NrIHVzaW5nIG11bHRpcGxlIGZvc3NpbCBjYWxpYnJhdGlvbnMgd2l0aCBzb2Z0IGJvdW5kczsgY2l0YXRpb25fYXV0aG9yPVlhbmcgIFouOyBjaXRhdGlvbl9hdXRob3I9UmFubmFsYSAgQi47ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU1vbC4gQmlvbC4gRXZvbC47ICBjaXRhdGlvbl95ZWFyPTIwMDY7ICBjaXRhdGlvbl92b2x1bWU9MjM7ICBjaXRhdGlvbl9wYWdlcz0yMTItMjI2OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Db21wYXJpc29uIG9mIGxpa2VsaWhvb2QgYW5kIEJheWVzaWFuIG1ldGhvZHMgZm9yIGVzdGltYXRpbmcgZGl2ZXJnZW5jZSB0aW1lcyB1c2luZyBtdWx0aXBsZSBnZW5lIGxvY2kgYW5kIGNhbGlicmF0aW9uIHBvaW50cywgd2l0aCBhcHBsaWNhdGlvbiB0byBhIHJhZGlhdGlvbiBvZiBjdXRlLWxvb2tpbmcgbW91c2UgbGVtdXIgc3BlY2llczsgY2l0YXRpb25fYXV0aG9yPVlhbmcgIFouOyBjaXRhdGlvbl9hdXRob3I9WW9kZXIgIEEuRC47ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVN5c3QuIEJpb2wuOyAgY2l0YXRpb25feWVhcj0yMDAzOyAgY2l0YXRpb25fdm9sdW1lPTUyOyAgY2l0YXRpb25fcGFnZXM9NzA1LTcxNjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RXN0aW1hdGlvbiBvZiBwcmltYXRlIHNwZWNpYXRpb24gZGF0ZXMgdXNpbmcgbG9jYWwgbW9sZWN1bGFyIGNsb2NrczsgY2l0YXRpb25fYXV0aG9yPVlvZGVyICBBLkQuOyBjaXRhdGlvbl9hdXRob3I9WWFuZyAgWi47ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU1vbC4gQmlvbC4gRXZvbC47ICBjaXRhdGlvbl95ZWFyPTIwMDA7ICBjaXRhdGlvbl92b2x1bWU9MTc7ICBjaXRhdGlvbl9wYWdlcz0xMDgxLTEwOTA7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9IiBaaGFuZyBDLiAyMDE1LiBBIHR1dG9yaWFsIG9uIG1vbGVjdWxhciBjbG9jayBkYXRpbmcgdXNpbmcgTXJCYXllcyBodHRwczovL3NpdGVzLmdvb2dsZS5jb20vc2l0ZS96aGFuZ2NoaWNvb2wvICwgbGFzdCBhY2Nlc3NlZCBPY3RvYmVyIDMwLCAyMDE1LiAgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RXZvbHV0aW9uYXJ5IGRpdmVyZ2VuY2UgYW5kIGNvbnZlcmdlbmNlIGluIHByb3RlaW5zOyBjaXRhdGlvbl9hdXRob3I9WnVja2Vya2FuZGwgIEUuOyBjaXRhdGlvbl9hdXRob3I9UGF1bGluZyAgTC47IGNpdGF0aW9uX2F1dGhvcj1Ccnlzb24gIFYuOyBjaXRhdGlvbl9hdXRob3I9Vm9nZWwgIEguSi47IGNpdGF0aW9uX3B1Ymxpc2hlcj1BY2FkZW1pYyBQcmVzcywgTmV3IFlvcms7ICBjaXRhdGlvbl90aXRsZT1Fdm9sLiBHZW5lcyBQcm90ZWluczsgIGNpdGF0aW9uX3llYXI9MTk2NTsgIGNpdGF0aW9uX3BhZ2VzPTk3LTE2NjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2Z1bGx0ZXh0X3dvcmxkX3JlYWRhYmxlIiBjb250ZW50PSIiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcGRmX3VybCIgY29udGVudD0iaHR0cHM6Ly9hY2FkZW1pYy5vdXAuY29tL3N5c2Jpby9hcnRpY2xlLXBkZi82NS8yLzIyOC8yNDU4OTAyOC9zeXYwODAucGRmIiAvPjxtZXRhIG5hbWU9ImRlc2NyaXB0aW9uIiBjb250ZW50PSJBYnN0cmFjdC4gIEJheWVzaWFuIHRvdGFsLWV2aWRlbmNlIGRhdGluZyBpbnZvbHZlcyB0aGUgc2ltdWx0YW5lb3VzIGFuYWx5c2lzIG9mIG1vcnBob2xvZ2ljYWwgZGF0YSBmcm9tIHRoZSBmb3NzaWwgcmVjb3JkIGFuZCBtb3JwaG9sb2dpY2FsIGFuZCBzZXF1ZW5jZSBkYXRhIGZyIiAvPmQCBA9kFgJmD2QWAmYPFgQeBGhyZWYFcy8vb3VwLnNpbHZlcmNoYWlyLWNkbi5jb20vRG93bmxvYWRGaWxlL0Rvd25sb2FkSW1hZ2UuYXNweD9pbWFnZT0mUFBUdHlwZT1TbGlkZVNldCZhcj0yNDI3MTY0JnhzbHRQYXRoPX4vVUkvYXBwL1hTTFQeCWlubmVyaHRtbAUURG93bmxvYWQgYWxsIGZpZ3VyZXNkZNlcXDRF3+QpTQRnEghXO1T293AJ" type="hidden">
</div>

<div class="aspNetHidden">

	<input name="__VIEWSTATEGENERATOR" id="__VIEWSTATEGENERATOR" value="2173C2F0" type="hidden">
</div>
        </form>
        
        <input id="hdnSiteID" value="5349" type="hidden">
        <input id="hdnAdDelaySeconds" value="5000" type="hidden">
        <input id="hdnAdConfigurationTop" value="default" type="hidden">
        <input id="hdnAdConfigurationRightRail" value="sticky" type="hidden">

        

        
    <div class="master-container">
        <section class="master-header row">
                <div class="widget widget-SitePageHeader widget-instance-SitePageHeader">
        
    <div class="ad-banner js-ad-banner-header">
    <div class="widget widget-AdBlock widget-instance-HeaderAd">
        

<div id="adBlockHeader" style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockHeader'); });
    </script>
<div id="google_ads_iframe_/116097782/sysbio_Behind_Ad1_0__container__" style="border: 0pt none;"><iframe id="google_ads_iframe_/116097782/sysbio_Behind_Ad1_0" title="3rd party ad content" name="google_ads_iframe_/116097782/sysbio_Behind_Ad1_0" scrolling="no" marginwidth="0" marginheight="0" style="border: 0px none; vertical-align: bottom;" srcdoc="" height="90" frameborder="0" width="970"></iframe></div></div>

    </div>

    </div>

    <div class="oup-header">
        <div class="center-inner-row">

            <div class="oup-header-logo">
                <a href="https://academic.oup.com/"><img src="oxford-academic-logo.svg" alt="Oxford University Press" class="oup-header-image"></a>
            </div><!-- /.oup-header-logo -->

            <ul class="oup-header-menu account-menu">

                <!-- MOBILE SEARCH  -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-search-toggle">
                        <i class="icon-menu_search"><span class="screenreader-text">Search</span></i>
                    </button>
                </li>

                <!-- MOBILE MENU -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-account-toggle">
                        <i class="icon-menu_account"><span class="screenreader-text">Account Menu</span></i>
                    </button>
                </li>

                <!-- MOBILE NAV -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-nav-toggle">
                        <i class="icon-menu_hamburger"><span class="screenreader-text">Menu</span></i>
                    </button>
                </li>

                <!-- DESKTOP INSTITUTIONS -->
                    <li class="oup-header-menu-item desktop">
                        
                            <a href="" class="dropdown-toggle institution">Akademicka Knihovna JU <i class="icon-general-arrow-filled-down arrow-icon"></i></a>
                        <div class="dropdown-panel dropdown-panel-institution">
                            
                            <div class="dropdown-menu">
                                <ul>

                                        <li class="dropdown-institution-item">

Akademicka Knihovna JU
                                        </li>

                                </ul>

                            </div><!-- /.dropdown-menu -->
                        </div><!-- /.dropdown-panel .dropdown-panel-institution -->
                    </li><!-- /.oup-header-menu-item .desktop -->

                <!-- DESKTOP SIGN IN -->
                <li class="oup-header-menu-item desktop">
                    <a href="" class="dropdown-toggle signin" id="header-account-info-user-fullname">
                        
Sign In                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                    </a>
                    <div class="dropdown-panel dropdown-panel-signin dropdown-panel-form">
                        <div class="spinner"></div>
                    </div><!-- /.dropdown-panel -->
                </li>

                <!-- DESKTOP REGISTRATION -->
                    <li class="oup-header-menu-item desktop"><a href="https://academic.oup.com/my-account/register?siteId=5349&amp;returnUrl=%2fsysbio%2farticle%2f65%2f2%2f228%2f2427164" class="register">Register</a></li>

                <!-- LANGUAGE -->
                    <!-- <li class="oup-header-menu-item desktop">
                        <a href="" class="dropdown-toggle language">Language</a>
                        <div class="dropdown-panel">
                            <div class="dropdown-menu">
                                <ul>
                                    foreach (var lang in Model.LanguageOptions)
                                    {
                                        string selectedClass = (lang.SiteId == Model.SiteId ? " selected" : "");
                                        Html.Raw(String.Format("<li class=\"header-language-item{0}\"><a href=\"//{1}\">{2}</a></li>", selectedClass, lang.SiteUrl, lang.LanguageText))
                                        ;
                                    }
                                </ul>
                            </div>
                        </div>
                    </li> -->

            </ul><!-- /.oup-header-menu .account-menu -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-header -->

<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" selected="selected">This issue</option><option class="header-search-bar-filters-item" value="">All  Systematic Biology</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm" type="text">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            Akademicka Knihovna JU
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

Akademicka Knihovna JU
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="https://academic.oup.com/my-account/register?siteId=5349&amp;returnUrl=%2fsysbio%2farticle%2f65%2f2%2f228%2f2427164" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1126564">
            <a href="https://academic.oup.com/sysbio/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1126565">
            <a href="https://academic.oup.com/sysbio/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1126562">
            <a href="https://academic.oup.com/sysbio/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1126566">
            <a href="https://academic.oup.com/sysbio/pages/General_Instructions" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1126567">
            <a href="https://mc.manuscriptcentral.com/systbiol" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1126568">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1126569">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1126570">
            <a href="https://academic.oup.com/sysbio/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1126563">
            <a href="https://academic.oup.com/sysbio/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1126571">
            <a href="https://academic.oup.com/sysbio/pages/About" class="nav-link ">About Systematic Biology</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1126572">
            <a href="http://www.systbio.org/" class="nav-link ">About the Society of Systematic Biologists</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1126573">
            <a href="https://academic.oup.com/sysbio/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1126574">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1126575">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org/" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1126576">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_b" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1126577">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1126578">
            <a href="http://www.systbio.org/" class="nav-link ">Contact Us</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1126579">
            <a href="https://academic.oup.com/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1126580">
            <a href="https://academic.oup.com/sysbio/pages/membership_society_systematic_biologists" class="nav-link ">Join SSB</a>
            
        </li>
    </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->


    <div class="journal-header journal-bg">
        <div class="center-inner-row">
                <a href="https://academic.oup.com/sysbio" class=" journal-logo-container">

            <img id="logo-SystematicBiology" class="journal-logo" src="sysbio_title1175497244.svg" alt="Systematic Biology">

                </a>
            <div class="society-logo-block">
                    <div class="society-block-inner-wrap">
                            <a href="http://www.systbio.org/" target="" class="society-logo-container">
                                <img id="logo-SocietyofSystematicBiologists" class="society-logo" src="sysbio_h1-360035926.svg" alt="Society of Systematic Biologists">
                            </a>
                                            </div><!-- /.society-block-inner-wrap - NOTE: used to vertically center one or more society logos -->
            </div> <!-- /.society-logo-block - NOTE: We want this element on the page, even if there are no society logos -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-header -->

<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1126564">
            <a href="https://academic.oup.com/sysbio/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1126565">
            <a href="https://academic.oup.com/sysbio/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1126562">
            <a href="https://academic.oup.com/sysbio/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1126566">
            <a href="https://academic.oup.com/sysbio/pages/General_Instructions" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1126567">
            <a href="https://mc.manuscriptcentral.com/systbiol" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1126568">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1126569">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1126570">
            <a href="https://academic.oup.com/sysbio/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1126563">
            <a href="https://academic.oup.com/sysbio/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1126571">
            <a href="https://academic.oup.com/sysbio/pages/About" class="nav-link ">About Systematic Biology</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1126572">
            <a href="http://www.systbio.org/" class="nav-link ">About the Society of Systematic Biologists</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1126573">
            <a href="https://academic.oup.com/sysbio/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1126574">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1126575">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org/" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1126576">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_b" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1126577">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1126578">
            <a href="http://www.systbio.org/" class="nav-link ">Contact Us</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1126579">
            <a href="https://academic.oup.com/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1126580">
            <a href="https://academic.oup.com/sysbio/pages/membership_society_systematic_biologists" class="nav-link ">Join SSB</a>
            
        </li>
    </ul>
        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  Systematic Biology</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" maxlength="255" id="MicrositeSearchTerm" title="search input" type="text">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="https://academic.oup.com/sysbio/advanced-search?page=1&amp;fl_SiteID=5349&amp;SearchSourceType=1" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

    <input name="searchScope" id="hfSolrJournalID" value="" type="hidden">
    <input name="parentSiteName" id="hfParentSiteName" value="Journals" type="hidden">
    <input name="parentSiteUrl" id="hfparentSiteUrl" value="academic.oup.com/journals" type="hidden">
    <input name="siteID" id="hfsiteID" value="5349" type="hidden">
    <input name="parentSiteID" id="hfparentSiteID" value="5567" type="hidden">
    <input id="hfSolrJournalName" value="" type="hidden">
    <input id="hfSolrMaxAllowSearchChar" value="100" type="hidden">
    <input id="hfJournalShortName" value="" type="hidden">
    <input id="hfSearchPlaceholder" value="" type="hidden">
    <input name="hfGlobalSearchSiteURL" id="hfGlobalSearchSiteURL" value="" type="hidden">
    <input name="journalSiteScope" id="hfjournalSiteScope" value="Journals" type="hidden">
    <input name="parentSiteScope" id="hfparentSiteScope" value="Parent" type="hidden">
    <input name="defaultSearchURL" id="hfdefaultSearchURL" value="search-results?page=1&amp;q=" type="hidden">
    <input name="issueSearch" id="hfIssueSearch" value="&amp;fl_IssueID=83174" type="hidden">
    <input name="issueSiteScope" id="hfIssueSiteScope" value="Issue" type="hidden">
<input id="hfDefaultAdvancedSearchUrl" name="hfDefaultAdvancedSearchUrl" value="advanced-search?page=1&amp;q=" type="hidden">    <input name="hfSearchSiteURL" id="hfSiteURL" value="academic.oup.com/sysbio" type="hidden">
    <script type="text/javascript">
        (function () {
            var hfSiteUrl = document.getElementById('hfSiteURL');
            var siteUrl = hfSiteUrl.value;
            var subdomainIndex = siteUrl.indexOf('/');

            hfSiteUrl.value = location.host + (subdomainIndex >= 0 ? siteUrl.substring(subdomainIndex) : '');
        })();
    </script>
<input id="routename" name="RouteName" value="sysbio" type="hidden">
    </div>

        </section>
        <main id="main" tabindex="-1">
            <section class="master-main row">
                <div class="center-inner-row no-overflow">
                    
    <div class="page-column-wrap">
        <div id="InfoColumn" class="page-column page-column--left">
            <div class="mobile-content-topbar hide">
                <button class="toggle-left-col toggle-left-col__article">Article Navigation</button>
            </div>
            <div class="info-inner-wrap">
                
                <button class="toggle-left-col__close btn-as-icon icon-general-close">
                    <span class="screenreader-text">Close mobile search navigation</span>
                </button>
                <div class="responsive-nav-title">Article navigation</div>

                    <div class="info-widget-wrap">
                        <div class="widget widget-IssueInfo widget-instance-OUP_IssueInfo_Article">
        

<div id="issueInfo-OUP_IssueInfo_Article" class="article-info-wrap clearfix">
    <i class="icon-general-close mobile-nav-btn nav-open"></i>
    <div class="article-issue-img">
            <a href="https://academic.oup.com/sysbio/issue/65/2"><img id="issueImage" class="fb-featured-image" src="m_cover.gif" alt="Issue Cover"></a>
    </div>
    <div class="article-issue-info">
        <div class="volume-issue__wrap">

            <a href="https://academic.oup.com/sysbio/issue/65/2">
                        <div class="volume trailing-comma">Volume 65</div>
                        <div class="issue">Issue 2</div>

            </a>

        </div>
        <div class="ii-pub-date">
March 2016        </div>
        
    </div>

</div>
    </div>


                    <div class="content-nav">
                            <div class="widget widget-ArticleJumpLinks widget-instance-OUP_ArticleJumpLinks_Widget">
            <div class="contents-title">Article Contents</div>
    <ul class="jumplink-list">
            <li class="section-jump-link head-1" link-destination="115286517">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#115286517">Abstract</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="115286526">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#115286526"><span class="small-caps">Theory</span></a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="115286553">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#115286553"><span class="small-caps">Simulations</span></a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="115286576">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#115286576"><span class="small-caps">Total-evidence dating of hymenoptera</span></a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="115286601">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#115286601"><span class="small-caps">Discussion</span></a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="115286622">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#115286622"><span class="small-caps">Software availablity</span></a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="115286624">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#115286624"><span class="small-caps">Supplementary</span><span class="small-caps">Material</span></a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="115286626">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#115286626"><span class="small-caps">Funding</span></a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="115286628">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#115286628"><span class="small-caps">Acknowledgments</span></a>
                </div>
            </li>
            <li class="section-jump-link head-2" link-destination="115286631">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#115286631"><span class="small-caps">Appendix</span></a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="115286658">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#115286658"><span class="small-caps">References</span></a>
                </div>
            </li>
    </ul>

    </div>

	                </div>
            
                        <div class="widget widget-ArticleNavLinks widget-instance-OUP_ArticleNavLinks_Article">
        <ul class="inline-list">
        <li class="prev arrow">
            <a href="https://academic.oup.com/sysbio/article/65/2/212/2427099">&lt; Previous</a>
        </li>        
            <li class="next arrow">
            <a href="https://academic.oup.com/sysbio/article/65/2/250/2427272">Next &gt;</a>
        </li>
</ul>
    </div>

                </div>
            </div>
        </div> 
    
        <div id="ContentColumn" class="page-column page-column--center">
            
             <div class="article-browse-top article-browse-mobile-nav">
                <div class="article-browse-mobile-nav-inner">
                    <button class="toggle-left-col toggle-left-col__article btn-as-link">
                        Article Navigation
                    </button>
                </div>
            </div>

            <div class="content-inner-wrap">
                    <div class="widget widget-ArticleTopInfo widget-instance-OUP_ArticleTop_Info_Widget">
        <div class="module-widget article-top-widget">

    

    
        


    <div class="widget-items">
                <h1 class="wi-article-title article-title-main">Total-Evidence Dating under the Fossilized Birth–Death Process
<i class="icon-availability_open" title="Open Access"></i>                </h1>
                <div class="wi-authors">
                    <div class="al-authors-list">
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Chi Zhang</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label"><sup>1</sup></div> Department of Bioinformatics and Genetics, Swedish Museum of Natural History, SE-104 05 Stockholm, Sweden; </div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="https://academic.oup.com/sysbio/search-results?f_Authors=Chi+Zhang">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Zhang%20C">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:%22Zhang%20C%22">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Chi Zhang</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Tanja Stadler</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label"><sup>2</sup></div> Department of Biosystems Science and Engineering, Eidgenössische Technische Hochschule Zürich, 4053 Basel, Switzerland; </div><div class="aff"><div class="label -label"><sup>3</sup></div> Swiss Institute of Bioinformatics (SIB), Switzerland; </div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="https://academic.oup.com/sysbio/search-results?f_Authors=Tanja+Stadler">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Stadler%20T">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:%22Stadler%20T%22">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Tanja Stadler</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Seraina Klopfstein</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label"><sup>1</sup></div> Department of Bioinformatics and Genetics, Swedish Museum of Natural History, SE-104 05 Stockholm, Sweden; </div><div class="aff"><div class="label -label"><sup>4</sup></div> Department of Invertebrates, Natural History Museum Bern, CH-3005 Bern, Switzerland; </div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="https://academic.oup.com/sysbio/search-results?f_Authors=Seraina+Klopfstein">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Klopfstein%20S">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:%22Klopfstein%20S%22">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Seraina Klopfstein</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Tracy A. Heath</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label"><sup>5</sup></div> Department of Integrative Biology, University of California, Berkeley, CA 94720 USA; </div><div class="aff"><div class="label -label"><sup>6</sup></div> Department of Ecology and Evolutionary Biology, University of Kansas, Lawrence, KS 66045, USA; </div><div class="aff"><div class="label -label"><sup>7</sup></div> Department of Ecology, Evolution &amp; Organismal Biology, Iowa State University, Ames, IA 50011, USA </div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="https://academic.oup.com/sysbio/search-results?f_Authors=Tracy+A.+Heath">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Heath%20T">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:%22Heath%20T%22">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Tracy A. Heath</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Fredrik Ronquist</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label"><sup>1</sup></div> Department of Bioinformatics and Genetics, Swedish Museum of Natural History, SE-104 05 Stockholm, Sweden; </div>
        </div>

                <div class="info-card-note">
                    <a reveal-id="COR1" data-open="COR1" class="link link-ref link-reveal xref-default">*</a>
                </div>

        <div class="info-author-correspondence">
             *Correspondence to be sent to: Department of Bioinformatics
 and Genetics, Swedish Museum of Natural History, Box 50007, SE-10405 
Stockholm, Sweden; E-mail: <a href="mailto:fredrik.ronquist@nrm.se" target="">fredrik.ronquist@nrm.se</a> . 
        </div>



        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="https://academic.oup.com/sysbio/search-results?f_Authors=Fredrik+Ronquist">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Ronquist%20F">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:%22Ronquist%20F%22">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Fredrik Ronquist<i class="icon-general-mail"></i></a>
                                </span>

                                <div class="al-author-name al-author-footnotes">
                                    <div class="al-author-info-wrap arrow-up">
    <div class="widget widget-SingleSection widget-instance-OUP_FootnoteSection">
        <div content-id="FN1"><div class="fn"><div class="footnote-content">Associate Editor: Thomas Near</div></div></div>
    </div>
                                    </div>
                                    <a class="js-linked-footnotes" href="javascript:;">Author Notes</a>
                                </div>
                    </div>
                </div>
            <div class="pub-history-wrap clearfix">

                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-primary"><em>Systematic Biology</em>, Volume 65, Issue 2, 1 March 2016, Pages 228–249, <a href="https://doi.org/10.1093/sysbio/syv080">https://doi.org/10.1093/sysbio/syv080</a></div>
                    </div>
                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-date-wrap">
                            <div class="citation-label">Published:</div>
                            <div class="citation-date">22 October 2015</div>
                        </div>
                            <div class="ww-citation-history-wrap">
                                <div class="citation-label">
                                    Article history<i class="icon-general-arrow-filled-down arrow-icon"></i>
                                </div>
                            </div>
                    </div>
                                    <div class="ww-history">
                            <div class="history-entry">
                                <div class="wi-state">Received:</div>
                                <div class="wi-date">06 February 2015</div>
                            </div>
                            <div class="history-entry">
                                <div class="wi-state">Revision Requested:</div>
                                <div class="wi-date">12 October 2015</div>
                            </div>
                            <div class="history-entry">
                                <div class="wi-state">Accepted:</div>
                                <div class="wi-date">12 October 2015</div>
                            </div>

                        <div class="ww-history-close"><a>Close</a></div>
                    </div><!-- /.ww-history -->
            </div><!-- /.pub-history-wrap -->

    </div>
</div>

<script>
    $(document).ready(function () {
        $('.article-top-widget').on('click', '.ati-toggle-trigger', function () {
            $(this).find('.icon-general-add, .icon-minus').toggleClass('icon-minus icon-general-add');
            $(this).siblings('.ati-toggle-content').toggleClass('hide');
        });

        // In Chrome, an anchor tag with target="_blank" and a "mailto:" href opens a new tab/window as well as the email client
        // I suspect this behavior will be corrected in the future
        // Remove the target="_blank"
        $('ul.wi-affiliationList').find('a[href^="mailto:"]').each(function () {
            $(this).removeAttr('target');
        });
    });
</script>


    </div>
    <div class="widget widget-ArticleLinks widget-instance-OUP_Article_Links_Widget">
        


    </div>

        
                <div class="article-body">
                                    <div class="toolbar-wrap">
                    <div class="toolbar-inner-wrap"><!-- for mobile -->
                        <ul id="Toolbar" role="navigation">
                            
                            <li class="toolbar-item item-link">
                                <a class="split-view">
                                    <i class="icon-menu_split"></i>
                                    Split View
                                </a>
                            </li>
                            
                            <li class="toolbar-item item-with-dropdown item-views" style="display: none;">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="FilterDrop" aria-haspopup="true">
                                    <i class="icon-menu_views"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Views</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ViewsDrop" class="f-dropdown" data-dropdown-content="" aria-label="submenu">
                                    <div class="arrow-up" style="display: block;"></div>
                                    <li class="article-content-filter" data-content-filter="article-content" style="display: none;"><a href="javascript:;"><span>Article contents</span></a></li>
                                    <li class="article-content-filter" data-content-filter="figures-tables"><a href="javascript:;"><span>Figures &amp; tables</span></a></li>
                                    
                                    
                                    
                                </ul>
                            </li>
                                <li class="toolbar-item item-pdf">
        <a class="al-link pdf article-pdfLink" data-article-id="2427164" href="https://academic.oup.com/sysbio/article-pdf/65/2/228/24589028/syv080.pdf">
            <i class="icon-menu_pdf"></i><span>PDF</span>
        </a>
    </li>

                            <li class="toolbar-item">
                                    <div class="widget widget-ToolboxGetCitation widget-instance-OUP_Get_Citation">
        
<a href="#" data-reveal-id="getCitation" data-reveal=""><i class="icon-read-more"></i><span>Cite</span></a>

<div id="getCitation" class="reveal-modal" data-reveal="">
    <div class="modal-title">Citation</div>
    <p>Chi Zhang, Tanja Stadler, Seraina Klopfstein, Tracy A. Heath, 
Fredrik Ronquist;  Total-Evidence Dating under the Fossilized 
Birth–Death Process, <em>Systematic Biology</em>, Volume 65, Issue 2, 1 March 2016, Pages 228–249, <a href="https://doi.org/10.1093/sysbio/syv080">https://doi.org/10.1093/sysbio/syv080</a></p>
    <p class="citation-label">Download citation file:</p>  
    <ul>
        <li><a href="https://academic.oup.com/sysbio/downloadcitation/2427164?format=ris">RIS (Zotero)</a></li>
        <li><a href="https://academic.oup.com/sysbio/downloadcitation/2427164?format=ris">EndNote</a></li>
        <li><a href="https://academic.oup.com/sysbio/downloadcitation/2427164?format=bibtex">BibTex</a></li>
        <li><a href="https://academic.oup.com/sysbio/downloadcitation/2427164?format=txt">Medlars</a></li>
        <li><a href="https://academic.oup.com/sysbio/downloadcitation/2427164?format=ris">ProCite</a></li>
        <li><a href="https://academic.oup.com/sysbio/downloadcitation/2427164?format=txt">RefWorks</a></li>
        <li><a href="https://academic.oup.com/sysbio/downloadcitation/2427164?format=ris">Reference Manager</a></li>
    </ul>
    <hr>
    
    <p>© 2018 Oxford University Press</p>
    <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
</div>
    </div>
 
                            </li>
                            <li class="toolbar-item item-tools">
                                <i class="icon-menu_permissions"></i>
                                    <div class="widget widget-ToolboxPermissions widget-instance-">
            <div class="module-widget">
            <a href="https://s100.copyright.com/AppDispatchServlet?publisherName=OUP&amp;publication=1076-836X&amp;title=Total-Evidence%20Dating%20under%20the%20Fossilized%20Birth%E2%80%93Death%20Process&amp;publicationDate=2015-10-22&amp;volumeNum=65&amp;issueNum=2&amp;author=Zhang%2C%20Chi%3B%20Stadler%2C%20Tanja&amp;startPage=228&amp;endPage=249&amp;contentId=10.1093%2Fsysbio%2Fsyv080&amp;oa=CC%20BY&amp;copyright=Oxford%20University%20Press&amp;orderBeanReset=True" id="PermissionsLink" class="" target="">Permissions</a>
    </div>

    </div>

                            </li>
                            <li class="toolbar-item item-with-dropdown item-share last">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="ShareDrop" aria-haspopup="true">
                                    <i class="icon-menu_share"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Share</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ShareDrop" class="addthis_toolbox addthis_default_style addthis_20x20_style f-dropdown" data-dropdown-content="" aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li><a class="addthis_button_email at300b" target="" title="Email" href="#"><span>Email</span></a></li>
                                    <li><a class="addthis_button_twitter at300b" title="Twitter" href="#"><span>Twitter</span></a></li>
                                    <li><a class="addthis_button_facebook at300b" title="Facebook" href="#"><span>Facebook</span></a></li>
                                <div class="atclear"></div></ul>
                            </li>
                        </ul>
                    
                        <div class="toolbar-search">
                                <div class="widget widget-SitePageHeader widget-instance-OUP_ArticleToolbarSearchBox">
        


<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" selected="selected">This issue</option><option class="header-search-bar-filters-item" value="">All  </option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm" type="text">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                            <li class="site-menu-item">
                    <a href="https://academic.oup.com/my-account/register?siteId=0&amp;returnUrl=%2fsysbio%2farticle%2f65%2f2%2f228%2f2427164" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">

    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->



<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">

        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  </option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" maxlength="255" id="MicrositeSearchTerm" title="search input" type="text">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a class="advanced-search js-advanced-search" href="https://academic.oup.com/sysbio/advanced-search?page=1&amp;fl_SiteID=5349&amp;SearchSourceType=1">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

<input id="routename" name="RouteName" value="" type="hidden">
    </div>

                        </div>
                    </div><!-- /.toolbar-inner-wrap -->                    
                </div>
                    <div id="ContentTab" class="content active">
                            <div class="widget widget-ArticleFulltext widget-instance-OUP_Article_FullText_Widget">
        <div class="module-widget">
    <div class="widget-items" data-widgetname="ArticleFulltext">





                            <h2 scrollto-destination="115286517" id="115286517" class="abstract-title">Abstract</h2>
<section class="abstract"><p>Bayesian total-evidence dating involves the
 simultaneous analysis of morphological data from the fossil record and 
morphological and sequence data from recent organisms, and it 
accommodates the uncertainty in the placement of fossils while dating 
the phylogenetic tree. Due to the flexibility of the Bayesian approach, 
total-evidence dating can also incorporate additional sources of 
information. Here, we take advantage of this and expand the analysis to 
include information about fossilization and sampling processes. Our work
 is based on the recently described fossilized birth–death (FBD) 
process, which has been used to model speciation, extinction, and 
fossilization rates that can vary over time in a piecewise manner. So 
far, sampling of extant and fossil taxa has been assumed to be either 
complete or uniformly at random, an assumption which is only valid for a
 minority of data sets. We therefore extend the FBD process to 
accommodate diversified sampling of extant taxa, which is standard 
practice in studies of higher-level taxa. We verify the implementation 
using simulations and apply it to the early radiation of Hymenoptera 
(wasps, ants, and bees). Previous total-evidence dating analyses of this
 data set were based on a simple uniform tree prior and dated the 
initial radiation of extant Hymenoptera to the late Carboniferous (309 
Ma). The analyses using the FBD prior under diversified sampling, 
however, date the radiation to the Triassic and Permian (252 Ma), 
slightly older than the age of the oldest hymenopteran fossils. By 
exploring a variety of FBD model assumptions, we show that it is mainly 
the accommodation of diversified sampling that causes the push toward 
more recent divergence times. Accounting for diversified sampling thus 
has the potential to close the long-discussed gap between rocks and 
clocks. We conclude that the explicit modeling of fossilization and 
sampling processes can improve divergence time estimates, but only if 
all important model aspects, including sampling biases, are adequately 
addressed.</p></section>                        <div class="article-metadata-panel clearfix"><div class="kwd-group"><a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;Bayesian phylogenetic inference&quot;">Bayesian phylogenetic inference</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;birth–death process&quot;">birth–death process</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="MCMC">MCMC</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;relaxed clock&quot;">relaxed clock</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;total-evidence dating&quot;">total-evidence dating</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;tree prior&quot;">tree prior</a></div><div class="article-metadata-tocSections">
            <div class="article-metadata-tocSections-title">Issue Section:</div>
                <a href="https://academic.oup.com/sysbio/search-results?f_TocHeadingTitle=Regular%20articles">Regular articles</a>
        </div></div>
<p> In recent years, there has been increasing interest in dated 
phylogenies, as they inform a wide range of questions in evolutionary 
biology. Advances in sequencing technologies coupled with new, 
statistically rigorous inference methods have greatly enhanced our 
ability to investigate phylogenetic relationships, but this is only the 
first step toward a dated phylogeny. Molecular data only provide 
evolutionary distances in units of evolutionary change, such as 
substitutions per site. Branch lengths measured in this way are the 
product of the geological time duration (e.g., in myr) and the 
evolutionary rate (e.g., in substitutions per site per myr). To estimate
 rates and times separately (on a relative scale), it is necessary to 
introduce additional model assumptions that account for branch-rate 
variation across the tree and the distribution of speciation events over
 time. Early studies achieved this by considering the evolutionary rate 
to be constant over time, that is, assuming a global molecular clock 
(also called a strict clock; <a reveal-id="B63" data-open="B63" class="link link-ref link-reveal xref-bibr">Zuckerkandl and Pauling 1965</a>
 ). More recent methods allow the rate to vary over time under 
constraints specified by a relaxed-clock model, typically using a 
Bayesian inference framework ( <a reveal-id="B15" data-open="B15" class="link link-ref link-reveal xref-bibr">Hasegawa et al. 1989</a> ; <a reveal-id="B26" data-open="B26" class="link link-ref link-reveal xref-bibr">Kishino et al. 1990</a> ; <a reveal-id="B54" data-open="B54" class="link link-ref link-reveal xref-bibr">Thorne et al. 1998</a> ; <a reveal-id="B23" data-open="B23" class="link link-ref link-reveal xref-bibr">Huelsenbeck et al. 2000</a> ; <a reveal-id="B61" data-open="B61" class="link link-ref link-reveal xref-bibr">Yoder and Yang 2000</a> ; <a reveal-id="B27" data-open="B27" class="link link-ref link-reveal xref-bibr">Kishino et al. 2001</a> ; <a reveal-id="B53" data-open="B53" class="link link-ref link-reveal xref-bibr">Thorne and Kishino 2002</a> ; <a reveal-id="B3" data-open="B3" class="link link-ref link-reveal xref-bibr">Aris-Brosou and Yang 2002</a> ; <a reveal-id="B60" data-open="B60" class="link link-ref link-reveal xref-bibr">Yang and Yoder 2003</a> ; <a reveal-id="B9" data-open="B9" class="link link-ref link-reveal xref-bibr">Drummond et al. 2006</a> ; <a reveal-id="B31" data-open="B31" class="link link-ref link-reveal xref-bibr">Lepage et al. 2006</a> ; <a reveal-id="B32" data-open="B32" class="link link-ref link-reveal xref-bibr">2007</a> ; <a reveal-id="B40" data-open="B40" class="link link-ref link-reveal xref-bibr">Rannala and Yang 2007</a> ; <a reveal-id="B8" data-open="B8" class="link link-ref link-reveal xref-bibr">Drummond and Suchard 2010</a> ; <a reveal-id="B17" data-open="B17" class="link link-ref link-reveal xref-bibr">Heath et al. 2012</a> ; <a reveal-id="B19" data-open="B19" class="link link-ref link-reveal xref-bibr">Heath and Moore 2014</a> ). </p><p>
 Regardless of whether a strict or a relaxed-clock model is used, the 
result is an estimate of species divergence times on a relative scale. 
To convert the relative times into absolute times, it has been customary
 to rely on user-specified internal nodes that are calibrated using 
additional information, typically from biogeographic events or from the 
fossil record. Using one or more such calibration nodes, it is possible 
to estimate the ages of all other nodes in the tree. This calibration 
technique—referred to here as “node dating”—can treat the ages of the 
calibration nodes as known without error ( <a reveal-id="B13" data-open="B13" class="link link-ref link-reveal xref-bibr">Graur and Martin 2004</a> ; <a reveal-id="B20" data-open="B20" class="link link-ref link-reveal xref-bibr">Hedges and Kumar 2004</a> ), or assign probability distributions to them ( <a reveal-id="B54" data-open="B54" class="link link-ref link-reveal xref-bibr">Thorne et al. 1998</a> ; <a reveal-id="B52" data-open="B52" class="link link-ref link-reveal xref-bibr">Tavaré et al. 2002</a> ; <a reveal-id="B59" data-open="B59" class="link link-ref link-reveal xref-bibr">Yang and Rannala 2006</a> ; <a reveal-id="B21" data-open="B21" class="link link-ref link-reveal xref-bibr">Ho and Phillips 2009</a> ; <a reveal-id="B55" data-open="B55" class="link link-ref link-reveal xref-bibr">Wilkinson et al. 2011</a> ; <a reveal-id="B16" data-open="B16" class="link link-ref link-reveal xref-bibr">Heath 2012</a> ). </p><p> An alternative approach to estimating divergence times was proposed recently and has been termed “total-evidence dating” ( <a reveal-id="B37" data-open="B37" class="link link-ref link-reveal xref-bibr">Pyron 2011</a> ; <a reveal-id="B44" data-open="B44" class="link link-ref link-reveal xref-bibr">Ronquist et al. 2012b</a>
 ). Compared with node dating methods, the total-evidence approach 
introduces a number of innovations in terms of how the fossil 
information is incorporated in the analysis. Specifically, homologous 
morphological characters are coded for fossil and extant taxa and 
included in a combined matrix. Age estimates are assigned to individual 
fossils based on the dating of the strata in which they are found. These
 data, together with molecular sequences sampled from extant taxa, are 
analyzed in an integrative framework to directly inform the inference of
 divergence times, while accounting for uncertainty in the placement of 
the fossils in the phylogeny. </p><p> One of the essential strengths of 
the total-evidence dating approach is that it allows the probabilistic 
model to be expanded to include additional sources of information that 
could be important in dating but have not been modeled explicitly 
before. Here, we exploit this to address the fossilization process and 
the sampling procedure, both of which potentially have a major impact on
 divergence time estimates. This has to be done in the context of a tree
 model capable of accommodating speciation, extinction, fossilization, 
and sampling. The early implementations of total-evidence dating ( <a reveal-id="B30" data-open="B30" class="link link-ref link-reveal xref-bibr">Lee et al. 2009</a> ; <a reveal-id="B37" data-open="B37" class="link link-ref link-reveal xref-bibr">Pyron 2011</a> ; <a reveal-id="B44" data-open="B44" class="link link-ref link-reveal xref-bibr">Ronquist et al. 2012b</a> ), in contrast, relied on simple tree priors that did not address fossilization and sampling. </p><p>
 The standard birth–death model used in phylogenetics, that is, the 
constant-rate reconstructed birth–death process, assumes that birth and 
death rates (or speciation and extinction rates) are constant over time 
and that no individuals (fossils) are sampled in the past ( <a reveal-id="B25" data-open="B25" class="link link-ref link-reveal xref-bibr">Kendall 1948</a> ; <a reveal-id="B36" data-open="B36" class="link link-ref link-reveal xref-bibr">Nee et al. 1994</a> ; <a reveal-id="B58" data-open="B58" class="link link-ref link-reveal xref-bibr">Yang and Rannala 1997</a> ; <a reveal-id="B12" data-open="B12" class="link link-ref link-reveal xref-bibr">Gernhard 2008</a> ; <a reveal-id="B45" data-open="B45" class="link link-ref link-reveal xref-bibr">Stadler 2009</a> ). <a reveal-id="B46" data-open="B46" class="link link-ref link-reveal xref-bibr">Stadler (2010a)</a> extended this process to account for serially sampled lineages (also see <a reveal-id="B6" data-open="B6" class="link link-ref link-reveal xref-bibr">Didier et al. 2012</a>
 ); this extension is called the “fossilized birth–death” (FBD) process.
 The FBD process prior simultaneously models the speciation and 
extinction patterns and observations of fossils in a birth–death 
macroevolutionary framework. <a reveal-id="B18" data-open="B18" class="link link-ref link-reveal xref-bibr">Heath et al. (2014)</a>
 applied the FBD prior to estimate speciation times, assuming constant 
rates of birth, death, and fossil sampling, when morphological character
 data for extant and fossil taxa are unavailable. Their approach is more
 similar to node dating than total-evidence dating in that fossils are 
associated with specific clades a priori, and the lengths of the fossil 
branches are only informed by the FBD prior, not by character data. 
Recently, <a reveal-id="B11" data-open="B11" class="link link-ref link-reveal xref-bibr">Gavryushkina et al. (2014)</a>
 extended the FBD model to allow speciation, extinction, and sampling 
rates to change through time in a piecewise manner, as previously 
described for extant species phylogenies ( <a reveal-id="B48" data-open="B48" class="link link-ref link-reveal xref-bibr">Stadler 2011</a> ) and virus phylogenies ( <a reveal-id="B51" data-open="B51" class="link link-ref link-reveal xref-bibr">Stadler et al. 2013</a> ). </p><p>
 So far, work on the FBD process has assumed either complete or 
uniformly random sampling of fossils and extant taxa. Although this is 
convenient from a mathematical perspective, it potentially ignores an 
important bias in the data resulting from the common non-random choice 
of terminals for dating analyses. <a reveal-id="B22" data-open="B22" class="link link-ref link-reveal xref-bibr">Höhna et al. (2011)</a>
 showed that incorrect modeling of the sampling process can cause major 
problems in inferring speciation and extinction rates under standard 
birth–death models, especially in cases where investigators tried to 
maximize the taxonomic diversity in the sample. Such diversified 
sampling is arguably more common than random sampling in species-level 
data sets, and is even explicit in all studies at higher taxonomic 
levels. Consider a strongly asymmetric radiation like Hymenoptera, for 
instance, where a ladder with six or seven side branches leads to a 
crown group with 95% or more of the total species diversity, which 
numbers in the 100,000's at least. No evolutionary biologist would study
 the early radiation of the group without trying to include all of the 
basal divergence events, but the chance of hitting all of them in a 
uniform-random sampling scheme would be negligible. </p><p><a reveal-id="B22" data-open="B22" class="link link-ref link-reveal xref-bibr">Höhna et al. (2011)</a>
 incorporated diversified sampling into the birth–death process by 
assuming that the investigator has succeeded in maximizing diversity, 
that is, has sampled exactly one extant descendant for each branch that 
was present at a specific time in the past (and has given rise to extant
 species). We take the same approach here to accommodate diversified 
sampling of extant taxa under the piecewise-constant FBD model. We use 
simulations to validate our implementation of the extended FBD model and
 to investigate its performance. We then apply total-evidence dating 
under this model to reinvestigate the early radiation of Hymenoptera, 
which was originally analyzed under a simple uniform tree prior ( <a reveal-id="B44" data-open="B44" class="link link-ref link-reveal xref-bibr">Ronquist et al. 2012b</a>
 ). We assess the influence of assumptions about the fossilization and 
sampling processes on the inferred divergence times, especially focusing
 on random and diversified sampling strategies. </p>                            <h2 scrollto-destination="115286526" id="115286526" class="section-title"><span class="small-caps">Theory</span></h2>
                            <h3 scrollto-destination="115286527" id="115286527" class="section-title">The FBD Process</h3>
<p> We use the FBD process as a macroevolutionary model describing the tree topology and node ages ( <a reveal-id="B46" data-open="B46" class="link link-ref link-reveal xref-bibr">Stadler 2010a</a> ; <a reveal-id="B18" data-open="B18" class="link link-ref link-reveal xref-bibr">Heath et al. 2014</a> ; <a reveal-id="B11" data-open="B11" class="link link-ref link-reveal xref-bibr">Gavryushkina et al. 2014</a> ). The model gives rise to extant species phylogenies with fossils. The process starts at time <span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><msub><mi>t</mi><mrow><mi>o</mi><mi>r</mi></mrow></msub></mrow></math></span>
 (time of origin or stem age) in the past with a single lineage 
(species). Lineages give birth to new lineages with a constant rate λ 
(speciation events), and die with a constant rate μ (extinction events).
 </p><p> Along branches, fossils are observed with a constant rate ψ. In
 addition, fossils can be observed with a constant probability ρ at 
pre-specified times in the past, accounting for extensive fossil 
sampling in particular stratigraphic layers (moments in time). The 
process is stopped at the present (i.e., after time <span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><msub><mi>t</mi><mrow><mi>o</mi><mi>r</mi></mrow></msub></mrow></math></span>
 ) when extant taxa are sampled (two sampling schemes—random and 
diversified sampling—are detailed below). The observed FBD tree is the 
tree that results when all lineages without a fossil or sampled extant 
descendant have been pruned away ( <a reveal-id="F1" data-open="F1" class="link link-reveal link-table xref-fig">Fig. 1</a>
 ). In order to distinguish between branches in the tree, we arbitrarily
 label the two lineages descending from each branching event with “left”
 or “right”; such trees are called oriented trees ( <a reveal-id="B10" data-open="B10" class="link link-ref link-reveal xref-bibr">Ford et al. 2009</a> ). For mathematical convenience, we derive probability densities on oriented trees. </p>                        <a id="115286530" scrollto-destination="115286530"></a>
<div id="F1" data-id="F1" class="fig fig-section"><div class="label fig-label">Figure 1.</div><div class="graphic-wrap"><img class="content-image" src="m_syv080f1.png" alt=" The complete tree (oriented) is generated from the birth–death process. Time from the root ( tmrca ) to the present ( tl=0 ) is divided into l intervals by time ti ( i=1,…,l ). The birth, death, and fossil sampling rates are λi , μi , and ψi respectively in each interval, whereas the sampling probability at time ti is ρi . The sampled tree on the right is called FBD tree. Fossil tips not sampled at any ti are at time y1,…,ym . Interior nodes are at time x1,…,xm+M−1 with x1 being the root age ( tmrca ). There are two sampling strategies for extant taxa: a) random sampling, and b) diversified sampling. " path-from-xml="syv080f1.tiff"><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="https://academic.oup.com/view-large/figure/115286530/syv080f1.tiff" path-from-xml="syv080f1.tiff" target="_blank">View large</a><a section="115286530" href="https://oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/sysbio/65/2/10.1093_sysbio_syv080/3/syv080f1.png?Expires=2147483647&amp;Signature=h1QFUE4YMyihEvm6XdEMHQtOHXV3HTpYrpPrBnqBgYD2Kz1j0PIdMXwpekm%7E-h%7E6uDsRm8a7ECNLLelVixyoeyf3sdl1H7iL4VJ6ori6OT5UzvrpWIy0tLsIeAuc%7EtYqtX78xzLd2aR9NlQ09NYc-f8Dba9bRg8ySk3JEuAoVc1fPvRtFjqCqgVNtT%7ERTKmciH50qKSXIrs2DqkrRM%7ERrE8ELvJpRSwNtlsKYGBr21lOGOfbBtATPGM190WcdhAx7snS5LjIig9SeYMcY9l4L0Joy8kqTpLGBN305e6MzlDJLhHZnbZW4vaA0U6Irsca5OlyqGZMnKOM-fgsRAAsGw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=115286530&amp;ar=2427164&amp;xsltPath=%7E/UI/app/XSLT&amp;imagename=" class="download-slide" path-from-xml="syv080f1.tiff" target="">Download slide</a></div></div><div class="caption fig-caption"><p> The complete tree (oriented) is generated from the birth–death process. Time from the root ( <span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><msub><mi>t</mi><mrow><mi>m</mi><mi>r</mi><mi>c</mi><mi>a</mi></mrow></msub></mrow></math></span> ) to the present ( <span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><msub><mi>t</mi><mi>l</mi></msub><mo>=</mo><mn>0</mn></mrow></math></span> ) is divided into <span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mi xmlns="">l</mi></math></span> intervals by time <span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><msub><mi>t</mi><mi>i</mi></msub></mrow></math></span> ( <span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><mi>i</mi><mo>=</mo><mn>1</mn><mo>,</mo><mo>…</mo><mo>,</mo><mi>l</mi></mrow></math></span> ). The birth, death, and fossil sampling rates are <span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><msub><mi>λ</mi><mi>i</mi></msub></mrow></math></span> , <span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><msub><mi>μ</mi><mi>i</mi></msub></mrow></math></span> , and <span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><msub><mi>ψ</mi><mi>i</mi></msub></mrow></math></span> respectively in each interval, whereas the sampling probability at time <span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><msub><mi>t</mi><mi>i</mi></msub></mrow></math></span> is <span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><msub><mi>ρ</mi><mi>i</mi></msub></mrow></math></span> . The sampled tree on the right is called FBD tree. Fossil tips not sampled at any <span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><msub><mi>t</mi><mi>i</mi></msub></mrow></math></span> are at time <span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><msub><mi>y</mi><mn>1</mn></msub><mo>,</mo><mo>…</mo><mo>,</mo><msub><mi>y</mi><mi>m</mi></msub></mrow></math></span> . Interior nodes are at time <span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><msub><mi>x</mi><mn>1</mn></msub><mo>,</mo><mo>…</mo><mo>,</mo><msub><mi>x</mi><mrow><mi>m</mi><mo>+</mo><mi>M</mi><mo>−</mo><mn>1</mn></mrow></msub></mrow></math></span> with <span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><msub><mi>x</mi><mn>1</mn></msub></mrow></math></span> being the root age ( <span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><msub><mi>t</mi><mrow><mi>m</mi><mi>r</mi><mi>c</mi><mi>a</mi></mrow></msub></mrow></math></span> ). There are two sampling strategies for extant taxa: a) random sampling, and b) diversified sampling. </p></div></div><div content-id="F1" class="fig fig-modal reveal-modal" style="display: block;"><div class="label fig-label">Figure 1.</div><div class="graphic-wrap"><img class="content-image" src="m_syv080f1.png" alt=" The complete tree (oriented) is generated from the birth–death process. Time from the root ( tmrca ) to the present ( tl=0 ) is divided into l intervals by time ti ( i=1,…,l ). The birth, death, and fossil sampling rates are λi , μi , and ψi respectively in each interval, whereas the sampling probability at time ti is ρi . The sampled tree on the right is called FBD tree. Fossil tips not sampled at any ti are at time y1,…,ym . Interior nodes are at time x1,…,xm+M−1 with x1 being the root age ( tmrca ). There are two sampling strategies for extant taxa: a) random sampling, and b) diversified sampling. " path-from-xml="syv080f1.tiff"><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="https://academic.oup.com/view-large/figure/115286530/syv080f1.tiff" path-from-xml="syv080f1.tiff" target="_blank">View large</a><a section="115286530" href="https://oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/sysbio/65/2/10.1093_sysbio_syv080/3/syv080f1.png?Expires=2147483647&amp;Signature=h1QFUE4YMyihEvm6XdEMHQtOHXV3HTpYrpPrBnqBgYD2Kz1j0PIdMXwpekm%7E-h%7E6uDsRm8a7ECNLLelVixyoeyf3sdl1H7iL4VJ6ori6OT5UzvrpWIy0tLsIeAuc%7EtYqtX78xzLd2aR9NlQ09NYc-f8Dba9bRg8ySk3JEuAoVc1fPvRtFjqCqgVNtT%7ERTKmciH50qKSXIrs2DqkrRM%7ERrE8ELvJpRSwNtlsKYGBr21lOGOfbBtATPGM190WcdhAx7snS5LjIig9SeYMcY9l4L0Joy8kqTpLGBN305e6MzlDJLhHZnbZW4vaA0U6Irsca5OlyqGZMnKOM-fgsRAAsGw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=115286530&amp;ar=2427164&amp;xsltPath=%7E/UI/app/XSLT&amp;imagename=" class="download-slide" path-from-xml="syv080f1.tiff" target="">Download slide</a></div></div><div class="caption fig-caption"><p> The complete tree (oriented) is generated from the birth–death process. Time from the root ( <span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><msub><mi>t</mi><mrow><mi>m</mi><mi>r</mi><mi>c</mi><mi>a</mi></mrow></msub></mrow></math></span> ) to the present ( <span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><msub><mi>t</mi><mi>l</mi></msub><mo>=</mo><mn>0</mn></mrow></math></span> ) is divided into <span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mi xmlns="">l</mi></math></span> intervals by time <span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><msub><mi>t</mi><mi>i</mi></msub></mrow></math></span> ( <span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><mi>i</mi><mo>=</mo><mn>1</mn><mo>,</mo><mo>…</mo><mo>,</mo><mi>l</mi></mrow></math></span> ). The birth, death, and fossil sampling rates are <span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><msub><mi>λ</mi><mi>i</mi></msub></mrow></math></span> , <span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><msub><mi>μ</mi><mi>i</mi></msub></mrow></math></span> , and <span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><msub><mi>ψ</mi><mi>i</mi></msub></mrow></math></span> respectively in each interval, whereas the sampling probability at time <span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><msub><mi>t</mi><mi>i</mi></msub></mrow></math></span> is <span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><msub><mi>ρ</mi><mi>i</mi></msub></mrow></math></span> . The sampled tree on the right is called FBD tree. Fossil tips not sampled at any <span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><msub><mi>t</mi><mi>i</mi></msub></mrow></math></span> are at time <span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><msub><mi>y</mi><mn>1</mn></msub><mo>,</mo><mo>…</mo><mo>,</mo><msub><mi>y</mi><mi>m</mi></msub></mrow></math></span> . Interior nodes are at time <span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><msub><mi>x</mi><mn>1</mn></msub><mo>,</mo><mo>…</mo><mo>,</mo><msub><mi>x</mi><mrow><mi>m</mi><mo>+</mo><mi>M</mi><mo>−</mo><mn>1</mn></mrow></msub></mrow></math></span> with <span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><msub><mi>x</mi><mn>1</mn></msub></mrow></math></span> being the root age ( <span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><msub><mi>t</mi><mrow><mi>m</mi><mi>r</mi><mi>c</mi><mi>a</mi></mrow></msub></mrow></math></span> ). There are two sampling strategies for extant taxa: a) random sampling, and b) diversified sampling. </p></div></div><p> In our implementation, we focus on trees conditioned on the crown age ( <span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><msub><mi>t</mi><mrow><mi>m</mi><mi>r</mi><mi>c</mi><mi>a</mi></mrow></msub></mrow></math></span> ) rather than the stem age ( <span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><msub><mi>t</mi><mrow><mi>o</mi><mi>r</mi></mrow></msub></mrow></math></span> ). Thus, at time <span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><msub><mi>t</mi><mrow><mi>m</mi><mi>r</mi><mi>c</mi><mi>a</mi></mrow></msub></mrow></math></span>
 in the past, we start with two lineages, each sharing the same time of 
origin (i.e, the root age). The process is further conditioned on both 
lineages producing sampled descendants, as otherwise the time <span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><msub><mi>t</mi><mrow><mi>m</mi><mi>r</mi><mi>c</mi><mi>a</mi></mrow></msub></mrow></math></span> would not be a crown age. In a hierarchical Bayesian model, <span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><msub><mi>t</mi><mrow><mi>m</mi><mi>r</mi><mi>c</mi><mi>a</mi></mrow></msub></mrow></math></span>
 would typically be considered a random variable drawn from a root age 
prior distribution, and the FBD distribution would then be used to 
calculate the probability of the rest of the tree conditional on the 
value of <span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><msub><mi>t</mi><mrow><mi>m</mi><mi>r</mi><mi>c</mi><mi>a</mi></mrow></msub></mrow></math></span>
 . Unlike some other implementations of the birth–death model, we do not
 condition the FBD probability on the number of extant taxa. </p>                            <h3 scrollto-destination="115286532" id="115286532" class="section-title">Modeling Parameter Shifts Over Time and Random Lineage Sampling</h3>
<p> Similarly to <a reveal-id="B11" data-open="B11" class="link link-ref link-reveal xref-bibr">Gavryushkina et al. (2014)</a> ; we allow variable (piecewise constant) birth, death, and sampling rates in the complete tree (also see <a reveal-id="B48" data-open="B48" class="link link-ref link-reveal xref-bibr">Stadler 2011</a> ; <a reveal-id="B51" data-open="B51" class="link link-ref link-reveal xref-bibr">Stadler et al. 2013</a> . The time from the root ( <span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><msub><mi>t</mi><mrow><mi>m</mi><mi>r</mi><mi>c</mi><mi>a</mi></mrow></msub></mrow></math></span> ) to the present ( <span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><msub><mi>t</mi><mi>l</mi></msub><mo>=</mo><mn>0</mn></mrow></math></span> ) is divided into <span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mi xmlns="">l</mi></math></span> intervals by time <span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><msub><mi>t</mi><mi>i</mi></msub></mrow></math></span> ( <span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><mi>i</mi><mo>=</mo><mn>1</mn><mo>,</mo><mo>…</mo><mo>,</mo><mi>l</mi></mrow></math></span> ). The birth, death, and fossil sampling rates are <span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><msub><mi>λ</mi><mi>i</mi></msub></mrow></math></span> , <span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><msub><mi>μ</mi><mi>i</mi></msub></mrow></math></span> , and <span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><msub><mi>ψ</mi><mi>i</mi></msub></mrow></math></span> in interval <span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><mo stretchy="false">[</mo><msub><mi>t</mi><mrow><mi>i</mi><mo>−</mo><mn>1</mn></mrow></msub><mo>,</mo><msub><mi>t</mi><mi>i</mi></msub><mo stretchy="false">)</mo></mrow></math></span> , whereas the (uniformly random) sampling probability at time <span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><msub><mi>t</mi><mi>i</mi></msub></mrow></math></span> is <span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><msub><mi>ρ</mi><mi>i</mi></msub></mrow></math></span> ( <a reveal-id="F1" data-open="F1" class="link link-reveal link-table xref-fig">Fig. 1</a> a). </p><p> In order to state the probability of the FBD tree under this model, we need some additional notation. We use <span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><msub><mi>M</mi><mi>i</mi></msub></mrow></math></span> for the number of sampled tips at time <span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><msub><mi>t</mi><mi>i</mi></msub></mrow></math></span> and <span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><msub><mi>K</mi><mi>i</mi></msub></mrow></math></span> for the number of sampled fossils with sampled descendants at time <span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><msub><mi>t</mi><mi>i</mi></msub></mrow></math></span> . <span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><msub><mi>N</mi><mi>i</mi></msub><mo>=</mo><msub><mi>M</mi><mi>i</mi></msub><mo>+</mo><msub><mi>K</mi><mi>i</mi></msub></mrow></math></span> is the total number of samples at time <span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><msub><mi>t</mi><mi>i</mi></msub></mrow></math></span> , and <span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><msub><mi>n</mi><mi>i</mi></msub></mrow></math></span> is the number of lineages present in the tree at time <span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><msub><mi>t</mi><mi>i</mi></msub></mrow></math></span> but not sampled at this time ( <span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><mi>i</mi><mo>=</mo><mn>1</mn><mo>,</mo><mo>…</mo><mo>,</mo><mi>l</mi></mrow></math></span> ). For example, in the FBD tree of <a reveal-id="F1" data-open="F1" class="link link-reveal link-table xref-fig">Figure 1</a> a, <span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><msub><mi>K</mi><mn>1</mn></msub><mo>=</mo><msub><mi>M</mi><mn>2</mn></msub><mo>=</mo><msub><mi>n</mi><mn>1</mn></msub><mo>=</mo><mn>1</mn></mrow></math></span> , <span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><msub><mi>M</mi><mn>3</mn></msub><mo>=</mo><msub><mi>n</mi><mn>2</mn></msub><mo>=</mo><mn>3</mn></mrow></math></span> , and <span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><msub><mi>n</mi><mn>3</mn></msub><mo>=</mo><mn>0</mn></mrow></math></span> . Let <span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><mi>M</mi><mo>=</mo><mstyle displaystyle="true"><munderover><mo>∑</mo><mrow><mi>i</mi><mo>=</mo><mn>1</mn></mrow><mi>l</mi></munderover><mrow><msub><mi>M</mi><mi>i</mi></msub></mrow></mstyle></mrow></math></span> and <span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><mi>K</mi><mo>=</mo><mstyle displaystyle="true"><munderover><mo>∑</mo><mrow><mi>i</mi><mo>=</mo><mn>1</mn></mrow><mi>l</mi></munderover><mrow><msub><mi>K</mi><mi>i</mi></msub></mrow></mstyle></mrow></math></span> . Between <span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><msub><mi>t</mi><mrow><mi>i</mi><mo>−</mo><mn>1</mn></mrow></msub></mrow></math></span> and <span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><msub><mi>t</mi><mi>i</mi></msub></mrow></math></span> , we use <span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><msub><mi>m</mi><mi>i</mi></msub></mrow></math></span> for the number of sampled fossil tips and <span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><msub><mi>k</mi><mi>i</mi></msub></mrow></math></span> for the number of sampled fossils with sampled descendants ( <span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><mi>i</mi><mo>=</mo><mn>1</mn><mo>,</mo><mo>…</mo><mo>,</mo><mi>l</mi></mrow></math></span> ). Let <span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><mi>m</mi><mo>=</mo><mstyle displaystyle="true"><munderover><mo>∑</mo><mrow><mi>i</mi><mo>=</mo><mn>1</mn></mrow><mi>l</mi></munderover><mrow><msub><mi>m</mi><mi>i</mi></msub></mrow></mstyle></mrow></math></span> and <span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><mi>k</mi><mo>=</mo><mstyle displaystyle="true"><munderover><mo>∑</mo><mrow><mi>i</mi><mo>=</mo><mn>1</mn></mrow><mi>l</mi></munderover><mrow><msub><mi>k</mi><mi>i</mi></msub></mrow></mstyle></mrow></math></span> . Fossil tips not occurring at time <span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><msub><mi>t</mi><mi>i</mi></msub></mrow></math></span> (i.e., the tips sampled with rate ψ) are at time <span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><msub><mi>y</mi><mn>1</mn></msub><mo>,</mo><mo>…</mo><mo>,</mo><msub><mi>y</mi><mi>m</mi></msub></mrow></math></span> . Interior branching times are at time <span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><msub><mi>x</mi><mn>1</mn></msub><mo>,</mo><mo>…</mo><mo>,</mo><msub><mi>x</mi><mrow><mi>m</mi><mo>+</mo><mi>M</mi><mo>−</mo><mn>1</mn></mrow></msub></mrow></math></span> with <span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><msub><mi>x</mi><mn>1</mn></msub></mrow></math></span> being the root age ( <span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><msub><mi>t</mi><mrow><mi>m</mi><mi>r</mi><mi>c</mi><mi>a</mi></mrow></msub></mrow></math></span> ). </p><div class="block-child-p"> The probability density of the FBD tree, <span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mi class="calligraphy" xmlns="" mathvariant="script">T</mi></math></span> , conditioned on the crown age being at time <span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><msub><mi>x</mi><mn>1</mn></msub></mrow></math></span> , a derivation analogous to that in ( <a reveal-id="B51" data-open="B51" class="link link-ref link-reveal xref-bibr">Stadler et al. 2013</a> ), is <a id="syv080-M1" class="jumplink-placeholder">&nbsp;</a><div class="formula-wrap"><div class="disp-formula" content-id="syv080-M1"><div class="math"><math xmlns="http://www.w3.org/1998/Math/MathML"><mtable columnalign="left" xmlns=""><mtr><mtd><mi>f</mi><mo stretchy="false">(</mo><mi class="calligraphy" mathvariant="script">T</mi><mi> </mi><mo stretchy="false">|</mo><msub><mi>t</mi><mrow><mi>m</mi><mi>r</mi><mi>c</mi><mi>a</mi></mrow></msub><mo>=</mo><msub><mi>x</mi><mn>1</mn></msub><mo>,</mo><mo>λ</mo><mo>,</mo><mo>μ</mo><mo>,</mo><mo>ψ</mo><mo>,</mo><mo>ρ</mo><mo>,</mo><mi>t</mi><mo stretchy="false">)</mo><mo>=</mo></mtd></mtr><mtr><mtd><mfrac><mrow><msubsup><mi>q</mi><mn>1</mn><mn>2</mn></msubsup><mo stretchy="false">(</mo><msub><mi>x</mi><mn>1</mn></msub><mo stretchy="false">)</mo></mrow><mrow><msup><mrow><mo stretchy="false">(</mo><mn>1</mn><mo>−</mo><msub><mi>p</mi><mn>1</mn></msub><mo stretchy="false">(</mo><msub><mi>x</mi><mn>1</mn></msub><mo stretchy="false">)</mo><mo stretchy="false">)</mo></mrow><mn>2</mn></msup></mrow></mfrac><mo>​</mo><mo>​</mo><mstyle displaystyle="true"><munderover><mo>∏</mo><mrow><mi>i</mi><mo>=</mo><mn>1</mn></mrow><mi>k</mi></munderover><mrow><msub><mi>ψ</mi><mrow><mi>I</mi><mo stretchy="false">(</mo><msub><mi>y</mi><mi>i</mi></msub><mo stretchy="false">)</mo></mrow></msub></mrow></mstyle><mo>​</mo><mo>​</mo><mstyle displaystyle="true"><munderover><mo>∏</mo><mrow><mi>i</mi><mo>=</mo><mn>2</mn></mrow><mrow><mi>m</mi><mo>+</mo><mi>M</mi><mo>−</mo><mn>1</mn></mrow></munderover><mrow><msub><mi>λ</mi><mrow><mi>I</mi><mo stretchy="false">(</mo><msub><mi>x</mi><mi>i</mi></msub><mo stretchy="false">)</mo></mrow></msub></mrow></mstyle><msub><mi>q</mi><mrow><mi>I</mi><mo stretchy="false">(</mo><msub><mi>x</mi><mi>i</mi></msub><mo stretchy="false">)</mo></mrow></msub><mo stretchy="false">(</mo><msub><mi>x</mi><mi>i</mi></msub><mo stretchy="false">)</mo><mo>​</mo><mo>​</mo><mstyle displaystyle="true"><munderover><mo>∏</mo><mrow><mi>i</mi><mo>=</mo><mn>1</mn></mrow><mi>m</mi></munderover><mrow><mfrac><mrow><msub><mi>ψ</mi><mrow><mi>I</mi><mo stretchy="false">(</mo><msub><mi>y</mi><mi>i</mi></msub><mo stretchy="false">)</mo></mrow></msub><msub><mi>p</mi><mrow><mi>I</mi><mo stretchy="false">(</mo><msub><mi>y</mi><mi>i</mi></msub><mo stretchy="false">)</mo></mrow></msub><mo stretchy="false">(</mo><msub><mi>y</mi><mi>i</mi></msub><mo stretchy="false">)</mo></mrow><mrow><msub><mi>q</mi><mrow><mi>I</mi><mo stretchy="false">(</mo><msub><mi>y</mi><mi>i</mi></msub><mo stretchy="false">)</mo></mrow></msub><mo stretchy="false">(</mo><msub><mi>y</mi><mi>i</mi></msub><mo stretchy="false">)</mo></mrow></mfrac></mrow></mstyle></mtd></mtr><mtr><mtd><mstyle displaystyle="true"><munderover><mo>∏</mo><mrow><mi>i</mi><mo>=</mo><mn>1</mn></mrow><mi>l</mi></munderover><mrow><msup><mrow><mo stretchy="false">[</mo><mo stretchy="false">(</mo><mn>1</mn><mo>−</mo><msub><mo>ρ</mo><mi>i</mi></msub><mo stretchy="false">)</mo><msub><mi>q</mi><mrow><mi>i</mi><mo>+</mo><mn>1</mn></mrow></msub><mo stretchy="false">(</mo><msub><mi>t</mi><mi>i</mi></msub><mo stretchy="false">)</mo><mo stretchy="false">]</mo></mrow><mrow><msub><mi>n</mi><mi>i</mi></msub></mrow></msup><msubsup><mo>ρ</mo><mi>i</mi><mrow><msub><mi>N</mi><mi>i</mi></msub></mrow></msubsup><msubsup><mi>q</mi><mrow><mi>i</mi><mo>+</mo><mn>1</mn></mrow><mrow><msub><mi>K</mi><mi>i</mi></msub></mrow></msubsup><mo stretchy="false">(</mo><msub><mi>t</mi><mi>i</mi></msub><mo stretchy="false">)</mo><msubsup><mi>p</mi><mrow><mi>i</mi><mo>+</mo><mn>1</mn></mrow><mrow><msub><mi>M</mi><mi>i</mi></msub></mrow></msubsup><mo stretchy="false">(</mo><msub><mi>t</mi><mi>i</mi></msub><mo stretchy="false">)</mo><mo>,</mo></mrow></mstyle></mtd></mtr></mtable></math></div></div><div class="label">(1)</div></div> with <a id="" class="jumplink-placeholder">&nbsp;</a><div class="formula-wrap"><div class="disp-formula" content-id=""><div class="math"><math xmlns="http://www.w3.org/1998/Math/MathML"><mtable columnalign="left" xmlns=""><mtr><mtd><mi>I</mi><mo stretchy="false">(</mo><mi>t</mi><mo stretchy="false">)</mo><mo>=</mo><mi>i</mi><mo>,</mo><mo>&nbsp;iff&nbsp;</mo><msub><mi>t</mi><mi>i</mi></msub><mo>&lt;</mo><mi>t</mi><mo>≤</mo><msub><mi>t</mi><mrow><mi>i</mi><mo>−</mo><mn>1</mn></mrow></msub><mo>,</mo></mtd></mtr><mtr><mtd><msub><mi>A</mi><mi>i</mi></msub><mo>=</mo><msqrt><mrow><msup><mrow><mo stretchy="false">(</mo><msub><mi>λ</mi><mi>i</mi></msub><mo>−</mo><msub><mi>μ</mi><mi>i</mi></msub><mo>−</mo><msub><mi>ψ</mi><mi>i</mi></msub><mo stretchy="false">)</mo></mrow><mn>2</mn></msup><mo>+</mo><mn>4</mn><msub><mi>λ</mi><mi>i</mi></msub><msub><mi>ψ</mi><mi>i</mi></msub></mrow></msqrt><mo>,</mo></mtd></mtr><mtr><mtd><msub><mi>B</mi><mi>i</mi></msub><mo>=</mo><mfrac><mrow><mo stretchy="false">(</mo><mn>1</mn><mo>−</mo><mn>2</mn><mo stretchy="false">(</mo><mn>1</mn><mo>−</mo><msub><mi>ρ</mi><mi>i</mi></msub><mo stretchy="false">)</mo><msub><mi>p</mi><mrow><mi>i</mi><mo>+</mo><mn>1</mn></mrow></msub><mo stretchy="false">(</mo><msub><mi>t</mi><mi>i</mi></msub><mo stretchy="false">)</mo><mo stretchy="false">)</mo><msub><mi>λ</mi><mi>i</mi></msub><mo>+</mo><msub><mi>μ</mi><mi>i</mi></msub><mo>+</mo><msub><mi>ψ</mi><mi>i</mi></msub></mrow><mrow><msub><mi>A</mi><mi>i</mi></msub></mrow></mfrac><mo>,</mo></mtd></mtr><mtr><mtd><msub><mi>p</mi><mi>i</mi></msub><mo stretchy="false">(</mo><mi>t</mi><mo stretchy="false">)</mo><mo>=</mo><mfrac><mrow><msub><mi>λ</mi><mi>i</mi></msub><mo>+</mo><msub><mi>μ</mi><mi>i</mi></msub><mo>+</mo><msub><mi>ψ</mi><mi>i</mi></msub><mo>−</mo><msub><mi>A</mi><mi>i</mi></msub><mfrac><mrow><mn>1</mn><mo>+</mo><msub><mi>B</mi><mi>i</mi></msub><mo>−</mo><mo stretchy="false">(</mo><mn>1</mn><mo>−</mo><msub><mi>B</mi><mi>i</mi></msub><mo stretchy="false">)</mo><msup><mi>e</mi><mrow><msub><mi>A</mi><mi>i</mi></msub><mo stretchy="false">(</mo><msub><mi>t</mi><mi>i</mi></msub><mo>−</mo><mi>t</mi><mo stretchy="false">)</mo></mrow></msup></mrow><mrow><mn>1</mn><mo>+</mo><msub><mi>B</mi><mi>i</mi></msub><mo>+</mo><mo stretchy="false">(</mo><mn>1</mn><mo>−</mo><msub><mi>B</mi><mi>i</mi></msub><mo stretchy="false">)</mo><msup><mi>e</mi><mrow><msub><mi>A</mi><mi>i</mi></msub><mo stretchy="false">(</mo><msub><mi>t</mi><mi>i</mi></msub><mo>−</mo><mi>t</mi><mo stretchy="false">)</mo></mrow></msup></mrow></mfrac></mrow><mrow><mn>2</mn><msub><mi>λ</mi><mi>i</mi></msub></mrow></mfrac><mo>,</mo></mtd></mtr><mtr><mtd><msub><mi>q</mi><mi>i</mi></msub><mo stretchy="false">(</mo><mi>t</mi><mo stretchy="false">)</mo><mo>=</mo><mfrac><mrow><mn>4</mn><msup><mi>e</mi><mrow><msub><mi>A</mi><mi>i</mi></msub><mo stretchy="false">(</mo><msub><mi>t</mi><mi>i</mi></msub><mo>−</mo><mi>t</mi><mo stretchy="false">)</mo></mrow></msup></mrow><mrow><msup><mrow><mo stretchy="false">(</mo><mn>1</mn><mo>+</mo><msub><mi>B</mi><mi>i</mi></msub><mo>+</mo><mo stretchy="false">(</mo><mn>1</mn><mo>−</mo><msub><mi>B</mi><mi>i</mi></msub><mo stretchy="false">)</mo><msup><mi>e</mi><mrow><msub><mi>A</mi><mi>i</mi></msub><mo stretchy="false">(</mo><msub><mi>t</mi><mi>i</mi></msub><mo>−</mo><mi>t</mi><mo stretchy="false">)</mo></mrow></msup><mo stretchy="false">)</mo></mrow><mn>2</mn></msup></mrow></mfrac><mo>,</mo></mtd></mtr></mtable></math></div></div></div> for <span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><mi>i</mi><mo>=</mo><mn>1</mn><mo>,</mo><mo>…</mo><mo>,</mo><mi>l</mi></mrow></math></span> , and <span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><msub><mi>p</mi><mrow><mi>l</mi><mo>+</mo><mn>1</mn></mrow></msub><mo stretchy="false">(</mo><msub><mi>t</mi><mi>l</mi></msub><mo stretchy="false">)</mo><mo>=</mo><msub><mi>q</mi><mrow><mi>l</mi><mo>+</mo><mn>1</mn></mrow></msub><mo stretchy="false">(</mo><msub><mi>t</mi><mi>l</mi></msub><mo stretchy="false">)</mo><mo>=</mo><mn>1</mn></mrow></math></span> . </div><div class="block-child-p"> If the condition is on the stem age ( <span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><msub><mi>t</mi><mrow><mi>o</mi><mi>r</mi></mrow></msub></mrow></math></span> ) instead of crown age ( <span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><msub><mi>t</mi><mrow><mi>m</mi><mi>r</mi><mi>c</mi><mi>a</mi></mrow></msub></mrow></math></span> ) ( <a reveal-id="B11" data-open="B11" class="link link-ref link-reveal xref-bibr">Gavryushkina et al. 2014</a> ), then the probability density is <a id="syv080-M2" class="jumplink-placeholder">&nbsp;</a><div class="formula-wrap"><div class="disp-formula" content-id="syv080-M2"><div class="math"><math xmlns="http://www.w3.org/1998/Math/MathML"><mtable columnalign="left" xmlns=""><mtr><mtd><mi>f</mi><mo stretchy="false">(</mo><mi class="calligraphy" mathvariant="script">T</mi><mo stretchy="false">|</mo><msub><mi>t</mi><mrow><mi>o</mi><mi>r</mi></mrow></msub><mo>=</mo><msub><mi>x</mi><mn>0</mn></msub><mo>,</mo><mi>λ</mi><mo>,</mo><mi>μ</mi><mo>,</mo><mi>ψ</mi><mo>,</mo><mi>ρ</mi><mo>,</mo><mi>t</mi><mo stretchy="false">)</mo><mo>=</mo></mtd></mtr><mtr><mtd><mfrac><mrow><msub><mi>q</mi><mn>1</mn></msub><mo stretchy="false">(</mo><msub><mi>x</mi><mn>0</mn></msub><mo stretchy="false">)</mo></mrow><mrow><mn>1</mn><mo>−</mo><msub><mi>p</mi><mn>1</mn></msub><mo stretchy="false">(</mo><msub><mi>x</mi><mn>0</mn></msub><mo stretchy="false">)</mo></mrow></mfrac><mstyle displaystyle="true"><munderover><mo>∏</mo><mrow><mi>i</mi><mo>=</mo><mn>1</mn></mrow><mi>k</mi></munderover><mrow><msub><mi>ψ</mi><mrow><mi>I</mi><mo stretchy="false">(</mo><msub><mi>y</mi><mi>i</mi></msub><mo stretchy="false">)</mo></mrow></msub></mrow></mstyle><mstyle displaystyle="true"><munderover><mo>∏</mo><mrow><mi>i</mi><mo>=</mo><mn>1</mn></mrow><mrow><mi>m</mi><mo>+</mo><mi>M</mi><mo>−</mo><mn>1</mn></mrow></munderover><mrow><msub><mi>λ</mi><mrow><mi>I</mi><mo stretchy="false">(</mo><msub><mi>x</mi><mi>i</mi></msub><mo stretchy="false">)</mo></mrow></msub></mrow></mstyle><msub><mi>q</mi><mrow><mi>I</mi><mo stretchy="false">(</mo><msub><mi>x</mi><mi>i</mi></msub><mo stretchy="false">)</mo></mrow></msub><mo stretchy="false">(</mo><msub><mi>x</mi><mi>i</mi></msub><mo stretchy="false">)</mo><mstyle displaystyle="true"><munderover><mo>∏</mo><mrow><mi>i</mi><mo>=</mo><mn>1</mn></mrow><mi>m</mi></munderover><mrow><mfrac><mrow><msub><mi>ψ</mi><mrow><mi>I</mi><mo stretchy="false">(</mo><msub><mi>y</mi><mi>i</mi></msub><mo stretchy="false">)</mo></mrow></msub><msub><mi>p</mi><mrow><mi>I</mi><mo stretchy="false">(</mo><msub><mi>y</mi><mi>i</mi></msub><mo stretchy="false">)</mo></mrow></msub><mo stretchy="false">(</mo><msub><mi>y</mi><mi>i</mi></msub><mo stretchy="false">)</mo></mrow><mrow><msub><mi>q</mi><mrow><mi>I</mi><mo stretchy="false">(</mo><msub><mi>y</mi><mi>i</mi></msub><mo stretchy="false">)</mo></mrow></msub><mo stretchy="false">(</mo><msub><mi>y</mi><mi>i</mi></msub><mo stretchy="false">)</mo></mrow></mfrac></mrow></mstyle></mtd></mtr><mtr><mtd><mstyle displaystyle="true"><munderover><mo>∏</mo><mrow><mi>i</mi><mo>=</mo><mn>1</mn></mrow><mi>l</mi></munderover><mrow><msup><mrow><mo stretchy="false">[</mo><mo stretchy="false">(</mo><mn>1</mn><mo>−</mo><msub><mi>ρ</mi><mi>i</mi></msub><mo stretchy="false">)</mo><msub><mi>q</mi><mrow><mi>i</mi><mo>+</mo><mn>1</mn></mrow></msub><mo stretchy="false">(</mo><msub><mi>t</mi><mi>i</mi></msub><mo stretchy="false">)</mo><mo stretchy="false">]</mo></mrow><mrow><msub><mi>n</mi><mi>i</mi></msub></mrow></msup><msubsup><mi>ρ</mi><mi>i</mi><mrow><msub><mi>N</mi><mi>i</mi></msub></mrow></msubsup><msubsup><mi>q</mi><mrow><mi>i</mi><mo>+</mo><mn>1</mn></mrow><mrow><msub><mi>K</mi><mi>i</mi></msub></mrow></msubsup><mo stretchy="false">(</mo><msub><mi>t</mi><mi>i</mi></msub><mo stretchy="false">)</mo><msubsup><mi>p</mi><mrow><mi>i</mi><mo>+</mo><mn>1</mn></mrow><mrow><msub><mi>M</mi><mi>i</mi></msub></mrow></msubsup><mo stretchy="false">(</mo><msub><mi>t</mi><mi>i</mi></msub><mo stretchy="false">)</mo><mo>.</mo></mrow></mstyle></mtd></mtr></mtable></math></div></div><div class="label">(2)</div></div></div><div class="block-child-p"> The joint posterior density under the FBD prior is <a id="syv080-M3" class="jumplink-placeholder">&nbsp;</a><div class="formula-wrap"><div class="disp-formula" content-id="syv080-M3"><div class="math"><math xmlns="http://www.w3.org/1998/Math/MathML"><mtable columnalign="left" xmlns=""><mtr><mtd><mi>f</mi><mo stretchy="false">(</mo><mi class="calligraphy" mathvariant="script">T</mi><mo>,</mo><mi>λ</mi><mo>,</mo><mi>μ</mi><mo>,</mo><mi>ψ</mi><mo>,</mo><mi>ρ</mi><mo>,</mo><mi>t</mi><mo>,</mo><mi>θ</mi><mo stretchy="false">|</mo><mi>D</mi><mo stretchy="false">)</mo><mo>∝</mo><mi>f</mi><mo stretchy="false">(</mo><mi>D</mi><mo stretchy="false">|</mo><mi class="calligraphy" mathvariant="script">T</mi><mo>,</mo><mi>θ</mi><mo stretchy="false">)</mo></mtd></mtr><mtr><mtd><mi>f</mi><mo stretchy="false">(</mo><mi class="calligraphy" mathvariant="script">T</mi><mo stretchy="false">|</mo><mi>t</mi><mo>,</mo><mi>λ</mi><mo>,</mo><mi>μ</mi><mo>,</mo><mi>ψ</mi><mo>,</mo><mi>ρ</mi><mo stretchy="false">)</mo><mi>f</mi><mo stretchy="false">(</mo><mi>λ</mi><mo>,</mo><mi>μ</mi><mo>,</mo><mi>ψ</mi><mo>,</mo><mi>ρ</mi><mo>,</mo><mi>t</mi><mo stretchy="false">)</mo><mi>f</mi><mo stretchy="false">(</mo><mi>θ</mi><mo stretchy="false">)</mo><mo>.</mo></mtd></mtr></mtable></math></div></div><div class="label">(3)</div></div></div><p> For λ, μ, ψ being constant and <span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><mi>l</mi><mo>=</mo><mn>1</mn></mrow></math></span> (no ρ sampling of fossils), this model simplifies to the model described in Stadler ( <a reveal-id="B46" data-open="B46" class="link link-ref link-reveal xref-bibr">2010a</a> ; Equation (5)) and <a reveal-id="B18" data-open="B18" class="link link-ref link-reveal xref-bibr">Heath et al. (2014)</a> . </p>                            <h3 scrollto-destination="115286539" id="115286539" class="section-title">Diversified Sampling of Extant Taxa</h3>
<div class="block-child-p"> To model diversified sampling of extant 
taxa, we assume that exactly one representative extant species per clade
 descending from some cutoff time <span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><msub><mi>x</mi><mrow><mi>c</mi><mi>u</mi><mi>t</mi></mrow></msub></mrow></math></span> is selected ( <a reveal-id="B29" data-open="B29" class="link link-ref link-reveal xref-bibr">Lambert and Stadler 2013</a> ; section “Higher-level phylogenies”) ( <a reveal-id="F1" data-open="F1" class="link link-reveal link-table xref-fig">Fig. 1</a> b), and state the probability of such a sampled tree as <span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><msub><mi class="calligraphy" mathvariant="script">T</mi><mrow><msub><mi>x</mi><mrow><mi>c</mi><mi>u</mi><mi>t</mi></mrow></msub></mrow></msub></mrow></math></span>
 . Such a sample maximizes the diversity, if diversity is measured as 
the total time length of the branches in the tree. We assume that the 
fossil sampling rate ψ is 0 in <span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><mo stretchy="false">[</mo><msub><mi>x</mi><mrow><mi>c</mi><mi>u</mi><mi>t</mi></mrow></msub><mo>,</mo><msub><mi>t</mi><mi>l</mi></msub><mo stretchy="false">)</mo></mrow></math></span> , and that <span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><msub><mi>ρ</mi><mi>i</mi></msub><mo>=</mo><mn>0</mn></mrow></math></span> for <span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><msub><mi>t</mi><mi>i</mi></msub><mo>∈</mo><mo stretchy="false">[</mo><msub><mi>x</mi><mrow><mi>c</mi><mi>u</mi><mi>t</mi></mrow></msub><mo>,</mo><msub><mi>t</mi><mi>l</mi></msub><mo stretchy="false">)</mo></mrow></math></span> . Following the notation above, the tree <span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><msub><mi class="calligraphy" mathvariant="script">T</mi><mrow><msub><mi>x</mi><mrow><mi>c</mi><mi>u</mi><mi>t</mi></mrow></msub></mrow></msub></mrow></math></span> has <span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><msub><mi>N</mi><mi>l</mi></msub></mrow></math></span> extant sampled tips and <span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><msub><mi>η</mi><mi>l</mi></msub></mrow></math></span> extant tips not sampled, so that the overall number of extant taxa in the complete tree is <span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><msub><mi>N</mi><mi>l</mi></msub><mo>+</mo><msub><mi>η</mi><mi>l</mi></msub></mrow></math></span> . For example, in <a reveal-id="F1" data-open="F1" class="link link-reveal link-table xref-fig">Figure 1</a> b, <span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><msub><mi>N</mi><mn>3</mn></msub><mo>=</mo><mn>3</mn></mrow></math></span> and <span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><msub><mi>η</mi><mn>3</mn></msub><mo>=</mo><mn>2</mn></mrow></math></span> . Let lineage <span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mi xmlns="">i</mi></math></span> ( <span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><mi>i</mi><mo>=</mo><mn>1</mn><mo>,</mo><mo>…</mo><mo>,</mo><msub><mi>N</mi><mi>l</mi></msub></mrow></math></span> ) at time <span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><msub><mi>x</mi><mrow><mi>c</mi><mi>u</mi><mi>t</mi></mrow></msub></mrow></math></span> have <span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><msub><mi>η</mi><mrow><msub><mi>l</mi><mi>i</mi></msub></mrow></msub><mo>+</mo><mn>1</mn></mrow></math></span> descendants at present, with <span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><mstyle displaystyle="true"><munderover><mo>∑</mo><mrow><mi>i</mi><mo>=</mo><mn>1</mn></mrow><mrow><msub><mi>N</mi><mi>l</mi></msub></mrow></munderover><mrow><msub><mi>η</mi><mrow><msub><mi>l</mi><mi>i</mi></msub></mrow></msub></mrow></mstyle><mo>=</mo><msub><mi>η</mi><mi>l</mi></msub></mrow></math></span> . Let <span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><mi>p</mi><mo stretchy="false">(</mo><mi>k</mi><mo stretchy="false">|</mo><msub><mi>x</mi><mrow><mi>c</mi><mi>u</mi><mi>t</mi></mrow></msub><mo stretchy="false">)</mo></mrow></math></span> be the probability that a lineage after time <span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><msub><mi>x</mi><mrow><mi>c</mi><mi>u</mi><mi>t</mi></mrow></msub></mrow></math></span> has <span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mi xmlns="">k</mi></math></span> descendants. Then, in analogy to ( <a reveal-id="B50" data-open="B50" class="link link-ref link-reveal xref-bibr">Stadler and Bokma 2013</a> ), <a id="syv080-M4" class="jumplink-placeholder">&nbsp;</a><div class="formula-wrap"><div class="disp-formula" content-id="syv080-M4"><div class="math"><math xmlns="http://www.w3.org/1998/Math/MathML"><mtable columnalign="left" xmlns=""><mtr><mtd><mi>f</mi><mo stretchy="false">(</mo><msub><mi class="calligraphy" mathvariant="script">T</mi><mrow><msub><mi>x</mi><mrow><mi>c</mi><mi>u</mi><mi>t</mi></mrow></msub></mrow></msub><mo stretchy="false">)</mo><mo>=</mo><mi>f</mi><mo stretchy="false">(</mo><mi class="calligraphy" mathvariant="script">T</mi><mo stretchy="false">)</mo><mstyle displaystyle="true"><munderover><mo>∏</mo><mrow><mi>i</mi><mo>=</mo><mn>1</mn></mrow><mrow><msub><mi>N</mi><mi>l</mi></msub></mrow></munderover><mi>p</mi></mstyle><mo stretchy="false">(</mo><msub><mi>η</mi><mrow><msub><mi>l</mi><mi>i</mi></msub></mrow></msub><mo>+</mo><mn>1</mn><mo stretchy="false">|</mo><msub><mi>x</mi><mrow><mi>c</mi><mi>u</mi><mi>t</mi></mrow></msub><mo stretchy="false">)</mo><mo>/</mo><mi>p</mi><mo stretchy="false">(</mo><mn>1</mn><mo stretchy="false">|</mo><msub><mi>x</mi><mrow><mi>c</mi><mi>u</mi><mi>t</mi></mrow></msub><mo stretchy="false">)</mo></mtd></mtr><mtr><mtd><mo>=</mo><mi>f</mi><mo stretchy="false">(</mo><mi class="calligraphy" mathvariant="script">T</mi><mo stretchy="false">)</mo><msup><mrow><mo>(</mo><mrow><mn>1</mn><mo>−</mo><mfrac><mn>1</mn><mrow><mi>F</mi><mo stretchy="false">(</mo><msub><mi>x</mi><mrow><mi>c</mi><mi>u</mi><mi>t</mi></mrow></msub><mo stretchy="false">)</mo></mrow></mfrac></mrow><mo>)</mo></mrow><mrow><msub><mi>η</mi><mi>l</mi></msub></mrow></msup><mo>,</mo></mtd></mtr></mtable></math></div></div><div class="label">(4)</div></div> where <span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><mi>f</mi><mo stretchy="false">(</mo><mi class="calligraphy" mathvariant="script">T</mi><mo stretchy="false">)</mo></mrow></math></span> is the probability density of the tree on <span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><msub><mi>N</mi><mi>l</mi></msub></mrow></math></span> tips assuming complete sampling (i.e., using Equation ( <a href="#syv080-M1" class="sectionLink xref-sec">1</a> ) with <span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><msub><mi>ρ</mi><mi>l</mi></msub><mo>=</mo><mn>1</mn></mrow></math></span> ), and where the last equality follows from Lambert and Stadler ( <a reveal-id="B29" data-open="B29" class="link link-ref link-reveal xref-bibr">2013</a> ; Equation (1)). </div><div class="block-child-p"><span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><mi>F</mi><mo stretchy="false">(</mo><mi>t</mi><mo stretchy="false">)</mo></mrow></math></span> is calculated as follows (Smrckova and Stadler, personal communication). For <span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mi xmlns="">t</mi></math></span> in <span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><mo stretchy="false">[</mo><msub><mi>t</mi><mrow><mi>i</mi><mo>−</mo><mn>1</mn></mrow></msub><mo>,</mo><msub><mi>t</mi><mi>i</mi></msub><mo stretchy="false">)</mo></mrow></math></span> , and denoting <span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mi xmlns="">t</mi></math></span> by <span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><msub><mi>t</mi><mrow><mi>i</mi><mo>−</mo><mn>1</mn></mrow></msub></mrow></math></span> , we have <a id="syv080-M5" class="jumplink-placeholder">&nbsp;</a><div class="formula-wrap"><div class="disp-formula" content-id="syv080-M5"><div class="math"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><mi>F</mi><mo stretchy="false">(</mo><mi>t</mi><mo stretchy="false">)</mo><mo>=</mo><mn>1</mn><mo>+</mo><mstyle displaystyle="true"><munderover><mo>∑</mo><mrow><mi>k</mi><mo>=</mo><mi>l</mi></mrow><mi>i</mi></munderover><mi>G</mi></mstyle><mo stretchy="false">(</mo><msub><mi>t</mi><mi>k</mi></msub><mo stretchy="false">)</mo><mo>,</mo></mrow></math></div></div><div class="label">(5)</div></div> with <a id="syv080-M6" class="jumplink-placeholder">&nbsp;</a><div class="formula-wrap"><div class="disp-formula" content-id="syv080-M6"><div class="math"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><mi>G</mi><mo stretchy="false">(</mo><msub><mi>t</mi><mi>k</mi></msub><mo stretchy="false">)</mo><mo>=</mo><mfrac><mrow><msub><mi>λ</mi><mi>k</mi></msub></mrow><mrow><msub><mi>λ</mi><mi>k</mi></msub><mo>−</mo><msub><mi>μ</mi><mi>k</mi></msub></mrow></mfrac><mrow><mo>(</mo><mrow><msup><mi>e</mi><mrow><mo stretchy="false">(</mo><msub><mi>λ</mi><mi>k</mi></msub><mo>−</mo><msub><mi>μ</mi><mi>k</mi></msub><mo stretchy="false">)</mo><mo stretchy="false">(</mo><msub><mi>t</mi><mrow><mi>k</mi><mo>−</mo><mn>1</mn></mrow></msub><mo>−</mo><msub><mi>t</mi><mi>k</mi></msub><mo stretchy="false">)</mo></mrow></msup><mo>−</mo><mn>1</mn></mrow><mo>)</mo></mrow><msup><mi>e</mi><mrow><mstyle displaystyle="true"><munderover><mo>∑</mo><mrow><mi>j</mi><mo>=</mo><mi>k</mi><mo>+</mo><mn>1</mn></mrow><mi>l</mi></munderover><mrow><mo stretchy="false">(</mo><msub><mi>λ</mi><mi>j</mi></msub><mo>−</mo><msub><mi>μ</mi><mi>j</mi></msub><mo stretchy="false">)</mo><mo stretchy="false">(</mo><msub><mi>t</mi><mrow><mi>j</mi><mo>−</mo><mn>1</mn></mrow></msub><mo>−</mo><msub><mi>t</mi><mi>j</mi></msub><mo stretchy="false">)</mo></mrow></mstyle></mrow></msup><mo>.</mo></mrow></math></div></div><div class="label">(6)</div></div> Note that for <span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><mi>l</mi><mo>=</mo><mi>k</mi></mrow></math></span> , we have <span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><msup><mi>e</mi><mrow><mstyle displaystyle="true"><munderover><mo>∑</mo><mrow><mi>j</mi><mo>=</mo><mi>l</mi><mo>+</mo><mn>1</mn></mrow><mi>l</mi></munderover><mrow><mo stretchy="false">(</mo><msub><mi>λ</mi><mi>j</mi></msub><mo>−</mo><msub><mi>μ</mi><mi>j</mi></msub><mo stretchy="false">)</mo><mo stretchy="false">(</mo><msub><mi>t</mi><mrow><mi>j</mi><mo>−</mo><mn>1</mn></mrow></msub><mo>−</mo><msub><mi>t</mi><mi>j</mi></msub><mo stretchy="false">)</mo></mrow></mstyle></mrow></msup><mo>=</mo><mn>1</mn></mrow></math></span> . </div><div class="block-child-p"> For λ and μ being constant in <span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><mo stretchy="false">[</mo><msub><mi>x</mi><mrow><mi>c</mi><mi>u</mi><mi>t</mi></mrow></msub><mo>,</mo><msub><mi>t</mi><mi>l</mi></msub><mo stretchy="false">)</mo></mrow></math></span> ( <span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><msub><mi>t</mi><mi>l</mi></msub><mo>=</mo><mn>0</mn></mrow></math></span> ) and <span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><msub><mi>t</mi><mrow><mi>l</mi><mo>−</mo><mn>1</mn></mrow></msub><mo>=</mo><msub><mi>x</mi><mrow><mi>c</mi><mi>u</mi><mi>t</mi></mrow></msub></mrow></math></span> , then <span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><mi>F</mi><mo stretchy="false">(</mo><msub><mi>x</mi><mrow><mi>c</mi><mi>u</mi><mi>t</mi></mrow></msub><mo stretchy="false">)</mo><mo>=</mo><mn>1</mn><mo>+</mo><mi>G</mi><mo stretchy="false">(</mo><msub><mi>t</mi><mi>l</mi></msub><mo stretchy="false">)</mo></mrow></math></span> , and <a id="syv080-M7" class="jumplink-placeholder">&nbsp;</a><div class="formula-wrap"><div class="disp-formula" content-id="syv080-M7"><div class="math"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><mi>f</mi><mo stretchy="false">(</mo><msub><mi class="calligraphy" mathvariant="script">T</mi><mrow><msub><mi>x</mi><mrow><mi>c</mi><mi>u</mi><mi>t</mi></mrow></msub></mrow></msub><mo stretchy="false">)</mo><mo>=</mo><mi>f</mi><mo stretchy="false">(</mo><mi class="calligraphy" mathvariant="script">T</mi><mo stretchy="false">)</mo><msup><mrow><mrow><mo>(</mo><mrow><mfrac><mrow><msub><mi>λ</mi><mi>l</mi></msub><mo stretchy="false">(</mo><mn>1</mn><mo>−</mo><msup><mi>e</mi><mrow><mo>−</mo><mo stretchy="false">(</mo><msub><mi>λ</mi><mi>l</mi></msub><mo>−</mo><msub><mi>μ</mi><mi>l</mi></msub><mo stretchy="false">)</mo><msub><mi>t</mi><mrow><mi>l</mi><mo>−</mo><mn>1</mn></mrow></msub></mrow></msup><mo stretchy="false">)</mo></mrow><mrow><msub><mi>λ</mi><mi>l</mi></msub><mo>−</mo><msub><mi>μ</mi><mi>l</mi></msub><msup><mi>e</mi><mrow><mo>−</mo><mo stretchy="false">(</mo><msub><mi>λ</mi><mi>l</mi></msub><mo>−</mo><msub><mi>μ</mi><mi>l</mi></msub><mo stretchy="false">)</mo><msub><mi>t</mi><mrow><mi>l</mi><mo>−</mo><mn>1</mn></mrow></msub></mrow></msup></mrow></mfrac></mrow><mo>)</mo></mrow></mrow><mrow><msub><mi>η</mi><mi>l</mi></msub></mrow></msup><mo>.</mo></mrow></math></div></div><div class="label">(7)</div></div></div><p>
 For inference, we typically consider trees where the samples (i.e., 
nodes corresponding to observed fossils or extant taxa) are labeled, 
instead of oriented trees. To convert an oriented tree into a labeled 
tree, we multiply the probability density of an oriented FBD tree (i.e.,
 Equation ( <a href="#syv080-M1" class="sectionLink xref-sec">1</a> ) or ( <a href="#syv080-M7" class="sectionLink xref-sec">7</a> )) with <span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><msup><mn>2</mn><mrow><mo stretchy="false">(</mo><mi>m</mi><mo>+</mo><mi>M</mi><mo>−</mo><mn>1</mn><mo stretchy="false">)</mo></mrow></msup><mo>/</mo><mo stretchy="false">(</mo><mi>m</mi><mo>+</mo><mi>M</mi><mo>+</mo><mi>k</mi><mo>+</mo><mi>K</mi><mo stretchy="false">)</mo><mo>!</mo></mrow></math></span> . Rather than sampling <span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><msub><mi>λ</mi><mi>i</mi></msub></mrow></math></span> , <span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><msub><mi>μ</mi><mi>i</mi></msub></mrow></math></span> , and <span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><msub><mi>ψ</mi><mi>i</mi></msub></mrow></math></span> , we operate on <span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><msub><mi>d</mi><mi>i</mi></msub><mo>=</mo><msub><mi>λ</mi><mi>i</mi></msub><mo>−</mo><msub><mi>μ</mi><mi>i</mi></msub></mrow></math></span> (net diversification), <span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><msub><mi>r</mi><mi>i</mi></msub><mo>=</mo><msub><mi>μ</mi><mi>i</mi></msub><mo>/</mo><msub><mi>λ</mi><mi>i</mi></msub></mrow></math></span> (turnover), <span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><msub><mi>s</mi><mi>i</mi></msub><mo>=</mo><msub><mi>ψ</mi><mi>i</mi></msub><mo>/</mo><mo stretchy="false">(</mo><msub><mi>μ</mi><mi>i</mi></msub><mo>+</mo><msub><mi>ψ</mi><mi>i</mi></msub><mo stretchy="false">)</mo></mrow></math></span> (fossil sampling proportion) ( <span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><mi>i</mi><mo>=</mo><mn>1</mn><mo>,</mo><mo>…</mo><mo>,</mo><mi>l</mi></mrow></math></span> ). With the sampling probabilities (the ρ values), there are potentially <span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><mn>4</mn><mi>l</mi></mrow></math></span> parameters in the full FBD model ( <a reveal-id="F1" data-open="F1" class="link link-reveal link-table xref-fig">Fig. 1</a>
 ). However, it is usually not possible to estimate the sampling 
probabilities of a birth–death model independently of the speciation and
 extinction rates ( <a reveal-id="B49" data-open="B49" class="link link-ref link-reveal xref-bibr">Stadler 2012</a> ). In the empirical analyses, we fix the sampling probability of extant taxa ( <span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><msub><mi>ρ</mi><mi>l</mi></msub></mrow></math></span>
 ) to a value based on estimates of the current species diversity. 
Further, we set all other ρ values to zero and instead use the 
fossilization rate <span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><msub><mi>ψ</mi><mi>i</mi></msub></mrow></math></span> in each interval <span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mi xmlns="">i</mi></math></span> to model the product of fossilization rate and fossil sampling probability in that time interval. We thus only estimate <span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><mn>3</mn><mi>l</mi></mrow></math></span> parameters in our analyses. When assuming constant birth, death, and fossilization rates ( <span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><mi>l</mi><mo>=</mo><mn>1</mn></mrow></math></span> ), the number of parameters reduces to three. </p>                            <h3 scrollto-destination="115286544" id="115286544" class="section-title">MCMC Proposals for FBD Trees</h3>
<p> To allow Markov chain Monte Carlo (MCMC) sampling of the FBD process
 model, we implemented two reversible-jump Markov chain Monte Carlo 
(rjMCMC) proposals, the “add-branch” and “delete-branch” moves of <a reveal-id="B18" data-open="B18" class="link link-ref link-reveal xref-bibr">Heath et al. (2014)</a> (see also <a reveal-id="B34" data-open="B34" class="link link-ref link-reveal xref-bibr">Lewis et al. 2005</a> ), in MrBayes ( <a reveal-id="B43" data-open="B43" class="link link-ref link-reveal xref-bibr">Ronquist et al. 2012a</a>
 ). The add-branch move selects an ancestral fossil and then adds a 
speciation event and a branch leading to that fossil, effectively 
changing the ancestral fossil to one that represents a sampled extinct 
lineage ( <a reveal-id="F2" data-open="F2" class="link link-reveal link-table xref-fig">Fig. 2</a> ). The delete-branch move positions a tip fossil on its sibling lineage by deleting the branch leading to it ( <a reveal-id="F2" data-open="F2" class="link link-reveal link-table xref-fig">Fig. 2</a>
 ). It is not allowed if the tip fossil is younger than its sibling. The
 two moves are chosen with equal probability, but we abort the 
add-branch move if there is no ancestral fossil, or abort the 
delete-branch move if there is no fossil tip. For example, on the left 
tree of <a reveal-id="F2" data-open="F2" class="link link-reveal link-table xref-fig">Figure 2</a> , we have one fossil tip ( <span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><msub><mi>F</mi><mn>1</mn></msub></mrow></math></span> ) and one fossil ancestor ( <span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><msub><mi>F</mi><mn>2</mn></msub></mrow></math></span>
 ), whereas on the right tree, obtained after an add-branch move, both 
fossils are tips. The Hastings ratios and Jacobian factors for each move
 are detailed below ( <span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><msup><mi>k</mi><mo>′</mo></msup></mrow></math></span> is the number of fossil ancestors, and <span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><msup><mi>m</mi><mo>′</mo></msup></mrow></math></span> is the number of fossil tips in the tree). </p>                        <a id="115286546" scrollto-destination="115286546"></a>
<div id="F2" data-id="F2" class="fig fig-section"><div class="label fig-label">Figure 2.</div><div class="graphic-wrap"><img class="content-image" src="m_syv080f2.png" alt="F1 and F2 are two fossils, T1 and T2 are extant taxa. F2 is either split from the lineage of T1 using the add-branch move (left to right) or merged to the lineage of T1 using the delete-branch move (right to left). The two crosses on lineages of F1 and T2 at time yf represent two alternative positions to which we could move fossil F2 using a SPR move. " path-from-xml="syv080f2.tiff"><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="https://academic.oup.com/view-large/figure/115286546/syv080f2.tiff" path-from-xml="syv080f2.tiff" target="_blank">View large</a><a section="115286546" href="https://oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/sysbio/65/2/10.1093_sysbio_syv080/3/syv080f2.png?Expires=2147483647&amp;Signature=YAEOf7vRV9RksuTfDzMBzfqGs5xn-u8dAzysjy7qJzVDDuFDWfeaG1rPwPjjbc3-qSh8J6lypmZTAW18UEbKO02WlMPq1tur7lQMnxNR90a5-8mOSMouHA2i1HyztS8s03GvrAHyzIESAmnTN-9LX%7Es72XA9yJOGMnz59q4GgCd3r7aJMytianMZ3Tu72i-1Lhy1d9sjhIboNT8j-nU4xmhgVoZ4uCUW7JGBzlwnCTtX89Brt2r1DU%7Eid846EChQJmLEpMOSfnvfkAeSXJvQrixoo6dIumauum7rbI0Q4Jig%7Ez4STVeMdvqocEzvw-9vOM2T-p1mbUudjg3yW%7EIwDg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=115286546&amp;ar=2427164&amp;xsltPath=%7E/UI/app/XSLT&amp;imagename=" class="download-slide" path-from-xml="syv080f2.tiff" target="">Download slide</a></div></div><div class="caption fig-caption"><p><span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><msub><mi>F</mi><mn>1</mn></msub></mrow></math></span> and <span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><msub><mi>F</mi><mn>2</mn></msub></mrow></math></span> are two fossils, <span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><msub><mi>T</mi><mn>1</mn></msub></mrow></math></span> and <span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><msub><mi>T</mi><mn>2</mn></msub></mrow></math></span> are extant taxa. <span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><msub><mi>F</mi><mn>2</mn></msub></mrow></math></span> is either split from the lineage of <span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><msub><mi>T</mi><mn>1</mn></msub></mrow></math></span> using the add-branch move (left to right) or merged to the lineage of <span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><msub><mi>T</mi><mn>1</mn></msub></mrow></math></span> using the delete-branch move (right to left). The two crosses on lineages of <span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><msub><mi>F</mi><mn>1</mn></msub></mrow></math></span> and <span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><msub><mi>T</mi><mn>2</mn></msub></mrow></math></span> at time <span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><msub><mi>y</mi><mi>f</mi></msub></mrow></math></span> represent two alternative positions to which we could move fossil <span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><msub><mi>F</mi><mn>2</mn></msub></mrow></math></span> using a SPR move. </p></div></div><div content-id="F2" class="fig fig-modal reveal-modal" style="display: block;"><div class="label fig-label">Figure 2.</div><div class="graphic-wrap"><img class="content-image" src="m_syv080f2.png" alt="F1 and F2 are two fossils, T1 and T2 are extant taxa. F2 is either split from the lineage of T1 using the add-branch move (left to right) or merged to the lineage of T1 using the delete-branch move (right to left). The two crosses on lineages of F1 and T2 at time yf represent two alternative positions to which we could move fossil F2 using a SPR move. " path-from-xml="syv080f2.tiff"><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="https://academic.oup.com/view-large/figure/115286546/syv080f2.tiff" path-from-xml="syv080f2.tiff" target="_blank">View large</a><a section="115286546" href="https://oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/sysbio/65/2/10.1093_sysbio_syv080/3/syv080f2.png?Expires=2147483647&amp;Signature=YAEOf7vRV9RksuTfDzMBzfqGs5xn-u8dAzysjy7qJzVDDuFDWfeaG1rPwPjjbc3-qSh8J6lypmZTAW18UEbKO02WlMPq1tur7lQMnxNR90a5-8mOSMouHA2i1HyztS8s03GvrAHyzIESAmnTN-9LX%7Es72XA9yJOGMnz59q4GgCd3r7aJMytianMZ3Tu72i-1Lhy1d9sjhIboNT8j-nU4xmhgVoZ4uCUW7JGBzlwnCTtX89Brt2r1DU%7Eid846EChQJmLEpMOSfnvfkAeSXJvQrixoo6dIumauum7rbI0Q4Jig%7Ez4STVeMdvqocEzvw-9vOM2T-p1mbUudjg3yW%7EIwDg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=115286546&amp;ar=2427164&amp;xsltPath=%7E/UI/app/XSLT&amp;imagename=" class="download-slide" path-from-xml="syv080f2.tiff" target="">Download slide</a></div></div><div class="caption fig-caption"><p><span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><msub><mi>F</mi><mn>1</mn></msub></mrow></math></span> and <span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><msub><mi>F</mi><mn>2</mn></msub></mrow></math></span> are two fossils, <span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><msub><mi>T</mi><mn>1</mn></msub></mrow></math></span> and <span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><msub><mi>T</mi><mn>2</mn></msub></mrow></math></span> are extant taxa. <span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><msub><mi>F</mi><mn>2</mn></msub></mrow></math></span> is either split from the lineage of <span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><msub><mi>T</mi><mn>1</mn></msub></mrow></math></span> using the add-branch move (left to right) or merged to the lineage of <span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><msub><mi>T</mi><mn>1</mn></msub></mrow></math></span> using the delete-branch move (right to left). The two crosses on lineages of <span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><msub><mi>F</mi><mn>1</mn></msub></mrow></math></span> and <span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><msub><mi>T</mi><mn>2</mn></msub></mrow></math></span> at time <span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><msub><mi>y</mi><mi>f</mi></msub></mrow></math></span> represent two alternative positions to which we could move fossil <span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><msub><mi>F</mi><mn>2</mn></msub></mrow></math></span> using a SPR move. </p></div></div>                            <h4 scrollto-destination="115286547" id="115286547" class="section-title">Add-branch move</h4>
<p> Choose an ancestral fossil with probability <span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><mn>1</mn><mo>/</mo><msup><mi>k</mi><mo>′</mo></msup></mrow></math></span> . Choose a branch length for the fossil from a uniform distribution on (0, <span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><msub><mi>c</mi><mi>f</mi></msub><mo>−</mo><msub><mi>y</mi><mi>f</mi></msub></mrow></math></span> ), where <span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><msub><mi>c</mi><mi>f</mi></msub></mrow></math></span> is the age of the ancestral node of the fossil and <span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><msub><mi>y</mi><mi>f</mi></msub></mrow></math></span> is the age of the fossil. For the corresponding delete-branch move, choose a tip fossil (probability <span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><mfrac><mn>1</mn><mrow><msup><mi>m</mi><mo>′</mo></msup><mo>+</mo><mn>1</mn></mrow></mfrac></mrow></math></span> ) and put it on the sister branch without changing the age of the fossil. The Hastings ratio is <span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><mfrac><mrow><msup><mi>k</mi><mo>′</mo></msup></mrow><mrow><msup><mi>m</mi><mo>′</mo></msup><mo>+</mo><mn>1</mn></mrow></mfrac></mrow></math></span> and the Jacobian is <span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><msub><mi>c</mi><mi>f</mi></msub><mo>−</mo><msub><mi>y</mi><mi>f</mi></msub></mrow></math></span> . </p>                            <h4 scrollto-destination="115286549" id="115286549" class="section-title">Delete-branch move</h4>
<p> Choose a tip fossil with probability <span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><mn>1</mn><mo>/</mo><msup><mi>m</mi><mo>′</mo></msup></mrow></math></span>
 . Put the fossil on the sister branch without changing its age. For the
 corresponding add-branch move, choose an ancestral fossil (probability <span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><mfrac><mn>1</mn><mrow><msup><mi>k</mi><mo>′</mo></msup><mo>+</mo><mn>1</mn></mrow></mfrac></mrow></math></span> ) and obtain the length of its subtending branch from a uniform distribution on (0, <span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><msub><mi>c</mi><mi>f</mi></msub><mo>−</mo><msub><mi>y</mi><mi>f</mi></msub></mrow></math></span> ), where <span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><msub><mi>c</mi><mi>f</mi></msub></mrow></math></span> is the age of the ancestral node of the fossil and <span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><msub><mi>y</mi><mi>f</mi></msub></mrow></math></span> is the age of the fossil. The Hastings ratio is <span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><mfrac><mrow><msup><mi>m</mi><mo>′</mo></msup></mrow><mrow><msup><mi>k</mi><mo>′</mo></msup><mo>+</mo><mn>1</mn></mrow></mfrac></mrow></math></span> , and the Jacobian is <span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><mfrac><mn>1</mn><mrow><msub><mi>c</mi><mi>f</mi></msub><mo>−</mo><msub><mi>y</mi><mi>f</mi></msub></mrow></mfrac></mrow></math></span> . </p><p>
 We also extended two topology proposals, the node-slider and the 
subtree-pruning-and-regrafting (SPR) moves, to update the position of 
ancestral fossils on the FBD tree. These proposals do not change the 
number of branches. The node-slider move changes the position of an 
ancestral fossil along the branch on which it sits using a sliding 
window with reflection. This move does not change the topology, and its 
Hastings ratio is 1. For example, in the tree to the left, <span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><msub><mi>F</mi><mn>2</mn></msub></mrow></math></span> can be moved along the subtending branch of <span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><msub><mi>T</mi><mn>1</mn></msub></mrow></math></span> with the upper boundary of min( <span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><msub><mi>c</mi><mi>f</mi></msub></mrow></math></span> , <span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><msub><mi>u</mi><mi>f</mi></msub></mrow></math></span> ) and the lower boundary <span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><msub><mi>l</mi><mi>f</mi></msub></mrow></math></span> , where <span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><msub><mi>u</mi><mi>f</mi></msub></mrow></math></span> and <span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><msub><mi>l</mi><mi>f</mi></msub></mrow></math></span> are the upper and lower ages of fossil <span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><msub><mi>F</mi><mn>2</mn></msub></mrow></math></span> ( <a reveal-id="F2" data-open="F2" class="link link-reveal link-table xref-fig">Fig. 2</a> ). </p><p> The SPR proposal can move an ancestral fossil to another branch without changing its age ( <a reveal-id="F2" data-open="F2" class="link link-reveal link-table xref-fig">Fig. 2</a> ). One of the two candidate branches crossing time <span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><msub><mi>y</mi><mi>f</mi></msub></mrow></math></span> (lineage <span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><msub><mi>F</mi><mn>1</mn></msub></mrow></math></span> and <span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><msub><mi>T</mi><mn>2</mn></msub></mrow></math></span> ) is chosen with equal probabilities for attaching <span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><msub><mi>F</mi><mn>2</mn></msub></mrow></math></span> at time <span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><msub><mi>y</mi><mi>f</mi></msub></mrow></math></span>
 . The Hastings ratio is 1. Note that the nearest-neighbor-interchange 
(NNI) move is a special case of SPR. In this example, moving <span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><msub><mi>F</mi><mn>2</mn></msub></mrow></math></span> to the branch subtending <span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><msub><mi>F</mi><mn>1</mn></msub></mrow></math></span> is a NNI move. </p>                            <h2 scrollto-destination="115286553" id="115286553" class="section-title"><span class="small-caps">Simulations</span></h2>
<p> We performed two rounds of simulations, primarily to verify our 
implementation of the FBD priors (see also Supplementary Table S3 on 
Dryad at <a class="link link-uri" href="http://dx.doi.org/10.5061/dryad.26820" target="">http://dx.doi.org/10.5061/dryad.26820</a> for empirical validation of our implementation against BEAST2 ( <a reveal-id="B5" data-open="B5" class="link link-ref link-reveal xref-bibr">Bouckaert et al. 2014</a> ; <a reveal-id="B11" data-open="B11" class="link link-ref link-reveal xref-bibr">Gavryushkina et al. 2014</a>
 ), using models available in both softwares). Therefore, we focus on 
scenarios where the fossils provide a substantial amount of dating 
information. However, we also look at the behavior of the model in 
various situations where there is a slight mismatch between the 
generating model and the model used for inference. </p>                            <h3 scrollto-destination="115286555" id="115286555" class="section-title">Simulation I: No Sequence Data</h3>
<p> The TreeSim package in R ( <a reveal-id="B38" data-open="B38" class="link link-ref link-reveal xref-bibr">R Development Core Team 2009</a> ; <a reveal-id="B47" data-open="B47" class="link link-ref link-reveal xref-bibr">Stadler 2010b</a>
 ) was used to simulate a complete tree under the constant-rate 
birth–death process with birth rate λ = 0.3, death rate μ = 0.2, and 100
 extant taxa. We then sampled extant taxa and fossils in the complete 
tree with one of the following four strategies: </p><ol class="order"><li><p>Sample extant taxa with probability ρ = 0.5, and sample fossils with a constant rate ψ = 0.1.</p></li><li><p> Sample extant taxa with probability <span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><msub><mi>ρ</mi><mn>2</mn></msub></mrow></math></span> = 0.5, and sample fossils with a constant rate ψ = 0.1. One additional time for sampling fossils is set at <span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><msub><mi>t</mi><mn>1</mn></msub></mrow></math></span> = 10 in the past, where each lineage is sampled with probability <span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><msub><mi>ρ</mi><mn>1</mn></msub></mrow></math></span> = 0.5. </p></li><li><p> Sample extant taxa with proportion 0.5 to maximize diversity ( <span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><msub><mi>x</mi><mrow><mi>c</mi><mi>u</mi><mi>t</mi></mrow></msub><mo>=</mo><msub><mi>t</mi><mn>1</mn></msub></mrow></math></span> is adjusted to sample half of the extant taxa, and <span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><msub><mi>ρ</mi><mn>1</mn></msub><mo>=</mo><mn>0</mn></mrow></math></span> ), and sample fossils with a constant rate <span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><msub><mi>ψ</mi><mn>1</mn></msub></mrow></math></span>  = 0.1 prior to <span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><msub><mi>t</mi><mn>1</mn></msub></mrow></math></span> , and <span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><msub><mi>ψ</mi><mn>2</mn></msub><mo>=</mo><mn>0</mn></mrow></math></span> in (0, <span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><msub><mi>t</mi><mn>1</mn></msub></mrow></math></span> ). </p></li><li><p> Sample extant taxa with proportion 0.5 to maximize diversity ( <span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><msub><mi>x</mi><mrow><mi>c</mi><mi>u</mi><mi>t</mi></mrow></msub><mo>=</mo><msub><mi>t</mi><mn>2</mn></msub></mrow></math></span> is adjusted to sample half of the extant taxa, and <span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><msub><mi>ρ</mi><mn>2</mn></msub><mo>=</mo><mn>0</mn></mrow></math></span> ), and sample fossils with a constant rate <span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><msub><mi>ψ</mi><mn>1</mn></msub><mo>=</mo><msub><mi>ψ</mi><mn>2</mn></msub></mrow></math></span> = 0.1, and <span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><msub><mi>ψ</mi><mn>3</mn></msub><mo>=</mo><mn>0</mn></mrow></math></span> in (0, <span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><msub><mi>t</mi><mn>2</mn></msub></mrow></math></span> ). At <span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><msub><mi>t</mi><mn>1</mn></msub></mrow></math></span> = 10 ( <span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><mo>&gt;</mo><msub><mi>t</mi><mn>2</mn></msub></mrow></math></span> ), fossils are sampled with probability <span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><msub><mi>ρ</mi><mn>1</mn></msub></mrow></math></span> = 0.5. </p></li></ol><p>
 For each generated FBD tree, we first fixed the tree topology and 
branch lengths, and ran MCMC without sequence data to sample net 
diversification ( <span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><mi>d</mi><mo>=</mo><mi>λ</mi><mo>−</mo><mi>μ</mi></mrow></math></span> ), turnover ( <span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><mi>r</mi><mo>=</mo><mi>μ</mi><mo>/</mo><mi>λ</mi></mrow></math></span> ), and fossil sampling proportion ( <span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><mi>s</mi><mo>=</mo><mi>ψ</mi><mo>/</mo><mo stretchy="false">(</mo><mi>μ</mi><mo>+</mo><mi>ψ</mi><mo stretchy="false">)</mo></mrow></math></span>
 ) from the FBD prior in MrBayes. Four chains (one cold, three heated) 
were run for 1 million iterations and sampled every 200 iterations, with
 the first 25% samples discarded as burn-in. The following hyperpriors 
were used for the parameters: <span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><mi>d</mi><mo>∼</mo></mrow></math></span>
 Exponential(10) (we use the rate parameterization for the exponential 
distribution throughout the article, unless noted otherwise), <span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><mi>r</mi><mo>∼</mo></mrow></math></span> Beta(1, 1), <span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><mi>s</mi><mo>∼</mo></mrow></math></span> Beta(1, 1), and <span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><msub><mi>ρ</mi><mi>i</mi></msub></mrow></math></span> 's were fixed to their true values. We generated 1000 replicates for each sampling strategy (4000 FBD trees in total). </p><p>
 We then added the tree proposals described above to also sample the FBD
 trees from the prior (without sequence data). In these runs, the prior 
for <span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><msub><mi>t</mi><mrow><mi>m</mi><mi>r</mi><mi>c</mi><mi>a</mi></mrow></msub></mrow></math></span> and for the fossil ages were fixed to their true values, and the chain length was enlarged to 2 million. </p><p>
 The mean value of the posterior estimates from these runs is very close
 to the corresponding true value in each case when the trees were fixed (
 <a reveal-id="F3" data-open="F3" class="link link-reveal link-table xref-fig">Fig. 3</a> ). The coverage probabilities for the parameters are high when we also sampled the trees from the priors ( <a reveal-id="T1" data-open="T1" class="link link-reveal link-table xref-fig">Table 1</a> ). Thus, these results indicate that our implementation of the FBD prior-density calculations and MCMC proposals is correct. </p>                        <a id="115286561" scrollto-destination="115286561"></a>
<div id="F3" data-id="F3" class="fig fig-section"><div class="label fig-label">Figure 3.</div><div class="graphic-wrap"><img class="content-image" src="m_syv080f3.png" alt=" Box-plot of the posterior means of d=λ−μ , r=μ/λ , and s=ψ/(μ+ψ) for 1000 replicates under strategy 1 to 4 in simulation I (fixed trees). The true values of d , r , and s are 0.1, 0.667, and 0.333, respectively (horizontal lines). " path-from-xml="syv080f3.tiff"><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="https://academic.oup.com/view-large/figure/115286561/syv080f3.tiff" path-from-xml="syv080f3.tiff" target="_blank">View large</a><a section="115286561" href="https://oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/sysbio/65/2/10.1093_sysbio_syv080/3/syv080f3.png?Expires=2147483647&amp;Signature=HdIYq0NmAirzcGxsn2sUSJvFX3CxjXWHmujfBhzz42kHbjZ6jJGWSPH7Nomb02TGj4CV%7EmiccelHxdbPW5%7EwfNHS1Y6jOQOh97MDCjYUeDjUyZPQxoU5dAya1qu1uiTG8ODpyMkYUq63l8A-Dr1pfpj-VuXuY5tuR15wwpMPNgNnZndcEPeyMVO7O3xFnkkewjELgliSPn1KhG4l2Rt3ZQIejrAVG6ZNqN9rMX3IE8PkoMkVJ9iYJR1VE65RFeCIAbpbQrFjtaHjgtjRdEWpUp8zLdzKunmT%7E0h5AdwtDYc-KiaQCaCnZr%7ELPX8RFjFcVuNM-L1b2MTxhe2sKOwaoQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=115286561&amp;ar=2427164&amp;xsltPath=%7E/UI/app/XSLT&amp;imagename=" class="download-slide" path-from-xml="syv080f3.tiff" target="">Download slide</a></div></div><div class="caption fig-caption"><p> Box-plot of the posterior means of <span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><mi>d</mi><mo>=</mo><mi>λ</mi><mo>−</mo><mi>μ</mi></mrow></math></span> , <span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><mi>r</mi><mo>=</mo><mi>μ</mi><mo>/</mo><mi>λ</mi></mrow></math></span> , and <span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><mi>s</mi><mo>=</mo><mi>ψ</mi><mo>/</mo><mo stretchy="false">(</mo><mi>μ</mi><mo>+</mo><mi>ψ</mi><mo stretchy="false">)</mo></mrow></math></span> for 1000 replicates under strategy 1 to 4 in simulation I (fixed trees). The true values of <span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mi xmlns="">d</mi></math></span> , <span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mi xmlns="">r</mi></math></span> , and <span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mi xmlns="">s</mi></math></span> are 0.1, 0.667, and 0.333, respectively (horizontal lines). </p></div></div><div content-id="F3" class="fig fig-modal reveal-modal" style="display: block;"><div class="label fig-label">Figure 3.</div><div class="graphic-wrap"><img class="content-image" src="m_syv080f3.png" alt=" Box-plot of the posterior means of d=λ−μ , r=μ/λ , and s=ψ/(μ+ψ) for 1000 replicates under strategy 1 to 4 in simulation I (fixed trees). The true values of d , r , and s are 0.1, 0.667, and 0.333, respectively (horizontal lines). " path-from-xml="syv080f3.tiff"><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="https://academic.oup.com/view-large/figure/115286561/syv080f3.tiff" path-from-xml="syv080f3.tiff" target="_blank">View large</a><a section="115286561" href="https://oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/sysbio/65/2/10.1093_sysbio_syv080/3/syv080f3.png?Expires=2147483647&amp;Signature=HdIYq0NmAirzcGxsn2sUSJvFX3CxjXWHmujfBhzz42kHbjZ6jJGWSPH7Nomb02TGj4CV%7EmiccelHxdbPW5%7EwfNHS1Y6jOQOh97MDCjYUeDjUyZPQxoU5dAya1qu1uiTG8ODpyMkYUq63l8A-Dr1pfpj-VuXuY5tuR15wwpMPNgNnZndcEPeyMVO7O3xFnkkewjELgliSPn1KhG4l2Rt3ZQIejrAVG6ZNqN9rMX3IE8PkoMkVJ9iYJR1VE65RFeCIAbpbQrFjtaHjgtjRdEWpUp8zLdzKunmT%7E0h5AdwtDYc-KiaQCaCnZr%7ELPX8RFjFcVuNM-L1b2MTxhe2sKOwaoQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=115286561&amp;ar=2427164&amp;xsltPath=%7E/UI/app/XSLT&amp;imagename=" class="download-slide" path-from-xml="syv080f3.tiff" target="">Download slide</a></div></div><div class="caption fig-caption"><p> Box-plot of the posterior means of <span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><mi>d</mi><mo>=</mo><mi>λ</mi><mo>−</mo><mi>μ</mi></mrow></math></span> , <span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><mi>r</mi><mo>=</mo><mi>μ</mi><mo>/</mo><mi>λ</mi></mrow></math></span> , and <span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><mi>s</mi><mo>=</mo><mi>ψ</mi><mo>/</mo><mo stretchy="false">(</mo><mi>μ</mi><mo>+</mo><mi>ψ</mi><mo stretchy="false">)</mo></mrow></math></span> for 1000 replicates under strategy 1 to 4 in simulation I (fixed trees). The true values of <span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mi xmlns="">d</mi></math></span> , <span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mi xmlns="">r</mi></math></span> , and <span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mi xmlns="">s</mi></math></span> are 0.1, 0.667, and 0.333, respectively (horizontal lines). </p></div></div>                        <a id="115286562" scrollto-destination="115286562"></a>
<div content-id="T1" class="table-modal" style="display: block;"><div class="table-wrap"><div id="T1" class="table-wrap-title" data-id="T1"><div class="label -label">Table 1.</div><div class="caption"><p>Coverage probabilities of parameter estimates in analyses of trees from Simulation I (no sequence data)</p></div> </div><div class="table-overflow"><table><thead name="thead" align="left"><tr><th>&nbsp;</th><th colspan="4">Coverage Prob.&nbsp;</th></tr><tr><th>Prior&nbsp;</th><th><span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mi xmlns="">d</mi></math></span>&nbsp;</th><th><span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mi xmlns="">r</mi></math></span>&nbsp;</th><th><span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mi xmlns="">s</mi></math></span>&nbsp;</th><th>% Ancestral&nbsp;</th></tr></thead><tbody><tr><td>I.1&nbsp;</td><td>0.97&nbsp;</td><td>0.84&nbsp;</td><td>0.86&nbsp;</td><td>0.87&nbsp;</td></tr><tr><td>I.2&nbsp;</td><td>0.95&nbsp;</td><td>0.85&nbsp;</td><td>0.83&nbsp;</td><td>0.83&nbsp;</td></tr><tr><td>I.3&nbsp;</td><td>0.91&nbsp;</td><td>0.87&nbsp;</td><td>0.84&nbsp;</td><td>0.89&nbsp;</td></tr><tr><td>I.4&nbsp;</td><td>0.94&nbsp;</td><td>0.90&nbsp;</td><td>0.89&nbsp;</td><td>0.91&nbsp;</td></tr></tbody></table></div><div content-id="T1" class="table-modal"><table><thead name="thead" align="left"><tr><th>&nbsp;</th><th colspan="4">Coverage Prob.&nbsp;</th></tr><tr><th>Prior&nbsp;</th><th><span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mi xmlns="">d</mi></math></span>&nbsp;</th><th><span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mi xmlns="">r</mi></math></span>&nbsp;</th><th><span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mi xmlns="">s</mi></math></span>&nbsp;</th><th>% Ancestral&nbsp;</th></tr></thead><tbody><tr><td>I.1&nbsp;</td><td>0.97&nbsp;</td><td>0.84&nbsp;</td><td>0.86&nbsp;</td><td>0.87&nbsp;</td></tr><tr><td>I.2&nbsp;</td><td>0.95&nbsp;</td><td>0.85&nbsp;</td><td>0.83&nbsp;</td><td>0.83&nbsp;</td></tr><tr><td>I.3&nbsp;</td><td>0.91&nbsp;</td><td>0.87&nbsp;</td><td>0.84&nbsp;</td><td>0.89&nbsp;</td></tr><tr><td>I.4&nbsp;</td><td>0.94&nbsp;</td><td>0.90&nbsp;</td><td>0.89&nbsp;</td><td>0.91&nbsp;</td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id="FN2"><div class="fn"><p> Notes: The coverage probabilities (rate of the true value falling into the 95% HPD interval) for <span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><mi>d</mi><mo>=</mo><mi>λ</mi><mo>−</mo><mi>μ</mi></mrow></math></span> , <span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><mi>r</mi><mo>=</mo><mi>μ</mi><mo>/</mo><mi>λ</mi></mrow></math></span> , <span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><mi>s</mi><mo>=</mo><mi>ψ</mi><mo>/</mo><mo stretchy="false">(</mo><mi>μ</mi><mo>+</mo><mi>ψ</mi><mo stretchy="false">)</mo></mrow></math></span>
 , and the proportion of ancestral fossils are summarized for each prior
 setting. I.x stands for Simulation I prior x (see Simulation section 
for details). FBD model parameters <span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mi xmlns="">d</mi></math></span> , <span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mi xmlns="">r</mi></math></span> , and <span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mi xmlns="">s</mi></math></span> were constant through time in both the generating model and the inference model. </p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="https://academic.oup.com/view-large/115286562" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="T1" class="table-wrap-title" data-id="T1"><div class="label -label">Table 1.</div><div class="caption"><p>Coverage probabilities of parameter estimates in analyses of trees from Simulation I (no sequence data)</p></div> </div><div class="table-overflow"><table><thead name="thead" align="left"><tr><th>&nbsp;</th><th colspan="4">Coverage Prob.&nbsp;</th></tr><tr><th>Prior&nbsp;</th><th><span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mi xmlns="">d</mi></math></span>&nbsp;</th><th><span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mi xmlns="">r</mi></math></span>&nbsp;</th><th><span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mi xmlns="">s</mi></math></span>&nbsp;</th><th>% Ancestral&nbsp;</th></tr></thead><tbody><tr><td>I.1&nbsp;</td><td>0.97&nbsp;</td><td>0.84&nbsp;</td><td>0.86&nbsp;</td><td>0.87&nbsp;</td></tr><tr><td>I.2&nbsp;</td><td>0.95&nbsp;</td><td>0.85&nbsp;</td><td>0.83&nbsp;</td><td>0.83&nbsp;</td></tr><tr><td>I.3&nbsp;</td><td>0.91&nbsp;</td><td>0.87&nbsp;</td><td>0.84&nbsp;</td><td>0.89&nbsp;</td></tr><tr><td>I.4&nbsp;</td><td>0.94&nbsp;</td><td>0.90&nbsp;</td><td>0.89&nbsp;</td><td>0.91&nbsp;</td></tr></tbody></table></div><div content-id="T1" class="table-modal"><table><thead name="thead" align="left"><tr><th>&nbsp;</th><th colspan="4">Coverage Prob.&nbsp;</th></tr><tr><th>Prior&nbsp;</th><th><span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mi xmlns="">d</mi></math></span>&nbsp;</th><th><span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mi xmlns="">r</mi></math></span>&nbsp;</th><th><span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mi xmlns="">s</mi></math></span>&nbsp;</th><th>% Ancestral&nbsp;</th></tr></thead><tbody><tr><td>I.1&nbsp;</td><td>0.97&nbsp;</td><td>0.84&nbsp;</td><td>0.86&nbsp;</td><td>0.87&nbsp;</td></tr><tr><td>I.2&nbsp;</td><td>0.95&nbsp;</td><td>0.85&nbsp;</td><td>0.83&nbsp;</td><td>0.83&nbsp;</td></tr><tr><td>I.3&nbsp;</td><td>0.91&nbsp;</td><td>0.87&nbsp;</td><td>0.84&nbsp;</td><td>0.89&nbsp;</td></tr><tr><td>I.4&nbsp;</td><td>0.94&nbsp;</td><td>0.90&nbsp;</td><td>0.89&nbsp;</td><td>0.91&nbsp;</td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id="FN2"><div class="fn"><p> Notes: The coverage probabilities (rate of the true value falling into the 95% HPD interval) for <span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><mi>d</mi><mo>=</mo><mi>λ</mi><mo>−</mo><mi>μ</mi></mrow></math></span> , <span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><mi>r</mi><mo>=</mo><mi>μ</mi><mo>/</mo><mi>λ</mi></mrow></math></span> , <span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><mi>s</mi><mo>=</mo><mi>ψ</mi><mo>/</mo><mo stretchy="false">(</mo><mi>μ</mi><mo>+</mo><mi>ψ</mi><mo stretchy="false">)</mo></mrow></math></span>
 , and the proportion of ancestral fossils are summarized for each prior
 setting. I.x stands for Simulation I prior x (see Simulation section 
for details). FBD model parameters <span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mi xmlns="">d</mi></math></span> , <span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mi xmlns="">r</mi></math></span> , and <span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mi xmlns="">s</mi></math></span> were constant through time in both the generating model and the inference model. </p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="https://academic.oup.com/view-large/115286562" target="_blank">View Large</a></div></div>                            <h3 scrollto-destination="115286563" id="115286563" class="section-title">Simulation II: Trees Inferred from Sequence Data</h3>
<p> We picked 100 trees randomly from the 1000 trees generated using 
strategy 2 and strategy 4 in Simulation I. We simulated DNA sequences on
 each tree using Seq-gen ( <a reveal-id="B39" data-open="B39" class="link link-ref link-reveal xref-bibr">Rambaut and Grass 1997</a> ) under the JC69 model ( <a reveal-id="B24" data-open="B24" class="link link-ref link-reveal xref-bibr">Jukes and Cantor 1969</a>
 ). The mutation rate was arbitrarily set to 0.003 (each branch length 
of the FBD tree was multiplied by this rate). To put this in a real-time
 perspective, if one time unit is assumed to correspond to 10 myr, then 
time 10 means 100 Ma, and the mutation rate would be 0.03% per site per 
myr. If one time unit is assumed to be shorter, then the corresponding 
absolute rate would increase proportionally, and vice versa. The 
sequence length was set to 500 bp. </p><p> We then used MCMC sampling in MrBayes to infer the node ages (root age in particular), net diversification ( <span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><mi>d</mi><mo>=</mo><mi>λ</mi><mo>−</mo><mi>μ</mi></mrow></math></span> ), turnover ( <span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><mi>r</mi><mo>=</mo><mi>μ</mi><mo>/</mo><mi>λ</mi></mrow></math></span> ), and fossil sampling proportion ( <span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><mi>s</mi><mo>=</mo><mi>ψ</mi><mo>/</mo><mo stretchy="false">(</mo><mi>μ</mi><mo>+</mo><mi>ψ</mi><mo stretchy="false">)</mo></mrow></math></span> ) from the sequence data, alongside the tree. The following priors were used for the parameters: <span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><mi>d</mi><mo>∼</mo></mrow></math></span> Exponential(10), <span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><mi>r</mi><mo>∼</mo></mrow></math></span> Beta(1, 1), <span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><mi>s</mi><mo>∼</mo></mrow></math></span> Beta(1, 1), and <span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><msub><mi>t</mi><mrow><mi>m</mi><mi>r</mi><mi>c</mi><mi>a</mi></mrow></msub><mo>∼</mo></mrow></math></span>
 Uniform (0, 1000). The evolutionary model was set to JC69, the true 
model. The prior for the global clock rate was set to a Gamma(3, 1000) 
distribution, which has a mean equal to the true clock rate. </p><p> We 
also investigated the impact of different prior assumptions concerning 
the sampling process. For alignments generated from trees in strategy 2 
(random sampling of extant taxa), the following prior parameters were 
used:

For alignments generated from trees in strategy 4 (diversified 
sampling), the same prior settings were used (runs 1′ – 4′), except that
 we used diversified sampling for 1′ (T) and 2′ (C), and random sampling
 for 4′ (D). Note that the uniform tree prior (U) ( <a reveal-id="B44" data-open="B44" class="link link-ref link-reveal xref-bibr">Ronquist et al. 2012b</a>
 ), which posits a uniform prior density on speciation times given the 
fossil and root age constraints, does not include parameters specific to
 the FBD model, that is, diversification, turnover, and fossil sampling 
rates. </p><ol class="order"><li><p>Random sampling of extant taxa with 
probability 0.5, extra fossil sampling at time 10 with probability 0.5 
(denoted T for “true model”).</p></li><li><p>Random sampling of extant taxa with probability 0.5, no extra fossil sampling at time 10 (denoted C for “constant rate”).</p></li><li><p>Larger
 sampling fraction (probability 1.0, i.e., complete sampling) of extant 
taxa, no extra fossil sampling at time 10 (denoted L for “larger 
sampling fraction”).</p></li><li><p>Diversified sampling of extant taxa 
with proportion 0.5, no extra fossil sampling at time 10 (denoted D for 
“different sampling strategy”).</p></li><li><p>Uniform tree prior (denoted U for “uniform tree prior”).</p></li></ol><p> For priors 1 and 1′ (piecewise-constant rates), we inferred the rate parameters ( <span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mi xmlns="">d</mi></math></span> , <span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mi xmlns="">r</mi></math></span> , <span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mi xmlns="">s</mi></math></span>
 ) before and after time 10 separately, and accounted for the sampling 
effort at time 10 and 0 (present). For these cases, the prior models 
used in MCMC inference were thus consistent with the simulation settings
 (the true model), except for the uncoupling of the rate parameters ( <span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mi xmlns="">d</mi></math></span> , <span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mi xmlns="">r</mi></math></span> , <span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mi xmlns="">s</mi></math></span>
 ) in the two time intervals. For the other FBD priors, we assumed 
constant diversification, turnover and fossil sampling rates and ignored
 the extra fossil sampling effort at time 10, causing various types of 
misfit between the prior (the model assumed in inference) and the model 
used in the simulation (the true model). Under diversified sampling, the
 cutoff time, <span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><msub><mi>x</mi><mrow><mi>c</mi><mi>u</mi><mi>t</mi></mrow></msub></mrow></math></span>
 , was adjusted to be slightly younger than the youngest internal node 
and also the youngest fossil. The fossil ages were fixed to the true 
values used in the simulation. The tree topology and branch lengths were
 not fixed but inferred from the simulated sequence data using MCMC 
sampling, employing the add-branch and delete-branch moves among others.
 The MCMC chains (one cold, three heated) were run for 2 million 
iterations and sampled every 200 iterations, with the first 25% samples 
discarded as burn-in. </p><p> The results show that when the FBD prior 
is consistent with the simulation settings (II.1 and II.1′), the 
estimates are close to the true values, with coverage probabilities for 
the root age being 0.88, and &gt;0.9 for the other parameters ( <a reveal-id="T2" data-open="T2" class="link link-reveal link-table xref-fig">Table 2</a> , <a reveal-id="F4" data-open="F4" class="link link-reveal link-table xref-fig">Figs. 4</a> and <a reveal-id="F5" data-open="F5" class="link link-reveal link-table xref-fig">5</a>
 , T). When assuming no extra fossil sampling at time 10, the accuracy 
of those estimates is almost the same as in the previous case ( <a reveal-id="T2" data-open="T2" class="link link-reveal link-table xref-fig">Table 2</a>
 , C). When assuming a larger sampling fraction of extant taxa (complete
 sampling), the rate estimates can be biased, causing the underlying 
parameters λ, μ, and ψ to also be biased ( <a reveal-id="T2" data-open="T2" class="link link-reveal link-table xref-fig">Table 2</a> , L). </p>                        <a id="115286570" scrollto-destination="115286570"></a>
<div id="F4" data-id="F4" class="fig fig-section"><div class="label fig-label">Figure 4.</div><div class="graphic-wrap"><img class="content-image" src="m_syv080f4.png" alt=" The posterior means of the root age and proportion of ancestral fossils against their true values, and box-plot of the posterior means of d , r , and s for the four scenarios in Simulation II. The true strategy is random sampling of extant taxa. " path-from-xml="syv080f4.tiff"><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="https://academic.oup.com/view-large/figure/115286570/syv080f4.tiff" path-from-xml="syv080f4.tiff" target="_blank">View large</a><a section="115286570" href="https://oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/sysbio/65/2/10.1093_sysbio_syv080/3/syv080f4.png?Expires=2147483647&amp;Signature=hei5lqRaKY5NFjJpeQ9NXscrT6927QFt3gbi64OcpcIq1zRrBUrYwDxUhsmnAahnn7yEGWDWAWsqZCDQwWUvMufL%7Ei%7EzKB0sCDt56fhEda-Y7t1aTccMnAe5XlskyPSw3oXzM4kXJU7vDxvrhLchjlTBibO7zOmOMMlgTvUlvaw3T6DnEuNFo%7EZnLAUxgJR4aFDfiXMsC8Zc7eZ1WkmjOrCZnz%7EpeMhczhMixHS%7EykGBeLiNrcru32vUrXtgDx7cG79odZxAkVAbWvVEs2NYYTr8ONQJ3pOneddnYMR6XMnygzxHdYYvc1xD0wiqC4bT6x0BZjxSwsNE5c-9Nphxew__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=115286570&amp;ar=2427164&amp;xsltPath=%7E/UI/app/XSLT&amp;imagename=" class="download-slide" path-from-xml="syv080f4.tiff" target="">Download slide</a></div></div><div class="caption fig-caption"><p>
 The posterior means of the root age and proportion of ancestral fossils
 against their true values, and box-plot of the posterior means of <span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mi xmlns="">d</mi></math></span> , <span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mi xmlns="">r</mi></math></span> , and <span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mi xmlns="">s</mi></math></span> for the four scenarios in Simulation II. The true strategy is random sampling of extant taxa. </p></div></div><div content-id="F4" class="fig fig-modal reveal-modal" style="display: block;"><div class="label fig-label">Figure 4.</div><div class="graphic-wrap"><img class="content-image" src="m_syv080f4.png" alt=" The posterior means of the root age and proportion of ancestral fossils against their true values, and box-plot of the posterior means of d , r , and s for the four scenarios in Simulation II. The true strategy is random sampling of extant taxa. " path-from-xml="syv080f4.tiff"><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="https://academic.oup.com/view-large/figure/115286570/syv080f4.tiff" path-from-xml="syv080f4.tiff" target="_blank">View large</a><a section="115286570" href="https://oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/sysbio/65/2/10.1093_sysbio_syv080/3/syv080f4.png?Expires=2147483647&amp;Signature=hei5lqRaKY5NFjJpeQ9NXscrT6927QFt3gbi64OcpcIq1zRrBUrYwDxUhsmnAahnn7yEGWDWAWsqZCDQwWUvMufL%7Ei%7EzKB0sCDt56fhEda-Y7t1aTccMnAe5XlskyPSw3oXzM4kXJU7vDxvrhLchjlTBibO7zOmOMMlgTvUlvaw3T6DnEuNFo%7EZnLAUxgJR4aFDfiXMsC8Zc7eZ1WkmjOrCZnz%7EpeMhczhMixHS%7EykGBeLiNrcru32vUrXtgDx7cG79odZxAkVAbWvVEs2NYYTr8ONQJ3pOneddnYMR6XMnygzxHdYYvc1xD0wiqC4bT6x0BZjxSwsNE5c-9Nphxew__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=115286570&amp;ar=2427164&amp;xsltPath=%7E/UI/app/XSLT&amp;imagename=" class="download-slide" path-from-xml="syv080f4.tiff" target="">Download slide</a></div></div><div class="caption fig-caption"><p>
 The posterior means of the root age and proportion of ancestral fossils
 against their true values, and box-plot of the posterior means of <span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mi xmlns="">d</mi></math></span> , <span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mi xmlns="">r</mi></math></span> , and <span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mi xmlns="">s</mi></math></span> for the four scenarios in Simulation II. The true strategy is random sampling of extant taxa. </p></div></div>                        <a id="115286571" scrollto-destination="115286571"></a>
<div id="F5" data-id="F5" class="fig fig-section"><div class="label fig-label">Figure 5.</div><div class="graphic-wrap"><img class="content-image" src="m_syv080f5.png" alt=" The posterior means of the root age and proportion of ancestral fossils against their true values, and box-plot of the posterior means of d , r , and s for the four scenarios in Simulation II. The true strategy is diversified sampling of extant taxa. " path-from-xml="syv080f5.tiff"><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="https://academic.oup.com/view-large/figure/115286571/syv080f5.tiff" path-from-xml="syv080f5.tiff" target="_blank">View large</a><a section="115286571" href="https://oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/sysbio/65/2/10.1093_sysbio_syv080/3/syv080f5.png?Expires=2147483647&amp;Signature=On-fEfPzkp7DwDC3B-jB183RGFn8iOEO4eYKJfa4JqrfrMo5E6Am00S6iy7Z5vUxQkxCTXSNLOm8nYQf-QcEyweXrLEpm2huj-%7E7kPegBmDpxrO4w5VLkcRCalShOqDnOnabBz75KGr12DtLUi7wGzniftI3nSNg-bSo5iihLA96J7jYFsigvax5gCVfXLvyr55f1CK1CVHTKoY5gNA38qS-GRacdGOoKvhv7Q2jIltLLuewSdcZ1xN6tKyCaGmObWtU5uz4wPVwlX90JaMmbz2Xj3WnjrVKA72COcjF%7EdoGqVfgTe8R2PTjMXqB00SSm07zdmzIIOR60OOs7c9m8A__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=115286571&amp;ar=2427164&amp;xsltPath=%7E/UI/app/XSLT&amp;imagename=" class="download-slide" path-from-xml="syv080f5.tiff" target="">Download slide</a></div></div><div class="caption fig-caption"><p>
 The posterior means of the root age and proportion of ancestral fossils
 against their true values, and box-plot of the posterior means of <span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mi xmlns="">d</mi></math></span> , <span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mi xmlns="">r</mi></math></span> , and <span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mi xmlns="">s</mi></math></span> for the four scenarios in Simulation II. The true strategy is diversified sampling of extant taxa. </p></div></div><div content-id="F5" class="fig fig-modal reveal-modal" style="display: block;"><div class="label fig-label">Figure 5.</div><div class="graphic-wrap"><img class="content-image" src="m_syv080f5.png" alt=" The posterior means of the root age and proportion of ancestral fossils against their true values, and box-plot of the posterior means of d , r , and s for the four scenarios in Simulation II. The true strategy is diversified sampling of extant taxa. " path-from-xml="syv080f5.tiff"><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="https://academic.oup.com/view-large/figure/115286571/syv080f5.tiff" path-from-xml="syv080f5.tiff" target="_blank">View large</a><a section="115286571" href="https://oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/sysbio/65/2/10.1093_sysbio_syv080/3/syv080f5.png?Expires=2147483647&amp;Signature=On-fEfPzkp7DwDC3B-jB183RGFn8iOEO4eYKJfa4JqrfrMo5E6Am00S6iy7Z5vUxQkxCTXSNLOm8nYQf-QcEyweXrLEpm2huj-%7E7kPegBmDpxrO4w5VLkcRCalShOqDnOnabBz75KGr12DtLUi7wGzniftI3nSNg-bSo5iihLA96J7jYFsigvax5gCVfXLvyr55f1CK1CVHTKoY5gNA38qS-GRacdGOoKvhv7Q2jIltLLuewSdcZ1xN6tKyCaGmObWtU5uz4wPVwlX90JaMmbz2Xj3WnjrVKA72COcjF%7EdoGqVfgTe8R2PTjMXqB00SSm07zdmzIIOR60OOs7c9m8A__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=115286571&amp;ar=2427164&amp;xsltPath=%7E/UI/app/XSLT&amp;imagename=" class="download-slide" path-from-xml="syv080f5.tiff" target="">Download slide</a></div></div><div class="caption fig-caption"><p>
 The posterior means of the root age and proportion of ancestral fossils
 against their true values, and box-plot of the posterior means of <span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mi xmlns="">d</mi></math></span> , <span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mi xmlns="">r</mi></math></span> , and <span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mi xmlns="">s</mi></math></span> for the four scenarios in Simulation II. The true strategy is diversified sampling of extant taxa. </p></div></div>                        <a id="115286572" scrollto-destination="115286572"></a>
<div content-id="T2" class="table-modal" style="display: block;"><div class="table-wrap"><div id="T2" class="table-wrap-title" data-id="T2"><div class="label -label">Table 2.</div><div class="caption"><p>Coverage probabilities of parameter estimates in analyses of data from Simulation II (trees inferred from sequences)</p></div> </div><div class="table-overflow"><table><thead name="thead" align="left"><tr><th>&nbsp;</th><th colspan="5">Coverage Prob.&nbsp;</th></tr><tr><th>Prior&nbsp;</th><th>Root Age&nbsp;</th><th><span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mi xmlns="">d</mi></math></span>&nbsp;</th><th><span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mi xmlns="">r</mi></math></span>&nbsp;</th><th><span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mi xmlns="">s</mi></math></span>&nbsp;</th><th>% Ancestral&nbsp;</th></tr></thead><tbody><tr><td colspan="6">FBD trees from Simulation I.2 (random sampling)&nbsp;</td></tr><tr><td>II.1 (T)&nbsp;</td><td>0.88&nbsp;</td><td>0.93, 0.94&nbsp;</td><td>0.96, 0.94&nbsp;</td><td>0.93, 0.94&nbsp;</td><td>0.97&nbsp;</td></tr><tr><td>II.2 (C)&nbsp;</td><td>0.88&nbsp;</td><td>0.96&nbsp;</td><td>0.95&nbsp;</td><td>0.95&nbsp;</td><td>0.93&nbsp;</td></tr><tr><td>II.3 (L)&nbsp;</td><td>0.86&nbsp;</td><td>0.51&nbsp;</td><td>0.66&nbsp;</td><td>0.91&nbsp;</td><td>0.94&nbsp;</td></tr><tr><td>II.4 (D)&nbsp;</td><td>0.84&nbsp;</td><td>0.79&nbsp;</td><td>0.00&nbsp;</td><td>0.00&nbsp;</td><td>0.04&nbsp;</td></tr><tr><td>II.5 (U)&nbsp;</td><td>0.46&nbsp;</td><td>—&nbsp;</td><td>—&nbsp;</td><td>—&nbsp;</td><td>—&nbsp;</td></tr><tr><td colspan="6">FBD trees from Simulation I.4 (diversified sampling)&nbsp;</td></tr><tr><td>II.1′ (T)&nbsp;</td><td>0.88&nbsp;</td><td>0.94, 0.93&nbsp;</td><td>0.96, 0.93&nbsp;</td><td>0.95, 0.96&nbsp;</td><td>0.89&nbsp;</td></tr><tr><td>II.2′ (C)&nbsp;</td><td>0.87&nbsp;</td><td>0.95&nbsp;</td><td>0.94&nbsp;</td><td>0.93&nbsp;</td><td>0.92&nbsp;</td></tr><tr><td>II.3′ (L)&nbsp;</td><td>0.84&nbsp;</td><td>0.43&nbsp;</td><td>0.94&nbsp;</td><td>0.24&nbsp;</td><td>0.70&nbsp;</td></tr><tr><td>II.4′ (D)&nbsp;</td><td>0.85&nbsp;</td><td>0.89&nbsp;</td><td>0.93&nbsp;</td><td>0.93&nbsp;</td><td>0.93&nbsp;</td></tr><tr><td>II.5′ (U)&nbsp;</td><td>0.48&nbsp;</td><td>—&nbsp;</td><td>—&nbsp;</td><td>—&nbsp;</td><td>—&nbsp;</td></tr></tbody></table></div><div content-id="T2" class="table-modal"><table><thead name="thead" align="left"><tr><th>&nbsp;</th><th colspan="5">Coverage Prob.&nbsp;</th></tr><tr><th>Prior&nbsp;</th><th>Root Age&nbsp;</th><th><span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mi xmlns="">d</mi></math></span>&nbsp;</th><th><span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mi xmlns="">r</mi></math></span>&nbsp;</th><th><span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mi xmlns="">s</mi></math></span>&nbsp;</th><th>% Ancestral&nbsp;</th></tr></thead><tbody><tr><td colspan="6">FBD trees from Simulation I.2 (random sampling)&nbsp;</td></tr><tr><td>II.1 (T)&nbsp;</td><td>0.88&nbsp;</td><td>0.93, 0.94&nbsp;</td><td>0.96, 0.94&nbsp;</td><td>0.93, 0.94&nbsp;</td><td>0.97&nbsp;</td></tr><tr><td>II.2 (C)&nbsp;</td><td>0.88&nbsp;</td><td>0.96&nbsp;</td><td>0.95&nbsp;</td><td>0.95&nbsp;</td><td>0.93&nbsp;</td></tr><tr><td>II.3 (L)&nbsp;</td><td>0.86&nbsp;</td><td>0.51&nbsp;</td><td>0.66&nbsp;</td><td>0.91&nbsp;</td><td>0.94&nbsp;</td></tr><tr><td>II.4 (D)&nbsp;</td><td>0.84&nbsp;</td><td>0.79&nbsp;</td><td>0.00&nbsp;</td><td>0.00&nbsp;</td><td>0.04&nbsp;</td></tr><tr><td>II.5 (U)&nbsp;</td><td>0.46&nbsp;</td><td>—&nbsp;</td><td>—&nbsp;</td><td>—&nbsp;</td><td>—&nbsp;</td></tr><tr><td colspan="6">FBD trees from Simulation I.4 (diversified sampling)&nbsp;</td></tr><tr><td>II.1′ (T)&nbsp;</td><td>0.88&nbsp;</td><td>0.94, 0.93&nbsp;</td><td>0.96, 0.93&nbsp;</td><td>0.95, 0.96&nbsp;</td><td>0.89&nbsp;</td></tr><tr><td>II.2′ (C)&nbsp;</td><td>0.87&nbsp;</td><td>0.95&nbsp;</td><td>0.94&nbsp;</td><td>0.93&nbsp;</td><td>0.92&nbsp;</td></tr><tr><td>II.3′ (L)&nbsp;</td><td>0.84&nbsp;</td><td>0.43&nbsp;</td><td>0.94&nbsp;</td><td>0.24&nbsp;</td><td>0.70&nbsp;</td></tr><tr><td>II.4′ (D)&nbsp;</td><td>0.85&nbsp;</td><td>0.89&nbsp;</td><td>0.93&nbsp;</td><td>0.93&nbsp;</td><td>0.93&nbsp;</td></tr><tr><td>II.5′ (U)&nbsp;</td><td>0.48&nbsp;</td><td>—&nbsp;</td><td>—&nbsp;</td><td>—&nbsp;</td><td>—&nbsp;</td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id="FN3"><div class="fn"><p> Notes: The coverage probabilities (rate of the true value falling into the 95% HPD interval) for the root age ( <span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><msub><mi>t</mi><mrow><mi>m</mi><mi>r</mi><mi>c</mi><mi>a</mi></mrow></msub></mrow></math></span> ), <span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><mi>d</mi><mo>=</mo><mi>λ</mi><mo>−</mo><mi>μ</mi></mrow></math></span> , <span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><mi>r</mi><mo>=</mo><mi>μ</mi><mo>/</mo><mi>λ</mi></mrow></math></span> , <span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><mi>s</mi><mo>=</mo><mi>ψ</mi><mo>/</mo><mo stretchy="false">(</mo><mi>μ</mi><mo>+</mo><mi>ψ</mi><mo stretchy="false">)</mo></mrow></math></span>
 , and the proportion of ancestral fossils are summarized for each prior
 setting. II.x stands for Simulation II prior x (see Simulation section 
for details). FBD model parameters <span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mi xmlns="">d</mi></math></span> , <span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mi xmlns="">r</mi></math></span> , and <span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mi xmlns="">s</mi></math></span>
 were inferred separately for the two time intervals in 1 (T) and 1′ 
(T); therefore, two values are given for these. In the other analyses 
(C, L, D), a single value was assumed for both time intervals. There are
 no birth, death, or sampling parameters for the uniform tree prior (U).
 </p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="https://academic.oup.com/view-large/115286572" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="T2" class="table-wrap-title" data-id="T2"><div class="label -label">Table 2.</div><div class="caption"><p>Coverage probabilities of parameter estimates in analyses of data from Simulation II (trees inferred from sequences)</p></div> </div><div class="table-overflow"><table><thead name="thead" align="left"><tr><th>&nbsp;</th><th colspan="5">Coverage Prob.&nbsp;</th></tr><tr><th>Prior&nbsp;</th><th>Root Age&nbsp;</th><th><span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mi xmlns="">d</mi></math></span>&nbsp;</th><th><span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mi xmlns="">r</mi></math></span>&nbsp;</th><th><span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mi xmlns="">s</mi></math></span>&nbsp;</th><th>% Ancestral&nbsp;</th></tr></thead><tbody><tr><td colspan="6">FBD trees from Simulation I.2 (random sampling)&nbsp;</td></tr><tr><td>II.1 (T)&nbsp;</td><td>0.88&nbsp;</td><td>0.93, 0.94&nbsp;</td><td>0.96, 0.94&nbsp;</td><td>0.93, 0.94&nbsp;</td><td>0.97&nbsp;</td></tr><tr><td>II.2 (C)&nbsp;</td><td>0.88&nbsp;</td><td>0.96&nbsp;</td><td>0.95&nbsp;</td><td>0.95&nbsp;</td><td>0.93&nbsp;</td></tr><tr><td>II.3 (L)&nbsp;</td><td>0.86&nbsp;</td><td>0.51&nbsp;</td><td>0.66&nbsp;</td><td>0.91&nbsp;</td><td>0.94&nbsp;</td></tr><tr><td>II.4 (D)&nbsp;</td><td>0.84&nbsp;</td><td>0.79&nbsp;</td><td>0.00&nbsp;</td><td>0.00&nbsp;</td><td>0.04&nbsp;</td></tr><tr><td>II.5 (U)&nbsp;</td><td>0.46&nbsp;</td><td>—&nbsp;</td><td>—&nbsp;</td><td>—&nbsp;</td><td>—&nbsp;</td></tr><tr><td colspan="6">FBD trees from Simulation I.4 (diversified sampling)&nbsp;</td></tr><tr><td>II.1′ (T)&nbsp;</td><td>0.88&nbsp;</td><td>0.94, 0.93&nbsp;</td><td>0.96, 0.93&nbsp;</td><td>0.95, 0.96&nbsp;</td><td>0.89&nbsp;</td></tr><tr><td>II.2′ (C)&nbsp;</td><td>0.87&nbsp;</td><td>0.95&nbsp;</td><td>0.94&nbsp;</td><td>0.93&nbsp;</td><td>0.92&nbsp;</td></tr><tr><td>II.3′ (L)&nbsp;</td><td>0.84&nbsp;</td><td>0.43&nbsp;</td><td>0.94&nbsp;</td><td>0.24&nbsp;</td><td>0.70&nbsp;</td></tr><tr><td>II.4′ (D)&nbsp;</td><td>0.85&nbsp;</td><td>0.89&nbsp;</td><td>0.93&nbsp;</td><td>0.93&nbsp;</td><td>0.93&nbsp;</td></tr><tr><td>II.5′ (U)&nbsp;</td><td>0.48&nbsp;</td><td>—&nbsp;</td><td>—&nbsp;</td><td>—&nbsp;</td><td>—&nbsp;</td></tr></tbody></table></div><div content-id="T2" class="table-modal"><table><thead name="thead" align="left"><tr><th>&nbsp;</th><th colspan="5">Coverage Prob.&nbsp;</th></tr><tr><th>Prior&nbsp;</th><th>Root Age&nbsp;</th><th><span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mi xmlns="">d</mi></math></span>&nbsp;</th><th><span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mi xmlns="">r</mi></math></span>&nbsp;</th><th><span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mi xmlns="">s</mi></math></span>&nbsp;</th><th>% Ancestral&nbsp;</th></tr></thead><tbody><tr><td colspan="6">FBD trees from Simulation I.2 (random sampling)&nbsp;</td></tr><tr><td>II.1 (T)&nbsp;</td><td>0.88&nbsp;</td><td>0.93, 0.94&nbsp;</td><td>0.96, 0.94&nbsp;</td><td>0.93, 0.94&nbsp;</td><td>0.97&nbsp;</td></tr><tr><td>II.2 (C)&nbsp;</td><td>0.88&nbsp;</td><td>0.96&nbsp;</td><td>0.95&nbsp;</td><td>0.95&nbsp;</td><td>0.93&nbsp;</td></tr><tr><td>II.3 (L)&nbsp;</td><td>0.86&nbsp;</td><td>0.51&nbsp;</td><td>0.66&nbsp;</td><td>0.91&nbsp;</td><td>0.94&nbsp;</td></tr><tr><td>II.4 (D)&nbsp;</td><td>0.84&nbsp;</td><td>0.79&nbsp;</td><td>0.00&nbsp;</td><td>0.00&nbsp;</td><td>0.04&nbsp;</td></tr><tr><td>II.5 (U)&nbsp;</td><td>0.46&nbsp;</td><td>—&nbsp;</td><td>—&nbsp;</td><td>—&nbsp;</td><td>—&nbsp;</td></tr><tr><td colspan="6">FBD trees from Simulation I.4 (diversified sampling)&nbsp;</td></tr><tr><td>II.1′ (T)&nbsp;</td><td>0.88&nbsp;</td><td>0.94, 0.93&nbsp;</td><td>0.96, 0.93&nbsp;</td><td>0.95, 0.96&nbsp;</td><td>0.89&nbsp;</td></tr><tr><td>II.2′ (C)&nbsp;</td><td>0.87&nbsp;</td><td>0.95&nbsp;</td><td>0.94&nbsp;</td><td>0.93&nbsp;</td><td>0.92&nbsp;</td></tr><tr><td>II.3′ (L)&nbsp;</td><td>0.84&nbsp;</td><td>0.43&nbsp;</td><td>0.94&nbsp;</td><td>0.24&nbsp;</td><td>0.70&nbsp;</td></tr><tr><td>II.4′ (D)&nbsp;</td><td>0.85&nbsp;</td><td>0.89&nbsp;</td><td>0.93&nbsp;</td><td>0.93&nbsp;</td><td>0.93&nbsp;</td></tr><tr><td>II.5′ (U)&nbsp;</td><td>0.48&nbsp;</td><td>—&nbsp;</td><td>—&nbsp;</td><td>—&nbsp;</td><td>—&nbsp;</td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id="FN3"><div class="fn"><p> Notes: The coverage probabilities (rate of the true value falling into the 95% HPD interval) for the root age ( <span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><msub><mi>t</mi><mrow><mi>m</mi><mi>r</mi><mi>c</mi><mi>a</mi></mrow></msub></mrow></math></span> ), <span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><mi>d</mi><mo>=</mo><mi>λ</mi><mo>−</mo><mi>μ</mi></mrow></math></span> , <span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><mi>r</mi><mo>=</mo><mi>μ</mi><mo>/</mo><mi>λ</mi></mrow></math></span> , <span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><mi>s</mi><mo>=</mo><mi>ψ</mi><mo>/</mo><mo stretchy="false">(</mo><mi>μ</mi><mo>+</mo><mi>ψ</mi><mo stretchy="false">)</mo></mrow></math></span>
 , and the proportion of ancestral fossils are summarized for each prior
 setting. II.x stands for Simulation II prior x (see Simulation section 
for details). FBD model parameters <span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mi xmlns="">d</mi></math></span> , <span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mi xmlns="">r</mi></math></span> , and <span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mi xmlns="">s</mi></math></span>
 were inferred separately for the two time intervals in 1 (T) and 1′ 
(T); therefore, two values are given for these. In the other analyses 
(C, L, D), a single value was assumed for both time intervals. There are
 no birth, death, or sampling parameters for the uniform tree prior (U).
 </p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="https://academic.oup.com/view-large/115286572" target="_blank">View Large</a></div></div><p> Somewhat surprisingly, the estimates are poor under diversified sampling when the true strategy is random sampling ( <a reveal-id="F4" data-open="F4" class="link link-reveal link-table xref-fig">Fig. 4</a> , D), but accurate under random sampling when the truth is diversified sampling ( <a reveal-id="F5" data-open="F5" class="link link-reveal link-table xref-fig">Fig. 5</a> , D) (see also <a reveal-id="T2" data-open="T2" class="link link-reveal link-table xref-fig">Table 2</a>
 ). This is probably explained by the asymmetry between these scenarios.
 Given the same fossil sampling rate, there are more fossils recovered 
under random sampling than under diversified sampling of extant taxa 
because we assume that no fossils are sampled after the cutoff time in 
the latter case. Thus, the data from random sampling are more 
informative and conflict strongly with the prior model assuming 
diversified sampling in 4 (D). In contrast, the data from diversified 
sampling are less informative and therefore conflict less strongly with 
the prior model in 4′ (D). We note that the accuracy of the root age 
estimate is almost the same in the four cases (T, C, S, D) ( <a reveal-id="T2" data-open="T2" class="link link-reveal link-table xref-fig">Table 2</a> , <a reveal-id="F4" data-open="F4" class="link link-reveal link-table xref-fig">Figs. 4</a> and <a reveal-id="F5" data-open="F5" class="link link-reveal link-table xref-fig">5</a>
 ), even when the prior is biased so that the other estimates may be 
biased. This is important as the focus in dating analyses is the 
inference of absolute node ages. </p><p> Under the uniform prior, the 
posterior mean of the root age is usually smaller than the true value. 
Although the absolute difference between the inferred and true values is
 small, the credibility interval is narrow so that the coverage rate is 
low (∼0.5) ( <a reveal-id="T2" data-open="T2" class="link link-reveal link-table xref-fig">Table 2</a>
 ). Apparently, the fact that the uniform prior does not model ancestral
 fossils causes a slight but consistent bias toward a younger estimate 
of the root age when many fossils are actually ancestral. One can 
possibly explain this as an effect of the different visibility of 
multiple substitutions along certain branches under these scenarios. An 
ancestral fossil can reveal that evolution took a more complicated route
 than is immediately apparent, resulting in longer branch-length 
estimates and deeper divergence times. When the same fossil sits on a 
side branch, which is always the case in the uniform tree prior, such 
complicated routes can be simplified by pushing apparently unique 
changes onto the side branch and thus shortening the estimated length of
 the trunk of the tree. This might be the cause of the underestimation 
of the root age under the uniform tree prior when there are ancestral 
fossils in the data. </p><p>We conclude by noting that our simulated 
data are highly informative about divergence times, as the fossil 
sampling is rate-constant and rich, the fossils have complete sequences 
with known fixed ages, and there is no rate variation across the tree. 
These settings are appropriate for validating the algorithms, and they 
may fit empirical scenarios where we have access to an abundance of 
well-preserved fossils or ancient DNA, and when there is little rate 
variation across the tree. However, divergence time estimates are likely
 to be less robust under more realistic settings, and additional 
simulations exploring the performance of FBD-based inferences in these 
cases would be valuable. In particular, when the fossil sampling is 
stratigraphic and the fossils are poorly preserved, as is the rule for 
empirical data sets, the FBD prior might have considerable impact on the
 divergence-time estimates (see Hymenoptera analysis below) so that it 
becomes important to model the fossilization and sampling processes 
appropriately. Furthermore, rate variation across the tree can cause 
both loss of accuracy and problems with inference biases, especially if 
the rate variation is not properly addressed by a relaxed-clock model. 
Finally, under realistic data-poor scenarios, there may also be problems
 associated with overparameterization (overfitting).</p>                            <h2 scrollto-destination="115286576" id="115286576" class="section-title"><span class="small-caps">Total-evidence dating of hymenoptera</span></h2>
<p> To investigate the performance of the FBD model in an empirical 
setting, we reanalyzed a data set on the early radiation of Hymenoptera (
 <a reveal-id="B44" data-open="B44" class="link link-ref link-reveal xref-bibr">Ronquist et al. 2012b</a> ). This data set includes 60 extant and 45 fossil Hymenoptera and eight outgroup taxa ( <a reveal-id="T3" data-open="T3" class="link link-reveal link-table xref-fig">Table 3</a> ; see also <a reveal-id="T1" data-open="T1" class="link link-reveal link-table xref-fig">Tables 1</a> and <a reveal-id="T2" data-open="T2" class="link link-reveal link-table xref-fig">2</a> in <a reveal-id="B44" data-open="B44" class="link link-ref link-reveal xref-bibr">Ronquist et al. (2012b)</a>
 ). The data were divided into eight partitions as follows: (i) 
morphology, (ii) 12S and 16S, (iii) 18S, (iv) 28S, (v) 1st and 2nd codon
 positions of CO1, (vi) 3rd codon position of CO1 (but not used in our 
analyses), (vii) 1st and 2nd codon positions of both copies of EF1 <span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mi xmlns="">α</mi></math></span> ( <a reveal-id="B28" data-open="B28" class="link link-ref link-reveal xref-bibr">Klopfstein and Ronquist 2013</a> ), and (viii) 3rd codon position of both copies of EF1 <span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mi xmlns="">α</mi></math></span> . As in <a reveal-id="B44" data-open="B44" class="link link-ref link-reveal xref-bibr">Ronquist et al. (2012b)</a> , we used the <span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><mi>M</mi><mi>k</mi></mrow></math></span> model ( <a reveal-id="B33" data-open="B33" class="link link-ref link-reveal xref-bibr">Lewis 2001</a>
 ) with the “variable” ascertainment bias and gamma-distributed rate 
variation across characters for the morphology partition. For the 
molecular partitions, we used the general time-reversible model with 
gamma rate variation (GTR +  <span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mi xmlns="">Γ</mi></math></span> ), except for 28S where we employed the SYM model with equal base frequencies and symmetric exchange rates (SYM +  <span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mi xmlns="">Γ</mi></math></span>
 ). Parameters of the substitution models and among-site rate variation 
were unlinked across partitions, and partition-specific rate multipliers
 were used to account for variation of evolutionary rates across 
partitions. Specifically, a flat Dirichlet prior was used for the 
relative partition rate multipliers (the partition rate multiplier times
 the proportion of sites in that partition). The relative partition rate
 multipliers were constrained to sum to 1.0, so that the average 
partition rate multiplier was 1.0 across sites (characters) in the data 
set. </p>                        <a id="115286578" scrollto-destination="115286578"></a>
<div content-id="T3" class="table-modal" style="display: block;"><div class="table-wrap"><div id="T3" class="table-wrap-title" data-id="T3"><div class="label -label">Table 3.</div><div class="caption"><p>Summary of Hymenoptera data</p></div> </div><div class="table-overflow"><table><thead name="thead" align="left"><tr><th>Taxa&nbsp;</th><th>Time (Ma)&nbsp;</th><th>Number&nbsp;</th></tr></thead><tbody><tr><td>Fossils&nbsp;</td><td>235 (228, 242)&nbsp;</td><td>2&nbsp;</td></tr><tr><td>&nbsp;</td><td>187.5 (174, 201)&nbsp;</td><td>1&nbsp;</td></tr><tr><td>&nbsp;</td><td>182.5 (182, 183)&nbsp;</td><td>4&nbsp;</td></tr><tr><td>&nbsp;</td><td>179.5 (168, 191)&nbsp;</td><td>4&nbsp;</td></tr><tr><td>&nbsp;</td><td>164.5 (161, 168)&nbsp;</td><td>3&nbsp;</td></tr><tr><td>&nbsp;</td><td>157.5 (152, 163)&nbsp;</td><td>17&nbsp;</td></tr><tr><td>&nbsp;</td><td>148.5 (145, 152)&nbsp;</td><td>1&nbsp;</td></tr><tr><td>&nbsp;</td><td>138.5 (125, 152)&nbsp;</td><td>1&nbsp;</td></tr><tr><td>&nbsp;</td><td>135 (125, 145)&nbsp;</td><td>6&nbsp;</td></tr><tr><td>&nbsp;</td><td>134 (133, 135)&nbsp;</td><td>2&nbsp;</td></tr><tr><td>&nbsp;</td><td>119 (113, 125)&nbsp;</td><td>2&nbsp;</td></tr><tr><td>&nbsp;</td><td>97 (94, 100)&nbsp;</td><td>1&nbsp;</td></tr><tr><td>&nbsp;</td><td>83 (80, 86)&nbsp;</td><td>1&nbsp;</td></tr><tr><td>Hymenoptera&nbsp;</td><td>0&nbsp;</td><td>60&nbsp;</td></tr><tr><td>Outgroups&nbsp;</td><td>0&nbsp;</td><td>8&nbsp;</td></tr><tr><td>Sum&nbsp;</td><td>&nbsp;</td><td>113&nbsp;</td></tr></tbody></table></div><div content-id="T3" class="table-modal"><table><thead name="thead" align="left"><tr><th>Taxa&nbsp;</th><th>Time (Ma)&nbsp;</th><th>Number&nbsp;</th></tr></thead><tbody><tr><td>Fossils&nbsp;</td><td>235 (228, 242)&nbsp;</td><td>2&nbsp;</td></tr><tr><td>&nbsp;</td><td>187.5 (174, 201)&nbsp;</td><td>1&nbsp;</td></tr><tr><td>&nbsp;</td><td>182.5 (182, 183)&nbsp;</td><td>4&nbsp;</td></tr><tr><td>&nbsp;</td><td>179.5 (168, 191)&nbsp;</td><td>4&nbsp;</td></tr><tr><td>&nbsp;</td><td>164.5 (161, 168)&nbsp;</td><td>3&nbsp;</td></tr><tr><td>&nbsp;</td><td>157.5 (152, 163)&nbsp;</td><td>17&nbsp;</td></tr><tr><td>&nbsp;</td><td>148.5 (145, 152)&nbsp;</td><td>1&nbsp;</td></tr><tr><td>&nbsp;</td><td>138.5 (125, 152)&nbsp;</td><td>1&nbsp;</td></tr><tr><td>&nbsp;</td><td>135 (125, 145)&nbsp;</td><td>6&nbsp;</td></tr><tr><td>&nbsp;</td><td>134 (133, 135)&nbsp;</td><td>2&nbsp;</td></tr><tr><td>&nbsp;</td><td>119 (113, 125)&nbsp;</td><td>2&nbsp;</td></tr><tr><td>&nbsp;</td><td>97 (94, 100)&nbsp;</td><td>1&nbsp;</td></tr><tr><td>&nbsp;</td><td>83 (80, 86)&nbsp;</td><td>1&nbsp;</td></tr><tr><td>Hymenoptera&nbsp;</td><td>0&nbsp;</td><td>60&nbsp;</td></tr><tr><td>Outgroups&nbsp;</td><td>0&nbsp;</td><td>8&nbsp;</td></tr><tr><td>Sum&nbsp;</td><td>&nbsp;</td><td>113&nbsp;</td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id="FN4"><div class="fn"><p>Notes:
 The data include 60 extant and 45 fossil hymenopteran taxa and 8 
outgroups. The estimated fossil ages are given as the midpoint and the 
time interval of the corresponding geographical stratum. The bounds of 
the stratum were used as parameters to a uniform prior on the fossil age
 in the total-evidence dating analyses.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="https://academic.oup.com/view-large/115286578" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="T3" class="table-wrap-title" data-id="T3"><div class="label -label">Table 3.</div><div class="caption"><p>Summary of Hymenoptera data</p></div> </div><div class="table-overflow"><table><thead name="thead" align="left"><tr><th>Taxa&nbsp;</th><th>Time (Ma)&nbsp;</th><th>Number&nbsp;</th></tr></thead><tbody><tr><td>Fossils&nbsp;</td><td>235 (228, 242)&nbsp;</td><td>2&nbsp;</td></tr><tr><td>&nbsp;</td><td>187.5 (174, 201)&nbsp;</td><td>1&nbsp;</td></tr><tr><td>&nbsp;</td><td>182.5 (182, 183)&nbsp;</td><td>4&nbsp;</td></tr><tr><td>&nbsp;</td><td>179.5 (168, 191)&nbsp;</td><td>4&nbsp;</td></tr><tr><td>&nbsp;</td><td>164.5 (161, 168)&nbsp;</td><td>3&nbsp;</td></tr><tr><td>&nbsp;</td><td>157.5 (152, 163)&nbsp;</td><td>17&nbsp;</td></tr><tr><td>&nbsp;</td><td>148.5 (145, 152)&nbsp;</td><td>1&nbsp;</td></tr><tr><td>&nbsp;</td><td>138.5 (125, 152)&nbsp;</td><td>1&nbsp;</td></tr><tr><td>&nbsp;</td><td>135 (125, 145)&nbsp;</td><td>6&nbsp;</td></tr><tr><td>&nbsp;</td><td>134 (133, 135)&nbsp;</td><td>2&nbsp;</td></tr><tr><td>&nbsp;</td><td>119 (113, 125)&nbsp;</td><td>2&nbsp;</td></tr><tr><td>&nbsp;</td><td>97 (94, 100)&nbsp;</td><td>1&nbsp;</td></tr><tr><td>&nbsp;</td><td>83 (80, 86)&nbsp;</td><td>1&nbsp;</td></tr><tr><td>Hymenoptera&nbsp;</td><td>0&nbsp;</td><td>60&nbsp;</td></tr><tr><td>Outgroups&nbsp;</td><td>0&nbsp;</td><td>8&nbsp;</td></tr><tr><td>Sum&nbsp;</td><td>&nbsp;</td><td>113&nbsp;</td></tr></tbody></table></div><div content-id="T3" class="table-modal"><table><thead name="thead" align="left"><tr><th>Taxa&nbsp;</th><th>Time (Ma)&nbsp;</th><th>Number&nbsp;</th></tr></thead><tbody><tr><td>Fossils&nbsp;</td><td>235 (228, 242)&nbsp;</td><td>2&nbsp;</td></tr><tr><td>&nbsp;</td><td>187.5 (174, 201)&nbsp;</td><td>1&nbsp;</td></tr><tr><td>&nbsp;</td><td>182.5 (182, 183)&nbsp;</td><td>4&nbsp;</td></tr><tr><td>&nbsp;</td><td>179.5 (168, 191)&nbsp;</td><td>4&nbsp;</td></tr><tr><td>&nbsp;</td><td>164.5 (161, 168)&nbsp;</td><td>3&nbsp;</td></tr><tr><td>&nbsp;</td><td>157.5 (152, 163)&nbsp;</td><td>17&nbsp;</td></tr><tr><td>&nbsp;</td><td>148.5 (145, 152)&nbsp;</td><td>1&nbsp;</td></tr><tr><td>&nbsp;</td><td>138.5 (125, 152)&nbsp;</td><td>1&nbsp;</td></tr><tr><td>&nbsp;</td><td>135 (125, 145)&nbsp;</td><td>6&nbsp;</td></tr><tr><td>&nbsp;</td><td>134 (133, 135)&nbsp;</td><td>2&nbsp;</td></tr><tr><td>&nbsp;</td><td>119 (113, 125)&nbsp;</td><td>2&nbsp;</td></tr><tr><td>&nbsp;</td><td>97 (94, 100)&nbsp;</td><td>1&nbsp;</td></tr><tr><td>&nbsp;</td><td>83 (80, 86)&nbsp;</td><td>1&nbsp;</td></tr><tr><td>Hymenoptera&nbsp;</td><td>0&nbsp;</td><td>60&nbsp;</td></tr><tr><td>Outgroups&nbsp;</td><td>0&nbsp;</td><td>8&nbsp;</td></tr><tr><td>Sum&nbsp;</td><td>&nbsp;</td><td>113&nbsp;</td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id="FN4"><div class="fn"><p>Notes:
 The data include 60 extant and 45 fossil hymenopteran taxa and 8 
outgroups. The estimated fossil ages are given as the midpoint and the 
time interval of the corresponding geographical stratum. The bounds of 
the stratum were used as parameters to a uniform prior on the fossil age
 in the total-evidence dating analyses.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="https://academic.oup.com/view-large/115286578" target="_blank">View Large</a></div></div><p> In contrast to the previous analysis ( <a reveal-id="B44" data-open="B44" class="link link-ref link-reveal xref-bibr">Ronquist et al. 2012b</a>
 ), which treated fossil ages as known without error, here we accounted 
for the uncertainty in those estimates. The fossil ages were assigned 
uniform prior distributions with ranges corresponding to the age ranges 
of the respective strata ( <a reveal-id="T3" data-open="T3" class="link link-reveal link-table xref-fig">Table 3</a> ). </p><p> We used both the uniform tree prior ( <a reveal-id="B43" data-open="B43" class="link link-ref link-reveal xref-bibr">Ronquist et al. 2012a</a>
 ), as in the previous analysis, and different versions of the FBD 
prior. For the FBD prior, we first assumed constant birth and death 
rates. For random sampling of extant taxa, the fossil sampling rate was 
assumed to be constant through time, whereas for diversified sampling, 
fossil sampling was constant (nonzero) before the cutoff time ( <span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><msub><mi>x</mi><mrow><mi>c</mi><mi>u</mi><mi>t</mi></mrow></msub></mrow></math></span> ) and zero thereafter. The value of <span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><msub><mi>x</mi><mrow><mi>c</mi><mi>u</mi><mi>t</mi></mrow></msub></mrow></math></span>
 was adjusted during the MCMC so that the cutoff was just after the 
youngest internal node (and the youngest fossil) in the FBD tree. </p><p>
 Because all fossils included in our data set were sampled from the 
Mesozoic (Triassic, Jurassic, and Cretaceous) and none from younger or 
older strata, we also used piecewise-constant rates in the FBD prior. 
For random sampling, the two rate-shifting times were fixed to 252 Ma 
(separating the Permian and the Triassic) and 66 Ma (separating the 
Cretaceous and the Paleocene). The birth, death, and sampling rates 
shift at the same time, so that we inferred three rates each for 
speciation, extinction, and fossil sampling, respectively. For 
diversified sampling, we only used one rate-shifting time for birth and 
death rates at 252 Ma, because the cutoff time ( <span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><msub><mi>x</mi><mrow><mi>c</mi><mi>u</mi><mi>t</mi></mrow></msub></mrow></math></span>
 , the additional shifting time for the fossil sampling rate) is very 
close to or slightly older than 66 Ma (see results below). To infer 
separate birth and death rates after time <span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><msub><mi>x</mi><mrow><mi>c</mi><mi>u</mi><mi>t</mi></mrow></msub></mrow></math></span>
 would lead to very uncertain estimates, because that time interval by 
definition consists of single lineages leading to each extant taxon. The
 fossil sampling rate after time <span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><msub><mi>x</mi><mrow><mi>c</mi><mi>u</mi><mi>t</mi></mrow></msub></mrow></math></span>
 is zero by definition. Under this approach, we thus inferred only two 
rates each for speciation, extinction, and fossil sampling, 
respectively. </p><p> We used an Exponential(100) prior for net diversification ( <span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mi xmlns="">d</mi></math></span> ), and a Beta(1, 1) prior for turnover ( <span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mi xmlns="">r</mi></math></span> ) and fossil sampling proportion ( <span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mi xmlns="">s</mi></math></span>
 ). The sampling probability (or sampling proportion) for the extant 
taxa was set to 0.0005; this estimate is based on the number of 
described extant hymenopteran species ( <a reveal-id="B1" data-open="B1" class="link link-ref link-reveal xref-bibr">Aguiar et al. 2013</a> ). The prior for the root age ( <span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><msub><mi>t</mi><mrow><mi>m</mi><mi>r</mi><mi>c</mi><mi>a</mi></mrow></msub></mrow></math></span>
 ) was set to “offsetExp(315, 396)”, that is, an offset exponential 
distribution with mean 396 (oldest insect fossil) and minimum 315 
(oldest neopteran fossil). The holometabolan taxa were constrained to be
 monophyletic, and the prior for their age was set to “offsetExp(302, 
396)” with minimum 302 (age of the oldest putative Holometabola). The 
holometabolan constraint are enforced to root the tree properly, as the 
clock model itself is not sufficient to provide the correct rooting for 
this data ( <a reveal-id="B43" data-open="B43" class="link link-ref link-reveal xref-bibr">Ronquist et al. 2012a</a> ). </p><p>
 We used a lognormal(−7.1, 0.5) prior for the substitution rate (the 
base rate of the clock) with mean 0.0009, median 0.0008, and mode 
0.0006. These settings were chosen by comparing the age of the oldest 
insect fossil with the root age estimation from uncalibrated clock 
analyses as discussed in <a reveal-id="B43" data-open="B43" class="link link-ref link-reveal xref-bibr">Ronquist et al. (2012a)</a>
 . We performed sensitivity analyses to determine the robustness of our 
estimates to changes in the priors on root age and clock rate (see 
results below). <a reveal-id="B43" data-open="B43" class="link link-ref link-reveal xref-bibr">Ronquist et al. (2012a)</a>
 noted that different relaxed-clock models produced very different 
divergence time estimates for this data. To explore whether these 
differences would be affected by replacing the simplistic uniform tree 
prior with the more realistic FBD model, we used two relaxed-clock 
models: the autocorrelated lognormal rates (TK02) model ( <a reveal-id="B53" data-open="B53" class="link link-ref link-reveal xref-bibr">Thorne and Kishino 2002</a>
 ) with an Exponential(0.2) prior for the variance increase parameter, 
and the uncorrelated independent gamma rates or “white noise” (IGR) 
model ( <a reveal-id="B32" data-open="B32" class="link link-ref link-reveal xref-bibr">Lepage et al. 2007</a>
 ) with an Exponential(37) prior for the variance increase parameter. As
 currently implemented in MrBayes, the compound Poisson process (CPP) 
model ( <a reveal-id="B23" data-open="B23" class="link link-ref link-reveal xref-bibr">Huelsenbeck et al. 2000</a> ) is computationally not compatible with the FBD model, and was not included in this study. </p><p>We
 executed four independent runs in parallel for each analysis, each 
consisting of four Metropolis-coupled MCMC chains (one cold, three 
heated). The length of each run was initially set to 50 million 
iterations, but enlarged to 100 million for the piecewise-constant FBD 
priors to achieve better convergence. Convergence was assessed by the 
MrBayes built-in diagnostics: the average standard deviation of split 
frequencies (ASDSF; target value 0.05) and the estimated effective 
sample size (ESS; target value 100). We also examined trace plots of 
likelihoods and parameter samples. The chain was sampled every 1000 
iterations, with the first 25% (or 50% for the longer chains) samples 
discarded as burn-in. The MCMC samples from the four parallel runs were 
then combined.</p>                            <h3 scrollto-destination="115286585" id="115286585" class="section-title">Results</h3>
<p> We first checked the induced tree prior by running the MCMC without 
data (by setting the sequence and morphology data likelihood to 1), 
under the model priors described above with fixed clock rate (0.001). We
 also explored a more relaxed (Exponential(1)) and a more constrained 
prior (Exponential(10000)) on net diversification ( <span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mi xmlns="">d</mi></math></span> ). It turns out that the induced divergence time priors vary widely under different prior assumptions ( <a reveal-id="T4" data-open="T4" class="link link-reveal link-table xref-fig">Table 4</a>
 ). Under the uniform tree model, the mean of the root age in the 
induced prior is much older than 396, the expectation of the root age 
prior. Under the FBD model, the induced divergence times vary widely: 
they are sensitive both to the model details and to the prior for <span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mi xmlns="">d</mi></math></span>
 . The expected proportion of ancestral fossils is consistently close to
 zero in the induced priors except for two cases involving the same type
 of FBD model ( <a reveal-id="T4" data-open="T4" class="link link-reveal link-table xref-fig">Table 4</a>
 , pcFBD\_Rnd). In both these cases, the induced prior favors young 
trees, thus apparently increasing the probability of fossils being 
ancestors. This phenomenon seems to be coupled with a high net 
diversification rate in the induced prior, as the expected proportion of
 ancestral fossils decreased dramatically when the prior was focused on 
small values of <span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mi xmlns="">d</mi></math></span> . </p>                        <a id="115286587" scrollto-destination="115286587"></a>
<div content-id="T4" class="table-modal" style="display: block;"><div class="table-wrap"><div id="T4" class="table-wrap-title" data-id="T4"><div class="label -label">Table 4.</div><div class="caption"><p> Induced prior distribution on root age ( <span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><msub><mi>t</mi><mrow><mi>m</mi><mi>r</mi><mi>c</mi><mi>a</mi></mrow></msub></mrow></math></span> ) and age of Hymenoptera </p></div> </div><div class="table-overflow"><table><thead name="thead" align="left"><tr><th>Tree Prior&nbsp;</th><th>Prior Root Age&nbsp;</th><th>Prior Age of Hymenoptera (Ma)&nbsp;</th><th>Proportion of Ancestral Fossils&nbsp;</th></tr></thead><tbody><tr><td>Uniform&nbsp;</td><td>486.6 (395.9, 650.3)&nbsp;</td><td>482.4 (392.2, 648.5)&nbsp;</td><td>—&nbsp;</td></tr><tr><td colspan="4"><span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><mi>d</mi><mo>∼</mo></mrow></math></span> Exp(1) &nbsp;</td></tr><tr><td>FBD (Rnd)&nbsp;</td><td>1235 (920.1, 1532.4)&nbsp;</td><td>1194 (893.6, 1476.9)&nbsp;</td><td>0.00 (0.0, 0.009)&nbsp;</td></tr><tr><td>FBD (Div)&nbsp;</td><td>321.6 (315.0, 343.3)&nbsp;</td><td>265.1 (222.8, 316.2)&nbsp;</td><td>0.00 (0.00, 0.01)&nbsp;</td></tr><tr><td>pcFBD (Rnd)&nbsp;</td><td>346.7 (315.0, 477.2)&nbsp;</td><td>170.4 (54.35, 288.7)&nbsp;</td><td>0.51 (0.14, 0.82)&nbsp;</td></tr><tr><td>pcFBD (Div)&nbsp;</td><td>350.9 (315.0, 484.9)&nbsp;</td><td>239.5 (196.4, 315.6)&nbsp;</td><td>0.00 (0.00, 0.02)&nbsp;</td></tr><tr><td colspan="4"><span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><mi>d</mi><mo>∼</mo></mrow></math></span> Exp(100) &nbsp;</td></tr><tr><td>FBD (Rnd)&nbsp;</td><td>861.8 (315.0, 1483.)&nbsp;</td><td>832.2 (25.4, 1436.2)&nbsp;</td><td>0.00 (0.0, 0.009)&nbsp;</td></tr><tr><td>FBD (Div)&nbsp;</td><td>321.7 (315.0, 343.1)&nbsp;</td><td>265.3 (223.2, 315.6)&nbsp;</td><td>0.00 (0.00, 0.02)&nbsp;</td></tr><tr><td>pcFBD (Rnd)&nbsp;</td><td>355.9 (315.0, 494.3)&nbsp;</td><td>209.3 (61.69, 322.4)&nbsp;</td><td>0.40 (0.07, 0.78)&nbsp;</td></tr><tr><td>pcFBD (Div)&nbsp;</td><td>358.8 (315.0, 517.0)&nbsp;</td><td>233.5 (202.7, 252.0)&nbsp;</td><td>0.00 (0.00, 0.03)&nbsp;</td></tr><tr><td colspan="4"><span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><mi>d</mi><mo>∼</mo></mrow></math></span> Exp(10000) &nbsp;</td></tr><tr><td>FBD (Rnd)&nbsp;</td><td>326.5 (315.0, 365.5)&nbsp;</td><td>279.2 (64.86, 345.0)&nbsp;</td><td>0.00 (0.00, 0.01)&nbsp;</td></tr><tr><td>FBD (Div)&nbsp;</td><td>320.0 (315.0, 335.7)&nbsp;</td><td>235.8 (196.0, 286.0)&nbsp;</td><td>0.00 (0.00, 0.02)&nbsp;</td></tr><tr><td>pcFBD (Rnd)&nbsp;</td><td>570.2 (315.0, 1098.)&nbsp;</td><td>458.3 (235.0, 1025.)&nbsp;</td><td>0.00 (0.00, 0.01)&nbsp;</td></tr><tr><td>pcFBD (Div)&nbsp;</td><td>360.4 (315.0, 519.6)&nbsp;</td><td>246.0 (219.9, 252.0)&nbsp;</td><td>0.00 (0.00, 0.01)&nbsp;</td></tr></tbody></table></div><div content-id="T4" class="table-modal"><table><thead name="thead" align="left"><tr><th>Tree Prior&nbsp;</th><th>Prior Root Age&nbsp;</th><th>Prior Age of Hymenoptera (Ma)&nbsp;</th><th>Proportion of Ancestral Fossils&nbsp;</th></tr></thead><tbody><tr><td>Uniform&nbsp;</td><td>486.6 (395.9, 650.3)&nbsp;</td><td>482.4 (392.2, 648.5)&nbsp;</td><td>—&nbsp;</td></tr><tr><td colspan="4"><span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><mi>d</mi><mo>∼</mo></mrow></math></span> Exp(1) &nbsp;</td></tr><tr><td>FBD (Rnd)&nbsp;</td><td>1235 (920.1, 1532.4)&nbsp;</td><td>1194 (893.6, 1476.9)&nbsp;</td><td>0.00 (0.0, 0.009)&nbsp;</td></tr><tr><td>FBD (Div)&nbsp;</td><td>321.6 (315.0, 343.3)&nbsp;</td><td>265.1 (222.8, 316.2)&nbsp;</td><td>0.00 (0.00, 0.01)&nbsp;</td></tr><tr><td>pcFBD (Rnd)&nbsp;</td><td>346.7 (315.0, 477.2)&nbsp;</td><td>170.4 (54.35, 288.7)&nbsp;</td><td>0.51 (0.14, 0.82)&nbsp;</td></tr><tr><td>pcFBD (Div)&nbsp;</td><td>350.9 (315.0, 484.9)&nbsp;</td><td>239.5 (196.4, 315.6)&nbsp;</td><td>0.00 (0.00, 0.02)&nbsp;</td></tr><tr><td colspan="4"><span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><mi>d</mi><mo>∼</mo></mrow></math></span> Exp(100) &nbsp;</td></tr><tr><td>FBD (Rnd)&nbsp;</td><td>861.8 (315.0, 1483.)&nbsp;</td><td>832.2 (25.4, 1436.2)&nbsp;</td><td>0.00 (0.0, 0.009)&nbsp;</td></tr><tr><td>FBD (Div)&nbsp;</td><td>321.7 (315.0, 343.1)&nbsp;</td><td>265.3 (223.2, 315.6)&nbsp;</td><td>0.00 (0.00, 0.02)&nbsp;</td></tr><tr><td>pcFBD (Rnd)&nbsp;</td><td>355.9 (315.0, 494.3)&nbsp;</td><td>209.3 (61.69, 322.4)&nbsp;</td><td>0.40 (0.07, 0.78)&nbsp;</td></tr><tr><td>pcFBD (Div)&nbsp;</td><td>358.8 (315.0, 517.0)&nbsp;</td><td>233.5 (202.7, 252.0)&nbsp;</td><td>0.00 (0.00, 0.03)&nbsp;</td></tr><tr><td colspan="4"><span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><mi>d</mi><mo>∼</mo></mrow></math></span> Exp(10000) &nbsp;</td></tr><tr><td>FBD (Rnd)&nbsp;</td><td>326.5 (315.0, 365.5)&nbsp;</td><td>279.2 (64.86, 345.0)&nbsp;</td><td>0.00 (0.00, 0.01)&nbsp;</td></tr><tr><td>FBD (Div)&nbsp;</td><td>320.0 (315.0, 335.7)&nbsp;</td><td>235.8 (196.0, 286.0)&nbsp;</td><td>0.00 (0.00, 0.02)&nbsp;</td></tr><tr><td>pcFBD (Rnd)&nbsp;</td><td>570.2 (315.0, 1098.)&nbsp;</td><td>458.3 (235.0, 1025.)&nbsp;</td><td>0.00 (0.00, 0.01)&nbsp;</td></tr><tr><td>pcFBD (Div)&nbsp;</td><td>360.4 (315.0, 519.6)&nbsp;</td><td>246.0 (219.9, 252.0)&nbsp;</td><td>0.00 (0.00, 0.01)&nbsp;</td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id="FN5"><div class="fn"><p> Notes: Numbers represent the median and the 95% HPD interval. The hyperprior for <span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><mi>d</mi><mo>=</mo><mi>λ</mi><mo>−</mo><mi>μ</mi></mrow></math></span> is Exponential (rate parameter specified in the table), for <span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><mi>r</mi><mo>=</mo><mi>μ</mi><mo>/</mo><mi>λ</mi></mrow></math></span> and <span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><mi>s</mi><mo>=</mo><mi>ψ</mi><mo>/</mo><mo stretchy="false">(</mo><mi>μ</mi><mo>+</mo><mi>ψ</mi><mo stretchy="false">)</mo></mrow></math></span>
 is Beta(1, 1), for the root age is Exponential with mean 396 and offset
 315, and for the fossils are uniform on the time interval of the 
stratum in which they were found ( <a reveal-id="T3" data-open="T3" class="link link-reveal link-table xref-fig">Table 3</a> ). For the full empirical analysis ( <a reveal-id="T5" data-open="T5" class="link link-reveal link-table xref-fig">Table 5</a> ), the intermediate prior on <span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mi xmlns="">d</mi></math></span> was used. </p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="https://academic.oup.com/view-large/115286587" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="T4" class="table-wrap-title" data-id="T4"><div class="label -label">Table 4.</div><div class="caption"><p> Induced prior distribution on root age ( <span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><msub><mi>t</mi><mrow><mi>m</mi><mi>r</mi><mi>c</mi><mi>a</mi></mrow></msub></mrow></math></span> ) and age of Hymenoptera </p></div> </div><div class="table-overflow"><table><thead name="thead" align="left"><tr><th>Tree Prior&nbsp;</th><th>Prior Root Age&nbsp;</th><th>Prior Age of Hymenoptera (Ma)&nbsp;</th><th>Proportion of Ancestral Fossils&nbsp;</th></tr></thead><tbody><tr><td>Uniform&nbsp;</td><td>486.6 (395.9, 650.3)&nbsp;</td><td>482.4 (392.2, 648.5)&nbsp;</td><td>—&nbsp;</td></tr><tr><td colspan="4"><span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><mi>d</mi><mo>∼</mo></mrow></math></span> Exp(1) &nbsp;</td></tr><tr><td>FBD (Rnd)&nbsp;</td><td>1235 (920.1, 1532.4)&nbsp;</td><td>1194 (893.6, 1476.9)&nbsp;</td><td>0.00 (0.0, 0.009)&nbsp;</td></tr><tr><td>FBD (Div)&nbsp;</td><td>321.6 (315.0, 343.3)&nbsp;</td><td>265.1 (222.8, 316.2)&nbsp;</td><td>0.00 (0.00, 0.01)&nbsp;</td></tr><tr><td>pcFBD (Rnd)&nbsp;</td><td>346.7 (315.0, 477.2)&nbsp;</td><td>170.4 (54.35, 288.7)&nbsp;</td><td>0.51 (0.14, 0.82)&nbsp;</td></tr><tr><td>pcFBD (Div)&nbsp;</td><td>350.9 (315.0, 484.9)&nbsp;</td><td>239.5 (196.4, 315.6)&nbsp;</td><td>0.00 (0.00, 0.02)&nbsp;</td></tr><tr><td colspan="4"><span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><mi>d</mi><mo>∼</mo></mrow></math></span> Exp(100) &nbsp;</td></tr><tr><td>FBD (Rnd)&nbsp;</td><td>861.8 (315.0, 1483.)&nbsp;</td><td>832.2 (25.4, 1436.2)&nbsp;</td><td>0.00 (0.0, 0.009)&nbsp;</td></tr><tr><td>FBD (Div)&nbsp;</td><td>321.7 (315.0, 343.1)&nbsp;</td><td>265.3 (223.2, 315.6)&nbsp;</td><td>0.00 (0.00, 0.02)&nbsp;</td></tr><tr><td>pcFBD (Rnd)&nbsp;</td><td>355.9 (315.0, 494.3)&nbsp;</td><td>209.3 (61.69, 322.4)&nbsp;</td><td>0.40 (0.07, 0.78)&nbsp;</td></tr><tr><td>pcFBD (Div)&nbsp;</td><td>358.8 (315.0, 517.0)&nbsp;</td><td>233.5 (202.7, 252.0)&nbsp;</td><td>0.00 (0.00, 0.03)&nbsp;</td></tr><tr><td colspan="4"><span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><mi>d</mi><mo>∼</mo></mrow></math></span> Exp(10000) &nbsp;</td></tr><tr><td>FBD (Rnd)&nbsp;</td><td>326.5 (315.0, 365.5)&nbsp;</td><td>279.2 (64.86, 345.0)&nbsp;</td><td>0.00 (0.00, 0.01)&nbsp;</td></tr><tr><td>FBD (Div)&nbsp;</td><td>320.0 (315.0, 335.7)&nbsp;</td><td>235.8 (196.0, 286.0)&nbsp;</td><td>0.00 (0.00, 0.02)&nbsp;</td></tr><tr><td>pcFBD (Rnd)&nbsp;</td><td>570.2 (315.0, 1098.)&nbsp;</td><td>458.3 (235.0, 1025.)&nbsp;</td><td>0.00 (0.00, 0.01)&nbsp;</td></tr><tr><td>pcFBD (Div)&nbsp;</td><td>360.4 (315.0, 519.6)&nbsp;</td><td>246.0 (219.9, 252.0)&nbsp;</td><td>0.00 (0.00, 0.01)&nbsp;</td></tr></tbody></table></div><div content-id="T4" class="table-modal"><table><thead name="thead" align="left"><tr><th>Tree Prior&nbsp;</th><th>Prior Root Age&nbsp;</th><th>Prior Age of Hymenoptera (Ma)&nbsp;</th><th>Proportion of Ancestral Fossils&nbsp;</th></tr></thead><tbody><tr><td>Uniform&nbsp;</td><td>486.6 (395.9, 650.3)&nbsp;</td><td>482.4 (392.2, 648.5)&nbsp;</td><td>—&nbsp;</td></tr><tr><td colspan="4"><span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><mi>d</mi><mo>∼</mo></mrow></math></span> Exp(1) &nbsp;</td></tr><tr><td>FBD (Rnd)&nbsp;</td><td>1235 (920.1, 1532.4)&nbsp;</td><td>1194 (893.6, 1476.9)&nbsp;</td><td>0.00 (0.0, 0.009)&nbsp;</td></tr><tr><td>FBD (Div)&nbsp;</td><td>321.6 (315.0, 343.3)&nbsp;</td><td>265.1 (222.8, 316.2)&nbsp;</td><td>0.00 (0.00, 0.01)&nbsp;</td></tr><tr><td>pcFBD (Rnd)&nbsp;</td><td>346.7 (315.0, 477.2)&nbsp;</td><td>170.4 (54.35, 288.7)&nbsp;</td><td>0.51 (0.14, 0.82)&nbsp;</td></tr><tr><td>pcFBD (Div)&nbsp;</td><td>350.9 (315.0, 484.9)&nbsp;</td><td>239.5 (196.4, 315.6)&nbsp;</td><td>0.00 (0.00, 0.02)&nbsp;</td></tr><tr><td colspan="4"><span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><mi>d</mi><mo>∼</mo></mrow></math></span> Exp(100) &nbsp;</td></tr><tr><td>FBD (Rnd)&nbsp;</td><td>861.8 (315.0, 1483.)&nbsp;</td><td>832.2 (25.4, 1436.2)&nbsp;</td><td>0.00 (0.0, 0.009)&nbsp;</td></tr><tr><td>FBD (Div)&nbsp;</td><td>321.7 (315.0, 343.1)&nbsp;</td><td>265.3 (223.2, 315.6)&nbsp;</td><td>0.00 (0.00, 0.02)&nbsp;</td></tr><tr><td>pcFBD (Rnd)&nbsp;</td><td>355.9 (315.0, 494.3)&nbsp;</td><td>209.3 (61.69, 322.4)&nbsp;</td><td>0.40 (0.07, 0.78)&nbsp;</td></tr><tr><td>pcFBD (Div)&nbsp;</td><td>358.8 (315.0, 517.0)&nbsp;</td><td>233.5 (202.7, 252.0)&nbsp;</td><td>0.00 (0.00, 0.03)&nbsp;</td></tr><tr><td colspan="4"><span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><mi>d</mi><mo>∼</mo></mrow></math></span> Exp(10000) &nbsp;</td></tr><tr><td>FBD (Rnd)&nbsp;</td><td>326.5 (315.0, 365.5)&nbsp;</td><td>279.2 (64.86, 345.0)&nbsp;</td><td>0.00 (0.00, 0.01)&nbsp;</td></tr><tr><td>FBD (Div)&nbsp;</td><td>320.0 (315.0, 335.7)&nbsp;</td><td>235.8 (196.0, 286.0)&nbsp;</td><td>0.00 (0.00, 0.02)&nbsp;</td></tr><tr><td>pcFBD (Rnd)&nbsp;</td><td>570.2 (315.0, 1098.)&nbsp;</td><td>458.3 (235.0, 1025.)&nbsp;</td><td>0.00 (0.00, 0.01)&nbsp;</td></tr><tr><td>pcFBD (Div)&nbsp;</td><td>360.4 (315.0, 519.6)&nbsp;</td><td>246.0 (219.9, 252.0)&nbsp;</td><td>0.00 (0.00, 0.01)&nbsp;</td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id="FN5"><div class="fn"><p> Notes: Numbers represent the median and the 95% HPD interval. The hyperprior for <span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><mi>d</mi><mo>=</mo><mi>λ</mi><mo>−</mo><mi>μ</mi></mrow></math></span> is Exponential (rate parameter specified in the table), for <span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><mi>r</mi><mo>=</mo><mi>μ</mi><mo>/</mo><mi>λ</mi></mrow></math></span> and <span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><mi>s</mi><mo>=</mo><mi>ψ</mi><mo>/</mo><mo stretchy="false">(</mo><mi>μ</mi><mo>+</mo><mi>ψ</mi><mo stretchy="false">)</mo></mrow></math></span>
 is Beta(1, 1), for the root age is Exponential with mean 396 and offset
 315, and for the fossils are uniform on the time interval of the 
stratum in which they were found ( <a reveal-id="T3" data-open="T3" class="link link-reveal link-table xref-fig">Table 3</a> ). For the full empirical analysis ( <a reveal-id="T5" data-open="T5" class="link link-reveal link-table xref-fig">Table 5</a> ), the intermediate prior on <span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mi xmlns="">d</mi></math></span> was used. </p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="https://academic.oup.com/view-large/115286587" target="_blank">View Large</a></div></div>                        <a id="115286588" scrollto-destination="115286588"></a>
<div content-id="T5" class="table-modal" style="display: block;"><div class="table-wrap"><div id="T5" class="table-wrap-title" data-id="T5"><div class="label -label">Table 5.</div><div class="caption"><p> Posterior distribution on root age ( <span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><msub><mi>t</mi><mrow><mi>m</mi><mi>r</mi><mi>c</mi><mi>a</mi></mrow></msub></mrow></math></span> ) and age of Hymenoptera </p></div> </div><div class="table-overflow"><table><thead name="thead" align="left"><tr><th>Tree Prior&nbsp;</th><th>Posterior Root Age (Ma)&nbsp;</th><th>Posterior Age of Hymenoptera (Ma)&nbsp;</th><th>Proportion of Ancestral Fossils&nbsp;</th></tr></thead><tbody><tr><td colspan="4">IGR relaxed clock&nbsp;</td></tr><tr><td>Uniform (U)&nbsp;</td><td>345.1 (315.0, 402.2)&nbsp;</td><td>306.0 (289.3, 341.3)&nbsp;</td><td>—&nbsp;</td></tr><tr><td>FBD (Rnd)&nbsp;</td><td>406.4 (328.1, 500.3)&nbsp;</td><td>346.6 (291.9, 426.5)&nbsp;</td><td>0.00 (0.00, 0.02)&nbsp;</td></tr><tr><td>FBD (Div)&nbsp;</td><td>322.7 (315.1, 342.6)&nbsp;</td><td>279.4 (255.1, 304.3)&nbsp;</td><td>0.02 (0.00, 0.07)&nbsp;</td></tr><tr><td>pcFBD (Rnd)&nbsp;</td><td>369.6 (315.6, 433.3)&nbsp;</td><td>325.6 (282.5, 374.9)&nbsp;</td><td>0.33 (0.20, 0.47)&nbsp;</td></tr><tr><td>pcFBD (Div)&nbsp;</td><td>328.2 (315.0, 358.9)&nbsp;</td><td>251.7 (237.9, 310.9)&nbsp;</td><td>0.02 (0.00, 0.07)&nbsp;</td></tr><tr><td colspan="4">TK02 relaxed clock&nbsp;</td></tr><tr><td>Uniform (U)&nbsp;</td><td>462.5 (368.3, 549.8)&nbsp;</td><td>372.5 (310.1, 446.8)&nbsp;</td><td>—&nbsp;</td></tr><tr><td>FBD (Rnd)&nbsp;</td><td>523.1 (438.2, 605.4)&nbsp;</td><td>402.9 (344.7, 457.0)&nbsp;</td><td>0.00 (0.00, 0.00)&nbsp;</td></tr><tr><td>FBD (Div)&nbsp;</td><td>435.1 (371.3, 497.5)&nbsp;</td><td>343.5 (301.5, 385.0)&nbsp;</td><td>0.00 (0.00, 0.00)&nbsp;</td></tr><tr><td>pcFBD (Rnd)&nbsp;</td><td>484.1 (411.2, 551.5)&nbsp;</td><td>365.3 (324.3, 413.1)&nbsp;</td><td>0.02 (0.00, 0.07)&nbsp;</td></tr><tr><td>pcFBD (Div)&nbsp;</td><td>447.4 (324.9, 564.5)&nbsp;</td><td>346.1 (265.4, 442.7)&nbsp;</td><td>0.00 (0.00, 0.00)&nbsp;</td></tr></tbody></table></div><div content-id="T5" class="table-modal"><table><thead name="thead" align="left"><tr><th>Tree Prior&nbsp;</th><th>Posterior Root Age (Ma)&nbsp;</th><th>Posterior Age of Hymenoptera (Ma)&nbsp;</th><th>Proportion of Ancestral Fossils&nbsp;</th></tr></thead><tbody><tr><td colspan="4">IGR relaxed clock&nbsp;</td></tr><tr><td>Uniform (U)&nbsp;</td><td>345.1 (315.0, 402.2)&nbsp;</td><td>306.0 (289.3, 341.3)&nbsp;</td><td>—&nbsp;</td></tr><tr><td>FBD (Rnd)&nbsp;</td><td>406.4 (328.1, 500.3)&nbsp;</td><td>346.6 (291.9, 426.5)&nbsp;</td><td>0.00 (0.00, 0.02)&nbsp;</td></tr><tr><td>FBD (Div)&nbsp;</td><td>322.7 (315.1, 342.6)&nbsp;</td><td>279.4 (255.1, 304.3)&nbsp;</td><td>0.02 (0.00, 0.07)&nbsp;</td></tr><tr><td>pcFBD (Rnd)&nbsp;</td><td>369.6 (315.6, 433.3)&nbsp;</td><td>325.6 (282.5, 374.9)&nbsp;</td><td>0.33 (0.20, 0.47)&nbsp;</td></tr><tr><td>pcFBD (Div)&nbsp;</td><td>328.2 (315.0, 358.9)&nbsp;</td><td>251.7 (237.9, 310.9)&nbsp;</td><td>0.02 (0.00, 0.07)&nbsp;</td></tr><tr><td colspan="4">TK02 relaxed clock&nbsp;</td></tr><tr><td>Uniform (U)&nbsp;</td><td>462.5 (368.3, 549.8)&nbsp;</td><td>372.5 (310.1, 446.8)&nbsp;</td><td>—&nbsp;</td></tr><tr><td>FBD (Rnd)&nbsp;</td><td>523.1 (438.2, 605.4)&nbsp;</td><td>402.9 (344.7, 457.0)&nbsp;</td><td>0.00 (0.00, 0.00)&nbsp;</td></tr><tr><td>FBD (Div)&nbsp;</td><td>435.1 (371.3, 497.5)&nbsp;</td><td>343.5 (301.5, 385.0)&nbsp;</td><td>0.00 (0.00, 0.00)&nbsp;</td></tr><tr><td>pcFBD (Rnd)&nbsp;</td><td>484.1 (411.2, 551.5)&nbsp;</td><td>365.3 (324.3, 413.1)&nbsp;</td><td>0.02 (0.00, 0.07)&nbsp;</td></tr><tr><td>pcFBD (Div)&nbsp;</td><td>447.4 (324.9, 564.5)&nbsp;</td><td>346.1 (265.4, 442.7)&nbsp;</td><td>0.00 (0.00, 0.00)&nbsp;</td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id="FN6"><div class="fn"><p>Notes:
 Numbers represent the median and 95% HPD interval of the estimated 
posterior distribution under the uniform tree prior (U), constant-rate 
FBD prior with random (Rnd) or diversified (Div) sampling, and 
piecewise-constant FBD prior (pcFBD) with random (Rnd) or diversified 
(Div) sampling, and under either the IGR or the TK02 relaxed-clock 
model.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="https://academic.oup.com/view-large/115286588" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="T5" class="table-wrap-title" data-id="T5"><div class="label -label">Table 5.</div><div class="caption"><p> Posterior distribution on root age ( <span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><msub><mi>t</mi><mrow><mi>m</mi><mi>r</mi><mi>c</mi><mi>a</mi></mrow></msub></mrow></math></span> ) and age of Hymenoptera </p></div> </div><div class="table-overflow"><table><thead name="thead" align="left"><tr><th>Tree Prior&nbsp;</th><th>Posterior Root Age (Ma)&nbsp;</th><th>Posterior Age of Hymenoptera (Ma)&nbsp;</th><th>Proportion of Ancestral Fossils&nbsp;</th></tr></thead><tbody><tr><td colspan="4">IGR relaxed clock&nbsp;</td></tr><tr><td>Uniform (U)&nbsp;</td><td>345.1 (315.0, 402.2)&nbsp;</td><td>306.0 (289.3, 341.3)&nbsp;</td><td>—&nbsp;</td></tr><tr><td>FBD (Rnd)&nbsp;</td><td>406.4 (328.1, 500.3)&nbsp;</td><td>346.6 (291.9, 426.5)&nbsp;</td><td>0.00 (0.00, 0.02)&nbsp;</td></tr><tr><td>FBD (Div)&nbsp;</td><td>322.7 (315.1, 342.6)&nbsp;</td><td>279.4 (255.1, 304.3)&nbsp;</td><td>0.02 (0.00, 0.07)&nbsp;</td></tr><tr><td>pcFBD (Rnd)&nbsp;</td><td>369.6 (315.6, 433.3)&nbsp;</td><td>325.6 (282.5, 374.9)&nbsp;</td><td>0.33 (0.20, 0.47)&nbsp;</td></tr><tr><td>pcFBD (Div)&nbsp;</td><td>328.2 (315.0, 358.9)&nbsp;</td><td>251.7 (237.9, 310.9)&nbsp;</td><td>0.02 (0.00, 0.07)&nbsp;</td></tr><tr><td colspan="4">TK02 relaxed clock&nbsp;</td></tr><tr><td>Uniform (U)&nbsp;</td><td>462.5 (368.3, 549.8)&nbsp;</td><td>372.5 (310.1, 446.8)&nbsp;</td><td>—&nbsp;</td></tr><tr><td>FBD (Rnd)&nbsp;</td><td>523.1 (438.2, 605.4)&nbsp;</td><td>402.9 (344.7, 457.0)&nbsp;</td><td>0.00 (0.00, 0.00)&nbsp;</td></tr><tr><td>FBD (Div)&nbsp;</td><td>435.1 (371.3, 497.5)&nbsp;</td><td>343.5 (301.5, 385.0)&nbsp;</td><td>0.00 (0.00, 0.00)&nbsp;</td></tr><tr><td>pcFBD (Rnd)&nbsp;</td><td>484.1 (411.2, 551.5)&nbsp;</td><td>365.3 (324.3, 413.1)&nbsp;</td><td>0.02 (0.00, 0.07)&nbsp;</td></tr><tr><td>pcFBD (Div)&nbsp;</td><td>447.4 (324.9, 564.5)&nbsp;</td><td>346.1 (265.4, 442.7)&nbsp;</td><td>0.00 (0.00, 0.00)&nbsp;</td></tr></tbody></table></div><div content-id="T5" class="table-modal"><table><thead name="thead" align="left"><tr><th>Tree Prior&nbsp;</th><th>Posterior Root Age (Ma)&nbsp;</th><th>Posterior Age of Hymenoptera (Ma)&nbsp;</th><th>Proportion of Ancestral Fossils&nbsp;</th></tr></thead><tbody><tr><td colspan="4">IGR relaxed clock&nbsp;</td></tr><tr><td>Uniform (U)&nbsp;</td><td>345.1 (315.0, 402.2)&nbsp;</td><td>306.0 (289.3, 341.3)&nbsp;</td><td>—&nbsp;</td></tr><tr><td>FBD (Rnd)&nbsp;</td><td>406.4 (328.1, 500.3)&nbsp;</td><td>346.6 (291.9, 426.5)&nbsp;</td><td>0.00 (0.00, 0.02)&nbsp;</td></tr><tr><td>FBD (Div)&nbsp;</td><td>322.7 (315.1, 342.6)&nbsp;</td><td>279.4 (255.1, 304.3)&nbsp;</td><td>0.02 (0.00, 0.07)&nbsp;</td></tr><tr><td>pcFBD (Rnd)&nbsp;</td><td>369.6 (315.6, 433.3)&nbsp;</td><td>325.6 (282.5, 374.9)&nbsp;</td><td>0.33 (0.20, 0.47)&nbsp;</td></tr><tr><td>pcFBD (Div)&nbsp;</td><td>328.2 (315.0, 358.9)&nbsp;</td><td>251.7 (237.9, 310.9)&nbsp;</td><td>0.02 (0.00, 0.07)&nbsp;</td></tr><tr><td colspan="4">TK02 relaxed clock&nbsp;</td></tr><tr><td>Uniform (U)&nbsp;</td><td>462.5 (368.3, 549.8)&nbsp;</td><td>372.5 (310.1, 446.8)&nbsp;</td><td>—&nbsp;</td></tr><tr><td>FBD (Rnd)&nbsp;</td><td>523.1 (438.2, 605.4)&nbsp;</td><td>402.9 (344.7, 457.0)&nbsp;</td><td>0.00 (0.00, 0.00)&nbsp;</td></tr><tr><td>FBD (Div)&nbsp;</td><td>435.1 (371.3, 497.5)&nbsp;</td><td>343.5 (301.5, 385.0)&nbsp;</td><td>0.00 (0.00, 0.00)&nbsp;</td></tr><tr><td>pcFBD (Rnd)&nbsp;</td><td>484.1 (411.2, 551.5)&nbsp;</td><td>365.3 (324.3, 413.1)&nbsp;</td><td>0.02 (0.00, 0.07)&nbsp;</td></tr><tr><td>pcFBD (Div)&nbsp;</td><td>447.4 (324.9, 564.5)&nbsp;</td><td>346.1 (265.4, 442.7)&nbsp;</td><td>0.00 (0.00, 0.00)&nbsp;</td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id="FN6"><div class="fn"><p>Notes:
 Numbers represent the median and 95% HPD interval of the estimated 
posterior distribution under the uniform tree prior (U), constant-rate 
FBD prior with random (Rnd) or diversified (Div) sampling, and 
piecewise-constant FBD prior (pcFBD) with random (Rnd) or diversified 
(Div) sampling, and under either the IGR or the TK02 relaxed-clock 
model.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="https://academic.oup.com/view-large/115286588" target="_blank">View Large</a></div></div><p> The full empirical analyses with the sequence and morphology data only used the intermediate Exponential(100) prior on <span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mi xmlns="">d</mi></math></span> . The posterior estimates of root age and age of Hymenoptera in these analyses are summarized in <a reveal-id="T5" data-open="T5" class="link link-reveal link-table xref-fig">Table 5</a>
 for a range of analyses under two kinds of relaxed-clock models (IGR 
and TK02) and two kinds of tree priors (uniform and FBD). In the case of
 the FBD prior, we assumed that sampling of extant taxa was either 
random or diversified. </p><p> Under the uniform tree prior ( <a reveal-id="T5" data-open="T5" class="link link-reveal link-table xref-fig">Table 5</a> U, see also Supplementary Figure S1 on Dryad at <a class="link link-uri" href="http://dx.doi.org/10.5061/dryad.26820" target="">http://dx.doi.org/10.5061/dryad.26820</a>
 ), accounting for the uncertainties of fossil ages using uniform 
priors, the node age estimates are very similar to those in the previous
 study ( <a reveal-id="B43" data-open="B43" class="link link-ref link-reveal xref-bibr">Ronquist et al. 2012a</a>
 ; Fig. 9) in which the fossil ages were treated as known without error.
 Under the IGR model, the median age of Hymenoptera is inferred to be in
 the Carboniferous, with the 95% HPD stretching into the Permian. The 
estimates are much older under the TK02 model, with the median age of 
Hymenoptera dating back to the Devonian, and the 95% HPD spanning the 
Carboniferous, Devonian, and Silurian. The same pattern was observed in <a reveal-id="B44" data-open="B44" class="link link-ref link-reveal xref-bibr">Ronquist et al. (2012b)</a>
 . The reason for the difference is apparently that the IGR model allows
 occasional extreme rate changes on adjacent branches, which makes it 
easier for this model to accommodate the obviously rapid rate shifts 
close to the root of the hymenopteran tree. The autocorrelated TK02 
model has a rate smoothing effect on adjacent branches, which tends to 
produce longer trees. As discussed in <a reveal-id="B44" data-open="B44" class="link link-ref link-reveal xref-bibr">Ronquist et al. (2012b)</a>
 , several lines of evidence suggest that the IGR model fits the data 
better. Therefore, we will focus on the IGR model in the following. </p><p>
 For the FBD priors, we first describe the results obtained under the 
assumption of constant speciation, extinction, and fossilization rates (
 <a reveal-id="T5" data-open="T5" class="link link-reveal link-table xref-fig">Table 5</a> FBD, see also Supplementary Figures S2 and S3 on Dryad at <a class="link link-uri" href="http://dx.doi.org/10.5061/dryad.26820" target="">http://dx.doi.org/10.5061/dryad.26820</a>
 ). Under these conditions, the modeling of the sampling process has a 
dramatic impact on the inferred divergence times. When extant taxa are 
assumed to have been sampled at random, the inferred age of Hymenoptera 
is older than under the uniform prior (347 Ma versus 306 Ma) and has a 
wider HPD interval. When changing from random to diversified sampling, 
the age of Hymenoptera becomes younger (279 Ma), now lying in the 
Permian. Compared with the random sampling prior, the diversified 
sampling assumption has the effect of stretching more recent and 
shrinking more ancestral branches, resulting in younger age estimates 
near the root of the tree (e.g., compare Supplementary Figures S2 and S3
 on Dryad at <a class="link link-uri" href="http://dx.doi.org/10.5061/dryad.26820" target="">http://dx.doi.org/10.5061/dryad.26820</a> ). </p><p>
 Finally, we applied the piecewise-constant FBD priors to account for 
variable rates of fossil sampling. We allowed net diversification <span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mi xmlns="">d</mi></math></span> and turnover <span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mi xmlns="">r</mi></math></span> to shift at the same time as <span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mi xmlns="">s</mi></math></span> at 252 Ma. For random sampling, <span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mi xmlns="">d</mi></math></span> , <span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mi xmlns="">r</mi></math></span> , and <span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mi xmlns="">s</mi></math></span> were allowed to change again at 66 Ma; for diversified sampling, only <span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mi xmlns="">s</mi></math></span>
 changed to zero at the cutoff time of about 70 Ma. Consequently, all 
fossils are included in the second time interval. The effect of allowing
 diversification, turnover, and fossilization rates to vary over time is
 to decrease the age estimates of Hymenoptera, by ∼20 myr ( <a reveal-id="T5" data-open="T5" class="link link-reveal link-table xref-fig">Table 5</a> , <a reveal-id="F6" data-open="F6" class="link link-reveal link-table xref-fig">Figs. 6</a> and <a reveal-id="F7" data-open="F7" class="link link-reveal link-table xref-fig">7</a>
 ). The estimate of 252 Ma is just at the geological time separating the
 Triassic and Permian. It is slightly older than the age of the oldest 
hymenopteran fossils, <em>Triassoxyela</em> and <em>Asioxyela</em> , which are dated to 228–242 Ma ( <a reveal-id="F7" data-open="F7" class="link link-reveal link-table xref-fig">Fig. 7</a> ). </p>                        <a id="115286593" scrollto-destination="115286593"></a>
<div id="F6" data-id="F6" class="fig fig-section"><div class="label fig-label">Figure 6.</div><div class="graphic-wrap"><img class="content-image" src="m_syv080f6.png" alt=" Majority-rule consensus tree, a) including all fossils and b) including only extant taxa, from total-evidence dating analysis under the piecewise-constant FBD prior with random sampling and under the IGR relaxed-clock model. Node bars indicate HPD intervals of estimated divergence times (cf. Table 5 , pcFBD_Rnd). " path-from-xml="syv080f6.tiff"><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="https://academic.oup.com/view-large/figure/115286593/syv080f6.tiff" path-from-xml="syv080f6.tiff" target="_blank">View large</a><a section="115286593" href="https://oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/sysbio/65/2/10.1093_sysbio_syv080/3/syv080f6.png?Expires=2147483647&amp;Signature=gbl2MmlEmOH5ZvQDesZd2rIQaisC0Vd1NDNW5zbgCHCfw70j9hwCoAbL2X8JG4DckLNMUyI-As2hhyUoAp20hzy9WjN0I6VoRNc14mLxPG%7ESwhyY6aGG5%7EoBNx%7E9vC-A62Ib0dl93HdgKwtJ%7E-BZLCEemXO7zvUhku0UkYYX%7EoJKDOOc2i8A2Wkk11zGmI1UpEazyN8PxYKu%7EKM9PUoav%7E0I2lMzCC9tNii%7EdVFVxo8Tgxzk7doxTdbhEXQKfUVkknfx0X0vpiM-F%7E1tG6Ha6ydP985Uzydkj86QY-sdsopk4jEvQqxv7utNwZoVVklBA65jKDWW3y6d26le%7EFentg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=115286593&amp;ar=2427164&amp;xsltPath=%7E/UI/app/XSLT&amp;imagename=" class="download-slide" path-from-xml="syv080f6.tiff" target="">Download slide</a></div></div><div class="caption fig-caption"><p>
 Majority-rule consensus tree, a) including all fossils and b) including
 only extant taxa, from total-evidence dating analysis under the 
piecewise-constant FBD prior with random sampling and under the IGR 
relaxed-clock model. Node bars indicate HPD intervals of estimated 
divergence times (cf. <a reveal-id="T5" data-open="T5" class="link link-reveal link-table xref-fig">Table 5</a> , pcFBD_Rnd). </p></div></div><div content-id="F6" class="fig fig-modal reveal-modal" style="display: block;"><div class="label fig-label">Figure 6.</div><div class="graphic-wrap"><img class="content-image" src="m_syv080f6.png" alt=" Majority-rule consensus tree, a) including all fossils and b) including only extant taxa, from total-evidence dating analysis under the piecewise-constant FBD prior with random sampling and under the IGR relaxed-clock model. Node bars indicate HPD intervals of estimated divergence times (cf. Table 5 , pcFBD_Rnd). " path-from-xml="syv080f6.tiff"><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="https://academic.oup.com/view-large/figure/115286593/syv080f6.tiff" path-from-xml="syv080f6.tiff" target="_blank">View large</a><a section="115286593" href="https://oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/sysbio/65/2/10.1093_sysbio_syv080/3/syv080f6.png?Expires=2147483647&amp;Signature=gbl2MmlEmOH5ZvQDesZd2rIQaisC0Vd1NDNW5zbgCHCfw70j9hwCoAbL2X8JG4DckLNMUyI-As2hhyUoAp20hzy9WjN0I6VoRNc14mLxPG%7ESwhyY6aGG5%7EoBNx%7E9vC-A62Ib0dl93HdgKwtJ%7E-BZLCEemXO7zvUhku0UkYYX%7EoJKDOOc2i8A2Wkk11zGmI1UpEazyN8PxYKu%7EKM9PUoav%7E0I2lMzCC9tNii%7EdVFVxo8Tgxzk7doxTdbhEXQKfUVkknfx0X0vpiM-F%7E1tG6Ha6ydP985Uzydkj86QY-sdsopk4jEvQqxv7utNwZoVVklBA65jKDWW3y6d26le%7EFentg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=115286593&amp;ar=2427164&amp;xsltPath=%7E/UI/app/XSLT&amp;imagename=" class="download-slide" path-from-xml="syv080f6.tiff" target="">Download slide</a></div></div><div class="caption fig-caption"><p>
 Majority-rule consensus tree, a) including all fossils and b) including
 only extant taxa, from total-evidence dating analysis under the 
piecewise-constant FBD prior with random sampling and under the IGR 
relaxed-clock model. Node bars indicate HPD intervals of estimated 
divergence times (cf. <a reveal-id="T5" data-open="T5" class="link link-reveal link-table xref-fig">Table 5</a> , pcFBD_Rnd). </p></div></div>                        <a id="115286594" scrollto-destination="115286594"></a>
<div id="F7" data-id="F7" class="fig fig-section"><div class="label fig-label">Figure 7.</div><div class="graphic-wrap"><img class="content-image" src="m_syv080f7.png" alt=" Majority-rule consensus tree, a) including all fossils and b) including only extant taxa, from total-evidence dating analysis under the piecewise-constant FBD prior with diversified sampling and under the IGR relaxed-clock model. Node bars indicate HPD intervals of estimated divergence times (cf. Table 5 , pcFBD_Div). " path-from-xml="syv080f7.tiff"><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="https://academic.oup.com/view-large/figure/115286594/syv080f7.tiff" path-from-xml="syv080f7.tiff" target="_blank">View large</a><a section="115286594" href="https://oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/sysbio/65/2/10.1093_sysbio_syv080/3/syv080f7.png?Expires=2147483647&amp;Signature=HeyL-CLTBqOhMmpuim8x%7EyDVBLGaQ9j-XQrI53QbhWWGmvxAs6oFYEQt4FGgaLNq98o37Rsimoq4v3WMSaDK3y8Lk8dPi4h7Arwm5cUAvHhffcImCjw3wZ1gaKrNrdudcILK8R5iUAz7COWPbpniKOEH0W3XHfK5jDjvV5h4SQ2%7EM4egjduVAf3f43F7OfzrGQWThTwDsSu8xw5XGHpqxWQjDVY%7EaGWDcjt%7Eo1zXHzzTgtDpkEJpntNH2J5jgQ3zE41f4jWfkUPkSv-ojzzpjaCgUddqaXJNNUt47SWBTrCoRHLqc2M5zzE%7E0nfV4gu4JnBA7FSbKYUpbm3xpaslhQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=115286594&amp;ar=2427164&amp;xsltPath=%7E/UI/app/XSLT&amp;imagename=" class="download-slide" path-from-xml="syv080f7.tiff" target="">Download slide</a></div></div><div class="caption fig-caption"><p>
 Majority-rule consensus tree, a) including all fossils and b) including
 only extant taxa, from total-evidence dating analysis under the 
piecewise-constant FBD prior with diversified sampling and under the IGR
 relaxed-clock model. Node bars indicate HPD intervals of estimated 
divergence times (cf. <a reveal-id="T5" data-open="T5" class="link link-reveal link-table xref-fig">Table 5</a> , pcFBD_Div). </p></div></div><div content-id="F7" class="fig fig-modal reveal-modal" style="display: block;"><div class="label fig-label">Figure 7.</div><div class="graphic-wrap"><img class="content-image" src="m_syv080f7.png" alt=" Majority-rule consensus tree, a) including all fossils and b) including only extant taxa, from total-evidence dating analysis under the piecewise-constant FBD prior with diversified sampling and under the IGR relaxed-clock model. Node bars indicate HPD intervals of estimated divergence times (cf. Table 5 , pcFBD_Div). " path-from-xml="syv080f7.tiff"><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="https://academic.oup.com/view-large/figure/115286594/syv080f7.tiff" path-from-xml="syv080f7.tiff" target="_blank">View large</a><a section="115286594" href="https://oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/sysbio/65/2/10.1093_sysbio_syv080/3/syv080f7.png?Expires=2147483647&amp;Signature=HeyL-CLTBqOhMmpuim8x%7EyDVBLGaQ9j-XQrI53QbhWWGmvxAs6oFYEQt4FGgaLNq98o37Rsimoq4v3WMSaDK3y8Lk8dPi4h7Arwm5cUAvHhffcImCjw3wZ1gaKrNrdudcILK8R5iUAz7COWPbpniKOEH0W3XHfK5jDjvV5h4SQ2%7EM4egjduVAf3f43F7OfzrGQWThTwDsSu8xw5XGHpqxWQjDVY%7EaGWDcjt%7Eo1zXHzzTgtDpkEJpntNH2J5jgQ3zE41f4jWfkUPkSv-ojzzpjaCgUddqaXJNNUt47SWBTrCoRHLqc2M5zzE%7E0nfV4gu4JnBA7FSbKYUpbm3xpaslhQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=115286594&amp;ar=2427164&amp;xsltPath=%7E/UI/app/XSLT&amp;imagename=" class="download-slide" path-from-xml="syv080f7.tiff" target="">Download slide</a></div></div><div class="caption fig-caption"><p>
 Majority-rule consensus tree, a) including all fossils and b) including
 only extant taxa, from total-evidence dating analysis under the 
piecewise-constant FBD prior with diversified sampling and under the IGR
 relaxed-clock model. Node bars indicate HPD intervals of estimated 
divergence times (cf. <a reveal-id="T5" data-open="T5" class="link link-reveal link-table xref-fig">Table 5</a> , pcFBD_Div). </p></div></div><p>
 We examined the impact of the priors on root age and clock rate, and 
that of the assumed sampling fraction. In the previous analyses, the 
prior for the root age was an offset exponential distribution with mean 
396 Ma and offset 315 Ma. We halved and doubled the distance from the 
mean to the minimum to obtain more and less restrictive priors. The more
 restrictive root-age prior had an expectation of 355 Ma, and the less 
restrictive prior an expectation of 477 Ma. The calibration prior for 
the holometabolan age was also changed so that the expectation was the 
same as for the root. The results show that the impact of the root 
calibration on the divergence-time estimates is rather small ( <a reveal-id="F8" data-open="F8" class="link link-reveal link-table xref-fig">Fig. 8</a> a). We also used a smaller (0.2) and larger variance (1.0) in the lognormal prior for the base rate of the clock. As shown in <a reveal-id="F8" data-open="F8" class="link link-reveal link-table xref-fig">Figure 8</a> b, the posterior age estimates are not sensitive to the variance parameter in the clock-rate prior. </p>                        <a id="115286596" scrollto-destination="115286596"></a>
<div id="F8" data-id="F8" class="fig fig-section"><div class="label fig-label">Figure 8.</div><div class="graphic-wrap"><img class="content-image" src="m_syv080f8.png" alt="The impact of a) tree age prior, b) clock rate prior, and c) sampling fraction on posterior ages of the root and of Hymenoptera under the uniform tree prior (if applicable), FBD with random sampling (FBD_Rnd) or diversified sampling (FBD_Div), and piecewise-constant FBD with random sampling (pcFBD_Rnd) or diversified sampling (pcFBD_Div)." path-from-xml="syv080f8.tiff"><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="https://academic.oup.com/view-large/figure/115286596/syv080f8.tiff" path-from-xml="syv080f8.tiff" target="_blank">View large</a><a section="115286596" href="https://oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/sysbio/65/2/10.1093_sysbio_syv080/3/syv080f8.png?Expires=2147483647&amp;Signature=qmLPlGAEI3RG2sdcPaJkZ5bNT62rph6VaRNufa%7Ez--DF9cHOLO1rczumK4DXOyVZPVuE1BqEzbz79xRSSk%7E3-QyyxGrN5aCG47jU4NEP745kUr2zPJPl6xbo8FQux64shPw%7EaEi%7EVwFyYZ89gfYAfb5rEeneKz%7EWi8XOV5ULCtSBNXw8glh9Jpj4szRFyyBxHT8AW78FyA2BVewKvbLKbS99eaImhXtdgRCDpgBIcnPKhRltcoeoxYsRyfDustu9Xytl8x-Bb2O5TZoBjSXDUG-uLS%7Ec%7EJSFKDiTUPbBoTQ1bDx%7Ek0qGl65LOcJDHrkAOPoCgTN-r1xICjjyYv326Q__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=115286596&amp;ar=2427164&amp;xsltPath=%7E/UI/app/XSLT&amp;imagename=" class="download-slide" path-from-xml="syv080f8.tiff" target="">Download slide</a></div></div><div class="caption fig-caption"><p>The
 impact of a) tree age prior, b) clock rate prior, and c) sampling 
fraction on posterior ages of the root and of Hymenoptera under the 
uniform tree prior (if applicable), FBD with random sampling (FBD_Rnd) 
or diversified sampling (FBD_Div), and piecewise-constant FBD with 
random sampling (pcFBD_Rnd) or diversified sampling (pcFBD_Div).</p></div></div><div content-id="F8" class="fig fig-modal reveal-modal" style="display: block;"><div class="label fig-label">Figure 8.</div><div class="graphic-wrap"><img class="content-image" src="m_syv080f8.png" alt="The impact of a) tree age prior, b) clock rate prior, and c) sampling fraction on posterior ages of the root and of Hymenoptera under the uniform tree prior (if applicable), FBD with random sampling (FBD_Rnd) or diversified sampling (FBD_Div), and piecewise-constant FBD with random sampling (pcFBD_Rnd) or diversified sampling (pcFBD_Div)." path-from-xml="syv080f8.tiff"><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="https://academic.oup.com/view-large/figure/115286596/syv080f8.tiff" path-from-xml="syv080f8.tiff" target="_blank">View large</a><a section="115286596" href="https://oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/sysbio/65/2/10.1093_sysbio_syv080/3/syv080f8.png?Expires=2147483647&amp;Signature=qmLPlGAEI3RG2sdcPaJkZ5bNT62rph6VaRNufa%7Ez--DF9cHOLO1rczumK4DXOyVZPVuE1BqEzbz79xRSSk%7E3-QyyxGrN5aCG47jU4NEP745kUr2zPJPl6xbo8FQux64shPw%7EaEi%7EVwFyYZ89gfYAfb5rEeneKz%7EWi8XOV5ULCtSBNXw8glh9Jpj4szRFyyBxHT8AW78FyA2BVewKvbLKbS99eaImhXtdgRCDpgBIcnPKhRltcoeoxYsRyfDustu9Xytl8x-Bb2O5TZoBjSXDUG-uLS%7Ec%7EJSFKDiTUPbBoTQ1bDx%7Ek0qGl65LOcJDHrkAOPoCgTN-r1xICjjyYv326Q__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=115286596&amp;ar=2427164&amp;xsltPath=%7E/UI/app/XSLT&amp;imagename=" class="download-slide" path-from-xml="syv080f8.tiff" target="">Download slide</a></div></div><div class="caption fig-caption"><p>The
 impact of a) tree age prior, b) clock rate prior, and c) sampling 
fraction on posterior ages of the root and of Hymenoptera under the 
uniform tree prior (if applicable), FBD with random sampling (FBD_Rnd) 
or diversified sampling (FBD_Div), and piecewise-constant FBD with 
random sampling (pcFBD_Rnd) or diversified sampling (pcFBD_Div).</p></div></div>                        <a id="115286597" scrollto-destination="115286597"></a>
<div id="F9" data-id="F9" class="fig fig-section"><div class="label fig-label">Figure 9.</div><div class="graphic-wrap"><img class="content-image" src="m_syv080f9.png" alt="Majority-rule consensus tree of extant taxa from total-evidence dating analysis under the IGR model and the piecewise-constant FBD prior with diversified sampling. The tree above has 9 taxa removed from the original data, whereas the tree below has 20 taxa removed, to eliminate young splits that might be inconsistent with the diversified sampling model." path-from-xml="syv080f9.tiff"><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="https://academic.oup.com/view-large/figure/115286597/syv080f9.tiff" path-from-xml="syv080f9.tiff" target="_blank">View large</a><a section="115286597" href="https://oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/sysbio/65/2/10.1093_sysbio_syv080/3/syv080f9.png?Expires=2147483647&amp;Signature=3i68vwaRNhYuGLZ2OWoxj2pA4RoC87eA67lO59J-GNdnpLyA7htjIqHkvOXJqefuyAw0VLBLUQNnXAJO5YwpGhrmtJ9liYqYdYaegI5gR3y5XKYcyPpxY0GNAQfy%7E1Zj8gyyAHxkdQ6ClJuLBnk%7EQRjouSL3XwIK%7EEoCb4xeNsHbb4z%7EzELyL4zHwI1v2KDkR3C1QoLpR2jBXSjsOALFpqQcsLOqBqN6dXnpDn%7EouIgWy%7EAsFJz7U4A07AJOHOHw9UpBtdhibrajVSNaDnheQdMfNlC8d28OIK6rwGh-AUbLNpQw7yE%7ELksdTxER1Iw5Ns6zCQYrZhO3AHFwbGyfeg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=115286597&amp;ar=2427164&amp;xsltPath=%7E/UI/app/XSLT&amp;imagename=" class="download-slide" path-from-xml="syv080f9.tiff" target="">Download slide</a></div></div><div class="caption fig-caption"><p>Majority-rule
 consensus tree of extant taxa from total-evidence dating analysis under
 the IGR model and the piecewise-constant FBD prior with diversified 
sampling. The tree above has 9 taxa removed from the original data, 
whereas the tree below has 20 taxa removed, to eliminate young splits 
that might be inconsistent with the diversified sampling model.</p></div></div><div content-id="F9" class="fig fig-modal reveal-modal" style="display: block;"><div class="label fig-label">Figure 9.</div><div class="graphic-wrap"><img class="content-image" src="m_syv080f9.png" alt="Majority-rule consensus tree of extant taxa from total-evidence dating analysis under the IGR model and the piecewise-constant FBD prior with diversified sampling. The tree above has 9 taxa removed from the original data, whereas the tree below has 20 taxa removed, to eliminate young splits that might be inconsistent with the diversified sampling model." path-from-xml="syv080f9.tiff"><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="https://academic.oup.com/view-large/figure/115286597/syv080f9.tiff" path-from-xml="syv080f9.tiff" target="_blank">View large</a><a section="115286597" href="https://oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/sysbio/65/2/10.1093_sysbio_syv080/3/syv080f9.png?Expires=2147483647&amp;Signature=3i68vwaRNhYuGLZ2OWoxj2pA4RoC87eA67lO59J-GNdnpLyA7htjIqHkvOXJqefuyAw0VLBLUQNnXAJO5YwpGhrmtJ9liYqYdYaegI5gR3y5XKYcyPpxY0GNAQfy%7E1Zj8gyyAHxkdQ6ClJuLBnk%7EQRjouSL3XwIK%7EEoCb4xeNsHbb4z%7EzELyL4zHwI1v2KDkR3C1QoLpR2jBXSjsOALFpqQcsLOqBqN6dXnpDn%7EouIgWy%7EAsFJz7U4A07AJOHOHw9UpBtdhibrajVSNaDnheQdMfNlC8d28OIK6rwGh-AUbLNpQw7yE%7ELksdTxER1Iw5Ns6zCQYrZhO3AHFwbGyfeg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=115286597&amp;ar=2427164&amp;xsltPath=%7E/UI/app/XSLT&amp;imagename=" class="download-slide" path-from-xml="syv080f9.tiff" target="">Download slide</a></div></div><div class="caption fig-caption"><p>Majority-rule
 consensus tree of extant taxa from total-evidence dating analysis under
 the IGR model and the piecewise-constant FBD prior with diversified 
sampling. The tree above has 9 taxa removed from the original data, 
whereas the tree below has 20 taxa removed, to eliminate young splits 
that might be inconsistent with the diversified sampling model.</p></div></div><p>
 The sampling probability of extant taxa was fixed to 0.0005 in the 
previous analyses, an estimate based on a compilation of the number of 
currently described hymenopteran species ( <a reveal-id="B1" data-open="B1" class="link link-ref link-reveal xref-bibr">Aguiar et al. 2013</a>
 ). To assess the sensitivity of the results to changes in the assumed 
sampling fraction, we enlarged and reduced it by an order of magnitude 
(to 0.005 and 0.00005, respectively). For random sampling of extant 
taxa, increasing the sampling probability decreases the posterior ages 
slightly, whereas it increases the posterior ages considerably for 
diversified sampling ( <a reveal-id="F8" data-open="F8" class="link link-reveal link-table xref-fig">Fig. 8</a>
 c). Eventually, the age estimates will approach those under complete 
sampling. These results contrast with our simulations, in which the 
sampling probability had only a minor effect on the age estimates ( <a reveal-id="T2" data-open="T2" class="link link-reveal link-table xref-fig">Table 2</a> , <a reveal-id="F4" data-open="F4" class="link link-reveal link-table xref-fig">Figs. 4</a> and <a reveal-id="F5" data-open="F5" class="link link-reveal link-table xref-fig">5</a>
 ). However, the sampling fractions examined in the empirical analyses 
are several orders of magnitude smaller than those studied in the 
simulations. </p><p> In the empirical analyses, almost all fossils are inferred to be tips rather than ancestral nodes in the tree ( <a reveal-id="T5" data-open="T5" class="link link-reveal link-table xref-fig">Table 5</a> , see Supplementary Tables S1 and S2 on Dryad at <a class="link link-uri" href="http://dx.doi.org/10.5061/dryad.26820" target="">http://dx.doi.org/10.5061/dryad.26820</a>
 for the estimated probability of each fossil being ancestral). The only
 exception involves the piecewise-constant FBD prior with random 
sampling under the IGR model, where approximately one-third of the 
fossils were estimated to be ancestral. This is the same model where the
 induced prior puts a high probability on fossils being ancestors ( <a reveal-id="T4" data-open="T4" class="link link-reveal link-table xref-fig">Table 4</a> ). </p><p> The consensus trees including all fossils are partially unresolved, regardless of the tree prior ( <a reveal-id="F6" data-open="F6" class="link link-reveal link-table xref-fig">Figs. 6</a> and <a reveal-id="F7" data-open="F7" class="link link-reveal link-table xref-fig">7</a> , and Supplementary Figures S1 to S10 on Dryad at <a class="link link-uri" href="http://dx.doi.org/10.5061/dryad.26820" target="">http://dx.doi.org/10.5061/dryad.26820</a>
 ). However, when the fossils are excluded from the sampled trees to 
compute the consensus, the consensus trees are fully resolved. Thus, the
 polytomies are entirely due to uncertainty in the placement of the 
fossils (see also <a reveal-id="B44" data-open="B44" class="link link-ref link-reveal xref-bibr">Ronquist et al. 2012b</a>
 ). MCMC convergence was slightly slower for the piecewise-constant FBD 
prior than for the uniform tree prior or the constant-rate FBD prior. 
This is especially true for the topology parameter. Nevertheless, all 
model parameters appeared to have converged after 100 million iterations
 according to the trace plots, and judging from the ASDSF ( <span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mo xmlns="">&lt;</mo></math></span> 0.05), ESS ( <span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mo xmlns="">≥</mo></math></span> 100), and PSRF values. </p>                            <h2 scrollto-destination="115286601" id="115286601" class="section-title"><span class="small-caps">Discussion</span></h2>
                            <h3 scrollto-destination="115286602" id="115286602" class="section-title">Modeling Fossilization and Sampling</h3>
<p> In this article, we examined the importance of incorporating 
information about speciation, extinction, and sampling of fossil and 
extant taxa in total-evidence dating. Clearly, these processes influence
 our prior beliefs concerning the shape of the tree and the placement of
 fossils therein. For instance, if the fossilization rate is high and 
the extinction rate is low, we expect most fossils to sit on branches 
leading to extant taxa. Conversely, if the fossilization rate is low and
 the extinction rate high, most fossils are instead likely to represent 
extinct side branches. The sampling of extant taxa also affects our 
prior expectations concerning the structure of the tree. For instance, 
if we sample extant taxa so that the chosen exemplars span as much 
diversity as possible (a diversified sample) ( <a reveal-id="B22" data-open="B22" class="link link-ref link-reveal xref-bibr">Höhna et al. 2011</a>
 ), then the tree should be ‘`bush-like” with long terminal branches. If
 extant taxa are chosen randomly instead, the branching events should be
 more evenly distributed over time. </p><p>Our expectations concerning 
the structure of the tree are also influenced by differences over time 
in speciation, extinction, and fossilization rates. The proportion of 
fossils included from a particular time horizon is strongly affected by 
decisions about fossil sampling, for example, which sites to explore in 
the search for fossils, and which fossil specimens to select for 
analysis. If the fossilization rate itself varies over time, this will 
cause some strata to be richer in fossils than others and thus better 
represented in the tree. Similarly, variations in speciation and 
extinction rates over time will cause the density of branching events in
 the observed tree to vary over time.</p><p> Together, all these 
phenomena affect the shape and structure of the tree and therefore 
potentially influence the estimation of divergence times. However, the 
tree priors used previously in total-evidence dating (e.g., <a reveal-id="B37" data-open="B37" class="link link-ref link-reveal xref-bibr">Pyron 2011</a> ; <a reveal-id="B44" data-open="B44" class="link link-ref link-reveal xref-bibr">Ronquist et al. 2012b</a> ; <a reveal-id="B56" data-open="B56" class="link link-ref link-reveal xref-bibr">Wood et al. 2013</a> ; <a reveal-id="B7" data-open="B7" class="link link-ref link-reveal xref-bibr">Dornburg et al. 2015</a> ; <a reveal-id="B2" data-open="B2" class="link link-ref link-reveal xref-bibr">Arcila et al. 2015</a> ; <a reveal-id="B14" data-open="B14" class="link link-ref link-reveal xref-bibr">Grimm et al. 2015</a>
 ) do not account for them at all. To explore their influence in 
total-evidence dating, the recent FBD process is an obvious choice ( <a reveal-id="B46" data-open="B46" class="link link-ref link-reveal xref-bibr">Stadler 2010a</a> ; <a reveal-id="B18" data-open="B18" class="link link-ref link-reveal xref-bibr">Heath et al. 2014</a> ; <a reveal-id="B11" data-open="B11" class="link link-ref link-reveal xref-bibr">Gavryushkina et al. 2014</a>
 ). The formulation of the FBD model allows us to examine the effects of
 speciation, extinction and fossilization processes, and to accommodate 
different sampling strategies. Even though birth–death models like the 
FBD process have their limitations, they provide a starting point in the
 search of more realistic tree priors. </p><p> In comparing the 
performance of the FBD model to previous models used in total-evidence 
dating, it might first seem like an obvious choice to use Bayes-factor 
tests, a standard Bayesian approach to model choice. Unfortunately, 
Bayes factors have severe limitations in this context. First, the models
 have different parameters and different dimensionality, which means 
that the outcome of a Bayes-factor test is decided to a large extent by 
the priors and not necessarily by model performance. The sensitivity of 
Bayes-factor tests to priors is illustrated well by the apparent 
contradictions that may arise in the testing of topology hypotheses ( <a reveal-id="B4" data-open="B4" class="link link-ref link-reveal xref-bibr">Bergsten et al. 2013</a>
 ). Second, even if it were possible to find suitable priors for 
appropriate Bayes-factor tests, it is difficult to estimate the Bayes 
factors accurately enough to allow reliable comparison of the complex 
models used in total-evidence dating. For instance, <a reveal-id="B44" data-open="B44" class="link link-ref link-reveal xref-bibr">Ronquist et al. (2012b)</a>
 tried to distinguish three different relaxed-clock models for 
total-evidence dating analyses using the state-of-the-art stepping-stone
 sampling algorithm ( <a reveal-id="B57" data-open="B57" class="link link-ref link-reveal xref-bibr">Xie et al. 2011</a>
 ) to estimate the marginal model likelihoods needed for computing Bayes
 factors. Despite considerable computational effort, however, the 
variance in the estimated marginal likelihoods between independent runs 
of the algorithm was larger than the differences between the models, 
making it impossible to distinguish them. </p><p>A pragmatic approach in
 these situations is to compare models with respect to their ability to 
account for the data. The comparison is simplified if we are comparing 
models defined on the same or similar parameter spaces, which is 
arguably the case here. At first sight, the FBD model might seem 
fundamentally different in that it allows fossils to represent both side
 branches and ancestors of other taxa in the tree, while previous models
 considered all fossils to be side branches (tip fossils). However, 
although ancestral fossils appear distinct from tip fossils, it is 
possible to consider them simply as the boundary case of tip fossils, 
when the length of the side branch goes to zero. In fact, we take this 
approach in our computational machinery, where ancestral fossils in the 
FBD model sit on side branches with length zero. Viewed in this way, the
 FBD model just specifies a different prior probability distribution on 
the same tree space used by the simpler models.</p><p> Given this 
similarity between the models, it is relatively straightforward to 
compare their ability to account for the data. At one extreme, the data 
could be so informative that they were able to pull the simpler 
tree-model priors toward the same posterior obtained under the FBD 
model. If so, there would only be minor differences in divergence-time 
estimates between models. In other words, the FBD model would not add 
much to the analysis except that we might be able to estimate some 
additional parameters of interest, like the fossilization rate. At the 
other extreme, the data might carry very little information about the 
shape of the tree, in which case we would essentially retrieve the prior
 under each model. The interesting case occurs when the data fit the FBD
 model better, but are not informative enough to modify the simple tree 
model priors toward the posterior observed under the FBD model. This may
 result in significant improvements in divergence-time estimation under 
the FBD model. Our results suggest that this is indeed the case we are 
facing. In particular, the fact that we observe informative posterior 
estimates of FBD model parameters indicates that the model is picking up
 relevant signal in the data, and thus fits the data better than 
previous tree priors (e.g., comparing <a reveal-id="T4" data-open="T4" class="link link-reveal link-table xref-fig">Table 4</a> with <a reveal-id="T5" data-open="T5" class="link link-reveal link-table xref-fig">5</a>
 ). It also suggests that the FBD models examined here are not 
overparameterized; if they were, the posterior distributions would be 
more similar to the induced prior distributions. </p><p> Our results 
also show that the specific details of the FBD prior can have a strong 
influence on divergence-time estimates. Notably, it seems that the most 
important factor is how the sampling of extant taxa is modeled ( <a reveal-id="T5" data-open="T5" class="link link-reveal link-table xref-fig">Table 5</a>
 ). Under the assumption of random sampling, the FBD model gives results
 that are older than those of the uniform model. When we accommodate the
 fact that the sample is diversified, however, the results change 
substantially. Under the IGR model, which appears to be the better of 
the two relaxed-clock models examined here, the estimate of the crown 
age of Hymenoptera shifts from around 325 Ma to 252 Ma when diversified 
sampling is accounted for. A similar shift toward younger ages also 
occurs under the TK02 relaxed-clock model. </p><p> Beyond the sampling 
issue, our results confirm that the relaxed-clock model significantly 
affects the divergence-time estimates. The introduction of the FBD model
 did not affect the difference between the two relaxed-clock models, IGR
 and TK02. However, there are several reasons to prefer the IGR results 
over the TK02 results ( <a reveal-id="B43" data-open="B43" class="link link-ref link-reveal xref-bibr">Ronquist et al. 2012a</a>
 ). Unlike the IGR model, TK02 assumes autocorrelated evolutionary 
rates. Such rates are not expected to shift very rapidly, and TK02 
therefore has difficulties accommodating the apparently rapid shifts in 
evolutionary rates close to the root of the hymenopteran tree ( <a reveal-id="B43" data-open="B43" class="link link-ref link-reveal xref-bibr">Ronquist et al. 2012a</a>
 ). This results in longer branches close to the root of the 
hymenopteran subtree, and is associated with substantially older 
divergence time estimates for the crown age of Hymenoptera. Given that 
the oldest known Hymenoptera fossils are around 235 Ma, the TK02 
estimates appear unrealistically old ( <a reveal-id="T5" data-open="T5" class="link link-reveal link-table xref-fig">Table 5</a> , TK02 vs. IGR), further strengthening the conclusion that the model does not fit the data well. </p><p>
 It is interesting to note that the result under the piecewise-constant 
FBD model with random sampling and IGR rates suggests that one third of 
the fossils are ancestors ( <a reveal-id="T5" data-open="T5" class="link link-reveal link-table xref-fig">Table 5</a>
 ). This proportion decreases to 2% or less under all other models, 
which is more in line with expectations. If one third of the fossils 
were indeed ancestors of extant lineages, then the diversity of 
Hymenoptera must have been very low during their early diversification 
and a large fraction of the ancient lineages must have left current 
descendants, both of which seem unlikely. The high proportion of 
inferred ancestors under this particular model is apparently due to a 
combination of two factors. First, the model puts considerably more 
prior probability on fossils being ancestral than any of the other 
models ( <a reveal-id="T4" data-open="T4" class="link link-reveal link-table xref-fig">Table 4</a>
 ). Second, the scarcity of character data for the fossils is not enough
 to pull the posterior away from this prior. Fossil data were only 
available for 4–20% of the morphological characters, depending on the 
completeness of the specimens (Supplementary Tables S1 and S2 on Dryad 
at <a class="link link-uri" href="http://dx.doi.org/10.5061/dryad.26820" target="">http://dx.doi.org/10.5061/dryad.26820</a>
 ), and molecular data were completely missing for the fossils. Thus, it
 is not surprising that the inferred proportion of ancestral fossils is 
influenced to a large extent by the tree prior. </p>                            <h3 scrollto-destination="115286612" id="115286612" class="section-title">Implications for the Age of Hymenoptera</h3>
<p> The previous total-evidence dating analysis of the early radiation 
of Hymenoptera suggested that the extant taxa started to diversify about
 309 Ma, almost immediately after splitting off from the remainder of 
Holometabola in the Carboniferous ( <a reveal-id="B44" data-open="B44" class="link link-ref link-reveal xref-bibr">Ronquist et al. 2012b</a>
 ). In contrast, paleontological reconstructions indicated a time lag of
 at least 75 myr from the origin of the order in the Carboniferous to 
the first separation of extant lineages in the Triassic ( <a reveal-id="B41" data-open="B41" class="link link-ref link-reveal xref-bibr">Rasnitsyn 1988</a> ; <a reveal-id="B42" data-open="B42" class="link link-ref link-reveal xref-bibr">2010</a> . The results under the diversified sampling FBD model ( <a reveal-id="F7" data-open="F7" class="link link-reveal link-table xref-fig">Fig. 7</a>
 ) essentially confirm the paleontological reconstructions, suggesting 
that the previous total-evidence analysis was biased by an unrealistic 
tree prior. In fact, the diversified sampling results are rather close 
to the dates suggested by Rasnitsyn in his attempt to fit the fossil 
record to the hymenopteran phylogeny, minimizing the extent of “ghost 
lineages” (lineages undocumented in the fossil record) ( <a reveal-id="B42" data-open="B42" class="link link-ref link-reveal xref-bibr">Rasnitsyn 2010</a>
 ). For instance, Rasnitsyn suggested that crown Tenthredinoidea date 
back to 140 Ma (223 Ma in our analysis), Pamphilioidea to 180 Ma (202 Ma
 in our analysis), and Apocrita to 180 Ma (187 Ma in our analysis). </p><p>
 Given the obvious gaps in the early fossil record of hymenopterans, 
with some lineages having sister groups that are absent for 100 myr or 
more ( <a reveal-id="B42" data-open="B42" class="link link-ref link-reveal xref-bibr">Rasnitsyn 2010</a>
 ), the 252 Ma estimate may appear surprisingly close to the age of the 
oldest known hymenopteran fossils. Is it really plausible that 
hymenopterans started to diversify only just before we observe the first
 evidence of their presence in the fossil record? There is actually some
 reason to suspect that the age estimates obtained under the diversified
 FBD process may be biased toward the recent because of imperfections in
 the model. Consider that, for mathematical convenience, our model of 
diversified sampling assumes that the investigator is able to find the 
sample representing the maximum amount of phylogenetic diversity given 
the chosen number of tips. In practice, however, the investigator is 
likely to miss a few of the oldest splits in the tree, and instead 
include a few splits that are younger than the ideal cutoff value, 
perhaps much younger. </p><p> To investigate whether such a bias could 
influence our analyses, we tried to reduce the effect by eliminating the
 speciation-time outliers from the tree. Specifically, we inferred an 
uncalibrated relaxed-clock tree of extant taxa under the uniform tree 
prior and IGR model. The priors for the root age and the substitution 
rate were unchanged. We first removed nine taxa representing extremely 
young splits from the tree. We then eliminated eleven additional taxa 
representing moderately young splits, resulting in the removal of twenty
 extant taxa in total (Supplementary Figure S12 on Dryad at <a class="link link-uri" href="http://dx.doi.org/10.5061/dryad.26820" target="">http://dx.doi.org/10.5061/dryad.26820</a>
 ). In the latter case, almost all splits expected to be younger than 
the youngest fossil (83 Ma) were eliminated. We then repeated the dating
 analysis under the piecewise-constant FBD model with diversified 
sampling. The estimated dates were very similar to those of the original
 analysis of the complete tree, especially for the deepest splits in the
 tree ( <a reveal-id="F9" data-open="F9" class="link link-reveal link-table xref-fig">Fig. 9</a> , see also <a reveal-id="F7" data-open="F7" class="link link-reveal link-table xref-fig">Fig. 7</a>
 ). This indicates that the biasing effect of the strict 
maximum-diversity assumption is negligible, at least for the older 
divergence times in the tree. Nevertheless, this may not always be the 
case, and relaxing the maximum-diversity assumption would appear to be 
an important area for further improvement of total-evidence dating under
 the FBD prior; indeed, for any type of dating under a birth–death prior
 assuming diversified sampling. </p><p> A recent phylogenomic study of insects ( <a reveal-id="B35" data-open="B35" class="link link-ref link-reveal xref-bibr">Misof et al. 2014</a> ) placed the earliest hymenopteran radiation in the Triassic, which is consistent with Rasnitsyn ( <a reveal-id="B41" data-open="B41" class="link link-ref link-reveal xref-bibr">1988</a> ; <a reveal-id="B42" data-open="B42" class="link link-ref link-reveal xref-bibr">2010</a>
 ) and with our results under the FBD model with diversified sampling. 
However, despite the impressive amounts of sequence data, the 
divergence-time estimates from this phylogenomic study are not 
necessarily more accurate, and cannot be taken as reliable confirmation 
of our results. This is because sequence data primarily inform the 
estimation of evolutionary branch lengths, while the major sources of 
dating uncertainty stem from the placement of calibration fossils and 
from the modeling of processes such as fossilization, sampling of recent
 and fossil taxa, and rate variation across the tree. Although the 
diversified FBD analysis presented here addresses all of these sources 
of uncertainty, the phylogenomic study was based on traditional node 
dating with narrow calibration priors ( <a reveal-id="B35" data-open="B35" class="link link-ref link-reveal xref-bibr">Misof et al. 2014</a>
 ), which potentially compromised the results. In addition, the dating 
of the hymenopteran part of the tree is limited by the sparse sampling 
of lineages from this order. For instance, no member of Xyelidae was 
included in the analysis, even though they are the sister group of all 
other extant hymenopteran lineages. </p><p>To further improve dating of 
the early hymenopteran radiation, it is important to focus on the major 
remaining sources of uncertainty, and address them in a total-evidence 
dating framework. More sequence data can help expand the number of 
extant terminals, increase the precision of evolutionary branch-length 
estimates, and possibly improve the modeling of speciation, extinction, 
and rate variation across the tree. However, much of the dating 
uncertainty will remain unless we become better at incorporating the 
information from the fossil record. For instance, a better understanding
 of morphological evolution in Hymenoptera will help place the fossils 
with more certainty in the phylogeny of extant taxa.</p><p> Most 
importantly, increasing the number of fossils included in the analysis 
can help elucidate the speciation, extinction, and fossil sampling 
processes, all of which can contribute strongly to divergence-time 
estimation. Our study has only scratched the surface with respect to 
fossils that could be informative about the early evolution of 
Hymenoptera, and the same could be said for many other dating studies. 
Some of the fossils that were not analyzed by us are poorly preserved, 
but we have shown earlier that even very incomplete fossils can 
contribute to the precision of divergence-time estimates in a 
total-evidence dating analysis ( <a reveal-id="B44" data-open="B44" class="link link-ref link-reveal xref-bibr">Ronquist et al. 2012b</a>
 ). Another possibility for improvement is to incorporate more 
stratigraphic information in the analysis, such as data on the variation
 in fossil sampling intensities over time. For example, the absence of 
insects that clearly belong to the hymenopteran crown group from strata 
older than the Triassic is potentially informative about the age of the 
crown group, yet we have not addressed this simple fact appropriately 
even in the most sophisticated models we explored here. To do so, we 
would have had to extend the sampling of fossils to the entire 
Holometabola and to older strata. </p>                            <h3 scrollto-destination="115286619" id="115286619" class="section-title">Conclusions</h3>
<p><a reveal-id="B43" data-open="B43" class="link link-ref link-reveal xref-bibr">Ronquist et al. (2012a)</a>
 demonstrated that total-evidence dating can use more information from 
the fossil record, may avoid biases caused by erroneous placement of 
fossils, and can improve the precision and presumably the accuracy of 
divergence-time estimates. With this study, we have shown that expanding
 the total-evidence analysis to include information about speciation, 
extinction, fossilization, and sampling can result in further 
improvements. In particular, modeling the sampling strategy of extant 
taxa in a realistic way appears to have a substantial impact on 
divergence-time estimates. Diversified sampling, which arguably is the 
rule rather than the exception in dating studies, results in trees with 
long terminal branches and most speciation times clustered close to the 
root of the tree. Such trees have low prior probability under most tree 
priors, including FBD models assuming complete or random sampling. If 
extant taxa have been sampled to maximize diversity, but this is not 
accounted for in the model, the low prior probability of “bush-like” 
trees may cause significant biases in divergence-time estimates, in 
particular, unrealistically old age estimates for the deeper nodes in 
the tree. This is well illustrated by our analyses of the early 
hymenopteran radiation, where accounting for diversified sampling of 
extant taxa results in a major shift in the age estimate of Hymenoptera 
toward more recent times. Interestingly, the new estimates that 
accommodate the sampling bias remove much of the misfit observed 
previously between molecular divergence-time estimates and the fossil 
record. The diversity-sampled piecewise-constant FBD model on which 
these results are based provides the best a priori fit to our data, at 
least among the models examined here, and it shows all the signs of a 
well-behaved model in inference, including the ability to pick up 
relevant signal that is not detected by simpler models. Future studies 
will have to show to what extent this model is equally successful also 
in other dating analyses. </p><p> Taken together, this study and that of <a reveal-id="B43" data-open="B43" class="link link-ref link-reveal xref-bibr">Ronquist et al. (2012a)</a>
 highlight two important sources of error that can affect divergence 
time estimates in a major way: imperfect relaxed-clock models and a 
failure to accommodate the effect of sampling biases on the expected 
tree shape. Neither of them is unique to total-evidence dating; both are
 equally relevant to node dating. In our analyses, both these errors 
have tended to push the estimate of the crown age of Hymenoptera toward 
older time intervals. However, they could also bias age estimates in the
 other direction. For instance, even in our analyses, the failure to 
account for diversified sampling apparently causes many of the younger 
splits in the hymenopteran tree to be estimated too young (compare <a reveal-id="F6" data-open="F6" class="link link-reveal link-table xref-fig">Figs. 6</a> and <a reveal-id="F7" data-open="F7" class="link link-reveal link-table xref-fig">7</a>
 ). Because of these complexities and the many other sources of error 
involved in divergence-time estimation, we refrain from speculation on 
how imperfect relaxed-clock models and failure to model sampling biases 
might have affected previous dating studies. Regardless of potential 
problems with past analyses, it is clear that total-evidence dating 
provides an ideal platform for going forward in exploring and further 
improving the models used for Bayesian divergence-time estimation. The 
processes of speciation, extinction, and fossilization are the 
real-world evolutionary mechanisms that affect present-day diversity and
 the observation of lineages as fossils through time. Thus, 
incorporating parameters that account for these processes is obviously 
important for epistemological reasons. From an empirical perspective, 
perhaps the most important aspect of the total-evidence approach is that
 it provides a common analytical platform, helping neontologists and 
paleontologists to include more information from the fossil record in 
future dating studies. </p>                            <h2 scrollto-destination="115286622" id="115286622" class="section-title"><span class="small-caps">Software availablity</span></h2>
<p> The FBD priors are implemented in version 3.2.6 and newer of MrBayes ( <a class="link link-uri" href="http://mrbayes.net/" target="">http://mrbayes.net</a> ; last accessed October 30, 2015). The commands are listed in the Appendix and in <a reveal-id="B62" data-open="B62" class="link link-ref link-reveal xref-bibr">Zhang (2015)</a> . </p>                            <h2 scrollto-destination="115286624" id="115286624" class="backsection-title"><span class="small-caps">Supplementary</span><span class="small-caps">Material</span></h2>
<p> Supplementary tables and figures related to this article have been deposited at Dryad under <a class="link link-uri" href="http://dx.doi.org/10.5061/dryad.26820" target="">http://dx.doi.org/10.5061/dryad.26820</a> . </p>                            <h2 scrollto-destination="115286626" id="115286626" class="backsection-title"><span class="small-caps">Funding</span></h2>
<p>This research was supported by Swedish Research Council Grant 
VR-2011-5622 (to F.R.). Computations were performed using the Swedish 
National Infrastructure for Computing under project SNIC 2013/1-194 and 
SNIC 2014/1-323. S.K. was supported by Swiss National Science Foundation
 fellowship grant PA00P3_145377 and PZ00P3_154791. T.A.H. was funded by 
National Science Foundation (USA) grant DEB-1256993.</p>                            <h2 scrollto-destination="115286628" id="115286628" class="backsection-title"><span class="small-caps">Acknowledgments</span></h2>
<p>We are indebted to Alex Rasnitsyn for invaluable help with 
information on the fossil record of Hymenoptera and to Nicolas Lartillot
 for detailed discussion on modeling issues related to this article. We 
sincerely thank Frank Anderson, Thomas Near, Alex Pyron, and an 
anonymous reviewer for constructive criticism of the original article 
and excellent suggestions for improvement.</p>                            <h3 scrollto-destination="115286631" id="115286631" class="appendix-title" data-legacy-id="APP1"><span class="small-caps">Appendix</span></h3>
                            <h4 scrollto-destination="115286632" id="115286632" class="section-title">MrBayes Commands</h4>
<p><span class="monospace">[calibrate the fossils]</span></p><p><span class="monospace">calibrate Aulisca = unif(152,163)</span></p><p><span class="monospace"> Sogutia = unif(174,201);</span></p><p><span class="monospace">prset nodeagepr = calibrated;</span></p><p><span class="monospace">[enforce the prior after calibrating]</span></p><p><span class="monospace">prset brlenspr = clock:fossilization;</span></p><p><span class="monospace">[net diversification, lambda-mu]</span></p><p><span class="monospace">prset speciationpr = exp(100);</span></p><p><span class="monospace">[turnover, mu/lambda]</span></p><p><span class="monospace">prset extinctionpr = beta(1,1);</span></p><p><span class="monospace">[fossil sampling proportion, psi/(mu + psi)]\ prset fossilizationpr = beta(1,1);</span></p><p><span class="monospace">[extant sampling proportion, rho]</span></p><p><span class="monospace">prset sampleprob = 0.0005;</span></p><p><span class="monospace">[sampling strategy for extant taxa]</span></p><p><span class="monospace">prset samplestrat = diversity [random];</span></p><p><span class="monospace">[sampling strategy with rate shifting</span></p><p><span class="monospace"> times #: t1 rho1, t2 rho2]</span></p><p><span class="monospace">prset samplestrat = random 2: 252 0, 66 0;</span></p><p><span class="monospace">prset samplestrat = diversity 1: 252 0;</span></p><p><span class="monospace">[prior for tmrca]</span></p><p><span class="monospace">prset treeagepr = offsetexp(315,396);</span></p><p><span class="monospace">[relaxed-clock model]</span></p><p><span class="monospace">prset clockvarpr=igr;</span></p><p><span class="monospace">prset clockratepr=lognorm(-7.1,0.5);</span></p><p><span class="monospace">[use ‘help prset’ for more info]</span></p>                            <h2 scrollto-destination="115286658" id="115286658" class="backreferences-title"><span class="small-caps">References</span></h2>
<div class="ref-list"><div content-id="B1" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="B1" class="ref-content "><div class="citation mixed-citation">
          <div class="comment">Aguiar A., Deans A., Engel M., Forshage 
M., Huber J., Jennings J., Johnson N., Lelej A., Longino J., Miko I., 
Ohl M., Rasmussen C., Taeger A., Yu D. 2013. Order Hymenoptera. In: 
Zhang, Z.-Q. (Ed.) Animal biodiversity: an outline of higher-level 
classification and survey of taxonomic richness (Addenda 2013). Zootaxa 
3703:51–62.</div>
        </div></div></div></div><div content-id="B2" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="B2" class="ref-content "><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Arcila</div>
               <div class="given-names">D.</div>
            </div>
             <div class="name">
              <div class="surname">Alexander Pyron</div>
               <div class="given-names">R.</div>
            </div>
             <div class="name">
              <div class="surname">Tyler</div>
               <div class="given-names">J.C.</div>
            </div>
             <div class="name">
              <div class="surname">Ortí</div>
               <div class="given-names">G.</div>
            </div>
             <div class="name">
              <div class="surname">Betancur-R</div>
               <div class="given-names">R.</div>
            </div>
          </div>
          <div class="year">2015</div>
            . 
            <div class="article-title">An evaluation of fossil tip-dating versus node-age calibrations in tetraodontiform fishes (Teleostei: Percomorphaceae)</div>
            . 
            <div class="source">Mol. Phylogenet. Evol.</div>&nbsp;<div class="volume">82</div><div class="fpage">Pt A:131</div>
            –
            <div class="lpage">145</div>
            .
          <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=An%20evaluation%20of%20fossil%20tip-dating%20versus%20node-age%20calibrations%20in%20tetraodontiform%20fishes%20%28Teleostei%3A%20Percomorphaceae%29&amp;author=D.%20Arcila&amp;author=R.%20Alexander%20Pyron&amp;author=J.C.%20Tyler&amp;author=G.%20Ort%C3%AD&amp;author=R.%20Betancur-R&amp;publication_year=2015&amp;journal=Mol.%20Phylogenet.%20Evol.&amp;volume=82&amp;pages=Pt%20A:131-145" target="">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.ympev.2014.10.011" target="">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.ympev.2014.10.011" target="">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1016%2Fj.ympev.2014.10.011"> </span></div>&nbsp;</div></div></div></div></div><div content-id="B3" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="B3" class="ref-content "><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Aris-Brosou</div>
               <div class="given-names">S.</div>
            </div>
             <div class="name">
              <div class="surname">Yang</div>
               <div class="given-names">Z.</div>
            </div>
          </div>
          <div class="year">2002</div>
            . 
            <div class="article-title">Effects of models of rate 
evolution on estimation of divergence dates with special reference to 
the metazoan 18S ribosomal RNA phylogeny</div>
            . 
            <div class="source">Syst. Biol.</div>&nbsp;<div class="volume">51</div>
            :
            <div class="fpage">703</div>
            –
            <div class="lpage">714</div>
            .
          <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Effects%20of%20models%20of%20rate%20evolution%20on%20estimation%20of%20divergence%20dates%20with%20special%20reference%20to%20the%20metazoan%2018S%20ribosomal%20RNA%20phylogeny&amp;author=S.%20Aris-Brosou&amp;author=Z.%20Yang&amp;publication_year=2002&amp;journal=Syst.%20Biol.&amp;volume=51&amp;pages=703-714" target="">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1080/10635150290102375" target="">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1080%2F10635150290102375" target="">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1080%2F10635150290102375"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12396585" class="link link-pub-id" target="">PubMed</a></div>&nbsp;</div></div></div></div></div><div content-id="B4" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="B4" class="ref-content "><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Bergsten</div>
               <div class="given-names">J.</div>
            </div>
             <div class="name">
              <div class="surname">Nilsson</div>
               <div class="given-names">A.N.</div>
            </div>
             <div class="name">
              <div class="surname">Ronquist</div>
               <div class="given-names">F.</div>
            </div>
          </div>
          <div class="year">2013</div>
            . 
            <div class="article-title">Bayesian tests of topology hypotheses with an example from diving beetles</div>
            . 
            <div class="source">Syst. Biol.</div>&nbsp;<div class="volume">62</div>
            :
            <div class="fpage">660</div>
            –
            <div class="lpage">673</div>
            .
          <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Bayesian%20tests%20of%20topology%20hypotheses%20with%20an%20example%20from%20diving%20beetles&amp;author=J.%20Bergsten&amp;author=A.N.%20Nilsson&amp;author=F.%20Ronquist&amp;publication_year=2013&amp;journal=Syst.%20Biol.&amp;volume=62&amp;pages=660-673" target="">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/sysbio/syt029" target="">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fsysbio%2Fsyt029" target="">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1093%2Fsysbio%2Fsyt029"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23628960" class="link link-pub-id" target="">PubMed</a></div>&nbsp;</div></div></div></div></div><div content-id="B5" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="B5" class="ref-content "><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Bouckaert</div>
               <div class="given-names">R.</div>
            </div>
             <div class="name">
              <div class="surname">Heled</div>
               <div class="given-names">J.</div>
            </div>
             <div class="name">
              <div class="surname">Kühnert</div>
               <div class="given-names">D.</div>
            </div>
             <div class="name">
              <div class="surname">Vaughan</div>
               <div class="given-names">T.</div>
            </div>
             <div class="name">
              <div class="surname">Wu</div>
               <div class="given-names">C.-H.</div>
            </div>
             <div class="name">
              <div class="surname">Xie</div>
               <div class="given-names">D.</div>
            </div>
             <div class="name">
              <div class="surname">Suchard</div>
               <div class="given-names">M. A.</div>
            </div>
             <div class="name">
              <div class="surname">Rambaut</div>
               <div class="given-names">A.</div>
            </div>
             <div class="name">
              <div class="surname">Drummond</div>
               <div class="given-names">A. J.</div>
            </div>
          </div>
          <div class="year">2014</div>
            . 
            <div class="article-title">BEAST 2: a software platform for Bayesian evolutionary analysis</div>
            . 
            <div class="source">PLoS Comput. Biol.</div>&nbsp;<div class="volume">10</div>
            :
            <div class="fpage">e1003537</div>
            .
          <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=BEAST%202%3A%20a%20software%20platform%20for%20Bayesian%20evolutionary%20analysis&amp;author=R.%20Bouckaert&amp;author=J.%20Heled&amp;author=D.%20K%C3%BChnert&amp;author=T.%20Vaughan&amp;author=C.-H.%20Wu&amp;author=D.%20Xie&amp;author=M.%20A.%20Suchard&amp;author=A.%20Rambaut&amp;author=A.%20J.%20Drummond&amp;publication_year=2014&amp;journal=PLoS%20Comput.%20Biol.&amp;volume=10&amp;pages=e1003537" target="">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pcbi.1003537" target="">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pcbi.1003537" target="">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1371%2Fjournal.pcbi.1003537"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24722319" class="link link-pub-id" target="">PubMed</a></div>&nbsp;</div></div></div></div></div><div content-id="B6" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="B6" class="ref-content "><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Didier</div>
               <div class="given-names">G.</div>
            </div>
             <div class="name">
              <div class="surname">Royer-Carenzi</div>
               <div class="given-names">M.</div>
            </div>
             <div class="name">
              <div class="surname">Laurin</div>
               <div class="given-names">M.</div>
            </div>
          </div>
          <div class="year">2012</div>
            . 
            <div class="article-title">The reconstructed evolutionary process with the fossil record</div>
            . 
            <div class="source">J. Theoret. Biol.</div>&nbsp;<div class="volume">315</div>
            :
            <div class="fpage">26</div>
            –
            <div class="lpage">37</div>
            .
          <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20reconstructed%20evolutionary%20process%20with%20the%20fossil%20record&amp;author=G.%20Didier&amp;author=M.%20Royer-Carenzi&amp;author=M.%20Laurin&amp;publication_year=2012&amp;journal=J.%20Theoret.%20Biol.&amp;volume=315&amp;pages=26-37" target="">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.jtbi.2012.08.046" target="">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.jtbi.2012.08.046" target="">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1016%2Fj.jtbi.2012.08.046"> </span></div>&nbsp;</div></div></div></div></div><div content-id="B7" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="B7" class="ref-content "><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Dornburg</div>
               <div class="given-names">A.</div>
            </div>
             <div class="name">
              <div class="surname">Friedman</div>
               <div class="given-names">M.</div>
            </div>
             <div class="name">
              <div class="surname">Near</div>
               <div class="given-names">T.J.</div>
            </div>
          </div>
          <div class="year">2015</div>
            . 
            <div class="article-title">Phylogenetic analysis of 
molecular and morphological data highlights uncertainty in the 
relationships of fossil and living species of elopomorpha 
(Actinopterygii: Teleostei)</div>
            . 
            <div class="source">Mol. Phylogenet. Evol.</div>&nbsp;<div class="volume">89</div>
            :
            <div class="fpage">205</div>
            –
            <div class="lpage">218</div>
            .
          <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Phylogenetic%20analysis%20of%20molecular%20and%20morphological%20data%20highlights%20uncertainty%20in%20the%20relationships%20of%20fossil%20and%20living%20species%20of%20elopomorpha%20%28Actinopterygii%3A%20Teleostei%29&amp;author=A.%20Dornburg&amp;author=M.%20Friedman&amp;author=T.J.%20Near&amp;publication_year=2015&amp;journal=Mol.%20Phylogenet.%20Evol.&amp;volume=89&amp;pages=205-218" target="">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.ympev.2015.04.004" target="">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.ympev.2015.04.004" target="">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1016%2Fj.ympev.2015.04.004"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25899306" class="link link-pub-id" target="">PubMed</a></div>&nbsp;</div></div></div></div></div><div content-id="B8" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="B8" class="ref-content "><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Drummond</div>
               <div class="given-names">A.J.</div>
            </div>
             <div class="name">
              <div class="surname">Suchard</div>
               <div class="given-names">M.A.</div>
            </div>
          </div>
          <div class="year">2010</div>
            . 
            <div class="article-title">Bayesian random local clocks, or one rate to rule them all</div>
            . 
            <div class="source">BMC Biol.</div>&nbsp;<div class="volume">8</div>
            :
            <div class="fpage">114</div>
            .
          <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Bayesian%20random%20local%20clocks%2C%20or%20one%20rate%20to%20rule%20them%20all&amp;author=A.J.%20Drummond&amp;author=M.A.%20Suchard&amp;publication_year=2010&amp;journal=BMC%20Biol.&amp;volume=8&amp;pages=114" target="">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1741-7007-8-114" target="">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1741-7007-8-114" target="">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1186%2F1741-7007-8-114"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20807414" class="link link-pub-id" target="">PubMed</a></div>&nbsp;</div></div></div></div></div><div content-id="B9" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="B9" class="ref-content "><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Drummond</div>
               <div class="given-names">A.J.</div>
            </div>
             <div class="name">
              <div class="surname">Ho</div>
               <div class="given-names">S.Y.</div>
            </div>
             <div class="name">
              <div class="surname">Phillips</div>
               <div class="given-names">M.J.</div>
            </div>
             <div class="name">
              <div class="surname">Rambaut</div>
               <div class="given-names">A.</div>
            </div>
          </div>
          <div class="year">2006</div>
            . 
            <div class="article-title">Relaxed phylogenetics and dating with confidence</div>
            . 
            <div class="source">PLoS Biol.</div>&nbsp;<div class="volume">4</div>
            :
            <div class="fpage">e88</div>
            .
          <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Relaxed%20phylogenetics%20and%20dating%20with%20confidence&amp;author=A.J.%20Drummond&amp;author=S.Y.%20Ho&amp;author=M.J.%20Phillips&amp;author=A.%20Rambaut&amp;publication_year=2006&amp;journal=PLoS%20Biol.&amp;volume=4&amp;pages=e88" target="">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pbio.0040088" target="">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pbio.0040088" target="">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1371%2Fjournal.pbio.0040088"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16683862" class="link link-pub-id" target="">PubMed</a></div>&nbsp;</div></div></div></div></div><div content-id="B10" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="B10" class="ref-content "><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Ford</div>
               <div class="given-names">D.</div>
            </div>
             <div class="name">
              <div class="surname">Gernhard</div>
               <div class="given-names">T.</div>
            </div>
             <div class="name">
              <div class="surname">Matsen</div>
               <div class="given-names">E.</div>
            </div>
          </div>
          <div class="year">2009</div>
            . 
            <div class="article-title">A method for investigating relative timing information on phylogenetic trees</div>
            . 
            <div class="source">Syst. Biol.</div>&nbsp;<div class="volume">58</div>
            :
            <div class="fpage">167</div>
            .
          <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20method%20for%20investigating%20relative%20timing%20information%20on%20phylogenetic%20trees&amp;author=D.%20Ford&amp;author=T.%20Gernhard&amp;author=E.%20Matsen&amp;publication_year=2009&amp;journal=Syst.%20Biol.&amp;volume=58&amp;pages=167" target="">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/sysbio/syp018" target="">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fsysbio%2Fsyp018" target="">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1093%2Fsysbio%2Fsyp018"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20525576" class="link link-pub-id" target="">PubMed</a></div>&nbsp;</div></div></div></div></div><div content-id="B11" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="B11" class="ref-content "><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Gavryushkina</div>
               <div class="given-names">A.</div>
            </div>
             <div class="name">
              <div class="surname">Welch</div>
               <div class="given-names">D.</div>
            </div>
             <div class="name">
              <div class="surname">Stadler</div>
               <div class="given-names">T.</div>
            </div>
             <div class="name">
              <div class="surname">Drummond</div>
               <div class="given-names">A.</div>
            </div>
          </div>
          <div class="year">2014</div>
            . 
            <div class="article-title">Bayesian inference of sampled ancestor trees for epidemiology and fossil calibration</div>
            . 
            <div class="source">PLoS Comput. Biol.</div>&nbsp;<div class="volume">10</div>
            :
            <div class="fpage">e1003919</div>
            .
          <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Bayesian%20inference%20of%20sampled%20ancestor%20trees%20for%20epidemiology%20and%20fossil%20calibration&amp;author=A.%20Gavryushkina&amp;author=D.%20Welch&amp;author=T.%20Stadler&amp;author=A.%20Drummond&amp;publication_year=2014&amp;journal=PLoS%20Comput.%20Biol.&amp;volume=10&amp;pages=e1003919" target="">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pcbi.1003919" target="">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pcbi.1003919" target="">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1371%2Fjournal.pcbi.1003919"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25474353" class="link link-pub-id" target="">PubMed</a></div>&nbsp;</div></div></div></div></div><div content-id="B12" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="B12" class="ref-content "><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Gernhard</div>
               <div class="given-names">T.</div>
            </div>
          </div>
          <div class="year">2008</div>
            . 
            <div class="article-title">The conditioned reconstructed process</div>
            . 
            <div class="source">J. Theoret. Biol.</div>&nbsp;<div class="volume">253</div>
            :
            <div class="fpage">769</div>
            –
            <div class="lpage">778</div>
            .
          <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20conditioned%20reconstructed%20process&amp;author=T.%20Gernhard&amp;publication_year=2008&amp;journal=J.%20Theoret.%20Biol.&amp;volume=253&amp;pages=769-778" target="">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.jtbi.2008.04.005" target="">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.jtbi.2008.04.005" target="">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1016%2Fj.jtbi.2008.04.005"> </span></div>&nbsp;</div></div></div></div></div><div content-id="B13" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="B13" class="ref-content "><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Graur</div>
               <div class="given-names">D.</div>
            </div>
             <div class="name">
              <div class="surname">Martin</div>
               <div class="given-names">W.</div>
            </div>
          </div>
          <div class="year">2004</div>
            . 
            <div class="article-title">Reading the entrails of chickens: Molecular timescales of evolution and the illusion of precision</div>
            . 
            <div class="source">Trends Genet.</div>&nbsp;<div class="volume">20</div>
            :
            <div class="fpage">80</div>
            –
            <div class="lpage">86</div>
            .
          <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Reading%20the%20entrails%20of%20chickens%3A%20Molecular%20timescales%20of%20evolution%20and%20the%20illusion%20of%20precision&amp;author=D.%20Graur&amp;author=W.%20Martin&amp;publication_year=2004&amp;journal=Trends%20Genet.&amp;volume=20&amp;pages=80-86" target="">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.tig.2003.12.003" target="">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.tig.2003.12.003" target="">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1016%2Fj.tig.2003.12.003"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/14746989" class="link link-pub-id" target="">PubMed</a></div>&nbsp;</div></div></div></div></div><div content-id="B14" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="B14" class="ref-content "><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Grimm</div>
               <div class="given-names">G.W.</div>
            </div>
             <div class="name">
              <div class="surname">Kapli</div>
               <div class="given-names">P.</div>
            </div>
             <div class="name">
              <div class="surname">Bomfleur</div>
               <div class="given-names">B.</div>
            </div>
             <div class="name">
              <div class="surname">McLoughlin</div>
               <div class="given-names">S.</div>
            </div>
             <div class="name">
              <div class="surname">Renner</div>
               <div class="given-names">S.S.</div>
            </div>
          </div>
          <div class="year">2015</div>
            . 
            <div class="article-title">Using more than the oldest fossils: Dating osmundaceae with three Bayesian clock approaches</div>
            . 
            <div class="source">Syst. Biol.</div>&nbsp;<div class="volume">64</div>
            :
            <div class="fpage">396</div>
            –
            <div class="lpage">405</div>
            .
          <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Using%20more%20than%20the%20oldest%20fossils%3A%20Dating%20osmundaceae%20with%20three%20Bayesian%20clock%20approaches&amp;author=G.W.%20Grimm&amp;author=P.%20Kapli&amp;author=B.%20Bomfleur&amp;author=S.%20McLoughlin&amp;author=S.S.%20Renner&amp;publication_year=2015&amp;journal=Syst.%20Biol.&amp;volume=64&amp;pages=396-405" target="">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/sysbio/syu108" target="">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fsysbio%2Fsyu108" target="">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1093%2Fsysbio%2Fsyu108"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25503771" class="link link-pub-id" target="">PubMed</a></div>&nbsp;</div></div></div></div></div><div content-id="B15" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="B15" class="ref-content "><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Hasegawa</div>
               <div class="given-names">M.</div>
            </div>
             <div class="name">
              <div class="surname">Kishino</div>
               <div class="given-names">H.</div>
            </div>
             <div class="name">
              <div class="surname">Yano</div>
               <div class="given-names">T.</div>
            </div>
          </div>
          <div class="year">1989</div>
            . 
            <div class="article-title">Estimation of branching dates among primates by molecular clocks of nuclear DNA which slowed down in Hominoidea</div>
            . 
            <div class="source">J. Human Evol.</div>&nbsp;<div class="volume">18</div>
            :
            <div class="fpage">461</div>
            –
            <div class="lpage">476</div>
            .
          <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Estimation%20of%20branching%20dates%20among%20primates%20by%20molecular%20clocks%20of%20nuclear%20DNA%20which%20slowed%20down%20in%20Hominoidea&amp;author=M.%20Hasegawa&amp;author=H.%20Kishino&amp;author=T.%20Yano&amp;publication_year=1989&amp;journal=J.%20Human%20Evol.&amp;volume=18&amp;pages=461-476" target="">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/0047-2484%2889%2990075-4" target="">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2F0047-2484%2889%2990075-4" target="">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1016%2F0047-2484(89)90075-4"> </span></div>&nbsp;</div></div></div></div></div><div content-id="B16" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="B16" class="ref-content "><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Heath</div>
               <div class="given-names">T.A.</div>
            </div>
          </div>
          <div class="year">2012</div>
            . 
            <div class="article-title">A hierarchical Bayesian model for calibrating estimates of species divergence times</div>
            . 
            <div class="source">Syst. Biol.</div>&nbsp;<div class="volume">61</div>
            :
            <div class="fpage">793</div>
            –
            <div class="lpage">809</div>
            .
          <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20hierarchical%20Bayesian%20model%20for%20calibrating%20estimates%20of%20species%20divergence%20times&amp;author=T.A.%20Heath&amp;publication_year=2012&amp;journal=Syst.%20Biol.&amp;volume=61&amp;pages=793-809" target="">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/sysbio/sys032" target="">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fsysbio%2Fsys032" target="">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1093%2Fsysbio%2Fsys032"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22334343" class="link link-pub-id" target="">PubMed</a></div>&nbsp;</div></div></div></div></div><div content-id="B17" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="B17" class="ref-content "><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Heath</div>
               <div class="given-names">T.A.</div>
            </div>
             <div class="name">
              <div class="surname">Holder</div>
               <div class="given-names">M.T.</div>
            </div>
             <div class="name">
              <div class="surname">Huelsenbeck</div>
               <div class="given-names">J.P.</div>
            </div>
          </div>
          <div class="year">2012</div>
            . 
            <div class="article-title">A Dirichlet process prior for estimating lineage-specific substitution rates</div>
            . 
            <div class="source">Mole. Biol. Evol.</div>&nbsp;<div class="volume">29</div>
            :
            <div class="fpage">939</div>
            –
            <div class="lpage">255</div>
            .
          <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20Dirichlet%20process%20prior%20for%20estimating%20lineage-specific%20substitution%20rates&amp;author=T.A.%20Heath&amp;author=M.T.%20Holder&amp;author=J.P.%20Huelsenbeck&amp;publication_year=2012&amp;journal=Mole.%20Biol.%20Evol.&amp;volume=29&amp;pages=939-255" target="">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/molbev/msr255" target="">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fmolbev%2Fmsr255" target="">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1093%2Fmolbev%2Fmsr255"> </span></div>&nbsp;</div></div></div></div></div><div content-id="B18" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="B18" class="ref-content "><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Heath</div>
               <div class="given-names">T.A.</div>
            </div>
             <div class="name">
              <div class="surname">Huelsenbeck</div>
               <div class="given-names">J.P.</div>
            </div>
             <div class="name">
              <div class="surname">Stadler</div>
               <div class="given-names">T.</div>
            </div>
          </div>
          <div class="year">2014</div>
            . 
            <div class="article-title">The fossilized birth–death process for coherent calibration of divergence-time estimates</div>
            . 
            <div class="source">Proc. Natl Acad. Sci., USA</div>&nbsp;<div class="volume">111</div>
            :
            <div class="fpage">E2957</div>
            –
            <div class="lpage">E2966</div>
            .
          <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20fossilized%20birth%E2%80%93death%20process%20for%20coherent%20calibration%20of%20divergence-time%20estimates&amp;author=T.A.%20Heath&amp;author=J.P.%20Huelsenbeck&amp;author=T.%20Stadler&amp;publication_year=2014&amp;journal=Proc.%20Natl%20Acad.%20Sci.%2C%20USA&amp;volume=111&amp;pages=E2957-E2966" target="">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.1319091111" target="">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.1319091111" target="">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1073%2Fpnas.1319091111"> </span></div>&nbsp;</div></div></div></div></div><div content-id="B19" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="B19" class="ref-content "><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Heath</div>
               <div class="given-names">T.A.</div>
            </div>
             <div class="name">
              <div class="surname">Moore</div>
               <div class="given-names">B.R.</div>
            </div>
          </div>
          <div class="person-group">
            <div class="name">
              <div class="surname">Chen</div>
               <div class="given-names">M.-H.</div>
            </div>
             <div class="name">
              <div class="surname">Kuo</div>
               <div class="given-names">L.</div>
            </div>
             <div class="name">
              <div class="surname">Lewis</div>
               <div class="given-names">P.O.</div>
            </div>
          </div>
          <div class="year">2014</div>
            . 
            <div class="article-title">Bayesian inference of species divergence times</div>
            . In 
            <div class="source">Bayesian phylogenetics: methods, algorithms, and applications</div>&nbsp;
            . 
            <div class="publisher-loc">Florida, US</div>
            : 
            <div class="publisher-name">Chapman &amp; Hall/CRC</div>
            .
          </div></div></div></div><div content-id="B20" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="B20" class="ref-content "><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Hedges</div>
               <div class="given-names">S.B.</div>
            </div>
             <div class="name">
              <div class="surname">Kumar</div>
               <div class="given-names">S.</div>
            </div>
          </div>
          <div class="year">2004</div>
            . 
            <div class="article-title">Precision of molecular time estimates</div>
            . 
            <div class="source">Trends Genet.</div>&nbsp;<div class="volume">20</div>
            :
            <div class="fpage">242</div>
            –
            <div class="lpage">247</div>
            .
          <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Precision%20of%20molecular%20time%20estimates&amp;author=S.B.%20Hedges&amp;author=S.%20Kumar&amp;publication_year=2004&amp;journal=Trends%20Genet.&amp;volume=20&amp;pages=242-247" target="">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.tig.2004.03.004" target="">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.tig.2004.03.004" target="">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1016%2Fj.tig.2004.03.004"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15109778" class="link link-pub-id" target="">PubMed</a></div>&nbsp;</div></div></div></div></div><div content-id="B21" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="B21" class="ref-content "><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Ho</div>
               <div class="given-names">S.Y.W.</div>
            </div>
             <div class="name">
              <div class="surname">Phillips</div>
               <div class="given-names">M.J.</div>
            </div>
          </div>
          <div class="year">2009</div>
            . 
            <div class="article-title">Accounting for calibration uncertainty in phylogenetic estimation of evolutionary divergence times</div>
            . 
            <div class="source">Syst. Biol.</div>&nbsp;<div class="volume">58</div>
            :
            <div class="fpage">367</div>
            –
            <div class="lpage">380</div>
            .
          <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Accounting%20for%20calibration%20uncertainty%20in%20phylogenetic%20estimation%20of%20evolutionary%20divergence%20times&amp;author=S.Y.W.%20Ho&amp;author=M.J.%20Phillips&amp;publication_year=2009&amp;journal=Syst.%20Biol.&amp;volume=58&amp;pages=367-380" target="">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/sysbio/syp035" target="">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fsysbio%2Fsyp035" target="">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1093%2Fsysbio%2Fsyp035"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20525591" class="link link-pub-id" target="">PubMed</a></div>&nbsp;</div></div></div></div></div><div content-id="B22" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="B22" class="ref-content "><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Höhna</div>
               <div class="given-names">S.</div>
            </div>
             <div class="name">
              <div class="surname">Stadler</div>
               <div class="given-names">T.</div>
            </div>
             <div class="name">
              <div class="surname">Ronquist</div>
               <div class="given-names">F.</div>
            </div>
             <div class="name">
              <div class="surname">Britton</div>
               <div class="given-names">T.</div>
            </div>
          </div>
          <div class="year">2011</div>
            . 
            <div class="article-title">Inferring speciation and extinction rates under different sampling schemes</div>
            . 
            <div class="source">Mole. Biol. Evol.</div>&nbsp;<div class="volume">28</div>
            :
            <div class="fpage">2577</div>
            –
            <div class="lpage">2589</div>
            .
          <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Inferring%20speciation%20and%20extinction%20rates%20under%20different%20sampling%20schemes&amp;author=S.%20H%C3%B6hna&amp;author=T.%20Stadler&amp;author=F.%20Ronquist&amp;author=T.%20Britton&amp;publication_year=2011&amp;journal=Mole.%20Biol.%20Evol.&amp;volume=28&amp;pages=2577-2589" target="">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/molbev/msr095" target="">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fmolbev%2Fmsr095" target="">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1093%2Fmolbev%2Fmsr095"> </span></div>&nbsp;</div></div></div></div></div><div content-id="B23" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="B23" class="ref-content "><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Huelsenbeck</div>
               <div class="given-names">J.P.</div>
            </div>
             <div class="name">
              <div class="surname">Larget</div>
               <div class="given-names">B.</div>
            </div>
             <div class="name">
              <div class="surname">Swofford</div>
               <div class="given-names">D.L.</div>
            </div>
          </div>
          <div class="year">2000</div>
            . 
            <div class="article-title">A compound Poisson process for relaxing the molecular clock</div>
            . 
            <div class="source">Genetics</div>&nbsp;<div class="volume">154</div>
            :
            <div class="fpage">1879</div>
            –
            <div class="lpage">1892</div>
            .
          <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20compound%20Poisson%20process%20for%20relaxing%20the%20molecular%20clock&amp;author=J.P.%20Huelsenbeck&amp;author=B.%20Larget&amp;author=D.L.%20Swofford&amp;publication_year=2000&amp;journal=Genetics&amp;volume=154&amp;pages=1879-1892" target="">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10747076" class="link link-pub-id" target="">PubMed</a></div>&nbsp;</div></div></div></div></div><div content-id="B24" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="B24" class="ref-content "><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Jukes</div>
               <div class="given-names">T.H.</div>
            </div>
             <div class="name">
              <div class="surname">Cantor</div>
               <div class="given-names">C.R.</div>
            </div>
          </div>
          <div class="person-group">
            <div class="name">
              <div class="surname">Munro</div>
               <div class="given-names">H.N.</div>
            </div>
          </div>
            , 
            <div class="year">1969</div>
            . 
            <div class="article-title">Evolution of protein molecules</div>
            . In 
            <div class="source">Mammalian protein metabolism</div>&nbsp;
            . 
            <div class="publisher-loc">New York</div>
            : 
            <div class="publisher-name">Academic Press</div>
            , pp. 
            <div class="fpage">21</div>
            –
            <div class="lpage">123</div>
            .
          <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Evolution%20of%20protein%20molecules&amp;author=T.H.%20Jukes&amp;author=C.R.%20Cantor&amp;author=H.N.%20Munro&amp;publication_year=1969&amp;book=Mammalian%20protein%20metabolism" target="">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/B978-1-4832-3211-9.50009-7" target="">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2FB978-1-4832-3211-9.50009-7" target="">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1016%2FB978-1-4832-3211-9.50009-7"> </span></div>&nbsp;</div></div></div></div></div><div content-id="B25" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="B25" class="ref-content "><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Kendall</div>
               <div class="given-names">D.G.</div>
            </div>
          </div>
          <div class="year">1948</div>
            . 
            <div class="article-title">On the generalized “birth-and-death” process</div>
            . 
            <div class="source">Annals Math. Stat.</div>&nbsp;<div class="volume">19</div>
            :
            <div class="fpage">1</div>
            –
            <div class="lpage">15</div>
            .
          <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=On%20the%20generalized%20%E2%80%9Cbirth-and-death%E2%80%9D%20process&amp;author=D.G.%20Kendall&amp;publication_year=1948&amp;journal=Annals%20Math.%20Stat.&amp;volume=19&amp;pages=1-15" target="">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1214/aoms/1177730285" target="">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1214%2Faoms%2F1177730285" target="">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1214%2Faoms%2F1177730285"> </span></div>&nbsp;</div></div></div></div></div><div content-id="B26" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="B26" class="ref-content "><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Kishino</div>
               <div class="given-names">H.</div>
            </div>
             <div class="name">
              <div class="surname">Miyata</div>
               <div class="given-names">T.</div>
            </div>
             <div class="name">
              <div class="surname">Hasegawa</div>
               <div class="given-names">M.</div>
            </div>
          </div>
          <div class="year">1990</div>
            . 
            <div class="article-title">Maximum likelihood inference of protein phylogeny and th origin of chloroplasts</div>
            . 
            <div class="source">J. Mol. Evol.</div>&nbsp;<div class="volume">31</div>
            :
            <div class="fpage">151</div>
            –
            <div class="lpage">160</div>
            .
          <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Maximum%20likelihood%20inference%20of%20protein%20phylogeny%20and%20th%20origin%20of%20chloroplasts&amp;author=H.%20Kishino&amp;author=T.%20Miyata&amp;author=M.%20Hasegawa&amp;publication_year=1990&amp;journal=J.%20Mol.%20Evol.&amp;volume=31&amp;pages=151-160" target="">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/BF02109483" target="">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2FBF02109483" target="">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1007%2FBF02109483"> </span></div>&nbsp;</div></div></div></div></div><div content-id="B27" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="B27" class="ref-content "><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Kishino</div>
               <div class="given-names">H.</div>
            </div>
             <div class="name">
              <div class="surname">Thorne</div>
               <div class="given-names">J.L.</div>
            </div>
             <div class="name">
              <div class="surname">Bruno</div>
               <div class="given-names">W.</div>
            </div>
          </div>
          <div class="year">2001</div>
            . 
            <div class="article-title">Performance of a divergence time estimation method under a probabilistic model of rate evolution</div>
            . 
            <div class="source">Mol. Biol. Evol.</div>&nbsp;<div class="volume">18</div>
            :
            <div class="fpage">352</div>
            –
            <div class="lpage">361</div>
            .
          <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Performance%20of%20a%20divergence%20time%20estimation%20method%20under%20a%20probabilistic%20model%20of%20rate%20evolution&amp;author=H.%20Kishino&amp;author=J.L.%20Thorne&amp;author=W.%20Bruno&amp;publication_year=2001&amp;journal=Mol.%20Biol.%20Evol.&amp;volume=18&amp;pages=352-361" target="">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/oxfordjournals.molbev.a003811" target="">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Foxfordjournals.molbev.a003811" target="">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1093%2Foxfordjournals.molbev.a003811"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11230536" class="link link-pub-id" target="">PubMed</a></div>&nbsp;</div></div></div></div></div><div content-id="B28" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="B28" class="ref-content "><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Klopfstein</div>
               <div class="given-names">S.</div>
            </div>
             <div class="name">
              <div class="surname">Ronquist</div>
               <div class="given-names">F.</div>
            </div>
          </div>
          <div class="year">2013</div>
            . 
            <div class="article-title">Convergent intron gains in hymenopteran elongation factor-1α</div>
            . 
            <div class="source">Mol. Phylogenet. Evol.</div>&nbsp;<div class="volume">67</div>
            :
            <div class="fpage">266</div>
            –
            <div class="lpage">276</div>
            .
          <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Convergent%20intron%20gains%20in%20hymenopteran%20elongation%20factor-1%CE%B1&amp;author=S.%20Klopfstein&amp;author=F.%20Ronquist&amp;publication_year=2013&amp;journal=Mol.%20Phylogenet.%20Evol.&amp;volume=67&amp;pages=266-276" target="">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.ympev.2013.01.015" target="">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.ympev.2013.01.015" target="">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1016%2Fj.ympev.2013.01.015"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23396205" class="link link-pub-id" target="">PubMed</a></div>&nbsp;</div></div></div></div></div><div content-id="B29" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="B29" class="ref-content "><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Lambert</div>
               <div class="given-names">A.</div>
            </div>
             <div class="name">
              <div class="surname">Stadler</div>
               <div class="given-names">T.</div>
            </div>
          </div>
          <div class="year">2013</div>
            . 
            <div class="article-title">Birth–death models and coalescent point processes: The shape and probability of reconstructed phylogenies</div>
            . 
            <div class="source">Theoret. Popul. Biol.</div>&nbsp;<div class="volume">90</div>
            :
            <div class="fpage">113</div>
            –
            <div class="lpage">128</div>
            .
          <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Birth%E2%80%93death%20models%20and%20coalescent%20point%20processes%3A%20The%20shape%20and%20probability%20of%20reconstructed%20phylogenies&amp;author=A.%20Lambert&amp;author=T.%20Stadler&amp;publication_year=2013&amp;journal=Theoret.%20Popul.%20Biol.&amp;volume=90&amp;pages=113-128" target="">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.tpb.2013.10.002" target="">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.tpb.2013.10.002" target="">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1016%2Fj.tpb.2013.10.002"> </span></div>&nbsp;</div></div></div></div></div><div content-id="B30" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="B30" class="ref-content "><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Lee</div>
               <div class="given-names">M.S.Y.</div>
            </div>
             <div class="name">
              <div class="surname">Oliver</div>
               <div class="given-names">P.M.</div>
            </div>
             <div class="name">
              <div class="surname">Hutchinson</div>
               <div class="given-names">M.N.</div>
            </div>
          </div>
          <div class="year">2009</div>
            . 
            <div class="article-title">Phylogenetic uncertainty and molecular clock calibrations: a case study of legless lizards (Pygopodidae, Gekkota)</div>
            . 
            <div class="source">Mol. Phylogenet. Evol.</div>&nbsp;<div class="volume">50</div>
            :
            <div class="fpage">661</div>
            –
            <div class="lpage">666</div>
            .
          <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Phylogenetic%20uncertainty%20and%20molecular%20clock%20calibrations%3A%20a%20case%20study%20of%20legless%20lizards%20%28Pygopodidae%2C%20Gekkota%29&amp;author=M.S.Y.%20Lee&amp;author=P.M.%20Oliver&amp;author=M.N.%20Hutchinson&amp;publication_year=2009&amp;journal=Mol.%20Phylogenet.%20Evol.&amp;volume=50&amp;pages=661-666" target="">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.ympev.2008.11.024" target="">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.ympev.2008.11.024" target="">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1016%2Fj.ympev.2008.11.024"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19111935" class="link link-pub-id" target="">PubMed</a></div>&nbsp;</div></div></div></div></div><div content-id="B31" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="B31" class="ref-content "><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Lepage</div>
               <div class="given-names">T.</div>
            </div>
             <div class="name">
              <div class="surname">Lawi</div>
               <div class="given-names">S.</div>
            </div>
             <div class="name">
              <div class="surname">Tupper</div>
               <div class="given-names">P.</div>
            </div>
             <div class="name">
              <div class="surname">Bryant</div>
               <div class="given-names">D.</div>
            </div>
          </div>
          <div class="year">2006</div>
            . 
            <div class="article-title">Continuous and tractable models for the variation of evolutionary rates</div>
            . 
            <div class="source">Math. Biosci.</div>&nbsp;<div class="volume">199</div>
            :
            <div class="fpage">216</div>
            –
            <div class="lpage">233</div>
            .
          <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Continuous%20and%20tractable%20models%20for%20the%20variation%20of%20evolutionary%20rates&amp;author=T.%20Lepage&amp;author=S.%20Lawi&amp;author=P.%20Tupper&amp;author=D.%20Bryant&amp;publication_year=2006&amp;journal=Math.%20Biosci.&amp;volume=199&amp;pages=216-233" target="">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.mbs.2005.11.002" target="">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.mbs.2005.11.002" target="">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1016%2Fj.mbs.2005.11.002"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16406009" class="link link-pub-id" target="">PubMed</a></div>&nbsp;</div></div></div></div></div><div content-id="B32" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="B32" class="ref-content "><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Lepage</div>
               <div class="given-names">T.</div>
            </div>
             <div class="name">
              <div class="surname">Bryant</div>
               <div class="given-names">D.</div>
            </div>
             <div class="name">
              <div class="surname">Philippe</div>
               <div class="given-names">H.</div>
            </div>
             <div class="name">
              <div class="surname">Lartillot</div>
               <div class="given-names">N.</div>
            </div>
          </div>
          <div class="year">2007</div>
            . 
            <div class="article-title">A general comparison of relaxed molecular clock models</div>
            . 
            <div class="source">Mol. Biol. Evol.</div>&nbsp;<div class="volume">24</div>
            :
            <div class="fpage">2669</div>
            –
            <div class="lpage">2680</div>
            .
          <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20general%20comparison%20of%20relaxed%20molecular%20clock%20models&amp;author=T.%20Lepage&amp;author=D.%20Bryant&amp;author=H.%20Philippe&amp;author=N.%20Lartillot&amp;publication_year=2007&amp;journal=Mol.%20Biol.%20Evol.&amp;volume=24&amp;pages=2669-2680" target="">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/molbev/msm193" target="">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fmolbev%2Fmsm193" target="">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1093%2Fmolbev%2Fmsm193"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17890241" class="link link-pub-id" target="">PubMed</a></div>&nbsp;</div></div></div></div></div><div content-id="B33" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="B33" class="ref-content "><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Lewis</div>
               <div class="given-names">P.O.</div>
            </div>
          </div>
          <div class="year">2001</div>
            . 
            <div class="article-title">A likelihood approach to estimating phylogeny from discrete morphological character data</div>
            . 
            <div class="source">Syst. Biol.</div>&nbsp;<div class="volume">50</div>
            :
            <div class="fpage">913</div>
            –
            <div class="lpage">925</div>
            .
          <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20likelihood%20approach%20to%20estimating%20phylogeny%20from%20discrete%20morphological%20character%20data&amp;author=P.O.%20Lewis&amp;publication_year=2001&amp;journal=Syst.%20Biol.&amp;volume=50&amp;pages=913-925" target="">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1080/106351501753462876" target="">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1080%2F106351501753462876" target="">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1080%2F106351501753462876"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12116640" class="link link-pub-id" target="">PubMed</a></div>&nbsp;</div></div></div></div></div><div content-id="B34" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="B34" class="ref-content "><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Lewis</div>
               <div class="given-names">P.</div>
            </div>
             <div class="name">
              <div class="surname">Holder</div>
               <div class="given-names">M.</div>
            </div>
             <div class="name">
              <div class="surname">Holsinger</div>
               <div class="given-names">K.</div>
            </div>
          </div>
          <div class="year">2005</div>
            . 
            <div class="article-title">Polytomies and Bayesian phylogenetic inference</div>
            . 
            <div class="source">Syst. Biol.</div>&nbsp;<div class="volume">54</div>
            :
            <div class="fpage">241</div>
            –
            <div class="lpage">253</div>
            .
          <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Polytomies%20and%20Bayesian%20phylogenetic%20inference&amp;author=P.%20Lewis&amp;author=M.%20Holder&amp;author=K.%20Holsinger&amp;publication_year=2005&amp;journal=Syst.%20Biol.&amp;volume=54&amp;pages=241-253" target="">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1080/10635150590924208" target="">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1080%2F10635150590924208" target="">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1080%2F10635150590924208"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16012095" class="link link-pub-id" target="">PubMed</a></div>&nbsp;</div></div></div></div></div><div content-id="B35" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="B35" class="ref-content "><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Misof</div>
               <div class="given-names">B.</div>
            </div>
             <div class="name">
              <div class="surname">Liu</div>
               <div class="given-names">S.</div>
            </div>
             <div class="name">
              <div class="surname">Meusemann</div>
               <div class="given-names">K.</div>
            </div>
             <div class="name">
              <div class="surname">Peters</div>
               <div class="given-names">R.S.</div>
            </div>
             <div class="name">
              <div class="surname">Donath</div>
               <div class="given-names">A.</div>
            </div>
             <div class="name">
              <div class="surname">Mayer</div>
               <div class="given-names">C.</div>
            </div>
             <div class="name">
              <div class="surname">Frandsen</div>
               <div class="given-names">P.B.</div>
            </div>
             <div class="name">
              <div class="surname">Ware</div>
               <div class="given-names">J.</div>
            </div>
             <div class="name">
              <div class="surname">Flouri</div>
               <div class="given-names">T.</div>
            </div>
             <div class="name">
              <div class="surname">Beutel</div>
               <div class="given-names">R.G.</div>
            </div>
             <div class="name">
              <div class="surname">Niehuis</div>
               <div class="given-names">O.</div>
            </div>
             <div class="name">
              <div class="surname">Petersen</div>
               <div class="given-names">M.</div>
            </div>
             <div class="name">
              <div class="surname">Izquierdo-Carrasco</div>
               <div class="given-names">F.</div>
            </div>
             <div class="name">
              <div class="surname">Wappler</div>
               <div class="given-names">T.</div>
            </div>
             <div class="name">
              <div class="surname">Rust</div>
               <div class="given-names">J.</div>
            </div>
             <div class="name">
              <div class="surname">Aberer</div>
               <div class="given-names">A.J.</div>
            </div>
             <div class="name">
              <div class="surname">Aspöck</div>
               <div class="given-names">U.</div>
            </div>
             <div class="name">
              <div class="surname">Aspöck</div>
               <div class="given-names">H.</div>
            </div>
             <div class="name">
              <div class="surname">Bartel</div>
               <div class="given-names">D.</div>
            </div>
             <div class="name">
              <div class="surname">Blanke</div>
               <div class="given-names">A.</div>
            </div>
             <div class="name">
              <div class="surname">Berger</div>
               <div class="given-names">S.</div>
            </div>
             <div class="name">
              <div class="surname">Böhm</div>
               <div class="given-names">A.</div>
            </div>
             <div class="name">
              <div class="surname">Buckley</div>
               <div class="given-names">T.R.</div>
            </div>
             <div class="name">
              <div class="surname">Calcott</div>
               <div class="given-names">B.</div>
            </div>
             <div class="name">
              <div class="surname">Chen</div>
               <div class="given-names">J.</div>
            </div>
             <div class="name">
              <div class="surname">Friedrich</div>
               <div class="given-names">F.</div>
            </div>
             <div class="name">
              <div class="surname">Fukui</div>
               <div class="given-names">M.</div>
            </div>
             <div class="name">
              <div class="surname">Fujita</div>
               <div class="given-names">M.</div>
            </div>
             <div class="name">
              <div class="surname">Greve</div>
               <div class="given-names">C.</div>
            </div>
             <div class="name">
              <div class="surname">Grobe</div>
               <div class="given-names">P.</div>
            </div>
             <div class="name">
              <div class="surname">Gu</div>
               <div class="given-names">S.</div>
            </div>
             <div class="name">
              <div class="surname">Huang</div>
               <div class="given-names">Y.</div>
            </div>
             <div class="name">
              <div class="surname">Jermiin</div>
               <div class="given-names">L.S.</div>
            </div>
             <div class="name">
              <div class="surname">Kawahara</div>
               <div class="given-names">A.Y.</div>
            </div>
             <div class="name">
              <div class="surname">Krogmann</div>
               <div class="given-names">L.</div>
            </div>
             <div class="name">
              <div class="surname">Kubiak</div>
               <div class="given-names">M.</div>
            </div>
             <div class="name">
              <div class="surname">Lanfear</div>
               <div class="given-names">R.</div>
            </div>
             <div class="name">
              <div class="surname">Letsch</div>
               <div class="given-names">H.</div>
            </div>
             <div class="name">
              <div class="surname">Li</div>
               <div class="given-names">Y.</div>
            </div>
             <div class="name">
              <div class="surname">Li</div>
               <div class="given-names">Z.</div>
            </div>
             <div class="name">
              <div class="surname">Li</div>
               <div class="given-names">J.</div>
            </div>
             <div class="name">
              <div class="surname">Lu</div>
               <div class="given-names">H.</div>
            </div>
             <div class="name">
              <div class="surname">Machida</div>
               <div class="given-names">R.</div>
            </div>
             <div class="name">
              <div class="surname">Mashimo</div>
               <div class="given-names">Y.</div>
            </div>
             <div class="name">
              <div class="surname">Kapli</div>
               <div class="given-names">P.</div>
            </div>
             <div class="name">
              <div class="surname">McKenna</div>
               <div class="given-names">D.D.</div>
            </div>
             <div class="name">
              <div class="surname">Meng</div>
               <div class="given-names">G.</div>
            </div>
             <div class="name">
              <div class="surname">Nakagaki</div>
               <div class="given-names">Y.</div>
            </div>
             <div class="name">
              <div class="surname">Navarrete-Heredia</div>
               <div class="given-names">J.L.</div>
            </div>
             <div class="name">
              <div class="surname">Ott</div>
               <div class="given-names">M.</div>
            </div>
             <div class="name">
              <div class="surname">Ou</div>
               <div class="given-names">Y.</div>
            </div>
             <div class="name">
              <div class="surname">Pass</div>
               <div class="given-names">G.</div>
            </div>
             <div class="name">
              <div class="surname">Podsiadlowski</div>
               <div class="given-names">L.</div>
            </div>
             <div class="name">
              <div class="surname">Pohl</div>
               <div class="given-names">H.</div>
            </div>
             <div class="name">
              <div class="surname">Reumont von</div>
               <div class="given-names">B.M.</div>
            </div>
             <div class="name">
              <div class="surname">Schütte</div>
               <div class="given-names">K.</div>
            </div>
             <div class="name">
              <div class="surname">Sekiya</div>
               <div class="given-names">K.</div>
            </div>
             <div class="name">
              <div class="surname">Shimizu</div>
               <div class="given-names">S.</div>
            </div>
             <div class="name">
              <div class="surname">Slipinski</div>
               <div class="given-names">A.</div>
            </div>
             <div class="name">
              <div class="surname">Stamatakis</div>
               <div class="given-names">A.</div>
            </div>
             <div class="name">
              <div class="surname">Song</div>
               <div class="given-names">W.</div>
            </div>
             <div class="name">
              <div class="surname">Su</div>
               <div class="given-names">X.</div>
            </div>
             <div class="name">
              <div class="surname">Szucsich</div>
               <div class="given-names">N.U.</div>
            </div>
             <div class="name">
              <div class="surname">Tan</div>
               <div class="given-names">M.</div>
            </div>
             <div class="name">
              <div class="surname">Tan</div>
               <div class="given-names">X.</div>
            </div>
             <div class="name">
              <div class="surname">Tang</div>
               <div class="given-names">M.</div>
            </div>
             <div class="name">
              <div class="surname">Tang</div>
               <div class="given-names">J.</div>
            </div>
             <div class="name">
              <div class="surname">Timelthaler</div>
               <div class="given-names">G.</div>
            </div>
             <div class="name">
              <div class="surname">Tomizuka</div>
               <div class="given-names">S.</div>
            </div>
             <div class="name">
              <div class="surname">Trautwein</div>
               <div class="given-names">M.</div>
            </div>
             <div class="name">
              <div class="surname">Tong</div>
               <div class="given-names">X.</div>
            </div>
             <div class="name">
              <div class="surname">Uchifune</div>
               <div class="given-names">T.</div>
            </div>
             <div class="name">
              <div class="surname">Walzl</div>
               <div class="given-names">M.G.</div>
            </div>
             <div class="name">
              <div class="surname">Wiegmann</div>
               <div class="given-names">B.M.</div>
            </div>
             <div class="name">
              <div class="surname">Wilbrandt</div>
               <div class="given-names">J.</div>
            </div>
             <div class="name">
              <div class="surname">Wipfler</div>
               <div class="given-names">B.</div>
            </div>
             <div class="name">
              <div class="surname">Wong</div>
               <div class="given-names">T.K.F.</div>
            </div>
             <div class="name">
              <div class="surname">Wu</div>
               <div class="given-names">Q.</div>
            </div>
             <div class="name">
              <div class="surname">Wu</div>
               <div class="given-names">G.</div>
            </div>
             <div class="name">
              <div class="surname">Xie</div>
               <div class="given-names">Y.</div>
            </div>
             <div class="name">
              <div class="surname">Yang</div>
               <div class="given-names">S.</div>
            </div>
             <div class="name">
              <div class="surname">Yang</div>
               <div class="given-names">Q.</div>
            </div>
             <div class="name">
              <div class="surname">Yeates</div>
               <div class="given-names">D.K.</div>
            </div>
             <div class="name">
              <div class="surname">Yoshizawa</div>
               <div class="given-names">K.</div>
            </div>
             <div class="name">
              <div class="surname">Zhang</div>
               <div class="given-names">Q.</div>
            </div>
             <div class="name">
              <div class="surname">Zhang</div>
               <div class="given-names">R.</div>
            </div>
             <div class="name">
              <div class="surname">Zhang</div>
               <div class="given-names">W.</div>
            </div>
             <div class="name">
              <div class="surname">Zhang</div>
               <div class="given-names">Y.</div>
            </div>
             <div class="name">
              <div class="surname">Zhao</div>
               <div class="given-names">J.</div>
            </div>
             <div class="name">
              <div class="surname">Zhou</div>
               <div class="given-names">C.</div>
            </div>
             <div class="name">
              <div class="surname">Zhou</div>
               <div class="given-names">L.</div>
            </div>
             <div class="name">
              <div class="surname">Ziesmann</div>
               <div class="given-names">T.</div>
            </div>
             <div class="name">
              <div class="surname">Zou</div>
               <div class="given-names">S.</div>
            </div>
             <div class="name">
              <div class="surname">Li</div>
               <div class="given-names">Y.</div>
            </div>
             <div class="name">
              <div class="surname">Xu</div>
               <div class="given-names">X.</div>
            </div>
             <div class="name">
              <div class="surname">Zhang</div>
               <div class="given-names">Y.</div>
            </div>
             <div class="name">
              <div class="surname">Yang</div>
               <div class="given-names">H.</div>
            </div>
             <div class="name">
              <div class="surname">Wang</div>
               <div class="given-names">J.</div>
            </div>
             <div class="name">
              <div class="surname">Wang</div>
               <div class="given-names">J.</div>
            </div>
             <div class="name">
              <div class="surname">Kjer</div>
               <div class="given-names">K.M.</div>
            </div>
             <div class="name">
              <div class="surname">Zhou</div>
               <div class="given-names">X.</div>
            </div>
          </div>
          <div class="year">2014</div>
            . 
            <div class="article-title">Phylogenomics resolves the timing and pattern of insect evolution</div>
            . 
            <div class="source">Science</div>&nbsp;<div class="volume">346</div>
            :
            <div class="fpage">763</div>
            –
            <div class="lpage">767</div>
            .
          <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Phylogenomics%20resolves%20the%20timing%20and%20pattern%20of%20insect%20evolution&amp;author=B.%20Misof&amp;author=S.%20Liu&amp;author=K.%20Meusemann&amp;author=R.S.%20Peters&amp;author=A.%20Donath&amp;author=C.%20Mayer&amp;author=P.B.%20Frandsen&amp;author=J.%20Ware&amp;author=T.%20Flouri&amp;author=R.G.%20Beutel&amp;author=O.%20Niehuis&amp;author=M.%20Petersen&amp;author=F.%20Izquierdo-Carrasco&amp;author=T.%20Wappler&amp;author=J.%20Rust&amp;author=A.J.%20Aberer&amp;author=U.%20Asp%C3%B6ck&amp;author=H.%20Asp%C3%B6ck&amp;author=D.%20Bartel&amp;author=A.%20Blanke&amp;author=S.%20Berger&amp;author=A.%20B%C3%B6hm&amp;author=T.R.%20Buckley&amp;author=B.%20Calcott&amp;author=J.%20Chen&amp;author=F.%20Friedrich&amp;author=M.%20Fukui&amp;author=M.%20Fujita&amp;author=C.%20Greve&amp;author=P.%20Grobe&amp;author=S.%20Gu&amp;author=Y.%20Huang&amp;author=L.S.%20Jermiin&amp;author=A.Y.%20Kawahara&amp;author=L.%20Krogmann&amp;author=M.%20Kubiak&amp;author=R.%20Lanfear&amp;author=H.%20Letsch&amp;author=Y.%20Li&amp;author=Z.%20Li&amp;author=J.%20Li&amp;author=H.%20Lu&amp;author=R.%20Machida&amp;author=Y.%20Mashimo&amp;author=P.%20Kapli&amp;author=D.D.%20McKenna&amp;author=G.%20Meng&amp;author=Y.%20Nakagaki&amp;author=J.L.%20Navarrete-Heredia&amp;author=M.%20Ott&amp;author=Y.%20Ou&amp;author=G.%20Pass&amp;author=L.%20Podsiadlowski&amp;author=H.%20Pohl&amp;author=B.M.%20Reumont%20von&amp;author=K.%20Sch%C3%BCtte&amp;author=K.%20Sekiya&amp;author=S.%20Shimizu&amp;author=A.%20Slipinski&amp;author=A.%20Stamatakis&amp;author=W.%20Song&amp;author=X.%20Su&amp;author=N.U.%20Szucsich&amp;author=M.%20Tan&amp;author=X.%20Tan&amp;author=M.%20Tang&amp;author=J.%20Tang&amp;author=G.%20Timelthaler&amp;author=S.%20Tomizuka&amp;author=M.%20Trautwein&amp;author=X.%20Tong&amp;author=T.%20Uchifune&amp;author=M.G.%20Walzl&amp;author=B.M.%20Wiegmann&amp;author=J.%20Wilbrandt&amp;author=B.%20Wipfler&amp;author=T.K.F.%20Wong&amp;author=Q.%20Wu&amp;author=G.%20Wu&amp;author=Y.%20Xie&amp;author=S.%20Yang&amp;author=Q.%20Yang&amp;author=D.K.%20Yeates&amp;author=K.%20Yoshizawa&amp;author=Q.%20Zhang&amp;author=R.%20Zhang&amp;author=W.%20Zhang&amp;author=Y.%20Zhang&amp;author=J.%20Zhao&amp;author=C.%20Zhou&amp;author=L.%20Zhou&amp;author=T.%20Ziesmann&amp;author=S.%20Zou&amp;author=Y.%20Li&amp;author=X.%20Xu&amp;author=Y.%20Zhang&amp;author=H.%20Yang&amp;author=J.%20Wang&amp;author=J.%20Wang&amp;author=K.M.%20Kjer&amp;author=X.%20Zhou&amp;publication_year=2014&amp;journal=Science&amp;volume=346&amp;pages=763-767" target="">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1126/science.1257570" target="">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1126%2Fscience.1257570" target="">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1126%2Fscience.1257570"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25378627" class="link link-pub-id" target="">PubMed</a></div>&nbsp;</div></div></div></div></div><div content-id="B36" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="B36" class="ref-content "><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Nee</div>
               <div class="given-names">S.</div>
            </div>
             <div class="name">
              <div class="surname">May</div>
               <div class="given-names">R.M.</div>
            </div>
             <div class="name">
              <div class="surname">Harvey</div>
               <div class="given-names">P.H.</div>
            </div>
          </div>
          <div class="year">1994</div>
            . 
            <div class="article-title">The reconstructed evolutionary process</div>
            . 
            <div class="source">Philo. Trans. R. Soc. B</div>&nbsp;<div class="volume">344</div>
            :
            <div class="fpage">305</div>
            –
            <div class="lpage">311</div>
            .
          <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20reconstructed%20evolutionary%20process&amp;author=S.%20Nee&amp;author=R.M.%20May&amp;author=P.H.%20Harvey&amp;publication_year=1994&amp;journal=Philo.%20Trans.%20R.%20Soc.%20B&amp;volume=344&amp;pages=305-311" target="">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1098/rstb.1994.0068" target="">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1098%2Frstb.1994.0068" target="">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1098%2Frstb.1994.0068"> </span></div>&nbsp;</div></div></div></div></div><div content-id="B37" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="B37" class="ref-content "><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Pyron</div>
               <div class="given-names">R.A.</div>
            </div>
          </div>
          <div class="year">2011</div>
            . 
            <div class="article-title">Divergence time estimation using fossils as terminal taxa and the origins of Lissamphibia</div>
            . 
            <div class="source">Syst. Biol.</div>&nbsp;<div class="volume">60</div>
            :
            <div class="fpage">466</div>
            –
            <div class="lpage">481</div>
            .
          <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Divergence%20time%20estimation%20using%20fossils%20as%20terminal%20taxa%20and%20the%20origins%20of%20Lissamphibia&amp;author=R.A.%20Pyron&amp;publication_year=2011&amp;journal=Syst.%20Biol.&amp;volume=60&amp;pages=466-481" target="">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/sysbio/syr047" target="">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fsysbio%2Fsyr047" target="">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1093%2Fsysbio%2Fsyr047"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21540408" class="link link-pub-id" target="">PubMed</a></div>&nbsp;</div></div></div></div></div><div content-id="B38" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="B38" class="ref-content "><div class="citation mixed-citation">
          <div class="collab">R Development Core Team</div>
             (
            <div class="year">2009</div>
            ). 
            <div class="source">R: a language and environment for statistical computing</div>&nbsp;
            . 
            <div class="publisher-name">R Foundation for Statistical Computing</div>
            , 
            <div class="publisher-loc">Vienna, Austria</div>
            .
          </div></div></div></div><div content-id="B39" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="B39" class="ref-content "><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Rambaut</div>
               <div class="given-names">A.</div>
            </div>
             <div class="name">
              <div class="surname">Grass</div>
               <div class="given-names">N.C.</div>
            </div>
          </div>
          <div class="year">1997</div>
            . 
            <div class="article-title">Seq-Gen: an application for the Monte Carlo simulation of DNA sequence evolution along phylogenetic trees</div>
            . 
            <div class="source">Comput. Appl. Biosci.</div>&nbsp;<div class="volume">13</div>
            :
            <div class="fpage">235</div>
            –
            <div class="lpage">238</div>
            .
          <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Seq-Gen%3A%20an%20application%20for%20the%20Monte%20Carlo%20simulation%20of%20DNA%20sequence%20evolution%20along%20phylogenetic%20trees&amp;author=A.%20Rambaut&amp;author=N.C.%20Grass&amp;publication_year=1997&amp;journal=Comput.%20Appl.%20Biosci.&amp;volume=13&amp;pages=235-238" target="">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9183526" class="link link-pub-id" target="">PubMed</a></div>&nbsp;</div></div></div></div></div><div content-id="B40" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="B40" class="ref-content "><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Rannala</div>
               <div class="given-names">B.</div>
            </div>
             <div class="name">
              <div class="surname">Yang</div>
               <div class="given-names">Z.</div>
            </div>
          </div>
          <div class="year">2007</div>
            . 
            <div class="article-title">Inferring speciation times under an episodic molecular clock</div>
            . 
            <div class="source">Syst. Biol.</div>&nbsp;<div class="volume">56</div>
            :
            <div class="fpage">453</div>
            –
            <div class="lpage">466</div>
            .
          <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Inferring%20speciation%20times%20under%20an%20episodic%20molecular%20clock&amp;author=B.%20Rannala&amp;author=Z.%20Yang&amp;publication_year=2007&amp;journal=Syst.%20Biol.&amp;volume=56&amp;pages=453-466" target="">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1080/10635150701420643" target="">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1080%2F10635150701420643" target="">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1080%2F10635150701420643"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17558967" class="link link-pub-id" target="">PubMed</a></div>&nbsp;</div></div></div></div></div><div content-id="B41" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="B41" class="ref-content "><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Rasnitsyn</div>
               <div class="given-names">A.P.</div>
            </div>
          </div>
          <div class="year">1988</div>
            . 
            <div class="article-title">An outline of evolution of the hymenopterous insects (order vespida)</div>
            . 
            <div class="source">Orient. Insects</div>&nbsp;<div class="volume">22</div>
            :
            <div class="fpage">115</div>
            –
            <div class="lpage">145</div>
            .
          <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=An%20outline%20of%20evolution%20of%20the%20hymenopterous%20insects%20%28order%20vespida%29&amp;author=A.P.%20Rasnitsyn&amp;publication_year=1988&amp;journal=Orient.%20Insects&amp;volume=22&amp;pages=115-145" target="">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1080/00305316.1988.11835485" target="">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1080%2F00305316.1988.11835485" target="">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1080%2F00305316.1988.11835485"> </span></div>&nbsp;</div></div></div></div></div><div content-id="B42" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="B42" class="ref-content "><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Rasnitsyn</div>
               <div class="given-names">A.P.</div>
            </div>
          </div>
          <div class="year">2010</div>
            . 
            <div class="article-title">Molecular phylogenetics, morphological cladistics, and fossil record</div>
            . 
            <div class="source">Entomol. Rev.</div>&nbsp;<div class="volume">90</div>
            :
            <div class="fpage">263</div>
            –
            <div class="lpage">298</div>
            .
          <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Molecular%20phylogenetics%2C%20morphological%20cladistics%2C%20and%20fossil%20record&amp;author=A.P.%20Rasnitsyn&amp;publication_year=2010&amp;journal=Entomol.%20Rev.&amp;volume=90&amp;pages=263-298" target="">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1134/S0013873810030012" target="">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1134%2FS0013873810030012" target="">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1134%2FS0013873810030012"> </span></div>&nbsp;</div></div></div></div></div><div content-id="B43" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="B43" class="ref-content "><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Ronquist</div>
               <div class="given-names">F.</div>
            </div>
             <div class="name">
              <div class="surname">Teslenko</div>
               <div class="given-names">M.</div>
            </div>
             <div class="name">
              <div class="surname">van der Mark</div>
               <div class="given-names">P.</div>
            </div>
             <div class="name">
              <div class="surname">Ayres</div>
               <div class="given-names">D.L.</div>
            </div>
             <div class="name">
              <div class="surname">Darling</div>
               <div class="given-names">A.</div>
            </div>
             <div class="name">
              <div class="surname">Höhna</div>
               <div class="given-names">S.</div>
            </div>
             <div class="name">
              <div class="surname">Larget</div>
               <div class="given-names">B.</div>
            </div>
             <div class="name">
              <div class="surname">Liu</div>
               <div class="given-names">L.</div>
            </div>
             <div class="name">
              <div class="surname">Suchard</div>
               <div class="given-names">M.A.</div>
            </div>
             <div class="name">
              <div class="surname">Huelsenbeck</div>
               <div class="given-names">J.P.</div>
            </div>
          </div>
          <div class="year">2012a</div>
            . 
            <div class="article-title">Mrbayes 3.2: Efficient Bayesian phylogenetic inference and model choice across a large model space</div>
            . 
            <div class="source">Syst. Biol.</div>&nbsp;<div class="volume">61</div>
            :
            <div class="fpage">539</div>
            –
            <div class="lpage">542</div>
            .
          <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Mrbayes%203.2%3A%20Efficient%20Bayesian%20phylogenetic%20inference%20and%20model%20choice%20across%20a%20large%20model%20space&amp;author=F.%20Ronquist&amp;author=M.%20Teslenko&amp;author=P.%20van%20der%20Mark&amp;author=D.L.%20Ayres&amp;author=A.%20Darling&amp;author=S.%20H%C3%B6hna&amp;author=B.%20Larget&amp;author=L.%20Liu&amp;author=M.A.%20Suchard&amp;author=J.P.%20Huelsenbeck&amp;publication_year=2012a&amp;journal=Syst.%20Biol.&amp;volume=61&amp;pages=539-542" target="">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/sysbio/sys029" target="">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fsysbio%2Fsys029" target="">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1093%2Fsysbio%2Fsys029"> </span></div>&nbsp;</div></div></div></div></div><div content-id="B44" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="B44" class="ref-content "><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Ronquist</div>
               <div class="given-names">F.</div>
            </div>
             <div class="name">
              <div class="surname">Klopfstein</div>
               <div class="given-names">S.</div>
            </div>
             <div class="name">
              <div class="surname">Vilhelmsen</div>
               <div class="given-names">L.</div>
            </div>
             <div class="name">
              <div class="surname">Schulmeister</div>
               <div class="given-names">S.</div>
            </div>
             <div class="name">
              <div class="surname">Murray</div>
               <div class="given-names">D.L.</div>
            </div>
             <div class="name">
              <div class="surname">Rasnitsyn</div>
               <div class="given-names">A.P.</div>
            </div>
          </div>
          <div class="year">2012b</div>
            . 
            <div class="article-title">A total-evidence approach to dating with fossils, applied to the early radiation of the Hymenoptera</div>
            . 
            <div class="source">Syst. Biol.</div>&nbsp;<div class="volume">61</div>
            :
            <div class="fpage">973</div>
            –
            <div class="lpage">999</div>
            .
          <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20total-evidence%20approach%20to%20dating%20with%20fossils%2C%20applied%20to%20the%20early%20radiation%20of%20the%20Hymenoptera&amp;author=F.%20Ronquist&amp;author=S.%20Klopfstein&amp;author=L.%20Vilhelmsen&amp;author=S.%20Schulmeister&amp;author=D.L.%20Murray&amp;author=A.P.%20Rasnitsyn&amp;publication_year=2012b&amp;journal=Syst.%20Biol.&amp;volume=61&amp;pages=973-999" target="">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/sysbio/sys058" target="">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fsysbio%2Fsys058" target="">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1093%2Fsysbio%2Fsys058"> </span></div>&nbsp;</div></div></div></div></div><div content-id="B45" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="B45" class="ref-content "><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Stadler</div>
               <div class="given-names">T.</div>
            </div>
          </div>
          <div class="year">2009</div>
            . 
            <div class="article-title">On incomplete sampling under birth–death models and connections to the sampling-based coalescent</div>
            . 
            <div class="source">J. Theoret. Biol.</div>&nbsp;<div class="volume">261</div>
            :
            <div class="fpage">58</div>
            –
            <div class="lpage">66</div>
            .
          <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=On%20incomplete%20sampling%20under%20birth%E2%80%93death%20models%20and%20connections%20to%20the%20sampling-based%20coalescent&amp;author=T.%20Stadler&amp;publication_year=2009&amp;journal=J.%20Theoret.%20Biol.&amp;volume=261&amp;pages=58-66" target="">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.jtbi.2009.07.018" target="">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.jtbi.2009.07.018" target="">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1016%2Fj.jtbi.2009.07.018"> </span></div>&nbsp;</div></div></div></div></div><div content-id="B46" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="B46" class="ref-content "><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Stadler</div>
               <div class="given-names">T.</div>
            </div>
          </div>
          <div class="year">2010a</div>
            . 
            <div class="article-title">Sampling-through-time in birth–death trees</div>
            . 
            <div class="source">J. Theoret. Biol.</div>&nbsp;<div class="volume">267</div>
            :
            <div class="fpage">396</div>
            –
            <div class="lpage">404</div>
            .
          <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Sampling-through-time%20in%20birth%E2%80%93death%20trees&amp;author=T.%20Stadler&amp;publication_year=2010a&amp;journal=J.%20Theoret.%20Biol.&amp;volume=267&amp;pages=396-404" target="">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.jtbi.2010.09.010" target="">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.jtbi.2010.09.010" target="">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1016%2Fj.jtbi.2010.09.010"> </span></div>&nbsp;</div></div></div></div></div><div content-id="B47" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="B47" class="ref-content "><div class="citation mixed-citation">
          <div class="comment">Stadler T. 2010b. TreeSim: Simulating trees under the birth–death model. R package version 1.0.</div>
        </div></div></div></div><div content-id="B48" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="B48" class="ref-content "><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Stadler</div>
               <div class="given-names">T.</div>
            </div>
          </div>
          <div class="year">2011</div>
            . 
            <div class="article-title">Mammalian phylogeny reveals recent diversification rate shifts</div>
            . 
            <div class="source">Proc. Natl. Acad. Sci. USA</div>&nbsp;<div class="volume">108</div>
            :
            <div class="fpage">6187</div>
            –
            <div class="lpage">6192</div>
            .
          <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Mammalian%20phylogeny%20reveals%20recent%20diversification%20rate%20shifts&amp;author=T.%20Stadler&amp;publication_year=2011&amp;journal=Proc.%20Natl.%20Acad.%20Sci.%20USA&amp;volume=108&amp;pages=6187-6192" target="">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.1016876108" target="">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.1016876108" target="">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1073%2Fpnas.1016876108"> </span></div>&nbsp;</div></div></div></div></div><div content-id="B49" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="B49" class="ref-content "><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Stadler</div>
               <div class="given-names">T.</div>
            </div>
          </div>
          <div class="year">2012</div>
            . 
            <div class="article-title">How can we improve accuracy of macroevolutionary rate estimates? Syst</div>
            . 
            <div class="source">Biol.</div>&nbsp;<div class="volume">62</div>
            :
            <div class="fpage">321</div>
            .
          </div></div></div></div><div content-id="B50" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="B50" class="ref-content "><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Stadler</div>
               <div class="given-names">T.</div>
            </div>
             <div class="name">
              <div class="surname">Bokma</div>
               <div class="given-names">F.</div>
            </div>
          </div>
          <div class="year">2013</div>
            . 
            <div class="article-title">Estimating speciation and extinction rates for phylogenies of higher taxa</div>
            . 
            <div class="source">Syst. Biol.</div>&nbsp;<div class="volume">62</div>
            :
            <div class="fpage">220</div>
            –
            <div class="lpage">230</div>
            .
          <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Estimating%20speciation%20and%20extinction%20rates%20for%20phylogenies%20of%20higher%20taxa&amp;author=T.%20Stadler&amp;author=F.%20Bokma&amp;publication_year=2013&amp;journal=Syst.%20Biol.&amp;volume=62&amp;pages=220-230" target="">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/sysbio/sys087" target="">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fsysbio%2Fsys087" target="">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1093%2Fsysbio%2Fsys087"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23103589" class="link link-pub-id" target="">PubMed</a></div>&nbsp;</div></div></div></div></div><div content-id="B51" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="B51" class="ref-content "><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Stadler</div>
               <div class="given-names">T.</div>
            </div>
             <div class="name">
              <div class="surname">Kühnert</div>
               <div class="given-names">D.</div>
            </div>
             <div class="name">
              <div class="surname">Bonhoeffer</div>
               <div class="given-names">S.</div>
            </div>
             <div class="name">
              <div class="surname">Drummond</div>
               <div class="given-names">A.J.</div>
            </div>
          </div>
          <div class="year">2013</div>
            . 
            <div class="article-title">Birth–death skyline plot reveals temporal changes of epidemic spread in HIV and hepatitis C virus (HCV)</div>
            . 
            <div class="source">Proc. Natl Acad. Sci. USA</div>&nbsp;<div class="volume">110</div>
            : 
            <div class="fpage">228</div>
            –
            <div class="lpage">233</div>
            .
          <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Birth%E2%80%93death%20skyline%20plot%20reveals%20temporal%20changes%20of%20epidemic%20spread%20in%20HIV%20and%20hepatitis%20C%20virus%20%28HCV%29&amp;author=T.%20Stadler&amp;author=D.%20K%C3%BChnert&amp;author=S.%20Bonhoeffer&amp;author=A.J.%20Drummond&amp;publication_year=2013&amp;journal=Proc.%20Natl%20Acad.%20Sci.%20USA&amp;volume=110&amp;pages=228-233" target="">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.1207965110" target="">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.1207965110" target="">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1073%2Fpnas.1207965110"> </span></div>&nbsp;</div></div></div></div></div><div content-id="B52" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="B52" class="ref-content "><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Tavaré</div>
               <div class="given-names">S.</div>
            </div>
             <div class="name">
              <div class="surname">Marshall</div>
               <div class="given-names">C.R.</div>
            </div>
             <div class="name">
              <div class="surname">Will</div>
               <div class="given-names">O.</div>
            </div>
             <div class="name">
              <div class="surname">Soligo</div>
               <div class="given-names">C.</div>
            </div>
             <div class="name">
              <div class="surname">Martin</div>
               <div class="given-names">R.D.</div>
            </div>
          </div>
          <div class="year">2002</div>
            . 
            <div class="article-title">Using the fossil record to estimate the age of the last common ancestor of extant primates</div>
            . 
            <div class="source">Nature</div>&nbsp;<div class="volume">416</div>
            :
            <div class="fpage">726</div>
            –
            <div class="lpage">729</div>
            .
          <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Using%20the%20fossil%20record%20to%20estimate%20the%20age%20of%20the%20last%20common%20ancestor%20of%20extant%20primates&amp;author=S.%20Tavar%C3%A9&amp;author=C.R.%20Marshall&amp;author=O.%20Will&amp;author=C.%20Soligo&amp;author=R.D.%20Martin&amp;publication_year=2002&amp;journal=Nature&amp;volume=416&amp;pages=726-729" target="">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/416726a" target="">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2F416726a" target="">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1038%2F416726a"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11961552" class="link link-pub-id" target="">PubMed</a></div>&nbsp;</div></div></div></div></div><div content-id="B53" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="B53" class="ref-content "><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Thorne</div>
               <div class="given-names">J.</div>
            </div>
             <div class="name">
              <div class="surname">Kishino</div>
               <div class="given-names">H.</div>
            </div>
          </div>
          <div class="year">2002</div>
            . 
            <div class="article-title">Divergence time and evolutionary rate estimation with multilocus data</div>
            . 
            <div class="source">Syst. Biol.</div>&nbsp;<div class="volume">51</div>
            :
            <div class="fpage">689</div>
            –
            <div class="lpage">702</div>
            .
          <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Divergence%20time%20and%20evolutionary%20rate%20estimation%20with%20multilocus%20data&amp;author=J.%20Thorne&amp;author=H.%20Kishino&amp;publication_year=2002&amp;journal=Syst.%20Biol.&amp;volume=51&amp;pages=689-702" target="">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1080/10635150290102456" target="">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1080%2F10635150290102456" target="">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1080%2F10635150290102456"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12396584" class="link link-pub-id" target="">PubMed</a></div>&nbsp;</div></div></div></div></div><div content-id="B54" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="B54" class="ref-content "><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Thorne</div>
               <div class="given-names">J.</div>
            </div>
             <div class="name">
              <div class="surname">Kishino</div>
               <div class="given-names">H.</div>
            </div>
             <div class="name">
              <div class="surname">Painter</div>
               <div class="given-names">I.S.</div>
            </div>
          </div>
          <div class="year">1998</div>
            . 
            <div class="article-title">Estimating the rate of evolution of the rate of molecular evolution</div>
            . 
            <div class="source">Mol. Biol. Evol.</div>&nbsp;<div class="volume">15</div>
            :
            <div class="fpage">1647</div>
            –
            <div class="lpage">1657</div>
            .
          <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Estimating%20the%20rate%20of%20evolution%20of%20the%20rate%20of%20molecular%20evolution&amp;author=J.%20Thorne&amp;author=H.%20Kishino&amp;author=I.S.%20Painter&amp;publication_year=1998&amp;journal=Mol.%20Biol.%20Evol.&amp;volume=15&amp;pages=1647-1657" target="">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/oxfordjournals.molbev.a025892" target="">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Foxfordjournals.molbev.a025892" target="">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1093%2Foxfordjournals.molbev.a025892"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9866200" class="link link-pub-id" target="">PubMed</a></div>&nbsp;</div></div></div></div></div><div content-id="B55" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="B55" class="ref-content "><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Wilkinson</div>
               <div class="given-names">R.D.</div>
            </div>
             <div class="name">
              <div class="surname">Steiper</div>
               <div class="given-names">M.E.</div>
            </div>
             <div class="name">
              <div class="surname">Soligo</div>
               <div class="given-names">C.</div>
            </div>
             <div class="name">
              <div class="surname">Martin</div>
               <div class="given-names">R.D.</div>
            </div>
             <div class="name">
              <div class="surname">Yang</div>
               <div class="given-names">Z.</div>
            </div>
             <div class="name">
              <div class="surname">Tavaré</div>
               <div class="given-names">S.</div>
            </div>
          </div>
          <div class="year">2011</div>
            . 
            <div class="article-title">Dating primate divergences through an integrated analysis of palaeontological and molecular data</div>
            . 
            <div class="source">Syst. Biol.</div>&nbsp;<div class="volume">60</div>
            :
            <div class="fpage">16</div>
            –
            <div class="lpage">31</div>
            .
          <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Dating%20primate%20divergences%20through%20an%20integrated%20analysis%20of%20palaeontological%20and%20molecular%20data&amp;author=R.D.%20Wilkinson&amp;author=M.E.%20Steiper&amp;author=C.%20Soligo&amp;author=R.D.%20Martin&amp;author=Z.%20Yang&amp;author=S.%20Tavar%C3%A9&amp;publication_year=2011&amp;journal=Syst.%20Biol.&amp;volume=60&amp;pages=16-31" target="">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/sysbio/syq054" target="">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fsysbio%2Fsyq054" target="">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1093%2Fsysbio%2Fsyq054"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21051775" class="link link-pub-id" target="">PubMed</a></div>&nbsp;</div></div></div></div></div><div content-id="B56" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="B56" class="ref-content "><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Wood</div>
               <div class="given-names">H.M.</div>
            </div>
             <div class="name">
              <div class="surname">Matzke</div>
               <div class="given-names">N.J.</div>
            </div>
             <div class="name">
              <div class="surname">Gillespie</div>
               <div class="given-names">R.G.</div>
            </div>
             <div class="name">
              <div class="surname">Griswold</div>
               <div class="given-names">C.E.</div>
            </div>
          </div>
          <div class="year">2013</div>
            . 
            <div class="article-title">Treating fossils as terminal taxa in divergence time estimation reveals ancient vicariance patterns in the palpimanoid spiders</div>
            . 
            <div class="source">Syst. Biol.</div>&nbsp;<div class="volume">62</div>
            :
            <div class="fpage">264</div>
            –
            <div class="lpage">284</div>
            .
          <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Treating%20fossils%20as%20terminal%20taxa%20in%20divergence%20time%20estimation%20reveals%20ancient%20vicariance%20patterns%20in%20the%20palpimanoid%20spiders&amp;author=H.M.%20Wood&amp;author=N.J.%20Matzke&amp;author=R.G.%20Gillespie&amp;author=C.E.%20Griswold&amp;publication_year=2013&amp;journal=Syst.%20Biol.&amp;volume=62&amp;pages=264-284" target="">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/sysbio/sys092" target="">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fsysbio%2Fsys092" target="">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1093%2Fsysbio%2Fsys092"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23192838" class="link link-pub-id" target="">PubMed</a></div>&nbsp;</div></div></div></div></div><div content-id="B57" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="B57" class="ref-content "><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Xie</div>
               <div class="given-names">W.</div>
            </div>
             <div class="name">
              <div class="surname">Lewis</div>
               <div class="given-names">P.O.</div>
            </div>
             <div class="name">
              <div class="surname">Fan</div>
               <div class="given-names">Y.</div>
            </div>
             <div class="name">
              <div class="surname">Kuo</div>
               <div class="given-names">L.</div>
            </div>
             <div class="name">
              <div class="surname">Chen</div>
               <div class="given-names">M.-H.</div>
            </div>
          </div>
          <div class="year">2011</div>
            . 
            <div class="article-title">Improving marginal likelihood estimation for Bayesian phylogenetic model selection</div>
            . 
            <div class="source">Syst. Biol.</div>&nbsp;<div class="volume">60</div>
            :
            <div class="fpage">150</div>
            –
            <div class="lpage">160</div>
            .
          <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Improving%20marginal%20likelihood%20estimation%20for%20Bayesian%20phylogenetic%20model%20selection&amp;author=W.%20Xie&amp;author=P.O.%20Lewis&amp;author=Y.%20Fan&amp;author=L.%20Kuo&amp;author=M.-H.%20Chen&amp;publication_year=2011&amp;journal=Syst.%20Biol.&amp;volume=60&amp;pages=150-160" target="">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/sysbio/syq085" target="">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fsysbio%2Fsyq085" target="">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1093%2Fsysbio%2Fsyq085"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21187451" class="link link-pub-id" target="">PubMed</a></div>&nbsp;</div></div></div></div></div><div content-id="B58" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="B58" class="ref-content "><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Yang</div>
               <div class="given-names">Z.</div>
            </div>
             <div class="name">
              <div class="surname">Rannala</div>
               <div class="given-names">B.</div>
            </div>
          </div>
          <div class="year">1997</div>
            . 
            <div class="article-title">Bayesian phylogenetic inference using DNA sequences: a Markov chain Monte Carlo method</div>
            . 
            <div class="source">Mol. Biol. Evol.</div>&nbsp;<div class="volume">14</div>
            :
            <div class="fpage">717</div>
            –
            <div class="lpage">724</div>
            .
          <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Bayesian%20phylogenetic%20inference%20using%20DNA%20sequences%3A%20a%20Markov%20chain%20Monte%20Carlo%20method&amp;author=Z.%20Yang&amp;author=B.%20Rannala&amp;publication_year=1997&amp;journal=Mol.%20Biol.%20Evol.&amp;volume=14&amp;pages=717-724" target="">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/oxfordjournals.molbev.a025811" target="">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Foxfordjournals.molbev.a025811" target="">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1093%2Foxfordjournals.molbev.a025811"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9214744" class="link link-pub-id" target="">PubMed</a></div>&nbsp;</div></div></div></div></div><div content-id="B59" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="B59" class="ref-content "><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Yang</div>
               <div class="given-names">Z.</div>
            </div>
             <div class="name">
              <div class="surname">Rannala</div>
               <div class="given-names">B.</div>
            </div>
          </div>
          <div class="year">2006</div>
            . 
            <div class="article-title">Bayesian estimation of species divergence times under a molecular clock using multiple fossil calibrations with soft bounds</div>
            . 
            <div class="source">Mol. Biol. Evol.</div>&nbsp;<div class="volume">23</div>
            :
            <div class="fpage">212</div>
            –
            <div class="lpage">226</div>
            .
          <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Bayesian%20estimation%20of%20species%20divergence%20times%20under%20a%20molecular%20clock%20using%20multiple%20fossil%20calibrations%20with%20soft%20bounds&amp;author=Z.%20Yang&amp;author=B.%20Rannala&amp;publication_year=2006&amp;journal=Mol.%20Biol.%20Evol.&amp;volume=23&amp;pages=212-226" target="">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/molbev/msj024" target="">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fmolbev%2Fmsj024" target="">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1093%2Fmolbev%2Fmsj024"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16177230" class="link link-pub-id" target="">PubMed</a></div>&nbsp;</div></div></div></div></div><div content-id="B60" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="B60" class="ref-content "><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Yang</div>
               <div class="given-names">Z.</div>
            </div>
             <div class="name">
              <div class="surname">Yoder</div>
               <div class="given-names">A.D.</div>
            </div>
          </div>
          <div class="year">2003</div>
            . 
            <div class="article-title">Comparison of likelihood and 
Bayesian methods for estimating divergence times using multiple gene 
loci and calibration points, with application to a radiation of 
cute-looking mouse lemur species</div>
            . 
            <div class="source">Syst. Biol.</div>&nbsp;<div class="volume">52</div>
            :
            <div class="fpage">705</div>
            –
            <div class="lpage">716</div>
            .
          <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Comparison%20of%20likelihood%20and%20Bayesian%20methods%20for%20estimating%20divergence%20times%20using%20multiple%20gene%20loci%20and%20calibration%20points%2C%20with%20application%20to%20a%20radiation%20of%20cute-looking%20mouse%20lemur%20species&amp;author=Z.%20Yang&amp;author=A.D.%20Yoder&amp;publication_year=2003&amp;journal=Syst.%20Biol.&amp;volume=52&amp;pages=705-716" target="">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1080/10635150390235557" target="">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1080%2F10635150390235557" target="">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1080%2F10635150390235557"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/14530137" class="link link-pub-id" target="">PubMed</a></div>&nbsp;</div></div></div></div></div><div content-id="B61" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="B61" class="ref-content "><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Yoder</div>
               <div class="given-names">A.D.</div>
            </div>
             <div class="name">
              <div class="surname">Yang</div>
               <div class="given-names">Z.</div>
            </div>
          </div>
          <div class="year">2000</div>
            . 
            <div class="article-title">Estimation of primate speciation dates using local molecular clocks</div>
            . 
            <div class="source">Mol. Biol. Evol.</div>&nbsp;<div class="volume">17</div>
            :
            <div class="fpage">1081</div>
            –
            <div class="lpage">1090</div>
            .
          <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Estimation%20of%20primate%20speciation%20dates%20using%20local%20molecular%20clocks&amp;author=A.D.%20Yoder&amp;author=Z.%20Yang&amp;publication_year=2000&amp;journal=Mol.%20Biol.%20Evol.&amp;volume=17&amp;pages=1081-1090" target="">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/oxfordjournals.molbev.a026389" target="">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Foxfordjournals.molbev.a026389" target="">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1093%2Foxfordjournals.molbev.a026389"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10889221" class="link link-pub-id" target="">PubMed</a></div>&nbsp;</div></div></div></div></div><div content-id="B62" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="B62" class="ref-content "><div class="citation mixed-citation">
          <div class="comment">
              Zhang C. 2015. A tutorial on molecular clock dating using MrBayes 
              <a class="link link-uri" href="https://sites.google.com/site/zhangchicool/" target="">https://sites.google.com/site/zhangchicool/</a>
              , last accessed October 30, 2015.
            </div>
        </div></div></div></div><div content-id="B63" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="B63" class="ref-content "><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Zuckerkandl</div>
               <div class="given-names">E.</div>
            </div>
             <div class="name">
              <div class="surname">Pauling</div>
               <div class="given-names">L.</div>
            </div>
          </div>
          <div class="person-group">
            <div class="name">
              <div class="surname">Bryson</div>
               <div class="given-names">V.</div>
            </div>
             <div class="name">
              <div class="surname">Vogel</div>
               <div class="given-names">H.J.</div>
            </div>
          </div>
          <div class="year">1965</div>
            . 
            <div class="article-title">Evolutionary divergence and convergence in proteins</div>
            . In: 
            <div class="source">Evol. Genes Proteins</div>&nbsp;
            . 
            <div class="publisher-loc">New York</div>
            : 
            <div class="publisher-name">Academic Press</div>
            , p. 
            <div class="fpage">97</div>
            –
            <div class="lpage">166</div>
            .
          <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Evolutionary%20divergence%20and%20convergence%20in%20proteins&amp;author=E.%20Zuckerkandl&amp;author=L.%20Pauling&amp;author=V.%20Bryson&amp;author=H.J.%20Vogel&amp;publication_year=1965&amp;book=Evol.%20Genes%20Proteins" target="">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/B978-1-4832-2734-4.50017-6" target="">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2FB978-1-4832-2734-4.50017-6" target="">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1016%2FB978-1-4832-2734-4.50017-6"> </span></div>&nbsp;</div></div></div></div></div></div>        <!-- /foreach -->



        

        
<div class="copyright copyright-statement">©The Author(s) 2015. Published by Oxford University Press, on behalf of the Society of Systematic Biologists.</div><div class="license"><div class="license-p"> This is an Open Access article distributed under the terms of the Creative Commons Attribution License ( <a class="link link-uri" href="http://creativecommons.org/licenses/by/4.0/" target="">http://creativecommons.org/licenses/by/4.0/</a>
 ), which permits unrestricted reuse, distribution, and reproduction in 
any medium, provided the original work is properly cited. </div></div><!-- /foreach -->

        <span id="UserHasAccess" data-userhasaccess="True"></span>

    </div><!-- /.widget-items -->
</div><!-- /.module-widget -->
    </div>
    <div class="widget widget-SolrResourceMetadata widget-instance-OUP_Article_ResourceMetadata_Widget">
        
        


    </div>
    <div class="widget widget-EditorInformation widget-instance-OUP_Article_EditorInformation_Widget">
        

    </div>


                        <div id="ContentTabFilteredView"></div>
                        <div class="downloadImagesppt" style="display: none;">
                            <a href="https://oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=&amp;PPTtype=SlideSet&amp;ar=2427164&amp;xsltPath=%7E/UI/app/XSLT" id="BodyContent_PageContent_lnkDownloadAllImages">Download all figures</a>
                        </div>
                        <div></div>
                            <div class="widget widget-ArticleDataRepositories widget-instance-Article_DryadLink">
        
    </div>

                         <div class="comments">
                                <div class="widget widget-UserCommentBody widget-instance-UserCommentBody_Article">
        


    </div>
    <div class="widget widget-UserComment widget-instance-OUP_UserComment_Article">
        
    </div>

                        </div>
                    </div>
                   
                </div> 

                     


            </div><!-- /.content-inner-wrap -->
        </div> 

        <div id="Sidebar" class="page-column page-column--right" style="height: auto;">
                <div class="widget widget-AdBlock widget-instance-ArticlePageTopSidebar">
        

<div id="adBlockMainBodyTop" style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyTop'); });
    </script>
<div id="google_ads_iframe_/116097782/sysbio_Behind_Ad2_0__container__" style="border: 0pt none;"><iframe id="google_ads_iframe_/116097782/sysbio_Behind_Ad2_0" title="3rd party ad content" name="google_ads_iframe_/116097782/sysbio_Behind_Ad2_0" scrolling="no" marginwidth="0" marginheight="0" style="border: 0px none; vertical-align: bottom;" srcdoc="" height="250" frameborder="0" width="300"></iframe></div></div>

    </div>
<div class="widget widget-dynamic " data-count="1"> 

    <div class="widget-dynamic-inner-wrap">

<div class="widget widget-dynamic " data-count="10"> 

    <div class="widget-dynamic-inner-wrap">

    <div class="widget widget-ArticleLevelMetrics widget-instance-Article_RightRailB0Article_RightRail_ArticleLevelMetrics">
        





    <div class="artmet-condensed-wrap clearfix">
        <div class="artmet-condensed-stats clearfix">
            <div class="artmet-item artmet-views">
                <span class="artmet-number">1,448</span>
                <span class="artmet-text">Views</span>
            </div>

            <div class="artmet-item artmet-citations">
                <span class="artmet-number">
                     <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000369956400004&amp;DestLinkType=CitingArticles&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="">45</a>
                </span>
                <span class="artmet-text">Citations</span>
            </div>
    
            <div class="artmet-item artmet-altmetric">
    <div class="widget widget-AltmetricLink widget-instance-ArticleLevelMetrics_AltmetricLinkSummary">
            <!-- Altmetrics -->
    <div id="altmetricEmbedId" runat="server" class="altmetric-embed" data-badge-type="donut" data-hide-no-mentions="false" data-doi="10.1093/sysbio/syv080" data-uuid="52a44cfd-f995-5e45-3fd6-d3985793f7ab"><a target="_self" href="https://www.altmetric.com/details.php?domain=academic.oup.com&amp;citation_id=4661783" style="display:inline-block;">
        <img alt="Article has an altmetric score of 19" src="a.png" style="border:0; margin:0; max-width: none;" height="64px" width="64px">
</a></div>
         <script type="text/javascript" src="embed.js"></script>

    </div>

            </div>
        </div>

            <div class="artmet-modal-trigger-wrap clearfix">
                <a class="artmet-modal-trigger" data-article-id="2427164"><i class="icon-metrics"></i><span>View Metrics</span></a>
            </div>
    </div>
        <div class="artmet-modal" id="MetricsModal">
            <div class="artmet-modal-contents">
                <a class="artmet-close-modal">×</a>
            </div>
        </div>


    </div>
    <div class="widget widget-Alerts widget-instance-Article_RightRailB0Article_RightRail_Alerts">
        
    <div class="alertsWidget">
        <h3>Email alerts</h3>
                <div class="userAlert alertType-5">
                    <a role="link" aria-label="New issue alert" data-userloggedin="False" data-alerttype="5">New issue alert</a>
                </div>
                <div class="userAlert alertType-3">
                    <a role="link" aria-label="Advance article alerts" data-userloggedin="False" data-alerttype="3">Advance article alerts</a>
                </div>
                <div class="userAlert alertType-1">
                    <a role="link" aria-label="Article activity alert" data-userloggedin="False" data-alerttype="1">Article activity alert</a>
                </div>
                    <div class="userAlert alertType-MarketingLink">
                <a role="link" aria-label="Receive exclusive offers and updates from Oxford Academic" data-userloggedin="False" data-additionalurl="/my-account/communication-preferences">Receive exclusive offers and updates from Oxford Academic</a>
            </div>
        <div class="userAlertSignUpModal reveal-modal small" data-reveal="">
            <div class="userAlertSignUp"></div>
            <label for="alert-signup-modal-close" class="screenreader-text">Close</label>
            <a role="button" id="alert-signup-modal-close" class="close-reveal-modal" href="javascript:;">
                <i class="icon-general-close"></i>
            </a>
        </div>
    </div>

    </div>
    <div class="widget widget-ArticleLinks widget-instance-Article_RightRailB0Article_RightRail_ArticleLinks">
        


    </div>
    <div class="widget widget-RelatedContent widget-instance-Article_RightRailB0Article_RightRail_RelatedContent">
        
    </div>
    <div class="widget widget-RelatedArticleIn widget-instance-Article_RightRailB0Article_RightRail_RelatedArticleIn">
            <h3 class="relatedArticleIn-title">Related articles in</h3>
    <div class="relatedArticleIn-content">
        <ul>
                <li>
                    <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000369956400004&amp;DestLinkType=RelatedRecords&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a">Web of Science</a>
                </li>
                <li>
                    <a href="https://scholar.google.com/scholar?q=related:https%3a%2f%2facademic.oup.com%2fsysbio%2farticle%2f65%2f2%2f228%2f2427164">Google Scholar</a>
                </li>
        </ul>
    </div>

    </div>
    <div class="widget widget-ArticleCitedBy widget-instance-Article_RightRailB0Article_RightRail_ArticleCitedBy">
        <h3 class="article-cited-title">Citing articles via</h3>
<div class="widget-links_wrap">
        <div class="article-cited-link-wrap web-of-science">
            <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000369956400004&amp;DestLinkType=CitingArticles&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="">Web of Science (45)</a>
        </div>
            <div class="article-cited-link-wrap google-scholar-url">
            <a href="http://scholar.google.com/scholar?q=link:https%3A%2F%2Facademic.oup.com%2Fsysbio%2Farticle%2F65%2F2%2F228%2F2427164" target="">Google Scholar</a>
        </div>
            <div class="article-cited-link-wrap crossref-citedby">
            <a href="https://academic.oup.com/sysbio/CrossRef-CitedBy/2427164">CrossRef</a>
        </div>
</div>
    </div>
    <div class="widget widget-ArticleListNewAndPopular widget-instance-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined">
            <ul class="articleListNewAndPopularCombinedView">
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode active" data-mode="MostRecent">Latest</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostRead">Most Read</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostCited">Most Cited</a>
            </li>
    </ul>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRecent hasContent">




<div id="newPopularList-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined" class="fb">

    







<div class="widget-layout widget-layout--vert">






        <div class="widget-columns widget-col-1">
            <div class="col">

                       <div class="widget-dynamic-entry">
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="https://academic.oup.com/sysbio/advance-article/doi/10.1093/sysbio/syy057/5107025">Detecting the Dependence of Diversification on Multiple Traits from Phylogenetic Trees and Trait Data</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="https://academic.oup.com/sysbio/advance-article/doi/10.1093/sysbio/syy057/5107025">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry">
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="https://academic.oup.com/sysbio/advance-article/doi/10.1093/sysbio/syy061/5104882">Long-branch attraction in species tree estimation: inconsistency of partitioned likelihood and topology-based summary methods</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="https://academic.oup.com/sysbio/advance-article/doi/10.1093/sysbio/syy061/5104882">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry">
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="https://academic.oup.com/sysbio/advance-article/doi/10.1093/sysbio/syy054/5079844">EPA-ng: Massively Parallel Evolutionary Placement of Genetic Sequences</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="https://academic.oup.com/sysbio/advance-article/doi/10.1093/sysbio/syy054/5079844">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry">
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="https://academic.oup.com/sysbio/advance-article/doi/10.1093/sysbio/syy056/5100880">Book Review</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="https://academic.oup.com/sysbio/advance-article/doi/10.1093/sysbio/syy056/5100880">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry">
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="https://academic.oup.com/sysbio/advance-article/doi/10.1093/sysbio/syy064/5100879">Impacts
 Of Inference Method And Dataset Filtering On Phylogenomic Resolution In
 A Rapid Radiation of Ground Squirrels (Xerinae: Marmotini)</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="https://academic.oup.com/sysbio/advance-article/doi/10.1093/sysbio/syy064/5100879">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
            </div>
        </div>
</div><!-- /.widget-layout-[orientation] --></div>
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRead hide">
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostCited hide">
        </section>

    </div>

    </div>

</div>
    </div>

</div>    <div class="widget widget-AdBlock widget-instance-ArticlePageTopMainBodyBottom">
        

<div id="adBlockMainBodyBottom" style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyBottom'); });
    </script>
<div id="google_ads_iframe_/116097782/sysbio_Behind_Ad3_0__container__" style="border: 0pt none;"><iframe id="google_ads_iframe_/116097782/sysbio_Behind_Ad3_0" title="3rd party ad content" name="google_ads_iframe_/116097782/sysbio_Behind_Ad3_0" scrolling="no" marginwidth="0" marginheight="0" style="border: 0px none; vertical-align: bottom;" srcdoc="" height="250" frameborder="0" width="300"></iframe></div></div>

    </div>

        </div> 
    </div><!-- /.page-column-wrap -->
    <input name="ctl00$ctl00$BodyContent$PageContent$hfArticleTitle" id="hfArticleTitle" value="Total-Evidence Dating under the Fossilized Birth–Death Process" type="hidden">

                </div><!-- /.center-inner-row -->
            </section>
        </main>
    </div>
    

        
    
    <script type="text/javascript">
        $(document).ready(function () {
            SCM.ScholarlyiQ.trackArticlePageEvents();
        });
        googletag.cmd.push(function () {
            ////////////////////////////////////// IMPORTANT: //////////////////////////////////////
            // Article DOIs are passed to GPT to target specific articles. Because the DOI cannot //
            // be accessed from Site.Master, put off calling enableServices() until the targeting //
            // for DOI is set. Only call enableServices() if it is not going to be called later   //
            // in Article.aspx. See Site.Master                                                   //
            ////////////////////////////////////////////////////////////////////////////////////////
            googletag.pubads().setTargeting("jnlsdoi", "10.1093/sysbio/syv080");
            googletag.enableServices();
        });
    </script>

        <div class="widget widget-SitePageFooter widget-instance-SitePageFooter">
        <div class="ad-banner ad-banner-footer">
    <div class="widget widget-AdBlock widget-instance-FooterAd">
        

<div id="adBlockFooter" style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockFooter'); });
    </script>
<div id="google_ads_iframe_/116097782/sysbio_Behind_Ad4_0__container__" style="border: 0pt none;"><iframe id="google_ads_iframe_/116097782/sysbio_Behind_Ad4_0" title="3rd party ad content" name="google_ads_iframe_/116097782/sysbio_Behind_Ad4_0" scrolling="no" marginwidth="0" marginheight="0" style="border: 0px none; vertical-align: bottom;" srcdoc="" height="90" frameborder="0" width="728"></iframe></div></div>

    </div>

</div>

    <div class="journal-footer journal-bg">
        <div class="center-inner-row">

                <div class="journal-footer-menu">

                    <ul>
                            <li class="link-1">
                                <a href="https://academic.oup.com/sysbio/pages/About">About Systematic Biology</a>
                            </li>
                            <li class="link-2">
                                <a href="https://academic.oup.com/sysbio/pages/Editorial_Board">Editorial Board</a>
                            </li>
                            <li class="link-3">
                                <a href="https://academic.oup.com/sysbio/pages/General_Instructions">Author Guidelines</a>
                            </li>
                            <li class="link-4">
                                <a href="http://www.systbio.org/">Contact Us</a>
                            </li>
                            <li class="link-5">
                                <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
                            </li>
</ul><ul>                            <li class="link-1">
                                <a href="https://twitter.com/systbiol">Twitter</a>
                            </li>
                            <li class="link-2">
                                <a href="https://academic.oup.com/sysbio/subscribe">Purchase</a>
                            </li>
                            <li class="link-3">
                                <a href="http://www.oxfordjournals.org/en/library-recommendation-form.html">Recommend to your Library</a>
                            </li>
                            <li class="link-4">
                                <a href="http://www.oupmediainfo.com/">Advertising and Corporate Services</a>
                            </li>
                            <li class="link-5">
                                <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org/">Journals Career Network</a>
                            </li>

                    </ul>


                </div><!-- /.journal-footer-menu -->

            <div class="journal-footer-affiliations">
                <!-- <h3>Affiliations</h3> -->
            <a href="" target="">
                <img id="footer-logo-SystematicBiology" class="journal-footer-affiliations-logo" src="sysbio_f1-178201843.svg" alt="Systematic Biology">
            </a>
            </div><!-- /.journal-footer-affiliations -->

            <div class="journal-footer-colophon">
                <ul>
                        <li>Online ISSN 1076-836X</li>
                                            <li>Print ISSN 1063-5157</li>
                    <li>Copyright © 2018 Society of Systematic Biologists</li>
                </ul>
            </div><!-- /.journal-footer-colophon -->


        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-footer -->

    </div>

    <div class="oup-footer">
        <div class="center-inner-row">
                <div class="widget widget-SelfServeContent widget-instance-OupUmbrellaFooterSelfServe">
        
<div class="oup-footer-row journal-links">
<div class="global-footer selfservelinks">
<ul>
    <li>
    <a href="https://academic.oup.com/journals/pages/about_us">About Us</a>
    </li>
    <li>
    <a href="https://academic.oup.com/journals/pages/contact_us">Contact Us</a>
    </li>
    <li>
    <a href="http://global.oup.com/jobs">Careers</a>
    </li>
    <li>
    <a href="https://academic.oup.com/journals/pages/help">Help</a>
    </li>
    <li>
    <a href="https://academic.oup.com/journals/pages/access_purchase">Access &amp; Purchase</a></li>
    <li>
    <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions">Rights &amp; Permissions</a>
    </li>
    <li>
    <a href="https://academic.oup.com/journals/pages/open_access">Open Access</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4 id="global-footer-customer services" class="title footerheader">Connect</h4>
<ul>
    <li>
    <a href="https://academic.oup.com/my-account/communication-preferences">Join Our Mailing List</a>
    </li>
    <li>
    <a href="http://blog.oup.com/">OUPblog</a>
    </li>
    <li>
    <a href="http://twitter.com/oupacademic">Twitter</a>
    </li>
    <li>
    <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
    </li>
    <li>
    <a href="http://www.youtube.com/playlist?list=PL3MAPgqN8JWioLLTkU4mlFM4-SNt-f1Xs">YouTube</a>
    </li>
    <li>
    <a href="http://oupacademic.tumblr.com/">Tumblr</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Resources</h4>
<ul>
    <li><a href="https://academic.oup.com/journals/pages/authors">Authors</a></li>
    <li><a href="https://academic.oup.com/journals/pages/librarians">Librarians</a></li>
    <li><a href="https://academic.oup.com/journals/pages/societies">Societies</a></li>
    <li><a href="https://academic.oup.com/journals/pages/sponsorship_and_advertising">Sponsors &amp; Advertisers</a></li>
    <li><a href="https://academic.oup.com/journals/pages/press">Press &amp; Media</a></li>
    <li><a href="https://academic.oup.com/journals/pages/agents">Agents</a></li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Explore</h4>
<ul>
    <li><a href="http://global.oup.com/academic">Shop OUP Academic</a></li>
    <li><a href="http://www.oxforddictionaries.com/">Oxford Dictionaries</a></li>
    <li><a href="http://oxfordindex.oup.com/">Oxford Index</a></li>
    <li><a href="https://www.epigeum.com/">Epigeum</a></li>
    <li><a href="http://global.oup.com/">OUP Worldwide</a></li>
    <li><a href="http://www.ox.ac.uk/">University of Oxford</a></li>
</ul>
</div>
<div class="OUP-mission">
<p>Oxford University Press is a department of the University of Oxford. 
It furthers the University's objective of excellence in research, 
scholarship, and education by publishing worldwide</p>
<img class="journal-footer-logo" src="oup-logo.svg" alt="Oxford University Press" height="42" width="150">
</div>
</div>
<div class="oup-footer-row">
<div class="oup-footer-row-links">
<ul>
    <li>Copyright © 2018 Oxford University Press</li>
    <li><a href="https://global.oup.com/cookiepolicy/">Cookie Policy</a></li>
    <li><a href="https://global.oup.com/privacy">Privacy Policy</a></li>
    <li><a href="https://academic.oup.com/journals/pages/about_us/legal/notices">Legal Notice</a></li>
    <li><a href="https://academic.oup.com/journals/pages/sitemap">Site Map</a></li>
    <li><a href="https://academic.oup.com/journals/pages/about_us/legal/accessibility">Accessibility</a></li>
    <li><a href="https://get.adobe.com/reader/">Get Adobe Reader</a></li>
</ul>
</div>
</div>
<style type="text/css">
    /* Issue right column fix for tablet/mobile */
    @media (max-width: 1200px) {
    .pg_issue .widget-instance-OUP_Issue {
    width: 100%;
    }
    }
</style>
    
    
    
    </div>

        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-footer -->
    <div class="ss-ui-only">
        
    </div>


        
        <div id="revealModal" class="reveal-modal" data-reveal="">
            <div id="revealContent"></div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        <div id="NeedSubscription" class="reveal-modal small" data-reveal="">
            <div class="subscription-needed">
                <h5>This Feature Is Available To Subscribers Only</h5>
                <p><a href="https://academic.oup.com/sign-in">Sign In</a> or <a href="https://academic.oup.com/my-account/register?siteId=5349&amp;returnUrl=%2fsysbio%2farticle%2f65%2f2%2f228%2f2427164">Create an Account</a></p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        

        <div id="noAccessReveal" class="reveal-modal tiny" data-reveal="">
            <p>This PDF is available to Subscribers Only</p>
            <a class="hide-for-article-page" id="articleLinkToPurchase" data-reveal=""><span>View Article Abstract &amp; Purchase Options</span></a>
            <div class="issue-purchase-modal">
                <p>For full access to this pdf, sign in to an existing account, or purchase an annual subscription.</p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        
        
 
    
<script src="footerJS" type="text/javascript"></script>
<script src="OupCookiePolicyJS" type="text/javascript"></script>
<script src="MathJax.js" type="text/javascript"></script>
<script src="chartist.js" type="text/javascript"></script>
<script src="ArticlePageJS" type="text/javascript"></script>
<script src="addthis_widget.js" type="text/javascript"></script><div style="visibility: hidden; height: 1px; width: 1px; position: absolute; top: -9999px; z-index: 100000;" id="_atssh"><iframe id="_atssh718" title="AddThis utility frame" style="height: 1px; width: 1px; position: absolute; top: 0px; z-index: 100000; border: 0px none; left: 0px;" src="https://s7.addthis.com/static/sh.e4e8af4de595fdb10ec1459d.html#rand=0.151027161559394&amp;iit=1538151309960&amp;tmr=load%3D1538151309922%26core%3D1538151309950%26main%3D1538151309954%26ifr%3D1538151309964&amp;cb=0&amp;cdn=0&amp;md=0&amp;kw=&amp;ab=-&amp;dh=academic.oup.com&amp;dr=&amp;du=https%3A%2F%2Facademic.oup.com%2Fsysbio%2Farticle%2F65%2F2%2F228%2F2427164&amp;href=https%3A%2F%2Facademic.oup.com%2Fsysbio%2Farticle%2F65%2F2%2F228%2F2427164&amp;dt=Total-Evidence%20Dating%20under%20the%20Fossilized%20Birth%E2%80%93Death%20Process%20%7C%20Systematic%20Biology%20%7C%20Oxford%20Academic&amp;dbg=0&amp;cap=tc%3D0%26ab%3D0&amp;inst=1&amp;jsl=1&amp;prod=undefined&amp;lng=en&amp;ogt=site_name%2Cupdated_time%2Cimage%2Curl%2Cdescription%2Ctitle%2Ctype%3Darticle&amp;pc=men&amp;pub=xa-5265518246c10183&amp;ssl=1&amp;sid=5bae538d51fcad4f&amp;srf=0.01&amp;ver=300&amp;xck=0&amp;xtr=0&amp;og=type%3Darticle%26title%3DTotal-Evidence%2520Dating%2520under%2520the%2520Fossilized%2520Birth%25E2%2580%2593Death%2520Process%2520%257C%2520Systematic%2520Biology%2520%257C%2520Oxford%2520Academic%26description%3DAbstract.%2520%2520Bayesian%2520total-evidence%2520dating%2520involves%2520the%2520simultaneous%2520analysis%2520of%2520morphological%2520data%2520from%2520the%2520fossil%2520record%2520and%2520morphological%2520and%2520sequence%2520data%2520fr%26url%3Dhttps%253A%252F%252Facademic.oup.com%252Fsysbio%252Farticle%252F65%252F2%252F228%252F2427164%26image%3Dhttps%253A%252F%252Foup.silverchair-cdn.com%252Foup%252Fbackfile%252FContent_public%252FJournal%252Fsysbio%252FIssue%252F67%252F5%252F2%252Fm_sysbio_67_5cover.png%253FExpires%253D2147483647%2526Signature%253DiMgOj%7EozkXgiMv5ypBZDzosyeLzAPALO2wmmKLje0Ck54FjJm347WeTb3PKZWVrxYH9T2vN47duSgGc4dLVp5tBbPeDOQV99%7EfC0QoiuNqiJsKVtDjUdYShshHjSJhQalWrhtfVODxOV3i%7Eokx0xHvcYKoh0XaVEgY8howRgKsjKDokUzxP8A-IlCyuRe7%7ERCRWcLvUokOnZjMW6dIfPEHfg4NKlHFUTrgzwYmZcyCm1Idbn11KibNLt9Vs51khpe8c8%7EeRqAFQvD8TZZR2syjseNNvaC62SMaVI7aravRbjdk14RleGCLdGvpguCHY1RYMY5C%7EwSqHtyxn-d0GGQA__%2526Key-Pair-Id%253DAPKAIE5G5CRDK6RD3PGA%26updated_time%3D2015-10-22%26site_name%3DOUP%2520Academic&amp;csi=undefined&amp;rev=v8.3.5-wp&amp;ct=1&amp;xld=1&amp;xd=1"></iframe></div><style id="service-icons-0"></style>
<script src="api.js" type="text/javascript"></script>

    
    <!-- Enter Google Analytics or Tag Manager here -->
    
     
 

    <!-- BEGIN: Scholarly iQ Page Tag -->
    <!-- Copyright 2001-2010, IBM Corporation All rights reserved. -->
    <script language="JavaScript" src="ntpagetag.js"></script>
    <noscript>
       <img src="//ouptag.scholarlyiq.com/ntpagetag.gif?js=0" height="1" width="1" border="0" hspace="0" vspace="0" alt="Scholarly IQ">
    </noscript>
    <!-- END: Scholarly iQ Page Tag -->



<div data-reveal="math-jax-load" id="math-jax-modal" class="reveal-modal" style="display: none;"></div><script type="text/javascript" src="syv080.js"></script></body></html>