<!DOCTYPE html>
<html lang="en"><head>
<meta http-equiv="content-type" content="text/html; charset=UTF-8">
    <meta charset="utf-8">
    
    <title>Shannon diversity index: a call to replace the original 
Shannon’s formula with unbiased estimator in the population genetics 
studies [PeerJ]</title>

    
        <link rel="dns-prefetch" href="https://d2pdyyx74uypu5.cloudfront.net/">
    <link rel="dns-prefetch" href="http://static.peerj.com/">
<link rel="dns-prefetch" href="https://doi.org/">

        
            <meta name="citation_title" content="Shannon diversity index: a call to replace the original Shannon’s formula with unbiased estimator in the population genetics studies"><meta name="citation_date" content="2020-06-29"><meta name="citation_doi" content="10.7717/peerj.9391"><meta name="citation_language" content="en"><meta name="citation_pdf_url" content="https://peerj.com/articles/9391.pdf"><meta name="citation_fulltext_html_url" content="https://peerj.com/articles/9391"><meta name="citation_volume" content="8"><meta name="citation_firstpage" content="e9391"><meta name="citation_keywords" content="Genetic diversity; Shannon index; Coalescent simulations; Measures of genetic variation; Sample size effect; Statistical genetics"><meta name="citation_journal_title" content="PeerJ"><meta name="citation_journal_abbrev" content="PeerJ"><meta name="citation_publisher" content="PeerJ Inc."><meta name="citation_issn" content="2167-8359"><meta name="citation_author" content="Maciej K. Konopiński"><meta name="citation_author_email" content="konopinski@iop.krakow.pl">
        <meta name="description" content="Background The Shannon diversity index has been widely used in population genetics studies. Recently, it was proposed as a unifying measure of diversity at different levels—from genes and populations to whole species and ecosystems. The index, however, was proven to be negatively biased at small sample sizes. Modifications to the original Shannon’s formula have been proposed to obtain an unbiased estimator. Methods In this study, the performance of four different estimators of Shannon index—the original Shannon’s formula and those of Zahl, Chao and Shen and Chao et al.—was tested on simulated microsatellite data. Both the simulation and analysis of the results were performed in the R language environment. A new R function was created for the calculation of all four indices from the genind data format. Results Sample size dependence was detected in all the estimators analysed; however, the deviation from parametric values was substantially smaller in the derived measures than in the original Shannon’s formula. Error rate was negatively associated with population heterozygosity. Comparisons among loci showed that fast-mutating loci were less affected by the error, except for the original Shannon’s estimator which, in the smallest sample, was more strongly affected by loci with a higher number of alleles. The Zahl and Chao et al. estimators performed notably better than the original Shannon’s formula. Conclusion The results of this study show that the original Shannon index should no longer be used as a measure of genetic diversity and should be replaced by Zahl’s unbiased estimator.">

        
                                                    <meta property="og:image" content="https://dfzljdn9uc3pi.cloudfront.net/2020/9391/1/fig-5-1x.jpg">
                <meta name="twitter:image" content="https://dfzljdn9uc3pi.cloudfront.net/2020/9391/1/fig-5-1x.jpg">
                    
        <meta name="twitter:card" content="summary_large_image">
        <meta name="twitter:url" content="https://peerj.com/articles/9391">
        <meta name="twitter:site" content="@thePeerJ">
        <meta name="twitter:title" content="Shannon diversity index: a call to replace the original Shannon’s formula with unbiased estimator in the population genetics studies">
        <meta name="twitter:description" content="Background The Shannon diversity index has been widely used in population genetics studies. Recently, it was proposed as a unifying measure of diversity at different levels—from genes and populations to whole species and ecosystems. The index, however, was proven to be negatively biased at small sample sizes. Modifications to the original Shannon’s formula have been proposed to obtain an unbiased estimator. Methods In this study, the performance of four different estimators of Shannon index—the original Shannon’s formula and those of Zahl, Chao and Shen and Chao et al.—was tested on simulated microsatellite data. Both the simulation and analysis of the results were performed in the R language environment. A new R function was created for the calculation of all four indices from the genind data format. Results Sample size dependence was detected in all the estimators analysed; however, the deviation from parametric values was substantially smaller in the derived measures than in the original Shannon’s formula. Error rate was negatively associated with population heterozygosity. Comparisons among loci showed that fast-mutating loci were less affected by the error, except for the original Shannon’s estimator which, in the smallest sample, was more strongly affected by loci with a higher number of alleles. The Zahl and Chao et al. estimators performed notably better than the original Shannon’s formula. Conclusion The results of this study show that the original Shannon index should no longer be used as a measure of genetic diversity and should be replaced by Zahl’s unbiased estimator.">

        <meta property="og:type" content="article">
        <meta property="og:title" content="Shannon diversity index: a call to replace the original Shannon’s formula with unbiased estimator in the population genetics studies">
        <meta property="og:url" content="https://peerj.com/articles/9391">
        <meta property="og:site_name" content="PeerJ">

    
    <link rel="alternate" type="application/pdf" href="https://peerj.com/articles/9391.pdf">
    <link rel="alternate" type="application/rdf+xml" href="https://peerj.com/articles/9391.rdf">
    <link rel="alternate" type="application/json" href="https://peerj.com/articles/9391.json">
    <link rel="alternate" type="application/xml" href="https://peerj.com/articles/9391.xml">
    <link rel="alternate" type="application/unixref+xml" href="https://peerj.com/articles/9391.unixref">
    <link rel="alternate" type="application/vnd.citationstyles.csl+json" href="https://peerj.com/articles/9391.citeproc">
    <link rel="alternate" type="application/bibjson+json" href="https://peerj.com/articles/9391.bibjson">
    <link rel="alternate" type="text/html" href="https://peerj.com/articles/9391.html">

    <link rel="canonical" href="https://peerj.com/articles/9391/">

    <meta name="viewport" content="width=device-width,initial-scale=1">
    <meta property="fb:app_id" content="534542813234464">

                        <link rel="stylesheet" href="05b9c3d-d3e38cb.css" media="screen">

<!--[if lt IE 9]>
    <link rel="stylesheet" href="/assets/css/ie8.css" media="screen">
<![endif]-->

<!--[if lt IE 10]>
    <link rel="stylesheet" href="/assets/css/ie9.css" media="screen">
<![endif]-->

    <style media="screen">html, body { height: 100%; }</style>
                <link rel="stylesheet" href="vue-bundle.css">
    

    <link rel="stylesheet" href="a0c1a2c-04690d8.css" media="screen">

    <link rel="stylesheet" href="be477b9-1134171.css" media="screen">
        <link rel="stylesheet" href="3e4ba6d-c134b5f.css" media="print">
    <script async="" src="analytics.js"></script><script async="" src="analytics.js"></script><script type="text/javascript" async="" src="mixpanel-2-latest.js"></script><script src="36e5d51-2d7025c.js"></script>
<script src="includes.js"></script>
<script src="startsWith.js"></script><!--[if lt IE 9]>
<script src="/assets/js/html5shiv.js"></script>

<![endif]-->

<!--[if lt IE 8]>
<script src="/assets/js/json2.js"></script>
<![endif]-->

<script>
    var PeerJ = {
        Article: {},
        User: {
            anonymous: true        },
        Publication: {},
        Production: {},
        Event: {},
        Com: {},
        Payment: {},
        Annotation: {},
        Search: {},
        Home: {},
        Subjects: {},
        Advocacy: {},
        Job: {},
        ContentAlert: {},
        Tools: {}
    };
</script>

    
<script>
    var campaign_keywords = ['utm_source', 'utm_medium', 'utm_campaign', 'utm_content', 'utm_term'];
    var kw = '';
    var lastUtms = {};
    var firstUtms = {};
    var allUtms = {};

    function campaignParams() {
        var index;
        for (index = 0; index < campaign_keywords.length; ++index) {
            kw = getQueryParam(document.URL, campaign_keywords[index]);
            if (kw.length) {
                lastUtms[campaign_keywords[index] + '-last'] = kw;
                firstUtms[campaign_keywords[index] + '-first'] = kw;
                allUtms[campaign_keywords[index] + '-all'] = kw;
            }
        }
    }

    function updatePreregCookie(preregCookie, firstUtmKey) {
        var utmVal = firstUtms[firstUtmKey];
        if (utmVal) {
            var existingPreregCampaign = $.cookie(preregCookie);
            var appendPreregCampaign;
            if (!existingPreregCampaign) {
                appendPreregCampaign = utmVal;
            } else {
                appendPreregCampaign = existingPreregCampaign + ',' + utmVal;

            }
            $.cookie(preregCookie, appendPreregCampaign, {expires: 365, path: "/"});
        }
    }

    function getQueryParam(url, param) {
        // Expects a raw URL
        param = param.replace(/[[]/, "\[").replace(/[]]/, "\]");
        var regexS = "[\?&]" + param + "=([^&#]*)",
                regex = new RegExp( regexS ),
                results = regex.exec(url);
        if (results === null || (results && typeof(results[1]) !== 'string' && results[1].length)) {
            return '';
        } else {
            return decodeURIComponent(results[1]).replace(/\W/gi, ' ');
        }
    }

    function articlePageEvent() {
        var articleContainer = $('.publication-jsondata');
        if (articleContainer.length) {
            var data = articleContainer.data('publication-meta');

            // Must be public
            if (data.publicationSubjects.length) {

                var eventName = 'Viewed-article';
                var preprint = data.preprint;
                if (preprint) {
                    eventName = 'Viewed-preprint';
                }

                data['ip-hash'] = '8b99d5a580c26a22b089cddd676e62a3';
                mixpanel.track(eventName, data);
            }
        }
    }

    function sectionListViewEvent() {
            }
</script>
    <script>
        // User agrees to terms on signup, so Mixpanel is OK
        // On submit, update mixpanel distinct id
        setTimeout(function () {
            var regmixpanel = document.getElementById('fos_user_registration_form_mixpanelId');
            if (regmixpanel) {
                var distinctId = $.cookie('pj_mp_distinct');
                if (!distinctId) {
                    distinctId = mixpanel.get_distinct_id();
                }
                console.log(distinctId);
                regmixpanel.value = distinctId;
            }
        }, 1500);

        // If logged out then check if consented to analytics cookies (if applicable to country)
        // Run through cookieConsent only
        PeerJ.Com.Mixpanel = new function() {
            this.leadView = function() {
                mixpanel.init('776a79e14e8f05a81ca92536c83f08b4', {
                    'secure_cookie': true,
                    loaded: function (mixpanel) {
                        setTimeout(function () {
                            articlePageEvent();

                            sectionListViewEvent();

                            
                            
                                                    }, 1000);
                    }
                });
            }
        };

        campaignParams();
        updatePreregCookie('pj_prereg_campaign', 'utm_campaign-first');
        updatePreregCookie('pj_prereg_content', 'utm_content-first');
        updatePreregCookie('pj_prereg_term', 'utm_term-first');
    </script>



    <script>(function(p,u,s,h,x){p.pushpad=p.pushpad||function(){(p.pushpad.q=p.pushpad.q||[]).push(arguments)};h=u.getElementsByTagName('head')[0];x=u.createElement('script');x.async=1;x.src=s;h.appendChild(x);})(window,document,'https://pushpad.xyz/pushpad.js');
pushpad('init', 5977, {hostname: 'peerj.com'});
</script><script async="" src="pushpad.js"></script>
    
    <link rel="search" type="application/opensearchdescription+xml" href="https://peerj.com/articles/osd.xml" title="PeerJ">
    


                    
    
    <script>
        // Run through cookieConsent only
        PeerJ.Com.GA = new function() {
            this.disabletracking = function() {
                window['ga-disable-' + 'UA-31208920-1'] = true;
            };

            this.runGA = function() {
                                (function (i, s, o, g, r, a, m) {
                    i['GoogleAnalyticsObject'] = r;
                    i[r] = i[r] || function () {
                        (i[r].q = i[r].q || []).push(arguments)
                    }, i[r].l = 1 * new Date();
                    a = s.createElement(o),
                        m = s.getElementsByTagName(o)[0];
                    a.async = 1;
                    a.src = g;
                    m.parentNode.insertBefore(a, m)
                })(window, document, 'script', 'https://www.google-analytics.com/analytics.js', 'ga');

                                ga('create', 'UA\u002D31208920\u002D1', 'auto');

                // Removes last octet
                ga('set', 'anonymizeIp', true);

                
                                                                
                                
                                
                                
                                
                                
                                                                
                                                ga('set', 'dimension4', ';Biodiversity\u003BEvolutionary\u0020Studies\u003BGenetics\u003BPopulation\u0020Biology;');
                
                                ga('require', 'displayfeatures');

                                ga('send', 'pageview');

                                                window.setTimeout(function () {
                    ga('send', 'event', 'adjusted bounce rate', 'page visit 15 seconds or more');
                }, 15000);
                
                                
                                                                            }
        };
    </script>
    <script src="8548491-f0f5b7c.js"></script>
        
<link rel="apple-touch-icon" sizes="57x57" href="https://d2pdyyx74uypu5.cloudfront.net/images/favicon/peerj/apple-icon-57x57.png">
<link rel="apple-touch-icon" sizes="60x60" href="https://d2pdyyx74uypu5.cloudfront.net/images/favicon/peerj/apple-icon-60x60.png">
<link rel="apple-touch-icon" sizes="72x72" href="https://d2pdyyx74uypu5.cloudfront.net/images/favicon/peerj/apple-icon-72x72.png">
<link rel="apple-touch-icon" sizes="76x76" href="https://d2pdyyx74uypu5.cloudfront.net/images/favicon/peerj/apple-icon-76x76.png">
<link rel="apple-touch-icon" sizes="114x114" href="https://d2pdyyx74uypu5.cloudfront.net/images/favicon/peerj/apple-icon-114x114.png">
<link rel="apple-touch-icon" sizes="120x120" href="https://d2pdyyx74uypu5.cloudfront.net/images/favicon/peerj/apple-icon-120x120.png">
<link rel="apple-touch-icon" sizes="144x144" href="https://d2pdyyx74uypu5.cloudfront.net/images/favicon/peerj/apple-icon-144x144.png">
<link rel="apple-touch-icon" sizes="152x152" href="https://d2pdyyx74uypu5.cloudfront.net/images/favicon/peerj/apple-icon-152x152.png">
<link rel="apple-touch-icon" sizes="180x180" href="https://d2pdyyx74uypu5.cloudfront.net/images/favicon/peerj/apple-icon-180x180.png">
<link rel="icon" type="image/png" sizes="192x192" href="https://d2pdyyx74uypu5.cloudfront.net/images/favicon/peerj/android-icon-192x192.png">
<link rel="shortcut icon" href="https://d2pdyyx74uypu5.cloudfront.net/images/favicon/peerj/favicon.ico">
<link rel="icon" type="image/png" sizes="32x32" href="https://d2pdyyx74uypu5.cloudfront.net/images/favicon/peerj/favicon-32x32.png">
<link rel="icon" type="image/png" sizes="96x96" href="https://d2pdyyx74uypu5.cloudfront.net/images/favicon/peerj/favicon-96x96.png">
<link rel="icon" type="image/png" sizes="16x16" href="https://d2pdyyx74uypu5.cloudfront.net/images/favicon/peerj/favicon-16x16.png">
<link rel="manifest" href="https://d2pdyyx74uypu5.cloudfront.net/images/favicon/peerj/manifest.json">
<meta name="msapplication-TileColor" content="#ffffff">
<meta name="msapplication-TileImage" content="https://d2pdyyx74uypu5.cloudfront.net/images/favicon/peerj/ms-icon-144x144.png">
<meta name="msapplication-config" content="https://d2pdyyx74uypu5.cloudfront.net/images/favicon/peerj/browserconfig.xml">
<meta name="theme-color" content="#ffffff"><link rel="stylesheet" href="badge.css"><link rel="stylesheet" type="text/css" href="trendmd.css" media="all"><style></style><style type="text/css">.MathJax_Hover_Frame {border-radius: .25em; -webkit-border-radius: .25em; -moz-border-radius: .25em; -khtml-border-radius: .25em; box-shadow: 0px 0px 15px #83A; -webkit-box-shadow: 0px 0px 15px #83A; -moz-box-shadow: 0px 0px 15px #83A; -khtml-box-shadow: 0px 0px 15px #83A; border: 1px solid #A6D ! important; display: inline-block; position: absolute}
.MathJax_Menu_Button .MathJax_Hover_Arrow {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 4px; -webkit-border-radius: 4px; -moz-border-radius: 4px; -khtml-border-radius: 4px; font-family: 'Courier New',Courier; font-size: 9px; color: #F0F0F0}
.MathJax_Menu_Button .MathJax_Hover_Arrow span {display: block; background-color: #AAA; border: 1px solid; border-radius: 3px; line-height: 0; padding: 4px}
.MathJax_Hover_Arrow:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_Hover_Arrow:hover span {background-color: #CCC!important}
</style><style type="text/css">#MathJax_About {position: fixed; left: 50%; width: auto; text-align: center; border: 3px outset; padding: 1em 2em; background-color: #DDDDDD; color: black; cursor: default; font-family: message-box; font-size: 120%; font-style: normal; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 15px; -webkit-border-radius: 15px; -moz-border-radius: 15px; -khtml-border-radius: 15px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_About.MathJax_MousePost {outline: none}
.MathJax_Menu {position: absolute; background-color: white; color: black; width: auto; padding: 5px 0px; border: 1px solid #CCCCCC; margin: 0; cursor: default; font: menu; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 5px; -webkit-border-radius: 5px; -moz-border-radius: 5px; -khtml-border-radius: 5px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
.MathJax_MenuItem {padding: 1px 2em; background: transparent}
.MathJax_MenuArrow {position: absolute; right: .5em; padding-top: .25em; color: #666666; font-size: .75em}
.MathJax_MenuActive .MathJax_MenuArrow {color: white}
.MathJax_MenuArrow.RTL {left: .5em; right: auto}
.MathJax_MenuCheck {position: absolute; left: .7em}
.MathJax_MenuCheck.RTL {right: .7em; left: auto}
.MathJax_MenuRadioCheck {position: absolute; left: .7em}
.MathJax_MenuRadioCheck.RTL {right: .7em; left: auto}
.MathJax_MenuLabel {padding: 1px 2em 3px 1.33em; font-style: italic}
.MathJax_MenuRule {border-top: 1px solid #DDDDDD; margin: 4px 3px}
.MathJax_MenuDisabled {color: GrayText}
.MathJax_MenuActive {background-color: #606872; color: white}
.MathJax_MenuDisabled:focus, .MathJax_MenuLabel:focus {background-color: #E8E8E8}
.MathJax_ContextMenu:focus {outline: none}
.MathJax_ContextMenu .MathJax_MenuItem:focus {outline: none}
#MathJax_AboutClose {top: .2em; right: .2em}
.MathJax_Menu .MathJax_MenuClose {top: -10px; left: -10px}
.MathJax_MenuClose {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; font-family: 'Courier New',Courier; font-size: 24px; color: #F0F0F0}
.MathJax_MenuClose span {display: block; background-color: #AAA; border: 1.5px solid; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; line-height: 0; padding: 8px 0 6px}
.MathJax_MenuClose:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_MenuClose:hover span {background-color: #CCC!important}
.MathJax_MenuClose:hover:focus {outline: none}
</style><style type="text/css">.MathJax_Preview .MJXf-math {color: inherit!important}
</style><style type="text/css">.MJX_Assistive_MathML {position: absolute!important; top: 0; left: 0; clip: rect(1px, 1px, 1px, 1px); padding: 1px 0 0 0!important; border: 0!important; height: 1px!important; width: 1px!important; overflow: hidden!important; display: block!important; -webkit-touch-callout: none; -webkit-user-select: none; -khtml-user-select: none; -moz-user-select: none; -ms-user-select: none; user-select: none}
.MJX_Assistive_MathML.MJX_Assistive_MathML_Block {width: 100%!important}
</style><style type="text/css">#MathJax_Zoom {position: absolute; background-color: #F0F0F0; overflow: auto; display: block; z-index: 301; padding: .5em; border: 1px solid black; margin: 0; font-weight: normal; font-style: normal; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; -webkit-box-sizing: content-box; -moz-box-sizing: content-box; box-sizing: content-box; box-shadow: 5px 5px 15px #AAAAAA; -webkit-box-shadow: 5px 5px 15px #AAAAAA; -moz-box-shadow: 5px 5px 15px #AAAAAA; -khtml-box-shadow: 5px 5px 15px #AAAAAA; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_ZoomOverlay {position: absolute; left: 0; top: 0; z-index: 300; display: inline-block; width: 100%; height: 100%; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
#MathJax_ZoomFrame {position: relative; display: inline-block; height: 0; width: 0}
#MathJax_ZoomEventTrap {position: absolute; left: 0; top: 0; z-index: 302; display: inline-block; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
</style><style type="text/css">.MathJax_Preview {color: #888}
#MathJax_Message {position: fixed; left: 1px; bottom: 2px; background-color: #E6E6E6; border: 1px solid #959595; margin: 0px; padding: 2px 8px; z-index: 102; color: black; font-size: 80%; width: auto; white-space: nowrap}
#MathJax_MSIE_Frame {position: absolute; top: 0; left: 0; width: 0px; z-index: 101; border: 0px; margin: 0px; padding: 0px}
.MathJax_Error {color: #CC0000; font-style: italic}
</style><style type="text/css">.MJXp-script {font-size: .8em}
.MJXp-right {-webkit-transform-origin: right; -moz-transform-origin: right; -ms-transform-origin: right; -o-transform-origin: right; transform-origin: right}
.MJXp-bold {font-weight: bold}
.MJXp-italic {font-style: italic}
.MJXp-scr {font-family: MathJax_Script,'Times New Roman',Times,STIXGeneral,serif}
.MJXp-frak {font-family: MathJax_Fraktur,'Times New Roman',Times,STIXGeneral,serif}
.MJXp-sf {font-family: MathJax_SansSerif,'Times New Roman',Times,STIXGeneral,serif}
.MJXp-cal {font-family: MathJax_Caligraphic,'Times New Roman',Times,STIXGeneral,serif}
.MJXp-mono {font-family: MathJax_Typewriter,'Times New Roman',Times,STIXGeneral,serif}
.MJXp-largeop {font-size: 150%}
.MJXp-largeop.MJXp-int {vertical-align: -.2em}
.MJXp-math {display: inline-block; line-height: 1.2; text-indent: 0; font-family: 'Times New Roman',Times,STIXGeneral,serif; white-space: nowrap; border-collapse: collapse}
.MJXp-display {display: block; text-align: center; margin: 1em 0}
.MJXp-math span {display: inline-block}
.MJXp-box {display: block!important; text-align: center}
.MJXp-box:after {content: " "}
.MJXp-rule {display: block!important; margin-top: .1em}
.MJXp-char {display: block!important}
.MJXp-mo {margin: 0 .15em}
.MJXp-mfrac {margin: 0 .125em; vertical-align: .25em}
.MJXp-denom {display: inline-table!important; width: 100%}
.MJXp-denom > * {display: table-row!important}
.MJXp-surd {vertical-align: top}
.MJXp-surd > * {display: block!important}
.MJXp-script-box > *  {display: table!important; height: 50%}
.MJXp-script-box > * > * {display: table-cell!important; vertical-align: top}
.MJXp-script-box > *:last-child > * {vertical-align: bottom}
.MJXp-script-box > * > * > * {display: block!important}
.MJXp-mphantom {visibility: hidden}
.MJXp-munderover {display: inline-table!important}
.MJXp-over {display: inline-block!important; text-align: center}
.MJXp-over > * {display: block!important}
.MJXp-munderover > * {display: table-row!important}
.MJXp-mtable {vertical-align: .25em; margin: 0 .125em}
.MJXp-mtable > * {display: inline-table!important; vertical-align: middle}
.MJXp-mtr {display: table-row!important}
.MJXp-mtd {display: table-cell!important; text-align: center; padding: .5em 0 0 .5em}
.MJXp-mtr > .MJXp-mtd:first-child {padding-left: 0}
.MJXp-mtr:first-child > .MJXp-mtd {padding-top: 0}
.MJXp-mlabeledtr {display: table-row!important}
.MJXp-mlabeledtr > .MJXp-mtd:first-child {padding-left: 0}
.MJXp-mlabeledtr:first-child > .MJXp-mtd {padding-top: 0}
.MJXp-merror {background-color: #FFFF88; color: #CC0000; border: 1px solid #CC0000; padding: 1px 3px; font-style: normal; font-size: 90%}
.MJXp-scale0 {-webkit-transform: scaleX(.0); -moz-transform: scaleX(.0); -ms-transform: scaleX(.0); -o-transform: scaleX(.0); transform: scaleX(.0)}
.MJXp-scale1 {-webkit-transform: scaleX(.1); -moz-transform: scaleX(.1); -ms-transform: scaleX(.1); -o-transform: scaleX(.1); transform: scaleX(.1)}
.MJXp-scale2 {-webkit-transform: scaleX(.2); -moz-transform: scaleX(.2); -ms-transform: scaleX(.2); -o-transform: scaleX(.2); transform: scaleX(.2)}
.MJXp-scale3 {-webkit-transform: scaleX(.3); -moz-transform: scaleX(.3); -ms-transform: scaleX(.3); -o-transform: scaleX(.3); transform: scaleX(.3)}
.MJXp-scale4 {-webkit-transform: scaleX(.4); -moz-transform: scaleX(.4); -ms-transform: scaleX(.4); -o-transform: scaleX(.4); transform: scaleX(.4)}
.MJXp-scale5 {-webkit-transform: scaleX(.5); -moz-transform: scaleX(.5); -ms-transform: scaleX(.5); -o-transform: scaleX(.5); transform: scaleX(.5)}
.MJXp-scale6 {-webkit-transform: scaleX(.6); -moz-transform: scaleX(.6); -ms-transform: scaleX(.6); -o-transform: scaleX(.6); transform: scaleX(.6)}
.MJXp-scale7 {-webkit-transform: scaleX(.7); -moz-transform: scaleX(.7); -ms-transform: scaleX(.7); -o-transform: scaleX(.7); transform: scaleX(.7)}
.MJXp-scale8 {-webkit-transform: scaleX(.8); -moz-transform: scaleX(.8); -ms-transform: scaleX(.8); -o-transform: scaleX(.8); transform: scaleX(.8)}
.MJXp-scale9 {-webkit-transform: scaleX(.9); -moz-transform: scaleX(.9); -ms-transform: scaleX(.9); -o-transform: scaleX(.9); transform: scaleX(.9)}
.MathJax_PHTML .noError {vertical-align: ; font-size: 90%; text-align: left; color: black; padding: 1px 3px; border: 1px solid}
</style><style type="text/css">.MathJax_Display {text-align: center; margin: 1em 0em; position: relative; display: block!important; text-indent: 0; max-width: none; max-height: none; min-width: 0; min-height: 0; width: 100%}
.MathJax .merror {background-color: #FFFF88; color: #CC0000; border: 1px solid #CC0000; padding: 1px 3px; font-style: normal; font-size: 90%}
.MathJax .MJX-monospace {font-family: monospace}
.MathJax .MJX-sans-serif {font-family: sans-serif}
#MathJax_Tooltip {background-color: InfoBackground; color: InfoText; border: 1px solid black; box-shadow: 2px 2px 5px #AAAAAA; -webkit-box-shadow: 2px 2px 5px #AAAAAA; -moz-box-shadow: 2px 2px 5px #AAAAAA; -khtml-box-shadow: 2px 2px 5px #AAAAAA; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true'); padding: 3px 4px; z-index: 401; position: absolute; left: 0; top: 0; width: auto; height: auto; display: none}
.MathJax {display: inline; font-style: normal; font-weight: normal; line-height: normal; font-size: 100%; font-size-adjust: none; text-indent: 0; text-align: left; text-transform: none; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; direction: ltr; max-width: none; max-height: none; min-width: 0; min-height: 0; border: 0; padding: 0; margin: 0}
.MathJax:focus, body :focus .MathJax {display: inline-table}
.MathJax.MathJax_FullWidth {text-align: center; display: table-cell!important; width: 10000em!important}
.MathJax img, .MathJax nobr, .MathJax a {border: 0; padding: 0; margin: 0; max-width: 5000em; max-height: 5000em; min-width: 0; min-height: 0; vertical-align: 0; line-height: normal; text-decoration: none}
img.MathJax_strut {border: 0!important; padding: 0!important; margin: 0!important; vertical-align: 0!important}
.MathJax span {display: inline; position: static; border: 0; padding: 0; margin: 0; vertical-align: 0; line-height: normal; text-decoration: none}
.MathJax nobr {white-space: nowrap!important}
.MathJax img {display: inline!important; float: none!important}
.MathJax * {transition: none; -webkit-transition: none; -moz-transition: none; -ms-transition: none; -o-transition: none}
.MathJax_Processing {visibility: hidden; position: fixed; width: 0; height: 0; overflow: hidden}
.MathJax_Processed {display: none!important}
.MathJax_ExBox {display: block!important; overflow: hidden; width: 1px; height: 60ex; min-height: 0; max-height: none}
.MathJax .MathJax_EmBox {display: block!important; overflow: hidden; width: 1px; height: 60em; min-height: 0; max-height: none}
.MathJax_LineBox {display: table!important}
.MathJax_LineBox span {display: table-cell!important; width: 10000em!important; min-width: 0; max-width: none; padding: 0; border: 0; margin: 0}
.MathJax .MathJax_HitBox {cursor: text; background: white; opacity: 0; filter: alpha(opacity=0)}
.MathJax .MathJax_HitBox * {filter: none; opacity: 1; background: transparent}
#MathJax_Tooltip * {filter: none; opacity: 1; background: transparent}
@font-face {font-family: MathJax_Blank; src: url('about:blank')}
.MathJax .noError {vertical-align: ; font-size: 90%; text-align: left; color: black; padding: 1px 3px; border: 1px solid}
</style></head>

<body class=""><div style="visibility: hidden; overflow: hidden; position: absolute; top: 0px; height: 1px; width: auto; padding: 0px; border: 0px none; margin: 0px; text-align: left; text-indent: 0px; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal;"><div id="MathJax_Hidden"></div></div><div id="MathJax_Message" style="display: none;"></div><div class="cc-revoke cc-bottom cc-animate cc-color-override--1762072787" style="">Cookie Policy</div><div role="dialog" aria-live="polite" aria-label="cookieconsent" aria-describedby="cookieconsent:desc" class="cc-window cc-banner cc-type-opt-in cc-theme-block cc-bottom cc-color-override--1762072787 " style=""><!--googleoff: all--><span id="cookieconsent:desc" class="cc-message">We've
 disabled social media &amp; analytics cookies unless you decide to 
allow them or signup. If you allow analytics cookies then we can use 
anonymous statistics to help us improve our website. <a aria-label="learn more about cookies" tabindex="0" class="cc-link" href="https://peerj.com/about/cookies/" target="_blank">Learn more</a> <div class="cookie-setting-container"><span id="cookie-settings" class="cookie-setting-link">Cookie settings</span> <form id="cookie-form" class="cookie-setting-hide"><div class="cookie-setting-form-title">I will allow:</div><div style="display: inline-block;margin-right: 30px"><input name="social-cookie" id="social-cookie" checked="checked" type="checkbox"> <label for="social-cookie" class="cookie-setting-form-label">social media</label></div><div style="display: inline-block"><input name="analytics-cookie" id="analytics-cookie" checked="checked" type="checkbox"> <label for="analytics-cookie" class="cookie-setting-form-label">anonymous statistics</label></div></form></div></span><div class="cc-compliance cc-highlight"><a aria-label="dismiss cookie message" role="button" tabindex="0" class="cc-btn cc-dismiss">✖</a><a aria-label="allow cookies" role="button" tabindex="0" class="cc-btn cc-allow">Allow cookies</a></div><!--googleon: all--></div>
    
                <!-- FreshDesk variable (TODO: move elsewhere) -->


<nav class="navbar navbar-fixed-top navbar-inverse navbar-alpha visible" role="navigation"><div class="navbar-inner"><!-- .btn-navbar is used as the toggle for collapsed navbar content --><a class="btn btn-navbar pull-right" data-toggle="collapse" data-target=".nav-collapse"><span class="icon-bar"></span><span class="icon-bar"></span><span class="icon-bar"></span></a><!-- logo --><ul class="nav pull-left nav-sections nav-journal"><li class="dropdown"><a href="https://peerj.com/" class="dropdown-toggle   active" data-toggle="dropdown"><span id="navJournalTitle">PeerJ – Life &amp; Environment</span><b class="caret"></b></a><ul class="dropdown-menu journal-list"><li><a href="https://peerj.com/">PeerJ Publishing Overview</a></li><li class="dropdown-submenu hidden-phone"><a tabindex="-1" href="#">PeerJ – the Journal of Life &amp; Environmental Sciences</a><ul class="dropdown-menu"><li><a href="https://peerj.com/sections/">About the journal Sections</a></li><li class="divider"></li><li><a href="https://peerj.com/sections/aquatic-biology/">Aquatic Biology</a></li><li><a href="https://peerj.com/sections/biochemistry-biophysics-molecular-biology/">Biochemistry, Biophysics and Molecular Biology</a></li><li><a href="https://peerj.com/sections/biodiversity-conservation/">Biodiversity and Conservation</a></li><li><a href="https://peerj.com/sections/bioinformatics-genomics/">Bioinformatics and Genomics</a></li><li><a href="https://peerj.com/sections/brain-cognition/">Brain and Cognition</a></li><li><a href="https://peerj.com/sections/ecology/">Ecology</a></li><li><a href="https://peerj.com/sections/environ-sci/">Environmental Science</a></li><li><a href="https://peerj.com/sections/microbiology/">Microbiology</a></li><li><a href="https://peerj.com/sections/paleontology-evolutionary-science/">Paleontology and Evolutionary Science</a></li><li><a href="https://peerj.com/sections/plant-biology/">Plant Biology</a></li><li><a href="https://peerj.com/sections/zoological-science/">Zoological Science</a></li></ul></li><li><a href="https://peerj.com/computer-science/">
                                    PeerJ Computer Science
                                </a></li><li><a href="https://peerj.com/chemistry/">
                                    PeerJ Physical Chemistry
                                </a></li><li><a href="https://peerj.com/chemistry/">
                                    PeerJ Organic Chemistry
                                </a></li><li><a href="https://peerj.com/chemistry/">
                                    PeerJ Inorganic Chemistry
                                </a></li><li><a href="https://peerj.com/chemistry/">
                                    PeerJ Analytical Chemistry
                                </a></li><li><a href="https://peerj.com/chemistry/">
                                    PeerJ Materials Science
                                </a></li><li class="divider"></li><li><a href="https://peerj.org/" target="_blank">Visit PeerJ.org and get involved</a></li></ul></li></ul><!-- mobile-only top nav items --><ul class="nav pull-left nav-about-phone hidden-desktop"><li class="dropdown"><a tabindex="-1" href="#" class="dropdown-toggle" data-toggle="dropdown">About <b class="caret"></b></a><ul class="dropdown-menu"><li id="about-overview"><a href="https://peerj.com/benefits/">PeerJ Journals Overview</a></li><li id="about-faq"><a href="https://peerj.com/about/FAQ/">PeerJ Journals FAQ</a></li><li id="about-what-publish"><a href="https://peerj.com/about/publications/">What we publish</a></li><li id="8yrs-publishing"><a href="https://peerj.com/benefits/peerj-timeline/">8 Years publishing</a></li><li class="divider"></li><li role="presentation" class="dropdown-header">Solutions for authors</li><li id="about-reputation"><a href="https://peerj.com/benefits/reputation/">Reputation</a></li><li id="about-peer-review"><a href="https://peerj.com/benefits/peer-review-timeline/">High quality peer review</a></li><li id="about-speed"><a href="https://peerj.com/benefits/fast-publishing/">Fast publishing</a></li><li id="about-impact"><a href="https://peerj.com/benefits/indexing-and-impact-factor/">Indexing and Impact Factor</a></li><li id="about-readership"><a href="https://peerj.com/benefits/broad-audience/">Global readership</a></li><li id="about-features"><a href="https://peerj.com/benefits/peerj-feature-comparison/">Feature comparison</a></li><li id="about-cost"><a href="https://peerj.com/benefits/reduced-cost-publishing/">Reduced cost publishing</a></li><li id="about-feedback"><a href="https://peerj.com/benefits/feedback/">Author feedback</a></li><li id="about-ecr-benefits"><a href="https://peerj.com/benefits/early-career-researchers/">Early career researcher benefits</a></li><li id="about-senior-researcher-benefits"><a href="https://peerj.com/benefits/senior-researchers/">Senior researcher benefits</a></li><li id="about-open-review"><a href="https://peerj.com/benefits/review-history-and-peer-review/">Open review (optional)</a></li><li id="about-rebuttal"><a href="https://peerj.com/benefits/academic-rebuttal-letters/">Rebuttal letters</a></li></ul></li><li><!-- checkout items --></li><li><!-- notifications --></li></ul><!-- sections --><ul class="nav pull-left nav-collapse nav-sections nav-sections-main collapse search-hide"><li class="dropdown visible-desktop"><a tabindex="-1" href="#" class="dropdown-toggle" data-toggle="dropdown">About <b class="caret"></b></a><ul class="dropdown-menu"><li id="about-overview"><a href="https://peerj.com/benefits/">PeerJ Journals Overview</a></li><li id="about-faq"><a href="https://peerj.com/about/FAQ/">PeerJ Journals FAQ</a></li><li id="about-what-publish"><a href="https://peerj.com/about/publications/">What we publish</a></li><li id="8yrs-publishing"><a href="https://peerj.com/benefits/peerj-timeline/">8 Years publishing</a></li><li class="divider"></li><li role="presentation" class="dropdown-header">Solutions for authors</li><li id="about-reputation"><a href="https://peerj.com/benefits/reputation/">Reputation</a></li><li id="about-peer-review"><a href="https://peerj.com/benefits/peer-review-timeline/">High quality peer review</a></li><li id="about-speed"><a href="https://peerj.com/benefits/fast-publishing/">Fast publishing</a></li><li id="about-impact"><a href="https://peerj.com/benefits/indexing-and-impact-factor/">Indexing and Impact Factor</a></li><li id="about-readership"><a href="https://peerj.com/benefits/broad-audience/">Global readership</a></li><li id="about-features"><a href="https://peerj.com/benefits/peerj-feature-comparison/">Feature comparison</a></li><li id="about-cost"><a href="https://peerj.com/benefits/reduced-cost-publishing/">Reduced cost publishing</a></li><li id="about-feedback"><a href="https://peerj.com/benefits/feedback/">Author feedback</a></li><li id="about-ecr-benefits"><a href="https://peerj.com/benefits/early-career-researchers/">Early career researcher benefits</a></li><li id="about-senior-researcher-benefits"><a href="https://peerj.com/benefits/senior-researchers/">Senior researcher benefits</a></li><li id="about-open-review"><a href="https://peerj.com/benefits/review-history-and-peer-review/">Open review (optional)</a></li><li id="about-rebuttal"><a href="https://peerj.com/benefits/academic-rebuttal-letters/">Rebuttal letters</a></li></ul></li><!-- more --><li class="dropdown"><a href="#" class="dropdown-toggle" data-toggle="dropdown">More <b class="caret"></b></a><ul class="dropdown-menu" role="menu" aria-labelledby="dLabel"><li><a href="https://peerj.com/expertrxiv/"><img src="expertrxiv.png" style="width: 80px"></a></li><li><a href="https://peerj.com/subjects/">Subjects</a></li><li class="dropdown-submenu hidden-phone"><a tabindex="-1" href="#">Search articles</a><ul class="dropdown-menu"><li role="presentation" class="dropdown-header">Peer-reviewed Journals</li><li><a tabindex="-1" href="https://peerj.com/articles/?journal=peerj">PeerJ (Life, Biological, Environmental and Health Sciences)</a></li><li><a tabindex="-1" href="https://peerj.com/articles/?journal=cs">PeerJ Computer Science</a></li><li><a tabindex="-1" href="https://peerj.com/articles/?journal=pchem">PeerJ Physical Chemistry</a></li><li><a tabindex="-1" href="https://peerj.com/articles/?journal=ochem">PeerJ Organic Chemistry</a></li><li><a tabindex="-1" href="https://peerj.com/articles/?journal=ichem">PeerJ Inorganic Chemistry</a></li><li><a tabindex="-1" href="https://peerj.com/articles/?journal=achem">PeerJ Analytical Chemistry</a></li><li><a tabindex="-1" href="https://peerj.com/articles/?journal=matsci">PeerJ Materials Science</a></li><li role="presentation" class="dropdown-header">Preprints</li><li><a tabindex="-1" href="https://peerj.com/preprints/">PeerJ Preprints</a></li></ul></li><li class="dropdown-submenu hidden-phone"><a tabindex="-1" href="#">Table of contents</a><ul class="dropdown-menu"><li role="presentation" class="dropdown-header">Table of Contents - current and archives</li><li><a tabindex="-1" href="https://peerj.com/medicine/">PeerJ - Medicine articles</a></li><li><a tabindex="-1" href="https://peerj.com/biology/">PeerJ - Biology &amp; Life science articles</a></li><li><a tabindex="-1" href="https://peerj.com/environment/">PeerJ - Environmental Science articles</a></li><li><a tabindex="-1" href="https://peerj.com/general/">PeerJ - General bio (stats, legal, policy, edu)</a></li><li class="divider"></li><li><a tabindex="-1" href="https://peerj.com/cs/">PeerJ Computer Science</a></li><li class="divider"></li><li><a tabindex="-1" href="https://peerj.com/preprints-toc/">PeerJ Preprints</a></li></ul></li><li><a href="https://peerj.com/academic-boards/advisors/">Academic advisors</a></li><li><a href="https://peerj.com/reviewer-match/">Volunteer to review</a></li><li><a href="https://peerj.com/collections/">Collections</a></li><li><a href="https://peerj.com/questions/">Discussions</a></li><li><a href="https://peerj.com/blog/">Blog</a></li><li><a href="https://peerj.com/prepaid-publishing/">Prepaid Publishing</a></li><li><a href="https://peerj.com/about/reviews/">Reviews and awards</a></li><li><a href="https://peerj.com/spread-the-word/">Spread the word</a></li><li><a href="https://peerj.com/about/">Who are we?</a></li><li><a href="https://peerj.com/about/contact/">Contact</a></li></ul></li></ul><!-- search --><div class="nav nav-collapse collapse pull-right nav-search"><form class="navbar-search" action="/search/"><input name="q" data-autocomplete-url="/search/" class="search-query ui-autocomplete-input" placeholder="Search" autocomplete="off" type="search"><!--<i class="icon-search"></i>--></form></div><ul class="nav pull-right nav-collapse collapse search-hide nav-utilities"><!-- login desktop --><li><a id="front-page-login" href="https://peerj.com/login">Login</a></li></ul><ul class="nav pull-right search-hide nav-shifter" style="width: 535.5px;"></ul><!-- for authors, my manuscripts --><ul class="nav nav-center nav-collapse collapse search-hide pull-right" style="visibility: visible;"><!-- for authors --><li class="dropdown nav-authors"><a href="#" class="dropdown-toggle" data-toggle="dropdown"><i class="icon-info4 icon-large nav-icon icomoon"></i><span class="visible-wide">AUTHORS</span><b class="caret"></b></a><ul class="dropdown-menu"><li><a href="https://peerj.com/benefits/">Peer Journals Overview</a></li><li><a href="https://peerj.com/about/author-instructions/">Submission Guidelines</a></li><li><a href="https://peerj.com/subjects/">Subject Areas</a></li><li><a href="https://peerj.com/academic-boards/">Editorial Board</a></li><li><a href="https://peerj.com/about/editorial-criteria/">Editorial Criteria</a></li><li><a href="https://peerj.com/pricing/">Pricing</a></li><li><a href="https://peerj.com/about/FAQ/">General FAQ</a></li><li><a href="https://peerj.com/computer-science/faq-cs/">Computer Science FAQ</a></li><li><a href="https://peerj.com/about/aims-and-scope/">Aims and Scope</a></li><li><a href="https://peerj.com/about/author-interviews/">Author Interviews</a></li><li><a href="https://peerj.com/about/policies-and-procedures/">Policies and Procedures</a></li><!--<li><a href="#">Why PeerJ?</a></li>--></ul></li><!-- my manuscripts --><!-- note: dropdown classes used just to maintain display --><li class="nav-manuscripts dropdown"><a href="https://peerj.com/new/" class="dropdown-toggle"><span>SUBMIT ARTICLE</span></a></li></ul></div></nav>
    
        <div class="item-top-navbar headroom headroom--top">
    <div class="item-top-navbar-inner">
        <div class="container-fluid">
            <div class="row-fluid">
                <div class="span12">
                       <div class="item-metrics-counts-top-nav article-item-metrics-counts">
                                                                    <span class="article-item-metrics-count" style="display: inline;">
                                        <span data-count="visitors">813</span>
                                        <span class="article-item-metrics-label">Visitors</span>
                                    </span>
                                
                            <span class="article-item-metrics-count" style="display: inline;">
                                <span data-count="views-html">1,040</span>
                                <span class="article-item-metrics-label">Views</span>
                            </span>

                            <span class="article-item-metrics-count" style="display: inline;">
                                <span data-count="views-pdf">252</span>
                                <span class="article-item-metrics-label">Downloads</span>
                            </span>
                       </div>
                </div>
            </div>
        </div>
    </div>
</div>

    <div id="wrap">

        

        <div id="nav-pad"></div>

        
        <div class="container">

            <noscript class="js-disabled-warning">
                <div class="alert alert-danger">
                    <i class="icon icon-warning-sign"></i> Javascript is disabled in your browser. Please <a href="https://www.enable-javascript.com" target="_blank">enable Javascript</a> to view PeerJ.
                </div>
            </noscript>

            
    <div class="row publication-jsondata" data-publication-meta="{&quot;publicationId&quot;:&quot;9391&quot;,&quot;Article-section&quot;:&quot;Bioinformatics and Genomics&quot;,&quot;journal&quot;:&quot;PeerJ&quot;,&quot;published&quot;:&quot;2020-06-29 08:39:13&quot;,&quot;preprint&quot;:false,&quot;publicationSubjects&quot;:[&quot;Biodiversity&quot;,&quot;Evolutionary Studies&quot;,&quot;Genetics&quot;,&quot;Population Biology&quot;],&quot;publicationInstitutions&quot;:null,&quot;publicationTop20Institution&quot;:false,&quot;publicationInstitutionPlan&quot;:false}">
        <!-- Left sidebar -->
        <div class="span1 article-sidebar">
            <div class="article-sidebar-left">
                <div class="sidebar-box sidebar-box--journal">
                    <a href="https://peerj.com/" class="sidebar-box--journal-mask"></a>
                    <img src="article-logo-peerj.png">
                </div>

                                    <div id="btn-view-tweets" class="sidebar-box sidebar-box--tweet">
                        <div class="text-center">View 8 tweets <i class="icon-twitter"></i></div>
                    </div>
                
                <a href="#related-research" class="sidebar-box sidebar-box--related text-center">
                    Related research
                    <i class="icon-angle-down"></i>
                </a>

                <!-- mobile only -->
                <div class="item-leftside-actions">
                     <div class="sidebar-box sidebar-box--action js-download-modal-trigger">Download</div>

                     <div id="notification-actions-mobile" class="sidebar-box sidebar-box--action" data-href="/following/publication/9696/">
    <span class="follow-btn  " id="item-left-follow-btn" title="" data-toggle="tooltip" data-success-modal="#followModal" data-href="/follow/publication/9696/0/" data-original-title="Receive article updates">
    <span class="button_text_follow">Follow</span></span>
</div>

                     
  
                        <div class="sidebar-box sidebar-box--social visible-desktop">
    <div class="sidebar-box--social-title">Share</div>
    <div class="d-flex">
        <a class="pj-socialism tw-soc" href="http://twitter.com/share?url=https%3A%2F%2Fpeerj.com%2Farticles%2F9391%2F&amp;via=thePeerJ&amp;text=Shannon%20diversity%20index%20estimators&amp;related=" target="_blank" onclick="window.open(this.href, 'popupwindow', 'width=500,height=500,scrollbars,resizable'); return false;">Twitter</a>
        <a class="pj-socialism fb-soc" href="http://www.facebook.com/sharer.php?u=https%3A%2F%2Fpeerj.com%2Farticles%2F9391%2F" target="_blank" onclick="window.open(this.href, 'popupwindow', 'width=500,height=500,scrollbars,resizable'); return false;">Facebook</a>
        <a class="pj-socialism em-soc" href="mailto:?Subject=Relevant%20research%20paper%20in%20PeerJ&amp;Body=Shannon%20diversity%20index%3A%20a%20call%20to%20replace%20the%20original%20Shannon%E2%80%99s%20formula%20with%20unbiased%20estimator%20in%20the%20population%20genetics%20studies%20https%3A%2F%2Fpeerj.com%2Farticles%2F9391%2F" target="_blank" onclick="window.open(this.href, 'popupwindow', 'width=500,height=500,scrollbars,resizable'); return false;">Email</a>
    </div>
</div>

<div class="btn-group sidebar-box sidebar-box--action">
    <a href="#" class="btn-share dropdown-toggle" data-toggle="dropdown">Share</a>

    <ul class="dropdown-menu">
        <li>
            <a href="http://twitter.com/share?url=https%3A%2F%2Fpeerj.com%2Farticles%2F9391%2F&amp;via=thePeerJ&amp;text=Shannon%20diversity%20index%20estimators&amp;related=" target="_blank" onclick="window.open(this.href, 'popupwindow', 'width=500,height=500,scrollbars,resizable'); return false;">Twitter</a>
        </li>
        <li>
            <a href="http://www.facebook.com/sharer.php?u=https%3A%2F%2Fpeerj.com%2Farticles%2F9391%2F" target="_blank" onclick="window.open(this.href, 'popupwindow', 'width=500,height=500,scrollbars,resizable'); return false;">Facebook</a>
        </li>
        <li>
            <a href="mailto:?Subject=Relevant%20research%20paper%20in%20PeerJ&amp;Body=Shannon%20diversity%20index%3A%20a%20call%20to%20replace%20the%20original%20Shannon%E2%80%99s%20formula%20with%20unbiased%20estimator%20in%20the%20population%20genetics%20studies%20https%3A%2F%2Fpeerj.com%2Farticles%2F9391%2F" target="_blank" onclick="window.open(this.href, 'popupwindow', 'width=500,height=500,scrollbars,resizable'); return false;">Email</a>
        </li>
    </ul>
</div>

                                    </div>

            </div>

            <div class="peer-reviewed visible-phone">
                <i class="icon-ok"></i> PEER-REVIEWED
            </div>

        </div>

        <div id="annotations-sidebar" class="span5"><div class="annotations annotations-positioned" data-selector="#p-1"><a href="https://peerj.com/questions.form?format=html&amp;target=articles/9391&amp;_counts=1&amp;selector=%23p-1" class="btn btn-primary add-annotation" data-toggle="annotation-form" rel="nofollow">Ask a question about this section</a><button class="btn btn-default close-annotations">Close</button><div class="help-block annotation-learn-more"><a href="https://peerj.com/about/FAQ/academic-contribution/" target="_blank">Learn more about Q&amp;A</a></div><div class="annotation-form-container" style="display: block;"></div></div><div class="annotations annotations-positioned" data-selector="#p-2"><a href="https://peerj.com/questions.form?format=html&amp;target=articles/9391&amp;_counts=1&amp;selector=%23p-2" class="btn btn-primary add-annotation" data-toggle="annotation-form" rel="nofollow">Ask a question about this section</a><button class="btn btn-default close-annotations">Close</button><div class="help-block annotation-learn-more"><a href="https://peerj.com/about/FAQ/academic-contribution/" target="_blank">Learn more about Q&amp;A</a></div><div class="annotation-form-container" style="display: block;"></div></div><div class="annotations annotations-positioned" data-selector="#p-3"><a href="https://peerj.com/questions.form?format=html&amp;target=articles/9391&amp;_counts=1&amp;selector=%23p-3" class="btn btn-primary add-annotation" data-toggle="annotation-form" rel="nofollow">Ask a question about this section</a><button class="btn btn-default close-annotations">Close</button><div class="help-block annotation-learn-more"><a href="https://peerj.com/about/FAQ/academic-contribution/" target="_blank">Learn more about Q&amp;A</a></div><div class="annotation-form-container" style="display: block;"></div></div><div class="annotations annotations-positioned" data-selector="#p-4"><a href="https://peerj.com/questions.form?format=html&amp;target=articles/9391&amp;_counts=1&amp;selector=%23p-4" class="btn btn-primary add-annotation" data-toggle="annotation-form" rel="nofollow">Ask a question about this section</a><button class="btn btn-default close-annotations">Close</button><div class="help-block annotation-learn-more"><a href="https://peerj.com/about/FAQ/academic-contribution/" target="_blank">Learn more about Q&amp;A</a></div><div class="annotation-form-container" style="display: block;"></div></div><div class="annotations annotations-positioned" data-selector="#p-5"><a href="https://peerj.com/questions.form?format=html&amp;target=articles/9391&amp;_counts=1&amp;selector=%23p-5" class="btn btn-primary add-annotation" data-toggle="annotation-form" rel="nofollow">Ask a question about this section</a><button class="btn btn-default close-annotations">Close</button><div class="help-block annotation-learn-more"><a href="https://peerj.com/about/FAQ/academic-contribution/" target="_blank">Learn more about Q&amp;A</a></div><div class="annotation-form-container" style="display: block;"></div></div><div class="annotations annotations-positioned" data-selector="#p-10"><a href="https://peerj.com/questions.form?format=html&amp;target=articles/9391&amp;_counts=1&amp;selector=%23p-10" class="btn btn-primary add-annotation" data-toggle="annotation-form" rel="nofollow">Ask a question about this section</a><button class="btn btn-default close-annotations">Close</button><div class="help-block annotation-learn-more"><a href="https://peerj.com/about/FAQ/academic-contribution/" target="_blank">Learn more about Q&amp;A</a></div><div class="annotation-form-container" style="display: block;"></div></div><div class="annotations annotations-positioned" data-selector="#p-11"><a href="https://peerj.com/questions.form?format=html&amp;target=articles/9391&amp;_counts=1&amp;selector=%23p-11" class="btn btn-primary add-annotation" data-toggle="annotation-form" rel="nofollow">Ask a question about this section</a><button class="btn btn-default close-annotations">Close</button><div class="help-block annotation-learn-more"><a href="https://peerj.com/about/FAQ/academic-contribution/" target="_blank">Learn more about Q&amp;A</a></div><div class="annotation-form-container" style="display: block;"></div></div><div class="annotations annotations-positioned" data-selector="#p-12"><a href="https://peerj.com/questions.form?format=html&amp;target=articles/9391&amp;_counts=1&amp;selector=%23p-12" class="btn btn-primary add-annotation" data-toggle="annotation-form" rel="nofollow">Ask a question about this section</a><button class="btn btn-default close-annotations">Close</button><div class="help-block annotation-learn-more"><a href="https://peerj.com/about/FAQ/academic-contribution/" target="_blank">Learn more about Q&amp;A</a></div><div class="annotation-form-container" style="display: block;"></div></div><div class="annotations annotations-positioned" data-selector="#p-17"><a href="https://peerj.com/questions.form?format=html&amp;target=articles/9391&amp;_counts=1&amp;selector=%23p-17" class="btn btn-primary add-annotation" data-toggle="annotation-form" rel="nofollow">Ask a question about this section</a><button class="btn btn-default close-annotations">Close</button><div class="help-block annotation-learn-more"><a href="https://peerj.com/about/FAQ/academic-contribution/" target="_blank">Learn more about Q&amp;A</a></div><div class="annotation-form-container" style="display: block;"></div></div><div class="annotations annotations-positioned" data-selector="#p-18"><a href="https://peerj.com/questions.form?format=html&amp;target=articles/9391&amp;_counts=1&amp;selector=%23p-18" class="btn btn-primary add-annotation" data-toggle="annotation-form" rel="nofollow">Ask a question about this section</a><button class="btn btn-default close-annotations">Close</button><div class="help-block annotation-learn-more"><a href="https://peerj.com/about/FAQ/academic-contribution/" target="_blank">Learn more about Q&amp;A</a></div><div class="annotation-form-container" style="display: block;"></div></div><div class="annotations annotations-positioned" data-selector="#p-19"><a href="https://peerj.com/questions.form?format=html&amp;target=articles/9391&amp;_counts=1&amp;selector=%23p-19" class="btn btn-primary add-annotation" data-toggle="annotation-form" rel="nofollow">Ask a question about this section</a><button class="btn btn-default close-annotations">Close</button><div class="help-block annotation-learn-more"><a href="https://peerj.com/about/FAQ/academic-contribution/" target="_blank">Learn more about Q&amp;A</a></div><div class="annotation-form-container" style="display: block;"></div></div><div class="annotations annotations-positioned" data-selector="#p-20"><a href="https://peerj.com/questions.form?format=html&amp;target=articles/9391&amp;_counts=1&amp;selector=%23p-20" class="btn btn-primary add-annotation" data-toggle="annotation-form" rel="nofollow">Ask a question about this section</a><button class="btn btn-default close-annotations">Close</button><div class="help-block annotation-learn-more"><a href="https://peerj.com/about/FAQ/academic-contribution/" target="_blank">Learn more about Q&amp;A</a></div><div class="annotation-form-container" style="display: block;"></div></div><div class="annotations annotations-positioned" data-selector="#p-21"><a href="https://peerj.com/questions.form?format=html&amp;target=articles/9391&amp;_counts=1&amp;selector=%23p-21" class="btn btn-primary add-annotation" data-toggle="annotation-form" rel="nofollow">Ask a question about this section</a><button class="btn btn-default close-annotations">Close</button><div class="help-block annotation-learn-more"><a href="https://peerj.com/about/FAQ/academic-contribution/" target="_blank">Learn more about Q&amp;A</a></div><div class="annotation-form-container" style="display: block;"></div></div><div class="annotations annotations-positioned" data-selector="#p-22"><a href="https://peerj.com/questions.form?format=html&amp;target=articles/9391&amp;_counts=1&amp;selector=%23p-22" class="btn btn-primary add-annotation" data-toggle="annotation-form" rel="nofollow">Ask a question about this section</a><button class="btn btn-default close-annotations">Close</button><div class="help-block annotation-learn-more"><a href="https://peerj.com/about/FAQ/academic-contribution/" target="_blank">Learn more about Q&amp;A</a></div><div class="annotation-form-container" style="display: block;"></div></div><div class="annotations annotations-positioned" data-selector="#p-23"><a href="https://peerj.com/questions.form?format=html&amp;target=articles/9391&amp;_counts=1&amp;selector=%23p-23" class="btn btn-primary add-annotation" data-toggle="annotation-form" rel="nofollow">Ask a question about this section</a><button class="btn btn-default close-annotations">Close</button><div class="help-block annotation-learn-more"><a href="https://peerj.com/about/FAQ/academic-contribution/" target="_blank">Learn more about Q&amp;A</a></div><div class="annotation-form-container" style="display: block;"></div></div><div class="annotations annotations-positioned" data-selector="#p-24"><a href="https://peerj.com/questions.form?format=html&amp;target=articles/9391&amp;_counts=1&amp;selector=%23p-24" class="btn btn-primary add-annotation" data-toggle="annotation-form" rel="nofollow">Ask a question about this section</a><button class="btn btn-default close-annotations">Close</button><div class="help-block annotation-learn-more"><a href="https://peerj.com/about/FAQ/academic-contribution/" target="_blank">Learn more about Q&amp;A</a></div><div class="annotation-form-container" style="display: block;"></div></div><div class="annotations annotations-positioned" data-selector="#p-25"><a href="https://peerj.com/questions.form?format=html&amp;target=articles/9391&amp;_counts=1&amp;selector=%23p-25" class="btn btn-primary add-annotation" data-toggle="annotation-form" rel="nofollow">Ask a question about this section</a><button class="btn btn-default close-annotations">Close</button><div class="help-block annotation-learn-more"><a href="https://peerj.com/about/FAQ/academic-contribution/" target="_blank">Learn more about Q&amp;A</a></div><div class="annotation-form-container" style="display: block;"></div></div><div class="annotations annotations-positioned" data-selector="#p-26"><a href="https://peerj.com/questions.form?format=html&amp;target=articles/9391&amp;_counts=1&amp;selector=%23p-26" class="btn btn-primary add-annotation" data-toggle="annotation-form" rel="nofollow">Ask a question about this section</a><button class="btn btn-default close-annotations">Close</button><div class="help-block annotation-learn-more"><a href="https://peerj.com/about/FAQ/academic-contribution/" target="_blank">Learn more about Q&amp;A</a></div><div class="annotation-form-container" style="display: block;"></div></div><div class="annotations annotations-positioned" data-selector="#p-28"><a href="https://peerj.com/questions.form?format=html&amp;target=articles/9391&amp;_counts=1&amp;selector=%23p-28" class="btn btn-primary add-annotation" data-toggle="annotation-form" rel="nofollow">Ask a question about this section</a><button class="btn btn-default close-annotations">Close</button><div class="help-block annotation-learn-more"><a href="https://peerj.com/about/FAQ/academic-contribution/" target="_blank">Learn more about Q&amp;A</a></div><div class="annotation-form-container" style="display: block;"></div></div><div class="annotations annotations-positioned" data-selector="#p-30"><a href="https://peerj.com/questions.form?format=html&amp;target=articles/9391&amp;_counts=1&amp;selector=%23p-30" class="btn btn-primary add-annotation" data-toggle="annotation-form" rel="nofollow">Ask a question about this section</a><button class="btn btn-default close-annotations">Close</button><div class="help-block annotation-learn-more"><a href="https://peerj.com/about/FAQ/academic-contribution/" target="_blank">Learn more about Q&amp;A</a></div><div class="annotation-form-container" style="display: block;"></div></div><div class="annotations annotations-positioned" data-selector="#p-31"><a href="https://peerj.com/questions.form?format=html&amp;target=articles/9391&amp;_counts=1&amp;selector=%23p-31" class="btn btn-primary add-annotation" data-toggle="annotation-form" rel="nofollow">Ask a question about this section</a><button class="btn btn-default close-annotations">Close</button><div class="help-block annotation-learn-more"><a href="https://peerj.com/about/FAQ/academic-contribution/" target="_blank">Learn more about Q&amp;A</a></div><div class="annotation-form-container" style="display: block;"></div></div><div class="annotations annotations-positioned" data-selector="#p-32"><a href="https://peerj.com/questions.form?format=html&amp;target=articles/9391&amp;_counts=1&amp;selector=%23p-32" class="btn btn-primary add-annotation" data-toggle="annotation-form" rel="nofollow">Ask a question about this section</a><button class="btn btn-default close-annotations">Close</button><div class="help-block annotation-learn-more"><a href="https://peerj.com/about/FAQ/academic-contribution/" target="_blank">Learn more about Q&amp;A</a></div><div class="annotation-form-container" style="display: block;"></div></div><div class="annotations annotations-positioned" data-selector="#p-33"><a href="https://peerj.com/questions.form?format=html&amp;target=articles/9391&amp;_counts=1&amp;selector=%23p-33" class="btn btn-primary add-annotation" data-toggle="annotation-form" rel="nofollow">Ask a question about this section</a><button class="btn btn-default close-annotations">Close</button><div class="help-block annotation-learn-more"><a href="https://peerj.com/about/FAQ/academic-contribution/" target="_blank">Learn more about Q&amp;A</a></div><div class="annotation-form-container" style="display: block;"></div></div><div class="annotations annotations-positioned" data-selector="#fig-1"><a href="https://peerj.com/questions.form?format=html&amp;target=articles/9391&amp;_counts=1&amp;selector=%23fig-1" class="btn btn-primary add-annotation" data-toggle="annotation-form" rel="nofollow">Ask a question about this section</a><button class="btn btn-default close-annotations">Close</button><div class="help-block annotation-learn-more"><a href="https://peerj.com/about/FAQ/academic-contribution/" target="_blank">Learn more about Q&amp;A</a></div><div class="annotation-form-container" style="display: block;"></div></div><div class="annotations annotations-positioned" data-selector="#fig-2"><a href="https://peerj.com/questions.form?format=html&amp;target=articles/9391&amp;_counts=1&amp;selector=%23fig-2" class="btn btn-primary add-annotation" data-toggle="annotation-form" rel="nofollow">Ask a question about this section</a><button class="btn btn-default close-annotations">Close</button><div class="help-block annotation-learn-more"><a href="https://peerj.com/about/FAQ/academic-contribution/" target="_blank">Learn more about Q&amp;A</a></div><div class="annotation-form-container" style="display: block;"></div></div><div class="annotations annotations-positioned" data-selector="#fig-3"><a href="https://peerj.com/questions.form?format=html&amp;target=articles/9391&amp;_counts=1&amp;selector=%23fig-3" class="btn btn-primary add-annotation" data-toggle="annotation-form" rel="nofollow">Ask a question about this section</a><button class="btn btn-default close-annotations">Close</button><div class="help-block annotation-learn-more"><a href="https://peerj.com/about/FAQ/academic-contribution/" target="_blank">Learn more about Q&amp;A</a></div><div class="annotation-form-container" style="display: block;"></div></div><div class="annotations annotations-positioned" data-selector="#fig-4"><a href="https://peerj.com/questions.form?format=html&amp;target=articles/9391&amp;_counts=1&amp;selector=%23fig-4" class="btn btn-primary add-annotation" data-toggle="annotation-form" rel="nofollow">Ask a question about this section</a><button class="btn btn-default close-annotations">Close</button><div class="help-block annotation-learn-more"><a href="https://peerj.com/about/FAQ/academic-contribution/" target="_blank">Learn more about Q&amp;A</a></div><div class="annotation-form-container" style="display: block;"></div></div><div class="annotations annotations-positioned" data-selector="#fig-5"><a href="https://peerj.com/questions.form?format=html&amp;target=articles/9391&amp;_counts=1&amp;selector=%23fig-5" class="btn btn-primary add-annotation" data-toggle="annotation-form" rel="nofollow">Ask a question about this section</a><button class="btn btn-default close-annotations">Close</button><div class="help-block annotation-learn-more"><a href="https://peerj.com/about/FAQ/academic-contribution/" target="_blank">Learn more about Q&amp;A</a></div><div class="annotation-form-container" style="display: block;"></div></div><div class="annotations annotations-positioned" data-selector="#table-1"><a href="https://peerj.com/questions.form?format=html&amp;target=articles/9391&amp;_counts=1&amp;selector=%23table-1" class="btn btn-primary add-annotation" data-toggle="annotation-form" rel="nofollow">Ask a question about this section</a><button class="btn btn-default close-annotations">Close</button><div class="help-block annotation-learn-more"><a href="https://peerj.com/about/FAQ/academic-contribution/" target="_blank">Learn more about Q&amp;A</a></div><div class="annotation-form-container" style="display: block;"></div></div><div class="annotations annotations-positioned" data-selector="#table-2"><a href="https://peerj.com/questions.form?format=html&amp;target=articles/9391&amp;_counts=1&amp;selector=%23table-2" class="btn btn-primary add-annotation" data-toggle="annotation-form" rel="nofollow">Ask a question about this section</a><button class="btn btn-default close-annotations">Close</button><div class="help-block annotation-learn-more"><a href="https://peerj.com/about/FAQ/academic-contribution/" target="_blank">Learn more about Q&amp;A</a></div><div class="annotation-form-container" style="display: block;"></div></div><div class="annotations annotations-positioned" data-selector="#table-3"><a href="https://peerj.com/questions.form?format=html&amp;target=articles/9391&amp;_counts=1&amp;selector=%23table-3" class="btn btn-primary add-annotation" data-toggle="annotation-form" rel="nofollow">Ask a question about this section</a><button class="btn btn-default close-annotations">Close</button><div class="help-block annotation-learn-more"><a href="https://peerj.com/about/FAQ/academic-contribution/" target="_blank">Learn more about Q&amp;A</a></div><div class="annotation-form-container" style="display: block;"></div></div><div class="annotations annotations-positioned" data-selector="#table-4"><a href="https://peerj.com/questions.form?format=html&amp;target=articles/9391&amp;_counts=1&amp;selector=%23table-4" class="btn btn-primary add-annotation" data-toggle="annotation-form" rel="nofollow">Ask a question about this section</a><button class="btn btn-default close-annotations">Close</button><div class="help-block annotation-learn-more"><a href="https://peerj.com/about/FAQ/academic-contribution/" target="_blank">Learn more about Q&amp;A</a></div><div class="annotation-form-container" style="display: block;"></div></div><div class="annotations annotations-positioned" data-selector="#table-5"><a href="https://peerj.com/questions.form?format=html&amp;target=articles/9391&amp;_counts=1&amp;selector=%23table-5" class="btn btn-primary add-annotation" data-toggle="annotation-form" rel="nofollow">Ask a question about this section</a><button class="btn btn-default close-annotations">Close</button><div class="help-block annotation-learn-more"><a href="https://peerj.com/about/FAQ/academic-contribution/" target="_blank">Learn more about Q&amp;A</a></div><div class="annotation-form-container" style="display: block;"></div></div></div>

        <!-- Middle col -->
        <div id="article-item-middle" class="span7" data-ms-type-entity="articles" data-ms-type-id="research-article" data-ms-type-text="Research-article">
            
                            <div id="article-tweets-container">
    <div class="row-fluid article-tweets-header">
        <div class="span9">
            <h2><em>Shannon diversity index: a call to replace the 
original Shannon’s formula with unbiased estimator in the population 
genetics studies</em></h2>
        </div>
        <div class="span3">
                            <div class="btn btn-inverse pull-right" id="btn-view-article"><span class="icon-file"></span> View article</div>
                    </div>
    </div>
    <div class="tweet-items">        <div class="row-fluid tweet-item">
        <div class="span1 offset1"><img src="default_profile_normal.png"></div>
        <div class="span8">
            <div>
                <a class="twitter-profile-url" href="https://twitter.com/intent/user/?user_id=1280488540242169858" target="_blank"><strong>Maciej</strong> <span class="twitter-handle">@KonopinskiMK</span></a>
                <span class="item-tweet-date">79 days ago</span>
            </div>
            <div>Shannon diversity index estimators https://t.co/I2fLNi0tTF</div>
            <div class="item-tweet-cta">
                <span class="item-tweet-cta-action"><a href="https://twitter.com/KonopinskiMK/status/1280492914754359300" target="_blank"><i class="icon-comment-alt"></i> reply</a></span>
                <span class="item-tweet-cta-action"><a href="https://twitter.com/KonopinskiMK/status/1280492914754359300" target="_blank"><i class="icon-retweet"></i> retweet</a></span>
                <span><a href="https://twitter.com/KonopinskiMK/status/1280492914754359300" target="_blank"><i class="icon-heart-empty"></i> like</a></span>
            </div>
        </div>
    </div>
        <div class="row-fluid tweet-item">
        <div class="span1 offset1"><img src="default_profile_normal.png"></div>
        <div class="span8">
            <div>
                <a class="twitter-profile-url" href="https://twitter.com/intent/user/?user_id=1280488540242169858" target="_blank"><strong>Maciej</strong> <span class="twitter-handle">@KonopinskiMK</span></a>
                <span class="item-tweet-date">79 days ago</span>
            </div>
            <div>Shannon diversity index estimators https://t.co/I2fLNi0tTF</div>
            <div class="item-tweet-cta">
                <span class="item-tweet-cta-action"><a href="https://twitter.com/KonopinskiMK/status/1280489753109057537" target="_blank"><i class="icon-comment-alt"></i> reply</a></span>
                <span class="item-tweet-cta-action"><a href="https://twitter.com/KonopinskiMK/status/1280489753109057537" target="_blank"><i class="icon-retweet"></i> retweet</a></span>
                <span><a href="https://twitter.com/KonopinskiMK/status/1280489753109057537" target="_blank"><i class="icon-heart-empty"></i> like</a></span>
            </div>
        </div>
    </div>
        <div class="row-fluid tweet-item">
        <div class="span1 offset1"><img src="hDMoyXNd_normal.jpg"></div>
        <div class="span8">
            <div>
                <a class="twitter-profile-url" href="https://twitter.com/intent/user/?user_id=1266258624" target="_blank"><strong>Abraham Carrizosa V حقل عنب</strong> <span class="twitter-handle">@CAbrahamV</span></a>
                <span class="item-tweet-date">86 days ago</span>
            </div>
            <div>RT @thePeerJ: Just published in #PeerJ - Shannon 
diversity index: a call to replace the original Shannon’s formula with 
unbiased estimator…</div>
            <div class="item-tweet-cta">
                <span class="item-tweet-cta-action"><a href="https://twitter.com/CAbrahamV/status/1278017395366510592" target="_blank"><i class="icon-comment-alt"></i> reply</a></span>
                <span class="item-tweet-cta-action"><a href="https://twitter.com/CAbrahamV/status/1278017395366510592" target="_blank"><i class="icon-retweet"></i> retweet</a></span>
                <span><a href="https://twitter.com/CAbrahamV/status/1278017395366510592" target="_blank"><i class="icon-heart-empty"></i> like</a></span>
            </div>
        </div>
    </div>
        <div class="row-fluid tweet-item">
        <div class="span1 offset1"><img src="pA-QvYuC_normal.jpg"></div>
        <div class="span8">
            <div>
                <a class="twitter-profile-url" href="https://twitter.com/intent/user/?user_id=2415892688" target="_blank"><strong>Blake Stamps</strong> <span class="twitter-handle">@BlakeStamps</span></a>
                <span class="item-tweet-date">86 days ago</span>
            </div>
            <div>RT @thePeerJ: Just published in #PeerJ - Shannon 
diversity index: a call to replace the original Shannon’s formula with 
unbiased estimator…</div>
            <div class="item-tweet-cta">
                <span class="item-tweet-cta-action"><a href="https://twitter.com/BlakeStamps/status/1278016966855360514" target="_blank"><i class="icon-comment-alt"></i> reply</a></span>
                <span class="item-tweet-cta-action"><a href="https://twitter.com/BlakeStamps/status/1278016966855360514" target="_blank"><i class="icon-retweet"></i> retweet</a></span>
                <span><a href="https://twitter.com/BlakeStamps/status/1278016966855360514" target="_blank"><i class="icon-heart-empty"></i> like</a></span>
            </div>
        </div>
    </div>
        <div class="row-fluid tweet-item">
        <div class="span1 offset1"><img src="RwW320zF_normal.jpg"></div>
        <div class="span8">
            <div>
                <a class="twitter-profile-url" href="https://twitter.com/intent/user/?user_id=405101510" target="_blank"><strong>Dr. Carole-Anne Gillis</strong> <span class="twitter-handle">@CaroleAnnGillis</span></a>
                <span class="item-tweet-date">86 days ago</span>
            </div>
            <div>RT @thePeerJ: Just published in #PeerJ - Shannon 
diversity index: a call to replace the original Shannon’s formula with 
unbiased estimator…</div>
            <div class="item-tweet-cta">
                <span class="item-tweet-cta-action"><a href="https://twitter.com/CaroleAnnGillis/status/1278016659131940864" target="_blank"><i class="icon-comment-alt"></i> reply</a></span>
                <span class="item-tweet-cta-action"><a href="https://twitter.com/CaroleAnnGillis/status/1278016659131940864" target="_blank"><i class="icon-retweet"></i> retweet</a></span>
                <span><a href="https://twitter.com/CaroleAnnGillis/status/1278016659131940864" target="_blank"><i class="icon-heart-empty"></i> like</a></span>
            </div>
        </div>
    </div>
        <div class="row-fluid tweet-item">
        <div class="span1 offset1"><img src="0EUgsdRN_normal.jpg"></div>
        <div class="span8">
            <div>
                <a class="twitter-profile-url" href="https://twitter.com/intent/user/?user_id=161497413" target="_blank"><strong>Diogo Proietti </strong> <span class="twitter-handle">@DiogoProvete</span></a>
                <span class="item-tweet-date">86 days ago</span>
            </div>
            <div>RT @thePeerJ: Just published in #PeerJ - Shannon 
diversity index: a call to replace the original Shannon’s formula with 
unbiased estimator…</div>
            <div class="item-tweet-cta">
                <span class="item-tweet-cta-action"><a href="https://twitter.com/DiogoProvete/status/1278016523454603265" target="_blank"><i class="icon-comment-alt"></i> reply</a></span>
                <span class="item-tweet-cta-action"><a href="https://twitter.com/DiogoProvete/status/1278016523454603265" target="_blank"><i class="icon-retweet"></i> retweet</a></span>
                <span><a href="https://twitter.com/DiogoProvete/status/1278016523454603265" target="_blank"><i class="icon-heart-empty"></i> like</a></span>
            </div>
        </div>
    </div>
        <div class="row-fluid tweet-item">
        <div class="span1 offset1"><img src="wjHxfQAv_normal.jpeg"></div>
        <div class="span8">
            <div>
                <a class="twitter-profile-url" href="https://twitter.com/intent/user/?user_id=322889899" target="_blank"><strong>Brian Hayden</strong> <span class="twitter-handle">@DrHaydo</span></a>
                <span class="item-tweet-date">86 days ago</span>
            </div>
            <div>RT @thePeerJ: Just published in #PeerJ - Shannon 
diversity index: a call to replace the original Shannon’s formula with 
unbiased estimator…</div>
            <div class="item-tweet-cta">
                <span class="item-tweet-cta-action"><a href="https://twitter.com/DrHaydo/status/1278016293262766081" target="_blank"><i class="icon-comment-alt"></i> reply</a></span>
                <span class="item-tweet-cta-action"><a href="https://twitter.com/DrHaydo/status/1278016293262766081" target="_blank"><i class="icon-retweet"></i> retweet</a></span>
                <span><a href="https://twitter.com/DrHaydo/status/1278016293262766081" target="_blank"><i class="icon-heart-empty"></i> like</a></span>
            </div>
        </div>
    </div>
        <div class="row-fluid tweet-item">
        <div class="span1 offset1"><img src="xgyGFDJk_normal.jpg"></div>
        <div class="span8">
            <div>
                <a class="twitter-profile-url" href="https://twitter.com/intent/user/?user_id=402875257" target="_blank"><strong>PeerJ — Life and Environment Journal</strong> <span class="twitter-handle">@thePeerJ</span></a>
                <span class="item-tweet-date">86 days ago</span>
            </div>
            <div>Just published in #PeerJ - Shannon diversity index: a 
call to replace the original Shannon’s formula with unbiased estimator 
in the population genetics studies https://t.co/l0Wg6eH02Z
#biodiversity #evolutionarystudies #genetics https://t.co/PZZVfuFNx2</div>
            <div class="item-tweet-cta">
                <span class="item-tweet-cta-action"><a href="https://twitter.com/thePeerJ/status/1278016206042206211" target="_blank"><i class="icon-comment-alt"></i> reply</a></span>
                <span class="item-tweet-cta-action"><a href="https://twitter.com/thePeerJ/status/1278016206042206211" target="_blank"><i class="icon-retweet"></i> retweet</a></span>
                <span><a href="https://twitter.com/thePeerJ/status/1278016206042206211" target="_blank"><i class="icon-heart-empty"></i> like</a></span>
            </div>
        </div>
    </div>

<div class="tweet-pagination pagination">
    

    <hr>
</div></div>
</div>            
            <div id="article-main-container">
                <div class="article-section-breadcrumb">
                                            <span class="icon-angle-left"></span>
                        <span>
                            <a href="https://peerj.com/sections/bioinformatics-genomics/latest/">
                                <span class="article-section-breadcrumb-name">Bioinformatics and Genomics</span>
                            </a>
                        </span>
                                    </div>


                <div class="hidden-print">
                    
                                                                                                </div>

                <!-- Main article -->
                <article itemscope="itemscope" itemtype="http://schema.org/ScholarlyArticle"><header class="article-meta front"><h1 class="article-title" itemprop="name headline">Shannon
 diversity index: a call to replace the original Shannon’s formula with 
unbiased estimator in the population genetics studies</h1><div class="subjects-navigation"><div class="subject-item manuscriptType-item"><a href="https://peerj.com/search/?type=articles&amp;manuscriptType=research-article">Research article</a></div><div class="subject-item"><a href="https://peerj.com/subjects/?filter=Biodiversity">Biodiversity</a></div><div class="subject-item"><a href="https://peerj.com/subjects/?filter=Evolutionary%20Studies">Evolutionary Studies</a></div><div class="subject-item"><a href="https://peerj.com/subjects/?filter=Genetics">Genetics</a></div><div class="subject-item"><a href="https://peerj.com/subjects/?filter=Population%20Biology">Population Biology</a></div></div>
<div class="article-authors"><span class="contrib" itemscope="itemscope" itemtype="http://schema.org/Person" id="author-1" data-jats-contrib-type="author" data-jats-corresp="yes" itemprop="author"><a href="https://peerj.com/articles/9391/author-1" rel="author" itemprop="url"><span class="name" itemprop="name"><span class="given-names" itemprop="givenName">Maciej K.</span> <span class="surname" itemprop="familyName">Konopiński</span></span></a><a class="corresp" href="mailto:konopinski@iop.krakow.pl" target="_blank" title="" data-toggle="tooltip" itemprop="email" data-original-title="email the corresponding author"><i class="icon-envelope">​</i></a></span></div><div class="article-publication-date"><span class="article-meta-name">Published </span><span class="article-meta-value">June 29, 2020</span></div><div id="article-identifiers">
        <span class="article-meta-name">PubMed</span>
        <a href="https://www.ncbi.nlm.nih.gov/pubmed/32655992" id="article-identifier-pmid" target="_blank">32655992</a>
    </div>
<div id="article-item-article-information-container"><div class="row-fluid row-article-item-section-heading"><div class="span1 article-main-left-span1"><i class="article-section-indicator icon-chevron-right"></i></div><h2>Author and article information</h2><div id="article-item-article-information" class="article-item-section-toggle"></div></div><div class="row-fluid row-article-item-section" style="display: none;"><div class="span1 article-main-left-span1">&nbsp;</div><div class="span11 article-item-section-content"><div id="article-information" class="article-item-section">
<div class="article-notes"><div itemscope="itemscope" itemtype="http://schema.org/Organization" id="aff-1">
<span class="article-label-container">&nbsp;</span><span itemprop="address"><span class="institution">Institute of Nature Conservation, Polish Academy of Sciences</span>, <span class="city">Kraków</span>, <span class="country">Poland</span></span>
</div></div>
<dl class="article-identifiers">
<dt> DOI</dt>
<dd>
<a href="https://doi.org/10.7717/peerj.9391" itemprop="sameAs">10.7717/peerj.9391</a><meta itemprop="sameAs" content="info:doi/10.7717/peerj.9391">
</dd>
</dl>
<dl class="article-dates">
<dt>Published</dt>
<dd><time itemprop="datePublished">2020-06-29</time></dd>
<dt>Accepted</dt>
<dd><time data-itemprop="dateAccepted">2020-05-29</time></dd>
<dt>Received</dt>
<dd><time itemprop="dateCreated">2019-12-17</time></dd>
</dl>
<dl class="article-editors">
<dt>Academic Editor</dt>
<dd itemprop="editor" itemscope="itemscope" itemtype="http://schema.org/Person"><a itemprop="url" href="https://peerj.com/articles/9391/editor-1" class="contrib" data-jats-contrib-type="editor"><span class="name" itemprop="name"><span class="given-names" itemprop="givenName">Patricia</span> <span class="surname" itemprop="familyName">Gandini</span></span></a></dd>
</dl>
<dl class="article-subjects">
<dt>Subject Areas</dt>
<dd>
<a class="subject" itemprop="about" href="https://peerj.com/subjects/?filter=Biodiversity">Biodiversity</a>, <a class="subject" itemprop="about" href="https://peerj.com/subjects/?filter=Evolutionary%20Studies">Evolutionary Studies</a>, <a class="subject" itemprop="about" href="https://peerj.com/subjects/?filter=Genetics">Genetics</a>, <a class="subject" itemprop="about" href="https://peerj.com/subjects/?filter=Population%20Biology">Population Biology</a>
</dd>
<dt>Keywords</dt>
<dd>
<span class="kwd" itemprop="keywords">Genetic diversity</span>, <span class="kwd" itemprop="keywords">Shannon index</span>, <span class="kwd" itemprop="keywords">Coalescent simulations</span>, <span class="kwd" itemprop="keywords">Measures of genetic variation</span>, <span class="kwd" itemprop="keywords">Sample size effect</span>, <span class="kwd" itemprop="keywords">Statistical genetics</span>
</dd>
</dl>
<dl class="article-license">
<dt>Copyright</dt>
<dd>© <span itemprop="copyrightYear">2020</span> <span itemprop="copyrightHolder">Konopiński</span>
</dd>
<dt>Licence</dt>
<dd>
               <span class="license-p">This is an open access article distributed under the terms of the <a class="ext-link" href="https://creativecommons.org/licenses/by/4.0/" rel="license" data-jats-ext-link-type="uri">Creative Commons Attribution License</a>,
 which permits unrestricted use, distribution, reproduction and 
adaptation in any medium and for any purpose provided that it is 
properly attributed. For attribution, the original author(s), title, 
publication source (PeerJ) and either DOI or URL of the article must be 
cited.</span>
            </dd>
</dl>
<dl class="self-citation">
<dt>Cite this article</dt>
<dd>
<span class="self-citation-authors">Konopiński MK.</span> <span class="self-citation-year">2020</span>. <span class="self-citation-title">Shannon
 diversity index: a call to replace the original Shannon’s formula with 
unbiased estimator in the population genetics studies</span>. <span itemprop="isPartOf" itemscope="itemscope" itemtype="http://schema.org/PublicationVolume"><span class="self-citation-journal" itemprop="isPartOf" itemscope="itemscope" itemtype="http://schema.org/Periodical"><span itemprop="name">PeerJ</span></span> <span class="self-citation-volume" itemprop="volumeNumber">8</span></span>:<span class="self-citation-elocation" itemprop="pageStart">e9391</span> <a href="https://doi.org/10.7717/peerj.9391" itemprop="url">https://doi.org/10.7717/peerj.9391</a>
</dd>
</dl>
</div></div></div></div><div id="article-item-abstract-container"><div class="row-fluid row-article-item-section-heading"><div class="span1 article-main-left-span1"><i class="article-section-indicator icon-chevron-down"></i></div><h2>Abstract</h2><div id="article-item-abstract" class="article-item-section-toggle"></div></div><div class="row-fluid row-article-item-section"><div class="span1 article-main-left-span1">&nbsp;</div><div class="span11 article-item-section-content"><div class="abstract" itemprop="description">
            <section class="sec">
               <h2 class="heading">Background</h2>
               <p>The Shannon diversity index has been widely used in 
population genetics studies. Recently, it was proposed as a unifying 
measure of diversity at different levels—from genes and populations to 
whole species and ecosystems. The index, however, was proven to be 
negatively biased at small sample sizes. Modifications to the original 
Shannon’s formula have been proposed to obtain an unbiased estimator.</p>
            </section>
            <section class="sec">
               <h2 class="heading">Methods</h2>
               <p>In this study, the performance of four different 
estimators of Shannon index—the original Shannon’s formula and those of 
Zahl, Chao and Shen and Chao et al.—was tested on simulated 
microsatellite data. Both the simulation and analysis of the results 
were performed in the R language environment. A new R function was 
created for the calculation of all four indices from the genind data 
format.</p>
            </section>
            <section class="sec">
               <h2 class="heading">Results</h2>
               <p>Sample size dependence was detected in all the 
estimators analysed; however, the deviation from parametric values was 
substantially smaller in the derived measures than in the original 
Shannon’s formula. Error rate was negatively associated with population 
heterozygosity. Comparisons among loci showed that fast-mutating loci 
were less affected by the error, except for the original Shannon’s 
estimator which, in the smallest sample, was more strongly affected by 
loci with a higher number of alleles. The Zahl and Chao et al. 
estimators performed notably better than the original Shannon’s formula.</p>
            </section>
            <section class="sec">
               <h2 class="heading">Conclusion</h2>
               <p>The results of this study show that the original 
Shannon index should no longer be used as a measure of genetic diversity
 and should be replaced by Zahl’s unbiased estimator.</p>
            </section>
         </div><div style="margin-top:20px" class="alert alert-info self-citation"><p style="color:#444;font-weight:bold;font-size:14px;">Cite this as</p>
<span class="self-citation-authors">Konopiński MK.</span> <span class="self-citation-year">2020</span>. <span class="self-citation-title">Shannon
 diversity index: a call to replace the original Shannon’s formula with 
unbiased estimator in the population genetics studies</span>. <span itemprop="isPartOf" itemscope="itemscope" itemtype="http://schema.org/PublicationVolume"><span class="self-citation-journal" itemprop="isPartOf" itemscope="itemscope" itemtype="http://schema.org/Periodical"><span itemprop="name">PeerJ</span></span> <span class="self-citation-volume" itemprop="volumeNumber">8</span></span>:<span class="self-citation-elocation" itemprop="pageStart">e9391</span> <a href="https://doi.org/10.7717/peerj.9391" itemprop="url">https://doi.org/10.7717/peerj.9391</a>
</div></div></div></div>
</header><main><div class="body" lang="en">
      
      
      
      
      
   </div><div><div class="row-fluid row-article-item-section-heading"><div class="span1 article-main-left-span1"><i class="article-section-indicator icon-chevron-down"></i></div><h2>Main article text</h2><div id="article-item-main-text" class="article-item-section-toggle"></div></div><div class="row-fluid row-article-item-section"><div class="span1 article-main-left-span1">&nbsp;</div><div class="span11 article-item-section-content"><div id="article-item-main-sections"><section class="sec" id="intro">
         <h2 class="heading">Introduction</h2>
         <p id="p-1" class="has-plink"><a class="plink" href="#p-1" rel="nofollow"><i class="icon-comment icon-flip-horizontal"></i></a>The Shannon diversity index (<a class="xref xref-bibr" href="https://doi.org/10.1002%2Fj.1538-7305.1948.tb01338.x" title="" data-jats-ref-type="bibr" data-jats-rid="ref-36" data-original-title="A mathematical theory of communication">Shannon, 1948</a>), also known as the Shannon-Wiener index, Shannon entropy or, incorrectly, the Shannon-Weaver index (<a class="xref xref-bibr" href="https://doi.org/10.1046%2Fj.1466-822X.2003.00015.x" title="" data-jats-ref-type="bibr" data-jats-rid="ref-39" data-original-title="A tribute to Claude Shannon (1916–2001) and a plea for more rigorous use of species richness, species diversity and the ‘Shannon–Wiener’ index">Spellerberg &amp; Fedor, 2003</a>),
 has been used to estimate genetic diversity in numerous studies. It can
 be utilised to describe variation at multiple levels of genetic 
organisation from single nucleotide polymorphisms (SNP), through whole 
species or larger taxonomic units to ecosystems. Due to its additive 
properties (<a class="xref xref-bibr" href="https://doi.org/10.1890%2F06-1736.1" title="" data-jats-ref-type="bibr" data-jats-rid="ref-21" data-original-title="Partitioning diversity into independent alpha and beta components">Jost, 2007</a>), the Shannon index has recently been postulated as a unifying measure for the partitioning of diversity at those levels (<a class="xref xref-bibr" href="https://doi.org/10.1111%2Feva.12593" title="" data-jats-ref-type="bibr" data-jats-rid="ref-15" data-original-title="Diversity from genes to ecosystems: a unifying framework to study variation across biological metrics and scales">Gaggiotti et al., 2018</a>; <a class="xref xref-bibr" href="https://doi.org/10.3390%2Fe20100727" title="" data-jats-ref-type="bibr" data-jats-rid="ref-37" data-original-title="Entropy, or information, unifies ecology and evolution and beyond">Sherwin, 2018</a>). Additionally, <a class="xref xref-bibr" href="https://doi.org/10.1016%2Fj.tree.2017.09.012" title="" data-jats-ref-type="bibr" data-jats-rid="ref-38" data-original-title="Information theory broadens the spectrum of molecular ecology and evolution">Sherwin et al. (2017)</a> showed its potential utility in genomic studies. Several population genetics programs and <i>R</i> packages calculate Shannon’s <i>H,</i> e.g.,&nbsp;GenAlEx (<a class="xref xref-bibr" href="https://doi.org/10.1093%2Fbioinformatics%2Fbts460" title="" data-jats-ref-type="bibr" data-jats-rid="ref-30" data-original-title="GenAlEx 6.5: genetic analysis in Excel. Population genetic software for teaching and research—an update">Peakall &amp; Smouse, 2012</a>), <i>DartR</i> (<a class="xref xref-bibr" href="https://doi.org/10.1111%2F1755-0998.12745" title="" data-jats-ref-type="bibr" data-jats-rid="ref-18" data-original-title="dartr: an r package to facilitate analysis of SNP data generated from reduced representation genome sequencing">Gruber et al., 2018</a>), <i>SpadeR</i> (<a class="xref xref-bibr" href="https://cran.r-project.org/package=SpadeR" title="" data-jats-ref-type="bibr" data-jats-rid="ref-7" data-original-title="">Chao, Ma &amp; Chiu, 2016</a>), <i>vegan</i> (<a class="xref xref-bibr" href="https://cran.r-project.org/package=vegan" title="" data-jats-ref-type="bibr" data-jats-rid="ref-28" data-original-title="">Oksanen et al., 2019</a>), <i>poppr</i> (<a class="xref xref-bibr" href="https://doi.org/10.7717%2Fpeerj.281" title="" data-jats-ref-type="bibr" data-jats-rid="ref-23" data-original-title="Poppr: an R package for genetic analysis of populations with clonal, partially clonal, and/or sexual reproduction">Kamvar, Tabima &amp; Grünwald, 2014</a>), GenoDive (<a class="xref xref-bibr" href="https://doi.org/10.1111%2Fj.1471-8286.2004.00770.x" title="" data-jats-ref-type="bibr" data-jats-rid="ref-27" data-original-title="genotype and genodive: two programs for the analysis of genetic diversity of asexual organisms">Meirmans &amp; Tienderen, 2004</a>), <i>HierDpart</i> (<a class="xref xref-bibr" href="https://cran.r-project.org/package=HierDpart" title="" data-jats-ref-type="bibr" data-jats-rid="ref-34" data-original-title="">Qin, 2019</a>) and the index is still in use (e.g.,&nbsp;&nbsp;<a class="xref xref-bibr" href="https://doi.org/10.1007%2Fs12686-018-1079-z" title="" data-jats-ref-type="bibr" data-jats-rid="ref-29" data-original-title="Predicting Shannon’s information for genes in finite populations: new uses for old equations">O’Reilly et al., 2018</a>; <a class="xref xref-bibr" href="https://doi.org/10.1002%2Fece3.5618" title="" data-jats-ref-type="bibr" data-jats-rid="ref-42" data-original-title="Population genetic structure, migration, and polyploidy origin of a medicinal species Gynostemma pentaphyllum (Cucurbitaceae)">Zhang et al., 2019</a>; <a class="xref xref-bibr" href="https://doi.org/10.1007%2Fs10592-019-01193-1" title="" data-jats-ref-type="bibr" data-jats-rid="ref-6" data-original-title="Genetic evidence of the southward founder speciation of Cycas taitungensis from ancestral C. revoluta along the Ryukyu Archipelagos">Chang, Huang &amp; Liao, 2019</a>).</p>
         <p id="p-2" class="has-plink"><a class="plink" href="#p-2" rel="nofollow"><i class="icon-comment icon-flip-horizontal"></i></a>The measure was initially developed within information theory (<a class="xref xref-bibr" href="https://doi.org/10.1002%2Fj.1538-7305.1948.tb01338.x" title="" data-jats-ref-type="bibr" data-jats-rid="ref-36" data-original-title="A mathematical theory of communication">Shannon, 1948</a>) but it was soon adopted in studies on species diversity (e.g.,&nbsp;&nbsp;<a class="xref xref-bibr" href="https://doi.org/10.1093%2Fbiomet%2F40.3-4.237" title="" data-jats-ref-type="bibr" data-jats-rid="ref-16" data-original-title="The population frequencies of species and the estimation of population parameters">Good, 1953</a>; <a class="xref xref-bibr" href="https://scholar.google.com/scholar_lookup?title=La%20teor%C3%ADa%20de%20la%20informaci%C3%B3n%20en%20Ecolog%C3%ADa&amp;author=Margalef&amp;publication_year=1957" title="" data-jats-ref-type="bibr" data-jats-rid="ref-25" data-original-title="La teoría de la información en Ecología">Margalef, 1957</a>; <a class="xref xref-bibr" href="https://doi.org/10.1073%2Fpnas.47.2.240" title="" data-jats-ref-type="bibr" data-jats-rid="ref-10" data-original-title="The effects of reduced competition in birds">Crowell, 1961</a>) and in population genetics (<a class="xref xref-bibr" href="https://doi.org/10.2135%2Fcropsci1975.0011183X001500050026x" title="" data-jats-ref-type="bibr" data-jats-rid="ref-20" data-original-title="Geographical patterns of phenotypic diversity in a world collection of durum wheats 1">Jain et al., 1975</a>). In principle, Shannon’s <i>H</i>
 takes into account the proportion of each species in an ecosystem 
studied; hence, it gives a better description of an ecosystem’s 
diversity than a plain number of species. When the number of species is 
equal in two locations, the index is capable of distinguishing between 
sites dominated by a single or only a few predominant species and those 
where each species has comparable input to the whole biodiversity (<a class="xref xref-bibr" href="https://scholar.google.com/scholar_lookup?title=La%20teor%C3%ADa%20de%20la%20informaci%C3%B3n%20en%20Ecolog%C3%ADa&amp;author=Margalef&amp;publication_year=1957" title="" data-jats-ref-type="bibr" data-jats-rid="ref-25" data-original-title="La teoría de la información en Ecología">Margalef, 1957</a>). Similarly, in population genetics studies, Shannon’s <i>H</i>
 allows distinguishing the level of variation between populations with 
the same number of alleles, when in some populations loci are dominated 
by only a few common alleles while in others variation is contributed 
more evenly by all alleles. The Shannon index is more sensitive to the 
loss of rare variants (e.g.,&nbsp;due to genetic bottlenecks) than 
heterozygosity, and more informative than allelic richness or a plain 
number of alleles (<a class="xref xref-bibr" href="https://doi.org/10.1016%2Fj.tree.2017.09.012" title="" data-jats-ref-type="bibr" data-jats-rid="ref-38" data-original-title="Information theory broadens the spectrum of molecular ecology and evolution">Sherwin et al., 2017</a>).</p>
         <p id="p-3" class="has-plink"><a class="plink" href="#p-3" rel="nofollow"><i class="icon-comment icon-flip-horizontal"></i></a>In
 the original formula of the Shannon index developed within information 
theory, it is assumed a researcher is capable of counting all words or 
letters in a text studied. In biological studies, however, researchers 
depend on a sample from the population and use it as a proxy for the 
population parameters. The index changes rapidly when the number of 
low-frequency occurrences grows, while their number depends on the 
sample size (<a class="xref xref-bibr" href="https://doi.org/10.1137%2F1104033" title="" data-jats-ref-type="bibr" data-jats-rid="ref-4" data-original-title="On a statistical estimate for the entropy of a sequence of independent random variables">Basharin, 1959</a>; <a class="xref xref-bibr" href="https://doi.org/10.1023%2FA%3A1026096204727" title="" data-jats-ref-type="bibr" data-jats-rid="ref-8" data-original-title="Nonparametric estimation of Shannon’s index of diversity when there are unseen species in sample">Chao &amp; Shen, 2003</a>).
 The probability that all the alleles are sampled falls dramatically 
when the sample size is small. At the same time, in small samples, the 
lack of some allele inflates the frequencies of the alleles that have 
been sampled. As a solution to that, a few unbiased estimators of <i>H</i> have been proposed. The methods use jack-knifing (<a class="xref xref-bibr" href="https://doi.org/10.2307%2F1936227" title="" data-jats-ref-type="bibr" data-jats-rid="ref-41" data-original-title="Jackknifing an index of diversity">Zahl, 1977</a>), rarefaction (<a class="xref xref-bibr" href="https://doi.org/10.1023%2FA%3A1026096204727" title="" data-jats-ref-type="bibr" data-jats-rid="ref-8" data-original-title="Nonparametric estimation of Shannon’s index of diversity when there are unseen species in sample">Chao &amp; Shen, 2003</a>) or the Good-Turing frequency formula (<a class="xref xref-bibr" href="https://doi.org/10.1093%2Fbiomet%2F40.3-4.237" title="" data-jats-ref-type="bibr" data-jats-rid="ref-16" data-original-title="The population frequencies of species and the estimation of population parameters">Good, 1953</a>; <a class="xref xref-bibr" href="https://doi.org/10.1111%2F2041-210X.12108" title="" data-jats-ref-type="bibr" data-jats-rid="ref-9" data-original-title="Entropy and the species accumulation curve: a novel entropy estimator via discovery rates of new species">Chao, Wang &amp; Jost, 2013</a>)
 to account for unsampled components of the system (i.e.,&nbsp;species 
or alleles). Although the issue of sample size in population genetics 
has been addressed in several studies (e.g.,&nbsp;&nbsp;<a class="xref xref-bibr" href="https://doi.org/10.2135%2Fcropsci1970.0011183X001000040012x" title="" data-jats-ref-type="bibr" data-jats-rid="ref-26" data-original-title="Influence of sample size on the estimation of genetic variances in a synthetic variety of maize. I. Grain yield 1">Marquez-Sanchez &amp; Hallauer, 1970</a>; <a class="xref xref-bibr" href="https://doi.org/10.2307%2F1443409" title="" data-jats-ref-type="bibr" data-jats-rid="ref-17" data-original-title="Genetic distance and heterozygosity estimates in electrophoretic studies: effects of sample size">Gorman &amp; Renzi, 1979</a>; <a class="xref xref-bibr" href="https://scholar.google.com/scholar_lookup?title=Sample%20size%20requirements%20for%20addressing%20the%20population%20genetic%20issues%20of%20forensic%20use%20of%20DNA%20typing&amp;author=Chakraborty&amp;publication_year=1992" title="" data-jats-ref-type="bibr" data-jats-rid="ref-5" data-original-title="Sample size requirements for addressing the population genetic issues of forensic use of DNA typing">Chakraborty, 1992</a>; <a class="xref xref-bibr" href="https://doi.org/10.1007%2FBF00221895" title="" data-jats-ref-type="bibr" data-jats-rid="ref-11" data-original-title="High level of genetic differentiation for allelic richness among populations of the argan tree [Argania spinosa (L.) Skeels] endemic to Morocco">El Mousadik &amp; Petit, 1996</a>; <a class="xref xref-bibr" href="https://doi.org/10.1046%2Fj.1365-294X.2002.01612.x" title="" data-jats-ref-type="bibr" data-jats-rid="ref-24" data-original-title="Estimating allelic richness: effects of sample size and bottlenecks">Leberg, 2002</a>; <a class="xref xref-bibr" href="https://doi.org/10.1111%2Fj.0908-8857.2008.04094.x" title="" data-jats-ref-type="bibr" data-jats-rid="ref-33" data-original-title="The effects of sample size on population genetic diversity estimates in song sparrows Melospiza melodia">Pruett &amp; Winker, 2008</a>),
 the dependence of Shannon’s diversity estimation on sample size has 
never been thoroughly discussed with regard to genetic data. <a class="xref xref-bibr" href="https://doi.org/10.1186%2F1471-2156-10-84" title="" data-jats-ref-type="bibr" data-jats-rid="ref-3" data-original-title="A simple method for estimating genetic diversity in large populations from finite sample sizes">Bashalkhanov, Pandey &amp; Rajora (2009)</a> noticed an increase in the deviation of Shannon’s <i>H</i> at small sample sizes; however, the only solution they suggested was increasing the sample size to 60–90 genotypes. However, <a class="xref xref-bibr" href="https://doi.org/10.1016%2Fj.tree.2017.09.012" title="" data-jats-ref-type="bibr" data-jats-rid="ref-38" data-original-title="Information theory broadens the spectrum of molecular ecology and evolution">Sherwin et al. (2017)</a> pointed out that although methods for sampling correction of <i>H</i> exist, unbiased estimators remain rarely applied in population genetics studies.</p>
         <p id="p-4" class="has-plink"><a class="plink" href="#p-4" rel="nofollow"><i class="icon-comment icon-flip-horizontal"></i></a>The
 aim of this study was to assess the effect of sample size and locus 
properties (mutation rate and the maximum possible number of allelic 
states) on the estimation of Shannon’s original index (<i>H</i><sub><i>MLE</i></sub>) and its three unbiased estimators proposed by (<a class="xref xref-bibr" href="https://doi.org/10.2307%2F1936227" title="" data-jats-ref-type="bibr" data-jats-rid="ref-41" data-original-title="Jackknifing an index of diversity">Zahl, 1977</a>) (<i>H</i><sub><i>Z</i></sub>), (<a class="xref xref-bibr" href="https://doi.org/10.1023%2FA%3A1026096204727" title="" data-jats-ref-type="bibr" data-jats-rid="ref-8" data-original-title="Nonparametric estimation of Shannon’s index of diversity when there are unseen species in sample">Chao &amp; Shen, 2003</a>) (<i>H</i><sub><i>CS</i></sub>), (<a class="xref xref-bibr" href="https://doi.org/10.1111%2F2041-210X.12108" title="" data-jats-ref-type="bibr" data-jats-rid="ref-9" data-original-title="Entropy and the species accumulation curve: a novel entropy estimator via discovery rates of new species">Chao, Wang &amp; Jost, 2013</a>) (<i>H</i><sub><i>Chao</i></sub>).
 The performance of the four indices was tested extensively on data 
generated using coalescent simulations. The relative effects of sample 
size, locus properties and population diversity were analysed with a 
Generalised Linear Model. A wrapper R function was written to allow for 
estimation of the four indices directly on <i>adegenet</i>’s ‘genind’ objects.</p>
      </section><section class="sec" id="materials|methods">
         <h2 class="heading">Materials &amp; Methods</h2>
         <p id="p-5" class="has-plink"><a class="plink" href="#p-5" rel="nofollow"><i class="icon-comment icon-flip-horizontal"></i></a>Analyses were conducted in R 3.6.2 (<a class="xref xref-bibr" href="#ref-35" title="" data-jats-ref-type="bibr" data-jats-rid="ref-35" data-original-title="">R Development Core Team, 2009</a>). Coalescent simulations as implemented in <i>fastsimcoal2</i> ver. 2.6 (<a class="xref xref-bibr" href="https://doi.org/10.1371%2Fjournal.pgen.1003905" title="" data-jats-ref-type="bibr" data-jats-rid="ref-12" data-original-title="Robust demographic inference from genomic and SNP data">Excoffier et al., 2013</a>)
 were used to generate populations differing in levels of genetic 
variation due to their demographic histories. The program was called 
from within the R environment using the <i>strataG</i> package ver. 2.0.2 command <i>fastsimcoal</i> (<a class="xref xref-bibr" href="https://doi.org/10.1111%2F1755-0998.12559" title="" data-jats-ref-type="bibr" data-jats-rid="ref-1" data-original-title="stratag: an r package for manipulating, summarizing and analysing population genetic data">Archer, Adams &amp; Schneiders, 2017</a>).
 Twenty-four microsatellite loci with four different mutation rates 
(0.0001, 0.0002, 0.0005 and 0.001 mutations per generation) and six 
different maximum numbers of alleles (3, 6, 9, 12, 15 and 20 alleles) 
were simulated (<a class="xref xref-table" href="#table-1" data-jats-ref-type="table" data-jats-rid="table-1">Table 1</a>).
 The model assumed a large population of 10,000 diploid individuals 
divided into four populations containing 10,000 individuals each. Three 
of those underwent bottlenecks of different sizes (20, 50 and 500 
individuals in populations <i>P</i><sub>20</sub>, <i>P</i><sub>50</sub> and <i>P</i><sub>500</sub> respectively) while the fourth, the control population (<i>P</i><sub><i>C</i></sub>
 remained at a stable size of 10,000 individuals. Each of the 
bottlenecked populations after 20 generations recovered to the original 
size of 10,000 and was simulated for another 20 generations until time 
T0 when samples equalling the whole populations were saved both from 
bottlenecked and control populations. Reference parametric values of 
four different Shannon’s estimators were calculated for each total 
population sample:</p>
         <figure class="table-wrap has-plink" id="table-1"><a class="plink" href="#table-1" rel="nofollow"><i class="icon-comment icon-flip-horizontal"></i></a><div class="caption">
<span class="caption-label">Table 1: </span>
               <div class="title">Combination of mutation rates and a maximum number of allelic states in the 24 loci simulated in <i>fastsimcoal2</i>.</div>
            </div>
            
            <div class="table-container"><table class="table table-bordered table-condensed table-hover">
                  <colgroup>
                     <col>
                     <col>
                     <col>
                     <col>
                     <col>
                     <col>
                     <col>
                     <col>
                  </colgroup>
                  <tbody>
                     <tr>
                        <td></td>
                        <td></td>
                        <td style="text-align:center;" colspan="6">Max. number of alleles</td>
                     </tr>
                     <tr>
                        <td></td>
                        <td></td>
                        <td>3</td>
                        <td>6</td>
                        <td>9</td>
                        <td>12</td>
                        <td>15</td>
                        <td>20</td>
                     </tr>
                     <tr>
                        <td rowspan="4">Mutation rate</td>
                        <td>0.0001</td>
                        <td>L01</td>
                        <td>L02</td>
                        <td>L03</td>
                        <td>L04</td>
                        <td>L05</td>
                        <td>L06</td>
                     </tr>
                     <tr>
                        <td>0.0002</td>
                        <td>L07</td>
                        <td>L08</td>
                        <td>L09</td>
                        <td>L10</td>
                        <td>L11</td>
                        <td>L12</td>
                     </tr>
                     <tr>
                        <td>0.0005</td>
                        <td>L13</td>
                        <td>L14</td>
                        <td>L15</td>
                        <td>L16</td>
                        <td>L17</td>
                        <td>L18</td>
                     </tr>
                     <tr>
                        <td>0.001</td>
                        <td>L19</td>
                        <td>L20</td>
                        <td>L21</td>
                        <td>L22</td>
                        <td>L23</td>
                        <td>L24</td>
                     </tr>
                  </tbody>
               </table></div>
<div class="object-id article-component-doi">DOI: <a href="https://doi.org/10.7717/peerj.9391/table-1" data-toggle="tooltip" title="" data-original-title="Cite this object using this DOI">10.7717/peerj.9391/table-1</a>
</div>
         </figure>
         <ol class="list" id="list-1" data-jats-list-type="order">
            <li class="list-item">
<p id="p-6"><i>H</i><sub><i>MLE</i></sub>—the maximum likelihood estimate of <i>H</i> based on the original Shannon’s formula (<a class="xref xref-bibr" href="https://doi.org/10.1002%2Fj.1538-7305.1948.tb01338.x" title="" data-jats-ref-type="bibr" data-jats-rid="ref-36" data-original-title="A mathematical theory of communication">Shannon, 1948</a>, Theorem 2)—probably the most widely used in population genetics;</p>
            </li>
            <li class="list-item">
<p id="p-7"><i>H</i><sub><i>CS</i></sub>—unbiased estimator proposed by <a class="xref xref-bibr" href="https://doi.org/10.1023%2FA%3A1026096204727" title="" data-jats-ref-type="bibr" data-jats-rid="ref-8" data-original-title="Nonparametric estimation of Shannon’s index of diversity when there are unseen species in sample">Chao &amp; Shen</a> (<a class="xref xref-bibr" href="https://doi.org/10.1023%2FA%3A1026096204727" title="" data-jats-ref-type="bibr" data-jats-rid="ref-8" data-original-title="Nonparametric estimation of Shannon’s index of diversity when there are unseen species in sample">2003</a>, equation 8);</p>
            </li>
            <li class="list-item">
<p id="p-8"><i>H</i><sub><i>Chao</i></sub>—unbiased estimator proposed by <a class="xref xref-bibr" href="https://doi.org/10.1111%2F2041-210X.12108" title="" data-jats-ref-type="bibr" data-jats-rid="ref-9" data-original-title="Entropy and the species accumulation curve: a novel entropy estimator via discovery rates of new species">Chao, Wang &amp; Jost</a> (<a class="xref xref-bibr" href="https://doi.org/10.1111%2F2041-210X.12108" title="" data-jats-ref-type="bibr" data-jats-rid="ref-9" data-original-title="Entropy and the species accumulation curve: a novel entropy estimator via discovery rates of new species">2013</a>, equation 7);</p>
            </li>
            <li class="list-item">
<p id="p-9"><i>H</i><sub><i>Z</i></sub>—jackknife estimate proposed by <a class="xref xref-bibr" href="https://doi.org/10.2307%2F1936227" title="" data-jats-ref-type="bibr" data-jats-rid="ref-41" data-original-title="Jackknifing an index of diversity">Zahl</a> (<a class="xref xref-bibr" href="https://doi.org/10.2307%2F1936227" title="" data-jats-ref-type="bibr" data-jats-rid="ref-41" data-original-title="Jackknifing an index of diversity">1977</a>, equation 7).</p>
            </li>
         </ol>
         <p id="p-10" class="has-plink"><a class="plink" href="#p-10" rel="nofollow"><i class="icon-comment icon-flip-horizontal"></i></a>Additionally, expected heterozygosity (<i>He</i>) was calculated as a reference measure of the genetic variation of the total populations.</p>
         <p id="p-11" class="has-plink"><a class="plink" href="#p-11" rel="nofollow"><i class="icon-comment icon-flip-horizontal"></i></a>The four <i>H</i> estimators are calculated using the function <i>Diversity</i> in R package <i>SpadeR</i> (<a class="xref xref-bibr" href="https://cran.r-project.org/package=SpadeR" title="" data-jats-ref-type="bibr" data-jats-rid="ref-7" data-original-title="">Chao, Ma &amp; Chiu, 2016</a>).
 Initial attempts to apply the function to the simulated set showed that
 the function often fails due to a problem with another nested function 
that estimates species richness. Moreover, the function has to be run 
separately for each locus to obtain locus-specific <i>H</i>. Therefore, a generic function (<i>ShannonGen</i>) was written using the formulas from the <i>shannon_index</i> function nested in <i>Diversity</i>. The function takes <i>genind</i> objects as the input and transforms them into abundance data, which is required by functions taken from <i>shannon_index</i>. <i>ShannonGen</i> returns a list containing user-selected estimators of <i>H</i> for all loci and populations included in the input object. The function can be acquired from GitHub (<a class="ext-link" href="https://github.com/konopinski/Shannon/" data-jats-ext-link-type="uri" data-original-title="" title="">https://github.com/konopinski/Shannon/</a>).</p>
         <p id="p-12" class="has-plink"><a class="plink" href="#p-12" rel="nofollow"><i class="icon-comment icon-flip-horizontal"></i></a>The Shannon diversity index estimators were calculated for samples of <i>Ns</i>
 = 5, 20, 80 and 200 genotypes drawn randomly without replacement from 
the simulated populations, The numbers of samples used were selected to 
represent four sampling scenarios:</p>
         <ol class="list" id="list-2" data-jats-list-type="order">
            <li class="list-item">
<p id="p-13">limited availability of samples—a situation often faced in studies on rare or elusive animals—<i>Ns</i> = 5 samples;</p>
            </li>
            <li class="list-item">
<p id="p-14">a minimum acceptable number of samples as suggested by <a class="xref xref-bibr" href="https://doi.org/10.1111%2Fj.0908-8857.2008.04094.x" title="" data-jats-ref-type="bibr" data-jats-rid="ref-33" data-original-title="The effects of sample size on population genetic diversity estimates in song sparrows Melospiza melodia">Pruett &amp; Winker (2008)</a> commonly occurring in population genetics studies—<i>Ns</i> = 20 samples;</p>
            </li>
            <li class="list-item">
<p id="p-15">optimal sampling, according to <a class="xref xref-bibr" href="https://doi.org/10.1186%2F1471-2156-10-84" title="" data-jats-ref-type="bibr" data-jats-rid="ref-3" data-original-title="A simple method for estimating genetic diversity in large populations from finite sample sizes">Bashalkhanov, Pandey &amp; Rajora (2009)</a>—<i>Ns</i> = 80 samples.</p>
            </li>
            <li class="list-item">
<p id="p-16">a very large sample with presumably low sampling error—<i>Ns</i> = 200 samples.</p>
            </li>
         </ol>
         <p id="p-17" class="has-plink"><a class="plink" href="#p-17" rel="nofollow"><i class="icon-comment icon-flip-horizontal"></i></a>Additionally, the parametric values were obtained from the simulated populations.</p>
         <p id="p-18" class="has-plink"><a class="plink" href="#p-18" rel="nofollow"><i class="icon-comment icon-flip-horizontal"></i></a>Sampling variance of each <i>H</i>
 estimator was assessed using 500 sets of samples randomly drawn from 
each of the simulated populations. The standard deviations (<i>SD</i>) of the results were calculated for each sample size and demographic scenario. The <i>SD</i>
 values distributions were compared pairwise between metrics. The level 
of significance was assessed based on 100,000 comparisons of the 
randomly drawn pairs of the <i>SD</i> values.</p>
         <p id="p-19" class="has-plink"><a class="plink" href="#p-19" rel="nofollow"><i class="icon-comment icon-flip-horizontal"></i></a>For each <i>H</i> estimator, a relative bias was calculated as <i>rB</i> = ((<span class="inline-formula"><span class="MathJax_Preview" style="color: inherit;"><span class="MJXp-math" id="MJXp-Span-1"><span class="MJXp-mover" id="MJXp-Span-2"><span><span class="MJXp-over"><span class=""><span class="MJXp-mrow" id="MJXp-Span-5" style="margin-right: 0px; margin-left: 0px;"><span class="MJXp-mo" id="MJXp-Span-6" style="margin-left: 0em; margin-right: 0em;">ˆ</span></span></span><span class=""><span class="MJXp-mrow" id="MJXp-Span-3"><span class="MJXp-mi MJXp-italic" id="MJXp-Span-4">H</span></span></span></span></span></span><span class="MJXp-mo" id="MJXp-Span-7" style="margin-left: 0.267em; margin-right: 0.267em;">−</span><span class="MJXp-mi MJXp-italic" id="MJXp-Span-8">H</span></span></span><span class="MathJax MathJax_Processed" id="MathJax-Element-1-Frame" tabindex="0" style=""><nobr><span class="math" id="MathJax-Span-1"><span style="display: inline-block; position: relative; width: 0em; height: 0px; font-size: 116%;"><span style="position: absolute;"><span class="mrow" id="MathJax-Span-2"><span class="mover" id="MathJax-Span-3"><span style="display: inline-block; position: relative; width: 1.118em; height: 0px;"><span style="position: absolute; clip: rect(3.196em, 1000.89em, 4.167em, -1000em); top: -4.023em; left: 0.056em;"><span class="mrow" id="MathJax-Span-4"><span class="mi" id="MathJax-Span-5" style="font-family: MathJax_Math; font-style: italic;">H<span style="display: inline-block; overflow: hidden; height: 1px; width: 0.057em;"></span></span></span><span style="display: inline-block; width: 0px; height: 4.023em;"></span></span><span style="position: absolute; clip: rect(3.107em, 1001em, 3.602em, -1000em); top: -4.261em; left: 0.118em;"><span class="mrow" id="MathJax-Span-6"><span class="mo" id="MathJax-Span-7" style=""><span style="font-family: MathJax_Size2;">ˆ</span></span></span><span style="display: inline-block; width: 0px; height: 4.023em;"></span></span></span></span><span class="mo" id="MathJax-Span-8" style="font-family: MathJax_Main; padding-left: 0.222em;">−</span><span class="mi" id="MathJax-Span-9" style="font-family: MathJax_Math; font-style: italic; padding-left: 0.222em;">H<span style="display: inline-block; overflow: hidden; height: 1px; width: 0.057em;"></span></span></span></span></span></span></nobr></span><script type="math/mml" id="MathJax-Element-1"><math xmlns="http://www.w3.org/1998/Math/MathML" id="mml-ieqn-19">
                  <mover accent="true">
                     <mrow>
                        <mi>H</mi>
                     </mrow>
                     <mrow>
                        <mo> ˆ</mo>
                     </mrow>
                  </mover>
                  <mo>−</mo>
                  <mi>H</mi>
               </math></script></span>)/<i>H</i>), where <span class="inline-formula"><span class="MathJax_Preview" style="color: inherit;"><span class="MJXp-math" id="MJXp-Span-9"><span class="MJXp-mover" id="MJXp-Span-10"><span><span class="MJXp-over"><span class=""><span class="MJXp-mrow" id="MJXp-Span-13" style="margin-right: 0px; margin-left: 0px;"><span class="MJXp-mo" id="MJXp-Span-14" style="margin-left: 0em; margin-right: 0em;">ˆ</span></span></span><span class=""><span class="MJXp-mrow" id="MJXp-Span-11"><span class="MJXp-mi MJXp-italic" id="MJXp-Span-12">H</span></span></span></span></span></span></span></span><span class="MathJax MathJax_Processed" id="MathJax-Element-2-Frame" tabindex="0" style=""><nobr><span class="math" id="MathJax-Span-10"><span style="display: inline-block; position: relative; width: 0em; height: 0px; font-size: 116%;"><span style="position: absolute;"><span class="mrow" id="MathJax-Span-11"><span class="mover" id="MathJax-Span-12"><span style="display: inline-block; position: relative; width: 1.118em; height: 0px;"><span style="position: absolute; clip: rect(3.196em, 1000.89em, 4.167em, -1000em); top: -4.023em; left: 0.056em;"><span class="mrow" id="MathJax-Span-13"><span class="mi" id="MathJax-Span-14" style="font-family: MathJax_Math; font-style: italic;">H<span style="display: inline-block; overflow: hidden; height: 1px; width: 0.057em;"></span></span></span><span style="display: inline-block; width: 0px; height: 4.023em;"></span></span><span style="position: absolute; clip: rect(3.107em, 1001em, 3.602em, -1000em); top: -4.261em; left: 0.118em;"><span class="mrow" id="MathJax-Span-15"><span class="mo" id="MathJax-Span-16" style=""><span style="font-family: MathJax_Size2;">ˆ</span></span></span><span style="display: inline-block; width: 0px; height: 4.023em;"></span></span></span></span></span></span></span></span></nobr></span><script type="math/mml" id="MathJax-Element-2"><math xmlns="http://www.w3.org/1998/Math/MathML" id="mml-ieqn-20">
                  <mover accent="true">
                     <mrow>
                        <mi>H</mi>
                     </mrow>
                     <mrow>
                        <mo> ˆ</mo>
                     </mrow>
                  </mover>
               </math></script></span> is the estimate of a given metric in a sample, and <i>H</i>
 is the parametric value of a given estimator. The bias was estimated 
only once per each metric/population/iteration (i.e., 16,000 times). The
 error of the metric was estimated as a mean relative squared error 
(MRSE):</p>
         <p id="p-20" class="has-plink"><a class="plink" href="#p-20" rel="nofollow"><i class="icon-comment icon-flip-horizontal"></i></a><span class="inline-formula"><span class="MathJax_Preview" style="color: inherit;"><span class="MJXp-math" id="MJXp-Span-15"><span class="MJXp-mi MJXp-italic" id="MJXp-Span-16">M</span><span class="MJXp-mi MJXp-italic" id="MJXp-Span-17">R</span><span class="MJXp-mi MJXp-italic" id="MJXp-Span-18">S</span><span class="MJXp-mi MJXp-italic" id="MJXp-Span-19">E</span><span class="MJXp-mo" id="MJXp-Span-20" style="margin-left: 0.333em; margin-right: 0.333em;">=</span><span class="MJXp-mfrac" id="MJXp-Span-21" style="vertical-align: 0.25em;"><span class="MJXp-box MJXp-script"><span class="MJXp-mrow" id="MJXp-Span-22"><span class="MJXp-mn" id="MJXp-Span-23">1</span></span></span><span class="MJXp-box" style="margin-top: -0.9em;"><span class="MJXp-denom"><span><span class="MJXp-rule" style="height: 1em; border-top: medium none; border-bottom: 1px solid; margin: 0.1em 0px;"></span></span><span><span class="MJXp-box MJXp-script"><span class="MJXp-mrow" id="MJXp-Span-24"><span class="MJXp-mn" id="MJXp-Span-25">500</span></span></span></span></span></span></span><span class="MJXp-msubsup" id="MJXp-Span-26"><span class="MJXp-mrow" id="MJXp-Span-27" style="margin-right: 0.05em;"><span class="MJXp-mo" id="MJXp-Span-28" style="margin-left: 0.111em; margin-right: 0.167em;">∑</span></span><span class="MJXp-script-box" style="height: 1.86em; vertical-align: -0.64em;"><span class=" MJXp-script"><span><span style="margin-bottom: -0.25em;"><span class="MJXp-mrow" id="MJXp-Span-33"><span class="MJXp-mn" id="MJXp-Span-34">500</span></span></span></span></span><span class=" MJXp-script"><span><span style="margin-top: -0.85em;"><span class="MJXp-mrow" id="MJXp-Span-29"><span class="MJXp-mi MJXp-italic" id="MJXp-Span-30">i</span><span class="MJXp-mo" id="MJXp-Span-31">=</span><span class="MJXp-mn" id="MJXp-Span-32">1</span></span></span></span></span></span></span><span class="MJXp-mfrac" id="MJXp-Span-35" style="vertical-align: 0.25em;"><span class="MJXp-box MJXp-script"><span class="MJXp-mrow" id="MJXp-Span-36"><span class="MJXp-msup" id="MJXp-Span-37"><span class="MJXp-mrow" id="MJXp-Span-38" style="margin-right: 0.05em;"><span class="MJXp-mfenced" id="MJXp-Span-39"><span class="MJXp-mo" id="MJXp-Span-40" style="vertical-align: -0.644em;"><span style="font-size: 3.578em; margin-left: -0.17em;" class="MJXp-right MJXp-scale4">(</span></span><span class="MJXp-mrow" id="MJXp-Span-41"><span class="MJXp-msub" id="MJXp-Span-42"><span class="MJXp-mrow" id="MJXp-Span-43" style="margin-right: 0.05em;"><span class="MJXp-mover" id="MJXp-Span-44"><span><span class="MJXp-over"><span class=" MJXp-script"><span class="MJXp-mrow" id="MJXp-Span-47" style="margin-right: 0px; margin-left: 0px;"><span class="MJXp-mo" id="MJXp-Span-48">ˆ</span></span></span><span class=""><span class="MJXp-mrow" id="MJXp-Span-45"><span class="MJXp-mi MJXp-italic" id="MJXp-Span-46">H</span></span></span></span></span></span></span><span class="MJXp-mrow MJXp-script" id="MJXp-Span-49" style="vertical-align: -0.4em;"><span class="MJXp-mi MJXp-italic" id="MJXp-Span-50">i</span></span></span><span class="MJXp-mo" id="MJXp-Span-51">−</span><span class="MJXp-mi MJXp-italic" id="MJXp-Span-52">H</span></span><span class="MJXp-mo" id="MJXp-Span-53" style="vertical-align: -0.644em;"><span style="font-size: 3.578em; margin-left: -0.17em;" class="MJXp-right MJXp-scale4">)</span></span></span></span><span class="MJXp-mrow MJXp-script" id="MJXp-Span-54" style="vertical-align: 1.46em;"><span class="MJXp-mn" id="MJXp-Span-55">2</span></span></span></span></span><span class="MJXp-box" style="margin-top: -0.9em;"><span class="MJXp-denom"><span><span class="MJXp-rule" style="height: 1em; border-top: medium none; border-bottom: 1px solid; margin: 0.1em 0px;"></span></span><span><span class="MJXp-box MJXp-script"><span class="MJXp-mrow" id="MJXp-Span-56"><span class="MJXp-mi MJXp-italic" id="MJXp-Span-57">H</span></span></span></span></span></span></span></span></span><span class="MathJax MathJax_Processed" id="MathJax-Element-3-Frame" tabindex="0" style=""><nobr><span class="math" id="MathJax-Span-17"><span style="display: inline-block; position: relative; width: 0em; height: 0px; font-size: 116%;"><span style="position: absolute;"><span class="mrow" id="MathJax-Span-18"><span class="mi" id="MathJax-Span-19" style="font-family: MathJax_Math; font-style: italic;">M<span style="display: inline-block; overflow: hidden; height: 1px; width: 0.081em;"></span></span><span class="mi" id="MathJax-Span-20" style="font-family: MathJax_Math; font-style: italic;">R</span><span class="mi" id="MathJax-Span-21" style="font-family: MathJax_Math; font-style: italic;">S<span style="display: inline-block; overflow: hidden; height: 1px; width: 0.032em;"></span></span><span class="mi" id="MathJax-Span-22" style="font-family: MathJax_Math; font-style: italic;">E<span style="display: inline-block; overflow: hidden; height: 1px; width: 0.026em;"></span></span><span class="mo" id="MathJax-Span-23" style="font-family: MathJax_Main; padding-left: 0.278em;">=</span><span class="mfrac" id="MathJax-Span-24" style="padding-left: 0.278em;"><span style="display: inline-block; position: relative; width: 1.181em; height: 0px; margin-right: 0.12em; margin-left: 0.12em;"><span style="position: absolute; clip: rect(3.408em, 1000.3em, 4.167em, -1000em); top: -4.423em; left: 50%; margin-left: -0.177em;"><span class="mrow" id="MathJax-Span-25"><span class="mn" id="MathJax-Span-26" style="font-size: 70.7%; font-family: MathJax_Main;">1</span></span><span style="display: inline-block; width: 0px; height: 4.023em;"></span></span><span style="position: absolute; clip: rect(3.408em, 1001.03em, 4.182em, -1000em); top: -3.652em; left: 50%; margin-left: -0.53em;"><span class="mrow" id="MathJax-Span-27"><span class="mn" id="MathJax-Span-28" style="font-size: 70.7%; font-family: MathJax_Main;">500</span></span><span style="display: inline-block; width: 0px; height: 4.023em;"></span></span><span style="position: absolute; clip: rect(0.85em, 1001.18em, 1.197em, -1000em); top: -1.274em; left: 0em;"><span style="display: inline-block; overflow: hidden; vertical-align: 0em; border-top: 1.3px solid; width: 1.181em; height: 0px;"></span><span style="display: inline-block; width: 0px; height: 1.054em;"></span></span></span></span><span class="msubsup" id="MathJax-Span-29" style="padding-left: 0.167em;"><span style="display: inline-block; position: relative; width: 2.279em; height: 0px;"><span style="position: absolute; clip: rect(3.129em, 1001em, 4.417em, -1000em); top: -4.023em; left: 0em;"><span class="mrow" id="MathJax-Span-30"><span class="mo" id="MathJax-Span-31" style="font-family: MathJax_Size1; vertical-align: 0em;">∑</span></span><span style="display: inline-block; width: 0px; height: 4.023em;"></span></span><span style="position: absolute; clip: rect(3.408em, 1001.14em, 4.182em, -1000em); top: -4.5em; left: 1.056em;"><span class="mrow" id="MathJax-Span-32"><span class="mn" id="MathJax-Span-33" style="font-size: 70.7%; font-family: MathJax_Main;">500</span></span><span style="display: inline-block; width: 0px; height: 4.023em;"></span></span><span style="position: absolute; clip: rect(3.408em, 1001.22em, 4.174em, -1000em); top: -3.738em; left: 1.056em;"><span class="mrow" id="MathJax-Span-34"><span class="mi" id="MathJax-Span-35" style="font-size: 70.7%; font-family: MathJax_Math; font-style: italic;">i</span><span class="mo" id="MathJax-Span-36" style="font-size: 70.7%; font-family: MathJax_Main;">=</span><span class="mn" id="MathJax-Span-37" style="font-size: 70.7%; font-family: MathJax_Main;">1</span></span><span style="display: inline-block; width: 0px; height: 4.023em;"></span></span></span></span><span class="mfrac" id="MathJax-Span-38" style="padding-left: 0.167em;"><span style="display: inline-block; position: relative; width: 3.485em; height: 0px; margin-right: 0.12em; margin-left: 0.12em;"><span style="position: absolute; clip: rect(2.927em, 1003.37em, 4.626em, -1000em); top: -4.882em; left: 50%; margin-left: -1.682em;"><span class="mrow" id="MathJax-Span-39"><span class="msup" id="MathJax-Span-40"><span style="display: inline-block; position: relative; width: 3.365em; height: 0px;"><span style="position: absolute; clip: rect(3.066em, 1002.93em, 4.626em, -1000em); top: -4.023em; left: 0em;"><span class="mrow" id="MathJax-Span-41"><span class="mfenced" id="MathJax-Span-42"><span class="mo" id="MathJax-Span-43" style="vertical-align: 0em;"><span><span style="font-size: 70.7%; font-family: MathJax_Size2;">(</span></span></span><span class="mrow" id="MathJax-Span-44"><span class="msub" id="MathJax-Span-45"><span style="display: inline-block; position: relative; width: 1.039em; height: 0px;"><span style="position: absolute; clip: rect(3.13em, 1000.82em, 4.167em, -1000em); top: -4.023em; left: 0em;"><span class="mrow" id="MathJax-Span-46"><span class="mover" id="MathJax-Span-47"><span style="display: inline-block; position: relative; width: 0.814em; height: 0px;"><span style="position: absolute; clip: rect(3.396em, 1000.63em, 4.167em, -1000em); top: -4.023em; left: 0.04em;"><span class="mrow" id="MathJax-Span-48"><span class="mi" id="MathJax-Span-49" style="font-size: 70.7%; font-family: MathJax_Math; font-style: italic;">H<span style="display: inline-block; overflow: hidden; height: 1px; width: 0.04em;"></span></span></span><span style="display: inline-block; width: 0px; height: 4.023em;"></span></span><span style="position: absolute; clip: rect(3.333em, 1000.71em, 3.767em, -1000em); top: -4.226em; left: 0.107em;"><span class="mrow" id="MathJax-Span-50"><span class="mo" id="MathJax-Span-51" style=""><span><span style="font-size: 70.7%; font-family: MathJax_Size2;">ˆ</span></span></span></span><span style="display: inline-block; width: 0px; height: 4.023em;"></span></span></span></span></span><span style="display: inline-block; width: 0px; height: 4.023em;"></span></span><span style="position: absolute; top: -3.917em; left: 0.814em;"><span class="mrow" id="MathJax-Span-52"><span class="mi" id="MathJax-Span-53" style="font-size: 50%; font-family: MathJax_Math; font-style: italic;">i</span></span><span style="display: inline-block; width: 0px; height: 4.023em;"></span></span></span></span><span class="mo" id="MathJax-Span-54" style="font-size: 70.7%; font-family: MathJax_Main;">−</span><span class="mi" id="MathJax-Span-55" style="font-size: 70.7%; font-family: MathJax_Math; font-style: italic;">H<span style="display: inline-block; overflow: hidden; height: 1px; width: 0.04em;"></span></span></span><span class="mo" id="MathJax-Span-56" style="vertical-align: 0em;"><span><span style="font-size: 70.7%; font-family: MathJax_Size2;">)</span></span></span></span></span><span style="display: inline-block; width: 0px; height: 4.023em;"></span></span><span style="position: absolute; top: -4.643em; left: 3.062em;"><span class="mrow" id="MathJax-Span-57"><span class="mn" id="MathJax-Span-58" style="font-size: 50%; font-family: MathJax_Main;">2</span></span><span style="display: inline-block; width: 0px; height: 4.023em;"></span></span></span></span></span><span style="display: inline-block; width: 0px; height: 4.023em;"></span></span><span style="position: absolute; clip: rect(3.396em, 1000.63em, 4.167em, -1000em); top: -3.64em; left: 50%; margin-left: -0.314em;"><span class="mrow" id="MathJax-Span-59"><span class="mi" id="MathJax-Span-60" style="font-size: 70.7%; font-family: MathJax_Math; font-style: italic;">H<span style="display: inline-block; overflow: hidden; height: 1px; width: 0.04em;"></span></span></span><span style="display: inline-block; width: 0px; height: 4.023em;"></span></span><span style="position: absolute; clip: rect(0.85em, 1003.49em, 1.197em, -1000em); top: -1.274em; left: 0em;"><span style="display: inline-block; overflow: hidden; vertical-align: 0em; border-top: 1.3px solid; width: 3.485em; height: 0px;"></span><span style="display: inline-block; width: 0px; height: 1.054em;"></span></span></span></span></span></span></span></span></nobr></span><script type="math/mml" id="MathJax-Element-3"><math xmlns="http://www.w3.org/1998/Math/MathML" id="mml-ieqn-21">
                  <mi>M</mi>
                  <mi>R</mi>
                  <mi>S</mi>
                  <mi>E</mi>
                  <mo>=</mo>
                  <mfrac>
                     <mrow>
                        <mn>1</mn>
                     </mrow>
                     <mrow>
                        <mn>500</mn>
                     </mrow>
                  </mfrac>
                  <msubsup>
                     <mrow>
                        <mo>∑</mo>
                     </mrow>
                     <mrow>
                        <mi>i</mi>
                        <mo>=</mo>
                        <mn>1</mn>
                     </mrow>
                     <mrow>
                        <mn>500</mn>
                     </mrow>
                  </msubsup>
                  <mfrac>
                     <mrow>
                        <msup>
                           <mrow>
                              <mfenced separators="" open="(" close=")">
                                 <mrow>
                                    <msub>
                                       <mrow>
                                          <mover accent="true">
                                             <mrow>
                                                <mi>H</mi>
                                             </mrow>
                                             <mrow>
                                                <mo> ˆ</mo>
                                             </mrow>
                                          </mover>
                                       </mrow>
                                       <mrow>
                                          <mi>i</mi>
                                       </mrow>
                                    </msub>
                                    <mo>−</mo>
                                    <mi>H</mi>
                                 </mrow>
                              </mfenced>
                           </mrow>
                           <mrow>
                              <mn>2</mn>
                           </mrow>
                        </msup>
                     </mrow>
                     <mrow>
                        <mi>H</mi>
                     </mrow>
                  </mfrac>
               </math></script></span>, where <span class="inline-formula"><span class="MathJax_Preview" style="color: inherit;"><span class="MJXp-math" id="MJXp-Span-58"><span class="MJXp-mover" id="MJXp-Span-59"><span><span class="MJXp-over"><span class=""><span class="MJXp-mrow" id="MJXp-Span-62" style="margin-right: 0px; margin-left: 0px;"><span class="MJXp-mo" id="MJXp-Span-63" style="margin-left: 0em; margin-right: 0em;">ˆ</span></span></span><span class=""><span class="MJXp-mrow" id="MJXp-Span-60"><span class="MJXp-mi MJXp-italic" id="MJXp-Span-61">H</span></span></span></span></span></span></span></span><span class="MathJax MathJax_Processed" id="MathJax-Element-4-Frame" tabindex="0" style=""><nobr><span class="math" id="MathJax-Span-61"><span style="display: inline-block; position: relative; width: 0em; height: 0px; font-size: 116%;"><span style="position: absolute;"><span class="mrow" id="MathJax-Span-62"><span class="mover" id="MathJax-Span-63"><span style="display: inline-block; position: relative; width: 1.118em; height: 0px;"><span style="position: absolute; clip: rect(3.196em, 1000.89em, 4.167em, -1000em); top: -4.023em; left: 0.056em;"><span class="mrow" id="MathJax-Span-64"><span class="mi" id="MathJax-Span-65" style="font-family: MathJax_Math; font-style: italic;">H<span style="display: inline-block; overflow: hidden; height: 1px; width: 0.057em;"></span></span></span><span style="display: inline-block; width: 0px; height: 4.023em;"></span></span><span style="position: absolute; clip: rect(3.107em, 1001em, 3.602em, -1000em); top: -4.261em; left: 0.118em;"><span class="mrow" id="MathJax-Span-66"><span class="mo" id="MathJax-Span-67" style=""><span style="font-family: MathJax_Size2;">ˆ</span></span></span><span style="display: inline-block; width: 0px; height: 4.023em;"></span></span></span></span></span></span></span></span></nobr></span><script type="math/mml" id="MathJax-Element-4"><math xmlns="http://www.w3.org/1998/Math/MathML" id="mml-ieqn-22">
                  <mover accent="true">
                     <mrow>
                        <mi>H</mi>
                     </mrow>
                     <mrow>
                        <mo> ˆ</mo>
                     </mrow>
                  </mover>
               </math></script></span><sub><i>i</i></sub> is an estimate of a metric in the <i>i</i>-th sample of the 500 resamplings that were carried out to estimate the mean.</p>
         <p id="p-21" class="has-plink"><a class="plink" href="#p-21" rel="nofollow"><i class="icon-comment icon-flip-horizontal"></i></a>To explore the factors influencing the errors of the analysed indices, the generalised linear model, <i>glm</i>, a function from R’s <i>stats</i> package, was used. The model included <i>MRSE</i> as the response and four explanatory variables: <i>H</i>
 estimator, sample size, population gene diversity and locus. To avoid 
overparametrisation, the GLM analyses were performed in two steps. 
Firstly, <i>MRSE</i> of mean <i>H</i> values over the 24 simulated loci were provided to the model as an effect; secondly, <i>MRSE</i> calculated for each locus separately was used.</p>
         <p id="p-22" class="has-plink"><a class="plink" href="#p-22" rel="nofollow"><i class="icon-comment icon-flip-horizontal"></i></a>Because the relation between sample size and <i>MRSE</i>
 is asymptotic, and the effect size may depend on arbitrarily selected 
number of samples, the values were provided to the model as categorical 
values (factors). The best-fitting model was selected based on the 
Akaike Information Criterion (AICc) as implemented in the <i>model.sel</i> function from the R package <i>MuMIn</i> (<a class="xref xref-bibr" href="https://cran.r-project.org/package=MuMIn" title="" data-jats-ref-type="bibr" data-jats-rid="ref-2" data-original-title="">Bartoń, 2019</a>). The GLM results were analysed using the <i>Anova</i> function from the package <i>car</i> (<a class="xref xref-bibr" href="https://scholar.google.com/scholar_lookup?title=An%20R%20companion%20to%20applied%20regression&amp;author=Fox&amp;publication_year=2019" title="" data-jats-ref-type="bibr" data-jats-rid="ref-14" data-original-title="An R companion to applied regression">Fox &amp; Weisberg, 2019</a>). The effects of the explanatory variables were visualised using the R package <i>effects</i> (<a class="xref xref-bibr" href="https://doi.org/10.18637%2Fjss.v087.i09" title="" data-jats-ref-type="bibr" data-jats-rid="ref-13" data-original-title="Visualizing fit and lack of fit in complex regression models with predictor effect plots and partial residuals">Fox &amp; Weisberg, 2018</a>; <a class="xref xref-bibr" href="https://scholar.google.com/scholar_lookup?title=An%20R%20companion%20to%20applied%20regression&amp;author=Fox&amp;publication_year=2019" title="" data-jats-ref-type="bibr" data-jats-rid="ref-14" data-original-title="An R companion to applied regression">Fox &amp; Weisberg, 2019</a>). Tukey’s Honest Significant Difference (Tukey’s HSD; <a class="xref xref-bibr" href="https://scholar.google.com/scholar_lookup?title=Exploratory%20data%20analysis&amp;author=Tukey&amp;publication_year=1977" title="" data-jats-ref-type="bibr" data-jats-rid="ref-40" data-original-title="Exploratory data analysis">Tukey, 1977</a>) method was used to test whether the differences between the factors were significant. The function <i>glht</i> from the R package <i>multcomp</i> (<a class="xref xref-bibr" href="https://doi.org/10.1002%2Fbimj.200810425" title="" data-jats-ref-type="bibr" data-jats-rid="ref-19" data-original-title="Simultaneous inference in general parametric models">Hothorn, Bretz &amp; Westfall, 2008</a>) was used to perform the analysis, while <i>cld</i> was used to summarise results and present them as compact letter displays (<a class="xref xref-bibr" href="https://doi.org/10.1198%2F1061860043515" title="" data-jats-ref-type="bibr" data-jats-rid="ref-32" data-original-title="An algorithm for a letter-based representation of all-pairwise comparisons">Piepho, 2004</a>). The script used for simulations is deposited at Github (<a class="ext-link" href="https://github.com/konopinski/Shannon/" data-jats-ext-link-type="uri" data-original-title="" title="">https://github.com/konopinski/Shannon/</a>).</p>
      </section><section class="sec" id="results">
         <h2 class="heading">Results</h2>
         <p id="p-23" class="has-plink"><a class="plink" href="#p-23" rel="nofollow"><i class="icon-comment icon-flip-horizontal"></i></a>Each metric was estimated altogether 192 096 000 times in 24 loci, 4 populations (<i>P</i><sub><i>C</i></sub>, <i>P</i><sub>500</sub>, <i>P</i><sub>50</sub>, <i>P</i><sub>20</sub>),
 5 sample sizes (i.e., 5, 20, 80 and 200 genotypes and for the whole 
simulated population to obtain parametric values), 500 randomizations 
and 1000 simulation repetitions. Mean expected heterozygosities 
calculated for the simulated total populations ranged from <i>He</i> = 0.318 to <i>He</i> = 0.789 with median <i>He</i> = 0.683. The parametric values of the four <i>H</i> indices were similar within each of the simulated demographic scenario both in terms of their median values and their ranges (<a class="xref xref-table" href="#table-2" data-jats-ref-type="table" data-jats-rid="table-2">Table 2</a>).</p>
         <figure class="table-wrap has-plink" id="table-2"><a class="plink" href="#table-2" rel="nofollow"><i class="icon-comment icon-flip-horizontal"></i></a><div class="caption">
<span class="caption-label">Table 2: </span>
               <div class="title">Summary of the simulations. Minimum, maximum and median values of the <i>H</i> estimators and the Nei’s gene diversity (<i>Hs</i>) calculated for the four simulated demographic scenarios.</div>
            </div>
            
            <div class="table-container"><table class="table table-bordered table-condensed table-hover">
                  <colgroup>
                     <col>
                     <col>
                     <col>
                     <col>
                     <col>
                     <col>
                     <col>
                  </colgroup>
                  <thead>
                     <tr>
                        <th>Population</th>
                        <th></th>
                        <th>
<i>H</i><sub><i>MLE</i></sub><i></i>
</th>
                        <th>
<i>H</i><sub><i>Z</i></sub><i></i>
</th>
                        <th>
<i>H</i><sub><i>CS</i></sub><i></i>
</th>
                        <th>
<i>H</i><sub><i>Chao</i></sub><i></i>
</th>
                        <th><i>Hs</i></th>
                     </tr>
                  </thead>
                  <tbody>
                     <tr>
                        <td></td>
                        <td>min</td>
                        <td>1.5159</td>
                        <td>1.5161</td>
                        <td>1.5160</td>
                        <td>1.5161</td>
                        <td>0.6996</td>
                     </tr>
                     <tr>
                        <td>
<i>P</i><sub><i>C</i></sub><i></i>
</td>
                        <td>median</td>
                        <td>1.6706</td>
                        <td>1.6708</td>
                        <td>1.6706</td>
                        <td>1.6708</td>
                        <td>0.7568</td>
                     </tr>
                     <tr>
                        <td></td>
                        <td>max</td>
                        <td>1.7834</td>
                        <td>1.7836</td>
                        <td>1.7834</td>
                        <td>1.7836</td>
                        <td>0.7889</td>
                     </tr>
                     <tr>
                        <td></td>
                        <td>min</td>
                        <td>1.4220</td>
                        <td>1.4221</td>
                        <td>1.4221</td>
                        <td>1.4221</td>
                        <td>0.6764</td>
                     </tr>
                     <tr>
                        <td>
<i>P</i><sub>500</sub><i></i>
</td>
                        <td>median</td>
                        <td>1.6075</td>
                        <td>1.6077</td>
                        <td>1.6075</td>
                        <td>1.6077</td>
                        <td>0.7428</td>
                     </tr>
                     <tr>
                        <td></td>
                        <td>max</td>
                        <td>1.7409</td>
                        <td>1.7411</td>
                        <td>1.7409</td>
                        <td>1.7411</td>
                        <td>0.7795</td>
                     </tr>
                     <tr>
                        <td></td>
                        <td>min</td>
                        <td>0.9940</td>
                        <td>0.9941</td>
                        <td>0.9941</td>
                        <td>0.9941</td>
                        <td>0.5347</td>
                     </tr>
                     <tr>
                        <td>
<i>P</i><sub>50</sub><i></i>
</td>
                        <td>median</td>
                        <td>1.2007</td>
                        <td>1.2009</td>
                        <td>1.2008</td>
                        <td>1.2009</td>
                        <td>0.6266</td>
                     </tr>
                     <tr>
                        <td></td>
                        <td>max</td>
                        <td>1.3441</td>
                        <td>1.3443</td>
                        <td>1.3442</td>
                        <td>1.3443</td>
                        <td>0.6843</td>
                     </tr>
                     <tr>
                        <td></td>
                        <td>min</td>
                        <td>0.5927</td>
                        <td>0.5928</td>
                        <td>0.5929</td>
                        <td>0.5928</td>
                        <td>0.3179</td>
                     </tr>
                     <tr>
                        <td>
<i>P</i><sub>20</sub><i></i>
</td>
                        <td>median</td>
                        <td>0.8281</td>
                        <td>0.8282</td>
                        <td>0.8282</td>
                        <td>0.8282</td>
                        <td>0.4731</td>
                     </tr>
                     <tr>
                        <td></td>
                        <td>max</td>
                        <td>1.0077</td>
                        <td>1.0079</td>
                        <td>1.0079</td>
                        <td>1.0079</td>
                        <td>0.5734</td>
                     </tr>
                  </tbody>
               </table></div>
<div class="object-id article-component-doi">DOI: <a href="https://doi.org/10.7717/peerj.9391/table-2" data-toggle="tooltip" title="" data-original-title="Cite this object using this DOI">10.7717/peerj.9391/table-2</a>
</div>
         </figure>
         <p id="p-24" class="has-plink"><a class="plink" href="#p-24" rel="nofollow"><i class="icon-comment icon-flip-horizontal"></i></a>Attempts to estimate the sampling variance of <i>H</i><sub><i>CS</i></sub>
 failed in 1,689 out of 16,000 resampling attempts, i.e., roughly 10%. 
The problem occurred only in the smallest simulated samples (<i>Ns</i> = 5) and only in the most variable populations: 937 in <i>P</i><sub><i>C</i></sub> and 752 in <i>P</i><sub>500</sub>. Similarly, mean <i>H</i><sub><i>CS</i></sub>
 could not be estimated in 10 cases in the 16 000 population sampling 
simulations extracted from the whole set to estimate bias. The problem 
occurred only in the most variable populations (7 failures in <i>P</i><sub><i>C</i></sub> and 3 in <i>P</i><sub>500</sub>)
 and in the smallest sample size. The loci that caused the problem were 
those simulated with a large number of possible allelic states (12, 15 
and 20 alleles) and the problem was more frequent in the loci with 
higher mutation rates (<a class="xref xref-supplementary-material" href="#supp-1" data-jats-ref-type="supplementary-material" data-jats-rid="supp-1">Table S1</a>). Due to the large proportion of missing data, the estimates of <i>H</i><sub><i>CS</i></sub> from the populations <i>P</i><sub><i>C</i></sub> and <i>P</i><sub>500</sub> simulated with the smallest sample size <i>Ns = 5</i> were excluded from the sampling variance comparisons, and the 10 samples that failed at estimation of <i>H</i><sub><i>CS</i></sub> in the simulations of population sampling were excluded from assessment of the performance of the <i>H</i> indices.</p>
         <p id="p-25" class="has-plink"><a class="plink" href="#p-25" rel="nofollow"><i class="icon-comment icon-flip-horizontal"></i></a>The standard deviation of the results distribution from the repeated sampling depended on sample size, demographic scenario and <i>H</i> estimator (<a class="xref xref-fig" href="#fig-1" data-jats-ref-type="fig" data-jats-rid="fig-1" data-original-title="" title="">Fig. 1</a>). Standard deviations of <i>H</i><sub><i>MLE</i></sub> were significantly lower in all pairwise comparisons (<a class="xref xref-table" href="#table-3" data-jats-ref-type="table" data-jats-rid="table-3">Table 3</a>). Among the remaining metrics, the estimates of <i>H</i><sub><i>Z</i></sub> had significantly narrower distribution then <i>H</i><sub><i>Chao</i></sub> in control populations (<i>P</i><sub><i>C</i></sub>) and the populations that underwent a bottleneck of 500 genotypes (<i>P</i><sub>500</sub>). In larger sample sizes, <i>Ns</i> = 20, 80 and 200, the distributions of standard deviations of <i>H</i><sub><i>Z</i></sub>, <i>H</i><sub><i>Chao</i></sub> and <i>H</i><sub><i>CS</i></sub> were not significantly different in any of the pairwise comparisons.</p>
         <p id="p-26" class="has-plink"><a class="plink" href="#p-26" rel="nofollow"><i class="icon-comment icon-flip-horizontal"></i></a>The median relative bias (<i>rB</i>) of the <i>H</i> estimates averaged over the 24 loci was inversely associated with the sample size in all cases (<a class="xref xref-fig" href="#fig-2" data-jats-ref-type="fig" data-jats-rid="fig-2" data-original-title="" title="">Fig. 2</a>, <a class="xref xref-table" href="#table-4" data-jats-ref-type="table" data-jats-rid="table-4">Table 4</a>). As compared to other <i>H</i>
 estimators at all sample sizes, the strongest negative departure from 
parametric values (i.e., calculated from the total population) was 
observed in <i>H</i><sub><i>MLE</i></sub> estimates. Among the remaining three <i>H</i> estimates, <i>H</i><sub><i>Chao</i></sub> and <i>H</i><sub><i>Z</i></sub> were the least biased (<a class="xref xref-table" href="#table-4" data-jats-ref-type="table" data-jats-rid="table-4">Table 4</a>). Except for the <i>H</i><sub><i>MLE</i></sub> estimates at sample sizes below 80 genotypes, 95% confidence intervals of <i>H</i>
 estimators always spanned the parametric value of the simulated data. 
The observed relative bias ranges were markedly wider in the smallest 
samples.</p>
         <figure class="fig has-plink" itemprop="image" itemscope="itemscope" itemtype="https://schema.org/ImageObject" id="fig-1"><a class="plink" href="#fig-1" rel="nofollow"><i class="icon-comment icon-flip-horizontal"></i></a><div class="image-container"><a href="https://dfzljdn9uc3pi.cloudfront.net/2020/9391/1/fig-1-2x.jpg" title="View the full image" class="fresco" data-fresco-caption="Figure 1: Density plots of SD distributions obtained through repeated sampling of Ns&nbsp;=&nbsp;5, 20, 80 and 200 genotypes from each population representing different demographic scenario in each simulation." data-fresco-group="figure" data-fresco-options="fit: 'width', ui: 'outside', thumbnails: false, loop: true, position: true, overflow: true, preload: false"><img class="graphic" src="fig-1-1x.jpg" itemprop="contentUrl" sizes="(min-width: 1200px) 581px, (max-width: 1199px) and (min-width: 980px) 462px, (max-width: 979px) and (min-width: 768px) 347px, (max-width: 767px) calc(100vw - 50px)" srcset="https://dfzljdn9uc3pi.cloudfront.net/2020/9391/1/fig-1-2x.jpg 1200w, https://dfzljdn9uc3pi.cloudfront.net/2020/9391/1/fig-1-1x.jpg 600w, https://dfzljdn9uc3pi.cloudfront.net/2020/9391/1/fig-1-small.jpg 355w" data-image-id="fig-1" alt="Density plots of SD distributions obtained through repeated sampling of Ns&nbsp;=&nbsp;5, 20, 80 and 200 genotypes from each population representing different demographic scenario in each simulation." data-full="https://dfzljdn9uc3pi.cloudfront.net/2020/9391/1/fig-1-full.png" data-thumb="https://dfzljdn9uc3pi.cloudfront.net/2020/9391/1/fig-1-thumb.jpg" data-original="https://dfzljdn9uc3pi.cloudfront.net/2020/9391/1/fig-1.png" data-image-type="figure" data-jats-mimetype="image" data-jats-mime-subtype="png" width="594" height="600"></a></div>
<figcaption itemprop="description">
               <h3 class="heading">
<span class="caption-label">Figure 1: </span>Density plots of SD distributions obtained through repeated sampling of <i>Ns</i>&nbsp;=&nbsp;5, 20, 80 and 200 genotypes from each population representing different demographic scenario in each simulation.</h3>
               <span class="p">Sample sizes (columns): (A–D) <i>Ns</i>&nbsp;=&nbsp;5, (E–H) <i>Ns</i>&nbsp;=&nbsp;20, (I–L) <i>Ns</i>&nbsp;=&nbsp;80, (M–P) <i>Ns</i>&nbsp;=&nbsp;200. Demographic scenarios (rows): (A), (E), (I), (M) <i>P</i><sub>20</sub>, (B), (F), (J), (N) <i>P</i><sub>50</sub>, (C), (G) ,(K), (O) <i>P</i><sub>500</sub>, (D), (H), (L), (P) <i>P</i><sub><i>C</i></sub>.</span>
            <div class="figcaption-footer">
<div class="article-image-download"><a href="https://dfzljdn9uc3pi.cloudfront.net/2020/9391/1/fig-1-full.png" class="btn btn-mini" download="peerj-9391-fig-1.png" itemprop="url"><i class="icon-large icon-picture">&nbsp;</i> Download full-size image</a></div>
<div class="object-id article-component-doi">DOI: <a href="https://doi.org/10.7717/peerj.9391/fig-1" data-toggle="tooltip" title="" data-original-title="Cite this object using this DOI">10.7717/peerj.9391/fig-1</a>
</div>
</div></figcaption></figure>
         <p id="p-28" class="has-plink"><a class="plink" href="#p-28" rel="nofollow"><i class="icon-comment icon-flip-horizontal"></i></a>The analyses of relative error (<i>MRSE</i>) provided similar findings. Based on AICc summarised by MuMIn function, the gamma distribution of <i>MRSE</i>
 with a log link function was used in the GLM. According to ANOVA test 
of the GLM results, all four factors—locus, metric, sample size and 
expected heterozygosity of the total population (<i>He</i>)—were significantly associated with the error levels (<i>p</i> = 10<sup>−15</sup>). The median <i>MRSE</i> of <i>H</i> estimators was negatively associated with the sample size. When compared to <i>Ns</i> = 200 genotypes, the slope of the relation, <i>β</i>, increased on decreasing sample size, from <i>β</i> = 0.97 for <i>Ns</i> = 80 genotypes, to <i>β</i>&nbsp;=&nbsp;4.46 for <i>Ns</i> = 5 individuals (<i>p</i>&nbsp;=&nbsp;10<sup>−15</sup>).
 Tukey’s HSD analysis of GLM results confirmed the differences between 
error levels among all the different sample sizes were significant for 
all metrics. The strongest effect of sample size on <i>MRSE</i> was observed for <i>H</i><sub><i>MLE</i></sub> (<a class="xref xref-fig" href="#fig-3" data-jats-ref-type="fig" data-jats-rid="fig-3" data-original-title="" title="">Fig. 3</a>). The remaining <i>H</i> estimators were markedly less affected by a small sample size with <i>H</i><sub><i>CS</i></sub> performing slightly worse than <i>H</i><sub><i>Z</i></sub> and <i>H</i><sub><i>Chao</i></sub>. Analysis of GLM results using Tukey’s HSD showed that among all the metrics, <i>H</i><sub><i>Chao</i></sub> and <i>H</i><sub><i>Z</i></sub> were significantly less affected by error than the other two estimators at the majority of sample sizes (<a class="xref xref-table" href="#table-5" data-jats-ref-type="table" data-jats-rid="table-5">Table 5</a>). Only at the smallest sample size, the difference between <i>H</i><sub><i>Chao</i></sub>, <i>H</i><sub><i>Z</i></sub> and <i>H</i><sub><i>CS</i></sub> was not significant. The error levels were also strongly negatively associated with the <i>He</i> of the total population (<i>β</i> = −0.71, <i>p</i>&nbsp;=&nbsp;10<sup>−15</sup>, <a class="xref xref-fig" href="#fig-4" data-jats-ref-type="fig" data-jats-rid="fig-4" data-original-title="" title="">Fig. 4</a>). In the case of <i>H</i><sub><i>MLE</i></sub> and <i>H</i><sub><i>Z</i></sub>, the effect depended on the sample size, being, positive at the smaller sample sizes: <i>Ns</i>  ≤ 80 in <i>H</i><sub><i>MLE</i></sub> and Ns = 5 in <i>H</i><sub><i>Z</i></sub>.</p>
         <figure class="table-wrap has-plink" id="table-3"><a class="plink" href="#table-3" rel="nofollow"><i class="icon-comment icon-flip-horizontal"></i></a><div class="caption">
<span class="caption-label">Table 3: </span>
               <div class="title">Sampling variance of the <i>H</i> estimators calculated for <i>Ns</i>= 5 genotypes. Mean <i>SD</i> of each estimator in each demographic scenario, and the <i>p</i>-values of pairwise comparisons of the <i>SD</i> distributions.</div>
            </div>
            
            <div class="table-container"><table class="table table-bordered table-condensed table-hover">
                  <colgroup>
                     <col>
                     <col>
                     <col>
                     <col>
                     <col>
                     <col>
                  </colgroup>
                  <thead>
                     <tr>
                        <th>Population</th>
                        <th>Metric</th>
                        <th>Mean SD of metric</th>
                        <th style="text-align:center;" colspan="3">
<i>p</i>-values</th>
                     </tr>
                     <tr>
                        <th></th>
                        <th></th>
                        <th></th>
                        <th>
<i>H</i><sub><i>Z</i></sub><i></i>
</th>
                        <th>
<i>H</i><sub><i>CS</i></sub><i></i>
</th>
                        <th>
<i>H</i><sub><i>Chao</i></sub><i></i>
</th>
                     </tr>
                  </thead>
                  <tbody>
                     <tr>
                        <td>
<i>P</i><sub><i>C</i></sub><i></i>
</td>
                        <td>
<i>H</i><sub><i>MLE</i></sub><i></i>
</td>
                        <td>0.0436</td>
                        <td>10<sup>−5</sup>
</td>
                        <td>–</td>
                        <td>10<sup>−5</sup>
</td>
                     </tr>
                     <tr>
                        <td></td>
                        <td>
<i>H</i><sub><i>Z</i></sub><i></i>
</td>
                        <td>0.0599</td>
                        <td></td>
                        <td>–</td>
                        <td>0.0038</td>
                     </tr>
                     <tr>
                        <td></td>
                        <td>
<i>H</i><sub><i>CS</i></sub><i></i>
</td>
                        <td>–</td>
                        <td></td>
                        <td></td>
                        <td>–</td>
                     </tr>
                     <tr>
                        <td></td>
                        <td>
<i>H</i><sub><i>Chao</i></sub><i></i>
</td>
                        <td>0.0704</td>
                        <td></td>
                        <td></td>
                        <td></td>
                     </tr>
                     <tr>
                        <td><i></i></td>
                        <td><i></i></td>
                        <td></td>
                        <td></td>
                        <td></td>
                        <td></td>
                     </tr>
                     <tr>
                        <td>
<i>P</i><sub>500</sub><i></i>
</td>
                        <td>
<i>H</i><sub><i>MLE</i></sub><i></i>
</td>
                        <td>0.0439</td>
                        <td>10<sup>−5</sup>
</td>
                        <td>–</td>
                        <td>10<sup>−5</sup>
</td>
                     </tr>
                     <tr>
                        <td></td>
                        <td>
<i>H</i><sub><i>Z</i></sub><i></i>
</td>
                        <td>0.0596</td>
                        <td></td>
                        <td>–</td>
                        <td>0.0191</td>
                     </tr>
                     <tr>
                        <td></td>
                        <td>
<i>H</i><sub><i>CS</i></sub><i></i>
</td>
                        <td>–</td>
                        <td></td>
                        <td></td>
                        <td>–</td>
                     </tr>
                     <tr>
                        <td></td>
                        <td>
<i>H</i><sub><i>Chao</i></sub><i></i>
</td>
                        <td>0.0683</td>
                        <td></td>
                        <td></td>
                        <td></td>
                     </tr>
                     <tr>
                        <td><i></i></td>
                        <td><i></i></td>
                        <td></td>
                        <td></td>
                        <td></td>
                        <td></td>
                     </tr>
                     <tr>
                        <td>
<i>P</i><sub>50</sub><i></i>
</td>
                        <td>
<i>H</i><sub><i>MLE</i></sub><i></i>
</td>
                        <td>0.0439</td>
                        <td>0.0001</td>
                        <td>0.0004</td>
                        <td>0.0016</td>
                     </tr>
                     <tr>
                        <td></td>
                        <td>
<i>H</i><sub><i>Z</i></sub><i></i>
</td>
                        <td>0.0547</td>
                        <td></td>
                        <td>0.6397</td>
                        <td>0.5189</td>
                     </tr>
                     <tr>
                        <td></td>
                        <td>
<i>H</i><sub><i>CS</i></sub><i></i>
</td>
                        <td>0.0565</td>
                        <td></td>
                        <td></td>
                        <td>0.8547</td>
                     </tr>
                     <tr>
                        <td></td>
                        <td>
<i>H</i><sub><i>Chao</i></sub><i></i>
</td>
                        <td>0.0572</td>
                        <td></td>
                        <td></td>
                        <td></td>
                     </tr>
                     <tr>
                        <td><i></i></td>
                        <td><i></i></td>
                        <td></td>
                        <td></td>
                        <td></td>
                        <td></td>
                     </tr>
                     <tr>
                        <td>
<i>P</i><sub>20</sub><i></i>
</td>
                        <td>
<i>H</i><sub><i>MLE</i></sub><i></i>
</td>
                        <td>0.0399</td>
                        <td>0.0418</td>
                        <td>0.0345</td>
                        <td>0.0498</td>
                     </tr>
                     <tr>
                        <td></td>
                        <td>
<i>H</i><sub><i>Z</i></sub><i></i>
</td>
                        <td>0.0472</td>
                        <td></td>
                        <td>0.8478</td>
                        <td>0.9290</td>
                     </tr>
                     <tr>
                        <td></td>
                        <td>
<i>H</i><sub><i>CS</i></sub><i></i>
</td>
                        <td>0.0480</td>
                        <td></td>
                        <td></td>
                        <td>0.9344</td>
                     </tr>
                     <tr>
                        <td></td>
                        <td>
<i>H</i><sub><i>Chao</i></sub><i></i>
</td>
                        <td>0.0479</td>
                        <td></td>
                        <td></td>
                        <td></td>
                     </tr>
                  </tbody>
               </table></div>
<div class="object-id article-component-doi">DOI: <a href="https://doi.org/10.7717/peerj.9391/table-3" data-toggle="tooltip" title="" data-original-title="Cite this object using this DOI">10.7717/peerj.9391/table-3</a>
</div>
         </figure>
         <figure class="fig has-plink" itemprop="image" itemscope="itemscope" itemtype="https://schema.org/ImageObject" id="fig-2"><a class="plink" href="#fig-2" rel="nofollow"><i class="icon-comment icon-flip-horizontal"></i></a><div class="image-container"><a href="https://dfzljdn9uc3pi.cloudfront.net/2020/9391/1/fig-2-2x.jpg" title="View the full image" class="fresco" data-fresco-caption="Figure 2: Box-whisker plot of relative bias (rB) of the four Shannon H estimators in all the simulated demographic scenarios and sample sizes. (A) HMLE, (B) HMLE, (C) HCS, (D) HChao." data-fresco-group="figure" data-fresco-options="fit: 'width', ui: 'outside', thumbnails: false, loop: true, position: true, overflow: true, preload: false"><img class="graphic" src="fig-2-1x.jpg" itemprop="contentUrl" sizes="(min-width: 1200px) 581px, (max-width: 1199px) and (min-width: 980px) 462px, (max-width: 979px) and (min-width: 768px) 347px, (max-width: 767px) calc(100vw - 50px)" srcset="https://dfzljdn9uc3pi.cloudfront.net/2020/9391/1/fig-2-2x.jpg 1200w, https://dfzljdn9uc3pi.cloudfront.net/2020/9391/1/fig-2-1x.jpg 600w, https://dfzljdn9uc3pi.cloudfront.net/2020/9391/1/fig-2-small.jpg 355w" data-image-id="fig-2" alt="Box-whisker plot of relative bias (rB) of the four Shannon H estimators in all the simulated demographic scenarios and sample sizes. (A) HMLE, (B) HMLE, (C) HCS, (D) HChao." data-full="https://dfzljdn9uc3pi.cloudfront.net/2020/9391/1/fig-2-full.png" data-thumb="https://dfzljdn9uc3pi.cloudfront.net/2020/9391/1/fig-2-thumb.jpg" data-original="https://dfzljdn9uc3pi.cloudfront.net/2020/9391/1/fig-2.png" data-image-type="figure" data-jats-mimetype="image" data-jats-mime-subtype="png" width="583" height="600"></a></div>
<figcaption itemprop="description">
               <h3 class="heading">
<span class="caption-label">Figure 2: </span>Box-whisker plot of relative bias (<i>rB</i>) of the four Shannon <i>H</i> estimators in all the simulated demographic scenarios and sample sizes. (A) <i>H</i><sub><i>MLE</i></sub>, (B) <i>H</i><sub><i>MLE</i></sub>, (C) <i>H</i><sub><i>CS</i></sub>, (D) <i>H</i><sub><i>Chao</i></sub>.</h3>
               <span class="p">The whiskers represent the range of 
maximum and minimum values, the top and bottom of the boxes represent 
the 75% and 25% quartiles. White—non-bottlenecked, control population (<i>P</i><sub><i>C</i></sub>), light-grey—bottleneck of 500 individuals (<i>P</i><sub>500</sub>), dark-grey—bottleneck of 50 individuals (<i>P</i><sub>50</sub>), anthracite—bottleneck of 20 individuals (<i>P</i><sub>20</sub>).</span>
            <div class="figcaption-footer">
<div class="article-image-download"><a href="https://dfzljdn9uc3pi.cloudfront.net/2020/9391/1/fig-2-full.png" class="btn btn-mini" download="peerj-9391-fig-2.png" itemprop="url"><i class="icon-large icon-picture">&nbsp;</i> Download full-size image</a></div>
<div class="object-id article-component-doi">DOI: <a href="https://doi.org/10.7717/peerj.9391/fig-2" data-toggle="tooltip" title="" data-original-title="Cite this object using this DOI">10.7717/peerj.9391/fig-2</a>
</div>
</div></figcaption></figure>
         <p id="p-30" class="has-plink"><a class="plink" href="#p-30" rel="nofollow"><i class="icon-comment icon-flip-horizontal"></i></a>In
 the second analysis, locus properties were tested. ANOVA of the GLM 
results showed that locus predictor was significantly associated with 
the <i>MRSE</i> (<i>p</i>&nbsp;=&nbsp;10<sup>−15</sup>) in all metrics. The size of the effect depended on mutation rates, the maximum number of alleles (<a class="xref xref-fig" href="#fig-5" data-jats-ref-type="fig" data-jats-rid="fig-5" data-original-title="" title="">Fig.&nbsp;5</a>) and expected heterozygosity at a given locus (<a class="xref xref-supplementary-material" href="#supp-3" data-jats-ref-type="supplementary-material" data-jats-rid="supp-3">Figs. S1</a>–<a class="xref xref-supplementary-material" href="#supp-6" data-jats-ref-type="supplementary-material" data-jats-rid="supp-6">S4</a>). Mutation rates had a more substantial effect on <i>MRSE</i> (line colors in <a class="xref xref-fig" href="#fig-5" data-jats-ref-type="fig" data-jats-rid="fig-5" data-original-title="" title="">Fig. 5</a>) than the maximum number of allelic states at a locus (line types in <a class="xref xref-fig" href="#fig-5" data-jats-ref-type="fig" data-jats-rid="fig-5" data-original-title="" title="">Fig. 5</a>; <a class="xref xref-supplementary-material" href="#supp-1" data-jats-ref-type="supplementary-material" data-jats-rid="supp-1">Table S1</a>), except for <i>H</i><sub><i>MLE</i></sub>
 at the smallest sample size in which case the error increased at fast 
mutating loci with the number of allelic states possible (L13-L24). The 
error level of <i>H</i><sub><i>Z</i></sub>, <i>H</i><sub><i>Chao</i></sub> and <i>H</i><sub><i>MLE</i></sub> decreased with the total population’s <i>He</i> of the locus (<a class="xref xref-supplementary-material" href="#supp-3" data-jats-ref-type="supplementary-material" data-jats-rid="supp-3">Figs. S1</a>–<a class="xref xref-supplementary-material" href="#supp-6" data-jats-ref-type="supplementary-material" data-jats-rid="supp-6">S4</a>). The slope of this relation was steeper in loci with fewer allelic states allowed (e.g., L01, L07, L13 and L19). In case of <i>H</i><sub><i>CS</i></sub> the MRSE at the smallest sample size, <i>Ns</i> = 5, was positively correlated with <i>He</i> at fast mutating loci with more allelic states allowed, while at the remaining sample sizes the relation was negative.</p>
      </section><section class="sec" id="discussion">
         <h2 class="heading">Discussion</h2>
         <p id="p-31" class="has-plink"><a class="plink" href="#p-31" rel="nofollow"><i class="icon-comment icon-flip-horizontal"></i></a>The
 problem of sample depencence of genetic diversity measures has been 
observed in estimation the number of alleles in populations; to tackle 
the issue, the rarefaction method was proposed for estimating allelic 
richness instead of the plain number of alleles (<a class="xref xref-bibr" href="https://doi.org/10.1007%2FBF00221895" title="" data-jats-ref-type="bibr" data-jats-rid="ref-11" data-original-title="High level of genetic differentiation for allelic richness among populations of the argan tree [Argania spinosa (L.) Skeels] endemic to Morocco">El Mousadik &amp; Petit, 1996</a>; <a class="xref xref-bibr" href="https://doi.org/10.1023%2FB%3ACOGE.0000041021.91777.1a" title="" data-jats-ref-type="bibr" data-jats-rid="ref-22" data-original-title="Counting alleles with rarefaction: private alleles and hierarchical sampling designs">Kalinowski, 2004</a>).
 Allelic richness quickly gained attention and became a popular 
estimator of genetic variation. On the other hand, the advances in 
Shannon diversity index estimation proposed by <a class="xref xref-bibr" href="https://doi.org/10.2307%2F1936227" title="" data-jats-ref-type="bibr" data-jats-rid="ref-41" data-original-title="Jackknifing an index of diversity">Zahl (1977)</a>, <a class="xref xref-bibr" href="https://doi.org/10.1023%2FA%3A1026096204727" title="" data-jats-ref-type="bibr" data-jats-rid="ref-8" data-original-title="Nonparametric estimation of Shannon’s index of diversity when there are unseen species in sample">Chao &amp; Shen (2003)</a> and <a class="xref xref-bibr" href="https://doi.org/10.1111%2F2041-210X.12108" title="" data-jats-ref-type="bibr" data-jats-rid="ref-9" data-original-title="Entropy and the species accumulation curve: a novel entropy estimator via discovery rates of new species">Chao, Wang &amp; Jost (2013)</a> remain unnoticed in population genetics studies.</p>
         <figure class="table-wrap has-plink" id="table-4"><a class="plink" href="#table-4" rel="nofollow"><i class="icon-comment icon-flip-horizontal"></i></a><div class="caption">
<span class="caption-label">Table 4: </span>
               <div class="title">Minimum, maximum and mean values of the relative bias (<i>rB</i>) of all Shannon <i>H</i> estimators and sample sizes tested.</div>
            </div>
            
            <div class="table-container"><table class="table table-bordered table-condensed table-hover">
                  <colgroup>
                     <col>
                     <col>
                     <col>
                     <col>
                     <col>
                     <col>
                  </colgroup>
                  <thead>
                     <tr>
                        <th>Sample size</th>
                        <th></th>
                        <th>
<i>H</i><sub><i>MLE</i></sub><i></i>
</th>
                        <th>
<i>H</i><sub><i>Z</i></sub><i></i>
</th>
                        <th>
<i>H</i><sub><i>CS</i></sub><i></i>
</th>
                        <th>
<i>H</i><sub><i>Chao</i></sub><i></i>
</th>
                     </tr>
                  </thead>
                  <tbody>
                     <tr>
                        <td></td>
                        <td>min</td>
                        <td>−0.3222</td>
                        <td>−0.2097</td>
                        <td>NA</td>
                        <td>−0.2232</td>
                     </tr>
                     <tr>
                        <td>5</td>
                        <td>median</td>
                        <td>−0.1872</td>
                        <td>−0.0229</td>
                        <td>NA</td>
                        <td>−0.0186</td>
                     </tr>
                     <tr>
                        <td></td>
                        <td>max</td>
                        <td>−0.0073</td>
                        <td>0.1599</td>
                        <td>NA</td>
                        <td>0.1344</td>
                     </tr>
                     <tr>
                        <td></td>
                        <td>min</td>
                        <td>−0.1252</td>
                        <td>−0.0939</td>
                        <td>−0.1063</td>
                        <td>−0.0956</td>
                     </tr>
                     <tr>
                        <td>20</td>
                        <td>median</td>
                        <td>−0.0508</td>
                        <td>−0.0024</td>
                        <td>−0.0142</td>
                        <td>−0.0041</td>
                     </tr>
                     <tr>
                        <td></td>
                        <td>max</td>
                        <td>0.0335</td>
                        <td>0.0809</td>
                        <td>0.0808</td>
                        <td>0.0777</td>
                     </tr>
                     <tr>
                        <td></td>
                        <td>min</td>
                        <td>−0.0527</td>
                        <td>−0.0423</td>
                        <td>−0.0420</td>
                        <td>−0.0424</td>
                     </tr>
                     <tr>
                        <td>80</td>
                        <td>median</td>
                        <td>−0.0139</td>
                        <td>−0.0009</td>
                        <td>−0.0059</td>
                        <td>−0.0013</td>
                     </tr>
                     <tr>
                        <td></td>
                        <td>max</td>
                        <td>0.0258</td>
                        <td>0.0403</td>
                        <td>0.0478</td>
                        <td>0,0407</td>
                     </tr>
                     <tr>
                        <td></td>
                        <td>min</td>
                        <td>−0.0318</td>
                        <td>−0.0259</td>
                        <td>−0.0222</td>
                        <td>−0.0261</td>
                     </tr>
                     <tr>
                        <td>200</td>
                        <td>median</td>
                        <td>−0.0058</td>
                        <td>−0.0004</td>
                        <td>−0.0021</td>
                        <td>−0.0005</td>
                     </tr>
                     <tr>
                        <td></td>
                        <td>max</td>
                        <td>0.0185</td>
                        <td>0.0250</td>
                        <td>0.0296</td>
                        <td>0.0248</td>
                     </tr>
                  </tbody>
               </table></div>
<div class="object-id article-component-doi">DOI: <a href="https://doi.org/10.7717/peerj.9391/table-4" data-toggle="tooltip" title="" data-original-title="Cite this object using this DOI">10.7717/peerj.9391/table-4</a>
</div>
         </figure>
         <figure class="fig has-plink" itemprop="image" itemscope="itemscope" itemtype="https://schema.org/ImageObject" id="fig-3"><a class="plink" href="#fig-3" rel="nofollow"><i class="icon-comment icon-flip-horizontal"></i></a><div class="image-container"><a href="https://dfzljdn9uc3pi.cloudfront.net/2020/9391/1/fig-3-2x.jpg" title="View the full image" class="fresco" data-fresco-caption="Figure 3: GLM results: the effect of the sample size predictor on the mean relative squared error (MRSE) of the four Shannon H estimators." data-fresco-group="figure" data-fresco-options="fit: 'width', ui: 'outside', thumbnails: false, loop: true, position: true, overflow: true, preload: false"><img class="graphic" src="fig-3-1x.jpg" itemprop="contentUrl" sizes="(min-width: 1200px) 581px, (max-width: 1199px) and (min-width: 980px) 462px, (max-width: 979px) and (min-width: 768px) 347px, (max-width: 767px) calc(100vw - 50px)" srcset="https://dfzljdn9uc3pi.cloudfront.net/2020/9391/1/fig-3-2x.jpg 1200w, https://dfzljdn9uc3pi.cloudfront.net/2020/9391/1/fig-3-1x.jpg 600w, https://dfzljdn9uc3pi.cloudfront.net/2020/9391/1/fig-3-small.jpg 355w" data-image-id="fig-3" alt="GLM results: the effect of the sample size predictor on the mean relative squared error (MRSE) of the four Shannon H estimators." data-full="https://dfzljdn9uc3pi.cloudfront.net/2020/9391/1/fig-3-full.png" data-thumb="https://dfzljdn9uc3pi.cloudfront.net/2020/9391/1/fig-3-thumb.jpg" data-original="https://dfzljdn9uc3pi.cloudfront.net/2020/9391/1/fig-3.png" data-image-type="figure" data-jats-mimetype="image" data-jats-mime-subtype="png" width="600" height="599"></a></div>
<figcaption itemprop="description">
               <h3 class="heading">
<span class="caption-label">Figure 3: </span>GLM results: the effect of the sample size predictor on the mean relative squared error (<i>MRSE</i>) of the four Shannon <i>H</i> estimators.</h3>
            <div class="figcaption-footer">
<div class="article-image-download"><a href="https://dfzljdn9uc3pi.cloudfront.net/2020/9391/1/fig-3-full.png" class="btn btn-mini" download="peerj-9391-fig-3.png" itemprop="url"><i class="icon-large icon-picture">&nbsp;</i> Download full-size image</a></div>
<div class="object-id article-component-doi">DOI: <a href="https://doi.org/10.7717/peerj.9391/fig-3" data-toggle="tooltip" title="" data-original-title="Cite this object using this DOI">10.7717/peerj.9391/fig-3</a>
</div>
</div></figcaption></figure>
         <figure class="table-wrap has-plink" id="table-5"><a class="plink" href="#table-5" rel="nofollow"><i class="icon-comment icon-flip-horizontal"></i></a><div class="caption">
<span class="caption-label">Table 5: </span>
               <div class="title">Compact letter display of Tukey HSD <i>post hoc</i> test of all pair-wise comparisons between the effects of the <i>H</i> estimators on <i>MRSE</i>. Four independent tests were performed on data with fixed sample sizes.</div>
            </div>
            
            <div class="table-container"><table class="table table-bordered table-condensed table-hover">
                  <colgroup>
                     <col>
                     <col>
                     <col>
                     <col>
                     <col>
                  </colgroup>
                  <thead>
                     <tr>
                        <th>Sample size</th>
                        <th style="text-align:center;" colspan="4">Estimator</th>
                     </tr>
                     <tr>
                        <th></th>
                        <th>
<i>H</i><sub><i>MLE</i></sub><i></i>
</th>
                        <th>
<i>H</i><sub><i>Z</i></sub><i></i>
</th>
                        <th>
<i>H</i><sub><i>CS</i></sub><i></i>
</th>
                        <th>
<i>H</i><sub><i>Chao</i></sub><i></i>
</th>
                     </tr>
                  </thead>
                  <tbody>
                     <tr>
                        <td>5</td>
                        <td>d</td>
                        <td>a</td>
                        <td>b</td>
                        <td>c</td>
                     </tr>
                     <tr>
                        <td>20</td>
                        <td>d</td>
                        <td>a</td>
                        <td>c</td>
                        <td>b</td>
                     </tr>
                     <tr>
                        <td>80</td>
                        <td>d</td>
                        <td>a</td>
                        <td>c</td>
                        <td>b</td>
                     </tr>
                     <tr>
                        <td>200</td>
                        <td>c</td>
                        <td>a</td>
                        <td>b</td>
                        <td>a</td>
                     </tr>
                  </tbody>
               </table></div>
<div class="object-id article-component-doi">DOI: <a href="https://doi.org/10.7717/peerj.9391/table-5" data-toggle="tooltip" title="" data-original-title="Cite this object using this DOI">10.7717/peerj.9391/table-5</a>
</div>
         </figure>
         <figure class="fig has-plink" itemprop="image" itemscope="itemscope" itemtype="https://schema.org/ImageObject" id="fig-4"><a class="plink" href="#fig-4" rel="nofollow"><i class="icon-comment icon-flip-horizontal"></i></a><div class="image-container"><a href="https://dfzljdn9uc3pi.cloudfront.net/2020/9391/1/fig-4-2x.jpg" title="View the full image" class="fresco" data-fresco-caption="Figure 4: GLM results: the effects of total population’s expected heterozygosity (He) and the sample size (Ns) predictors on the mean relative squared error (MRSE) of (A) HMLE, (B) HZ, (C) HCS and (D) HCh." data-fresco-group="figure" data-fresco-options="fit: 'width', ui: 'outside', thumbnails: false, loop: true, position: true, overflow: true, preload: false"><img class="graphic" src="fig-4-1x.jpg" itemprop="contentUrl" sizes="(min-width: 1200px) 581px, (max-width: 1199px) and (min-width: 980px) 462px, (max-width: 979px) and (min-width: 768px) 347px, (max-width: 767px) calc(100vw - 50px)" srcset="https://dfzljdn9uc3pi.cloudfront.net/2020/9391/1/fig-4-2x.jpg 1200w, https://dfzljdn9uc3pi.cloudfront.net/2020/9391/1/fig-4-1x.jpg 600w, https://dfzljdn9uc3pi.cloudfront.net/2020/9391/1/fig-4-small.jpg 355w" data-image-id="fig-4" alt="GLM results: the effects of total population’s expected heterozygosity (He) and the sample size (Ns) predictors on the mean relative squared error (MRSE) of (A) HMLE, (B) HZ, (C) HCS and (D) HCh." data-full="https://dfzljdn9uc3pi.cloudfront.net/2020/9391/1/fig-4-full.png" data-thumb="https://dfzljdn9uc3pi.cloudfront.net/2020/9391/1/fig-4-thumb.jpg" data-original="https://dfzljdn9uc3pi.cloudfront.net/2020/9391/1/fig-4.png" data-image-type="figure" data-jats-mimetype="image" data-jats-mime-subtype="png" width="552" height="600"></a></div>
<figcaption itemprop="description">
               <h3 class="heading">
<span class="caption-label">Figure 4: </span>GLM results: the effects of total population’s expected heterozygosity (<i>He</i>) and the sample size (<i>Ns</i>) predictors on the mean relative squared error (<i>MRSE</i>) of (A) <i>H</i><sub><i>MLE</i></sub>, (B) <i>H</i><sub><i>Z</i></sub>, (C) <i>H</i><sub><i>CS</i></sub> and (D) <i>H</i><sub><i>Ch</i></sub>.</h3>
            <div class="figcaption-footer">
<div class="article-image-download"><a href="https://dfzljdn9uc3pi.cloudfront.net/2020/9391/1/fig-4-full.png" class="btn btn-mini" download="peerj-9391-fig-4.png" itemprop="url"><i class="icon-large icon-picture">&nbsp;</i> Download full-size image</a></div>
<div class="object-id article-component-doi">DOI: <a href="https://doi.org/10.7717/peerj.9391/fig-4" data-toggle="tooltip" title="" data-original-title="Cite this object using this DOI">10.7717/peerj.9391/fig-4</a>
</div>
</div></figcaption></figure>
         <figure class="fig has-plink" itemprop="image" itemscope="itemscope" itemtype="https://schema.org/ImageObject" id="fig-5"><a class="plink" href="#fig-5" rel="nofollow"><i class="icon-comment icon-flip-horizontal"></i></a><div class="image-container"><a href="https://dfzljdn9uc3pi.cloudfront.net/2020/9391/1/fig-5-2x.jpg" title="View the full image" class="fresco" data-fresco-caption="Figure 5: GLM results: locus (L01–L24) effects on mean relative squared error (MRSE) of the four Shannon H estimators at four sample sizes (A) Ns&nbsp;= 5, (B) Ns&nbsp;=&nbsp;20, (C) Ns&nbsp;=&nbsp;80 and (D) Ns&nbsp;=&nbsp;200." data-fresco-group="figure" data-fresco-options="fit: 'width', ui: 'outside', thumbnails: false, loop: true, position: true, overflow: true, preload: false"><img class="graphic" src="fig-5-1x.jpg" itemprop="contentUrl" sizes="(min-width: 1200px) 581px, (max-width: 1199px) and (min-width: 980px) 462px, (max-width: 979px) and (min-width: 768px) 347px, (max-width: 767px) calc(100vw - 50px)" srcset="https://dfzljdn9uc3pi.cloudfront.net/2020/9391/1/fig-5-2x.jpg 1200w, https://dfzljdn9uc3pi.cloudfront.net/2020/9391/1/fig-5-1x.jpg 600w, https://dfzljdn9uc3pi.cloudfront.net/2020/9391/1/fig-5-small.jpg 355w" data-image-id="fig-5" alt="GLM results: locus (L01–L24) effects on mean relative squared error (MRSE) of the four Shannon H estimators at four sample sizes (A) Ns&nbsp;= 5, (B) Ns&nbsp;=&nbsp;20, (C) Ns&nbsp;=&nbsp;80 and (D) Ns&nbsp;=&nbsp;200." data-full="https://dfzljdn9uc3pi.cloudfront.net/2020/9391/1/fig-5-full.png" data-thumb="https://dfzljdn9uc3pi.cloudfront.net/2020/9391/1/fig-5-thumb.jpg" data-original="https://dfzljdn9uc3pi.cloudfront.net/2020/9391/1/fig-5.png" data-image-type="figure" data-jats-mimetype="image" data-jats-mime-subtype="png" width="600" height="561"></a></div>
<figcaption itemprop="description">
               <h3 class="heading">
<span class="caption-label">Figure 5: </span>GLM results: locus (L01–L24) effects on mean relative squared error (<i>MRSE</i>) of the four Shannon <i>H</i> estimators at four sample sizes (A) <i>Ns</i>&nbsp;= 5, (B) <i>Ns</i>&nbsp;=&nbsp;20, (C) <i>Ns</i>&nbsp;=&nbsp;80 and (D) <i>Ns</i>&nbsp;=&nbsp;200.</h3>
            <div class="figcaption-footer">
<div class="article-image-download"><a href="https://dfzljdn9uc3pi.cloudfront.net/2020/9391/1/fig-5-full.png" class="btn btn-mini" download="peerj-9391-fig-5.png" itemprop="url"><i class="icon-large icon-picture">&nbsp;</i> Download full-size image</a></div>
<div class="object-id article-component-doi">DOI: <a href="https://doi.org/10.7717/peerj.9391/fig-5" data-toggle="tooltip" title="" data-original-title="Cite this object using this DOI">10.7717/peerj.9391/fig-5</a>
</div>
</div></figcaption></figure>
         <p id="p-32" class="has-plink"><a class="plink" href="#p-32" rel="nofollow"><i class="icon-comment icon-flip-horizontal"></i></a>The results of the present study confirm what has been known from species diversity studies (<a class="xref xref-bibr" href="https://doi.org/10.1086%2F282439" title="" data-jats-ref-type="bibr" data-jats-rid="ref-31" data-original-title="Shannon’s formula as a measure of specific diversity: its use and misuse">Pielou, 1966</a>), that the original Shannon index, <i>H</i><sub><i>MLE</i></sub>
 is strongly dependent on sample size. This phenomenon is stronger both 
in more genetically variable populations and in more variable loci, 
particularly at small sample sizes. It is not possible to estimate the 
true value of Shannon index using <i>H</i><sub><i>MLE</i></sub> when the
 sample size is small. The most likely explanation for it is that in 
small samples, the probability that all alleles have been captured is 
lower than when the sample is large. The so-called <i>nearly unbiased</i>
 estimators also showed some level of bias, even in samples as big as 
200 genotypes; however, the difference was negligible (less than 1‰), 
and the parametric values were well within the 95% confidence intervals 
of results from the simulated samples. Those measures performed better 
at more diverse loci and populations, where both <i>rB</i> and <i>MRSE</i>
 were, on average, smaller. On the other hand, the error levels of 
unbiased metrics were more dependent on mutation rates rather than the 
maximum possible number of allelic states at the locus, which may 
suggest that the occurrence of low-frequency alleles stemming from 
numerous mutation events has a stabilising effect on those estimators. 
Using <i>H</i><sub><i>CS</i></sub> bears a high risk of encountering 
problems when the sample sizes are small, and the level of genetic 
variation is high, which makes this metric hardly useful in population 
genetics studies. The performance of this metric is similar to that of 
the other two unbiased <i>H</i> estimators; however, it proved to be less precise than <i>H</i><sub><i>Chao</i></sub> and <i>H</i><sub><i>Z</i></sub>. The analysis of the simulations results suggest the Zahl jackknife estimator <i>H</i><sub><i>Z</i></sub>
 as the most suitable estimator of Shannon diversity index to describe 
variation at multiallelic loci such as microsatellites. Among the three 
unbiased estimators, <i>H</i><sub><i>Z</i></sub> had the lowest sampling
 variance and the smallest bias, which also results in the lowest error 
as compared to the other metrics. For this reason, <i>H</i><sub><i>Z</i></sub> should replace traditionally used <i>H</i><sub><i>MLE</i></sub> in population genetics studies using microsatellites.</p>
         <p id="p-33" class="has-plink"><a class="plink" href="#p-33" rel="nofollow"><i class="icon-comment icon-flip-horizontal"></i></a>Although
 all the results presented here were derived from simulating 
microsatellite loci, the pattern of differences among them shows that 
the estimates of the Shannon index for less variable loci are more 
error-prone than for multi-allelic markers. However, as SNP markers are 
mostly biallelic and the <i>H</i> estimates might be more affected by 
error at individual loci, the effect of the errors averaged over a large
 number of loci usually used in SNP-based studies may become negligible.
 Further simulation and empirical tests are necessary to investigate the
 performance of the Shannon index in SNP loci. While the cost NGS 
analyses has dropped significantly in recent years, and the present 
computational power enables analyses of a large amount of data, the 
problem of small sample sizes in genomic studies remains critical in 
studies of vulnerable or elusive species, i.e., the cases where the 
Shannon index is still widely used.</p>
      </section><section class="sec" id="supplemental-information">
         <h2 class="heading"> Supplemental Information</h2>
         <div class="supplementary-material well well-small" id="supp-1" data-jats-mimetype="application" data-jats-mime-subtype="vnd.openxmlformats-officedocument.wordprocessingml.document">
<h3 class="heading">The number of failed attempts to estimate HCS at 24 simulated loci (L01-L24) for the sample of Ns = 5 genotypes</h3>
               
               <p id="p-34">PC and P500 - two demographic scenarios at which the problems were detected.</p>
            <div class="object-id article-component-doi">DOI: <a href="https://doi.org/10.7717/peerj.9391/supp-1" data-toggle="tooltip" title="" data-original-title="Cite this object using this DOI">10.7717/peerj.9391/supp-1</a>
</div>
<div><a href="https://dfzljdn9uc3pi.cloudfront.net/2020/9391/1/TableS1.docx" class="btn article-supporting-download" data-rel="supplement" download="TableS1.docx" data-filename="TableS1.docx"><i class="icon-large icon-download-alt">&nbsp;</i> Download</a></div>
</div>
         <div class="supplementary-material well well-small" id="supp-2" data-jats-mimetype="application" data-jats-mime-subtype="vnd.openxmlformats-officedocument.wordprocessingml.document">
<h3 class="heading">Tukey’s HSD <i>post-hoc</i> test of GLM results of 
locus dependence of MRSE, summarised as compact letter display. The 
tests were performed independently for each sample size/estimator 
combination (columns)</h3>
               
               <p id="p-35">Letters from <i>a</i> to <i>m</i> represent ranges of the overappling confidence intervals.</p>
            <div class="object-id article-component-doi">DOI: <a href="https://doi.org/10.7717/peerj.9391/supp-2" data-toggle="tooltip" title="" data-original-title="Cite this object using this DOI">10.7717/peerj.9391/supp-2</a>
</div>
<div><a href="https://dfzljdn9uc3pi.cloudfront.net/2020/9391/1/TableS2.docx" class="btn article-supporting-download" data-rel="supplement" download="TableS2.docx" data-filename="TableS2.docx"><i class="icon-large icon-download-alt">&nbsp;</i> Download</a></div>
</div>
         <div class="supplementary-material well well-small" id="supp-3" data-jats-mimetype="application" data-jats-mime-subtype="pdf">
<h3 class="heading">GLM results: effects of Nei’s diversity (<i>Hs</i>) on mean relative squared error (<i>MRSE</i>) of four <i>H</i> estimators at the 24 simulated loci <i>Ns</i> = 5 genotypes</h3>
               
            <div class="object-id article-component-doi">DOI: <a href="https://doi.org/10.7717/peerj.9391/supp-3" data-toggle="tooltip" title="" data-original-title="Cite this object using this DOI">10.7717/peerj.9391/supp-3</a>
</div>
<div><a href="https://dfzljdn9uc3pi.cloudfront.net/2020/9391/1/FigureS1.pdf" class="btn article-supporting-download" data-rel="supplement" download="FigureS1.pdf" data-filename="FigureS1.pdf"><i class="icon-large icon-download-alt">&nbsp;</i> Download</a></div>
</div>
         <div class="supplementary-material well well-small" id="supp-4" data-jats-mimetype="application" data-jats-mime-subtype="pdf">
<h3 class="heading">GLM results: effects of Nei’s diversity (<i>Hs</i>) on mean relative squared error (<i>MRSE</i>) of four <i>H</i> estimators at the 24 simulated loci <i>Ns</i> = 20 genotypes</h3>
               
            <div class="object-id article-component-doi">DOI: <a href="https://doi.org/10.7717/peerj.9391/supp-4" data-toggle="tooltip" title="" data-original-title="Cite this object using this DOI">10.7717/peerj.9391/supp-4</a>
</div>
<div><a href="https://dfzljdn9uc3pi.cloudfront.net/2020/9391/1/FigureS2.pdf" class="btn article-supporting-download" data-rel="supplement" download="FigureS2.pdf" data-filename="FigureS2.pdf"><i class="icon-large icon-download-alt">&nbsp;</i> Download</a></div>
</div>
         <div class="supplementary-material well well-small" id="supp-5" data-jats-mimetype="application" data-jats-mime-subtype="pdf">
<h3 class="heading">GLM results: effects of Nei’s diversity (<i>Hs</i>) on mean relative squared error (<i>MRSE</i>) of four <i>H</i> estimators at the 24 simulated loci <i>Ns</i> = 80 genotypes</h3>
               
            <div class="object-id article-component-doi">DOI: <a href="https://doi.org/10.7717/peerj.9391/supp-5" data-toggle="tooltip" title="" data-original-title="Cite this object using this DOI">10.7717/peerj.9391/supp-5</a>
</div>
<div><a href="https://dfzljdn9uc3pi.cloudfront.net/2020/9391/1/FigureS3.pdf" class="btn article-supporting-download" data-rel="supplement" download="FigureS3.pdf" data-filename="FigureS3.pdf"><i class="icon-large icon-download-alt">&nbsp;</i> Download</a></div>
</div>
         <div class="supplementary-material well well-small" id="supp-6" data-jats-mimetype="application" data-jats-mime-subtype="pdf">
<h3 class="heading">GLM results: effects of Nei’s diversity (<i>Hs</i>) on mean relative squared error (<i>MRSE</i>) of four <i>H</i> estimators at the 24 simulated loci <i>Ns</i> = 200 genotypes</h3>
               
            <div class="object-id article-component-doi">DOI: <a href="https://doi.org/10.7717/peerj.9391/supp-6" data-toggle="tooltip" title="" data-original-title="Cite this object using this DOI">10.7717/peerj.9391/supp-6</a>
</div>
<div><a href="https://dfzljdn9uc3pi.cloudfront.net/2020/9391/1/FigureS4.pdf" class="btn article-supporting-download" data-rel="supplement" download="FigureS4.pdf" data-filename="FigureS4.pdf"><i class="icon-large icon-download-alt">&nbsp;</i> Download</a></div>
</div>
         <div class="supplementary-material well well-small" id="supp-7">
<h3 class="heading">R code</h3>
               
            <div class="object-id article-component-doi">DOI: <a href="https://doi.org/10.7717/peerj.9391/supp-7" data-toggle="tooltip" title="" data-original-title="Cite this object using this DOI">10.7717/peerj.9391/supp-7</a>
</div>
<div><a href="https://dfzljdn9uc3pi.cloudfront.net/2020/9391/1/Shannon4.R" class="btn article-supporting-download" data-rel="supplement" download="Shannon4.R" data-filename="Shannon4.R"><i class="icon-large icon-download-alt">&nbsp;</i> Download</a></div>
</div>
      </section><div class="sec" id="additional-information">
         <h2 class="heading">Additional Information and Declarations</h2>
         <div class="fn-group" data-jats-content-type="competing-interests">
            <h3 class="heading">Competing Interests</h3>
<div class="fn" id="conflict-1" data-jats-fn-type="conflict"><p>The author declares he has no competing interests.</p></div>
</div>
         <div class="fn-group" data-jats-content-type="author-contributions">
            <h3 class="heading">Author Contributions</h3>
<div class="fn" id="contribution-1" data-jats-fn-type="con"><p><a class="xref xref-contrib" href="#author-1" data-jats-ref-type="contrib" data-jats-rid="author-1">Maciej K. Konopiński</a>
 conceived and designed the experiments, performed the experiments, 
analyzed the data, prepared figures and/or tables, authored or reviewed 
drafts of the paper, and approved the final draft.</p></div>
</div>
         <div class="fn-group" data-jats-content-type="other">
            <h3 class="heading">Data Availability</h3>
<div class="fn" id="addinfo-1">
<p>The following information was supplied regarding data availability:</p>
            <p>The code used for simulations is available as a <a class="xref xref-supplementary-material" href="#supplemental-information" data-jats-ref-type="supplementary-material" data-jats-rid="supplemental-information">Supplemental File</a>.</p>
</div>
</div>
      <h3 class="heading">Funding</h3>
<p>The work was entirely funded by the Institute of Nature Conservation,
 Polish Academy of Sciences. The funders had no role in study design, 
data collection and analysis, decision to publish, or preparation of the
 manuscript.</p>
</div></div></div></div></div></main><footer class="back">
      
      
      
   <div><div class="row-fluid row-article-item-section-heading"><div class="span1 article-main-left-span1"><i class="article-section-indicator icon-chevron-down"></i></div><h2>Acknowledgements</h2><div id="article-item-acknowledgements-text" class="article-item-section-toggle"></div></div><div class="row-fluid row-article-item-section"><div class="span1 article-main-left-span1">&nbsp;</div><div class="span11 article-item-section-content"><section class="ack" id="acknowledgements"><h2 class="heading" style="display: none;">Acknowledgements</h2>
         <p>I would like to thank Kamil Bartoń, Aleksandra Biedrzycka, 
Adam Ćmiel and Piotr Skórka for fruitful discussions and valuable 
comments they made on the manuscript. I would like to thank the editor 
(Patricia Gandini), Oscar Gaggiotti and the two anonymous referees for 
help on improving this manuscript.</p>
      </section></div></div></div><div><div class="row-fluid row-article-item-section-heading"><div class="span1 article-main-left-span1"><i class="article-section-indicator icon-chevron-down"></i></div><h2>References</h2><div id="article-item-references-text" class="article-item-section-toggle"></div></div><div class="row-fluid row-article-item-section"><div class="span1 article-main-left-span1">&nbsp;</div><div class="span11 article-item-section-content"><section class="ref-list-container" id="references"><h2 class="heading" style="display: none;">References</h2>
<ul class="ref-list" data-jats-content-type="authoryear">
<li class="ref"><div class="citation" itemprop="citation" itemscope="itemscope" itemtype="http://schema.org/ScholarlyArticle" id="ref-1">
<span class="citation-authors-year"><b><span class="name" itemprop="author" itemscope="itemscope" itemtype="http://schema.org/Person"><span class="surname" itemprop="familyName">Archer</span> <span class="given-names" itemprop="givenName">FI</span></span>, <span class="name" itemprop="author" itemscope="itemscope" itemtype="http://schema.org/Person"><span class="surname" itemprop="familyName">Adams</span> <span class="given-names" itemprop="givenName">PE</span></span>, <span class="name" itemprop="author" itemscope="itemscope" itemtype="http://schema.org/Person"><span class="surname" itemprop="familyName">Schneiders</span> <span class="given-names" itemprop="givenName">BB</span></span>.</b> <b class="year" itemprop="datePublished">2017</b>.</span> <cite itemprop="name"><a class="article-title" target="_blank" itemprop="url" href="https://doi.org/10.1111%2F1755-0998.12559">stratag: an r package for manipulating, summarizing and analysing population genetic data</a>.</cite> <span><span class="volume" itemprop="isPartOf" itemscope="itemscope" itemtype="http://schema.org/PublicationVolume"><span class="source" itemprop="isPartOf" itemscope="itemscope" itemtype="http://schema.org/Periodical"><span itemprop="name">Molecular Ecology Resources</span></span> <b itemprop="volumeNumber">17</b></span>:<span class="fpage" itemprop="pageStart">5</span>-<span class="lpage" itemprop="pageEnd">11</span> </span>
</div></li>
<li class="ref"><div class="citation" itemprop="citation" itemscope="itemscope" id="ref-2">
<span class="citation-authors-year"><b><span class="name" itemprop="author" itemscope="itemscope" itemtype="http://schema.org/Person"><span class="surname" itemprop="familyName">Bartoń</span> <span class="given-names" itemprop="givenName">K</span></span>.</b> <b class="year" itemprop="datePublished">2019</b>.</span> <cite class="article-title"><a class="data-title" target="_blank" itemprop="url" href="https://cran.r-project.org/package=MuMIn" data-original-title="" title="">MuMIn: multi-model inference</a>.</cite><span> <span class="version-container"><span class="version">R package version 1.43.15</span></span> </span> <span class="label label-software">software</span>
</div></li>
<li class="ref"><div class="citation" itemprop="citation" itemscope="itemscope" itemtype="http://schema.org/ScholarlyArticle" id="ref-3">
<span class="citation-authors-year"><b><span class="name" itemprop="author" itemscope="itemscope" itemtype="http://schema.org/Person"><span class="surname" itemprop="familyName">Bashalkhanov</span> <span class="given-names" itemprop="givenName">S</span></span>, <span class="name" itemprop="author" itemscope="itemscope" itemtype="http://schema.org/Person"><span class="surname" itemprop="familyName">Pandey</span> <span class="given-names" itemprop="givenName">M</span></span>, <span class="name" itemprop="author" itemscope="itemscope" itemtype="http://schema.org/Person"><span class="surname" itemprop="familyName">Rajora</span> <span class="given-names" itemprop="givenName">OP</span></span>.</b> <b class="year" itemprop="datePublished">2009</b>.</span> <cite itemprop="name"><a class="article-title" target="_blank" itemprop="url" href="https://doi.org/10.1186%2F1471-2156-10-84">A simple method for estimating genetic diversity in large populations from finite sample sizes</a>.</cite> <span><span class="volume" itemprop="isPartOf" itemscope="itemscope" itemtype="http://schema.org/PublicationVolume"><span class="source" itemprop="isPartOf" itemscope="itemscope" itemtype="http://schema.org/Periodical"><span itemprop="name">BMC Genetics</span></span> <b itemprop="volumeNumber">10</b></span>:<span class="fpage" itemprop="pageStart">84</span> </span>
</div></li>
<li class="ref"><div class="citation" itemprop="citation" itemscope="itemscope" itemtype="http://schema.org/ScholarlyArticle" id="ref-4">
<span class="citation-authors-year"><b><span class="name" itemprop="author" itemscope="itemscope" itemtype="http://schema.org/Person"><span class="surname" itemprop="familyName">Basharin</span> <span class="given-names" itemprop="givenName">GP</span></span>.</b> <b class="year" itemprop="datePublished">1959</b>.</span> <cite itemprop="name"><a class="article-title" target="_blank" itemprop="url" href="https://doi.org/10.1137%2F1104033">On a statistical estimate for the entropy of a sequence of independent random variables</a>.</cite> <span><span class="volume" itemprop="isPartOf" itemscope="itemscope" itemtype="http://schema.org/PublicationVolume"><span class="source" itemprop="isPartOf" itemscope="itemscope" itemtype="http://schema.org/Periodical"><span itemprop="name">Theory of Probability &amp; Its Applications</span></span> <b itemprop="volumeNumber">4</b></span>:<span class="fpage" itemprop="pageStart">333</span>-<span class="lpage" itemprop="pageEnd">336</span> </span>
</div></li>
<li class="ref"><div class="citation" itemprop="citation" itemscope="itemscope" itemtype="http://schema.org/ScholarlyArticle" id="ref-5">
<span class="citation-authors-year"><b><span class="name" itemprop="author" itemscope="itemscope" itemtype="http://schema.org/Person"><span class="surname" itemprop="familyName">Chakraborty</span> <span class="given-names" itemprop="givenName">R</span></span>.</b> <b class="year" itemprop="datePublished">1992</b>.</span> <cite itemprop="name"><a class="article-title" target="_blank" itemprop="url" href="https://scholar.google.com/scholar_lookup?title=Sample%20size%20requirements%20for%20addressing%20the%20population%20genetic%20issues%20of%20forensic%20use%20of%20DNA%20typing&amp;author=Chakraborty&amp;publication_year=1992">Sample size requirements for addressing the population genetic issues of forensic use of DNA typing</a>.</cite> <span><span class="volume" itemprop="isPartOf" itemscope="itemscope" itemtype="http://schema.org/PublicationVolume"><span class="source" itemprop="isPartOf" itemscope="itemscope" itemtype="http://schema.org/Periodical"><span itemprop="name">Human Biology</span></span> <b itemprop="volumeNumber">64</b></span>:<span class="fpage" itemprop="pageStart">141</span>-<span class="lpage" itemprop="pageEnd">159</span> </span>
</div></li>
<li class="ref"><div class="citation" itemprop="citation" itemscope="itemscope" itemtype="http://schema.org/ScholarlyArticle" id="ref-6">
<span class="citation-authors-year"><b><span class="name" itemprop="author" itemscope="itemscope" itemtype="http://schema.org/Person"><span class="surname" itemprop="familyName">Chang</span> <span class="given-names" itemprop="givenName">J-T</span></span>, <span class="name" itemprop="author" itemscope="itemscope" itemtype="http://schema.org/Person"><span class="surname" itemprop="familyName">Huang</span> <span class="given-names" itemprop="givenName">B-H</span></span>, <span class="name" itemprop="author" itemscope="itemscope" itemtype="http://schema.org/Person"><span class="surname" itemprop="familyName">Liao</span> <span class="given-names" itemprop="givenName">P-C</span></span>.</b> <b class="year" itemprop="datePublished">2019</b>.</span> <cite itemprop="name"><a class="article-title" target="_blank" itemprop="url" href="https://doi.org/10.1007%2Fs10592-019-01193-1">Genetic
 evidence of the southward founder speciation of Cycas taitungensis from
 ancestral C. revoluta along the Ryukyu Archipelagos</a>.</cite> <span><span class="volume" itemprop="isPartOf" itemscope="itemscope" itemtype="http://schema.org/PublicationVolume"><span class="source" itemprop="isPartOf" itemscope="itemscope" itemtype="http://schema.org/Periodical"><span itemprop="name">Conservation Genetics</span></span> <b itemprop="volumeNumber">20</b></span>:<span class="fpage" itemprop="pageStart">1045</span>-<span class="lpage" itemprop="pageEnd">1056</span> </span>
</div></li>
<li class="ref"><div class="citation" itemprop="citation" itemscope="itemscope" id="ref-7">
<span class="citation-authors-year"><b><span class="name" itemprop="author" itemscope="itemscope" itemtype="http://schema.org/Person"><span class="surname" itemprop="familyName">Chao</span> <span class="given-names" itemprop="givenName">A</span></span>, <span class="name" itemprop="author" itemscope="itemscope" itemtype="http://schema.org/Person"><span class="surname" itemprop="familyName">Ma</span> <span class="given-names" itemprop="givenName">KH</span></span>, <span class="name" itemprop="author" itemscope="itemscope" itemtype="http://schema.org/Person"><span class="surname" itemprop="familyName">Chiu</span> <span class="given-names" itemprop="givenName">TCH</span></span>.</b> <b class="year" itemprop="datePublished">2016</b>.</span> <cite class="article-title"><a class="data-title" target="_blank" itemprop="url" href="https://cran.r-project.org/package=SpadeR" data-original-title="" title="">SpadeR: species-richness prediction and diversity estimation with R. R package version 0.1.1</a>.</cite><span></span> <span class="label label-software">software</span>
</div></li>
<li class="ref"><div class="citation" itemprop="citation" itemscope="itemscope" itemtype="http://schema.org/ScholarlyArticle" id="ref-8">
<span class="citation-authors-year"><b><span class="name" itemprop="author" itemscope="itemscope" itemtype="http://schema.org/Person"><span class="surname" itemprop="familyName">Chao</span> <span class="given-names" itemprop="givenName">A</span></span>, <span class="name" itemprop="author" itemscope="itemscope" itemtype="http://schema.org/Person"><span class="surname" itemprop="familyName">Shen</span> <span class="given-names" itemprop="givenName">T-J</span></span>.</b> <b class="year" itemprop="datePublished">2003</b>.</span> <cite itemprop="name"><a class="article-title" target="_blank" itemprop="url" href="https://doi.org/10.1023%2FA%3A1026096204727">Nonparametric estimation of Shannon’s index of diversity when there are unseen species in sample</a>.</cite> <span><span class="volume" itemprop="isPartOf" itemscope="itemscope" itemtype="http://schema.org/PublicationVolume"><span class="source" itemprop="isPartOf" itemscope="itemscope" itemtype="http://schema.org/Periodical"><span itemprop="name">Environmental and Ecological Statistics</span></span> <b itemprop="volumeNumber">10</b></span>:<span class="fpage" itemprop="pageStart">429</span>-<span class="lpage" itemprop="pageEnd">443</span> </span>
</div></li>
<li class="ref"><div class="citation" itemprop="citation" itemscope="itemscope" itemtype="http://schema.org/ScholarlyArticle" id="ref-9">
<span class="citation-authors-year"><b><span class="name" itemprop="author" itemscope="itemscope" itemtype="http://schema.org/Person"><span class="surname" itemprop="familyName">Chao</span> <span class="given-names" itemprop="givenName">A</span></span>, <span class="name" itemprop="author" itemscope="itemscope" itemtype="http://schema.org/Person"><span class="surname" itemprop="familyName">Wang</span> <span class="given-names" itemprop="givenName">YT</span></span>, <span class="name" itemprop="author" itemscope="itemscope" itemtype="http://schema.org/Person"><span class="surname" itemprop="familyName">Jost</span> <span class="given-names" itemprop="givenName">L</span></span>.</b> <b class="year" itemprop="datePublished">2013</b>.</span> <cite itemprop="name"><a class="article-title" target="_blank" itemprop="url" href="https://doi.org/10.1111%2F2041-210X.12108">Entropy and the species accumulation curve: a novel entropy estimator via discovery rates of new species</a>.</cite> <span><span class="volume" itemprop="isPartOf" itemscope="itemscope" itemtype="http://schema.org/PublicationVolume"><span class="source" itemprop="isPartOf" itemscope="itemscope" itemtype="http://schema.org/Periodical"><span itemprop="name">Methods in Ecology and Evolution</span></span> <b itemprop="volumeNumber">4</b></span>:<span class="fpage" itemprop="pageStart">1091</span>-<span class="lpage" itemprop="pageEnd">1100</span> </span>
</div></li>
<li class="ref"><div class="citation" itemprop="citation" itemscope="itemscope" itemtype="http://schema.org/ScholarlyArticle" id="ref-10">
<span class="citation-authors-year"><b><span class="name" itemprop="author" itemscope="itemscope" itemtype="http://schema.org/Person"><span class="surname" itemprop="familyName">Crowell</span> <span class="given-names" itemprop="givenName">K</span></span>.</b> <b class="year" itemprop="datePublished">1961</b>.</span> <cite itemprop="name"><a class="article-title" target="_blank" itemprop="url" href="https://doi.org/10.1073%2Fpnas.47.2.240">The effects of reduced competition in birds</a>.</cite> <span><span class="volume" itemprop="isPartOf" itemscope="itemscope" itemtype="http://schema.org/PublicationVolume"><span class="source" itemprop="isPartOf" itemscope="itemscope" itemtype="http://schema.org/Periodical"><span itemprop="name">Proceedings of the National Academy of Sciences of the United States of America</span></span> <b itemprop="volumeNumber">47</b></span>:<span class="fpage" itemprop="pageStart">240</span>-<span class="lpage" itemprop="pageEnd">243</span> </span>
</div></li>
<li class="ref"><div class="citation" itemprop="citation" itemscope="itemscope" itemtype="http://schema.org/ScholarlyArticle" id="ref-11">
<span class="citation-authors-year"><b><span class="name" itemprop="author" itemscope="itemscope" itemtype="http://schema.org/Person"><span class="surname" itemprop="familyName">El Mousadik</span> <span class="given-names" itemprop="givenName">A</span></span>, <span class="name" itemprop="author" itemscope="itemscope" itemtype="http://schema.org/Person"><span class="surname" itemprop="familyName">Petit</span> <span class="given-names" itemprop="givenName">RJ</span></span>.</b> <b class="year" itemprop="datePublished">1996</b>.</span> <cite itemprop="name"><a class="article-title" target="_blank" itemprop="url" href="https://doi.org/10.1007%2FBF00221895">High
 level of genetic differentiation for allelic richness among populations
 of the argan tree [Argania spinosa (L.) Skeels] endemic to Morocco</a>.</cite> <span><span class="volume" itemprop="isPartOf" itemscope="itemscope" itemtype="http://schema.org/PublicationVolume"><span class="source" itemprop="isPartOf" itemscope="itemscope" itemtype="http://schema.org/Periodical"><span itemprop="name">Theoretical and Applied Genetics</span></span> <b itemprop="volumeNumber">92</b></span>:<span class="fpage" itemprop="pageStart">832</span>-<span class="lpage" itemprop="pageEnd">839</span> </span>
</div></li>
<li class="ref"><div class="citation" itemprop="citation" itemscope="itemscope" itemtype="http://schema.org/ScholarlyArticle" id="ref-12">
<span class="citation-authors-year"><b><span class="name" itemprop="author" itemscope="itemscope" itemtype="http://schema.org/Person"><span class="surname" itemprop="familyName">Excoffier</span> <span class="given-names" itemprop="givenName">L</span></span>, <span class="name" itemprop="author" itemscope="itemscope" itemtype="http://schema.org/Person"><span class="surname" itemprop="familyName">Dupanloup</span> <span class="given-names" itemprop="givenName">I</span></span>, <span class="name" itemprop="author" itemscope="itemscope" itemtype="http://schema.org/Person"><span class="surname" itemprop="familyName">Huerta-Sánchez</span> <span class="given-names" itemprop="givenName">E</span></span>, <span class="name" itemprop="author" itemscope="itemscope" itemtype="http://schema.org/Person"><span class="surname" itemprop="familyName">Sousa</span> <span class="given-names" itemprop="givenName">VC</span></span>, <span class="name" itemprop="author" itemscope="itemscope" itemtype="http://schema.org/Person"><span class="surname" itemprop="familyName">Foll</span> <span class="given-names" itemprop="givenName">M</span></span>.</b> <b class="year" itemprop="datePublished">2013</b>.</span> <cite itemprop="name"><a class="article-title" target="_blank" itemprop="url" href="https://doi.org/10.1371%2Fjournal.pgen.1003905">Robust demographic inference from genomic and SNP data</a>.</cite> <span><span class="volume" itemprop="isPartOf" itemscope="itemscope" itemtype="http://schema.org/PublicationVolume"><span class="source" itemprop="isPartOf" itemscope="itemscope" itemtype="http://schema.org/Periodical"><span itemprop="name">PLOS Genetics</span></span> <b itemprop="volumeNumber">9</b></span>:<span class="elocation-id" itemprop="pageStart">e1003905</span> </span>
</div></li>
<li class="ref"><div class="citation" itemprop="citation" itemscope="itemscope" itemtype="http://schema.org/ScholarlyArticle" id="ref-13">
<span class="citation-authors-year"><b><span class="name" itemprop="author" itemscope="itemscope" itemtype="http://schema.org/Person"><span class="surname" itemprop="familyName">Fox</span> <span class="given-names" itemprop="givenName">J</span></span>, <span class="name" itemprop="author" itemscope="itemscope" itemtype="http://schema.org/Person"><span class="surname" itemprop="familyName">Weisberg</span> <span class="given-names" itemprop="givenName">S</span></span>.</b> <b class="year" itemprop="datePublished">2018</b>.</span> <cite itemprop="name"><a class="article-title" target="_blank" itemprop="url" href="https://doi.org/10.18637%2Fjss.v087.i09">Visualizing fit and lack of fit in complex regression models with predictor effect plots and partial residuals</a>.</cite> <span><span class="volume" itemprop="isPartOf" itemscope="itemscope" itemtype="http://schema.org/PublicationVolume"><span class="source" itemprop="isPartOf" itemscope="itemscope" itemtype="http://schema.org/Periodical"><span itemprop="name">Journal of Statistical Software</span></span> <b itemprop="volumeNumber">87</b></span>:<span class="fpage" itemprop="pageStart">1</span>-<span class="lpage" itemprop="pageEnd">27</span> </span>
</div></li>
<li class="ref"><div class="citation" itemprop="citation" itemscope="itemscope" itemtype="http://schema.org/Book" id="ref-14">
<span class="citation-authors-year"><b><span class="name" itemprop="author" itemscope="itemscope" itemtype="http://schema.org/Person"><span class="surname" itemprop="familyName">Fox</span> <span class="given-names" itemprop="givenName">J</span></span>, <span class="name" itemprop="author" itemscope="itemscope" itemtype="http://schema.org/Person"><span class="surname" itemprop="familyName">Weisberg</span> <span class="given-names" itemprop="givenName">S</span></span>.</b> <b class="year" itemprop="datePublished">2019</b>.</span> <cite class="article-title"></cite> <span itemprop="name"><a class="source" target="_blank" href="https://scholar.google.com/scholar_lookup?title=An%20R%20companion%20to%20applied%20regression&amp;author=&amp;publication_year=2019">An R companion to applied regression</a>.</span><span>  Thousand Oaks: <span class="publisher">Sage</span>.  </span>
</div></li>
<li class="ref"><div class="citation" itemprop="citation" itemscope="itemscope" itemtype="http://schema.org/ScholarlyArticle" id="ref-15">
<span class="citation-authors-year"><b><span class="name" itemprop="author" itemscope="itemscope" itemtype="http://schema.org/Person"><span class="surname" itemprop="familyName">Gaggiotti</span> <span class="given-names" itemprop="givenName">OE</span></span>, <span class="name" itemprop="author" itemscope="itemscope" itemtype="http://schema.org/Person"><span class="surname" itemprop="familyName">Chao</span> <span class="given-names" itemprop="givenName">A</span></span>, <span class="name" itemprop="author" itemscope="itemscope" itemtype="http://schema.org/Person"><span class="surname" itemprop="familyName">Peres-Neto</span> <span class="given-names" itemprop="givenName">P</span></span>, <span class="name" itemprop="author" itemscope="itemscope" itemtype="http://schema.org/Person"><span class="surname" itemprop="familyName">Chiu</span> <span class="given-names" itemprop="givenName">C-H</span></span>, <span class="name" itemprop="author" itemscope="itemscope" itemtype="http://schema.org/Person"><span class="surname" itemprop="familyName">Edwards</span> <span class="given-names" itemprop="givenName">C</span></span>, <span class="name" itemprop="author" itemscope="itemscope" itemtype="http://schema.org/Person"><span class="surname" itemprop="familyName">Fortin</span> <span class="given-names" itemprop="givenName">M-J</span></span>, <span class="name" itemprop="author" itemscope="itemscope" itemtype="http://schema.org/Person"><span class="surname" itemprop="familyName">Jost</span> <span class="given-names" itemprop="givenName">L</span></span>, <span class="name" itemprop="author" itemscope="itemscope" itemtype="http://schema.org/Person"><span class="surname" itemprop="familyName">Richards</span> <span class="given-names" itemprop="givenName">CM</span></span>, <span class="name" itemprop="author" itemscope="itemscope" itemtype="http://schema.org/Person"><span class="surname" itemprop="familyName">Selkoe</span> <span class="given-names" itemprop="givenName">KA</span></span>.</b> <b class="year" itemprop="datePublished">2018</b>.</span> <cite itemprop="name"><a class="article-title" target="_blank" itemprop="url" href="https://doi.org/10.1111%2Feva.12593">Diversity from genes to ecosystems: a unifying framework to study variation across biological metrics and scales</a>.</cite> <span><span class="volume" itemprop="isPartOf" itemscope="itemscope" itemtype="http://schema.org/PublicationVolume"><span class="source" itemprop="isPartOf" itemscope="itemscope" itemtype="http://schema.org/Periodical"><span itemprop="name">Evolutionary Applications</span></span> <b itemprop="volumeNumber">11</b></span>:<span class="fpage" itemprop="pageStart">1176</span>-<span class="lpage" itemprop="pageEnd">1193</span> </span>
</div></li>
<li class="ref"><div class="citation" itemprop="citation" itemscope="itemscope" itemtype="http://schema.org/ScholarlyArticle" id="ref-16">
<span class="citation-authors-year"><b><span class="name" itemprop="author" itemscope="itemscope" itemtype="http://schema.org/Person"><span class="surname" itemprop="familyName">Good</span> <span class="given-names" itemprop="givenName">IJ</span></span>.</b> <b class="year" itemprop="datePublished">1953</b>.</span> <cite itemprop="name"><a class="article-title" target="_blank" itemprop="url" href="https://doi.org/10.1093%2Fbiomet%2F40.3-4.237">The population frequencies of species and the estimation of population parameters</a>.</cite> <span><span class="volume" itemprop="isPartOf" itemscope="itemscope" itemtype="http://schema.org/PublicationVolume"><span class="source" itemprop="isPartOf" itemscope="itemscope" itemtype="http://schema.org/Periodical"><span itemprop="name">Biometrika</span></span> <b itemprop="volumeNumber">40</b></span>:<span class="fpage" itemprop="pageStart">237</span>-<span class="lpage" itemprop="pageEnd">264</span> </span>
</div></li>
<li class="ref"><div class="citation" itemprop="citation" itemscope="itemscope" itemtype="http://schema.org/ScholarlyArticle" id="ref-17">
<span class="citation-authors-year"><b><span class="name" itemprop="author" itemscope="itemscope" itemtype="http://schema.org/Person"><span class="surname" itemprop="familyName">Gorman</span> <span class="given-names" itemprop="givenName">GC</span></span>, <span class="name" itemprop="author" itemscope="itemscope" itemtype="http://schema.org/Person"><span class="surname" itemprop="familyName">Renzi</span> <span class="given-names" itemprop="givenName">J</span></span>.</b> <b class="year" itemprop="datePublished">1979</b>.</span> <cite itemprop="name"><a class="article-title" target="_blank" itemprop="url" href="https://doi.org/10.2307%2F1443409">Genetic distance and heterozygosity estimates in electrophoretic studies: effects of sample size</a>.</cite> <span><span class="volume" itemprop="isPartOf" itemscope="itemscope" itemtype="http://schema.org/PublicationVolume"><span class="source" itemprop="isPartOf" itemscope="itemscope" itemtype="http://schema.org/Periodical"><span itemprop="name">Copeia</span></span> <b itemprop="volumeNumber">1979</b></span>:<span class="fpage" itemprop="pageStart">242</span>-<span class="lpage" itemprop="pageEnd">249</span> </span>
</div></li>
<li class="ref"><div class="citation" itemprop="citation" itemscope="itemscope" itemtype="http://schema.org/ScholarlyArticle" id="ref-18">
<span class="citation-authors-year"><b><span class="name" itemprop="author" itemscope="itemscope" itemtype="http://schema.org/Person"><span class="surname" itemprop="familyName">Gruber</span> <span class="given-names" itemprop="givenName">B</span></span>, <span class="name" itemprop="author" itemscope="itemscope" itemtype="http://schema.org/Person"><span class="surname" itemprop="familyName">Unmack</span> <span class="given-names" itemprop="givenName">PJ</span></span>, <span class="name" itemprop="author" itemscope="itemscope" itemtype="http://schema.org/Person"><span class="surname" itemprop="familyName">Berry</span> <span class="given-names" itemprop="givenName">OF</span></span>, <span class="name" itemprop="author" itemscope="itemscope" itemtype="http://schema.org/Person"><span class="surname" itemprop="familyName">Georges</span> <span class="given-names" itemprop="givenName">A</span></span>.</b> <b class="year" itemprop="datePublished">2018</b>.</span> <cite itemprop="name"><a class="article-title" target="_blank" itemprop="url" href="https://doi.org/10.1111%2F1755-0998.12745">dartr: an r package to facilitate analysis of SNP data generated from reduced representation genome sequencing</a>.</cite> <span><span class="volume" itemprop="isPartOf" itemscope="itemscope" itemtype="http://schema.org/PublicationVolume"><span class="source" itemprop="isPartOf" itemscope="itemscope" itemtype="http://schema.org/Periodical"><span itemprop="name">Molecular Ecology Resources</span></span> <b itemprop="volumeNumber">18</b></span>:<span class="fpage" itemprop="pageStart">691</span>-<span class="lpage" itemprop="pageEnd">699</span> </span>
</div></li>
<li class="ref"><div class="citation" itemprop="citation" itemscope="itemscope" itemtype="http://schema.org/ScholarlyArticle" id="ref-19">
<span class="citation-authors-year"><b><span class="name" itemprop="author" itemscope="itemscope" itemtype="http://schema.org/Person"><span class="surname" itemprop="familyName">Hothorn</span> <span class="given-names" itemprop="givenName">T</span></span>, <span class="name" itemprop="author" itemscope="itemscope" itemtype="http://schema.org/Person"><span class="surname" itemprop="familyName">Bretz</span> <span class="given-names" itemprop="givenName">F</span></span>, <span class="name" itemprop="author" itemscope="itemscope" itemtype="http://schema.org/Person"><span class="surname" itemprop="familyName">Westfall</span> <span class="given-names" itemprop="givenName">P</span></span>.</b> <b class="year" itemprop="datePublished">2008</b>.</span> <cite itemprop="name"><a class="article-title" target="_blank" itemprop="url" href="https://doi.org/10.1002%2Fbimj.200810425">Simultaneous inference in general parametric models</a>.</cite> <span><span class="volume" itemprop="isPartOf" itemscope="itemscope" itemtype="http://schema.org/PublicationVolume"><span class="source" itemprop="isPartOf" itemscope="itemscope" itemtype="http://schema.org/Periodical"><span itemprop="name">Biometrical Journal. Biometrische Zeitschrift</span></span> <b itemprop="volumeNumber">50</b></span>:<span class="fpage" itemprop="pageStart">346</span>-<span class="lpage" itemprop="pageEnd">363</span> </span>
</div></li>
<li class="ref"><div class="citation" itemprop="citation" itemscope="itemscope" itemtype="http://schema.org/ScholarlyArticle" id="ref-20">
<span class="citation-authors-year"><b><span class="name" itemprop="author" itemscope="itemscope" itemtype="http://schema.org/Person"><span class="surname" itemprop="familyName">Jain</span> <span class="given-names" itemprop="givenName">SK</span></span>, <span class="name" itemprop="author" itemscope="itemscope" itemtype="http://schema.org/Person"><span class="surname" itemprop="familyName">Qualset</span> <span class="given-names" itemprop="givenName">CO</span></span>, <span class="name" itemprop="author" itemscope="itemscope" itemtype="http://schema.org/Person"><span class="surname" itemprop="familyName">Bhatt</span> <span class="given-names" itemprop="givenName">GM</span></span>, <span class="name" itemprop="author" itemscope="itemscope" itemtype="http://schema.org/Person"><span class="surname" itemprop="familyName">Wu</span> <span class="given-names" itemprop="givenName">KK</span></span>.</b> <b class="year" itemprop="datePublished">1975</b>.</span> <cite itemprop="name"><a class="article-title" target="_blank" itemprop="url" href="https://doi.org/10.2135%2Fcropsci1975.0011183X001500050026x">Geographical patterns of phenotypic diversity in a world collection of durum wheats 1</a>.</cite> <span><span class="volume" itemprop="isPartOf" itemscope="itemscope" itemtype="http://schema.org/PublicationVolume"><span class="source" itemprop="isPartOf" itemscope="itemscope" itemtype="http://schema.org/Periodical"><span itemprop="name">Crop Science</span></span> <b itemprop="volumeNumber">15</b></span>:<span class="fpage" itemprop="pageStart">700</span>-<span class="lpage" itemprop="pageEnd">704</span> </span>
</div></li>
<li class="ref"><div class="citation" itemprop="citation" itemscope="itemscope" itemtype="http://schema.org/ScholarlyArticle" id="ref-21">
<span class="citation-authors-year"><b><span class="name" itemprop="author" itemscope="itemscope" itemtype="http://schema.org/Person"><span class="surname" itemprop="familyName">Jost</span> <span class="given-names" itemprop="givenName">L</span></span>.</b> <b class="year" itemprop="datePublished">2007</b>.</span> <cite itemprop="name"><a class="article-title" target="_blank" itemprop="url" href="https://doi.org/10.1890%2F06-1736.1">Partitioning diversity into independent alpha and beta components</a>.</cite> <span><span class="volume" itemprop="isPartOf" itemscope="itemscope" itemtype="http://schema.org/PublicationVolume"><span class="source" itemprop="isPartOf" itemscope="itemscope" itemtype="http://schema.org/Periodical"><span itemprop="name">Ecology</span></span> <b itemprop="volumeNumber">88</b></span>:<span class="fpage" itemprop="pageStart">2427</span>-<span class="lpage" itemprop="pageEnd">2439</span> </span>
</div></li>
<li class="ref"><div class="citation" itemprop="citation" itemscope="itemscope" itemtype="http://schema.org/ScholarlyArticle" id="ref-22">
<span class="citation-authors-year"><b><span class="name" itemprop="author" itemscope="itemscope" itemtype="http://schema.org/Person"><span class="surname" itemprop="familyName">Kalinowski</span> <span class="given-names" itemprop="givenName">ST</span></span>.</b> <b class="year" itemprop="datePublished">2004</b>.</span> <cite itemprop="name"><a class="article-title" target="_blank" itemprop="url" href="https://doi.org/10.1023%2FB%3ACOGE.0000041021.91777.1a">Counting alleles with rarefaction: private alleles and hierarchical sampling designs</a>.</cite> <span><span class="volume" itemprop="isPartOf" itemscope="itemscope" itemtype="http://schema.org/PublicationVolume"><span class="source" itemprop="isPartOf" itemscope="itemscope" itemtype="http://schema.org/Periodical"><span itemprop="name">Conservation Genetics</span></span> <b itemprop="volumeNumber">5</b></span>:<span class="fpage" itemprop="pageStart">539</span>-<span class="lpage" itemprop="pageEnd">543</span> </span>
</div></li>
<li class="ref"><div class="citation" itemprop="citation" itemscope="itemscope" itemtype="http://schema.org/ScholarlyArticle" id="ref-23">
<span class="citation-authors-year"><b><span class="name" itemprop="author" itemscope="itemscope" itemtype="http://schema.org/Person"><span class="surname" itemprop="familyName">Kamvar</span> <span class="given-names" itemprop="givenName">ZN</span></span>, <span class="name" itemprop="author" itemscope="itemscope" itemtype="http://schema.org/Person"><span class="surname" itemprop="familyName">Tabima</span> <span class="given-names" itemprop="givenName">JF</span></span>, <span class="name" itemprop="author" itemscope="itemscope" itemtype="http://schema.org/Person"><span class="surname" itemprop="familyName">Grünwald</span> <span class="given-names" itemprop="givenName">NJ</span></span>.</b> <b class="year" itemprop="datePublished">2014</b>.</span> <cite itemprop="name"><a class="article-title" target="_blank" itemprop="url" href="https://doi.org/10.7717%2Fpeerj.281">Poppr: an R package for genetic analysis of populations with clonal, partially clonal, and/or sexual reproduction</a>.</cite> <span><span class="volume" itemprop="isPartOf" itemscope="itemscope" itemtype="http://schema.org/PublicationVolume"><span class="source" itemprop="isPartOf" itemscope="itemscope" itemtype="http://schema.org/Periodical"><span itemprop="name">PeerJ</span></span> <b itemprop="volumeNumber">2</b></span>:<span class="elocation-id" itemprop="pageStart">e281</span> </span>
</div></li>
<li class="ref"><div class="citation" itemprop="citation" itemscope="itemscope" itemtype="http://schema.org/ScholarlyArticle" id="ref-24">
<span class="citation-authors-year"><b><span class="name" itemprop="author" itemscope="itemscope" itemtype="http://schema.org/Person"><span class="surname" itemprop="familyName">Leberg</span> <span class="given-names" itemprop="givenName">PL</span></span>.</b> <b class="year" itemprop="datePublished">2002</b>.</span> <cite itemprop="name"><a class="article-title" target="_blank" itemprop="url" href="https://doi.org/10.1046%2Fj.1365-294X.2002.01612.x">Estimating allelic richness: effects of sample size and bottlenecks</a>.</cite> <span><span class="volume" itemprop="isPartOf" itemscope="itemscope" itemtype="http://schema.org/PublicationVolume"><span class="source" itemprop="isPartOf" itemscope="itemscope" itemtype="http://schema.org/Periodical"><span itemprop="name">Molecular Ecology</span></span> <b itemprop="volumeNumber">11</b></span>:<span class="fpage" itemprop="pageStart">2445</span>-<span class="lpage" itemprop="pageEnd">2449</span> </span>
</div></li>
<li class="ref"><div class="citation" itemprop="citation" itemscope="itemscope" itemtype="http://schema.org/ScholarlyArticle" id="ref-25">
<span class="citation-authors-year"><b><span class="name" itemprop="author" itemscope="itemscope" itemtype="http://schema.org/Person"><span class="surname" itemprop="familyName">Margalef</span> <span class="given-names" itemprop="givenName">R</span></span>.</b> <b class="year" itemprop="datePublished">1957</b>.</span> <cite itemprop="name"><a class="article-title" target="_blank" itemprop="url" href="https://scholar.google.com/scholar_lookup?title=La%20teor%C3%ADa%20de%20la%20informaci%C3%B3n%20en%20Ecolog%C3%ADa&amp;author=Margalef&amp;publication_year=1957">La teoría de la información en Ecología</a>.</cite> <span><span class="volume" itemprop="isPartOf" itemscope="itemscope" itemtype="http://schema.org/PublicationVolume"><span class="source" itemprop="isPartOf" itemscope="itemscope" itemtype="http://schema.org/Periodical"><span itemprop="name">Memorias de la Real Academia de Ciencias y Artes de Barcelona</span></span> <b itemprop="volumeNumber">32</b></span>:<span class="fpage" itemprop="pageStart">373</span>-<span class="lpage" itemprop="pageEnd">436</span> </span>
</div></li>
<li class="ref"><div class="citation" itemprop="citation" itemscope="itemscope" itemtype="http://schema.org/ScholarlyArticle" id="ref-26">
<span class="citation-authors-year"><b><span class="name" itemprop="author" itemscope="itemscope" itemtype="http://schema.org/Person"><span class="surname" itemprop="familyName">Marquez-Sanchez</span> <span class="given-names" itemprop="givenName">F</span></span>, <span class="name" itemprop="author" itemscope="itemscope" itemtype="http://schema.org/Person"><span class="surname" itemprop="familyName">Hallauer</span> <span class="given-names" itemprop="givenName">AR</span></span>.</b> <b class="year" itemprop="datePublished">1970</b>.</span> <cite itemprop="name"><a class="article-title" target="_blank" itemprop="url" href="https://doi.org/10.2135%2Fcropsci1970.0011183X001000040012x">Influence of sample size on the estimation of genetic variances in a synthetic variety of maize. I. Grain yield 1</a>.</cite> <span><span class="volume" itemprop="isPartOf" itemscope="itemscope" itemtype="http://schema.org/PublicationVolume"><span class="source" itemprop="isPartOf" itemscope="itemscope" itemtype="http://schema.org/Periodical"><span itemprop="name">Crop Science</span></span> <b itemprop="volumeNumber">10</b></span>:<span class="fpage" itemprop="pageStart">357</span>-<span class="lpage" itemprop="pageEnd">361</span> </span>
</div></li>
<li class="ref"><div class="citation" itemprop="citation" itemscope="itemscope" itemtype="http://schema.org/ScholarlyArticle" id="ref-27">
<span class="citation-authors-year"><b><span class="name" itemprop="author" itemscope="itemscope" itemtype="http://schema.org/Person"><span class="surname" itemprop="familyName">Meirmans</span> <span class="given-names" itemprop="givenName">PG</span></span>, <span class="name" itemprop="author" itemscope="itemscope" itemtype="http://schema.org/Person"><span class="surname" itemprop="familyName">Tienderen</span> <span class="given-names" itemprop="givenName">PHV</span></span>.</b> <b class="year" itemprop="datePublished">2004</b>.</span> <cite itemprop="name"><a class="article-title" target="_blank" itemprop="url" href="https://doi.org/10.1111%2Fj.1471-8286.2004.00770.x">genotype and genodive: two programs for the analysis of genetic diversity of asexual organisms</a>.</cite> <span><span class="volume" itemprop="isPartOf" itemscope="itemscope" itemtype="http://schema.org/PublicationVolume"><span class="source" itemprop="isPartOf" itemscope="itemscope" itemtype="http://schema.org/Periodical"><span itemprop="name">Molecular Ecology Notes</span></span> <b itemprop="volumeNumber">4</b></span>:<span class="fpage" itemprop="pageStart">792</span>-<span class="lpage" itemprop="pageEnd">794</span> </span>
</div></li>
<li class="ref"><div class="citation" itemprop="citation" itemscope="itemscope" id="ref-28">
<span class="citation-authors-year"><b><span class="name" itemprop="author" itemscope="itemscope" itemtype="http://schema.org/Person"><span class="surname" itemprop="familyName">Oksanen</span> <span class="given-names" itemprop="givenName">J</span></span>, <span class="name" itemprop="author" itemscope="itemscope" itemtype="http://schema.org/Person"><span class="surname" itemprop="familyName">Blanchet</span> <span class="given-names" itemprop="givenName">FG</span></span>, <span class="name" itemprop="author" itemscope="itemscope" itemtype="http://schema.org/Person"><span class="surname" itemprop="familyName">Friendly</span> <span class="given-names" itemprop="givenName">M</span></span>, <span class="name" itemprop="author" itemscope="itemscope" itemtype="http://schema.org/Person"><span class="surname" itemprop="familyName">Kindt</span> <span class="given-names" itemprop="givenName">R</span></span>, <span class="name" itemprop="author" itemscope="itemscope" itemtype="http://schema.org/Person"><span class="surname" itemprop="familyName">Legendre</span> <span class="given-names" itemprop="givenName">P</span></span>, <span class="name" itemprop="author" itemscope="itemscope" itemtype="http://schema.org/Person"><span class="surname" itemprop="familyName">McGlinn</span> <span class="given-names" itemprop="givenName">D</span></span>, <span class="name" itemprop="author" itemscope="itemscope" itemtype="http://schema.org/Person"><span class="surname" itemprop="familyName">Minchin</span> <span class="given-names" itemprop="givenName">PR</span></span>, <span class="name" itemprop="author" itemscope="itemscope" itemtype="http://schema.org/Person"><span class="surname" itemprop="familyName">O’Hara</span> <span class="given-names" itemprop="givenName">RB</span></span>, <span class="name" itemprop="author" itemscope="itemscope" itemtype="http://schema.org/Person"><span class="surname" itemprop="familyName">Simpson</span> <span class="given-names" itemprop="givenName">GL</span></span>, <span class="name" itemprop="author" itemscope="itemscope" itemtype="http://schema.org/Person"><span class="surname" itemprop="familyName">Solymos</span> <span class="given-names" itemprop="givenName">P</span></span><span class="hide hidden-authors">, <span class="name" itemprop="author" itemscope="itemscope" itemtype="http://schema.org/Person"><span class="surname" itemprop="familyName">Stevens</span> <span class="given-names" itemprop="givenName">MHH</span></span>, <span class="name" itemprop="author" itemscope="itemscope" itemtype="http://schema.org/Person"><span class="surname" itemprop="familyName">Szoecs</span> <span class="given-names" itemprop="givenName">E</span></span>, <span class="name" itemprop="author" itemscope="itemscope" itemtype="http://schema.org/Person"><span class="surname" itemprop="familyName">Wagner</span> <span class="given-names" itemprop="givenName">H</span></span></span><a href="#" title="Show hidden authors" class="show-all-authors">+3 more</a>.</b> <b class="year" itemprop="datePublished">2019</b>.</span> <cite class="article-title"><a class="data-title" target="_blank" itemprop="url" href="https://cran.r-project.org/package=vegan" data-original-title="" title="">vegan: community ecology package</a>.</cite><span> <span class="version-container"><span class="version">R package version 2.5-6</span></span> </span> <span class="label label-software">software</span>
</div></li>
<li class="ref"><div class="citation" itemprop="citation" itemscope="itemscope" itemtype="http://schema.org/ScholarlyArticle" id="ref-29">
<span class="citation-authors-year"><b><span class="name" itemprop="author" itemscope="itemscope" itemtype="http://schema.org/Person"><span class="surname" itemprop="familyName">O’Reilly</span> <span class="given-names" itemprop="givenName">GD</span></span>, <span class="name" itemprop="author" itemscope="itemscope" itemtype="http://schema.org/Person"><span class="surname" itemprop="familyName">Jabot</span> <span class="given-names" itemprop="givenName">F</span></span>, <span class="name" itemprop="author" itemscope="itemscope" itemtype="http://schema.org/Person"><span class="surname" itemprop="familyName">Gunn</span> <span class="given-names" itemprop="givenName">MR</span></span>, <span class="name" itemprop="author" itemscope="itemscope" itemtype="http://schema.org/Person"><span class="surname" itemprop="familyName">Sherwin</span> <span class="given-names" itemprop="givenName">WB</span></span>.</b> <b class="year" itemprop="datePublished">2018</b>.</span> <cite itemprop="name"><a class="article-title" target="_blank" itemprop="url" href="https://doi.org/10.1007%2Fs12686-018-1079-z">Predicting Shannon’s information for genes in finite populations: new uses for old equations</a>.</cite> <span><span class="volume" itemprop="isPartOf" itemscope="itemscope" itemtype="http://schema.org/PublicationVolume"><span class="source" itemprop="isPartOf" itemscope="itemscope" itemtype="http://schema.org/Periodical"><span itemprop="name">Conservation Genetics Resources</span></span> <b itemprop="volumeNumber">12</b></span>:<span class="fpage" itemprop="pageStart">245</span>-<span class="lpage" itemprop="pageEnd">255</span> </span>
</div></li>
<li class="ref"><div class="citation" itemprop="citation" itemscope="itemscope" itemtype="http://schema.org/ScholarlyArticle" id="ref-30">
<span class="citation-authors-year"><b><span class="name" itemprop="author" itemscope="itemscope" itemtype="http://schema.org/Person"><span class="surname" itemprop="familyName">Peakall</span> <span class="given-names" itemprop="givenName">R</span></span>, <span class="name" itemprop="author" itemscope="itemscope" itemtype="http://schema.org/Person"><span class="surname" itemprop="familyName">Smouse</span> <span class="given-names" itemprop="givenName">PE</span></span>.</b> <b class="year" itemprop="datePublished">2012</b>.</span> <cite itemprop="name"><a class="article-title" target="_blank" itemprop="url" href="https://doi.org/10.1093%2Fbioinformatics%2Fbts460">GenAlEx 6.5: genetic analysis in Excel. Population genetic software for teaching and research—an update</a>.</cite> <span><span class="volume" itemprop="isPartOf" itemscope="itemscope" itemtype="http://schema.org/PublicationVolume"><span class="source" itemprop="isPartOf" itemscope="itemscope" itemtype="http://schema.org/Periodical"><span itemprop="name">Bioinformatics</span></span> <b itemprop="volumeNumber">28</b></span>:<span class="fpage" itemprop="pageStart">2537</span>-<span class="lpage" itemprop="pageEnd">2539</span> </span>
</div></li>
<li class="ref"><div class="citation" itemprop="citation" itemscope="itemscope" itemtype="http://schema.org/ScholarlyArticle" id="ref-31">
<span class="citation-authors-year"><b><span class="name" itemprop="author" itemscope="itemscope" itemtype="http://schema.org/Person"><span class="surname" itemprop="familyName">Pielou</span> <span class="given-names" itemprop="givenName">EC</span></span>.</b> <b class="year" itemprop="datePublished">1966</b>.</span> <cite itemprop="name"><a class="article-title" target="_blank" itemprop="url" href="https://doi.org/10.1086%2F282439">Shannon’s formula as a measure of specific diversity: its use and misuse</a>.</cite> <span><span class="volume" itemprop="isPartOf" itemscope="itemscope" itemtype="http://schema.org/PublicationVolume"><span class="source" itemprop="isPartOf" itemscope="itemscope" itemtype="http://schema.org/Periodical"><span itemprop="name">The American Naturalist</span></span> <b itemprop="volumeNumber">100</b></span>:<span class="fpage" itemprop="pageStart">463</span>-<span class="lpage" itemprop="pageEnd">465</span> </span>
</div></li>
<li class="ref"><div class="citation" itemprop="citation" itemscope="itemscope" itemtype="http://schema.org/ScholarlyArticle" id="ref-32">
<span class="citation-authors-year"><b><span class="name" itemprop="author" itemscope="itemscope" itemtype="http://schema.org/Person"><span class="surname" itemprop="familyName">Piepho</span> <span class="given-names" itemprop="givenName">H-P</span></span>.</b> <b class="year" itemprop="datePublished">2004</b>.</span> <cite itemprop="name"><a class="article-title" target="_blank" itemprop="url" href="https://doi.org/10.1198%2F1061860043515">An algorithm for a letter-based representation of all-pairwise comparisons</a>.</cite> <span><span class="volume" itemprop="isPartOf" itemscope="itemscope" itemtype="http://schema.org/PublicationVolume"><span class="source" itemprop="isPartOf" itemscope="itemscope" itemtype="http://schema.org/Periodical"><span itemprop="name">Journal of Computational and Graphical Statistics</span></span> <b itemprop="volumeNumber">13</b></span>:<span class="fpage" itemprop="pageStart">456</span>-<span class="lpage" itemprop="pageEnd">466</span> </span>
</div></li>
<li class="ref"><div class="citation" itemprop="citation" itemscope="itemscope" itemtype="http://schema.org/ScholarlyArticle" id="ref-33">
<span class="citation-authors-year"><b><span class="name" itemprop="author" itemscope="itemscope" itemtype="http://schema.org/Person"><span class="surname" itemprop="familyName">Pruett</span> <span class="given-names" itemprop="givenName">CL</span></span>, <span class="name" itemprop="author" itemscope="itemscope" itemtype="http://schema.org/Person"><span class="surname" itemprop="familyName">Winker</span> <span class="given-names" itemprop="givenName">K</span></span>.</b> <b class="year" itemprop="datePublished">2008</b>.</span> <cite itemprop="name"><a class="article-title" target="_blank" itemprop="url" href="https://doi.org/10.1111%2Fj.0908-8857.2008.04094.x">The effects of sample size on population genetic diversity estimates in song sparrows Melospiza melodia</a>.</cite> <span><span class="volume" itemprop="isPartOf" itemscope="itemscope" itemtype="http://schema.org/PublicationVolume"><span class="source" itemprop="isPartOf" itemscope="itemscope" itemtype="http://schema.org/Periodical"><span itemprop="name">Journal of Avian Biology</span></span> <b itemprop="volumeNumber">39</b></span>:<span class="fpage" itemprop="pageStart">252</span>-<span class="lpage" itemprop="pageEnd">256</span> </span>
</div></li>
<li class="ref"><div class="citation" itemprop="citation" itemscope="itemscope" id="ref-34">
<span class="citation-authors-year"><b><span class="name" itemprop="author" itemscope="itemscope" itemtype="http://schema.org/Person"><span class="surname" itemprop="familyName">Qin</span> <span class="given-names" itemprop="givenName">X</span></span>.</b> <b class="year" itemprop="datePublished">2019</b>.</span> <cite class="article-title"><a class="data-title" target="_blank" itemprop="url" href="https://cran.r-project.org/package=HierDpart" data-original-title="" title="">HierDpart: partitioning hierarchical diversity and differentiation across metrics and scales, from genes to ecosystems</a>.</cite><span> <span class="version-container"><span class="version">R package version 0.5.0</span></span> </span> <span class="label label-software">software</span>
</div></li>
<li class="ref"><div class="citation" itemprop="citation" itemscope="itemscope" id="ref-35">
<span class="citation-authors-year"><b><span class="collab" itemprop="author" itemscope="itemscope">R Development Core Team</span>.</b> <b class="year" itemprop="datePublished">2009</b>.</span> <cite class="article-title"><a class="data-title" target="_blank" itemprop="url" href="#ref-35" data-original-title="" title="">R: a language and environment for statistical computing</a>.</cite><span> Vienna: <span class="publisher">R Foundation for Statistical Computing</span>.</span> <span class="label label-software">software</span>
</div></li>
<li class="ref"><div class="citation" itemprop="citation" itemscope="itemscope" itemtype="http://schema.org/ScholarlyArticle" id="ref-36">
<span class="citation-authors-year"><b><span class="name" itemprop="author" itemscope="itemscope" itemtype="http://schema.org/Person"><span class="surname" itemprop="familyName">Shannon</span> <span class="given-names" itemprop="givenName">CE</span></span>.</b> <b class="year" itemprop="datePublished">1948</b>.</span> <cite itemprop="name"><a class="article-title" target="_blank" itemprop="url" href="https://doi.org/10.1002%2Fj.1538-7305.1948.tb01338.x">A mathematical theory of communication</a>.</cite> <span><span class="volume" itemprop="isPartOf" itemscope="itemscope" itemtype="http://schema.org/PublicationVolume"><span class="source" itemprop="isPartOf" itemscope="itemscope" itemtype="http://schema.org/Periodical"><span itemprop="name">Bell System Technical Journal</span></span> <b itemprop="volumeNumber">27</b></span>:<span class="fpage" itemprop="pageStart">379</span>-<span class="lpage" itemprop="pageEnd">423</span> </span>
</div></li>
<li class="ref"><div class="citation" itemprop="citation" itemscope="itemscope" itemtype="http://schema.org/ScholarlyArticle" id="ref-37">
<span class="citation-authors-year"><b><span class="name" itemprop="author" itemscope="itemscope" itemtype="http://schema.org/Person"><span class="surname" itemprop="familyName">Sherwin</span> <span class="given-names" itemprop="givenName">WB</span></span>.</b> <b class="year" itemprop="datePublished">2018</b>.</span> <cite itemprop="name"><a class="article-title" target="_blank" itemprop="url" href="https://doi.org/10.3390%2Fe20100727">Entropy, or information, unifies ecology and evolution and beyond</a>.</cite> <span><span class="volume" itemprop="isPartOf" itemscope="itemscope" itemtype="http://schema.org/PublicationVolume"><span class="source" itemprop="isPartOf" itemscope="itemscope" itemtype="http://schema.org/Periodical"><span itemprop="name">Entropy</span></span> <b itemprop="volumeNumber">20</b></span>:<span class="fpage" itemprop="pageStart">727</span> </span>
</div></li>
<li class="ref"><div class="citation" itemprop="citation" itemscope="itemscope" itemtype="http://schema.org/ScholarlyArticle" id="ref-38">
<span class="citation-authors-year"><b><span class="name" itemprop="author" itemscope="itemscope" itemtype="http://schema.org/Person"><span class="surname" itemprop="familyName">Sherwin</span> <span class="given-names" itemprop="givenName">WB</span></span>, <span class="name" itemprop="author" itemscope="itemscope" itemtype="http://schema.org/Person"><span class="surname" itemprop="familyName">Chao</span> <span class="given-names" itemprop="givenName">A</span></span>, <span class="name" itemprop="author" itemscope="itemscope" itemtype="http://schema.org/Person"><span class="surname" itemprop="familyName">Jost</span> <span class="given-names" itemprop="givenName">L</span></span>, <span class="name" itemprop="author" itemscope="itemscope" itemtype="http://schema.org/Person"><span class="surname" itemprop="familyName">Smouse</span> <span class="given-names" itemprop="givenName">PE</span></span>.</b> <b class="year" itemprop="datePublished">2017</b>.</span> <cite itemprop="name"><a class="article-title" target="_blank" itemprop="url" href="https://doi.org/10.1016%2Fj.tree.2017.09.012">Information theory broadens the spectrum of molecular ecology and evolution</a>.</cite> <span><span class="volume" itemprop="isPartOf" itemscope="itemscope" itemtype="http://schema.org/PublicationVolume"><span class="source" itemprop="isPartOf" itemscope="itemscope" itemtype="http://schema.org/Periodical"><span itemprop="name">Trends in Ecology &amp; Evolution</span></span> <b itemprop="volumeNumber">32</b></span>:<span class="fpage" itemprop="pageStart">948</span>-<span class="lpage" itemprop="pageEnd">963</span> </span>
</div></li>
<li class="ref"><div class="citation" itemprop="citation" itemscope="itemscope" itemtype="http://schema.org/ScholarlyArticle" id="ref-39">
<span class="citation-authors-year"><b><span class="name" itemprop="author" itemscope="itemscope" itemtype="http://schema.org/Person"><span class="surname" itemprop="familyName">Spellerberg</span> <span class="given-names" itemprop="givenName">IF</span></span>, <span class="name" itemprop="author" itemscope="itemscope" itemtype="http://schema.org/Person"><span class="surname" itemprop="familyName">Fedor</span> <span class="given-names" itemprop="givenName">PJ</span></span>.</b> <b class="year" itemprop="datePublished">2003</b>.</span> <cite itemprop="name"><a class="article-title" target="_blank" itemprop="url" href="https://doi.org/10.1046%2Fj.1466-822X.2003.00015.x">A
 tribute to Claude Shannon (1916–2001) and a plea for more rigorous use 
of species richness, species diversity and the ‘Shannon–Wiener’ index</a>.</cite> <span><span class="volume" itemprop="isPartOf" itemscope="itemscope" itemtype="http://schema.org/PublicationVolume"><span class="source" itemprop="isPartOf" itemscope="itemscope" itemtype="http://schema.org/Periodical"><span itemprop="name">Global Ecology and Biogeography</span></span> <b itemprop="volumeNumber">12</b></span>:<span class="fpage" itemprop="pageStart">177</span>-<span class="lpage" itemprop="pageEnd">179</span> </span>
</div></li>
<li class="ref"><div class="citation" itemprop="citation" itemscope="itemscope" itemtype="http://schema.org/Book" id="ref-40">
<span class="citation-authors-year"><b><span class="name" itemprop="author" itemscope="itemscope" itemtype="http://schema.org/Person"><span class="surname" itemprop="familyName">Tukey</span> <span class="given-names" itemprop="givenName">JW</span></span>.</b> <b class="year" itemprop="datePublished">1977</b>.</span> <cite class="article-title"></cite> <span itemprop="name"><a class="source" target="_blank" href="https://scholar.google.com/scholar_lookup?title=Exploratory%20data%20analysis&amp;author=&amp;publication_year=1977">Exploratory data analysis</a>.</span><span>  Reading: <span class="publisher">Addison-Wesley</span>.  </span>
</div></li>
<li class="ref"><div class="citation" itemprop="citation" itemscope="itemscope" itemtype="http://schema.org/ScholarlyArticle" id="ref-41">
<span class="citation-authors-year"><b><span class="name" itemprop="author" itemscope="itemscope" itemtype="http://schema.org/Person"><span class="surname" itemprop="familyName">Zahl</span> <span class="given-names" itemprop="givenName">S</span></span>.</b> <b class="year" itemprop="datePublished">1977</b>.</span> <cite itemprop="name"><a class="article-title" target="_blank" itemprop="url" href="https://doi.org/10.2307%2F1936227">Jackknifing an index of diversity</a>.</cite> <span><span class="volume" itemprop="isPartOf" itemscope="itemscope" itemtype="http://schema.org/PublicationVolume"><span class="source" itemprop="isPartOf" itemscope="itemscope" itemtype="http://schema.org/Periodical"><span itemprop="name">Ecology</span></span> <b itemprop="volumeNumber">58</b></span>:<span class="fpage" itemprop="pageStart">907</span>-<span class="lpage" itemprop="pageEnd">913</span> </span>
</div></li>
<li class="ref"><div class="citation" itemprop="citation" itemscope="itemscope" itemtype="http://schema.org/ScholarlyArticle" id="ref-42">
<span class="citation-authors-year"><b><span class="name" itemprop="author" itemscope="itemscope" itemtype="http://schema.org/Person"><span class="surname" itemprop="familyName">Zhang</span> <span class="given-names" itemprop="givenName">X</span></span>, <span class="name" itemprop="author" itemscope="itemscope" itemtype="http://schema.org/Person"><span class="surname" itemprop="familyName">Su</span> <span class="given-names" itemprop="givenName">H</span></span>, <span class="name" itemprop="author" itemscope="itemscope" itemtype="http://schema.org/Person"><span class="surname" itemprop="familyName">Yang</span> <span class="given-names" itemprop="givenName">J</span></span>, <span class="name" itemprop="author" itemscope="itemscope" itemtype="http://schema.org/Person"><span class="surname" itemprop="familyName">Feng</span> <span class="given-names" itemprop="givenName">L</span></span>, <span class="name" itemprop="author" itemscope="itemscope" itemtype="http://schema.org/Person"><span class="surname" itemprop="familyName">Li</span> <span class="given-names" itemprop="givenName">Z</span></span>, <span class="name" itemprop="author" itemscope="itemscope" itemtype="http://schema.org/Person"><span class="surname" itemprop="familyName">Zhao</span> <span class="given-names" itemprop="givenName">G</span></span>.</b> <b class="year" itemprop="datePublished">2019</b>.</span> <cite itemprop="name"><a class="article-title" target="_blank" itemprop="url" href="https://doi.org/10.1002%2Fece3.5618">Population genetic structure, migration, and polyploidy origin of a medicinal species Gynostemma pentaphyllum (Cucurbitaceae)</a></cite> <span><span class="volume" itemprop="isPartOf" itemscope="itemscope" itemtype="http://schema.org/PublicationVolume"><span class="source" itemprop="isPartOf" itemscope="itemscope" itemtype="http://schema.org/Periodical"><span itemprop="name">Ecology and Evolution</span></span> <b itemprop="volumeNumber">9</b></span>:<span class="fpage" itemprop="pageStart">11145</span>-<span class="lpage" itemprop="pageEnd">11170</span> </span>
</div></li>
</ul></section></div></div></div></footer></article>
            </div>


            <div id="related-research"><div class="trendmd-widget-container trendmd-widget_title-larger trendmd-widget_body-larger" data-trendmd-journal-id="52926" data-trendmd-article-id="5f14d993-3652-4a16-8f05-9ca90e7408fb" data-trendmd-swoop="e30=" similar-nonsponsored-merged="true" style="display: block;" data-trendmd-widget-id="0" similar-sponsored-merged="true">
  <div class="trendmd-widget trendmd-widget_horizontal">
    <div class="trendmd-widget-inner">
        <div class="trendmd-widget-section">
          <div class="trendmd-widget-header">
                          <h3 class="trendmd-widget-header__heading">Related research</h3>
                      </div>
          <ol class="trendmd-widget-list"><li class="trendmd-widget-list-item">
                <a class="trendmd-widget-list-item__link" data-trendmd-id="30099812-59d3-4fd6-901b-910be6dd4b82" data-trendmd-journal-id="52926" data-trendmd-content-category="" data-trendmd-campaign-id="0" data-trendmd-ct="%7B%22i%22:%5B%5D,%22c%22:%5B%5D,%22n%22:%5B%5D%7D" data-trendmd-sponsored="false" href="https://peerj.com/articles/1634/?utm_source=TrendMD&amp;utm_campaign=PeerJ_TrendMD_0&amp;utm_medium=TrendMD" rel="noindex, nofollow" target="_self">Estimating and comparing microbial diversity in the presence of sequencing errors.</a>
                                <div class="trendmd-widget-list-item__meta">
                                                                        
  PeerJ                                                    </div>
              </li><li class="trendmd-widget-list-item">
                <a class="trendmd-widget-list-item__link" data-trendmd-id="5bd5f8a8-5241-4c94-ad6f-b33098b2b240" data-trendmd-journal-id="52926" data-trendmd-content-category="" data-trendmd-campaign-id="0" data-trendmd-ct="%7B%22i%22:%5B%5D,%22c%22:%5B%5D,%22n%22:%5B%5D%7D" data-trendmd-sponsored="false" href="https://peerj.com/articles/8498/?utm_source=TrendMD&amp;utm_campaign=PeerJ_TrendMD_0&amp;utm_medium=TrendMD" rel="noindex, nofollow" target="_self">Analysis
 of the genetic diversity of the coastal and island endangered plant 
species Elaeagnus macrophylla via conserved DNA-derived polymorphism 
marker</a>
                                <div class="trendmd-widget-list-item__meta">
                                                                        
  PeerJ                                                    </div>
              </li><li class="trendmd-widget-list-item">
                <a class="trendmd-widget-list-item__link" data-trendmd-id="d710cc38-1700-45fc-83c6-1eea11a23004" data-trendmd-journal-id="52926" data-trendmd-content-category="" data-trendmd-campaign-id="0" data-trendmd-ct="%7B%22i%22:%5B%5D,%22c%22:%5B%5D,%22n%22:%5B%5D%7D" data-trendmd-sponsored="false" href="https://peerj.com/articles/6035/?utm_source=TrendMD&amp;utm_campaign=PeerJ_TrendMD_0&amp;utm_medium=TrendMD" rel="noindex, nofollow" target="_self">A direct approach to estimating false discovery rates conditional on covariates</a>
                                <div class="trendmd-widget-list-item__meta">
                                                                        
  PeerJ                                                    </div>
              </li><li class="trendmd-widget-list-item">
                <a class="trendmd-widget-list-item__link" data-trendmd-id="eabdffc0-9f7b-4ae3-a758-1b0b08f553f6" data-trendmd-journal-id="52926" data-trendmd-content-category="" data-trendmd-campaign-id="0" data-trendmd-ct="%7B%22i%22:%5B%5D,%22c%22:%5B%5D,%22n%22:%5B%5D%7D" data-trendmd-sponsored="false" href="https://peerj.com/articles/9593/?utm_source=TrendMD&amp;utm_campaign=PeerJ_TrendMD_0&amp;utm_medium=TrendMD" rel="noindex, nofollow" target="_self">Improved
 normalization of species count data in ecology by scaling with ranked 
subsampling (SRS): application to microbial communities</a>
                                <div class="trendmd-widget-list-item__meta">
                                                                        
  PeerJ                                                    </div>
              </li><li class="trendmd-widget-list-item">
                <a class="trendmd-widget-list-item__link" data-trendmd-id="c6ee04bd-8b67-45ff-a431-e460d204f8cd" data-trendmd-journal-id="52926" data-trendmd-content-category="" data-trendmd-campaign-id="0" data-trendmd-ct="%7B%22i%22:%5B%5D,%22c%22:%5B%5D,%22n%22:%5B%5D%7D" data-trendmd-sponsored="false" href="https://peerj.com/articles/1793/?utm_source=TrendMD&amp;utm_campaign=PeerJ_TrendMD_0&amp;utm_medium=TrendMD" rel="noindex, nofollow" target="_self">Genetic diversity and structure in the Endangered Allen Cays Rock Iguana, Cyclura cychlura inornata.</a>
                                <div class="trendmd-widget-list-item__meta">
                                                                        
  PeerJ                                                    </div>
              </li></ol>
        </div>
      
        <div class="trendmd-widget-section">
          <div class="trendmd-widget-header">
                      </div>
          <ol class="trendmd-widget-list"><li class="trendmd-widget-list-item">
                <a class="trendmd-widget-list-item__link" data-trendmd-id="b39f32d4-bf36-4b81-a21c-a89254143e98" data-trendmd-journal-id="59836" data-trendmd-content-category="" data-trendmd-campaign-id="14992" data-trendmd-ct="%7B%22i%22:true,%22c%22:true,%22n%22:%7B%22specificNpi%22:false,%22anyNpi%22:false%7D%7D" data-trendmd-sponsored="true" href="https://www.emerald.com/insight/content/doi/10.1108/03068290410555381/full/html?utm_source=TrendMD&amp;utm_medium=cpc&amp;utm_campaign=International_Journal_of_Social_Economics_TrendMD_1&amp;WT.mc_id=Emerald_TrendMD_1" rel="noindex, nofollow" target="_blank">Empirical study in finite correlation coefficient in two phase estimation<span class="trendmd-widget-list-item__link__external"></span></a>
                                <div class="trendmd-widget-list-item__meta">
                                                        <span class="js_authors">M. Khoshnevisan et al.,</span>
                                                        International 
Journal of Social Economics,                                            
             <span class="js_publication_date">2004</span>
                                  </div>
              </li><li class="trendmd-widget-list-item">
                <a class="trendmd-widget-list-item__link" data-trendmd-id="b61569b8-f173-4292-bba0-cf4c4b1376ec" data-trendmd-journal-id="74362" data-trendmd-content-category="None" data-trendmd-campaign-id="11621" data-trendmd-ct="%7B%22i%22:true,%22c%22:true,%22n%22:%7B%22specificNpi%22:false,%22anyNpi%22:false%7D%7D" data-trendmd-sponsored="true" href="https://www.tandfonline.com/doi/full/10.1080/24754269.2019.1677085?utm_source=TrendMD&amp;utm_medium=cpc&amp;utm_campaign=Statistical_Theory_and_Related_Fields_TrendMD_1" rel="noindex, nofollow" target="_blank">Using
 state space models as a statistical impact measurement of survey 
redesigns: a case study of the labour force survey of the Australian 
Bureau of Statistics<span class="trendmd-widget-list-item__link__external"></span></a>
                                <div class="trendmd-widget-list-item__meta">
                                                        <span class="js_authors">Xichuan (Mark) Zhang et al.,</span>
                                                        Statistical 
Theory and Related Fields,                                              
           <span class="js_publication_date">2019</span>
                                  </div>
              </li><li class="trendmd-widget-list-item">
                <a class="trendmd-widget-list-item__link" data-trendmd-id="f7f59b90-8699-4123-ae1f-6893a2155c64" data-trendmd-journal-id="59596" data-trendmd-content-category="" data-trendmd-campaign-id="14992" data-trendmd-ct="%7B%22i%22:true,%22c%22:true,%22n%22:%7B%22specificNpi%22:false,%22anyNpi%22:false%7D%7D" data-trendmd-sponsored="true" href="http://www.emeraldinsight.com/doi/abs/10.1108/SRJ-05-2016-0061?utm_source=TrendMD&amp;utm_medium=cpc&amp;utm_campaign=Social_Responsibility_Journal_TrendMD_1&amp;WT.mc_id=Emerald_TrendMD_1" rel="noindex, nofollow" target="_blank">Corporate governance: evidence from Islamic banks<span class="trendmd-widget-list-item__link__external"></span></a>
                                <div class="trendmd-widget-list-item__meta">
                                                        <span class="js_authors">Ali R. Almutairi,</span>
                                                        Social 
Responsibility Journal,                                                 
        <span class="js_publication_date">2017</span>
                                  </div>
              </li><li class="trendmd-widget-list-item">
                <a class="trendmd-widget-list-item__link" data-trendmd-id="b691b149-2d4e-48d9-aa7d-4217d3b35f79" data-trendmd-journal-id="69640" data-trendmd-content-category="" data-trendmd-campaign-id="15169" data-trendmd-ct="%7B%22i%22:true,%22c%22:true,%22n%22:%7B%22specificNpi%22:false,%22anyNpi%22:false%7D%7D" data-trendmd-sponsored="true" href="https://www.precisiononcologynews.com/regulatory-news-fda-approvals/fda-approves-braftovi-erbitux-combination-braf-v600e-mutated?utm_source=TrendMD&amp;utm_medium=TrendMD&amp;utm_campaign=1&amp;trendmd-shared=1#.Xo9OZYhKjIU" rel="noindex, nofollow" target="_blank">FDA Approves Braftovi, Erbitux Combination for BRAF V600E-Mutated Colorectal Cancer Patients<span class="trendmd-widget-list-item__link__external"></span></a>
                                <div class="trendmd-widget-list-item__meta">
                                                                        
  Precision Oncology News,                                              
           <span class="js_publication_date">2020</span>
                                  </div>
              </li><li class="trendmd-widget-list-item">
                <a class="trendmd-widget-list-item__link" data-trendmd-id="63deb5f5-c634-4f26-84f7-6e4afa5116c6" data-trendmd-journal-id="74362" data-trendmd-content-category="None" data-trendmd-campaign-id="11621" data-trendmd-ct="%7B%22i%22:true,%22c%22:true,%22n%22:%7B%22specificNpi%22:false,%22anyNpi%22:false%7D%7D" data-trendmd-sponsored="true" href="https://www.tandfonline.com/doi/full/10.1080/24754269.2019.1675409?utm_source=TrendMD&amp;utm_medium=cpc&amp;utm_campaign=Statistical_Theory_and_Related_Fields_TrendMD_1" rel="noindex, nofollow" target="_blank">Nearest neighbour imputation under single index models<span class="trendmd-widget-list-item__link__external"></span></a>
                                <div class="trendmd-widget-list-item__meta">
                                                        <span class="js_authors">Jun Shao et al.,</span>
                                                        Statistical 
Theory and Related Fields,                                              
           <span class="js_publication_date">2019</span>
                                  </div>
              </li></ol>
        </div>
      
    </div>
    <div class="trendmd-widget-footer">
      <div class="trendmd-widget-footer-inner">
        <div class="trendmd-widget-footer-inner-left">
          <a href="https://www.trendmd.com/how-it-works-readers" target="_blank" class="trendmd-widget-brand" title="TrendMD">
            <span class="trendmd-widget-brand__recommended">Powered by</span>
            <span class="trendmd-widget-brand__logo trendmd-widget-brand__logo__faded" style="display: inline-block; width: 93px">
              <svg viewBox="0 0 234 57" version="1.1" xmlns="http://www.w3.org/2000/svg" xlink="http://www.w3.org/1999/xlink">
                <g id="logo">
                  <path d="M227.7,56.9 L149.7,56.9 C146.7,56.9 143.7,53.8 143.7,50.9 L143.7,6.9 C143.7,4 146.7,0.9 149.7,0.9 L227.7,0.9 C230.7,0.9 233.7,4 233.7,6.9 L233.7,50.9 C233.7,53.9 230.6,56.9 227.7,56.9 L227.7,56.9 Z" id="shape0" fill="#CA2127"></path>
                  <g id="group1" transform="translate(0.000000, 14.000000)" fill="#16325c">
                    <path d="M15.8,5.9 L15.8,29.6 L9.2,29.6 L9.2,5.9 L0.9,5.9 L0.9,0.2 L24.1,0.2 L24.1,5.9 L15.8,5.9 L15.8,5.9 Z" id="shape1"></path>
                    <path d="M50.2,10 C50.2,14.7 48.3,17.7 44.6,19.1 L52,29.6 L44,29.6 L37.5,20.2 L33,20.2 L33,29.6 L26.4,29.6 L26.4,0.2 L37.5,0.2 C42.1,0.2 45.3,1 47.3,2.5 C49.2,4.1 50.2,6.6 50.2,10 L50.2,10 Z M42.3,13.5 C43.1,12.8 43.5,11.6 43.5,10 C43.5,8.4 43.1,7.3 42.2,6.7 C41.4,6.1 39.9,5.8 37.8,5.8 L32.9,5.8 L32.9,14.5 L37.7,14.5 C40,14.6 41.5,14.2 42.3,13.5 L42.3,13.5 Z" id="shape2"></path>
                    <path d="M75.6,0.3 L75.6,6.1 L61,6.1 L61,12.1 L74.1,12.1 L74.1,17.7 L61,17.7 L61,23.8 L76,23.8 L76,29.6 L54.4,29.6 L54.4,0.2 L75.6,0.2 L75.6,0.3 Z" id="shape3"></path>
                    <path d="M99.9,0.3 L106.5,0.3 L106.5,29.7 L99.9,29.7 L85.9,11.3 L85.9,29.7 L79.3,29.7 L79.3,0.3 L85.4,0.3 L99.8,19.2 L99.8,0.3 L99.9,0.3 Z" id="shape4"></path>
                    <path d="M133.6,4.1 C136.4,6.7 137.8,10.2 137.8,14.8 C137.8,19.4 136.4,23 133.7,25.6 C131,28.3 126.8,29.6 121.2,29.6 L111.2,29.6 L111.2,0.2 L121.6,0.2 C126.7,0.3 130.8,1.5 133.6,4.1 L133.6,4.1 Z M128.8,21.6 C130.4,20.1 131.2,17.9 131.2,15 C131.2,12.1 130.4,9.9 128.8,8.3 C127.2,6.7 124.7,6 121.4,6 L117.7,6 L117.7,23.9 L121.9,23.9 C124.8,23.8 127.1,23.1 128.8,21.6 L128.8,21.6 Z" id="shape5"></path>
                  </g>
                  <g id="group2" transform="translate(156.000000, 14.000000)" fill="#FFFFFF">
                    <path d="M26.9,11.1 L19,27.1 L15,27.1 L7.1,11.1 L7.1,29.6 L0.5,29.6 L0.5,0.2 L9.4,0.2 L17,16.3 L24.6,0.2 L33.4,0.2 L33.4,29.6 L26.8,29.6 L26.8,11.1 L26.9,11.1 Z" id="shape6"></path>
                    <path d="M60.6,4.1 C63.4,6.7 64.8,10.2 64.8,14.8 C64.8,19.4 63.4,23 60.7,25.6 C58,28.3 53.8,29.6 48.2,29.6 L38.2,29.6 L38.2,0.2 L48.6,0.2 C53.7,0.3 57.8,1.5 60.6,4.1 L60.6,4.1 Z M55.8,21.6 C57.4,20.1 58.2,17.9 58.2,15 C58.2,12.1 57.4,9.9 55.8,8.3 C54.2,6.7 51.7,6 48.4,6 L44.7,6 L44.7,23.9 L48.9,23.9 C51.8,23.8 54.1,23.1 55.8,21.6 L55.8,21.6 Z" id="shape7"></path>
                  </g>
                </g>
              </svg>
            </span>
          </a>
        </div>
        <div class="trendmd-widget-footer-inner-right">
                      <span class="trendmd-widget-footer__login-text" style="display:none">
              <a class="trendmd-widget-footer__login-link" href="https://reader.trendmd.com/login">Sign in</a> or
              <a class="trendmd-widget-footer__login-link" href="https://reader.trendmd.com/sign_up?wid=52926">create an account</a> for a better, personalized experience
            </span>
                    <div class="trendmd-widget-settings"><span class="trendmd-widget-settings__cog"></span>
            <div class="trendmd-widget-settings__modal"><ul class="trendmd-widget-settings__list">
                                                  <li class="trendmd-widget-settings__item">
                    <a href="https://www.trendmd.com/privacy" target="_blank" class="trendmd-widget-settings__link">Privacy policy</a>
                  </li>
                                <li class="trendmd-widget-settings__item trendmd-widget-dns" style="display:none">
                  <a href="https://www.trendmd.com/personaldata" target="_blank" class="trendmd-widget-settings__link">Do not sell my personal information</a>
                </li>
                <li class="trendmd-widget-settings__item">
                  <a href="https://www.trendmd.com/google-analytics" target="_blank" class="trendmd-widget-settings__link">Google Analytics settings</a>
                </li>
                              </ul></div>
          </div>
        </div>
        <div class="trendmd-widget-footer-inner-clear"></div>
        <div class="trendmd-widget-footer-inner-bottom">
                      <div class="trendmd-widget-cookie-notification" style="display: block!important">
              <div class="trendmd-widget-cookie-notification__info"><span>I consent to the use of Google Analytics and related cookies across the TrendMD network (widget, website, blog). </span><a href="https://www.trendmd.com/google-analytics" target="_blank">Learn more</a></div>
              <div class="trendmd-widget-cookie-notification__actions">
                <span></span>
                <button onclick="(function() { window.TrendMD.gdpr().setGaConsent(true); })(); return false;" style="display: inline-block;">Yes</button>
                <button onclick="(function() { window.TrendMD.gdpr().setGaConsent(false); })(); return false;" style="display: inline-block;">No</button>
              </div>
            </div>
                  </div>
      </div>
    </div>
  </div>
  
    <div id="trendmd-check">
      <div id="ad-holder"></div>
    </div>
  
</div>
</div>

            <!-- annotations -->
            <ul class="nav nav-tabs annotation-tabs-nav">
                <li class="active"><a href="#questions" data-toggle="tab"><i class="icon-comments"></i> Questions
                        <span class="annotation-counter annotation-counter-questioning" style="display: none;">0</span></a></li>
                <li><a href="#links" data-toggle="tab"><i class="icon-link"></i> Links
                        <span class="annotation-counter annotation-counter-linking" style="display: none;">0</span></a></li>
            </ul>

            <div class="tab-content annotation-tab-content">
                <div class="tab-pane active" id="questions">
                    <div id="questions-summaries"></div><div class="annotations" id="questions" data-target="articles/9391" data-counts="1">
    <div class="row-fluid row-article-item-section">
        <div class="span1 article-main-left-span1">&nbsp;</div>
        <div class="span11 article-item-section-content">
            
            <div>
                <div class="annotations" data-total-results="0" data-collection-motivation="questioning">
    </div>
            </div>

                            <a class="btn btn-primary annotation-create-button add-annotation" id="annotation-create-question" data-toggle="annotation-form" data-target="#annotation-question-create-container" rel="nofollow" href="https://peerj.com/questions.form?format=html&amp;target=articles/9391&amp;_counts=1"><i class="icon-plus"></i> Ask a question</a>
                <div class="help-block annotation-learn-more"><a href="https://peerj.com/about/FAQ/academic-contribution/" target="_blank">Learn more about Q&amp;A</a></div>
                <div class="annotation-form-container" id="annotation-question-create-container"></div>
                    </div>
    </div>
</div>
                </div>

                <div class="tab-pane" id="links">
                    <div class="annotations" id="links" data-target="articles/9391" data-counts="1">
    <div class="row-fluid row-article-item-section">
        <div class="span1 article-main-left-span1">&nbsp;</div>
        <div class="span11 article-item-section-content">
            
            <div>
                <div class="annotations" data-total-results="0" data-collection-motivation="linking">
    </div>
            </div>

            <a class="btn btn-primary annotation-create-button add-annotation" id="annotation-create-link" data-toggle="annotation-form" data-target="#annotation-link-create-container" rel="nofollow" href="https://peerj.com/links.form?format=html&amp;target=articles/9391&amp;_counts=1"><i class="icon-plus"></i> Add a link</a>
            <div class="annotation-form-container" id="annotation-link-create-container"></div>
        </div>
    </div>
</div>
                </div>
            </div>

            <div class="hidden-desktop" id="mobile-featured-jobs"><div class="featured-job-container"><p class="featured-job-header">Research Jobs, Collaborations, &amp; Tasks</p>
                    List &amp; find research jobs, collaborations, or tasks on PeerJ for free.<br><a class="job-featured-cta" href="https://peerj.com/expertrxiv/">Learn more <i class="icon-angle-right"></i></a></div></div>
        </div>

        <!-- Right sidebar -->
        <div class="span3 offset1 article-sidebar visible-desktop">
                <div id="article-sidebar-main-content" data-todo-href="/todos/44110/">
                    <div class="dimensions-stats-container">
            <span class="__dimensions_badge_embed__" data-doi="10.7717/peerj.9391" data-hide-zero-citations="true" data-legend="always" data-style="small_circle"></span>
        </div>
    
    
        <form action="/search/" id="institution-page-cta"><div class="institution-cta-search-title">Enter your institution</div><div>To find colleagues at PeerJ</div><input name="q" placeholder="Enter to search" type="search"><input name="type" value="institutions" type="hidden"></form><div class="row-fluid item-action-buttons article-sidebar-item">
            <div class="span12">
                <a href="https://peerj.com/benefits/" class="author-quote article-author-quote-link">
    <div class="author-quote-text">
        <span class="lead-in">I published in PeerJ</span> and it is very
 fast, has good editors, has consistently given good quality and 
rigorous reviews of my work, and produces visually appealing 
manuscripts.</div>
    <div class="author-quote-details">
        <span class="author-quote-name">Matthew Jackson</span><br>
        PeerJ author
    </div>
</a>                <div class="article-free-publishing-cta">
    <div class="article-free-publishing-cta-title">Publish Free in 2020</div>
    <div class="article-free-publishing-cta-subline">In PeerJ Chemistry Journals</div>
    <a href="https://peerj.com/blog/post/115284881305/free-open-access-publishing-for-chemistry-and-computer-science-subject-areas" class="btn btn-article article-free-publishing-cta-btn">
        Learn more
    </a>
</div>                                                                                                        <div id="download-modal-trigger" class="js-download-modal-trigger btn btn-article btn-download btn-success mb-3  ">
    Download
</div>                <!--<div class="content-cta-intro-text">Want alerts from articles like this?</div>-->
<div id="content-alert-link" class="content-alert-link-btn" data-href="/content-alert/?aid=44110">
    <div id="content-alert-button-label">
        <i class="icon-envelope btn-content-alert-icon"></i>
        Content <div class="content-alert-btn-lastword">Alert</div>
    </div>
    <div id="content-alert-button-loading" style="display:none;"><i class="icon-spin icon-spinner"></i> Loading...</div>
</div>
    <div class="content-cta-help-text">
        Just enter your email
    </div>
                            </div>
        </div>
    
    
                
        
        <nav class="article-sidebar-block">
        <div class="sidebar-heading">
            <i class="icon-wrench"></i> Tools &amp; info
        </div>
        <ul class="nav nav-list article-item-metrics-counts" data-src="/articles/9391/counter/">
                            <li>
                    <a href="https://peerj.com/articles/9391/reviews/" rel="version-history">Peer Review information</a>
                </li>
            
                            
                                    <li><a href="https://scholar.google.com/scholar?cites=https%3A%2F%2Fpeerj.com%2Farticles%2F9391%2F" target="_blank">Citations in Google Scholar</a></li>
                            
            
            <li><a href="#questions">Ask questions
                    <span class="metric-counter annotation-counter-questioning" style="display: none;">0</span></a></li>

            <li><a href="#links">Add links
                    <span class="metric-counter annotation-counter-linking" style="display: none;">0</span></a></li>

                            <li class="article-item-metrics-count" style="display: list-item;"><a data-toggle="modal" href="#metricsModal">Visitors <span class="metric-counter" data-count="visitors">813</span> <span class="pull-right metric-counter-details-cta">click for details</span></a></li>
    <li class="article-item-metrics-count" style="display: list-item;"><a data-toggle="modal" href="#metricsModal">Views <span class="metric-counter" data-count="views-html">1,040</span></a></li>
    <li class="article-item-metrics-count" style="display: list-item;"><a data-toggle="modal" href="#metricsModal">Downloads <span class="metric-counter" data-count="views-pdf">252</span></a></li>

                            <li><a id="item-flag-button" data-toggle="modal" href="#flagModal">Report problem with article</a></li>
                    </ul>
    </nav>

        
    <div id="related-research-sidebar"><div class="trendmd-widget-container trendmd-widget_title-larger trendmd-widget_body-larger" data-trendmd-journal-id="52926" data-trendmd-article-id="5f14d993-3652-4a16-8f05-9ca90e7408fb" data-trendmd-swoop="e30=" similar-sponsored-merged="true" style="display: block;" data-trendmd-widget-id="0" similar-nonsponsored-merged="true">
  <div class="trendmd-widget">
    <div class="trendmd-widget-inner">
        <div class="trendmd-widget-section">
          <div class="trendmd-widget-header">
                          <h3 class="trendmd-widget-header__heading">Related research</h3>
                      </div>
          <ol class="trendmd-widget-list"><li class="trendmd-widget-list-item">
                <a class="trendmd-widget-list-item__link" data-trendmd-id="30099812-59d3-4fd6-901b-910be6dd4b82" data-trendmd-journal-id="52926" data-trendmd-content-category="" data-trendmd-campaign-id="0" data-trendmd-ct="%7B%22i%22:%5B%5D,%22c%22:%5B%5D,%22n%22:%5B%5D%7D" data-trendmd-sponsored="false" href="https://peerj.com/articles/1634/?utm_source=TrendMD&amp;utm_campaign=PeerJ_TrendMD_0&amp;utm_medium=TrendMD" rel="noindex, nofollow" target="_self">Estimating and comparing microbial diversity in the presence of sequencing errors.</a>
                                <div class="trendmd-widget-list-item__meta">
                                                                        
  PeerJ                                                    </div>
              </li><li class="trendmd-widget-list-item">
                <a class="trendmd-widget-list-item__link" data-trendmd-id="5bd5f8a8-5241-4c94-ad6f-b33098b2b240" data-trendmd-journal-id="52926" data-trendmd-content-category="" data-trendmd-campaign-id="0" data-trendmd-ct="%7B%22i%22:%5B%5D,%22c%22:%5B%5D,%22n%22:%5B%5D%7D" data-trendmd-sponsored="false" href="https://peerj.com/articles/8498/?utm_source=TrendMD&amp;utm_campaign=PeerJ_TrendMD_0&amp;utm_medium=TrendMD" rel="noindex, nofollow" target="_self">Analysis
 of the genetic diversity of the coastal and island endangered plant 
species Elaeagnus macrophylla via conserved DNA-derived polymorphism 
marker</a>
                                <div class="trendmd-widget-list-item__meta">
                                                                        
  PeerJ                                                    </div>
              </li><li class="trendmd-widget-list-item">
                <a class="trendmd-widget-list-item__link" data-trendmd-id="eabdffc0-9f7b-4ae3-a758-1b0b08f553f6" data-trendmd-journal-id="52926" data-trendmd-content-category="" data-trendmd-campaign-id="0" data-trendmd-ct="%7B%22i%22:%5B%5D,%22c%22:%5B%5D,%22n%22:%5B%5D%7D" data-trendmd-sponsored="false" href="https://peerj.com/articles/9593/?utm_source=TrendMD&amp;utm_campaign=PeerJ_TrendMD_0&amp;utm_medium=TrendMD" rel="noindex, nofollow" target="_self">Improved
 normalization of species count data in ecology by scaling with ranked 
subsampling (SRS): application to microbial communities</a>
                                <div class="trendmd-widget-list-item__meta">
                                                                        
  PeerJ                                                    </div>
              </li><li class="trendmd-widget-list-item" style="display: none;">
                <a class="trendmd-widget-list-item__link" data-trendmd-id="c6ee04bd-8b67-45ff-a431-e460d204f8cd" data-trendmd-journal-id="52926" data-trendmd-content-category="" data-trendmd-campaign-id="0" data-trendmd-ct="%7B%22i%22:%5B%5D,%22c%22:%5B%5D,%22n%22:%5B%5D%7D" data-trendmd-sponsored="false" href="https://peerj.com/articles/1793/?utm_source=TrendMD&amp;utm_campaign=PeerJ_TrendMD_0&amp;utm_medium=TrendMD" rel="noindex, nofollow" target="_self">Genetic diversity and structure in the Endangered Allen Cays Rock Iguana, Cyclura cychlura inornata.</a>
                                <div class="trendmd-widget-list-item__meta">
                                                                        
  PeerJ                                                    </div>
              </li><li class="trendmd-widget-list-item" style="display: none;">
                <a class="trendmd-widget-list-item__link" data-trendmd-id="d710cc38-1700-45fc-83c6-1eea11a23004" data-trendmd-journal-id="52926" data-trendmd-content-category="" data-trendmd-campaign-id="0" data-trendmd-ct="%7B%22i%22:%5B%5D,%22c%22:%5B%5D,%22n%22:%5B%5D%7D" data-trendmd-sponsored="false" href="https://peerj.com/articles/6035/?utm_source=TrendMD&amp;utm_campaign=PeerJ_TrendMD_0&amp;utm_medium=TrendMD" rel="noindex, nofollow" target="_self">A direct approach to estimating false discovery rates conditional on covariates</a>
                                <div class="trendmd-widget-list-item__meta">
                                                                        
  PeerJ                                                    </div>
              </li></ol>
        </div>
      
        <div class="trendmd-widget-section">
          <div class="trendmd-widget-header">
                      </div>
          <ol class="trendmd-widget-list"><li class="trendmd-widget-list-item" style="display: none;">
                <a class="trendmd-widget-list-item__link" data-trendmd-id="b61569b8-f173-4292-bba0-cf4c4b1376ec" data-trendmd-journal-id="74362" data-trendmd-content-category="None" data-trendmd-campaign-id="11621" data-trendmd-ct="%7B%22i%22:true,%22c%22:true,%22n%22:%7B%22specificNpi%22:false,%22anyNpi%22:false%7D%7D" data-trendmd-sponsored="true" href="https://www.tandfonline.com/doi/full/10.1080/24754269.2019.1677085?utm_source=TrendMD&amp;utm_medium=cpc&amp;utm_campaign=Statistical_Theory_and_Related_Fields_TrendMD_1" rel="noindex, nofollow" target="_blank">Using
 state space models as a statistical impact measurement of survey 
redesigns: a case study of the labour force survey of the Australian 
Bureau of Statistics<span class="trendmd-widget-list-item__link__external"></span></a>
                                <div class="trendmd-widget-list-item__meta">
                                                        <span class="js_authors">Xichuan (Mark) Zhang et al.,</span>
                                                        Statistical 
Theory and Related Fields,                                              
           <span class="js_publication_date">2019</span>
                                  </div>
              </li><li class="trendmd-widget-list-item" style="display: none;">
                <a class="trendmd-widget-list-item__link" data-trendmd-id="b39f32d4-bf36-4b81-a21c-a89254143e98" data-trendmd-journal-id="59836" data-trendmd-content-category="" data-trendmd-campaign-id="14992" data-trendmd-ct="%7B%22i%22:true,%22c%22:true,%22n%22:%7B%22specificNpi%22:false,%22anyNpi%22:false%7D%7D" data-trendmd-sponsored="true" href="https://www.emerald.com/insight/content/doi/10.1108/03068290410555381/full/html?utm_source=TrendMD&amp;utm_medium=cpc&amp;utm_campaign=International_Journal_of_Social_Economics_TrendMD_1&amp;WT.mc_id=Emerald_TrendMD_1" rel="noindex, nofollow" target="_blank">Empirical study in finite correlation coefficient in two phase estimation<span class="trendmd-widget-list-item__link__external"></span></a>
                                <div class="trendmd-widget-list-item__meta">
                                                        <span class="js_authors">M. Khoshnevisan et al.,</span>
                                                        International 
Journal of Social Economics,                                            
             <span class="js_publication_date">2004</span>
                                  </div>
              </li><li class="trendmd-widget-list-item" style="display: none;">
                <a class="trendmd-widget-list-item__link" data-trendmd-id="f7f59b90-8699-4123-ae1f-6893a2155c64" data-trendmd-journal-id="59596" data-trendmd-content-category="" data-trendmd-campaign-id="14992" data-trendmd-ct="%7B%22i%22:true,%22c%22:true,%22n%22:%7B%22specificNpi%22:false,%22anyNpi%22:false%7D%7D" data-trendmd-sponsored="true" href="http://www.emeraldinsight.com/doi/abs/10.1108/SRJ-05-2016-0061?utm_source=TrendMD&amp;utm_medium=cpc&amp;utm_campaign=Social_Responsibility_Journal_TrendMD_1&amp;WT.mc_id=Emerald_TrendMD_1" rel="noindex, nofollow" target="_blank">Corporate governance: evidence from Islamic banks<span class="trendmd-widget-list-item__link__external"></span></a>
                                <div class="trendmd-widget-list-item__meta">
                                                        <span class="js_authors">Ali R. Almutairi,</span>
                                                        Social 
Responsibility Journal,                                                 
        <span class="js_publication_date">2017</span>
                                  </div>
              </li><li class="trendmd-widget-list-item" style="display: none;">
                <a class="trendmd-widget-list-item__link" data-trendmd-id="63deb5f5-c634-4f26-84f7-6e4afa5116c6" data-trendmd-journal-id="74362" data-trendmd-content-category="None" data-trendmd-campaign-id="11621" data-trendmd-ct="%7B%22i%22:true,%22c%22:true,%22n%22:%7B%22specificNpi%22:false,%22anyNpi%22:false%7D%7D" data-trendmd-sponsored="true" href="https://www.tandfonline.com/doi/full/10.1080/24754269.2019.1675409?utm_source=TrendMD&amp;utm_medium=cpc&amp;utm_campaign=Statistical_Theory_and_Related_Fields_TrendMD_1" rel="noindex, nofollow" target="_blank">Nearest neighbour imputation under single index models<span class="trendmd-widget-list-item__link__external"></span></a>
                                <div class="trendmd-widget-list-item__meta">
                                                        <span class="js_authors">Jun Shao et al.,</span>
                                                        Statistical 
Theory and Related Fields,                                              
           <span class="js_publication_date">2019</span>
                                  </div>
              </li><li class="trendmd-widget-list-item" style="display: none;">
                <a class="trendmd-widget-list-item__link" data-trendmd-id="a301b8a1-f2f4-4a75-bef4-6d0c20eab921" data-trendmd-journal-id="54916" data-trendmd-content-category="" data-trendmd-campaign-id="15171" data-trendmd-ct="%7B%22i%22:true,%22c%22:true,%22n%22:%7B%22specificNpi%22:false,%22anyNpi%22:false%7D%7D" data-trendmd-sponsored="true" href="https://www.360dx.com/business-news/invitaes-transition-genomic-information-company-likely-include-more-ma-activity?utm_medium=TrendMD&amp;utm_campaign=1&amp;utm_source=TrendMD&amp;trendmd-shared=1#.Wwz0WxMrIWp" rel="noindex, nofollow" target="_blank">Invitae's Transition to 'Genomic Information Company' Likely to Include More M&amp;A Activity<span class="trendmd-widget-list-item__link__external"></span></a>
                                <div class="trendmd-widget-list-item__meta">
                                                        <span class="js_authors">Turna Ray,</span>
                                                        360Dx,                                                         <span class="js_publication_date">2017</span>
                                  </div>
              </li></ol>
        </div>
      
    </div>
    <div class="trendmd-widget-footer">
      <div class="trendmd-widget-footer-inner">
        <div class="trendmd-widget-footer-inner-left">
          <a href="https://www.trendmd.com/how-it-works-readers" target="_blank" class="trendmd-widget-brand" title="TrendMD">
            <span class="trendmd-widget-brand__recommended">Powered by</span>
            <span class="trendmd-widget-brand__logo trendmd-widget-brand__logo__faded" style="display: inline-block; width: 93px">
              <svg viewBox="0 0 234 57" version="1.1" xmlns="http://www.w3.org/2000/svg" xlink="http://www.w3.org/1999/xlink">
                <g id="logo">
                  <path d="M227.7,56.9 L149.7,56.9 C146.7,56.9 143.7,53.8 143.7,50.9 L143.7,6.9 C143.7,4 146.7,0.9 149.7,0.9 L227.7,0.9 C230.7,0.9 233.7,4 233.7,6.9 L233.7,50.9 C233.7,53.9 230.6,56.9 227.7,56.9 L227.7,56.9 Z" id="shape0" fill="#CA2127"></path>
                  <g id="group1" transform="translate(0.000000, 14.000000)" fill="#16325c">
                    <path d="M15.8,5.9 L15.8,29.6 L9.2,29.6 L9.2,5.9 L0.9,5.9 L0.9,0.2 L24.1,0.2 L24.1,5.9 L15.8,5.9 L15.8,5.9 Z" id="shape1"></path>
                    <path d="M50.2,10 C50.2,14.7 48.3,17.7 44.6,19.1 L52,29.6 L44,29.6 L37.5,20.2 L33,20.2 L33,29.6 L26.4,29.6 L26.4,0.2 L37.5,0.2 C42.1,0.2 45.3,1 47.3,2.5 C49.2,4.1 50.2,6.6 50.2,10 L50.2,10 Z M42.3,13.5 C43.1,12.8 43.5,11.6 43.5,10 C43.5,8.4 43.1,7.3 42.2,6.7 C41.4,6.1 39.9,5.8 37.8,5.8 L32.9,5.8 L32.9,14.5 L37.7,14.5 C40,14.6 41.5,14.2 42.3,13.5 L42.3,13.5 Z" id="shape2"></path>
                    <path d="M75.6,0.3 L75.6,6.1 L61,6.1 L61,12.1 L74.1,12.1 L74.1,17.7 L61,17.7 L61,23.8 L76,23.8 L76,29.6 L54.4,29.6 L54.4,0.2 L75.6,0.2 L75.6,0.3 Z" id="shape3"></path>
                    <path d="M99.9,0.3 L106.5,0.3 L106.5,29.7 L99.9,29.7 L85.9,11.3 L85.9,29.7 L79.3,29.7 L79.3,0.3 L85.4,0.3 L99.8,19.2 L99.8,0.3 L99.9,0.3 Z" id="shape4"></path>
                    <path d="M133.6,4.1 C136.4,6.7 137.8,10.2 137.8,14.8 C137.8,19.4 136.4,23 133.7,25.6 C131,28.3 126.8,29.6 121.2,29.6 L111.2,29.6 L111.2,0.2 L121.6,0.2 C126.7,0.3 130.8,1.5 133.6,4.1 L133.6,4.1 Z M128.8,21.6 C130.4,20.1 131.2,17.9 131.2,15 C131.2,12.1 130.4,9.9 128.8,8.3 C127.2,6.7 124.7,6 121.4,6 L117.7,6 L117.7,23.9 L121.9,23.9 C124.8,23.8 127.1,23.1 128.8,21.6 L128.8,21.6 Z" id="shape5"></path>
                  </g>
                  <g id="group2" transform="translate(156.000000, 14.000000)" fill="#FFFFFF">
                    <path d="M26.9,11.1 L19,27.1 L15,27.1 L7.1,11.1 L7.1,29.6 L0.5,29.6 L0.5,0.2 L9.4,0.2 L17,16.3 L24.6,0.2 L33.4,0.2 L33.4,29.6 L26.8,29.6 L26.8,11.1 L26.9,11.1 Z" id="shape6"></path>
                    <path d="M60.6,4.1 C63.4,6.7 64.8,10.2 64.8,14.8 C64.8,19.4 63.4,23 60.7,25.6 C58,28.3 53.8,29.6 48.2,29.6 L38.2,29.6 L38.2,0.2 L48.6,0.2 C53.7,0.3 57.8,1.5 60.6,4.1 L60.6,4.1 Z M55.8,21.6 C57.4,20.1 58.2,17.9 58.2,15 C58.2,12.1 57.4,9.9 55.8,8.3 C54.2,6.7 51.7,6 48.4,6 L44.7,6 L44.7,23.9 L48.9,23.9 C51.8,23.8 54.1,23.1 55.8,21.6 L55.8,21.6 Z" id="shape7"></path>
                  </g>
                </g>
              </svg>
            </span>
          </a>
        </div>
        <div class="trendmd-widget-footer-inner-right">
                      <span class="trendmd-widget-footer__login-text" style="display:none">
              <a class="trendmd-widget-footer__login-link" href="https://reader.trendmd.com/login">Sign in</a> or
              <a class="trendmd-widget-footer__login-link" href="https://reader.trendmd.com/sign_up?wid=52926">create an account</a> for a better, personalized experience
            </span>
                    <div class="trendmd-widget-settings"><span class="trendmd-widget-settings__cog"></span>
            <div class="trendmd-widget-settings__modal">
              <ul class="trendmd-widget-settings__list">
                                                  <li class="trendmd-widget-settings__item">
                    <a href="https://www.trendmd.com/privacy" target="_blank" class="trendmd-widget-settings__link">Privacy policy</a>
                  </li>
                                <li class="trendmd-widget-settings__item trendmd-widget-dns" style="display:none">
                  <a href="https://www.trendmd.com/personaldata" target="_blank" class="trendmd-widget-settings__link">Do not sell my personal information</a>
                </li>
                <li class="trendmd-widget-settings__item">
                  <a href="https://www.trendmd.com/google-analytics" target="_blank" class="trendmd-widget-settings__link">Google Analytics settings</a>
                </li>
                              </ul>
            </div>
          </div>
        </div>
        <div class="trendmd-widget-footer-inner-clear"></div>
        <div class="trendmd-widget-footer-inner-bottom">
                      <div class="trendmd-widget-cookie-notification">
              <div class="trendmd-widget-cookie-notification__info"><span>I consent to the use of Google Analytics and related cookies across the TrendMD network (widget, website, blog). </span><a href="https://www.trendmd.com/google-analytics" target="_blank">Learn more</a></div>
              <div class="trendmd-widget-cookie-notification__actions">
                <span></span>
                <button onclick="(function() { window.TrendMD.gdpr().setGaConsent(true); })(); return false;" style="display: inline-block;">Yes</button>
                <button onclick="(function() { window.TrendMD.gdpr().setGaConsent(false); })(); return false;" style="display: inline-block;">No</button>
              </div>
            </div>
                  </div>
      </div>
    </div>
  </div>
  
    <div id="trendmd-check">
      <div id="ad-holder"></div>
    </div>
  
</div>
</div>

</div>
<nav class="article-sidebar-block follow affix-top">
            <div class="sidebar-heading">
            <i class="icon-list-ul"></i> Outline
        </div>
        <div class="article-navigation"><ul class="nav nav-list"><li class="article-anchor"><a href="#intro">Introduction</a></li><li class="article-anchor"><a href="#materials|methods">Materials &amp; Methods</a></li><li class="article-anchor"><a href="#results">Results</a></li><li class="article-anchor"><a href="#discussion">Discussion</a></li><li class="article-anchor"><a href="#supplemental-information"> Supplemental Information</a></li></ul></div>
    <div id="top-return" class="top-return">
        <i class="icon-arrow-up"></i> Return to top
    </div>
            <div data-clone="#mobile-featured-jobs" data-source="/expertrxiv/featured/?subjects=Biodiversity%2C%20Evolutionary%20Studies%2C%20Genetics%2C%20Population%20Biology"><div class="featured-job-container"><p class="featured-job-header">Research Jobs, Collaborations, &amp; Tasks</p>
                    List &amp; find research jobs, collaborations, or tasks on PeerJ for free.<br><a class="job-featured-cta" href="https://peerj.com/expertrxiv/">Learn more <i class="icon-angle-right"></i></a></div></div>
    </nav>



    
        </div>
    </div>

        
<style>
    .modal-loading-container{
        display:flex;
        justify-content:center;
        color:#999;
        padding:3rem;
    }
</style>

<div id="download-article-modal" class="modal hide fade peer-review-article">

    <div class="modal-header">
        <button type="button" class="close" data-dismiss="modal" aria-hidden="true">×</button>
        <h3>Download article</h3>
    </div>

    <div class="modal-body">
        <div id="download-article-modal-loading" class="modal-loading-container" style="display:none;">
            <i class="icon-spin icon-3x icon-spinner"></i>
        </div>
        <div id="download-article-modal-body">
                    <div id="download-modal-buttons-container">
        <div class="download-modal-article-title">Shannon diversity 
index: a call to replace the original Shannon’s formula with unbiased 
estimator in the population genetics studies</div>
            <div class="mt-2 download-buttons">
                                                                                <a target="_blank" download="" data-format="PDF" data-download-confirm-text="PDF downloaded" href="https://peerj.com/articles/9391.pdf" class="btn btn-primary js-download-btn btn-block btn-large mb-2 "><i class="icon-cloud-download mr-1"></i> PDF (3.0MB)</a>
                                                    <a target="_blank" data-download-confirm-text="Mendeley opened" href="http://www.mendeley.com/import/?doi=10.7717/peerj.9391" class="btn btn-primary js-download-btn btn-block btn-large mb-2"><i class="icon-cloud-download mr-1"></i> Save to Mendeley</a>
                                    <a target="_blank" data-download-confirm-text="Readcube article opened" href="http://www.readcube.com/articles/10.7717/peerj.9391" class="btn btn-primary js-download-btn btn-block btn-large mb-2"><i class="icon-cloud-download mr-1"></i> Read in ReadCube</a>
                                                    <a target="_blank" data-format="RIS" data-download-confirm-text="RIS file downloaded" href="https://peerj.com/articles/9391.ris" class="btn btn-primary js-download-btn btn-block btn-large mb-2 "><i class="icon-cloud-download mr-1"></i> RIS</a>
                    <a target="_blank" data-format="XML" data-download-confirm-text="XML file downloaded" href="https://peerj.com/articles/9391.xml" class="btn btn-primary js-download-btn btn-block btn-large mb-2 "><i class="icon-cloud-download mr-1"></i> XML</a>
                    <a target="_blank" data-format="BibText" data-download-confirm-text="BibText file downloaded" href="https://peerj.com/articles/9391.bib" class="btn btn-primary js-download-btn btn-block btn-large mb-2 "><i class="icon-cloud-download mr-1"></i> BibTeX</a>
                
            </div>
        </div>

        <div id="download-modal-downloading-message" style="display:none;">
            <div class="text-center pt-4 pb-4">
                <div>
                    <strong>Your download will start in a moment...</strong>
                </div>
                <div class="btn btn-secondary mt-4 js-close-download-modal">Close</div>
            </div>
        </div>

        <div id="download-modal-signup-container" style="display:none;">
            
<div class="download-modal-cta-container">

    <div class="download-modal-confirm">
        <div class="download-modal-confirm-title">
            <i class="icon-tickcircle downloaded-tick"></i> <span class="download-modal-confirm-title-text"></span>
            <i class="icon-chevron-down show-download-link"></i>
        </div>
        <a class="article-modal-download-url" href=""></a>
    </div>

            <div class="download-modal-cta-title text-center">
            <a href="https://peerj.com/sections/bioinformatics-genomics/latest/?modalv=3">
                Bioinformatics and Genomics
            </a>
        </div>
    
    <div class="download-modal-cta-subtitle-small mt-2 mb-4 text-center">
        Subscribe for section  updates
    </div>

            <div class="section-subscribe-container mb-2" style="display: flex;justify-content:center;">
            <div>
                <input placeholder="Email address" name="email" class="form-control" id="download-subscribe-email" type="text">
            </div>
            <div class="ml-1">
                <select name="freq" class="form-control" style="width: 100%;" id="download-subscribe-freq">
                    <option value="daily" selected="selected">Daily</option>
                    <option value="weekly">Weekly</option>
                </select>
            </div>
        </div>
    
    <div id="download-subscribe-error-container" class="mb-2 text-center text-error" style="display:none;"></div>

                
    <button class="btn btn-primary btn-block btn-large mb-2 btn-modal-cta" style="display: block;" id="download-subscribe-submit" data-url="/content-alert/download-subscribe?section=5&amp;aid=44110" data-signed-in="" data-section-name="Bioinformatics and Genomics">
        Subscribe
    </button>

    <a href="#" class="btn btn-block btn-link btn-large btn-modal-close js-close-download-modal mb-2">
        Close
    </a>

</div>

<script>
    (function(){
        $('#download-subscribe-submit').click(function(){

            var button = $(this);
            var url = button.data('url');
            if(button.attr('disabled')) return;

            $.get(url, function(response){

                if(!response.token){
                    errorContainer.html('Server error, you have not been subscribed').show();
                    button.html('Subscribe').removeAttr('disabled');
                    return;
                }

                var errorContainer = $('#download-subscribe-error-container');
                errorContainer.html('').hide();
                button.html('<i class="icon-spin icon-spinner"></i>').attr('disabled', true);

                var signedIn = button.data('signed-in');
                var sectionName = button.data('section-name');
                var data = {
                    _token: response.token
                };

                if(!signedIn) {
                    var email = $('#download-subscribe-email').val();
                    data.email = email;
                    data.freq = $('download-subscribe-freq').val();
                }

                $.ajax({
                    url: url,
                    method: 'POST',
                    data: data
                }).success(function(response){
                    button.hide();
                    $('.js-close-download-modal').trigger('click');

                    PeerJ.Tools.ToastNotifications.add({
                        type: 'success',
                        title: 'Subscribed',
                        text: sectionName ? 'You subscribed to ' + sectionName : 'You subscribed to this article\'s subjects'
                    });

                }).error(function(response){
                    if(response.responseJSON && response.responseJSON.errors){
                        errorContainer.html(response.responseJSON.errors[0]).show();
                    }
                }).complete(function(){
                    button.html('Subscribe').removeAttr('disabled');
                });

            });
        });

    }());
</script>
        </div>
            </div>
    </div>

    <div class="modal-footer" style="display:none;">
        <div class="pull-right">
                    </div>

                    <span class="submit-copy submit-copy-btn btn cancel pull-left" id="modal-cancel" data-dismiss="modal">Cancel</span>
            </div>
</div>

        <div id="ajax-form"></div>

    <!-- Flag Modal -->
        <div id="flagModal" class="modal hide" style="max-height:none">
      <div class="modal-header" style="text-align: center">
        <button type="button" class="close" data-dismiss="modal" aria-hidden="true">×</button>
        <h3 class="slim">Report a problem</h3>
      </div>

      <form id="article-flag-form" data-href="/issues/9391/flag/" method="post">

          <div class="modal-body" style="max-height:350px;overflow-y:auto">
              <div class="alert alert-info">
                    <p><strong>Common use cases</strong><br>
                    Typos, corrections needed, missing information, abuse, etc
                    </p>

                    <p><strong>Our promise</strong><br>
                    PeerJ promises to address all issues as quickly and professionally as possible. We
                    thank you in advance for your patience and understanding.
                    </p>
              </div>

                <div id="flag-modal-result" style="margin-left:45px;">

                                            <div>
                            <label><strong>Type of problem</strong></label>
                            <p>
                            <select id="moderation_flag_category" name="moderation_flag[category]" class="span4"><option value="typo" selected="selected">Typo</option><option value="metadata">Missing or incorrect metadata</option><option value="quality">Quality: PDF, figure, table, or data quality</option><option value="download">Download issues</option><option value="abuse">Abusive behavior</option><option value="misconduct">Research misconduct</option><option value="other">Other issue not listed above</option></select>
                            
                            </p>
                         </div>
                        <div>
                            <label><strong>Details</strong> <i class="icon-large icon-question-sign" title="" data-toggle="tooltip" data-original-title="Please be as detailed as possible within the 500 character limit. Any details you provide will not be shown publicly."></i></label>
                            <div>
                                <textarea id="moderation_flag_detail" name="moderation_flag[detail]" required="required" maxlength="500" class="span4" placeholder="Enter any details about this issue. Kept confidential with PeerJ staff." rows="5" data-counter-target="#flag-counter"></textarea>
                                
                                <div style="margin:10px 0 0 0; color:#777777; float: left; display: block"><span id="flag-counter" class="label label-info">500</span> characters remaining</div>
                            </div>
                         </div>
                        
                                    </div>

          </div>
        </form>
        <div id="flag-modal-footer" class="modal-footer">
        <button class="btn" data-dismiss="modal" aria-hidden="true">Cancel</button>
        <input class="btn btn-success save-flag-btn" value="Send report" type="submit">
        </div>
</div>

    <!-- Follow Publication Modal -->
        <div id="followModal" class="modal hide" style="max-height:none">
      <div class="modal-header" style="text-align:center">
        <button type="button" class="close" data-dismiss="modal" aria-hidden="true">×</button>
                    <h3 class="slim" id="followModalLabel">Follow this publication for updates</h3>
              </div>

    <div>
        <div class="modal-body" style="max-height:350px;overflow-y:auto">
            <div class="row-fluid" style="margin-bottom: 15px">
                <div class="span1">
                    <i class="icon-large icon-bullhorn"></i>
                </div>
                <div class="span11">
                    "Following" is like subscribing to any updates related to a publication.
                    These updates will appear in your home dashboard each time you visit PeerJ.
                </div>
            </div>

            <div class="row-fluid">
                <div class="span1">
                    <i class="icon-large icon-envelope"></i>
                </div>
                <div class="span11">
                    <p>
                    You can also choose to receive updates via daily or weekly email digests.
                    If you are following multiple publications then we will send you
                    no more than one email per day or week based on your preferences.
                    </p>
                    <p>
                    <em>Note: You are now also subscribed to the subject areas of this publication</em>
                    and will receive updates in the daily or weekly email digests if turned on.
                    You can <a href="https://peerj.com/settings/details/">add specific subject areas</a> through your profile settings.
                    </p>
                </div>
            </div>

                     <hr>
                <div id="follow-modal-result" style="margin-left:-40px;padding-top:7px;">
                                    </div>

          </div>

        </div>

        <div id="follow-modal-footer" class="modal-footer">
            <button class="btn" data-dismiss="modal" aria-hidden="true">Close</button>
        </div>
    </div>

    <!-- Unfollow Publication Modal -->
    <div id="unfollowModal" class="modal hide">
    <div class="modal-header">
        <button type="button" class="close" data-dismiss="modal" aria-hidden="true">×</button>
        <h3>Change notification settings or unfollow</h3>
    </div>

    <form id="article-unfollow-form" data-href="/follow/publication/9391/1/" method="put" class="form-horizontal">


        <div id="unfollow-form-load-result" class="modal-body" data-href="/follow/publication/9391/edit/" style="max-height:350px;overflow-y:auto">
            <p>Loading ...</p>
        </div>

    </form>
        <div class="modal-footer">
            <button class="btn follow-close-btn" data-dismiss="modal" aria-hidden="true">Close</button>
            <input class="btn btn-success update-follow-btn" value="Update" type="submit">
        </div>
</div>

    <!-- Metrics Modal -->
    <div id="metricsModal" class="modal hide">
    <div class="modal-body" style="max-height:330px;overflow-y:auto">

        <div class="row-fluid">
            <div class="span12">
                <p class="leadh2">Usage since published - updated daily</p>
            </div>
        </div>

        <div class="row-fluid">
                            <div class="span8">
                    <h3 style="margin-bottom:10px">Social referrals <small>unique visitors</small></h3>
                                            <div class="row-fluid" style="font-size: 16px; color: #444; border-bottom: 1px solid #ccc; margin-bottom: 5px;">
                            <div class="span8" style="min-height:0">Facebook</div>
                            <div class="span3" style="text-align:right;min-height:0">42</div>
                        </div>
                                            <div class="row-fluid" style="font-size: 16px; color: #444; border-bottom: 1px solid #ccc; margin-bottom: 5px;">
                            <div class="span8" style="min-height:0">Twitter</div>
                            <div class="span3" style="text-align:right;min-height:0">13</div>
                        </div>
                                        
                    <h3 style="margin:30px 0 10px 0">Top referrals <small>unique visitors</small></h3>
                                                                        
                                                                        
                                                                        
                                                                        
                                                                        
                                                                        
                                                                        
                                                                        
                                                                        
                                                                        
                                                                        
                                                                        
                                                                        
                                                                        
                                                                        
                                                                        
                                                                        
                                                                        
                                                                    <div class="row-fluid" style="font-size: 16px;  color: #444; border-bottom: 1px solid #ccc; margin-bottom: 5px;">
                            <div class="span8" style="min-height:0">
                                                                    From bookmark or typed URL
                                                            </div>
                            <div class="span3" style="text-align:right;min-height:0">890</div>
                        </div>
                                            <div class="row-fluid" style="font-size: 16px;  color: #444; border-bottom: 1px solid #ccc; margin-bottom: 5px;">
                            <div class="span8" style="min-height:0">
                                                                    From PeerJ Content Alert Emails
                                                            </div>
                            <div class="span3" style="text-align:right;min-height:0">152</div>
                        </div>
                                            <div class="row-fluid" style="font-size: 16px;  color: #444; border-bottom: 1px solid #ccc; margin-bottom: 5px;">
                            <div class="span8" style="min-height:0">
                                                                    Google search
                                                            </div>
                            <div class="span3" style="text-align:right;min-height:0">73</div>
                        </div>
                                            <div class="row-fluid" style="font-size: 16px;  color: #444; border-bottom: 1px solid #ccc; margin-bottom: 5px;">
                            <div class="span8" style="min-height:0">
                                                                    Twitter
                                                            </div>
                            <div class="span3" style="text-align:right;min-height:0">13</div>
                        </div>
                                                        </div>
            
            <div class="span4" style="overflow-x:hidden;">
                <h3 style="margin-bottom:10px">Share this publication</h3>

                                    
    
                        <ul class="unstyled">
    <li>
        <a class="pj-socialism tw-soc" href="http://twitter.com/share?url=https%3A%2F%2Fpeerj.com%2Farticles%2F9391%2F&amp;via=thePeerJ&amp;text=Shannon%20diversity%20index%20estimators&amp;related=" target="_blank" onclick="window.open(this.href, 'popupwindow', 'width=500,height=500,scrollbars,resizable'); return false;">Twitter</a>
    </li>
    <li>
        <a class="pj-socialism fb-soc" href="http://www.facebook.com/sharer.php?u=https%3A%2F%2Fpeerj.com%2Farticles%2F9391%2F" target="_blank" onclick="window.open(this.href, 'popupwindow', 'width=500,height=500,scrollbars,resizable'); return false;">Facebook</a>
    </li>
    <li>
        <a class="pj-socialism em-soc" href="mailto:?Subject=Relevant%20research%20paper%20in%20PeerJ&amp;Body=Shannon%20diversity%20index%3A%20a%20call%20to%20replace%20the%20original%20Shannon%E2%80%99s%20formula%20with%20unbiased%20estimator%20in%20the%20population%20genetics%20studies%20https%3A%2F%2Fpeerj.com%2Farticles%2F9391%2F" target="_blank" onclick="window.open(this.href, 'popupwindow', 'width=500,height=500,scrollbars,resizable'); return false;">Email</a>
    </li>
</ul>                
                <h3 style="margin-bottom:10px;margin-top:10px">Metrics</h3>

                <!-- Altmetric -->
                <div class="altmetric-embed" data-badge-popover="right" data-link-target="_blank" data-doi="10.7717/peerj.9391"></div>
            </div>
        </div>

    </div>

    <div class="modal-footer">
        <button class="btn" data-dismiss="modal" aria-hidden="true">Close</button>
    </div>
</div>

    <!-- Wiki Modal -->
    
    <!-- Links Modal -->
    <div class="modal hide fade" id="article-links-modal">
    <div class="modal-header">
        <a rel="nofollow" data-dismiss="modal" aria-hidden="true" class="close">×</a>

        <h3 class="modal-title">Links</h3>
    </div>

    <div class="modal-body"></div>

    <div class="modal-footer">
        <a rel="nofollow" href="https://peerj.com/links.form?target=articles/9391" class="btn btn-primary">Add a link</a>
        <button class="btn follow-close-btn" data-dismiss="modal" aria-hidden="true">Close</button>
    </div>
</div>

    <!-- Citing Modal -->
    <div id="citing-modal" class="modal hide">
    <div class="modal-header">
        <button type="button" class="close" data-dismiss="modal" aria-hidden="true">×</button>
        <h2 class="slim"><i class="icon-copy"></i> Articles citing this paper</h2>
    </div>
  <div class="modal-body">Loading citing articles… <i class="icon icon-spinner icon-spin"></i></div>
</div>

    <!-- Graphical abstract modal -->
    
        </div>

        
                <div id="push"></div>
    </div>

    <footer id="footer">
        <div class="foot">
            <div class="container">
                
                                    <div class="row">
	<div class="span7">
		<b>About us -</b> <a href="https://peerj.com/about/" class="aboutLink" data-target="team">PeerJ team</a>
		| <a href="https://peerj.com/about/publications/" class="aboutLink" data-target="journals">Our publications</a> |
		<a href="https://peerj.com/benefits/">Benefits</a> | <a href="https://peerj.com/about/partnerships/" class="aboutLink" data-target="partnership">Partnerships</a> | <a href="https://peerj.com/about/endorsements/" class="aboutLink" data-target="endorsements">Endorsements</a>
		 <i class="icon-trophy"></i> <a href="https://peerj.com/about/reviews/" class="aboutLink" data-target="reviews">Awards</a>
	</div>
	<div class="span5">
		<b>Resources -</b> <a href="https://peerj.com/about/FAQ/">FAQ</a> | <a href="https://peerj.com/about/careers/">Careers</a> | <a href="https://peerj.com/about/press/">Press
		room</a> | <a href="https://peerj.com/about/terms/">Terms of use</a> | <a href="https://peerj.com/about/privacy/">Privacy</a> | <a href="https://peerj.com/about/contact/" class="aboutLink" data-target="contact">Contact</a>
	</div>
	<div class="span7">
		<b>Academic boards -</b> <a href="https://peerj.com/academic-boards/advisors/">Advisors</a> | <a href="https://peerj.com/academic-boards/editors/">Editors</a> |
		<a href="https://peerj.com/academic-boards/subjects/">Subject areas</a>
	</div>
	<div class="span5">
		<b>Follow us -</b>
		<a href="https://peerj.com/blog/">PeerJ blog</a> |
		<a href="http://twitter.com/thePeerJ/" title="" data-toggle="tooltip" data-original-title="Follow on Twitter">Twitter</a>
		|
		<a href="http://facebook.com/thePeerJ/" title="" data-toggle="tooltip" data-original-title="Follow on Facebook">Facebook</a>
		|
		<a href="http://www.linkedin.com/company/peerj" title="" data-toggle="tooltip" data-original-title="Follow on LinkedIn">LinkedIn</a>
        |
        <a href="https://www.instagram.com/thepeerj" title="" data-toggle="tooltip" data-original-title="Follow on Instagram">Instagram</a>
        |
        <a href="http://www.pinterest.com/thepeerj/boards/" title="" data-toggle="tooltip" data-original-title="Follow on Pinterest">Pinterest</a>
	</div>
	<div class="span7">
		<b>Submission guides -</b>
		<a href="https://peerj.com/about/aims-and-scope"><em>PeerJ – Life and Environment</em></a> |
		<a href="https://peerj.com/about/aims-and-scope/cs"><em>PeerJ Computer Science</em></a> |
		<a href="https://peerj.com/about/aims-and-scope/chemistry"><em>PeerJ Chemistry</em></a>
	</div>
	<div class="span5">
		<b>Spread the word</b> -
		<a href="https://peerj.com/spread-the-word/activities/">Activities</a> |
		<a href="https://peerj.com/spread-the-word/resources/">Resources</a>
	</div>
    <div class="span7">&nbsp;</div>
    <div class="span5">
	<b>PeerJ feeds <i class="icon-rss"></i> - </b>
	<a href="https://peerj.com/articles/index.atom" rel="alternate" title="Articles (Atom)" type="application/atom+xml">Atom</a> |
	<a href="https://peerj.com/articles/index.rss1">RSS 1.0</a> |
	<a href="https://peerj.com/articles/index.rss2">RSS 2.0</a> |
	<a href="https://peerj.com/articles/index.json">JSON</a>
	<br>

	<b>PeerJ Computer Science feeds <i class="icon-rss"></i> - </b>
	<a href="https://peerj.com/articles/index.atom?journal=cs" rel="alternate" title="PeerJ Computer Science articles (Atom)" type="application/atom+xml">Atom</a> |
	<a href="https://peerj.com/articles/index.rss1?journal=cs">RSS 1.0</a> |
	<a href="https://peerj.com/articles/index.rss2?journal=cs">RSS 2.0</a> |
	<a href="https://peerj.com/articles/index.json?journal=cs">JSON</a>
	<br>
        <b>Archives - </b>
        <a href="https://peerj.com/archives/" rel="archives"><em>PeerJ – Life and Environment</em></a> |
        <a href="https://peerj.com/archives/?journal=cs" rel="archives"><em>PeerJ Computer Science</em></a>
    </div>

</div>

<div id="fb-root"></div>
                
                <div class="row" style="margin-top:10px;font-size:12px">
    <div class="span12" style="color:#888">
        
        <div>
            <span style="margin-right:7px"><span style="font-style:italic">PeerJ</span> ISSN: 2167-8359</span>
            <span style="margin-right:7px"><span style="font-style:italic">PeerJ Comput. Sci.</span> ISSN: 2376-5992</span>
            <span><span style="font-style:italic">PeerJ Preprints</span> ISSN: 2167-9843</span>
        </div>
    </div>
</div>
            </div>
        </div>
    </footer>

        <div id="alerts" data-async-alerts="/alerts/"></div>

                    <script src="8d39319-39a0771.js"></script>
                <script src="runtime.js"></script><script src="0.js"></script><script src="1.js"></script><script src="vue-bundle.js"></script>
    

    <script src="5d3c493-193ec0b.js"></script>

    <script src="c1dacd9-f146d62.js"></script>
        <!--[if gt IE 8]><!-->
        <script src="highlight.js"></script>

        <script>
            $(function () {
                // syntax highlighting for code blocks
                $("pre > code").each(function() {
                    var node = $(this);

                    var language;

                    // JATS >=1.1
                    language = node.data('jats-language');

                    if (!language) {
                        // JATS <1.1
                        language = node.data('jats-preformat-type');

                        // ignore default 'code' type
                        if (language === 'code') {
                            language = null;
                        }
                    }

                    if (language) {
                        node.addClass('language-' + language);
                    }

                    hljs.highlightBlock(this);
                });
            });
        </script>
    <!--<![endif]-->

    <!-- initialise the follow button -->
    <script>
        $(function() {
            PeerJ.Event.Follow.init();
        });
    </script>

    
<script type="text/x-mathjax-config;executed=true">
    MathJax.Hub.Config({
        messageStyle: "none",
        imageFont: null,
        "CommonHTML": {
            linebreaks: { automatic: true },
            scale: 95
        },
        "HTML-CSS": {
            linebreaks: { automatic: true },
            scale: 90
        },
        menuSettings: {
            zoom: "Click"
        }
    });

    MathJax.Ajax.config.root = "/bundles/peerjmathjax/MathJax/";
</script>

<script src="MathJax.js"></script>

    <script defer="defer" src="trendmd.js" data-trendmdconfig="{&quot;journal_id&quot;:&quot;52926&quot;,&quot;element&quot;:&quot;#related-research&quot;}"></script>
    <script defer="defer" src="trendmd.js" data-trendmdconfig="{&quot;journal_id&quot;:&quot;52926&quot;,&quot;element&quot;:&quot;#related-research-sidebar&quot;}"></script>
    <script async="" src="badge.js" charset="utf-8"></script>

    <div id="content-alert-container"></div>

    <div id="toast-container"></div>

    <div id="vue-notifications"></div>

    <div id="vue-confirm-modal"></div>

            <script>
            $(PeerJ.Home.Banner.init);
        </script>
    
    

<ul class="ui-autocomplete ui-front ui-menu ui-widget ui-widget-content" id="ui-id-1" tabindex="0" style="display: none;"></ul><span role="status" aria-live="assertive" aria-relevant="additions" class="ui-helper-hidden-accessible"></span><div style="position: absolute; width: 0px; height: 0px; overflow: hidden; padding: 0px; border: 0px none; margin: 0px;"><div id="MathJax_Font_Test" style="position: absolute; visibility: hidden; top: 0px; left: 0px; width: auto; padding: 0px; border: 0px none; margin: 0px; white-space: nowrap; text-align: left; text-indent: 0px; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; font-size: 40px; font-weight: normal; font-style: normal; font-size-adjust: none; font-family: MathJax_Size4, sans-serif;"></div></div></body></html>