<!DOCTYPE html> <html class=pb-page data-request-id=0062433a-7a21-4348-8403-a653d1e30e8f style lang=en><!--
 Page saved with SingleFileZ 
 url: https://onlinelibrary.wiley.com/doi/10.1111/j.1600-0587.2009.05880.x 
 saved date: Tue Oct 20 2020 18:42:54 GMT+0200 (CEST)
--><head data-pb-dropzone=head><meta charset=utf-8><meta http-equiv=X-UA-Compatible content="IE=edge">
<meta name=pbContext content=";article:article:doi\:10.1111/j.1600-0587.2009.05880.x;page:string:Article/Chapter View;ctype:string:Journal Content;website:website:pericles;journal:journal:16000587;issue:issue:doi\:10.1111/eco.2010.33.issue-1;wgroup:string:Publication Websites;requestedJournal:journal:16000587;pageGroup:string:Publication Pages;subPage:string:Full Text">
<meta name=robots content=noarchive><meta name=pb-robots-disabled><meta property=og:title content="A diversity of beta diversities: straightening up a concept gone awry. Part 1. Defining beta diversity as a function of alpha and gamma diversity"><meta property=og:type content=Article><meta property=og:url content=https://onlinelibrary.wiley.com/doi/10.1111/j.1600-0587.2009.05880.x><meta property=og:image content=https://onlinelibrary.wiley.com/cms/asset/1eb6a05b-f2c4-4732-ae7b-df20b9535f83/eco.2010.33.issue-1.cover.gif><meta property=og:site_name content="Wiley Online Library"><meta property=og:description content="The term beta diversity has been used to refer to a wide variety of phenomena. Although all of these encompass some kind of compositional heterogeneity between places, many are not related to each ot..."><meta name=viewport content="width=device-width,initial-scale=1"><meta name=apple-itunes-app content="app-id=782998940"><meta name=citation_doi content=10.1111/j.1600-0587.2009.05880.x><meta name=dc.identifier content=10.1111/j.1600-0587.2009.05880.x><meta name=citation_journal_title content=Ecography><meta name=citation_title content="A diversity of beta diversities: straightening up a concept gone awry. Part 1. Defining beta diversity as a function of alpha and gamma diversity"><meta name=citation_volume content=33><meta name=citation_issue content=1><meta name=citation_firstpage content=2><meta name=citation_lastpage content=22><meta name=citation_publication_date content=2010/02/01><meta name=citation_online_date content=2010/02/11><meta name=citation_issn content=1600-0587><meta name=citation_language content=en><meta name=citation_author content="Hanna Tuomisto"><meta name=citation_abstract_html_url content=https://onlinelibrary.wiley.com/doi/abs/10.1111/j.1600-0587.2009.05880.x><meta name=citation_fulltext_html_url content=https://onlinelibrary.wiley.com/doi/full/10.1111/j.1600-0587.2009.05880.x><meta name=citation_pdf_url content=https://onlinelibrary.wiley.com/doi/pdf/10.1111/j.1600-0587.2009.05880.x><meta name=citation_publisher content="John Wiley &amp; Sons, Ltd"><meta name=article_references content="Tuomisto, H. (2010), A diversity of beta diversities: straightening up a concept gone awry. Part 1. Defining beta diversity as a function of alpha and gamma diversity. Ecography, 33: 2-22. doi:10.1111/j.1600-0587.2009.05880.x"><meta name=epdf_available content=false><meta name=Description content="The term beta diversity has been used to refer to a wide variety of phenomena. Although all of these encompass some kind of compositional heterogeneity between places, many are not related to each ..."><title>A diversity of beta diversities: straightening up a concept gone awry. Part 1. Defining beta diversity as a function of alpha and gamma diversity - Tuomisto - 2010 - Ecography - Wiley Online Library</title><meta name=referrer content=origin-when-cross-origin><meta name=google-site-verification content=ffz4i4DxIJCdGpRXe0XN6KG6o4FYI-nnFW9I_dg8piM><link rel="shortcut icon" href=images/19.ico><style>.sf-hidden{display:none!important}</style><link rel=canonical href=https://onlinelibrary.wiley.com/doi/10.1111/j.1600-0587.2009.05880.x><body class=pb-ui><noscript>
				<a href="/action/clickThrough?id=1013&url=https%3A%2F%2Fpubads.g.doubleclick.net%2Fgampad%2Fjump%3Fiu%3D%24googlePublisherCategory%26amp%3Bsz%3D728x90%26amp%3Btile%3D1%26amp%3Bc%3D%24random&loc=%2Fdoi%2F10.1111%2Fj.1600-0587.2009.05880.x&pubId=58070517&placeholderId=1015&productId=101018" target="_blank">
					<img src="https://pubads.g.doubleclick.net/gampad/ad?iu=$googlePublisherCategory&amp;sz=728x90&amp;tile=1&amp;c=$random">
				</a>
			</noscript>
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
<link rel=stylesheet type=text/css href=stylesheet_0.css>
 <link rel=stylesheet href=stylesheet_1.css>
<link rel=stylesheet type=text/css href=stylesheet_2.css>
<link rel=stylesheet type=text/css href=stylesheet_3.css>
<link rel=stylesheet type=text/css href=stylesheet_4.css>
<link rel=stylesheet type=text/css href=stylesheet_5.css>
<style>.publication-cover-info{margin-bottom:20px}.article-row-right>.pb-ad{margin-left:0;margin-bottom:10px}.admin-message{background-color:#005274;color:white}.banner-wrapper .pb-ad{padding:0}.advert.advert-leaderboard{margin:15px 0}.banner-wrapper{box-shadow:none}.cover-image__image img{max-width:150px;max-height:150px}h1.citation__title{font-size:1.375rem}@media (max-width:991px){h1.citation__title{font-size:1.125rem}h1.citation__title:after{content:""}}.cart-indicator__wrapper{padding:0}</style>
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
<style>.at-icon-wrapper{overflow:hidden}</style><link href=stylesheet_6.css type=text/css rel=stylesheet><style>#MathJax_Message{position:fixed;left:1px;bottom:2px;background-color:#E6E6E6;border:1px solid #959595;margin:0px;padding:2px 8px;z-index:102;color:black;font-size:80%;width:auto;white-space:nowrap}</style><link id=altmetric-embed-css rel=stylesheet type=text/css href=stylesheet_7.css>
<div id=MathJax_Message style=display:none></div>
<div class=skip-links>
 <ul class=rlist>
 
 <li><a href=#article__content tabindex=1>Skip to Article Content</a></li>
 <li><a href=#pane-pcw-detailscon tabindex=1>Skip to Article Information</a></li>
 
 </ul>
</div>
 <div id=pb-page-content data-ng-non-bindable>
 <div data-pb-dropzone=main data-pb-dropzone-name=Main>
 <div data-widget-def=pageHeader data-widget-id=62175478-b98a-43d5-90f2-1f2045ac1a7e class=pageHeader>
 
 
 <header class=page-header>
 <div data-pb-dropzone=main>
 
 <div data-widget-def=ux3-layout-widget data-widget-id=bb5b5e2c-d3d6-4015-b586-eeade38f59d4 class=admin-message>
 
 
 <div class=container><div class=row><div>
 
 
 <div class=pb-ad>
 <div style="background=blue;">
 <div class=container>
 <p style=text-align:center><b>Working off-campus?</b> Learn about our <a href="https://onlinelibrary.wiley.com/action/clickThrough?id=101169&amp;url=https%3A%2F%2Fonlinelibrary.wiley.com%2Fresearchers%2Fread%2Faccess-options&amp;loc=%2Fdoi%2F10.1111%2Fj.1600-0587.2009.05880.x&amp;pubId=58070517&amp;placeholderId=1029&amp;productId=1011" target=_blank><font color=white><u>remote access options</u></font></a></p>
 </div>
</div>
 </div>
 
</div></div></div>
 </div>
 
 
 <div class=container><div class=row><div class="contents col-sm-12 col-md-12">
 <div data-widget-def=UX3InstitutionBanner data-widget-id=0a735273-4141-44d8-801a-2b89143bed55 class="institution-wrapper-mobile hidden-lg padded-content sf-hidden" aria-hidden=true>
 
 
 
 </div>
 
 
 <div class=pull-left>
 <div data-widget-def=ux3-general-image data-widget-id=ff262197-0fb8-4973-aaa2-2fceba4b5a61 class="logo-container hidden-xs hidden-sm hidden-md">
 
 
 <a href=https://onlinelibrary.wiley.com/ title="Wiley Online Library" tabindex=2><img id=mainLogo alt="Wiley Online Library" src=images/20.png></a>
 </div>
 
 <div data-widget-def=ux3-general-image data-widget-id=9a75b789-224c-4745-833a-930cf6c36f35 class="logo-container-mobile hidden-lg hidden-md hidden-sm sf-hidden">
 
 
 
 </div>
 
 <div data-widget-def=UX3InstitutionBanner data-widget-id=14d0e2f8-27d3-42d0-a477-72015e14e818 class="institution-wrapper hidden-xs hidden-sm hidden-md institution--block">
 
 
 <div class=institution><div class=institution-info-wrapper><span class=institution__intro>Access by</span><span class=institution__name>Czechs- Biological Institution Of Cas</span></div></div>
 </div>
 
</div><div class=pull-right>
 <div data-widget-def=UX3QuickSearchWidget data-widget-id=48175bb0-0f70-46e6-bb89-7e2781fdb8e4 class=header-quicksearch>
 
 
 <div class=quick-search-wrapper><a href=# title=search aria-label=Search class="quick-search__item sf-hidden"></a><div data-linkto-page=true class="quick-search quickSearchFormContainer"><div class=main-search-field-wrapper><div class="tab clearfix"><input placeholder=Search value tabindex=13 class="fakeQSInput quickSearchFilter" type=search><label for=search-within class=sr-only>Search within</label><select name=search-within id=search-within tabindex=10 class=search-within><option value=#pane-67065c09-4a88-49cd-934c-ac707951d35c0 selected>This Journal<option value=#pane-67065c09-4a88-49cd-934c-ac707951d35c1>Anywhere</select><ul role=tablist class="rlist tab__content"><li id=pane-67065c09-4a88-49cd-934c-ac707951d35c0 aria-labelledby=pane-67065c09-4a88-49cd-934c-ac707951d35c0-con role=tab class="tab__pane quick-search-pane__anywhere active"><form action=/action/doSearch name=thisJournalQuickSearch title="Quick Search"><div class="input-group option-0"><label for=searchField0 class=hiddenLabel>Search term</label><input name=AllField id=searchField0 placeholder=Search data-auto-complete-max-words=7 data-auto-complete-max-chars=32 data-contributors-conf=3 data-topics-conf=3 data-publication-titles-conf=3 data-history-items-conf=3 value tabindex=9 class="autocomplete actualQSInput quickSearchFilter ui-autocomplete-input" autocomplete=off type=search><span role=status aria-live=polite class=ui-helper-hidden-accessible></span><div class="search-options-wrapper quickSearchFilter sf-hidden"></div></div><button id=quick-search-btn type=submit title=Search tabindex=11 aria-label="Submit Search" class="btn quick-search__button icon-search"></button></form><li id=pane-67065c09-4a88-49cd-934c-ac707951d35c1 aria-labelledby=pane-67065c09-4a88-49cd-934c-ac707951d35c1-con role=tab class="tab__pane quick-search-pane__anywhere sf-hidden"></ul></div></div></div></div>
 </div>
 
 <div data-widget-def=ux3-layout-widget data-widget-id=5ef13ed1-f4ac-4b62-a426-07ec9576a586 class=cart-indicator__wrapper>
 
 
 <div class=pull-left>
 <div data-widget-def=eCommerceCartIndicatorWidget data-widget-id=4a9f7c04-a273-4a25-91fb-8e7bd9970288 class=cart-indicator__wrapper>
 
 
 
 
 
 </div>
 
</div><div class=pull-right>
 <div data-widget-def=literatumNavigationLoginBar data-widget-id=d384ca92-7137-45d6-b3c2-78c9c0a98c2b class=login-bar-container>
 
 
 
 
 
 
 <div class="loginBar login-open">
 <a href="https://onlinelibrary.wiley.com/action/showLogin?uri=%2Fdoi%2F10.1111%2Fj.1600-0587.2009.05880.x&amp;aria-label=Log+in+or+Register" class=show-login role=button uri=/doi/10.1111/j.1600-0587.2009.05880.x aria-label="Log in or Register">
 <span class="icon-profile_empty sf-hidden"></span>
 <span class=sign-in-label>Login / Register</span>
 </a>
 
 </div>
 
 </div>
 
</div>
 </div>
 
</div>
</div></div></div>
 </div>
</header>
 </div>
 
 <div data-widget-def=pageBody data-widget-id=72100436-7a82-49fc-933b-c6c9d8c42914 class="pageBody article-page-tablet body-text">
 
 
 <main id=main-content tabindex=-1>
 
 <div class="page-body pagefulltext">
 <div data-pb-dropzone=main>
 
 
 <div>
 <div data-widget-def=literatumAd data-widget-id=0220a30b-90c0-4b51-8637-141edf12f86a class=banner-wrapper>
 
 
 
 <div class=pb-ad>
 <aside role=complementary>
 <div id=advert-leaderboard class="advert advert-leaderboard" data-google-query-id=CPbszNzOw-wCFU7AEQgdrsoAqg style=display:none>
 
 
 
 </div>
 </aside>
 </div>
 
 </div>
 
 
 
</div>
 <section data-widget-def=ux3-layout-widget data-widget-id=0f7bccba-8328-4dbd-a1fa-e84c526dd9ab>
 
 
 <div class=container><div class=row><div class=col-md-12>
 
 <div>
 
 <article data-figures="/action/ajaxShowFigures?widgetId=5cf4c79f-0ae9-4dc5-96ce-77f62de7ada9&amp;ajax=true&amp;doi=10.1111%2Fj.1600-0587.2009.05880.x&amp;pbContext=%3Barticle%3Aarticle%3Adoi%5C%3A10.1111%2Fj.1600-0587.2009.05880.x%3Bpage%3Astring%3AArticle%2FChapter+View%3Bctype%3Astring%3AJournal+Content%3Bwebsite%3Awebsite%3Apericles%3Bjournal%3Ajournal%3A16000587%3Bissue%3Aissue%3Adoi%5C%3A10.1111%2Feco.2010.33.issue-1%3Bwgroup%3Astring%3APublication+Websites%3BrequestedJournal%3Ajournal%3A16000587%3BpageGroup%3Astring%3APublication+Pages%3BsubPage%3Astring%3AFull+Text&amp;accordionHeadingWrapper=h2" data-references="/action/ajaxShowEnhancedAbstract?widgetId=5cf4c79f-0ae9-4dc5-96ce-77f62de7ada9&amp;ajax=true&amp;doi=10.1111%2Fj.1600-0587.2009.05880.x&amp;pbContext=%3Barticle%3Aarticle%3Adoi%5C%3A10.1111%2Fj.1600-0587.2009.05880.x%3Bpage%3Astring%3AArticle%2FChapter+View%3Bctype%3Astring%3AJournal+Content%3Bwebsite%3Awebsite%3Apericles%3Bjournal%3Ajournal%3A16000587%3Bissue%3Aissue%3Adoi%5C%3A10.1111%2Feco.2010.33.issue-1%3Bwgroup%3Astring%3APublication+Websites%3BrequestedJournal%3Ajournal%3A16000587%3BpageGroup%3Astring%3APublication+Pages%3BsubPage%3Astring%3AFull+Text&amp;accordionHeadingWrapper=h2" data-related="/action/ajaxShowRecommended?widgetId=5cf4c79f-0ae9-4dc5-96ce-77f62de7ada9&amp;ajax=true&amp;doi=10.1111%2Fj.1600-0587.2009.05880.x&amp;pbContext=%3Barticle%3Aarticle%3Adoi%5C%3A10.1111%2Fj.1600-0587.2009.05880.x%3Bpage%3Astring%3AArticle%2FChapter+View%3Bctype%3Astring%3AJournal+Content%3Bwebsite%3Awebsite%3Apericles%3Bjournal%3Ajournal%3A16000587%3Bissue%3Aissue%3Adoi%5C%3A10.1111%2Feco.2010.33.issue-1%3Bwgroup%3Astring%3APublication+Websites%3BrequestedJournal%3Ajournal%3A16000587%3BpageGroup%3Astring%3APublication+Pages%3BsubPage%3Astring%3AFull+Text&amp;accordionHeadingWrapper=h2&amp;showSubjects=true&amp;taxonomiesCodes=enter+your+taxonomy+codes+here&amp;displayCitedByLink=true&amp;displayAlmetricDropzone=true&amp;taxonomyUri=publication-features&amp;topicUri=hide-issue-metadata" data-details="/action/ajaxShowPubInfo?widgetId=5cf4c79f-0ae9-4dc5-96ce-77f62de7ada9&amp;ajax=true&amp;doi=10.1111%2Fj.1600-0587.2009.05880.x&amp;pbContext=%3Barticle%3Aarticle%3Adoi%5C%3A10.1111%2Fj.1600-0587.2009.05880.x%3Bpage%3Astring%3AArticle%2FChapter+View%3Bctype%3Astring%3AJournal+Content%3Bwebsite%3Awebsite%3Apericles%3Bjournal%3Ajournal%3A16000587%3Bissue%3Aissue%3Adoi%5C%3A10.1111%2Feco.2010.33.issue-1%3Bwgroup%3Astring%3APublication+Websites%3BrequestedJournal%3Ajournal%3A16000587%3BpageGroup%3Astring%3APublication+Pages%3BsubPage%3Astring%3AFull+Text&amp;accordionHeadingWrapper=h2&amp;showSubjects=true&amp;taxonomiesCodes=enter+your+taxonomy+codes+here&amp;displayCitedByLink=true&amp;displayAlmetricDropzone=true&amp;taxonomyUri=publication-features&amp;topicUri=hide-issue-metadata" data-mathjax="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.5/MathJax.js?config=TeX-AMS-MML_HTMLorMML" class=hide_CASA><div class="row article-row"><div id=article__content tabindex=-1 class="col-sm-12 col-md-8 col-lg-8 article__content article-row-left"><div data-pb-dropzone=mrwBannerDropZone class=mrw-banner-dropzone>
 
 
</div><div class=article-citation><div class=citation>
 <div class=pb-dropzone data-pb-dropzone=publicaitonContent-series-title>
 <div id=journal-banner-text class=journal-banner-text style=font-size:22px><a href=https://onlinelibrary.wiley.com/journal/16000587 title="Ecography homepage">Ecography</a></div><a href=https://onlinelibrary.wiley.com/toc/16000587/2010/33/1 class="volume-issue sf-hidden">Volume 33, Issue 1</a><a href=https://onlinelibrary.wiley.com/journal/16000587 title="Ecography homepage" class=citation--logo><img id=journal-banner-image test=test-value alt=Ecography src="data:image/gif;base64,R0lGODlhAQABAAAAACH5BAEKAAEALAAAAAABAAEAAAICTAEAOw==" class=journal-banner-image style=display:none></a>
 </div>
 <div class="doi-access-container clearfix"><span class=primary-heading></span>
 <div class=doi-access-wrapper><div class="free-access access-type"> <i aria-hidden=true class=icon-icon-lock_open></i> <div class=doi-access> <div class=doi-access tabindex=0>Free Access</div></div></div></div></div>
 <h1 class=citation__title>A diversity of beta diversities: straightening up a concept gone awry. Part 1. Defining beta diversity as a function of alpha and gamma diversity</h1>
 <div class="rlist--inline loa comma visible-xs mobile-authors loa-authors-trunc sf-hidden">
 
 </div>
 <div class="loa-wrapper loa-authors hidden-xs">
 <div id=sb-1 class=accordion>
 <div class=accordion-tabbed>
 <div class="accordion-tabbed__tab-mobile accordion__closed"><a href="https://onlinelibrary.wiley.com/action/doSearch?ContribAuthorStored=Tuomisto%2C+Hanna" class="author-name accordion-tabbed__control" data-id=a1 data-db-target-for=a1 aria-controls=a1 aria-haspopup=true id=a1_Ctrl role=button><span>Hanna Tuomisto</span><i aria-hidden=true class="icon-section_arrow_d sf-hidden"></i></a><div class="author-info accordion-tabbed__content sf-hidden" data-db-target-of=a1 aria-labelledby=a1_Ctrl role=region id=a1></div>
 </div>
 </div>
 </div>
 </div>
 <div class=epub-sections><div class=epub-section><span class=epub-state>First published: </span><span class=epub-date>04 March 2010</span></div>
 <div class=epub-section><a class=epub-doi aria-label="Digital Object Identifier" href=https://doi.org/10.1111/j.1600-0587.2009.05880.x>https://doi.org/10.1111/j.1600-0587.2009.05880.x</a></div><div class="epub-section cited-by-count"><span>Citations: <a href=#citedby-section>488</a></span></div>
 </div>
 <div class=article-header__correspondence-to><i aria-hidden=true class="at-icon-wrapper icon-mail_outline"></i>H. Tuomisto (<span>E-mail address: </span><a title="
                Link to email address
            " href=mailto:hanna.tuomisto@utu.fi>hanna.tuomisto@utu.fi</a>), Dept of Biology, FI‐20014 Univ. of Turku, Finland.
 </div>
</div></div><div class=fixedCoolBar style=padding-top:0px><nav aria-label="Article navigation and tools" class="coolBar stickybar" style=top:auto;width:auto;left:auto><div class="stickybar__wrapper coolBar__wrapper clearfix"><div class="rlist coolBar__zone"><div class=coolBar__section><a href=# data-db-target-for=article data-db-switch=icon-close_thin aria-haspopup=true aria-controls=article_Pop role=button id=article_Ctrl class="coolBar__ctrl hidden-lg sf-hidden"></a><div data-db-target-of=article aria-labelledby=article_Ctrl role=menu id=article_Pop class="coolBar__drop fixed rlist dropBlock__holder sf-hidden"></div></div><div class="coolBar__section coolBar--sections"><a href=# data-db-target-for=sections data-db-switch=icon-close_thin aria-haspopup=true aria-controls=sections_Pop role=button id=sections_Ctrl class=coolBar__ctrl><i class=icon-list-unordered></i><span>Sections</span></a><div data-db-target-of=sections aria-labelledby=sections_Ctrl role=menu id=sections_Pop class="coolBar__drop dropBlock__holder rlist fixed sf-hidden"></div></div></div><div class="coolBar__second rlist"><div class="coolBar__section coolBar--download PdfLink scroll-to-target cloned"><a href=https://onlinelibrary.wiley.com/doi/epdf/10.1111/j.1600-0587.2009.05880.x title=ePDF class="coolBar__ctrl pdf-download"><img src=images/21.png alt=PDF>PDF</a></div>
 
 <div class=relative><a href=# data-db-target-for=0e406ce9-5c43-4e2d-b741-e0b43d1f1865 data-db-switch=icon-close_thin aria-haspopup=true aria-controls=0e406ce9-5c43-4e2d-b741-e0b43d1f1865_Pop role=button id=0e406ce9-5c43-4e2d-b741-e0b43d1f1865_Ctrl class=article-tools__ctrl><i aria-hidden=true class=icon-Icon_Tools></i><span>Tools</span></a><div data-db-target-of=0e406ce9-5c43-4e2d-b741-e0b43d1f1865 aria-labelledby=0e406ce9-5c43-4e2d-b741-e0b43d1f1865_Ctrl role=menu id=0e406ce9-5c43-4e2d-b741-e0b43d1f1865_Pop class="article-tools__block fixed dropBlock__holder sf-hidden"></div></div>
 
 <div class=relative><a id=share-access__ctrl href=# data-db-target-for=dff42abe-4b17-4fe9-a9c5-625405cd1007 data-db-switch=icon-close_thin aria-haspopup=true aria-controls=share_Pop role=button class=share-access__ctrl><i aria-hidden=true class=icon-Icon_Share></i><span>Share</span></a><a id=share__ctrl href=# data-db-target-for=dff42abe-4b17-4fe9-a9c5-625405cd1007 data-db-switch=icon-close_thin aria-haspopup=true aria-controls=share_Pop role=button class="share__ctrl hidden sf-hidden"></a><div data-db-target-of=dff42abe-4b17-4fe9-a9c5-625405cd1007 aria-labelledby=share-access__ctrl role=menu id=share_Pop class="share__block dropBlock__holder fixed sf-hidden">
 
 
</div></div>
</div></div></nav></div><div id=accessDenialslot class=accessDenialslot></div><div class=article__body><article>
 
 
 <div class=abstract-group>
 <section class="article-section article-section__abstract" data-lang=en id=section-1-en lang=en>
 <h2 id=d25731123 class="article-section__header section__title main abstractlang_en">Abstract</h2>
 <div class="article-section__content en main">
 <p>The term <i>beta diversity</i> has been used to refer to a wide variety of phenomena. Although all of these encompass some kind of compositional heterogeneity between places, many are not related to each other in any predictable way. The present two‐part review aims to put the different phenomena that have been called a beta component of diversity into a common conceptual framework, and to explain what each of them measures. In this first part, the focus is on defining beta diversity. This involves deciding what diversity is and how the observed total or gamma diversity (γ) is partitioned into alpha (α) and beta (β) components. Several different definitions of “beta diversity” that result from these decisions have been used in the ecological literature. <i>True beta diversity</i> is obtained when the total effective number of species in a dataset (<i>true gamma diversity</i>γ) is multiplicatively partitioned into the effective number of species per compositionally distinct virtual sampling unit (<i>true alpha diversity</i>α<sub>d</sub>) and the effective number of such compositional units (β<sub>Md</sub>=γ/α<sub>d</sub>). All true diversities quantify the effective number of types of entities. Because the other variants of “beta diversity” that have been used by ecologists quantify other phenomena, an alternative nomenclature is proposed here for the seven most popular beta components: regional‐to‐local diversity ratio, two‐way diversity ratio, absolute effective species turnover (=regional diversity excess), Whittaker's effective species turnover, proportional effective species turnover, regional entropy excess and regional variance excess. In the second part of the review, the focus will be on how to quantify these phenomena in practice. This involves deciding how the sampling units that contribute to total diversity are selected, and whether the entity that is quantified is all of “beta diversity”, a specific part of “beta diversity”, the rate of change in “beta diversity” in relation to a given external factor, or something else.</p>
 
 </div>
 </section>
 </div>
 <div class=pb-dropzone data-pb-dropzone=below-abstract-group>
 
 </div>
 
 <section class="article-section article-section__full">
 <section class=article-section__content id=ss1>
 
 <p>In a seminal paper, Whittaker (<span><a href=#b127 id=#b127R class="bibLink tab-link" data-tab=pane-pcw-references>1960</a></span>, p. 320) defined <i>beta diversity</i> as “The extent of change in community composition, or degree of community differentiation, in relation to a complex‐gradient of environment, or a pattern of environments”. He then proceeded to quantify beta diversity in different ways. His first two cases concerned Jaccard index (example 1) and percentage similarity (example 2) values between vegetation samples differing in geological formation and/or local moisture conditions. His third example quantified beta diversity as the ratio of gamma diversity (diversity in a set of sampling units) to alpha diversity (average diversity within sampling units) within each geological formation. His fourth example quantified beta diversity as the number of half‐change units, the half‐change unit being the distance along a transect by which similarity between two sampling units decreases to one‐half of the value estimated for similar environments.
 </p>
 
 <p>Obviously, <span><a href=#b129 id=#b129R class="bibLink tab-link" data-tab=pane-pcw-references>Whittaker (1960)</a></span> did not have an exact definition of beta diversity in mind, but used the term in a rather vague sense to refer to compositional heterogeneity among places. His verbal definition of beta diversity was broad, and each new example added a new phenomenon to it. All of the quantitative definitions are ratios that can be interpreted as unitless, but in fact their measurement units are species/species (example 1), (% abundance)/(% abundance) (example 2), (unit of diversity index)/(unit of diversity index) (example 3) and (unit of external gradient)/(unit of external gradient) (example 4). Under these definitions, beta diversity can be calculated for sampling units representing different habitat classes (examples 1 and 2), the same habitat class (example 3), or a gradient that has not been divided into habitat classes (example 4). Beta diversity may be explicitly dependent on a specified external gradient (distance along a transect in example 4) or not (examples 1, 2 and 3). Either species presence‐absence data (examples 1 and 4) or quantitative abundance data (examples 2, 3 and 4) can be used in different mathematical formulae.
 </p>
 
 <p> <span><a href=#b129 id=#b129R class="bibLink tab-link" data-tab=pane-pcw-references>Whittaker (1960)</a></span> introduced the term beta diversity (β) together with <i>alpha diversity</i> (α) and <i>gamma diversity</i> (γ). Both α and γ represent species diversity, but α is the mean species diversity at the local, within‐site or within‐habitat scale, whereas γ is the total species diversity at the regional or landscape scale. <span><a href=#b17 id=#b17R class="bibLink tab-link" data-tab=pane-pcw-references>Cody (1975)</a></span> redefined <i>beta diversity</i> as the rate of compositional turnover along a habitat gradient within one geographical region, and <i>gamma diversity</i> as the rate of compositional turnover with geographical distance within one habitat. <span><a href=#b10 id=#b10R class="bibLink tab-link" data-tab=pane-pcw-references>Bratton (1975)</a></span> also used the term beta diversity to refer to the rate of species turnover along a gradient. <span><a href=#b131 id=#b131R class="bibLink tab-link" data-tab=pane-pcw-references>Whittaker (1977)</a></span> accepted this expansion of <i>beta diversity</i>, which then became the “extent or rate of change in composition”. To account for different spatial scales, <span><a href=#b131 id=#b131R class="bibLink tab-link" data-tab=pane-pcw-references>Whittaker (1977)</a></span> proposed a hierarchical nomenclature in which alpha diversity refers to within‐habitat diversity, beta diversity to among‐habitat differentiation in a landscape, gamma diversity to total within‐landscape diversity, <i>delta diversity</i> to among‐landscape differentiation in a region, and <i>epsilon diversity</i> to total within‐region diversity.
 </p>
 
 <p>No wonder, therefore, that researchers have had a hard time agreeing on which quantitative interpretation of beta diversity is the correct one. Sources of contention include how beta diversity should be calculated, whether it should be measured within habitats or between habitats, and what spatial scales are appropriate. Several attempts have been made to tie alpha and gamma diversity to specific spatial scales (reviewed in <span><a href=#b132 id=#b132R class="bibLink tab-link" data-tab=pane-pcw-references>Whittaker et al. 2001</a></span> and <span><a href=#b78 id=#b78R class="bibLink tab-link" data-tab=pane-pcw-references>Magurran 2004</a></span>), but little consensus has emerged on this point. However, usually researchers are interested in habitats and regions so extensive that full species inventories are a practical impossiblity, so that obtaining accurate estimates of alpha and gamma diversity is a major concern (<span><a href=#b18 id=#b18R class="bibLink tab-link" data-tab=pane-pcw-references>Colwell and Coddington 1994</a></span>, <span><a href=#b96 id=#b96R class="bibLink tab-link" data-tab=pane-pcw-references>Plotkin and Muller‐Landau 2002</a></span>, <span><a href=#b12 id=#b12R class="bibLink tab-link" data-tab=pane-pcw-references>Chao et al. 2006</a></span>).
 </p>
 
 <p>Here I do not wish to dwell on how to delimit <i>local</i>, <i>regional</i> or <i>habitat</i> in practice, or on issues of data representativeness. Although these are important questions, the logical definition of the phenomenon that is to be measured needs to be established before it is useful to discuss practical sampling problems. Therefore I will treat all diversity components as properties of a dataset: once it has been decided which data points form the dataset of interest, all diversity components can be exactly quantified for that dataset. In this context, spatial scales are arbitrary and can be selected freely. The data need to come from discrete sampling units embedded in a study region, but whether lag (distance between neighboring sampling units) is small or large is irrelevant. The extent of the study corresponds to the size of the study region and the grain to the size of the sampling unit. Delta and epsilon diversities become unnecessary, because they simply refer to beta and gamma diversity, respectively, in a study system with large grain and extent. Grain and extent affect the numerical values of the diversity components, and lag needs to be taken into account when extrapolating results from an existing dataset to uninventoried areas. These issues will be discussed in the second part of the present review (<span><a href=#b119 id=#b119R class="bibLink tab-link" data-tab=pane-pcw-references>Tuomisto 2010</a></span>), once the basic concepts have been defined.
 </p>
 
 <p>Defining <i>beta diversity</i> is the main topic of the present paper. All variants of the umbrella concept encompass some kind of heterogeneity, differentiation or complementarity, but they actually refer to quite different phenomena. Some of these phenomena vary independently of each other among datasets, and “beta diversity” values based on different variants may therefore not be correlated. The situation is similar to that for the umbrella concept <i>size</i>. If we did not have separate words for weight, length, height, area and volume, discussions on size would become very confusing. The situation would become even worse if the term size were not only used for both weight and height, but also for growth rate. For example, consider animal A whose size is 100 cm, and animal B whose size is 10 kg. Which animal is bigger? It is impossible to say, because the given values are not commensurate. If the weight of A is revealed to be 5 kg, we know that B is heavier than A. But finding out that the size of B is 100 allows no conclusions, because we do not know if the unit of measurement was inches, meters, grams, pounds or something else. Furthermore, different aspects of size may rank animals differently; a snake that is larger than an elephant by the body length criterion is probably smaller by the body height or body weight criterion. If ranking is done using growth rate as the size criterion, it may be found that the younger the animal, the “larger” it is, and that in old animals “size” can even obtain a negative value.
 </p>
 
 <p>Comparing measurements based on different variants of the umbrella concept is equally useless in the case of “beta diversity”. Some variants of “beta diversity” are ratios in which the measurement units cancel out, whereas the measurement units in others can be, for example, virtual sampling units, species, (virtual species)<sup>−1</sup>, bits, species per unit sampling effort, km<sup>−1</sup>, species per km or species per unit habitat gradient. Because all variants of “beta diversity” are both multivariate and abstract, such crucial differences among them are much more difficult to spot than in the case of size. Consequently, the beta diversity literature is replete with studies that commit errors analogous to drawing conclusions on height on the basis of results on weight, or to comparing how two studies ranked different animal species without noticing that one study had measured body length whereas the other had measured the rate at which the animals' weight increased over time.
 </p>
 
 <p>Many authors have commented on the confusion around the concept of beta diversity (<span><a href=#b43 id=#b43R class="bibLink tab-link" data-tab=pane-pcw-references>Gray 2000</a></span>, <span><a href=#b126 id=#b126R class="bibLink tab-link" data-tab=pane-pcw-references>Vellend 2001</a></span>, <span><a href=#b66 id=#b66R class="bibLink tab-link" data-tab=pane-pcw-references>Koleff et al. 2003a</a></span>, <span><a href=#b65 id=#b65R class="bibLink tab-link" data-tab=pane-pcw-references>b</a></span>, <span><a href=#b85 id=#b85R class="bibLink tab-link" data-tab=pane-pcw-references>Novotny and Weiblen 2005</a></span>). The most thorough review to date seems to be that by <span><a href=#b59 id=#b59R class="bibLink tab-link" data-tab=pane-pcw-references>Jurasinski et al. (2009)</a></span>, who classified several beta diversity concepts into two categories. My aim in the present review is to explain what those and many other variants of “beta diversity” actually mean, and to put them into a common conceptual framework. It is crucial to use a metric that appropriately represents the phenomenon of interest, and knowledge about the logical relationships among alternatives can help in making that choice.
 </p>
 
 <p>In the present two‐part review, “beta diversity” variants are thought of as either 1) basic definitions or 2) approaches to applying a basic definition to a particular dataset. The variants of the first category will be discussed in the first part of the review and those of the second category in the second part. A basic definition specifies what “diversity” is and how the total or gamma “diversity” is partitioned into alpha and beta components. Once a basic definition has been chosen, it needs to be decided how the sampling units are selected, and whether to quantify all of “beta diversity”, a part of “beta diversity”, the rate of change in “beta diversity” in relation to some external factor, or someting else. Each approach leads to quantifying a different phenomenon, but all have been called <i>beta diversity</i>. To facilitate more accurate communication in the future, other names will here be proposed for all variants other than true beta diversity.
 </p>
 
 </section>
 <section class=article-section__content id=ss2>
 
 <h2 class="article-section__title section__title section1" id=ss2-title> The starting point: what is diversity?</h2>
 
 <section class=article-section__sub-content id=ss3>
 
 <h3 class="article-section__sub-title section2" id=ss3-title> Diversity in relation to a single classification</h3>
 
 <p>In order to measure diversity in a dataset of interest, the entities of which it is composed (such as individuals) need to be classified into types (such as species). Let us call the classification of individuals (or other appropriate units of abundance) into species the <i>γ‐classification</i>. Total diversity in relation to the γ‐classification is <i>gamma diversity</i> (γ), or total species diversity. The simplest measure of diversity is the number of types recognised, in this case species richness <i>S</i>. This equals the number of columns in a sites by species table (<a href=#t1 class=scrollableLink>Table 1</a>). In the present paper, <i>S</i> itself is used as a unitless number; the annotation <i>S</i> sp will be used, when necessary, to make the measurement unit (species) explicit. The number of types has the important doubling property, which can be understood by a thought experiment (<span><a href=#b51 id=#b51R class="bibLink tab-link" data-tab=pane-pcw-references>Hill 1973</a></span>, <span><a href=#b134 id=#b134R class="bibLink tab-link" data-tab=pane-pcw-references>Wilson and Shmida 1984</a></span>). Imagine that each column in <a href=#t1 class=scrollableLink>Table 1</a> is split into two columns, and each proportional abundance value of the original species is evenly divided between the two new species. Intuitively, the species diversity of the dataset has thereby doubled, and so has its species richness.
 </p>
 
 <div class=article-table-content id=t1>
 <header class=article-table-caption><span class=table-caption__label>Table 1.
 </span>A raw data table indicating how the <i>m</i> observed entities of interest have been classified into species according to the γ‐classification (columns) and into sampling units (SU) according to the ω‐classification (rows). The absolute abundance of species <i>i</i> in sampling unit <i>j</i> is annotated <i>m</i><sub><i>ij</i></sub>, and each cell value in the table (<i>p</i><sub><i>ij</i></sub>) gives this as a proportion of the total abundance <i>m</i> of all species in the dataset. Absolute abundance can be quantified using any measure that is appropriate for the questions at hand, for example number of individuals, surface area or biomass. The row totals show the proportion of the total abundance contributed by sampling unit <i>j</i>, i.e. the weight of sampling unit <i>j</i> in the dataset. The column totals show the proportion of the total abundance contributed by species <i>i</i>, i.e. the weight of species <i>i</i> in the dataset. The proportional abundance of species <i>i</i> within the limits of sampling unit <i>j</i> is <i>p</i><sub><i>i∣j</i></sub>=<i>p</i><sub><i>ij</i></sub>/<i>w</i><sub><i>j</i></sub> from which follows that <i>p</i><sub><i>ij</i></sub>=<i>w</i><sub><i>j</i></sub><i>p</i><sub><i>i∣j</i></sub>.
 </header>
 
 
 
 
 <div class=article-table-content-wrapper>
 <table class="table article-section__table">
 
 
 
 
 
 
 
 <thead>
 
 <tr>
 
 <th class="bottom-bordered-cell left-aligned"></th>
 
 <th class="bottom-bordered-cell center-aligned">Sp 1</th>
 
 <th class="bottom-bordered-cell center-aligned">Sp 2</th>
 
 <th class="bottom-bordered-cell center-aligned">Sp <i>i</i></th>
 
 <th class="bottom-bordered-cell center-aligned">Sp <i>S</i></th>
 
 <th class="bottom-bordered-cell center-aligned">Sampling unit weight</th>
 
 </tr>
 
 </thead>
 
 <tbody>
 
 <tr>
 
 <td class=left-aligned>SU 1</td>
 
 <td class=left-aligned> <i>p</i>
 <sub>11</sub>=<i>m</i><sub>11</sub>/<i>m</i></td>
 
 <td class=left-aligned> <i>p</i>
 <sub>21</sub>=<i>m</i><sub>21</sub>/<i>m</i></td>
 
 <td class=left-aligned> <i>p</i>
 <sub>
 <i>i</i>1</sub>=<i>m</i><sub><i>i</i>1</sub>/<i>m</i></td>
 
 <td class=left-aligned> <i>p</i>
 <sub>
 <i>S</i>1</sub>=<i>m</i><sub><i>S</i>1</sub>/<i>m</i></td>
 
 <td class=left-aligned> <img alt="inline image" class=section_image src=images/22.gif> 
 </td>
 
 </tr>
 
 <tr>
 
 <td class=left-aligned>SU 2</td>
 
 <td class=left-aligned> <i>p</i>
 <sub>12</sub>=<i>m</i><sub>12</sub>/<i>m</i></td>
 
 <td class=left-aligned> <i>p</i>
 <sub>22</sub>=<i>m</i><sub>22</sub>/<i>m</i></td>
 
 <td class=left-aligned> <i>p</i>
 <sub>
 <i>i</i>2</sub>=<i>m</i><sub><i>i</i>2</sub>/<i>m</i></td>
 
 <td class=left-aligned> <i>p</i>
 <sub>
 <i>S</i>2</sub>=<i>m</i><sub><i>S</i>2</sub>/<i>m</i></td>
 
 <td class=left-aligned> <img alt="inline image" class=section_image src=images/23.gif> 
 </td>
 
 </tr>
 
 <tr>
 
 <td class=left-aligned>SU <i>j</i></td>
 
 <td class=left-aligned> <i>p</i>
 <sub>1<i>j</i></sub>=<i>m</i><sub>1<i>j</i></sub>/<i>m</i></td>
 
 <td class=left-aligned> <i>p</i>
 <sub>2<i>j</i></sub>=<i>m</i><sub>2<i>j</i></sub>/<i>m</i></td>
 
 <td class=left-aligned> <i>p</i>
 <sub>
 <i>ij</i>
 </sub>=<i>m</i><sub><i>ij</i></sub>/<i>m</i></td>
 
 <td class=left-aligned> <i>p</i>
 <sub>
 <i>Sj</i>
 </sub>=<i>m</i><sub><i>Sj</i></sub>/<i>m</i></td>
 
 <td class=left-aligned> <img alt="inline image" class=section_image src=images/24.gif> 
 </td>
 
 </tr>
 
 <tr>
 
 <td class=left-aligned>SU <i>N</i></td>
 
 <td class=left-aligned> <i>p</i>
 <sub>1<i>N</i></sub>=<i>m</i><sub>1<i>N</i></sub>/<i>m</i></td>
 
 <td class=left-aligned> <i>p</i>
 <sub>2<i>N</i></sub>=<i>m</i><sub>2<i>N</i></sub>/<i>m</i></td>
 
 <td class=left-aligned> <i>p</i>
 <sub>
 <i>iN</i>
 </sub>=<i>m</i><sub><i>iN</i></sub>/<i>m</i></td>
 
 <td class=left-aligned> <i>p</i>
 <sub>
 <i>SN</i>
 </sub>=<i>m</i><sub><i>SN</i></sub>/<i>m</i></td>
 
 <td class=left-aligned> <img alt="inline image" class=section_image src=images/25.gif> 
 </td>
 
 </tr>
 
 <tr>
 
 <td class=left-aligned>Species weight</td>
 
 <td class=left-aligned> <img alt="inline image" class=section_image src=images/26.gif> 
 </td>
 
 <td class=left-aligned> <img alt="inline image" class=section_image src=images/27.gif> 
 </td>
 
 <td class=left-aligned> <img alt="inline image" class=section_image src=images/28.gif> 
 </td>
 
 <td class=left-aligned> <img alt="inline image" class=section_image src=images/29.gif> 
 </td>
 
 <td class=left-aligned> <img alt="inline image" class=section_image src=images/30.gif> 
 </td>
 
 </tr>
 
 </tbody>
 
 </table>
 </div>
 
 
 <div class=article-section__table-source></div>
 </div>
 
 <p>If all species are equally abundant, each of their proportional abundances (column totals in <a href=#t1 class=scrollableLink>Table 1</a>) equal 1/<i>S</i>. Mean proportional abundance then also equals 1/<i>S</i>. When proportional abundances vary, their mean can be expressed 1/<i>S</i><sub>E</sub>. The inverse of this mean, <i>S</i><sub>E</sub>, is the number of equally‐abundant virtual species (<i>effective species</i>) in the dataset, also known as the effective number of species or species diversity (<span><a href=#b77 id=#b77R class="bibLink tab-link" data-tab=pane-pcw-references>MacArthur 1965</a></span>, <span><a href=#b51 id=#b51R class="bibLink tab-link" data-tab=pane-pcw-references>Hill 1973</a></span>, <span><a href=#b56 id=#b56R class="bibLink tab-link" data-tab=pane-pcw-references>Jost 2006</a></span>). The measurement unit becomes sp<sub>E</sub>, where subscript “E” refers to effective. True species diversity hence quantifies <i>how many effective species the dataset represents, given the mean proportional abundance of the actual species</i>. In fact, diversity in general can be defined in this way: <i>the true diversity of the types of entities of interest is the inverse of the mean of their proportional abundances</i>.
 </p>
 
 <div class=paragraph-displayedItem-text-type>A mean can be calculated in different ways, with some kinds of mean giving more weight to small values and others to large values. The weighted generalised mean with exponent <i>q</i>−1 allows a balance to be chosen that is appropriate for the questions at hand (<span><a href=#b51 id=#b51R class="bibLink tab-link" data-tab=pane-pcw-references>Hill 1973</a></span>):
 <div class=inline-equation id=di-fml-1><span class="inline-equation__construct figure"><img class=figure__image src=images/31.gif alt=image title=image></span><span class=inline-equation__label></span></div> 
 </div>
 
 <p>Here <i>p</i><sub><i>i</i></sub> is the proportional abundance of the <i>i</i>th species (see <a href=#t1 class=scrollableLink>Table 1</a> for annotation of proportional abundances). This expression becomes the harmonic mean when <i>q</i>=0, the geometric mean in the limit as <i>q</i> approaches unity, the arithmetic mean when <i>q</i>=2 and the maximum value in the limit as <i>q</i> approches infinity. Each species is nominally weighted by the proportion of the data it contributes to the dataset, i.e. by <i>p</i><sub><i>i</i></sub> itself, but the effective species weights also depend on <i>q</i>. When <i>q</i>=1, the effective weights equal the nominal weights, and each species <i>i</i> affects the mean exactly in proportion to <i>p</i><sub><i>i</i></sub>. As <i>q</i> is increased, the most abundant species gain more effective weight and the mean gradually approaches the largest <i>p</i><sub><i>i</i></sub> value in the dataset no matter how many species the dataset contains. As <i>q</i> is decreased, the least abundant species gain more effective weight, such that at <i>q</i>=0 all effective weights are the same and the mean equals 1/<i>S</i> no matter how unequal the <i>p</i><sub><i>i</i></sub> values. When <i>q</i>&lt;0, the least abundant species would get more effective weight than the most abundant species, and the effective number of species would exceed the actual number of species observed, so <i>q</i> must logically be restricted to nonnegative values.
 </p>
 
 <div class=paragraph-displayedItem-text-type>True diversity is the inverse of <img alt="inline image" class=section_image src=images/32.gif> and equals (<span><a href=#b51 id=#b51R class="bibLink tab-link" data-tab=pane-pcw-references>Hill 1973</a></span>)
 <div class=inline-equation id=di-fml-3><span class="inline-equation__construct figure"><img class=figure__image src=images/33.gif alt=image title=image></span><span class=inline-equation__label></span></div> 
 </div>
 
 <p>where subscript “γ” indicates that the calculations are based on the γ‐classification (see <a href=#t2 class=scrollableLink>Table 2</a> for annotation of derived variables). When either <i>q</i>=0 or all <i>p</i><sub><i>i</i></sub> are equal, then <i>S</i><sub>E</sub>=<i>S</i> and hence <sup><i>q</i></sup><i>D</i><sub>γ</sub>=<i>S</i> sp<sub>E</sub>. Otherwise, <i>S</i><sub>E</sub>&lt;<i>S</i> and therefore <sup><i>q</i></sup><i>D</i><sub>γ</sub>&lt;<i>S</i> sp<sub>E</sub>. The difference between the actual and effective number of species increases as the value of <i>q</i> and/or the inequality among the <i>p</i><sub><i>i</i></sub> values increase.
 </p>
 
 <div class=article-table-content id=t2>
 <header class=article-table-caption><span class=table-caption__label>Table 2.
 </span>Summary of the annotation used for the variables derived from species proportional abundances. Synonymous expressions are listed on the same line. Annotation related to the proportional abundances themselves is explained in <span class=section-footNote><a href=#t1 class=scrollableLink>Table 1</a></span>.
 </header>
 
 
 
 
 <div class=article-table-content-wrapper>
 <table class="table article-section__table">
 
 
 
 
 
 <thead>
 
 <tr>
 
 <th class="bottom-bordered-cell left-aligned">Symbol</th>
 
 <th class="bottom-bordered-cell center-aligned"></th>
 
 <th class="bottom-bordered-cell center-aligned"></th>
 
 <th class="bottom-bordered-cell center-aligned">Explanation</th>
 
 </tr>
 
 </thead>
 
 <tbody>
 
 <tr>
 
 <td class=left-aligned> <img alt="inline image" class=section_image src=images/34.gif> 
 </td>
 
 <td class=left-aligned> <img alt="inline image" class=section_image src=images/35.gif> 
 </td>
 
 <td class=left-aligned> <img alt="inline image" class=section_image src=images/36.gif> 
 </td>
 
 <td class=left-aligned>mean proportional species abundance; the weighted generalised mean with exponent <i>q</i>−1 of the <i>p</i><sub><i>i</i></sub> values
 </td>
 
 </tr>
 
 <tr>
 
 <td class=left-aligned> <sup>
 <i>q</i>
 </sup>λ<sub>γ</sub></td>
 
 <td class=left-aligned> <img alt="inline image" class=section_image src=images/37.gif> 
 </td>
 
 <td class=left-aligned> <img alt="inline image" class=section_image src=images/38.gif> 
 </td>
 
 <td class=left-aligned>basic sum of order <i>q</i> in relation to the γ‐classification: the sum of <i>p</i><sub><i>i</i></sub><sup><i>q</i></sup> values over all species <i>i</i>, or mean proportional species abundance raised to the power <i>q</i>−1
 </td>
 
 </tr>
 
 <tr>
 
 <td class=left-aligned> <sup>
 <i>q</i>
 </sup>
 <i>D</i> 
 </td>
 
 <td class=left-aligned> <img alt="inline image" class=section_image src=images/39.gif> 
 </td>
 
 <td class=left-aligned> <sup>
 <i>q</i>
 </sup>λ<sup>1/(1−<i>q</i>)</sup></td>
 
 <td class=left-aligned>true diversity of order <i>q</i>: the inverse of mean proportional abundances of the types of interest, or the numbers equivalent of <sup><i>q</i></sup>λ
 </td>
 
 </tr>
 
 <tr>
 
 <td class=left-aligned> <i>H</i>′
 </td>
 
 <td class=left-aligned>log(<sup>1</sup><i>D</i>)
 </td>
 
 <td class=left-aligned> <img alt="inline image" class=section_image src=images/40.gif> 
 </td>
 
 <td class=left-aligned>Shannon entropy: the logarithm of true diversity of order 1, or the logarithm of the inverse of the geometric mean of the proportional abundances of the types of interest</td>
 
 </tr>
 
 <tr>
 
 <td class=left-aligned>γ′</td>
 
 <td class=left-aligned></td>
 
 <td class=left-aligned></td>
 
 <td class=left-aligned>the raw value of a species diversity index as calculated using the entire dataset, e.g. <sup><i>q</i></sup>λ<sub>γ</sub> or 
<img alt="inline image" class=section_image src=images/41.gif></td>
 
 </tr>
 
 <tr>
 
 <td class=left-aligned>α′</td>
 
 <td class=left-aligned></td>
 
 <td class=left-aligned></td>
 
 <td class=left-aligned>the alpha component obtained when partitioning γ′</td>
 
 </tr>
 
 <tr>
 
 <td class=left-aligned>β′</td>
 
 <td class=left-aligned></td>
 
 <td class=left-aligned></td>
 
 <td class=left-aligned>the beta component obtained when partitioning γ′</td>
 
 </tr>
 
 <tr>
 
 <td class=left-aligned> <sup>
 <i>q</i>
 </sup>
 <i>D</i>
 <sub>γ</sub> 
 </td>
 
 <td class=left-aligned> <img alt="inline image" class=section_image src=images/42.gif> 
 </td>
 
 <td class=left-aligned> <sup>
 <i>q</i>
 </sup>λ<sub>γ</sub><sup>1/(1−<i>q</i>)</sup></td>
 
 <td class=left-aligned>true gamma diversity: total effective number of species in the dataset (measurement unit: effective species or sp<sub>E</sub>)
 </td>
 
 </tr>
 
 <tr>
 
 <td class=left-aligned> <img alt="inline image" class=section_image src=images/43.gif> 
 </td>
 
 <td class=left-aligned> <img alt="inline image" class=section_image src=images/44.gif> 
 </td>
 
 <td class=left-aligned></td>
 
 <td class=left-aligned>mean proportional species abundance within sampling unit <i>j</i>; the weighted generalised mean with exponent <i>q</i>−1 of the <i>p</i><sub>i∣j</sub> values corresponding to sampling unit <i>j</i></td>
 
 </tr>
 
 <tr>
 
 <td class=left-aligned> <sup>
 <i>q</i>
 </sup>
 <i>D</i>
 <sub>γ<i>j</i></sub> 
 </td>
 
 <td class=left-aligned> <img alt="inline image" class=section_image src=images/45.gif> 
 </td>
 
 <td class=left-aligned>γ<sub><i>j</i></sub>=<sup><i>q</i></sup>λ<sub>γ<i>j</i></sub><sup>1/(1−<i>q</i>)</sup></td>
 
 <td class=left-aligned>gamma diversity (=effective number of species) within sampling unit <i>j</i> (measurement unit: sp<sub>E</sub>)
 </td>
 
 </tr>
 
 <tr>
 
 <td class=left-aligned> <img alt="inline image" class=section_image src=images/46.gif> 
 </td>
 
 <td class=left-aligned> <img alt="inline image" class=section_image src=images/47.gif> 
 </td>
 
 <td class=left-aligned></td>
 
 <td class=left-aligned>mean proportional species abundance in the dataset; the weighted generalised mean with exponent <i>q</i>−1 of all <i>p</i><sub>i∣j</sub> values
 </td>
 
 </tr>
 
 <tr>
 
 <td class=left-aligned>α<sub>t</sub></td>
 
 <td class=left-aligned> <img alt="inline image" class=section_image src=images/48.gif> 
 </td>
 
 <td class=left-aligned> <img alt="inline image" class=section_image src=images/49.gif> 
 </td>
 
 <td class=left-aligned>mean species diversity within sampling units: weighted generalised mean with exponent 1−<i>q</i> of the <sup><i>q</i></sup><i>D</i><sub>γ<i>j</i></sub> values with <i>w</i><sub><i>j</i></sub> used as weights (measurement unit: sp<sub>E</sub>)
 </td>
 
 </tr>
 
 <tr>
 
 <td class=left-aligned>α<sub>d</sub></td>
 
 <td class=left-aligned> <sup>
 <i>q</i>
 </sup>
 <i>D</i>
 <sub>α</sub> 
 </td>
 
 <td class=left-aligned>α<sub>t</sub>/CU
 </td>
 
 <td class=left-aligned>true alpha diversity: effective number of species per virtual sampling unit of mean species diversity, or per compositional unit (measurement unit: sp<sub>E</sub>/CU)
 </td>
 
 </tr>
 
 <tr>
 
 <td class=left-aligned>α<sub>R</sub></td>
 
 <td class=left-aligned> <sup>
 <i>q</i>
 </sup>
 <i>D</i>
 <sub>‘γω’/ω</sub> 
 </td>
 
 <td class=left-aligned> <sup>
 <i>q</i>
 </sup>
 <i>D</i>
 <sub>‘γω’</sub>/<sup><i>q</i></sup><i>D</i><sub>ω</sub></td>
 
 <td class=left-aligned>effective number of species abundance values per effective sampling unit (measurement unit: sp<sub>E</sub>SU<sub>E</sub>/SU<sub>E</sub>)
 </td>
 
 </tr>
 
 <tr>
 
 <td class=left-aligned>β<sub>Md</sub></td>
 
 <td class=left-aligned> <sup>
 <i>q</i>
 </sup>
 <i>D</i>
 <sub>β</sub>=<sup><i>q</i></sup><i>D</i><sub>γ</sub>/<sup><i>q</i></sup><i>D</i><sub>α</sub></td>
 
 <td class=left-aligned>γ/α<sub>d</sub></td>
 
 <td class=left-aligned>true beta diversity: number of compositional units in the dataset (measurement unit: CU)</td>
 
 </tr>
 
 <tr>
 
 <td class=left-aligned>β<sub>Mt</sub></td>
 
 <td class=left-aligned> <img alt="inline image" class=section_image src=images/50.gif> 
 </td>
 
 <td class=left-aligned>γ/α<sub>t</sub></td>
 
 <td class=left-aligned>regional‐to‐local diversity ratio (measurement unit: sp<sub>E</sub>/sp<sub>E</sub>)
 </td>
 
 </tr>
 
 <tr>
 
 <td class=left-aligned>β<sub>R</sub></td>
 
 <td class=left-aligned> <sup>
 <i>q</i>
 </sup>
 <i>D</i>
 <sub>γ</sub>
 <sup>
 <i>q</i>
 </sup>
 <i>D</i>
 <sub>ω</sub>/<sup><i>q</i></sup><i>D</i><sub>‘γω’</sub></td>
 
 <td class=left-aligned>γ/α<sub>R</sub></td>
 
 <td class=left-aligned>two‐way diversity ratio (measurement unit: sp<sub>E</sub>SU<sub>E</sub>/sp<sub>E</sub>SU<sub>E</sub>)
 </td>
 
 </tr>
 
 <tr>
 
 <td class=left-aligned>β<sub>At</sub></td>
 
 <td class=left-aligned> <img alt="inline image" class=section_image src=images/51.gif> 
 </td>
 
 <td class=left-aligned>γ−α<sub>t</sub></td>
 
 <td class=left-aligned>regional diversity excess; absolute effective species turnover (measurement unit: sp<sub>E</sub>)
 </td>
 
 </tr>
 
 <tr>
 
 <td class=left-aligned>β<sub>Mt−1</sub></td>
 
 <td class=left-aligned>γ/α<sub>t</sub>−1
 </td>
 
 <td class=left-aligned>(γ−α<sub>t</sub>)/α<sub>t</sub></td>
 
 <td class=left-aligned>Whittaker's species turnover: effective species turnover expressed in multiples of the species diversity in a single compositional unit (measurement unit: sp<sub>E</sub>/sp<sub>E</sub>)
 </td>
 
 </tr>
 
 <tr>
 
 <td class=left-aligned>β<sub>Pt</sub></td>
 
 <td class=left-aligned>1−α<sub>t</sub>/γ
 </td>
 
 <td class=left-aligned>(γ−α<sub>t</sub>)/γ
 </td>
 
 <td class=left-aligned>proportional species turnover: effective species turnover expressed as a proportion of total species diversity (measurement unit sp<sub>E</sub>/sp<sub>E</sub>)
 </td>
 
 </tr>
 
 <tr>
 
 <td class=left-aligned> <img alt="inline image" class=section_image src=images/52.gif> 
 </td>
 
 <td class=left-aligned> <img alt="inline image" class=section_image src=images/53.gif> 
 </td>
 
 <td class=left-aligned>log(<sup>1</sup>β<sub>Md</sub>)=log(γ)−log(α<sub>d</sub>)
 </td>
 
 <td class=left-aligned>beta Shannon entropy (measurement unit: depends on the base of the logarithm)</td>
 
 </tr>
 
 <tr>
 
 <td class=left-aligned> <img alt="inline image" class=section_image src=images/54.gif> 
 </td>
 
 <td class=left-aligned> <img alt="inline image" class=section_image src=images/55.gif> 
 </td>
 
 <td class=left-aligned>log(<sup>1</sup>β<sub>Mt</sub>)=log(γ)−log(α<sub>t</sub>)
 </td>
 
 <td class=left-aligned>regional Shannon entropy excess (measurement unit: depends on the base of the logarithm)</td>
 
 </tr>
 
 <tr>
 
 <td class=left-aligned> <img alt="inline image" class=section_image src=images/56.gif> 
 </td>
 
 <td class=left-aligned> <img alt="inline image" class=section_image src=images/57.gif> 
 </td>
 
 <td class=left-aligned>(γ−α<sub>t</sub>)/γα<sub>t</sub></td>
 
 <td class=left-aligned>regional variance excess (measurement unit: sp<sub>E</sub>/sp<sub>E</sub><sup>2</sup>)
 </td>
 
 </tr>
 
 </tbody>
 
 </table>
 </div>
 
 
 <div class=article-section__table-source></div>
 </div>
 
 <p>The term <img alt="inline image" class=section_image src=images/58.gif>, known as the <i>basic sum</i>, is important because most of the popular species diversity indices can be derived from it (<span><a href=#b51 id=#b51R class="bibLink tab-link" data-tab=pane-pcw-references>Hill 1973</a></span>, <span><a href=#b60 id=#b60R class="bibLink tab-link" data-tab=pane-pcw-references>Keylock 2005</a></span>). For example, <sup>0</sup>λ<sub>γ</sub> equals species richness, <sup>2</sup>λ<sub>γ</sub> Simpson's index, 1/<sup>2</sup>λ<sub>γ</sub> the inverse Simpson index, 1−<sup>2</sup>λ<sub>γ</sub> the Gini‐Simpson index and <sup>∞</sup>λ<sub>γ</sub> the Berger‐Parker index. The Shannon entropy 
<img alt="inline image" class=section_image src=images/59.gif> (also known as the Shannon index or Shannon‐Wiener index) equals log(<sup>1</sup><i>D</i><sub>γ</sub>) (Mathematical Proof 1). The (1−<i>q</i>)th root of <sup><i>q</i></sup>λ equals true diversity <sup><i>q</i></sup><i>D</i>, which is also known as Hill's (diversity) number or the numbers equivalent of the corresponding diversity index (<span><a href=#b51 id=#b51R class="bibLink tab-link" data-tab=pane-pcw-references>Hill 1973</a></span>, <span><a href=#b91 id=#b91R class="bibLink tab-link" data-tab=pane-pcw-references>Peet 1974</a></span>, <span><a href=#b109 id=#b109R class="bibLink tab-link" data-tab=pane-pcw-references>Routledge 1979</a></span>, <span><a href=#b103 id=#b103R class="bibLink tab-link" data-tab=pane-pcw-references>Ricotta 2005</a></span>, <span><a href=#b56 id=#b56R class="bibLink tab-link" data-tab=pane-pcw-references>Jost 2006</a></span>, <span><a href=#b57 id=#b57R class="bibLink tab-link" data-tab=pane-pcw-references>Jost 2007</a></span>, <span><a href=#b44 id=#b44R class="bibLink tab-link" data-tab=pane-pcw-references>Gregorius and Gillet 2008</a></span>). All diversity indices based on <sup><i>q</i></sup>λ with the same value of <i>q</i> have the same numbers equivalent (<span><a href=#b51 id=#b51R class="bibLink tab-link" data-tab=pane-pcw-references>Hill 1973</a></span>, <span><a href=#b56%20#b57 id="#b56 #b57R" class="bibLink tab-link" data-tab=pane-pcw-references>Jost 2006, 2007</a></span>).
 </p>
 
 <p> <span><a href=#b51 id=#b51R class="bibLink tab-link" data-tab=pane-pcw-references>Hill (1973)</a></span>, <span><a href=#b109 id=#b109R class="bibLink tab-link" data-tab=pane-pcw-references>Routledge (1979)</a></span> and <span><a href=#b56%20#b57 id="#b56 #b57R" class="bibLink tab-link" data-tab=pane-pcw-references>Jost (2006, 2007)</a></span> made a strong case for quantifying diversity using <sup><i>q</i></sup><i>D</i> rather than <sup><i>q</i></sup>λ or any of its transformations other than <sup><i>q</i></sup>λ<sup>1/(1−<i>q</i>)</sup>. With all values of <i>q</i>, true gamma diversity <sup><i>q</i></sup><i>D</i><sub>γ</sub> is positively correlated with the number of actual species <i>S</i> and has the doubling property. This gives it a uniform and intuitive interpretation. In contrast, the interpretation of <sup><i>q</i></sup>λ<sub>γ</sub> changes with the value of <i>q</i>: when <i>q</i>&lt;1, <sup><i>q</i></sup>λ<sub>γ</sub> is positively correlated with <i>S</i> whereas when <i>q</i>&gt;1, <sup><i>q</i></sup>λ<sub>γ</sub> is negatively correlated with <i>S</i>. When 
<img alt="inline image" class=section_image src=images/60.gif> equals unity by definition, whatever the values of <i>S</i> and <i>p</i><sub><i>i</i></sub>, so the Shannon entropy needs to be used instead if a diversity index with <i>q</i>=1 is desired. Furthermore, <sup><i>q</i></sup>λ<sub>γ</sub> has the doubling property only when <i>q</i>=0, which easily leads to misinterpeting differences in <sup><i>q</i></sup>λ<sub>γ</sub> when <i>q</i>&gt;0 (<span><a href=#b56%20#b57 id="#b56 #b57R" class="bibLink tab-link" data-tab=pane-pcw-references>Jost 2006, 2007</a></span>).
 </p>
 
 <p>Given the advantages of <sup><i>q</i></sup><i>D</i><sub>γ</sub>, it is surprising how few ecological studies have used it when a diversity measure that takes species abundances into account is needed (but see <span><a href=#b77 id=#b77R class="bibLink tab-link" data-tab=pane-pcw-references>MacArthur 1964, 1965</a></span>, <span><a href=#b112 id=#b112R class="bibLink tab-link" data-tab=pane-pcw-references>Schlacher et al. 1998</a></span>, <span><a href=#b43 id=#b43R class="bibLink tab-link" data-tab=pane-pcw-references>Gray 2000</a></span>, Ellingsen 2001, 2002, <span><a href=#b11 id=#b11R class="bibLink tab-link" data-tab=pane-pcw-references>Chandy et al. 2006</a></span>, <span><a href=#b29 id=#b29R class="bibLink tab-link" data-tab=pane-pcw-references>Economo and Keitt 2008</a></span>). In the present review, the focus will be on true diversities <sup><i>q</i></sup><i>D</i> and mean proportional abundances 
<img alt="inline image" class=section_image src=images/61.gif>, because this simplifies the discussion on diversity considerably. Diversity indices derived from <sup><i>q</i></sup>λ will be mentioned only to make connections to earlier literature.
 </p>
 
 </section>
 
 <section class=article-section__sub-content id=ss4>
 
 <h3 class="article-section__sub-title section2" id=ss4-title> Diversity in relation to two classifications</h3>
 
 <p>In the previous section, the only classification of interest was the γ‐classification (the classification of individuals, or other units of abundance, into species). All calculations were done using overall species proportional abundances <i>p</i><sub><i>i</i></sub>, as if the dataset were a single sampling unit and <a href=#t1 class=scrollableLink>Table 1</a> consisted of a single row. Expanding that row into multiple rows in effect introduces a second classification of the observed entities, namely their classification into sampling units at a more local scale of observation. This classification is here called the <i>ω‐classification</i> (omega‐classification).
 </p>
 
 <div class=paragraph-displayedItem-text-type>The proportional abundance of the <i>i</i>th species in the entire dataset is obtained as the sum of the <i>p</i><sub><i>ij</i></sub> values in the <i>i</i>th column, which can be rewritten as the weighted arithmetic mean of the corresponding proportional abundances within the <i>N</i> sampling units (<a href=#t1 class=scrollableLink>Table 1</a>):
 <div class=inline-equation id=di-fml-8><span class="inline-equation__construct figure"><img class=figure__image src=images/62.gif alt=image title=image></span><span class=inline-equation__label></span></div> 
 </div>
 
 <p>Here <i>p</i><sub><i>i∣j</i></sub>=<i>m</i><sub><i>ij</i></sub>/<i>m</i><sub><i>j</i></sub> is the proportional abundance of species <i>i</i> within sampling unit <i>j</i>, and each <i>p</i><sub><i>i∣j</i></sub> value is weighted by the proportion of the total abundance contained in the corresponding sampling unit <i>w</i><sub><i>j</i></sub>=<i>m</i><sub><i>j</i></sub>/<i>m</i>. If the total abundance is evenly distributed among the sampling units, all <i>w</i><sub><i>j</i></sub> equal 1/<i>N</i> and the weighted mean simplifies to an unweighted mean. Using modified sampling unit weights <i>w</i><sub><i>j</i>new</sub> (such as 1/<i>N</i> when the <i>w</i><sub><i>j</i></sub> are not equal) leads to quantifying <i>p</i><sub><i>i</i></sub> in a new table in which the proportion of the total abundance contained in sampling unit <i>j</i> is <i>w</i><sub><i>j</i>new</sub>/<i>w</i><sub><i>j</i></sub> times that in the original table. Conceptually, this corresponds to stretching the absolute abundance observed in sampling unit <i>j</i> from <i>m</i><sub><i>j</i></sub> to <i>m</i><sub><i>j</i>new</sub> abundance units. For example, a single bird could be treated as either one or more individuals, depending on which sampling unit it was observed in. The potential difference in <i>p</i><sub><i>i</i></sub> (and hence in mean <i>p</i><sub><i>i</i></sub> and in gamma diversity) between the actual and modified dataset increases as the value of <i>q</i> and/or the deviation of the <i>w</i><sub><i>j</i>new</sub>/<i>w</i><sub><i>j</i></sub> ratios from unity increase.
 </p>
 
 <p>Modifying sampling unit weights by stretching may be justified if the <i>w</i><sub><i>j</i>new</sub> values represent relative sampling effort as quantified, for example, by plot area or duration of observation period. The results need to be interpreted with caution, however, because in reality the number of species and their proportional abundances change with absolute observed abundance. Therefore, rarefaction to a new, smaller absolute abundance <i>m</i><sub><i>j</i>new</sub> is often a better approach to modifying sampling unit weights. This is especially the case when the questions at hand are such that the effect of, for example, variation in bird observability or plant stem density is considered noise rather than a phenomenon of interest. The difference in <i>p</i><sub><i>i</i></sub> between the actual and rarefied dataset increases as the value of <i>q</i> decreases and/or the differences between <i>m</i><sub><i>j</i>new</sub> and the corresponding <i>m</i><sub><i>j</i></sub> increase. Obviously, any conclusions about diversity depend on appropriate weighting of sampling units.
 </p>
 
 <div class=paragraph-displayedItem-text-type>Mean species diversity within the sampling units, or <i>alpha diversity</i>, will often also be of interest. This is quantified by first taking the weighted generalised mean of all within‐sampling unit species proportional abundances in the dataset
 <div class=inline-equation id=di-fml-9><span class="inline-equation__construct figure"><img class=figure__image src=images/63.gif alt=image title=image></span><span class=inline-equation__label></span></div> 
 </div>
 
 <p>The nominal weight <i>p</i><sub><i>ij</i></sub> equals the proportion of the entire dataset that was contributed by the corresponding <i>p</i><sub><i>i∣j</i></sub> value. The inverse of 
<img alt="inline image" class=section_image src=images/64.gif> quantifies within‐sampling unit species diversity α.
 </p>
 
 <p>The measurement unit of α depends on which classifications are relevant for the questions at hand, and this will be indicated by lowercase subscripts. If only the γ‐classification is of interest, the measurement unit is sp<sub>E</sub> and the annotation α<sub>t</sub> is used (the subscript “t” refers to turnover; Sections 3–5). Because this actually quantifies mean gamma diversity within the sampling units, the annotation 
<img alt="inline image" class=section_image src=images/65.gif> can also be used, where subscript “<i>j”</i> specifies that gamma diversity is quantified at the extent of a single sampling unit rather than at the extent of the entire dataset. If both the γ‐classification and the ω‐classification are of interest, the measurement unit is sp<sub>E</sub>/SU where SU stands for <i>sampling unit</i>. When gamma diversity is partitioned into alpha and beta components, conclusions will actually be made about <i>compositional units</i> (CU). These are obtained by classifying the observed <i>S</i><sub>E</sub> effective species evenly into <i>N</i><sub>CU</sub> virtual sampling units (=compositional units) such that each compositional unit receives α<sub>t</sub> effective species shared by no other compositional unit. The classification of effective species into compositional units will be referred to as the β‐classification. Taking into account both the γ‐classification and the β‐classification gives the measurement unit sp<sub>E</sub>/CU. This is the hallmark of true alpha diversity α<sub>d</sub> (=<sup><i>q</i></sup><i>D</i><sub>α</sub>), where subscript “d” refers to diversity. The numerical values of α<sub>d</sub> and α<sub>t</sub> are identical, and the unsubscripted notation α will be used to refer to both.
 </p>
 
 <p>It is also possible to calculate α<sub>t</sub> as the weighted generalised mean with exponent 1−<i>q</i> of the γ<sub><i>j</i></sub> (=<sup><i>q</i></sup><i>D</i><sub>γ<i>j</i></sub>) values (Proof 2). This corresponds to the arithmetic mean when <i>q</i>=0, the geometric mean when <i>q</i>=1 and the harmonic mean when <i>q</i>=2. Whenever mean species diversity within sampling units is mentioned in the present paper, the <i>mean</i> therefore refers to <i>the weighted generalised mean with exponent 1</i>−q <i>and sampling unit weights equal to</i> w<sub>j</sub>. The same mean can be obtained as the numbers equivalent of the weighted arithmetic mean of the basic sums <sup><i>q</i></sup>λ<sub>γ<i>j</i></sub> or (when <i>q</i>=1) Shannon entropies 
<img alt="inline image" class=section_image src=images/66.gif> (Proof 3).
 </p>
 
 <p>Mean within‐sampling unit species diversity <img alt="inline image" class=section_image src=images/67.gif> can never be larger than gamma diversity of the entire dataset γ, with equality being attained when each species is found in all sampling units at a constant proportional abundance. How much smaller than γ the weighted mean 
<img alt="inline image" class=section_image src=images/68.gif> is depends both on the γ<sub><i>j</i></sub> values and on the effective sampling unit weights. As <i>q</i> increases, the nominal weights <i>w</i><sub><i>j</i></sub> lose importance in the generalised mean used to calculate 
<img alt="inline image" class=section_image src=images/69.gif> and the sampling units with the smallest γ<sub><i>j</i></sub> gain progressively more weight. When calculating γ, in contrast, the sampling unit weights are used in an arithmetic mean no matter what the value of <i>q</i>, and the effective weights therefore always equal the nominal weights. The effects of the nominal weights on γ and α are therefore different, except in two special cases: when all <i>w</i><sub><i>j</i></sub> are equal (in which case the weighted mean simplifies to the unweighted mean), and when <i>q</i>=1 (in which case the effective weights in the generalised mean equal the nominal weights by definition). In these special cases, α<sub>t</sub> is restricted to the range [γ/<i>N</i>, γ], where the minimum value is obtained when the <i>N</i> actual sampling units share no species. When the <i>w</i><sub><i>j</i></sub> vary and <i>q</i>≠1, <i>N</i><sub>CU</sub> can exceed <i>N</i> with some combinations of γ<sub><i>j</i></sub>, <i>w</i><sub><i>j</i></sub> and <i>q</i>. If nominal weights other than the row totals <i>w</i><sub><i>j</i></sub> are used, 
<img alt="inline image" class=section_image src=images/70.gif> will be quantified for a new dataset derived by modifying the observed abundance values.
 </p>
 
 <p>Choosing to quantify α<sub>d</sub> implies that the γ‐classification is the primary classification of interest, and the ω‐classification is used only to define the limits of the sampling units within which species diversity is quantified. An alternative approach is to consider both classifications equally interesting, and to treat them symmetrically (<span><a href=#b109 id=#b109R class="bibLink tab-link" data-tab=pane-pcw-references>Routledge 1979</a></span>, <span><a href=#b57 id=#b57R class="bibLink tab-link" data-tab=pane-pcw-references>Jost 2007</a></span>). Then a measure of mean diversity per sampling unit can be obtained by first calculating the total diversity in relation to the γ‐ and ω‐classifications simultaneously (‘γω’), and then dividing this by the total diversity in relation to the ω‐classification (ω). The numerator is obtained as the inverse of the mean of all species proportional abundance values <i>p</i><sub><i>ij</i></sub> and the denominator as the inverse of the mean of the sampling unit weights <i>w</i><sub><i>j</i></sub> (Proof 4). Consequently, the ratio ‘γω’/ω=<sup><i>q</i></sup><i>D</i><sub>‘γω’/ω</sub> quantifies the effective number of species abundance values (virtual cells with mean <i>p</i><sub><i>ij</i></sub> in <a href=#t1 class=scrollableLink>Table 1</a>) per effective sampling unit (virtual rows with mean <i>w</i><sub><i>j</i></sub> in <a href=#t1 class=scrollableLink>Table 1</a>). <span><a href=#b57 id=#b57R class="bibLink tab-link" data-tab=pane-pcw-references>Jost (2007)</a></span> used <sup><i>q</i></sup><i>D</i><sub>‘γω’/ω</sub> as a measure of “alpha diversity”, so the annotation α<sub>R</sub> will also be used here. The subscript “R” refers to a ratio of two true diversities, and uppercase indicates that α<sub>R</sub> differs from α<sub>t</sub> and α<sub>d</sub> both in numerical value and in measurement unit.
 </p>
 
 <p> <span><a href=#b57 id=#b57R class="bibLink tab-link" data-tab=pane-pcw-references>Jost (2007)</a></span> derived α<sub>R</sub> by taking the numbers equivalent of the weighted arithmetic mean of the basic sums <sup><i>q</i></sup>λ<sub>γ<i>j</i></sub> or Shannon entropies 
<img alt="inline image" class=section_image src=images/71.gif>, and then <i>w</i><sub><i>j</i></sub><sup><i>q</i></sup> values are used as weights, instead of <i>w</i><sub><i>j</i></sub> values as when calculating α<sub>t</sub> (Proof 4). Consequently, α<sub>R</sub> equals α<sub>t</sub> when either all sampling units have equal weights or <i>q</i>=1. When <i>q</i>=0, α<sub>R</sub> of the original dataset equals α<sub>t</sub> of a new dataset derived by modifying the observed abundance values. <sup><i>q</i></sup><i>D</i><sub>‘γω’/ω</sub> (=α<sub>R</sub>) may exceed γ with some combinations of <i>p</i><sub><i>ij</i></sub>, <i>w</i><sub><i>j</i></sub> and <i>q</i> but it has a minimum value of γ/<i>N</i>, which is reached when none of the sampling units share any species. If all <i>w</i><sub><i>j</i></sub> are the same or <i>q</i> equals zero or unity, α<sub>R</sub> is constrained to the interval [γ/<i>N</i>, γ].
 </p>
 
 <p>With these basic concepts, we are ready to tackle the problem of “beta diversity”.</p>
 
 </section>
 
 </section>
 <section class=article-section__content id=ss5>
 
 <h2 class="article-section__title section__title section1" id=ss5-title> 1. True beta diversity β<sub>Md</sub>=γ/α<sub>d</sub> and regional‐to‐local diversity ratio β<sub>Mt</sub>=γ/α<sub>t</sub></h2>
 
 <section class=article-section__sub-content id=ss6>
 
 <h3 class="article-section__sub-title section2" id=ss6-title> The definitions</h3>
 
 <p>In his example 3, <span><a href=#b129 id=#b129R class="bibLink tab-link" data-tab=pane-pcw-references>Whittaker (1960)</a></span> introduced the equation β=γ/α as a quantitative definition of beta diversity. This corresponds to a multiplicative partitioning of total species diversity γ=αβ<sub>M</sub> where subscript “M” refers to multiplicative. β<sub>M</sub> is independent of the species richness of the system, as can be verified by the thought experiment of splitting each species: both α and γ will double, but their ratio will remain unchanged. At first, <span><a href=#b129 id=#b129R class="bibLink tab-link" data-tab=pane-pcw-references>Whittaker (1960)</a></span> used raw values of a diversity index (Fisher's alpha), but later he realised that numbers equivalents of diversity indices should be used (<span><a href=#b130 id=#b130R class="bibLink tab-link" data-tab=pane-pcw-references>Whittaker 1972</a></span>). Otherwise the gamma and alpha components may lack the doubling property, which would make the beta component dependent on the species richness of the system.
 </p>
 
 <p> <span><a href=#b130 id=#b130R class="bibLink tab-link" data-tab=pane-pcw-references>Whittaker (1972)</a></span> discussed both gamma diversity and alpha diversity in terms of species diversity, and hence calculated the ratio 
<img alt="inline image" class=section_image src=images/72.gif>. The resulting beta component β<sub>Mt</sub> (=<sup><i>q</i></sup>β<sub>Mt</sub>) quantifies <i>how many times as rich in effective species an entire dataset is than its constituent sampling units are on average</i>. Because species diversity in each compositional unit equals mean species diversity in the actual sampling units, β<sub>Mt</sub> also quantifies <i>how many times as rich in effective species the dataset is than one of its constituent compositional units</i>. β<sub>Mt</sub> can be called the <i>regional‐to‐local diversity ratio</i>; it is a unitless scalar that quantifies the ratio of gamma diversities at two different levels of observation.
 </p>
 
 <p>Using true alpha diversity α<sub>d</sub> instead of α<sub>t</sub> gives true beta diversity β<sub>Md</sub>=<sup><i>q</i></sup>β<sub>Md</sub>=γ/α<sub>d</sub>. True beta diversity quantifies <i>the number of compositional units in the dataset</i>. In the previous section, the classification of effective species into compositional units was defined as the β‐classification, so true beta diversity also quantifies <i>the total diversity in the dataset in relation to the β‐classification</i>. This shows that β<sub>Md</sub> is a true diversity, which can be emphasised by using the annotation <sup><i>q</i></sup><i>D</i><sub>β</sub>=<sup><i>q</i></sup><i>D</i><sub>γ</sub>/<sup><i>q</i></sup><i>D</i><sub>α</sub>. Its measurement unit is sp<sub>E</sub>/(sp<sub>E</sub>/CU)=CU.
 </p>
 
 <p>Discussions about alpha, beta and gamma diversity have usually ignored measurement units, which has probably contributed to the confusion around the concepts. After all, the plain numbers 1, 2 and 3 are much more easily compared as if they quantified the same phenomenon than are values such as 1 CU, 2 sp<sub>E</sub> and 3 bits. The difference between true beta diversity β<sub>Md</sub> and regional‐to‐local diversity ratio β<sub>Mt</sub> is subtle, as their numerical values are the same. However, the difference in measurement unit is conceptually important. <span><a href=#b129 id=#b129R class="bibLink tab-link" data-tab=pane-pcw-references>Whittaker (1960)</a></span> observed that “The same types of measurements may be applied to ‘gamma’ as to ‘alpha’ diversity; ‘beta’ diversity represents a different problem”. <span><a href=#b131 id=#b131R class="bibLink tab-link" data-tab=pane-pcw-references>Whittaker (1977)</a></span> referred to α and γ as <i>inventory diversity</i> and to β as <i>differentiation diversity</i>, which has since become a common practice (<span><a href=#b78 id=#b78R class="bibLink tab-link" data-tab=pane-pcw-references>Magurran 2004</a></span>). Some researchers have even argued that beta diversity should not be called diversity at all (<span><a href=#b68 id=#b68R class="bibLink tab-link" data-tab=pane-pcw-references>Lande 1996</a></span>, <span><a href=#b61 id=#b61R class="bibLink tab-link" data-tab=pane-pcw-references>Kiflawi and Spencer 2004</a></span>, <span><a href=#b44 id=#b44R class="bibLink tab-link" data-tab=pane-pcw-references>Gregorius and Gillet 2008</a></span>). This statement is justified if “beta diversity” is defined in terms of the regional‐to‐local diversity ratio β<sub>Mt</sub> or species turnover (to be discussed in Sections 3–5), which really are conceptually different from true alpha and gamma diversity. However, true beta diversity β<sub>Md</sub> is the number of compositional units, which is diversity in the very same sense as is the number of effective species.
 </p>
 
 <p>Indeed, α<sub>d</sub> (=<sup><i>q</i></sup><i>D</i><sub>α</sub>), β<sub>Md</sub> (=<sup><i>q</i></sup><i>D</i><sub>β</sub>) and γ (=<sup><i>q</i></sup><i>D</i><sub>γ</sub>) differ only because they focus on different entities (individuals in α<sub>d</sub> and γ vs. effective species in β<sub>Md</sub>) or on quantifying the diversity of the types into which those entities are classified at a different level (in the entire dataset in β<sub>Md</sub> and γ vs per compositional unit in α<sub>d</sub>). This justifies singling β<sub>Md</sub> out as the sole measure of true beta diversity, and recommending that all other definitions of “beta diversity” be called something else.
 </p>
 
 <p>As we saw above, β<sub>Mt</sub>=<i>N</i><sub>CU</sub>. The minimum value of <i>N</i><sub>CU</sub> equals unity, obtained when all sampling units have the same species in the same proportional abundances. The maximum value that <i>N</i><sub>CU</sub> can take depends on the effective sampling unit weights, which determine the minimum value of α in relation to γ (<i>Diversity in relation to two classifications</i>, above). The values that β<sub>Mt</sub> and β<sub>Md</sub> take when no sampling units share any species therefore depend on both the ω‐classification and the γ‐classification. When either all <i>w</i><sub><i>j</i></sub> are equal or <i>q</i>=1, <i>N</i><sub>CU</sub> cannot exceed <i>N</i> which constrains β<sub>Mt</sub> to the interval [1, <i>N</i>] and β<sub>Md</sub> to the interval [1 CU, <i>N</i> CU]. These ranges depend only on the ω‐classification.
 </p>
 
 <p>The generic term <i>multiplicative beta component</i> (β<sub>M</sub>) will here be used to refer to either β<sub>Md</sub> or β<sub>Mt</sub>. Increasing the value of <i>q</i> makes <sup><i>q</i></sup>β<sub>M</sub> more sensitive to the variation among sampling units in the proportional abundances of species and less sensitive to variation in species composition. Consequently, <sup>0</sup>β<sub>M</sub> is not affected by changes in species abundances (as long as presence‐absence patterns do not change), and <sup>∞</sup>β<sub>M</sub> is not affected by changes in species composition (as long as the abundance of the single most abundant species does not change).
 </p>
 
 <p>It is important to notice that the logical consistency of β<sub>M</sub> necessitates that both γ and α are based on the same dataset. This implies that the weight given to sampling unit <i>j</i> has to be the same when calculating γ and when calculating α. Using the row totals from <a href=#t1 class=scrollableLink>Table 1</a> as sampling unit weights gives the alpha, beta and gamma diversities of that table. Using some other weights gives the diversity components of a new table in which observed abundances have been modified according to the weights. If different weights are used when calculating γ and when calculating α, these will be quantified for different datasets. Dividing γ of one dataset by α of another dataset produces a ratio that does not correspond to β<sub>M</sub> for either dataset.
 </p>
 
 <p>If each sampling unit represents a community (or a habitat) and all sampling units together represent a region, true beta diversity represents the number of compositionally non‐overlapping community (or habitat) types in the region. This interpretation is ecologically accurate only if each sampling unit is sufficiently large to be truly representative of its community (or habitat), and if enough sampling units have been inventoried for them to be truly representative of the entire region. In practical applications these conditions are seldom met, but evaluating how sampling problems may affect the reliability of extrapolations beyond the dataset at hand is deferred to the second part of the present review (<span><a href=#b119 id=#b119R class="bibLink tab-link" data-tab=pane-pcw-references>Tuomisto 2010</a></span>).
 </p>
 
 <p>β<sub>M</sub>, especially as applied to presence‐absence data, is one of the most popular definitions of “beta diversity” in ecology (<span><a href=#b108%20#b109 id="#b108 #b109R" class="bibLink tab-link" data-tab=pane-pcw-references>Routledge 1977, 1979</a></span>, <span><a href=#b69 id=#b69R class="bibLink tab-link" data-tab=pane-pcw-references>Lee and La Roi 1979</a></span>, <span><a href=#b79 id=#b79R class="bibLink tab-link" data-tab=pane-pcw-references>McCune and Antos 1981</a></span>, <span><a href=#b115 id=#b115R class="bibLink tab-link" data-tab=pane-pcw-references>Stoms 1994</a></span>, <span><a href=#b128 id=#b128R class="bibLink tab-link" data-tab=pane-pcw-references>Weiher and Boylen 1994</a></span>, <span><a href=#b43 id=#b43R class="bibLink tab-link" data-tab=pane-pcw-references>Gray 2000</a></span>, <span><a href=#b94 id=#b94R class="bibLink tab-link" data-tab=pane-pcw-references>Perelman et al. 2001</a></span>, <span><a href=#b126 id=#b126R class="bibLink tab-link" data-tab=pane-pcw-references>Vellend 2001</a></span>, <span><a href=#b4 id=#b4R class="bibLink tab-link" data-tab=pane-pcw-references>Arita and Rodríguez 2002</a></span>, <span><a href=#b32 id=#b32R class="bibLink tab-link" data-tab=pane-pcw-references>Ellingsen and Gray 2002</a></span>, <span><a href=#b47 id=#b47R class="bibLink tab-link" data-tab=pane-pcw-references>Harrison and Inoye 2002</a></span>, <span><a href=#b106 id=#b106R class="bibLink tab-link" data-tab=pane-pcw-references>Rodríguez and Arita 2004</a></span>, <span><a href=#b133 id=#b133R class="bibLink tab-link" data-tab=pane-pcw-references>Wiersma and Urban 2005</a></span>, <span><a href=#b74 id=#b74R class="bibLink tab-link" data-tab=pane-pcw-references>Lira‐Noriega et al. 2007</a></span>, <span><a href=#b89 id=#b89R class="bibLink tab-link" data-tab=pane-pcw-references>Passy and Blanchet 2007</a></span>, <span><a href=#b3 id=#b3R class="bibLink tab-link" data-tab=pane-pcw-references>Arita et al. 2008</a></span>, <span><a href=#b36 id=#b36R class="bibLink tab-link" data-tab=pane-pcw-references>Gallardo‐Cruz et al. 2009</a></span>). Some of these studies clearly discussed true beta diversity β<sub>Md</sub> and others regional‐to‐local diversity ratio β<sub>Mt</sub>, but not all specified their interpretation.
 </p>
 
 </section>
 
 <section class=article-section__sub-content id=ss7>
 
 <h3 class="article-section__sub-title section2" id=ss7-title> Hierarchical diversity partitioning</h3>
 
 <p>Above, true gamma diversity was partitioned into two independent components. If the γ‐ or ω‐classifications are hierarchically structured, more than two independent components can be obtained. Hierarchical ω‐classification means that each row in <a href=#t1 class=scrollableLink>Table 1</a> represents data that have been pooled from a number of smaller sampling units, possibly over several hierarchical levels. Let us identify the hierarchical levels such that the highest hierarchical level below that of the entire dataset is level 1, the next more local level is level 2, and so on. If the number of level‐1 sampling units is <i>N</i><sub>1</sub> and the number of level‐2 sampling units is <i>N</i><sub>2</sub>, <a href=#t1 class=scrollableLink>Table 1</a> could contain either <i>N</i><sub>1</sub> or <i>N</i><sub>2</sub> rows, depending on which level is shown. The cell values would then be adapted accordingly, such that the <i>p</i><sub><i>ij</i></sub> values in a given level‐1 sampling unit are the species‐wise sums of the <i>p</i><sub><i>ij</i></sub> values in its constituent level‐2 sampling units. Partitioning gamma diversity (species diversity in the entire dataset) at level 1 gives the true diversity components α<sub>1</sub> (mean species diversity per level‐1 compositional unit) and β<sub>1</sub> (effective number of level‐1 compositional units in the entire dataset). Level‐1 alpha diversity can be further partitioned into the true diversity components α<sub>2</sub> (mean species diversity per level‐2 compositional unit) and σ<sub>2/1</sub> (mean level‐2 compositional unit diversity per level‐1 compositional unit). Similarly, α<sub>2</sub> can be partitioned into α<sub>3</sub> and σ<sub>3/2</sub> and so on.
 </p>
 
 <p>The new diversity component σ (sigma) is analogous to α because it is quantified as a mean of diversities observed per compositional unit, rather than as a single value for the entire dataset. The relationship between beta diversity and sigma diversity is therefore similar to that between gamma diversity and alpha diversity. The measurement unit of γ is sp<sub>E</sub> and that of α<sub><i>h</i></sub> is sp<sub>E</sub>/CU<sub><i>h</i></sub> where CU<sub><i>h</i></sub> stands for level‐<i>h</i> compositional unit. Analogously, the measurement unit of β<sub>1</sub> is CU<sub>1</sub> and that of σ<sub>(<i>h</i>+1)/<i>h</i></sub> is CU<sub><i>h</i>+1</sub>/CU<sub><i>h</i></sub>.
 </p>
 
 <div class=paragraph-displayedItem-text-type>These true diversities are multiplicatively related by 
 <div class=inline-equation id=di-fml-19><span class="inline-equation__construct figure"><img class=figure__image src=images/73.gif alt=image title=image></span><span class=inline-equation__label></span></div> 
 </div>
 
 <div class=paragraph-displayedItem-text-type>The units of measurement on the right side of this equation are 
 <div class=inline-equation id=di-fml-20><span class="inline-equation__construct figure"><img class=figure__image src=images/74.gif alt=image title=image></span><span class=inline-equation__label></span></div> 
 </div>
 
 <p>as we would expect of gamma diversity.</p>
 
 <div class=paragraph-displayedItem-text-type>Gamma diversity can be partitioned at any hierarchical level into alpha and beta diversities at the same level, or alpha and beta diversities at different levels complemented by sigma diversity of appropriate level(s). For example, 
 <div class=inline-equation id=di-fml-21><span class="inline-equation__construct figure"><img class=figure__image src=images/75.gif alt=image title=image></span><span class=inline-equation__label></span></div> 
 </div>
 
 <p>A hierarchically structured γ‐classification can be used in a similar way. This allows quantifying what proportion of the observed total species diversity is due to, for example, species diversity within genera, genus diversity within families and so on (<span><a href=#b95 id=#b95R class="bibLink tab-link" data-tab=pane-pcw-references>Pielou 1975</a></span>, pp. 17–18).
 </p>
 
 </section>
 
 <section class=article-section__sub-content id=ss8>
 
 <h3 class="article-section__sub-title section2" id=ss8-title> Heterogeneity measures</h3>
 
 <p>Recently, <span><a href=#b59 id=#b59R class="bibLink tab-link" data-tab=pane-pcw-references>Jurasinski et al. (2009)</a></span> argued that there is a conceptual difference between beta diversity as calculated from the relationship between alpha and gamma diversity, and beta diversity as quantified using distance coefficients. In fact, many (dis)similarity coefficients can be derived from alpha and gamma diversity as calculated for a dataset that consists of two sampling units. Different definitions of “beta diversity” therefore naturally give rise to different dissimilarity coefficients.
 </p>
 
 <div class=paragraph-displayedItem-text-type> <span><a href=#b77 id=#b77R class="bibLink tab-link" data-tab=pane-pcw-references>MacArthur (1965)</a></span> measured the faunal difference between two censuses by 
<img alt="inline image" class=section_image src=images/76.gif>In the annotation of the present paper, MacArthur's measure equals
 <div class=inline-equation id=di-fml-23><span class="inline-equation__construct figure"><img class=figure__image src=images/77.gif alt=image title=image></span><span class=inline-equation__label></span></div> 
 </div>
 
 <div class=paragraph-displayedItem-text-type>More generally, <sup><i>q</i></sup>β<sub>M</sub> can be used as an index of total compositional heterogeneity in a dataset at the scale represented by the sampling units. However, <sup><i>q</i></sup>β<sub>M</sub> does not measure compositional heterogeneity among the sampling units themselves. This is because it can obtain the same value with a small number of compositionally dissimilar samping units or with a larger number of more similar sampling units. If compositional heterogeneity among sampling units is of interest, <sup><i>q</i></sup>β<sub>M</sub> can be partitioned into two independent components:
 <div class=inline-equation id=di-fml-24><span class="inline-equation__construct figure"><img class=figure__image src=images/78.gif alt=image title=image></span><span class=inline-equation__label></span></div> 
 </div>
 
 <p> <i>N</i>
 <sub>CU</sub>/<i>N</i> quantifies mean heterogeneity per sampling unit, i.e. how many compositional units there are for each actual sampling unit in the dataset. If all sampling units have the same weight or <i>q</i>=1, <sup><i>q</i></sup>β<sub>Mt</sub> is constrained to values in the interval [1, <i>N</i>] and <i>N</i><sub>CU</sub>/<i>N</i> becomes constrained to [1/<i>N</i>, 1].
 </p>
 
 <div class=paragraph-displayedItem-text-type>When <sup><i>q</i></sup>β<sub>Mt</sub> is calculated for two sampling units (<i>N</i>=2) using presence‐absence data (<i>q</i>=0) and equal sampling unit weights, it is inversely related with the Jaccard index (<i>C</i><sub>J</sub>) and linearly (and negatively) related with the Sørensen index (<i>C</i><sub>S</sub>). In the equations below (and in others that will follow), <i>a</i> is the number of species shared by both sampling units, <i>b</i> is the number of species unique to the first sampling unit and <i>c</i> is the number of species unique to the second sampling unit.
 <div class=inline-equation id=di-fml-25><span class="inline-equation__construct figure"><img class=figure__image src=images/79.gif alt=image title=image></span><span class=inline-equation__label></span></div> 
 </div>
 
 <p>Although both the Jaccard and the Sørensen index are monotonic transformations of <sup>0</sup>β<sub>M</sub>, they are still transformations and therefore do not quantify either <sup>0</sup>β<sub>Mt</sub> or <sup>0</sup>β<sub>Md</sub>. It is also important to keep in mind that the relevant <sup>0</sup>β<sub>M</sub> here is that of two equally weighted sampling units (<i>Diversity in relation to two classifications</i>, above).
 </p>
 
 </section>
 
 </section>
 <section class=article-section__content id=ss9>
 
 <h2 class="article-section__title section__title section1" id=ss9-title> 2. Two‐way diversity ratio β<sub>R</sub>=γ/α<sub>R</sub></h2>
 
 <div class=paragraph-displayedItem-text-type> <span><a href=#b57 id=#b57R class="bibLink tab-link" data-tab=pane-pcw-references>Jost (2007)</a></span> required that in addition to being independent of alpha diversity, beta diversity should be monotonic with respect to compositional differentiation among the sampling units (given the ω‐classification). This led him to divide gamma diversity by the effective number of species abundance values per effective sampling unit (α<sub>R</sub>=<sup><i>q</i></sup><i>D</i><sub>‘γω’/ω</sub>; <i>Diversity in relation to two classifications</i>, above). Jost's definition of “beta diversity” is therefore <sup><i>q</i></sup>β<sub>R</sub>=γ/α<sub>R</sub> which equals
 <div class=inline-equation id=di-fml-26><span class="inline-equation__construct figure"><img class=figure__image src=images/80.gif alt=image title=image></span><span class=inline-equation__label></span></div> 
 </div>
 
 <p>The numerator equals the number of effective species (=number of virtual columns with mean <i>p</i><sub><i>i</i></sub> in <a href=#t1 class=scrollableLink>Table 1</a>) multiplied by the number of effective sampling units (=number of virtual rows with mean <i>w</i><sub><i>j</i></sub>). The denominator equals the number of effective species–sampling unit combinations (=number of virtual cells with mean <i>p</i><sub><i>ij</i></sub>). <span><a href=#b57 id=#b57R class="bibLink tab-link" data-tab=pane-pcw-references>Jost (2007)</a></span> showed that <sup><i>q</i></sup><i>D</i><sub>γω/‘γω’</sub> is monotonically related with compositional differentiation among sampling units only when either all sampling unit weights <i>w</i><sub><i>j</i></sub> are equal or <i>q</i> equals zero or unity, and therefore restricted its use to these special cases.
 </p>
 
 <p>However, <sup>q</sup>D<sub>γω/‘γω’</sub> has a logical interpretation in terms of diversity with any w<sub>j</sub> and q values. It quantifies <i>how many times as much diversity in relation to the γ‐ and ω‐classifications there is in the dataset when the classifications are considered separately vs. when they are considered together.</i><sup>q</sup>D<sub>γω/‘γω’</sub>=<sup>q</sup>β<sub>R</sub> can be called <i>two‐way diversity ratio</i>, since it compares diversities in relation to two different classifications, calculated in two different ways. The logical measurement unit of both the numerator and the denominator is sp<sub>E</sub>SU<sub>E</sub> (the product of effective species and effective sampling units), so <sup>q</sup>β<sub>R</sub> simplifies to a unitless scalar. It has a maximum value of N, which is obtained when no sampling units share species, but no fixed lower limit except in the special cases when it is monotonically related with differentiation. Then the minimum value is unity, which is obtained when all sampling units have the same species in the same proportional abundances. When all sampling unit weights are equal or q=1, <sup>q</sup>β<sub>R</sub> equals regional‐to‐local diversity ratio <sup>q</sup>β<sub>Mt</sub>. When q=0, <sup>q</sup>β<sub>R</sub> of the original dataset equals <sup>q</sup>β<sub>Mt</sub> of a new dataset in which the abundances observed in all sampling units have been modified so as to be equal.
 </p>
 
 </section>
 <section class=article-section__content id=ss10>
 
 <h2 class="article-section__title section__title section1" id=ss10-title> 3. Regional diversity excess (absolute effective species turnover) β<sub>At</sub>=γ−α<sub>t</sub></h2>
 
 <p> <i>Regional diversity excess</i>β<sub>At</sub> (or <sup><i>q</i></sup>β<sub>At</sub>) corresponds to additive partitioning of total diversity γ=α+β and equals 
<img alt="inline image" class=section_image src=images/81.gif> where subscript “A” indicates additive. This quantifies <i>the amount by which the effective species richness of the entire (regional) dataset exceeds that of a single sampling unit of mean effective species richness</i>.
 </p>
 
 <div class=paragraph-displayedItem-text-type>Both α<sub>t</sub>+β<sub>At</sub> and α<sub>t</sub>β<sub>Mt</sub> equal gamma diversity, from which follows
 <div class=inline-equation id=di-fml-28><span class="inline-equation__construct figure"><img class=figure__image src=images/82.gif alt=image title=image></span><span class=inline-equation__label></span></div> 
 </div>
 
 <p>All effective species not present in one compositional unit have to be present in the other <i>N</i><sub>CU</sub>−1 compositional units, which causes turnover of effective species. Therefore, β<sub>At</sub> can also be interpreted as <i>the absolute amount of effective species turnover among the compositional units of the dataset</i> (hence the subscript “t”).
 </p>
 
 <p>The minimum value of <sup><i>q</i></sup>β<sub>At</sub> is zero effective species, which is obtained when all actual sampling units have the same species in constant proportional abundances. The maximum value depends on both the number of compositional units and on α<sub>t</sub>. If either all sampling unit weights are equal or <i>q</i>=1, then <i>N</i><sub>CU</sub> cannot exceed <i>N</i> and <sup><i>q</i></sup>β<sub>At</sub> is constrained not to exceed (<i>N</i>−1)α<sub>t</sub>. In these special cases, <sup><i>q</i></sup>β<sub>At</sub> also quantifies absolute effective species turnover among the <i>N</i> actual sampling units. If <i>q</i>=0, effective species equal actual species, so if all sampling unit weights are equal then <sup>0</sup>β<sub>At</sub> can also be interpreted as absolute actual species turnover among the <i>N</i> actual sampling units.
 </p>
 
 <p>Because β<sub>At</sub>=α<sub>t</sub>(β<sub>Mt</sub>−1) and, equivalently, β<sub>Mt</sub>=β<sub>At</sub>/α<sub>t</sub>+1, it is obvious that absolute effective species turnover β<sub>At</sub> is not monotonically related with β<sub>Mt</sub> and β<sub>Md</sub> when the datasets to be compared differ in α. Whereas β<sub>M</sub> is independent of the species diversity of the observed system, β<sub>At</sub> is not. Consider the thought experiment of duplicating each species: not only α<sub>t</sub> and γ will double, but β<sub>At</sub> will also double. Therefore, absolute effective species turnover may be smaller in a species‐poor region with many compositional units than in a species‐rich region with few compositional units, and conflicting results may be obtained if regional datasets are ranked on the basis of both β<sub>At</sub> and β<sub>M</sub>.
 </p>
 
 <div class=paragraph-displayedItem-text-type>When calculated for two sampling units with <i>q</i>=0 and equal sampling unit weights, absolute effective species turnover is related to the Manhattan metric (<i>M</i>) and the Euclidean distance (<i>E</i>) as calculated using presence‐absence data (<i>a</i>, <i>b</i>, and <i>c</i> as in Section 1):
 <div class=inline-equation id=di-fml-29><span class="inline-equation__construct figure"><img class=figure__image src=images/83.gif alt=image title=image></span><span class=inline-equation__label></span></div> 
 </div>
 
 <p>In these equations, <i>m</i><sub><i>i</i>1</sub> is the abundance of species <i>i</i> in the first sampling unit and <i>m</i><sub><i>i</i>2</sub> in the second (the abundance data have to be binary: 0 for absence, 1 for presence). Absolute effective species turnover can therefore be used to generalise either the Manhattan metric or the squared Euclidean distance to a presence‐absence dataset with multiple equally‐weighted sampling units. Both dissimilarity indices have properties that are not desirable when applied to compositional data (<span><a href=#b70 id=#b70R class="bibLink tab-link" data-tab=pane-pcw-references>Legendre and Legendre 1998</a></span>), so they have not been particularly popular in beta diversity studies (but see <span><a href=#b128 id=#b128R class="bibLink tab-link" data-tab=pane-pcw-references>Weiher and Boylen 1994</a></span>, <span><a href=#b112 id=#b112R class="bibLink tab-link" data-tab=pane-pcw-references>Schlacher et al. 1998</a></span>, <span><a href=#b67 id=#b67R class="bibLink tab-link" data-tab=pane-pcw-references>Koleff et al. 2003b</a></span>).
 </p>
 
 <div class=paragraph-displayedItem-text-type> <span><a href=#b77 id=#b77R class="bibLink tab-link" data-tab=pane-pcw-references>MacArthur (1964, 1965)</a></span> seems to have been the first one to partition species diversity data using an additive equation, but he restricted its use to the Shannon entropy. The additive equation
 <div class=inline-equation id=di-fml-30><span class="inline-equation__construct figure"><img class=figure__image src=images/84.gif alt=image title=image></span><span class=inline-equation__label></span></div> 
 </div>
 
 <div class=paragraph-displayedItem-text-type>can be rewritten 
 <div class=inline-equation id=di-fml-31><span class="inline-equation__construct figure"><img class=figure__image src=images/85.gif alt=image title=image></span><span class=inline-equation__label></span></div> 
 </div>
 
 <div class=paragraph-displayedItem-text-type>which equals (Proof 1) 
 <div class=inline-equation id=di-fml-32><span class="inline-equation__construct figure"><img class=figure__image src=images/86.gif alt=image title=image></span><span class=inline-equation__label></span></div> 
 </div>
 
 <p>Both <span><a href=#b77 id=#b77R class="bibLink tab-link" data-tab=pane-pcw-references>MacArthur (1965)</a></span> and <span><a href=#b108%20#b109 id="#b108 #b109R" class="bibLink tab-link" data-tab=pane-pcw-references>Routledge (1977, 1979)</a></span> observed that converting Shannon entropies to their numbers equivalents leads to Whittaker's multiplicative diversity components. This relationship has been discussed several times recently (<span><a href=#b103 id=#b103R class="bibLink tab-link" data-tab=pane-pcw-references>Ricotta 2005</a></span>, <span><a href=#b56%20#b57 id="#b56 #b57R" class="bibLink tab-link" data-tab=pane-pcw-references>Jost 2006, 2007</a></span>).
 </p>
 
 <div class=paragraph-displayedItem-text-type>Although it is possible to rephrase 
 <div class=inline-equation id=di-fml-33><span class="inline-equation__construct figure"><img class=figure__image src=images/87.gif alt=image title=image></span><span class=inline-equation__label></span></div> 
 </div>
 
 <p>this easily leads to confusing true diversities with indices of diversity, and to applying the additive partitioning to diversity indices for which a beta component that is independent on the species richness of the system cannot be obtained (see also <span><a href=#b57 id=#b57R class="bibLink tab-link" data-tab=pane-pcw-references>Jost 2007</a></span>). In the present paper, the symbols α, β and γ are used only when referring to the components of true diversity, and the symbols α′, β′ and γ′ when referring to raw diversity index values.
 </p>
 
 <p>Regional diversity excess β<sub>At</sub> was hardly used until <span><a href=#b68 id=#b68R class="bibLink tab-link" data-tab=pane-pcw-references>Lande (1996)</a></span> and <span><a href=#b125 id=#b125R class="bibLink tab-link" data-tab=pane-pcw-references>Veech et al. (2002)</a></span> argued that measuring alpha and beta “diversity” in the same units (in this case, sp or sp<sub>E</sub>) is an advantage. <span><a href=#b68 id=#b68R class="bibLink tab-link" data-tab=pane-pcw-references>Lande (1996)</a></span> followed <span><a href=#b83 id=#b83R class="bibLink tab-link" data-tab=pane-pcw-references>Nei (1973)</a></span> and <span><a href=#b90 id=#b90R class="bibLink tab-link" data-tab=pane-pcw-references>Patil and Taillie (1982)</a></span> in applying the additive partitioning γ′=α′+β′ not only to the Shannon entropy but also to other diversity indices such as <sup>0</sup>λ. Because <sup>0</sup>λ=<sup>0</sup><i>D</i>, this leads to an additive rather than multiplicative partitioning of true gamma diversity, and the meaning of the beta component is thereby changed. Regional diversity excess β<sub>At</sub> has become a popular measure of “beta diversity”, especially when partitioning regional species richness at multiple spatial scales (<span><a href=#b127 id=#b127R class="bibLink tab-link" data-tab=pane-pcw-references>Wagner et al. 2000</a></span>, <span><a href=#b38 id=#b38R class="bibLink tab-link" data-tab=pane-pcw-references>Gering and Crist 2002</a></span>, <span><a href=#b23 id=#b23R class="bibLink tab-link" data-tab=pane-pcw-references>Crist et al. 2003</a></span>, <span><a href=#b39 id=#b39R class="bibLink tab-link" data-tab=pane-pcw-references>Gering et al. 2003</a></span>, <span><a href=#b116 id=#b116R class="bibLink tab-link" data-tab=pane-pcw-references>Summerville et al. 2003</a></span>, <span><a href=#b107 id=#b107R class="bibLink tab-link" data-tab=pane-pcw-references>Roschewitz et al. 2005</a></span>, <span><a href=#b11 id=#b11R class="bibLink tab-link" data-tab=pane-pcw-references>Chandy et al. 2006</a></span>, <span><a href=#b22 id=#b22R class="bibLink tab-link" data-tab=pane-pcw-references>Crist and Veech 2006</a></span>, <span><a href=#b35 id=#b35R class="bibLink tab-link" data-tab=pane-pcw-references>Freestone and Inouye 2006</a></span>, <span><a href=#b122 id=#b122R class="bibLink tab-link" data-tab=pane-pcw-references>Tylianakis et al. 2006</a></span>, <span><a href=#b8 id=#b8R class="bibLink tab-link" data-tab=pane-pcw-references>Belmaker et al. 2008</a></span>, <span><a href=#b14 id=#b14R class="bibLink tab-link" data-tab=pane-pcw-references>Chiarucchi et al. 2008</a></span>, <span><a href=#b37 id=#b37R class="bibLink tab-link" data-tab=pane-pcw-references>Gardezi and Gonzalez 2008</a></span>, <span><a href=#b62 id=#b62R class="bibLink tab-link" data-tab=pane-pcw-references>Klimek et al. 2008</a></span>, <span><a href=#b101 id=#b101R class="bibLink tab-link" data-tab=pane-pcw-references>Ribeiro et al. 2008</a></span>, <span><a href=#b114 id=#b114R class="bibLink tab-link" data-tab=pane-pcw-references>Sobek et al. 2009</a></span>).
 </p>
 
 <p>Regional diversity excess has been referred to as “additive beta diversity” (<span><a href=#b61 id=#b61R class="bibLink tab-link" data-tab=pane-pcw-references>Kiflawi and Spencer 2004</a></span>, <span><a href=#b103%20#b104 id="#b103 #b104R" class="bibLink tab-link" data-tab=pane-pcw-references>Ricotta 2005, 2008</a></span>, <span><a href=#b29 id=#b29R class="bibLink tab-link" data-tab=pane-pcw-references>Economo and Keitt 2008</a></span>), but it is conceptually very different from true beta diversity. Whereas β<sub>Md</sub> is a true diversity (=effective number of types), β<sub>At</sub> is not. Instead, β<sub>At</sub> quantifies the difference in true species diversity between the entire dataset and an average sampling unit. Using the generic term beta diversity to refer to both phenomena leads to confusion and should be avoided. It is also important to distinguish absolute effective species turnover from relative effective species turnover, which will be discussed next.
 </p>
 
 </section>
 <section class=article-section__content id=ss11>
 
 <h2 class="article-section__title section__title section1" id=ss11-title> 4. Effective species turnover expressed in multiples of mean species diversity of a single sampling unit (Whittaker's effective species turnover) β<sub>Mt−1</sub>=(γ−α<sub>t</sub>)/α<sub>t</sub></h2>
 
 <div class=paragraph-displayedItem-text-type>Regional‐to‐local diversity ratio β<sub>Mt</sub> obtains its minimum value of unity when all sampling units are compositionally identical and there is no species turnover among them. <span><a href=#b130 id=#b130R class="bibLink tab-link" data-tab=pane-pcw-references>Whittaker (1972)</a></span> developed from β<sub>Mt</sub> a species turnover measure with a minimum value of zero that quantifies <i>the number of complete effective species turnovers among compositional units in the dataset</i> and equals
 <div class=inline-equation id=di-fml-34><span class="inline-equation__construct figure"><img class=figure__image src=images/88.gif alt=image title=image></span><span class=inline-equation__label></span></div> 
 </div>
 
 <p>Rephrasing this equation gives β<sub>Mt−1</sub>=(γ−α<sub>t</sub>)/α<sub>t</sub>=β<sub>At</sub>/α<sub>t</sub>, which shows that β<sub>Mt−1</sub> simply relates absolute effective species turnover to mean species diversity of the sampling units. Equivalently, β<sub>At</sub>=α<sub>t</sub>β<sub>Mt−1</sub> (see also <span><a href=#b61 id=#b61R class="bibLink tab-link" data-tab=pane-pcw-references>Kiflawi and Spencer 2004</a></span>; note that β<sub>M</sub> refers to both β<sub>Mt</sub> and β<sub>Mt−1</sub> in their text). In other words, β<sub>Mt−1</sub> expresses <i>effective species turnover among the compositional units of the dataset in multiples</i><i>of their effective species richness</i>. Unlike β<sub>At</sub>, β<sub>Mt−1</sub> is independent of the species richness of the system, so the two are not monotonically related when the datasets to be compared differ in α<sub>t</sub>. Absolute and relative effective species turnover can therefore lead to different rankings of datasets.
 </p>
 
 <p>Given that both β<sub>Mt</sub> and β<sub>Mt−1</sub> were proposed by <span><a href=#b129%20#b130 id="#b129 #b130R" class="bibLink tab-link" data-tab=pane-pcw-references>Whittaker (1960, 1972)</a></span>, it is not surprising that both have been referred to as “Whittaker's beta diversity”. However, β<sub>Mt−1</sub> quantifies a specific kind of species turnover rather than a true diversity, so it is better referred to as <i>Whittaker's (effective) species turnover</i>. Nevertheless, β<sub>Mt−1</sub> has been used as a measure of “beta diversity” in several papers (<span><a href=#b134 id=#b134R class="bibLink tab-link" data-tab=pane-pcw-references>Wilson and Shmida 1984</a></span>, <span><a href=#b9 id=#b9R class="bibLink tab-link" data-tab=pane-pcw-references>Blackburn and Gaston 1996</a></span>, <span><a href=#b24 id=#b24R class="bibLink tab-link" data-tab=pane-pcw-references>Davis et al. 1999</a></span>, <span><a href=#b16 id=#b16R class="bibLink tab-link" data-tab=pane-pcw-references>Clarke and Lidgard 2000</a></span>, Ellingsen 2001, 2002, <span><a href=#b65 id=#b65R class="bibLink tab-link" data-tab=pane-pcw-references>Koleff and Gaston 2001</a></span>, <span><a href=#b117 id=#b117R class="bibLink tab-link" data-tab=pane-pcw-references>Sweeney and Cook 2001</a></span>, <span><a href=#b25 id=#b25R class="bibLink tab-link" data-tab=pane-pcw-references>Davis 2005</a></span>, <span><a href=#b80 id=#b80R class="bibLink tab-link" data-tab=pane-pcw-references>Mena and Vázquez‐Domínguez 2005</a></span>, <span><a href=#b81 id=#b81R class="bibLink tab-link" data-tab=pane-pcw-references>Munari and Mistri 2008</a></span>).
 </p>
 
 <div class=paragraph-displayedItem-text-type>When calculated for two sampling units using species richness (<i>q</i>=0) and equal sampling unit weights, Whittaker's species turnover is linearly (and negatively) related with the Sørensen index (<i>C</i><sub>S</sub>):
 <div class=inline-equation id=di-fml-35><span class="inline-equation__construct figure"><img class=figure__image src=images/89.gif alt=image title=image></span><span class=inline-equation__label></span></div> 
 </div>
 
 <p>Consequently, all studies that aimed to quantify “beta diversity” and chose the one‐complement of the Sørensen index to do so actually quantified Whittaker's species turnover (<span><a href=#b123 id=#b123R class="bibLink tab-link" data-tab=pane-pcw-references>Vazquez and Givnish 1998</a></span>, <span><a href=#b98 id=#b98R class="bibLink tab-link" data-tab=pane-pcw-references>Price et al. 1999</a></span>, <span><a href=#b19 id=#b19R class="bibLink tab-link" data-tab=pane-pcw-references>Condit et al. 2002</a></span>, <span><a href=#b133 id=#b133R class="bibLink tab-link" data-tab=pane-pcw-references>Wiersma and Urban 2005</a></span>, <span><a href=#b42 id=#b42R class="bibLink tab-link" data-tab=pane-pcw-references>Graham et al. 2006</a></span>, <span><a href=#b84 id=#b84R class="bibLink tab-link" data-tab=pane-pcw-references>Normand et al. 2006</a></span>, <span><a href=#b87 id=#b87R class="bibLink tab-link" data-tab=pane-pcw-references>Ødegaard 2006</a></span>, Novotny et al. 2007, <span><a href=#b110 id=#b110R class="bibLink tab-link" data-tab=pane-pcw-references>Ruokolainen et al. 2007</a></span>, <span><a href=#b50 id=#b50R class="bibLink tab-link" data-tab=pane-pcw-references>Hernández et al. 2008</a></span>, <span><a href=#b63 id=#b63R class="bibLink tab-link" data-tab=pane-pcw-references>Klop and Prins 2008</a></span>, <span><a href=#b73 id=#b73R class="bibLink tab-link" data-tab=pane-pcw-references>Linares‐Palomino and Kessler 2009</a></span>).
 </p>
 
 <div class=paragraph-displayedItem-text-type>If either all sampling unit weights are equal or <i>q</i>=1, Whittaker's effective species turnover obtains a maximum value of <i>N</i>−1 when the <i>N</i> sampling units share no species. Then β<sub>Mt−1</sub> can be ranged to the interval [0, 1]:
 <div class=inline-equation id=di-fml-36><span class="inline-equation__construct figure"><img class=figure__image src=images/90.gif alt=image title=image></span><span class=inline-equation__label></span></div> 
 </div>
 
 <div class=paragraph-displayedItem-text-type>Here “min” and “max” refer to the minimum and maximum values, respectively, that can be obtained given the number of sampling units <i>N</i>. Ranging Whittaker's effective species turnover and ranging regional‐to‐local diversity ratio lead to exactly the same result, because the two are linearly related. Since the ranged index is constrained to a fixed maximum value irrespective of <i>N</i>, it does not quantify Whittaker's effective species turnover even though it is based on it. Instead, 
<img alt="inline image" class=section_image src=images/91.gif> quantifies the amount by which β<sub>Mt−1</sub> exceeds its minimum possible value, expressed as a proportion of the total possible range of values (given <i>N</i> and the constraint of equal sampling unit weights when <i>q</i>≠1). For <i>q</i>=0, the ranged index has been proposed by <span><a href=#b48 id=#b48R class="bibLink tab-link" data-tab=pane-pcw-references>Harrison et al. (1992)</a></span>, who called it beta‐1. 
<img alt="inline image" class=section_image src=images/92.gif> can be interpreted in terms of compositional differentiation among the <i>N</i> sampling units, so its one‐complement is a measure of compositional overlap:
 <div class=inline-equation id=di-fml-39><span class="inline-equation__construct figure"><img class=figure__image src=images/93.gif alt=image title=image></span><span class=inline-equation__label></span></div> 
 </div>
 
 <p>When <i>q</i>=0 and <i>N</i>=2, <sup><i>q</i></sup><i>C</i><sub>S<i>N</i></sub> simplifies to the Sørensen index. In a dataset with more than two sampling units, pairwise Sørensen index values are not sensitive to whether or not some species are shared by three or more sampling units, but Whittaker's effective species turnover, its ranged version and <sup><i>q</i></sup><i>C</i><sub>S<i>N</i></sub> are. The Sørensen index has been generalised to <i>N</i> sampling units by <span><a href=#b28 id=#b28R class="bibLink tab-link" data-tab=pane-pcw-references>Diserud and Ødegaard (2007)</a></span>, who derived the ranging equation shown above for <sup>0</sup><i>C</i><sub>S<i>N</i></sub>. <span><a href=#b56 id=#b56R class="bibLink tab-link" data-tab=pane-pcw-references>Jost (2006)</a></span> derived a pairwise overlap measure for any value of <i>q</i> by ranging a monotonic transformation of <sup><i>q</i></sup>β<sub>Mt</sub>, and <span><a href=#b13 id=#b13R class="bibLink tab-link" data-tab=pane-pcw-references>Chao et al. (2008)</a></span> generalised it to <i>N</i>≥2 sampling units. This generalised index, <i>C</i><sub><i>qN</i></sub>, also yields the Sørensen index as a special case when <i>q</i>=0 and <i>N</i>=2. With other values of <i>q</i>, <i>C</i><sub><i>qN</i></sub> and <sup><i>q</i></sup><i>C</i><sub>S<i>N</i></sub> behave in different ways, as will be seen presently.
 </p>
 
 <p>Both Whittaker's species turnover <sup><i>q</i></sup>β<sub>Mt−1</sub> and its ranged version 
<img alt="inline image" class=section_image src=images/94.gif> depend on <i>N</i>, but in different ways (the same is true of <sup><i>q</i></sup>β<sub>Mt</sub> and 
<img alt="inline image" class=section_image src=images/95.gif>). This is easiest to visualise for <i>q</i>=0, equal sampling unit weights and constant <sup>0</sup><i>D</i><sub>γ<i>j</i></sub>. When <i>N</i> increases, <sup>0</sup>β<sub>Mt−1</sub> remains constant if the new sampling units do not introduce any new species to the dataset. Average compositional overlap among the sampling units then has to increase (unless they were identical to start with), and 
<img alt="inline image" class=section_image src=images/96.gif> has to decrease (the numerator remains constant while the denominator increases). Conversely, 
<img alt="inline image" class=section_image src=images/97.gif> remains constant if the newly added sampling units have the same average overlap with the original set of sampling units as these previously had among themselves. This is possible only if the new sampling units do introduce some new species to the dataset, in which case <sup>0</sup>β<sub>Mt−1</sub> necessarily increases. In general, <sup><i>q</i></sup>β<sub>Mt−1</sub> and 
<img alt="inline image" class=section_image src=images/98.gif> are linearly related when <i>N</i> is constant, but the correlation between them grows weaker with increasing variation in <i>N</i>. Consequently, the two do not measure the same phenomenon and can rank datasets differently.
 </p>
 
 </section>
 <section class=article-section__content id=ss12>
 
 <h2 class="article-section__title section__title section1" id=ss12-title> 5. Effective species turnover expressed as a proportion of total species diversity (proportional effective species turnover) β<sub>Pt</sub>=(γ−α<sub>t</sub>)/γ
 </h2>
 
 <p>To obtain Whittaker's effective species turnover, absolute effective species turnover β<sub>At</sub> is divided by the mean species diversity of the compositional units. Equally well, β<sub>At</sub> can be divided by total species diversity. Doing so leads to a new kind of relative effective species turnover measure, namely β<sub>Pt</sub>=(γ−α<sub>t</sub>)/γ. The equation can be rephrased 
<img alt="inline image" class=section_image src=images/99.gif>. This quantifies <i>the proportion of the effective species of the entire dataset that is not contained in a single compositional unit</i>. When <i>q</i>=0 and all sampling unit weights are equal, the term α<sub>t</sub>/γ=1/β<sub>Mt</sub> also indicates the proportion of sampling units in which the average species occurs (mean species frequency; <span><a href=#b130 id=#b130R class="bibLink tab-link" data-tab=pane-pcw-references>Whittaker 1972</a></span>, <span><a href=#b108 id=#b108R class="bibLink tab-link" data-tab=pane-pcw-references>Routledge 1977</a></span>, <span><a href=#b3 id=#b3R class="bibLink tab-link" data-tab=pane-pcw-references>Arita et al. 2008</a></span>).
 </p>
 
 <p>The relationship between the diversity components can also be written γ=α<sub>t</sub>/(1−β<sub>Pt</sub>) where the subscript “P” refers to a proportional partitioning of gamma diversity. From the above equations it follows that β<sub>Pt</sub>=1−1/β<sub>Mt</sub>=β<sub>Mt−1</sub>/β<sub>Mt</sub> or equivalently β<sub>Mt</sub>=1/(1−1/β<sub>Pt</sub>). Furthermore, β<sub>Pt</sub>=β<sub>At</sub>/γ or equivalently β<sub>At</sub>=γβ<sub>Pt</sub>. Proportional effective species turnover β<sub>Pt</sub> is independent of the species richness of the system, so it may give results that are in conflict with those obtained with absolute effective species turnover β<sub>At</sub> if the datasets to be compared differ in gamma diversity. In contrast, β<sub>Pt</sub>, β<sub>Mt−1</sub>, β<sub>Mt</sub> and β<sub>Md</sub> are monotonically related to each other. Consequently, if one wishes to rank datasets on the basis of their relative effective species turnover or multiplicative beta component, identical results will be obtained. However, the relationship between the proportional and multiplicative measures is not linear.
 </p>
 
 <div class=paragraph-displayedItem-text-type>When calculated for two sampling units using species richness (<i>q</i>=0) and equal weights, <sup><i>q</i></sup>β<sub>Pt</sub> is linearly (and negatively) related with the Jaccard index and inversely related with the Sørensen index:
 <div class=inline-equation id=di-fml-46><span class="inline-equation__construct figure"><img class=figure__image src=images/100.gif alt=image title=image></span><span class=inline-equation__label></span></div> 
 </div>
 
 <div class=paragraph-displayedItem-text-type>The Jaccard index can hence be expressed <i>C</i><sub>J</sub>=1−2(<sup>0</sup>β<sub>Pt</sub>). By allowing <i>q</i> to vary, the general similarity index <sup><i>q</i></sup><i>C</i><sub>J</sub>=1−2(<sup><i>q</i></sup>β<sub>Pt</sub>) is obtained, which can also be expressed <sup><i>q</i></sup><i>C</i><sub>J</sub>=2/<sup><i>q</i></sup>β<sub>Mt</sub>−1. The minimum value of β<sub>Pt</sub> is zero, which is obtained when all sampling units have the same species in constant proportional abundances. When all sampling units have the same weight or <i>q</i>=1, β<sub>Pt</sub> is constrained not to exceed 1−1/<i>N</i>, with this maximum being obtained when none of the <i>N</i> sampling units share any species. Then <sup><i>q</i></sup>β<sub>Pt</sub> can be ranged to the interval [0, 1] by the equation
 <div class=inline-equation id=di-fml-47><span class="inline-equation__construct figure"><img class=figure__image src=images/101.gif alt=image title=image></span><span class=inline-equation__label></span></div> 
 </div>
 
 <div class=paragraph-displayedItem-text-type>The one‐complement of this ranged index is 
 <div class=inline-equation id=di-fml-48><span class="inline-equation__construct figure"><img class=figure__image src=images/102.gif alt=image title=image></span><span class=inline-equation__label></span></div> 
 </div>
 
 <p>where <sup><i>q</i></sup><i>C</i><sub>J<i>N</i></sub> is the generalisation of the Jaccard index to any value of <i>q</i> and any number of sampling units (under the constraint of equal sampling unit weights when <i>q</i>≠1). <sup><i>q</i></sup><i>C</i><sub>J<i>N</i></sub> simplifies to the classic Jaccard index when <i>q</i>=0 and <i>N</i>=2. The index of biotal dispersity proposed by <span><a href=#b64 id=#b64R class="bibLink tab-link" data-tab=pane-pcw-references>Koch (1957)</a></span> equals <sup>0</sup><i>C</i><sub>J<i>N</i></sub>. The general ranging equation for <sup><i>q</i></sup><i>C</i><sub>J<i>N</i></sub> was derived by <span><a href=#b56 id=#b56R class="bibLink tab-link" data-tab=pane-pcw-references>Jost (2006)</a></span>, who showed that for <i>N</i>=2 and <i>q</i>=2 it equals the Morisita‐Horn index. The <i>C</i><sub><i>qN</i></sub> measure of <span><a href=#b13 id=#b13R class="bibLink tab-link" data-tab=pane-pcw-references>Chao et al. (2008)</a></span> also yields <sup>2</sup><i>C</i><sub>J<i>N</i></sub> at <i>q</i>=2, but as we saw in Section 4, at <i>q</i>=0 it equals <sup>0</sup><i>C</i><sub>S<i>N</i></sub> instead. As always, ranging changes the numerical values such that the ecological interpretation of the ranged index (<sup><i>q</i></sup><i>C</i><sub>J<i>N</i></sub>) is not the same as that of the original measure (1−<sup><i>q</i></sup>β<sub>Pt</sub> or 1/<sup><i>q</i></sup>β<sub>Mt</sub>).
 </p>
 
 <p>Because example 1 of <span><a href=#b129 id=#b129R class="bibLink tab-link" data-tab=pane-pcw-references>Whittaker (1960)</a></span> applied the Jaccard index, it was in fact based on ranged values of proportional species turnover rather than on true beta diversity. Many other studies have followed suit, as the Jaccard index (or its one‐complement, also known as the complementarity of <span><a href=#b18 id=#b18R class="bibLink tab-link" data-tab=pane-pcw-references>Colwell and Coddington 1994</a></span>) has been a popular measure of “beta diversity” (e.g., <span><a href=#b111 id=#b111R class="bibLink tab-link" data-tab=pane-pcw-references>Scheiner and Rey‐Benayas 1994</a></span>, <span><a href=#b100 id=#b100R class="bibLink tab-link" data-tab=pane-pcw-references>Rey Benayas 1995</a></span>, <span><a href=#b97 id=#b97R class="bibLink tab-link" data-tab=pane-pcw-references>Porembski et al. 1996</a></span>, <span><a href=#b46 id=#b46R class="bibLink tab-link" data-tab=pane-pcw-references>Harrison 1997</a></span>, <span><a href=#b16 id=#b16R class="bibLink tab-link" data-tab=pane-pcw-references>Clarke and Lidgard 2000</a></span>, <span><a href=#b54 id=#b54R class="bibLink tab-link" data-tab=pane-pcw-references>Izsak and Price 2001</a></span>, <span><a href=#b88 id=#b88R class="bibLink tab-link" data-tab=pane-pcw-references>Pärtel et al. 2001</a></span>, <span><a href=#b5 id=#b5R class="bibLink tab-link" data-tab=pane-pcw-references>Balvanera et al. 2002</a></span>, <span><a href=#b121 id=#b121R class="bibLink tab-link" data-tab=pane-pcw-references>Tuomisto et al. 2003</a></span>, <span><a href=#b120 id=#b120R class="bibLink tab-link" data-tab=pane-pcw-references>Tuomisto and Ruokolainen 2005</a></span>, <span><a href=#b15 id=#b15R class="bibLink tab-link" data-tab=pane-pcw-references>Chust et al. 2006</a></span>, <span><a href=#b35 id=#b35R class="bibLink tab-link" data-tab=pane-pcw-references>Freestone and Inouye 2006</a></span>, <span><a href=#b49 id=#b49R class="bibLink tab-link" data-tab=pane-pcw-references>Harrison et al. 2006</a></span>, <span><a href=#b87 id=#b87R class="bibLink tab-link" data-tab=pane-pcw-references>Ødegaard 2006</a></span>, <span><a href=#b33 id=#b33R class="bibLink tab-link" data-tab=pane-pcw-references>Flores‐Palacios and García‐Franco 2008</a></span>).
 </p>
 
 <p> <span><a href=#b107 id=#b107R class="bibLink tab-link" data-tab=pane-pcw-references>Roschewitz et al. (2005)</a></span> presented <sup>0</sup>β<sub>Pt</sub> values under the name “relative beta diversity”, which they derived using the additive partitioning of total species richness as a starting point. Several other authors have also noticed that dividing “additive beta diversity”<sup>0</sup>β<sub>At</sub> by gamma diversity makes the beta component independent of alpha diversity, and consequently presented proportional species turnover values as “additive beta diversity” values (<span><a href=#b22 id=#b22R class="bibLink tab-link" data-tab=pane-pcw-references>Crist and Veech 2006</a></span>, <span><a href=#b124 id=#b124R class="bibLink tab-link" data-tab=pane-pcw-references>Veech and Crist 2007</a></span>, <span><a href=#b52 id=#b52R class="bibLink tab-link" data-tab=pane-pcw-references>Hof et al. 2008</a></span>, <span><a href=#b104 id=#b104R class="bibLink tab-link" data-tab=pane-pcw-references>Ricotta 2008</a></span>).
 </p>
 
 </section>
 <section class=article-section__content id=ss13>
 
 <h2 class="article-section__title section__title section1" id=ss13-title> 6. Beta Shannon entropy <span><img alt="inline image" class=section_image src=images/103.gif></span> and regional Shannon entropy excess <span><img alt="inline image" class=section_image src=images/104.gif></span></h2>
 
 <p>As we saw in Section 3 (<i>Regional diversity excess</i>), multiplicative partitioning of true diversities of order 1 (<sup>1</sup><i>D</i><sub>γ</sub>=<sup>1</sup><i>D</i><sub>α</sub><sup>1</sup><i>D</i><sub>β</sub>) corresponds to additive partitioning of Shannon entropies 
<img alt="inline image" class=section_image src=images/105.gif>. Using the beta component of the Shannon entropy without converting it to its numbers equivalent <sup>1</sup><i>D</i><sub>β</sub> leads to a new definition of “beta diversity”, namely 
<img alt="inline image" class=section_image src=images/106.gif>. Here 
<img alt="inline image" class=section_image src=images/107.gif> is Shannon entropy related to the γ‐classification, 
<img alt="inline image" class=section_image src=images/108.gif> is Shannon entropy related to the γ‐classification that is conditional on the β‐classification, and 
<img alt="inline image" class=section_image src=images/109.gif> is Shannon entropy related to the β‐classification. In other words, 
<img alt="inline image" class=section_image src=images/110.gif> quantifes the mean uncertainty regarding which effective species is picked when one individual (or other unit of abundance) is taken at random from the entire dataset. 
<img alt="inline image" class=section_image src=images/111.gif> quantifes the mean uncertainty regarding which effective species is picked when one individual (or other unit of abundance) is taken at random from the entire dataset, but the uncertainty is quantified only within the limits of the compositional unit that contains the effective species to which the chosen individual belongs. Beta Shannon entropy 
<img alt="inline image" class=section_image src=images/112.gif> quantifies <i>the uncertainty regarding which compositional unit is picked when one effective species is taken at random from the entire dataset</i>.
 </p>
 
 <p>The Shannon entropy corresponding to mean species diversity in the sampling units <img alt="inline image" class=section_image src=images/113.gif> quantifies the mean uncertainty regarding which effective species is picked when one individual (or other unit of abundance) is taken at random from within a randomly preselected sampling unit. If 
<img alt="inline image" class=section_image src=images/114.gif> is used instead of 
<img alt="inline image" class=section_image src=images/115.gif> when partitioning gamma Shannon entropy, the beta component becomes regional Shannon entropy excess 
<img alt="inline image" class=section_image src=images/116.gif>. This quantifies <i>the amount by which the Shannon entropy of the regional dataset (in relation to the γ‐classification) exceeds that of a single sampling unit of arithmetic mean entropy</i>. Although the interpretations of 
<img alt="inline image" class=section_image src=images/117.gif> and 
<img alt="inline image" class=section_image src=images/118.gif> are different, their numerical values are the same.
 </p>
 
 <p>The minimum value of both <img alt="inline image" class=section_image src=images/119.gif> and 
<img alt="inline image" class=section_image src=images/120.gif> is zero, which is obtained when there is no variation in species proportional abundances among sampling units. The maximum value is log(<i>N</i>) for 
<img alt="inline image" class=section_image src=images/121.gif> and log(<i>N</i> CU) for 
<img alt="inline image" class=section_image src=images/122.gif>, which are obtained when none of the <i>N</i> sampling units share any species. Depending on which base is chosen for the logarithm, 
<img alt="inline image" class=section_image src=images/123.gif> and 
<img alt="inline image" class=section_image src=images/124.gif> are measured in different units, such as bits, nats or decits (<span><a href=#b113 id=#b113R class="bibLink tab-link" data-tab=pane-pcw-references>Shannon 1948</a></span>; Proof 1).
 </p>
 
 <p>Although Shannon entropy related to the β‐classification and regional Shannon entropy excess are monotonic transformations of true beta diversity <sup>1</sup>β<sub>Md</sub>, they do not equal true beta diversity. The relationship is strongly curvilinear, because the logarithm of a variable increases much more slowly than the variable itself. This can easily lead to errors of interpretation if entropy is confused with true diversity (see <span><a href=#b56%20#b57 id="#b56 #b57R" class="bibLink tab-link" data-tab=pane-pcw-references>Jost 2006, 2007</a></span> for further discussion). Nevertheless, it has been rather common in the ecological literature to use 
<img alt="inline image" class=section_image src=images/125.gif> or 
<img alt="inline image" class=section_image src=images/126.gif> as a measure of “beta diversity” (<span><a href=#b71 id=#b71R class="bibLink tab-link" data-tab=pane-pcw-references>Levins 1968</a></span>, Allan 1975a,b, <span><a href=#b53 id=#b53R class="bibLink tab-link" data-tab=pane-pcw-references>Holland and Jain 1981</a></span>, <span><a href=#b6 id=#b6R class="bibLink tab-link" data-tab=pane-pcw-references>Barker et al. 1983</a></span>, <span><a href=#b40 id=#b40R class="bibLink tab-link" data-tab=pane-pcw-references>Gimaret‐Carpentier et al. 1998</a></span>, <span><a href=#b127 id=#b127R class="bibLink tab-link" data-tab=pane-pcw-references>Wagner et al. 2000</a></span>, <span><a href=#b23 id=#b23R class="bibLink tab-link" data-tab=pane-pcw-references>Crist et al. 2003</a></span>, <span><a href=#b39 id=#b39R class="bibLink tab-link" data-tab=pane-pcw-references>Gering et al. 2003</a></span>, <span><a href=#b116 id=#b116R class="bibLink tab-link" data-tab=pane-pcw-references>Summerville et al. 2003</a></span>, <span><a href=#b20 id=#b20R class="bibLink tab-link" data-tab=pane-pcw-references>Couteron and Pélissier 2004</a></span>, <span><a href=#b92 id=#b92R class="bibLink tab-link" data-tab=pane-pcw-references>Pélissier and Couteron 2007</a></span>, <span><a href=#b7 id=#b7R class="bibLink tab-link" data-tab=pane-pcw-references>Basset et al. 2008</a></span>).
 </p>
 
 <div class=paragraph-displayedItem-text-type> <span><a href=#b54 id=#b54R class="bibLink tab-link" data-tab=pane-pcw-references>Horn (1966)</a></span> developed the following indices on the basis of Shannon entropies:
 <div class=inline-equation id=di-fml-73><span class="inline-equation__construct figure"><img class=figure__image src=images/127.gif alt=image title=image></span><span class=inline-equation__label></span></div> 
 </div>
 
 <div class=paragraph-displayedItem-text-type> <i>R</i>
 <sub>
 <i>o</i>
 </sub> stands for an overlap index and <i>R</i><sub><i>h</i></sub> for the corresponding heterogeneity index. In the annotation of the present paper, Horn's heterogeneity index can be rewritten
 <div class=inline-equation id=di-fml-74><span class="inline-equation__construct figure"><img class=figure__image src=images/128.gif alt=image title=image></span><span class=inline-equation__label></span></div> 
 </div>
 
 <div class=paragraph-displayedItem-text-type>Here the numerator equals the regional Shannon entropy excess that was actually observed in the region consisting of the two sampling units, and the denominator equals the regional Shannon entropy excess that would have been obtained if the two sampling units had shared no species. In other words, the Horn index of heterogeneity ranges Shannon entropy excess to the interval [0, 1], and therefore expresses the observed Shannon entropy excess as a proportion of its theoretical maximum value (given the observed mean species diversity in the sampling units and the number of sampling units observed). Although <span><a href=#b54 id=#b54R class="bibLink tab-link" data-tab=pane-pcw-references>Horn (1966)</a></span> derived the index for <i>N</i>=2, it can easily be applied to datasets with <i>N</i>≥2. The generalisations of both Horn indices to <i>N</i> sampling units are simply
 <div class=inline-equation id=di-fml-75><span class="inline-equation__construct figure"><img class=figure__image src=images/129.gif alt=image title=image></span><span class=inline-equation__label></span></div> 
 </div>
 
 <p>The <i>C</i><sub><i>qN</i></sub> measure of <span><a href=#b13 id=#b13R class="bibLink tab-link" data-tab=pane-pcw-references>Chao et al. (2008)</a></span> yields <i>R</i><sub><i>oN</i></sub> in the limit as <i>q</i> approaches unity (<span><a href=#b56 id=#b56R class="bibLink tab-link" data-tab=pane-pcw-references>Jost 2006</a></span>), but as we saw in Sections 4 and 5, <i>C</i><sub><i>qN</i></sub> corresponds to other definitions of “beta diversity” with other values of <i>q</i>.
 </p>
 
 <p>It is obvious from the above equation that <i>R</i><sub><i>hN</i></sub> does not equal <sup>1</sup>β<sub>Mt</sub> even though it is derivable from <sup>1</sup>β<sub>Mt</sub>. As mentioned in Section 1 (<i>Heterogeneity measures</i>), <span><a href=#b77 id=#b77R class="bibLink tab-link" data-tab=pane-pcw-references>MacArthur (1965)</a></span> used 
<img alt="inline image" class=section_image src=images/130.gif> as a measure of the faunal difference between two censuses. The relationship with MacArthur's index was pointed out by <span><a href=#b54 id=#b54R class="bibLink tab-link" data-tab=pane-pcw-references>Horn (1966)</a></span> when discussing the new <i>R</i><sub><i>h</i></sub> measure.
 </p>
 
 </section>
 <section class=article-section__content id=ss14>
 
 <h2 class="article-section__title section__title section1" id=ss14-title> 7. Regional variance excess<span><img alt="inline image" class=section_image src=images/131.gif></span></h2>
 
 <div class=paragraph-displayedItem-text-type>Let us now apply the additive diversity index partitioning to the Gini‐Simpson index 1−<sup>2</sup>λ:
 <div class=inline-equation id=di-fml-78><span class="inline-equation__construct figure"><img class=figure__image src=images/132.gif alt=image title=image></span><span class=inline-equation__label></span></div> 
 </div>
 
 <div class=paragraph-displayedItem-text-type>where <img alt="inline image" class=section_image src=images/133.gif> equals the weighted arithmetic mean of the Gini‐Simpson index values quantified for each sampling unit separately. Replacing β′ with the more explicit annotation 
<img alt="inline image" class=section_image src=images/134.gif> and inserting <sup>2</sup>λ=1/<sup>2</sup><i>D</i>, the above equation can be re‐expressed
 <div class=inline-equation id=di-fml-81><span class="inline-equation__construct figure"><img class=figure__image src=images/135.gif alt=image title=image></span><span class=inline-equation__label></span></div> 
 </div>
 
 <div class=paragraph-displayedItem-text-type>where <img alt="inline image" class=section_image src=images/136.gif> is the weighted harmonic mean of the 
<img alt="inline image" class=section_image src=images/137.gif> values. The equation can also be written
 <div class=inline-equation id=di-fml-84><span class="inline-equation__construct figure"><img class=figure__image src=images/138.gif alt=image title=image></span><span class=inline-equation__label></span></div> 
 </div>
 
 <p>When an individual is repeatedly picked at random from the sample, the Gini‐Simpson index equals the variance in observed species identity, i.e. <img alt="inline image" class=section_image src=images/139.gif> (<span><a href=#b68 id=#b68R class="bibLink tab-link" data-tab=pane-pcw-references>Lande 1996</a></span>, <span><a href=#b20 id=#b20R class="bibLink tab-link" data-tab=pane-pcw-references>Couteron and Pélissier 2004</a></span>). 
<img alt="inline image" class=section_image src=images/140.gif> can be called <i>regional variance excess</i> because it quantifies <i>the amount by which the variance in species identity of a randomly picked individual as calculated in the regional dataset exceeds that within a single sampling unit of arithmetic mean variance</i>.
 </p>
 
 <p>From the formulation <img alt="inline image" class=section_image src=images/141.gif> it can be seen that the Gini‐Simpson index also quantifies the probability that two individuals drawn at random (with replacement) from a sample are different species. Therefore, regional variance excess can also be interpreted as <i>the difference in the probabilities of drawing two individuals of different species in the entire dataset vs within a sampling unit of mean probability</i>. The generalisation of the Gini‐Simpson index to other values of <i>q</i>&gt;1 is 1−<sup><i>q</i></sup>λ, which quantifies the probability that <i>q</i> individuals drawn at random (with replacement) from a sample represent at least two different species. Consequently, inserting 1−<sup><i>q</i></sup>λ into the equation β′=γ′−α′ would yield a different definition of the beta component for each different value of <i>q</i>. To my knowledge, values of <i>q</i> other than 2 have not been used in connection with “additive beta diversity”.
 </p>
 
 <p>Regional variance excess is related to regional diversity excess by <img alt="inline image" class=section_image src=images/142.gif>, to Whittaker's effective species turnover by 
<img alt="inline image" class=section_image src=images/143.gif> and to proportional effective species turnover by 
<img alt="inline image" class=section_image src=images/144.gif>. These equations show that regional variance excess quantifies <i>the amount of relative effective species turnover per effective species in the dataset</i>. Consequently, the amount of regional variance excess is mostly determined by the species diversity of the system: when α<sub>t</sub> and γ increase, 
<img alt="inline image" class=section_image src=images/145.gif> necessarily approaches zero (see also <span><a href=#b56%20#b57 id="#b56 #b57R" class="bibLink tab-link" data-tab=pane-pcw-references>Jost 2006, 2007</a></span>). If all sampling units have equal weights, regional variance excess is constrained by an upper limit of (<i>N</i>−1)/<i>N</i>α<sub>t</sub>, which is obtained when none of the <i>N</i> sampling units share any species. Its absolute minimum value of zero is obtained when there is no variation in species proportional abundances among sampling units.
 </p>
 
 <p>The dependence of regional variance excess on species diversity can be verified by the thought experiment of duplicating each species in the system: α and γ will double, but <img alt="inline image" class=section_image src=images/146.gif> will decrease by one‐half. This is in contrast to the situation with absolute effective species turnover, which increases with increasing alpha diversity, and to that with the multiplicative beta components and the relative species turnover measures, which vary independently of alpha diversity. Consequently, if one wishes to rank regional datasets on the basis of their “beta diversity”, regional variance excess will generally give results that are in conflict with results based on any of the other definitions. Interpreting trends in regional variance excess in terms of species turnover or compositional differences among sampling units can therefore be very misleading.
 </p>
 
 <p>When quantified for two equally‐weighted sampling units, regional variance excess equals the squared Euclidean distance between them as calculated using proportional abundance data (<span><a href=#b118 id=#b118R class="bibLink tab-link" data-tab=pane-pcw-references>ter Braak 1983</a></span>). The <i>D</i><sub>ST</sub> index, which is commonly used in genetical studies (<span><a href=#b83 id=#b83R class="bibLink tab-link" data-tab=pane-pcw-references>Nei 1973</a></span>, <span><a href=#b58 id=#b58R class="bibLink tab-link" data-tab=pane-pcw-references>Jost 2008</a></span>), is mathematically the same as regional variance excess, although <i>D</i><sub>ST</sub> uses a γ‐classification based on alleles rather than species.
 </p>
 
 <p>Just like regional diversity excess <sup>0</sup>β<sub>At</sub> and regional entropy excess 
<img alt="inline image" class=section_image src=images/147.gif>, regional variance excess 
<img alt="inline image" class=section_image src=images/148.gif> has been used as a measure of “additive beta diversity” with the justification that measuring α′, β′ and γ′ in the same units (in this case, 1/sp<sub>E</sub>) is an advantage (<span><a href=#b68 id=#b68R class="bibLink tab-link" data-tab=pane-pcw-references>Lande 1996</a></span>, <span><a href=#b40 id=#b40R class="bibLink tab-link" data-tab=pane-pcw-references>Gimaret‐Carpentier et al. 1998</a></span>, <span><a href=#b127 id=#b127R class="bibLink tab-link" data-tab=pane-pcw-references>Wagner et al. 2000</a></span>, <span><a href=#b34 id=#b34R class="bibLink tab-link" data-tab=pane-pcw-references>Fournier and Loreau 2001</a></span>, <span><a href=#b23 id=#b23R class="bibLink tab-link" data-tab=pane-pcw-references>Crist et al. 2003</a></span>, <span><a href=#b39 id=#b39R class="bibLink tab-link" data-tab=pane-pcw-references>Gering et al. 2003</a></span>, <span><a href=#b93 id=#b93R class="bibLink tab-link" data-tab=pane-pcw-references>Pélissier et al. 2003</a></span>, <span><a href=#b116 id=#b116R class="bibLink tab-link" data-tab=pane-pcw-references>Summerville et al. 2003</a></span>, <span><a href=#b20 id=#b20R class="bibLink tab-link" data-tab=pane-pcw-references>Couteron and Pélissier 2004</a></span>, <span><a href=#b25 id=#b25R class="bibLink tab-link" data-tab=pane-pcw-references>Davis 2005</a></span>, <span><a href=#b92 id=#b92R class="bibLink tab-link" data-tab=pane-pcw-references>Pélissier and Couteron 2007</a></span>). The interpretational problems that follow have been discussed by <span><a href=#b56%20#b57 id="#b56 #b57R" class="bibLink tab-link" data-tab=pane-pcw-references>Jost (2006, 2007)</a></span>.
 </p>
 
 </section>
 <section class=article-section__content id=ss15>
 
 <h2 class="article-section__title section__title section1" id=ss15-title> ∞. How (not) to generate more definitions of “beta diversity”</h2>
 
 <p>Exploring the basic definitions of “beta diversity” has revealed that new definitions can be created by at least four strategies: 1) by defining “diversity” in a new way, 2) by defining the alpha component in a new way, 3) by defining the relationship between gamma and alpha in a new way, and 4) by forgetting about alpha and gamma and defining beta by some other means. In strategy 1, numerous different measures can be used to quantify “diversity”. For example:</p>
 
 <div class=paragraph-element>
 
 <ul class="rlist hanging">
 
 <li><span class=number>A </span>
 
 
 <p>true diversity <sup><i>q</i></sup><i>D</i></p>
 
 </li>
 
 <li><span class=number>B </span>
 
 
 <p>Shannon entropy <i>H′</i>=log(<sup>1</sup><i>D</i>)
 </p>
 
 </li>
 
 <li><span class=number>C </span>
 
 
 <p>Gini‐Simpson index 1−<sup>2</sup>λ=1−1/<sup>2</sup><i>D</i></p>
 
 </li>
 
 <li><span class=number>D </span>
 
 
 <p>Fisher's alphaE)–Z)raw value of some other diversity index.</p>
 
 </li>
 
 </ul>
 
 </div>
 
 <p>Obviously, it is important to make an explicit choice among the definitions of “diversity”, because each of them quantifies a fundamentally different phenomenon. Option A provides the most suitable diversity measure for a general standard for three main reasons. Firstly, <sup><i>q</i></sup><i>D</i> is the only measure of those listed that has the doubling property: when the intuitive diversity of a system doubles, the value of <sup><i>q</i></sup><i>D</i> also doubles (<span><a href=#b51 id=#b51R class="bibLink tab-link" data-tab=pane-pcw-references>Hill 1973</a></span>, <span><a href=#b57 id=#b57R class="bibLink tab-link" data-tab=pane-pcw-references>Jost 2007</a></span>). Secondly, the interpretation of <sup><i>q</i></sup><i>D</i> is logical, for example “effective number of species” when quantifying species diversity and “effective number of habitats” when quantifying habitat diversity. Thirdly, the sensitivity of <sup><i>q</i></sup><i>D</i> to rare vs abundant species (or any other types of interest) can be adjusted by varying the value of <i>q</i>, without changing the interpretation of the measure or its measurement unit. The options corresponding to different values of <i>q</i> can be annotated A<sub><i>q</i></sub>. For example, in option A<sub>0</sub> no importance is given to species abundances whereas in option A<sub>1</sub> the diversity value is affected by each species in proportion to its proportional abundance, but what is being measured in both cases is the number of effective species. In contrast, each of the other “diversity” measures corresponds to a different concept of “diversity” and will be associated with a different measurement unit. In the case of option B, the variants corresponding to different logarithm bases <i>b</i> can be annotated B<sub><i>b</i></sub>.
 </p>
 
 <p>In strategy 2, at least three different definitions of the alpha component of total “diversity” have been used, which can be derived from:</p>
 
 <div class=paragraph-element>
 
 <ul class="rlist hanging">
 
 <li><span class=number>A </span>
 
 
 <p>diversity per compositional unit α<sub>d</sub>, e.g. <sup><i>q</i></sup><i>D</i><sub>α</sub> or <img alt="inline image" class=section_image src=images/149.gif></p>
 
 </li>
 
 <li><span class=number>B </span>
 
 
 <p>mean diversity in the original sampling units α<sub>t</sub>, e.g. <img alt="inline image" class=section_image src=images/150.gif> or <img alt="inline image" class=section_image src=images/151.gif></p>
 
 </li>
 
 <li><span class=number>C </span>
 
 
 <p>diversity of species‐sampling unit combinations divided by diversity of sampling units α<sub>R</sub>, e.g. <sup><i>q</i></sup><i>D</i><sub>‘γω’/ω</sub></p>
 
 </li>
 
 </ul>
 
 </div>
 
 <p>The difference between options A and B is in the ecological interpretation and measurement units, as their numerical values are the same. In contrast, option C leads to numerically different results except in some special cases when it converges on option B.</p>
 
 <p>In strategy 3, at least four equations have been used to calculate the beta component:</p>
 
 <div class=paragraph-element>
 
 <ul class="rlist hanging">
 
 <li><span class=number>A </span>
 
 
 <p>β=γ/α</p>
 
 </li>
 
 <li><span class=number>B </span>
 
 
 <p>β=γ−α</p>
 
 </li>
 
 <li><span class=number>C </span>
 
 
 <p>β=(γ−α)/α</p>
 
 </li>
 
 <li><span class=number>D </span>
 
 
 <p>β=(γ−α)/γ</p>
 
 </li>
 
 </ul>
 
 </div>
 
 <p>Each of these equations leads to measuring something different, and therefore to defining a conceptually different beta component. Option A provides the most suitable equation for a general standard because it is the simplest way of producing independent alpha and beta components. Specific choices made with respect to strategies 1 and 2 limit the choices available for strategy 3. Let's say that option A was chosen in strategy 1. If we then choose option B in strategy 2, the measurement unit in both gamma and alpha diversity is sp<sub>E</sub> and hence any of the strategy 3 options can be chosen. However, if we instead choose option A in strategy 2, then alpha diversity is measured using the unit sp<sub>E</sub>/CU, which is not commensurate with sp<sub>E</sub>. Thereby α cannot be subtracted from γ, which forces us to choose option A in strategy 3. The resulting beta component corresponds to true beta diversity <sup><i>q</i></sup>β<sub>Md</sub>, which can also be expressed as beta diversity based on strategy 1A<sub><i>q</i></sub>, 2A, 3A, or strategy β(A<sub><i>q</i></sub>AA), for short (<a href=#f1 class=scrollableLink>Fig. 1</a>).
 </p>
 
 <section class=article-section__inline-figure>
 <figure class=figure id=f1><a target=_blank href=https://onlinelibrary.wiley.com/cms/asset/b94e7c18-db30-4a6a-bf79-11cbd9a06e66/ecog_5880_f1.gif>
 <picture>
 <img class=figure__image src=images/152.gif data-lg-src=/cms/asset/b94e7c18-db30-4a6a-bf79-11cbd9a06e66/ecog_5880_f1.gif alt=image title=image srcset sizes></picture></a><figcaption class=figure__caption>
 <div class=figure__caption__header><strong class=figure__title>Figure 1</strong><div class=figure-extra><a href=# class=open-figure-link>Open in figure viewer</a><a href="https://onlinelibrary.wiley.com/action/downloadFigures?id=f1&amp;doi=10.1111%2Fj.1600-0587.2009.05880.x" class=ppt-figure-link><i aria-hidden=true class=icon-Icon_Download></i><span>PowerPoint</span></a></div>
 </div>
 <div class="figure__caption figure__caption-text">
 <p>The beta component of “diversity” is defined by choosing 1) a diversity measure, 2) a definition for the alpha component and 3) a relationship between the alpha and gamma components. When option A is chosen in each case, true alpha, beta and gamma diversities are obtained; these choices are connected with thick black lines. The choices made by <span class="figureLink bibLink tab-link"><a href=#b129%20#b130 id="#b129 #b130R" class="bibLink tab-link" data-tab=pane-pcw-references>Whittaker (1960, 1972)</a></span> when discussing “beta diversity” are connected with thin black lines; of these, the ones with stippled lines were rejected by <span class="figureLink bibLink tab-link"><a href=#b130 id=#b130R class="bibLink tab-link" data-tab=pane-pcw-references>Whittaker (1972)</a></span>. The choices connected by gray lines correspond to definitions of “beta diversity” proposed by <span class="figureLink bibLink tab-link"><a href=#b68 id=#b68R class="bibLink tab-link" data-tab=pane-pcw-references>Lande (1996)</a></span>. For explanation of the symbols, see <a href=#t2 class=scrollableLink>Table 2</a>.</p>
 
 </div>
 </figcaption>
 </figure>
 </section>
 
 <p>The fourth strategy boils down to inventing a new dissimilarity index and calling whatever it quantifies “beta diversity”. This strategy has been quite popular ever since <span><a href=#b129%20#b130 id="#b129 #b130R" class="bibLink tab-link" data-tab=pane-pcw-references>Whittaker (1960, 1972)</a></span> used the Jaccard and Sørensen indices and the percentage similarity to quantify “beta diversity”. The review of <span><a href=#b66 id=#b66R class="bibLink tab-link" data-tab=pane-pcw-references>Koleff et al. (2003a)</a></span> listed 24 different indices that have been used to quantify “beta diversity” for presence‐absence data, and <span><a href=#b105 id=#b105R class="bibLink tab-link" data-tab=pane-pcw-references>Ricotta and Burrascano (2009)</a></span> suggested that any meaningful dissimilarity index can be used for the purpose. The problem with this approach is that it leads to a proliferation of conceptually different definitions of “beta diversity”. The dissimilarity indices that were mentioned in Sections 1 through 7, above, can be derived from alpha and gamma diversity. Therefore, each of them is compatible with one definition of “beta diversity”, but a different definition in each case. Many other popular dissimilarity indices cannot be expressed as a function of γ and α at all. This seems to be the case with the chi‐square metric, chord distance and Hellinger distance. The use of these indices may still be justified when addressing ecological questions related to compositional differences, but the results cannot be interpreted in terms of the alpha, beta and gamma components of diversity.
 </p>
 
 <p>The conceptual interpretation of some dissimilarity measures changes depending on which kind of abundance data are used. This is especially obvious in the case of the squared Euclidean distance: using presence‐absence data leads to quantifying regional diversity excess <sup>0</sup>β<sub>At</sub>, using proportional abundance data leads to quantifying regional variance excess 
<img alt="inline image" class=section_image src=images/153.gif> and using absolute abundance data leads to a dissimilarity measure that cannot be expressed as a function of α and γ. A number of indices converge on the Sørensen index (or its one‐complement) when applied to presence‐absence data, and then quantify Whittaker's species turnover <sup>0</sup>β<sub>Mt−1</sub>. This is true of the percentage similarity used by <span><a href=#b129 id=#b129R class="bibLink tab-link" data-tab=pane-pcw-references>Whittaker (1960)</a></span> in his example 2, and the measures known as the Renkonen, Steinhaus, Odum and Bray‐Curtis indices. However, there is no obvious way to express these indices in terms of α and γ when proportional or absolute abundance data are used.
 </p>
 
 <p>A few examples of studies that have (unintentionally) used strategies 1 through 4 to create new definitions of “beta diversity” may be mentioned. <span><a href=#b129%20#b130 id="#b129 #b130R" class="bibLink tab-link" data-tab=pane-pcw-references>Whittaker (1960, 1972)</a></span> himself used most of them. One of the “beta diversity” measures that he proposed is regional‐to‐local diversity ratio β<sub>Mt</sub>, which corresponds to strategy β(ABA) (<a href=#f1 class=scrollableLink>Fig. 1</a>). Whittaker used β<sub>Mt</sub> with both <i>q</i>=0 and <i>q</i>=1, which can be expressed β(A<sub>0</sub>BA) and β(A<sub>1</sub>BA), respectively. Whittaker's species turnover β<sub>Mt−1</sub> makes the same choices in strategies 1 and 2 but partitions gamma using a different equation, and corresponds to strategy β(A<sub>0</sub>BC) when applied to presence‐absence data. Whittaker's use of the Jaccard index corresponds to using the ranged version of β(A<sub>0</sub>BD). Whittaker's discarded definitions were based on a ratio of Fisher's alphas or Shannon entropies, and correspond to strategies β(DBA) and β(B<sub><i>e</i></sub>BA), respectively. Whittaker's use of percentage similarity applied strategy 4, as this measure is not a function of α and γ. Whittaker's use of the half‐change unit introduced a completely new approach, namely quantifying the amount of change in the chosen kind of beta component in relation to an external gradient; this and other derived approaches will be considered in the second part of the present review (<span><a href=#b119 id=#b119R class="bibLink tab-link" data-tab=pane-pcw-references>Tuomisto 2010</a></span>).
 </p>
 
 <div class=paragraph-displayedItem-text-type>Although <span><a href=#b51 id=#b51R class="bibLink tab-link" data-tab=pane-pcw-references>Hill (1973)</a></span> made it obvious that he preferred numbers equivalents of diversity indices as the measure of diversity, he tentatively conceded that Rényi entropies might be used as well. The Rényi entropy of order <i>q</i> equals (<span><a href=#b99 id=#b99R class="bibLink tab-link" data-tab=pane-pcw-references>Rényi 1960</a></span>, <span><a href=#b51 id=#b51R class="bibLink tab-link" data-tab=pane-pcw-references>Hill 1973</a></span>)
 <div class=inline-equation id=di-fml-99><span class="inline-equation__construct figure"><img class=figure__image src=images/154.gif alt=image title=image></span><span class=inline-equation__label></span></div> 
 </div>
 
 <p>and is therefore a generalisation of the Shannon entropy log(<sup>1</sup><i>D</i>) to values of <i>q</i> other than unity. Measuring “diversity” with the Rényi entropy therefore logically leads to a whole family of “beta diversity” concepts, namely β(B<sub><i>q,b</i></sub>++) where the expanded option B of strategy 1 (with subscripts to specify the value of <i>q</i> and the log base <i>b</i>) is combined with each of the options in strategies 2 and 3 in turn. It is clearly not desirable to call all these measures “beta diversity”, because none of them quantifies the same phenomenon as does true beta diversity, strategy β(A<sub><i>q</i></sub>AA).
 </p>
 
 <p>When <span><a href=#b68 id=#b68R class="bibLink tab-link" data-tab=pane-pcw-references>Lande (1996)</a></span> proposed defining “beta diversity” on the basis of additive partitioning of concave diversity indices (β′=γ′−α′) instead of multiplicative partitioning of true diversities (β=γ/α), in effect he introduced a host of new definitions of “beta diversity”. He discussed three of these in detail, namely the ones based on species richness, Shannon entropy and the Gini‐Simpson index. These correspond to strategies β(A<sub>0</sub>BB)=<sup>0</sup>β<sub>At</sub> (Section 3, above), 
<img alt="inline image" class=section_image src=images/155.gif> (Section 6) and 
<img alt="inline image" class=section_image src=images/156.gif> (Section 7), respectively. Many studies have applied two or three of these measures in parallel with the intention of studying how the alpha and beta components of diversity are affected when different relative weights are given to common vs rare species (<span><a href=#b27 id=#b27R class="bibLink tab-link" data-tab=pane-pcw-references>DeVries et al. 1997</a></span>, <span><a href=#b127 id=#b127R class="bibLink tab-link" data-tab=pane-pcw-references>Wagner et al. 2000</a></span>, <span><a href=#b26 id=#b26R class="bibLink tab-link" data-tab=pane-pcw-references>DeVries and Walla 2001</a></span>, <span><a href=#b23 id=#b23R class="bibLink tab-link" data-tab=pane-pcw-references>Crist et al. 2003</a></span>, <span><a href=#b39 id=#b39R class="bibLink tab-link" data-tab=pane-pcw-references>Gering et al. 2003</a></span>, <span><a href=#b116 id=#b116R class="bibLink tab-link" data-tab=pane-pcw-references>Summerville et al. 2003</a></span>, <span><a href=#b101 id=#b101R class="bibLink tab-link" data-tab=pane-pcw-references>Ribeiro et al. 2008</a></span>), or have discussed related analytical methods (<span><a href=#b68 id=#b68R class="bibLink tab-link" data-tab=pane-pcw-references>Lande 1996</a></span>, <span><a href=#b93 id=#b93R class="bibLink tab-link" data-tab=pane-pcw-references>Pélissier et al. 2003</a></span>, <span><a href=#b20 id=#b20R class="bibLink tab-link" data-tab=pane-pcw-references>Couteron and Pélissier 2004</a></span>, <span><a href=#b21 id=#b21R class="bibLink tab-link" data-tab=pane-pcw-references>Couteron and Ollier 2005</a></span>, <span><a href=#b92 id=#b92R class="bibLink tab-link" data-tab=pane-pcw-references>Pélissier and Couteron 2007</a></span>). However, when raw diversity index values are used, each diversity index not only weights species differently but also defines diversity itself in a different way. This causes a new definition of “beta diversity” to be created for each new diversity index, namely strategy β(+BB). I have found only one study (<span><a href=#b11 id=#b11R class="bibLink tab-link" data-tab=pane-pcw-references>Chandy et al. 2006</a></span>) that conducted additive diversity partitioning for several values of <i>q</i> in parallel such that <sup><i>q</i></sup><i>D</i> was used as the diversity measure. This is necessary for the results obtained with different diversity indices to quantify the same phenomenon (<sup><i>q</i></sup>β<sub>At</sub> in the case of <span><a href=#b11 id=#b11R class="bibLink tab-link" data-tab=pane-pcw-references>Chandy et al. 2006</a></span>) and hence to shed light on the effect of rare vs abundant species on that phenomenon. In genetic studies (where the γ‐classification is based on alleles rather than species), β(CBB) is known as the <i>D</i><sub>ST</sub> index (<span><a href=#b83 id=#b83R class="bibLink tab-link" data-tab=pane-pcw-references>Nei 1973</a></span>, <span><a href=#b58 id=#b58R class="bibLink tab-link" data-tab=pane-pcw-references>Jost 2008</a></span>).
 </p>
 
 <p> <span><a href=#b68 id=#b68R class="bibLink tab-link" data-tab=pane-pcw-references>Lande (1996)</a></span> further proposed using α′/γ′=(γ′−β′)/γ′=1−β′/γ′ as a measure of community similarity, and observed that when species richness is used as the diversity index, this equals the inverse of Whittaker's beta diversity (1/<sup>0</sup>β<sub>Mt</sub> in the current annotation). The corresponding dissimilarity index is β′/γ′=(γ′−α′)/γ′=1−α′/γ′. When based on species richness, this measure equals 
<img alt="inline image" class=section_image src=images/157.gif>, which was discussed as a variant of “beta diversity” in Section 5 (above), and can now be classified as strategy β(A<sub>0</sub>BD). With Shannon entropy, β′/γ′ becomes 
<img alt="inline image" class=section_image src=images/158.gif> for which the strategic annotation is β(B<sub>1,<i>e</i></sub>BD). This measure was used in a classic study on human genetic diversity by <span><a href=#b72 id=#b72R class="bibLink tab-link" data-tab=pane-pcw-references>Lewontin (1972)</a></span>. With the Gini‐Simpson index, β′/γ′ becomes 
<img alt="inline image" class=section_image src=images/159.gif> whose strategic annotation is β(CBD). In genetic studies, β(CBD) is known as the <i>G</i><sub>ST</sub> index (<span><a href=#b83 id=#b83R class="bibLink tab-link" data-tab=pane-pcw-references>Nei 1973</a></span>, <span><a href=#b58 id=#b58R class="bibLink tab-link" data-tab=pane-pcw-references>Jost 2008</a></span>).
 </p>
 
 <p>If β′/γ′ is equated with “beta diversity”, then a new variant of “beta diversity” is generated for each new diversity index used, namely strategy β(+BD). Several researchers have used these measures to quantify compositional (dis)similarity between sampling units (<span><a href=#b27 id=#b27R class="bibLink tab-link" data-tab=pane-pcw-references>DeVries et al. 1997</a></span>, <span><a href=#b26 id=#b26R class="bibLink tab-link" data-tab=pane-pcw-references>DeVries and Walla 2001</a></span>, <span><a href=#b34 id=#b34R class="bibLink tab-link" data-tab=pane-pcw-references>Fournier and Loreau 2001</a></span>, <span><a href=#b102 id=#b102R class="bibLink tab-link" data-tab=pane-pcw-references>Ricotta 2003</a></span>, <span><a href=#b82 id=#b82R class="bibLink tab-link" data-tab=pane-pcw-references>Munos et al. 2008</a></span>). However, the interval over which the values of (γ′−α′)/γ′ vary depends on which measure is used, which complicates interpretation. In the case of β(B<sub>1,<i>e</i></sub>BD), the possible range of values is [0, ln(<i>N</i>)/ln(<i>N</i>α<sub>t</sub>)]. In the other cases, <i>q</i>≠1 so a maximum value can be defined only when all sampling unit weights are equal. If this is the case, the range of values is [0, (<i>N</i>−1)/<i>N</i>] for β(A<sub>0</sub>BD) and [0, (<i>N</i>−1)/(<i>N</i>α<sub>t</sub>−1)] for β(CBD). Note the presence of α<sub>t</sub> in two of the range definitions; it indicates that the value of (γ′−α′)/γ′ is independent of alpha diversity only when the diversity index in question has the doubling property. This easily causes interpretational problems when α varies among datasets (<span><a href=#b56%20#b57%20#b58 id="#b56 #b57 #b58R" class="bibLink tab-link" data-tab=pane-pcw-references>Jost 2006, 2007, 2008</a></span>). Note also that all maximum values are dependent on <i>N</i>, which easily causes interpretational problems when <i>N</i> varies among datasets.
 </p>
 
 <p>Each one of the β(+BB) and β(+BD) measures proposed by <span><a href=#b68 id=#b68R class="bibLink tab-link" data-tab=pane-pcw-references>Lande (1996)</a></span> quantifies some real property of the data, but a different property in each case. Neither family of measures quantifies either true beta diversity of a dataset or compositional differentiation among sampling units, so interpreting them as if they did leads to erroneous conclusions.
 </p>
 
 <div class=paragraph-displayedItem-text-type>The <i>C</i><sub><i>qN</i></sub> measure of <span><a href=#b13 id=#b13R class="bibLink tab-link" data-tab=pane-pcw-references>Chao et al. (2008)</a></span>, referred to in Sections 4, 5 and 6 (above), provides a compositional overlap index that is independent of the species richness of the system at all values of <i>q</i> (if <i>q</i>≠1, all sampling unit weights have to be equal). <i>C</i><sub><i>qN</i></sub> is calculated as follows:
 <div class=inline-equation id=di-fml-105><span class="inline-equation__construct figure"><img class=figure__image src=images/160.gif alt=image title=image></span><span class=inline-equation__label></span></div> 
 </div>
 
 <div class=paragraph-displayedItem-text-type>At <i>q</i>=2, this gives the same result as the equation of <sup><i>q</i></sup><i>C</i><sub>J<i>N</i></sub> (Section 5). The exponent is positive at <i>q</i>&gt;1, but becomes negative at <i>q</i>&lt;1 and therefore causes all terms to be inverted. The equation can then be restated with non‐negative exponents as
 <div class=inline-equation id=di-fml-106><span class="inline-equation__construct figure"><img class=figure__image src=images/161.gif alt=image title=image></span><span class=inline-equation__label></span></div> 
 </div>
 
 <p>At <i>q</i>=0, this gives the same result as the equation of <sup><i>q</i></sup><i>C</i><sub>S<i>N</i></sub> (Section 4), and in this case <i>C</i><sub><i>qN</i></sub> is linearly related with <sup><i>q</i></sup>β<sub>Mt</sub>. As <i>q</i> increases, the exponent causes the relationship between <i>C</i><sub><i>qN</i></sub> and <sup><i>q</i></sup>β<sub>Mt</sub> to become increasingly curvilinear.
 </p>
 
 <p>As an index of compositional overlap, <i>C</i><sub><i>qN</i></sub> has attractive properties (<span><a href=#b56 id=#b56R class="bibLink tab-link" data-tab=pane-pcw-references>Jost 2006</a></span>, <span><a href=#b13 id=#b13R class="bibLink tab-link" data-tab=pane-pcw-references>Chao et al. 2008</a></span>; see also Sections 4, 5 and 6). The general tendency in the ecological literature to interpret one‐complements of similarity indices as “beta diversity” calls for a few words of caution, however. <i>C</i><sub><i>qN</i></sub> is based on <sup><i>q</i></sup>β<sub>Mt</sub><sup>1−<i>q</i></sup> rather than on <sup><i>q</i></sup>β<sub>Mt</sub> itself, so its interpretation in terms of beta diversity is different for each value of <i>q</i>. When <i>q</i>=0, <i>C</i><sub>0<i>N</i></sub> is a linear transformation (given <i>N</i>) of regional‐to‐local diversity ratio <sup>0</sup>β<sub>Mt</sub> and Whittaker's species turnover <sup>0</sup>β<sub>Mt−1</sub> (Section 4). Its one‐complement then equals the ranged version of strategies β(A<sub>0</sub>BA) and β(A<sub>0</sub>BC), respectively. In turn, <i>C</i><sub>2<i>N</i></sub> is a linear transformation (given <i>N</i>) of proportional species turnover <sup>2</sup>β<sub>Pt</sub> (Section 5), and 1−<i>C</i><sub>2<i>N</i></sub> equals the ranged version of strategy 1/β(A<sub>2</sub>BA). In the limit, as <i>q</i> approaches unity, <i>C</i><sub>1<i>N</i></sub> is a linear transformation (given <i>N</i>) of regional Shannon entropy excess 
<img alt="inline image" class=section_image src=images/162.gif> (Section 6), and 1−<i>C</i><sub>1<i>N</i></sub> equals the ranged version of strategies β(B<sub>1,<i>b</i></sub>BA) and log[β(A<sub>1</sub>BA)]. In general, <i>C</i><sub><i>qN</i></sub> is linearly related with strategy 1/[β(A<sub><i>q</i></sub>BA)]<sup><i>q</i>−1</sup>=[β(A<sub><i>q</i></sub>BA)]<sup>1−<i>q</i></sup>. This is linearly related with regional‐to‐local diversity ratio β(A<sub><i>q</i></sub>BA) and true beta diversity β(A<sub><i>q</i></sub>AA) only at <i>q</i>=0. If <i>C</i><sub><i>qN</i></sub> were mistakenly (contrary to its intended use) assumed to represent either “beta diversity” or a linear transformation of “beta diversity” with all <i>q</i>, a different definition of “beta diversity” would be created for each different value of <i>q</i>.
 </p>
 
 <p>Several recent papers have suggested taking phylogenetic relatedness into account when quantifying “beta diversity”. Some of the proposed measures are not related to alpha and gamma diversity in any way (<span><a href=#b55 id=#b55R class="bibLink tab-link" data-tab=pane-pcw-references>Izsak and Price 2001</a></span>, <span><a href=#b75 id=#b75R class="bibLink tab-link" data-tab=pane-pcw-references>Lozupone et al. 2007</a></span>) and therefore create new definitions of “beta diversity” by strategy 4. Others use as a starting point the decomposition of total diversity (or the decomposition of a diversity index), and then modify the calculations by applying weights to species pairs according to the phylogenetic distance between them (<span><a href=#b45 id=#b45R class="bibLink tab-link" data-tab=pane-pcw-references>Hardy and Senterre 2007</a></span>). <span><a href=#b41 id=#b41R class="bibLink tab-link" data-tab=pane-pcw-references>Graham and Fine (2008)</a></span> promoted the use of measures that incorporate phylogenetic information to study the link between ecological and evolutionary processes, and suggested using the terms “phylogenetic beta diversity” and “phylobetadiversity” for them. Such measures may indeed be useful, but great care is needed to avoid muddling these new concepts as badly as has happened with the traditional measures of “beta diversity”. It seems that a transparent and straightforward way of examining the contribution of higher taxonomical levels to beta diversity (or to the other multiplicative diversity components) could be achieved simply by using a hierarchical γ‐classification, as discussed in Section 1 (<i>Hierarchical diversity partitioning</i>).
 </p>
 
 </section>
 <section class=article-section__content id=ss16>
 
 <h2 class="article-section__title section__title section1" id=ss16-title> Conclusions</h2>
 
 <p>Any diversity component (α, β or γ) can be computed for any group of organisms. To allow accurate communication about what the components are, a few crucial definitions need to be stated at the outset. First, quantifying gamma diversity necessitates specifying the γ‐classification: which of the entities of interest belong to which of the types of interest. Most ecological applications have been interested in species diversity, in which case the γ‐classification is the classification of observed individuals (or other units of abundance) into species. However, the concept of diversity is general and the same mathematical arguments can be applied to any dataset in which entities are classified into types. For example, individuals could be classified into taxa of some other rank than species (such as genera or families), into types defined by the presence of different alleles of a gene, or into ecological guilds, functional groups or size classes (as long as any given individual can belong to only one type). Similarly, entries in a species list could be classified into types on the basis of, say, the number of letters they contain, or characters in a species list could be classified into types on the basis of which letter they represent. To avoid confusion, it seems safest to restrict the use of the terms <i>γ‐classification</i> and <i>gamma diversity</i> to taxonomical classes such as species, and to use other terms to refer to diversity related to other kinds of classifications.
 </p>
 
 <p>If gamma diversity is to be partitioned into alpha and beta components, a second classification of the entities of interest is needed. The ω‐classification defines which entities were observed in which sampling units. Changing the ω‐classification by splitting or pooling sampling units leads to different proportions of the total diversity being apportioned to the alpha vs. the beta component. Attention must also be paid to sampling unit weights when calculating alpha and gamma diversity: these define whether the diversity components are quantified for the original dataset or for a new dataset with modified abundance values.</p>
 
 <p>Once the γ‐classification and the ω‐classification have been defined, it is possible to proceed to defining diversity and its components. First, one must specify which measure is being used to quantify “diversity”, what the alpha component represents, and which equation is being used to relate alpha and gamma diversities to each other when quantifying the beta component. True beta diversity is the number of compositional units in the dataset. It is obtained as β<sub>Md</sub>=γ/α<sub>d</sub> where true gamma diversity is the total effective number of species in the dataset, and true alpha diversity is the effective number of species per compositional unit, with each compositional unit having the same effective number of species as the real sampling units do on average. These diversity components are ecologically meaningful and provide a unified foundation for discussions concerning patterns and processes in diversity.
 </p>
 
 <p>Each of the other ways of partitioning true gamma diversity or a gamma diversity index value can also be useful in specific ecological applications. However, their beta components do not quantify true beta diversity but other phenomena, such as absolute or relative turnover of effective species, ratio of regional to local species diversity, or the amount by which total entropy or variance exceeds the average entropy or variance of a single sampling unit. Accurate communication about these phenomena requires that each is identified by its own unique name.</p>
 
 </section>
 <section class=article-section__content id=ss17>
 
 <h2 class="article-section__title section__title section1" id=ss17-title> Mathematical proofs</h2>
 
 <div class=paragraph-displayedItem-text-type> <b>Proof 1</b>. The Shannon entropy <i>H</i>′ is calculated as
 <div class=inline-equation id=di-fml-108><span class="inline-equation__construct figure"><img class=figure__image src=images/163.gif alt=image title=image></span><span class=inline-equation__label></span></div> 
 </div>
 
 <div class=paragraph-displayedItem-text-type>This equals 
 <div class=inline-equation id=di-fml-109><span class="inline-equation__construct figure"><img class=figure__image src=images/164.gif alt=image title=image></span><span class=inline-equation__label></span></div> 
 </div>
 
 <div class=paragraph-displayedItem-text-type>Since <img alt="inline image" class=section_image src=images/165.gif> equals unity by definition, 
<img alt="inline image" class=section_image src=images/166.gif> is the weighted geometric mean of the <i>p</i><sub><i>i</i></sub> values and <i>H′</i> therefore equals
 <div class=inline-equation id=di-fml-112><span class="inline-equation__construct figure"><img class=figure__image src=images/167.gif alt=image title=image></span><span class=inline-equation__label></span></div> 
 </div>
 
 <p>Choosing the base of the logarithm corresponds to choosing the unit of measurement (<span><a href=#b113 id=#b113R class="bibLink tab-link" data-tab=pane-pcw-references>Shannon 1948</a></span>). With base 2 the unit is binary digit or bit, with base <i>e</i> natural digit or nat and with base 10 decimal digit or decit. The geometric mean itself is not affected by the unit of the entropy, so when <i>H</i>′ is converted to its numbers equivalent exp(<i>H′</i>)=exp(log(<sup>1</sup><i>D</i>)), the same true diversity is recovered no matter which log base is used, provided that the same base is used both in the exponential function and the logarithm.
 </p>
 
 <div class=paragraph-displayedItem-text-type> <b>Proof 2</b>. To calculate mean species diversity within the sampling units α<sub>t</sub>, the weighted generalised mean with exponent <i>q</i>−1 of the proportional abundances of all species within all sampling units is first obtained as
 <div class=inline-equation id=di-fml-113><span class="inline-equation__construct figure"><img class=figure__image src=images/168.gif alt=image title=image></span><span class=inline-equation__label></span></div> 
 </div>
 
 <div class=paragraph-displayedItem-text-type>Because <i>p</i><sub><i>ij</i></sub>=<i>w</i><sub><i>j</i></sub><i>p</i><sub><i>i∣j</i></sub> (<a href=#t1 class=scrollableLink>Table 1</a>), this can be rewritten
 <div class=inline-equation id=di-fml-114><span class="inline-equation__construct figure"><img class=figure__image src=images/169.gif alt=image title=image></span><span class=inline-equation__label></span></div> 
 </div>
 
 <div class=paragraph-displayedItem-text-type>On the other hand, species diversity within sampling unit <i>j</i> equals
 <div class=inline-equation id=di-fml-115><span class="inline-equation__construct figure"><img class=figure__image src=images/170.gif alt=image title=image></span><span class=inline-equation__label></span></div> 
 </div>
 
 <div class=paragraph-displayedItem-text-type>from which follows that 
 <div class=inline-equation id=di-fml-116><span class="inline-equation__construct figure"><img class=figure__image src=images/171.gif alt=image title=image></span><span class=inline-equation__label></span></div> 
 </div>
 
 <div class=paragraph-displayedItem-text-type>The overall mean of the within‐sampling unit proportional abundances can therefore be written 
 <div class=inline-equation id=di-fml-117><span class="inline-equation__construct figure"><img class=figure__image src=images/172.gif alt=image title=image></span><span class=inline-equation__label></span></div> 
 </div>
 
 <div class=paragraph-displayedItem-text-type>The inverse of this gives mean species diversity within the sampling units 
 <div class=inline-equation id=di-fml-118><span class="inline-equation__construct figure"><img class=figure__image src=images/173.gif alt=image title=image></span><span class=inline-equation__label></span></div> 
 </div>
 
 <p>This is the weighted generalised mean with exponent 1−<i>q</i> of the <sup><i>q</i></sup><i>D</i><sub>γ<i>j</i></sub> values.
 </p>
 
 <p>Above, only the γ‐classification is considered important, so the measurement unit of <i>p</i><sub><i>i∣j</i></sub> is (individuals/effective species)/individuals and the measurement unit of α<sub>t</sub> becomes sp<sub>E</sub>. If the β‐classification is also considered important, the measurement unit of <i>p</i><sub><i>i∣j</i></sub> is (individuals/effective species)/(individuals/compositional unit), and the result corresponds to α<sub>d</sub> with measurement unit sp<sub>E</sub>/CU. Although the unit of abundance (here individual) is not explicit in the final measurement unit, different units of abundance lead to numerically different results.
 </p>
 
 <div class=paragraph-displayedItem-text-type> <b>Proof 3</b>. The overall mean of the within‐sampling unit proportional abundances can be written
 <div class=inline-equation id=di-fml-119><span class="inline-equation__construct figure"><img class=figure__image src=images/174.gif alt=image title=image></span><span class=inline-equation__label></span></div> 
 </div>
 
 <div class=paragraph-displayedItem-text-type>Mean species diversity within the sampling units is the inverse of this generalised mean and equals 
 <div class=inline-equation id=di-fml-120><span class="inline-equation__construct figure"><img class=figure__image src=images/175.gif alt=image title=image></span><span class=inline-equation__label></span></div> 
 </div>
 
 <div class=paragraph-displayedItem-text-type>which is the numbers equivalent of the weighted arithmetic mean of the <sup><i>q</i></sup>λ<sub>γ<i>j</i></sub> values, with the row totals of <a href=#t1 class=scrollableLink>Table 1</a> used as weights. At <i>q</i>=1, the mean species diversity within sampling units is obtained as the exponential of the weighted arithmetic mean of the corresponding Shannon entropies (Proof 1 and Routledge 1977, 1979)
 <div class=inline-equation id=di-fml-121><span class="inline-equation__construct figure"><img class=figure__image src=images/176.gif alt=image title=image></span><span class=inline-equation__label></span></div> 
 </div>
 
 <div class=paragraph-displayedItem-text-type> <b>Proof 4</b>. <span><a href=#b57 id=#b57R class="bibLink tab-link" data-tab=pane-pcw-references>Jost (2007)</a></span> calculated “alpha diversity” as
 <div class=inline-equation id=di-fml-122><span class="inline-equation__construct figure"><img class=figure__image src=images/177.gif alt=image title=image></span><span class=inline-equation__label></span></div> 
 </div>
 
 <div class=paragraph-displayedItem-text-type>The denominator can be rewritten 
 <div class=inline-equation id=di-fml-123><span class="inline-equation__construct figure"><img class=figure__image src=images/178.gif alt=image title=image></span><span class=inline-equation__label></span></div> 
 </div>
 
 <p>which is the effective number of sampling units in the dataset (virtual rows, each with mean <i>w</i><sub><i>j</i></sub> in <a href=#t1 class=scrollableLink>Table 1</a>).
 </p>
 
 <div class=paragraph-displayedItem-text-type>By inserting <img alt="inline image" class=section_image src=images/179.gif>, the numerator of <sup><i>q</i></sup><i>D</i><sub>‘γω’/γ</sub> can be rewritten
 <div class=inline-equation id=di-fml-125><span class="inline-equation__construct figure"><img class=figure__image src=images/180.gif alt=image title=image></span><span class=inline-equation__label></span></div> 
 </div>
 
 <div class=paragraph-displayedItem-text-type>Because <i>p</i><sub><i>ij</i></sub>=<i>w</i><sub><i>j</i></sub><i>p</i><sub><i>i∣j</i></sub> (<a href=#t1 class=scrollableLink>Table 1</a>), this equals
 <div class=inline-equation id=di-fml-126><span class="inline-equation__construct figure"><img class=figure__image src=images/181.gif alt=image title=image></span><span class=inline-equation__label></span></div> 
 </div>
 
 <p>which is the effective number of species‐sampling unit combinations in the dataset (virtual cells, each with mean <i>p</i><sub><i>ij</i></sub> in <a href=#t1 class=scrollableLink>Table 1</a>).
 </p>
 
 </section>
 <div class=article-section__content>
 
 <h2 class="article-section__title section__title section1" id=ss18-title> Acknowledgements</h2>
 
 <p>I thank Robert K. Colwell for inviting me to write this review, for support during the process and for many useful comments on several versions of the manuscript. Numerous discussions with Kalle Ruokolainen on beta diversity and related issues have inspired and helped structure the contents of this paper. Helpful suggestions were also given by Mark Higgins, Mirkka Jones, Juhani Karhumäki, Esa Lehikoinen and especially four anonymous reviewers. The Academy of Finland provided funding that enabled my almost total immersion in the theme of beta diversity.</p>
 
 <ol></ol>
 </div>
 <section class="article-section article-section__references" id=references-section>
 <div class="accordion article-accordion">
 <h2>
 <div tabindex=0 role=button class=accordion__control aria-controls=idss19 aria-expanded=false data-references="/action/ajaxShowEnhancedAbstract?widgetId=5cf4c79f-0ae9-4dc5-96ce-77f62de7ada9&amp;ajax=true&amp;doi=10.1111%2Fj.1600-0587.2009.05880.x&amp;pbContext=%3Barticle%3Aarticle%3Adoi%5C%3A10.1111%2Fj.1600-0587.2009.05880.x%3Bpage%3Astring%3AArticle%2FChapter+View%3Bctype%3Astring%3AJournal+Content%3Bwebsite%3Awebsite%3Apericles%3Bjournal%3Ajournal%3A16000587%3Bissue%3Aissue%3Adoi%5C%3A10.1111%2Feco.2010.33.issue-1%3Bwgroup%3Astring%3APublication+Websites%3BrequestedJournal%3Ajournal%3A16000587%3BpageGroup%3Astring%3APublication+Pages%3BsubPage%3Astring%3AFull+Text&amp;accordionHeadingWrapper=h2&amp;onlyLog=true"><span class=section__title id=references-section-1>References</span><i aria-hidden=true class=icon-section_arrow_d></i></div>
 </h2>
 <div class=accordion__content style=display:none role=region aria-labelledby=references-section-1 id=ss19>
 
 </div>
 </div>
 </section>
 </section>
 <div style=display:none>
 
 </div>
 <div class=pb-dropzone data-pb-dropzone=citedBy-dropzone>
 <section id=cited-by class="article-section article-section__citedBy cited-by"><div class="accordion article-accordion"><h2><div tabindex=0 role=button aria-controls=cited-by__content aria-expanded=false class=accordion__control><span id=citedby-section class=section__title>Citing Literature</span><i aria-hidden=true class=icon-section_arrow_d></i></div></h2><div id=cited-by__content class="accordion__content cited-by__content" style=display:none></div></div></section>
 </div>
 
</article></div></div><div class="col-lg-4 col-md-4 col-sm-4 hidden-xs hidden-sm hidden-md article-row-right fixed-element" style=top:-444px><div class="publication-cover-info clearfix">
 
 <div class="cover-image-wrapper cover-image__image hasDetails"><a href=https://onlinelibrary.wiley.com/toc/16000587/2010/33/1 title="View Volume 33, Issue 1"><img src=images/182.gif alt></a></div><div class="extra-info-wrapper cover-image__details"><p class=volume-issue><a href=https://onlinelibrary.wiley.com/toc/16000587/2010/33/1 title="View Volume 33, Issue 1"><span>Volume</span><span class=val>33</span>, <span>Issue</span><span class=val>1</span></a><p>February 2010<p class=page-range><span>Pages</span><span> 2-22</span></p>
 
 
</div>
</div>
 
 
 <div class=pb-ad>
 <div id=advert-rail-1 class="advert advert-rail" data-google-query-id=CPfszNzOw-wCFU7AEQgdrsoAqg>
 
 
 
 <div id=google_ads_iframe_/2215/wly.ecology_000046/ecography_ecog_1__container__ style="border:0pt none"></div></div>
 </div>
 
<div class="sticko__parent scrollThenFix" style="height:calc(0px + 100vh)"><div class="tab tab--slide tab--flex sticko__md tabs--xs dynamic-sticko"><ul data-mobile-toggle=slide role=tablist class="rlist tab__nav w-slide--list cloned"><li role=presentation><a data-slide-clone=self href=#pane-pcw-figures aria-controls=#pane-pcw-figures role=tab data-toggle=tab id=pane-pcw-figurescon data-slide-target=#pane-pcw-figures class="article-row-right__tabLinks w-slide__btn figures-tab empty" tabindex=-1 aria-selected=false><i aria-hidden=true class=icon-multi-figure></i><h2>Figures</h2></a><li role=presentation><a data-slide-clone=self href=#pane-pcw-references aria-controls=#pane-pcw-references role=tab data-toggle=tab id=pane-pcw-referencescon data-slide-target=#pane-pcw-references class="article-row-right__tabLinks w-slide__btn references-tab empty" tabindex=-1 aria-selected=false><i aria-hidden=true class=icon-chain></i><h2>References</h2></a><li role=presentation><a data-slide-clone=self href=#pane-pcw-related aria-controls=#pane-pcw-related role=tab data-toggle=tab id=pane-pcw-relatedcon data-slide-target=#pane-pcw-related class="article-row-right__tabLinks w-slide__btn related-tab empty" tabindex=-1 aria-selected=false><i aria-hidden=true class=icon-related-circles-extend-button></i><h2>Related</h2></a><li role=presentation class=active><a data-slide-clone=self href=#pane-pcw-details aria-controls=#pane-pcw-details role=tab data-toggle=tab id=pane-pcw-detailscon data-slide-target=#pane-pcw-details class="article-row-right__tabLinks w-slide__btn details-tab" tabindex=-1 aria-selected=false><i aria-hidden=true class=icon-Icon_Information></i><h2>Information</h2></a></ul><ul role=presentation class="rlist tab__content sticko__child"><li id=pane-pcw-figures aria-labelledby=pane-pcw-figurescon role=tabpanel class="tab__pane article-row-right__panes empty sf-hidden" aria-hidden=true><li id=pane-pcw-references aria-labelledby=pane-pcw-referencescon role=tabpanel class="tab__pane article-row-right__panes empty sf-hidden" aria-hidden=true><li id=pane-pcw-related aria-labelledby=pane-pcw-relatedcon role=tabpanel class="tab__pane article-row-right__panes empty sf-hidden" aria-hidden=true><li id=pane-pcw-details aria-labelledby=pane-pcw-detailscon role=tabpanel class="tab__pane article-row-right__panes active" aria-hidden=true>
<section class=info-tab-wrapper>
<section class=metrics>
<h3 class=info-header>Metrics</h3>
<div class=section__body><div class=cited-by-count><label>Citations: </label><span>488</span></div>
<div class=pb-dropzone data-pb-dropzone=publicaitonContent-info-tab-metrics>
<div data-badge-popover=left data-badge-type=1 data-doi=10.1111/j.1600-0587.2009.05880.x data-hide-no-mentions=false class=altmetric-embed data-uuid=c70534d8-67d0-e4e3-3208-911c8aa80c6b><a target=_self href="https://www.altmetric.com/details.php?domain=onlinelibrary.wiley.com&amp;citation_id=689014" rel=popover data-content="<div>    <div style='padding-left: 10px; line-height:18px; border-left: 16px solid #ffd140;'>
      <a class='link-to-altmetric-details-tab' target='_self' href='https://www.altmetric.com/details.php?domain=onlinelibrary.wiley.com&amp;citation_id=689014&amp;tab=blogs'>
          Blogged by <b>3</b>
      </a>
    </div>
    <div style='padding-left: 10px; line-height:18px; border-left: 16px solid #74CFED;'>
      <a class='link-to-altmetric-details-tab' target='_self' href='https://www.altmetric.com/details.php?domain=onlinelibrary.wiley.com&amp;citation_id=689014&amp;tab=twitter'>
          Tweeted by <b>4</b>
      </a>
    </div>
    <div style='padding-left: 10px; line-height:18px; border-left: 16px solid #958899;'>
      <a class='link-to-altmetric-details-tab' target='_self' href='https://www.altmetric.com/details.php?domain=onlinelibrary.wiley.com&amp;citation_id=689014&amp;tab=wikipedia'>
          Referenced in <b>5</b> Wikipedia pages
      </a>
    </div>

    <div class='altmetric-embed readers' style='margin-top: 10px;'>
          <div class='altmetric-embed tip_mendeley'
               style='padding-left: 10px; line-height:18px; border-left: 16px solid #A60000;'>
              <b>1865</b> readers on Mendeley
          </div>
          <div class='altmetric-embed tip_citeulike'
               style='padding-left: 10px; line-height:18px; border-left: 16px solid #BCD2EF;'>
              <b>4</b> readers on CiteULike
          </div>
    </div>
    <div style='margin-top: 10px; text-align: center;'>
        <a class='altmetric_details' target='_self' href='https://www.altmetric.com/details.php?domain=onlinelibrary.wiley.com&amp;citation_id=689014'>
            See more details
        </a>
    </div>
</div>" style=display:inline-block data-badge-popover=left>
 <img alt="Article has an altmetric score of 27" src=images/183.png style=border:0;margin:0;max-width:none width=110 height=20>
</a></div>
</div>
</div>
</section>
<section class=details>
<h3 class=info-header>Details</h3>
<p class=copyright>© 2010 The Authors</p>
<p></p>
<ul class="cross-mark rlist">
</ul>
</section>
<section class=publication-history>
<h3 class=info-header>Publication History</h3>
<ul class=rlist>
<li><label>Issue Online: </label>04 March 2010
</li>
<li><label>Version of Record online: </label>11 February 2010
</li>
<li><label>Editorial history: </label>Manuscript Accepted 18 November 2009
</li>
</ul>
</section>
</section>
<li class=tab__spinner style=display:none></ul></div></div></div></div></article><div id=figure-viewer data-ux3-wrapper=figure-viewer data-ux3-transformed-by=figureInit data-ux3-role=parent role=dialog class="figure-viewer sf-hidden"></div>
 
 <div style=text-align:right!important><a style=font-weight:300!important;font-size:90% href=https://onlinelibrary.wiley.com/doi/pdf/10.1111/j.1600-0587.2009.05880.x>Download PDF</a></div>
</div>
</div></div></div>
 </section>
 
 </div>
 </div>
<div class="w-slide padding-top"><div class=w-slide_head><a href=# class=w-slide__back><i class=icon-arrow_l aria-hidden=true></i>back</a><span class=w-slide__title></span></div><div class=w-slide__content></div></div></main>
 </div>
 
 <div data-widget-def=pageFooter data-widget-id=053994f3-7d80-4bb1-97a4-c67fbc4768cd class=pageFooter>
 
 
 <footer class=page-footer>
 <div data-pb-dropzone=main>
 
 
 <div class=main-footer>
 
 <style class=sf-hidden>.footer-links{text-align:center}.footer-links h3{font-weight:300}.footer-links a{font-weight:600}@media (min-width:992px){.footer-links{text-align:left}}</style>
 <div data-widget-def=ux3-layout-widget data-widget-id=2753bec1-a79d-4403-827b-6c788ddd6db3 class=footer-sections>
 
 
 <h2 class=sr-only>
 Additional links
 </h2>
 
 <div class=container><div class=row><div class=col-md-3>
 
 <div class="footer-links mb-3">
 <h3>About Wiley Online Library</h3>
 <ul class=rlist>
 <li><a href=https://www.wiley.com/privacy>Privacy Policy</a></li>
 <li><a href=https://onlinelibrary.wiley.com/terms-and-conditions>Terms of Use</a></li>
 <li><a href=https://onlinelibrary.wiley.com/cookies>Cookies</a></li>
 <li><a href=https://onlinelibrary.wiley.com/accessibility>Accessibility</a></li>
 </ul>
</div>
</div><div class=col-md-3>
 
 <div class="footer-links mb-3">
 <h3>Help &amp; Support</h3>
 <ul class=rlist>
 <li><a href=https://hub.wiley.com/community/support/onlinelibrary>Contact Us</a></li>
 </ul>
 </div>
</div><div class=col-md-3>
 
 <div class="footer-links mb-3">
 <h3>Opportunities</h3>
 <ul class=rlist>
 <li><a href=https://onlinelibrary.wiley.com/agents>Subscription Agents</a></li>
 <li><a href=https://onlinelibrary.wiley.com/advertisers>Advertisers &amp; Corporate Partners</a></li>
 </ul>
 </div>
</div><div class=col-md-3>
 
 <div class="footer-links mb-3">
 <h3>Connect with Wiley</h3>
 <ul class=rlist>
 <li><a href=https://www.wiley.com/network>The Wiley Network</a></li>
 <li><a href=https://newsroom.wiley.com/>Wiley Press Room</a></li>
 </ul>
 </div>
</div></div></div>
 </div>
 
 <div data-widget-def=ux3-layout-widget data-widget-id=86584e25-e4db-4118-9550-9b4f5341a4ac class=copyright-container>
 
 
 <div class=container><div class=row><div class=col-md-12>
 <div data-widget-def=UX3HTMLWidget data-widget-id=bfbf6c5d-a650-4662-828f-5275fada9120 class=copyright>
 
 
 <div class="copyright pull-left">Copyright © 1999-2020 <a href=https://www.wiley.com/>John Wiley &amp; Sons, Inc</a>. All rights reserved</div>
 </div>
 
 
 <div class="footer-logo pull-right">
 
<a href=https://www.wiley.com/ title="Wiley Home Page" aria-label=Wiley><img alt="Wiley Home Page" src=images/184.png></a>
</div>
</div></div></div>
 </div>
 
</div>
 </div>
</footer>
 </div>
 
 
 <div id=iucr-failure class=sf-hidden> </div> 
 
 
<div class="popup login-popup hidden sf-hidden" role=dialog tabindex=-1 aria-labelledby=loginPopupHead>
 
</div>
 
 
<div class="popup change-password-drawer hidden sf-hidden" role=dialog tabindex=-1 aria-labelledby=changePasswordPopupHead>
 
</div>
 
 
<div class="popup registration-popup hidden sf-hidden" role=dialog tabindex=-1 aria-labelledby=createAccountPopupHead>
 
</div>
 
 
<div class="popup top-drawer request-reset-password-drawer hidden sf-hidden" role=dialog tabindex=-1 aria-labelledby=resetPasswordPopupHead>
 
</div>
 
 <div><div class=pb-dropzone data-pb-dropzone=col-0 title=col-0></div></div>
 
 
<div class="popup top-drawer request-username-drawer username-popup hidden sf-hidden" role=dialog tabindex=-1 aria-labelledby=requestUsernamePopupHead>
 
 
</div>
 
 
 </div>
 </div><img src=images/185.x>
 
<div style=visibility:hidden;height:1px;width:1px;position:absolute;top:-9999px;z-index:100000 id=_atssh class=sf-hidden></div><div id=crossmark-widget style=display:none></div><ul class="ui-autocomplete ui-front ui-menu ui-widget ui-widget-content ui-corner-all quickSearchAutocomplete" id=ui-id-1 tabindex=0 style=display:none></ul><ul class="ui-autocomplete ui-front ui-menu ui-widget ui-widget-content ui-corner-all quickSearchAutocomplete" id=ui-id-2 tabindex=0 style=display:none></ul><div class="altmetric-embed altmetric-popover altmetric-left" data-uuid=c70534d8-67d0-e4e3-3208-911c8aa80c6b id=_altmetric_popover_el_c70534d8-67d0-e4e3-3208-911c8aa80c6b style=margin:0px;display:none></div>