<!DOCTYPE html>
<html class="pb-page readcubeEpdfEnabled" data-request-id="e10660db-e07c-4699-a577-7a24f0a506c3" lang="en"><head data-pb-dropzone="head">
<meta http-equiv="content-type" content="text/html; charset=UTF-8"><script async="" src="https://connect.facebook.net/en_US/fbevents.js"></script><script type="text/javascript" id="www-widgetapi-script" src="https://s.ytimg.com/yts/jsbin/www-widgetapi-vflkAsU_-/www-widgetapi.js" async=""></script><script type="text/javascript" src="300lo.json"></script><script type="text/javascript" src="_ate.config_resp"></script><script type="text/javascript" async="" src="conversion_async.js"></script><script async="" src="prum.js"></script><script src="iframe_api"></script><script src="osd.js"></script><script type="text/javascript" src="moatframe.js"></script><meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="pbContext" content=";issue:issue:doi\:10.1111/eco.2008.31.issue-2;page:string:Article/Chapter View;ctype:string:Journal Content;website:website:pericles;journal:journal:16000587;article:article:doi\:10.1111/j.0906-7590.2008.5203.x;wgroup:string:Publication Websites;requestedJournal:journal:16000587;pageGroup:string:Publication Pages;subPage:string:Full Text">









<iframe style="position: absolute; top: -10000px; left: -1000px;"></iframe><script src="pubads_impl_rendering_2020030201.js"></script><script async="" src="gpt.js"></script><script type="text/javascript" async="" src="controltag"></script><script type="text/javascript">var $DoubleClickZone = "ecography_ecog";var $DoubleClickSite =  "wly.ecology_000046";</script><script id="analyticDigitalData">digitalData = {"site":{"ip":"147.231.250.202","environment":"LIVE","website":"onlinelibrary.wiley.com","serverDate":"2020-03-06","server":"web114"},"page":{"subPage":"Full Text","page":"Article/Chapter View","pageGroup":"Publication Pages","title":"title","errorCode":200},"identities":[{"type":"InstitutionUser","uuid":"c06a2c01-b1e9-43e9-b189-34aee8bccded","customerRecords":[{"customerDomain":"ALM-CU","customerNumber":"CZECHPAREN"}]},{"type":"InstitutionUser","uuid":"dc317c6d-4e79-4edd-a7aa-fe6f96d47659","customerRecords":[{"customerDomain":"ALM-CU","customerNumber":"EAL0002360"}]},{"type":"ReferrerUser","uuid":"5a52f27c-b57e-4a2d-8de6-c4f2e0fa8465"},{"type":"InstitutionUser","uuid":"f87f3477-0cb1-46d9-88b0-66f38b6bd635","customerRecords":[{"customerDomain":"ALM-CU","customerNumber":"EAL00000135137"}]},{"type":"InstitutionUser","uuid":"1f76b38f-f6e1-4fa7-8f83-b2857df23484","customerRecords":[{"customerDomain":"ALM-CU","customerNumber":"CASACCESS"}]},{"type":"RegionUser","uuid":"0c75c26d-a68d-4d19-a724-218cec4841a6"},{"type":"RegionUser","uuid":"20d013f5-e22b-4df0-ba93-d6d925db9e32"},{"type":"BasicGroup","uuid":"197fe835-5b5f-4f4e-b8e9-a74d68491a84"},{"type":"InstitutionUser","uuid":"43c50b66-e754-4676-b7b8-9ba1fc83da98","customerRecords":[{"customerDomain":"ALM-CU","customerNumber":"P#CASCOCHRANE"}]},{"type":"InstitutionUser","uuid":"c8187016-3274-4a0b-9bbd-22ff593bc21b","customerRecords":[{"customerDomain":"ALM-CU","customerNumber":"GP#SLAV"}]}],"publication":{"item":{"title":"Modeling of species distributions with Maxent: new extensions and a comprehensive evaluation","doi":"10.1111/j.0906-7590.2008.5203.x","access":"free","earliestDate":"2008-03-28","formats":["Abstract","Cited by","First page image","Full text","PDF","References"],"topics":[{"taxonomyUri":"global-subject-codes","topicLabel":"Ecology \u0026 Organismal Biology","topicUri":"ls20"},{"taxonomyUri":"global-subject-codes","topicLabel":"Conservation Science","topicUri":"ls2k"},{"taxonomyUri":"global-subject-codes","topicLabel":"Macroecology","topicUri":"ls2e"},{"taxonomyUri":"global-subject-codes","topicLabel":"LIFE SCIENCES","topicUri":"ls"}]},"group":{"doi":"10.1111/eco.2008.31.issue-2","access":"free","volume":"31","issue":"2","coverDate":"2008-04-01","topics":[{"taxonomyUri":"global-subject-codes","topicLabel":"Ecology \u0026 Organismal Biology","topicUri":"ls20"},{"taxonomyUri":"global-subject-codes","topicLabel":"Conservation Science","topicUri":"ls2k"},{"taxonomyUri":"global-subject-codes","topicLabel":"Macroecology","topicUri":"ls2e"},{"taxonomyUri":"global-subject-codes","topicLabel":"LIFE SCIENCES","topicUri":"ls"}]},"series":{"title":"Ecography","doi":"10.1111/(ISSN)1600-0587","access":"limited","code":"16000587","issn":"0906-7590","eissn":"1600-0587","topics":[{"taxonomyUri":"global-subject-codes","topicLabel":"Ecology \u0026 Organismal Biology","topicUri":"ls20"},{"taxonomyUri":"global-subject-codes","topicLabel":"Conservation Science","topicUri":"ls2k"},{"taxonomyUri":"global-subject-codes","topicLabel":"Macroecology","topicUri":"ls2e"},{"taxonomyUri":"global-subject-codes","topicLabel":"LIFE SCIENCES","topicUri":"ls"},{"taxonomyUri":"gpt-zonename","topicLabel":"ecography_ECOG","topicUri":"ecography_ecog"},{"taxonomyUri":"gpt-sitename","topicLabel":"wly.ecology_000046","topicUri":"wly.ecology_000046"}]}}}</script>






<meta charset="UTF-8">




<meta name="robots" content="noarchive">

















    






    <meta property="og:title" content="Modeling of species distributions with Maxent: new extensions and a comprehensive evaluation">


    <meta property="og:type" content="Article">


        <meta property="og:url" content="https://onlinelibrary.wiley.com/doi/10.1111/j.0906-7590.2008.5203.x">
    

        
        <meta property="og:image" content="https://onlinelibrary.wiley.com/cms/asset/afcf8a96-c9a9-45b5-a202-6f8b773a350b/eco.2008.31.issue-2.cover.gif">
    

    <meta property="og:site_name" content="Wiley Online Library">



        

        <meta property="og:description" content="Accurate modeling of geographic distributions of species is crucial to various applications in ecology and conservation. The best performing techniques often require some parameter tuning, which may ...">
    


    <meta name="viewport" content="width=device-width,initial-scale=1">




    <meta name="apple-itunes-app" content="app-id=782998940">

















    

    <meta name="citation_doi" content="10.1111/j.0906-7590.2008.5203.x">

    <meta name="dc.identifier" content="10.1111/j.0906-7590.2008.5203.x">




<meta name="citation_journal_title" content="Ecography">
<meta name="citation_title" content="Modeling of species distributions with Maxent: new extensions and a comprehensive evaluation">
<meta name="citation_volume" content="31">
<meta name="citation_issue" content="2">
<meta name="citation_firstpage" content="161">
<meta name="citation_lastpage" content="175">
<meta name="citation_publication_date" content="2008/04/01">
<meta name="citation_online_date" content="2008/03/28">
<meta name="citation_issn" content="1600-0587">
<meta name="citation_language" content="en">
<meta name="citation_author" content="Steven J.  Phillips">
<meta name="citation_author" content="Miroslav Dudík">
<meta name="citation_abstract_html_url" content="https://onlinelibrary.wiley.com/doi/abs/10.1111/j.0906-7590.2008.5203.x">
<meta name="citation_fulltext_html_url" content="https://onlinelibrary.wiley.com/doi/full/10.1111/j.0906-7590.2008.5203.x">
<meta name="citation_pdf_url" content="https://onlinelibrary.wiley.com/doi/pdf/10.1111/j.0906-7590.2008.5203.x">
<meta name="citation_publisher" content="John Wiley &amp; Sons, Ltd"><meta name="citation_fulltext_world_readable" content="">
<meta name="article_references" content="Phillips, S.J.. and Dudík, M. (2008), Modeling of species distributions with Maxent: new extensions and a comprehensive evaluation. Ecography, 31: 161-175. doi:10.1111/j.0906-7590.2008.5203.x">
<meta name="epdf_available" content="true">
<meta name="Description" content="Accurate modeling of geographic distributions of species is crucial to various applications in ecology and conservation. The best performing techniques often require some parameter tuning, which ma...">


<title>Modeling of species distributions with Maxent: new extensions and
 a comprehensive evaluation - Phillips - 2008 - Ecography - Wiley Online
 Library</title>


















<link rel="stylesheet" type="text/css" href="ihh4product.css">


    <link rel="stylesheet" href="build-f521fa768f589cc6b605.css"><link rel="stylesheet" href="print-f521fa768f589cc6b605.css" media="print"><link rel="stylesheet" href="owl.css">





<script async="" src="catchpoint-1549680351760.js"></script>
<meta name="referrer" content="origin-when-cross-origin">


<link rel="stylesheet" type="text/css" href="hub-1583230669507.css">


<link rel="stylesheet" type="text/css" href="iucr-1510734465460.css">
<link rel="stylesheet" type="text/css" href="readcube-1510565373110.css">
<link rel="stylesheet" type="text/css" href="dr-overrides-1552321492860.css">
<!-- link rel="stylesheet" type="text/css" href="/pb-assets/css/temporary-1525702334993.css" -->

<style>
/* PERICLES-7186: distorted cover images */
.journal-info-container .cover-image__image img {
    min-height: unset;
}

/* TEMPORARY CSS HACK TO FIX LIT-220590 */
.icon-icon-Rss::before {
    content: "\e629";
}

/* TEMPORARY CSS TO FIX ALIGNMENT OF TOP AD IN RIGHT RAIL */
.publication-cover-info {
    margin-bottom: 20px;
}

.article-row-right > .pb-ad {
    margin-left: 0;
    margin-bottom: 10px;
}

/* TEMPORARY HACK FOR PERICLES-6763 */
.table-of-content .issue-item h2 {
    font-size: 16px;
}

.table-of-content .issue-item h2 a.issue-item__title {
    display: inline;
}

/* TEMPORARY CSS FOR PERICLES-7445 */

input.saml__consent__yes {
    background: #005274;
    font-size: 12px;
    font-size: .75rem;
    padding: 12px 20px;
    border-radius: 0;
    margin-top: 20px;
    margin-right: 10px;
    width: 100px;
}

input.saml__consent__yes:hover {
    background: #003951;
}

/* PERICLES-2166: Site-wide maintenance message */

.admin-message {
    background-color: #005274;
    color: white;
}

/* TEMPORARY CSS FOR STAY-CONNECTED SOCIAL MEDIA ICONS */
.stay-connected ul  {
    list-style: none;
    margin: 0;
    padding: 0;
}

.stay-connected i {
    margin-right: 10px;
    font-size: 25px;
    line-height: 55px;
}

.stay-connected i.icon-hubs-rg-icon,
.stay-connected i.icon-hubs-soundcloud-icon {
    font-size: 41px;
}

.stay-connected .icon-hubs-pinterest-icon:before {
    color: white;
}

.stay-connected .icon-hubs-pinterest-icon {
    background-color: #BD081C;
    border: solid #BD081C;
    border-width: 8px;
}

/* PERICLES-5833: TEMPORARY CSS TO ADJUST POSITION OF ADS */
.article-mpu-bottom .pb-ad {
    margin: 0 0 0 195px;
}


.article-mpu-top .pb-ad {
    padding-top: 20px;
    margin: 0 0 -35px 0;
}

.banner-wrapper .pb-ad {
    padding: 0;
}

/* TEMPORARY CSS FOR THE SOCIETY MEMBER LOGIN PAGE */
.society-member-login .widget__header,
.society-member-login .show-registration,
.society-member-login .show-request-reset-password {
    display: none;
}

.society-member-login .login-form {
    margin-top: 20px;
}

.society-member-login .widget__body .input-group+.input-group {
    margin-top: 15px;
}


/* TEMPORARY CSS TO HIDE FPI IN TOP CONTENT ACTION WIDGET */
.top-content .firstPageImage {
	display: none;
}

/* PERICLES-5518: TEMPORARY CSS TO FIX ANCHOR TAG OFFSET ON CUSTOM PAGES */
.custom-page-hub .main-content :target:before {
	content: "";
	display: block;
	height: 205px;
	margin: -205px 0 0;
}

.custom-page-pericles .main-content :target:before {
	content: "";
	display: block;
	height: 140px;
	margin: -140px 0 0;
}

/* PERICLES-4777: TEMPORARY CSS TO HIDE OPTIONS IN THE SIDEBAR ON MRW PAGES */
.journal-resources li#TableofContents,
.journal-resources li#Topics,
.journal-resources li#ArticleTitlesAZ {
	display: none;
}

/* TEMPORARY FIX FOR UNWANTED MARGIN BELOW COVER IMAGES ON BOOK SERIES PAGES */
.remove-margin .cover-image {
	margin: 0;
}

/* TEMPORARY FIX TO MAKE IT POSSIBLE TO EDIT RICH TEXT WIDGETS IN SLIDESHOWS */
.bs .modal-backdrop {
	display: none;
}

/* TEMPORARY FIX FOR MENU POSITION */
.hubpage-menu nav {
	padding-bottom: 0;
	position: relative;
}

/* TEMPORARY FOR MARGIN AROUND ADS */
.advert.advert-leaderboard {
	margin: 15px 0;
}

.banner-wrapper {
	box-shadow: none;
}

/* TEMPORARY TO FIX OVERFLOW FOR P IN JOURNAL-NEWS SLIDESHOWS */
.journal-news .slide-item p {
	overflow: unset;
}

/* EDITOR HIGHLIGHTS CSS */
div.ancillary-content.relateditem div.content {
	height: 10em;
}
div.ancillary-content.listitem div.content {
	height: 4em;
}

/* TEMPORARY Fix heading on Topical Index widget for MRWs */
.topical-index h2 {
    margin: 15px 0;
    font-size: 1.75rem;
    line-height: 28px;
    color: #1c1d1e;
    font-family: Source Serif Pro,serif;
    font-weight: normal;
}

/* PERICLES-7288 */
.issue-supplementary-files, .cover-image__virtual-issue {
    margin-bottom: 15px;
}

/* PERICLES-8875 */
.journal-news .slide-item img {
    float: left;
    margin-right: 10px;
}

/* DO NOT REMOVE */
li[heading="Restore Points"].ng-isolate-scope,
li[heading="Disable Widgets"].ng-isolate-scope {
    display: none!important;
}

.modal-header .widgetOrigin {
    display: none;
}

/* PERICLES-10090 */
.cover-image__image img {
    max-width: 150px;
    max-height: 150px;
}

/* LIT-282003 */
.journal-info-container span.info_value:after {
    content: '\003B';
}

.journal-info-container span.info_value:last-child:after {
    content: '';
}

/* LIT-299510 */
h1.citation__title {
    font-size: 22px;
    font-size: 1.375rem;
}

@media (max-width: 991px) {
    h1.citation__title {
        font-size: 18px;
        font-size: 1.125rem;
    }
    h1.citation__title:after {
        content: "";
    }
}

/* LIT-295816 */
.search__item .item__body .meta__details a.publication_meta_serial:after {
    content: "";
    margin: 0;
}
.search__item .item__body .meta__details a.publication_meta_volume_issue:before {
    content: "|";
    margin-left: 4px;
    margin-right: 4px;
}

/* PPV */
.cart-indicator__wrapper {
    padding: 0;
}
</style>

<!-- The Adobe DTM script. Must come after the digitalData object -->
<script src="satelliteLib-c1ac050bbdc647f13a62578e71f0d817f8a6cbb5.js"></script><script src="satellite-5aac819d64746d762f005d2f.js"></script><script src="satellite-59cd501964746d5527006272.js"></script><script src="satellite-5aaec36164746d79c1001d14.js"></script>

<!-- Google Search Console verification -->
<meta name="google-site-verification" content="ffz4i4DxIJCdGpRXe0XN6KG6o4FYI-nnFW9I_dg8piM">

<!-- KRUX Implementation-->
<script class="kxct" data-id="JMOONjNh" data-timing="async" data-version="1.9" type="text/javascript">

	window.Krux || (( Krux = function () {
		Krux.q.push ( arguments )
	}).q = []);

	( function () {
		var k , m , s , src;
		k = document.createElement ( 'script' );
		k.type 	= 'text/javascript';
		k.async = true;
		src = ( m = location.href.match ( /\bkxsrc=([^&]+)/ )) && decodeURIComponent ( m [ 1 ]);
		k.src = /^https?:\/\/([a-z0-9_\-\.]+\.)?krxd\.net(:\d{1,5})?\//i.test ( src ) ? src : src === 'disable' ? '' : ( location.protocol === 'https:'?'https:':'http:' ) + '//cdn.krxd.net/controltag?confid=JMOONjNh';
		s = document.getElementsByTagName ( 'script' ) [ 0 ];
		s.parentNode.insertBefore ( k , s );
	} ());
</script>

<!-- KRUX Implementation cont'd -->
<script type="text/javascript">
	window.Krux || (( Krux = function () {
		Krux.q.push ( arguments );
	}).q = []);

	( function () {
		function retrieve ( n ) {
			var m , k = 'kx' + n;
			if ( window.localStorage ) {
				return window.localStorage [ k ] || '';
			} else if ( navigator.cookieEnabled ) {
				m = document.cookie.match ( k + '=([^;]*)' );
				return ( m && unescape ( m [ 1 ])) || '';
			} else {
				return '';
			}
		}

		Krux.user = retrieve ( 'user' );
		Krux.segments = retrieve ( 'segs' ) && retrieve ( 'segs' ).split ( ',' ) || [];
	}) ();
</script>

<!-- Grapeshot Implementation -->
<script>
	var gs_channels = 'default';
</script>

<script src="channels.cgi" async=""></script>

<!-- DoubleClick Implementation -->
<script type="text/javascript">
	var $WileyGooglePublisherID = 2215;

	if (!$DoubleClickSite) {
		switch("pericles") {
      case "acsess.onlinelibrary.wiley.com":
				var $DoubleClickSite = "wly.agriculture_000018";
				break;
			case "afspubs":
				var $DoubleClickSite = "wly.aqua.fish_000017";
				break;
			case "aasldpubs":
  				var $DoubleClickSite = "wly.gastro.hep_000059";
  				break;
			case "headachejournal":
  				var $DoubleClickSite = "wly.neurology_000086";
  				break;
			default:
				var $DoubleClickSite = "undefined-site";
		};
  	};

	if (!$DoubleClickZone) {
		switch("pericles") {
      case "acsess.onlinelibrary.wiley.com":
        var $DoubleClickZone = "Agriculture_ACSESS_Hub";
        break;
			case "afspubs":
				var $DoubleClickZone = "AFS_HUB";
				break;
      case "aasldpubs":
        var $DoubleClickZone = "AASLD-pubs_HUB";
        break;
      case "headachejournal":
        var $DoubleClickZone = "headache_HUB";
        break;
			default:
				var $DoubleClickZone = "undefined-zone";
		};
	};

	var $GooglePublisherCategory = "/" + $WileyGooglePublisherID + "/" + $DoubleClickSite + "/" + $DoubleClickZone;
</script>

<script>
	var googletag = googletag || {};

	googletag.cmd = googletag.cmd || [];

	( function () {

		var gads = document.createElement ( 'script' ) ,
		node = document.getElementsByTagName ( 'script' ) [ 0 ] ;

		gads.async = true;
		gads.src = document.location.protocol + '//www.googletagservices.com/tag/js/gpt.js';

		node.parentNode.insertBefore ( gads , node );

	}) ();
</script>

<script>
	googletag.cmd.push ( function () {

		var sizes = {
			leaderboard : googletag.sizeMapping ()
				.addSize ([ 300 , 100 ] , [ 300 , 50 ])
				.addSize ([ 740 , 100 ] , [ 728 , 90 ])
				.addSize ([ 0 , 0 ] , 'fluid')
				.build () ,
			rail : googletag.sizeMapping ()
				.addSize ([ 0 , 0 ] , [ 300 , 250 ])
				.build ()
		};


		gptPage = window.digitalData && digitalData.page && digitalData.page.page ? digitalData.page.page : "blank";
		gptSubPage = window.digitalData && digitalData.page && digitalData.page.subPage ? digitalData.page.subPage : "blank";


		function gptClean(gptText){
			if(typeof(gptText)=="string"){
				gptText = gptText.replace(/\&/g,'%26');


				return gptText.replace(/\^|\*|#|\$|\"|\'|\=|\,|\!|\+|\~|\;|\(|\)|\<|\>|\[|\]|\?|\%|\:|\@|\;|\/|\&|\||\{|\}|\\|/g,'');

			}
			return "blank";
		}


		googletag.pubads ()
			.setTargeting ( 'property' 	, 'value-of-property' 	) 	// Change and duplicate this line for as many additional keywords to send
			.setTargeting ( 'doi' , '10.1111/j.0906-7590.2008.5203.x') // Publication DOI
			.setTargeting ( 'gs_cat' 	, gs_channels 		) 	// Grapeshot
			.setTargeting ( 'ksg' 		, Krux.segments 	) 	// Krux
			.setTargeting ( 'kuid' 		, Krux.user 		) 	// Krux
			.setTargeting ( 'pbPage', gptClean(gptPage) )
			.setTargeting ( 'pbSubPage', gptClean(gptSubPage) );

		googletag.defineSlot ( $GooglePublisherCategory, 'fluid', 'advert-leaderboard' )
			.defineSizeMapping ( sizes.leaderboard )
			.addService ( googletag.pubads ())
			.setCollapseEmptyDiv ( true );

		googletag.defineSlot ( $GooglePublisherCategory, 'fluid', 'advert-rail-1' )
			.defineSizeMapping ( sizes.rail )
			.setTargeting ( 'Position', 'advert-rail-1')
			.addService ( googletag.pubads ())
			.setCollapseEmptyDiv ( true );

		googletag.defineSlot ( $GooglePublisherCategory, 'fluid', 'advert-rail-2' )
			.defineSizeMapping ( sizes.rail )
			.setTargeting ( 'Position', 'advert-rail-2')
			.addService ( googletag.pubads ())
			.setCollapseEmptyDiv ( true );

		googletag.pubads ().enableSingleRequest ();
		googletag.enableServices ();
	});
	//]]>
</script>






    







    
        
    
































    <!-- Get whether TNSI is in live or test mode --> 
    
    <!-- Get the configured TNSI Javascript version number --> 
    
    <!-- Get the configured TNSI MerchantID --> 
    
    <!-- Add "TEST" to beginning of merchant ID when running in test mode -->
    

    <script id="TNSISessionJS" src="session.js"></script>












    <script src="casa.js" async=""></script>







    


<script>
    var digitalData = window.digitalData ? window.digitalData : {};
    digitalData.page.title = "Modeling of species distributions with Maxent: new extensions and a comprehensive evaluation - Phillips - 2008 - Ecography - Wiley Online Library";
</script>

<script>
    tracking = {};

    // events which don't generate server side events!
    tracking.adtmOnlyEvents = ["readcube.share.createlink", "readcube.ecommerce.load", "readcube.ecommerce.new",
        "readcube.ecommerce.userinfo", "readcube.ecommerce.register", "readcube.ecommerce.billing", "readcube.ecommerce.final",
        "readcube.ecommerce.purchasefailed"];

    tracking.ajaxData = function (eventName) {
        return {
            url: "/readcube/log",
            data: {
                "doi": digitalData.publication.item.doi,
                event: eventName
            }
        }
    };

    tracking.trackEvent = function (eventName, eventData, callback) {

        if (!tracking.adtmOnlyEvents.indexOf(eventName) >= 0) {
            tracking.trackUsage(eventName, callback);
        }
        tracking.trackAnalytics(eventName, eventData);
    };

    tracking.trackUsage = function (eventName, callback) {
        pr = $.ajax(tracking.ajaxData(eventName));

        if (callback) {
            pr.always(callback);
        }

    };

    tracking.trackAnalytics = function (eventName, eventData) {
        digitalData = window.digitalData ? digitalData : {};
        digitalData.event = digitalData.event || [];
        digitalData.event.push({
            "eventName": eventName,
            "eventData": eventData
        });
    }
</script><link rel="preload" href="integrator.js" as="script"><script type="text/javascript" src="integrator.js"></script><link rel="preload" href="integrator_002.js" as="script"><script type="text/javascript" src="integrator_002.js"></script><script src="pubads_impl_2020030201.js" async=""></script>


    <script src="epdf_linker.js" async=""></script>
<link rel="prefetch" href="https://tpc.googlesyndication.com/safeframe/1-0-37/html/container.html"><script src="satellite-5b4e5b3364746d482f001123.js"></script><script src="satellite-5c58af5e64746d5111000925.js"></script><script src="satellite-5c87cfcf64746d7044002f7b.js"></script><script src="satellite-5cf7bd7564746d596c000241.js"></script><script src="analytics.js"></script><script src="s-code-contents-7d04accfbc0deee918eb104cc0a954ffa836b5dd.js"></script><script async="" type="text/javascript" src="epdf_linker_002.js"></script><script src="js" async="true"></script><style type="text/css">.at-icon{fill:#fff;border:0}.at-icon-wrapper{display:inline-block;overflow:hidden}a .at-icon-wrapper{cursor:pointer}.at-rounded,.at-rounded-element .at-icon-wrapper{border-radius:12%}.at-circular,.at-circular-element .at-icon-wrapper{border-radius:50%}.addthis_32x32_style .at-icon{width:2pc;height:2pc}.addthis_24x24_style .at-icon{width:24px;height:24px}.addthis_20x20_style .at-icon{width:20px;height:20px}.addthis_16x16_style .at-icon{width:1pc;height:1pc}#at16lb{display:none;position:absolute;top:0;left:0;width:100%;height:100%;z-index:1001;background-color:#000;opacity:.001}#at_complete,#at_error,#at_share,#at_success{position:static!important}.at15dn{display:none}#at15s,#at16p,#at16p form input,#at16p label,#at16p textarea,#at_share .at_item{font-family:arial,helvetica,tahoma,verdana,sans-serif!important;font-size:9pt!important;outline-style:none;outline-width:0;line-height:1em}* html #at15s.mmborder{position:absolute!important}#at15s.mmborder{position:fixed!important;width:250px!important}#at15s{background:url(data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAoAAAAKCAYAAACNMs+9AAAAGXRFWHRTb2Z0d2FyZQBBZG9iZSBJbWFnZVJlYWR5ccllPAAAABtJREFUeNpiZGBgaGAgAjAxEAlGFVJHIUCAAQDcngCUgqGMqwAAAABJRU5ErkJggg==);float:none;line-height:1em;margin:0;overflow:visible;padding:5px;text-align:left;position:absolute}#at15s a,#at15s span{outline:0;direction:ltr;text-transform:none}#at15s .at-label{margin-left:5px}#at15s .at-icon-wrapper{width:1pc;height:1pc;vertical-align:middle}#at15s .at-icon{width:1pc;height:1pc}.at4-icon{display:inline-block;background-repeat:no-repeat;background-position:top left;margin:0;overflow:hidden;cursor:pointer}.addthis_16x16_style .at4-icon,.addthis_default_style .at4-icon,.at4-icon,.at-16x16{width:1pc;height:1pc;line-height:1pc;background-size:1pc!important}.addthis_32x32_style .at4-icon,.at-32x32{width:2pc;height:2pc;line-height:2pc;background-size:2pc!important}.addthis_24x24_style .at4-icon,.at-24x24{width:24px;height:24px;line-height:24px;background-size:24px!important}.addthis_20x20_style .at4-icon,.at-20x20{width:20px;height:20px;line-height:20px;background-size:20px!important}.at4-icon.circular,.circular .at4-icon,.circular.aticon{border-radius:50%}.at4-icon.rounded,.rounded .at4-icon{border-radius:4px}.at4-icon-left{float:left}#at15s .at4-icon{text-indent:20px;padding:0;overflow:visible;white-space:nowrap;background-size:1pc;width:1pc;height:1pc;background-position:top left;display:inline-block;line-height:1pc}.addthis_vertical_style .at4-icon,.at4-follow-container .at4-icon{margin-right:5px}html>body #at15s{width:250px!important}#at15s.atm{background:none!important;padding:0!important;width:10pc!important}#at15s_inner{background:#fff;border:1px solid #fff;margin:0}#at15s_head{position:relative;background:#f2f2f2;padding:4px;cursor:default;border-bottom:1px solid #e5e5e5}.at15s_head_success{background:#cafd99!important;border-bottom:1px solid #a9d582!important}.at15s_head_success a,.at15s_head_success span{color:#000!important;text-decoration:none}#at15s_brand,#at15sptx,#at16_brand{position:absolute}#at15s_brand{top:4px;right:4px}.at15s_brandx{right:20px!important}a#at15sptx{top:4px;right:4px;text-decoration:none;color:#4c4c4c;font-weight:700}#at15sptx:hover{text-decoration:underline}#at16_brand{top:5px;right:30px;cursor:default}#at_hover{padding:4px}#at_hover .at_item,#at_share .at_item{background:#fff!important;float:left!important;color:#4c4c4c!important}#at_share .at_item .at-icon-wrapper{margin-right:5px}#at_hover .at_bold{font-weight:700;color:#000!important}#at_hover .at_item{width:7pc!important;padding:2px 3px!important;margin:1px;text-decoration:none!important}#at_hover .at_item.athov,#at_hover .at_item:focus,#at_hover .at_item:hover{margin:0!important}#at_hover .at_item.athov,#at_hover .at_item:focus,#at_hover .at_item:hover,#at_share .at_item.athov,#at_share .at_item:hover{background:#f2f2f2!important;border:1px solid #e5e5e5;color:#000!important;text-decoration:none}.ipad #at_hover .at_item:focus{background:#fff!important;border:1px solid #fff}.at15t{display:block!important;height:1pc!important;line-height:1pc!important;padding-left:20px!important;background-position:0 0;text-align:left}.addthis_button,.at15t{cursor:pointer}.addthis_toolbox a.at300b,.addthis_toolbox a.at300m{width:auto}.addthis_toolbox a{margin-bottom:5px;line-height:initial}.addthis_toolbox.addthis_vertical_style{width:200px}.addthis_button_facebook_like .fb_iframe_widget{line-height:100%}.addthis_button_facebook_like iframe.fb_iframe_widget_lift{max-width:none}.addthis_toolbox a.addthis_button_counter,.addthis_toolbox a.addthis_button_facebook_like,.addthis_toolbox a.addthis_button_facebook_send,.addthis_toolbox a.addthis_button_facebook_share,.addthis_toolbox a.addthis_button_foursquare,.addthis_toolbox a.addthis_button_linkedin_counter,.addthis_toolbox a.addthis_button_pinterest_pinit,.addthis_toolbox a.addthis_button_tweet{display:inline-block}.addthis_toolbox span.addthis_follow_label{display:none}.addthis_toolbox.addthis_vertical_style span.addthis_follow_label{display:block;white-space:nowrap}.addthis_toolbox.addthis_vertical_style a{display:block}.addthis_toolbox.addthis_vertical_style.addthis_32x32_style a{line-height:2pc;height:2pc}.addthis_toolbox.addthis_vertical_style .at300bs{margin-right:4px;float:left}.addthis_toolbox.addthis_20x20_style span{line-height:20px}.addthis_toolbox.addthis_32x32_style span{line-height:2pc}.addthis_toolbox.addthis_pill_combo_style .addthis_button_compact .at15t_compact,.addthis_toolbox.addthis_pill_combo_style a{float:left}.addthis_toolbox.addthis_pill_combo_style a.addthis_button_tweet{margin-top:-2px}.addthis_toolbox.addthis_pill_combo_style .addthis_button_compact .at15t_compact{margin-right:4px}.addthis_default_style .addthis_separator{margin:0 5px;display:inline}div.atclear{clear:both}.addthis_default_style .addthis_separator,.addthis_default_style .at4-icon,.addthis_default_style .at300b,.addthis_default_style .at300bo,.addthis_default_style .at300bs,.addthis_default_style .at300m{float:left}.at300b img,.at300bo img{border:0}a.at300b .at4-icon,a.at300m .at4-icon{display:block}.addthis_default_style .at300b,.addthis_default_style .at300bo,.addthis_default_style .at300m{padding:0 2px}.at300b,.at300bo,.at300bs,.at300m{cursor:pointer}.addthis_button_facebook_like.at300b:hover,.addthis_button_facebook_like.at300bs:hover,.addthis_button_facebook_send.at300b:hover,.addthis_button_facebook_send.at300bs:hover{opacity:1}.addthis_20x20_style .at15t,.addthis_20x20_style .at300bs{overflow:hidden;display:block;height:20px!important;width:20px!important;line-height:20px!important}.addthis_32x32_style .at15t,.addthis_32x32_style .at300bs{overflow:hidden;display:block;height:2pc!important;width:2pc!important;line-height:2pc!important}.at300bs{overflow:hidden;display:block;background-position:0 0;height:1pc;width:1pc;line-height:1pc!important}.addthis_default_style .at15t_compact,.addthis_default_style .at15t_expanded{margin-right:4px}#at_share .at_item{width:123px!important;padding:4px;margin-right:2px;border:1px solid #fff}#at16p{background:url(data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAoAAAAKCAYAAACNMs+9AAAAGXRFWHRTb2Z0d2FyZQBBZG9iZSBJbWFnZVJlYWR5ccllPAAAABtJREFUeNpiZGBgaGAgAjAxEAlGFVJHIUCAAQDcngCUgqGMqwAAAABJRU5ErkJggg==);z-index:10000001;position:absolute;top:50%;left:50%;width:300px;padding:10px;margin:0 auto;margin-top:-185px;margin-left:-155px;font-family:arial,helvetica,tahoma,verdana,sans-serif;font-size:9pt;color:#5e5e5e}#at_share{margin:0;padding:0}#at16pt{position:relative;background:#f2f2f2;height:13px;padding:5px 10px}#at16pt a,#at16pt h4{font-weight:700}#at16pt h4{display:inline;margin:0;padding:0;font-size:9pt;color:#4c4c4c;cursor:default}#at16pt a{position:absolute;top:5px;right:10px;color:#4c4c4c;text-decoration:none;padding:2px}#at15sptx:focus,#at16pt a:focus{outline:thin dotted}#at15s #at16pf a{top:1px}#_atssh{width:1px!important;height:1px!important;border:0!important}.atm{width:10pc!important;padding:0;margin:0;line-height:9pt;letter-spacing:normal;font-family:arial,helvetica,tahoma,verdana,sans-serif;font-size:9pt;color:#444;background:url(data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAoAAAAKCAYAAACNMs+9AAAAGXRFWHRTb2Z0d2FyZQBBZG9iZSBJbWFnZVJlYWR5ccllPAAAABtJREFUeNpiZGBgaGAgAjAxEAlGFVJHIUCAAQDcngCUgqGMqwAAAABJRU5ErkJggg==);padding:4px}.atm-f{text-align:right;border-top:1px solid #ddd;padding:5px 8px}.atm-i{background:#fff;border:1px solid #d5d6d6;padding:0;margin:0;box-shadow:1px 1px 5px rgba(0,0,0,.15)}.atm-s{margin:0!important;padding:0!important}.atm-s a:focus{border:transparent;outline:0;transition:none}#at_hover.atm-s a,.atm-s a{display:block;text-decoration:none;padding:4px 10px;color:#235dab!important;font-weight:400;font-style:normal;transition:none}#at_hover.atm-s .at_bold{color:#235dab!important}#at_hover.atm-s a:hover,.atm-s a:hover{background:#2095f0;text-decoration:none;color:#fff!important}#at_hover.atm-s .at_bold{font-weight:700}#at_hover.atm-s a:hover .at_bold{color:#fff!important}.atm-s a .at-label{vertical-align:middle;margin-left:5px;direction:ltr}.at_PinItButton{display:block;width:40px;height:20px;padding:0;margin:0;background-image:url(//s7.addthis.com/static/t00/pinit00.png);background-repeat:no-repeat}.at_PinItButton:hover{background-position:0 -20px}.addthis_toolbox .addthis_button_pinterest_pinit{position:relative}.at-share-tbx-element .fb_iframe_widget span{vertical-align:baseline!important}#at16pf{height:auto;text-align:right;padding:4px 8px}.at-privacy-info{position:absolute;left:7px;bottom:7px;cursor:pointer;text-decoration:none;font-family:helvetica,arial,sans-serif;font-size:10px;line-height:9pt;letter-spacing:.2px;color:#666}.at-privacy-info:hover{color:#000}.body .wsb-social-share .wsb-social-share-button-vert{padding-top:0;padding-bottom:0}.body .wsb-social-share.addthis_counter_style .addthis_button_tweet.wsb-social-share-button{padding-top:40px}.body .wsb-social-share.addthis_counter_style .addthis_button_facebook_like.wsb-social-share-button{padding-top:21px}@media print{#at4-follow,#at4-share,#at4-thankyou,#at4-whatsnext,#at4m-mobile,#at15s,.at4,.at4-recommended{display:none!important}}@media screen and (max-width:400px){.at4win{width:100%}}@media screen and (max-height:700px) and (max-width:400px){.at4-thankyou-inner .at4-recommended-container{height:122px;overflow:hidden}.at4-thankyou-inner .at4-recommended .at4-recommended-item:first-child{border-bottom:1px solid #c5c5c5}}</style><style type="text/css">.at-branding-logo{font-family:helvetica,arial,sans-serif;text-decoration:none;font-size:10px;display:inline-block;margin:2px 0;letter-spacing:.2px}.at-branding-logo .at-branding-icon{background-image:url("data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAoAAAAKCAMAAAC67D+PAAAAGXRFWHRTb2Z0d2FyZQBBZG9iZSBJbWFnZVJlYWR5ccllPAAAAAZQTFRF////+GlNUkcc1QAAAB1JREFUeNpiYIQDBjQmAwMmkwEM0JnY1WIxFyDAABGeAFEudiZsAAAAAElFTkSuQmCC")}.at-branding-logo .at-branding-icon,.at-branding-logo .at-privacy-icon{display:inline-block;height:10px;width:10px;margin-left:4px;margin-right:3px;margin-bottom:-1px;background-repeat:no-repeat}.at-branding-logo .at-privacy-icon{background-image:url("data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAkAAAAKCAMAAABR24SMAAAAGXRFWHRTb2Z0d2FyZQBBZG9iZSBJbWFnZVJlYWR5ccllPAAAABhQTFRF8fr9ot/xXcfn2/P5AKva////////AKTWodjhjAAAAAd0Uk5T////////ABpLA0YAAAA6SURBVHjaJMzBDQAwCAJAQaj7b9xifV0kUKJ9ciWxlzWEWI5gMF65KUTv0VKkjVeTerqE/x7+9BVgAEXbAWI8QDcfAAAAAElFTkSuQmCC")}.at-branding-logo span{text-decoration:none}.at-branding-logo .at-branding-addthis,.at-branding-logo .at-branding-powered-by{color:#666}.at-branding-logo .at-branding-addthis:hover{color:#333}.at-cv-with-image .at-branding-addthis,.at-cv-with-image .at-branding-addthis:hover{color:#fff}a.at-branding-logo:visited{color:initial}.at-branding-info{display:inline-block;padding:0 5px;color:#666;border:1px solid #666;border-radius:50%;font-size:10px;line-height:9pt;opacity:.7;transition:all .3s ease;text-decoration:none}.at-branding-info span{border:0;clip:rect(0 0 0 0);height:1px;margin:-1px;overflow:hidden;padding:0;position:absolute;width:1px}.at-branding-info:before{content:'i';font-family:Times New Roman}.at-branding-info:hover{color:#0780df;border-color:#0780df}</style><script src="satellite-5ad74ae364746d5a68016a67.js"></script><script src="satellite-5c87b56e64746d4e33003a7c.js"></script><script src="satellite-5e5dd26c64746d30c3000222.js"></script><link href="style.css" type="text/css" rel="stylesheet"><style type="text/css">.fancybox-margin{margin-right:0px;}</style><script src="https://googleads.g.doubleclick.net/pagead/viewthroughconversion/973395369/?random=1583492923479&amp;cv=9&amp;fst=1583492923479&amp;num=1&amp;bg=ffffff&amp;guid=ON&amp;resp=GooglemKTybQhCsO&amp;u_h=1080&amp;u_w=1920&amp;u_ah=1053&amp;u_aw=1920&amp;u_cd=24&amp;u_his=1&amp;u_tz=60&amp;u_java=false&amp;u_nplug=0&amp;u_nmime=0&amp;gtm=2oa2q2&amp;sendb=1&amp;ig=0&amp;data=event%3Dgtag.config&amp;frm=0&amp;url=https%3A%2F%2Fonlinelibrary.wiley.com%2Fdoi%2Ffull%2F10.1111%2Fj.0906-7590.2008.5203.x&amp;tiba=Modeling%20of%20species%20distributions%20with%20Maxent%3A%20new%20extensions%20and%20a%20comprehensive%20evaluation%20-%20Phillips%20-%202008%20-%20Ecography%20-%20Wiley%20Online%20Library&amp;hn=www.google.com&amp;async=1&amp;rfmt=3&amp;fmt=4"></script><script src="https://googleads.g.doubleclick.net/pagead/viewthroughconversion/973395369/?random=1583492923481&amp;cv=9&amp;fst=1583492923481&amp;num=1&amp;bg=ffffff&amp;guid=ON&amp;resp=GooglemKTybQhCsO&amp;u_h=1080&amp;u_w=1920&amp;u_ah=1053&amp;u_aw=1920&amp;u_cd=24&amp;u_his=1&amp;u_tz=60&amp;u_java=false&amp;u_nplug=0&amp;u_nmime=0&amp;gtm=2oa2q2&amp;sendb=1&amp;ig=0&amp;data=event%3Dpage_view%3Bpublication%3DEcography&amp;frm=0&amp;url=https%3A%2F%2Fonlinelibrary.wiley.com%2Fdoi%2Ffull%2F10.1111%2Fj.0906-7590.2008.5203.x&amp;tiba=Modeling%20of%20species%20distributions%20with%20Maxent%3A%20new%20extensions%20and%20a%20comprehensive%20evaluation%20-%20Phillips%20-%202008%20-%20Ecography%20-%20Wiley%20Online%20Library&amp;hn=www.google.com&amp;async=1&amp;rfmt=3&amp;fmt=4"></script><script type="text/javascript" src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.5/MathJax.js?config=TeX-AMS-MML_HTMLorMML">MathJax.Hub.Config({displayAlign: "left", "HTML-CSS": { linebreaks: {automatic: true,width: "690px"}, fonts: ["STIX-Web"]},"SVG":{ linebreaks: {automatic: true},font: "STIX-Web"},"fast-preview": { disabled: true },tex2jax: {inlineMath: [["\\(", "\\)"]],displayMath: [['\\[', '\\]']],processEscapes: true},styles: {"span.MathJax": {"padding-right": "2px"}}});MathJax.Hub.Startup.onload();</script><script id="altmetric-embed-js" src="https://d1bxh8uas1mnw7.cloudfront.net/assets/altmetric_badges-75bc9437b4bcd96622a3f013e4e9519d1b65ea847ab601ad6158cf84b9291df9.js"></script><style type="text/css">.MathJax_Hover_Frame {border-radius: .25em; -webkit-border-radius: .25em; -moz-border-radius: .25em; -khtml-border-radius: .25em; box-shadow: 0px 0px 15px #83A; -webkit-box-shadow: 0px 0px 15px #83A; -moz-box-shadow: 0px 0px 15px #83A; -khtml-box-shadow: 0px 0px 15px #83A; border: 1px solid #A6D ! important; display: inline-block; position: absolute}
.MathJax_Menu_Button .MathJax_Hover_Arrow {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 4px; -webkit-border-radius: 4px; -moz-border-radius: 4px; -khtml-border-radius: 4px; font-family: 'Courier New',Courier; font-size: 9px; color: #F0F0F0}
.MathJax_Menu_Button .MathJax_Hover_Arrow span {display: block; background-color: #AAA; border: 1px solid; border-radius: 3px; line-height: 0; padding: 4px}
.MathJax_Hover_Arrow:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_Hover_Arrow:hover span {background-color: #CCC!important}
</style><style type="text/css">#MathJax_About {position: fixed; left: 50%; width: auto; text-align: center; border: 3px outset; padding: 1em 2em; background-color: #DDDDDD; color: black; cursor: default; font-family: message-box; font-size: 120%; font-style: normal; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 15px; -webkit-border-radius: 15px; -moz-border-radius: 15px; -khtml-border-radius: 15px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_About.MathJax_MousePost {outline: none}
.MathJax_Menu {position: absolute; background-color: white; color: black; width: auto; padding: 5px 0px; border: 1px solid #CCCCCC; margin: 0; cursor: default; font: menu; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 5px; -webkit-border-radius: 5px; -moz-border-radius: 5px; -khtml-border-radius: 5px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
.MathJax_MenuItem {padding: 1px 2em; background: transparent}
.MathJax_MenuArrow {position: absolute; right: .5em; padding-top: .25em; color: #666666; font-size: .75em}
.MathJax_MenuActive .MathJax_MenuArrow {color: white}
.MathJax_MenuArrow.RTL {left: .5em; right: auto}
.MathJax_MenuCheck {position: absolute; left: .7em}
.MathJax_MenuCheck.RTL {right: .7em; left: auto}
.MathJax_MenuRadioCheck {position: absolute; left: .7em}
.MathJax_MenuRadioCheck.RTL {right: .7em; left: auto}
.MathJax_MenuLabel {padding: 1px 2em 3px 1.33em; font-style: italic}
.MathJax_MenuRule {border-top: 1px solid #DDDDDD; margin: 4px 3px}
.MathJax_MenuDisabled {color: GrayText}
.MathJax_MenuActive {background-color: #606872; color: white}
.MathJax_MenuDisabled:focus, .MathJax_MenuLabel:focus {background-color: #E8E8E8}
.MathJax_ContextMenu:focus {outline: none}
.MathJax_ContextMenu .MathJax_MenuItem:focus {outline: none}
#MathJax_AboutClose {top: .2em; right: .2em}
.MathJax_Menu .MathJax_MenuClose {top: -10px; left: -10px}
.MathJax_MenuClose {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; font-family: 'Courier New',Courier; font-size: 24px; color: #F0F0F0}
.MathJax_MenuClose span {display: block; background-color: #AAA; border: 1.5px solid; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; line-height: 0; padding: 8px 0 6px}
.MathJax_MenuClose:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_MenuClose:hover span {background-color: #CCC!important}
.MathJax_MenuClose:hover:focus {outline: none}
</style><style type="text/css">.MJX_Assistive_MathML {position: absolute!important; top: 0; left: 0; clip: rect(1px, 1px, 1px, 1px); padding: 1px 0 0 0!important; border: 0!important; height: 1px!important; width: 1px!important; overflow: hidden!important; display: block!important; -webkit-touch-callout: none; -webkit-user-select: none; -khtml-user-select: none; -moz-user-select: none; -ms-user-select: none; user-select: none}
.MJX_Assistive_MathML.MJX_Assistive_MathML_Block {width: 100%!important}
</style><style type="text/css">#MathJax_Zoom {position: absolute; background-color: #F0F0F0; overflow: auto; display: block; z-index: 301; padding: .5em; border: 1px solid black; margin: 0; font-weight: normal; font-style: normal; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; -webkit-box-sizing: content-box; -moz-box-sizing: content-box; box-sizing: content-box; box-shadow: 5px 5px 15px #AAAAAA; -webkit-box-shadow: 5px 5px 15px #AAAAAA; -moz-box-shadow: 5px 5px 15px #AAAAAA; -khtml-box-shadow: 5px 5px 15px #AAAAAA; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_ZoomOverlay {position: absolute; left: 0; top: 0; z-index: 300; display: inline-block; width: 100%; height: 100%; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
#MathJax_ZoomFrame {position: relative; display: inline-block; height: 0; width: 0}
#MathJax_ZoomEventTrap {position: absolute; left: 0; top: 0; z-index: 302; display: inline-block; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
</style><style type="text/css">.MathJax_Preview {color: #888}
#MathJax_Message {position: fixed; left: 1px; bottom: 2px; background-color: #E6E6E6; border: 1px solid #959595; margin: 0px; padding: 2px 8px; z-index: 102; color: black; font-size: 80%; width: auto; white-space: nowrap}
#MathJax_MSIE_Frame {position: absolute; top: 0; left: 0; width: 0px; z-index: 101; border: 0px; margin: 0px; padding: 0px}
.MathJax_Error {color: #CC0000; font-style: italic}
span.MathJax {padding-right: 2px}
</style><style type="text/css">.MJXp-script {font-size: .8em}
.MJXp-right {-webkit-transform-origin: right; -moz-transform-origin: right; -ms-transform-origin: right; -o-transform-origin: right; transform-origin: right}
.MJXp-bold {font-weight: bold}
.MJXp-italic {font-style: italic}
.MJXp-scr {font-family: MathJax_Script,'Times New Roman',Times,STIXGeneral,serif}
.MJXp-frak {font-family: MathJax_Fraktur,'Times New Roman',Times,STIXGeneral,serif}
.MJXp-sf {font-family: MathJax_SansSerif,'Times New Roman',Times,STIXGeneral,serif}
.MJXp-cal {font-family: MathJax_Caligraphic,'Times New Roman',Times,STIXGeneral,serif}
.MJXp-mono {font-family: MathJax_Typewriter,'Times New Roman',Times,STIXGeneral,serif}
.MJXp-largeop {font-size: 150%}
.MJXp-largeop.MJXp-int {vertical-align: -.2em}
.MJXp-math {display: inline-block; line-height: 1.2; text-indent: 0; font-family: 'Times New Roman',Times,STIXGeneral,serif; white-space: nowrap; border-collapse: collapse}
.MJXp-display {display: block; text-align: center; margin: 1em 0}
.MJXp-math span {display: inline-block}
.MJXp-box {display: block!important; text-align: center}
.MJXp-box:after {content: " "}
.MJXp-rule {display: block!important; margin-top: .1em}
.MJXp-char {display: block!important}
.MJXp-mo {margin: 0 .15em}
.MJXp-mfrac {margin: 0 .125em; vertical-align: .25em}
.MJXp-denom {display: inline-table!important; width: 100%}
.MJXp-denom > * {display: table-row!important}
.MJXp-surd {vertical-align: top}
.MJXp-surd > * {display: block!important}
.MJXp-script-box > *  {display: table!important; height: 50%}
.MJXp-script-box > * > * {display: table-cell!important; vertical-align: top}
.MJXp-script-box > *:last-child > * {vertical-align: bottom}
.MJXp-script-box > * > * > * {display: block!important}
.MJXp-mphantom {visibility: hidden}
.MJXp-munderover, .MJXp-munder {display: inline-table!important}
.MJXp-over {display: inline-block!important; text-align: center}
.MJXp-over > * {display: block!important}
.MJXp-munderover > *, .MJXp-munder > * {display: table-row!important}
.MJXp-mtable {vertical-align: .25em; margin: 0 .125em}
.MJXp-mtable > * {display: inline-table!important; vertical-align: middle}
.MJXp-mtr {display: table-row!important}
.MJXp-mtd {display: table-cell!important; text-align: center; padding: .5em 0 0 .5em}
.MJXp-mtr > .MJXp-mtd:first-child {padding-left: 0}
.MJXp-mtr:first-child > .MJXp-mtd {padding-top: 0}
.MJXp-mlabeledtr {display: table-row!important}
.MJXp-mlabeledtr > .MJXp-mtd:first-child {padding-left: 0}
.MJXp-mlabeledtr:first-child > .MJXp-mtd {padding-top: 0}
.MJXp-merror {background-color: #FFFF88; color: #CC0000; border: 1px solid #CC0000; padding: 1px 3px; font-style: normal; font-size: 90%}
.MJXp-scale0 {-webkit-transform: scaleX(.0); -moz-transform: scaleX(.0); -ms-transform: scaleX(.0); -o-transform: scaleX(.0); transform: scaleX(.0)}
.MJXp-scale1 {-webkit-transform: scaleX(.1); -moz-transform: scaleX(.1); -ms-transform: scaleX(.1); -o-transform: scaleX(.1); transform: scaleX(.1)}
.MJXp-scale2 {-webkit-transform: scaleX(.2); -moz-transform: scaleX(.2); -ms-transform: scaleX(.2); -o-transform: scaleX(.2); transform: scaleX(.2)}
.MJXp-scale3 {-webkit-transform: scaleX(.3); -moz-transform: scaleX(.3); -ms-transform: scaleX(.3); -o-transform: scaleX(.3); transform: scaleX(.3)}
.MJXp-scale4 {-webkit-transform: scaleX(.4); -moz-transform: scaleX(.4); -ms-transform: scaleX(.4); -o-transform: scaleX(.4); transform: scaleX(.4)}
.MJXp-scale5 {-webkit-transform: scaleX(.5); -moz-transform: scaleX(.5); -ms-transform: scaleX(.5); -o-transform: scaleX(.5); transform: scaleX(.5)}
.MJXp-scale6 {-webkit-transform: scaleX(.6); -moz-transform: scaleX(.6); -ms-transform: scaleX(.6); -o-transform: scaleX(.6); transform: scaleX(.6)}
.MJXp-scale7 {-webkit-transform: scaleX(.7); -moz-transform: scaleX(.7); -ms-transform: scaleX(.7); -o-transform: scaleX(.7); transform: scaleX(.7)}
.MJXp-scale8 {-webkit-transform: scaleX(.8); -moz-transform: scaleX(.8); -ms-transform: scaleX(.8); -o-transform: scaleX(.8); transform: scaleX(.8)}
.MJXp-scale9 {-webkit-transform: scaleX(.9); -moz-transform: scaleX(.9); -ms-transform: scaleX(.9); -o-transform: scaleX(.9); transform: scaleX(.9)}
.MathJax_PHTML .noError {vertical-align: ; font-size: 90%; text-align: left; color: black; padding: 1px 3px; border: 1px solid}
</style><link id="altmetric-embed-css" rel="stylesheet" type="text/css" href="https://d1bxh8uas1mnw7.cloudfront.net/assets/embed-a458e1093bac38cf5adc51eff02159de22f9e260b3636d0ecefba2667a43520a.css"></head>
<body class="pb-ui"><div id="MathJax_Message" style="display: none;"></div>

<div class="skip-links">
    <ul class="rlist">
        
                <li><a href="#article__content" tabindex="1">Skip to Article Content</a></li>
                <li><a href="#pane-pcw-detailscon" tabindex="1">Skip to Article Information</a></li>
            
    </ul>
</div>







    <div id="pb-page-content" data-ng-non-bindable="">
        <div data-pb-dropzone="main" data-pb-dropzone-name="Main">










        <div data-widget-def="pageHeader" data-widget-id="62175478-b98a-43d5-90f2-1f2045ac1a7e" class="pageHeader fixed-element">
        



        
        <header class="page-header">
    <div data-pb-dropzone="main">
    









        <div data-widget-def="ux3-layout-widget" data-widget-id="bb5b5e2c-d3d6-4015-b586-eeade38f59d4" class="admin-message">
        



        
        <div class="container"><div class="row"><div>



        
        



        
    

</div></div></div>

        </div>
    




        
        <div class="container"><div class="row"><div class="contents col-sm-12 col-md-12">









        <div data-widget-def="UX3InstitutionBanner" data-widget-id="0a735273-4141-44d8-801a-2b89143bed55" class="institution-wrapper-mobile hidden-lg padded-content">
        



        
        <div class="institution"><div class="institution-info-wrapper"><span class="institution__intro">Access By</span><span class="institution__name">Czechs- Biological Institution Of Cas</span></div></div>

        </div>
    




        
        <div class="pull-left">









        <div data-widget-def="ux3-general-image" data-widget-id="ff262197-0fb8-4973-aaa2-2fceba4b5a61" class="logo-container hidden-xs hidden-sm hidden-md">
        



        
        <a href="https://onlinelibrary.wiley.com/" title="Wiley Online Library"><img id="mainLogo" alt="Wiley Online Library" src="logo-header-1526603583437.png"></a>

        </div>
    










        <div data-widget-def="ux3-general-image" data-widget-id="9a75b789-224c-4745-833a-930cf6c36f35" class="logo-container-mobile hidden-lg hidden-md hidden-sm">
        



        
        <a href="https://onlinelibrary.wiley.com/" title="Wiley Online Library"><img id="mobileLogo" alt="Wiley Online Library" src="mobilehublogo-1518781861020.png"></a>

        </div>
    










        <div data-widget-def="UX3InstitutionBanner" data-widget-id="14d0e2f8-27d3-42d0-a477-72015e14e818" class="institution-wrapper hidden-xs hidden-sm hidden-md institution--block">
        



        
        <div class="institution"><div class="institution-info-wrapper"><span class="institution__intro">Access by</span><span class="institution__name">Czechs- Biological Institution Of Cas</span></div></div>

        </div>
    
</div><div class="pull-right">









        <div data-widget-def="UX3QuickSearchWidget" data-widget-id="48175bb0-0f70-46e6-bb89-7e2781fdb8e4" class="header-quicksearch">
        



        
        <div class="quick-search-wrapper"><a href="#" title="search" aria-label="Search" class="quick-search__item"><i aria-hidden="true" class="icon-search"></i><span class="quick-search__item__arrow"></span></a><div data-linkto-page="true" class="quick-search quickSearchFormContainer"><div class="main-search-field-wrapper"><div class="tab clearfix"><input placeholder="Search" tabindex="13" class="fakeQSInput quickSearchFilter" type="search"><label for="search-within" class="sr-only">Search within</label><select name="search-within" id="search-within" role="tablist" tabindex="10" class="search-within"><option value="#pane-67065c09-4a88-49cd-934c-ac707951d35c0" selected="selected">This Journal</option><option value="#pane-67065c09-4a88-49cd-934c-ac707951d35c1">Anywhere</option></select><ul role="tablist" class="rlist tab__content"><li id="pane-67065c09-4a88-49cd-934c-ac707951d35c0" aria-labelledby="pane-67065c09-4a88-49cd-934c-ac707951d35c0-con" role="tab" class="tab__pane quick-search-pane__anywhere active"><form action="/action/doSearch" name="thisJournalQuickSearch" method="get" title="Quick Search"><div class="input-group option-0"><label for="searchField0" class="hiddenLabel">Search term</label><input name="AllField" id="searchField0" placeholder="Search" onfocus="this.value = this.value;" data-auto-complete-max-words="7" data-auto-complete-max-chars="32" data-contributors-conf="3" data-topics-conf="3" data-publication-titles-conf="3" data-history-items-conf="3" tabindex="9" class="autocomplete actualQSInput quickSearchFilter ui-autocomplete-input" autocomplete="off" type="search"><span role="status" aria-live="polite" class="ui-helper-hidden-accessible"></span><input name="SeriesKey" value="16000587" type="hidden"><div class="search-options-wrapper quickSearchFilter"><a href="https://onlinelibrary.wiley.com/search/advanced?publication=16000587" title="" tabindex="12" class="advanced">  Advanced Search</a><a href="https://onlinelibrary.wiley.com/search/advanced?publication=16000587#citation" title="" tabindex="12" class="citation">  Citation Search</a></div></div><button id="quick-search-btn" type="submit" title="Search" tabindex="11" aria-label="Submit Search" class="btn quick-search__button icon-search"></button></form></li><li id="pane-67065c09-4a88-49cd-934c-ac707951d35c1" aria-labelledby="pane-67065c09-4a88-49cd-934c-ac707951d35c1-con" role="tab" class="tab__pane quick-search-pane__anywhere"><form action="/action/doSearch" name="defaultQuickSearch" method="get" title="Quick Search"><div class="input-group option-1"><label for="searchField1" class="hiddenLabel">Search term</label><input name="AllField" id="searchField1" placeholder="Search" onfocus="this.value = this.value;" data-auto-complete-max-words="7" data-auto-complete-max-chars="32" data-contributors-conf="3" data-topics-conf="3" data-publication-titles-conf="3" data-history-items-conf="3" tabindex="9" class="autocomplete actualQSInput quickSearchFilter ui-autocomplete-input" autocomplete="off" type="search"><span role="status" aria-live="polite" class="ui-helper-hidden-accessible"></span><div class="search-options-wrapper quickSearchFilter"><a href="https://onlinelibrary.wiley.com/search/advanced?" title="" tabindex="12" class="advanced">  Advanced Search</a><a href="https://onlinelibrary.wiley.com/search/advanced?#citation" title="" tabindex="12" class="citation">  Citation Search</a></div></div><button id="quick-search-btn" type="submit" title="Search" tabindex="11" aria-label="Submit Search" class="btn quick-search__button icon-search"></button></form></li></ul></div></div></div></div>

        </div>
    










        <div data-widget-def="ux3-layout-widget" data-widget-id="5ef13ed1-f4ac-4b62-a426-07ec9576a586" class="cart-indicator__wrapper">
        



        
        <div class="pull-left">









        <div data-widget-def="eCommerceCartIndicatorWidget" data-widget-id="4a9f7c04-a273-4a25-91fb-8e7bd9970288" class="cart-indicator__wrapper">
        



        
        
















        
    

        </div>
    
</div><div class="pull-right">









        <div data-widget-def="literatumNavigationLoginBar" data-widget-id="d384ca92-7137-45d6-b3c2-78c9c0a98c2b" class="login-bar-container">
        



        
        




    









    <div class="loginBar login-open">
        <a href="https://onlinelibrary.wiley.com/action/showLogin?uri=%2Fdoi%2Ffull%2F10.1111%2Fj.0906-7590.2008.5203.x&amp;aria-label=Log+in+or+Register" class="show-login" role="button" uri="/doi/full/10.1111/j.0906-7590.2008.5203.x" aria-label="Log in or Register">
            <span class="icon-profile_empty"></span>
            <span class="sign-in-label">Login / Register</span>
        </a>
            
    </div>



        </div>
    
</div>

        </div>
    
</div>
</div></div></div>
    </div>
</header>


        </div>
    










        <div data-widget-def="pageBody" data-widget-id="72100436-7a82-49fc-933b-c6c9d8c42914" class="pageBody article-page-tablet body-text">
        



        
        <main id="main-content" tabindex="-1">
    <!--begin pagefulltext-->
    <div class="page-body pagefulltext">
        <div data-pb-dropzone="main">
        



        
        <div>









        <div data-widget-def="literatumAd" data-widget-id="0220a30b-90c0-4b51-8637-141edf12f86a" class="banner-wrapper">
        



        
        



        <div class="pb-ad">
            <aside role="complementary">
		<div id="advert-leaderboard" class="advert advert-leaderboard" data-google-query-id="CPqwpOnahegCFcqLdwodQosEJQ" style="display: none;">
			<script>
				googletag.cmd.push 		( function () {
					googletag.display 	( 'advert-leaderboard' );
				});
			</script><div id="google_ads_iframe_/2215/wly.ecology_000046/ecography_ecog_0__container__" style="border: 0pt none; width: 100%; height: 0%;"></div>

			<!-- No JavaScript support is a call to an image -->
			<noscript>
				<a href="/action/clickThrough?id=1013&url=https%3A%2F%2Fpubads.g.doubleclick.net%2Fgampad%2Fjump%3Fiu%3D%24googlePublisherCategory%26amp%3Bsz%3D728x90%26amp%3Btile%3D1%26amp%3Bc%3D%24random&loc=%2Fdoi%2Ffull%2F10.1111%2Fj.0906-7590.2008.5203.x&pubId=59085228&placeholderId=1015&productId=101018" target="_blank">
					<img src="https://pubads.g.doubleclick.net/gampad/ad?iu=$googlePublisherCategory&amp;sz=728x90&amp;tile=1&amp;c=$random">
				</a>
			</noscript>
		</div>
	</aside>
        </div>
    


        </div>
    




        
        <style type="text/css">li.menu-item[data-menu-label="Accepted Articles"] {
  display: none;
}</style>
</div>










        <section data-widget-def="ux3-layout-widget" data-widget-id="0f7bccba-8328-4dbd-a1fa-e84c526dd9ab">
        



        
        <div class="container"><div class="row"><div class=" col-md-12">



        
        <div>



        
        <article data-figures="/action/ajaxShowFigures?widgetId=5cf4c79f-0ae9-4dc5-96ce-77f62de7ada9&amp;ajax=true&amp;doi=10.1111%2Fj.0906-7590.2008.5203.x&amp;pbContext=%3Bissue%3Aissue%3Adoi%5C%3A10.1111%2Feco.2008.31.issue-2%3Bpage%3Astring%3AArticle%2FChapter+View%3Bctype%3Astring%3AJournal+Content%3Bwebsite%3Awebsite%3Apericles%3Bjournal%3Ajournal%3A16000587%3Barticle%3Aarticle%3Adoi%5C%3A10.1111%2Fj.0906-7590.2008.5203.x%3Bwgroup%3Astring%3APublication+Websites%3BrequestedJournal%3Ajournal%3A16000587%3BpageGroup%3Astring%3APublication+Pages%3BsubPage%3Astring%3AFull+Text&amp;accordionHeadingWrapper=h2" data-references="/action/ajaxShowEnhancedAbstract?widgetId=5cf4c79f-0ae9-4dc5-96ce-77f62de7ada9&amp;ajax=true&amp;doi=10.1111%2Fj.0906-7590.2008.5203.x&amp;pbContext=%3Bissue%3Aissue%3Adoi%5C%3A10.1111%2Feco.2008.31.issue-2%3Bpage%3Astring%3AArticle%2FChapter+View%3Bctype%3Astring%3AJournal+Content%3Bwebsite%3Awebsite%3Apericles%3Bjournal%3Ajournal%3A16000587%3Barticle%3Aarticle%3Adoi%5C%3A10.1111%2Fj.0906-7590.2008.5203.x%3Bwgroup%3Astring%3APublication+Websites%3BrequestedJournal%3Ajournal%3A16000587%3BpageGroup%3Astring%3APublication+Pages%3BsubPage%3Astring%3AFull+Text&amp;accordionHeadingWrapper=h2" data-related="/action/ajaxShowRecommended?widgetId=5cf4c79f-0ae9-4dc5-96ce-77f62de7ada9&amp;ajax=true&amp;doi=10.1111%2Fj.0906-7590.2008.5203.x&amp;pbContext=%3Bissue%3Aissue%3Adoi%5C%3A10.1111%2Feco.2008.31.issue-2%3Bpage%3Astring%3AArticle%2FChapter+View%3Bctype%3Astring%3AJournal+Content%3Bwebsite%3Awebsite%3Apericles%3Bjournal%3Ajournal%3A16000587%3Barticle%3Aarticle%3Adoi%5C%3A10.1111%2Fj.0906-7590.2008.5203.x%3Bwgroup%3Astring%3APublication+Websites%3BrequestedJournal%3Ajournal%3A16000587%3BpageGroup%3Astring%3APublication+Pages%3BsubPage%3Astring%3AFull+Text&amp;accordionHeadingWrapper=h2&amp;showSubjects=true&amp;taxonomiesCodes=enter+your+taxonomy+codes+here&amp;displayCitedByLink=true&amp;displayAlmetricDropzone=true&amp;taxonomyUri=publication-features&amp;topicUri=hide-issue-metadata" data-details="/action/ajaxShowPubInfo?widgetId=5cf4c79f-0ae9-4dc5-96ce-77f62de7ada9&amp;ajax=true&amp;doi=10.1111%2Fj.0906-7590.2008.5203.x&amp;pbContext=%3Bissue%3Aissue%3Adoi%5C%3A10.1111%2Feco.2008.31.issue-2%3Bpage%3Astring%3AArticle%2FChapter+View%3Bctype%3Astring%3AJournal+Content%3Bwebsite%3Awebsite%3Apericles%3Bjournal%3Ajournal%3A16000587%3Barticle%3Aarticle%3Adoi%5C%3A10.1111%2Fj.0906-7590.2008.5203.x%3Bwgroup%3Astring%3APublication+Websites%3BrequestedJournal%3Ajournal%3A16000587%3BpageGroup%3Astring%3APublication+Pages%3BsubPage%3Astring%3AFull+Text&amp;accordionHeadingWrapper=h2&amp;showSubjects=true&amp;taxonomiesCodes=enter+your+taxonomy+codes+here&amp;displayCitedByLink=true&amp;displayAlmetricDropzone=true&amp;taxonomyUri=publication-features&amp;topicUri=hide-issue-metadata" data-mathjax="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.5/MathJax.js?config=TeX-AMS-MML_HTMLorMML"><div class="row article-row"><div id="article__content" tabindex="-1" class="col-sm-12 col-md-8 col-lg-8 article__content article-row-left"><div data-pb-dropzone="mrwBannerDropZone" class="mrw-banner-dropzone">



        
        
</div><div class="article-citation"><div class="citation">
   <div class="pb-dropzone" data-pb-dropzone="publicaitonContent-series-title">
      <div id="journal-banner-text" class="journal-banner-text" style="font-size: 22px;"><a href="https://onlinelibrary.wiley.com/journal/16000587" title="Ecography homepage">Ecography</a></div><a href="https://onlinelibrary.wiley.com/toc/16000587/2008/31/2" class="volume-issue">Volume 31, Issue 2</a><a href="https://onlinelibrary.wiley.com/journal/16000587" title="Ecography homepage" class="citation--logo"><img id="journal-banner-image" test="test-value" alt="Ecography" src="16000587-1569831440893.jpg" class="journal-banner-image" style="display: none;"></a><script>var image = new Image();

image.onerror = function() {
  // if the image does not exist
  //document.getElementById("journal-banner-text").style.marginBottom = "15px";
  document.getElementById("journal-banner-text").style.fontSize = "22px";
  document.getElementById("journal-banner-image").style.display = "none";
}

image.src = "/pb-assets/journal-banners/16000587-1569831440893.jpg";</script>
   </div>
   <div class="doi-access-container clearfix"><span class="primary-heading"></span>
      <div class="doi-access-wrapper"><div class="free-access access-type"> <i aria-hidden="true" class="icon-icon-lock_open"></i> <div class="doi-access"> <div class="doi-access" tabindex="0">Free Access</div></div></div></div></div>
   <h1 class="citation__title">Modeling of species distributions with Maxent: new extensions and a comprehensive evaluation</h1>
   <div class="rlist--inline loa comma visible-xs mobile-authors loa-authors-trunc">
      <div class="accordion-tabbed__tab-mobile  accordion__closed"><a href="https://onlinelibrary.wiley.com/action/doSearch?ContribAuthorStored=Phillips%2C+Steven+J" class="author-name accordion-tabbed__control" data-id="am1" data-db-target-for="am1" aria-controls="am1" aria-haspopup="true" id="am1_Ctrl" role="button"><span>Steven J.  Phillips</span><i aria-hidden="true" class="icon-section_arrow_d"></i></a><div class="author-info accordion-tabbed__content" data-db-target-of="am1" aria-labelledby="am1_Ctrl" role="region" id="am1"><a class="moreInfoLink" href="https://onlinelibrary.wiley.com/action/doSearch?ContribAuthorStored=Phillips%2C+Steven+J">Search for more papers by this author</a></div>
      </div>
      <div class="accordion-tabbed__tab-mobile  accordion__closed"><a href="https://onlinelibrary.wiley.com/action/doSearch?ContribAuthorStored=Dud%C3%ADk%2C+Miroslav" class="author-name accordion-tabbed__control" data-id="am2" data-db-target-for="am2" aria-controls="am2" aria-haspopup="true" id="am2_Ctrl" role="button"><span>Miroslav Dudík</span><i aria-hidden="true" class="icon-section_arrow_d"></i></a><div class="author-info accordion-tabbed__content" data-db-target-of="am2" aria-labelledby="am2_Ctrl" role="region" id="am2"><a class="moreInfoLink" href="https://onlinelibrary.wiley.com/action/doSearch?ContribAuthorStored=Dud%C3%ADk%2C+Miroslav">Search for more papers by this author</a></div>
      </div>
   </div>
   <div class="loa-wrapper loa-authors hidden-xs">
      <div id="sb-1" class="accordion">
         <div class="accordion-tabbed">
            <div class="accordion-tabbed__tab-mobile  accordion__closed"><a href="https://onlinelibrary.wiley.com/action/doSearch?ContribAuthorStored=Phillips%2C+Steven+J" class="author-name accordion-tabbed__control" data-id="a1" data-db-target-for="a1" aria-controls="a1" aria-haspopup="true" id="a1_Ctrl" role="button"><span>Steven J.  Phillips</span><i aria-hidden="true" class="icon-section_arrow_d"></i></a><div class="author-info accordion-tabbed__content" data-db-target-of="a1" aria-labelledby="a1_Ctrl" role="region" id="a1"><a class="moreInfoLink" href="https://onlinelibrary.wiley.com/action/doSearch?ContribAuthorStored=Phillips%2C+Steven+J">Search for more papers by this author</a></div>
            </div>
            <div class="accordion-tabbed__tab-mobile  accordion__closed"><a href="https://onlinelibrary.wiley.com/action/doSearch?ContribAuthorStored=Dud%C3%ADk%2C+Miroslav" class="author-name accordion-tabbed__control" data-id="a2" data-db-target-for="a2" aria-controls="a2" aria-haspopup="true" id="a2_Ctrl" role="button"><span>Miroslav Dudík</span><i aria-hidden="true" class="icon-section_arrow_d"></i></a><div class="author-info accordion-tabbed__content" data-db-target-of="a2" aria-labelledby="a2_Ctrl" role="region" id="a2"><a class="moreInfoLink" href="https://onlinelibrary.wiley.com/action/doSearch?ContribAuthorStored=Dud%C3%ADk%2C+Miroslav">Search for more papers by this author</a></div>
            </div>
         </div>
      </div>
   </div>
   <div class="epub-sections"><div class="epub-section"><span class="epub-state">First published: </span><span class="epub-date">28 March 2008</span></div>
      <div class="epub-section"><a class="epub-doi" aria-label="Digital Object Identifier" href="https://doi.org/10.1111/j.0906-7590.2008.5203.x">https://doi.org/10.1111/j.0906-7590.2008.5203.x</a></div>
      <div class="epub-section cited-by-count"><span>Citations: <a href="#citedby-section">2479</a></span></div>
   </div>
   <div class="article-header__correspondence-to"><i aria-hidden="true" class="at-icon-wrapper icon-mail_outline"></i>
S. J. Phillips (<a title="
                        Link to email address
                    " href="mailto:phillips@research.att.com">phillips@research.att.com</a>),
 AT&amp;T Labs Research, 180 Park Avenue, Florham Park, NJ 07932, USA. –
 M. Dudík, Computer Science Dept, Princeton Univ., 35 Olden Street, 
Princeton, NJ 08540, USA.
   </div>
</div></div><div class="fixed-element"><nav aria-label="Article navigation and tools" class="coolBar stickybar"><div class="stickybar__wrapper coolBar__wrapper clearfix"><div class="rlist coolBar__zone"><div class="coolBar__section"><a href="#" data-db-target-for="article" data-db-switch="icon-close_thin" aria-haspopup="true" aria-controls="article_Pop" role="button" id="article_Ctrl" class="coolBar__ctrl hidden-lg"><i class="icon-abstract"></i><span>About</span></a><div data-db-target-of="article" aria-labelledby="article_Ctrl" role="menu" id="article_Pop" class="coolBar__drop fixed rlist dropBlock__holder"><div data-target=".article-row-right .tab__nav, .coolBar--download" data-remove="false" data-targetclass="hidden-xs hidden-sm" data-toggle="transplant" data-direction="from" data-transplant="self" class="transplant showit"><ul class="transplanted-clone"><li><ul data-mobile-toggle="slide" role="tablist" class="rlist tab__nav w-slide--list"><li role="presentation"><a data-slide-clone="self" href="#pane-pcw-figures" aria-controls="#pane-pcw-figures" role="tab" data-toggle="tab" id="pane-pcw-figurescon" data-slide-target="#pane-pcw-figures" class="article-row-right__tabLinks w-slide__btn figures-tab empty" tabindex="-1" aria-selected="false"><i aria-hidden="true" class="icon-multi-figure"></i><h2>Figures</h2></a></li><li role="presentation"><a data-slide-clone="self" href="#pane-pcw-references" aria-controls="#pane-pcw-references" role="tab" data-toggle="tab" id="pane-pcw-referencescon" data-slide-target="#pane-pcw-references" class="article-row-right__tabLinks w-slide__btn references-tab empty" tabindex="-1" aria-selected="false"><i aria-hidden="true" class="icon-chain"></i><h2>References</h2></a></li><li role="presentation"><a data-slide-clone="self" href="#pane-pcw-related" aria-controls="#pane-pcw-related" role="tab" data-toggle="tab" id="pane-pcw-relatedcon" data-slide-target="#pane-pcw-related" class="article-row-right__tabLinks w-slide__btn related-tab empty" tabindex="-1" aria-selected="false"><i aria-hidden="true" class="icon-related-circles-extend-button"></i><h2>Related</h2></a></li><li role="presentation"><a data-slide-clone="self" href="#pane-pcw-details" aria-controls="#pane-pcw-details" role="tab" data-toggle="tab" id="pane-pcw-detailscon" data-slide-target="#pane-pcw-details" class="article-row-right__tabLinks w-slide__btn details-tab" tabindex="-1" aria-selected="false"><i aria-hidden="true" class="icon-Icon_Information"></i><h2>Information</h2></a></li></ul></li><li><div class="coolBar__section coolBar--download readcubeEPdfLink"><a href="https://onlinelibrary.wiley.com/doi/epdf/10.1111/j.0906-7590.2008.5203.x" title="Article ePDF" class="coolBar__ctrl pdf-download"><img src="pdf-icon.png" alt="ePDF">PDF</a></div><div class="coolBar__section coolBar--download PdfLink scroll-to-target pdfLink--is-hidden"><a href="https://onlinelibrary.wiley.com/doi/pdf/10.1111/j.0906-7590.2008.5203.x" title="Article PDF" class="coolBar__ctrl pdf-download"><img src="pdf-icon.png" alt="PDF">PDF</a></div></li></ul></div></div></div><div class="coolBar__section coolBar--sections"><a href="#" data-db-target-for="sections" data-db-switch="icon-close_thin" aria-haspopup="true" aria-controls="sections_Pop" role="button" id="sections_Ctrl" class="coolBar__ctrl"><i class="icon-list-unordered"></i><span>Sections</span></a><div data-db-target-of="sections" aria-labelledby="sections_Ctrl" role="menu" id="sections_Pop" class="coolBar__drop dropBlock__holder rlist fixed"><ul class="rlist w-slide--list"><li role="menuitem"><a class="w-slide__hide" href="#d12494444"><span>Abstract</span></a></li><li role="menuitem"><a class="w-slide__hide" href="#ss2-title"><span> Maxent modeling of distributions</span></a></li><li role="menuitem"><a class="w-slide__hide" href="#ss6-title"><span> Experimental methods</span></a></li><li role="menuitem"><a class="w-slide__hide" href="#ss9-title"><span> Experiments for tuning Maxent</span></a></li><li role="menuitem"><a class="w-slide__hide" href="#ss15-title"><span> Experiments for new extensions to Maxent</span></a></li><li role="menuitem"><a class="w-slide__hide" href="#ss20-title"><span> Results</span></a></li><li role="menuitem"><a class="w-slide__hide" href="#ss27-title"><span> Discussion and conclusions</span></a></li><li role="menuitem"><a class="w-slide__hide" href="#ss32-title"><span> Acknowledgements</span></a></li><li role="menuitem"><a class="w-slide__hide" href="#references-section-1"><span>References</span></a></li><li role="menuitem"><a class="w-slide__hide" href="#citedby-section"><span>Citing Literature</span></a></li></ul></div></div></div><div class="coolBar__second rlist"><div class="coolBar__section coolBar--download readcubeEPdfLink cloned"><a href="https://onlinelibrary.wiley.com/doi/epdf/10.1111/j.0906-7590.2008.5203.x" title="Article ePDF" class="coolBar__ctrl pdf-download"><img src="pdf-icon.png" alt="ePDF">PDF</a></div><div class="coolBar__section coolBar--download PdfLink scroll-to-target pdfLink--is-hidden cloned"><a href="https://onlinelibrary.wiley.com/doi/pdf/10.1111/j.0906-7590.2008.5203.x" title="Article PDF" class="coolBar__ctrl pdf-download"><img src="pdf-icon.png" alt="PDF">PDF</a></div>



        
        <div class="relative"><a href="#" data-db-target-for="0e406ce9-5c43-4e2d-b741-e0b43d1f1865" data-db-switch="icon-close_thin" aria-haspopup="true" aria-controls="0e406ce9-5c43-4e2d-b741-e0b43d1f1865_Pop" role="button" id="0e406ce9-5c43-4e2d-b741-e0b43d1f1865_Ctrl" class="article-tools__ctrl"><i aria-hidden="true" class="icon-Icon_Tools"></i><span>Tools</span></a><div data-db-target-of="0e406ce9-5c43-4e2d-b741-e0b43d1f1865" aria-labelledby="0e406ce9-5c43-4e2d-b741-e0b43d1f1865_Ctrl" role="menu" id="0e406ce9-5c43-4e2d-b741-e0b43d1f1865_Pop" class="article-tools__block fixed dropBlock__holder"><ul class="rlist w-slide--list"><li role="presentation" class="article-tool"><a href="https://onlinelibrary.wiley.com/action/rightsLink?doi=10.1111%2Fj.0906-7590.2008.5203.x&amp;mode=" role="menuitem" target="_blank"><i aria-hidden="true" class="icon-permissions-key"></i><span>Request permission</span></a></li><li role="presentation" class="article-tool"><a href="https://onlinelibrary.wiley.com/action/showCitFormats?doi=10.1111%2Fj.0906-7590.2008.5203.x" role="menuitem" target="_self"><i aria-hidden="true" class="icon-export-icon"></i><span>Export citation</span></a></li><li role="presentation" class="article-tool"><a href="https://onlinelibrary.wiley.com/personalize/addFavoritePublication?doi=10.1111%2Fj.0906-7590.2008.5203.x" role="menuitem" target="_self"><i aria-hidden="true" class="icon-favorites"></i><span>Add to favorites</span></a></li><li role="presentation" class="article-tool"><a href="https://onlinelibrary.wiley.com/action/addCitationAlert?doi=10.1111%2Fj.0906-7590.2008.5203.x" role="menuitem" target="_self"><i aria-hidden="true" class="icon-Icon_Track-citations"></i><span>Track citation</span></a></li></ul></div></div>




        
        <!-- Go to www.addthis.com/dashboard to customize your tools --><script type="text/javascript" async="async" src="addthis_widget.js"></script><div class="relative"><a id="share-access__ctrl" href="#" data-db-target-for="dff42abe-4b17-4fe9-a9c5-625405cd1007" data-db-switch="icon-close_thin" aria-haspopup="true" aria-controls="share_Pop" role="button" class="share-access__ctrl hidden"><i aria-hidden="true" class="icon-Icon_Share"></i><span>Share</span></a><a id="share__ctrl" href="#" data-db-target-for="dff42abe-4b17-4fe9-a9c5-625405cd1007" data-db-switch="icon-close_thin" aria-haspopup="true" aria-controls="share_Pop" role="button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i><span>Share</span></a><div data-db-target-of="dff42abe-4b17-4fe9-a9c5-625405cd1007" aria-labelledby="share__ctrl" role="menu" id="share_Pop" class="share__block dropBlock__holder fixed">



        
        <div data-readcube="/pb/widgets/readCubeP2PUrl/10.1111/j.0906-7590.2008.5203.x/" class="readCube-sharing hidden"><p>Give access</p><a href="#" data-toggle="modal" data-target="#readCube" class="readCube-sharing__link"><i class="icon-users"></i>Share full text access</a></div><div class="ux-modal-container readCube-sharing__modal"><div id="readCube" class="modal"><div class="modal__dialog"><div class="modal__header"><button type="button" data-dismiss="modal" class="close"><i aria-hidden="true" class="icon-close_thin"></i><span class="sr-only">Close modal</span></button><p>Share full text access</p></div><div class="modal__body"><form method="post"><fieldset><legend>Please review our <a href="https://onlinelibrary.wiley.com/termsAndConditions" target="_blank">Terms and Conditions of Use</a> and check box below to share full-text version of article.</legend><div class="input-group"><label for="terms-and-conditions" class="checkbox--primary"><input id="terms-and-conditions" value="yes" required="" name="terms-and-conditions" type="checkbox"><span class="label-txt">I have read and accept the Wiley Online Library Terms and Conditions of Use.</span></label></div></fieldset><hr class="separator"><fieldset><label>Shareable Link</label><p>Use the link below to share a full-text version of this article with your friends and colleagues. <a href="https://onlinelibrary.wiley.com/researchers/tools-resources/sharing" target="_blank" class="emphasis">Learn more.</a></p><div class="shareable-link"><div class="shareable-link__field"><span id="shareable-link__text"></span></div></div><button type="submit" disabled="disabled" class="btn shareable-link__btn">Copy URL</button></fieldset></form></div></div></div></div>
<p>Share a link</p><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_email" target="_blank" title="Email" href="#"><i aria-hidden="true" class="at-icon-wrapper icon-Email"></i>Email</a></li><li><a role="link" class="addthis_button_facebook" title="Facebook" href="#"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter" title="Twitter" href="#"><i aria-hidden="true" class="at-icon-wrapper icon-twitter-t"></i>Twitter</a></li><li><a role="link" class="addthis_button_linkedin" target="_blank" title="LinkedIn" href="#"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit" target="_blank" title="Reddit" href="#"><i aria-hidden="true" class="at-icon-wrapper icon-reddit-alien"></i>Reddit</a></li><li><a role="link" class="addthis_button_wechat" target="_blank" title="WeChat" href="#"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>Wechat</a></li><div class="atclear"></div></ul></div></div>
</div></div></nav></div><div id="accessDenialslot" class="accessDenialslot"></div><div class="article__body "><article>
   
   
   <div class="abstract-group">
      <section class="article-section article-section__abstract" data-lang="en" id="section-1-en" style="" lang="en">
         <h2 id="d12494444" class="article-section__header section__title main abstractlang_en main">Abstract</h2>
         <div class="article-section__content en main">
            <p>Accurate modeling of geographic distributions of species 
is crucial to various applications in ecology and conservation. The best
 performing techniques often require some parameter tuning, which may be
 prohibitively time‐consuming to do separately for each species, or 
unreliable for small or biased datasets. Additionally, even with the 
abundance of good quality data, users interested in the application of 
species models need not have the statistical knowledge required for 
detailed tuning. In such cases, it is desirable to use “default 
settings”, tuned and validated on diverse datasets. Maxent is a recently
 introduced modeling technique, achieving high predictive accuracy and 
enjoying several additional attractive properties. The performance of 
Maxent is influenced by a moderate number of parameters. The first 
contribution of this paper is the empirical tuning of these parameters. 
Since many datasets lack information about species absence, we present a
 tuning method that uses presence‐only data. We evaluate our method on 
independently collected high‐quality presence‐absence data. In addition 
to tuning, we introduce several concepts that improve the predictive 
accuracy and running time of Maxent. We introduce “hinge features” that 
model more complex relationships in the training data; we describe a new
 logistic output format that gives an estimate of probability of 
presence; finally we explore “background sampling” strategies that cope 
with sample selection bias and decrease model‐building time. Our 
evaluation, based on a diverse dataset of 226 species from 6 regions, 
shows: 1) default settings tuned on presence‐only data achieve 
performance which is almost as good as if they had been tuned on the 
evaluation data itself; 2) hinge features substantially improve model 
performance; 3) logistic output improves model calibration, so that 
large differences in output values correspond better to large 
differences in suitability; 4) “target‐group” background sampling can 
give much better predictive performance than random background sampling;
 5) random background sampling results in a dramatic decrease in running
 time, with no decrease in model performance.</p>
            
         </div>
      </section>
   </div>
   <div class="pb-dropzone" data-pb-dropzone="below-abstract-group">
      <!-- Empty dropzone -->
   </div>
   
   <section class="article-section article-section__full">
      <section class="article-section__content" id="ss1">
         
         <p>Predictive models of species geographic distributions are important for a variety of applications in ecology and conservation (<span><a href="#b12" id="#b12R" class="bibLink tab-link" data-tab="pane-pcw-references">Graham et al. 2004</a></span>).
 For example they have been applied to study spread of invasive species 
(Thuiller et al. 2005), impacts of climate change (Thomas et al. 2004), 
and spatial patterns of species diversity (<span><a href="#b13" id="#b13R" class="bibLink tab-link" data-tab="pane-pcw-references">Graham et al. 2006</a></span>).
 The increasing availability of large quantities of occurrence data, 
especially from natural history museum and herbarium collections, is 
fuelling the active exploration of methods which do not require data on 
species absence, since absence data is missing in those large datasets 
and is expensive to collect. Presence‐only modeling methods only require
 a set of known occurrences together with predictor variables such as 
topographic, climatic, edaphic, biogeographic and remotely sensed 
variables. Many techniques are available for this task, ranging from 
climatic envelopes and logistic regression, to boosted regression trees 
and multivariate regression splines. For some applications, different 
modeling methods may give very different predictions (<span><a href="#b21" id="#b21R" class="bibLink tab-link" data-tab="pane-pcw-references">Pearson et al. 2006</a></span>, <span><a href="#b29" id="#b29R" class="bibLink tab-link" data-tab="pane-pcw-references">Randin et al. 2006</a></span>),
 so it is important to develop guidelines for producing the most 
accurate models of species’ distributions. A recent comprehensive 
comparison of presence‐only modeling techniques (<span><a href="#b8" id="#b8R" class="bibLink tab-link" data-tab="pane-pcw-references">Elith et al. 2006</a></span>)
 found that some new methods have better predictive accuracy than the 
established methods. A distinguishing feature of the new methods is that
 they can fit more complex models from smaller datasets, using explicit 
“regularization” mechanisms to prevent model complexity from increasing 
beyond what is supported by the empirical data. In this paper we focus 
on one of these new methods, called Maxent (Phillips et al. 2004, 2006),
 and in particular, some new extensions to the method and the tuning of 
its regularization mechanism to optimize predictive accuracy.
         </p>
         
         <p>Tuning the parameters of a modeling method poses several 
challenges: when the number of tuned parameters is large, it may be 
prohibitively time‐consuming to tune the method on each species 
separately. Furthermore, tuning is typically based on performance on a 
randomly selected held‐out portion of a dataset. If the dataset is too 
small or highly biased, performance on the held‐out subset need not be a
 good indicator of predictive performance. Therefore, it may be 
desirable to limit species‐specific or dataset‐specific tuning, and use 
“default” settings. The use of default settings is justified provided 
that they have been validated over a wide range of species, 
environmental conditions, numbers of occurrences, and amounts of sample 
selection bias.</p>
         
         <p>In Maxent, several settings affect model accuracy by 
determining the type and complexity of dependencies on the environment 
that Maxent tries to fit. The dependencies are described by simple 
functions derived from environmental variables, called “features”. More 
complex features allow fitting more complex dependencies, but they may 
require more data. The complexity of dependencies is controlled by the 
choice of feature types, and by settings called “regularization 
parameters”. These parameters prevent Maxent from matching the input 
data too closely, which is known as “overfitting” and has a detrimental 
effect on predictive performance (<span><a href="#b17" id="#b17R" class="bibLink tab-link" data-tab="pane-pcw-references">Hastie et al. 2001</a></span>).
         </p>
         
         <p>The contributions of this paper are two‐fold. First, we 
explore tuning of Maxent settings with the comprehensive dataset used by
 <span><a href="#b8" id="#b8R" class="bibLink tab-link" data-tab="pane-pcw-references">Elith et al. (2006)</a></span>.
 The dataset contains presence‐only data derived from collections of 
natural history museums and herbaria, together with presence‐absence 
data obtained independently from rigorous surveys. We use the 
presence‐only data to determine appropriate feature types and 
regularization parameters depending on the number of occurrence records.
 We then evaluate the tuned settings on the independent presence‐absence
 data. The resulting configuration is used for the default settings of 
the Maxent software (<span><a href="#b26" id="#b26R" class="bibLink tab-link" data-tab="pane-pcw-references">Phillips et al. 2005</a></span>) ver. 1.8.3 through the time of writing (at least ver. 3.1.0), with a minor exception noted in the Discussion.
         </p>
         
         <p>In the second set of contributions, we introduce and examine
 the effects of several extensions of Maxent. Our first extension aims 
to capture more general dependencies on the environment and thus 
increase the predictive accuracy of Maxent models. Specifically, we 
introduce a new type of features –“hinge” features – to model arbitrary 
piecewise linear responses to the environmental variables. Hinge 
features are similar to “threshold” features (<span><a href="#b27" id="#b27R" class="bibLink tab-link" data-tab="pane-pcw-references">Phillips et al. 2006</a></span>),
 which are used to model piecewise constant responses. An analogous step
 up from piecewise constant responses to piecewise linear responses has 
been helpful in the regression setting (<span><a href="#b10" id="#b10R" class="bibLink tab-link" data-tab="pane-pcw-references">Friedman 1991</a></span>).
         </p>
         
         <p>The second extension is a new logistic output format, which 
addresses the fact that the existing raw and cumulative formats can be 
hard to interpret. For many modeling applications, we are interested in 
the probability that the species is present, conditional on the 
environmental conditions. This probability of presence is what is 
estimated by the logistic format.</p>
         
         <p>The third extension addresses the problem of sample 
selection bias. Occurrence data are frequently biased, for example 
towards areas easier to access such as areas near roads, towns, 
airports, and waterways (<span><a href="#b30" id="#b30R" class="bibLink tab-link" data-tab="pane-pcw-references">Reddy and Dávalos 2003</a></span>).
 When the bias is large, presence‐only models approximate the biased 
sampling distribution as much as they approximate the species 
distribution. This can be avoided by having the background sample 
reflect the same bias as the presence data (<span><a href="#b37" id="#b37R" class="bibLink tab-link" data-tab="pane-pcw-references">Zaniewski et al. 2002</a></span>, <span><a href="#b5" id="#b5R" class="bibLink tab-link" data-tab="pane-pcw-references">Dudík et al. 2005</a></span>)
 – specifically, the bias shared by the collectors of a particular group
 of species. To implement this idea, we use as background the set of 
occurrences for an entire group of species that may be captured or 
observed using the same methods (<span><a href="#b28" id="#b28R" class="bibLink tab-link" data-tab="pane-pcw-references">Ponder et al. 2001</a></span>, <span><a href="#b1" id="#b1R" class="bibLink tab-link" data-tab="pane-pcw-references">Anderson 2003</a></span>). We investigate the extent to which such data, called <i>target‐group</i> background (<span><a href="#b7" id="#b7R" class="bibLink tab-link" data-tab="pane-pcw-references">Elith and Leathwick 2007</a></span>,
 Phillips et al. unpubl.), can be regarded as a random sample from the 
sampling distribution and thus used to improve Maxent predictions.
         </p>
         
         <p>The final extension simply reduces model building time. 
Rather than performing calculations involving all sites in the study 
area, we make use of a random subset of sites, called “background data”,
 chosen to be large enough to sufficiently represent the distribution of
 environmental conditions in the study region.</p>
         
      </section>
      <section class="article-section__content" id="ss2">
         
         <h2 class="article-section__title section__title section1" id="ss2-title"> Maxent modeling of distributions</h2>
         
         <section class="article-section__sub-content" id="ss3">
            
            <h3 class="article-section__sub-title section2" id="ss3-title"> Basic concepts</h3>
            
            <p>In maximum entropy density estimation, the true 
distribution of a species is represented as a probability distribution π
 over the set X of sites in the study area. Thus, π assigns a 
non‐negative value to every site x and the values π(x) sum to one. We 
produce a model of π, a probability distribution that respects a set of 
constraints derived from the occurrence data. The constraints are 
expressed in terms of simple functions of the environmental variables, 
called features<i>.</i> Specifically, the mean of each feature is 
required to be close (within some error bounds) to the empirical average
 over the union of the presence sites. For example, for a feature 
“annual precipitation”, the corresponding constraint says that the mean 
annual precipitation predicted by the model should be close to the 
average observed precipitation. Since the set of constraints typically 
under‐specifies the model, among all probability distributions 
satisfying the constraints, we choose the one of maximum entropy, i.e. 
the most unconstrained one (<span><a href="#b18" id="#b18R" class="bibLink tab-link" data-tab="pane-pcw-references">Jaynes 1957</a></span>).
            </p>
            
            <p>Maximum entropy density estimation can also be explained 
from a decision theoretic perspective as robust Bayes estimation. 
Specifically, consider the scenario where the goal of the modeler is to 
optimize the expected log likelihood (see “Performance measures”, 
below), and the only fact known about the true distribution π is that it
 satisfies a certain set of constraints. The strategy which guarantees 
the best performance regardless of π, also called the minimax strategy, 
is to choose the maximum entropy distribution subject to the given 
constraints (<span><a href="#b34" id="#b34R" class="bibLink tab-link" data-tab="pane-pcw-references">Topsøe 1979</a></span>, <span><a href="#b14" id="#b14R" class="bibLink tab-link" data-tab="pane-pcw-references">Grünwald 2000</a></span>, <span><a href="#b15" id="#b15R" class="bibLink tab-link" data-tab="pane-pcw-references">Grünwald and Dawid 2004</a></span>).
            </p>
            
            <p>To understand how π represents the realized distribution 
of the species, consider the following (idealized) sampling strategy. An
 observer picks a random site x from the set X of sites in the study 
area, and records 1 if the species is present at x<i>,</i> and 0 if it is absent. If we denote the response variable (presence or absence) as y<i>,</i> then π(x) is the conditional probability P(x∣y=1), i.e. the probability of the observer being at x<i>,</i> given that the species is present. According to Bayes’ rule,

               </p><div class="inline-equation" id="m1"><span class="inline-equation__construct"><img class="" src="ecog_5203_m1.gif" alt="image" title="image"></span><span class="inline-equation__label">(1)</span></div>

since according to our sampling strategy P(x)=1/∣X∣ for all x<i>.</i> 
Here P(y=1) is the overall prevalence of the species in the study area. 
The quantity P(y=1∣x) is the probability that the species is present at 
the site x<i>,</i> which is 0 or 1 for plants, but may be between 0 and 1
 for vagile organisms. Equation 1 shows that π is proportional to 
probability of presence. However, if we have only occurrence data, we 
cannot determine the species’ prevalence (<span><a href="#b27" id="#b27R" class="bibLink tab-link" data-tab="pane-pcw-references">Phillips et al. 2006</a></span>, <span><a href="#b35" id="#b35R" class="bibLink tab-link" data-tab="pane-pcw-references">Ward et al. 2007</a></span>).
 Therefore, instead of estimating P(y=1∣x) directly, we estimate the 
distribution π. We emphasize that here x is a site, rather than a vector
 of environmental conditions. This treatment differs from more 
traditional statistical methods, such as logistic regression; later we 
will bring these two viewpoints together and present a new way of 
estimating probability of presence from the Maxent model (see below).
            <p></p>
            
            <p>According to Section 2 of <span><a href="#b4" id="#b4R" class="bibLink tab-link" data-tab="pane-pcw-references">Dudík et al. (2004)</a></span>, the Maxent distribution belongs to the family of Gibbs distributions derived from the set of features f<sub>1</sub>, …, f<sub>n</sub>. Gibbs distributions are exponential distributions parameterized by a vector of feature weights λ=(λ<sub>1</sub>, …,λ<sub>n</sub>) and defined by

               </p><div class="inline-equation" id="m2"><span class="inline-equation__construct"><img class="" src="ecog_5203_m2.gif" alt="image" title="image"></span><span class="inline-equation__label">(2)</span></div>

where Z<sub>λ</sub> is a normalization constant ensuring that probabilities q<sub>λ</sub>(x) sum to one over the study area. Therefore, the value of the Maxent model q<sub>λ</sub> at a site x depends only on the feature values at x<i>,</i> and hence only on the environmental variables at x<i>.</i>
 This means that the Maxent model, which we originally defined with 
strict reference to the set X of training sites, can also be “projected”
 to other sites where the same environmental variables are available. 
The Maxent distribution is the Gibbs distribution q<sub>λ</sub> that maximizes a penalized log likelihood of the presence sites, namely

               <div class="inline-equation" id="di-fml-3"><span class="inline-equation__construct"><img class="" src="ecog_5203_mu1.gif" alt="image" title="image"></span><span class="inline-equation__label"></span></div>

where the regularization parameter β<sub>j</sub> is the width of the error bound for feature f<sub>j</sub> and x<sub>1</sub>, …,x<sub>m</sub>
 are the presence sites. The first term, log likelihood, gets larger as 
we obtain a better fit to the data. This gives insight into how Maxent 
uses background data: the first term is larger for models that give more
 probability to the presence sites and less to the rest of the sites, 
i.e. models that best distinguish the presence sites from the 
background. The second term, regularization (also known as the lasso 
penalty; <span><a href="#b33" id="#b33R" class="bibLink tab-link" data-tab="pane-pcw-references">Tibshirani 1996</a></span>) gets larger as the weights λ<sub>j</sub> get larger. Larger weights λ<sub>j</sub>
 typically mean that the model is more complex and is thus more likely 
to overfit. Maximizing the difference between log likelihood and 
regularization can be viewed as seeking a Gibbs distribution which fits 
the data well, but which is not too complex. The tradeoff is controlled 
by the regularization parameters.
            <p></p>
            
            <p>For large sample sizes, the performance of Maxent (as 
measured by log likelihood of test data) converges to that of the best 
Gibbs distribution, as long as the presence sites are drawn 
independently at random according to π (<span><a href="#b4" id="#b4R" class="bibLink tab-link" data-tab="pane-pcw-references">Dudík et al. 2004</a></span>). The theoretical analysis gives the best performance guarantees when the regularization parameters β<sub>j</sub>
 are as small as possible, while keeping the true feature means (under 
π) within the error bounds. Thus, we have an incentive to obtain error 
bounds that are as tight as possible. To simplify the process of tuning 
parameters, we reduce the number of parameters from one per feature to 
one per feature class by setting

               </p><div class="inline-equation" id="di-fml-4"><span class="inline-equation__construct"><img class="" src="ecog_5203_mu2.gif" alt="image" title="image"></span><span class="inline-equation__label"></span></div>

where β is a regularization parameter that depends only on the feature class and s<sup>2</sup>[f<sub>j</sub>] is the empirical variance of feature f<sub>j</sub>, so 
<img alt="inline image" class="section_image" src="ecog_5203_mu3.gif"> is an estimate of the standard deviation of the empirical average. According to the theoretical guarantees we expect that 
<img alt="inline image" class="section_image" src="ecog_5203_mu4.gif"> 
will give good performance. However, the value of β that optimizes the 
theoretical bounds may not necessarily give the best model performance 
in practice. Therefore, we fine‐tune the regularization parameter for 
each feature class separately, using empirical tuning as described 
below.
            <p></p>
            
         </section>
         
         <section class="article-section__sub-content" id="ss4">
            
            <h3 class="article-section__sub-title section2" id="ss4-title"> Environmental variables and feature classes in Maxent</h3>
            
            <p>Features in Maxent are derived from environmental variables of two types: continuous and categorical<i>.</i>
 Continuous variables take arbitrary real values which correspond to 
measured quantities such as altitude, annual precipitation, and maximum 
temperature. Categorical variables take only a limited number of 
discrete values such as soil type or vegetation type. Some categorical 
variables quantify the degree of some property (on a discrete scale), 
for example soil fertility. This type of variable is referred to as 
discrete ordinal<i>.</i> We will typically treat discrete ordinal variables as if they were continuous.
            </p>
            
            <p>The Maxent software (<span><a href="#b26" id="#b26R" class="bibLink tab-link" data-tab="pane-pcw-references">Phillips et al. 2005</a></span>) implements features of six classes: <i>linear</i> (L), <i>quadratic</i> (Q), <i>product</i> (P), <i>threshold</i> (T), <i>hinge</i> (H), and <i>category indicator</i> (C) features. Hinge features are introduced in this paper, while the other five classes were introduced in <span><a href="#b27" id="#b27R" class="bibLink tab-link" data-tab="pane-pcw-references">Phillips et al. (2006)</a></span>.
 Linear, quadratic, product, threshold, and hinge features are derived 
from continuous variables. Linear features are equal to continuous 
environmental variables, quadratic features equal their squares, and 
product features equal products of pairs of continuous environmental 
variables. Respectively, they constrain means, variances, and 
covariances of the respective variables to match their empirical values (<span><a href="#b27" id="#b27R" class="bibLink tab-link" data-tab="pane-pcw-references">Phillips et al. 2006</a></span>).
 Category indicator features are derived from categorical variables. 
Specifically, if a categorical variable has k categories, it is used to 
derive k category indicator features. For each of the k categories, the 
corresponding category indicator equals 1 if the variable has the 
corresponding value and 0 if it has any of the remaining k<i>−</i>1 values.
            </p>
            
            <p>Threshold and hinge features allow Maxent to model an 
arbitrary response of the species to an environmental variable from 
which they are derived. If f is a continuous variable then for any value
 h (called the knot<i>),</i> we define the threshold feature threshold<sub>f,h</sub> by

               </p><div class="inline-equation" id="di-fml-7"><span class="inline-equation__construct"><img class="" src="ecog_5203_mu5.gif" alt="image" title="image"></span><span class="inline-equation__label"></span></div> 
            <p></p>
            
            <p>The forward hinge feature <i>forwardhinge</i><sub>f,h</sub> is 0 if f(x) <i>≤</i>h<i>,</i> then increases linearly to 1 at the maximum value of f:

               </p><div class="inline-equation" id="di-fml-8"><span class="inline-equation__construct"><img class="" src="ecog_5203_mu6.gif" alt="image" title="image"></span><span class="inline-equation__label"></span></div> 
            <p></p>
            
            <p>In a similar way, we define a reverse hinge feature, which is 1 at the minimum value of f, drops linearly to 0 at f(x)=h<i>,</i> and is 0 afterwards. Examples of a forward hinge feature and a reverse hinge feature are shown graphically in <a href="#f1" class="scrollableLink">Fig. 1</a>.
 Forward and reverse hinge features are collectively referred to as 
hinge features, and we have coined this term to evoke the shapes seen in
 the figure. In the terminology of splines, threshold features are base 
functions of splines of order 1 (piecewise constant splines) while hinge
 features are base functions of splines of order 2 (piecewise linear 
splines).
            </p>
            
            <section class="article-section__inline-figure">
               <figure class="figure" id="f1"><a target="_blank" href="https://onlinelibrary.wiley.com/cms/asset/798aeeec-30d8-4c95-97f7-84a7f02c8bb5/ecog_5203_f1.gif"><img class="figure__image" src="ecog_5203_f1.gif" alt="image" title="image"></a><figcaption class="figure__caption">
                     <div class="figure__caption__header"><strong class="figure-title">Figure 1</strong> 
                        <div class="figure-extra"><a href="#" class="open-figure-link">Open in figure viewer</a><a href="https://onlinelibrary.wiley.com/action/downloadFigures?id=f1&amp;doi=10.1111%2Fj.0906-7590.2008.5203.x" class="ppt-figure-link">PowerPoint</a></div>
                     </div>
                     <div>
                        <p>A forward hinge feature (left) and a reverse hinge feature (right), defined for a continuous environmental variable f.</p>
                        
                     </div>
                  </figcaption>
               </figure>
            </section>
            
            <p>As the number of features increases, Gibbs distributions 
become more complex, and may be more prone to overfitting. We therefore 
expect that more complex feature classes will require more 
regularization to yield accurate predictions. Many combinations of 
feature classes are possible; common combinations used are LC, LQC, HC, 
HQC, TC, and HQPTC. Linear features are special cases of hinge features,
 so it is redundant to use L and H features simultaneously.</p>
            
         </section>
         
         <section class="article-section__sub-content" id="ss5">
            
            <h3 class="article-section__sub-title section2" id="ss5-title"> Maxent output formats and logistic models</h3>
            
            <p>The primary output of Maxent is the exponential function q<sub>λ</sub>(x)
 that assigns a probability (referred to as a “raw” value) to each site 
used during model training. Raw values are not intuitive to work with 
though: in particular, it is hard to interpret “projected” values 
obtained by applying q<sub>λ</sub> to environmental conditions at sites 
not used during model training. Raw values are also scale‐dependent, in 
the sense that using more background data results in smaller raw values,
 since they must sum to one over a larger number of background points. 
For these reasons, raw values have generally been converted into the 
“cumulative” format (<span><a href="#b27" id="#b27R" class="bibLink tab-link" data-tab="pane-pcw-references">Phillips et al. 2006</a></span>).
            </p>
            
            <p>The cumulative format is defined in terms of omission rates predicted by the Maxent distribution q<sub>λ</sub>. Specifically, we consider 0–1 prediction rules that threshold raw outputs at a level p<i>.</i> Each raw threshold p is transformed into the omission percentage c(p) predicted by q<sub>λ</sub> for the corresponding rule, i.e.

               </p><div class="inline-equation" id="di-fml-9"><span class="inline-equation__construct"><img class="" src="ecog_5203_mu7.gif" alt="image" title="image"></span><span class="inline-equation__label"></span></div> 
            <p></p>
            
            <p>Therefore, if we make a 0–1 prediction from the Maxent distribution q<sub>λ</sub> using a cumulative threshold of c, the omission rate is c% for test sites drawn from q<sub>λ</sub>.
 The cumulative format is scale‐independent, and is more easily 
interpreted when projected, but it is not necessarily proportional to 
probability of presence.
            </p>
            
            <p>For example, consider a generalist species whose 
probability of presence is close to 1 across the whole study area, with 
slight variations that avoid ties. Since the probability values are 
similar across the entire region, the cumulative values of individual 
sites will be roughly proportional to their rank, and hence they will 
range evenly from 0 to 100. Thus, big variations in cumulative value do 
not necessarily represent big variations in suitability or probability 
of presence.</p>
            
            <p>We therefore introduce a new logistic output format that 
gives an estimate of probability of presence. Let z denote a vector of 
environmental variables, and let z(x) be the value of z at a site x<i>.</i> Traditional statistical methods such as logistic regression estimate P(y=1∣z)<i>,</i>
 the conditional probability of presence given the environmental 
conditions, which is closely related to the quantity we estimate, 
P(y=1∣x):

               </p><div class="inline-equation" id="m3"><span class="inline-equation__construct"><img class="" src="ecog_5203_m3.gif" alt="image" title="image"></span><span class="inline-equation__label">(3)</span></div>

where X(z) denotes the set of locations with environmental conditions z<i>.</i> Therefore, in order to estimate P(y=1∣z)<i>,</i> it suffices to focus on P(y=1∣x)<i>.</i>
 Indeed, in the special case that π(x) is only a function of the 
environmental conditions, if we let x(z) denote an arbitrary element 
from X(z), eq. 3 simplifies to

               <div class="inline-equation" id="m4"><span class="inline-equation__construct"><img class="" src="ecog_5203_m4.gif" alt="image" title="image"></span><span class="inline-equation__label">(4)</span></div> 
            <p></p>
            
            <p>Combining eq. 1 and 4, it is tempting to use our estimate q<sub>λ</sub> of π to derive the following estimate of

               </p><div class="inline-equation" id="di-fml-12"><span class="inline-equation__construct"><img class="" src="ecog_5203_mu8.gif" alt="image" title="image"></span><span class="inline-equation__label"></span></div>
               
               <div class="inline-equation" id="di-fml-13"><span class="inline-equation__construct"><img class="" src="ecog_5203_mu9.gif" alt="image" title="image"></span><span class="inline-equation__label"></span></div> 
            <p></p>
            
            <p>However, this approximation has two difficulties. First, 
we may not know or be able to estimate P(y=1), since this quantity is 
not determinable from presence‐only data (<span><a href="#b35" id="#b35R" class="bibLink tab-link" data-tab="pane-pcw-references">Ward et al. 2007</a></span>). Second, the approximation may result in probabilities greater than one, since Maxent does not guarantee that q<sub>λ</sub>(x) is smaller than 1/(P(y=1)∣X∣).
            </p>
            
            <p>We resolve these difficulties with a novel application of
 the maximum entropy principle. Rather than applying the principle to 
estimate a distribution over sites, we apply it to a joint distribution 
P(x, y) representing both a sampling distribution over sites (assumed 
the same for data collection and evaluation) and the presence/absence of
 the species. In particular, we estimate P(x, y) by a distribution Q(x, 
y) of maximum entropy subject to constraints on the conditional 
distribution P(x∣y<i>=</i>1), i.e. the same constraints we applied to estimate π. Once we obtain the joint estimate Q<i>,</i> we have enough information to derive the conditional probability Q(y<i>=</i>1∣x), which turns out to be

               </p><div class="inline-equation" id="di-fml-14"><span class="inline-equation__construct"><img class="" src="ecog_5203_mu10.gif" alt="image" title="image"></span><span class="inline-equation__label"></span></div>

where q<sub>λ</sub> is the maximum entropy estimate of π and H is the entropy of q<sub>λ</sub><i>.</i> Similarly,

               <div class="inline-equation" id="di-fml-15"><span class="inline-equation__construct"><img class="" src="ecog_5203_mu11.gif" alt="image" title="image"></span><span class="inline-equation__label"></span></div>

(for the derivation see Dudík and Phillips unpubl.). Thus, Q(y=1∣z) 
takes the form of a logistic regression model with the same set of 
parameters λ as the Maxent model and with the intercept determined by 
the entropy of q<sub>λ</sub>. Because of the robust Bayes interpretation
 of Maxent (see “Basic concepts”, above), we expect that the estimate 
Q(y=1∣z) will perform well against a range of sampling distributions and
 prevalence values.
            <p></p>
            
            <p>The model Q(y=1∣z) can also be interpreted from the point
 of view of information theory as follows. Suppose that we receive a 
sequence of independent samples from the Maxent distribution q<sub>λ</sub>,
 corresponding to a sequence of observations. Then the average of their 
log probabilities will be very close to −H, the negative entropy (<span><a href="#b3" id="#b3R" class="bibLink tab-link" data-tab="pane-pcw-references">Cover and Thomas 2006</a></span>), because −H is simply the mean log probability: 
<img alt="inline image" class="section_image" src="ecog_5203_mu12.gif">. Thus, for “typical” sites whose log probabilities are close to this mean, we obtain q<sub>λ</sub>(x)≈e<sup>‐H</sup>. The model Q therefore assigns typical presence sites probability of presence close to 0.5.
            </p>
            
            <p>We may have a prior expectation about the probability of 
presence at typical presence sites: for example, extensive collecting 
effort may have been required to obtain the known occurrence records for
 a rare species, suggesting that its probability of presence is low 
everywhere. This information could potentially be incorporated as a 
constraint on P(x, y). However, probability of presence depends on 
sampling effort, and in particular on site size and, for vagile 
organisms, on observation time. Therefore, we can more simply 
incorporate knowledge of sampling effort by interpreting Q(y=1∣z) as 
probability of presence under a similar level of sampling effort as was 
required to obtain the known occurrence data.</p>
            
            <p>Note that the raw, cumulative and logistic formats are 
all monotonically related, so they rank sites in the same order and 
therefore result in identical performance, when measured using 
rank‐based statistics such as AUC (<span><a href="#b9" id="#b9R" class="bibLink tab-link" data-tab="pane-pcw-references">Fielding and Bell 1997</a></span>).
 However, their predictive performance will vary when measured by 
statistics that depend on actual output values such as Pearson's 
correlation (<span><a href="#b38" id="#b38R" class="bibLink tab-link" data-tab="pane-pcw-references">Zheng and Agresti 2000</a></span>).
            </p>
            
         </section>
         
      </section>
      <section class="article-section__content" id="ss6">
         
         <h2 class="article-section__title section__title section1" id="ss6-title"> Experimental methods</h2>
         
         <section class="article-section__sub-content" id="ss7">
            
            <h3 class="article-section__sub-title section2" id="ss7-title"> Species occurrence data and environmental predictors</h3>
            
            <p>We used a comprehensive collection of data developed by a
 working group at the National Center for Ecological Analysis and 
Synthesis (NCEAS) as part of a large‐scale comparison of species 
distribution modeling methods (<span><a href="#b8" id="#b8R" class="bibLink tab-link" data-tab="pane-pcw-references">Elith et al. 2006</a></span>).
 We refer to the data as “the NCEAS data”, and the comparison of methods
 as “the NCEAS comparison”. The NCEAS data consists of two independent 
datasets for 226 species from 6 regions of the world (<a href="#t1" class="scrollableLink">Table 1</a>).
            </p>
            
            <div class="article-table-content" id="t1">
               <header class="article-table-caption"><span class="table-caption__label">Table 1.
            </span>The six study regions and their corresponding numbers
 of species and environmental variable types: categorical (categ.), 
discrete ordinal (ord.), continuous (cont.) and binary, which can be 
considered as discrete ordinal or categorical.
               </header>
               
               
               
               
               <div class="article-table-content-wrapper">
                  <table class="table article-section__table">
                     
                     
                     
                     
                     
                     
                     <thead>
                        
                        <tr>
                           
                           <th class="bottom-bordered-cell left-aligned">Code</th>
                           
                           <th class="bottom-bordered-cell center-aligned">Region</th>
                           
                           <th class="bottom-bordered-cell center-aligned">Taxonomic groups</th>
                           
                           <th class="bottom-bordered-cell center-aligned">Environmental variables</th>
                           
                           <th class="bottom-bordered-cell center-aligned">Species</th>
                           
                        </tr>
                        
                     </thead>
                     
                     <tbody>
                        
                        <tr>
                           
                           <td class="left-aligned">awt</td>
                           
                           <td class="left-aligned">Australian wet tropics</td>
                           
                           <td class="left-aligned">Bird, plant</td>
                           
                           <td class="left-aligned">13 cont.</td>
                           
                           <td class="center-aligned">40</td>
                           
                        </tr>
                        
                        <tr>
                           
                           <td class="left-aligned">can</td>
                           
                           <td class="left-aligned">Ontario, Canada</td>
                           
                           <td class="left-aligned">Bird</td>
                           
                           <td class="left-aligned">10 cont., 1 categ.</td>
                           
                           <td class="center-aligned">20</td>
                           
                        </tr>
                        
                        <tr>
                           
                           <td class="left-aligned">nsw</td>
                           
                           <td class="left-aligned">NE New South Wales</td>
                           
                           <td class="left-aligned">Bird, mammal, plant, reptile</td>
                           
                           <td class="left-aligned">10 cont., 1 categ., 2 ord.</td>
                           
                           <td class="center-aligned">54</td>
                           
                        </tr>
                        
                        <tr>
                           
                           <td class="left-aligned">nz</td>
                           
                           <td class="left-aligned">New Zealand</td>
                           
                           <td class="left-aligned">Plant</td>
                           
                           <td class="left-aligned">11 cont., 2 ord.</td>
                           
                           <td class="center-aligned">52</td>
                           
                        </tr>
                        
                        <tr>
                           
                           <td class="left-aligned">sa</td>
                           
                           <td class="left-aligned">South America</td>
                           
                           <td class="left-aligned">Plant</td>
                           
                           <td class="left-aligned">11 cont.</td>
                           
                           <td class="center-aligned">30</td>
                           
                        </tr>
                        
                        <tr>
                           
                           <td class="left-aligned">swi</td>
                           
                           <td class="left-aligned">Switzerland</td>
                           
                           <td class="left-aligned">Plant</td>
                           
                           <td class="left-aligned">12 cont., 1 binary</td>
                           
                           <td class="center-aligned">30</td>
                           
                        </tr>
                        
                     </tbody>
                     
                  </table>
               </div>
               
               
               <div class="article-section__table-source"></div>
            </div>
            
            <p>The first dataset contains presence‐only data, i.e. a set
 of geographic coordinates of recorded presence sites for each species 
together with a set of environmental variables for each of the 6 
regions. The environmental variables describe grids of 2 million to 27 
million sites, each of which we downscaled to 1.5 million sites in order
 to reduce computation time. The number of presence sites per species 
ranges from 2 to 5822, with a median of 57. These presence‐only data 
constitute the training data used to make models. The second dataset is 
presence‐absence evaluation data: for each species, the evaluation data 
contains a set of sites of observed presence and a set of sites of 
observed absence. The number of test sites (presence and absence 
combined) ranges from 102 to 19 120. The presence‐only data are derived 
from museum and herbarium‐type collections, while the presence‐absence 
data are derived from rigorous surveys that sample across both 
environmental and geographic space. For more details, see <span><a href="#b8" id="#b8R" class="bibLink tab-link" data-tab="pane-pcw-references">Elith et al. (2006)</a></span>.
            </p>
            
         </section>
         
         <section class="article-section__sub-content" id="ss8">
            
            <h3 class="article-section__sub-title section2" id="ss8-title"> Performance measures</h3>
            
            <p>One measure of performance applicable to Maxent is log 
loss. It equals the negative log likelihood of the test data, i.e. the 
sum of the negative log probabilities that the Maxent model assigns to 
test sites. Log loss is the quantity that Maxent optimizes (<span><a href="#b4" id="#b4R" class="bibLink tab-link" data-tab="pane-pcw-references">Dudík et al. 2004</a></span>).
 It is always non‐negative and can be arbitrarily large. Smaller values 
correspond to better prediction (higher likelihood). The uniform 
distribution over a set of N sites achieves a log loss of ln N. The true
 distribution π achieves the minimum log loss equal to the entropy of π,
 which is always smaller than or equal to ln N. We can use log loss on 
presence‐only test data to evaluate a Maxent model, but (at least in the
 form used here) it is not suitable for use with presence‐absence test 
data.
            </p>
            
            <p>Another performance measure, applicable to any species 
modeling method, is the area under the ROC curve (AUC), which measures 
the quality of a ranking of sites (<span><a href="#b9" id="#b9R" class="bibLink tab-link" data-tab="pane-pcw-references">Fielding and Bell 1997</a></span>).
 The AUC is the probability that a randomly chosen presence site will be
 ranked above a randomly chosen absence site. A random ranking has on 
average an AUC of 0.5, and a perfect ranking achieves the best possible 
AUC of 1.0. Models with values above 0.75 are considered potentially 
useful (<span><a href="#b6" id="#b6R" class="bibLink tab-link" data-tab="pane-pcw-references">Elith 2002</a></span>).
 Sometimes we do not have any absence data with which to measure AUC: 
this is true in the present study only in our tuning experiments, where 
the tuning is being performed using presence‐only data. In this case, an
 AUC can be calculated by using background data (also called 
pseudo‐absences) chosen uniformly at random from the study area, in 
place of true absences. The interpretation changes accordingly: the AUC 
is now the probability that a randomly chosen presence site is ranked 
above a random background site (<span><a href="#b27" id="#b27R" class="bibLink tab-link" data-tab="pane-pcw-references">Phillips et al. 2006</a></span>).
            </p>
            
            <p>The correlation between a model's predictions and 
presence/absence in test data (regarded as a 0–1 variable) is known as 
the point biserial correlation, and can be calculated as a Pearson 
correlation coefficient (COR; <span><a href="#b38" id="#b38R" class="bibLink tab-link" data-tab="pane-pcw-references">Zheng and Agresti 2000</a></span>).
 While AUC is rank‐based, depending only on the relative ordering of 
predictions, COR measures how well calibrated the prediction values are 
(up to scaling and translation), in other words, whether big differences
 in prediction values correspond to big differences in probability of 
presence of the species.
            </p>
            
            <p>Lastly, if a model is interpreted as estimating species' 
probability of presence, rather than just giving an index of habitat 
suitability, then the model predictions can be evaluated using deviance,
 defined as −2 times the log probability of the test data. In 
particular, if the model predicts a value of p for a test site, that 
site contributes −2 ln p to the deviance if the species is present 
there, and −2 ln(1−p) otherwise. To combine deviance values from 
different regions with different amounts of test data, we normalize the 
deviance by dividing by the number of test sites. Like COR, deviance 
measures how well calibrated prediction values are, but in addition, it 
penalizes errors in scaling of prediction values.</p>
            
         </section>
         
      </section>
      <section class="article-section__content" id="ss9">
         
         <h2 class="article-section__title section__title section1" id="ss9-title"> Experiments for tuning Maxent</h2>
         
         <p>In this section, we describe the tuning of regularization 
parameters and selection of the best‐performing sets of feature classes,
 using presence‐only data. We selected a small subset of species in each
 region to use for tuning purposes. Our goal was to include a diverse 
set of biological groups and a wide set of sample sizes. <a href="#t2" class="scrollableLink">Table 2</a> lists the selected species as well as experiments where they were used (experiments are described below).
         </p>
         
         <div class="article-table-content" id="t2">
            <header class="article-table-caption"><span class="table-caption__label">Table 2.
            </span>Species used for tuning regularization parameters. 
The fourth column gives the number of occurrence records in the 
presence‐only data used for the parameter tuning; the last column lists 
sections that describe experiments in which the species was included.
            </header>
            
            
            
            
            <div class="article-table-content-wrapper">
               <table class="table article-section__table">
                  
                  
                  
                  
                  
                  
                  <thead>
                     
                     <tr>
                        
                        <th class="bottom-bordered-cell left-aligned">Region/code</th>
                        
                        <th class="bottom-bordered-cell center-aligned">Species</th>
                        
                        <th class="bottom-bordered-cell center-aligned">Group</th>
                        
                        <th class="bottom-bordered-cell center-aligned">#PO</th>
                        
                        <th class="bottom-bordered-cell center-aligned">Experiments</th>
                        
                     </tr>
                     
                  </thead>
                  
                  <tbody>
                     
                     <tr>
                        
                        <td class="left-aligned">awt/ausrob</td>
                        
                        <td class="left-aligned"> <i>Austrochaperina robusta</i> 
                        </td>
                        
                        <td class="left-aligned">Frog</td>
                        
                        <td class="left-aligned">193</td>
                        
                        <td class="left-aligned">Reg L, LQ, LQP, T; Opt</td>
                        
                     </tr>
                     
                     <tr>
                        
                        <td class="left-aligned">awt/bhe</td>
                        
                        <td class="left-aligned"> <i>Lichenostomus frenatus</i> 
                        </td>
                        
                        <td class="left-aligned">bird</td>
                        
                        <td class="left-aligned">351</td>
                        
                        <td class="left-aligned">Opt</td>
                        
                     </tr>
                     
                     <tr>
                        
                        <td class="left-aligned">awt/coporn</td>
                        
                        <td class="left-aligned"> <i>Cophixalus ornatus</i> 
                        </td>
                        
                        <td class="left-aligned">frog</td>
                        
                        <td class="left-aligned">337</td>
                        
                        <td class="left-aligned">Opt</td>
                        
                     </tr>
                     
                     <tr>
                        
                        <td class="left-aligned">awt/cryliv</td>
                        
                        <td class="left-aligned"> <i>Cryptocarya lividula</i> 
                        </td>
                        
                        <td class="left-aligned">plant</td>
                        
                        <td class="left-aligned">44</td>
                        
                        <td class="left-aligned">Opt</td>
                        
                     </tr>
                     
                     <tr>
                        
                        <td class="left-aligned">awt/ghr</td>
                        
                        <td class="left-aligned"> <i>Heteromyias albispecularis</i> 
                        </td>
                        
                        <td class="left-aligned">bird</td>
                        
                        <td class="left-aligned">484</td>
                        
                        <td class="left-aligned">Opt</td>
                        
                     </tr>
                     
                     <tr>
                        
                        <td class="left-aligned">awt/guiacu</td>
                        
                        <td class="left-aligned"> <i>Guioa acutifolia</i> 
                        </td>
                        
                        <td class="left-aligned">plant</td>
                        
                        <td class="left-aligned">56</td>
                        
                        <td class="left-aligned">Reg L, LQ, LQP; Opt</td>
                        
                     </tr>
                     
                     <tr>
                        
                        <td class="left-aligned">awt/lamcog</td>
                        
                        <td class="left-aligned"> <i>Lampropholis coggeri</i> 
                        </td>
                        
                        <td class="left-aligned">reptile</td>
                        
                        <td class="left-aligned">165</td>
                        
                        <td class="left-aligned">Opt</td>
                        
                     </tr>
                     
                     <tr>
                        
                        <td class="left-aligned">awt/sapbas</td>
                        
                        <td class="left-aligned"> <i>Saproscincus basiliscus</i> 
                        </td>
                        
                        <td class="left-aligned">reptile</td>
                        
                        <td class="left-aligned">177</td>
                        
                        <td class="left-aligned">Opt</td>
                        
                     </tr>
                     
                     <tr>
                        
                        <td class="left-aligned">can/amcr</td>
                        
                        <td class="left-aligned"> <i>Corvus brachyrhynchos</i> 
                        </td>
                        
                        <td class="left-aligned">bird</td>
                        
                        <td class="left-aligned">483</td>
                        
                        <td class="left-aligned">Opt; Cat</td>
                        
                     </tr>
                     
                     <tr>
                        
                        <td class="left-aligned">can/cogr</td>
                        
                        <td class="left-aligned"> <i>Quiscalus quiscula</i> 
                        </td>
                        
                        <td class="left-aligned">bird</td>
                        
                        <td class="left-aligned">721</td>
                        
                        <td class="left-aligned">Reg L, LQ, LQP, C, T; Cat; Opt; Cat</td>
                        
                     </tr>
                     
                     <tr>
                        
                        <td class="left-aligned">can/eato</td>
                        
                        <td class="left-aligned"> <i>Pipilo crythrophthalmus</i> 
                        </td>
                        
                        <td class="left-aligned">bird</td>
                        
                        <td class="left-aligned">119</td>
                        
                        <td class="left-aligned">Cat; Opt; Cat</td>
                        
                     </tr>
                     
                     <tr>
                        
                        <td class="left-aligned">can/gcki</td>
                        
                        <td class="left-aligned"> <i>Regulus satrapa</i> 
                        </td>
                        
                        <td class="left-aligned">bird</td>
                        
                        <td class="left-aligned">18</td>
                        
                        <td class="left-aligned">Opt; Cat</td>
                        
                     </tr>
                     
                     <tr>
                        
                        <td class="left-aligned">can/hosp</td>
                        
                        <td class="left-aligned"> <i>Passer domesticus</i> 
                        </td>
                        
                        <td class="left-aligned">bird</td>
                        
                        <td class="left-aligned">615</td>
                        
                        <td class="left-aligned">Opt; Cat</td>
                        
                     </tr>
                     
                     <tr>
                        
                        <td class="left-aligned">can/inbu</td>
                        
                        <td class="left-aligned"> <i>Passerina cyanea</i> 
                        </td>
                        
                        <td class="left-aligned">bird</td>
                        
                        <td class="left-aligned">138</td>
                        
                        <td class="left-aligned">Reg L, LQ, LQP, C; Cat; Opt; Cat</td>
                        
                     </tr>
                     
                     <tr>
                        
                        <td class="left-aligned">can/modo</td>
                        
                        <td class="left-aligned"> <i>Zenaida macrovra</i> 
                        </td>
                        
                        <td class="left-aligned">bird</td>
                        
                        <td class="left-aligned">749</td>
                        
                        <td class="left-aligned">Cat; Opt; Cat</td>
                        
                     </tr>
                     
                     <tr>
                        
                        <td class="left-aligned">can/wtsp</td>
                        
                        <td class="left-aligned"> <i>Zonotrichia albicollis</i> 
                        </td>
                        
                        <td class="left-aligned">bird</td>
                        
                        <td class="left-aligned">313</td>
                        
                        <td class="left-aligned">Cat; Opt; Cat</td>
                        
                     </tr>
                     
                     <tr>
                        
                        <td class="left-aligned">nsw/basp2</td>
                        
                        <td class="left-aligned"> <i>Falsistrellus tasmaniensis</i> 
                        </td>
                        
                        <td class="left-aligned">mammal</td>
                        
                        <td class="left-aligned">28</td>
                        
                        <td class="left-aligned">Opt; Cat</td>
                        
                     </tr>
                     
                     <tr>
                        
                        <td class="left-aligned">nsw/dbsp2</td>
                        
                        <td class="left-aligned"> <i>Calyptorhynchus lathami</i> 
                        </td>
                        
                        <td class="left-aligned">bird</td>
                        
                        <td class="left-aligned">426</td>
                        
                        <td class="left-aligned">Reg L, LQ, LQP, C, T; Cat; Opt; Cat</td>
                        
                     </tr>
                     
                     <tr>
                        
                        <td class="left-aligned">nsw/dbsp7</td>
                        
                        <td class="left-aligned"> <i>Myzomela sanguinolenta</i> 
                        </td>
                        
                        <td class="left-aligned">bird</td>
                        
                        <td class="left-aligned">315</td>
                        
                        <td class="left-aligned">Cat; Ord; Opt; Cat</td>
                        
                     </tr>
                     
                     <tr>
                        
                        <td class="left-aligned">nsw/nbsp2</td>
                        
                        <td class="left-aligned"> <i>Tyto tenebricosa</i> 
                        </td>
                        
                        <td class="left-aligned">bird</td>
                        
                        <td class="left-aligned">120</td>
                        
                        <td class="left-aligned">Cat; Ord; Opt; Cat</td>
                        
                     </tr>
                     
                     <tr>
                        
                        <td class="left-aligned">nsw/otsp7</td>
                        
                        <td class="left-aligned"> <i>Eucalyptus campanulata</i> 
                        </td>
                        
                        <td class="left-aligned">plant</td>
                        
                        <td class="left-aligned">69</td>
                        
                        <td class="left-aligned">Cat; Ord; Opt; Cat</td>
                        
                     </tr>
                     
                     <tr>
                        
                        <td class="left-aligned">nsw/rusp2</td>
                        
                        <td class="left-aligned"> <i>Cyathea leichhardtiana</i> 
                        </td>
                        
                        <td class="left-aligned">plant</td>
                        
                        <td class="left-aligned">42</td>
                        
                        <td class="left-aligned">Opt; Cat</td>
                        
                     </tr>
                     
                     <tr>
                        
                        <td class="left-aligned">nsw/srsp5</td>
                        
                        <td class="left-aligned"> <i>Eulamprus murrayi</i> 
                        </td>
                        
                        <td class="left-aligned">reptile</td>
                        
                        <td class="left-aligned">186</td>
                        
                        <td class="left-aligned">Opt; Cat</td>
                        
                     </tr>
                     
                     <tr>
                        
                        <td class="left-aligned">nsw/srsp7</td>
                        
                        <td class="left-aligned"> <i>Pseudechis porphyricaus</i> 
                        </td>
                        
                        <td class="left-aligned">reptile</td>
                        
                        <td class="left-aligned">118</td>
                        
                        <td class="left-aligned">Reg L, LQ, LQP, C; Cat; Opt; Cat</td>
                        
                     </tr>
                     
                     <tr>
                        
                        <td class="left-aligned">nz/clefor</td>
                        
                        <td class="left-aligned"> <i>Clematis forsteri</i> 
                        </td>
                        
                        <td class="left-aligned">plant</td>
                        
                        <td class="left-aligned">36</td>
                        
                        <td class="left-aligned">Opt</td>
                        
                     </tr>
                     
                     <tr>
                        
                        <td class="left-aligned">nz/coppro</td>
                        
                        <td class="left-aligned"> <i>Coprosma propinqua</i> 
                        </td>
                        
                        <td class="left-aligned">plant</td>
                        
                        <td class="left-aligned">40</td>
                        
                        <td class="left-aligned">Opt</td>
                        
                     </tr>
                     
                     <tr>
                        
                        <td class="left-aligned">nz/drauni</td>
                        
                        <td class="left-aligned"> <i>Dracophyllum uniflorum</i> 
                        </td>
                        
                        <td class="left-aligned">plant</td>
                        
                        <td class="left-aligned">174</td>
                        
                        <td class="left-aligned">Reg L, LQ, LQP, T; Ord; Opt</td>
                        
                     </tr>
                     
                     <tr>
                        
                        <td class="left-aligned">nz/libbid</td>
                        
                        <td class="left-aligned"> <i>Libocedrus bidwillii</i> 
                        </td>
                        
                        <td class="left-aligned">plant</td>
                        
                        <td class="left-aligned">105</td>
                        
                        <td class="left-aligned">Opt</td>
                        
                     </tr>
                     
                     <tr>
                        
                        <td class="left-aligned">nz/metper</td>
                        
                        <td class="left-aligned"> <i>Metrosideros perforata</i> 
                        </td>
                        
                        <td class="left-aligned">plant</td>
                        
                        <td class="left-aligned">87</td>
                        
                        <td class="left-aligned">Opt</td>
                        
                     </tr>
                     
                     <tr>
                        
                        <td class="left-aligned">nz/metrob</td>
                        
                        <td class="left-aligned"> <i>Metrosideros robusta</i> 
                        </td>
                        
                        <td class="left-aligned">plant</td>
                        
                        <td class="left-aligned">48</td>
                        
                        <td class="left-aligned">Opt</td>
                        
                     </tr>
                     
                     <tr>
                        
                        <td class="left-aligned">nz/phyalp</td>
                        
                        <td class="left-aligned"> <i>Phyllocladus alpinus</i> 
                        </td>
                        
                        <td class="left-aligned">plant</td>
                        
                        <td class="left-aligned">211</td>
                        
                        <td class="left-aligned">Opt</td>
                        
                     </tr>
                     
                     <tr>
                        
                        <td class="left-aligned">nz/prutax</td>
                        
                        <td class="left-aligned"> <i>Prumnopitys taxifolia</i> 
                        </td>
                        
                        <td class="left-aligned">plant</td>
                        
                        <td class="left-aligned">130</td>
                        
                        <td class="left-aligned">Reg L, LQ, LQP; Ord; Opt</td>
                        
                     </tr>
                     
                     <tr>
                        
                        <td class="left-aligned">sa/amphpani</td>
                        
                        <td class="left-aligned"> <i>Amphilophium paniculatum</i> 
                        </td>
                        
                        <td class="left-aligned">plant</td>
                        
                        <td class="left-aligned">88</td>
                        
                        <td class="left-aligned">Reg L, LQ, LQP, T; Opt</td>
                        
                     </tr>
                     
                     <tr>
                        
                        <td class="left-aligned">sa/arrabrac</td>
                        
                        <td class="left-aligned"> <i>Arrabidaea brachypoda</i> 
                        </td>
                        
                        <td class="left-aligned">plant</td>
                        
                        <td class="left-aligned">203</td>
                        
                        <td class="left-aligned">Reg L, LQ, LQP; Opt</td>
                        
                     </tr>
                     
                     <tr>
                        
                        <td class="left-aligned">sa/arracinn</td>
                        
                        <td class="left-aligned"> <i>Arrabidaea cinnomomea</i> 
                        </td>
                        
                        <td class="left-aligned">plant</td>
                        
                        <td class="left-aligned">49</td>
                        
                        <td class="left-aligned">Opt</td>
                        
                     </tr>
                     
                     <tr>
                        
                        <td class="left-aligned">sa/cydiaequ</td>
                        
                        <td class="left-aligned"> <i>Cydista aequinoctialis</i> 
                        </td>
                        
                        <td class="left-aligned">plant</td>
                        
                        <td class="left-aligned">138</td>
                        
                        <td class="left-aligned">Opt</td>
                        
                     </tr>
                     
                     <tr>
                        
                        <td class="left-aligned">sa/distmagn</td>
                        
                        <td class="left-aligned"> <i>Distictella magnoliifolia</i> 
                        </td>
                        
                        <td class="left-aligned">plant</td>
                        
                        <td class="left-aligned">81</td>
                        
                        <td class="left-aligned">Opt</td>
                        
                     </tr>
                     
                     <tr>
                        
                        <td class="left-aligned">sa/fridspec</td>
                        
                        <td class="left-aligned"> <i>Fridericia speciosa</i> 
                        </td>
                        
                        <td class="left-aligned">plant</td>
                        
                        <td class="left-aligned">57</td>
                        
                        <td class="left-aligned">Opt</td>
                        
                     </tr>
                     
                     <tr>
                        
                        <td class="left-aligned">sa/lundvirg</td>
                        
                        <td class="left-aligned"> <i>Lundia virginalis</i> 
                        </td>
                        
                        <td class="left-aligned">plant</td>
                        
                        <td class="left-aligned">36</td>
                        
                        <td class="left-aligned">Opt</td>
                        
                     </tr>
                     
                     <tr>
                        
                        <td class="left-aligned">sa/parapyra</td>
                        
                        <td class="left-aligned"> <i>Paragonia pyramidata</i> 
                        </td>
                        
                        <td class="left-aligned">plant</td>
                        
                        <td class="left-aligned">216</td>
                        
                        <td class="left-aligned">Opt</td>
                        
                     </tr>
                     
                     <tr>
                        
                        <td class="left-aligned">swi/abialb</td>
                        
                        <td class="left-aligned"> <i>Abies alba</i> 
                        </td>
                        
                        <td class="left-aligned">plant</td>
                        
                        <td class="left-aligned">3357</td>
                        
                        <td class="left-aligned">Opt</td>
                        
                     </tr>
                     
                     <tr>
                        
                        <td class="left-aligned">swi/acepse</td>
                        
                        <td class="left-aligned"> <i>Acer pseudoplatanus</i> 
                        </td>
                        
                        <td class="left-aligned">plant</td>
                        
                        <td class="left-aligned">2800</td>
                        
                        <td class="left-aligned">Ord; Opt</td>
                        
                     </tr>
                     
                     <tr>
                        
                        <td class="left-aligned">swi/betpen</td>
                        
                        <td class="left-aligned"> <i>Betula pendula</i> 
                        </td>
                        
                        <td class="left-aligned">plant</td>
                        
                        <td class="left-aligned">468</td>
                        
                        <td class="left-aligned">Opt</td>
                        
                     </tr>
                     
                     <tr>
                        
                        <td class="left-aligned">swi/fagsyl</td>
                        
                        <td class="left-aligned"> <i>Fagus sylvatica</i> 
                        </td>
                        
                        <td class="left-aligned">plant</td>
                        
                        <td class="left-aligned">5528</td>
                        
                        <td class="left-aligned">Reg L, LQ, LQP, T; Ord; Opt</td>
                        
                     </tr>
                     
                     <tr>
                        
                        <td class="left-aligned">swi/pincem</td>
                        
                        <td class="left-aligned"> <i>Pinus cembra</i> 
                        </td>
                        
                        <td class="left-aligned">plant</td>
                        
                        <td class="left-aligned">279</td>
                        
                        <td class="left-aligned">Reg L, LQ, LQP; Ord; Opt</td>
                        
                     </tr>
                     
                     <tr>
                        
                        <td class="left-aligned">swi/pinunc</td>
                        
                        <td class="left-aligned"> <i>Pinus uncinata</i> 
                        </td>
                        
                        <td class="left-aligned">plant</td>
                        
                        <td class="left-aligned">291</td>
                        
                        <td class="left-aligned">Opt</td>
                        
                     </tr>
                     
                     <tr>
                        
                        <td class="left-aligned">swi/poptre</td>
                        
                        <td class="left-aligned"> <i>Populus tremula</i> 
                        </td>
                        
                        <td class="left-aligned">plant</td>
                        
                        <td class="left-aligned">154</td>
                        
                        <td class="left-aligned">Opt</td>
                        
                     </tr>
                     
                     <tr>
                        
                        <td class="left-aligned">swi/pruavi</td>
                        
                        <td class="left-aligned"> <i>Prunus avium</i> 
                        </td>
                        
                        <td class="left-aligned">plant</td>
                        
                        <td class="left-aligned">613</td>
                        
                        <td class="left-aligned">Opt</td>
                        
                     </tr>
                     
                  </tbody>
                  
               </table>
            </div>
            
            
            <div class="article-section__table-source"></div>
         </div>
         
         <p>In the tuning experiments, we measured the performance of 
Maxent for various parameter settings as follows. We randomly 
partitioned occurrence records of every species into a training set with
 60% of the records and the test set with 40% of the records. We ran 
Maxent on the training set and evaluated its performance on the test set
 and took the average over 5–10 random partitions (see below). In all 
the tuning experiments, the performance is measured in terms of log loss
 and AUC, where the latter is measured using background data in place of
 true absences. Since the downscaled grids have size N=1.5×10<sup>6</sup>, the uniform distribution has a log loss of ln 1.5×10<sup>6</sup>=14.22. The AUC of the uniform distribution is 0.5.
         </p>
         
         <section class="article-section__sub-content" id="ss10">
            
            <h3 class="article-section__sub-title section2" id="ss10-title"> Tuning regularization parameters using presence‐only data (Reg)</h3>
            
            <p>The goal of the first set of experiments was to determine
 regularization parameter values for individual feature classes – called
 β<sub>L</sub>, β<sub>Q</sub>, β<sub>P</sub>, β<sub>T</sub>, β<sub>C</sub>–
 yielding good performance in all six regions for varying numbers of 
occurrence records. We generated models for different feature class 
settings (L, LQ, LQP, T, C; hinge features are considered separately in 
“Experiments for new extensions to Maxent”, below). For each setting, we
 assumed a single regularization parameter; in particular, for the LQ 
setting we kept β<sub>L</sub>=β<sub>Q</sub> and for the LQP setting we kept β<sub>L</sub>=β<sub>Q</sub>=β<sub>P</sub>.
            </p>
            
            <p>For each feature class setting, we varied the number of 
occurrences (by considering nested subsets of the full training set) and
 the value of the regularization parameter β. The number of occurrences 
was chosen from the geometrically increasing sequence {6, 10, 17, 30, 
55, 100, 178, 316, 1000, 3162}, and the values of β from the 
geometrically increasing sequence {0.02, 0.05, 0.10, 0.22, 0.46, 1.0, 
2.2, 4.6} chosen to bracket the range of suitable values suggested by 
theory (<span><a href="#b4" id="#b4R" class="bibLink tab-link" data-tab="pane-pcw-references">Dudík et al. 2004</a></span>).
 For each number of occurrences, we determined the average performance 
over five random partitions as a function of β. We call the 
corresponding plots β‐curves. Peaks of β‐curves (minima of log loss 
curves and maxima of AUC curves) correspond to optimal choices of β for 
each particular species. Some prior knowledge was used to restrict the 
set of β's and numbers of occurrences to intervals where the peaks are 
likely to occur; in particular, L runs were not configured on large 
sample sizes and LQP runs were not configured on small sample sizes. 
Also note that C runs were only possible for regions can and nsw, with 
one categorical variable in each region.
            </p>
            
            <p>For each feature class setting and number of occurrences,
 we selected the best β by visual inspection of β‐curves. The goal was 
to choose β performing well in terms of both log loss and AUC on all of 
the evaluated species. We first excluded curves where Maxent was not 
performing well: log loss β‐curves where Maxent never reached 
performance below 14.2 (the log loss of the uniform distribution) and 
AUC curves that remained below 0.7 (based on the recommendations of <span><a href="#b6" id="#b6R" class="bibLink tab-link" data-tab="pane-pcw-references">Elith (2002)</a></span>
 that values above 0.75 are considered potentially useful). On the 
remaining curves we used visual inspection to employ two strategies that
 could be loosely termed as “mean criterion” and “median criterion”. 
According to the former criterion, we chose the value β that was close 
to the peak of β‐curves of as many species as possible. According to the
 latter criterion, we chose β at which about half of the β‐curves were 
increasing and half were decreasing. The latter criterion was used 
whenever the peak was not identifiable in β‐curves (they were monotone).
 The optimal values of β for numbers of occurrences between those in the
 evaluated sequence were obtained by linear interpolation.
            </p>
            
         </section>
         
         <section class="article-section__sub-content" id="ss11">
            
            <h3 class="article-section__sub-title section2" id="ss11-title"> Combining continuous and categorical variables (Cat)</h3>
            
            <p>In the initial block of experiments (see above), the regularization parameter for category indicators β<sub>C</sub>
 was determined in Maxent runs with a single categorical variable. When 
categorical variables are used with additional continuous variables, the
 total number of features increases, so we expect that higher values of β<sub>C</sub>
 will yield better performance (see the discussion in “Maxent modeling 
of distributions”, above). In this set of experiments, we explored a 
range of alternative settings of β<sub>C</sub> in runs including L, Q and P features derived from continuous variables.
            </p>
            
            <p>We carried out LC, LQC and LQPC runs with β<sub>L</sub>, β<sub>Q</sub>, β<sub>p</sub> equal to the previously determined optimum and for three different settings of β<sub>C</sub>. The first β<sub>C</sub> setting, β<sub>C</sub>=<i>low</i>, corresponds to the value determined in the initial block of experiments, the second setting β<sub>C</sub>=β<sub>LQP</sub>
 corresponds to using the same regularization for category indicators as
 for L, Q and P features, and the third setting, an approximation of the
 geometric average of the previous two settings, is an intermediate 
regularization choice.
            </p>
            
            <p>For each setting of β<sub>C</sub>, we plotted the average
 performance over 10 partitions as a function of an increasing number of
 samples from nested subsets of sizes {5, 10, 20, 40, 75, 150, 300, 750,
 2000}. (This sequence differs from the sequence used in “Tuning 
regularization parameters using presence‐only data”, above). The 
rationale behind choosing a different sequence was to evaluate Maxent 
for the training set sizes where the interpolated values of β are used.)
 The resulting plots are referred to as m‐curves. The best setting of β<sub>C</sub>
 was again chosen by visual inspection of graphs with the goal being to 
perform well on all evaluated species both in terms of AUC and log loss.
 In the current block of experiments, we marked all discrete ordinal 
variables as categorical to obtain a larger number of categorical 
variables and hence a more reliable tuning of β<sub>C</sub>.
            </p>
            
         </section>
         
         <section class="article-section__sub-content" id="ss12">
            
            <h3 class="article-section__sub-title section2" id="ss12-title"> Using discrete ordinal variables (Ord)</h3>
            
            <p>Next, we explored the effect of treating discrete ordinal
 variables as categorical or continuous. For the former case, we used 
the optimal β<sub>C</sub> determined in the previous experiment. For the latter case, we consider two settings of β<sub>C</sub>: the previously determined optimal setting and the baseline setting β<sub>C</sub>=β<sub>LQP</sub>
 which uses a single regularization parameter for all features (this was
 the setting in versions of Maxent prior to 1.8.3). The optimal setting 
is determined by visual inspection of m‐curves for LC, LQC and LQPC 
runs.
            </p>
            
         </section>
         
         <section class="article-section__sub-content" id="ss13">
            
            <h3 class="article-section__sub-title section2" id="ss13-title"> Choosing optimal feature sets (Opt)</h3>
            
            <p>The final goal of the tuning experiments was to decide 
which sets of feature classes to use for what numbers of species 
occurrences. We used the previously determined regularization parameters
 for the LC, LQC, LQPC and LQPTC runs. The optimal ranges for different 
feature class settings were determined by visual inspection of m‐curves.</p>
            
         </section>
         
         <section class="article-section__sub-content" id="ss14">
            
            <h3 class="article-section__sub-title section2" id="ss14-title"> Evaluating Maxent tuning using presence‐absence test data</h3>
            
            <p>We evaluated our tuning of Maxent's parameters by 
comparing with the best possible settings for the given evaluation data:
 those that yield the best performance when models are built from the 
presence‐only training data and evaluated on the presence‐absence test 
data. We call the latter parameter settings “pa‐tuned”, in contrast to 
the “po‐tuned” values determined in “Tuning regularization parameters 
using presence‐only data”, above and “Combining continuous and 
categorial variables”, above. In po‐tuning, a single set of 
regularization parameters was determined for application across all 
regions. However, it is conceivable that different sets of parameters 
may be appropriate for different regions, and better performance may be 
obtained by tuning the parameters for each region separately. In 
pa‐tuning, we therefore distinguished two cases: regional tuning and 
global tuning. In the former case, a separate set of regularization 
parameters was chosen to maximize the average AUC of the species in the 
relevant region only. In the latter case, a single set of regularization
 parameters was chosen to maximize the average AUC across all species.</p>
            
            <p>Sets of pa‐tuned regularization parameters were obtained 
by local search a heuristic optimization technique that tries to improve
 the value of the objective (in our case the AUC on the presence‐absence
 data) by making incremental changes in parameters. We began with the 
po‐tuned parameter values and then cyclically iterated through feature 
classes, trying to increase or decrease the corresponding regularization
 parameter. This was repeated until no changes in parameters yielded an 
improvement. We considered multiplicative changes in regularization 
parameters by a factor of √2 or 1/√2. We allowed at most an 8‐fold 
increase or decrease relative to the po‐tuned parameter setting. Each 
feature class was applied in the same range of numbers of occurrences as
 determined by po‐tuning in “Choosing optimal features sets”, above.</p>
            
            <p>Both regional and global tuning were performed using two 
different choices of training background sets. We used a random sample 
of 10 000 sites from the study area (random background) and the set of 
all occurrence records for the target group (target‐group background, 
see below).</p>
            
         </section>
         
      </section>
      <section class="article-section__content" id="ss15">
         
         <h2 class="article-section__title section__title section1" id="ss15-title"> Experiments for new extensions to Maxent</h2>
         
         <section class="article-section__sub-content" id="ss16">
            
            <h3 class="article-section__sub-title section2" id="ss16-title"> Hinge features</h3>
            
            <p>To evaluate the benefit of using hinge features, we determined their regularization parameter β<sub>H</sub>
 and the minimum number of occurrence records for which they should be 
used, using presence‐only data and the same methodology as in 
“Experiments for tuning Maxent”, above. We then generated models for all
 226 species and evaluated them using the independent presence‐absence 
test data, comparing the resulting AUC and COR values with those 
obtained without hinge features.
            </p>
            
         </section>
         
         <section class="article-section__sub-content" id="ss17">
            
            <h3 class="article-section__sub-title section2" id="ss17-title"> Logistic output</h3>
            
            <p>Logistic, cumulative and raw outputs are all 
monotonically rated, i.e. they rank sites in the same order, so their 
statistical performance is identical when measured using AUC. On the 
other hand, their performance may differ when measured using COR, which 
depends on the actual output values rather than just their ordering. We 
therefore ran Maxent with all three output formats on the full suite of 
226 species, and compared the three sets of predictions using the COR 
statistic evaluated using the independent presence/absence test data.</p>
            
         </section>
         
         <section class="article-section__sub-content" id="ss18">
            
            <h3 class="article-section__sub-title section2" id="ss18-title"> Sample selection bias and target‐group background</h3>
            
            <p>Section “Maxent modeling of distributions” above assumes 
that occurrences are unbiased samples from the species’ distribution, 
but that assumption is often not valid for occurrence data (<span><a href="#b30" id="#b30R" class="bibLink tab-link" data-tab="pane-pcw-references">Reddy and Dávalos 2003</a></span>).
 A simple strategy to remove sample selection bias is to replace the 
uniform background data by a random sample of background data drawn from
 the sampling distribution. As a result, both the background data and 
species presences become biased in the same manner. Since Maxent is 
choosing the distribution of maximum entropy relative to the provided 
background, the sample selection bias is effectively factored out (<span><a href="#b5" id="#b5R" class="bibLink tab-link" data-tab="pane-pcw-references">Dudík et al. 2005</a></span>).
 When using such a background sample, it may happen that there is no 
probability distribution over the background sites that satisfies all 
the constraints derived from the occurrence data. Therefore, in order to
 ensure that the optimization problem as described in “Maxent modeling 
of distributions”, above remains feasible, the species’ presence sites 
must be added to the background sample if they are not already there. 
The Maxent distribution is thus calculated over the union of the 
background and presence sites, yielding a Gibbs distribution which can 
then be projected onto the rest of the study area (or any other area). 
We investigated whether the set of occurrence data for an entire “target
 group” of species that may be captured or observed using the same 
methods (<span><a href="#b28" id="#b28R" class="bibLink tab-link" data-tab="pane-pcw-references">Ponder et al. 2001</a></span>, <span><a href="#b1" id="#b1R" class="bibLink tab-link" data-tab="pane-pcw-references">Anderson 2003</a></span>)
 can be regarded as a random sample from the sampling distribution and 
thus used as background data. We generated models for all 226 species 
using target‐group background data and compared their predictive 
performance with models generated using random background. The regions 
awt and nsw contained multiple target groups: region awt contained birds
 and plants; region nsw contained birds, reptiles, mammals, and plants. 
The remaining regions had only a single target group.
            </p>
            
         </section>
         
         <section class="article-section__sub-content" id="ss19">
            
            <h3 class="article-section__sub-title section2" id="ss19-title"> Random background sampling</h3>
            
            <p>The runtime of Maxent scales linearly with the size of 
the background set. Thus, for large, fine‐grain maps, Maxent may be 
prohibitively slow. However, because of the considerable uncertainty in 
the presence data, a large amount of background data may not be 
necessary. Instead, we can use a small subsample of the background to 
significantly reduce runtime without sacrificing predictive performance.
 Here we investigate how large a background sample is needed. We chose a
 geometric progression of background sizes, namely the set {63, 125, 
250, 500, 1000, 2000, 4000, 8000, 16 000, 32 000, 64 000, 128 000, 
256 000}. For each background size, we generated models for all 226 
species for each of 10 random background sets of that size, determining 
the average AUC using the independent presence/absence test data. All 
models were generated using regularization parameters and feature 
classes as tuned in “Experiments for tuning Maxent” and “Hinge 
features”, above.</p>
            
         </section>
         
      </section>
      <section class="article-section__content" id="ss20">
         
         <h2 class="article-section__title section__title section1" id="ss20-title"> Results</h2>
         
         <section class="article-section__sub-content" id="ss21">
            
            <h3 class="article-section__sub-title section2" id="ss21-title"> Tuning of Maxent's parameters</h3>
            
            <p>By visual inspection of β‐curves for L, LQ, LQP, T, and C
 runs, we propose the regularization parameter settings given in the top
 portion of <a href="#t3" class="scrollableLink">Table 3</a>. Values set
 in boldface were determined exactly, while the others were obtained 
from the boldface values by interpolation or extrapolation. Values 
between the highest and lowest bold settings are linearly interpolated 
between the two closest bold settings. Values above the highest bold 
settings are kept the same as the highest bold settings, and the values 
below the lowest bold settings are linearly extrapolated from the two 
lowest bold settings. Note that feature classes with larger numbers of 
features generally have higher β values, consistent with the theory in 
“Maxent modeling of distributions”, above. However, for each feature 
class, the best value of β is monotonically non‐increasing in the number
 of occurrence records. In other words, model performance is optimized 
if we use error bounds that decrease in width somewhat faster than 
1/√sample size, as the theory suggests. The β‐curves were generally 
smooth and unimodal (<a href="#f2" class="scrollableLink">Fig. 2</a>).
            </p>
            
            <div class="article-table-content" id="t3">
               <header class="article-table-caption"><span class="table-caption__label">Table 3.
            </span>Parameter settings tuned using presence‐only data. 
Values in boldface were determined exactly, values in italics are 
linearly interpolated or extrapolated, with the exception that the 
values to the right of the listed ranges remain constant. For category 
indicator features, the “low” settings were determined using a single 
(categorical) variable, while the intermediate settings were chosen to 
approximate the geometric average of the “low” setting and β<sub>L</sub>.
               </header>
               
               
               
               
               <div class="article-table-content-wrapper">
                  <table class="table article-section__table">
                     
                     
                     
                     
                     
                     
                     
                     
                     <thead>
                        
                        <tr>
                           
                           <th class="bottom-bordered-cell left-aligned"></th>
                           
                           <th colspan="6" class="bottom-bordered-cell center-aligned">Number of occcurrence records</th>
                           
                        </tr>
                        
                        <tr>
                           
                           <th class="bottom-bordered-cell left-aligned"></th>
                           
                           <th class="bottom-bordered-cell left-aligned">0</th>
                           
                           <th class="bottom-bordered-cell left-aligned">6</th>
                           
                           <th class="bottom-bordered-cell left-aligned">10</th>
                           
                           <th class="bottom-bordered-cell left-aligned">17</th>
                           
                           <th class="bottom-bordered-cell left-aligned">30</th>
                           
                           <th class="bottom-bordered-cell left-aligned">100</th>
                           
                        </tr>
                        
                     </thead>
                     
                     <tbody>
                        
                        <tr>
                           
                           <td class="left-aligned">Linear features: β<sub>L</sub></td>
                           
                           <td class="left-aligned"> <i>1.0</i> 
                           </td>
                           
                           <td class="left-aligned"> <b>1.0</b> 
                           </td>
                           
                           <td class="left-aligned"> <b>1.0</b> 
                           </td>
                           
                           <td class="left-aligned"> <i>0.72</i> 
                           </td>
                           
                           <td class="left-aligned"> <b>0.2</b> 
                           </td>
                           
                           <td class="left-aligned"> <b>0.05</b> 
                           </td>
                           
                        </tr>
                        
                        <tr>
                           
                           <td class="left-aligned">Linear and quadratic features: β<sub>L</sub>, β<sub>Q</sub></td>
                           
                           <td class="center-aligned"> <i>1.3</i> 
                           </td>
                           
                           <td class="center-aligned"> <b>1.0</b> 
                           </td>
                           
                           <td class="center-aligned"> <b>0.8</b> 
                           </td>
                           
                           <td class="center-aligned"> <b>0.5</b> 
                           </td>
                           
                           <td class="center-aligned"> <b>0.25</b> 
                           </td>
                           
                           <td class="center-aligned"> <b>0.05</b> 
                           </td>
                           
                        </tr>
                        
                        <tr>
                           
                           <td class="left-aligned">Linear, quadratic and product features: β<sub>L</sub>, β<sub>Q</sub>, β<sub>P</sub></td>
                           
                           <td class="center-aligned"> <i>2.6</i> 
                           </td>
                           
                           <td class="center-aligned"> <b>2.0</b> 
                           </td>
                           
                           <td class="center-aligned"> <b>1.6</b> 
                           </td>
                           
                           <td class="center-aligned"> <b>0.9</b> 
                           </td>
                           
                           <td class="center-aligned"> <b>0.55</b> 
                           </td>
                           
                           <td class="center-aligned"> <b>0.05</b> 
                           </td>
                           
                        </tr>
                        
                        <tr>
                           
                           <td class="left-aligned">Threshold features: β<sub>T</sub></td>
                           
                           <td class="center-aligned"> <i>2.0</i> 
                           </td>
                           
                           <td class="center-aligned"> <i>1.94</i> 
                           </td>
                           
                           <td class="center-aligned"> <b>1.9</b> 
                           </td>
                           
                           <td class="center-aligned"> <i>1.83</i> 
                           </td>
                           
                           <td class="center-aligned"> <i>1.7</i> 
                           </td>
                           
                           <td class="center-aligned"> <b>1.0</b> 
                           </td>
                           
                        </tr>
                        
                        <tr>
                           
                           <td class="left-aligned">Category indicators, a single categorical variable, “low”: β<sub>C</sub></td>
                           
                           <td class="center-aligned">0.2</td>
                           
                           <td class="center-aligned"> <b>0.2</b> 
                           </td>
                           
                           <td class="center-aligned"> <b>0.2</b> 
                           </td>
                           
                           <td class="center-aligned"> <b>0.1</b> 
                           </td>
                           
                           <td class="center-aligned"> <b>0.05</b> 
                           </td>
                           
                           <td class="center-aligned"> <i>0.05</i> 
                           </td>
                           
                        </tr>
                        
                        <tr>
                           
                           <td class="left-aligned">Category indicators, “intermediate”: β<sub>C</sub></td>
                           
                           <td class="center-aligned">0.65</td>
                           
                           <td class="center-aligned">0.53</td>
                           
                           <td class="center-aligned">0.45</td>
                           
                           <td class="center-aligned">0.25</td>
                           
                           <td class="center-aligned">0.15</td>
                           
                           <td class="center-aligned">0.05</td>
                           
                        </tr>
                        
                        <tr>
                           
                           <td class="left-aligned">Hinge features: β<sub>H</sub></td>
                           
                           <td class="center-aligned"> <b>0.5</b> 
                           </td>
                           
                           <td class="center-aligned"> <b>0.5</b> 
                           </td>
                           
                           <td class="center-aligned"> <b>0.5</b> 
                           </td>
                           
                           <td class="center-aligned"> <b>0.5</b> 
                           </td>
                           
                           <td class="center-aligned"> <b>0.5</b> 
                           </td>
                           
                           <td class="center-aligned"> <b>0.5</b> 
                           </td>
                           
                        </tr>
                        
                     </tbody>
                     
                  </table>
               </div>
               
               
               <div class="article-section__table-source"></div>
            </div>
            
            <section class="article-section__inline-figure">
               <figure class="figure" id="f2"><a target="_blank" href="https://onlinelibrary.wiley.com/cms/asset/bbd3bb46-671a-4e16-b129-588b548fbcb1/ecog_5203_f2.gif"><img class="figure__image" src="ecog_5203_f2.gif" alt="image" title="image"></a><figcaption class="figure__caption">
                     <div class="figure__caption__header"><strong class="figure-title">Figure 2</strong> 
                        <div class="figure-extra"><a href="#" class="open-figure-link">Open in figure viewer</a><a href="https://onlinelibrary.wiley.com/action/downloadFigures?id=f2&amp;doi=10.1111%2Fj.0906-7590.2008.5203.x" class="ppt-figure-link">PowerPoint</a></div>
                     </div>
                     <div>
                        <p>A subset of the β‐curves for LQ runs of 
Maxent. Average performance of Maxent is evaluated in terms of log loss 
and AUC over five random partitions as a function of the regularization 
parameter β (where β<sub>L</sub>=β<sub>Q</sub><i>=</i>β<i>)</i> for varying number of occurrences m<i>.</i> Default values of β<sub>L</sub>, β<sub>Q</sub> for Maxent ver 1.8.3 through at least 3.1.0 (listed in <a href="#t3" class="scrollableLink">Table 3</a>) were chosen to obtain satisfactory performance on the evaluated subset of species. For explanations of species codes, see <a href="#t2" class="scrollableLink">Table 2</a>.</p>
                        
                     </div>
                  </figcaption>
               </figure>
            </section>
            
            <p>When using both continuous and categorical variables, the intermediate setting for β<sub>C</sub> (<a href="#t3" class="scrollableLink">Table 3</a>)
 gave the best performance in more than half of the m‐curves and never 
gave the worst performance. Discrete ordinal variables performed best 
when viewed as continuous (details not shown).
            </p>
            
            <p>Finally, we determined optimal combinations of feature classes. <a href="#f3" class="scrollableLink">Fig. 3</a>
 shows the performance of four feature class settings. From the figures,
 we determined ranges of individual feature classes as follows: LC 
features for 2–9 samples, LQC features for 10–79 samples, LQPTC features
 for 80 and more samples.
            </p>
            
            <section class="article-section__inline-figure">
               <figure class="figure" id="f3"><a target="_blank" href="https://onlinelibrary.wiley.com/cms/asset/7e1c3abd-7d64-48eb-88eb-f31814bd2b8f/ecog_5203_f3.gif"><img class="figure__image" src="ecog_5203_f3.gif" alt="image" title="image"></a><figcaption class="figure__caption">
                     <div class="figure__caption__header"><strong class="figure-title">Figure 3</strong> 
                        <div class="figure-extra"><a href="#" class="open-figure-link">Open in figure viewer</a><a href="https://onlinelibrary.wiley.com/action/downloadFigures?id=f3&amp;doi=10.1111%2Fj.0906-7590.2008.5203.x" class="ppt-figure-link">PowerPoint</a></div>
                     </div>
                     <div>
                        <p>Curves showing average performance of Maxent 
as a function of sample size (m‐curves) averaged over all species 
selected for tuning. By visual inspection, we determined the ranges of 
sample sizes in which to use the different sets of feature classes as: 
LC features for 2–9 samples, LQC features for 10–79 samples and LQPTC 
features for 80 and more samples. AUC values were calculated using 
background data in place of true absences, as the data used to generate 
these curves is presence‐only. Each region is shown separately; for 
region names, see <a href="#t1" class="scrollableLink">Table 1</a>.</p>
                        
                     </div>
                  </figcaption>
               </figure>
            </section>
            
            <section class="article-section__sub-content" id="ss22">
               
               <h4 class="article-section__sub-title section3" id="ss22-title"> Evaluation of presence‐only tuning using presence‐absence data</h4>
               
               <p>In <a href="#t5" class="scrollableLink">Table 5</a>, 
we compare performance of po‐tuned Maxent parameters and pa‐tuned 
parameters. Global pa‐tuning results in an improvement of average AUC by
 0.006 (both for random background and target‐group background) whereas 
regional pa‐tuning improves the AUC for random background by 0.014 and 
the AUC for target‐group background by 0.012.
               </p>
               
               <div class="article-table-content" id="t5">
                  <header class="article-table-caption"><span class="table-caption__label">Table 5.
            </span>Maxent performance in terms of AUC under four 
different training‐evaluation scenarios: presence‐only training on 
random background and target‐group background, and evaluation with 
respect to globally or regionally presence‐absence optimized parameters.
 AUC values were determined using the presence‐absence data.
                  </header>
                  
                  
                  
                  
                  <div class="article-table-content-wrapper">
                     <table class="table article-section__table">
                        
                        
                        
                        
                        
                        
                        
                        
                        <thead>
                           
                           <tr>
                              
                              <th class="bottom-bordered-cell left-aligned"></th>
                              
                              <th colspan="3" class="bottom-bordered-cell center-aligned">Random background</th>
                              
                              <th colspan="3" class="bottom-bordered-cell center-aligned">Target‐group background</th>
                              
                           </tr>
                           
                           <tr>
                              
                              <th class="bottom-bordered-cell left-aligned"></th>
                              
                              <th class="bottom-bordered-cell center-aligned"></th>
                              
                              <th colspan="2" class="bottom-bordered-cell center-aligned">improvement from default</th>
                              
                              <th class="bottom-bordered-cell center-aligned"></th>
                              
                              <th colspan="2" class="bottom-bordered-cell center-aligned">improvement from default</th>
                              
                           </tr>
                           
                           <tr>
                              
                              <th class="bottom-bordered-cell left-aligned"></th>
                              
                              <th class="bottom-bordered-cell center-aligned">default settings</th>
                              
                              <th class="bottom-bordered-cell center-aligned">globally optimized settings</th>
                              
                              <th class="bottom-bordered-cell center-aligned">regionally optimized settings</th>
                              
                              <th class="bottom-bordered-cell center-aligned">default settings</th>
                              
                              <th class="bottom-bordered-cell center-aligned">globally Optimized settings</th>
                              
                              <th class="bottom-bordered-cell center-aligned">regionally Optimized settings</th>
                              
                           </tr>
                           
                        </thead>
                        
                        <tbody>
                           
                           <tr>
                              
                              <td class="left-aligned">awt</td>
                              
                              <td class="center-aligned">0.693</td>
                              
                              <td class="center-aligned">0.004</td>
                              
                              <td class="center-aligned">0.015</td>
                              
                              <td class="center-aligned">0.729</td>
                              
                              <td class="center-aligned">0.000</td>
                              
                              <td class="center-aligned">0.009</td>
                              
                           </tr>
                           
                           <tr>
                              
                              <td class="left-aligned">can</td>
                              
                              <td class="center-aligned">0.594</td>
                              
                              <td class="center-aligned">0.008</td>
                              
                              <td class="center-aligned">0.023</td>
                              
                              <td class="center-aligned">0.719</td>
                              
                              <td class="center-aligned">0.011</td>
                              
                              <td class="center-aligned">0.020</td>
                              
                           </tr>
                           
                           <tr>
                              
                              <td class="left-aligned">nsw</td>
                              
                              <td class="center-aligned">0.711</td>
                              
                              <td class="center-aligned">0.005</td>
                              
                              <td class="center-aligned">0.022</td>
                              
                              <td class="center-aligned">0.742</td>
                              
                              <td class="center-aligned">0.009</td>
                              
                              <td class="center-aligned">0.020</td>
                              
                           </tr>
                           
                           <tr>
                              
                              <td class="left-aligned">nz</td>
                              
                              <td class="center-aligned">0.733</td>
                              
                              <td class="center-aligned">0.008</td>
                              
                              <td class="center-aligned">0.009</td>
                              
                              <td class="center-aligned">0.741</td>
                              
                              <td class="center-aligned">0.009</td>
                              
                              <td class="center-aligned">0.011</td>
                              
                           </tr>
                           
                           <tr>
                              
                              <td class="left-aligned">sa</td>
                              
                              <td class="center-aligned">0.796</td>
                              
                              <td class="center-aligned">0.007</td>
                              
                              <td class="center-aligned">0.014</td>
                              
                              <td class="center-aligned">0.793</td>
                              
                              <td class="center-aligned">0.003</td>
                              
                              <td class="center-aligned">0.005</td>
                              
                           </tr>
                           
                           <tr>
                              
                              <td class="left-aligned">swi</td>
                              
                              <td class="center-aligned">0.803</td>
                              
                              <td class="center-aligned">0.003</td>
                              
                              <td class="center-aligned">0.001</td>
                              
                              <td class="center-aligned">0.837</td>
                              
                              <td class="center-aligned">0.006</td>
                              
                              <td class="center-aligned">0.006</td>
                              
                           </tr>
                           
                           <tr>
                              
                              <td class="left-aligned">all species</td>
                              
                              <td class="center-aligned">0.726</td>
                              
                              <td class="center-aligned">0.006</td>
                              
                              <td class="center-aligned">0.014</td>
                              
                              <td class="center-aligned">0.757</td>
                              
                              <td class="center-aligned">0.006</td>
                              
                              <td class="center-aligned">0.012</td>
                              
                           </tr>
                           
                        </tbody>
                        
                     </table>
                  </div>
                  
                  
                  <div class="article-section__table-source"></div>
               </div>
               
               <p>In <a href="#t6" class="scrollableLink">Table 6</a>, 
we report parameters obtained by global pa‐tuning and medians of 
parameters obtained by regional pa‐tuning. For regions awt and nsw, the 
regional tuning was performed separately for each taxonomic group (birds
 and plants in awt, and small mammals, reptiles, birds and plants in 
nsw), resulting in a total of 10 regionally optimized parameter sets. 
Each median is thus taken over a set of 10 values.
               </p>
               
               <div class="article-table-content" id="t6">
                  <header class="article-table-caption"><span class="table-caption__label">Table 6.
            </span>Overview of pa‐tuned parameters: globally optimized 
parameters and medians of 10 regionally optimized parameters. The global
 settings optimize the average performance across all species. The 
regional settings are optimized separately for each of 10 taxonomic 
groups in the 6 regions.
                  </header>
                  
                  
                  
                  
                  <div class="article-table-content-wrapper">
                     <table class="table article-section__table">
                        
                        
                        
                        
                        
                        
                        
                        
                        
                        
                        
                        <thead>
                           
                           <tr>
                              
                              <th class="bottom-bordered-cell left-aligned"></th>
                              
                              <th class="bottom-bordered-cell left-aligned"></th>
                              
                              <th colspan="4" class="bottom-bordered-cell center-aligned">Random background</th>
                              
                              <th colspan="4" class="bottom-bordered-cell center-aligned">Target‐group background</th>
                              
                           </tr>
                           
                           <tr>
                              
                              <th class="bottom-bordered-cell left-aligned"></th>
                              
                              <th class="bottom-bordered-cell left-aligned"></th>
                              
                              <th colspan="4" class="bottom-bordered-cell center-aligned">number of occurrences</th>
                              
                              <th colspan="4" class="bottom-bordered-cell center-aligned">number of occurrences</th>
                              
                           </tr>
                           
                           <tr>
                              
                              <th class="bottom-bordered-cell left-aligned"></th>
                              
                              <th class="bottom-bordered-cell left-aligned"></th>
                              
                              <th class="bottom-bordered-cell center-aligned">2–9</th>
                              
                              <th class="bottom-bordered-cell center-aligned">10–14</th>
                              
                              <th class="bottom-bordered-cell center-aligned">15–79</th>
                              
                              <th class="bottom-bordered-cell center-aligned">≥80</th>
                              
                              <th class="bottom-bordered-cell center-aligned">2–9</th>
                              
                              <th class="bottom-bordered-cell center-aligned">10–14</th>
                              
                              <th class="bottom-bordered-cell center-aligned">15–79</th>
                              
                              <th class="bottom-bordered-cell center-aligned">≥80</th>
                              
                           </tr>
                           
                        </thead>
                        
                        <tbody>
                           
                           <tr>
                              
                              <td class="left-aligned">β<sub>L</sub>:
                              </td>
                              
                              <td class="left-aligned">global optimum</td>
                              
                              <td class="center-aligned">1.00</td>
                              
                              <td class="center-aligned">1.41</td>
                              
                              <td class="center-aligned"></td>
                              
                              <td class="center-aligned"></td>
                              
                              <td class="center-aligned">2.00</td>
                              
                              <td class="center-aligned">1.41</td>
                              
                              <td class="center-aligned"></td>
                              
                              <td class="center-aligned"></td>
                              
                           </tr>
                           
                           <tr>
                              
                              <td class="left-aligned"></td>
                              
                              <td class="left-aligned">regional median</td>
                              
                              <td class="center-aligned">1.00</td>
                              
                              <td class="center-aligned">1.00</td>
                              
                              <td class="center-aligned"></td>
                              
                              <td class="center-aligned"></td>
                              
                              <td class="center-aligned">1.00</td>
                              
                              <td class="center-aligned">1.00</td>
                              
                              <td class="center-aligned"></td>
                              
                              <td class="center-aligned"></td>
                              
                           </tr>
                           
                           <tr>
                              
                              <td class="left-aligned"></td>
                              
                              <td class="left-aligned">default*</td>
                              
                              <td class="center-aligned">1.00</td>
                              
                              <td class="center-aligned">0.71</td>
                              
                              <td class="center-aligned"></td>
                              
                              <td class="center-aligned"></td>
                              
                              <td class="center-aligned">1.00</td>
                              
                              <td class="center-aligned">0.71</td>
                              
                              <td class="center-aligned"></td>
                              
                              <td class="center-aligned"></td>
                              
                           </tr>
                           
                           <tr>
                              
                              <td class="left-aligned">β<sub>Q</sub>:
                              </td>
                              
                              <td class="left-aligned">global optimum</td>
                              
                              <td class="center-aligned"></td>
                              
                              <td class="center-aligned">1.41</td>
                              
                              <td class="center-aligned">0.50</td>
                              
                              <td class="center-aligned">0.35†</td>
                              
                              <td class="center-aligned"></td>
                              
                              <td class="center-aligned">2.00</td>
                              
                              <td class="center-aligned">1.00</td>
                              
                              <td class="center-aligned">0.35†</td>
                              
                           </tr>
                           
                           <tr>
                              
                              <td class="left-aligned"></td>
                              
                              <td class="left-aligned">regional median</td>
                              
                              <td class="center-aligned"></td>
                              
                              <td class="center-aligned">1.00</td>
                              
                              <td class="center-aligned">0.85</td>
                              
                              <td class="center-aligned">0.05</td>
                              
                              <td class="center-aligned"></td>
                              
                              <td class="center-aligned">1.00</td>
                              
                              <td class="center-aligned">0.71</td>
                              
                              <td class="center-aligned">0.07</td>
                              
                           </tr>
                           
                           <tr>
                              
                              <td class="left-aligned"></td>
                              
                              <td class="left-aligned">default*</td>
                              
                              <td class="center-aligned"></td>
                              
                              <td class="center-aligned">0.71</td>
                              
                              <td class="center-aligned">0.23</td>
                              
                              <td class="center-aligned">0.05</td>
                              
                              <td class="center-aligned"></td>
                              
                              <td class="center-aligned">0.71</td>
                              
                              <td class="center-aligned">0.23</td>
                              
                              <td class="center-aligned">0.05</td>
                              
                           </tr>
                           
                           <tr>
                              
                              <td class="left-aligned">β<sub>P</sub>:
                              </td>
                              
                              <td class="left-aligned">global optimum</td>
                              
                              <td class="center-aligned"></td>
                              
                              <td class="center-aligned"></td>
                              
                              <td class="center-aligned"></td>
                              
                              <td class="center-aligned">0.35†</td>
                              
                              <td class="center-aligned"></td>
                              
                              <td class="center-aligned"></td>
                              
                              <td class="center-aligned"></td>
                              
                              <td class="center-aligned">0.35†</td>
                              
                           </tr>
                           
                           <tr>
                              
                              <td class="left-aligned"></td>
                              
                              <td class="left-aligned">regional median</td>
                              
                              <td class="center-aligned"></td>
                              
                              <td class="center-aligned"></td>
                              
                              <td class="center-aligned"></td>
                              
                              <td class="center-aligned">0.04</td>
                              
                              <td class="center-aligned"></td>
                              
                              <td class="center-aligned"></td>
                              
                              <td class="center-aligned"></td>
                              
                              <td class="center-aligned">0.11</td>
                              
                           </tr>
                           
                           <tr>
                              
                              <td class="left-aligned"></td>
                              
                              <td class="left-aligned">default*</td>
                              
                              <td class="center-aligned"></td>
                              
                              <td class="center-aligned"></td>
                              
                              <td class="center-aligned"></td>
                              
                              <td class="center-aligned">0.05</td>
                              
                              <td class="center-aligned"></td>
                              
                              <td class="center-aligned"></td>
                              
                              <td class="center-aligned"></td>
                              
                              <td class="center-aligned">0.05</td>
                              
                           </tr>
                           
                           <tr>
                              
                              <td class="left-aligned">β<sub>T</sub>:
                              </td>
                              
                              <td class="left-aligned">global optimum</td>
                              
                              <td class="center-aligned"></td>
                              
                              <td class="center-aligned"></td>
                              
                              <td class="center-aligned"></td>
                              
                              <td class="center-aligned">2.00</td>
                              
                              <td class="center-aligned"></td>
                              
                              <td class="center-aligned"></td>
                              
                              <td class="center-aligned"></td>
                              
                              <td class="center-aligned">8.00†</td>
                              
                           </tr>
                           
                           <tr>
                              
                              <td class="left-aligned"></td>
                              
                              <td class="left-aligned">regional median</td>
                              
                              <td class="center-aligned"></td>
                              
                              <td class="center-aligned"></td>
                              
                              <td class="center-aligned"></td>
                              
                              <td class="center-aligned">1.21</td>
                              
                              <td class="center-aligned"></td>
                              
                              <td class="center-aligned"></td>
                              
                              <td class="center-aligned"></td>
                              
                              <td class="center-aligned">1.71</td>
                              
                           </tr>
                           
                           <tr>
                              
                              <td class="left-aligned"></td>
                              
                              <td class="left-aligned">default*</td>
                              
                              <td class="center-aligned"></td>
                              
                              <td class="center-aligned"></td>
                              
                              <td class="center-aligned"></td>
                              
                              <td class="center-aligned">1.00</td>
                              
                              <td class="center-aligned"></td>
                              
                              <td class="center-aligned"></td>
                              
                              <td class="center-aligned"></td>
                              
                              <td class="center-aligned">1.00</td>
                              
                           </tr>
                           
                           <tr>
                              
                              <td class="left-aligned">β<sub>H</sub>:
                              </td>
                              
                              <td class="left-aligned">global optimum</td>
                              
                              <td class="center-aligned"></td>
                              
                              <td class="center-aligned"></td>
                              
                              <td class="center-aligned">0.35</td>
                              
                              <td class="center-aligned">0.50</td>
                              
                              <td class="center-aligned"></td>
                              
                              <td class="center-aligned"></td>
                              
                              <td class="center-aligned">0.71</td>
                              
                              <td class="center-aligned">1.41</td>
                              
                           </tr>
                           
                           <tr>
                              
                              <td class="left-aligned"></td>
                              
                              <td class="left-aligned">regional median</td>
                              
                              <td class="center-aligned"></td>
                              
                              <td class="center-aligned"></td>
                              
                              <td class="center-aligned">0.85</td>
                              
                              <td class="center-aligned">0.50</td>
                              
                              <td class="center-aligned"></td>
                              
                              <td class="center-aligned"></td>
                              
                              <td class="center-aligned">1.21</td>
                              
                              <td class="center-aligned">0.50</td>
                              
                           </tr>
                           
                           <tr>
                              
                              <td class="left-aligned"></td>
                              
                              <td class="left-aligned">default*</td>
                              
                              <td class="center-aligned"></td>
                              
                              <td class="center-aligned"></td>
                              
                              <td class="center-aligned">0.50</td>
                              
                              <td class="center-aligned">0.50</td>
                              
                              <td class="center-aligned"></td>
                              
                              <td class="center-aligned"></td>
                              
                              <td class="center-aligned">0.50</td>
                              
                              <td class="center-aligned">0.50</td>
                              
                           </tr>
                           
                           <tr>
                              
                              <td class="left-aligned">β<sub>C</sub>:
                              </td>
                              
                              <td class="left-aligned">global optimum</td>
                              
                              <td class="center-aligned">1.41</td>
                              
                              <td class="center-aligned">0.50</td>
                              
                              <td class="center-aligned">0.03**</td>
                              
                              <td class="center-aligned">0.03</td>
                              
                              <td class="center-aligned">0.50</td>
                              
                              <td class="center-aligned">1.00</td>
                              
                              <td class="center-aligned">0.13</td>
                              
                              <td class="center-aligned">0.25</td>
                              
                           </tr>
                           
                           <tr>
                              
                              <td class="left-aligned"></td>
                              
                              <td class="left-aligned">regional median</td>
                              
                              <td class="center-aligned">0.71</td>
                              
                              <td class="center-aligned">0.50</td>
                              
                              <td class="center-aligned">0.18</td>
                              
                              <td class="center-aligned">0.04</td>
                              
                              <td class="center-aligned">0.71</td>
                              
                              <td class="center-aligned">0.50</td>
                              
                              <td class="center-aligned">1.00</td>
                              
                              <td class="center-aligned">0.04</td>
                              
                           </tr>
                           
                           <tr>
                              
                              <td class="left-aligned"></td>
                              
                              <td class="left-aligned">default*</td>
                              
                              <td class="center-aligned">0.53</td>
                              
                              <td class="center-aligned">0.39</td>
                              
                              <td class="center-aligned">0.14</td>
                              
                              <td class="center-aligned">0.05</td>
                              
                              <td class="center-aligned">0.53</td>
                              
                              <td class="center-aligned">0.39</td>
                              
                              <td class="center-aligned">0.14</td>
                              
                              <td class="center-aligned">0.05</td>
                              
                           </tr>
                           
                        </tbody>
                        
                     </table>
                  </div>
                  
                  
                  
                  <div class="article-section__table-footnotes">
                     <ul>
                        <li id="t6_note198">
                           * po‐tuned values are listed for the median 
training set size appearing in each range: 6, 12, 36, and 221.
                           
                        </li>
                        <li id="t6_note199">
                           † the largest possible value in local search.
                           
                        </li>
                        <li id="t6_note200">
                           ** the smallest possible value in local search.
                           
                        </li>
                     </ul>
                  </div>
                  <div class="article-section__table-source"></div>
               </div>
               
               <p>To compare pa‐tuned regularization parameters with 
po‐tuned regularization parameters, we determined the median training 
set size in each range and report the corresponding po‐tuned values. 
Note that the pa‐tuned values are almost always larger than the po‐tuned
 values. Larger regularization represents our increased uncertainty in 
feature‐expectation estimates as a result of differences between 
training and test distributions.</p>
               
            </section>
            
         </section>
         
         <section class="article-section__sub-content" id="ss23">
            
            <h3 class="article-section__sub-title section2" id="ss23-title"> New extensions to Maxent</h3>
            
            <section class="article-section__sub-content" id="ss24">
               
               <h4 class="article-section__sub-title section3" id="ss24-title"> Hinge features</h4>
               
               <p>Tuning on the presence‐only data indicated that hinge 
features should be used when the number of presence records is at least 
15, with a regularization parameter of 0.5. Using these parameters, a 
full run on the 226 species with random background yielded an average 
AUC of 0.726, which was significantly better than the average of 0.721 
obtained without hinge features (p&lt;0.001, two‐tailed Wilcoxon 
signed‐rank test, pairing by species). Similarly significant improvement
 was observed for the COR statistic (from 0.198 to 0.210, p&lt;0.001).</p>
               
            </section>
            
            <section class="article-section__sub-content" id="ss25">
               
               <h4 class="article-section__sub-title section3" id="ss25-title"> Logistic output</h4>
               
               <p>When compared using the COR statistic, the logistic 
output format achieved the best performance, followed by cumulative 
output format and then raw output (<a href="#t4" class="scrollableLink">Table 4</a>),
 and the ordering was consistent when using either random background or 
target‐group background. The performance of logistic output was 
significantly better than raw output when using random background, and 
significantly better than cumulative output when using target‐group 
background (p&lt;0.01, two‐tailed Wilcoxon signed‐rank test, pairing by 
species).
               </p>
               
               <div class="article-table-content" id="t4">
                  <header class="article-table-caption"><span class="table-caption__label">Table 4.
            </span>Average values of the COR statistic for Maxent run 
with either random or target‐group background, and with output given in 
either raw, cumulative or logistic format.
                  </header>
                  
                  
                  
                  
                  <div class="article-table-content-wrapper">
                     <table class="table article-section__table">
                        
                        
                        
                        
                        
                        <thead>
                           
                           <tr>
                              
                              <th class="bottom-bordered-cell left-aligned">Background type</th>
                              
                              <th class="bottom-bordered-cell center-aligned">Raw</th>
                              
                              <th class="bottom-bordered-cell center-aligned">Cumulative</th>
                              
                              <th class="bottom-bordered-cell center-aligned">Logistic</th>
                              
                           </tr>
                           
                        </thead>
                        
                        <tbody>
                           
                           <tr>
                              
                              <td class="left-aligned">Random</td>
                              
                              <td class="center-aligned">0.184</td>
                              
                              <td class="center-aligned">0.206</td>
                              
                              <td class="center-aligned">0.210</td>
                              
                           </tr>
                           
                           <tr>
                              
                              <td class="left-aligned">Target‐group</td>
                              
                              <td class="center-aligned">0.238</td>
                              
                              <td class="center-aligned">0.239</td>
                              
                              <td class="center-aligned">0.245</td>
                              
                           </tr>
                           
                        </tbody>
                        
                     </table>
                  </div>
                  
                  
                  <div class="article-section__table-source"></div>
               </div>
               
            </section>
            
            <section class="article-section__sub-content" id="ss26">
               
               <h4 class="article-section__sub-title section3" id="ss26-title"> Background treatments</h4>
               
               <p>Using target‐group background instead of random 
background raised the average AUC from 0.726 to 0.757, as measured using
 independent presence‐absence test data. This improvement is highly 
significant (p&lt;10<sup>−6</sup>, one‐tailed sign test, pairing by species). The results of the background sampling experiment (<a href="#f4" class="scrollableLink">Fig. 4</a>)
 indicate that predictive performance increases substantially as the 
number of background sites increases, reaching a plateau after 8000 
sites.
               </p>
               
               <section class="article-section__inline-figure">
                  <figure class="figure" id="f4"><a target="_blank" href="https://onlinelibrary.wiley.com/cms/asset/8f686231-f3a6-43d2-b2af-8019a63fb60c/ecog_5203_f4.gif"><img class="figure__image" src="ecog_5203_f4.gif" alt="image" title="image"></a><figcaption class="figure__caption">
                        <div class="figure__caption__header"><strong class="figure-title">Figure 4</strong> 
                           <div class="figure-extra"><a href="#" class="open-figure-link">Open in figure viewer</a><a href="https://onlinelibrary.wiley.com/action/downloadFigures?id=f4&amp;doi=10.1111%2Fj.0906-7590.2008.5203.x" class="ppt-figure-link">PowerPoint</a></div>
                        </div>
                        <div>
                           <p>Average performance of Maxent for varying 
background sample sizes. For each sample size, 10 random background 
samples were taken, and for each such sample, models were generated for 
all 226 species. Values shown are average AUC values across all random 
samples and species. The AUC values were calculated using independent 
presence‐absence data.</p>
                           
                        </div>
                     </figcaption>
                  </figure>
               </section>
               
            </section>
            
         </section>
         
      </section>
      <section class="article-section__content" id="ss27">
         
         <h2 class="article-section__title section__title section1" id="ss27-title"> Discussion and conclusions</h2>
         
         <section class="article-section__sub-content" id="ss28">
            
            <h3 class="article-section__sub-title section2" id="ss28-title"> Maxent tuning</h3>
            
            <p>A major focus of this work is the tuning of 
regularization parameters and choice of feature classes, both viewed as 
functions of the number of presence samples available during training. 
Tuning was performed on presence‐only data (po‐tuned parameters) and 
then compared to parameters optimized for presence‐absence evaluation 
data (pa‐tuned parameters). The po‐tuning procedure was robust: 
five‐fold cross‐validation reduced the variance of parameter estimates, 
as did the fact that we took averages or medians over several species to
 determine each parameter. In addition, the po‐tuned values were 
evaluated on many more species in the presence‐absence evaluation data, 
and we found that the po‐tuned parameters resulted in model performance 
which is almost as good as if the parameters had been tuned on the 
evaluation data itself. The datasets used here cover a wide range of 
species, taxonomic groups, numbers of occurrence records and species 
prevalence (fraction of the study area occupied by the species). We 
conclude therefore that the tuning methodology based on presence‐only 
data was very effective, and the resulting regularization parameters, 
which are the defaults in Maxent software ver. 1.8.3 through at least 
3.1.0 (with one exception, described below), are well suited for a wide 
range of presence‐only datasets. It is tempting to use the pa‐tuned 
settings as default settings in the software, since they give marginally
 better performance on the evaluation data. However, we believe that 
doing so may result in overfitting to this particular evaluation 
dataset, since the pa‐tuned settings are being evaluated here on the 
same data on which they were tuned. We believe it is preferable to 
continue using the po‐tuned settings, which have been validated on 
independent test data in this study.</p>
            
            <p>We emphasize, though, that datasets that deviate 
significantly from those used in this study may require further 
parameter tuning (for example, using the tuning approach demonstrated 
here) to optimize Maxent model performance. The six regional datasets we
 used all contain a similar number of environmental variables (11–13). 
The theory in “Maxent modeling of distributions”, above suggests that 
more regularization may be needed if the number of environmental 
variables is much larger. Similarly, &gt;10 000 background sites may be 
needed if the number of presence sites is much greater than for the 
species studied here. In addition, the datasets used here made little 
use of categorical data, and since the time when we performed this 
study, some other datasets with more categorical data have shown strong 
signs of overfitting with the regularization as tuned here. For this 
reason, software ver. 2.3.35 and later use higher categorical 
regularization, with the second last line of <a href="#t3" class="scrollableLink">Table 3</a>
 being increased to {0.65, 0.56, 0.5, 0.25, 0.25, 0.25}. This change has
 minimal effect on Maxent's statistical performance on the data we have 
used here.
            </p>
            
         </section>
         
         <section class="article-section__sub-content" id="ss29">
            
            <h3 class="article-section__sub-title section2" id="ss29-title"> Possible reasons for Maxent's good performance</h3>
            
            <p>The parameter settings developed in this study were used 
(without hinge features or logistic output, which came later) in the 
modeling comparison of <span><a href="#b8" id="#b8R" class="bibLink tab-link" data-tab="pane-pcw-references">Elith et al. (2006)</a></span>,
 and we feel that our careful parameter tuning, especially for small 
sample sizes, contributed to Maxent's good showing in that comparison. 
There are a number of other factors that may help explain Maxent's good 
performance.
            </p>
            
            <p>First, Maxent uses 𝓁<sub>1</sub> regularization, which tends to produce models with few non‐zero coefficients (<span><a href="#b36" id="#b36R" class="bibLink tab-link" data-tab="pane-pcw-references">Williams 1995</a></span>, <span><a href="#b33" id="#b33R" class="bibLink tab-link" data-tab="pane-pcw-references">Tibshirani 1996</a></span>)
 and therefore encourages parsimony. Regularization appears to prevent 
overfitting better than variable‐selection methods commonly used for 
regression‐based models such as generalized additive and generalized 
linear models. These models can also use regularization –𝓁<sub>1</sub> 
regularization applied to such methods is known as the lasso – but the 
regularized variants have not been used for species distribution 
modeling to date.
            </p>
            
            <p>Perhaps more importantly, when used on presence‐only 
data, regression‐based methods suffer from the problem of “contaminated 
controls” (<span><a href="#b19" id="#b19R" class="bibLink tab-link" data-tab="pane-pcw-references">Keating and Cherry 2004</a></span>),
 in which background data is treated as absence data, even though it is 
contaminated with presences. Recent work addresses this issue by using 
the EM algorithm to essentially decontaminate the background (<span><a href="#b35" id="#b35R" class="bibLink tab-link" data-tab="pane-pcw-references">Ward et al. 2007</a></span>),
 but this approach has not been extensively compared to other 
presence‐only modeling methods because it requires knowledge of the 
species prevalence in the study region, which cannot be inferred from 
presence‐only data. A final possible explanation for Maxent's good 
performance is that it is a generative approach, modeling the species 
distribution P(x∣y=1) directly, whereas regression‐based methods are 
discriminative, modeling P(y=1∣x). Maxent's good performance on small 
samples is in line with previous studies indicating that generative 
methods give better predictions than discriminative methods when the 
amount of training data is small (<span><a href="#b20" id="#b20R" class="bibLink tab-link" data-tab="pane-pcw-references">Ng and Jordan 2001</a></span>).
            </p>
            
            <p>Lastly, we note that boosted regression trees (<span><a href="#b11" id="#b11R" class="bibLink tab-link" data-tab="pane-pcw-references">Friedman 2001</a></span>) slightly out‐performed Maxent in the comparison of <span><a href="#b8" id="#b8R" class="bibLink tab-link" data-tab="pane-pcw-references">Elith et al. (2006)</a></span>.
 Both methods have been further developed since the comparison: for BRT,
 there is improved tuning for presence‐only data (J. Elith pers. comm.),
 and for Maxent, we introduced hinge features and logistic output 
format. The current versions of BRT and Maxent achieve average AUC and 
COR scores that are extremely close; however, Maxent achieves somewhat 
better average deviance scores (<a href="#t7" class="scrollableLink">Table 7</a>).
 This discrepancy is probably because Maxent's logistic format is robust
 to the unknown prevalence (“Maxent output formats and logistic models”,
 above), while BRT (as applied to presence‐only data) gives equal weight
 to presences and absences (<span><a href="#b8" id="#b8R" class="bibLink tab-link" data-tab="pane-pcw-references">Elith et al. 2006</a></span>), which seems to inflate estimates of probability of presence.
            </p>
            
            <div class="article-table-content" id="t7">
               <header class="article-table-caption"><span class="table-caption__label">Table 7.
            </span>Average performance for current versions and settings
 of Maxent and boosted regression trees (BRT), evaluated on independent 
presence/absence test data using area under the receiver operating 
characteristic curve (AUC), correlation with 1/0 values for 
presence/absence (COR), and average deviance per test site. Each value 
shown is the average over 226 species.
               </header>
               
               
               
               
               <div class="article-table-content-wrapper">
                  <table class="table article-section__table">
                     
                     
                     
                     
                     
                     
                     <thead>
                        
                        <tr>
                           
                           <th class="bottom-bordered-cell left-aligned">Algorithm</th>
                           
                           <th class="bottom-bordered-cell center-aligned">Background type</th>
                           
                           <th class="bottom-bordered-cell center-aligned">AUC</th>
                           
                           <th class="bottom-bordered-cell center-aligned">COR</th>
                           
                           <th class="bottom-bordered-cell center-aligned">Average deviance</th>
                           
                        </tr>
                        
                     </thead>
                     
                     <tbody>
                        
                        <tr>
                           
                           <td class="left-aligned">Maxent</td>
                           
                           <td class="left-aligned">Random</td>
                           
                           <td class="center-aligned">0.7276</td>
                           
                           <td class="center-aligned">0.2100</td>
                           
                           <td class="center-aligned">0.9929</td>
                           
                        </tr>
                        
                        <tr>
                           
                           <td class="left-aligned">Maxent</td>
                           
                           <td class="left-aligned">Target‐group</td>
                           
                           <td class="center-aligned">0.7569</td>
                           
                           <td class="center-aligned">0.2446</td>
                           
                           <td class="center-aligned">0.8585</td>
                           
                        </tr>
                        
                        <tr>
                           
                           <td class="left-aligned">BRT</td>
                           
                           <td class="left-aligned">Random</td>
                           
                           <td class="center-aligned">0.7275</td>
                           
                           <td class="center-aligned">0.2130</td>
                           
                           <td class="center-aligned">1.1249</td>
                           
                        </tr>
                        
                        <tr>
                           
                           <td class="left-aligned">BRT</td>
                           
                           <td class="left-aligned">Target‐group</td>
                           
                           <td class="center-aligned">0.7544</td>
                           
                           <td class="center-aligned">0.2435</td>
                           
                           <td class="center-aligned">0.9635</td>
                           
                        </tr>
                        
                     </tbody>
                     
                  </table>
               </div>
               
               
               <div class="article-section__table-source"></div>
            </div>
            
         </section>
         
         <section class="article-section__sub-content" id="ss30">
            
            <h3 class="article-section__sub-title section2" id="ss30-title"> Maxent extensions</h3>
            
            <p>We have shown that target‐group background can 
significantly improve model performance. The magnitude of the increase 
in AUC that results from using target‐group background is similar to the
 differences in AUC between modeling methods in <span><a href="#b8" id="#b8R" class="bibLink tab-link" data-tab="pane-pcw-references">Elith et al. (2006)</a></span>.
 The performance improvement was greatest in Ontario, Canada, the region
 with the most glaring sample selection bias in the presence‐only 
training data. Target‐group background therefore appears effective at 
countering sample selection bias, matching the theoretical prediction (<span><a href="#b5" id="#b5R" class="bibLink tab-link" data-tab="pane-pcw-references">Dudík et al. 2005</a></span>)
 and suggesting that in large natural history museum and herbarium 
collections, target‐group background may be interpreted as a random 
sample from the (biased) sampling distribution. Target‐group background 
can also be used for other modeling methods, and is analyzed in detail 
elsewhere (Phillips et al. unpubl.). We have also shown that using a 
random sample of around 10 000 background sites achieves the same model 
performance as using the full set of sites from the whole study area, 
offering a dramatic reduction in processing requirements for large 
datasets.
            </p>
            
            <p>The new hinge features introduced here significantly 
improve model performance. Hinge features can be used with any 
presence‐only dataset, and are used by default in the software whenever 
there are at least 15 presence sites. In fact, hinge features can 
effectively replace quadratic, product and threshold features: when 
hinge features are used, omitting Q, P and T features hardly changes 
predictive performance, with average AUC and COR scores changing by 
&lt;0.0001 for both random and target‐group background. Interestingly, 
hinge features do not significantly increase the complexity of models 
that Maxent can produce, since threshold features already allow an 
arbitrary response to each environmental variable. However, hinge 
features make piecewise linear contributions to the exponent of the 
Maxent model, which appear to incur less regularization penalty when 
approximating species' true response to the environmental variables than
 the step‐function response produced by threshold features. In other 
words, hinge features allow simpler and more succinct approximations of 
the true species response to the environment. Building on this 
principle, an interesting topic for future research would be to develop 
new feature classes that allow even more succinct response 
approximations, such as the splines commonly used in generalized 
additive models (<span><a href="#b16" id="#b16R" class="bibLink tab-link" data-tab="pane-pcw-references">Hastie and Tibshirani 1990</a></span>).
            </p>
            
            <p>The logistic output format, introduced here, is easier to
 interpret than previous Maxent output formats: it can be interpreted as
 estimating a species probability of presence, conditioned on 
environmental variables. For sites with small logistic value, i.e. that 
are predicted to be unsuitable or marginally suitable for the species, 
the logistic value is proportional to Maxent's raw output. For the most 
suitable sites, the logistic value is capped so that it remains below 
one. This addresses a problem with raw output, that the exponential 
model is not bounded above and can therefore give unreasonably high 
values to some sites, corresponding to probability of presence greater 
than one. Because the logistic format fixes this problem, it is better 
calibrated, in the sense that it has improved average COR.</p>
            
         </section>
         
         <section class="article-section__sub-content" id="ss31">
            
            <h3 class="article-section__sub-title section2" id="ss31-title"> Beyond the realized distribution</h3>
            
            <p>The evaluation and tuning we have described measures 
model performance according to the ability to predict the realized 
distribution of a species, and the parameter tunings are therefore 
optimized for predicting realized distributions. It is important to note
 that many applications of species distribution models depend on 
predicting potential distributions, rather than realized distributions. A
 species may have failed to disperse due to geographic barriers, or be 
excluded from an area due to competition. In the current evaluation, 
models which predict into such areas would be penalized, though it is 
not clear how many of the species considered here are absent from 
significant portions of their potential distribution. Some applications,
 however, require prediction into unoccupied areas, for instance, when 
measuring the inter‐predictivity of models of sister species (<span><a href="#b23" id="#b23R" class="bibLink tab-link" data-tab="pane-pcw-references">Peterson et al. 1999</a></span>).
 In such cases, good performance on presence‐absence data from the 
realized distribution may not necessarily imply effective prediction of 
the potential distribution. Thus, it is possible that different 
parameter settings would be needed than those developed here. When 
attempting to predict potential distributions, care should be taken to 
avoid indirect predictor variables whose effect on species may vary 
across the modeled region, such as altitude or the climatic conditions 
in particular months of the year (<span><a href="#b22" id="#b22R" class="bibLink tab-link" data-tab="pane-pcw-references">Peterson 2006</a></span>, <span><a href="#b27" id="#b27R" class="bibLink tab-link" data-tab="pane-pcw-references">Phillips et al. 2006</a></span>).
 Care is also needed with the choice of background data used during 
training, for example by removing areas where the species is excluded by
 known geographic barriers.
            </p>
            
            <p>Other uses of species distribution models involve 
“transferral”: producing a model over one study area and then applying 
it to another area, or to changed environmental conditions in the same 
area. Example applications include predicting the effect of climate 
change on species distributions (<span><a href="#b31" id="#b31R" class="bibLink tab-link" data-tab="pane-pcw-references">Thomas et al. 2004</a></span>, <span><a href="#b2" id="#b2R" class="bibLink tab-link" data-tab="pane-pcw-references">Araujo et al. 2005</a></span>) and predicting areas at risk for species invasions (<span><a href="#b24" id="#b24R" class="bibLink tab-link" data-tab="pane-pcw-references">Peterson et al. 2003</a></span>, <span><a href="#b32" id="#b32R" class="bibLink tab-link" data-tab="pane-pcw-references">Thuiller et al. 2005</a></span>).
 Such applications may require different choices of feature types and 
regularization parameters from those defined here. In particular, it has
 been shown that modeling methods that otherwise produce reasonably 
similar predictions can make wildly different predictions of species 
range size under climate change (<span><a href="#b21" id="#b21R" class="bibLink tab-link" data-tab="pane-pcw-references">Pearson et al. 2006</a></span>).
 It is an important avenue of future research to determine guidelines 
for using Maxent (and other modeling methods) to create models that can 
reliably be transferred to alternate climatic conditions or geographic 
areas.
            </p>
            
         </section>
         
      </section>
      <div class="article-section__content">
         
         <h2 class="article-section__title section__title section1" id="ss32-title"> Acknowledgements</h2>
         
         <p>We thank the other members of the working group on “Testing 
alternative methodologies for modelling species’ ecological niches and 
predicting geographical distributions”, led by A. T. Peterson and C. 
Moritz and hosted by the National Center for Ecological Analysis and 
Synthesis in Santa Barbara. Their kind inclusion of Maxent in the study 
enabled much of the research described here. We further thank those who 
supplied data for the working group, including working group members as 
well as A. Ford, CSIRO Atherton, for AWT data; M. Peck and G. Peck, 
Royal Ontario Museum and M. Cadman, Bird Studies Canada, Canadian 
Wildlife Service of Environment Canada, for CAN data; the National 
Vegetation Survey Databank and the Allan Herbarium, for NZ data; 
Missouri Botanical Garden, especially R. Magill and T. Consiglio, for SA
 data; and T. Wohlgemuth and U. Braendi from WSL Switzerland for SWI 
data. We also wish to thank Rob Anderson and Catherine Graham for 
extremely helpful commentary on the manuscript. Miroslav Dudík received 
support through NSF grant CCR‐0325463.</p>
         
         <ol></ol>
      </div>
      <section class="article-section article-section__references" id="references-section">
         <div class="accordion article-accordion">
            <h2>
               <div tabindex="0" role="button" class="accordion__control" aria-controls="idss33" aria-expanded="false" data-references="/action/ajaxShowEnhancedAbstract?widgetId=5cf4c79f-0ae9-4dc5-96ce-77f62de7ada9&amp;ajax=true&amp;doi=10.1111%2Fj.0906-7590.2008.5203.x&amp;pbContext=%3Bissue%3Aissue%3Adoi%5C%3A10.1111%2Feco.2008.31.issue-2%3Bpage%3Astring%3AArticle%2FChapter+View%3Bctype%3Astring%3AJournal+Content%3Bwebsite%3Awebsite%3Apericles%3Bjournal%3Ajournal%3A16000587%3Barticle%3Aarticle%3Adoi%5C%3A10.1111%2Fj.0906-7590.2008.5203.x%3Bwgroup%3Astring%3APublication+Websites%3BrequestedJournal%3Ajournal%3A16000587%3BpageGroup%3Astring%3APublication+Pages%3BsubPage%3Astring%3AFull+Text&amp;accordionHeadingWrapper=h2&amp;onlyLog=true"><span class="section__title" id="references-section-1">References</span><i aria-hidden="true" class="icon-section_arrow_d"></i></div>
            </h2>
            <div class="accordion__content" style="display: none;" role="
                            region
                        " aria-labelledby="
                            references-section-1" id="ss33">
               <ul class="rlist separator">
                  
                  
                  <li data-bib-id="b1">
                     <span class="author">Anderson, R. P.</span>. <span class="pubYear">2003</span>.  <span class="articleTitle">Real vs. artefactual absences in species distributions: tests for <i>Oryzomys albigularis</i> (Rodentia: Muridae) in Venezuela</span>. 
<i>Journal of Biogeography</i> 
                      <span class="vol">30</span>:  <span class="pageFirst">591</span>– <span class="pageLast">605</span>.

                     <div class="extra-links" id="e_1_2_9_2_1"><span>
                           <a href="https://onlinelibrary.wiley.com/doi/10.1046/j.1365-2699.2003.00867.x" onclick="newWindow(this.href);return false" target="_blank">Wiley Online Library</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1111%2Fj.0906-7590.2008.5203.x&amp;key=000182456800010" onclick="newWindow(this.href);return false" target="_blank">Web of Science®</a></span><span><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=30&amp;publication_year=2003&amp;pages=591-605&amp;journal=Journal+of+Biogeography&amp;author=R.+P.+Anderson&amp;title=Real+vs.+artefactual+absences+in+species+distributions%3A+tests+for+Oryzomys+albigularis+%28Rodentia%3A+Muridae%29+in+Venezuela" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="b2">
                     <span class="author">Araujo, M. B.</span>
                     						<i>et al</i>. <span class="pubYear">2005</span>.  <span class="articleTitle">Validation of species‐climate impact models under climate change</span>. 
<i>Global Change Biology</i> 
                      <span class="vol">11</span>:  <span class="pageFirst">1504</span>– <span class="pageLast">1513</span>.

                     <div class="extra-links" id="e_1_2_9_3_1"><span>
                           <a href="https://onlinelibrary.wiley.com/doi/10.1111/j.1365-2486.2005.01000.x" onclick="newWindow(this.href);return false" target="_blank">Wiley Online Library</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1111%2Fj.0906-7590.2008.5203.x&amp;key=000231396700009" onclick="newWindow(this.href);return false" target="_blank">Web of Science®</a></span><span><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=11&amp;publication_year=2005&amp;pages=1504-1513&amp;journal=Global+Change+Biology&amp;author=M.+B.+Araujo&amp;title=Validation+of+species%E2%80%90climate+impact+models+under+climate+change" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="b3">
                     <span class="author">Cover, T. M.</span> and 
<span class="author">Thomas, J. A.</span>. <span class="pubYear">2006</span>.  <span class="bookTitle">Elements of information theory</span>,  <span class="edition">2nd ed</span>.  Wiley.

                     <div class="extra-links" id="e_1_2_9_4_1"><span><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;publication_year=2006&amp;author=T.+M.+Cover&amp;author=J.+A.+Thomas&amp;title=Elements+of+information+theory" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="b4">
                     <span class="author">Dudík, M.</span>
                     						<i>et al</i>. <span class="pubYear">2004</span>.  <span class="chapterTitle">Performance guarantees for regularized maximum entropy density estimation</span>. 
 <span class="bookTitle">Proceedings of the Seventeenth Annual Conference on Computational Learning Theory</span>.  ACM Press, New York, pp.  <span class="pageFirst">655</span>– <span class="pageLast">662</span>.

                     <div class="extra-links" id="e_1_2_9_5_1"><span><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;publication_year=2004&amp;pages=655-662&amp;author=M.+Dud%C3%ADk&amp;title=Proceedings+of+the+Seventeenth+Annual+Conference+on+Computational+Learning+Theory" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="b5">
                     <span class="author">Dudík, M.</span>
                     						<i>et al</i>. <span class="pubYear">2005</span>.  <span class="chapterTitle">Correcting sample selection bias in maximum entropy density estimation</span>. 
 <span class="bookTitle">Advances in Neural Information Processing Systems 18</span>.  The MIT Press, pp.  <span class="pageFirst">323</span>– <span class="pageLast">330</span>.

                     <div class="extra-links" id="e_1_2_9_6_1"><span><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;publication_year=2005&amp;pages=323-330&amp;author=M.+Dud%C3%ADk&amp;title=Advances+in+Neural+Information+Processing+Systems+18" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="b6">
                     <span class="author">Elith, J.</span>. <span class="pubYear">2002</span>.  <span class="chapterTitle">Quantitative methods for modeling species habitat: comparative performance and an application to Australian plants</span>. – In:  <span class="editor">S. Ferson</span> and 
 <span class="editor">M. Burgman</span> (eds),  <span class="bookTitle">Quantitative methods for conservation biology</span>.  Springer, pp.  <span class="pageFirst">39</span>– <span class="pageLast">58</span>.

                     <div class="extra-links" id="e_1_2_9_7_1"><span><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;publication_year=2002&amp;pages=39-58&amp;author=J.+Elith&amp;title=Quantitative+methods+for+conservation+biology" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="b7">
                     <span class="author">Elith, J.</span> and 
<span class="author">Leathwick, J.</span>. <span class="pubYear">2007</span>.  <span class="articleTitle">Predicting
 species distributions from museum and herbarium records using 
multiresponse models fitted with multivariate adaptive regression 
splines</span>. 
<i>Diversity and Distributions</i> 
                      <span class="vol">13</span>:  <span class="pageFirst">265</span>– <span class="pageLast">275</span>.

                     <div class="extra-links" id="e_1_2_9_8_1"><span>
                           <a href="https://onlinelibrary.wiley.com/doi/10.1111/j.1472-4642.2007.00340.x" onclick="newWindow(this.href);return false" target="_blank">Wiley Online Library</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1111%2Fj.0906-7590.2008.5203.x&amp;key=000245812100003" onclick="newWindow(this.href);return false" target="_blank">Web of Science®</a></span><span><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=13&amp;publication_year=2007&amp;pages=265-275&amp;journal=Diversity+and+Distributions&amp;author=J.+Elith&amp;author=J.+Leathwick&amp;title=Predicting+species+distributions+from+museum+and+herbarium+records+using+multiresponse+models+fitted+with+multivariate+adaptive+regression+splines" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="b8">
                     <span class="author">Elith, J.</span>
                     						<i>et al</i>. <span class="pubYear">2006</span>.  <span class="articleTitle">Novel methods improve prediction of species’ distributions from occurrence data</span>. 
<i>Ecography</i> 
                      <span class="vol">29</span>:  <span class="pageFirst">129</span>– <span class="pageLast">151</span>.

                     <div class="extra-links" id="e_1_2_9_9_1"><span>
                           <a href="https://onlinelibrary.wiley.com/doi/10.1111/j.2006.0906-7590.04596.x" onclick="newWindow(this.href);return false" target="_blank">Wiley Online Library</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1111%2Fj.0906-7590.2008.5203.x&amp;key=000236767000001" onclick="newWindow(this.href);return false" target="_blank">Web of Science®</a></span><span><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=29&amp;publication_year=2006&amp;pages=129-151&amp;journal=Ecography&amp;author=J.+Elith&amp;title=Novel+methods+improve+prediction+of+species%E2%80%99+distributions+from+occurrence+data" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="b9">
                     <span class="author">Fielding, A. H.</span> and 
<span class="author">Bell, J. F.</span>. <span class="pubYear">1997</span>.  <span class="articleTitle">A review of methods for the assessment of prediction errors in conservation presence/absence models</span>. 
<i>Environmental Conservation</i> 
                      <span class="vol">24</span>:  <span class="pageFirst">38</span>– <span class="pageLast">49</span>.

                     <div class="extra-links" id="e_1_2_9_10_1"><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=16&amp;doi=10.1111%2Fj.0906-7590.2008.5203.x&amp;key=10.1017%2FS0376892997000088" onclick="newWindow(this.href);return false" target="_blank">Crossref</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1111%2Fj.0906-7590.2008.5203.x&amp;key=A1997XV97800008" onclick="newWindow(this.href);return false" target="_blank">Web of Science®</a></span><span><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=24&amp;publication_year=1997&amp;pages=38-49&amp;journal=Environmental+Conservation&amp;author=A.+H.+Fielding&amp;author=J.+F.+Bell&amp;title=A+review+of+methods+for+the+assessment+of+prediction+errors+in+conservation+presence%2Fabsence+models" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="b10">
                     <span class="author">Friedman, J.</span>. <span class="pubYear">1991</span>.  <span class="articleTitle">Multivariate adaptive regression splines (with discussion)</span>. 
<i>Ann. Stat.</i> 
                      <span class="vol">19</span>:  <span class="pageFirst">1</span>– <span class="pageLast">141</span>.

                     <div class="extra-links" id="e_1_2_9_11_1"><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=16&amp;doi=10.1111%2Fj.0906-7590.2008.5203.x&amp;key=10.1214%2Faos%2F1176347963" onclick="newWindow(this.href);return false" target="_blank">Crossref</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1111%2Fj.0906-7590.2008.5203.x&amp;key=A1991FF04700001" onclick="newWindow(this.href);return false" target="_blank">Web of Science®</a></span><span><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=19&amp;publication_year=1991&amp;pages=1-141&amp;journal=Ann.+Stat.&amp;author=J.+Friedman&amp;title=Multivariate+adaptive+regression+splines+%28with+discussion%29" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="b11">
                     <span class="author">Friedman, J. H.</span>. <span class="pubYear">2001</span>.  <span class="articleTitle">Greedy function approximation: a gradient boosting machine</span>. 
<i>Ann. Stat.</i> 
                      <span class="vol">29</span>:  <span class="pageFirst">1189</span>– <span class="pageLast">1232</span>.

                     <div class="extra-links" id="e_1_2_9_12_1"><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=16&amp;doi=10.1111%2Fj.0906-7590.2008.5203.x&amp;key=10.1214%2Faos%2F1013203451" onclick="newWindow(this.href);return false" target="_blank">Crossref</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1111%2Fj.0906-7590.2008.5203.x&amp;key=000173361700001" onclick="newWindow(this.href);return false" target="_blank">Web of Science®</a></span><span><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=29&amp;publication_year=2001&amp;pages=1189-1232&amp;journal=Ann.+Stat.&amp;author=J.+H.+Friedman&amp;title=Greedy+function+approximation%3A+a+gradient+boosting+machine" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="b12">
                     <span class="author">Graham, C. H.</span>
                     						<i>et al</i>. <span class="pubYear">2004</span>.  <span class="articleTitle">New developments in museum‐based informatics and applications in biodiversity analysis</span>. 
<i>Trends Ecol. Evol.</i> 
                      <span class="vol">19</span>:  <span class="pageFirst">497</span>– <span class="pageLast">503</span>.

                     <div class="extra-links" id="e_1_2_9_13_1"><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=16&amp;doi=10.1111%2Fj.0906-7590.2008.5203.x&amp;key=10.1016%2Fj.tree.2004.07.006" onclick="newWindow(this.href);return false" target="_blank">Crossref</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=8&amp;doi=10.1111%2Fj.0906-7590.2008.5203.x&amp;key=16701313" onclick="newWindow(this.href);return false" target="_blank">PubMed</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1111%2Fj.0906-7590.2008.5203.x&amp;key=000223899600011" onclick="newWindow(this.href);return false" target="_blank">Web of Science®</a></span><span><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=19&amp;publication_year=2004&amp;pages=497-503&amp;journal=Trends+Ecol.+Evol.&amp;author=C.+H.+Graham&amp;title=New+developments+in+museum%E2%80%90based+informatics+and+applications+in+biodiversity+analysis" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="b13">
                     <span class="author">Graham, C. H.</span>
                     						<i>et al</i>. <span class="pubYear">2006</span>.  <span class="articleTitle">Habitat history improves prediction of biodiversity in a rainforest fauna</span>. 
<i>Proc. Nat. Acad. Sci. USA</i> 
                      <span class="vol">103</span>:  <span class="pageFirst">632</span>– <span class="pageLast">636</span>.

                     <div class="extra-links" id="e_1_2_9_14_1"><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=16&amp;doi=10.1111%2Fj.0906-7590.2008.5203.x&amp;key=10.1073%2Fpnas.0505754103" onclick="newWindow(this.href);return false" target="_blank">Crossref</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=32&amp;doi=10.1111%2Fj.0906-7590.2008.5203.x&amp;key=1%3ACAS%3A528%3ADC%252BD28XhtVOiurc%253D" onclick="newWindow(this.href);return false" target="_blank">CAS</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=8&amp;doi=10.1111%2Fj.0906-7590.2008.5203.x&amp;key=16407139" onclick="newWindow(this.href);return false" target="_blank">PubMed</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1111%2Fj.0906-7590.2008.5203.x&amp;key=000234727800025" onclick="newWindow(this.href);return false" target="_blank">Web of Science®</a></span><span><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=103&amp;publication_year=2006&amp;pages=632-636&amp;journal=Proc.+Nat.+Acad.+Sci.+USA&amp;author=C.+H.+Graham&amp;title=Habitat+history+improves+prediction+of+biodiversity+in+a+rainforest+fauna" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="b14">
                     <span class="author">Grünwald, P.</span>. <span class="pubYear">2000</span>.  <span class="chapterTitle">Maximum entropy and the glasses you are looking through</span>. 
 <span class="bookTitle">Proceedings of the Sixteenth Annual Conference on Uncertainty in Artificial Intelligence (UAI2000)</span>, pp.  <span class="pageFirst">238</span>– <span class="pageLast">246</span>.

                     <div class="extra-links" id="e_1_2_9_15_1"><span><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;publication_year=2000&amp;pages=238-246&amp;author=P.+Gr%C3%BCnwald&amp;title=Proceedings+of+the+Sixteenth+Annual+Conference+on+Uncertainty+in+Artificial+Intelligence+%28UAI2000%29" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="b15">
                     <span class="author">Grünwald, P. D.</span> and 
<span class="author">Dawid, A. P.</span>. <span class="pubYear">2004</span>.  <span class="articleTitle">Game theory, maximum entropy, minimum discrepancy, and robust Bayesian decision theory</span>. 
<i>Ann. Stat.</i> 
                      <span class="vol">32</span>:  <span class="pageFirst">1367</span>– <span class="pageLast">1433</span>.

                     <div class="extra-links" id="e_1_2_9_16_1"><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=16&amp;doi=10.1111%2Fj.0906-7590.2008.5203.x&amp;key=10.1214%2F009053604000000553" onclick="newWindow(this.href);return false" target="_blank">Crossref</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1111%2Fj.0906-7590.2008.5203.x&amp;key=000223519100003" onclick="newWindow(this.href);return false" target="_blank">Web of Science®</a></span><span><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=32&amp;publication_year=2004&amp;pages=1367-1433&amp;journal=Ann.+Stat.&amp;author=P.+D.+Gr%C3%BCnwald&amp;author=A.+P.+Dawid&amp;title=Game+theory%2C+maximum+entropy%2C+minimum+discrepancy%2C+and+robust+Bayesian+decision+theory" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="b16">
                     <span class="author">Hastie, T.</span> and 
<span class="author">Tibshirani, R.</span>. <span class="pubYear">1990</span>.  <span class="bookTitle">Generalized additive models</span>.  Chapman and Hall.

                     <div class="extra-links" id="e_1_2_9_17_1"><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1111%2Fj.0906-7590.2008.5203.x&amp;key=A1990EK04600007" onclick="newWindow(this.href);return false" target="_blank">Web of Science®</a></span><span><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;publication_year=1990&amp;author=T.+Hastie&amp;author=R.+Tibshirani&amp;title=Generalized+additive+models" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="b17">
                     <span class="author">Hastie, T.</span>
                     						<i>et al</i>. <span class="pubYear">2001</span>.  <span class="bookTitle">The elements of statistical learning: data mining, inference, and prediction</span>.  Springer.

                     <div class="extra-links" id="e_1_2_9_18_1"><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=16&amp;doi=10.1111%2Fj.0906-7590.2008.5203.x&amp;key=10.1007%2F978-0-387-21606-5" onclick="newWindow(this.href);return false" target="_blank">Crossref</a></span><span><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;publication_year=2001&amp;author=T.+Hastie&amp;title=The+elements+of+statistical+learning%3A+data+mining%2C+inference%2C+and+prediction" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="b18">
                     <span class="author">Jaynes, E. T.</span>. <span class="pubYear">1957</span>.  <span class="articleTitle">Information theory and statistical mechanics</span>. 
<i>Phys. Rev.</i> 
                      <span class="vol">106</span>:  <span class="pageFirst">620</span>– <span class="pageLast">630</span>.

                     <div class="extra-links" id="e_1_2_9_19_1"><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=16&amp;doi=10.1111%2Fj.0906-7590.2008.5203.x&amp;key=10.1103%2FPhysRev.106.620" onclick="newWindow(this.href);return false" target="_blank">Crossref</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1111%2Fj.0906-7590.2008.5203.x&amp;key=A1957WB72300004" onclick="newWindow(this.href);return false" target="_blank">Web of Science®</a></span><span><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=106&amp;publication_year=1957&amp;pages=620-630&amp;journal=Phys.+Rev.&amp;author=E.+T.+Jaynes&amp;title=Information+theory+and+statistical+mechanics" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="b19">
                     <span class="author">Keating, K. A.</span> and 
<span class="author">Cherry, S.</span>. <span class="pubYear">2004</span>.  <span class="articleTitle">Use and interpretation of logistic regression in habitat‐selection studies</span>. 
<i>J. Wildl. Manage.</i> 
                      <span class="vol">68</span>:  <span class="pageFirst">774</span>– <span class="pageLast">789</span>.

                     <div class="extra-links" id="e_1_2_9_20_1"><span>
                           <a href="https://wildlife.onlinelibrary.wiley.com/doi/10.2193/0022-541X%282004%29068%5B0774%3AUAIOLR%5D2.0.CO%3B2" onclick="newWindow(this.href);return false" target="_blank">Wiley Online Library</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1111%2Fj.0906-7590.2008.5203.x&amp;key=000226322800003" onclick="newWindow(this.href);return false" target="_blank">Web of Science®</a></span><span><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=68&amp;publication_year=2004&amp;pages=774-789&amp;journal=J.+Wildl.+Manage.&amp;author=K.+A.+Keating&amp;author=S.+Cherry&amp;title=Use+and+interpretation+of+logistic+regression+in+habitat%E2%80%90selection+studies" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="b20">
                     <span class="author">Ng, A. Y.</span> and 
<span class="author">Jordan, M. I.</span>. <span class="pubYear">2001</span>.  <span class="articleTitle">On discriminative vs. generative classifiers: a comparison of logistic regression and naive Bayes</span>. 
<i>Adv. Neural Inform. Process. Syst.</i> 
                      <span class="vol">14</span>:  <span class="pageFirst">605</span>– <span class="pageLast">610</span>.

                     <div class="extra-links" id="e_1_2_9_21_1"><span><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=14&amp;publication_year=2001&amp;pages=605-610&amp;journal=Adv.+Neural+Inform.+Process.+Syst.&amp;author=A.+Y.+Ng&amp;author=M.+I.+Jordan&amp;title=On+discriminative+vs.+generative+classifiers%3A+a+comparison+of+logistic+regression+and+naive+Bayes" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="b21">
                     <span class="author">Pearson, R. G.</span>
                     						<i>et al</i>. <span class="pubYear">2006</span>.  <span class="articleTitle">Model‐based uncertainty in species range prediction</span>. 
<i>J. Biogeogr.</i> 
                      <span class="vol">33</span>:  <span class="pageFirst">1704</span>– <span class="pageLast">1711</span>.

                     <div class="extra-links" id="e_1_2_9_22_1"><span>
                           <a href="https://onlinelibrary.wiley.com/doi/10.1111/j.1365-2699.2006.01460.x" onclick="newWindow(this.href);return false" target="_blank">Wiley Online Library</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1111%2Fj.0906-7590.2008.5203.x&amp;key=000240367800003" onclick="newWindow(this.href);return false" target="_blank">Web of Science®</a></span><span><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=33&amp;publication_year=2006&amp;pages=1704-1711&amp;journal=J.+Biogeogr.&amp;author=R.+G.+Pearson&amp;title=Model%E2%80%90based+uncertainty+in+species+range+prediction" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="b22">
                     <span class="author">Peterson, A. T.</span>. <span class="pubYear">2006</span>.  <span class="articleTitle">Uses and requirements of ecological niche models and related distributional models</span>. 
<i>Biodiv. Inform.</i> 
                      <span class="vol">3</span>:  <span class="pageFirst">59</span>– <span class="pageLast">72</span>.

                     <div class="extra-links" id="e_1_2_9_23_1"><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=16&amp;doi=10.1111%2Fj.0906-7590.2008.5203.x&amp;key=10.17161%2Fbi.v3i0.29" onclick="newWindow(this.href);return false" target="_blank">Crossref</a></span><span><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=3&amp;publication_year=2006&amp;pages=59-72&amp;journal=Biodiv.+Inform.&amp;author=A.+T.+Peterson&amp;title=Uses+and+requirements+of+ecological+niche+models+and+related+distributional+models" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="b23">
                     <span class="author">Peterson, A. T.</span>
                     						<i>et al</i>. <span class="pubYear">1999</span>.  <span class="articleTitle">Conservatism of ecological niches in evolutionary time</span>. 
<i>Science</i> 
                      <span class="vol">285</span>:  <span class="pageFirst">1265</span>– <span class="pageLast">1267</span>.

                     <div class="extra-links" id="e_1_2_9_24_1"><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=16&amp;doi=10.1111%2Fj.0906-7590.2008.5203.x&amp;key=10.1126%2Fscience.285.5431.1265" onclick="newWindow(this.href);return false" target="_blank">Crossref</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=32&amp;doi=10.1111%2Fj.0906-7590.2008.5203.x&amp;key=1%3ACAS%3A528%3ADyaK1MXls1Sitrs%253D" onclick="newWindow(this.href);return false" target="_blank">CAS</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=8&amp;doi=10.1111%2Fj.0906-7590.2008.5203.x&amp;key=10455053" onclick="newWindow(this.href);return false" target="_blank">PubMed</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1111%2Fj.0906-7590.2008.5203.x&amp;key=000082130200043" onclick="newWindow(this.href);return false" target="_blank">Web of Science®</a></span><span><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=285&amp;publication_year=1999&amp;pages=1265-1267&amp;journal=Science&amp;author=A.+T.+Peterson&amp;title=Conservatism+of+ecological+niches+in+evolutionary+time" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="b24">
                     <span class="author">Peterson, A. T.</span>
                     						<i>et al</i>. <span class="pubYear">2003</span>.  <span class="articleTitle">Predicting the potential invasive distributions of four alien plant species in North America</span>. 
<i>Weed Sci.</i> 
                      <span class="vol">51</span>:  <span class="pageFirst">863</span>– <span class="pageLast">868</span>.

                     <div class="extra-links" id="e_1_2_9_25_1"><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=16&amp;doi=10.1111%2Fj.0906-7590.2008.5203.x&amp;key=10.1614%2FP2002-081" onclick="newWindow(this.href);return false" target="_blank">Crossref</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=32&amp;doi=10.1111%2Fj.0906-7590.2008.5203.x&amp;key=1%3ACAS%3A528%3ADC%252BD3sXpsFams7Y%253D" onclick="newWindow(this.href);return false" target="_blank">CAS</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1111%2Fj.0906-7590.2008.5203.x&amp;key=000186783600005" onclick="newWindow(this.href);return false" target="_blank">Web of Science®</a></span><span><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=51&amp;publication_year=2003&amp;pages=863-868&amp;journal=Weed+Sci.&amp;author=A.+T.+Peterson&amp;title=Predicting+the+potential+invasive+distributions+of+four+alien+plant+species+in+North+America" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="b63">
                     <span class="author">Phillips, S. J.</span>
                     						<i>et al</i>. <span class="pubYear">2004</span>.  <span class="chapterTitle">A maximum entropy approach to species distribution modeling</span>. 
 <span class="bookTitle">Proceedings of the Twenty‐First International Conference on Machine Learning</span>.  ACM Press, New York, pp.  <span class="pageFirst">472</span>– <span class="pageLast">486</span>.

                     <div class="extra-links" id="e_1_2_9_26_1"><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1111%2Fj.0906-7590.2008.5203.x&amp;key=000222864400033" onclick="newWindow(this.href);return false" target="_blank">Web of Science®</a></span><span><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;publication_year=2004&amp;pages=472-486&amp;author=S.+J.+Phillips&amp;title=Proceedings+of+the+Twenty%E2%80%90First+International+Conference+on+Machine+Learning" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="b26">
                     <span class="author">Phillips, S. J.</span>
                     						<i>et al</i>. <span class="pubYear">2005</span>.  <span class="bookTitle">Maxent software for species distribution modeling</span>
					&lt;http://www.cs.princeton.edu/∼schapire/maxent/&gt;..

                     <div class="extra-links" id="e_1_2_9_27_1"><span><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;publication_year=2005&amp;author=S.+J.+Phillips&amp;title=Maxent+software+for+species+distribution+modeling" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="b27">
                     <span class="author">Phillips, S. J.</span>
                     						<i>et al</i>. <span class="pubYear">2006</span>.  <span class="articleTitle">Maximum entropy modeling of species geographic distributions</span>. 
<i>Ecol. Model.</i> 
                      <span class="vol">190</span>:  <span class="pageFirst">231</span>– <span class="pageLast">259</span>.

                     <div class="extra-links" id="e_1_2_9_28_1"><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=16&amp;doi=10.1111%2Fj.0906-7590.2008.5203.x&amp;key=10.1016%2Fj.ecolmodel.2005.03.026" onclick="newWindow(this.href);return false" target="_blank">Crossref</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1111%2Fj.0906-7590.2008.5203.x&amp;key=000233859600001" onclick="newWindow(this.href);return false" target="_blank">Web of Science®</a></span><span><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=190&amp;publication_year=2006&amp;pages=231-259&amp;journal=Ecol.+Model.&amp;author=S.+J.+Phillips&amp;title=Maximum+entropy+modeling+of+species+geographic+distributions" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="b28">
                     <span class="author">Ponder, W. F.</span>
                     						<i>et al</i>. <span class="pubYear">2001</span>.  <span class="articleTitle">Evaluation of museum collection data for use in biodiversity assessment</span>. 
<i>Conserv. Biol.</i> 
                      <span class="vol">15</span>:  <span class="pageFirst">648</span>– <span class="pageLast">657</span>.

                     <div class="extra-links" id="e_1_2_9_29_1"><span>
                           <a href="https://conbio.onlinelibrary.wiley.com/doi/10.1046/j.1523-1739.2001.015003648.x" onclick="newWindow(this.href);return false" target="_blank">Wiley Online Library</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1111%2Fj.0906-7590.2008.5203.x&amp;key=000169226000013" onclick="newWindow(this.href);return false" target="_blank">Web of Science®</a></span><span><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=15&amp;publication_year=2001&amp;pages=648-657&amp;journal=Conserv.+Biol.&amp;author=W.+F.+Ponder&amp;title=Evaluation+of+museum+collection+data+for+use+in+biodiversity+assessment" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="b29">
                     <span class="author">Randin, C. F.</span>
                     						<i>et al</i>. <span class="pubYear">2006</span>.  <span class="articleTitle">Are niche‐based species distribution models transferable in space?</span>. 
<i>J. Biogeogr.</i> 
                      <span class="vol">33</span>:  <span class="pageFirst">1689</span>– <span class="pageLast">1703</span>.

                     <div class="extra-links" id="e_1_2_9_30_1"><span>
                           <a href="https://onlinelibrary.wiley.com/doi/10.1111/j.1365-2699.2006.01466.x" onclick="newWindow(this.href);return false" target="_blank">Wiley Online Library</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1111%2Fj.0906-7590.2008.5203.x&amp;key=000240367800002" onclick="newWindow(this.href);return false" target="_blank">Web of Science®</a></span><span><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=33&amp;publication_year=2006&amp;pages=1689-1703&amp;journal=J.+Biogeogr.&amp;author=C.+F.+Randin&amp;title=Are+niche%E2%80%90based+species+distribution+models+transferable+in+space%3F" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="b30">
                     <span class="author">Reddy, S.</span> and 
<span class="author">Dávalos, L. M.</span>. <span class="pubYear">2003</span>.  <span class="articleTitle">Geographical sampling bias and its implications for conservation priorities in Africa</span>. 
<i>J. Biogeogr.</i> 
                      <span class="vol">30</span>:  <span class="pageFirst">1719</span>– <span class="pageLast">1727</span>.

                     <div class="extra-links" id="e_1_2_9_31_1"><span>
                           <a href="https://onlinelibrary.wiley.com/doi/10.1046/j.1365-2699.2003.00946.x" onclick="newWindow(this.href);return false" target="_blank">Wiley Online Library</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1111%2Fj.0906-7590.2008.5203.x&amp;key=000186169500006" onclick="newWindow(this.href);return false" target="_blank">Web of Science®</a></span><span><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=30&amp;publication_year=2003&amp;pages=1719-1727&amp;journal=J.+Biogeogr.&amp;author=S.+Reddy&amp;author=L.+M.+D%C3%A1valos&amp;title=Geographical+sampling+bias+and+its+implications+for+conservation+priorities+in+Africa" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="b31">
                     <span class="author">Thomas, C. D.</span>
                     						<i>et al</i>. <span class="pubYear">2004</span>.  <span class="articleTitle">Extinction risk from climate change</span>. 
<i>Nature</i> 
                      <span class="vol">427</span>:  <span class="pageFirst">145</span>– <span class="pageLast">148</span>.

                     <div class="extra-links" id="e_1_2_9_32_1"><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=16&amp;doi=10.1111%2Fj.0906-7590.2008.5203.x&amp;key=10.1038%2Fnature02121" onclick="newWindow(this.href);return false" target="_blank">Crossref</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=32&amp;doi=10.1111%2Fj.0906-7590.2008.5203.x&amp;key=1%3ACAS%3A528%3ADC%252BD2cXhtFOgtQ%253D%253D" onclick="newWindow(this.href);return false" target="_blank">CAS</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=8&amp;doi=10.1111%2Fj.0906-7590.2008.5203.x&amp;key=14712274" onclick="newWindow(this.href);return false" target="_blank">PubMed</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1111%2Fj.0906-7590.2008.5203.x&amp;key=000187863900033" onclick="newWindow(this.href);return false" target="_blank">Web of Science®</a></span><span><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=427&amp;publication_year=2004&amp;pages=145-148&amp;journal=Nature&amp;author=C.+D.+Thomas&amp;title=Extinction+risk+from+climate+change" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="b32">
                     <span class="author">Thuiller, W.</span>
                     						<i>et al</i>. <span class="pubYear">2005</span>.  <span class="articleTitle">Niche‐based modelling as a tool for predicting the risk of alien plant invasions at a global scale</span>. 
<i>Global Change Biol.</i> 
                      <span class="vol">11</span>:  <span class="pageFirst">2234</span>– <span class="pageLast">2250</span>.

                     <div class="extra-links" id="e_1_2_9_33_1"><span>
                           <a href="https://onlinelibrary.wiley.com/doi/10.1111/j.1365-2486.2005.001018.x" onclick="newWindow(this.href);return false" target="_blank">Wiley Online Library</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1111%2Fj.0906-7590.2008.5203.x&amp;key=000234087300014" onclick="newWindow(this.href);return false" target="_blank">Web of Science®</a></span><span><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=11&amp;publication_year=2005&amp;pages=2234-2250&amp;journal=Global+Change+Biol.&amp;author=W.+Thuiller&amp;title=Niche%E2%80%90based+modelling+as+a+tool+for+predicting+the+risk+of+alien+plant+invasions+at+a+global+scale" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="b33">
                     <span class="author">Tibshirani, R.</span>. <span class="pubYear">1996</span>.  <span class="bookTitle">Bias, variance and prediction error for classification rules</span>.  Technical report, Univ. of Toronto.

                     <div class="extra-links" id="e_1_2_9_34_1"><span><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;publication_year=1996&amp;author=R.+Tibshirani&amp;title=Bias%2C+variance+and+prediction+error+for+classification+rules" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="b34">
                     <span class="author">Topsøe, F.</span>. <span class="pubYear">1979</span>.  <span class="articleTitle">Information theoretical optimization techniques</span>. 
<i>Kybernetika</i> 
                      <span class="vol">15</span>:  <span class="pageFirst">8</span>– <span class="pageLast">27</span>.

                     <div class="extra-links" id="e_1_2_9_35_1"><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1111%2Fj.0906-7590.2008.5203.x&amp;key=A1979GQ08900002" onclick="newWindow(this.href);return false" target="_blank">Web of Science®</a></span><span><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=15&amp;publication_year=1979&amp;pages=8-27&amp;journal=Kybernetika&amp;author=F.+Tops%C3%B8e&amp;title=Information+theoretical+optimization+techniques" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="b35">
                     <span class="author">Ward, G.</span>
                     						<i>et al</i>. <span class="pubYear">2007</span>.  <span class="bookTitle">Presence‐only data and the em algorithm</span>
					Biometrics, in press.

                     <div class="extra-links" id="e_1_2_9_36_1"><span><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;publication_year=2007&amp;author=G.+Ward&amp;title=Presence%E2%80%90only+data+and+the+em+algorithm" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="b36">
                     <span class="author">Williams, P. M.</span>. <span class="pubYear">1995</span>.  <span class="articleTitle">Bayesian regularization and pruning using a Laplace prior</span>. 
<i>Neural Comput.</i> 
                      <span class="vol">7</span>:  <span class="pageFirst">117</span>– <span class="pageLast">143</span>.

                     <div class="extra-links" id="e_1_2_9_37_1"><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=16&amp;doi=10.1111%2Fj.0906-7590.2008.5203.x&amp;key=10.1162%2Fneco.1995.7.1.117" onclick="newWindow(this.href);return false" target="_blank">Crossref</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1111%2Fj.0906-7590.2008.5203.x&amp;key=A1995QG83300010" onclick="newWindow(this.href);return false" target="_blank">Web of Science®</a></span><span><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=7&amp;publication_year=1995&amp;pages=117-143&amp;journal=Neural+Comput.&amp;author=P.+M.+Williams&amp;title=Bayesian+regularization+and+pruning+using+a+Laplace+prior" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="b37">
                     <span class="author">Zaniewski, A. E.</span>
                     						<i>et al</i>. <span class="pubYear">2002</span>.  <span class="articleTitle">Predicting species spatial distributions using presence‐only data: a case study of native New Zealand ferns</span>. 
<i>Ecol. Model.</i> 
                      <span class="vol">157</span>:  <span class="pageFirst">261</span>– <span class="pageLast">280</span>.

                     <div class="extra-links" id="e_1_2_9_38_1"><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=16&amp;doi=10.1111%2Fj.0906-7590.2008.5203.x&amp;key=10.1016%2FS0304-3800%2802%2900199-0" onclick="newWindow(this.href);return false" target="_blank">Crossref</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1111%2Fj.0906-7590.2008.5203.x&amp;key=000179241300012" onclick="newWindow(this.href);return false" target="_blank">Web of Science®</a></span><span><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=157&amp;publication_year=2002&amp;pages=261-280&amp;journal=Ecol.+Model.&amp;author=A.+E.+Zaniewski&amp;title=Predicting+species+spatial+distributions+using+presence%E2%80%90only+data%3A+a+case+study+of+native+New+Zealand+ferns" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="b38">
                     <span class="author">Zheng, B.</span> and 
<span class="author">Agresti, A.</span>. <span class="pubYear">2000</span>.  <span class="articleTitle">Summarizing the predictive power of a generalized linear model</span>. 
<i>Stat. Med.</i> 
                      <span class="vol">19</span>:  <span class="pageFirst">1771</span>– <span class="pageLast">1781</span>.

                     <div class="extra-links" id="e_1_2_9_39_1"><span>
                           <a href="https://onlinelibrary.wiley.com/doi/10.1002/1097-0258%2820000715%2919%3A13%3C1771%3A%3AAID-SIM485%3E3.0.CO%3B2-P" onclick="newWindow(this.href);return false" target="_blank">Wiley Online Library</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=32&amp;doi=10.1111%2Fj.0906-7590.2008.5203.x&amp;key=1%3ASTN%3A280%3ADC%252BD3M7islemtw%253D%253D" onclick="newWindow(this.href);return false" target="_blank">CAS</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=8&amp;doi=10.1111%2Fj.0906-7590.2008.5203.x&amp;key=10861777" onclick="newWindow(this.href);return false" target="_blank">PubMed</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1111%2Fj.0906-7590.2008.5203.x&amp;key=000088063000005" onclick="newWindow(this.href);return false" target="_blank">Web of Science®</a></span><span><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=19&amp;publication_year=2000&amp;pages=1771-1781&amp;journal=Stat.+Med.&amp;author=B.+Zheng&amp;author=A.+Agresti&amp;title=Summarizing+the+predictive+power+of+a+generalized+linear+model" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
               </ul>
            </div>
         </div>
      </section>
   </section>
   <div style="display:none">
      <ul class="custom-style-list noteList"></ul>
   </div>
   <section id="citedBy" class="article-section article-section__citedBy" data-source="https://onlinelibrary.wiley.com/action/ajaxShowCitedBy?doi=10.1111%2Fj.0906-7590.2008.5203.x">
      <div class="accordion article-accordion">
         <h2>
            <div tabindex="0" class="accordion__control" role="button" aria-controls="idss15" aria-expanded="false"><span class="section__title" id="citedby-section">Citing Literature</span><i aria-hidden="true" class="icon-section_arrow_d"></i></div>
         </h2>
         <div class="accordion__content" role="region" aria-labelledby="citedby-section" id="idss15" style="display: none;"></div>
      </div>
   </section>
   
</article></div></div><div class="col-lg-4 col-md-4 col-sm-4 hidden-xs hidden-sm hidden-md article-row-right fixed-element" style="top: -359px;"><div class="publication-cover-info clearfix">



        
        <div class="cover-image-wrapper cover-image__image hasDetails"><a href="https://onlinelibrary.wiley.com/toc/16000587/2008/31/2"><img src="eco.gif" alt="Publication cover image"></a></div><div class="extra-info-wrapper cover-image__details"><p class="volume-issue"><a href="https://onlinelibrary.wiley.com/toc/16000587/2008/31/2"><span>Volume</span><span class="val">31</span>, <span>Issue</span><span class="val">2</span></a></p><p>April 2008</p><p class="page-range"><span>Pages</span><span> 161-175</span></p>



        
        
</div>
</div>



        
        



        <div class="pb-ad">
            <div id="advert-rail-1" class="advert advert-rail" data-google-query-id="CPuwpOnahegCFcqLdwodQosEJQ">
			

			<!-- No JavaScript support is a call to an image -->
			
		<div id="google_ads_iframe_/2215/wly.ecology_000046/ecography_ecog_1__container__" style="border: 0pt none;"><iframe id="google_ads_iframe_/2215/wly.ecology_000046/ecography_ecog_1" title="3rd party ad content" name="google_ads_iframe_/2215/wly.ecology_000046/ecography_ecog_1" scrolling="no" marginwidth="0" marginheight="0" style="border: 0px none; vertical-align: bottom;" srcdoc="" data-google-container-id="2" data-load-complete="true" width="300" height="250" frameborder="0"></iframe></div></div>
        </div>
    

<div class="sticko__parent scrollThenFix" style="height: calc(-85px + 100vh);"><div class="tab tab--slide tab--flex sticko__md tabs--xs dynamic-sticko  tab--flex tabs--xs"><ul data-mobile-toggle="slide" role="tablist" class="rlist tab__nav w-slide--list cloned"><li role="presentation"><a data-slide-clone="self" href="#pane-pcw-figures" aria-controls="#pane-pcw-figures" role="tab" data-toggle="tab" id="pane-pcw-figurescon" data-slide-target="#pane-pcw-figures" class="article-row-right__tabLinks w-slide__btn figures-tab empty" tabindex="-1" aria-selected="false"><i aria-hidden="true" class="icon-multi-figure"></i><h2>Figures</h2></a></li><li role="presentation"><a data-slide-clone="self" href="#pane-pcw-references" aria-controls="#pane-pcw-references" role="tab" data-toggle="tab" id="pane-pcw-referencescon" data-slide-target="#pane-pcw-references" class="article-row-right__tabLinks w-slide__btn references-tab empty" tabindex="-1" aria-selected="false"><i aria-hidden="true" class="icon-chain"></i><h2>References</h2></a></li><li role="presentation"><a data-slide-clone="self" href="#pane-pcw-related" aria-controls="#pane-pcw-related" role="tab" data-toggle="tab" id="pane-pcw-relatedcon" data-slide-target="#pane-pcw-related" class="article-row-right__tabLinks w-slide__btn related-tab empty" tabindex="-1" aria-selected="false"><i aria-hidden="true" class="icon-related-circles-extend-button"></i><h2>Related</h2></a></li><li role="presentation" class="active"><a data-slide-clone="self" href="#pane-pcw-details" aria-controls="#pane-pcw-details" role="tab" data-toggle="tab" id="pane-pcw-detailscon" data-slide-target="#pane-pcw-details" class="article-row-right__tabLinks w-slide__btn details-tab" tabindex="-1" aria-selected="false"><i aria-hidden="true" class="icon-Icon_Information"></i><h2>Information</h2></a></li></ul><ul role="presentation" class="rlist tab__content sticko__child"><li id="pane-pcw-figures" aria-labelledby="pane-pcw-figurescon" role="tabpanel" class="tab__pane article-row-right__panes empty" aria-hidden="true"></li><li id="pane-pcw-references" aria-labelledby="pane-pcw-referencescon" role="tabpanel" class="tab__pane article-row-right__panes empty" aria-hidden="true"></li><li id="pane-pcw-related" aria-labelledby="pane-pcw-relatedcon" role="tabpanel" class="tab__pane article-row-right__panes empty" aria-hidden="true"></li><li id="pane-pcw-details" aria-labelledby="pane-pcw-detailscon" role="tabpanel" class="tab__pane article-row-right__panes active" aria-hidden="true"><div class="contentTab">
<section class="info-tab-wrapper">
<section class="metrics">
<h3 class="info-header">Metrics</h3>
<div class="section__body">
<div class="cited-by-count"><label>Citations: </label><span>2479</span></div>
<div class="pb-dropzone" data-pb-dropzone="publicaitonContent-info-tab-metrics">
<script type="text/javascript" src="https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js"></script>
<div data-badge-popover="left" data-badge-type="1" data-doi="10.1111/j.0906-7590.2008.5203.x" data-hide-no-mentions="false" class="altmetric-embed" data-uuid="1f394bd5-8b76-3dbf-f068-d6089ec8c5d2"><a target="_self" href="https://www.altmetric.com/details.php?domain=onlinelibrary.wiley.com&amp;citation_id=16592058" rel="popover" data-content="&lt;div&gt;    &lt;div style='padding-left: 10px; line-height:18px; border-left: 16px solid #ffd140;'&gt;
      &lt;a class='link-to-altmetric-details-tab' target='_self' href='https://www.altmetric.com/details.php?domain=onlinelibrary.wiley.com&amp;citation_id=16592058&amp;tab=blogs'&gt;
          Blogged by &lt;b&gt;3&lt;/b&gt;
      &lt;/a&gt;
    &lt;/div&gt;
    &lt;div style='padding-left: 10px; line-height:18px; border-left: 16px solid #9f79f2;'&gt;
      &lt;a class='link-to-altmetric-details-tab' target='_self' href='https://www.altmetric.com/details.php?domain=onlinelibrary.wiley.com&amp;citation_id=16592058&amp;tab=policy-documents'&gt;
          Referenced in &lt;b&gt;3&lt;/b&gt; policy sources
      &lt;/a&gt;
    &lt;/div&gt;
    &lt;div style='padding-left: 10px; line-height:18px; border-left: 16px solid #74CFED;'&gt;
      &lt;a class='link-to-altmetric-details-tab' target='_self' href='https://www.altmetric.com/details.php?domain=onlinelibrary.wiley.com&amp;citation_id=16592058&amp;tab=twitter'&gt;
          Tweeted by &lt;b&gt;1&lt;/b&gt;
      &lt;/a&gt;
    &lt;/div&gt;

    &lt;div class='altmetric-embed readers' style='margin-top: 10px;'&gt;
          &lt;div class='altmetric-embed tip_mendeley'
               style='padding-left: 10px; line-height:18px; border-left: 16px solid #A60000;'&gt;
              &lt;b&gt;3323&lt;/b&gt; readers on Mendeley
          &lt;/div&gt;
          &lt;div class='altmetric-embed tip_citeulike'
               style='padding-left: 10px; line-height:18px; border-left: 16px solid #BCD2EF;'&gt;
              &lt;b&gt;2&lt;/b&gt; readers on CiteULike
          &lt;/div&gt;
    &lt;/div&gt;
    &lt;div style='margin-top: 10px; text-align: center;'&gt;
        &lt;a class='altmetric_details' target='_self' href='https://www.altmetric.com/details.php?domain=onlinelibrary.wiley.com&amp;citation_id=16592058'&gt;
            See more details
        &lt;/a&gt;
    &lt;/div&gt;
&lt;/div&gt;" style="display:inline-block;" data-badge-popover="left">
        <img alt="Article has an altmetric score of 28" src="https://d1uo4w7k31k5mn.cloudfront.net/v1/28.png" style="border:0; margin:0; max-width: none;" width="110px" height="20px">
</a></div>
</div>
</div>
</section>
<section class="details">
<h3 class="info-header">Details</h3>
<p class="copyright">© 2008 The Authors</p>
<p></p>
<ul class="cross-mark rlist">
</ul>
</section>
<section class="publication-history">
<h3 class="info-header">Publication History</h3>
<ul class="rlist">
<li><label>Issue Online: </label>28 March 2008
</li>
<li><label>Version of Record online: </label>28 March 2008
</li>
<li><label>Editorial history: </label>Manuscript Accepted 13 December 2007
</li>
</ul>
</section>
</section>

</div><div class="emptyTab">



        
        



        <div class="pb-ad">
            <aside>
	<div id="advert-rail-2" class="advert advert-rail" data-google-query-id="CPywpOnahegCFcqLdwodQosEJQ">
		

		<!-- No JavaScript support is a call to an image -->
		
	<div id="google_ads_iframe_/2215/wly.ecology_000046/ecography_ecog_2__container__" style="border: 0pt none;"><iframe id="google_ads_iframe_/2215/wly.ecology_000046/ecography_ecog_2" title="3rd party ad content" name="google_ads_iframe_/2215/wly.ecology_000046/ecography_ecog_2" scrolling="no" marginwidth="0" marginheight="0" style="border: 0px none; vertical-align: bottom;" srcdoc="" data-google-container-id="3" data-load-complete="true" width="300" height="250" frameborder="0"></iframe></div></div>
</aside>
        </div>
    

</div></li><li class="tab__spinner" style="display: none;"><img src="spinner.gif" id="spinner" style="width: 100%" alt=""></li></ul></div></div></div></div></article><script>var articleRef = document.querySelector('.article__body:not(.show-references) .article__references');
if (articleRef) { articleRef.style.display = "none"; }

</script><div id="figure-viewer" data-ux3-wrapper="figure-viewer" data-ux3-transformed-by="figureInit" data-ux3-role="parent" role="dialog" class="figure-viewer"><div class="figure-viewer__reg__top clearfix"><div class="figure-viewer__top__right"><a href="#" data-ux3-role="controller" role="button" class="figure-viewer__ctrl__close"><span class="icon-close_thin"><span class="sr-only">Close Figure Viewer</span></span></a></div><div class="figure-viewer__top__left"><a href="#" role="button" class="figure-viewer__ctrl__browse"><span class="icon-allfigures"><span class="sr-only">Browse All Figures</span></span></a><a href="#" role="button" class="figure-viewer__ctrl__return is-hidden"><span class="icon-arrow-left"><span class="sr-only">Return to Figure</span></span></a><button title="zoom out" class="figure-viewer__label__zoom icon-zoom-out zoom-out"></button><span tabindex="1" class="zoomSlider js__zoom-slider ui-slider"><input class="zoom-range" value="50" type="range"></span><button title="zoom in" class="figure-viewer__label__zoom icon-zoom-in zoom-in"></button></div></div><div class="figure-viewer__reg__center" style="height: calc(-56px + 100vh);"><div class="figure-viewer__cent__left"><a href="#" role="button" class="figure-viewer__ctrl__prev"><span class="icon-arrow_l"><span class="sr-only">Previous Figure</span></span></a><a href="#" role="button" class="figure-viewer__ctrl__next"><span class="icon-arrow_r"><span class="sr-only">Next Figure</span></span></a><div class="figure-viewer__hold__fig"><figure class="holder"></figure></div><div class="figure-viewer__hold__list clearfix container"></div></div><div class="figure-viewer__cent__right"><div class="figure-viewer__title"><a title="Open/Close Caption" href="#" class="figure-viewer__ctrl__caption"><span class="icon-doublearrow"></span><span class="figure-viewer__caption__label">Caption</span></a><span class="figure-viewer__title__text"></span></div><div class="figure-viewer__hold__figcap"></div></div></div></div>
</div>
</div></div></div>

        </section>
    
        </div>
    </div><!--end pagefulltext-->
<div class="w-slide"><div class="w-slide_head"><a href="#" class="w-slide__back"><i class=" icon-arrow_l" aria-hidden="true"></i>back</a><span class="w-slide__title"></span></div><div class="w-slide__content"></div></div></main>

        </div>
    










        <div data-widget-def="pageFooter" data-widget-id="053994f3-7d80-4bb1-97a4-c67fbc4768cd" class="pageFooter">
        



        
        <footer class="page-footer">
    <div data-pb-dropzone="main">
    



        
        <div class="main-footer">



        
        <style>
    .footer-links {
        text-align: center;
    }
    .footer-links h3 {
        font-weight: 300;
    }
    .footer-links a {
        font-weight: 600;
    }
    @media (min-width: 992px) {
        .footer-links {
            text-align: left;
        }
    }
</style>










        <div data-widget-def="ux3-layout-widget" data-widget-id="2753bec1-a79d-4403-827b-6c788ddd6db3" class="footer-sections">
        



        
            <h2 class="sr-only">
                Additional links
            </h2>
        
        <div class="container"><div class="row"><div class=" col-md-3">



        
        <div class="footer-links mb-3">
  <h3>About Wiley Online Library</h3>
  <ul class="rlist">
    <li><a href="https://www.wiley.com/privacy">Privacy Policy</a></li>
    <li><a href="https://onlinelibrary.wiley.com/terms-and-conditions">Terms of Use</a></li>
    <li><a href="https://onlinelibrary.wiley.com/cookies">Cookies</a></li>
    <li><a href="https://onlinelibrary.wiley.com/accessibility">Accessibility</a></li>
  </ul>
</div>
</div><div class=" col-md-3">



        
         <div class="footer-links mb-3">
    <h3>Help &amp; Support</h3>
    <ul class="rlist">
      <li><a href="https://hub.wiley.com/community/support/onlinelibrary">Contact Us</a></li>
    </ul>
  </div>
</div><div class=" col-md-3">



        
         <div class="footer-links mb-3">
    <h3>Opportunities</h3>
    <ul class="rlist">
      <li><a href="https://onlinelibrary.wiley.com/agents">Subscription Agents</a></li>
      <li><a href="https://onlinelibrary.wiley.com/advertisers">Advertisers &amp; Corporate Partners</a></li>
    </ul>
  </div>
</div><div class=" col-md-3">



        
        <div class="footer-links mb-3">
    <h3>Connect with Wiley</h3>
    <ul class="rlist">
      <li><a href="https://hub.wiley.com/community/exchanges/">The Wiley Network</a></li>
      <li><a href="http://newsroom.wiley.com/">Wiley Press Room</a></li>
    </ul>
  </div>
</div></div></div>

        </div>
    










        <div data-widget-def="ux3-layout-widget" data-widget-id="86584e25-e4db-4118-9550-9b4f5341a4ac" class="copyright-container">
        



        
        <div class="container"><div class="row"><div class=" col-md-12">









        <div data-widget-def="UX3HTMLWidget" data-widget-id="bfbf6c5d-a650-4662-828f-5275fada9120" class="copyright">
        



        
        <div class="copyright pull-left">Copyright © 1999-2019 <a href="http://www.wiley.com/">John Wiley &amp; Sons, Inc</a>. All rights reserved</div>

        </div>
    




        
        <div class="footer-logo pull-right">
        
<a href="http://www.wiley.com/" title="Wiley Home Page" aria-label="Wiley"><img alt="Wiley Home Page" src="footer-logo-wiley-1510029248417.png"></a>

</div>
</div></div></div>

        </div>
    
</div>
    </div>
</footer>

        </div>
    




        
        <!-- [Tag{id&#x3D;320, code&#x3D;ls20, name&#x3D;Ecology &amp; Organismal Biology}, Tag{id&#x3D;1194, code&#x3D;ls2k, name&#x3D;Conservation Science}, Tag{id&#x3D;1195, code&#x3D;ls2e, name&#x3D;Macroecology}, Tag{id&#x3D;60, code&#x3D;ls, name&#x3D;LIFE SCIENCES}, Tag{id&#x3D;102280, code&#x3D;ecography_ecog, name&#x3D;ecography_ECOG}, Tag{id&#x3D;101738, code&#x3D;wly.ecology_000046, name&#x3D;wly.ecology_000046}] -->              <div id="iucr-failure"> <p><i class="icon-notice"></i>The full text of this article hosted at iucr.org is unavailable due to technical difficulties.<a href="#" class="close-iucr-failure"><i class="icon-x"></i></a> </p> </div> 




        
        
















<div class="popup login-popup hidden" role="dialog" tabindex="-1" aria-labelledby="loginPopupHead">



    <div class="content">
        <a href="#" class="close" role="button" aria-label="Cancel Login"><i class="icon-tools_close"></i></a>
        <div class="modal__header">
            <div class="header_logo">
    <img src="logo-header-1526603583437.png" alt="">
</div>
        </div>

        <div class="modal__body">
            <div class="ssoLogin">
                
            </div>

            <h2 id="loginPopupHead">Log in to Wiley Online Library</h2>

            































<div class="login-form">
    

    

    
    <form action="/action/doLogin?societyURLCode=" class="bordered" method="post"><input name="id" value="67065c09-4a88-49cd-934c-ac707951d35c" type="hidden">
        
            <input name="popup" value="true" type="hidden">
        
        

        
        <input name="loginUri" value="/doi/full/10.1111/j.0906-7590.2008.5203.x" type="hidden">
        
            




<input name="redirectUri" value="/doi/full/10.1111/j.0906-7590.2008.5203.x" type="hidden">
<input name="loginUri" value="/doi/full/10.1111/j.0906-7590.2008.5203.x" type="hidden">
            
        
        
        
                






















    
    
        
    <div class="input-group">
        <div class="label">

            
                    <label for="username">Email or Customer ID</label>
                
        </div>
        
        
                <input id="username" class="login" name="login" size="15" placeholder="Enter your email" autocorrect="off" spellcheck="false" autocapitalize="off" required="true" type="text">
            
        
                    
                        <div class="actions">
                            
                        </div>
                    
                
    </div>


                
                



















    
    <div class="input-group">
        <div class="label">
            <label for="password">Password</label>
        </div>
        
        
                <input id="password" class="password" name="password" value="" autocomplete="off" placeholder="Enter your password" autocorrect="off" spellcheck="false" autocapitalize="off" required="true" type="password">
            
        <span class="password-eye-icon icon-eye hidden"></span>
        
                
    </div>

                
                    
                        <div class="actions">
                            
                            <a href="https://onlinelibrary.wiley.com/action/requestResetPassword" class="link show-request-reset-password">
                                Forgot password?
                            </a>
                        </div>
                    
                
            
        
        <div class="loginExtraBeans-dropZone" data-pb-dropzone="loginExtraBeans">
            
        </div>

        
                <div class="align-end">
                    
                    
                            
                            <span class="submit " disabled="disabled">
                                <input class="button btn submit primary no-margin-bottom accessSubmit" name="submitButton" value="Log In" disabled="disabled" type="submit">
                            </span>
                        
                </div></form>
    <div class="societySpecificInstructions">
        
    </div>
</div>

            
                <div class="login-container clearfix form-btn">
                    <a href="https://onlinelibrary.wiley.com/action/registration" class="linkBtn float-left">
                        
                        NEW USER <span aria-hidden="true">&gt;</span>
                    </a>
                    
                        <a href="https://onlinelibrary.wiley.com/action/ssostart?redirectUri=%2Fdoi%2Ffull%2F10.1111%2Fj.0906-7590.2008.5203.x" class="linkBtn float-right instituation-login last-tab-item">
                            
                            INSTITUTIONAL LOGIN <span aria-hidden="true">&gt;</span>
                        </a><a class="fakeFocusoutElement" href="#"></a>
                    
                </div>
            

        </div>
    </div>
</div>




        
        






<div class="popup change-password-drawer hidden" role="dialog" tabindex="-1" aria-labelledby="changePasswordPopupHead">
    <div class="content">
        <a href="#" class="close cancel" role="button" aria-label="close dialog"><i class="icon-close_thin"></i></a>
        <div class="modal__header">
            <div class="header_logo">
    <img src="logo-header-1526603583437.png" alt="">
</div>

            <h2 id="changePasswordPopupHead">Change Password</h2>
        </div>
        <div class="modal__body">

            <div class="form">

                <form action="/action/changePassword" method="post"><div class="message error"></div>
                    <input name="submit" value="submit" type="hidden">
                    <div class="input-group">
                        <div class="label">
                            <label for="a589574e-bb98-4c6e-8fed-67365ff05357-old">Old Password</label>
                        </div>
                        <input id="a589574e-bb98-4c6e-8fed-67365ff05357-old" class="old" name="old" value="" autocomplete="off" type="password">
                        <span class="password-eye-icon icon-eye hidden"></span>
                    </div>
                    <div class="input-group">
                        <div class="label">
                            <label for="a589574e-bb98-4c6e-8fed-67365ff05357-new">New Password</label>
                        </div>
                        <input id="a589574e-bb98-4c6e-8fed-67365ff05357-new" class="pass-hint new" name="new" value="" autocomplete="off" type="password">
                        <span class="password-eye-icon icon-eye hidden"></span>
                        
                            







    


    


    


<div class="password-strength-indicator" data-min="10" data-max="32" data-strength="3">
    <span class="text too-short">Too Short</span>
    <span class="text weak">Weak</span>
    <span class="text medium">Medium</span>
    <span class="text strong">Strong</span>
    <span class="text very-strong">Very Strong</span>
    <span class="text too-long">Too Long</span>
</div>
                        
                    </div>
                    <input class="button primary submit" value="Submit" disabled="disabled" type="submit"></form>
                <a href="#" class="fakeFormLastElement last-tab-item"></a><a class="fakeFocusoutElement" href="#"></a>
            </div>
            <div class="success-template hidden">
                <h2>Password Changed Successfully</h2>
                <p>Your password has been changed</p>
            </div>
        </div>
    </div>
</div>





        
        





<div class="popup registration-popup hidden" role="dialog" tabindex="-1" aria-labelledby="createAccountPopupHead">



    <div class="content">
        <a href="#" class="close" role="button" aria-label="close dialog"><i class="icon-tools_close"></i></a>
        <div class="modal__header">
            <div class="logoDropZone" data-pb-dropzone="logoDropZone">
                
            </div>

            <h2 id="createAccountPopupHead">Create a new account</h2>
        </div>

        <div class="modal__body">
            <form action="/action/registration" class="registration-form" method="post"><input name="redirectUri" value="/doi/full/10.1111/j.0906-7590.2008.5203.x" type="hidden">
                <div class="input-group">
                    <div class="label">
                        <label for="4e647394-f751-4441-baa4-df426bca4b6e.email">Email or Customer ID</label>
                    </div>
                    <input id="4e647394-f751-4441-baa4-df426bca4b6e.email" class="email" name="email" size="15" type="text">
                </div>
                <div class="submit">
                    <input class="button submit primary" value="Register" disabled="disabled" type="submit">
                </div></form>
            <div class="center">
                <a href="https://onlinelibrary.wiley.com/action/showLogin" class="show-login last-tab-item">
                    
                    Returning user
                </a><a class="fakeFocusoutElement" href="#"></a>
            </div>
        </div>
    </div>
</div>




        
        



<div class="popup top-drawer request-reset-password-drawer hidden" role="dialog" tabindex="-1" aria-labelledby="resetPasswordPopupHead">
    <div class="content">
        <div class="modal__header">
            <a href="#" class="cancel close" role="button" aria-label="close dialog"><i class="icon-close_thin"></i></a>

            <div class="header_logo">
    <img src="logo-header-1526603583437.png" alt="">
</div>

            <h2 id="resetPasswordPopupHead">Forgot your password?</h2>

            <p class="sub">Enter your email address below. </p>

        </div>

        <div class="modal__body">

            









<div class="error"></div>


<form action="/action/requestResetPassword" class="request-reset-password-form" method="post"><input name="requestResetPassword" value="true" type="hidden">
    
            <div class="input-group">
                






















    
    
        
    <div class="input-group">
        <div class="label">

            
                    <label for="email">Email or Customer ID</label>
                
        </div>
        
        
                <input id="email" class="email" name="email" placeholder="Enter your email" autocorrect="off" spellcheck="false" autocapitalize="off" type="email">
            
        
    </div>

            </div>
        


    
            <div class="password-recaptcha-ajax"></div>
        

    <div class="form-btn">
        <input class="button btn primary submit" name="submit" value="RESET PASSWORD" disabled="disabled" type="submit">
    </div></form>
            

            <div class="message error"></div>

            <div class="success-template hidden">

                <p>"Please check your email for instructions on resetting your password.
                    If you do not receive an email within 10 minutes, your email address may not be registered,
                    and you may need to create a new Wiley Online Library account.</p>

            </div>
        </div>
    </div>
</div>





        
        <div><div class="pb-dropzone" data-pb-dropzone="col-0" title="col-0"></div></div>




        
        




<div class="popup top-drawer request-username-drawer username-popup hidden" role="dialog" tabindex="-1" aria-labelledby="requestUsernamePopupHead">
    <a href="#" class="close" role="button" aria-label="close dialog"><i class="icon-close_thin"></i></a>
    <div class="content">
        <h2 id="requestUsernamePopupHead">Request Username</h2>
        <div class="form">
            <p>Can't sign in? Forgot your username?</p>
            <p class="sub">Enter your email address below and we will send you your username</p>
            <div class="message error"></div>
            <form action="/action/requestUsername" method="post"><input name="requestUsername" value="requestUsername" type="hidden">
                <div class="input-group">
                    <div class="label">
                        <label for="ac834f24-aa07-4ad2-9d13-f77c843f21cb.email">Email or Customer ID</label>
                    </div>
                    <input id="ac834f24-aa07-4ad2-9d13-f77c843f21cb.email" class="email" name="email" size="15" type="text">
                </div>

                <div class="username-recaptcha-ajax">
                </div>

                <input class="button primary submit" name="submit" value="Submit" disabled="disabled" type="submit">
                <div class="center">
                    <a href="#" class="cancel">Close</a>
                </div></form>
        </div>
        <div class="success-template hidden note">
            <p>If the address matches an existing account you will receive an email with instructions to retrieve your username</p>
        </div>
    </div>
</div>




        
        <!-- Belongs at the bottom of the page -->

<script type="text/javascript">
if(window._satellite) { 
_satellite.pageBottom(); 
}
</script><script src="satellite-5dacec9364746d6c440013e7.js"></script>
        </div>
    </div>
<script src="build.js"></script><div style="visibility: hidden; height: 1px; width: 1px; position: absolute; top: -9999px; z-index: 100000;" id="_atssh"><iframe id="_atssh265" title="AddThis utility frame" style="height: 1px; width: 1px; position: absolute; top: 0px; z-index: 100000; border: 0px none; left: 0px;" src="https://s7.addthis.com/static/sh.f48a1a04fe8dbf021b4cda1d.html#rand=0.8040832068263212&amp;iit=1583492923075&amp;tmr=load%3D1583492922802%26core%3D1583492922817%26main%3D1583492923066%26ifr%3D1583492923079&amp;cb=0&amp;cdn=0&amp;md=0&amp;kw=&amp;ab=-&amp;dh=onlinelibrary.wiley.com&amp;dr=&amp;du=https%3A%2F%2Fonlinelibrary.wiley.com%2Fdoi%2Ffull%2F10.1111%2Fj.0906-7590.2008.5203.x&amp;href=https%3A%2F%2Fonlinelibrary.wiley.com%2Fdoi%2Ffull%2F10.1111%2Fj.0906-7590.2008.5203.x&amp;dt=Modeling%20of%20species%20distributions%20with%20Maxent%3A%20new%20extensions%20and%20a%20comprehensive%20evaluation&amp;dbg=0&amp;cap=tc%3D0%26ab%3D0&amp;inst=1&amp;jsl=1&amp;prod=undefined&amp;lng=en&amp;ogt=description%2Csite_name%2Cimage%2Curl%2Ctype%3DArticle%2Ctitle&amp;pc=men&amp;pub=xa-4faab26f2cff13a7&amp;ssl=1&amp;sid=5e622f3aa83f65b8&amp;srf=0.01&amp;ver=300&amp;xck=0&amp;xtr=0&amp;og=title%3DModeling%2520of%2520species%2520distributions%2520with%2520Maxent%253A%2520new%2520extensions%2520and%2520a%2520comprehensive%2520evaluation%26type%3DArticle%26url%3Dhttps%253A%252F%252Fonlinelibrary.wiley.com%252Fdoi%252F10.1111%252Fj.0906-7590.2008.5203.x%26image%3Dhttps%253A%252F%252Fonlinelibrary.wiley.com%252Fcms%252Fasset%252Fafcf8a96-c9a9-45b5-a202-6f8b773a350b%252Feco.2008.31.issue-2.cover.gif%26site_name%3DWiley%2520Online%2520Library%26description%3DAccurate%2520modeling%2520of%2520geographic%2520distributions%2520of%2520species%2520is%2520crucial%2520to%2520various%2520applications%2520in%2520ecology%2520and%2520conservation.%2520The%2520best%2520performing%2520techniques%2520often%2520require%2520some%2520parameter%2520tuning%252C%2520which%2520may%2520...&amp;csi=undefined&amp;rev=v8.28.3-wp&amp;ct=1&amp;xld=1&amp;xd=1"></iframe></div><style id="service-icons-0"></style><script src="main.js"></script><iframe id="google_osd_static_frame_1609065771995" name="google_osd_static_frame" style="display: none; width: 0px; height: 0px;"></iframe>
<script type="text/javascript" src="ihh4product.js"></script>
<script>var _prum=[['id','59e8fecb3847311aab7b23c6'],['mark','firstbyte',(new Date()).getTime()]];(function(){var s=document.getElementsByTagName('script')[0],p=document.createElement('script');p.async='async';p.src='//rum-static.pingdom.net/prum.min.js';s.parentNode.insertBefore(p,s);})();</script>
























    










    <script>
        $(document).ready(function () {
            var start = new Date();
            $(window).unload(function () {
                var end = new Date();
                $.ajax({
                    url: '/action/analytics',
                    method: 'post',
                    data: {
                        'Duration': end - start,
                        'DOI': digitalData.publication.item.doi,
                        'EventType': 'FullTextDuration'
                    },
                    async: false
                });
            });

           window.onafterprint = function () {
                $.ajax({
                    url: '/action/analytics',
                    method: 'post',
                    data: {
                        'DOI': digitalData.publication.item.doi,
                        'EventType': 'FullTextPrint'
                    }
                });
            };
        });
    </script>


<div id="crossmark-widget" style="display: none;"><div class="crossmark-reset crossmark-overlay"></div><div class="crossmark-reset crossmark-popup"><div class="crossmark-reset crossmark-popup__offset"><div class="crossmark-reset crossmark-popup__inner"><div class="crossmark-reset crossmark-popup__header"><button class="crossmark-reset crossmark-popup__btn-close"><span class="sr-only">Close crossmark popup</span></button></div><div class="crossmark-reset crossmark-popup__content-wrapper"><iframe title="Crossmark service - Intentionally blank" aria-hidden="true" class="crossmark-reset crossmark-popup__content"></iframe></div></div></div></div></div><ul class="ui-autocomplete ui-front ui-menu ui-widget ui-widget-content ui-corner-all quickSearchAutocomplete" id="ui-id-1" tabindex="0" style="display: none;"></ul><ul class="ui-autocomplete ui-front ui-menu ui-widget ui-widget-content ui-corner-all quickSearchAutocomplete" id="ui-id-2" tabindex="0" style="display: none;"></ul><script type="text/javascript" src="https://api.altmetric.com/v1/doi/10.1111/j.0906-7590.2008.5203.x?callback=_altmetric.embed_callback&amp;domain=onlinelibrary.wiley.com&amp;key=3c130976ca2b8f2e88f8377633751ba1&amp;cache_until=12-6"></script><div class="altmetric-embed altmetric-popover altmetric-left" data-uuid="1f394bd5-8b76-3dbf-f068-d6089ec8c5d2" id="_altmetric_popover_el_1f394bd5-8b76-3dbf-f068-d6089ec8c5d2" style="margin: 0px; display: none;"><div class="altmetric_container" id="_altmetric_container"><div class="altmetric_arrow altmetric-left"></div><div id="_altmetric_popover_inner" class="altmetric-embed altmetric-popover-inner altmetric-floating altmetric-left" data-uuid="1f394bd5-8b76-3dbf-f068-d6089ec8c5d2"><div class="altmetric-embed altmetric-popover-content altmetric-floating"><div>    <div style="padding-left: 10px; line-height:18px; border-left: 16px solid #ffd140;">
      <a class="link-to-altmetric-details-tab" target="_self" href="https://www.altmetric.com/details.php?domain=onlinelibrary.wiley.com&amp;citation_id=16592058&amp;tab=blogs">
          Blogged by <b>3</b>
      </a>
    </div>
    <div style="padding-left: 10px; line-height:18px; border-left: 16px solid #9f79f2;">
      <a class="link-to-altmetric-details-tab" target="_self" href="https://www.altmetric.com/details.php?domain=onlinelibrary.wiley.com&amp;citation_id=16592058&amp;tab=policy-documents">
          Referenced in <b>3</b> policy sources
      </a>
    </div>
    <div style="padding-left: 10px; line-height:18px; border-left: 16px solid #74CFED;">
      <a class="link-to-altmetric-details-tab" target="_self" href="https://www.altmetric.com/details.php?domain=onlinelibrary.wiley.com&amp;citation_id=16592058&amp;tab=twitter">
          Tweeted by <b>1</b>
      </a>
    </div>

    <div class="altmetric-embed readers" style="margin-top: 10px;">
          <div class="altmetric-embed tip_mendeley" style="padding-left: 10px; line-height:18px; border-left: 16px solid #A60000;">
              <b>3323</b> readers on Mendeley
          </div>
          <div class="altmetric-embed tip_citeulike" style="padding-left: 10px; line-height:18px; border-left: 16px solid #BCD2EF;">
              <b>2</b> readers on CiteULike
          </div>
    </div>
    <div style="margin-top: 10px; text-align: center;">
        <a class="altmetric_details" target="_self" href="https://www.altmetric.com/details.php?domain=onlinelibrary.wiley.com&amp;citation_id=16592058">
            See more details
        </a>
    </div>
</div></div></div></div></div></body><iframe src="https://injections.readcube.com/html/track_epdf.43d7bc64.html?action=Link&amp;publisher=wiley&amp;status=OK" style="position: absolute; visibility: hidden; width: 0px; height: 0px;"></iframe></html>