<!DOCTYPE html>
<html class="js" style="--font-family-sans: Europa; --font-family-serif: Noto Serif;" lang="en"><head>
<meta http-equiv="content-type" content="text/html; charset=UTF-8">
        <title>Phylogenomic methods outperform traditional multi-locus 
approaches in resolving deep evolutionary history: a case study of 
formicine ants | BMC Evolutionary Biology | Full Text</title>

        
            

    <meta name="citation_abstract" content="Ultraconserved elements (UCEs) have been successfully used in phylogenomics for a variety of taxa, but their power in phylogenetic inference has yet to be extensively compared with that of traditional Sanger sequencing data sets. Moreover, UCE data on invertebrates, including insects, are sparse. We compared the phylogenetic informativeness of 959 UCE loci with a multi-locus data set of ten nuclear markers obtained via Sanger sequencing, testing the ability of these two types of data to resolve and date the evolutionary history of the second most species-rich subfamily of ants in the world, the Formicinae. Phylogenetic analyses show that UCEs are superior in resolving ancient and shallow relationships in formicine ants, demonstrated by increased node support and a more resolved phylogeny. Phylogenetic informativeness metrics indicate a twofold improvement relative to the 10-gene data matrix generated from the identical set of taxa. We were able to significantly improve formicine classification based on our comprehensive UCE phylogeny. Our divergence age estimations, using both UCE and Sanger data, indicate that crown-group Formicinae are older (104–117&nbsp;Ma) than previously suggested. Biogeographic analyses infer that the diversification of the subfamily has occurred on all continents with no particular hub of cladogenesis. We found UCEs to be far superior to the multi-locus data set in estimating formicine relationships. The early history of the clade remains uncertain due to ancient rapid divergence events that are unresolvable even with our genomic-scale data, although this might be largely an effect of several problematic taxa subtended by long branches. Our comparison of divergence ages from both Sanger and UCE data demonstrates the effectiveness of UCEs for dating analyses. This comparative study highlights both the promise and limitations of UCEs for insect phylogenomics, and will prove useful to the growing number of evolutionary biologists considering the transition from Sanger to next-generation sequencing approaches.">

    <meta name="description" content="Ultraconserved elements (UCEs) have been successfully used in phylogenomics for a variety of taxa, but their power in phylogenetic inference has yet to be extensively compared with that of traditional Sanger sequencing data sets. Moreover, UCE data on invertebrates, including insects, are sparse. We compared the phylogenetic informativeness of 959 UCE loci with a multi-locus data set of ten nuclear markers obtained via Sanger sequencing, testing the ability of these two types of data to resolve and date the evolutionary history of the second most species-rich subfamily of ants in the world, the Formicinae. Phylogenetic analyses show that UCEs are superior in resolving ancient and shallow relationships in formicine ants, demonstrated by increased node support and a more resolved phylogeny. Phylogenetic informativeness metrics indicate a twofold improvement relative to the 10-gene data matrix generated from the identical set of taxa. We were able to significantly improve formicine classification based on our comprehensive UCE phylogeny. Our divergence age estimations, using both UCE and Sanger data, indicate that crown-group Formicinae are older (104–117&nbsp;Ma) than previously suggested. Biogeographic analyses infer that the diversification of the subfamily has occurred on all continents with no particular hub of cladogenesis. We found UCEs to be far superior to the multi-locus data set in estimating formicine relationships. The early history of the clade remains uncertain due to ancient rapid divergence events that are unresolvable even with our genomic-scale data, although this might be largely an effect of several problematic taxa subtended by long branches. Our comparison of divergence ages from both Sanger and UCE data demonstrates the effectiveness of UCEs for dating analyses. This comparative study highlights both the promise and limitations of UCEs for insect phylogenomics, and will prove useful to the growing number of evolutionary biologists considering the transition from Sanger to next-generation sequencing approaches.">

    <meta name="journal_id" content="12862">

    <meta name="dc.title" content="Phylogenomic methods outperform traditional multi-locus approaches in resolving deep evolutionary history: a case study of formicine ants">

    <meta name="dc.source" content="BMC Evolutionary Biology 2015 15:1">

    <meta name="dc.format" content="text/html">

    <meta name="dc.publisher" content="BioMed Central">

    <meta name="dc.date" content="2015-12-04">

    <meta name="dc.type" content="OriginalPaper">

    <meta name="dc.language" content="En">

    <meta name="dc.copyright" content="2015 Blaimer et al.">

    <meta name="dc.rightsAgent" content="reprints@biomedcentral.com">

    <meta name="dc.description" content="Ultraconserved elements (UCEs) have been successfully used in phylogenomics for a variety of taxa, but their power in phylogenetic inference has yet to be extensively compared with that of traditional Sanger sequencing data sets. Moreover, UCE data on invertebrates, including insects, are sparse. We compared the phylogenetic informativeness of 959 UCE loci with a multi-locus data set of ten nuclear markers obtained via Sanger sequencing, testing the ability of these two types of data to resolve and date the evolutionary history of the second most species-rich subfamily of ants in the world, the Formicinae. Phylogenetic analyses show that UCEs are superior in resolving ancient and shallow relationships in formicine ants, demonstrated by increased node support and a more resolved phylogeny. Phylogenetic informativeness metrics indicate a twofold improvement relative to the 10-gene data matrix generated from the identical set of taxa. We were able to significantly improve formicine classification based on our comprehensive UCE phylogeny. Our divergence age estimations, using both UCE and Sanger data, indicate that crown-group Formicinae are older (104–117&nbsp;Ma) than previously suggested. Biogeographic analyses infer that the diversification of the subfamily has occurred on all continents with no particular hub of cladogenesis. We found UCEs to be far superior to the multi-locus data set in estimating formicine relationships. The early history of the clade remains uncertain due to ancient rapid divergence events that are unresolvable even with our genomic-scale data, although this might be largely an effect of several problematic taxa subtended by long branches. Our comparison of divergence ages from both Sanger and UCE data demonstrates the effectiveness of UCEs for dating analyses. This comparative study highlights both the promise and limitations of UCEs for insect phylogenomics, and will prove useful to the growing number of evolutionary biologists considering the transition from Sanger to next-generation sequencing approaches.">

    <meta name="prism.issn" content="1471-2148">

    <meta name="prism.publicationName" content="BMC Evolutionary Biology">

    <meta name="prism.publicationDate" content="2015-12-04">

    <meta name="prism.volume" content="15">

    <meta name="prism.number" content="1">

    <meta name="prism.section" content="OriginalPaper">

    <meta name="prism.startingPage" content="271">

    <meta name="prism.copyright" content="2015 Blaimer et al.">

    <meta name="prism.rightsAgent" content="reprints@biomedcentral.com">

    <meta name="prism.url" content="https://bmcevolbiol.biomedcentral.com/articles/10.1186/s12862-015-0552-5">

    <meta name="prism.doi" content="doi:10.1186/s12862-015-0552-5">

    <meta name="citation_pdf_url" content="https://bmcevolbiol.biomedcentral.com/track/pdf/10.1186/s12862-015-0552-5">

    <meta name="citation_fulltext_html_url" content="https://bmcevolbiol.biomedcentral.com/articles/10.1186/s12862-015-0552-5">

    <meta name="citation_journal_title" content="BMC Evolutionary Biology">

    <meta name="citation_publisher" content="BioMed Central">

    <meta name="citation_issn" content="1471-2148">

    <meta name="citation_title" content="Phylogenomic methods outperform traditional multi-locus approaches in resolving deep evolutionary history: a case study of formicine ants">

    <meta name="citation_volume" content="15">

    <meta name="citation_issue" content="1">

    <meta name="citation_publication_date" content="2015/12">

    <meta name="citation_online_date" content="2015/12/04">

    <meta name="citation_firstpage" content="271">

    <meta name="citation_article_type" content="Research article">

    <meta name="citation_fulltext_world_readable" content="">

    <meta name="dc.identifier" content="doi:10.1186/s12862-015-0552-5">

    <meta name="DOI" content="10.1186/s12862-015-0552-5">

    <meta name="citation_doi" content="10.1186/s12862-015-0552-5">

    <meta name="dc.creator" content="Bonnie B. Blaimer">

    <meta name="dc.creator" content="Seán G. Brady">

    <meta name="dc.creator" content="Ted R. Schultz">

    <meta name="dc.creator" content="Michael W. Lloyd">

    <meta name="dc.creator" content="Brian L. Fisher">

    <meta name="dc.creator" content="Philip S. Ward">

    <meta name="dc.subject" content="Evolutionary Biology">

    <meta name="dc.subject" content="Animal Systematics/Taxonomy/Biogeography">

    <meta name="dc.subject" content="Entomology">

    <meta name="dc.subject" content="Genetics and Population Dynamics">

    <meta name="dc.subject" content="Life Sciences, general">

    <meta name="citation_reference" content="citation_journal_title=Gen Res; citation_title=Ultraconserved elements are novel phylogenomic markers that resolve placental mammal phylogeny when combined with species-tree analysis; citation_author=JE McCormack, BC Faircloth, NG Crawford, PA Gowaty, RT Brumfield, TC Glenn; citation_volume=22; citation_issue=4; citation_publication_date=2012; citation_pages=746-754; citation_doi=10.1101/gr.125864.111; citation_id=CR1">

    <meta name="citation_reference" content="citation_journal_title=Annu Rev Ecol Syst; citation_title=High-throughput genomic data in systematics and phylogenetics; citation_author=EM Lemmon, AR Lemmon; citation_volume=44; citation_publication_date=2013; citation_pages=99-121; citation_doi=10.1146/annurev-ecolsys-110512-135822; citation_id=CR2">

    <meta name="citation_reference" content="citation_journal_title=Syst Biol; citation_title=Target capture and massively parallel sequencing of ultraconserved elements for comparative studies at shallow evolutionary time scales; citation_author=BT Smith, MG Harvey, BC Faircloth, TC Glenn, RT Brumfield; citation_volume=63; citation_publication_date=2013; citation_pages=83-95; citation_doi=10.1093/sysbio/syt061; citation_id=CR3">

    <meta name="citation_reference" content="citation_journal_title=Nature; citation_title=The drivers of tropical speciation; citation_author=BT Smith, JE McCormack, AM Cuervo, MJ Hickerson, A Aleixo, CD Cadena, J Perez-Eman, CW Burney, X Xie, MG Harvey; citation_volume=515; citation_publication_date=2014; citation_pages=406-409; citation_doi=10.1038/nature13687; citation_id=CR4">

    <meta name="citation_reference" content="citation_journal_title=Biol Lett; citation_title=More than 1000 ultraconserved elements provide evidence that turtles are the sister group of archosaurs; citation_author=NG Crawford, BC Faircloth, JE McCormack, RT Brumfield, K Winker, TC Glenn; citation_volume=8; citation_publication_date=2012; citation_pages=783-786; citation_doi=10.1098/rsbl.2012.0331; citation_id=CR5">

    <meta name="citation_reference" content="citation_journal_title=Syst Biol; citation_title=Ultraconserved elements anchor thousands of genetic markers spanning multiple evolutionary timescales; citation_author=BC Faircloth, JE McCormack, NG Crawford, MG Harvey, RT Brumfield, TC Glenn; citation_volume=61; citation_publication_date=2012; citation_pages=717-726; citation_doi=10.1093/sysbio/sys004; citation_id=CR6">

    <meta name="citation_reference" content="citation_journal_title=Mol Ecol Res; citation_title=Target enrichment of ultraconserved elements from arthropods provides a genomic perspective on relationships among Hymenoptera; citation_author=BC Faircloth, MG Branstetter, ND White, SG Brady; citation_volume=15; citation_publication_date=2015; citation_pages=489-501; citation_doi=10.1111/1755-0998.12328; citation_id=CR7">

    <meta name="citation_reference" content="citation_journal_title=Mol Phylogenet Evol; citation_title=Genome-wide ultraconserved elements exhibit higher phylogenetic informativeness than traditional gene markers in percomorph fishes; citation_author=PS Gilbert, J Chang, C Pan, EM Sobel, JS Sinsheimer, BC Faircloth, ME Alfaro; citation_volume=92; citation_publication_date=2015; citation_pages=140-146; citation_doi=10.1016/j.ympev.2015.05.027; citation_id=CR8">

    <meta name="citation_reference" content="citation_journal_title=Proc Natl Acad Sci; citation_title=Evaluating alternative hypotheses for the early evolution and diversification of ants; citation_author=SG Brady, TR Schultz, BL Fisher, PS Ward; citation_volume=103; citation_publication_date=2006; citation_pages=18172-18177; citation_doi=10.1073/pnas.0605858103; citation_id=CR9">

    <meta name="citation_reference" content="citation_journal_title=Science; citation_title=Phylogeny of the ants: Diversification in the age of angiosperms; citation_author=CS Moreau, CD Bell, R Vila, SB Archibald, NE Pierce; citation_volume=312; citation_publication_date=2006; citation_pages=101-104; citation_doi=10.1126/science.1124891; citation_id=CR10">

    <meta name="citation_reference" content="citation_journal_title=Syst Biol; citation_title=Phylogeny and biogeography of dolichoderine ants: effects of data partitioning and relict taxa on historical inference; citation_author=PS Ward, SG Brady, BL Fisher, TR Schultz; citation_volume=59; citation_publication_date=2010; citation_pages=342-362; citation_doi=10.1093/sysbio/syq012; citation_id=CR11">

    <meta name="citation_reference" content="citation_journal_title=BMC Evol Biol; citation_title=The rise of army ants and their relatives: diversification of specialized predatory doryline ants; citation_author=SG Brady, BL Fisher, TR Schultz, PS Ward; citation_volume=14; citation_publication_date=2014; citation_pages=93; citation_doi=10.1186/1471-2148-14-93; citation_id=CR12">

    <meta name="citation_reference" content="citation_journal_title=Zootaxa; citation_title=The higher classification of the ant subfamily Ponerinae (Hymenoptera: Formicidae), with a review of ponerine ecology and behavior; citation_author=CA Schmidt, SO Shattuck; citation_volume=3817; citation_publication_date=2014; citation_pages=1-242; citation_doi=10.11646/zootaxa.3817.1.1; citation_id=CR13">

    <meta name="citation_reference" content="citation_journal_title=Syst Ent; citation_title=The evolution of myrmicine ants: phylogeny and biogeography of a hyperdiverse ant clade (Hymenoptera: Formicidae); citation_author=PS Ward, SG Brady, BL Fisher, TR Schultz; citation_volume=40; citation_publication_date=2015; citation_pages=61-81; citation_doi=10.1111/syen.12090; citation_id=CR14">

    <meta name="citation_reference" content="citation_journal_title=PLoS ONE; citation_title=Tracing the rise of ants-out of the ground; citation_author=A Lucky, MD Trautwein, BS Guenard, MD Weiser, RR Dunn; citation_volume=8; citation_issue=12; citation_publication_date=2013; citation_pages=e84012; citation_doi=10.1371/journal.pone.0084012; citation_id=CR15">

    <meta name="citation_reference" content="citation_journal_title=Evolution; citation_title=Testing the museum versus cradle tropical biological diversity hypothesis: phylogeny, diversification, and ancestral biogeographic range evolution of the ants; citation_author=CS Moreau, CD Bell; citation_volume=67; citation_publication_date=2013; citation_pages=2240-2257; citation_doi=10.1111/evo.12105; citation_id=CR16">

    <meta name="citation_reference" content="citation_journal_title=Ecography; citation_title=Functional and phylogenetic approaches reveal the evolution of diversity in a hyper diverse biota; citation_author=BB Blaimer, SG Brady, TR Schultz, BL Fisher; citation_volume=38; citation_publication_date=2015; citation_pages=901-912; citation_doi=10.1111/ecog.01370; citation_id=CR17">

    <meta name="citation_reference" content="citation_journal_title=Proc Natl Acad Sci; citation_title=Major evolutionary transitions in ant agriculture; citation_author=TR Schultz, SG Brady; citation_volume=105; citation_publication_date=2008; citation_pages=5435-5440; citation_doi=10.1073/pnas.0711024105; citation_id=CR18">

    <meta name="citation_reference" content="citation_journal_title=J Evol Biol; citation_title=Renewed diversification is associated with new ecological opportunity in the Neotropical turtle ants; citation_author=SL Price, S Powell, DJC Kronauer, LAP Tran, NE Pierce, RK Wayne; citation_volume=27; citation_publication_date=2014; citation_pages=242-258; citation_doi=10.1111/jeb.12300; citation_id=CR19">

    <meta name="citation_reference" content="citation_journal_title=Annu Rev Ecol Syst; citation_title=The phylogeny and evolution of ants; citation_author=PS Ward; citation_volume=45; citation_publication_date=2014; citation_pages=23-43; citation_doi=10.1146/annurev-ecolsys-120213-091824; citation_id=CR20">

    <meta name="citation_reference" content="Schmidt JO. Chemistry, pharmacology, and chemical ecology of ant venoms. In: Piek T, editor. Venoms of the Hymenoptera. London: Academic Press; 1986: 425–508">

    <meta name="citation_reference" content="citation_journal_title=Proc Natl Acad Sci; citation_title=Formicine ants: An arthropod source for the pumiliotoxin alkaloids of dendrobatid poison frogs; citation_author=RA Saporito, HM Garraffo, MA Donnelly, AL Edwards, JT Longino, JW Daly; citation_volume=101; citation_publication_date=2004; citation_pages=8045-8050; citation_doi=10.1073/pnas.0402365101; citation_id=CR22">

    <meta name="citation_reference" content="citation_journal_title=Myrmecol News; citation_title=Social parasitism among ants: a review (Hymenoptera: Formicidae); citation_author=A Buschinger; citation_volume=12; citation_publication_date=2009; citation_pages=219-235; citation_id=CR23">

    <meta name="citation_reference" content="citation_journal_title=Syst Ent; citation_title=Phylogeny and taxonomy of the Prenolepis genus-group of ants (Hymenoptera: Formicidae); citation_author=JS LaPolla, SG Brady, SO Shattuck; citation_volume=35; citation_publication_date=2010; citation_pages=118-131; citation_doi=10.1111/j.1365-3113.2009.00492.x; citation_id=CR24">

    <meta name="citation_reference" content="citation_journal_title=Syst Ent; citation_title=A new ant genus from the Greater Antilles and Central America, Zatania (Hymenoptera: Formicidae), exemplifies the utility of male and molecular character systems; citation_author=JS Lapolla, RJ Kallal, SG Brady; citation_volume=37; citation_publication_date=2012; citation_pages=200-214; citation_doi=10.1111/j.1365-3113.2011.00605.x; citation_id=CR25">

    <meta name="citation_reference" content="Blumenstiel B, Cibulskis K, Fisher S, DeFelice M, Barry A, Fennell T, et al. Targeted exon sequencing by in-solution hybrid selection. Curr Protoc Hum Genet. 2010;Unit 18.4.1–18.4.24, Supplement 66.">

    <meta name="citation_reference" content="citation_journal_title=Syst Ent; citation_title=The ant subfamily Pseudomyrmecinae (Hymenoptera: Formicidae): phylogeny and evolution of big-eyed arboreal ants; citation_author=PS Ward, DA Downie; citation_volume=30; citation_publication_date=2005; citation_pages=310-335; citation_doi=10.1111/j.1365-3113.2004.00281.x; citation_id=CR27">

    <meta name="citation_reference" content="citation_journal_title=Myrmecol News; citation_title=Molecular and morphological evidence for three sympatric species of Leptanilla (Hymenoptera: Formicidae) on the Greek island of Rhodes; citation_author=PS Ward, TP Sumnicht; citation_volume=17; citation_publication_date=2012; citation_pages=5-11; citation_id=CR28">

    <meta name="citation_reference" content="Katoh K, Asimenos G, Toh H. Multiple alignment of DNA sequences with MAFFT. In: Bioinformatics for DNA sequence analysis. Humana Press, New York City; 2009: 39–64">

    <meta name="citation_reference" content="Faircloth B. Illumiprocessor: a trimmomatic wrapper for parallel adapter and quality trimming. 2013. doi: 
                    10.6079/J9ILL
                    
                  
                        ">

    <meta name="citation_reference" content="Bolger AM, Lohse M, Usadel B. Trimmomatic: a flexible trimmer for Illumina sequence data. Bioinformatics 2014; btu170:1–7.">

    <meta name="citation_reference" content="Faircloth B. PHYLUCE is a software package for the analysis of conserved genomic loci. 2015. doi:
                    10.6079/J9PHYL
                    
                  
                        ">

    <meta name="citation_reference" content="citation_journal_title=Nat Biotech; citation_title=Full-length transcriptome assembly from RNA-Seq data without a reference genome; citation_author=MG Grabherr, BJ Haas, M Yassour, JZ Levin, DA Thompson, I Amit, X Adiconis, L Fan, R Raychowdhury, Q Zeng; citation_volume=29; citation_publication_date=2011; citation_pages=644-652; citation_doi=10.1038/nbt.1883; citation_id=CR33">

    <meta name="citation_reference" content="citation_journal_title=Mol Biol Evol; citation_title=Selection of conserved blocks from multiple alignments for their use in phylogenetic analysis; citation_author=J Castresana; citation_volume=17; citation_publication_date=2000; citation_pages=540-552; citation_doi=10.1093/oxfordjournals.molbev.a026334; citation_id=CR34">

    <meta name="citation_reference" content="citation_journal_title=Mol Biol Evol; citation_title=PartitionFinder: Combined selection of partitioning schemes and substitution models for phylogenetic analyses; citation_author=R Lanfear, B Calcott, SYW Ho, S Guindon; citation_volume=29; citation_publication_date=2012; citation_pages=1695-1701; citation_doi=10.1093/molbev/mss020; citation_id=CR35">

    <meta name="citation_reference" content="citation_journal_title=Bioinformatics; citation_title=RAxML-VI-HPC: maximum likelihood-based phylogenetic analyses with thousands of taxa and mixed models; citation_author=A Stamatakis; citation_volume=22; citation_publication_date=2006; citation_pages=2688-2690; citation_doi=10.1093/bioinformatics/btl446; citation_id=CR36">

    <meta name="citation_reference" content="Zwickl DJ. Genetic algorithm approaches for the phylogenetic analysis of large biological sequence datasets under the maximum likelihood criterion. The University of Texas at Austin; 2006; accessible at: 
                    https://repositories.lib.utexas.edu/handle/2152/2666
                    
                  
                        ">

    <meta name="citation_reference" content="citation_journal_title=Syst Biol; citation_title=MrBayes 3.2: efficient Bayesian phylogenetic inference and model choice across a large model space; citation_author=F Ronquist, M Teslenko, P Mark, DL Ayres, A Darling, S Höhna, B Larget, L Liu, MA Suchard, JP Huelsenbeck; citation_volume=61; citation_publication_date=2012; citation_pages=539-542; citation_doi=10.1093/sysbio/sys029; citation_id=CR38">

    <meta name="citation_reference" content="citation_journal_title=BMC Evol Biol; citation_title=Automatic selection of partitioning schemes for phylogenetic analyses using iterative k-means clustering of site rates; citation_author=PB Frandsen, B Calcott, C Mayer, R Lanfear; citation_volume=15; citation_publication_date=2015; citation_pages=13; citation_doi=10.1186/s12862-015-0283-7; citation_id=CR39">

    <meta name="citation_reference" content="citation_journal_title=Syst Biol; citation_title=Profiling Phylogenetic Informativeness; citation_author=JP Townsend; citation_volume=56; citation_publication_date=2007; citation_pages=222-231; citation_doi=10.1080/10635150701311362; citation_id=CR40">

    <meta name="citation_reference" content="Faircloth BC, Chang J, Alfaro ME. TAPIR enables high-throughput estimation and comparison of phylogenetic informativeness using locus-specific substitution models. 2012. arXiv preprint arXiv:12021215.">

    <meta name="citation_reference" content="citation_journal_title=BMC Evol Biol; citation_title=PhyDesign: an online application for profiling phylogenetic informativeness; citation_author=F López-Giráldez, JP Townsend; citation_volume=11; citation_publication_date=2011; citation_pages=152; citation_doi=10.1186/1471-2148-11-152; citation_id=CR42">

    <meta name="citation_reference" content="citation_journal_title=Mol Biol Evol; citation_title=Bayesian Phylogenetics with BEAUti and the BEAST 1.7; citation_author=AJ Drummond, MA Suchard, D Xie, A Rambaut; citation_volume=29; citation_publication_date=2012; citation_pages=1969-1973; citation_doi=10.1093/molbev/mss075; citation_id=CR43">

    <meta name="citation_reference" content="citation_journal_title=Syst Biol; citation_title=Maximum likelihood inference of geographic range evolution by dispersal, local extinction, and cladogenesis; citation_author=RH Ree, SA Smith; citation_volume=57; citation_publication_date=2008; citation_pages=4-14; citation_doi=10.1080/10635150701883881; citation_id=CR44">

    <meta name="citation_reference" content="citation_journal_title=BMC Evol Biol; citation_title=A Southern Hemisphere origin for campanulid angiosperms, with traces of the break-up of Gondwana; citation_author=JM Beaulieu, DC Tank, MJ Donoghue; citation_volume=13; citation_publication_date=2013; citation_pages=80; citation_doi=10.1186/1471-2148-13-80; citation_id=CR45">

    <meta name="citation_reference" content="citation_journal_title=Mol Phylogen Evol; citation_title=RASP (Reconstruct Ancestral State in Phylogenies): a tool for historical biogeography; citation_author=Y Yu, AJ Harris, C Blair, X He; citation_volume=87; citation_publication_date=2015; citation_pages=46-49; citation_doi=10.1016/j.ympev.2015.03.008; citation_id=CR46">

    <meta name="citation_reference" content="citation_journal_title=Syst Biol; citation_title=Pruning rogue taxa improves phylogenetic accuracy: an efficient algorithm and webservice; citation_author=AJ Aberer, D Krompass, A Stamatakis; citation_volume=62; citation_publication_date=2013; citation_pages=162-166; citation_doi=10.1093/sysbio/sys078; citation_id=CR47">

    <meta name="citation_reference" content="citation_journal_title=PLoS ONE; citation_title=Comparison of ultra-conserved elements in Drosophilids and Vertebrates; citation_author=IV Makunin, VV Shloma, SJ Stephen, M Pheasant, SN Belyakin; citation_volume=8; citation_publication_date=2013; citation_pages=e82362; citation_doi=10.1371/journal.pone.0082362; citation_id=CR48">

    <meta name="citation_reference" content="citation_journal_title=Mem Am Entomol Inst; citation_title=Synopsis and classification of Formicidae; citation_author=B Bolton; citation_volume=71; citation_publication_date=2003; citation_pages=1-370; citation_id=CR49">

    <meta name="citation_reference" content="citation_journal_title=Ann Rev Ent; citation_title=Ancient rapid radiations of insects: challenges for phylogenetic analysis; citation_author=JB Whitfield, KM Kjer; citation_volume=53; citation_publication_date=2008; citation_pages=449-472; citation_doi=10.1146/annurev.ento.53.103106.093304; citation_id=CR50">

    <meta name="citation_reference" content="citation_journal_title=Proc Natl Acad Sci; citation_title=Episodic radiations in the fly tree of life; citation_author=BM Wiegmann, MD Trautwein, IS Winkler, NB Barr, J-W Kim, C Lambkin, MA Bertone, BK Cassel, KM Bayless, AM Heimberg; citation_volume=108; citation_publication_date=2011; citation_pages=5690-5695; citation_doi=10.1073/pnas.1012675108; citation_id=CR51">

    <meta name="citation_reference" content="Bazinet AL, Cummings MP, Mitter KT, Mitter CW. Can RNA-Seq resolve the rapid radiation of advanced moths and butterflies (Hexapoda: Lepidoptera: Apoditrysia)? An exploratory study. PLoS ONE. 2013; 8(12):e82615.">

    <meta name="citation_reference" content="citation_journal_title=Curr Biol; citation_title=Phylogenomics resolves evolutionary relationships among ants, bees, and wasps; citation_author=BR Johnson, ML Borowiec, JC Chiu, EK Lee, J Atallah, PS Ward; citation_volume=23; citation_publication_date=2013; citation_pages=2058-2062; citation_doi=10.1016/j.cub.2013.08.050; citation_id=CR53">

    <meta name="citation_reference" content="citation_title=Systematics of the ant genus Camponotus (Hymenoptera: Formicidae): a preliminary analysis using data from the mitochondrial gene cytochrome oxidase I; citation_inbook_title=Hymenoptera: evolution, biodiversity and biological control; citation_publication_date=2000; citation_id=CR54; citation_author=SG Brady; citation_author=J Gadau; citation_author=PS Ward; citation_publisher=Fourth International Hymenoptera Conference">

    <meta name="citation_reference" content="citation_journal_title=Sociobiology; citation_title=Molecular phylogeny of the ant subfamily Formicinae (Hymenoptera, Formicidae) from China based on mitochondrial genes; citation_author=Z Chen, S Zhou, D Ye, Y Chen, C Lu; citation_volume=60; citation_publication_date=2013; citation_pages=135-144; citation_doi=10.13102/sociobiology.v60i2.135-144; citation_id=CR55">

    <meta name="citation_reference" content="citation_journal_title=PeerJ; citation_title=Genome evolution in an ancient bacteria-ant symbiosis: parallel gene loss among Blochmannia spanning the origin of the ant tribe Camponotini; citation_author=LE Williams, JJ Wernegreen; citation_volume=3; citation_publication_date=2015; citation_pages=e881; citation_doi=10.7717/peerj.881; citation_id=CR56">

    <meta name="citation_reference" content="citation_journal_title=BMC Evol Biol; citation_title=One nutritional symbiosis begat another: phylogenetic evidence that the ant tribe Camponotini acquired Blochmannia by tending sap-feeding insects; citation_author=JJ Wernegreen, SN Kauppinen, SG Brady, PS Ward; citation_volume=9; citation_publication_date=2009; citation_pages=292; citation_doi=10.1186/1471-2148-9-292; citation_id=CR57">

    <meta name="citation_reference" content="citation_journal_title=Philos Trans R Soc Lond B Biol Sci; citation_title=Assessing the quality of molecular divergence time estimates by fossil calibrations and fossil-based model selection; citation_author=TJ Near, MJ Sanderson; citation_volume=359; citation_publication_date=2004; citation_pages=1477-1483; citation_doi=10.1098/rstb.2004.1523; citation_id=CR58">

    <meta name="citation_reference" content="citation_journal_title=Syst Biol; citation_title=Accounting for calibration uncertainty in phylogenetic estimation of evolutionary divergence times; citation_author=SYW Ho, MJ Phillips; citation_volume=58; citation_publication_date=2009; citation_pages=367-380; citation_doi=10.1093/sysbio/syp035; citation_id=CR59">

    <meta name="citation_reference" content="citation_journal_title=Am Ent; citation_title=Effects of fossil calibration-uncertainty on divergence dating in ants and bees; citation_author=SG Brady; citation_volume=57; citation_publication_date=2011; citation_pages=56; citation_doi=10.1093/ae/57.1.56; citation_id=CR60">

    <meta name="citation_author" content="Bonnie B. Blaimer">

    <meta name="citation_author_institution" content="Department of Entomology, National Museum of Natural History, Smithsonian Institution, Washington, USA">

    <meta name="citation_author" content="Seán G. Brady">

    <meta name="citation_author_institution" content="Department of Entomology, National Museum of Natural History, Smithsonian Institution, Washington, USA">

    <meta name="citation_author" content="Ted R. Schultz">

    <meta name="citation_author_institution" content="Department of Entomology, National Museum of Natural History, Smithsonian Institution, Washington, USA">

    <meta name="citation_author" content="Michael W. Lloyd">

    <meta name="citation_author_institution" content="Department of Entomology, National Museum of Natural History, Smithsonian Institution, Washington, USA">

    <meta name="citation_author" content="Brian L. Fisher">

    <meta name="citation_author_institution" content="Department of Entomology, California Academy of Sciences, San Francisco, USA">

    <meta name="citation_author" content="Philip S. Ward">

    <meta name="citation_author_institution" content="Department of Entomology and Nematology, University of California-Davis, Davis, USA">


        

        <meta charset="UTF-8">
        <meta http-equiv="X-UA-Compatible" content="IE=edge">
        <meta name="viewport" content="width=device-width, initial-scale=1">
        <!-- For IE 9 and below -->
<!--[if IE]><link rel="shortcut icon" href=/static/images/favicons//favicon.ico><![endif]-->
<!-- Touch Icons - iOS and Android 2.1+ -->
<link rel="apple-touch-icon" href="https://bmcevolbiol.biomedcentral.com/static/images/favicons/bmc/favicon-3f5c56eaaa.png">
<!-- Firefox, Chrome, Safari, IE 11+ and Opera -->
<link rel="icon" href="https://bmcevolbiol.biomedcentral.com/static/images/favicons/bmc/favicon-3f5c56eaaa.png">


        <script type="text/javascript" src="https://consumer.krxd.net/consent/set/bd339c69-af54-4a21-b4f1-654bcfcd83ca?idt=device&amp;dt=kxcookie&amp;dc=1&amp;al=1&amp;tg=1&amp;cd=0&amp;sh=0&amp;re=0&amp;callback=Krux.ns.macmillan.kxjsonp_consent_set_1"></script><script type="text/javascript" src="https://consumer.krxd.net/consent/get/bd339c69-af54-4a21-b4f1-654bcfcd83ca?idt=device&amp;dt=kxcookie&amp;callback=Krux.ns.macmillan.kxjsonp_consent_get_0"></script><script async="" src="controltag.js"></script><script type="text/javascript" async="" src="analytics.js"></script><script type="text/javascript" async="" src="KDqylSLE.js"></script><script type="text/javascript" async="" src="MathJax.js"></script><script async="" src="gtm.js"></script><script>(function(H){H.className=H.className.replace(/\bno-js\b/,'js')})(document.documentElement)</script>
        <script>(function(d){if (sessionStorage.fontsLoaded){d.style.setProperty('--font-family-sans', 'Europa');d.style.setProperty('--font-family-serif', 'Noto Serif')}})(document.documentElement);</script>
        <link rel="stylesheet" media="screen" href="basic-b69f4da053.css">
<link rel="stylesheet" media="print" href="print-7866383638.css">

<link rel="stylesheet" class="js-ctm" href="main-7c1188aaa2.css" media="only screen and (-webkit-min-device-pixel-ratio:0) and (min-color-index:0), (-ms-high-contrast: none), only all and (min--moz-device-pixel-ratio:0) and (min-resolution: 3e1dpcm)">

        <script src="polyfill.js" defer="defer"></script>
        <script src="app-4d954207d9.js" defer="defer"></script>
        <script>
            window.airbrakeErrors = [];
            window.onerror = function(error) {
                window.airbrakeErrors.push(error);
            }
        </script>
        
    <script type="text/javascript">
        config = {
            env: 'live',
            site: 'bmcevolbiol.biomedcentral.com',
            siteWithPath: 'bmcevolbiol.biomedcentral.com' + window.location.pathname,
            twitterHashtag: 'bmcevolbiol',
            cmsPrefix: 'https://studio-cms.springernature.com/studio/',
            
            doi: '10.1186/s12862-015-0552-5',
            figshareScriptUrl: 'https://widgets.figshare.com/static/figshare.js',
            publisherBrand: 'BioMed Central'
        };
    </script>

        
            
    <script type="text/javascript">
        dataLayer = [{"content":{"article":{"doi":"10.1186/s12862-015-0552-5","articleType":"Research article","peerReviewType":"Closed","supplement":null},"contentInfo":{"imprint":"BioMed Central","title":"Phylogenomic methods outperform traditional multi-locus approaches in resolving deep evolutionary history: a case study of formicine ants","publishedAt":1449187200000,"publishedAtDate":"2015-12-04","author":["Bonnie B. Blaimer","Seán G. Brady","Ted R. Schultz","Michael W. Lloyd","Brian L. Fisher","Philip S. Ward"],"collection":[]},"attributes":{"deliveryPlatform":"oscar","template":"rebrand","cms":null,"copyright":{"creativeCommonsType":"CC BY + CC0","openAccess":true},"environment":"live"},"journal":{"siteKey":"bmcevolbiol.biomedcentral.com","volume":"15","issue":"1","title":"BMC Evolutionary Biology","type":"SERIES","journalID":12862,"gaCode":"UA-62065396-10","section":[{"sectionId":"AC_afc6783aebe4f218a9c2852d53878997","sectionName":"Phylogenetics and phylogeography"}]},"category":{"pmc":{"primarySubject":"Life Sciences"},"contentType":"Research article","publishingSegment":"BMC-series Biology"}},"session":{"authentication":{"authenticationID":[]}},"version":"1.0.0","page":{"category":{"pageType":"article"}}}]

    </script>

        
        <script src="7d6f4b4f-77fb-43b9-b429-eb1e1eeac6aa.js" id="onetrust-bmc-control" charset="UTF-8"></script><script type="text/javascript" src="jquery_002.js"></script><link type="text/css" href="optanon.css" rel="stylesheet"><style>#optanon ul#optanon-menu li { background-color: #F8F8F8 !important }#optanon ul#optanon-menu li.menu-item-selected { background-color: #C5F6F9 !important }#optanon #optanon-popup-wrapper .optanon-white-button-middle { background-color: #1B3051 !important }.optanon-alert-box-wrapper .optanon-alert-box-button-middle { background-color: #1B3051 !important; border-color: #1B3051 !important; }#optanon #optanon-popup-wrapper .optanon-white-button-middle a { color: #ffffff !important }.optanon-alert-box-wrapper .optanon-alert-box-button-middle a { color: #ffffff !important }#optanon #optanon-popup-bottom { background-color: #FFFFFF !important }#optanon.modern #optanon-popup-top, #optanon.modern #optanon-popup-body-left-shading { background-color: #FFFFFF !important }.optanon-alert-box-wrapper { background-color:#EFF6FC !important }.optanon-alert-box-wrapper .optanon-alert-box-bg p { color:#333333 !important }</style>
        
    
        
    


        
    
        
            
                <!-- Google Tag Manager -->
                <script>
                    (function (w, d, s, l, i) {
                    w[l] = w[l] || [];
                    w[l].push({'gtm.start': new Date().getTime(), event: 'gtm.js'});
                    var f = d.getElementsByTagName(s)[0],
                            j = d.createElement(s),
                            dl = l != 'dataLayer' ? '&l=' + l : '';
                    j.async = true;
                    j.src = 'https://www.googletagmanager.com/gtm.js?id=' + i + dl;
                    f.parentNode.insertBefore(j, f);
                })(window, document, 'script', 'dataLayer', 'GTM-TDGJHK');
                </script>
                <!-- End Google Tag Manager -->
            
        
    


        
    
    <link rel="canonical" href="https://bmcevolbiol.biomedcentral.com/articles/10.1186/s12862-015-0552-5">
    


        

        
            
        <meta property="og:url" content="https://bmcevolbiol.biomedcentral.com/articles/10.1186/s12862-015-0552-5">
        <meta property="og:type" content="article">
        <meta property="og:site_name" content="BMC Evolutionary Biology">
        <meta property="og:title" content="Phylogenomic methods outperform traditional multi-locus approaches in resolving deep evolutionary history: a case study of formicine ants">
        <meta property="og:description" content="Ultraconserved elements (UCEs) have been successfully used in phylogenomics for a variety of taxa, but their power in phylogenetic inference has yet to be extensively compared with that of traditional Sanger sequencing data sets. Moreover, UCE data on invertebrates, including insects, are sparse. We compared the phylogenetic informativeness of 959 UCE loci with a multi-locus data set of ten nuclear markers obtained via Sanger sequencing, testing the ability of these two types of data to resolve and date the evolutionary history of the second most species-rich subfamily of ants in the world, the Formicinae. Phylogenetic analyses show that UCEs are superior in resolving ancient and shallow relationships in formicine ants, demonstrated by increased node support and a more resolved phylogeny. Phylogenetic informativeness metrics indicate a twofold improvement relative to the 10-gene data matrix generated from the identical set of taxa. We were able to significantly improve formicine classification based on our comprehensive UCE phylogeny. Our divergence age estimations, using both UCE and Sanger data, indicate that crown-group Formicinae are older (104–117&amp;nbsp;Ma) than previously suggested. Biogeographic analyses infer that the diversification of the subfamily has occurred on all continents with no particular hub of cladogenesis. We found UCEs to be far superior to the multi-locus data set in estimating formicine relationships. The early history of the clade remains uncertain due to ancient rapid divergence events that are unresolvable even with our genomic-scale data, although this might be largely an effect of several problematic taxa subtended by long branches. Our comparison of divergence ages from both Sanger and UCE data demonstrates the effectiveness of UCEs for dating analyses. This comparative study highlights both the promise and limitations of UCEs for insect phylogenomics, and will prove useful to the growing number of evolutionary biologists considering the transition from Sanger to next-generation sequencing approaches.">
        <meta property="og:image" content="https://images.springer.com/sgw/journals/medium/12862.jpg">
    
        
    <script type="text/javascript" src="jquery.js"></script><script src="figshare.js"></script><link rel="preload" href="integrator.js"><script type="text/javascript" src="integrator.js"></script><link rel="preload" href="integrator_002.js"><script type="text/javascript" src="integrator_002.js"></script><script src="pubads_impl_249.js" async=""></script><style type="text/css">.MathJax_Hover_Frame {border-radius: .25em; -webkit-border-radius: .25em; -moz-border-radius: .25em; -khtml-border-radius: .25em; box-shadow: 0px 0px 15px #83A; -webkit-box-shadow: 0px 0px 15px #83A; -moz-box-shadow: 0px 0px 15px #83A; -khtml-box-shadow: 0px 0px 15px #83A; border: 1px solid #A6D ! important; display: inline-block; position: absolute}
.MathJax_Menu_Button .MathJax_Hover_Arrow {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 4px; -webkit-border-radius: 4px; -moz-border-radius: 4px; -khtml-border-radius: 4px; font-family: 'Courier New',Courier; font-size: 9px; color: #F0F0F0}
.MathJax_Menu_Button .MathJax_Hover_Arrow span {display: block; background-color: #AAA; border: 1px solid; border-radius: 3px; line-height: 0; padding: 4px}
.MathJax_Hover_Arrow:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_Hover_Arrow:hover span {background-color: #CCC!important}
</style><style type="text/css">#MathJax_About {position: fixed; left: 50%; width: auto; text-align: center; border: 3px outset; padding: 1em 2em; background-color: #DDDDDD; color: black; cursor: default; font-family: message-box; font-size: 120%; font-style: normal; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 15px; -webkit-border-radius: 15px; -moz-border-radius: 15px; -khtml-border-radius: 15px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_About.MathJax_MousePost {outline: none}
.MathJax_Menu {position: absolute; background-color: white; color: black; width: auto; padding: 5px 0px; border: 1px solid #CCCCCC; margin: 0; cursor: default; font: menu; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 5px; -webkit-border-radius: 5px; -moz-border-radius: 5px; -khtml-border-radius: 5px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
.MathJax_MenuItem {padding: 1px 2em; background: transparent}
.MathJax_MenuArrow {position: absolute; right: .5em; padding-top: .25em; color: #666666; font-size: .75em}
.MathJax_MenuActive .MathJax_MenuArrow {color: white}
.MathJax_MenuArrow.RTL {left: .5em; right: auto}
.MathJax_MenuCheck {position: absolute; left: .7em}
.MathJax_MenuCheck.RTL {right: .7em; left: auto}
.MathJax_MenuRadioCheck {position: absolute; left: .7em}
.MathJax_MenuRadioCheck.RTL {right: .7em; left: auto}
.MathJax_MenuLabel {padding: 1px 2em 3px 1.33em; font-style: italic}
.MathJax_MenuRule {border-top: 1px solid #DDDDDD; margin: 4px 3px}
.MathJax_MenuDisabled {color: GrayText}
.MathJax_MenuActive {background-color: #606872; color: white}
.MathJax_MenuDisabled:focus, .MathJax_MenuLabel:focus {background-color: #E8E8E8}
.MathJax_ContextMenu:focus {outline: none}
.MathJax_ContextMenu .MathJax_MenuItem:focus {outline: none}
#MathJax_AboutClose {top: .2em; right: .2em}
.MathJax_Menu .MathJax_MenuClose {top: -10px; left: -10px}
.MathJax_MenuClose {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; font-family: 'Courier New',Courier; font-size: 24px; color: #F0F0F0}
.MathJax_MenuClose span {display: block; background-color: #AAA; border: 1.5px solid; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; line-height: 0; padding: 8px 0 6px}
.MathJax_MenuClose:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_MenuClose:hover span {background-color: #CCC!important}
.MathJax_MenuClose:hover:focus {outline: none}
</style><style type="text/css">.MJX_Assistive_MathML {position: absolute!important; top: 0; left: 0; clip: rect(1px, 1px, 1px, 1px); padding: 1px 0 0 0!important; border: 0!important; height: 1px!important; width: 1px!important; overflow: hidden!important; display: block!important; -webkit-touch-callout: none; -webkit-user-select: none; -khtml-user-select: none; -moz-user-select: none; -ms-user-select: none; user-select: none}
.MJX_Assistive_MathML.MJX_Assistive_MathML_Block {width: 100%!important}
</style><style type="text/css">#MathJax_Zoom {position: absolute; background-color: #F0F0F0; overflow: auto; display: block; z-index: 301; padding: .5em; border: 1px solid black; margin: 0; font-weight: normal; font-style: normal; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; -webkit-box-sizing: content-box; -moz-box-sizing: content-box; box-sizing: content-box; box-shadow: 5px 5px 15px #AAAAAA; -webkit-box-shadow: 5px 5px 15px #AAAAAA; -moz-box-shadow: 5px 5px 15px #AAAAAA; -khtml-box-shadow: 5px 5px 15px #AAAAAA; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_ZoomOverlay {position: absolute; left: 0; top: 0; z-index: 300; display: inline-block; width: 100%; height: 100%; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
#MathJax_ZoomFrame {position: relative; display: inline-block; height: 0; width: 0}
#MathJax_ZoomEventTrap {position: absolute; left: 0; top: 0; z-index: 302; display: inline-block; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
</style><style type="text/css">.MathJax_Preview {color: #888}
#MathJax_Message {position: fixed; left: 1px; bottom: 2px; background-color: #E6E6E6; border: 1px solid #959595; margin: 0px; padding: 2px 8px; z-index: 102; color: black; font-size: 80%; width: auto; white-space: nowrap}
#MathJax_MSIE_Frame {position: absolute; top: 0; left: 0; width: 0px; z-index: 101; border: 0px; margin: 0px; padding: 0px}
.MathJax_Error {color: #CC0000; font-style: italic}
</style></head>

    <body class="journal journal-fulltext"><div id="MathJax_Message" style="display: none;"></div>
        
    
        
            
                <!-- Google Tag Manager (noscript) -->
                <noscript>
                    <iframe src="https://www.googletagmanager.com/ns.html?id=GTM-TDGJHK"
                            height="0" width="0" style="display:none;visibility:hidden"></iframe>
                </noscript>
                <!-- End Google Tag Manager (noscript) -->
            
        
    

        

    <div class="u-vh-full">
        <div class="u-visually-hidden" aria-hidden="true">
    
    <!--?xml version="1.0" encoding="UTF-8"?--><svg xmlns="http://www.w3.org/2000/svg"><defs><filter id="a" filterUnits="userSpaceOnUse" x="0" y="0" width="60.643" height="31.01"><feColorMatrix values="1 0 0 0 0 0 1 0 0 0 0 0 1 0 0 0 0 0 1 0"></feColorMatrix></filter></defs><symbol id="icon-arrow-down" viewBox="0 0 16 8"><path d="M8 6l6-6h2L8 8 0 0h2l6 6"></path></symbol><symbol id="icon-arrow-left-bullet" viewBox="0 0 8 16"><path d="M3 8l5 5v3L0 8l8-8v3L3 8z"></path></symbol><symbol id="icon-arrow-right-bullet" viewBox="0 0 8 16"><path d="M5 8L0 3V0l8 8-8 8v-3l5-5z"></path></symbol><symbol id="icon-arrow-right" viewBox="0 0 32 64"><path d="M0 0v64l32-32z"></path></symbol><symbol id="icon-close" viewBox="0 0 23.998 23.999"><path d="M20.998 0l-8.999 9L3 0 0 3l8.999 9L0 20.999l3 3L11.999 15l8.999 8.999 3-3L14.999 12l8.999-9z"></path></symbol><symbol id="icon-community" viewBox="0 0 60.643 31.01"><mask maskUnits="userSpaceOnUse" x="0" y="0" width="60.643" height="31.01" id="b"><path fill="#FFF" d="M0 31.01h60.643V0H0z" filter="url(#a)"></path></mask><path mask="url(#b)" d="M60.643 25.497L50.005 14.86a7.925 7.925 0 1 0-7.669 0l-6.847 6.847-1.334-1.334a7.925 7.925 0 0 0-3.834-14.861 7.925 7.925 0 0 0-3.834 14.861l-1.334 1.334-6.847-6.847a7.925 7.925 0 1 0-7.669 0L0 25.497h21.362L15.85 31.01h28.943l-5.514-5.513h21.364z"></path></symbol><symbol id="icon-download" viewBox="-301 390 9 14"><path d="M-301 395.6l4.5 5.1 4.5-5.1h-3V390h-3v5.6h-3zm0 6.5h9v1.9h-9z"></path></symbol><symbol id="icon-ext-link" viewBox="0 0 16 16"><path fill="#676767" d="M12.9 16H3.1C1.4 16 0 14.6 0 12.9V3.2C0 1.4 1.4 0 3.1 0h3.7v1H3.1C2 1 1 2 1 3.2v9.7C1 14 2 15 3.1 15h9.7c1.2 0 2.1-1 2.1-2.1V8.7h1v4.2c.1 1.7-1.3 3.1-3 3.1z"></path><path fill="#676767" d="M12.8 2.5l.7.7-9 8.9-.7-.7 9-8.9z"></path><path fill="#676767" d="M9.7 0L16 6.2V0z"></path></symbol><symbol id="icon-info-bordered" viewBox="470.812 270.868 18 18"><path d="M479.812 270.868c-4.972 0-9 4.029-9 9s4.028 9 9 9c4.971 0 9-4.029 9-9s-4.029-9-9-9zm0 16.875a7.875 7.875 0 0 1-7.875-7.875 7.875 7.875 0 1 1 15.75 0 7.875 7.875 0 0 1-7.875 7.875z"></path><path d="M479.284 279.586c.089-.238-.024-.361-.13-.361-.489 0-1.123 1.16-1.359 1.16-.093 0-.173-.092-.173-.174 0-.238.581-.801.751-.971.526-.506 1.217-.895 1.979-.895.567 0 1.174.346.703 1.639l-.952 2.604c-.079.199-.224.531-.224.746 0 .092.054.186.159.186.395 0 1.121-1.135 1.304-1.135.067 0 .158.086.158.199 0 .385-1.542 2.043-2.874 2.043-.477 0-.804-.225-.804-.732 0-.641.447-1.734.538-1.963l.924-2.346zm.727-3.41c0-.586.498-1.066 1.082-1.066.527 0 .91.357.91.906 0 .615-.501 1.068-1.098 1.068-.541-.002-.894-.361-.894-.908z"></path></symbol><symbol id="icon-opr" viewBox="0 0 18 18"><path d="M.9 2.5c-.2-.2-.4-.3-.6-.2-.3.2-.3.4-.3.6 0 .8 0 1.7.1 2.5 0 .5.3.7.7.6.9 0 1.7-.2 2.5-.3h.2c.5-.2.6-.5.3-.8-.2-.2-.4-.4-.6-.5-.1-.1-.2-.2-.4-.3.5-.6 1-1.1 1.7-1.5 4.6-3 10.9-.5 12 4.9.9 4.4-2.2 8.6-6.7 9.2-3.7.5-7.2-1.6-8.4-5.1 0-.1-.1-.2-.1-.3-.1-.3-.5-.5-.8-.4-.3.1-.5.4-.4.8.4 1.3 1.1 2.4 2 3.4 1.4 1.5 3.1 2.4 5.1 2.8 3 .5 5.6-.2 7.8-2.2 2.1-1.9 3-4.2 3-6.9-.1-4-3.1-7.6-7.1-8.4-3.3-.9-6.2 0-8.6 2.3-.2.1-.3.3-.5.5-.3-.2-.6-.5-.9-.7z"></path><path d="M13 4.7c-2.6 1.6-4.5 3.6-5.3 4.6L5.6 7.7l-.9.7L8.3 12c.6-1.6 2.6-4.6 4.9-6.8l-.2-.5z"></path></symbol><symbol id="icon-quotation" viewBox="0 0.333 55.932 35.619"><path d="M43.738.333L31.545 12.207v23.745h24.387V12.207H43.738zm-31.545 0L0 12.207v23.745h24.386V12.207H12.193z"></path></symbol><symbol id="icon-remove" viewBox="-296 388 18 18"><path d="M-291.7 396.1h9v2h-9z"></path><path d="M-287 405.5c-4.7 0-8.5-3.8-8.5-8.5s3.8-8.5 8.5-8.5 8.5 3.8 8.5 8.5-3.8 8.5-8.5 8.5zm0-16c-4.1 0-7.5 3.4-7.5 7.5s3.4 7.5 7.5 7.5 7.5-3.4 7.5-7.5-3.4-7.5-7.5-7.5z"></path></symbol><symbol id="icon-ribbon" viewBox="0 0 36 72"><path d="M0 0v72h.053L18 54.054 35.945 72H36V0H0zm25.8 11.4l3.001 3L15 28.2l-7.801-7.8 3-3L15 22.201 25.8 11.4z"></path></symbol><symbol id="icon-rss" viewBox="0 0 24 24"><g fill="#F26822"><ellipse cx="3.305" cy="20.702" rx="3.306" ry="3.298"></ellipse><path d="M15.978 24h-4.684c0-6.224-5.057-11.27-11.294-11.27V8.058c8.824 0 15.978 7.137 15.978 15.942z"></path><path d="M19.2 23.95C19.2 13.366 10.604 4.79 0 4.79V0c13.255 0 24 10.723 24 23.95h-4.8z"></path></g></symbol><symbol id="icon-search" viewBox="0 0 16 16"><path d="M6.276 12.257C2.815 12.257 0 9.508 0 6.128 0 2.749 2.815 0 6.276 0c3.462 0 6.277 2.749 6.277 6.128.002 3.38-2.815 6.129-6.277 6.129zm0-10.772c-2.622 0-4.755 2.083-4.755 4.643 0 2.562 2.133 4.644 4.755 4.644 2.623 0 4.756-2.082 4.756-4.644 0-2.56-2.134-4.643-4.756-4.643z"></path><path d="M16 13.751L13.641 16l-4.875-4.82 2.36-2.248L16 13.751z"></path></symbol><symbol id="icon-speech-bubble" viewBox="0 0 40 54"><path d="M0 0v40h6.667v13.334L20 40h20V0z"></path></symbol><symbol id="icon-error" viewBox="2.002 0 14 14"><path d="M16.002 11.949L13.951 14 9.002 9.051 4.053 14l-2.051-2.051L6.951 7 2.002 2.05 4.053 0l4.949 4.95L13.951 0l2.051 2.049L11.053 7l4.949 4.949z"></path></symbol><symbol id="icon-info" viewBox="5.502 0 7 14"><path d="M8.165 5.923c.146-.384-.04-.577-.209-.577-.783 0-1.798 1.856-2.177 1.856-.15 0-.277-.147-.277-.279 0-.383.933-1.282 1.203-1.557.846-.811 1.949-1.434 3.174-1.434.908 0 1.879.555 1.125 2.63l-1.526 4.167c-.126.32-.358.854-.358 1.198 0 .148.088.298.255.298.632 0 1.796-1.818 2.091-1.818.105 0 .252.134.252.321C11.717 11.345 9.245 14 7.11 14c-.763 0-1.289-.361-1.289-1.176 0-1.027.717-2.778.865-3.145l1.479-3.756zm1.144-4.211C9.309.771 10.111 0 11.045 0c.845 0 1.457.577 1.457 1.456 0 .983-.804 1.709-1.757 1.709-.866-.001-1.436-.578-1.436-1.453z"></path></symbol><symbol id="icon-success" viewBox="0 0 16 14"><path d="M15.592 0c-4.867 2.984-8.4 6.751-9.987 8.641L1.717 5.595 0 6.979l6.717 6.832c1.157-2.961 4.817-8.748 9.287-12.86L15.592 0z"></path></symbol><symbol id="icon-warning" viewBox="7.002 0 2.789 14"><path d="M7.002 11.211h2.789V14H7.002v-2.789zM7.002 0v3.436l.741 5.326h1.326l.723-5.326V0h-2.79z"></path></symbol><symbol id="icon-facebook-bordered" viewBox="463.812 263.868 32 32"><path d="M479.812 263.868c-8.837 0-16 7.163-16 16s7.163 16 16 16 16-7.163 16-16-7.163-16-16-16zm0 30c-7.732 0-14-6.269-14-14s6.268-14 14-14 14 6.269 14 14-6.267 14-14 14z"></path><path d="M483.025 280.48l.32-2.477h-2.453v-1.582c0-.715.199-1.207 1.227-1.207h1.311v-2.213a17.753 17.753 0 0 0-1.907-.098c-1.894 0-3.186 1.154-3.186 3.271V278h-2.142v2.477h2.142v6.354h2.557v-6.354l2.131.003z"></path></symbol><symbol id="icon-facebook" viewBox="0 0 24 24"><circle fill="#4D67A4" cx="12" cy="12" r="12"></circle><path fill="#FFF" d="M8.396 10.143h2.137V8.165s-.092-1.292.892-2.274c.979-.979 2.298-.887 4.177-.724v2.21h-1.387s-.586-.013-.861.303c-.271.315-.24.764-.24.875v1.588h2.41l-.311 2.458h-2.116v6.336h-2.56v-6.345H8.396v-2.449z"></path></symbol><symbol id="icon-gplus-bordered" viewBox="463.812 263.868 32 32"><path d="M479.812 263.868c-8.838 0-16 7.163-16 16s7.162 16 16 16c8.837 0 16-7.163 16-16s-7.163-16-16-16zm0 30c-7.732 0-14-6.269-14-14s6.268-14 14-14c7.731 0 14 6.269 14 14s-6.267 14-14 14z"></path><path d="M480.5 273.117c.123.076.26.185.416.334.146.152.289.343.432.562.135.211.258.459.359.746.086.286.129.619.129 1.004-.012.7-.164 1.262-.465 1.683a5.374 5.374 0 0 1-.461.562c-.186.176-.375.35-.58.527a2.65 2.65 0 0 0-.322.41.908.908 0 0 0-.176.553c0 .205.059.375.18.508.102.129.199.238.299.336l.662.545a6.48 6.48 0 0 1 1.086 1.119c.293.415.445.955.459 1.625 0 .949-.42 1.789-1.256 2.521-.869.76-2.123 1.15-3.762 1.177-1.373-.015-2.4-.308-3.074-.879-.682-.534-1.023-1.172-1.023-1.917 0-.359.111-.766.336-1.211.215-.443.602-.837 1.172-1.174a6.126 6.126 0 0 1 1.998-.725 15.415 15.415 0 0 1 1.723-.173 3.996 3.996 0 0 1-.377-.596 1.347 1.347 0 0 1-.193-.74c0-.174.023-.317.074-.433a8.83 8.83 0 0 1 .121-.338 5.973 5.973 0 0 1-.633.035c-1.043-.012-1.838-.339-2.381-.979-.574-.603-.861-1.3-.861-2.097 0-.961.404-1.832 1.217-2.616.559-.46 1.135-.757 1.736-.898a8.494 8.494 0 0 1 1.67-.182h3.916l-1.209.704h-1.207v.005l-.005.002zm.754 11.276c0-.498-.162-.932-.484-1.302-.346-.348-.881-.778-1.609-1.29a4.045 4.045 0 0 0-.439-.02c-.102-.016-.354 0-.768.039a8.141 8.141 0 0 0-1.244.275c-.1.036-.24.096-.42.17a2.713 2.713 0 0 0-.553.355 1.985 1.985 0 0 0-.459.592c-.145.25-.215.551-.215.901 0 .687.311 1.253.932 1.698.596.444 1.404.68 2.432.688.928-.014 1.629-.217 2.117-.612.476-.385.71-.886.71-1.494zm-2.729-5.743a1.843 1.843 0 0 0 1.289-.556c.164-.249.271-.506.318-.767a7.32 7.32 0 0 0 .041-.659c0-.772-.195-1.552-.594-2.336a2.611 2.611 0 0 0-.73-.921 1.894 1.894 0 0 0-1.064-.352 1.723 1.723 0 0 0-1.324.64c-.299.437-.438.92-.428 1.457 0 .709.209 1.444.623 2.211.201.356.459.659.775.912.323.248.686.371 1.094.371zm9.698.077h-2.055v-2.056h-.996v2.056h-2.053v.998h2.053v2.052h.996v-2.052h2.055z"></path></symbol><symbol id="icon-gplus" viewBox="0 0 24 24"><circle fill="#D84A36" cx="12" cy="12" r="12"></circle><g fill="#FFF"><path d="M12.012 12.111c-.304-.33-.07-.842.155-1.078.226-.241.639-.56.829-.74.193-.184.66-.605.674-1.868 0 0 .112-1.139-1.127-2.06h.977l1.038-.593h-3.507s-2.083-.067-3.085 1.207c-1 1.276-.966 2.871.114 3.77 1.08.898 2.371.648 2.371.648s-.197.422-.163.775c.036.353.421.849.504.965 0 0-2.386-.011-3.551 1.033-1.165 1.053-1.29 3.097 1.021 3.803 2.312.707 5.303-.055 5.906-2.277 0 0 .262-.908-.189-1.683-.459-.775-1.663-1.571-1.967-1.902zm-2.555-1.83c-.744-1-1.308-2.953 0-3.783 0 0 .955-.579 1.801.266.846.846 1.082 2.277.928 3.003-.15.727-.563.897-.739 1.008-.185.111-1.249.504-1.99-.494zm3.668 5.481c-.035.393-.083 1.254-1.491 1.664-1.409.408-3.826-.264-3.757-1.914 0 0 .006-.698.449-1.089.445-.396 1.094-.794 3.029-.819 0 0 .643.473 1.072.809.427.332.733.952.698 1.349z"></path><path d="M16.323 11.055V9.803h-.752v1.252h-1.255v.748h1.255v1.256h.752v-1.256h1.248v-.748z"></path></g></symbol><symbol id="icon-linkedin" viewBox="0 0 24 24"><circle fill="#0178B5" cx="12" cy="12" r="12"></circle><g fill="#FFF"><circle cx="8.022" cy="8.043" r="1.256"></circle><path d="M6.929 10.246h2.17v6.967h-2.17zm3.533 6.967h2.157v-3.594s-.078-1.627 1.219-1.627c1.301 0 1.211 1.231 1.211 1.635v3.586h2.183v-3.775s.129-2.745-1.472-3.191c-1.605-.445-2.661.071-3.219.985v-.985h-2.079v6.966z"></path></g></symbol><symbol id="icon-reddit" viewBox="0 0 24 24"><circle fill="#BCBCBC" cx="12" cy="12" r="12"></circle><path fill="#FFF" d="M4.661 9.741c.941 0 1.703.761 1.703 1.704 0 .938-.762 1.705-1.703 1.705s-1.704-.767-1.704-1.705c0-.943.763-1.704 1.704-1.704zm13.844 0a1.704 1.704 0 1 1 .001 3.409 1.704 1.704 0 0 1-.001-3.409z"></path><path fill="#FFF" d="M11.736 8.732c4.285 0 7.762 2.283 7.762 5.104 0 2.812-3.477 5.1-7.762 5.1-4.288 0-7.762-2.285-7.762-5.1.001-2.82 3.474-5.104 7.762-5.104z"></path><path fill="#010101" d="M11.736 19.262c-4.461 0-8.088-2.437-8.088-5.426 0-2.994 3.626-5.431 8.088-5.431 4.457 0 8.087 2.437 8.087 5.431 0 2.989-3.63 5.426-8.087 5.426zm0-10.205c-4.104 0-7.438 2.145-7.438 4.779 0 2.633 3.334 4.773 7.438 4.773 4.098 0 7.437-2.142 7.437-4.773 0-2.636-3.339-4.779-7.437-4.779z"></path><circle fill="#FFF" cx="18.014" cy="5.765" r="1.385"></circle><path fill="#010101" d="M18.014 7.456a1.694 1.694 0 0 1-1.695-1.691c0-.934.764-1.694 1.695-1.694s1.693.762 1.693 1.694c0 .934-.762 1.691-1.693 1.691zm0-2.734c-.574 0-1.043.469-1.043 1.043s.469 1.041 1.043 1.041a1.042 1.042 0 0 0 0-2.084zM3.818 13.275a2.068 2.068 0 0 1-1.068-1.811 2.074 2.074 0 0 1 3.538-1.466l-.458.462a1.412 1.412 0 0 0-1.007-.42 1.424 1.424 0 0 0-.688 2.669l-.317.566zm15.866 0l-.316-.566a1.43 1.43 0 0 0 .73-1.245c0-.784-.635-1.424-1.42-1.424-.381 0-.736.149-1.008.42l-.459-.462a2.074 2.074 0 1 1 2.473 3.277z"></path><path fill="#F04B23" d="M9.165 11.604c.704 0 1.272.564 1.272 1.27a1.273 1.273 0 1 1-1.272-1.27zm5.321 0a1.274 1.274 0 1 1-1.275 1.27c0-.704.57-1.27 1.275-1.27z"></path><path fill="#010101" d="M11.79 8.915a.327.327 0 0 1-.307-.44l1.472-4.181 3.616.862c.177.044.283.22.238.395a.32.32 0 0 1-.391.239l-3.045-.729-1.275 3.632a.334.334 0 0 1-.308.222zm-.069 8.376c-2.076 0-2.908-.939-2.943-.982a.332.332 0 0 1 .035-.461.334.334 0 0 1 .459.033c.019.021.713.758 2.449.758 1.766 0 2.539-.763 2.549-.767a.324.324 0 1 1 .469.446c-.04.043-.954.973-3.018.973z"></path></symbol><symbol id="icon-twitter-bordered" viewBox="463.812 263.868 32 32"><path d="M486.416 276.191a5.622 5.622 0 0 1-1.554.429 2.718 2.718 0 0 0 1.19-1.502 5.456 5.456 0 0 1-1.72.657 2.71 2.71 0 0 0-1.979-.854 2.711 2.711 0 0 0-2.642 3.326 7.681 7.681 0 0 1-5.586-2.831 2.714 2.714 0 0 0 .839 3.618 2.748 2.748 0 0 1-1.227-.339v.031a2.71 2.71 0 0 0 2.174 2.656 2.735 2.735 0 0 1-1.229.049 2.726 2.726 0 0 0 2.531 1.883 5.442 5.442 0 0 1-4.01 1.123 7.672 7.672 0 0 0 4.155 1.215c4.983 0 7.71-4.129 7.71-7.711 0-.115-.004-.232-.006-.351a5.41 5.41 0 0 0 1.354-1.399z"></path><path d="M479.812 263.868c-8.837 0-16 7.163-16 16s7.163 16 16 16 16-7.163 16-16-7.163-16-16-16zm0 30c-7.732 0-14-6.269-14-14s6.268-14 14-14 14 6.269 14 14-6.267 14-14 14z"></path></symbol><symbol id="icon-twitter" viewBox="0 0 24 24"><circle fill="#26A7DF" cx="12" cy="12" r="12"></circle><path fill="#FFF" d="M5.903 6.768s2.176 2.883 5.953 3.082c0 0-.51-1.702 1.058-3.004 1.568-1.305 3.508-.2 3.879.292 0 0 1.104-.172 1.875-.691 0 0-.252.995-1.18 1.594 0 0 1.086-.146 1.596-.439 0 0-.533.853-1.447 1.503 0 0 .381 3.595-2.709 6.57-3.09 2.971-8.176 2.352-10.012.797 0 0 2.495.277 4.249-1.219 0 0-2.074-.037-2.671-1.973 0 0 1.036.08 1.215-.119 0 0-2.232-.539-2.254-2.873 0 0 .68.339 1.297.359 0-.001-2.153-1.557-.849-3.879z"></path></symbol><symbol id="icon-weibo-bordered" viewBox="463.812 263.868 32 32"><path d="M479.812 263.868c-8.838 0-16 7.163-16 16s7.162 16 16 16c8.837 0 16-7.163 16-16s-7.163-16-16-16zm0 30c-7.732 0-14-6.269-14-14s6.268-14 14-14c7.731 0 14 6.269 14 14s-6.267 14-14 14z"></path><path d="M478.552 285.348c-2.616.261-4.876-.926-5.044-2.649-.167-1.722 1.814-3.33 4.433-3.588 2.609-.263 4.871.926 5.041 2.647.165 1.721-1.818 3.331-4.43 3.59m5.23-5.718c-.226-.065-.374-.109-.259-.403.25-.639.276-1.188.005-1.581-.515-.734-1.915-.693-3.521-.021 0 0-.508.224-.378-.181.247-.798.209-1.468-.178-1.852-.87-.878-3.194.032-5.183 2.027-1.489 1.494-2.357 3.082-2.357 4.453 0 2.619 3.354 4.213 6.631 4.213 4.297 0 7.154-2.504 7.154-4.493.001-1.198-1.007-1.881-1.914-2.162"></path><path fill="#020202" d="M486.637 274.833a4.176 4.176 0 0 0-3.982-1.291.608.608 0 0 0-.465.72.604.604 0 0 0 .72.466 2.968 2.968 0 0 1 2.827.92 3 3 0 0 1 .625 2.918.602.602 0 0 0 .39.762.603.603 0 0 0 .763-.391v-.001a4.218 4.218 0 0 0-.878-4.103"></path><path fill="#020202" d="M485.041 276.276a2.037 2.037 0 0 0-1.938-.63.518.518 0 0 0-.396.621.517.517 0 0 0 .617.398c.336-.071.702.03.947.307s.312.649.207.979a.52.52 0 0 0 .336.654.523.523 0 0 0 .657-.336 2.038 2.038 0 0 0-.43-1.993m-6.347 5.951c-.09.156-.293.233-.451.166-.151-.062-.204-.235-.115-.389.093-.155.284-.229.44-.168.157.056.214.235.126.391m-.832 1.074c-.253.405-.795.58-1.202.396-.403-.186-.521-.655-.27-1.051.248-.39.771-.566 1.176-.393.413.17.543.636.296 1.048m.95-2.864c-1.244-.326-2.65.294-3.19 1.396-.553 1.119-.021 2.369 1.236 2.775 1.303.42 2.84-.225 3.374-1.436.526-1.183-.132-2.402-1.42-2.735"></path></symbol><symbol id="icon-weibo" viewBox="4 4 24 24"><circle fill="#C8E8F9" cx="16" cy="16" r="12"></circle><path fill="#FFF" d="M9.098 18.194c0 1.981 2.574 3.593 5.757 3.593 3.178 0 5.756-1.611 5.756-3.593 0-1.989-2.578-3.601-5.756-3.601-3.183.001-5.757 1.612-5.757 3.601"></path><path fill="#DF0A21" d="M14.991 21.496c-2.817.278-5.244-.996-5.428-2.85-.183-1.855 1.95-3.586 4.767-3.862 2.813-.278 5.243.996 5.428 2.849.18 1.855-1.957 3.584-4.767 3.863m5.628-6.155c-.24-.067-.404-.116-.279-.432.27-.688.299-1.278.004-1.699-.551-.791-2.062-.748-3.789-.022 0-.001-.548.241-.406-.191.266-.859.224-1.577-.191-1.993-.937-.942-3.437.037-5.577 2.18-1.602 1.61-2.533 3.317-2.533 4.789 0 2.817 3.606 4.533 7.13 4.533 4.626 0 7.701-2.696 7.701-4.834.001-1.292-1.085-2.027-2.06-2.331"></path><path fill="#F4992C" d="M23.689 10.182a4.492 4.492 0 0 0-4.283-1.39.651.651 0 1 0 .269 1.275 3.2 3.2 0 0 1 3.045.99 3.224 3.224 0 0 1 .672 3.138.647.647 0 0 0 .418.823.647.647 0 0 0 .816-.421v-.003a4.51 4.51 0 0 0-.937-4.412"></path><path fill="#F4992C" d="M21.973 11.736a2.183 2.183 0 0 0-2.086-.679.56.56 0 0 0-.428.667.554.554 0 0 0 .662.43v.001c.361-.075.754.034 1.018.33.266.299.34.698.227 1.054a.563.563 0 0 0 .359.706.563.563 0 0 0 .709-.362 2.2 2.2 0 0 0-.461-2.147"></path><path fill="#13110C" d="M15.143 18.139c-.1.17-.315.251-.483.179-.168-.07-.222-.255-.128-.42.097-.167.311-.248.479-.182.165.062.228.252.132.423m-.896 1.155c-.271.434-.854.625-1.295.425-.433-.195-.561-.707-.285-1.128.269-.426.832-.609 1.265-.427.443.183.58.688.315 1.13m1.024-3.084c-1.34-.351-2.852.32-3.434 1.506-.597 1.206-.021 2.546 1.33 2.984 1.402.453 3.055-.243 3.629-1.543.569-1.275-.142-2.587-1.525-2.947"></path></symbol></svg>
</div>

        <a class="u-visually-hidden u-visually-hidden-focus" href="#main-content">
    <span class="c-banner c-banner--info">Skip to content</span>
</a>

        
            
    <div class="adsbox c-ad c-ad--LB1">
        <div class="c-ad__inner">
            <p class="c-ad__label">Advertisement</p>
            <div id="div-gpt-ad-LB1" data-gpt-unitpath="/270604982/bmc/bmcevolbiol/articles" data-gpt-sizes="728x90,970x90" data-gpt-targeting="pos=LB1;doi=10.1186/s12862-015-0552-5;kwrd=Ultraconserved elements,Insect phylogenomics,Ancient rapid radiations,Formicinae,Ant evolution;pmc=L21001,L2504X,L25090,M31010,L00004;"><div id="google_ads_iframe_/270604982/bmc/bmcevolbiol/articles_0__container__" style="border: 0pt none;"><iframe id="google_ads_iframe_/270604982/bmc/bmcevolbiol/articles_0" title="3rd party ad content" name="google_ads_iframe_/270604982/bmc/bmcevolbiol/articles_0" scrolling="no" marginwidth="0" marginheight="0" style="border: 0px none; vertical-align: bottom;" srcdoc="" height="90" frameborder="0" width="728"></iframe></div></div>
        </div>
    </div>

        
         
    <div id="membership-message-loader-desktop" class="placeholder" data-placeholder="/placeholder/v1/membership/message"></div>


<header class="c-popup-search ctx-publisher-header" id="top" data-test="publisher-header">
    <nav class="c-navbar">
        <div class="u-container">
            
            <div class="basic-white-logo c-navbar__logo u-margin-right-xxl" itemscope="" itemtype="http://schema.org/Organization" data-test="navbar-logo-header">
                <link itemprop="parentOrganization" href="#parentOrganization">
                <div class="c-logo">
    <a href="https://www.biomedcentral.com/" itemprop="url">
        <img src="logo-bmc-white-series-1da6525f4e.svg" alt="BMC" itemprop="logo">
        <div class="c-logo__strapline">
            <img src="logo-bmc-white-strapline-sn-e7b1d6e67f.svg" alt="Part of Springer Nature">
        </div>
    </a>
</div>
            </div>
            <button type="button" class="c-navbar__link c-dropdown js-publisher-menu-button u-hidden-at-lg" id="publisher-header-menu-button" data-toggle="collapse" data-test="publisher-header-menu-toggle" data-toggle-linked="publisher-header" data-target="publisher-header-search publisher-header-menu" aria-controls="publisher-header-menu publisher-header-search" aria-expanded="false">
                <span class="c-dropdown__toggle">Menu</span>
            </button>
            <div class="c-navbar__content c-collapse" id="publisher-header-menu">
                
    <ul class="c-navbar__nav" data-test="publisher-navigation">
        
        <li class="c-navbar__item">
            <a class="c-navbar__link" href="https://www.biomedcentral.com/journals">
                Explore journals
            </a>
        </li>
        
        <li class="c-navbar__item">
            <a class="c-navbar__link" href="https://www.biomedcentral.com/getpublished">
                Get published
            </a>
        </li>
        
        <li class="c-navbar__item">
            <a class="c-navbar__link" href="https://www.biomedcentral.com/about">
                About BMC
            </a>
        </li>
        
    </ul>


                <ul class="c-navbar__nav">
                    <li class="c-navbar__item">
                        <a data-replace="true" data-placeholder="/placeholder/v1/username" class="c-navbar__link" href="https://www.biomedcentral.com/login" data-test="login-link" data-track="click" data-track-category="Account" data-track-action="Click Login Link">
                            <span class="u-js-shown">Login</span>
                            <span class="u-js-hidden">My Account</span>
                        </a>
                    </li>
                    <li class="c-navbar__item">
                        <button type="button" class="c-navbar__link js-publisher-search-button u-hidden-at-lt-lg" data-toggle="collapse" data-test="header-search-button" data-target="publisher-header-search publisher-header-menu publisher-header-menu-button" aria-controls="publisher-header-search publisher-header-menu" aria-expanded="false">
                            <span class="c-flex c-flex--align-center">
                                Search
                                <svg class="c-icon u-margin-left-xs" width="14" height="14" aria-hidden="true">
                                    <use xlink:href="#icon-search"></use>
                                </svg>
                            </span>
                        </button>
                    </li>
                </ul>
            </div>
        </div>
    </nav>
    <div class="c-popup-search__content c-collapse js-publisher-search-bar" id="publisher-header-search">
        <div class="u-container">
            <div class="c-popup-search__container">
                <div class="ctx-search">
    <form role="search" class="c-form-field c-form-field--inline" method="GET" action="//www.biomedcentral.com/search" data-track="submit" data-track-category="Search and Results" data-track-action="Submit search" data-track-label="" data-test="global-search" novalidate="">
        <label for="publisherSearch" class="u-visually-hidden">Search all BMC articles</label>
        <input id="publisherSearch" class="c-form-field__input js-publisher-search-input" autocomplete="off" role="textbox" data-test="search-input" name="query" placeholder="Search all BMC articles" type="text">
        <button class="c-button c-button--secondary" type="submit" data-test="search-submit-button">
    <span class="u-visually-hidden">Search</span>
    <svg class="c-icon" width="16" height="16" aria-hidden="true">
        <use xlink:href="#icon-search"></use>
    </svg>
</button>


        <input name="searchType" value="publisherSearch" type="hidden">
    </form>
</div>

            </div>
        </div>
    </div>
</header>


        
            <header class="c-journal-header c-journal-header--bmc-evolutionary-biology ctx-journal-header">
                <div class="u-container">
                    <div class="c-journal-header__inner ">
                        
                        <div class="c-journal-title" id="journalTitle">
                            <a href="https://bmcevolbiol.biomedcentral.com/">
    
        
    
    <span class="c-journal-title__text ">BMC Evolutionary Biology</span>
    
</a>
                        </div>
                        
                    </div>
                </div>
                <div class="c-navbar">
                    <div class="u-container">
                        
                            <button type="button" class="c-navbar__link c-dropdown u-hidden-at-lg" data-toggle="collapse" data-test="journal-header-menu" data-toggle-linked="journal-header" data-target="journal-menu" aria-controls="journal-menu" aria-expanded="false">
                                    <span class="c-dropdown__toggle">Menu</span>
                            </button>
                            <div class="c-navbar__content c-collapse" id="journal-menu">
                                <nav class="c-navbar__nav">
                                    <ul class="c-navbar__nav c-navbar__nav--journal" role="menu" data-test="site-navigation">
                                        
                                            <li class="c-navbar__item" role="menuitem">
                                                <a class="c-navbar__link" href="https://bmcevolbiol.biomedcentral.com/">Home</a>
                                            </li>
                                        
                                            <li class="c-navbar__item" role="menuitem">
                                                <a class="c-navbar__link" href="https://bmcevolbiol.biomedcentral.com/about">About</a>
                                            </li>
                                        
                                            <li class="c-navbar__item" role="menuitem">
                                                <a class="c-navbar__link c-navbar__link--is-shown" href="https://bmcevolbiol.biomedcentral.com/articles">Articles</a>
                                            </li>
                                        
                                            <li class="c-navbar__item" role="menuitem">
                                                <a class="c-navbar__link" href="https://bmcevolbiol.biomedcentral.com/submission-guidelines">Submission Guidelines</a>
                                            </li>
                                        
                                    </ul>
                                </nav>
                            </div>
                        
                    </div>
                </div>
                <div class="c-journal-header__identity c-journal-header__identity--default">
                    
                </div>
            </header>
            
        

        <div class="u-container u-margin-top-xl u-margin-bottom-xl" id="main-content">
            <div class="c-content-layout c-content-layout--fulltext" data-component="sticky-container">
                <div class="c-content-layout__nav u-hidden-at-lt-xl">
                    <nav xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" id="ArticleNavigation" aria-labelledby="toc-menu-title" data-component="sticky"><h3 id="toc-menu-title" class="u-visually-hidden" data-test="toc-title">Table of Contents</h3><ol class="c-nav c-nav--stacked" role="menu" id="tocMenu" aria-expanded="true"><li role="menuitem" class="table-of-contents__item c-nav__item"><a class="table-of-contents__link c-nav__link" href="#Abs1" data-track-category="Table of Contents" data-track="click" data-track-action="Link Clicked" data-track-label="Abstract" data-test="abstract-link">Abstract</a></li><li role="menuitem" class="table-of-contents__item c-nav__item"><a class="table-of-contents__link c-nav__link" href="#Sec1" data-track-category="Table of Contents" data-track="click" data-track-action="Link Clicked" data-track-label="Background">Background</a></li><li role="menuitem" class="table-of-contents__item c-nav__item"><a class="table-of-contents__link c-nav__link" href="#Sec2" data-track-category="Table of Contents" data-track="click" data-track-action="Link Clicked" data-track-label="Methods">Methods</a></li><li role="menuitem" class="table-of-contents__item c-nav__item"><a class="table-of-contents__link c-nav__link" href="#Sec11" data-track-category="Table of Contents" data-track="click" data-track-action="Link Clicked" data-track-label="Results">Results</a></li><li role="menuitem" class="table-of-contents__item c-nav__item"><a class="table-of-contents__link c-nav__link" href="#Sec17" data-track-category="Table of Contents" data-track="click" data-track-action="Link Clicked" data-track-label="Discussion">Discussion</a></li><li role="menuitem" class="table-of-contents__item c-nav__item"><a class="table-of-contents__link c-nav__link" href="#Sec23" data-track-category="Table of Contents" data-track="click" data-track-action="Link Clicked" data-track-label="Conclusions">Conclusions</a></li><li role="menuitem" class="table-of-contents__item c-nav__item"><a class="table-of-contents__link c-nav__link" href="#Sec24" data-track-category="Table of Contents" data-track="click" data-track-action="Link Clicked" data-track-label="Availability of supporting data">Availability of supporting data</a></li><li role="menuitem" class="table-of-contents__item c-nav__item"><a class="table-of-contents__link c-nav__link" href="#Decs" data-test="declarations-link" data-track-category="Table of Contents" data-track="click" data-track-action="Link Clicked" data-track-label="Declarations">
                  Declarations
                </a></li><li role="menuitem" class="table-of-contents__item c-nav__item"><a class="table-of-contents__link c-nav__link" href="#Bib1" data-track-category="Table of Contents" data-track="click" data-track-action="Link Clicked" data-track-label="References">References</a></li><li role="menuitem" class="table-of-contents__item c-nav__item u-hidden" data-display-comments-link=""><a class="table-of-contents__link c-nav__link" href="#Comments" data-track-category="Table of Contents" data-track="click" data-track-action="Link Clicked" data-track-label="Comments">Comments</a></li></ol></nav>
                </div>

                <main class="c-content-layout__main">
                    <article itemscope="" itemtype="http://schema.org/ScholarlyArticle">
                        
                        
    <div class="FulltextWrapper">
        <ul class="c-article-identifiers u-reset-list">
            <li class="c-article-identifiers__item" data-test="article-category">Research article</li>
            <li class="c-article-identifiers__item" data-test="open-access">
                <span class="u-text-open-access">Open Access</span>
            </li>
            
            
        </ul>
    </div>

                        <div class="FulltextWrapper"><div xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" class="MainTitleSection"><h1 xmlns="http://www.w3.org/1999/xhtml" class="ArticleTitle" itemprop="headline" lang="en">Phylogenomic
 methods outperform traditional multi-locus approaches in resolving deep
 evolutionary history: a case study of formicine ants</h1></div><div xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" class="AuthorNames" data-author-affiliations="true"><ul class="u-listReset"><li class="Author hasAffil" data-jumpto="Aff" data-test="author-name"><span xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="AuthorName" itemprop="author" itemscope="" itemtype="http://schema.org/Person">Bonnie&nbsp;B.&nbsp;Blaimer</span><sup><a href="#Aff1" class="AffiliationID">1</a></sup><a href="mailto:bonnieblaimer@gmail.com" class="EmailAuthor" title="Email author"><span class="ContactIcon" aria-hidden="true"></span><span class="u-srOnly">Email author</span></a>, </li><li class="Author hasAffil" data-jumpto="Aff" data-test="author-name"><span xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="AuthorName" itemprop="author" itemscope="" itemtype="http://schema.org/Person">Seán&nbsp;G.&nbsp;Brady</span><sup><a href="#Aff1" class="AffiliationID">1</a></sup>, </li><li class="Author hasAffil" data-jumpto="Aff" data-test="author-name"><span xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="AuthorName" itemprop="author" itemscope="" itemtype="http://schema.org/Person">Ted&nbsp;R.&nbsp;Schultz</span><sup><a href="#Aff1" class="AffiliationID">1</a></sup>, </li><li class="Author hasAffil" data-jumpto="Aff" data-test="author-name"><span xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="AuthorName" itemprop="author" itemscope="" itemtype="http://schema.org/Person">Michael&nbsp;W.&nbsp;Lloyd</span><sup><a href="#Aff1" class="AffiliationID">1</a></sup>, </li><li class="Author hasAffil" data-jumpto="Aff" data-test="author-name"><span xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="AuthorName" itemprop="author" itemscope="" itemtype="http://schema.org/Person">Brian&nbsp;L.&nbsp;Fisher</span><sup><a href="#Aff2" class="AffiliationID">2</a></sup> and </li><li class="Author hasAffil" data-jumpto="Aff" data-test="author-name"><span xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="AuthorName" itemprop="author" itemscope="" itemtype="http://schema.org/Person">Philip&nbsp;S.&nbsp;Ward</span><sup><a href="#Aff3" class="AffiliationID">3</a></sup></li></ul></div><div xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" class="ArticleContextInformation"><div class="ContextInformation"><div class="ContextInformationJournalTitles"><span class="ArticleCitation"><span itemscope="" itemtype="http://schema.org/Periodical" itemid="#periodical"><span xmlns="http://www.w3.org/1999/xhtml" class="JournalTitle">BMC Evolutionary Biology</span></span><span class="ArticleCitation_Year">2015</span><span class="ArticleCitation_Volume"><strong itemprop="isPartOf" itemtype="http://schema.org/PublicationVolume"><link itemprop="isPartOf" href="#periodical">15</strong>:271</span></span></div><p class="ArticleDOI"><a itemprop="sameAs" class="u-text-inherit" href="https://doi.org/10.1186/s12862-015-0552-5">https://doi.org/10.1186/s12862-015-0552-5</a></p><p class="Copyright">© &nbsp;Blaimer et al.&nbsp;2015</p></div></div><ul class="ArticleHistory u-reset-list"><li class="History HistoryReceived"><strong>Received: </strong><span>8&nbsp;October&nbsp;2015</span></li><li class="History HistoryAccepted"><strong>Accepted: </strong><span>26&nbsp;November&nbsp;2015</span></li><li class="History HistoryOnlineDate"><strong>Published: </strong><span itemprop="datePublished">4&nbsp;December&nbsp;2015</span></li></ul></div>
                        
                        
                        
    

                        <div class="FulltextWrapper"><section xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" class="Abstract Section1 RenderAsSection1 c-section" id="Abs1" lang="en"><h2 class="c-section__heading"><a href="#Abstract" data-test="abstract-heading" data-disable-click-event-default="">Abstract</a></h2><div class="c-section__content" id="Abstract"><div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="AbstractSection" id="ASec1">
                <h3 xmlns="" class="Heading">Background</h3>
                <p id="Par1" class="Para">Ultraconserved elements (UCEs)
 have been successfully used in phylogenomics for a variety of taxa, but
 their power in phylogenetic inference has yet to be extensively 
compared with that of traditional Sanger sequencing data sets. Moreover,
 UCE data on invertebrates, including insects, are sparse. We compared 
the phylogenetic informativeness of 959 UCE loci with a multi-locus data
 set of ten nuclear markers obtained via Sanger sequencing, testing the 
ability of these two types of data to resolve and date the evolutionary 
history of the second most species-rich subfamily of ants in the world, 
the Formicinae.</p>
              </div><div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="AbstractSection" id="ASec2">
                <h3 xmlns="" class="Heading">Results</h3>
                <p id="Par2" class="Para">Phylogenetic analyses show 
that UCEs are superior in resolving ancient and shallow relationships in
 formicine ants, demonstrated by increased node support and a more 
resolved phylogeny. Phylogenetic informativeness metrics indicate a 
twofold improvement relative to the 10-gene data matrix generated from 
the identical set of taxa. We were able to significantly improve 
formicine classification based on our comprehensive UCE phylogeny. Our 
divergence age estimations, using both UCE and Sanger data, indicate 
that crown-group Formicinae are older (104–117&nbsp;Ma) than previously 
suggested. Biogeographic analyses infer that the diversification of the 
subfamily has occurred on all continents with no particular hub of 
cladogenesis.</p>
              </div><div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="AbstractSection" id="ASec3">
                <h3 xmlns="" class="Heading">Conclusions</h3>
                <p id="Par3" class="Para">We found UCEs to be far 
superior to the multi-locus data set in estimating formicine 
relationships. The early history of the clade remains uncertain due to 
ancient rapid divergence events that are unresolvable even with our 
genomic-scale data, although this might be largely an effect of several 
problematic taxa subtended by long branches. Our comparison of 
divergence ages from both Sanger and UCE data demonstrates the 
effectiveness of UCEs for dating analyses. This comparative study 
highlights both the promise and limitations of UCEs for insect 
phylogenomics, and will prove useful to the growing number of 
evolutionary biologists considering the transition from Sanger to 
next-generation sequencing approaches.</p>
              </div></div></section><section xmlns:func="http://oscar.fig.bmc.com" class="KeywordGroup Section1 RenderAsSection1 c-section" lang="en"><h2 xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" class="Heading js-ToggleCollapseSection c-section__heading" data-sticky-update="true"><a href="#Keywords" data-toggle="collapse" data-toggle-disable-clickoff="" data-target="Keywords">Keywords</a></h2><div class="js-CollapseSection CollapseSection c-section__content" data-test="collapse-section" id="Keywords"><ul class="c-keywords"><li class="c-keywords__item" itemprop="about">Ultraconserved elements</li><li class="c-keywords__item" itemprop="about">Insect phylogenomics</li><li class="c-keywords__item" itemprop="about">Ancient rapid radiations</li><li class="c-keywords__item" itemprop="about">Formicinae</li><li class="c-keywords__item" itemprop="about">Ant evolution</li></ul></div></section>
            <section xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" id="Sec1" class="Section1 RenderAsSection1 c-section"><h2 class="Heading js-ToggleCollapseSection c-section__heading" data-sticky-update="true"><a href="#Background" data-toggle="collapse" data-toggle-disable-clickoff="" data-target="Background">Background</a></h2><div class="js-CollapseSection CollapseSection c-section__content" id="Background" data-test="collapse-section"><p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par19" class="Para">Current
 target-enrichment and next-generation sequencing techniques allow for 
the rapid generation of hundreds of loci for use as phylogenetic 
markers. This is demonstrated by an increasing number of studies, 
largely conducted on vertebrates (e.g., [<span class="CitationRef"><a href="#CR1" data-toggle="citation">1</a></span><span>, <span class="CitationRef"><a href="#CR2" data-toggle="citation">2</a></span>, <span class="CitationRef"><a href="#CR3" data-toggle="citation">3</a></span>, </span><span class="CitationRef"><a href="#CR4" data-toggle="citation">4</a></span>]).
 One of the most promising approaches focuses on capturing 
ultraconserved elements (UCEs)—regions in the genome that have remained 
highly conserved across great evolutionary distances. Core UCEs are 
sequenced together with their more variable flanking regions, producing 
markers for phylogenetic reconstruction [<span class="CitationRef"><a href="#CR5" data-toggle="citation">5</a></span>, <span class="CitationRef"><a href="#CR6" data-toggle="citation">6</a></span>].
 Recently, this method has been adapted and applied to insects, 
informing family-level relationships among Hymenoptera (bees, ants and 
wasps) [<span class="CitationRef"><a href="#CR7" data-toggle="citation">7</a></span>].
 Although these prior studies report the successful use of UCEs in 
phylogenetics, we are not aware of any study directly comparing this 
phylogenomic method to the longstanding use of multi-locus sequence data
 in phylogenetics. In an important recent exercise, Gilbert et al. [<span class="CitationRef"><a href="#CR8" data-toggle="citation">8</a></span>]
 calculated and compared the phylogenetic informativeness of UCEs and 
several single-copy nuclear markers extracted in silico from eight 
published fish genomes. Here we address an unresolved phylogenetic 
problem by simultaneously generating both UCE and traditional 
Sanger-sequenced data for the same 82 ant species, estimating and 
directly comparing phylogenies separately produced by each source of 
information. Many biologists require such an applied comparison as they 
evaluate the costs and benefits of next-generation techniques over 
Sanger sequencing in advance of the data collection phases of their next
 projects.</p><p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par20" class="Para">The
 evolutionary history and ecological success of the ants (family 
Formicidae) have been illuminated recently in multiple studies using a 
variety of approaches. For example, recent molecular phylogenetic 
research has clarified relationships among and within subfamilies [<span class="CitationRef"><a href="#CR9" data-toggle="citation">9</a></span><span>, <span class="CitationRef"><a href="#CR10" data-toggle="citation">10</a></span>, <span class="CitationRef"><a href="#CR11" data-toggle="citation">11</a></span>, <span class="CitationRef"><a href="#CR12" data-toggle="citation">12</a></span>, <span class="CitationRef"><a href="#CR13" data-toggle="citation">13</a></span>, </span><span class="CitationRef"><a href="#CR14" data-toggle="citation">14</a></span>], while other research has focused on diversification patterns [<span class="CitationRef"><a href="#CR15" data-toggle="citation">15</a></span><span>, <span class="CitationRef"><a href="#CR16" data-toggle="citation">16</a></span>, </span><span class="CitationRef"><a href="#CR17" data-toggle="citation">17</a></span>] or the evolution of successful behaviors [<span class="CitationRef"><a href="#CR18" data-toggle="citation">18</a></span>, <span class="CitationRef"><a href="#CR19" data-toggle="citation">19</a></span>].
 The ant subfamily Formicinae is the second most species-rich subfamily 
of ants with around 3000 described species, trumped in diversity only by
 the Myrmicinae [<span class="CitationRef"><a href="#CR20" data-toggle="citation">20</a></span>]. The group contains the well-known, economically important carpenter ants of the genus <em xmlns="" class="EmphasisTypeItalic">Camponotus</em>,
 presently the most diverse genus-level clade of ants in the world with 
over 1,000 described species. Other prominent members of this group 
include the silk-spinning weaver ants (<em xmlns="" class="EmphasisTypeItalic">Oecophylla</em>) and spiny ants (<em xmlns="" class="EmphasisTypeItalic">Polyrhachis</em>), which dominate the forest canopies of the Old World, as well as the yellow crazy ant <em xmlns="" class="EmphasisTypeItalic">Anoplolepis gracilipes</em>,
 one of the world’s most destructive and invasive ant species. Despite 
being stingless, formicines have derived a defensive venom exceptional 
among the arthropods, formic acid (well described e.g. in <em xmlns="" class="EmphasisTypeItalic">Camponotus</em>, <em xmlns="" class="EmphasisTypeItalic">Formica</em>, <em xmlns="" class="EmphasisTypeItalic">Lasius</em> [<span class="CitationRef"><a href="#CR21" data-toggle="citation">21</a></span>]),
 and also have been identified recently as the only known dietary source
 of pumiliotoxins sequestered by dendrobatid poison dart frogs [<span class="CitationRef"><a href="#CR22" data-toggle="citation">22</a></span>]. Many formicine ants also exhibit intriguing slave-making behavior (e.g. <em xmlns="" class="EmphasisTypeItalic">Polyergus, Rossomyrmex</em>) or other forms of social parasitism (e.g. <em xmlns="" class="EmphasisTypeItalic">Lasius, Plagiolepis</em>) [<span class="CitationRef"><a href="#CR23" data-toggle="citation">23</a></span>].</p><p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par21" class="Para">Recent
 phylogenetic research has focused on resolving generic relationships 
within subfamily-level groups of ants such as the Myrmicinae [<span class="CitationRef"><a href="#CR14" data-toggle="citation">14</a></span>], Ponerinae [<span class="CitationRef"><a href="#CR13" data-toggle="citation">13</a></span>], and Dorylinae [<span class="CitationRef"><a href="#CR12" data-toggle="citation">12</a></span>].
 The evolution of the subfamily Formicinae, however, has not yet been 
comprehensively scrutinized, with the exception of one particular 
subgroup, the <em xmlns="" class="EmphasisTypeItalic">Prenolepis</em> genus-group [<span class="CitationRef"><a href="#CR24" data-toggle="citation">24</a></span>, <span class="CitationRef"><a href="#CR25" data-toggle="citation">25</a></span>].
 Prior studies of generic relationships within these subfamilies were 
based on data sets composed of multiple nuclear loci generated by 
traditional Sanger sequencing. Particularly in the cases of the 
Myrmicinae and the Dorylinae, these methods were not able to provide 
information adequate for entirely resolving lineage diversification [<span class="CitationRef"><a href="#CR12" data-toggle="citation">12</a></span>, <span class="CitationRef"><a href="#CR14" data-toggle="citation">14</a></span>].</p><p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par22" class="Para">We
 compare the efficacy of a UCE-based phylogenomic data set to that of a 
high-quality nuclear-gene data set for resolving phylogenetic 
relationships and obtaining divergence estimates within formicine ants. 
To do so, we assembled a data set of 959 UCE loci by means of target 
enrichment and multiplexed sequencing for 82 formicine taxa, and 
simultaneously generated a data set of ten PCR-amplified and 
Sanger-sequenced nuclear loci (eight of these protein-coding) for the 
same 82 taxa. We then use these combined results to investigate (i) the 
power of each data set for resolving the phylogeny of the subfamily 
Formicinae and (ii) the evolutionary and biogeographic history of the 
subfamily.</p></div></section>
            <section xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" id="Sec2" class="Section1 RenderAsSection1 c-section"><h2 class="Heading js-ToggleCollapseSection c-section__heading" data-sticky-update="true"><a href="#Methods" data-toggle="collapse" data-toggle-disable-clickoff="" data-target="Methods">Methods</a></h2><div class="js-CollapseSection CollapseSection c-section__content" id="Methods" data-test="collapse-section"><section id="Sec3" class="Section2 RenderAsSection2">
                <h3 class="Heading">Molecular data collection</h3>
                <section id="Sec4" class="Section3 RenderAsSection3">
                  <h4 class="Heading">Taxon sampling</h4>
                  <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par23" class="Para">A more extensive description of all methods can be found in Additional file <span class="InternalRef"><a href="#MOESM1">1</a></span>.
 Our data set comprised 82 ant species, which represent 48 of the 51 
currently-valid formicine genera. We further included eight outgroup 
taxa from seven other ant subfamilies (Myrmicinae, Ectatomminae, 
Heteroponerinae, Pseudomyrmecinae, Myrmeciinae, Aneuretinae, 
Dolichoderinae) belonging to the formicoid clade of ants (sensu Brady et
 al. [<span class="CitationRef"><a href="#CR9" data-toggle="citation">9</a></span>]),
 and trees were rooted using the four subfamilies most distantly related
 to the formicines. Ants for this study were collected at the following 
locations, and with respective institutions providing authorizations for
 the capture, collection and exportation: AUSTRALIA: Environmental 
Protection Agency, Queensland Parks and Wildlife Service; BRUNEI: 
Universiti Brunei Darussalam and the Brunei Museums; CENTRAL AFRICAN 
REPUBLIC: Ministère de l'Environnement des Eaux, Forest, Chasses et 
Pêche; COSTA RICA: Ministerio del Ambiente y Energia; Direction General 
de Vida Silvestre, Ministerio de Recursos Naturales Energia y Minas; 
FIJI: Ministry of Fisheries and Forests, Department of Forestry; GABON: 
National Center for Scientific and Technological Research; HONG KONG: 
Agriculture, Fisheries and Conservation Department, Kowloon; MADAGASCAR:
 Ministère de l'Environnement et des Forêts, Madagascar National Parks; 
MALAYSIA: Sabah Biodiversity Council; UGANDA: Uganda National Council 
for Science and Technology, Uganda Wildlife Authority; UNITED STATES: 
National Park Service; and State of California Natural Resources Agency,
 Department of Parks and Recreation. Vouchers have been deposited at the
 University of California, Davis, at the National Museum of Natural 
History, and at the California Academy of Sciences. Additional file <span class="InternalRef"><a href="#MOESM2">2</a></span> lists specimen identifiers; collection data can be found by searching for these CASENT numbers on the AntWeb (<span class="ExternalRef"><a href="http://www.antweb.org/"><span class="RefSource">www.antweb.org</span></a></span>)
 database. DNA was extracted destructively or non-destructively from 
worker ants or pupae using a DNeasy Blood and TissueKit (Qiagen, 
Valencia, CA, USA).</p>
                </section>
                <section id="Sec5" class="Section3 RenderAsSection3">
                  <h4 class="Heading">Library preparation, target enrichment and sequencing of UCEs</h4>
                  <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par24" class="Para">We
 sheared 2.8–497&nbsp;ng (139&nbsp;ng mean) DNA to a target size of 
approximately 500–600&nbsp;bp by sonication and used this sheared DNA as
 input for a modified genomic DNA library preparation protocol following
 Faircloth et al. ([<span class="CitationRef"><a href="#CR7" data-toggle="citation">7</a></span>], but see Additional file <span class="InternalRef"><a href="#MOESM1">1</a></span>).
 We enriched pooled libraries using a set of 2749 custom-designed probes
 (MYcroarray, Inc.) targeting 1510 UCE loci in Hymenoptera [<span class="CitationRef"><a href="#CR7" data-toggle="citation">7</a></span>]. We followed library enrichment procedures for the MYcroarray MYBaits kit [<span class="CitationRef"><a href="#CR26" data-toggle="citation">26</a></span>],
 except we used a 0.1X concentration of the standard MYBaits 
concentration, and added 0.7&nbsp;μL of 500&nbsp;μM custom blocking 
oligos designed against our custom sequence tags. We used the with-bead 
approach for PCR recovery of enriched libraries as described in 
Faircloth et al. [<span class="CitationRef"><a href="#CR7" data-toggle="citation">7</a></span>].
 Following post-enrichment PCR, we purified resulting reactions using 
1.0X speedbeads and rehydrated the enriched pools in 22&nbsp;μL&nbsp;EB.</p>
                  <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par25" class="Para">We performed qPCR using a SYBR® FAST qPCR kit (Kapa Biosystems) on a ViiA<sup>TM</sup>
 7 (Life Technologies), and based on the size-adjusted concentrations 
estimated by qPCR, we pooled libraries at equimolar concentrations and 
size-selected for 250–800 with a BluePippin (SageScience). The pooled 
libraries were sequenced using two partial lanes of a 150-bp paired-end 
Illumina HiSeq 2500 run (U Cornell Genomics Facility). All of the UCE 
laboratory work was conducted in and with support of the Laboratories of
 Analytical Biology (L.A.B.) facilities of the National Museum of 
Natural History (NMNH). Quality-trimmed sequence reads generated as part
 of this study are available from the NCBI Sequence Read Archive (<span class="ExternalRef"><a href="http://www.ncbi.nlm.nih.gov/sra"><span class="RefSource">http://www.ncbi.nlm.nih.gov/sra</span></a></span>; SUB1067415).</p>
                </section>
                <section id="Sec6" class="Section3 RenderAsSection3">
                  <h4 class="Heading">Amplification, Sanger sequencing, and alignment of nuclear loci</h4>
                  <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par26" class="Para">Ten nuclear markers commonly used in ant systematics were selected for amplification ([for primers see [<span class="CitationRef"><a href="#CR9" data-toggle="citation">9</a></span>, <span class="CitationRef"><a href="#CR11" data-toggle="citation">11</a></span>, <span class="CitationRef"><a href="#CR27" data-toggle="citation">27</a></span>, <span class="CitationRef"><a href="#CR28" data-toggle="citation">28</a></span>]): Long-wavelength rhodopsin (<em xmlns="" class="EmphasisTypeItalic">LW Rh</em>, 458&nbsp;bp), elongation factor 1-alpha F1 (<em xmlns="" class="EmphasisTypeItalic">EF1aF1</em>, 359&nbsp;bp), elongation factor 1-alpha F2 (<em xmlns="" class="EmphasisTypeItalic">EF1aF2</em>, 517&nbsp;bp), abdominal-A (<em xmlns="" class="EmphasisTypeItalic">abdA</em>, 606&nbsp;bp), arginine kinase (<em xmlns="" class="EmphasisTypeItalic">argK</em>, 673&nbsp;bp), ultrabithorax (<em xmlns="" class="EmphasisTypeItalic">Ubx</em>, 630&nbsp;bp), 18S ribosomal DNA (1851&nbsp;bp), 28S rDNA (825&nbsp;bp), wingless (<em xmlns="" class="EmphasisTypeItalic">Wg</em>, 412&nbsp;bp) and topoisomerase 1 (<em xmlns="" class="EmphasisTypeItalic">Top1</em>,
 883&nbsp;bp), for a total of 7214&nbsp;bp in the aligned data matrix. 
Amplifications were performed using standard PCR methods outlined in 
Ward and Downie [<span class="CitationRef"><a href="#CR27" data-toggle="citation">27</a></span>]
 and cycle sequencing reactions were performed using PCR primers and 
BigDye ® Terminator ver. 3.1 Cycle Sequencing chemistry. Amplicons were 
analyzed on ABI 3730 Sequencers © (2011 Life Technologies, Frederick, 
MA) housed at the College of Biological Sciences DNA Sequencing 
Facility, University of California, Davis, CA and at L.A.B. at the NMNH,
 Washington, DC. Sequence data were aligned with MAFFT v7.017 [<span class="CitationRef"><a href="#CR29" data-toggle="citation">29</a></span>]. All newly generated sequences have been deposited in GenBank, under accessions KT443144-KT443783 (see Additional file <span class="InternalRef"><a href="#MOESM2">2</a></span>).</p>
                </section>
              </section><section id="Sec7" class="Section2 RenderAsSection2">
                <h3 class="Heading">Processing and alignment of UCE data</h3>
                <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par27" class="Para">We trimmed the demultiplexed FASTQ data output for adapter contamination and low-quality bases using Illumiprocessor [<span class="CitationRef"><a href="#CR30" data-toggle="citation">30</a></span>], based on the package Trimmomatic [<span class="CitationRef"><a href="#CR31" data-toggle="citation">31</a></span>]. All further data processing described in the following relied on the PHYLUCE package [<span class="CitationRef"><a href="#CR6" data-toggle="citation">6</a></span>, <span class="CitationRef"><a href="#CR32" data-toggle="citation">32</a></span>]; a detailed description of this pipeline and its scripts can be found in Additional file <span class="InternalRef"><a href="#MOESM1">1</a></span>.</p>
                <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par28" class="Para">We computed summary statistics on the data and assembled the cleaned reads using Trinity (version trinityrnaseq_r20140717) [<span class="CitationRef"><a href="#CR33" data-toggle="citation">33</a></span>].
 To identify contigs representing enriched UCE loci from each species, 
species-specific contig assemblies were aligned to a FASTA file of all 
enrichment baits (min_coverage = 50, min_identity = 80), and sequence 
coverage statistics (avg, min, max) for contigs containing UCE loci were
 calculated. We created FASTA files for each UCE locus containing 
sequence data for taxa present at that particular locus and aligned 
these using MAFFT [<span class="CitationRef"><a href="#CR29" data-toggle="citation">29</a></span>] (min-length = 20, no-trim). We further trimmed our alignments using Gblocks [<span class="CitationRef"><a href="#CR34" data-toggle="citation">34</a></span>].
 Initially, we selected the following subsets of UCE alignments 
depending on the captured UCE loci across taxa: 1) 50&nbsp;% complete 
(containing data from ≥ 45 of the 90 taxa for each locus), 2) 60&nbsp;% 
complete (≥54 of 90 taxa), 3) 70&nbsp;% complete (≥63 of 90 taxa) and 4)
 95&nbsp;% (≥85 of 90 taxa).</p>
              </section><section id="Sec8" class="Section2 RenderAsSection2">
                <h3 class="Heading">Phylogenetic inference</h3>
                <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par29" class="Para">For the 10-gene data set, PartitionFinder v.1.1.1 [<span class="CitationRef"><a href="#CR35" data-toggle="citation">35</a></span>]
 was used to simultaneously select data partitions and estimate 
appropriate models of evolution, for subsequent analyses with maximum 
likelihood (ML) and Bayesian methods. ML analyses were carried out in 
the programs RAxML v7.7.7 [<span class="CitationRef"><a href="#CR36" data-toggle="citation">36</a></span>] and GARLI v.2.0 [<span class="CitationRef"><a href="#CR37" data-toggle="citation">37</a></span>] and included both best tree and bootstrap searches. Bayesian inference (BI) was performed in MRBAYES 3.2 [<span class="CitationRef"><a href="#CR38" data-toggle="citation">38</a></span>]
 with 2 independent runs of 40 million generations, summarizing 72000 
trees after discarding a burnin of 10&nbsp;%. MCMC convergence was 
checked visually and with Bayes Factor comparisons using TRACER v1.6 (<span class="ExternalRef"><a href="http://tree.bio.ed.ac.uk/software/tracer/"><span class="RefSource">http://tree.bio.ed.ac.uk/software/tracer/</span></a></span>)
 and by examining PSRF values in MrBayes .stat output files. All 
analyses were carried out using parallel processing (one chain per CPU) 
on a 12-core Intel-processor Apple computer or on the Smithsonian NMNH 
L.A.B Topaz network of Apple computers with Intel processors.</p>
                <div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par30" class="Para">To select data partitions for the UCE phylogenomic data set, we used a development version of PartitionFinder [<span class="CitationRef"><a href="#CR39" data-toggle="citation">39</a></span>] that depends on the software fast_TIGER (http://dx.doi.org/<span class="ExternalRef"><a href="https://doi.org/10.5281/zenodo.12914"><span class="RefSource">10.5281/zenodo.12914</span></a></span>)
 and is designed to handle large genome-scale data sets. The UCE data 
set was analyzed with ML best tree and bootstrap searches (N = 100) in 
RAxML v8.0.3 [<span class="CitationRef"><a href="#CR36" data-toggle="citation">36</a></span>],
 initially on a 50&nbsp;%, 60&nbsp;%, 70&nbsp;% and 95&nbsp;% complete 
UCE matrix (see above). For subsequent analyses, however, we elected to 
proceed with the 70&nbsp;% and 95&nbsp;% matrices. We also reconstructed
 gene trees for the 959 UCE loci in the 70&nbsp;% matrix by performing 
RAxML analyses (best tree and bootstrap) on individual loci, and used 
these to construct a subset of UCE data, representing the 100 loci with 
the best average bootstrap score (UCE-100best hereafter). The four main 
data sets used for downstream analyses are summarized in Table&nbsp;<span class="InternalRef"><a href="#Tab1">1</a></span>. We calculated phylogenetic informativeness (PI) [<span class="CitationRef"><a href="#CR40" data-toggle="citation">40</a></span>] per nucleotide site for the three UCE and the 10-gene data sets with the software package TAPIR [<span class="CitationRef"><a href="#CR41" data-toggle="citation">41</a></span>] (<span class="ExternalRef"><a href="http://faircloth-lab.github.com/tapir/"><span class="RefSource">http://faircloth-lab.github.com/tapir/</span></a></span>), a parallelized version of PhyDesign [<span class="CitationRef"><a href="#CR42" data-toggle="citation">42</a></span>].<figure xmlns="" id="Tab1" class="FigureTable"><figcaption class="Caption" lang="en"><div xmlns="http://www.w3.org/1999/xhtml" class="CaptionContent"><span class="CaptionNumber">Table 1</span>
                        <p xmlns="" class="SimplePara">Overview of UCE and Sanger data sets</p>
                      </div></figcaption><div class="Table"><table xmlns="http://www.w3.org/1999/xhtml" border="1"><colgroup><col><col><col><col><col></colgroup><thead><tr class="header"><th>&nbsp;</th><th><p xmlns="" class="SimplePara">Sanger - 10-gene</p></th><th><p xmlns="" class="SimplePara">UCE - 70&nbsp;%</p></th><th><p xmlns="" class="SimplePara">UCE - 100best</p></th><th><p xmlns="" class="SimplePara">UCE - 95&nbsp;%</p></th></tr></thead><tbody><tr class="noclass"><td><p xmlns="" class="SimplePara">Loci</p></td><td><p xmlns="" class="SimplePara">10</p></td><td><p xmlns="" class="SimplePara">959</p></td><td><p xmlns="" class="SimplePara">100</p></td><td><p xmlns="" class="SimplePara">50</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">Total bp</p></td><td><p xmlns="" class="SimplePara">7214&nbsp;bp</p></td><td><p xmlns="" class="SimplePara">589757&nbsp;bp</p></td><td><p xmlns="" class="SimplePara">71611&nbsp;bp</p></td><td><p xmlns="" class="SimplePara">35619&nbsp;bp</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">Mean PI (ingroup)</p></td><td><p xmlns="" class="SimplePara">4.09E-04</p></td><td><p xmlns="" class="SimplePara">7.86E-04</p></td><td><p xmlns="" class="SimplePara">9.39E-04</p></td><td><p xmlns="" class="SimplePara">6.65E-04</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">Data partitions</p></td><td><p xmlns="" class="SimplePara">12</p></td><td><p xmlns="" class="SimplePara">101</p></td><td><p xmlns="" class="SimplePara">18</p></td><td><p xmlns="" class="SimplePara">18</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">RAxML</p></td><td><p xmlns="" class="SimplePara">x</p></td><td><p xmlns="" class="SimplePara">x</p></td><td><p xmlns="" class="SimplePara">x</p></td><td><p xmlns="" class="SimplePara">x</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">GARLI</p></td><td><p xmlns="" class="SimplePara">x</p></td><td><p xmlns="" class="SimplePara">-</p></td><td><p xmlns="" class="SimplePara">-</p></td><td><p xmlns="" class="SimplePara">-</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">MrBayes</p></td><td><p xmlns="" class="SimplePara">40 Mgen</p></td><td><p xmlns="" class="SimplePara">-</p></td><td><p xmlns="" class="SimplePara">-</p></td><td><p xmlns="" class="SimplePara">-</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">BEAST</p></td><td><p xmlns="" class="SimplePara">500 Mgen</p></td><td><p xmlns="" class="SimplePara">-</p></td><td><p xmlns="" class="SimplePara">300 Mgen</p></td><td><p xmlns="" class="SimplePara">300 Mgen</p></td></tr></tbody></table></div><div xmlns="http://www.w3.org/1999/xhtml" class="TableFooter">
                      <p xmlns="" class="SimplePara">Summary of number 
of loci, length of matrix, mean PI, number of data partitions and type 
of analyses for the four data sets used in this study. PI = phylogenetic
 informativeness <em class="EmphasisTypeItalic">sensu</em> Townsend [<span xmlns="http://www.w3.org/1999/xhtml" class="CitationRef"><a href="#CR40" data-toggle="citation">40</a></span>], calculated for ingroup taxa only</p>
                    </div></figure>
                        </div>
                <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par31" class="Para">We
 identified five taxa, subtended by long branches, which influenced 
resolution in analyses of both the UCE and 10-gene data sets. In order 
to better understand the effects of these taxa on phylogenetic results, 
we carried out phylogenetic analyses (BI for 10-gene, ML for UCEs) with a
 series of taxon-reduced data sets. Data matrices as well as the 
resulting tree files for the four main data sets are deposited in 
Treebase (<span class="ExternalRef"><a href="http://purl.org/phylo/treebase/phylows/study/TB2:S18146"><span class="RefSource">http://purl.org/phylo/treebase/phylows/study/TB2:S18146</span></a></span>).</p>
              </section><section id="Sec9" class="Section2 RenderAsSection2">
                <h3 class="Heading">Dating analyses</h3>
                <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par32" class="Para">We
 inferred divergence dates within the Formicinae from the UCE-100best, 
UCE-95&nbsp;%, and the 10-gene data set with the program BEAST v1.8 [<span class="CitationRef"><a href="#CR43" data-toggle="citation">43</a></span>].
 We chose these smaller UCE data sets for the dating analysis because 
BEAST cannot currently handle larger data sets with hundreds of loci 
such as our full 70&nbsp;% matrix. We performed analyses on the 10-gene 
data set with four independent runs and 500 million generations; UCE 
analyses consisted of two runs of 300 million generations each for 
95&nbsp;% and 100best data sets (see Table&nbsp;<span class="InternalRef"><a href="#Tab1">1</a></span>). All divergence analyses were calibrated by placing calibration priors on nine nodes in the phylogeny (see Additional file <span class="InternalRef"><a href="#MOESM3">3</a></span>).
 Trace files were analyzed in Tracer v1.6 to determine chain convergence
 and burnin. Tree files were then summarized with LogCombiner v1.8.2 and
 TreeAnnotator v1.8.2 after discarding a burnin of 20&nbsp;%. These 
dating analyses and all phylogenetic analyses on UCEs were performed on 
the Smithsonian Institution high performance cluster (SI/HPC).</p>
              </section><section id="Sec10" class="Section2 RenderAsSection2">
                <h3 class="Heading">Biogeographic analyses</h3>
                <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par33" class="Para">We constructed a species distribution matrix to evaluate the biogeographic history of Formicinae (see Additional file <span class="InternalRef"><a href="#MOESM4">4</a></span>).
 We assigned to each terminal taxon the distribution of its species plus
 that of other species estimated to be more closely related to the 
terminal taxon than to any other species in our data set. We used the 
dispersal-extinction-cladogenesis model (DEC, “Lagrange,” [<span class="CitationRef"><a href="#CR44" data-toggle="citation">44</a></span>]) and the statistical DEC model (S-DEC, “Bayes-Lagrange”, [<span class="CitationRef"><a href="#CR45" data-toggle="citation">45</a></span>]) implemented in the program RASP [<span class="CitationRef"><a href="#CR46" data-toggle="citation">46</a></span>]
 to estimate ancestral ranges from the set of trees and the respective 
MCC tree from our BEAST analysis on the UCE-100best data set. Under both
 models, outgroups were removed before the analyses. We followed Ward et
 al. [<span class="CitationRef"><a href="#CR14" data-toggle="citation">14</a></span>]
 in designating six biogeographic areas (Neotropical, Nearctic, 
Palearctic, Afrotropical, Indomalayan and Australasian) and defined 
different dispersal constraints for two time slices (0–50&nbsp;Ma and 
50–105&nbsp;Ma) based on paleogeography (Scotese, 2010, PALEOMAP 
project; <span class="ExternalRef"><a href="http://www.scotese.com/"><span class="RefSource">http://www.scotese.com/</span></a></span>) (see Additional file <span class="InternalRef"><a href="#MOESM5">5</a></span>).</p>
              </section></div></section>
            <section xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" id="Sec11" class="Section1 RenderAsSection1 c-section"><h2 class="Heading js-ToggleCollapseSection c-section__heading" data-sticky-update="true"><a href="#Results" data-toggle="collapse" data-toggle-disable-clickoff="" data-target="Results">Results</a></h2><div class="js-CollapseSection CollapseSection c-section__content" id="Results" data-test="collapse-section"><section id="Sec12" class="Section2 RenderAsSection2">
                <h3 class="Heading">UCE capture statistics</h3>
                <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par34" class="Para">Multiplexed sequencing of UCEs resulted in an average of 1.6 million reads per sample (see Additional file <span class="InternalRef"><a href="#MOESM6">6</a></span>)
 with an average length of 290 base pairs (bp). An average of 29655 
contigs with a mean length of 359.2&nbsp;bp was assembled by Trinity 
after adapter- and quality-trimming, with an average coverage of 17.4X. 
From all of the assembled contigs, we recovered an average of 936 UCE 
loci per sample with a mean length of 805&nbsp;bp. The average coverage 
per captured UCE locus was 92.3X. Following alignment of individual UCE 
loci, we filtered these data for loci captured for ≥70&nbsp;% of taxa 
(UCE-70&nbsp;%), retaining 959 loci, and for loci captured for 
≥95&nbsp;% of taxa (UCE-95&nbsp;%), retaining 50 loci. We further 
selected a data set of 100 loci with the best average bootstrap support 
for subsequent dating analyses (UCE-100best), because this represented a
 manageable size for BEAST (whereas analysis of the full 959 loci was 
not feasible). Concatenation of UCE loci generated matrices of 
589757&nbsp;bp (UCE-70&nbsp;%), 71611&nbsp;bp (UCE-100best), and 
35619&nbsp;bp (UCE-95&nbsp;%). The ten Sanger-sequenced nuclear loci 
were concatenated into one matrix of 7214&nbsp;bp of protein-coding and 
ribosomal DNA data, with no missing data for any taxon. Table&nbsp;<span class="InternalRef"><a href="#Tab1">1</a></span> provides an overview of these data sets.</p>
              </section><section id="Sec13" class="Section2 RenderAsSection2">
                <h3 class="Heading">Phylogenetic results</h3>
                <div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par35" class="Para">PartitionFinder
 selected 12 data partitions as the best-fitting scheme for our 10-gene 
matrix, whereas the UCE-70&nbsp;%, UCE-95&nbsp;% and UCE-100best data 
sets were divided into 101, 18, and 18 partitions, respectively 
(Table&nbsp;<span class="InternalRef"><a href="#Tab1">1</a></span>). The
 results of maximum likelihood (ML) best tree and bootstrap searches on 
the partitioned UCE-70&nbsp;% data set and 10-gene data sets are 
summarized in Fig.&nbsp;<span class="InternalRef"><a href="#Fig1">1</a></span>.
 Analyses of both data sets identified six major, well-supported clades 
within the Formicinae, outlined below, as well as five isolated genera 
for which closest relatives remain uncertain. We propose tribal and 
genus-level revisions to the classification of the subfamily based on 
our phylogenetic results (as detailed in Additional file <span class="InternalRef"><a href="#MOESM7">7</a></span>), intended for formal publication elsewhere (Ward et al., in review).<ol xmlns="" class="OrderedList"><li class="ListItem"><span xmlns="http://www.w3.org/1999/xhtml" class="ItemNumber">1)</span><div xmlns="http://www.w3.org/1999/xhtml" class="ItemContent">
                        <p id="Par36" class="Para">Camponotini: This 
clade is recovered with high bootstrap support (BS = 100) in both UCE 
and 10-gene analyses, and includes the genera <em xmlns="" class="EmphasisTypeItalic">Camponotus</em>, <em xmlns="" class="EmphasisTypeItalic">Polyrhachis</em>, <em xmlns="" class="EmphasisTypeItalic">Opisthopsis</em>, <em xmlns="" class="EmphasisTypeItalic">Echinopla</em>, <em xmlns="" class="EmphasisTypeItalic">Phasmomyrmex</em>, and <em xmlns="" class="EmphasisTypeItalic">Forelophilus.</em>
                                    </p>
                      </div><div class="ClearBoth">&nbsp;</div></li><li class="ListItem"><span xmlns="http://www.w3.org/1999/xhtml" class="ItemNumber">2)</span><div xmlns="http://www.w3.org/1999/xhtml" class="ItemContent">
                        <p id="Par37" class="Para">Plagiolepidini (redefined): We recovered very good support (BS = 100 in both analyses) for a clade containing the genera <em xmlns="" class="EmphasisTypeItalic">Acropyga, Anoplolepis, Agraulomyrmex, Aphomomyrmex, Lepiosota, Petalomyrmex, Plagiolepis, Tapinolepis</em>, and an undescribed formicine genus. <em xmlns="" class="EmphasisTypeItalic">Lepisiota</em> was further recovered as paraphyletic with respect to <em xmlns="" class="EmphasisTypeItalic">Plagiolepis</em> (Fig.&nbsp;<span class="InternalRef"><a href="#Fig1">1</a></span>).</p>
                      </div><div class="ClearBoth">&nbsp;</div></li><li class="ListItem"><span xmlns="http://www.w3.org/1999/xhtml" class="ItemNumber">3)</span><div xmlns="http://www.w3.org/1999/xhtml" class="ItemContent">
                        <p id="Par38" class="Para">Formicini: All 
current members of the tribe Formicini form another highly-supported 
clade in both UCE and 10-gene analyses (BS = 100/100), including <em xmlns="" class="EmphasisTypeItalic">Bajcaridris</em>, <em xmlns="" class="EmphasisTypeItalic">Cataglyphis</em>, <em xmlns="" class="EmphasisTypeItalic">Formica, Ibericoformica</em>, <em xmlns="" class="EmphasisTypeItalic">Rossomyrmex</em>, <em xmlns="" class="EmphasisTypeItalic">Polyergus</em>, and <em xmlns="" class="EmphasisTypeItalic">Proformica</em>.</p>
                      </div><div class="ClearBoth">&nbsp;</div></li><li class="ListItem"><span xmlns="http://www.w3.org/1999/xhtml" class="ItemNumber">4)</span><div xmlns="http://www.w3.org/1999/xhtml" class="ItemContent">
                        <p id="Par39" class="Para">Melophorini (redefined): The UCE phylogeny reveals a well-supported clade (BS = 100) containing <em xmlns="" class="EmphasisTypeItalic">Lasiophanes, Melophorus, Myrmecorhynchus, Notoncus, Pseudonotoncus, Notostigma, Prolasius, Stigmacros</em>, and <em xmlns="" class="EmphasisTypeItalic">Teratomyrmex</em>.
 This clade is also recovered in analyses of the 10-gene data set, but 
with lower support (BS: GARLI = 64, RAxML = 55; BI/PP: 1.0).</p>
                      </div><div class="ClearBoth">&nbsp;</div></li><li class="ListItem"><span xmlns="http://www.w3.org/1999/xhtml" class="ItemNumber">5)</span><div xmlns="http://www.w3.org/1999/xhtml" class="ItemContent">
                        <p id="Par40" class="Para">Lasiini (redefined): Both UCE and 10-gene data sets further highly support (BS = 100/95) a clade consisting of ten genera: <em xmlns="" class="EmphasisTypeItalic">Cladomyrma, Euprenolepis, Lasius, Myrmecocystus, Nylanderia, Paraparatrechina, Paratrechina, Prenolepis, Pseudolasius</em> and <em xmlns="" class="EmphasisTypeItalic">Zatania</em>. Two genera, <em xmlns="" class="EmphasisTypeItalic">Prenolepis</em> and <em xmlns="" class="EmphasisTypeItalic">Nylanderia</em>, were further recovered as paraphyletic with respect to each other.</p>
                      </div><div class="ClearBoth">&nbsp;</div></li><li class="ListItem"><span xmlns="http://www.w3.org/1999/xhtml" class="ItemNumber">6)</span><div xmlns="http://www.w3.org/1999/xhtml" class="ItemContent">
                        <p id="Par41" class="Para">Myrmelachistini (resurrected): Both data sets recover <em xmlns="" class="EmphasisTypeItalic">Brachymyrmex</em> and <em xmlns="" class="EmphasisTypeItalic">Myrmelachista</em> as sister to all other formicines, forming a highly supported clade (BS = 100/100).</p>
                      </div><div class="ClearBoth">&nbsp;</div></li></ol>
                           <figure xmlns="" class="Figure" id="Fig1"><div xmlns="http://www.w3.org/1999/xhtml" class="MediaObject" id="MO1">
                      <a target="_blank" rel="noopener" href="https://media.springernature.com/full/springer-static/image/art%3A10.1186%2Fs12862-015-0552-5/MediaObjects/12862_2015_552_Fig1_HTML.gif"><img class="c-lazy-image u-js-shown" src="12862_2015_552_Fig1_HTML.gif" alt="Fig. 1" data-lazy-load="true" data-src="https://media.springernature.com/lw785/springer-static/image/art%3A10.1186%2Fs12862-015-0552-5/MediaObjects/12862_2015_552_Fig1_HTML.gif"><noscript><img src="https://media.springernature.com/lw785/springer-static/image/art%3A10.1186%2Fs12862-015-0552-5/MediaObjects/12862_2015_552_Fig1_HTML.gif" alt="Fig. 1"></img></noscript></a>
                    </div><figcaption class="Caption" lang="en"><div xmlns="http://www.w3.org/1999/xhtml" class="CaptionContent"><span class="CaptionNumber">Fig. 1</span>
                        <p xmlns="" class="SimplePara">Phylogeny of the subfamily Formicinae. Contrasting phylogenetic trees estimated by <strong class="EmphasisTypeBold">a</strong> the phylogenomic UCE-70&nbsp;% data set and <strong class="EmphasisTypeBold">b</strong>
 the "traditional" Sanger-sequencing-generated 10-nuclear-gene data set.
 Both figures are based on RAxML best tree searches, with RAxML 
bootstrap values mapped on the respective nodes. The bootstrap searches 
included 100 and 1152 replicates for UCE and 10-gene data set, 
respectively. The six larger formicine tribes are indicated. See also 
Additional file <span xmlns="http://www.w3.org/1999/xhtml" class="InternalRef"><a href="#MOESM8">8</a></span>
                                    </p>
                      </div></figcaption></figure>
                        </div>
              </section><section id="Sec14" class="Section2 RenderAsSection2">
                <h3 class="Heading">Performance of UCE versus 10-gene data sets</h3>
                <div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par42" class="Para">Overall,
 phylogenies resulting from maximum-likelihood analyses of each of our 
two main data sets (UCE-70&nbsp;% and 10-gene data set) are congruent in
 topology for all parts of the phylogeny that receive high support, with
 disagreements restricted only to poorly resolved areas. The single 
exception is the position of <em xmlns="" class="EmphasisTypeItalic">Myrmecocystus</em>. In the UCE-70&nbsp;% phylogeny <em xmlns="" class="EmphasisTypeItalic">Myrmecocystus</em> is sister to <em xmlns="" class="EmphasisTypeItalic">Lasius</em>, whereas in the 10-gene data set this taxon arises within <em xmlns="" class="EmphasisTypeItalic">Lasius</em>. The UCE-70&nbsp;% phylogeny (Fig.&nbsp;<span class="InternalRef"><a href="#Fig1">1a</a></span>) is highly supported with only 12 (out of 85) nodes with BS &lt; 100, whereas the 10-gene phylogeny (Fig.&nbsp;<span class="InternalRef"><a href="#Fig1">1b</a></span>),
 in contrast, retains 42 nodes with BS &lt; 100. For example, generic 
relationships within the tribe Melophorini are well supported in the UCE
 tree, whereas these remain fairly unresolved in the 10-gene analysis. 
Interestingly, neither of the phylogenies resulting from the two data 
sets is able to fully resolve the relationships between the 
above-described major formicine lineages, i.e., both contain an ancient,
 unresolved polytomy. The UCE data set, however, provides substantially 
more resolution in this area of the phylogeny (Fig.&nbsp;<span class="InternalRef"><a href="#Fig1">1a</a></span>) than does the 10-gene phylogeny (Fig.&nbsp;<span class="InternalRef"><a href="#Fig1">1b</a></span>),
 reconstructing the Melophorini as sister to a clade containing the 
Camponotini, Plagiolepidini, and Formicini (in a polytomy). Figs.&nbsp;<span class="InternalRef"><a href="#Fig2">2a</a></span>&amp;b
 provide contrasting summary sketches of the tribal relationships based 
on these two data sets. Bayesian analyses of the 10-gene data set 
produced very similar results (see Additional file <span class="InternalRef"><a href="#MOESM8">8</a></span>). Maximum-likelihood analyses for the UCE-100best and 95&nbsp;% data sets also show <em xmlns="" class="EmphasisTypeItalic">Lasius</em>
 as paraphyletic, and overall phylogenies from these smaller UCE subsets
 are less well supported than from the full 70&nbsp;% data set (see 
Additional file <span class="InternalRef"><a href="#MOESM9">9</a></span>).<figure xmlns="" class="Figure" id="Fig2"><div xmlns="http://www.w3.org/1999/xhtml" class="MediaObject" id="MO2">
                      <a target="_blank" rel="noopener" href="https://media.springernature.com/full/springer-static/image/art%3A10.1186%2Fs12862-015-0552-5/MediaObjects/12862_2015_552_Fig2_HTML.gif"><img class="c-lazy-image u-js-shown" src="12862_2015_552_Fig2_HTML.gif" alt="Fig. 2" data-lazy-load="true" data-src="https://media.springernature.com/lw785/springer-static/image/art%3A10.1186%2Fs12862-015-0552-5/MediaObjects/12862_2015_552_Fig2_HTML.gif"><noscript><img src="https://media.springernature.com/lw785/springer-static/image/art%3A10.1186%2Fs12862-015-0552-5/MediaObjects/12862_2015_552_Fig2_HTML.gif" alt="Fig. 2"></img></noscript></a>
                    </div><figcaption class="Caption" lang="en"><div xmlns="http://www.w3.org/1999/xhtml" class="CaptionContent"><span class="CaptionNumber">Fig. 2</span>
                        <p xmlns="" class="SimplePara">Comparison of 
support for major lineages within the Formicinae. Comparison of support 
for formicine tribes and the influence of the problematic taxa. Panel <strong class="EmphasisTypeBold">a</strong>) UCE-70&nbsp;% data set, all taxa included; <strong class="EmphasisTypeBold">b</strong>) 10-gene data set, all taxa included; <strong class="EmphasisTypeBold">c</strong>) UCE-70&nbsp;% data set, problematic genera excluded (<em class="EmphasisTypeItalic">Santschiella, Gigantiops, Myrmoteras, Oecophylla, Gesomyrmex</em>); <strong class="EmphasisTypeBold">d</strong>)
 10-gene data set, problematic genera excluded. Both figures are based 
on RAxML bootstrap searches, with 100 and 1152 replicates for UCE and 
10-gene data set, respectively. See also Additional file <span xmlns="http://www.w3.org/1999/xhtml" class="InternalRef"><a href="#MOESM10">10</a></span>
                                    </p>
                      </div></figcaption></figure>
                        </div>
                <div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par43" class="Para">Phylogenetic
 informativeness (PI) increases in both data sets asymptotically with 
increasing divergence ages, but is much higher in the UCE data sets than
 in the 10-gene data set (Fig.&nbsp;<span class="InternalRef"><a href="#Fig3">3a</a></span>).
 The UCE-70&nbsp;%, UCE-100best and UCE-95&nbsp;% data sets show a 2.0-,
 2.5- and 1.5-fold increase in PI relative to the 10-gene data set, 
respectively (Fig.&nbsp;<span class="InternalRef"><a href="#Fig3">3a</a></span> and Table&nbsp;<span class="InternalRef"><a href="#Tab1">1</a></span>).<figure xmlns="" class="Figure" id="Fig3"><div xmlns="http://www.w3.org/1999/xhtml" class="MediaObject" id="MO3">
                      <a target="_blank" rel="noopener" href="https://media.springernature.com/full/springer-static/image/art%3A10.1186%2Fs12862-015-0552-5/MediaObjects/12862_2015_552_Fig3_HTML.gif"><img class="c-lazy-image u-js-shown" src="12862_2015_552_Fig3_HTML.gif" alt="Fig. 3" data-lazy-load="true" data-src="https://media.springernature.com/lw785/springer-static/image/art%3A10.1186%2Fs12862-015-0552-5/MediaObjects/12862_2015_552_Fig3_HTML.gif"><noscript><img src="https://media.springernature.com/lw785/springer-static/image/art%3A10.1186%2Fs12862-015-0552-5/MediaObjects/12862_2015_552_Fig3_HTML.gif" alt="Fig. 3"></img></noscript></a>
                    </div><figcaption class="Caption" lang="en"><div xmlns="http://www.w3.org/1999/xhtml" class="CaptionContent"><span class="CaptionNumber">Fig. 3</span>
                        <p xmlns="" class="SimplePara">
                                       <strong class="EmphasisTypeBold">a</strong> Phylogenetic informativeness and <strong class="EmphasisTypeBold">b</strong>) comparison of divergence estimates. <strong class="EmphasisTypeBold">a</strong> Phylogenetic Informativeness (PI) as estimated with TAPIR [<span xmlns="http://www.w3.org/1999/xhtml" class="CitationRef"><a href="#CR41" data-toggle="citation">41</a></span>]
 for the three UCE data sets and the 10-gene data set. PI is here 
plotted per nucleotide site as it increases with increasing age of 
divergence (in Ma) between taxa. <strong class="EmphasisTypeBold">b</strong>
 Graphic comparison of divergence time estimates for three BEAST 
analyses (UCE-100best, UCE-95&nbsp;% and 10-gene data set); node labels 
correspond with those of Tables&nbsp;<span xmlns="http://www.w3.org/1999/xhtml" class="InternalRef"><a href="#Tab2">2</a></span>, Additional file <span xmlns="http://www.w3.org/1999/xhtml" class="InternalRef"><a href="#MOESM11">11</a></span>, and Fig.&nbsp;<span xmlns="http://www.w3.org/1999/xhtml" class="InternalRef"><a href="#Fig4">4</a></span>
                                    </p>
                      </div></figcaption></figure>
                        </div>
              </section><section id="Sec15" class="Section2 RenderAsSection2">
                <h3 class="Heading">Taxa with uncertain relationships</h3>
                <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par44" class="Para">We identified five taxa (<em xmlns="" class="EmphasisTypeItalic">Gigantiops</em>, <em xmlns="" class="EmphasisTypeItalic">Myrmoteras, Oecophylla</em>, <em xmlns="" class="EmphasisTypeItalic">Gesomyrmex</em>, <em xmlns="" class="EmphasisTypeItalic">Santschiella</em>) that are subtended by very long branches in phylogenies resulting from analyses of both data sets (Fig.&nbsp;<span class="InternalRef"><a href="#Fig1">1</a></span>). No analysis of either data set is decisively able to resolve the precise positions within the subfamily of <em xmlns="" class="EmphasisTypeItalic">Gigantiops</em> or <em xmlns="" class="EmphasisTypeItalic">Santschiella</em>. Strongly supported by the UCE but not by the ten-gene data, however, are a sister-group relationship between <em xmlns="" class="EmphasisTypeItalic">Myrmoteras</em> and the tribe Camponotini (BS = 100) and a grouping of <em xmlns="" class="EmphasisTypeItalic">Gesomyrmex</em> and <em xmlns="" class="EmphasisTypeItalic">Oecophylla</em> as sister taxa (BS = 100).</p>
                <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par45" class="Para">We
 investigated the effect of these potential rogue taxa on tree topology,
 especially on the deep polytomy between subfamilies, and summarize 
results in Fig.&nbsp;<span class="InternalRef"><a href="#Fig2">2</a></span> (see also Additional file <span class="InternalRef"><a href="#MOESM10">10</a></span>).
 Excluding all five taxa resulted in a fully resolved, well-supported 
UCE phylogeny for the remaining six formicine lineages (Fig.&nbsp;<span class="InternalRef"><a href="#Fig2">2c</a></span>).
 This tree resolves the major polytomy with a relatively well-supported 
(BS = 93) sister-group relationship between Formicini and Camponotini, 
and with Plagiolepidini as the sister to (Formicini + Camponotini). In 
analyses of the 10-gene data set, in contrast, resolution of 
relationships between these tribes is only slightly improved by 
excluding the five problematic taxa (Fig.&nbsp;<span class="InternalRef"><a href="#Fig2">2d</a></span>).</p>
              </section><section id="Sec16" class="Section2 RenderAsSection2">
                <h3 class="Heading">Divergence dating and biogeographic analyses</h3>
                <div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par46" class="Para">With
 the exception of the positions of the five problematic or rogue taxa, 
analyses using BEAST produced results similar to those of other analyses
 with regard to topology. Figure&nbsp;<span class="InternalRef"><a href="#Fig4">4</a></span>
 depicts the time-calibrated phylogeny as estimated from the UCE-100best
 data set, with the ancestral ranges estimated by the S-DEC model in 
RASP mapped onto each node. Support values, median crown group ages, 
select highest posterior density intervals (95&nbsp;% HPD), and 
ancestral ranges are summarized in Table&nbsp;<span class="InternalRef"><a href="#Tab2">2</a></span> (see also Additional file <span class="InternalRef"><a href="#MOESM11">11</a></span>).
 Median age estimates and their 95&nbsp;% HPD intervals are relatively 
similar across the three BEAST analyses, with ages differing by 15 MY at
 most (node 140, Fig.&nbsp;<span class="InternalRef"><a href="#Fig3">3b</a></span> and Table&nbsp;<span class="InternalRef"><a href="#Tab2">2</a></span>).
 Overall the two UCE data sets estimate slightly younger ages than the 
10-gene data set. Ancestral range estimates under the two models (DEC 
and S-DEC) also mostly agree with each other (Fig.&nbsp;<span class="InternalRef"><a href="#Fig4">4</a></span> and Table&nbsp;<span class="InternalRef"><a href="#Tab2">2</a></span>).<figure xmlns="" class="Figure" id="Fig4"><div xmlns="http://www.w3.org/1999/xhtml" class="MediaObject" id="MO4">
                      <a target="_blank" rel="noopener" href="https://media.springernature.com/full/springer-static/image/art%3A10.1186%2Fs12862-015-0552-5/MediaObjects/12862_2015_552_Fig4_HTML.gif"><img class="c-lazy-image u-js-shown" src="12862_2015_552_Fig4_HTML.gif" alt="Fig. 4" data-lazy-load="true" data-src="https://media.springernature.com/lw785/springer-static/image/art%3A10.1186%2Fs12862-015-0552-5/MediaObjects/12862_2015_552_Fig4_HTML.gif"><noscript><img src="https://media.springernature.com/lw785/springer-static/image/art%3A10.1186%2Fs12862-015-0552-5/MediaObjects/12862_2015_552_Fig4_HTML.gif" alt="Fig. 4"></img></noscript></a>
                    </div><figcaption class="Caption" lang="en"><div xmlns="http://www.w3.org/1999/xhtml" class="CaptionContent"><span class="CaptionNumber">Fig. 4</span>
                        <p xmlns="" class="SimplePara">Time-calibrated 
phylogeny and ancestral range estimates for the subfamily Formicinae. 
Maximum clade credibility tree summarized from 48000 trees as estimated 
with the UCE-100best data set under a relaxed-clock model with nine 
fossil calibrations. Blue bars show the 95&nbsp;% highest posterior 
density range for each node. Node numbers refer to Table&nbsp;<span xmlns="http://www.w3.org/1999/xhtml" class="InternalRef"><a href="#Tab2">2</a></span> and Additional file <span xmlns="http://www.w3.org/1999/xhtml" class="InternalRef"><a href="#MOESM11">11</a></span>.
 Ancestral ranges estimated by S-DEC are mapped on MRCA nodes for each 
tribe and genus (regardless of the level of support) and all other nodes
 that received high support (&gt;70) for reconstructions. 
T = Neotropical, N = Nearctic, P = Palearctic, E = Afrotropical, 
O = Indomalayan and A = Australasian</p>
                      </div></figcaption></figure>
                           <figure xmlns="" id="Tab2" class="FigureTable"><figcaption class="Caption" lang="en"><div xmlns="http://www.w3.org/1999/xhtml" class="CaptionContent"><span class="CaptionNumber">Table 2</span>
                        <p xmlns="" class="SimplePara">Summary of crown group divergence ages and estimated ancestral ranges</p>
                      </div></figcaption><div class="Table"><table xmlns="http://www.w3.org/1999/xhtml" border="1"><colgroup><col><col><col><col><col><col><col><col><col><col></colgroup><thead><tr class="header"><th>&nbsp;</th><th><p xmlns="" class="SimplePara">Node</p></th><th><p xmlns="" class="SimplePara">PP</p></th><th><p xmlns="" class="SimplePara">10 gene</p></th><th><p xmlns="" class="SimplePara">UCE - 95&nbsp;%</p></th><th><p xmlns="" class="SimplePara">UCE - 100best</p></th><th colspan="2"><p xmlns="" class="SimplePara">S-DEC</p></th><th colspan="2"><p xmlns="" class="SimplePara">DEC</p></th></tr><tr class="header"><th>&nbsp;</th><th>&nbsp;</th><th>&nbsp;</th><th>&nbsp;</th><th>&nbsp;</th><th>&nbsp;</th><th><p xmlns="" class="SimplePara">Area</p></th><th><p xmlns="" class="SimplePara">Prob.</p></th><th><p xmlns="" class="SimplePara">Area</p></th><th><p xmlns="" class="SimplePara">Prob.</p></th></tr></thead><tbody><tr class="noclass"><td><p xmlns="" class="SimplePara">Formicinae</p></td><td><p xmlns="" class="SimplePara">163</p></td><td><p xmlns="" class="SimplePara">0.9</p></td><td><p xmlns="" class="SimplePara">117.6 [100–136]</p></td><td><p xmlns="" class="SimplePara">108.7 [92–127]</p></td><td><p xmlns="" class="SimplePara">104.1 [88–125]</p></td><td><p xmlns="" class="SimplePara">TPOA</p></td><td><p xmlns="" class="SimplePara">19.0</p></td><td><p xmlns="" class="SimplePara">TPEOA</p></td><td><p xmlns="" class="SimplePara">22.0</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">Camponotini</p></td><td><p xmlns="" class="SimplePara">134</p></td><td><p xmlns="" class="SimplePara">1</p></td><td><p xmlns="" class="SimplePara">55.8 [43–70]</p></td><td><p xmlns="" class="SimplePara">55.6 [43–70]</p></td><td><p xmlns="" class="SimplePara">51.4 [40–64]</p></td><td><p xmlns="" class="SimplePara">O</p></td><td><p xmlns="" class="SimplePara">43.4</p></td><td><p xmlns="" class="SimplePara">O</p></td><td><p xmlns="" class="SimplePara">45.5</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">Formicini</p></td><td><p xmlns="" class="SimplePara">158</p></td><td><p xmlns="" class="SimplePara">1</p></td><td><p xmlns="" class="SimplePara">66.7 [55–80]</p></td><td><p xmlns="" class="SimplePara">64.3 [54–75]</p></td><td><p xmlns="" class="SimplePara">65.9 [55–79]</p></td><td><p xmlns="" class="SimplePara">
                              <strong class="EmphasisTypeBold">P</strong>
                            </p></td><td><p xmlns="" class="SimplePara">
                              <strong class="EmphasisTypeBold">89.9</strong>
                            </p></td><td><p xmlns="" class="SimplePara">
                              <strong class="EmphasisTypeBold">P</strong>
                            </p></td><td><p xmlns="" class="SimplePara">
                              <strong class="EmphasisTypeBold">100.0</strong>
                            </p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">Lasiini</p></td><td><p xmlns="" class="SimplePara">100</p></td><td><p xmlns="" class="SimplePara">1</p></td><td><p xmlns="" class="SimplePara">88.9 [72–106]</p></td><td><p xmlns="" class="SimplePara">77.4 [65–92]</p></td><td><p xmlns="" class="SimplePara">76.7 [65–91]</p></td><td><p xmlns="" class="SimplePara">O</p></td><td><p xmlns="" class="SimplePara">46.5</p></td><td><p xmlns="" class="SimplePara">O</p></td><td><p xmlns="" class="SimplePara">49.2</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">Melophorini</p></td><td><p xmlns="" class="SimplePara">108</p></td><td><p xmlns="" class="SimplePara">1</p></td><td><p xmlns="" class="SimplePara">62.9 [43–85]</p></td><td><p xmlns="" class="SimplePara">64.7 [49–84]</p></td><td><p xmlns="" class="SimplePara">52.6 [41–66]</p></td><td><p xmlns="" class="SimplePara">
                              <strong class="EmphasisTypeBold">A</strong>
                            </p></td><td><p xmlns="" class="SimplePara">
                              <strong class="EmphasisTypeBold">99.9</strong>
                            </p></td><td><p xmlns="" class="SimplePara">
                              <strong class="EmphasisTypeBold">A</strong>
                            </p></td><td><p xmlns="" class="SimplePara">
                              <strong class="EmphasisTypeBold">100.0</strong>
                            </p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">Myrmelachistini</p></td><td><p xmlns="" class="SimplePara">84</p></td><td><p xmlns="" class="SimplePara">1</p></td><td><p xmlns="" class="SimplePara">80.0 [47–112]</p></td><td><p xmlns="" class="SimplePara">75.7 [50–103]</p></td><td><p xmlns="" class="SimplePara">72.2 [45–101]</p></td><td><p xmlns="" class="SimplePara">
                              <strong class="EmphasisTypeBold">T</strong>
                            </p></td><td><p xmlns="" class="SimplePara">
                              <strong class="EmphasisTypeBold">95.3</strong>
                            </p></td><td><p xmlns="" class="SimplePara">
                              <strong class="EmphasisTypeBold">T</strong>
                            </p></td><td><p xmlns="" class="SimplePara">
                              <strong class="EmphasisTypeBold">100.0</strong>
                            </p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">Plagiolepidini</p></td><td><p xmlns="" class="SimplePara">149</p></td><td><p xmlns="" class="SimplePara">1</p></td><td><p xmlns="" class="SimplePara">84.4 [71–99]</p></td><td><p xmlns="" class="SimplePara">76.2 [66–88]</p></td><td><p xmlns="" class="SimplePara">74.0 [65–83]</p></td><td><p xmlns="" class="SimplePara">EO</p></td><td><p xmlns="" class="SimplePara">52.9</p></td><td><p xmlns="" class="SimplePara">E</p></td><td><p xmlns="" class="SimplePara">51.2</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">
                              <em class="EmphasisTypeItalic">Camponotus s.s.</em>
                            </p></td><td><p xmlns="" class="SimplePara">126</p></td><td><p xmlns="" class="SimplePara">1</p></td><td><p xmlns="" class="SimplePara">27.6</p></td><td><p xmlns="" class="SimplePara">23.8</p></td><td><p xmlns="" class="SimplePara">23.1</p></td><td><p xmlns="" class="SimplePara">TEO</p></td><td><p xmlns="" class="SimplePara">26.9</p></td><td><p xmlns="" class="SimplePara">PEO</p></td><td><p xmlns="" class="SimplePara">26.7</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">
                              <em class="EmphasisTypeItalic">Colobopsis</em>
                            </p></td><td><p xmlns="" class="SimplePara">115</p></td><td><p xmlns="" class="SimplePara">1</p></td><td><p xmlns="" class="SimplePara">34.2</p></td><td><p xmlns="" class="SimplePara">32.7</p></td><td><p xmlns="" class="SimplePara">28.5</p></td><td><p xmlns="" class="SimplePara">O</p></td><td><p xmlns="" class="SimplePara">31.5</p></td><td><p xmlns="" class="SimplePara">O</p></td><td><p xmlns="" class="SimplePara">27.7</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">
                              <em class="EmphasisTypeItalic">Echinopla</em>
                            </p></td><td><p xmlns="" class="SimplePara">127</p></td><td><p xmlns="" class="SimplePara">1</p></td><td><p xmlns="" class="SimplePara">13.8</p></td><td><p xmlns="" class="SimplePara">10.5</p></td><td><p xmlns="" class="SimplePara">10.3</p></td><td><p xmlns="" class="SimplePara">A</p></td><td><p xmlns="" class="SimplePara">51.4</p></td><td><p xmlns="" class="SimplePara">A</p></td><td><p xmlns="" class="SimplePara">50.1</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">
                              <em class="EmphasisTypeItalic">Calomyrmex</em>
                            </p></td><td><p xmlns="" class="SimplePara">128</p></td><td><p xmlns="" class="SimplePara">1</p></td><td><p xmlns="" class="SimplePara">13.9</p></td><td><p xmlns="" class="SimplePara">7.9</p></td><td><p xmlns="" class="SimplePara">7.9</p></td><td><p xmlns="" class="SimplePara">
                              <strong class="EmphasisTypeBold">A</strong>
                            </p></td><td><p xmlns="" class="SimplePara">
                              <strong class="EmphasisTypeBold">100.0</strong>
                            </p></td><td><p xmlns="" class="SimplePara">
                              <strong class="EmphasisTypeBold">A</strong>
                            </p></td><td><p xmlns="" class="SimplePara">
                              <strong class="EmphasisTypeBold">100.0</strong>
                            </p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">
                              <em class="EmphasisTypeItalic">Polyrhachis</em>
                            </p></td><td><p xmlns="" class="SimplePara">117</p></td><td><p xmlns="" class="SimplePara">1</p></td><td><p xmlns="" class="SimplePara">14.2</p></td><td><p xmlns="" class="SimplePara">16.0</p></td><td><p xmlns="" class="SimplePara">13.5</p></td><td><p xmlns="" class="SimplePara">OA</p></td><td><p xmlns="" class="SimplePara">38.1</p></td><td><p xmlns="" class="SimplePara">OA</p></td><td><p xmlns="" class="SimplePara">40.8</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">
                              <em class="EmphasisTypeItalic">Opisthopsis</em>
                            </p></td><td><p xmlns="" class="SimplePara">116</p></td><td><p xmlns="" class="SimplePara">1</p></td><td><p xmlns="" class="SimplePara">6.2</p></td><td><p xmlns="" class="SimplePara">4.0</p></td><td><p xmlns="" class="SimplePara">3.2</p></td><td><p xmlns="" class="SimplePara">
                              <strong class="EmphasisTypeBold">A</strong>
                            </p></td><td><p xmlns="" class="SimplePara">
                              <strong class="EmphasisTypeBold">100.0</strong>
                            </p></td><td><p xmlns="" class="SimplePara">
                              <strong class="EmphasisTypeBold">A</strong>
                            </p></td><td><p xmlns="" class="SimplePara">
                              <strong class="EmphasisTypeBold">100.0</strong>
                            </p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">Formicine_genus01</p></td><td><p xmlns="" class="SimplePara">143</p></td><td><p xmlns="" class="SimplePara">1</p></td><td><p xmlns="" class="SimplePara">8.3</p></td><td><p xmlns="" class="SimplePara">9.9</p></td><td><p xmlns="" class="SimplePara">10.0</p></td><td><p xmlns="" class="SimplePara">
                              <strong class="EmphasisTypeBold">E</strong>
                            </p></td><td><p xmlns="" class="SimplePara">
                              <strong class="EmphasisTypeBold">100.0</strong>
                            </p></td><td><p xmlns="" class="SimplePara">
                              <strong class="EmphasisTypeBold">E</strong>
                            </p></td><td><p xmlns="" class="SimplePara">
                              <strong class="EmphasisTypeBold">100.0</strong>
                            </p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">
                              <em class="EmphasisTypeItalic">Tapinolepis</em>
                            </p></td><td><p xmlns="" class="SimplePara">138</p></td><td><p xmlns="" class="SimplePara">1</p></td><td><p xmlns="" class="SimplePara">57.88<sup xmlns="http://www.w3.org/1999/xhtml">a</sup>
                                          </p></td><td><p xmlns="" class="SimplePara">56.1</p></td><td><p xmlns="" class="SimplePara">45.5</p></td><td><p xmlns="" class="SimplePara">
                              <strong class="EmphasisTypeBold">E</strong>
                            </p></td><td><p xmlns="" class="SimplePara">
                              <strong class="EmphasisTypeBold">100.0</strong>
                            </p></td><td><p xmlns="" class="SimplePara">
                              <strong class="EmphasisTypeBold">E</strong>
                            </p></td><td><p xmlns="" class="SimplePara">
                              <strong class="EmphasisTypeBold">100.0</strong>
                            </p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">
                              <em class="EmphasisTypeItalic">Acropyga</em>
                            </p></td><td><p xmlns="" class="SimplePara">145</p></td><td><p xmlns="" class="SimplePara">1</p></td><td><p xmlns="" class="SimplePara">27.7</p></td><td><p xmlns="" class="SimplePara">30.0</p></td><td><p xmlns="" class="SimplePara">23.9</p></td><td><p xmlns="" class="SimplePara">E</p></td><td><p xmlns="" class="SimplePara">31.6</p></td><td><p xmlns="" class="SimplePara">E</p></td><td><p xmlns="" class="SimplePara">17.3</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">
                              <em class="EmphasisTypeItalic">Plagiolepis</em>
                            </p></td><td><p xmlns="" class="SimplePara">140</p></td><td><p xmlns="" class="SimplePara">1</p></td><td><p xmlns="" class="SimplePara">26.1</p></td><td><p xmlns="" class="SimplePara">11.2</p></td><td><p xmlns="" class="SimplePara">9.6</p></td><td><p xmlns="" class="SimplePara">PEOA</p></td><td><p xmlns="" class="SimplePara">48.7</p></td><td><p xmlns="" class="SimplePara">PEOA</p></td><td><p xmlns="" class="SimplePara">51.6</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">
                              <em class="EmphasisTypeItalic">Anoplolepis</em>
                            </p></td><td><p xmlns="" class="SimplePara">136</p></td><td><p xmlns="" class="SimplePara">1</p></td><td><p xmlns="" class="SimplePara">31.8</p></td><td><p xmlns="" class="SimplePara">36.4</p></td><td><p xmlns="" class="SimplePara">29.7</p></td><td><p xmlns="" class="SimplePara">
                              <strong class="EmphasisTypeBold">EO</strong>
                            </p></td><td><p xmlns="" class="SimplePara">
                              <strong class="EmphasisTypeBold">99.1</strong>
                            </p></td><td><p xmlns="" class="SimplePara">
                              <strong class="EmphasisTypeBold">EO</strong>
                            </p></td><td><p xmlns="" class="SimplePara">
                              <strong class="EmphasisTypeBold">100.0</strong>
                            </p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">
                              <em class="EmphasisTypeItalic">Oecophylla</em>
                            </p></td><td><p xmlns="" class="SimplePara">110</p></td><td><p xmlns="" class="SimplePara">1</p></td><td><p xmlns="" class="SimplePara">9.2</p></td><td><p xmlns="" class="SimplePara">13.2</p></td><td><p xmlns="" class="SimplePara">13.2</p></td><td><p xmlns="" class="SimplePara">
                              <strong class="EmphasisTypeBold">EOA</strong>
                            </p></td><td><p xmlns="" class="SimplePara">
                              <strong class="EmphasisTypeBold">75.9</strong>
                            </p></td><td><p xmlns="" class="SimplePara">
                              <strong class="EmphasisTypeBold">EOA</strong>
                            </p></td><td><p xmlns="" class="SimplePara">
                              <strong class="EmphasisTypeBold">78.1</strong>
                            </p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">
                              <em class="EmphasisTypeItalic">Gesomyrmex</em>
                            </p></td><td><p xmlns="" class="SimplePara">109</p></td><td><p xmlns="" class="SimplePara">1</p></td><td><p xmlns="" class="SimplePara">3.7</p></td><td><p xmlns="" class="SimplePara">8.8</p></td><td><p xmlns="" class="SimplePara">4.2</p></td><td><p xmlns="" class="SimplePara">
                              <strong class="EmphasisTypeBold">O</strong>
                            </p></td><td><p xmlns="" class="SimplePara">
                              <strong class="EmphasisTypeBold">100.0</strong>
                            </p></td><td><p xmlns="" class="SimplePara">
                              <strong class="EmphasisTypeBold">O</strong>
                            </p></td><td><p xmlns="" class="SimplePara">
                              <strong class="EmphasisTypeBold">100.0</strong>
                            </p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">
                              <em class="EmphasisTypeItalic">Formica</em>
                            </p></td><td><p xmlns="" class="SimplePara">152</p></td><td><p xmlns="" class="SimplePara">1</p></td><td><p xmlns="" class="SimplePara">28.7</p></td><td><p xmlns="" class="SimplePara">40.6</p></td><td><p xmlns="" class="SimplePara">28.6</p></td><td><p xmlns="" class="SimplePara">NP</p></td><td><p xmlns="" class="SimplePara">68.2</p></td><td><p xmlns="" class="SimplePara">
                              <strong class="EmphasisTypeBold">NP</strong>
                            </p></td><td><p xmlns="" class="SimplePara">
                              <strong class="EmphasisTypeBold">71.8</strong>
                            </p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">
                              <em class="EmphasisTypeItalic">Paratrechina</em>
                            </p></td><td><p xmlns="" class="SimplePara">92</p></td><td><p xmlns="" class="SimplePara">1</p></td><td><p xmlns="" class="SimplePara">14.8</p></td><td><p xmlns="" class="SimplePara">18.5</p></td><td><p xmlns="" class="SimplePara">14.0</p></td><td><p xmlns="" class="SimplePara">
                              <strong class="EmphasisTypeBold">E</strong>
                            </p></td><td><p xmlns="" class="SimplePara">
                              <strong class="EmphasisTypeBold">100.0</strong>
                            </p></td><td><p xmlns="" class="SimplePara">
                              <strong class="EmphasisTypeBold">E</strong>
                            </p></td><td><p xmlns="" class="SimplePara">
                              <strong class="EmphasisTypeBold">100.0</strong>
                            </p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">
                              <em class="EmphasisTypeItalic">Nylanderia s.l.</em>
                            </p></td><td><p xmlns="" class="SimplePara">90</p></td><td><p xmlns="" class="SimplePara">1</p></td><td><p xmlns="" class="SimplePara">24.3</p></td><td><p xmlns="" class="SimplePara">24.2</p></td><td><p xmlns="" class="SimplePara">24.2</p></td><td><p xmlns="" class="SimplePara">TEO</p></td><td><p xmlns="" class="SimplePara">22.9</p></td><td><p xmlns="" class="SimplePara">TEO</p></td><td><p xmlns="" class="SimplePara">22.3</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">
                              <em class="EmphasisTypeItalic">Paraparatrechina</em>
                            </p></td><td><p xmlns="" class="SimplePara">85</p></td><td><p xmlns="" class="SimplePara">1</p></td><td><p xmlns="" class="SimplePara">31.3</p></td><td><p xmlns="" class="SimplePara">31.4</p></td><td><p xmlns="" class="SimplePara">26.8</p></td><td><p xmlns="" class="SimplePara">EO</p></td><td><p xmlns="" class="SimplePara">42.1</p></td><td><p xmlns="" class="SimplePara">EO</p></td><td><p xmlns="" class="SimplePara">40.4</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">
                              <em class="EmphasisTypeItalic">Lasius</em>
                            </p></td><td><p xmlns="" class="SimplePara">98</p></td><td><p xmlns="" class="SimplePara">1</p></td><td><p xmlns="" class="SimplePara">25.48<sup xmlns="http://www.w3.org/1999/xhtml">a</sup>
                                          </p></td><td><p xmlns="" class="SimplePara">21.18<sup xmlns="http://www.w3.org/1999/xhtml">a</sup>
                                          </p></td><td><p xmlns="" class="SimplePara">21.29<sup xmlns="http://www.w3.org/1999/xhtml">a</sup>
                                          </p></td><td><p xmlns="" class="SimplePara">
                              <strong class="EmphasisTypeBold">NP</strong>
                            </p></td><td><p xmlns="" class="SimplePara">
                              <strong class="EmphasisTypeBold">76.5</strong>
                            </p></td><td><p xmlns="" class="SimplePara">
                              <strong class="EmphasisTypeBold">NP</strong>
                            </p></td><td><p xmlns="" class="SimplePara">
                              <strong class="EmphasisTypeBold">81.8</strong>
                            </p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">
                              <em class="EmphasisTypeItalic">Brachymyrmex</em>
                            </p></td><td><p xmlns="" class="SimplePara">83</p></td><td><p xmlns="" class="SimplePara">1</p></td><td><p xmlns="" class="SimplePara">23.6</p></td><td><p xmlns="" class="SimplePara">28.6</p></td><td><p xmlns="" class="SimplePara">21.1</p></td><td><p xmlns="" class="SimplePara">T</p></td><td><p xmlns="" class="SimplePara">62.6</p></td><td><p xmlns="" class="SimplePara">T</p></td><td><p xmlns="" class="SimplePara">63.0</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">
                              <em class="EmphasisTypeItalic">Lepisiota</em>
                            </p></td><td><p xmlns="" class="SimplePara">141</p></td><td><p xmlns="" class="SimplePara">0.93</p></td><td><p xmlns="" class="SimplePara">49.51<sup xmlns="http://www.w3.org/1999/xhtml">a</sup>
                                          </p></td><td><p xmlns="" class="SimplePara">44.2</p></td><td><p xmlns="" class="SimplePara">42.9</p></td><td><p xmlns="" class="SimplePara">
                              <strong class="EmphasisTypeBold">E</strong>
                            </p></td><td><p xmlns="" class="SimplePara">
                              <strong class="EmphasisTypeBold">99.7</strong>
                            </p></td><td><p xmlns="" class="SimplePara">
                              <strong class="EmphasisTypeBold">E</strong>
                            </p></td><td><p xmlns="" class="SimplePara">
                              <strong class="EmphasisTypeBold">100.0</strong>
                            </p></td></tr></tbody></table></div><div xmlns="http://www.w3.org/1999/xhtml" class="TableFooter">
                      <p xmlns="" class="SimplePara">Table summarizing 
median crown group ages (in Ma, rounded to the first decimal) for 
selected formicine clades as estimated by BEAST analyses of different 
data sets. Bracketed numbers represent 95&nbsp;% HPD (highest posterior 
density) intervals for selected nodes, rounded to the nearest integer. 
PP = posterior probability as estimated from the UCE-100best data set; <sup xmlns="http://www.w3.org/1999/xhtml">a</sup>
 indicates this genus is not recovered as monophyletic in the particular
 analysis. Crown group ancestral ranges are further shown as estimated 
with the DEC and S-DEC models implemented in RASP for selected clades, 
bolded for probabilities &gt; 75. Node numbers correspond to Fig.&nbsp;<span xmlns="http://www.w3.org/1999/xhtml" class="InternalRef"><a href="#Fig3">3</a></span>;
 only ranges with highest probability are shown. T = Neotropical, 
N = Neartic, P = Palearctic, E = Afrotropical, O = South-East Asian, 
A = Australian</p>
                    </div></figure>
                        </div>
                <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par47" class="Para">Crown-group
 Formicinae are estimated to have evolved in the early Cretaceous, 
between 104.1–117.6&nbsp;Ma. Ancestral range models estimate a very 
broad distribution range (TPOA/TPEOA; Table&nbsp;<span class="InternalRef"><a href="#Tab2">2</a></span> and Fig.&nbsp;<span class="InternalRef"><a href="#Fig4">4</a></span>)
 for the most recent common ancestor (MRCA) of the Formicinae, although 
without much support. The six larger formicine tribes diversified 
throughout the late Cretaceous, Paleocene, and early Eocene, between 
51–88.9&nbsp;Ma (Fig.&nbsp;<span class="InternalRef"><a href="#Fig4">4</a></span> and Table&nbsp;<span class="InternalRef"><a href="#Tab2">2</a></span>).
 The ancestral range analyses did not provide much support for ancient 
dispersal events (nodes 159–162) leading to the current distribution of 
these lineages, in accord with the uncertainty of phylogenetic 
relationships between them. The Lasiini are the oldest crown formicine 
lineage (76.7–88.9&nbsp;Ma, node 100) and share an Indomalayan ancestor.
 Ancestral reconstructions and dispersal within this presently global 
lineage were not well supported. The sister group to all other 
Formicinae, the Myrmelachistini, is estimated to have a Neotropical 
origin between 72.2–80&nbsp;Ma (node 84). Our analyses suggest that the 
Plagiolepidini evolved around the same time (76.7–88.9&nbsp;Ma), but on a
 different continent: the Afrotropical and Indomalayan regions are 
reconstructed as ancestral ranges for crown-group Plagiolepidini. A 
Palearctic origin is further suggested for the Formicini in the 
Paleocene (64.3–66.7&nbsp;Ma), while an early Eocene origin 
(51.4–55.8&nbsp;Ma) of the Camponotini in the Indomalayan region 
received moderate support. Age estimates for crown-group Melophorini 
range from 52.6–62.9&nbsp;Ma with an Australasian origin. For extant 
Formicinae genera, our crown-group estimates range from 
3.2–56.1&nbsp;Ma. Notably, the oldest genera within the Formicinae are <em xmlns="" class="EmphasisTypeItalic">Lepisiota</em> (42.9–44.2&nbsp;Ma, node 141) and <em xmlns="" class="EmphasisTypeItalic">Tapinolepis</em> (45.5–56.1&nbsp;Ma, node 138), while <em xmlns="" class="EmphasisTypeItalic">Gesomyrmex</em> (4.2–8.8&nbsp;Ma) and <em xmlns="" class="EmphasisTypeItalic">Opisthopsis</em> (3.2–6.2&nbsp;Ma) are recovered as the youngest lineages.</p>
              </section></div></section>
            <section xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" id="Sec17" class="Section1 RenderAsSection1 c-section"><h2 class="Heading js-ToggleCollapseSection c-section__heading" data-sticky-update="true"><a href="#Discussion" data-toggle="collapse" data-toggle-disable-clickoff="" data-target="Discussion">Discussion</a></h2><div class="js-CollapseSection CollapseSection c-section__content" id="Discussion" data-test="collapse-section"><section id="Sec18" class="Section2 RenderAsSection2">
                <h3 class="Heading">Comparison of UCE vs multi-locus methods</h3>
                <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par48" class="Para">We
 reconstruct the evolution of the subfamily Formicinae based upon a 
next-generation, pan-genomic data set of UCEs, and provide a direct 
comparison of this targeted-enrichment phylogenomic approach to a much 
smaller traditional phylogenetic data set assembled by Sanger-sequencing
 methods using the same set of 82 exemplar species. The Sanger data set 
was 100&nbsp;% complete without missing data, while the UCE data set 
used for comparison was only 70&nbsp;% complete. Our results clearly 
demonstrate the advantage of using the nearly 1000 UCE loci over using 
10 genes to resolve formicine relationships. Only five nodes have less 
than 70&nbsp;% bootstrap support in the UCE phylogeny (Fig.&nbsp;<span class="InternalRef"><a href="#Fig1">1a</a></span>),
 whereas 28 (out of 85) of the nodes in the 10-gene phylogeny are poorly
 supported (BS &lt; 70). Such increased support in the UCE phylogeny 
compared to the 10-gene phylogeny is perhaps unsurprising, given the 
different scales of the data (~590000&nbsp;bp vs ~7200&nbsp;bp, 
Table&nbsp;<span class="InternalRef"><a href="#Tab1">1</a></span>). 
Furthermore, the superiority of the UCEs over the nuclear loci is not 
merely a function of sequence length, but can also be attributed to 
higher phylogenetic informativeness (PI). The full UCE-70&nbsp;% data 
set has nearly double the PI relative to the 10-gene data set, while 
filtering of the UCE data by average bootstrap support (UCE-100best) 
raised PI to a level about 2.5 times higher. These metrics are congruent
 with estimates from a recent study comparing phylogenetic 
informativeness across ten single-copy nuclear genes with UCE core and 
flanking regions [<span class="CitationRef"><a href="#CR8" data-toggle="citation">8</a></span>].</p>
                <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par49" class="Para">The
 remaining uncertainty in the UCE phylogeny could well be heavily 
influenced by the presence of the five problematic taxa subtended by 
long branches. Conversely, however, it should be stressed that although 
the exclusions of these taxa increase support for the remaining 
relationships, these exclusions could simultaneously lead to a decrease 
in phylogenetic accuracy due to less complete taxon sampling, and thus 
these results should not necessarily be interpreted as improved 
estimates of phylogenetic relationships (but see [<span class="CitationRef"><a href="#CR47" data-toggle="citation">47</a></span>]).</p>
              </section><section id="Sec19" class="Section2 RenderAsSection2">
                <h3 class="Heading">Dating with UCEs</h3>
                <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par50" class="Para">To
 our knowledge, divergence ages based on UCEs have never been 
systematically compared to those estimated from other types of data, 
such as our ten-gene nuclear sequence data. It is possible that 
functional differences between these two types of data may lead to 
incompatible branch length estimation. All of our Sanger data were 
protein-coding or ribosomal DNA sequence from nuclear genes, whereas 
many UCE loci in general do not overlap with protein-coding regions, but
 rather appear to act as enhancers or splicing regulators [<span class="CitationRef"><a href="#CR48" data-toggle="citation">48</a></span>].
 Methods that jointly estimate divergence ages and tree topology, such 
as BEAST, have further seldom been employed to date with UCE or other 
genomic-scale data due to computational constraints. We overcame this 
limitation by filtering our data to a manageable size (i.e. 100 and 50 
loci), and are thus able to compare for the first time age estimates 
derived from UCEs to those derived from our ten-gene nuclear data. For 
most nodes, the 10-gene data set estimated slightly older ages than the 
UCE data sets (Fig.&nbsp;<span class="InternalRef"><a href="#Fig3">3b</a></span> and Table&nbsp;<span class="InternalRef"><a href="#Tab2">2</a></span>).
 This is likely due to the high variance of evolutionary rates across 
loci included in the data sets, including the loci in the two different 
UCE data sets. Variance between estimates, however, is still much 
smaller than, for example, the 95&nbsp;% HPC intervals around any of 
these age estimates (Table&nbsp;<span class="InternalRef"><a href="#Tab2">2</a></span> and Fig.&nbsp;<span class="InternalRef"><a href="#Fig4">4</a></span>),
 suggesting that variance across loci is not the only factor influencing
 the differences in age estimates. We conclude that divergence dating 
with UCEs is both feasible and promising, and needs to be explored 
further as current methods and handling of genome-scale data sets 
continue to improve.</p>
              </section><section id="Sec20" class="Section2 RenderAsSection2">
                <h3 class="Heading">Practical considerations of UCEs vs Sanger sequencing</h3>
                <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par51" class="Para">While
 most researchers would agree that our results clearly show the 
advantage of using the nearly 1000 UCE loci over the 10-gene data set 
for phylogenetic inference, there are also practical aspects to consider
 regarding the cost and time spent in obtaining these data sets. We do 
not aim to provide a detailed analysis here, because both time and cost 
factors are highly variable and dependent on, e.g., sample DNA quality, 
available laboratory supplies, accumulated experience with a given 
technique, and sequencing cost at the respective genomic facility used. 
In our case, however, we found that the cost and time to generate both 
of these data sets are similar. Labor time associated with 
next-generation library preparation and target enrichment for UCEs for 
90 taxa (~3&nbsp;weeks for one full-time person) is roughly the same as 
for a single attempt at PCR-amplification and cycle sequencing of 10 
individual genes, if not less. Processing time of the sequence data 
through the bioinformatics pipeline further is negligible compared to 
the time spent editing individual sequences. Cost of supplies and 
sequencing to generate ~1000 UCE loci can be as low as ~ $40–60/sample, 
compared to an estimate of $5/sample/gene fragment for PCR 
amplifications and Sanger sequencing. Thus, from our perspective, we 
found the UCE methodology comparable with regard to cost and time input 
and superior in terms of data output when compared to Sanger sequencing.</p>
              </section><section id="Sec21" class="Section2 RenderAsSection2">
                <h3 class="Heading">Implications for formicine systematics</h3>
                <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par52" class="Para">Based on our UCE phylogeny, we propose several taxonomic changes at the tribal level (see Additional file <span class="InternalRef"><a href="#MOESM7">7</a></span>)
 for the subfamily Formicinae that aim to improve ant systematics while 
simultaneously keeping names fairly stable. These results partly agree 
with Bolton’s [<span class="CitationRef"><a href="#CR49" data-toggle="citation">49</a></span>]
 prior system of formicine tribes based on morphology, although major 
changes have to be made in the compositions of Lasiini, Melophorini and 
Plagiolepidini, and the Myrmelachistini must be resurrected.</p>
                <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par53" class="Para">The
 five problematic taxa were previously unassociated with any of the 
larger clades, and to some extent this ambiguity persists. However, the 
UCE data firmly support the close relationship of <em xmlns="" class="EmphasisTypeItalic">Gesomyrmex</em> and <em xmlns="" class="EmphasisTypeItalic">Oecophylla</em>, and of <em xmlns="" class="EmphasisTypeItalic">Myrmoteras</em>
 as the sister group of Camponotini; these relationships are poorly 
supported by the 10-gene data set. The phylogenetic positions of <em xmlns="" class="EmphasisTypeItalic">Gigantiops</em> and <em xmlns="" class="EmphasisTypeItalic">Santschiella</em> remain less clear, although the UCE data provide some support for a close relationship of these two taxa with Camponotini + <em xmlns="" class="EmphasisTypeItalic">Myrmoteras</em>. Ancient radiation events are common throughout the insect tree of life [<span class="CitationRef"><a href="#CR50" data-toggle="citation">50</a></span>], and other phylogenomic-scale studies have tried to resolve these with varying success ([e.g., [<span class="CitationRef"><a href="#CR51" data-toggle="citation">51</a></span><span>, <span class="CitationRef"><a href="#CR52" data-toggle="citation">52</a></span>, </span><span class="CitationRef"><a href="#CR53" data-toggle="citation">53</a></span>]).
 Deep lineage diversification within the Formicinae appears to have 
occurred very rapidly, over a period of 10–12 MY in the Cretaceous 
(Fig.&nbsp;<span class="InternalRef"><a href="#Fig4">4</a></span>), and 
may challenge the information content of even phylogenomic data. In 
addition, although our sampling comprised representatives of nearly all 
extant formicine genera, our results could have been influenced by 
limited taxon sampling within these lineages, and thus increased taxon 
sampling may be able to improve phylogenetic resolution.</p>
                <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par54" class="Para">Our analyses recovered four formicine genera as non-monophyletic: <em xmlns="" class="EmphasisTypeItalic">Nylanderia</em>, <em xmlns="" class="EmphasisTypeItalic">Prenolepis</em>, <em xmlns="" class="EmphasisTypeItalic">Lepisiota,</em> and <em xmlns="" class="EmphasisTypeItalic">Camponotus</em>.
 Additional taxon sampling will be necessary to resolve the generic 
limits of the first three, although other unpublished data suggest that <em xmlns="" class="EmphasisTypeItalic">Prenolepis emmae</em> may be misplaced in <em xmlns="" class="EmphasisTypeItalic">Prenolepis</em> and actually belongs in <em xmlns="" class="EmphasisTypeItalic">Nylanderia</em>
 (J. LaPolla, pers. comm.). We propose taxonomic changes here only for 
the carpenter ants (Ward et al., in review; see also Additional file <span class="InternalRef"><a href="#MOESM7">7</a></span>), <em xmlns="" class="EmphasisTypeItalic">Camponotus</em>, a genus for which paraphyly has been repeatedly indicated [<span class="CitationRef"><a href="#CR16" data-toggle="citation">16</a></span>, <span class="CitationRef"><a href="#CR53" data-toggle="citation">53</a></span><span>, <span class="CitationRef"><a href="#CR54" data-toggle="citation">54</a></span>, <span class="CitationRef"><a href="#CR55" data-toggle="citation">55</a></span>, </span><span class="CitationRef"><a href="#CR56" data-toggle="citation">56</a></span>]. Based on a strongly supported, well-sampled phylogeny, we resurrect the genera <em xmlns="" class="EmphasisTypeItalic">Colobopsis</em> and <em xmlns="" class="EmphasisTypeItalic">Dinomyrmex</em> for the two divergent lineages, and redefine <em xmlns="" class="EmphasisTypeItalic">Camponotus</em> to include <em xmlns="" class="EmphasisTypeItalic">Forelophilus</em> and <em xmlns="" class="EmphasisTypeItalic">Phasmomyrmex</em>, thus making it monophyletic (Additional file <span class="InternalRef"><a href="#MOESM7">7</a></span>). We found <em xmlns="" class="EmphasisTypeItalic">Colobopsis</em> to be well separated from other <em xmlns="" class="EmphasisTypeItalic">Camponotus</em> and sister to all other Camponotini, a result mirrored by phylogenetic analyses of their obligate bacterial endosymbionts, <em xmlns="" class="EmphasisTypeItalic">Blochmannia</em>, unique to Camponotini [<span class="CitationRef"><a href="#CR57" data-toggle="citation">57</a></span>]. The newly discovered sister relationship of <em xmlns="" class="EmphasisTypeItalic">Myrmoteras</em> with Camponotini now raises the intriguing question of whether the former also harbor <em xmlns="" class="EmphasisTypeItalic">Blochmannia</em> or related endosymbionts. Remarkably, we found the genera known to harbor the pumiliotoxins (<em xmlns="" class="EmphasisTypeItalic">Brachymyrmex</em> and <em xmlns="" class="EmphasisTypeItalic">Paratrechina</em>) sequestered by dendrobatid poison frogs [<span class="CitationRef"><a href="#CR22" data-toggle="citation">22</a></span>]
 to be part of the two earliest branching lineages within the 
Formicinae, Myrmelachistini and Lasiini. This interesting pattern calls 
for a wider sampling and thorough investigation of these chemicals 
throughout the subfamily.</p>
              </section><section id="Sec22" class="Section2 RenderAsSection2">
                <h3 class="Heading">Formicine biogeography</h3>
                <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par55" class="Para">Our
 dating analyses extend formicine evolution deep into the Cretaceous 
(104.1–117.6&nbsp;Ma). These median crown-group age estimates are 
considerably older than the fossil record suggests, with <em xmlns="" class="EmphasisTypeItalic">Kyromyrma</em>
 (~92&nbsp;Ma), the oldest known stem-group formicine fossil, relatively
 older than previous molecular dating estimates for the subfamily 
(77–83&nbsp;Ma, [<span class="CitationRef"><a href="#CR9" data-toggle="citation">9</a></span>]; 80–100&nbsp;Ma, [<span class="CitationRef"><a href="#CR10" data-toggle="citation">10</a></span>]; 75–90&nbsp;Ma, [<span class="CitationRef"><a href="#CR16" data-toggle="citation">16</a></span>]). The origin of the ant subfamily Myrmicinae was likewise recently estimated to be about 10 MY older than previous estimates [<span class="CitationRef"><a href="#CR14" data-toggle="citation">14</a></span>]. Divergence dating analyses can be sensitive with regard to incorrectly placed fossil calibrations [<span class="CitationRef"><a href="#CR58" data-toggle="citation">58</a></span><span>, <span class="CitationRef"><a href="#CR59" data-toggle="citation">59</a></span>, </span><span class="CitationRef"><a href="#CR60" data-toggle="citation">60</a></span>],
 but our analyses, sampling from the prior, show no indication of 
detrimental interactions between calibration priors. Another possibility
 is that an imbalance of ingroup vs. outgroup sampling and a lack of 
calibrations in the outgroup part of the phylogeny may be driving our 
age estimates, although we used outgroup taxa very similar to those in 
previous subfamily-level studies [<span class="CitationRef"><a href="#CR11" data-toggle="citation">11</a></span>, <span class="CitationRef"><a href="#CR12" data-toggle="citation">12</a></span>, <span class="CitationRef"><a href="#CR14" data-toggle="citation">14</a></span>].
 Conversely, our estimates may present a considerable improvement to 
previous studies for the very reason that our sampling of formicine 
lineages is more comprehensive.</p>
                <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par56" class="Para">The origin of the Formicinae was placed in the Neotropics by Moreau &amp; Bell [<span class="CitationRef"><a href="#CR16" data-toggle="citation">16</a></span>].
 Our inference of a Neotropical origin for the Myrmelachistini—the 
oldest tribe and the sister lineage to the remaining formicines—agrees 
with this hypothesis. Further inference of biogeographic range evolution
 in the Formicinae was impeded in our study by the remaining 
phylogenetic uncertainty surrounding tribal relationships, but 
nonetheless we obtained highly supported crown-group ancestral range 
estimates for a number of lineages. The evolution of Melophorini took 
place mainly in Australasia (Fig.&nbsp;<span class="InternalRef"><a href="#Fig4">4</a></span> and Table&nbsp;<span class="InternalRef"><a href="#Tab2">2</a></span>),
 which seems a natural result given that extant members of this tribe 
are largely confined to Australia. Along the same lines, the Formicini 
appeared to have had a history of evolution mainly in the Palearctic 
region, except for one dispersal to the Neotropics in the Eocene to 
Oligocene. For the Plagiolepidini, our analyses reconstructed an 
ancestral dispersal from the Oriental to the Afrotropical region 
(Fig.&nbsp;<span class="InternalRef"><a href="#Fig3">3</a></span>, node 
150 to 151 to 148) where this tribe then appears to have undergone the 
majority of its diversification. Camponotini and Lasiini are two 
species-rich clades of formicine ants with representatives across all 
continents. For both of these globally distributed groups our estimates 
point to an origin in the Oriental region, although with mediocre 
support (Table&nbsp;<span class="InternalRef"><a href="#Tab2">2</a></span>, 43.4–45.5&nbsp;% and 46.5–49.2&nbsp;%). Moreau &amp; Bell [<span class="CitationRef"><a href="#CR16" data-toggle="citation">16</a></span>]
 have suggested that the Neotropics functioned as a cradle for ant 
diversification; however, our biogeographic results are not fully 
consistent with this hypothesis. While there are indications of a 
Formicinae origin in the Neotropics, our analyses overall do not 
associate the diversification of formicine ants with any particular 
region.</p>
              </section></div></section>
            <section xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" id="Sec23" class="Section1 RenderAsSection1 c-section"><h2 class="Heading js-ToggleCollapseSection c-section__heading" data-sticky-update="true"><a href="#Conclusions" data-toggle="collapse" data-toggle-disable-clickoff="" data-target="Conclusions">Conclusions</a></h2><div class="js-CollapseSection CollapseSection c-section__content" id="Conclusions" data-test="collapse-section"><p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par57" class="Para">We
 compared the phylogenetic informativeness of a 10-nuclear-gene data set
 produced by Sanger sequencing with a next-generation, phylogenomic data
 set of nearly 1000 UCE loci. This comparison, executed within the 
context of a case study of the same 82 species, tested the ability of 
these two types of data to resolve the evolutionary history of formicine
 ants. We found UCEs to be far superior to the multi-locus data set in 
estimating formicine relationships and noted a 1.5–2.5-fold increase in 
phylogenetic informativeness relative to the Sanger-produced data. Some 
ancient rapid divergence events remained unresolved even by our genomic 
data, indicating that phylogenetic reconstruction may in these cases 
only be improved with whole-genome data or, alternatively, that 
genuinely rapid radiations may have produced unresolvable hard 
polytomies. We successfully used BEAST to infer divergence ages from the
 UCE data, overcoming computational limitations through data filtering. 
These analyses reconstructed formicine ants and their major lineages to 
be relatively older compared to previous estimates for the group. The 
subfamily appears to have diversified across all biogeographic regions 
and to have had no particular evolutionary cradle, although much of the 
early history of the clade remains unclear. UCEs were able to 
significantly improve formicine tribal classification based on the 
comprehensive phylogeny for the group estimated here. Our study 
highlights both the promise and possible limitations of UCEs for 
evolutionary biologists considering the transition from Sanger to 
next-generation sequencing approaches: Taken together, our findings 
indicate UCEs are highly useful for insect phylogenomics. The resulting 
phylogeny reveals exciting foci for the study of behavior and chemical 
ecology in formicine ants.</p></div></section>
            <section xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" id="Sec24" class="Section1 RenderAsSection1 c-section"><h2 class="Heading js-ToggleCollapseSection c-section__heading" data-sticky-update="true"><a href="#Availabilityofsupportingdata" data-toggle="collapse" data-toggle-disable-clickoff="" data-target="Availabilityofsupportingdata">Availability of supporting data</a></h2><div class="js-CollapseSection CollapseSection c-section__content" id="Availabilityofsupportingdata" data-test="collapse-section"><p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par58" class="Para">The
 data sets supporting the results of this article are available in 
GenBank (Accessions KT443144–KT443783) and in the Sequence Read Archive 
(SUB1067415); data matrices and associated tree files are deposited in 
Treebase (TB2:S18146).</p></div></section>
          <section xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" class="AbbreviationGroup Section1 RenderAsSection1 c-section"><h2 class="Heading js-ToggleCollapseSection c-section__heading" data-sticky-update="true"><a href="#Abbreviations" data-toggle="collapse" data-toggle-disable-clickoff="" data-target="Abbreviations">Abbreviations</a></h2><div class="js-CollapseSection CollapseSection c-section__content" data-test="collapse-section" id="Abbreviations"><div class="DefinitionList">
                <dl class="DefinitionListEntry"><dt class="Term"><dfn>BI:&nbsp;</dfn></dt><dd class="Description">
                    <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par4" class="Para">Bayesian inference</p>
                  </dd></dl>
                <dl class="DefinitionListEntry"><dt class="Term"><dfn>BS:&nbsp;</dfn></dt><dd class="Description">
                    <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par5" class="Para">Bootstrap support</p>
                  </dd></dl>
                <dl class="DefinitionListEntry"><dt class="Term"><dfn>bp:&nbsp;</dfn></dt><dd class="Description">
                    <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par6" class="Para">base pairs</p>
                  </dd></dl>
                <dl class="DefinitionListEntry"><dt class="Term"><dfn>DEC:&nbsp;</dfn></dt><dd class="Description">
                    <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par7" class="Para">Dispersal-extinction-cladogenesis</p>
                  </dd></dl>
                <dl class="DefinitionListEntry"><dt class="Term"><dfn>HPD:&nbsp;</dfn></dt><dd class="Description">
                    <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par8" class="Para">Highest posterior density</p>
                  </dd></dl>
                <dl class="DefinitionListEntry"><dt class="Term"><dfn>Ma:&nbsp;</dfn></dt><dd class="Description">
                    <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par9" class="Para">Million years ago</p>
                  </dd></dl>
                <dl class="DefinitionListEntry"><dt class="Term"><dfn>MCC:&nbsp;</dfn></dt><dd class="Description">
                    <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par10" class="Para">Maximum clade credibility</p>
                  </dd></dl>
                <dl class="DefinitionListEntry"><dt class="Term"><dfn>ML:&nbsp;</dfn></dt><dd class="Description">
                    <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par11" class="Para">Maximum Likelihood</p>
                  </dd></dl>
                <dl class="DefinitionListEntry"><dt class="Term"><dfn>MRCA:&nbsp;</dfn></dt><dd class="Description">
                    <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par12" class="Para">Most recent common ancestor</p>
                  </dd></dl>
                <dl class="DefinitionListEntry"><dt class="Term"><dfn>MY:&nbsp;</dfn></dt><dd class="Description">
                    <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par13" class="Para">Million years</p>
                  </dd></dl>
                <dl class="DefinitionListEntry"><dt class="Term"><dfn>PI:&nbsp;</dfn></dt><dd class="Description">
                    <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par14" class="Para">Phylogenetic informativeness</p>
                  </dd></dl>
                <dl class="DefinitionListEntry"><dt class="Term"><dfn>PP:&nbsp;</dfn></dt><dd class="Description">
                    <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par15" class="Para">Posterior Probability</p>
                  </dd></dl>
                <dl class="DefinitionListEntry"><dt class="Term"><dfn>S-DEC:&nbsp;</dfn></dt><dd class="Description">
                    <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par16" class="Para">Statistical Dispersal-extincton-cladogenesis</p>
                  </dd></dl>
                <dl class="DefinitionListEntry"><dt class="Term"><dfn>SI/HPC:&nbsp;</dfn></dt><dd class="Description">
                    <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par17" class="Para">Smithsonian Institution high performance cluster</p>
                  </dd></dl>
                <dl class="DefinitionListEntry"><dt class="Term"><dfn>UCEs:&nbsp;</dfn></dt><dd class="Description">
                    <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par18" class="Para">Ultraconserved elements</p>
                  </dd></dl>
              </div></div></section><section xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" class="Section1 RenderAsSection1 c-section" data-test="declarations-section" id="Decs"><h2 class="Heading js-ToggleCollapseSection c-section__heading" data-sticky-update="true"><a href="#Declarations" data-toggle="collapse" data-toggle-disable-clickoff="" data-target="Declarations">Declarations</a></h2><div class="js-CollapseSection CollapseSection c-section__content" data-test="collapse-section" id="Declarations"><div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Acknowledgments"><h3 xmlns="" class="Heading">Acknowledgments</h3><p xmlns="" class="SimplePara">This
 study was supported by NSF grants EF-0431330 (to PSW, SGB, TRS &amp; 
BLF), DEB-0743542 (to SGB), DEB-0949689 (to TRS), DEB-0842204 (to PSW), 
and the Competitive Grants Program for Science (Smithsonian Institution,
 to SGB &amp; TRS). BBB was partly supported by a Peter Buck 
Postdoctoral Fellowship. We are grateful to A. Jesovnik and J. 
Sosa-Calvo for help with laboratory procedures, and M. Branstetter for 
guidance on UCE methods. Portions of the laboratory and the computer 
work were conducted in and with the support of the L.A.B. facilities of 
the National Museum of Natural History.</p></div><div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="License">
                <p xmlns="" class="SimplePara">
                           <strong class="EmphasisTypeBold">Open Access</strong>This article is distributed under the terms of the Creative Commons Attribution 4.0 International License (<span xmlns="http://www.w3.org/1999/xhtml" class="ExternalRef"><a href="http://creativecommons.org/licenses/by/4.0/"><span class="RefSource">http://creativecommons.org/licenses/by/4.0/</span></a></span>),
 which permits unrestricted use, distribution, and reproduction in any 
medium, provided you give appropriate credit to the original author(s) 
and the source, provide a link to the Creative Commons license, and 
indicate if changes were made. The Creative Commons Public Domain 
Dedication waiver (<span xmlns="http://www.w3.org/1999/xhtml" class="ExternalRef"><a href="http://creativecommons.org/publicdomain/zero/1.0/"><span class="RefSource">http://creativecommons.org/publicdomain/zero/1.0/</span></a></span>) applies to the data made available in this article, unless otherwise stated.</p>
              </div><aside class="Appendix" id="App1">
              
                <h3 class="Heading">Additional files</h3><div id="widgetContainer" data-test="figshare-container"></div>
                <div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par59" class="Para">
                  <div class="MediaObject" id="MOESM1">
                    <div xmlns="" class="esm-item "><div xmlns="http://www.w3.org/1999/xhtml" class="caption-container"><a href="https://static-content.springer.com/esm/art%3A10.1186%2Fs12862-015-0552-5/MediaObjects/12862_2015_552_MOESM1_ESM.pdf"><span class="caption-number">Additional file 1:</span></a>
                          <span xmlns="" class="SimplePara">
                                          <strong class="EmphasisTypeBold">Comprehensive description of methods.</strong>
 An extensive version of the methods section, giving a more detailed 
account of laboratory and bioinformatics procedures. (PDF 132&nbsp;kb)</span>
                        </div></div>
                  </div>
                  <div class="MediaObject" id="MOESM2">
                    <div xmlns="" class="esm-item "><div xmlns="http://www.w3.org/1999/xhtml" class="caption-container"><a href="https://static-content.springer.com/esm/art%3A10.1186%2Fs12862-015-0552-5/MediaObjects/12862_2015_552_MOESM2_ESM.pdf"><span class="caption-number">Additional file 2:</span></a>
                          <span xmlns="" class="SimplePara">
                                          <strong class="EmphasisTypeBold">Genbank accessions for PCR-amplified sequences included in this study.</strong> Table including specimen identifiers and Genbank accession numbers for all study taxa. (PDF 94&nbsp;kb)</span>
                        </div></div>
                  </div>
                  <div class="MediaObject" id="MOESM3">
                    <div xmlns="" class="esm-item "><div xmlns="http://www.w3.org/1999/xhtml" class="caption-container"><a href="https://static-content.springer.com/esm/art%3A10.1186%2Fs12862-015-0552-5/MediaObjects/12862_2015_552_MOESM3_ESM.pdf"><span class="caption-number">Additional file 3:</span></a>
                          <span xmlns="" class="SimplePara">
                                          <strong class="EmphasisTypeBold">Calibration points used to define prior calibration densities for dating analyses with BEAST.</strong> (PDF 91&nbsp;kb)</span>
                        </div></div>
                  </div>
                  <div class="MediaObject" id="MOESM4">
                    <div xmlns="" class="esm-item "><div xmlns="http://www.w3.org/1999/xhtml" class="caption-container"><a href="https://static-content.springer.com/esm/art%3A10.1186%2Fs12862-015-0552-5/MediaObjects/12862_2015_552_MOESM4_ESM.pdf"><span class="caption-number">Additional file 4:</span></a>
                          <span xmlns="" class="SimplePara">
                                          <strong class="EmphasisTypeBold">Taxon distribution matrix used in the biogeographic analyses.</strong> (PDF 81&nbsp;kb)</span>
                        </div></div>
                  </div>
                  <div class="MediaObject" id="MOESM5">
                    <div xmlns="" class="esm-item "><div xmlns="http://www.w3.org/1999/xhtml" class="caption-container"><a href="https://static-content.springer.com/esm/art%3A10.1186%2Fs12862-015-0552-5/MediaObjects/12862_2015_552_MOESM5_ESM.pdf"><span class="caption-number">Additional file 5:</span></a>
                          <span xmlns="" class="SimplePara">
                                          <strong class="EmphasisTypeBold">Dispersal constraint matrices used for DEC and S-DEC analyses in RASP.</strong> (PDF 75&nbsp;kb)</span>
                        </div></div>
                  </div>
                  <div class="MediaObject" id="MOESM6">
                    <div xmlns="" class="esm-item "><div xmlns="http://www.w3.org/1999/xhtml" class="caption-container"><a href="https://static-content.springer.com/esm/art%3A10.1186%2Fs12862-015-0552-5/MediaObjects/12862_2015_552_MOESM6_ESM.pdf"><span class="caption-number">Additional file 6:</span></a>
                          <span xmlns="" class="SimplePara">
                                          <strong class="EmphasisTypeBold">Summary of UCE capture statistics.</strong> (PDF 95&nbsp;kb)</span>
                        </div></div>
                  </div>
                  <div class="MediaObject" id="MOESM7">
                    <div xmlns="" class="esm-item "><div xmlns="http://www.w3.org/1999/xhtml" class="caption-container"><a href="https://static-content.springer.com/esm/art%3A10.1186%2Fs12862-015-0552-5/MediaObjects/12862_2015_552_MOESM7_ESM.pdf"><span class="caption-number">Additional file 7:</span></a>
                          <span xmlns="" class="SimplePara">
                                          <strong class="EmphasisTypeBold">Revised tribal classification for the Formicinae.</strong> (PDF 84&nbsp;kb)</span>
                        </div></div>
                  </div>
                  <div class="MediaObject" id="MOESM8">
                    <div xmlns="" class="esm-item "><div xmlns="http://www.w3.org/1999/xhtml" class="caption-container"><a href="https://static-content.springer.com/esm/art%3A10.1186%2Fs12862-015-0552-5/MediaObjects/12862_2015_552_MOESM8_ESM.pdf"><span class="caption-number">Additional file 8:</span></a>
                          <span xmlns="" class="SimplePara">
                                          <strong class="EmphasisTypeBold">Phylogenetic trees from analyses not illustrated in the main text.</strong> Additional results from Bayesian and Maximum Likelihood bootstrap analyses. (PDF 2225&nbsp;kb)</span>
                        </div></div>
                  </div>
                  <div class="MediaObject" id="MOESM9">
                    <div xmlns="" class="esm-item "><div xmlns="http://www.w3.org/1999/xhtml" class="caption-container"><a href="https://static-content.springer.com/esm/art%3A10.1186%2Fs12862-015-0552-5/MediaObjects/12862_2015_552_MOESM9_ESM.pdf"><span class="caption-number">Additional file 9:</span></a>
                          <span xmlns="" class="SimplePara">
                                          <strong class="EmphasisTypeBold">Phylogenetic trees from analyses not illustrated in the main text, continued.</strong> Additional results from Maximum Likelihood analyses on UCE data subsets. (PDF 669&nbsp;kb)</span>
                        </div></div>
                  </div>
                  <div class="MediaObject" id="MOESM10">
                    <div xmlns="" class="esm-item "><div xmlns="http://www.w3.org/1999/xhtml" class="caption-container"><a href="https://static-content.springer.com/esm/art%3A10.1186%2Fs12862-015-0552-5/MediaObjects/12862_2015_552_MOESM10_ESM.pdf"><span class="caption-number">Additional file 10:</span></a>
                          <span xmlns="" class="SimplePara">
                                          <strong class="EmphasisTypeBold">Summary of taxon exclusion experiments.</strong>
 Summary sketches of phylogenetic relationships contrasting the 
placement of the seven rogue taxa between UCE and 10-gene data set. (PDF
 248&nbsp;kb)</span>
                        </div></div>
                  </div>
                  <div class="MediaObject" id="MOESM11">
                    <div xmlns="" class="esm-item "><div xmlns="http://www.w3.org/1999/xhtml" class="caption-container"><a href="https://static-content.springer.com/esm/art%3A10.1186%2Fs12862-015-0552-5/MediaObjects/12862_2015_552_MOESM11_ESM.pdf"><span class="caption-number">Additional file 11:</span></a>
                          <span xmlns="" class="SimplePara">
                                          <strong class="EmphasisTypeBold">Comprehensive results of ancestral range estimations under the DEC and S-DEC models.</strong> Extended version of Table&nbsp;<span xmlns="http://www.w3.org/1999/xhtml" class="InternalRef"><a href="#Tab2">2</a></span> including all crown group ancestral ranges as estimated with the DEC and S-DEC models implemented in RASP. (PDF 97&nbsp;kb)</span>
                        </div></div>
                  </div>
                </div>
              
            </aside><aside xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="ArticleNote ArticleNoteMisc"><h3 xmlns="" class="Heading">Competing interests</h3><p xmlns="" class="SimplePara">The authors declare that they have no competing interest.</p></aside><aside xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="ArticleNote ArticleNoteMisc"><h3 xmlns="" class="Heading">Authors’ contributions</h3><p xmlns="" class="SimplePara">PSW,
 SGB, BLF, TRS, and BBB conceived the study and participated in its 
design and coordination. BBB generated and analyzed the UCE data, 
conducted phylogenetic and divergence dating analyses, and led 
manuscript writing. SGB generated and edited Sanger data, and helped to 
draft the manuscript. BLF obtained specimens, databased specimen 
information and images, and helped to draft the manuscript. MWL 
generated and analyzed UCE data, and helped to draft the manuscript. TRS
 conducted phylogenetic analyses and helped to draft the manuscript. PSW
 obtained specimens, databased specimen information, generated and 
edited the Sanger data, and helped to draft the manuscript. All authors 
read and approved the final manuscript.</p></aside></div></section></div>
                        <div class="FulltextWrapper"><section xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" class="Section1 RenderAsSection1 c-section Affiliations" id="Aff"><h2 class="Heading js-ToggleCollapseSection c-section__heading" data-sticky-update="true"><a href="#Author-Affiliations" data-toggle="collapse" data-toggle-disable-clickoff="" data-target="Author-Affiliations">Authors’ Affiliations</a></h2><div class="js-CollapseSection CollapseSection c-section__content c-section__content" data-test="collapse-section" id="Author-Affiliations"><div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Affiliation" id="Aff1"><span class="AffiliationNumber">(1)</span><div class="AffiliationText">Department of Entomology, National Museum of Natural History, Smithsonian Institution, Washington, DC&nbsp;20560, USA</div></div><div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Affiliation" id="Aff2"><span class="AffiliationNumber">(2)</span><div class="AffiliationText">Department of Entomology, California Academy of Sciences, San Francisco, CA&nbsp;94118, USA</div></div><div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Affiliation" id="Aff3"><span class="AffiliationNumber">(3)</span><div class="AffiliationText">Department of Entomology and Nematology, University of California-Davis, Davis, CA&nbsp;95616, USA</div></div></div></section></div>
                        <div class="FulltextWrapper"><section xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" class="Section1 RenderAsSection1 c-section SectionTypeMaterialsAndMethods" id="Bib1"><h2 class="Heading js-ToggleCollapseSection c-section__heading" data-sticky-update="true"><a href="#References" data-toggle="collapse" data-toggle-disable-clickoff="" data-target="References">References</a></h2><div class="js-CollapseSection CollapseSection c-section__content" data-test="collapse-section" id="References"><ol class="BibliographyWrapper"><li class="Citation"><cite class="CitationContent" id="CR1" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">McCormack
 JE, Faircloth BC, Crawford NG, Gowaty PA, Brumfield RT, Glenn TC. 
Ultraconserved elements are novel phylogenomic markers that resolve 
placental mammal phylogeny when combined with species-tree analysis. Gen
 Res. 2012;22(4):746–54.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1101/gr.125864.111"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Ultraconserved%20elements%20are%20novel%20phylogenomic%20markers%20that%20resolve%20placental%20mammal%20phylogeny%20when%20combined%20with%20species-tree%20analysis&amp;author=JE.%20McCormack&amp;author=BC.%20Faircloth&amp;author=NG.%20Crawford&amp;author=PA.%20Gowaty&amp;author=RT.%20Brumfield&amp;author=TC.%20Glenn&amp;journal=Gen%20Res&amp;volume=22&amp;issue=4&amp;pages=746-754&amp;publication_year=2012"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR2" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Lemmon EM, Lemmon AR. High-throughput genomic data in systematics and phylogenetics. Annu Rev Ecol Syst. 2013;44:99–121.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1146/annurev-ecolsys-110512-135822"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=High-throughput%20genomic%20data%20in%20systematics%20and%20phylogenetics&amp;author=EM.%20Lemmon&amp;author=AR.%20Lemmon&amp;journal=Annu%20Rev%20Ecol%20Syst&amp;volume=44&amp;pages=99-121&amp;publication_year=2013"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR3" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Smith
 BT, Harvey MG, Faircloth BC, Glenn TC, Brumfield RT. Target capture and
 massively parallel sequencing of ultraconserved elements for 
comparative studies at shallow evolutionary time scales. Syst Biol. 
2013;63:83–95.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1093/sysbio/syt061"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=24021724"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Target%20capture%20and%20massively%20parallel%20sequencing%20of%20ultraconserved%20elements%20for%20comparative%20studies%20at%20shallow%20evolutionary%20time%20scales&amp;author=BT.%20Smith&amp;author=MG.%20Harvey&amp;author=BC.%20Faircloth&amp;author=TC.%20Glenn&amp;author=RT.%20Brumfield&amp;journal=Syst%20Biol&amp;volume=63&amp;pages=83-95&amp;publication_year=2013"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR4" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Smith
 BT, McCormack JE, Cuervo AM, Hickerson MJ, Aleixo A, Cadena CD, et al. 
The drivers of tropical speciation. Nature. 2014;515:406–9.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1038/nature13687"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=25209666"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=The%20drivers%20of%20tropical%20speciation&amp;author=BT.%20Smith&amp;author=JE.%20McCormack&amp;author=AM.%20Cuervo&amp;author=MJ.%20Hickerson&amp;author=A.%20Aleixo&amp;author=CD.%20Cadena&amp;author=J.%20Perez-Eman&amp;author=CW.%20Burney&amp;author=X.%20Xie&amp;author=MG.%20Harvey&amp;journal=Nature&amp;volume=515&amp;pages=406-409&amp;publication_year=2014"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR5" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Crawford
 NG, Faircloth BC, McCormack JE, Brumfield RT, Winker K, Glenn TC. More 
than 1000 ultraconserved elements provide evidence that turtles are the 
sister group of archosaurs. Biol Lett. 2012;8:783–6.</span><span class="Occurrences"><span class="Occurrence OccurrencePMCID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3440978"><span><span>PubMed Central</span></span></a></span><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1098/rsbl.2012.0331"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=22593086"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=More%20than%201000%20ultraconserved%20elements%20provide%20evidence%20that%20turtles%20are%20the%20sister%20group%20of%20archosaurs&amp;author=NG.%20Crawford&amp;author=BC.%20Faircloth&amp;author=JE.%20McCormack&amp;author=RT.%20Brumfield&amp;author=K.%20Winker&amp;author=TC.%20Glenn&amp;journal=Biol%20Lett&amp;volume=8&amp;pages=783-786&amp;publication_year=2012"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR6" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Faircloth
 BC, McCormack JE, Crawford NG, Harvey MG, Brumfield RT, Glenn TC. 
Ultraconserved elements anchor thousands of genetic markers spanning 
multiple evolutionary timescales. Syst Biol. 2012;61:717–26.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1093/sysbio/sys004"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=22232343"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Ultraconserved%20elements%20anchor%20thousands%20of%20genetic%20markers%20spanning%20multiple%20evolutionary%20timescales&amp;author=BC.%20Faircloth&amp;author=JE.%20McCormack&amp;author=NG.%20Crawford&amp;author=MG.%20Harvey&amp;author=RT.%20Brumfield&amp;author=TC.%20Glenn&amp;journal=Syst%20Biol&amp;volume=61&amp;pages=717-726&amp;publication_year=2012"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR7" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Faircloth
 BC, Branstetter MG, White ND, Brady SG. Target enrichment of 
ultraconserved elements from arthropods provides a genomic perspective 
on relationships among Hymenoptera. Mol Ecol Res. 2015;15:489–501.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1111/1755-0998.12328"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Target%20enrichment%20of%20ultraconserved%20elements%20from%20arthropods%20provides%20a%20genomic%20perspective%20on%20relationships%20among%20Hymenoptera&amp;author=BC.%20Faircloth&amp;author=MG.%20Branstetter&amp;author=ND.%20White&amp;author=SG.%20Brady&amp;journal=Mol%20Ecol%20Res&amp;volume=15&amp;pages=489-501&amp;publication_year=2015"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR8" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Gilbert
 PS, Chang J, Pan C, Sobel EM, Sinsheimer JS, Faircloth BC, et al. 
Genome-wide ultraconserved elements exhibit higher phylogenetic 
informativeness than traditional gene markers in percomorph fishes. Mol 
Phylogenet Evol. 2015;92:140–6.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1016/j.ympev.2015.05.027"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=26079130"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Genome-wide%20ultraconserved%20elements%20exhibit%20higher%20phylogenetic%20informativeness%20than%20traditional%20gene%20markers%20in%20percomorph%20fishes&amp;author=PS.%20Gilbert&amp;author=J.%20Chang&amp;author=C.%20Pan&amp;author=EM.%20Sobel&amp;author=JS.%20Sinsheimer&amp;author=BC.%20Faircloth&amp;author=ME.%20Alfaro&amp;journal=Mol%20Phylogenet%20Evol&amp;volume=92&amp;pages=140-146&amp;publication_year=2015"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR9" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Brady
 SG, Schultz TR, Fisher BL, Ward PS. Evaluating alternative hypotheses 
for the early evolution and diversification of ants. Proc Natl Acad Sci.
 2006;103:18172–7.</span><span class="Occurrences"><span class="Occurrence OccurrencePMCID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC1838725"><span><span>PubMed Central</span></span></a></span><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1073/pnas.0605858103"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=17079492"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Evaluating%20alternative%20hypotheses%20for%20the%20early%20evolution%20and%20diversification%20of%20ants&amp;author=SG.%20Brady&amp;author=TR.%20Schultz&amp;author=BL.%20Fisher&amp;author=PS.%20Ward&amp;journal=Proc%20Natl%20Acad%20Sci&amp;volume=103&amp;pages=18172-18177&amp;publication_year=2006"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR10" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Moreau
 CS, Bell CD, Vila R, Archibald SB, Pierce NE. Phylogeny of the ants: 
Diversification in the age of angiosperms. Science. 2006;312:101–4.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1126/science.1124891"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=16601190"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Phylogeny%20of%20the%20ants%3A%20Diversification%20in%20the%20age%20of%20angiosperms&amp;author=CS.%20Moreau&amp;author=CD.%20Bell&amp;author=R.%20Vila&amp;author=SB.%20Archibald&amp;author=NE.%20Pierce&amp;journal=Science&amp;volume=312&amp;pages=101-104&amp;publication_year=2006"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR11" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Ward
 PS, Brady SG, Fisher BL, Schultz TR. Phylogeny and biogeography of 
dolichoderine ants: effects of data partitioning and relict taxa on 
historical inference. Syst Biol. 2010;59:342–62.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1093/sysbio/syq012"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=20525640"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Phylogeny%20and%20biogeography%20of%20dolichoderine%20ants%3A%20effects%20of%20data%20partitioning%20and%20relict%20taxa%20on%20historical%20inference&amp;author=PS.%20Ward&amp;author=SG.%20Brady&amp;author=BL.%20Fisher&amp;author=TR.%20Schultz&amp;journal=Syst%20Biol&amp;volume=59&amp;pages=342-362&amp;publication_year=2010"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR12" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Brady
 SG, Fisher BL, Schultz TR, Ward PS. The rise of army ants and their 
relatives: diversification of specialized predatory doryline ants. BMC 
Evol Biol. 2014;14:93.</span><span class="Occurrences"><span class="Occurrence OccurrencePMCID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4021219"><span><span>PubMed Central</span></span></a></span><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1186/1471-2148-14-93"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=24886136"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=The%20rise%20of%20army%20ants%20and%20their%20relatives%3A%20diversification%20of%20specialized%20predatory%20doryline%20ants&amp;author=SG.%20Brady&amp;author=BL.%20Fisher&amp;author=TR.%20Schultz&amp;author=PS.%20Ward&amp;journal=BMC%20Evol%20Biol&amp;volume=14&amp;pages=93&amp;publication_year=2014"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR13" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Schmidt
 CA, Shattuck SO. The higher classification of the ant subfamily 
Ponerinae (Hymenoptera: Formicidae), with a review of ponerine ecology 
and behavior. Zootaxa. 2014;3817:1–242.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.11646/zootaxa.3817.1.1"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=24943802"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=The%20higher%20classification%20of%20the%20ant%20subfamily%20Ponerinae%20%28Hymenoptera%3A%20Formicidae%29%2C%20with%20a%20review%20of%20ponerine%20ecology%20and%20behavior&amp;author=CA.%20Schmidt&amp;author=SO.%20Shattuck&amp;journal=Zootaxa&amp;volume=3817&amp;pages=1-242&amp;publication_year=2014"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR14" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Ward
 PS, Brady SG, Fisher BL, Schultz TR. The evolution of myrmicine ants: 
phylogeny and biogeography of a hyperdiverse ant clade (Hymenoptera: 
Formicidae). Syst Ent. 2015;40:61–81.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1111/syen.12090"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=The%20evolution%20of%20myrmicine%20ants%3A%20phylogeny%20and%20biogeography%20of%20a%20hyperdiverse%20ant%20clade%20%28Hymenoptera%3A%20Formicidae%29&amp;author=PS.%20Ward&amp;author=SG.%20Brady&amp;author=BL.%20Fisher&amp;author=TR.%20Schultz&amp;journal=Syst%20Ent&amp;volume=40&amp;pages=61-81&amp;publication_year=2015"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR15" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Lucky A, Trautwein MD, Guenard BS, Weiser MD, Dunn RR. Tracing the rise of ants-out of the ground. PLoS ONE. 2013;8(12):e84012.</span><span class="Occurrences"><span class="Occurrence OccurrencePMCID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3873401"><span><span>PubMed Central</span></span></a></span><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1371/journal.pone.0084012"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=24386323"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Tracing%20the%20rise%20of%20ants-out%20of%20the%20ground&amp;author=A.%20Lucky&amp;author=MD.%20Trautwein&amp;author=BS.%20Guenard&amp;author=MD.%20Weiser&amp;author=RR.%20Dunn&amp;journal=PLoS%20ONE&amp;volume=8&amp;issue=12&amp;pages=e84012&amp;publication_year=2013"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR16" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Moreau
 CS, Bell CD. Testing the museum versus cradle tropical biological 
diversity hypothesis: phylogeny, diversification, and ancestral 
biogeographic range evolution of the ants. Evolution. 2013;67:2240–57.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1111/evo.12105"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=23888848"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Testing%20the%20museum%20versus%20cradle%20tropical%20biological%20diversity%20hypothesis%3A%20phylogeny%2C%20diversification%2C%20and%20ancestral%20biogeographic%20range%20evolution%20of%20the%20ants&amp;author=CS.%20Moreau&amp;author=CD.%20Bell&amp;journal=Evolution&amp;volume=67&amp;pages=2240-2257&amp;publication_year=2013"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR17" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Blaimer
 BB, Brady SG, Schultz TR, Fisher BL. Functional and phylogenetic 
approaches reveal the evolution of diversity in a hyper diverse biota. 
Ecography. 2015;38:901–12.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1111/ecog.01370"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Functional%20and%20phylogenetic%20approaches%20reveal%20the%20evolution%20of%20diversity%20in%20a%20hyper%20diverse%20biota&amp;author=BB.%20Blaimer&amp;author=SG.%20Brady&amp;author=TR.%20Schultz&amp;author=BL.%20Fisher&amp;journal=Ecography&amp;volume=38&amp;pages=901-912&amp;publication_year=2015"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR18" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Schultz TR, Brady SG. Major evolutionary transitions in ant agriculture. Proc Natl Acad Sci. 2008;105:5435–40.</span><span class="Occurrences"><span class="Occurrence OccurrencePMCID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2291119"><span><span>PubMed Central</span></span></a></span><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1073/pnas.0711024105"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=18362345"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Major%20evolutionary%20transitions%20in%20ant%20agriculture&amp;author=TR.%20Schultz&amp;author=SG.%20Brady&amp;journal=Proc%20Natl%20Acad%20Sci&amp;volume=105&amp;pages=5435-5440&amp;publication_year=2008"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR19" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Price
 SL, Powell S, Kronauer DJC, Tran LAP, Pierce NE, Wayne RK. Renewed 
diversification is associated with new ecological opportunity in the 
Neotropical turtle ants. J Evol Biol. 2014;27:242–58.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1111/jeb.12300"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=24417316"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Renewed%20diversification%20is%20associated%20with%20new%20ecological%20opportunity%20in%20the%20Neotropical%20turtle%20ants&amp;author=SL.%20Price&amp;author=S.%20Powell&amp;author=DJC.%20Kronauer&amp;author=LAP.%20Tran&amp;author=NE.%20Pierce&amp;author=RK.%20Wayne&amp;journal=J%20Evol%20Biol&amp;volume=27&amp;pages=242-258&amp;publication_year=2014"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR20" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Ward PS. The phylogeny and evolution of ants. Annu Rev Ecol Syst. 2014;45:23–43.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1146/annurev-ecolsys-120213-091824"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=The%20phylogeny%20and%20evolution%20of%20ants&amp;author=PS.%20Ward&amp;journal=Annu%20Rev%20Ecol%20Syst&amp;volume=45&amp;pages=23-43&amp;publication_year=2014"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR21" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Schmidt
 JO. Chemistry, pharmacology, and chemical ecology of ant venoms. In: 
Piek T, editor. Venoms of the Hymenoptera. London: Academic Press; 1986:
 425–508<span xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Occurrences"><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="https://scholar.google.com/scholar?hl=en&amp;q=Schmidt%20JO.%20Chemistry%2C%20pharmacology%2C%20and%20chemical%20ecology%20of%20ant%20venoms.%20In%3A%20Piek%20T%2C%20editor.%20Venoms%20of%20the%20Hymenoptera.%20London%3A%20Academic%20Press%3B%201986%3A%20425%E2%80%93508"><span><span>Google Scholar</span></span></a></span></span></span><span class="Occurrences"></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR22" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Saporito
 RA, Garraffo HM, Donnelly MA, Edwards AL, Longino JT, Daly JW. 
Formicine ants: An arthropod source for the pumiliotoxin alkaloids of 
dendrobatid poison frogs. Proc Natl Acad Sci. 2004;101:8045–50.</span><span class="Occurrences"><span class="Occurrence OccurrencePMCID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC419554"><span><span>PubMed Central</span></span></a></span><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1073/pnas.0402365101"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=15128938"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Formicine%20ants%3A%20An%20arthropod%20source%20for%20the%20pumiliotoxin%20alkaloids%20of%20dendrobatid%20poison%20frogs&amp;author=RA.%20Saporito&amp;author=HM.%20Garraffo&amp;author=MA.%20Donnelly&amp;author=AL.%20Edwards&amp;author=JT.%20Longino&amp;author=JW.%20Daly&amp;journal=Proc%20Natl%20Acad%20Sci&amp;volume=101&amp;pages=8045-8050&amp;publication_year=2004"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR23" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Buschinger A. Social parasitism among ants: a review (Hymenoptera: Formicidae). Myrmecol News. 2009;12:219–35.</span><span class="Occurrences"><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Social%20parasitism%20among%20ants%3A%20a%20review%20%28Hymenoptera%3A%20Formicidae%29&amp;author=A.%20Buschinger&amp;journal=Myrmecol%20News&amp;volume=12&amp;pages=219-235&amp;publication_year=2009"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR24" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">LaPolla JS, Brady SG, Shattuck SO. Phylogeny and taxonomy of the <em class="EmphasisTypeItalic">Prenolepis</em> genus-group of ants (Hymenoptera: Formicidae). Syst Ent. 2010;35:118–31.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1111/j.1365-3113.2009.00492.x"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Phylogeny%20and%20taxonomy%20of%20the%20Prenolepis%20genus-group%20of%20ants%20%28Hymenoptera%3A%20Formicidae%29&amp;author=JS.%20LaPolla&amp;author=SG.%20Brady&amp;author=SO.%20Shattuck&amp;journal=Syst%20Ent&amp;volume=35&amp;pages=118-131&amp;publication_year=2010"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR25" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Lapolla JS, Kallal RJ, Brady SG. A new ant genus from the Greater Antilles and Central America, <em class="EmphasisTypeItalic">Zatania</em> (Hymenoptera: Formicidae), exemplifies the utility of male and molecular character systems. Syst Ent. 2012;37:200–14.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1111/j.1365-3113.2011.00605.x"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=A%20new%20ant%20genus%20from%20the%20Greater%20Antilles%20and%20Central%20America%2C%20Zatania%20%28Hymenoptera%3A%20Formicidae%29%2C%20exemplifies%20the%20utility%20of%20male%20and%20molecular%20character%20systems&amp;author=JS.%20Lapolla&amp;author=RJ.%20Kallal&amp;author=SG.%20Brady&amp;journal=Syst%20Ent&amp;volume=37&amp;pages=200-214&amp;publication_year=2012"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR26" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Blumenstiel
 B, Cibulskis K, Fisher S, DeFelice M, Barry A, Fennell T, et al. 
Targeted exon sequencing by in-solution hybrid selection. Curr Protoc 
Hum Genet. 2010;Unit 18.4.1–18.4.24, Supplement 66.<span xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Occurrences"><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="https://scholar.google.com/scholar?hl=en&amp;q=Blumenstiel%20B%2C%20Cibulskis%20K%2C%20Fisher%20S%2C%20DeFelice%20M%2C%20Barry%20A%2C%20Fennell%20T%2C%20et%20al.%20Targeted%20exon%20sequencing%20by%20in-solution%20hybrid%20selection.%20Curr%20Protoc%20Hum%20Genet.%202010%3BUnit%2018.4.1%E2%80%9318.4.24%2C%20Supplement%2066."><span><span>Google Scholar</span></span></a></span></span></span><span class="Occurrences"></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR27" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Ward
 PS, Downie DA. The ant subfamily Pseudomyrmecinae (Hymenoptera: 
Formicidae): phylogeny and evolution of big-eyed arboreal ants. Syst 
Ent. 2005;30:310–35.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1111/j.1365-3113.2004.00281.x"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=The%20ant%20subfamily%20Pseudomyrmecinae%20%28Hymenoptera%3A%20Formicidae%29%3A%20phylogeny%20and%20evolution%20of%20big-eyed%20arboreal%20ants&amp;author=PS.%20Ward&amp;author=DA.%20Downie&amp;journal=Syst%20Ent&amp;volume=30&amp;pages=310-335&amp;publication_year=2005"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR28" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Ward PS, Sumnicht TP. Molecular and morphological evidence for three sympatric species of <em class="EmphasisTypeItalic">Leptanilla</em> (Hymenoptera: Formicidae) on the Greek island of Rhodes. Myrmecol News. 2012;17:5–11.</span><span class="Occurrences"><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Molecular%20and%20morphological%20evidence%20for%20three%20sympatric%20species%20of%20Leptanilla%20%28Hymenoptera%3A%20Formicidae%29%20on%20the%20Greek%20island%20of%20Rhodes&amp;author=PS.%20Ward&amp;author=TP.%20Sumnicht&amp;journal=Myrmecol%20News&amp;volume=17&amp;pages=5-11&amp;publication_year=2012"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR29" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Katoh
 K, Asimenos G, Toh H. Multiple alignment of DNA sequences with MAFFT. 
In: Bioinformatics for DNA sequence analysis. Humana Press, New York 
City; 2009: 39–64<span xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Occurrences"><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="https://scholar.google.com/scholar?hl=en&amp;q=Katoh%20K%2C%20Asimenos%20G%2C%20Toh%20H.%20Multiple%20alignment%20of%20DNA%20sequences%20with%20MAFFT.%20In%3A%20Bioinformatics%20for%20DNA%20sequence%20analysis.%20Humana%20Press%2C%20New%20York%20City%3B%202009%3A%2039%E2%80%9364"><span><span>Google Scholar</span></span></a></span></span></span><span class="Occurrences"></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR30" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Faircloth B. Illumiprocessor: a trimmomatic wrapper for parallel adapter and quality trimming. 2013. doi: <span xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="ExternalRef"><a href="https://doi.org/10.6079/J9ILL"><span class="RefSource">10.6079/J9ILL</span></a></span>
                        </span><span class="Occurrences"></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR31" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Bolger AM, Lohse M, Usadel B. Trimmomatic: a flexible trimmer for Illumina sequence data. Bioinformatics 2014; btu170:1–7.<span xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Occurrences"><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="https://scholar.google.com/scholar?hl=en&amp;q=Bolger%20AM%2C%20Lohse%20M%2C%20Usadel%20B.%20Trimmomatic%3A%20a%20flexible%20trimmer%20for%20Illumina%20sequence%20data.%20Bioinformatics%202014%3B%20btu170%3A1%E2%80%937."><span><span>Google Scholar</span></span></a></span></span></span><span class="Occurrences"></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR32" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Faircloth B. PHYLUCE is a software package for the analysis of conserved genomic loci. 2015. doi:<span xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="ExternalRef"><a href="https://doi.org/10.6079/J9PHYL"><span class="RefSource">10.6079/J9PHYL</span></a></span>
                        </span><span class="Occurrences"></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR33" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Grabherr
 MG, Haas BJ, Yassour M, Levin JZ, Thompson DA, Amit I, et al. 
Full-length transcriptome assembly from RNA-Seq data without a reference
 genome. Nat Biotech. 2011;29:644–52.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1038/nbt.1883"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Full-length%20transcriptome%20assembly%20from%20RNA-Seq%20data%20without%20a%20reference%20genome&amp;author=MG.%20Grabherr&amp;author=BJ.%20Haas&amp;author=M.%20Yassour&amp;author=JZ.%20Levin&amp;author=DA.%20Thompson&amp;author=I.%20Amit&amp;author=X.%20Adiconis&amp;author=L.%20Fan&amp;author=R.%20Raychowdhury&amp;author=Q.%20Zeng&amp;journal=Nat%20Biotech&amp;volume=29&amp;pages=644-652&amp;publication_year=2011"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR34" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Castresana
 J. Selection of conserved blocks from multiple alignments for their use
 in phylogenetic analysis. Mol Biol Evol. 2000;17:540–52.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1093/oxfordjournals.molbev.a026334"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=10742046"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Selection%20of%20conserved%20blocks%20from%20multiple%20alignments%20for%20their%20use%20in%20phylogenetic%20analysis&amp;author=J.%20Castresana&amp;journal=Mol%20Biol%20Evol&amp;volume=17&amp;pages=540-552&amp;publication_year=2000"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR35" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Lanfear
 R, Calcott B, Ho SYW, Guindon S. PartitionFinder: Combined selection of
 partitioning schemes and substitution models for phylogenetic analyses.
 Mol Biol Evol. 2012;29:1695–701.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1093/molbev/mss020"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=22319168"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=PartitionFinder%3A%20Combined%20selection%20of%20partitioning%20schemes%20and%20substitution%20models%20for%20phylogenetic%20analyses&amp;author=R.%20Lanfear&amp;author=B.%20Calcott&amp;author=SYW.%20Ho&amp;author=S.%20Guindon&amp;journal=Mol%20Biol%20Evol&amp;volume=29&amp;pages=1695-1701&amp;publication_year=2012"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR36" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Stamatakis
 A. RAxML-VI-HPC: maximum likelihood-based phylogenetic analyses with 
thousands of taxa and mixed models. Bioinformatics. 2006;22:2688–90.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1093/bioinformatics/btl446"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=16928733"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=RAxML-VI-HPC%3A%20maximum%20likelihood-based%20phylogenetic%20analyses%20with%20thousands%20of%20taxa%20and%20mixed%20models&amp;author=A.%20Stamatakis&amp;journal=Bioinformatics&amp;volume=22&amp;pages=2688-2690&amp;publication_year=2006"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR37" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Zwickl
 DJ. Genetic algorithm approaches for the phylogenetic analysis of large
 biological sequence datasets under the maximum likelihood criterion. 
The University of Texas at Austin; 2006; accessible at: <span xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="ExternalRef"><a href="https://repositories.lib.utexas.edu/handle/2152/2666"><span class="RefSource">https://repositories.lib.utexas.edu/handle/2152/2666</span></a></span>
                        </span><span class="Occurrences"></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR38" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Ronquist
 F, Teslenko M, van der Mark P, Ayres DL, Darling A, Höhna S, et al. 
MrBayes 3.2: efficient Bayesian phylogenetic inference and model choice 
across a large model space. Syst Biol. 2012;61:539–42.</span><span class="Occurrences"><span class="Occurrence OccurrencePMCID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3329765"><span><span>PubMed Central</span></span></a></span><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1093/sysbio/sys029"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=22357727"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=MrBayes%203.2%3A%20efficient%20Bayesian%20phylogenetic%20inference%20and%20model%20choice%20across%20a%20large%20model%20space&amp;author=F.%20Ronquist&amp;author=M.%20Teslenko&amp;author=P.%20Mark&amp;author=DL.%20Ayres&amp;author=A.%20Darling&amp;author=S.%20H%C3%B6hna&amp;author=B.%20Larget&amp;author=L.%20Liu&amp;author=MA.%20Suchard&amp;author=JP.%20Huelsenbeck&amp;journal=Syst%20Biol&amp;volume=61&amp;pages=539-542&amp;publication_year=2012"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR39" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Frandsen
 PB, Calcott B, Mayer C, Lanfear R. Automatic selection of partitioning 
schemes for phylogenetic analyses using iterative k-means clustering of 
site rates. BMC Evol Biol. 2015;15:13.</span><span class="Occurrences"><span class="Occurrence OccurrencePMCID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4327964"><span><span>PubMed Central</span></span></a></span><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1186/s12862-015-0283-7"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=25887041"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Automatic%20selection%20of%20partitioning%20schemes%20for%20phylogenetic%20analyses%20using%20iterative%20k-means%20clustering%20of%20site%20rates&amp;author=PB.%20Frandsen&amp;author=B.%20Calcott&amp;author=C.%20Mayer&amp;author=R.%20Lanfear&amp;journal=BMC%20Evol%20Biol&amp;volume=15&amp;pages=13&amp;publication_year=2015"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR40" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Townsend JP. Profiling Phylogenetic Informativeness. Syst Biol. 2007;56:222–31.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1080/10635150701311362"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=17464879"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Profiling%20Phylogenetic%20Informativeness&amp;author=JP.%20Townsend&amp;journal=Syst%20Biol&amp;volume=56&amp;pages=222-231&amp;publication_year=2007"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR41" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Faircloth
 BC, Chang J, Alfaro ME. TAPIR enables high-throughput estimation and 
comparison of phylogenetic informativeness using locus-specific 
substitution models. 2012. arXiv preprint arXiv:12021215.<span xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Occurrences"><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="https://scholar.google.com/scholar?hl=en&amp;q=Faircloth%20BC%2C%20Chang%20J%2C%20Alfaro%20ME.%20TAPIR%20enables%20high-throughput%20estimation%20and%20comparison%20of%20phylogenetic%20informativeness%20using%20locus-specific%20substitution%20models.%202012.%20arXiv%20preprint%20arXiv%3A12021215."><span><span>Google Scholar</span></span></a></span></span></span><span class="Occurrences"></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR42" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">López-Giráldez
 F, Townsend JP. PhyDesign: an online application for profiling 
phylogenetic informativeness. BMC Evol Biol. 2011;11:152.</span><span class="Occurrences"><span class="Occurrence OccurrencePMCID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3124428"><span><span>PubMed Central</span></span></a></span><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1186/1471-2148-11-152"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=21627831"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=PhyDesign%3A%20an%20online%20application%20for%20profiling%20phylogenetic%20informativeness&amp;author=F.%20L%C3%B3pez-Gir%C3%A1ldez&amp;author=JP.%20Townsend&amp;journal=BMC%20Evol%20Biol&amp;volume=11&amp;pages=152&amp;publication_year=2011"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR43" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Drummond AJ, Suchard MA, Xie D, Rambaut A. Bayesian Phylogenetics with BEAUti and the BEAST 1.7. Mol Biol Evol. 2012;29:1969–73.</span><span class="Occurrences"><span class="Occurrence OccurrencePMCID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3408070"><span><span>PubMed Central</span></span></a></span><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1093/molbev/mss075"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=22367748"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Bayesian%20Phylogenetics%20with%20BEAUti%20and%20the%20BEAST%201.7&amp;author=AJ.%20Drummond&amp;author=MA.%20Suchard&amp;author=D.%20Xie&amp;author=A.%20Rambaut&amp;journal=Mol%20Biol%20Evol&amp;volume=29&amp;pages=1969-1973&amp;publication_year=2012"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR44" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Ree
 RH, Smith SA. Maximum likelihood inference of geographic range 
evolution by dispersal, local extinction, and cladogenesis. Syst Biol. 
2008;57:4–14.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1080/10635150701883881"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=18253896"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Maximum%20likelihood%20inference%20of%20geographic%20range%20evolution%20by%20dispersal%2C%20local%20extinction%2C%20and%20cladogenesis&amp;author=RH.%20Ree&amp;author=SA.%20Smith&amp;journal=Syst%20Biol&amp;volume=57&amp;pages=4-14&amp;publication_year=2008"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR45" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Beaulieu
 JM, Tank DC, Donoghue MJ. A Southern Hemisphere origin for campanulid 
angiosperms, with traces of the break-up of Gondwana. BMC Evol Biol. 
2013;13:80.</span><span class="Occurrences"><span class="Occurrence OccurrencePMCID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3636071"><span><span>PubMed Central</span></span></a></span><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1186/1471-2148-13-80"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=23565668"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=A%20Southern%20Hemisphere%20origin%20for%20campanulid%20angiosperms%2C%20with%20traces%20of%20the%20break-up%20of%20Gondwana&amp;author=JM.%20Beaulieu&amp;author=DC.%20Tank&amp;author=MJ.%20Donoghue&amp;journal=BMC%20Evol%20Biol&amp;volume=13&amp;pages=80&amp;publication_year=2013"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR46" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Yu
 Y, Harris AJ, Blair C, He X. RASP (Reconstruct Ancestral State in 
Phylogenies): a tool for historical biogeography. Mol Phylogen Evol. 
2015;87:46–9.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1016/j.ympev.2015.03.008"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=RASP%20%28Reconstruct%20Ancestral%20State%20in%20Phylogenies%29%3A%20a%20tool%20for%20historical%20biogeography&amp;author=Y.%20Yu&amp;author=AJ.%20Harris&amp;author=C.%20Blair&amp;author=X.%20He&amp;journal=Mol%20Phylogen%20Evol&amp;volume=87&amp;pages=46-49&amp;publication_year=2015"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR47" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Aberer
 AJ, Krompass D, Stamatakis A. Pruning rogue taxa improves phylogenetic 
accuracy: an efficient algorithm and webservice. Syst Biol. 
2013;62:162–6.</span><span class="Occurrences"><span class="Occurrence OccurrencePMCID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3526802"><span><span>PubMed Central</span></span></a></span><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1093/sysbio/sys078"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=22962004"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Pruning%20rogue%20taxa%20improves%20phylogenetic%20accuracy%3A%20an%20efficient%20algorithm%20and%20webservice&amp;author=AJ.%20Aberer&amp;author=D.%20Krompass&amp;author=A.%20Stamatakis&amp;journal=Syst%20Biol&amp;volume=62&amp;pages=162-166&amp;publication_year=2013"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR48" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Makunin
 IV, Shloma VV, Stephen SJ, Pheasant M, Belyakin SN. Comparison of 
ultra-conserved elements in Drosophilids and Vertebrates. PLoS ONE. 
2013;8:e82362.</span><span class="Occurrences"><span class="Occurrence OccurrencePMCID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3862641"><span><span>PubMed Central</span></span></a></span><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1371/journal.pone.0082362"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=24349264"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Comparison%20of%20ultra-conserved%20elements%20in%20Drosophilids%20and%20Vertebrates&amp;author=IV.%20Makunin&amp;author=VV.%20Shloma&amp;author=SJ.%20Stephen&amp;author=M.%20Pheasant&amp;author=SN.%20Belyakin&amp;journal=PLoS%20ONE&amp;volume=8&amp;pages=e82362&amp;publication_year=2013"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR49" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Bolton B. Synopsis and classification of Formicidae. Mem Am Entomol Inst. 2003;71:1–370.</span><span class="Occurrences"><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Synopsis%20and%20classification%20of%20Formicidae&amp;author=B.%20Bolton&amp;journal=Mem%20Am%20Entomol%20Inst&amp;volume=71&amp;pages=1-370&amp;publication_year=2003"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR50" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Whitfield JB, Kjer KM. Ancient rapid radiations of insects: challenges for phylogenetic analysis. Ann Rev Ent. 2008;53:449–72.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1146/annurev.ento.53.103106.093304"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Ancient%20rapid%20radiations%20of%20insects%3A%20challenges%20for%20phylogenetic%20analysis&amp;author=JB.%20Whitfield&amp;author=KM.%20Kjer&amp;journal=Ann%20Rev%20Ent&amp;volume=53&amp;pages=449-472&amp;publication_year=2008"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR51" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Wiegmann
 BM, Trautwein MD, Winkler IS, Barr NB, Kim J-W, Lambkin C, et al. 
Episodic radiations in the fly tree of life. Proc Natl Acad Sci. 
2011;108:5690–5.</span><span class="Occurrences"><span class="Occurrence OccurrencePMCID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3078341"><span><span>PubMed Central</span></span></a></span><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1073/pnas.1012675108"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=21402926"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Episodic%20radiations%20in%20the%20fly%20tree%20of%20life&amp;author=BM.%20Wiegmann&amp;author=MD.%20Trautwein&amp;author=IS.%20Winkler&amp;author=NB.%20Barr&amp;author=J-W.%20Kim&amp;author=C.%20Lambkin&amp;author=MA.%20Bertone&amp;author=BK.%20Cassel&amp;author=KM.%20Bayless&amp;author=AM.%20Heimberg&amp;journal=Proc%20Natl%20Acad%20Sci&amp;volume=108&amp;pages=5690-5695&amp;publication_year=2011"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR52" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Bazinet
 AL, Cummings MP, Mitter KT, Mitter CW. Can RNA-Seq resolve the rapid 
radiation of advanced moths and butterflies (Hexapoda: Lepidoptera: 
Apoditrysia)? An exploratory study. PLoS ONE. 2013; 8(12):e82615.<span xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Occurrences"><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="https://scholar.google.com/scholar?hl=en&amp;q=Bazinet%20AL%2C%20Cummings%20MP%2C%20Mitter%20KT%2C%20Mitter%20CW.%20Can%20RNA-Seq%20resolve%20the%20rapid%20radiation%20of%20advanced%20moths%20and%20butterflies%20%28Hexapoda%3A%20Lepidoptera%3A%20Apoditrysia%29%3F%20An%20exploratory%20study.%20PLoS%20ONE.%202013%3B%208%2812%29%3Ae82615."><span><span>Google Scholar</span></span></a></span></span></span><span class="Occurrences"></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR53" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Johnson
 BR, Borowiec ML, Chiu JC, Lee EK, Atallah J, Ward PS. Phylogenomics 
resolves evolutionary relationships among ants, bees, and wasps. Curr 
Biol. 2013;23:2058–62.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1016/j.cub.2013.08.050"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=24094856"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Phylogenomics%20resolves%20evolutionary%20relationships%20among%20ants%2C%20bees%2C%20and%20wasps&amp;author=BR.%20Johnson&amp;author=ML.%20Borowiec&amp;author=JC.%20Chiu&amp;author=EK.%20Lee&amp;author=J.%20Atallah&amp;author=PS.%20Ward&amp;journal=Curr%20Biol&amp;volume=23&amp;pages=2058-2062&amp;publication_year=2013"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR54" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Brady SG, Gadau J, Ward PS. Systematics of the ant genus <em class="EmphasisTypeItalic">Camponotus</em>
 (Hymenoptera: Formicidae): a preliminary analysis using data from the 
mitochondrial gene cytochrome oxidase I. In: Austin AD, Dowton M, 
editors. Hymenoptera: evolution, biodiversity and biological control. 
Canberra: Fourth International Hymenoptera Conference; 2000.</span><span class="Occurrences"><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Systematics%20of%20the%20ant%20genus%20Camponotus%20%28Hymenoptera%3A%20Formicidae%29%3A%20a%20preliminary%20analysis%20using%20data%20from%20the%20mitochondrial%20gene%20cytochrome%20oxidase%20I&amp;author=SG.%20Brady&amp;author=J.%20Gadau&amp;author=PS.%20Ward&amp;publication_year=2000"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR55" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Chen
 Z, Zhou S, Ye D, Chen Y, Lu C. Molecular phylogeny of the ant subfamily
 Formicinae (Hymenoptera, Formicidae) from China based on mitochondrial 
genes. Sociobiology. 2013;60:135–44.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.13102/sociobiology.v60i2.135-144"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Molecular%20phylogeny%20of%20the%20ant%20subfamily%20Formicinae%20%28Hymenoptera%2C%20Formicidae%29%20from%20China%20based%20on%20mitochondrial%20genes&amp;author=Z.%20Chen&amp;author=S.%20Zhou&amp;author=D.%20Ye&amp;author=Y.%20Chen&amp;author=C.%20Lu&amp;journal=Sociobiology&amp;volume=60&amp;pages=135-144&amp;publication_year=2013"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR56" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Williams LE, Wernegreen JJ. Genome evolution in an ancient bacteria-ant symbiosis: parallel gene loss among <em class="EmphasisTypeItalic">Blochmannia</em> spanning the origin of the ant tribe Camponotini. PeerJ. 2015;3:e881.</span><span class="Occurrences"><span class="Occurrence OccurrencePMCID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4389277"><span><span>PubMed Central</span></span></a></span><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.7717/peerj.881"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=25861561"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Genome%20evolution%20in%20an%20ancient%20bacteria-ant%20symbiosis%3A%20parallel%20gene%20loss%20among%20Blochmannia%20spanning%20the%20origin%20of%20the%20ant%20tribe%20Camponotini&amp;author=LE.%20Williams&amp;author=JJ.%20Wernegreen&amp;journal=PeerJ&amp;volume=3&amp;pages=e881&amp;publication_year=2015"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR57" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Wernegreen
 JJ, Kauppinen SN, Brady SG, Ward PS. One nutritional symbiosis begat 
another: phylogenetic evidence that the ant tribe Camponotini acquired <em class="EmphasisTypeItalic">Blochmannia</em> by tending sap-feeding insects. BMC Evol Biol. 2009;9:292.</span><span class="Occurrences"><span class="Occurrence OccurrencePMCID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2810300"><span><span>PubMed Central</span></span></a></span><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1186/1471-2148-9-292"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=20015388"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=One%20nutritional%20symbiosis%20begat%20another%3A%20phylogenetic%20evidence%20that%20the%20ant%20tribe%20Camponotini%20acquired%20Blochmannia%20by%20tending%20sap-feeding%20insects&amp;author=JJ.%20Wernegreen&amp;author=SN.%20Kauppinen&amp;author=SG.%20Brady&amp;author=PS.%20Ward&amp;journal=BMC%20Evol%20Biol&amp;volume=9&amp;pages=292&amp;publication_year=2009"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR58" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Near
 TJ, Sanderson MJ. Assessing the quality of molecular divergence time 
estimates by fossil calibrations and fossil-based model selection. 
Philos Trans R Soc Lond B Biol Sci. 2004;359:1477–83.</span><span class="Occurrences"><span class="Occurrence OccurrencePMCID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC1693436"><span><span>PubMed Central</span></span></a></span><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1098/rstb.2004.1523"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=15519966"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Assessing%20the%20quality%20of%20molecular%20divergence%20time%20estimates%20by%20fossil%20calibrations%20and%20fossil-based%20model%20selection&amp;author=TJ.%20Near&amp;author=MJ.%20Sanderson&amp;journal=Philos%20Trans%20R%20Soc%20Lond%20B%20Biol%20Sci&amp;volume=359&amp;pages=1477-1483&amp;publication_year=2004"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR59" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Ho
 SYW, Phillips MJ. Accounting for calibration uncertainty in 
phylogenetic estimation of evolutionary divergence times. Syst Biol. 
2009;58:367–80.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1093/sysbio/syp035"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=20525591"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Accounting%20for%20calibration%20uncertainty%20in%20phylogenetic%20estimation%20of%20evolutionary%20divergence%20times&amp;author=SYW.%20Ho&amp;author=MJ.%20Phillips&amp;journal=Syst%20Biol&amp;volume=58&amp;pages=367-380&amp;publication_year=2009"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR60" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Brady SG. Effects of fossil calibration-uncertainty on divergence dating in ants and bees. Am Ent. 2011;57:56.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1093/ae/57.1.56"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Effects%20of%20fossil%20calibration-uncertainty%20on%20divergence%20dating%20in%20ants%20and%20bees&amp;author=SG.%20Brady&amp;journal=Am%20Ent&amp;volume=57&amp;pages=56&amp;publication_year=2011"><span><span>Google Scholar</span></span></a></span></span></cite></li></ol></div></section><section xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" class="Section1 RenderAsSection1 c-section" id="CopyrightMessage"><h2 class="Heading js-ToggleCollapseSection c-section__heading" data-sticky-update="true"><a href="#Blaimeretal" data-toggle="collapse" data-toggle-disable-clickoff="" data-target="Blaimeretal">Copyright</a></h2><div class="js-CollapseSection CollapseSection c-section__content" data-test="collapse-section" id="Blaimeretal"><div xmlns="http://www.w3.org/1999/xhtml" class="ArticleCopyright">©&nbsp;Blaimer et al.&nbsp;2015</div></div></section></div>
                        


                    </article>
                </main>

                <div class="c-content-layout__side u-text-sm">
                    <aside>
                        <div class="p-separator">
                            
        <p class="u-margin-bottom-sm">
            <a id="articlePdf" data-test="download-pdf" class="c-button c-button--primary c-button--full-width" target="_blank" rel="noopener" href="https://bmcevolbiol.biomedcentral.com/track/pdf/10.1186/s12862-015-0552-5" data-track="click" data-track-category="PDF Download" data-track-action="Click download article PDF" data-track-label="10.1186/s12862-015-0552-5">Download PDF</a>
        </p>
    
                            
                            
    <button type="button" class="c-button c-button--secondary c-button--full-width c-button--expandable" data-toggle="collapse" data-target="export-citations" data-test="export-citations-button" data-sticky-update="true" aria-controls="export-citations" aria-expanded="false">
        <span class="c-dropdown__toggle">Export citations</span>
    </button>
    <div id="export-citations" class="c-export-citations" data-test="export-citations-content">
        <div class="c-export-citations__section">
            <h4 class="c-export-citations__title">Papers, Zotero, Reference Manager, RefWorks (<strong>.RIS</strong>)</h4>
            <ul class="c-export-citations__list" id="articleReferences">
                <li><a data-track="click" data-track-category="Article Page" data-track-action="Download Papers, Zotero, Reference Manager, RefWorks article citations" data-track-label="10.1186/s12862-015-0552-5" href="http://citation-needed.springer.com/v2/references/10.1186/s12862-015-0552-5?format=refman&amp;flavour=citation">
                    <svg class="c-export-citations__icon" width="12" height="12" aria-hidden="true">
                        <use xlink:href="#icon-download"></use>
                    </svg>
                    <span class="u-visually-hidden">Download </span>Citations
                </a></li>
                <li><a data-track="click" data-track-category="Article Page" data-track-action="Download Papers, Zotero, Reference Manager, RefWorks references" data-track-label="10.1186/s12862-015-0552-5" href="http://citation-needed.springer.com/v2/references/10.1186/s12862-015-0552-5?format=refman&amp;flavour=references">
                    <svg class="c-export-citations__icon" width="12" height="12" aria-hidden="true">
                        <use xlink:href="#icon-download"></use>
                    </svg>
                    <span class="u-visually-hidden">Download </span>References
                </a></li>
                <li><a data-track="click" data-track-category="Article Page" data-track-action="Download Papers, Zotero, Reference Manager, RefWorks citations and references" data-track-label="10.1186/s12862-015-0552-5" href="http://citation-needed.springer.com/v2/references/10.1186/s12862-015-0552-5?format=refman&amp;flavour=full">
                    <svg class="c-export-citations__icon" width="12" height="12" aria-hidden="true">
                        <use xlink:href="#icon-download"></use>
                    </svg>
                    <span class="u-visually-hidden">Download </span>Both
                </a></li>
            </ul>
        </div>
        <div class="c-export-citations__section">
            <h4 class="c-export-citations__title">EndNote (<strong>.ENW</strong>)</h4>
            <ul class="c-export-citations__list">
                <li><a data-track="click" data-track-category="Article Page" data-track-action="Download EndNote article citations" data-track-label="10.1186/s12862-015-0552-5" href="http://citation-needed.springer.com/v2/references/10.1186/s12862-015-0552-5?format=endnote&amp;flavour=citation">
                    <svg class="c-export-citations__icon" width="12" height="12" aria-hidden="true">
                        <use xlink:href="#icon-download"></use>
                    </svg>
                    <span class="u-visually-hidden">Download </span>Citations
                </a></li>
                <li><a data-track="click" data-track-category="Article Page" data-track-action="Download EndNote references" data-track-label="10.1186/s12862-015-0552-5" href="http://citation-needed.springer.com/v2/references/10.1186/s12862-015-0552-5?format=endnote&amp;flavour=references">
                    <svg class="c-export-citations__icon" width="12" height="12" aria-hidden="true">
                        <use xlink:href="#icon-download"></use>
                    </svg>
                    <span class="u-visually-hidden">Download </span>References
                </a></li>
                <li><a data-track="click" data-track-category="Article Page" data-track-action="Download EndNote citations and references" data-track-label="10.1186/s12862-015-0552-5" href="http://citation-needed.springer.com/v2/references/10.1186/s12862-015-0552-5?format=endnote&amp;flavour=full">
                    <svg class="c-export-citations__icon" width="12" height="12" aria-hidden="true">
                        <use xlink:href="#icon-download"></use>
                    </svg>
                    <span class="u-visually-hidden">Download </span>Both
                </a></li>
            </ul>
        </div>
        <div class="c-export-citations__section">
            <h4 class="c-export-citations__title">Mendeley, JabRef (<strong>.BIB</strong>)</h4>
            <ul class="c-export-citations__list">
                <li><a data-track="click" data-track-category="Article Page" data-track-action="Download Mendeley, JabRef article citations" data-track-label="10.1186/s12862-015-0552-5" href="http://citation-needed.springer.com/v2/references/10.1186/s12862-015-0552-5?format=bibtex&amp;flavour=citation">
                    <svg class="c-export-citations__icon" width="12" height="12" aria-hidden="true">
                        <use xlink:href="#icon-download"></use>
                    </svg>
                    <span class="u-visually-hidden">Download </span>Citations
                </a></li>
                <li><a data-track="click" data-track-category="Article Page" data-track-action="Download Mendeley, JabRef references" data-track-label="10.1186/s12862-015-0552-5" href="http://citation-needed.springer.com/v2/references/10.1186/s12862-015-0552-5?format=bibtex&amp;flavour=references">
                    <svg class="c-export-citations__icon" width="12" height="12" aria-hidden="true">
                        <use xlink:href="#icon-download"></use>
                    </svg>
                    <span class="u-visually-hidden">Download </span>References
                </a></li>
                <li><a data-track="click" data-track-category="Article Page" data-track-action="Download Mendeley, JabRef citations and references" data-track-label="10.1186/s12862-015-0552-5" href="http://citation-needed.springer.com/v2/references/10.1186/s12862-015-0552-5?format=bibtex&amp;flavour=full">
                    <svg class="c-export-citations__icon" width="12" height="12" aria-hidden="true">
                        <use xlink:href="#icon-download"></use>
                    </svg>
                    <span class="u-visually-hidden">Download </span>Both
                </a></li>
            </ul>
        </div>
    </div>

                        </div>
                        
    
    <div id="articleSections" class="p-separator">
        <h3 class="p-section-title">Section</h3>
        <ul class="c-list-group c-list-group--bordered c-list-group c-list-group--md">
            
                <li class="c-list-group__item">
                    <a href="https://bmcevolbiol.biomedcentral.com/articles/sections/phylogenetics-and-phylogeography" data-track="click" data-track-category="Article Page" data-track-action="Click section link" data-track-label="10.1186/s12862-015-0552-5">Phylogenetics and phylogeography</a>
                </li>
            
        </ul>
    </div>
    
 
                        
    
 
                        <div class="p-separator">
                            <h3 class="p-section-title">Metrics</h3>
                            <ul class="c-list-group c-list-group--md">
                                
            <li class="c-list-group__item">
                Article accesses:
                
                    4274
                
            </li>
        
                                
            <li class="c-list-group__item">
                Citations: 51 <a href="http://citations.springer.com/item?doi=10.1186/s12862-015-0552-5" rel="noopener" target="_blank">more information</a>
            </li>
        
                                
    <li class="c-list-group__item">
        <p>Altmetric Attention Score: 15</p>
        <div class="altmetric">
            <a class="altmetric__link" href="http://www.altmetric.com/details.php?citation_id=4841718&amp;domain=www.biomedcentral.com" target="_blank" rel="noopener" data-track="click" data-track-category="Article Page" data-track-action="Click Altmetric" data-track-label="10.1186/s12862-015-0552-5">
                <img class="altmetric__img" src="a.png" alt="Altmetric Attention Score">
            </a>
        </div>
    </li>

                            </ul>
                        </div>
                        
    <div id="sociaMediaShare" class="p-separator u-js-shown">
        <h3 class="p-section-title">Share this article</h3>
        <ul id="shareButtons" class="c-social-links">
            <li class="c-social-links__item">
                <button id="shareOnTwitter" class="js-btnShareOnTwitter" data-track="click" data-track-category="Article Page" data-track-action="Share Twitter" data-track-label="10.1186/s12862-015-0552-5">
                    <span class="u-visually-hidden">Share on Twitter</span>
                    <svg class="c-icon c-social-links__icon" width="24" height="24" aria-hidden="true">
                        <use xlink:href="#icon-twitter"></use>
                    </svg>
                </button>
            </li>
            <li class="c-social-links__item">
                <button id="shareOnFacebook" class="js-btnShareOnFacebook" data-track="click" data-track-category="Article Page" data-track-action="Share Facebook" data-track-label="10.1186/s12862-015-0552-5">
                    <span class="u-visually-hidden">Share on Facebook</span>
                    <svg class="c-icon c-social-links__icon" width="24" height="24" aria-hidden="true">
                        <use xlink:href="#icon-facebook"></use>
                    </svg>
                </button>
            </li>
            <li class="c-social-links__item">
                <button id="shareOnLinkedIn" class="js-btnShareOnLinkedIn" data-track="click" data-track-category="Article Page" data-track-action="Share LinkedIn" data-track-label="10.1186/s12862-015-0552-5">
                    <span class="u-visually-hidden">Share on LinkedIn</span>
                    <svg class="c-icon c-social-links__icon" width="24" height="24" aria-hidden="true">
                        <use xlink:href="#icon-linkedin"></use>
                    </svg>
                </button>
            </li>
            <li class="c-social-links__item">
                <button id="shareOnWeibo" class="js-btnShareOnWeibo" data-track="click" data-track-category="Article Page" data-track-action="Share Weibo" data-track-label="10.1186/s12862-015-0552-5">
                    <span class="u-visually-hidden">Share on Weibo</span>
                    <svg class="c-icon c-social-links__icon" width="24" height="24" aria-hidden="true">
                        <use xlink:href="#icon-weibo"></use>
                    </svg>
                </button>
            </li>
            <li class="c-social-links__item">
                <button id="shareOnGooglePlus" class="js-btnShareOnGplus" data-track="click" data-track-category="Article Page" data-track-action="Share Google+" data-track-label="10.1186/s12862-015-0552-5">
                    <span class="u-visually-hidden">Share on Google Plus</span>
                    <svg class="c-icon c-social-links__icon" width="24" height="24" aria-hidden="true">
                        <use xlink:href="#icon-gplus"></use>
                    </svg>
                </button>
            </li>
            <li class="c-social-links__item">
                <button id="shareOnReddit" class="js-btnShareOnReddit" data-track="click" data-track-category="Article Page" data-track-action="Share Reddit" data-track-label="10.1186/s12862-015-0552-5">
                    <span class="u-visually-hidden">Share on Reddit</span>
                    <svg class="c-icon c-social-links__icon" width="24" height="24" aria-hidden="true">
                        <use xlink:href="#icon-reddit"></use>
                    </svg>
                </button>
            </li>
        </ul>
    </div>
 
                        
    
        <div class="p-separator u-js-shown">
            <h3 class="p-section-title">See updates</h3>
            <button class="c-crossmark-button" data-target="crossmark" data-test="crossmark" data-track="click" data-track-category="Article Page" data-track-action="Click CrossMark" data-track-label="10.1186/s12862-015-0552-5" style="cursor: pointer;">
                <img id="crossmark-icon" src="CROSSMARK_Color_horizontal.svg" alt="Crossmark logo">
            </button>
        </div>
    
 
                        <div class="p-separator">
                            <h3 class="p-section-title">Other Actions</h3>
                            <ul class="c-list-group c-list-group--bordered c-list-group c-list-group--md">
                                <li class="c-list-group__item">
                                    
    <a data-track="click" data-track-category="Article Page" data-track-action="Order article reprints" data-track-label="10.1186/s12862-015-0552-5" href="https://www.odysseypress.com/onlinehost/reprint_order.php?type=A&amp;page=0&amp;journal=738&amp;doi=10.1186%2Fs12862-015-0552-5&amp;volume=15&amp;issue=1&amp;title=Phylogenomic+methods+outperform+traditional+multi-locus+approaches+in+resolving+deep+evolutionary+history%3A+a+case+study+of+formicine+ants&amp;author_name=Bonnie+B.+Blaimer&amp;start_page=1&amp;end_page=14">Order reprint</a>

                                </li>
                            </ul>
                        </div>
                    </aside>
                    
    <div id="FulltextAd" class="adsbox c-ad c-ad--skyscraper">
        <div class="c-ad__inner" data-component="sticky" data-test="fulltext-ad">
            <p class="c-ad__label">Advertisement</p>
            <div class="c-ad__content">
                <span id="pbgrd-43415139"></span>
            </div>
        </div>
    </div>

                </div>
            </div>
        </div>


        
            <div class="c-journal-footer">
                <div class="c-journal-footer__inner">
                    <div class="c-journal-footer__summary">
                        <h4 class="c-journal-title c-journal-title--footer">
                            
                            <span class="c-journal-title__text">BMC Evolutionary Biology</span>
                        </h4>
                        <p class="c-journal-footer__issn">ISSN: 1471-2148</p>
                    </div>
                    
                        <div class="c-journal-footer__contact">
                            <h4 class="c-journal-footer__contact-title ">Contact us</h4>
                            <ul class="c-journal-footer__contact-list">
                                
                                    <li class="c-journal-footer__contact-item">Submission enquiries: <a href="mailto:bmcevolbiol@biomedcentral.com">bmcevolbiol@biomedcentral.com</a></li>
                                
                                
                                    <li class="c-journal-footer__contact-item">General enquiries: <a href="mailto:info@biomedcentral.com">info@biomedcentral.com</a></li>
                                
                            </ul>
                        </div>
                    
                </div>
            </div>
        
        
    <img rel="nofollow" class="tracker" style="display:none" src="s12862-015-0552-5.png" alt="">

         
    <footer>
        <div class="ctx-publisher-footer">
    <div class="c-navbar">
        <div class="u-container">
            <div class="c-navbar__logo u-margin-right-xl">
                <a href="https://www.biomedcentral.com/">
                    <img src="logo-bmc-series-9977e4f82f.svg" alt="BMC logo" data-test="navbar-logo-footer">

                </a>
            </div>
            <div class="c-navbar__content">
                
    <ul class="c-navbar__nav" data-test="publisher-navigation">
        
        <li class="c-navbar__item">
            <a class="c-navbar__link" href="https://www.biomedcentral.com/journals">
                Explore journals
            </a>
        </li>
        
        <li class="c-navbar__item">
            <a class="c-navbar__link" href="https://www.biomedcentral.com/getpublished">
                Get published
            </a>
        </li>
        
        <li class="c-navbar__item">
            <a class="c-navbar__link" href="https://www.biomedcentral.com/about">
                About BMC
            </a>
        </li>
        
    </ul>


            </div>
        </div>
    </div>
    <div class="c-publisher-footer" data-test="publisher-footer">
        <div class="u-container">
            
            <div class="c-flex c-flex--wrap c-flex--space-between" role="navigation" data-test="publisher-footer-menu">
                <div class="c-flex" aria-label="Publisher footer links">
                    
                        
                            <ul class="c-list-group c-list-group--sm u-margin-right-lg u-margin-bottom-md">
                                
                                    <li class="c-list-group__item">
                                        <a class="u-text-inherit" href="http://blogs.biomedcentral.com/">Read more on our blogs</a>
                                    </li>
                                
                                    <li class="c-list-group__item">
                                        <a class="u-text-inherit" href="https://www.biomedcentral.com/login">Receive BMC newsletters</a>
                                    </li>
                                
                                    <li class="c-list-group__item">
                                        <a class="u-text-inherit" href="https://www.biomedcentral.com/account">Manage article alerts</a>
                                    </li>
                                
                                    <li class="c-list-group__item">
                                        <a class="u-text-inherit" href="https://authorservices.springernature.com/go/10BMC">Language editing for authors</a>
                                    </li>
                                
                                    <li class="c-list-group__item">
                                        <a class="u-text-inherit" href="http://authorservices.springernature.com/scientific-editing/">Scientific editing for authors</a>
                                    </li>
                                
                            </ul>
                        
                            <ul class="c-list-group c-list-group--sm u-margin-right-lg u-margin-bottom-md">
                                
                                    <li class="c-list-group__item">
                                        <a class="u-text-inherit" href="https://www.biomedcentral.com/about/policies">Policies</a>
                                    </li>
                                
                                    <li class="c-list-group__item">
                                        <a class="u-text-inherit" href="https://www.biomedcentral.com/accessibility">Accessibility</a>
                                    </li>
                                
                                    <li class="c-list-group__item">
                                        <a class="u-text-inherit" href="https://www.biomedcentral.com/about/press-centre">Press center</a>
                                    </li>
                                
                            </ul>
                        
                            <ul class="c-list-group c-list-group--sm u-margin-right-lg u-margin-bottom-md">
                                
                                    <li class="c-list-group__item">
                                        <a class="u-text-inherit" href="https://www.biomedcentral.com/about/contact-us">Contact us</a>
                                    </li>
                                
                                    <li class="c-list-group__item">
                                        <a class="u-text-inherit" href="https://biomedcentral.typeform.com/to/VLXboo">Leave feedback</a>
                                    </li>
                                
                                    <li class="c-list-group__item">
                                        <a class="u-text-inherit" href="https://www.biomedcentral.com/about/jobs">Careers</a>
                                    </li>
                                
                            </ul>
                        
                    
                </div>
                <div class="u-margin-bottom-lg">
                    <h3 id="social-menu" class="u-text-sm u-reset-margin u-text-normal">Follow BMC</h3>
                    <ul class="c-flex u-reset-list" data-test="footer-social-links">
                        
                            <li class="u-margin-top-xs u-margin-right-xs">
                                <a href="https://twitter.com/biomedcentral" class="u-text-inherit" data-track="click" data-track-action="Click Twitter link" data-track-category="Publisher footer">
                                    <span class="u-visually-hidden">BMC Twitter page</span>
                                    <svg class="c-icon" width="24" height="24" aria-hidden="true">
                                        <use xlink:href="#icon-twitter-bordered"></use>
                                    </svg>
                                </a>
                            </li>
                        
                            <li class="u-margin-top-xs u-margin-right-xs">
                                <a href="https://www.facebook.com/BioMedCentral" class="u-text-inherit" data-track="click" data-track-action="Click Facebook link" data-track-category="Publisher footer">
                                    <span class="u-visually-hidden">BMC Facebook page</span>
                                    <svg class="c-icon" width="24" height="24" aria-hidden="true">
                                        <use xlink:href="#icon-facebook-bordered"></use>
                                    </svg>
                                </a>
                            </li>
                        
                            <li class="u-margin-top-xs u-margin-right-xs">
                                <a href="https://plus.google.com/106226337984152901734/" class="u-text-inherit" data-track="click" data-track-action="Click Google Plus link" data-track-category="Publisher footer">
                                    <span class="u-visually-hidden">BMC Google Plus page</span>
                                    <svg class="c-icon" width="24" height="24" aria-hidden="true">
                                        <use xlink:href="#icon-gplus-bordered"></use>
                                    </svg>
                                </a>
                            </li>
                        
                            <li class="u-margin-top-xs u-margin-right-xs">
                                <a href="http://www.weibo.com/biomedcentral" class="u-text-inherit" data-track="click" data-track-action="Click Weibo link" data-track-category="Publisher footer">
                                    <span class="u-visually-hidden">BMC Weibo page</span>
                                    <svg class="c-icon" width="24" height="24" aria-hidden="true">
                                        <use xlink:href="#icon-weibo-bordered"></use>
                                    </svg>
                                </a>
                            </li>
                        
                    </ul>
                </div>
            </div>
            <p class="u-reset-margin">
                By using this website, you agree to our
                <a class="u-text-inherit" href="https://www.biomedcentral.com/terms-and-conditions">Terms and Conditions</a>,
                <a class="u-text-inherit" href="https://www.biomedcentral.com/privacy-statement">Privacy
                    statement</a> and
                <a class="u-text-inherit" href="https://www.biomedcentral.com/cookies" data-test="cookie-link">Cookies</a> policy.
                
                    <a class="optanon-toggle-display u-text-inherit" href="javascript:void(0);">Manage the cookies</a> we use in the preference centre.
                
            </p>
        </div>
    </div>
</div>


        <div class="c-sn-footer c-sn-footer--neutral">
    <div class="u-container">
        <img src="logo-springernature-8bd4e8d82f.svg" class="c-sn-footer__logo" alt="Springer Nature Logo" itemprop="logo">
        <p data-test="copyright">© 2018 BioMed Central Ltd unless otherwise stated. Part of
            <a href="http://www.springernature.com/" data-track="click" data-track-action="Click Springer Nature link" data-track-category="Corporate footer" itemscope="" itemtype="http://schema.org/Organization" itemid="#parentOrganization">Springer Nature</a>.
        </p>
    </div>
</div>

    </footer>

        

    </div>

    
    <script type="text/javascript" src="bmc.js" async="async"></script>
    <noscript>
	<img hidden src="https://verify.nature.com/verify/nature.png" border="0" width="0" height="0" style="display: none">
</noscript>

    


<script type="text/javascript" id="polyfill-matches">Element.prototype.matches||(Element.prototype.matches=Element.prototype.matchesSelector||Element.prototype.mozMatchesSelector||Element.prototype.msMatchesSelector||Element.prototype.oMatchesSelector||Element.prototype.webkitMatchesSelector||function(a){a=(this.document||this.ownerDocument).querySelectorAll(a);for(var b=a.length;0<=--b&&a.item(b)!==this;);return-1<b});</script><script type="text/javascript" id="gpt-control" src="gpt.js"></script><script type="text/javascript" id="krux-control-bmc">window.Krux||((Krux=function(){Krux.q.push(arguments)}).q=[]);(function(){var a=document.createElement("script");a.type="text/javascript";a.async=!0;a.src=("https:"===location.protocol?"https:":"http:")+"//cdn.krxd.net/controltag/KDqylSLE.js";var b=document.getElementsByTagName("script")[0];b.parentNode.insertBefore(a,b)})();</script><script type="text/javascript" id="krux-interchange-bmc">window.Krux||((Krux=function(){Krux.q.push(arguments)}).q=[]);(function(){function c(a){a="kxmacmillan_"+a;try{var b=window.localStorage}catch(e){b=null}return b?b[a]||"":navigator.cookieEnabled?(b=document.cookie.match(a+"\x3d([^;]*)"))&&unescape(b[1])||"":""}Krux.user=c("user");Krux.segments=c("segs")?c("segs").split(","):[];var a=[];Krux.user&&a.push("kuid\x3d"+Krux.user);for(var d=0;d<Krux.segments.length;d++)a.push("ksg\x3d"+Krux.segments[d]);Krux.dfppKeyValues=a.length?a.join(";")+";":""})();</script><script type="text/javascript" id="krux-consent">var allowed=google_tag_manager["GTM-TDGJHK"].macro(23);allowed=!0===allowed?1:0;Krux("consent:set",{dc:allowed,al:allowed,tg:allowed,cd:!1,sh:!1,re:!1},function(a,b){});</script><script type="text/javascript" id="gpt-retrieve-ads">function splitKeys(a){var d=[],e="",g=[],h=a.split(";");for(a=0;a<h.length;++a){void 0!==k&&(e=k);var b=h[a].split("\x3d");var k=b[0];k!==e&&(0<e.length&&0<d.length&&g.push([e,d]),d=[]);if(2===b.length&&""!==b[0]&&""!==b[1]){var c=b[1].split(",");for(b=0;b<c.length;++b)d.push(c[b])}}0<k.length&&0<d.length&&g.push([k,d]);return g}
function splitSizes(a){var d=[];if(null!==a){var e=0<=a.indexOf("|")?a.split("|"):a.split(",");for(a=0;a<e.length;++a){var g=e[a].split("x");var h=parseInt(g[0],10);var b=parseInt(g[1],10);2===g.length&&!isNaN(h)&&!isNaN(b)&&0<=h&&0<=b&&d.push([h,b])}}return d}function debounce(a,d){var e=null,g=null;return function(){var h=this,b=Number(new Date),k=arguments;e&&b<e+d?(clearTimeout(g),g=setTimeout(function(){e=b;a.apply(h,k)},d)):(e=b,a.apply(h,k))}}
function addScrollEvent(a){window.addEventListener?window.addEventListener("scroll",a,!1):window.attachEvent("onscroll",a)}function removeScrollEvent(a){window.removeEventListener?window.removeEventListener("scroll",a,!1):window.detachEvent("scroll",a)}function getAdContainers(){return document.querySelectorAll("div[data-gpt-unitpath]")}function isLoggedIn(){return 0<=document.cookie.indexOf("OSCAR_SESSION_COOKIE")?"logged\x3dy;":"logged\x3dn;"}
(function(a,d){function e(f){var a="test";a=a.replace(/[\[\]]/g,"\\$\x26");a=new RegExp("[?\x26]"+a+"(\x3d([^\x26#]*)|\x26|#|$)");a=(a=a.exec(window.location.href))?a[2]?decodeURIComponent(a[2].replace(/\+/g," ")):"":null;a=a&&"ads"===a?"adtype\x3dtest;":"";f=f.getAttribute("data-gpt-targeting");var c=document.querySelector("[data-ad-targeting-search-terms]")?"search\x3d"+document.querySelector("[data-ad-targeting-search-terms]").innerText+";":!1;var d=isLoggedIn();a&&-1===f.indexOf(a)&&(f+=a);c&&
(f+=c);return f+d+b}function g(a,c){for(var f=a.length,b=[];f--;)c(a[f],f)&&(b.push(a[f].slot),a.splice(f,1));b.length&&googletag.pubads().refresh(b);return a}function h(f){var c=Math.max(document.documentElement.clientHeight,a.innerHeight||0);return g(f,function(a){a=document.getElementById(a.divId);a=a.getBoundingClientRect();a=a.top-300;return c>a})}var b,k=getAdContainers(),c=[];d&&(b=d.dfppKeyValues?d.dfppKeyValues:"");a.googletag=a.googletag||{};a.googletag.cmd=a.googletag.cmd||[];for(var m=
0;k[m];++m){var l=k[m];splitKeys(e(l));c.push({divId:l.getAttribute("id"),adUnitPath:l.getAttribute("data-gpt-unitpath"),sizeArray:splitSizes(l.getAttribute("data-gpt-sizes")),targeting:splitKeys(e(l))})}googletag.cmd.push(function(){googletag.pubads().setRequestNonPersonalizedAds(google_tag_manager["GTM-TDGJHK"].macro(27)?0:1);googletag.pubads().disableInitialLoad();googletag.enableServices()});googletag.cmd.push(function(){for(var a=0;c[a];++a)try{c[a].slot=googletag.defineSlot(c[a].adUnitPath,c[a].sizeArray,c[a].divId).addService(googletag.pubads());
for(var b=0,d=c[a].targeting.length;b<d;++b)2===c[a].targeting[b].length&&""!==c[a].targeting[b][0]&&""!==c[a].targeting[b][1]&&c[a].slot.setTargeting(c[a].targeting[b][0],c[a].targeting[b][1])}catch(p){console.log("failed to create slot for",c[a])}});googletag.cmd.push(function(){for(var a=0;c[a];++a)googletag.display(c[a].divId)});googletag.cmd.push(function(){c=h(c)});var n=debounce(function(){googletag.cmd.push(function(){c=h(c);c.length||removeScrollEvent(n)})},250);addScrollEvent(n)})(window,
window.Krux);</script>

<script type="text/javascript" id="ga-data-track-event-listeners">function addEvent(b){document.addEventListener(b,function(a){a=a.target;var c='[data-track\x3d"'+b+'"]';a=google_tag_manager["GTM-TDGJHK"].macro(35)(a,c);void 0!==a&&dataLayer.push({event:"data-track",eventAction:a.getAttribute("data-track-action"),eventCategory:a.getAttribute("data-track-category"),eventLabel:a.getAttribute("data-track-label")||null,eventValue:a.getAttribute("data-track-value")||null})},!0)}(function(){addEvent("click");addEvent("submit")})();</script><script type="text/javascript" id="crossmark-script" src="widget.js"></script><script type="text/javascript" id="recommended-script" src="entry-point"></script><iframe style="width: 0px; height: 0px; bottom: 0px; right: 0px; z-index: 100000; position: fixed;" scrolling="no" src="https://recommended.springernature.com/v2197/generated/client.html?source=https://bmcevolbiol.biomedcentral.com" name="uptodate-client" id="uptodate-client" title="Personalised recommendations" allowtransparency="true" frameborder="0"></iframe></body><iframe id="kx-proxy-KDqylSLE" src="https://cdn.krxd.net/partnerjs/xdi/proxy.3d2100fd7107262ecb55ce6847f01fa5.html#%21kxcid=KDqylSLE&amp;kxt=https%3A%2F%2Fbmcevolbiol.biomedcentral.com&amp;kxcl=cdn&amp;kxp=" style="display: none; visibility: hidden; height: 0; width: 0;"></iframe></html>