<!DOCTYPE html> <html class=pb-page data-request-id=3ad69415-3e80-40d9-a5e3-526873b6068b style lang=en><!--
 Page saved with SingleFileZ 
 url: https://onlinelibrary.wiley.com/doi/10.1111/j.1600-0587.2009.06148.x 
 saved date: Tue Oct 20 2020 18:43:30 GMT+0200 (CEST)
--><head data-pb-dropzone=head><meta charset=utf-8><meta http-equiv=X-UA-Compatible content="IE=edge">
<meta name=pbContext content=";page:string:Article/Chapter View;ctype:string:Journal Content;website:website:pericles;article:article:doi\:10.1111/j.1600-0587.2009.06148.x;journal:journal:16000587;issue:issue:doi\:10.1111/eco.2010.33.issue-1;wgroup:string:Publication Websites;requestedJournal:journal:16000587;pageGroup:string:Publication Pages;subPage:string:Full Text">
<meta name=robots content=noarchive><meta name=pb-robots-disabled><meta property=og:title content="A diversity of beta diversities: straightening up a concept gone awry. Part 2. Quantifying beta diversity and related phenomena"><meta property=og:type content=Article><meta property=og:url content=https://onlinelibrary.wiley.com/doi/10.1111/j.1600-0587.2009.06148.x><meta property=og:image content=https://onlinelibrary.wiley.com/cms/asset/1eb6a05b-f2c4-4732-ae7b-df20b9535f83/eco.2010.33.issue-1.cover.gif><meta property=og:site_name content="Wiley Online Library"><meta property=og:description content="The present two‐part review aims to put the different phenomena that have been called “beta diversity” over the years into a common conceptual framework and to explain what each of them measures. The..."><meta name=viewport content="width=device-width,initial-scale=1"><meta name=apple-itunes-app content="app-id=782998940"><meta name=citation_doi content=10.1111/j.1600-0587.2009.06148.x><meta name=dc.identifier content=10.1111/j.1600-0587.2009.06148.x><meta name=citation_journal_title content=Ecography><meta name=citation_title content="A diversity of beta diversities: straightening up a concept gone awry. Part 2. Quantifying beta diversity and related phenomena"><meta name=citation_volume content=33><meta name=citation_issue content=1><meta name=citation_firstpage content=23><meta name=citation_lastpage content=45><meta name=citation_publication_date content=2010/02/01><meta name=citation_online_date content=2010/02/11><meta name=citation_issn content=1600-0587><meta name=citation_language content=en><meta name=citation_author content="Hanna Tuomisto"><meta name=citation_abstract_html_url content=https://onlinelibrary.wiley.com/doi/abs/10.1111/j.1600-0587.2009.06148.x><meta name=citation_fulltext_html_url content=https://onlinelibrary.wiley.com/doi/full/10.1111/j.1600-0587.2009.06148.x><meta name=citation_pdf_url content=https://onlinelibrary.wiley.com/doi/pdf/10.1111/j.1600-0587.2009.06148.x><meta name=citation_publisher content="John Wiley &amp; Sons, Ltd"><meta name=article_references content="Tuomisto, H. (2010), A diversity of beta diversities: straightening up a concept gone awry. Part 2. Quantifying beta diversity and related phenomena. Ecography, 33: 23-45. doi:10.1111/j.1600-0587.2009.06148.x"><meta name=epdf_available content=false><meta name=Description content="The present two‐part review aims to put the different phenomena that have been called “beta diversity” over the years into a common conceptual framework and to explain what each of them measures. T..."><title>A diversity of beta diversities: straightening up a concept gone awry. Part 2. Quantifying beta diversity and related phenomena - Tuomisto - 2010 - Ecography - Wiley Online Library</title><meta name=referrer content=origin-when-cross-origin><meta name=google-site-verification content=ffz4i4DxIJCdGpRXe0XN6KG6o4FYI-nnFW9I_dg8piM><link rel="shortcut icon" href=images/19.ico><style>.sf-hidden{display:none!important}</style><link rel=canonical href=https://onlinelibrary.wiley.com/doi/10.1111/j.1600-0587.2009.06148.x><body class=pb-ui><noscript>
				<a href="/action/clickThrough?id=1013&url=https%3A%2F%2Fpubads.g.doubleclick.net%2Fgampad%2Fjump%3Fiu%3D%24googlePublisherCategory%26amp%3Bsz%3D728x90%26amp%3Btile%3D1%26amp%3Bc%3D%24random&loc=%2Fdoi%2F10.1111%2Fj.1600-0587.2009.06148.x&pubId=58070509&placeholderId=1015&productId=101018" target="_blank">
					<img src="https://pubads.g.doubleclick.net/gampad/ad?iu=$googlePublisherCategory&amp;sz=728x90&amp;tile=1&amp;c=$random">
				</a>
			</noscript>
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
<link rel=stylesheet type=text/css href=stylesheet_0.css>
 <link rel=stylesheet href=stylesheet_1.css>
<link rel=stylesheet type=text/css href=stylesheet_2.css>
<link rel=stylesheet type=text/css href=stylesheet_3.css>
<link rel=stylesheet type=text/css href=stylesheet_4.css>
<link rel=stylesheet type=text/css href=stylesheet_5.css>
<style>.publication-cover-info{margin-bottom:20px}.article-row-right>.pb-ad{margin-left:0;margin-bottom:10px}.admin-message{background-color:#005274;color:white}.banner-wrapper .pb-ad{padding:0}.advert.advert-leaderboard{margin:15px 0}.banner-wrapper{box-shadow:none}.cover-image__image img{max-width:150px;max-height:150px}h1.citation__title{font-size:1.375rem}@media (max-width:991px){h1.citation__title{font-size:1.125rem}h1.citation__title:after{content:""}}.cart-indicator__wrapper{padding:0}</style>
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
<style>.at-icon-wrapper{overflow:hidden}</style><link href=stylesheet_6.css type=text/css rel=stylesheet><style>#MathJax_Message{position:fixed;left:1px;bottom:2px;background-color:#E6E6E6;border:1px solid #959595;margin:0px;padding:2px 8px;z-index:102;color:black;font-size:80%;width:auto;white-space:nowrap}</style><link id=altmetric-embed-css rel=stylesheet type=text/css href=stylesheet_7.css>
<div id=MathJax_Message style=display:none></div>
<div class=skip-links>
 <ul class=rlist>
 
 <li><a href=#article__content tabindex=1>Skip to Article Content</a></li>
 <li><a href=#pane-pcw-detailscon tabindex=1>Skip to Article Information</a></li>
 
 </ul>
</div>
 <div id=pb-page-content data-ng-non-bindable>
 <div data-pb-dropzone=main data-pb-dropzone-name=Main>
 <div data-widget-def=pageHeader data-widget-id=62175478-b98a-43d5-90f2-1f2045ac1a7e class=pageHeader>
 
 
 <header class=page-header>
 <div data-pb-dropzone=main>
 
 <div data-widget-def=ux3-layout-widget data-widget-id=bb5b5e2c-d3d6-4015-b586-eeade38f59d4 class=admin-message>
 
 
 <div class=container><div class=row><div>
 
 
 <div class=pb-ad>
 <div style="background=blue;">
 <div class=container>
 <p style=text-align:center><b>Working off-campus?</b> Learn about our <a href="https://onlinelibrary.wiley.com/action/clickThrough?id=101169&amp;url=https%3A%2F%2Fonlinelibrary.wiley.com%2Fresearchers%2Fread%2Faccess-options&amp;loc=%2Fdoi%2F10.1111%2Fj.1600-0587.2009.06148.x&amp;pubId=58070509&amp;placeholderId=1029&amp;productId=1011" target=_blank><font color=white><u>remote access options</u></font></a></p>
 </div>
</div>
 </div>
 
</div></div></div>
 </div>
 
 
 <div class=container><div class=row><div class="contents col-sm-12 col-md-12">
 <div data-widget-def=UX3InstitutionBanner data-widget-id=0a735273-4141-44d8-801a-2b89143bed55 class="institution-wrapper-mobile hidden-lg padded-content sf-hidden" aria-hidden=true>
 
 
 
 </div>
 
 
 <div class=pull-left>
 <div data-widget-def=ux3-general-image data-widget-id=ff262197-0fb8-4973-aaa2-2fceba4b5a61 class="logo-container hidden-xs hidden-sm hidden-md">
 
 
 <a href=https://onlinelibrary.wiley.com/ title="Wiley Online Library" tabindex=2><img id=mainLogo alt="Wiley Online Library" src=images/20.png></a>
 </div>
 
 <div data-widget-def=ux3-general-image data-widget-id=9a75b789-224c-4745-833a-930cf6c36f35 class="logo-container-mobile hidden-lg hidden-md hidden-sm sf-hidden">
 
 
 
 </div>
 
 <div data-widget-def=UX3InstitutionBanner data-widget-id=14d0e2f8-27d3-42d0-a477-72015e14e818 class="institution-wrapper hidden-xs hidden-sm hidden-md institution--block">
 
 
 <div class=institution><div class=institution-info-wrapper><span class=institution__intro>Access by</span><span class=institution__name>Czechs- Biological Institution Of Cas</span></div></div>
 </div>
 
</div><div class=pull-right>
 <div data-widget-def=UX3QuickSearchWidget data-widget-id=48175bb0-0f70-46e6-bb89-7e2781fdb8e4 class=header-quicksearch>
 
 
 <div class=quick-search-wrapper><a href=# title=search aria-label=Search class="quick-search__item sf-hidden"></a><div data-linkto-page=true class="quick-search quickSearchFormContainer"><div class=main-search-field-wrapper><div class="tab clearfix"><input placeholder=Search value tabindex=13 class="fakeQSInput quickSearchFilter" type=search><label for=search-within class=sr-only>Search within</label><select name=search-within id=search-within tabindex=10 class=search-within><option value=#pane-67065c09-4a88-49cd-934c-ac707951d35c0 selected>This Journal<option value=#pane-67065c09-4a88-49cd-934c-ac707951d35c1>Anywhere</select><ul role=tablist class="rlist tab__content"><li id=pane-67065c09-4a88-49cd-934c-ac707951d35c0 aria-labelledby=pane-67065c09-4a88-49cd-934c-ac707951d35c0-con role=tab class="tab__pane quick-search-pane__anywhere active"><form action=/action/doSearch name=thisJournalQuickSearch title="Quick Search"><div class="input-group option-0"><label for=searchField0 class=hiddenLabel>Search term</label><input name=AllField id=searchField0 placeholder=Search data-auto-complete-max-words=7 data-auto-complete-max-chars=32 data-contributors-conf=3 data-topics-conf=3 data-publication-titles-conf=3 data-history-items-conf=3 value tabindex=9 class="autocomplete actualQSInput quickSearchFilter ui-autocomplete-input" autocomplete=off type=search><span role=status aria-live=polite class=ui-helper-hidden-accessible></span><div class="search-options-wrapper quickSearchFilter sf-hidden"></div></div><button id=quick-search-btn type=submit title=Search tabindex=11 aria-label="Submit Search" class="btn quick-search__button icon-search"></button></form><li id=pane-67065c09-4a88-49cd-934c-ac707951d35c1 aria-labelledby=pane-67065c09-4a88-49cd-934c-ac707951d35c1-con role=tab class="tab__pane quick-search-pane__anywhere sf-hidden"></ul></div></div></div></div>
 </div>
 
 <div data-widget-def=ux3-layout-widget data-widget-id=5ef13ed1-f4ac-4b62-a426-07ec9576a586 class=cart-indicator__wrapper>
 
 
 <div class=pull-left>
 <div data-widget-def=eCommerceCartIndicatorWidget data-widget-id=4a9f7c04-a273-4a25-91fb-8e7bd9970288 class=cart-indicator__wrapper>
 
 
 
 
 
 </div>
 
</div><div class=pull-right>
 <div data-widget-def=literatumNavigationLoginBar data-widget-id=d384ca92-7137-45d6-b3c2-78c9c0a98c2b class=login-bar-container>
 
 
 
 
 
 
 <div class="loginBar login-open">
 <a href="https://onlinelibrary.wiley.com/action/showLogin?uri=%2Fdoi%2F10.1111%2Fj.1600-0587.2009.06148.x&amp;aria-label=Log+in+or+Register" class=show-login role=button uri=/doi/10.1111/j.1600-0587.2009.06148.x aria-label="Log in or Register">
 <span class="icon-profile_empty sf-hidden"></span>
 <span class=sign-in-label>Login / Register</span>
 </a>
 
 </div>
 
 </div>
 
</div>
 </div>
 
</div>
</div></div></div>
 </div>
</header>
 </div>
 
 <div data-widget-def=pageBody data-widget-id=72100436-7a82-49fc-933b-c6c9d8c42914 class="pageBody article-page-tablet body-text">
 
 
 <main id=main-content tabindex=-1>
 
 <div class="page-body pagefulltext">
 <div data-pb-dropzone=main>
 
 
 <div>
 <div data-widget-def=literatumAd data-widget-id=0220a30b-90c0-4b51-8637-141edf12f86a class=banner-wrapper>
 
 
 
 <div class=pb-ad>
 <aside role=complementary>
 <div id=advert-leaderboard class="advert advert-leaderboard" data-google-query-id=CPyV3O3Ow-wCFdIw4AodNQQGwQ style=display:none>
 
 
 
 </div>
 </aside>
 </div>
 
 </div>
 
 
 
</div>
 <section data-widget-def=ux3-layout-widget data-widget-id=0f7bccba-8328-4dbd-a1fa-e84c526dd9ab>
 
 
 <div class=container><div class=row><div class=col-md-12>
 
 <div>
 
 <article data-figures="/action/ajaxShowFigures?widgetId=5cf4c79f-0ae9-4dc5-96ce-77f62de7ada9&amp;ajax=true&amp;doi=10.1111%2Fj.1600-0587.2009.06148.x&amp;pbContext=%3Bpage%3Astring%3AArticle%2FChapter+View%3Bctype%3Astring%3AJournal+Content%3Bwebsite%3Awebsite%3Apericles%3Barticle%3Aarticle%3Adoi%5C%3A10.1111%2Fj.1600-0587.2009.06148.x%3Bjournal%3Ajournal%3A16000587%3Bissue%3Aissue%3Adoi%5C%3A10.1111%2Feco.2010.33.issue-1%3Bwgroup%3Astring%3APublication+Websites%3BrequestedJournal%3Ajournal%3A16000587%3BpageGroup%3Astring%3APublication+Pages%3BsubPage%3Astring%3AFull+Text&amp;accordionHeadingWrapper=h2" data-references="/action/ajaxShowEnhancedAbstract?widgetId=5cf4c79f-0ae9-4dc5-96ce-77f62de7ada9&amp;ajax=true&amp;doi=10.1111%2Fj.1600-0587.2009.06148.x&amp;pbContext=%3Bpage%3Astring%3AArticle%2FChapter+View%3Bctype%3Astring%3AJournal+Content%3Bwebsite%3Awebsite%3Apericles%3Barticle%3Aarticle%3Adoi%5C%3A10.1111%2Fj.1600-0587.2009.06148.x%3Bjournal%3Ajournal%3A16000587%3Bissue%3Aissue%3Adoi%5C%3A10.1111%2Feco.2010.33.issue-1%3Bwgroup%3Astring%3APublication+Websites%3BrequestedJournal%3Ajournal%3A16000587%3BpageGroup%3Astring%3APublication+Pages%3BsubPage%3Astring%3AFull+Text&amp;accordionHeadingWrapper=h2" data-related="/action/ajaxShowRecommended?widgetId=5cf4c79f-0ae9-4dc5-96ce-77f62de7ada9&amp;ajax=true&amp;doi=10.1111%2Fj.1600-0587.2009.06148.x&amp;pbContext=%3Bpage%3Astring%3AArticle%2FChapter+View%3Bctype%3Astring%3AJournal+Content%3Bwebsite%3Awebsite%3Apericles%3Barticle%3Aarticle%3Adoi%5C%3A10.1111%2Fj.1600-0587.2009.06148.x%3Bjournal%3Ajournal%3A16000587%3Bissue%3Aissue%3Adoi%5C%3A10.1111%2Feco.2010.33.issue-1%3Bwgroup%3Astring%3APublication+Websites%3BrequestedJournal%3Ajournal%3A16000587%3BpageGroup%3Astring%3APublication+Pages%3BsubPage%3Astring%3AFull+Text&amp;accordionHeadingWrapper=h2&amp;showSubjects=true&amp;taxonomiesCodes=enter+your+taxonomy+codes+here&amp;displayCitedByLink=true&amp;displayAlmetricDropzone=true&amp;taxonomyUri=publication-features&amp;topicUri=hide-issue-metadata" data-details="/action/ajaxShowPubInfo?widgetId=5cf4c79f-0ae9-4dc5-96ce-77f62de7ada9&amp;ajax=true&amp;doi=10.1111%2Fj.1600-0587.2009.06148.x&amp;pbContext=%3Bpage%3Astring%3AArticle%2FChapter+View%3Bctype%3Astring%3AJournal+Content%3Bwebsite%3Awebsite%3Apericles%3Barticle%3Aarticle%3Adoi%5C%3A10.1111%2Fj.1600-0587.2009.06148.x%3Bjournal%3Ajournal%3A16000587%3Bissue%3Aissue%3Adoi%5C%3A10.1111%2Feco.2010.33.issue-1%3Bwgroup%3Astring%3APublication+Websites%3BrequestedJournal%3Ajournal%3A16000587%3BpageGroup%3Astring%3APublication+Pages%3BsubPage%3Astring%3AFull+Text&amp;accordionHeadingWrapper=h2&amp;showSubjects=true&amp;taxonomiesCodes=enter+your+taxonomy+codes+here&amp;displayCitedByLink=true&amp;displayAlmetricDropzone=true&amp;taxonomyUri=publication-features&amp;topicUri=hide-issue-metadata" data-mathjax="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.5/MathJax.js?config=TeX-AMS-MML_HTMLorMML" class=hide_CASA><div class="row article-row"><div id=article__content tabindex=-1 class="col-sm-12 col-md-8 col-lg-8 article__content article-row-left"><div data-pb-dropzone=mrwBannerDropZone class=mrw-banner-dropzone>
 
 
</div><div class=article-citation><div class=citation>
 <div class=pb-dropzone data-pb-dropzone=publicaitonContent-series-title>
 <div id=journal-banner-text class=journal-banner-text style=font-size:22px><a href=https://onlinelibrary.wiley.com/journal/16000587 title="Ecography homepage">Ecography</a></div><a href=https://onlinelibrary.wiley.com/toc/16000587/2010/33/1 class="volume-issue sf-hidden">Volume 33, Issue 1</a><a href=https://onlinelibrary.wiley.com/journal/16000587 title="Ecography homepage" class=citation--logo><img id=journal-banner-image test=test-value alt=Ecography src="data:image/gif;base64,R0lGODlhAQABAAAAACH5BAEKAAEALAAAAAABAAEAAAICTAEAOw==" class=journal-banner-image style=display:none></a>
 </div>
 <div class="doi-access-container clearfix"><span class=primary-heading></span>
 <div class=doi-access-wrapper><div class="free-access access-type"> <i aria-hidden=true class=icon-icon-lock_open></i> <div class=doi-access> <div class=doi-access tabindex=0>Free Access</div></div></div></div></div>
 <h1 class=citation__title>A diversity of beta diversities: straightening up a concept gone awry. Part 2. Quantifying beta diversity and related phenomena</h1>
 <div class="rlist--inline loa comma visible-xs mobile-authors loa-authors-trunc sf-hidden">
 
 </div>
 <div class="loa-wrapper loa-authors hidden-xs">
 <div id=sb-1 class=accordion>
 <div class=accordion-tabbed>
 <div class="accordion-tabbed__tab-mobile accordion__closed"><a href="https://onlinelibrary.wiley.com/action/doSearch?ContribAuthorStored=Tuomisto%2C+Hanna" class="author-name accordion-tabbed__control" data-id=a1 data-db-target-for=a1 aria-controls=a1 aria-haspopup=true id=a1_Ctrl role=button><span>Hanna Tuomisto</span><i aria-hidden=true class="icon-section_arrow_d sf-hidden"></i></a><div class="author-info accordion-tabbed__content sf-hidden" data-db-target-of=a1 aria-labelledby=a1_Ctrl role=region id=a1></div>
 </div>
 </div>
 </div>
 </div>
 <div class=epub-sections><div class=epub-section><span class=epub-state>First published: </span><span class=epub-date>04 March 2010</span></div>
 <div class=epub-section><a class=epub-doi aria-label="Digital Object Identifier" href=https://doi.org/10.1111/j.1600-0587.2009.06148.x>https://doi.org/10.1111/j.1600-0587.2009.06148.x</a></div><div class="epub-section cited-by-count"><span>Citations: <a href=#citedby-section>218</a></span></div>
 </div>
 <div class=article-header__correspondence-to><i aria-hidden=true class="at-icon-wrapper icon-mail_outline"></i>H. Tuomisto (<span>E-mail address: </span><a title="
                Link to email address
            " href=mailto:hanna.tuomisto@utu.fi>hanna.tuomisto@utu.fi</a>), Dept of Biology, FI‐20014 Univ. of Turku, Finland
 </div>
</div></div><div class=fixedCoolBar style=padding-top:0px><nav aria-label="Article navigation and tools" class="coolBar stickybar" style=top:auto;width:auto;left:auto><div class="stickybar__wrapper coolBar__wrapper clearfix"><div class="rlist coolBar__zone"><div class=coolBar__section><a href=# data-db-target-for=article data-db-switch=icon-close_thin aria-haspopup=true aria-controls=article_Pop role=button id=article_Ctrl class="coolBar__ctrl hidden-lg sf-hidden"></a><div data-db-target-of=article aria-labelledby=article_Ctrl role=menu id=article_Pop class="coolBar__drop fixed rlist dropBlock__holder sf-hidden"></div></div><div class="coolBar__section coolBar--sections"><a href=# data-db-target-for=sections data-db-switch=icon-close_thin aria-haspopup=true aria-controls=sections_Pop role=button id=sections_Ctrl class=coolBar__ctrl><i class=icon-list-unordered></i><span>Sections</span></a><div data-db-target-of=sections aria-labelledby=sections_Ctrl role=menu id=sections_Pop class="coolBar__drop dropBlock__holder rlist fixed sf-hidden"></div></div></div><div class="coolBar__second rlist"><div class="coolBar__section coolBar--download PdfLink scroll-to-target cloned"><a href=https://onlinelibrary.wiley.com/doi/epdf/10.1111/j.1600-0587.2009.06148.x title=ePDF class="coolBar__ctrl pdf-download"><img src=images/21.png alt=PDF>PDF</a></div>
 
 <div class=relative><a href=# data-db-target-for=0e406ce9-5c43-4e2d-b741-e0b43d1f1865 data-db-switch=icon-close_thin aria-haspopup=true aria-controls=0e406ce9-5c43-4e2d-b741-e0b43d1f1865_Pop role=button id=0e406ce9-5c43-4e2d-b741-e0b43d1f1865_Ctrl class=article-tools__ctrl><i aria-hidden=true class=icon-Icon_Tools></i><span>Tools</span></a><div data-db-target-of=0e406ce9-5c43-4e2d-b741-e0b43d1f1865 aria-labelledby=0e406ce9-5c43-4e2d-b741-e0b43d1f1865_Ctrl role=menu id=0e406ce9-5c43-4e2d-b741-e0b43d1f1865_Pop class="article-tools__block fixed dropBlock__holder sf-hidden"></div></div>
 
 <div class=relative><a id=share-access__ctrl href=# data-db-target-for=dff42abe-4b17-4fe9-a9c5-625405cd1007 data-db-switch=icon-close_thin aria-haspopup=true aria-controls=share_Pop role=button class=share-access__ctrl><i aria-hidden=true class=icon-Icon_Share></i><span>Share</span></a><a id=share__ctrl href=# data-db-target-for=dff42abe-4b17-4fe9-a9c5-625405cd1007 data-db-switch=icon-close_thin aria-haspopup=true aria-controls=share_Pop role=button class="share__ctrl hidden sf-hidden"></a><div data-db-target-of=dff42abe-4b17-4fe9-a9c5-625405cd1007 aria-labelledby=share-access__ctrl role=menu id=share_Pop class="share__block dropBlock__holder fixed sf-hidden">
 
 
</div></div>
</div></div></nav></div><div id=accessDenialslot class=accessDenialslot></div><div class=article__body><article>
 
 
 <div class=abstract-group>
 <section class="article-section article-section__abstract" data-lang=en id=section-1-en lang=en>
 <h2 id=d23383239 class="article-section__header section__title main abstractlang_en">Abstract</h2>
 <div class="article-section__content en main">
 <p>The present two‐part review aims to put the different phenomena that have been called “beta diversity” over the years into a common conceptual framework and to explain what each of them measures. The first part (Tuomisto 2010) discussed basic definitions of “beta diversity”. Each arises from a different way of combining a definition of “diversity” with a definition of its alpha component and with a mathematical relationship between the alpha and gamma components. This second part assumes that an appropriate basic definition of a beta component (which may or may not be true beta diversity) has been chosen, and the focus here will be on how to quantify it for a given dataset. About twenty different approaches have been used for this purpose. It turns out that only two of these approaches accurately quantify the selected beta component: one does so for the entire dataset, and the other for two sampling units at a time. The other approaches actually quantify other phenomena, such as mean species turnover between sampling units, compositional gradient length (with or without reference to an external gradient), distinctness of a focal sampling unit, rate of species accumulation with increasing sampling effort, rate of compositional turnover along an external gradient, or the rate of decay in compositional similarity with increasing geographical distance. Although most of these phenomena can be expressed as a function of a beta component of diversity, they do not equal a beta component of diversity. Many of these derived variables are not even numerically correlated with the beta component on which they are based, which needs to be taken into account when interpreting the results. The effects of sampling decisions when results are extrapolated beyond the available data will also be discussed.</p>
 
 </div>
 </section>
 </div>
 <div class=pb-dropzone data-pb-dropzone=below-abstract-group>
 
 </div>
 
 <section class="article-section article-section__full">
 <section class=article-section__content id=ss1>
 
 <p>The first part of the present review (<span><a href=#b110 id=#b110R class="bibLink tab-link" data-tab=pane-pcw-references>Tuomisto 2010</a></span>) discussed eight different basic definitions of “beta diversity”. This second part discusses derived definitions of “beta diversity” that arise from different approaches to quantifying the chosen kind of “beta diversity”, or a phenomenon related to it, for a given dataset. The focus in this part is mostly on five of the eight basic definitions of “beta diversity”, collectively referred to here as <i>beta components</i>. <a href=#t1 class=scrollableLink>Table 1</a> presents an overview of both the basic and the derived definitions of “beta diversity” and indicates in which part of the present review each definition is explained in detail.
 </p>
 
 <div class=article-table-content id=t1>
 <header class=article-table-caption><span class=table-caption__label>Table 1.
 </span>Summary of the notation and names used for different kinds of “beta diversity” in the present paper. The first five are basic definitions in which true gamma diversity (γ=<sup><i>q</i></sup><i>D</i><sub>γ</sub>) of the dataset of interest is partitioned into alpha and beta components using a simple mathematical function. The others are derived definitions, the notation for each of which summarises its relationship to the basic definitions, other components of diversity, and explanatory gradients (“n.n.” means no specific notation). The alpha component is either mean gamma diversity within sampling units 
<span><img alt="inline image" class=section_image src=images/22.gif></span>, whose measurement unit is sp<sub>E</sub>, or true alpha diversity α<sub>d</sub> (= <sup><i>q</i></sup><i>D</i><sub>α</sub>), whose value is the same but measurement unit is (effective species)/(compositional unit)=sp<sub>E</sub>/CU. When either all <i>N</i> sampling units are weighted equally or <i>q</i>=1, α<sub>t</sub> is constrained to the interval [γ/<i>N</i>, γ]. This allows expressing also the range of each beta component in terms of <i>N</i>; these ranges are shown together with the measurement units. The last column indicates in which subsection of the present two‐part review each kind of “beta diversity” is discussed in detail. Part 1 refers to <span class=section-footNote><span><a href=#b110 id=#b110R class="bibLink tab-link" data-tab=pane-pcw-references>Tuomisto (2010)</a></span></span> and Part 2 to the present paper.
 </header>
 
 
 
 
 <div class=article-table-content-wrapper>
 <table class="table article-section__table">
 
 
 
 
 
 <thead>
 
 <tr>
 
 <th class="bottom-bordered-cell left-aligned">Notation</th>
 
 <th class="bottom-bordered-cell center-aligned">Definition</th>
 
 <th class="bottom-bordered-cell center-aligned">Measurement unit [range]</th>
 
 <th class="bottom-bordered-cell center-aligned">Section</th>
 
 </tr>
 
 </thead>
 
 <tbody>
 
 <tr>
 
 <td class=left-aligned>β<sub>Md</sub></td>
 
 <td class=left-aligned>true beta diversity=γ/α<sub>d</sub></td>
 
 <td class=left-aligned>CU [1 CU to <i>N</i> CU]
 </td>
 
 <td class=left-aligned>Part 1: 1</td>
 
 </tr>
 
 <tr>
 
 <td class=left-aligned></td>
 
 <td class=left-aligned></td>
 
 <td class=left-aligned></td>
 
 <td class=left-aligned></td>
 
 </tr>
 
 <tr>
 
 <td class=left-aligned>β<sub>Mt</sub></td>
 
 <td class=left-aligned>regional‐to‐local diversity ratio=γ/α<sub>t</sub></td>
 
 <td class=left-aligned>sp<sub>E</sub>/sp<sub>E</sub> [1 to <i>N</i>]
 </td>
 
 <td class=left-aligned>Part 1: 1</td>
 
 </tr>
 
 <tr>
 
 <td class=left-aligned></td>
 
 <td class=left-aligned></td>
 
 <td class=left-aligned></td>
 
 <td class=left-aligned></td>
 
 </tr>
 
 <tr>
 
 <td class=left-aligned>β<sub>At</sub></td>
 
 <td class=left-aligned>absolute effective species turnover=γ−α<sub>t</sub></td>
 
 <td class=left-aligned>sp<sub>E</sub> [0 to (<i>N</i>−1)α<sub>t</sub>]
 </td>
 
 <td class=left-aligned>Part 1: 2</td>
 
 </tr>
 
 <tr>
 
 <td class=left-aligned></td>
 
 <td class=left-aligned></td>
 
 <td class=left-aligned></td>
 
 <td class=left-aligned></td>
 
 </tr>
 
 <tr>
 
 <td class=left-aligned>β<sub>Mt−1</sub></td>
 
 <td class=left-aligned>Whittaker's effective species turnover = (γ−α<sub>t</sub>)/α<sub>t</sub>=γ/α<sub>t</sub>−1
 </td>
 
 <td class=left-aligned>sp<sub>E</sub>/sp<sub>E</sub> [0 to <i>N</i>−1]
 </td>
 
 <td class=left-aligned>Part 1: 3</td>
 
 </tr>
 
 <tr>
 
 <td class=left-aligned></td>
 
 <td class=left-aligned></td>
 
 <td class=left-aligned></td>
 
 <td class=left-aligned></td>
 
 </tr>
 
 <tr>
 
 <td class=left-aligned>β<sub>Pt</sub></td>
 
 <td class=left-aligned>proportional effective species turnover = (γ−α<sub>t</sub>)/γ=1−α<sub>t</sub>/γ
 </td>
 
 <td class=left-aligned>sp<sub>E</sub>/sp<sub>E</sub> [0 to 1−1/<i>N</i>]
 </td>
 
 <td class=left-aligned>Part 1: 4</td>
 
 </tr>
 
 <tr>
 
 <td class=left-aligned></td>
 
 <td class=left-aligned></td>
 
 <td class=left-aligned></td>
 
 <td class=left-aligned></td>
 
 </tr>
 
 <tr>
 
 <td class=left-aligned>Δ<i>c</i></td>
 
 <td class=left-aligned>any of the effective species turnover measures, i.e. β<sub>At</sub>, β<sub>Mt−1</sub> or β<sub>Pt</sub></td>
 
 <td class=left-aligned>as in the chosen turnover</td>
 
 <td class=left-aligned>Part 2: Introduction</td>
 
 </tr>
 
 <tr>
 
 <td class=left-aligned></td>
 
 <td class=left-aligned></td>
 
 <td class=left-aligned></td>
 
 <td class=left-aligned></td>
 
 </tr>
 
 <tr>
 
 <td class=left-aligned>β<sub>Mtot</sub> or Δ<i>c</i><sub>tot</sub></td>
 
 <td class=left-aligned>a beta component quantified for the entire dataset</td>
 
 <td class=left-aligned>as in the chosen beta component</td>
 
 <td class=left-aligned>Part 2: 1.1</td>
 
 </tr>
 
 <tr>
 
 <td class=left-aligned></td>
 
 <td class=left-aligned></td>
 
 <td class=left-aligned></td>
 
 <td class=left-aligned></td>
 
 </tr>
 
 <tr>
 
 <td class=left-aligned>β<sub>M<i>j,k</i></sub> or Δ<i>c</i><sub><i>j,k</i></sub></td>
 
 <td class=left-aligned>a beta component quantified for a subset of the dataset that consists of the sampling units <i>j</i> and <i>k</i></td>
 
 <td class=left-aligned>as in the chosen beta component</td>
 
 <td class=left-aligned>Part 2: 1.2</td>
 
 </tr>
 
 <tr>
 
 <td class=left-aligned></td>
 
 <td class=left-aligned></td>
 
 <td class=left-aligned></td>
 
 <td class=left-aligned></td>
 
 </tr>
 
 <tr>
 
 <td class=left-aligned> <img alt="inline image" class=section_image src=images/23.gif> 
 </td>
 
 <td class=left-aligned>average of all the species turnover values that can be calculated for different sampling unit pairs in the dataset (with <i>j</i>≠<i>k</i>)
 </td>
 
 <td class=left-aligned>as in the chosen turnover</td>
 
 <td class=left-aligned>Part 2: 2.1</td>
 
 </tr>
 
 <tr>
 
 <td class=left-aligned></td>
 
 <td class=left-aligned></td>
 
 <td class=left-aligned></td>
 
 <td class=left-aligned></td>
 
 </tr>
 
 <tr>
 
 <td class=left-aligned> <img alt="inline image" class=section_image src=images/24.gif> 
 </td>
 
 <td class=left-aligned>average of all the species turnover values that can be calculated between a real sampling unit and a regional compositional centroid in the dataset</td>
 
 <td class=left-aligned>as in the chosen turnover</td>
 
 <td class=left-aligned>Part 2: 2.2</td>
 
 </tr>
 
 <tr>
 
 <td class=left-aligned></td>
 
 <td class=left-aligned></td>
 
 <td class=left-aligned></td>
 
 <td class=left-aligned></td>
 
 </tr>
 
 <tr>
 
 <td class=left-aligned>Δ<i>c</i><sub><i>j,k</i>max</sub> or Δ<i>c′</i><sub>max</sub></td>
 
 <td class=left-aligned>compositional gradient length in the dataset along the compositional dimension with most turnover</td>
 
 <td class=left-aligned>as in the chosen turnover</td>
 
 <td class=left-aligned>Part 2: 2.3</td>
 
 </tr>
 
 <tr>
 
 <td class=left-aligned></td>
 
 <td class=left-aligned></td>
 
 <td class=left-aligned></td>
 
 <td class=left-aligned></td>
 
 </tr>
 
 <tr>
 
 <td class=left-aligned>Δ<i>c</i><sub>(Δ<i>g</i>)</sub></td>
 
 <td class=left-aligned>compositional gradient length along a specified section of an external gradient <i>g</i></td>
 
 <td class=left-aligned>as in the chosen turnover</td>
 
 <td class=left-aligned>Part 2: 2.4</td>
 
 </tr>
 
 <tr>
 
 <td class=left-aligned></td>
 
 <td class=left-aligned></td>
 
 <td class=left-aligned></td>
 
 <td class=left-aligned></td>
 
 </tr>
 
 <tr>
 
 <td class=left-aligned>ΔΔ<i>g</i><sub>(Δlog(1–Δ<i>c</i>))</sub></td>
 
 <td class=left-aligned>number of half‐change units, i.e. observed amount of change in differences in explanatory gradient <i>g</i> expressed in terms of decrease in compositional similarity
 </td>
 
 <td class=left-aligned>(unit of <i>g</i>)/(unit of <i>g</i>)
 </td>
 
 <td class=left-aligned>Part 2: 2.5</td>
 
 </tr>
 
 <tr>
 
 <td class=left-aligned></td>
 
 <td class=left-aligned></td>
 
 <td class=left-aligned></td>
 
 <td class=left-aligned></td>
 
 </tr>
 
 <tr>
 
 <td class=left-aligned> <img alt="inline image" class=section_image src=images/25.gif> 
 </td>
 
 <td class=left-aligned>compositional distinctness of the focal sampling unit F</td>
 
 <td class=left-aligned>as in the chosen turnover</td>
 
 <td class=left-aligned>Part 2: 3.1</td>
 
 </tr>
 
 <tr>
 
 <td class=left-aligned></td>
 
 <td class=left-aligned></td>
 
 <td class=left-aligned></td>
 
 <td class=left-aligned></td>
 
 </tr>
 
 <tr>
 
 <td class=left-aligned>n.n.</td>
 
 <td class=left-aligned>compositional nestedness of a species‐poor sampling unit in a more species‐rich one</td>
 
 <td class=left-aligned>sp/sp</td>
 
 <td class=left-aligned>Part 2: 3.2</td>
 
 </tr>
 
 <tr>
 
 <td class=left-aligned></td>
 
 <td class=left-aligned></td>
 
 <td class=left-aligned></td>
 
 <td class=left-aligned></td>
 
 </tr>
 
 <tr>
 
 <td class=left-aligned>n.n.</td>
 
 <td class=left-aligned>logically inconsistent beta components in which α and γ are based on different datasets</td>
 
 <td class=left-aligned>as in the chosen beta component</td>
 
 <td class=left-aligned>Part 2: 3.3</td>
 
 </tr>
 
 <tr>
 
 <td class=left-aligned></td>
 
 <td class=left-aligned></td>
 
 <td class=left-aligned></td>
 
 <td class=left-aligned></td>
 
 </tr>
 
 <tr>
 
 <td class=left-aligned>n.n.</td>
 
 <td class=left-aligned>average of all pairwise beta component values with compositional data taken from outside the sampling units of interest</td>
 
 <td class=left-aligned>as in the chosen beta component</td>
 
 <td class=left-aligned>Part 2: 3.4</td>
 
 </tr>
 
 <tr>
 
 <td class=left-aligned></td>
 
 <td class=left-aligned></td>
 
 <td class=left-aligned></td>
 
 <td class=left-aligned></td>
 
 </tr>
 
 <tr>
 
 <td class=left-aligned>Δγ/Δ<i>x</i></td>
 
 <td class=left-aligned>rate of gamma diversity accumulation with increasing (logarithm of the) number of sampling units</td>
 
 <td class=left-aligned>sp<sub>E</sub>/SU or sp<sub>E</sub>/log(SU)
 </td>
 
 <td class=left-aligned>Part 2: 4.1.A</td>
 
 </tr>
 
 <tr>
 
 <td class=left-aligned></td>
 
 <td class=left-aligned></td>
 
 <td class=left-aligned></td>
 
 <td class=left-aligned></td>
 
 </tr>
 
 <tr>
 
 <td class=left-aligned>Δα<sub>t</sub>/Δ<i>x</i></td>
 
 <td class=left-aligned>rate of alpha diversity accumulation when sampling unit size increases in multiples of (logarithm of the) original size</td>
 
 <td class=left-aligned>sp<sub>E</sub>/SU or sp<sub>E</sub>/log(SU)
 </td>
 
 <td class=left-aligned>Part 2: 4.1.B</td>
 
 </tr>
 
 <tr>
 
 <td class=left-aligned></td>
 
 <td class=left-aligned></td>
 
 <td class=left-aligned></td>
 
 <td class=left-aligned></td>
 
 </tr>
 
 <tr>
 
 <td class=left-aligned>Δlog(γ)/Δ<i>x</i></td>
 
 <td class=left-aligned>rate of gamma entropy accumulation with increasing logarithm of the number of sampling units</td>
 
 <td class=left-aligned>log(sp<sub>E</sub>)/log(SU)
 </td>
 
 <td class=left-aligned>Part 2: 4.2.A</td>
 
 </tr>
 
 <tr>
 
 <td class=left-aligned></td>
 
 <td class=left-aligned></td>
 
 <td class=left-aligned></td>
 
 <td class=left-aligned></td>
 
 </tr>
 
 <tr>
 
 <td class=left-aligned>Δlog(α<sub>t</sub>)/Δ<i>x</i></td>
 
 <td class=left-aligned>rate of alpha entropy accumulation when sampling unit size increases in multiples of the logarithm of original size</td>
 
 <td class=left-aligned>log(sp<sub>E</sub>)/log(SU)
 </td>
 
 <td class=left-aligned>Part 2: 4.2.B</td>
 
 </tr>
 
 <tr>
 
 <td class=left-aligned></td>
 
 <td class=left-aligned></td>
 
 <td class=left-aligned></td>
 
 <td class=left-aligned></td>
 
 </tr>
 
 <tr>
 
 <td class=left-aligned>Δβ<sub>M</sub>/Δ<i>x</i> or ΔΔ<i>c</i>/Δ<i>x</i></td>
 
 <td class=left-aligned>rate of change in a beta component of diversity with increasing number of sampling units</td>
 
 <td class=left-aligned>(unit of the beta component)/SU</td>
 
 <td class=left-aligned>Part 2: 4.3.A</td>
 
 </tr>
 
 <tr>
 
 <td class=left-aligned></td>
 
 <td class=left-aligned></td>
 
 <td class=left-aligned></td>
 
 <td class=left-aligned></td>
 
 </tr>
 
 <tr>
 
 <td class=left-aligned>Δβ<sub>M</sub>/Δ<i>x</i> or ΔΔ<i>c</i>/Δ<i>x</i></td>
 
 <td class=left-aligned>decay rate of a beta component of diversity when sampling unit size increases in multiples of original size</td>
 
 <td class=left-aligned>(unit of the beta component)/SU</td>
 
 <td class=left-aligned>Part 2: 4.3.B</td>
 
 </tr>
 
 <tr>
 
 <td class=left-aligned></td>
 
 <td class=left-aligned></td>
 
 <td class=left-aligned></td>
 
 <td class=left-aligned></td>
 
 </tr>
 
 <tr>
 
 <td class=left-aligned>Δβ<sub>Pt</sub>/Δ<i>x</i></td>
 
 <td class=left-aligned>proportional effective species turnover accumulation rate when an increasing proportion of the available sampling units is taken into account</td>
 
 <td class=left-aligned>(sp<sub>E</sub>/sp<sub>E</sub>)/SU
 </td>
 
 <td class=left-aligned>Part 2: 4.3.C</td>
 
 </tr>
 
 <tr>
 
 <td class=left-aligned></td>
 
 <td class=left-aligned></td>
 
 <td class=left-aligned></td>
 
 <td class=left-aligned></td>
 
 </tr>
 
 <tr>
 
 <td class=left-aligned>Δlog(<sup><i>q</i></sup>β<sub>M</sub>)/Δ<i>x</i></td>
 
 <td class=left-aligned>rate of change in beta entropy or regional entropy excess with increasing logarithm of the number of sampling units</td>
 
 <td class=left-aligned>(unit of entropy)/log(SU), e.g. bits/log(SU)</td>
 
 <td class=left-aligned>Part 2: 4.4</td>
 
 </tr>
 
 <tr>
 
 <td class=left-aligned></td>
 
 <td class=left-aligned></td>
 
 <td class=left-aligned></td>
 
 <td class=left-aligned></td>
 
 </tr>
 
 <tr>
 
 <td class=left-aligned>n.n.</td>
 
 <td class=left-aligned>species diversity or entropy accumulation rate with alpha and gamma diversities based on different data</td>
 
 <td class=left-aligned>as in the chosen accumulation rate</td>
 
 <td class=left-aligned>Part 2: 4.5</td>
 
 </tr>
 
 <tr>
 
 <td class=left-aligned></td>
 
 <td class=left-aligned></td>
 
 <td class=left-aligned></td>
 
 <td class=left-aligned></td>
 
 </tr>
 
 <tr>
 
 <td class=left-aligned>Δ<i>c</i><sub>(Δ<i>g</i>)</sub>/Δ<i>g</i></td>
 
 <td class=left-aligned>compositional turnover rate along a specified section of an external gradient <i>g</i></td>
 
 <td class=left-aligned>(unit of chosen turnover)/(unit of external gradient)</td>
 
 <td class=left-aligned>Part 2: 4.6</td>
 
 </tr>
 
 <tr>
 
 <td class=left-aligned></td>
 
 <td class=left-aligned></td>
 
 <td class=left-aligned></td>
 
 <td class=left-aligned></td>
 
 </tr>
 
 <tr>
 
 <td class=left-aligned>ΔΔ<i>c</i><sub>(ΔΔ<i>g</i>)</sub>/ΔΔ<i>g</i> or Δlog(1–Δ<i>c</i>)<sub>(ΔΔ<i>g</i>)</sub>/ΔΔ<i>g</i></td>
 
 <td class=left-aligned>rate of change in (the logarithm of the one‐complement of) pairwise effective species turnover with increasing distance along an explanatory gradient <i>g</i> (slope of a distance decay regression)
 </td>
 
 <td class=left-aligned>(unit of chosen turnover)/(unit of external gradient) or log(unit of turnover)/(unit of external gradient)</td>
 
 <td class=left-aligned>Part 2: 4.7</td>
 
 </tr>
 
 </tbody>
 
 </table>
 </div>
 
 
 <div class=article-section__table-source></div>
 </div>
 
 <p>Two of the beta components treated in this paper measure <i>compositional heterogeneity</i> in the dataset. <i>True beta diversity</i> (<sup><i>q</i></sup><i>D</i><sub>β</sub>=<sup><i>q</i></sup><i>D</i><sub>γ</sub>/<sup><i>q</i></sup><i>D</i><sub>α</sub> or <sup><i>q</i></sup>β<sub>Md</sub>=γ/α<sub>d</sub>) quantifies the number of compositional units (compositionally distinct virtual sampling units that have the same species diversity as the actual sampling units do on average, abbreviated CU) in the dataset. <i>Regional‐to‐local diversity ratio</i> (
<img alt="inline image" class=section_image src=images/26.gif>) quantifies how many times as rich in effective species the entire dataset is than a single compositional unit. The two differ in measurement units but obtain the same numerical value when based on the same data; when both are referred to simultaneously, the notation <sup><i>q</i></sup>β<sub>M</sub> is used. The other three beta components measure <i>effective species turnover</i>. <i>Absolute effective species turnover</i> (<sup><i>q</i></sup>β<sub>At</sub>=γ−α<sub>t</sub>) quantifies the cumulative number of effective species that change among all compositional units in the dataset. <i>Relative effective species turnover</i> is obtained by dividing <sup><i>q</i></sup>β<sub>At</sub> by either α<sub>t</sub> or γ. <i>Whittaker's effective species turnover</i> (<sup><i>q</i></sup>β<sub>Mt−1</sub>=<sup><i>q</i></sup>β<sub>At</sub>/α<sub>t</sub>) quantifies the number of times that the effective species composition changes completely among all compositional units. <i>Proportional effective species turnover</i> (<sup><i>q</i></sup>β<sub>Pt</sub>=<sup><i>q</i></sup>β<sub>At</sub>/γ) quantifies what proportion of the effective species composition of the entire dataset changes among the compositional units. Effective species turnover can be thought of as change in effective community composition, so all three turnover measures are collectively referred to by Δ<i>c</i>.
 </p>
 
 <p> <i>True gamma diversity</i>
 <sup>
 <i>q</i>
 </sup>
 <i>D</i>
 <sub>γ</sub>=γ is the number of effective species in a dataset, quantified as the inverse of the mean of the proportional abundances of the actual species. The parameter <i>q</i> defines which kind of mean is used, and in practice controls to what degree the rarest species are taken into account (see <i>The starting point: what is diversity?</i> in <span><a href=#b110 id=#b110R class="bibLink tab-link" data-tab=pane-pcw-references>Tuomisto 2010</a></span>). When <i>q</i>=0, even the rarest of the <i>S</i> actual species are counted as one effective species, so species diversity equals species richness. When <i>q</i>=1, diversity equals the exponential of the Shannon entropy, and when <i>q</i>=2, the inverse of Simpson concentration. The larger the value of <i>q</i>, the smaller the fraction of an effective species that is contributed by the rare actual species. The same is true of alpha diversity (<sup><i>q</i></sup><i>D</i><sub>α</sub>=α<sub>d</sub> or <sup><i>q</i></sup><i>D</i><sub>γj</sub>=α<sub>t</sub>), so diversity values based on different values of <i>q</i> are not comparable, and α and γ based on different <i>q</i> cannot be used to calculate a beta component (<sup><i>q</i></sup>β). When <i>q</i>=0, effective species turnover equals actual species turnover, but increasing <i>q</i> causes the two to diverge as effective species turnover becomes increasingly determined by differences among sampling units in the proportional abundances of the most abundant species. Abundance itself can be measured in different ways that are not interchangeable. For example, when tree plots of a fixed surface area are used, increasing the value of <i>q</i> increases the differences between results based on the number of stems and those based on basal area or biomass. With these considerations in mind, the superscript <i>q</i> will be omitted in the present paper, unless a specific value of <i>q</i> needs to be indicated.
 </p>
 
 <p>Each definition of a beta component quantifies a different phenomenon, so their values are not commensurate. The minimum value is zero in some beta components and unity in others. The maximum values of all beta components vary with the number of sampling units <i>N</i>, but they also depend on the value of <i>q</i>, on sampling unit weights (which ideally reflect an appropriate measure of sampling effort) and, in one case, on alpha diversity. The beta components only have a consistent upper bound if all sampling units are weighted equally or <i>q</i>=1 (<a href=#t1 class=scrollableLink>Table 1</a> and <span><a href=#b110 id=#b110R class="bibLink tab-link" data-tab=pane-pcw-references>Tuomisto 2010</a></span>). In the present paper, it is therefore assumed that sampling unit weights are equal. Because the upper bound depends on <i>N</i>, comparing beta components based on different <i>N</i> is problematic. Therefore, the number of sampling units used when calculating the beta component will, in some cases, be made explicit with the help of a subscript.
 </p>
 
 <p>Some basic definitions of “beta diversity” are based on the raw value of a diversity index rather than on a true diversity. Such measures include <i>regional Shannon entropy excess</i> (
<img alt="inline image" class=section_image src=images/27.gif> where <i>H′</i> is the Shannon entropy) and <i>regional variance excess</i><img alt="inline image" class=section_image src=images/28.gif> where <sup>2</sup>λ is the Simpson concentration; see <span><a href=#b110 id=#b110R class="bibLink tab-link" data-tab=pane-pcw-references>Tuomisto 2010</a></span> for details). When regional Shannon entropy or variance excess is discussed in the present paper, this will be stated explicitly; the notation Δ<i>c</i> does <i>not</i> refer to them.
 </p>
 
 <p>A basic definition of a beta component can be applied to an existing dataset in several different ways. Many of the possible approaches do not actually quantify the chosen beta component itself. Instead, they may quantify just a part of the beta component, the rate of change in the beta component along some external gradient, or some other phenomenon. Nevertheless, the results from all of these approaches have been called “beta diversity” (some of these were discussed by <span><a href=#b52 id=#b52R class="bibLink tab-link" data-tab=pane-pcw-references>Jurasinski et al. 2009</a></span>). The purpose of the present paper is to review what the different approaches in fact quantify, and how they relate to one another. Attention is also given to sampling considerations when the purpose of the study is to draw conclusions on “beta diversity” in a region of interest that is too large to be entirely inventoried, and the available dataset therefore only provides a sample of it.
 </p>
 
 </section>
 <section class=article-section__content id=ss2>
 
 <h2 class="article-section__title section__title section1" id=ss2-title> 1. Accurate approaches to quantifying β<sub>M</sub> or Δ<i>c</i></h2>
 
 <section class=article-section__sub-content id=ss3>
 
 <h3 class="article-section__sub-title section2" id=ss3-title> 1.1 The regional approach β<sub>Mtot</sub> or Δ<i>c</i><sub>tot</sub></h3>
 
 <p>When the aim of a study is to quantify the total amount of compositional heterogeneity or effective species turnover in a dataset, the regional approach is the way to go. In this approach, the chosen basic definition of a beta component is applied such that all available data are used, and the subscript “tot” can be used to express this. The interpretation of the obtained β<sub>Mtot</sub> or Δ<i>c</i><sub>tot</sub> value is then exactly according to the chosen definition.
 </p>
 
 <p>Often the available dataset is a sample of a larger region of interest, and the purpose is to compare the beta components in separate regions. If so, it is important to take into account that the maximum values of all beta components depend on the number of sampling units <i>N</i> (<a href=#t1 class=scrollableLink>Table 1</a>). Erroneous conclusions on which regions are compositionally most heterogeneous or have most effective species turnover can easily be drawn if regions are represented by different sample sizes. This problem can be avoided by using the same <i>N</i> in every region, by way of rarefaction methods if necessary (<span><a href=#b85 id=#b85R class="bibLink tab-link" data-tab=pane-pcw-references>Perelman et al. 2001</a></span>). Rarefaction implicitly assumes that each sampling unit is equally good in documenting local species diversity. It is therefore best applied to datasets where each sampling unit contains the same total abundance of the organisms of interest (this can, in turn, be achieved by rarefaction within the sampling units), or sampling effort has been standardised in some other way that is appropriate for the questions at hand.
 </p>
 
 <p>Total absolute effective species turnover β<sub>At</sub> can be larger in a species‐rich dataset where all of the actual sampling units share species with all other sampling units than in a species‐poor dataset where no sampling units share any species. If a species turnover measure that is independent of alpha diversity is required, a relative effective species turnover measure (β<sub>Mt−1</sub> or β<sub>Pt</sub>) should be used. Conversely, relative effective species turnover and the compositional heterogeneity measures (β<sub>M</sub>) cannot be interpreted in terms of the absolute number of species involved.
 </p>
 
 </section>
 
 <section class=article-section__sub-content id=ss4>
 
 <h3 class="article-section__sub-title section2" id=ss4-title> 1.2 The pairwise approach β<sub>M<i>j,k</i></sub> or Δ<i>c</i><sub><i>j,k</i></sub></h3>
 
 <p>The regional approach to quantifying the beta component gives a single value for the entire dataset of interest. However, often researchers would like to study how the beta component varies and what external factors are correlated with it. This can be achieved by calculating values of the beta component for different subsets of the data, such as sampling unit pairs.</p>
 
 <p>Alpha and gamma diversity are derivable from a sites by species raw data table of <i>N</i> rows and <i>S</i> columns, where each cell value equals the proportional abundance of the <i>i</i>th species in the <i>j</i>th sampling unit (level 1 table A in <a href=#f1 class=scrollableLink>Fig. 1</a>). Subsets of the data can be formed by taking into account just two of the rows at a time; forming all possible pairwise combinations of the <i>j</i>th and the <i>k</i>th sampling unit yields <i>N</i><sup>2</sup> new raw data tables. The beta components of all these two‐row tables can be quantified and arranged into a derived data table of <i>N</i> rows and <i>N</i> columns (<a href=#f1 class=scrollableLink>Fig. 1</a>, level 2 table D). The raw data table corresponds to the first level of abstraction sensu <span><a href=#b113 id=#b113R class="bibLink tab-link" data-tab=pane-pcw-references>Tuomisto and Ruokolainen (2006, 2008)</a></span>; this is not the same as the first level of abstraction sensu <span><a href=#b62 id=#b62R class="bibLink tab-link" data-tab=pane-pcw-references>Legendre et al. 2005</a></span>), and the derived table to the second level of abstraction (of both authors).
 </p>
 
 <section class=article-section__inline-figure>
 <figure class=figure id=f1><a target=_blank href=https://onlinelibrary.wiley.com/cms/asset/88ec90d8-1015-43a8-9d5f-e3a665ad69ec/ecog_6148_f1.gif>
 <picture>
 <img class=figure__image src=images/29.gif data-lg-src=/cms/asset/88ec90d8-1015-43a8-9d5f-e3a665ad69ec/ecog_6148_f1.gif alt=image title=image srcset sizes></picture></a><figcaption class=figure__caption>
 <div class=figure__caption__header><strong class=figure__title>Figure 1</strong><div class=figure-extra><a href=# class=open-figure-link>Open in figure viewer</a><a href="https://onlinelibrary.wiley.com/action/downloadFigures?id=f1&amp;doi=10.1111%2Fj.1600-0587.2009.06148.x" class=ppt-figure-link><i aria-hidden=true class=icon-Icon_Download></i><span>PowerPoint</span></a></div>
 </div>
 <div class="figure__caption figure__caption-text">
 <p>How different approaches to quantifying effective species turnover and related measures correspond to the three levels of abstraction of <span class="figureLink bibLink tab-link"><a href=#b113 id=#b113R class="bibLink tab-link" data-tab=pane-pcw-references>Tuomisto and Ruokolainen (2006, 2008)</a></span>. <i>Level 1</i> consists of the raw data measurements at each sampling unit (SU); species abundances are shown on the left in table A and the values of an explanatory gradient on the right in table C. Community composition <i>c</i> is a summary of the species abundance information. The score of a sampling unit along the first axis of an ordination (cell values in table B) approximates its position along the most important compositional gradient, whose relationship with explanatory gradient <i>g</i> is shown in the level 1 scatterplot (the origin of the <i>y</i> axis is at average community composition). <i>Level 2</i> data are derived as dissimilarities based on the level 1 data. Effective species turnover corresponds to Δ<i>c</i>; pairwise turnover values are shown in the level 2 dissimilarity matrix D. Dissimilarities corresponding to the same sampling units but based on explanatory data are shown in the level 2 table E, and the relationship between the two dissimilarity matrices is shown in the level 2 scatterplot. <i>Level 3</i> data are derived as dissimilarities based on the level 2 data. SUP stands for sampling unit pair. Since the dissimilarity matrices are symmetric, values below the diagonal are not shown. Proposed definitions of “beta diversity” include variance in table A, the difference between the largest and smallest cell value in table B, a single cell value in table D, mean of all off‐diagonal cell values in table D, the slope of the level 1 regression line, the length of the vertical side of the triangle delimited by the level 1 regression line, the slope of the level 2 regression line, the length of the vertical side of the triangle delimited by the level 2 regression line, and the length of the horizontal side of the triangle delimited by the level 2 regression line. For further explanation, see text.</p>
 
 </div>
 </figcaption>
 </figure>
 </section>
 
 <p>The level 2 table corresponds to a dissimilarity matrix if the beta component has a minimum value of zero when the two sampling units are compositionally identical. This is the case when any of the effective species turnover measures Δ<i>c</i><sub><i>j,k</i></sub> is used. If the sampling units have equal weights, <sup>0</sup>β<sub>At</sub> multiplied by 2 equals the squared Euclidean distance and the Manhattan metric calculated using presence‐absence data, <sup>0</sup>β<sub>Mt−1</sub> the one‐complement of the Sørensen index and <sup>0</sup>β<sub>Pt</sub> ranged to the interval [0, 1] equals the one‐complement of the Jaccard index (Sections 3, 4 and 5, respectively, in <span><a href=#b110 id=#b110R class="bibLink tab-link" data-tab=pane-pcw-references>Tuomisto 2010</a></span>). Using a dissimilarity matrix based on <sup>0</sup>β<sub>Mt−1</sub> and <sup>0</sup>β<sub>Pt</sub> has been popular in studies related to “beta diversity”, but <sup>0</sup>β<sub>At</sub> has been used much less, because its dependence on alpha diversity complicates interpretation of the results.
 </p>
 
 <p>Many data analysis methods use a dissimilarity matrix as a starting point. Principal coordinates analysis (PCoA, also known as metric multidimensional scaling) and non‐metric multidimensional scaling (NMDS) can be used to visualise the Δ<i>c</i><sub><i>j,k</i></sub> values in an ordination diagram. Agglomerative clustering produces a hierarchical classification of the sampling units such that the sampling units with the smallest Δ<i>c</i><sub><i>j,k</i></sub> values are combined first. The Mantel test quantifies the correlation between the cell values in two dissimilarity matrices. Finally, multiple regression on dissimilarity matrices and generalised dissimilarity modelling quantify the relative contributions of alternative explanatory dissimilarity matrices to explaining variation in the response dissimilarity matrix and provide means for forecasting values of Δ<i>c</i><sub><i>j,k</i></sub>.
 </p>
 
 <p>Most of these analysis methods are included in numerical ecology textbooks (<span><a href=#b61 id=#b61R class="bibLink tab-link" data-tab=pane-pcw-references>Legendre and Legendre 1998</a></span>), and sometimes their use is explicitly described in terms of “beta diversity” (<span><a href=#b70 id=#b70R class="bibLink tab-link" data-tab=pane-pcw-references>Magurran 2004</a></span>, <span><a href=#b113 id=#b113R class="bibLink tab-link" data-tab=pane-pcw-references>Tuomisto and Ruokolainen 2006, 2008</a></span>, <span><a href=#b36 id=#b36R class="bibLink tab-link" data-tab=pane-pcw-references>Ferrier et al. 2007</a></span>). However, these methods would only address true beta diversity if the pairwise values were equal to β<sub>Md</sub>, which they never are; the minimum value of β<sub>Md</sub> is not zero and it is therefore not a dissimilarity measure. However, because Whittaker's effective species turnover β<sub>Mt−1</sub> is a linear transformation of β<sub>Md</sub>, it can be used to obtain results that are consistent with true beta diversity.
 </p>
 
 </section>
 
 </section>
 <section class=article-section__content id=ss5>
 
 <h2 class="article-section__title section__title section1" id=ss5-title> 2. Approaches that quantify a derivative of Δ<i>c</i></h2>
 
 <section class=article-section__sub-content id=ss6>
 
 <h3 class="article-section__sub-title section2" id=ss6-title> 2.1 Average of all pairwise values <span><img alt="inline image" class=section_image src=images/30.gif></span></h3>
 
 <p>One possible measure of compositional differentiation in a dataset is the average of all the pairwise effective species turnover values Δ<i>c</i><sub><i>j,k</i></sub> (<span><a href=#b118 id=#b118R class="bibLink tab-link" data-tab=pane-pcw-references>Whittaker 1972</a></span>). In <a href=#f1 class=scrollableLink>Fig. 1</a>, this corresponds to the average of the off‐diagonal cell values in the level 2 dissimilarity matrix D, or to the corresponding single <i>y</i> value in the level 2 scatterplot. This average quantifies <i>how much effective species turnover of the chosen kind (β</i><sub><i>Mt−1</i></sub>, <i>β</i><sub><i>Pt</i></sub><i>or β</i><sub><i>At</i></sub><i>) is expected between two sampling units drawn at random (without replacement) from the dataset</i>.
 </p>
 
 <p>When average pairwise values <img alt="inline image" class=section_image src=images/31.gif> from several datasets are compared with the corresponding regional values Δ<i>c</i><sub>tot</sub>, a positive correlation can be expected between the two kinds of measure. However, the pairwise values are constrained by <i>N</i>=2, whereas the total values are generally based on larger values of <i>N</i>, so 
<img alt="inline image" class=section_image src=images/32.gif> is usually much smaller than Δ<i>c</i><sub>tot</sub> (<a href=#f2 class=scrollableLink>Fig. 2</a>).
 </p>
 
 <section class=article-section__inline-figure>
 <figure class=figure id=f2><a target=_blank href=https://onlinelibrary.wiley.com/cms/asset/b2ba60c7-134d-4f61-bb7d-fd22e9250bcd/ecog_6148_f2.gif>
 <picture>
 <img class=figure__image src=images/33.gif data-lg-src=/cms/asset/b2ba60c7-134d-4f61-bb7d-fd22e9250bcd/ecog_6148_f2.gif alt=image title=image srcset sizes></picture></a><figcaption class=figure__caption>
 <div class=figure__caption__header><strong class=figure__title>Figure 2</strong><div class=figure-extra><a href=# class=open-figure-link>Open in figure viewer</a><a href="https://onlinelibrary.wiley.com/action/downloadFigures?id=f2&amp;doi=10.1111%2Fj.1600-0587.2009.06148.x" class=ppt-figure-link><i aria-hidden=true class=icon-Icon_Download></i><span>PowerPoint</span></a></div>
 </div>
 <div class="figure__caption figure__caption-text">
 <p>How different approaches to calculating compositional heterogeneity <sup>0</sup>β<sub>M</sub> and compositional turnover Δ<i>c</i> (as based on Whittaker's species turnover <sup>0</sup>β<sub>Mt−1</sub>) reflect the data from a 9‐cell window of a regular grid. All sampling units (identified by letters) with the same community type (numbers) have identical species compositions, and sampling units with different community types share no species. β<sub>Mtot</sub> and Δ<i>c</i><sub>tot</sub> are calculated for the entire window (Section 1.1 of the text); 
<span><img alt="inline image" class=section_image src=images/34.gif></span> and 
<span><img alt="inline image" class=section_image src=images/35.gif></span> are averages of all pairwise values between the cells (Section 2.1); 
<span><img alt="inline image" class=section_image src=images/36.gif></span> is the average of pairwise values between each cell and a regional compositional centroid (Section 2.2); and 
<span><img alt="inline image" class=section_image src=images/37.gif></span> is the compositional distinctness of the focal sampling unit F (Section 3.1).</p>
 
 </div>
 </figcaption>
 </figure>
 </section>
 
 <p>Following the suggestion of <span><a href=#b118 id=#b118R class="bibLink tab-link" data-tab=pane-pcw-references>Whittaker (1972)</a></span>, quantifying “beta diversity” with average pairwise compositional dissimilarity has been relatively common. The median has been used occasionally (<span><a href=#b19 id=#b19R class="bibLink tab-link" data-tab=pane-pcw-references>Clarke and Lidgard 2000</a></span>), but most studies have used the arithmetic mean. For presence‐absence data, 
<img alt="inline image" class=section_image src=images/38.gif> seems most commonly to have been based on mean <sup>0</sup>β<sub>Pt</sub> as ranged to the interval [0, 1] (which equals the one‐complement of the Jaccard index; <span><a href=#b97 id=#b97R class="bibLink tab-link" data-tab=pane-pcw-references>Scheiner 1990</a></span>, <span><a href=#b100 id=#b100R class="bibLink tab-link" data-tab=pane-pcw-references>Scheiner and Rey‐Benayas 1994</a></span>, <span><a href=#b19 id=#b19R class="bibLink tab-link" data-tab=pane-pcw-references>Clarke and Lidgard 2000</a></span>, <span><a href=#b7 id=#b7R class="bibLink tab-link" data-tab=pane-pcw-references>Balvanera et al. 2002</a></span>, <span><a href=#b34 id=#b34R class="bibLink tab-link" data-tab=pane-pcw-references>Ellingsen and Gray 2002</a></span>, <span><a href=#b67 id=#b67R class="bibLink tab-link" data-tab=pane-pcw-references>Mac Nally et al. 2004</a></span>, <span><a href=#b111 id=#b111R class="bibLink tab-link" data-tab=pane-pcw-references>Tuomisto and Ruokolainen 2005</a></span>, <span><a href=#b114 id=#b114R class="bibLink tab-link" data-tab=pane-pcw-references>Urban et al. 2006</a></span>, <span><a href=#b102 id=#b102R class="bibLink tab-link" data-tab=pane-pcw-references>Shurin et al. 2009</a></span>). Mean <sup>0</sup>β<sub>Mt−1</sub> (which equals the one‐complement of the Sørensen index) has also been used (<span><a href=#b115 id=#b115R class="bibLink tab-link" data-tab=pane-pcw-references>Vazquez and Givnish 1998</a></span>, <span><a href=#b45 id=#b45R class="bibLink tab-link" data-tab=pane-pcw-references>Hernández et al. 2008</a></span>). It has even been suggested that any dissimilarity index can be applied (<span><a href=#b113 id=#b113R class="bibLink tab-link" data-tab=pane-pcw-references>Tuomisto and Ruokolainen 2008</a></span>, <span><a href=#b94 id=#b94R class="bibLink tab-link" data-tab=pane-pcw-references>Ricotta and Burrascano 2009</a></span>), and some studies have indeed used indices that are not derivable from alpha and gamma diversity and therefore do not correspond to any beta component (<span><a href=#b82 id=#b82R class="bibLink tab-link" data-tab=pane-pcw-references>Oliver et al. 1998</a></span>, <span><a href=#b34 id=#b34R class="bibLink tab-link" data-tab=pane-pcw-references>Ellingsen and Gray 2002</a></span>, <span><a href=#b46 id=#b46R class="bibLink tab-link" data-tab=pane-pcw-references>Hewitt et al. 2005</a></span>, <span><a href=#b48 id=#b48R class="bibLink tab-link" data-tab=pane-pcw-references>Jankowski et al. 2009</a></span>, <span><a href=#b90 id=#b90R class="bibLink tab-link" data-tab=pane-pcw-references>Qian 2009</a></span>). However, even those dissimilarity measures that correspond to a beta component (β<sub>At</sub>, β<sub>Mt−1</sub> and β<sub>Pt</sub>) do not quantify true beta diversity (β<sub>Md</sub>).
 </p>
 
 <p>The mean of pairwise squared Euclidean distances, which equals the variance of the raw data table, has also been promoted as a measure of “beta diversity” (<span><a href=#b62 id=#b62R class="bibLink tab-link" data-tab=pane-pcw-references>Legendre et al. 2005</a></span>). However, this is only compatible with the concept of Δ<i>c</i> when presence–absence data are used, because in this case the squared Euclidean distance is a linear transformation of absolute species turnover <sup>0</sup>β<sub>At</sub> (Section 3 in <span><a href=#b110 id=#b110R class="bibLink tab-link" data-tab=pane-pcw-references>Tuomisto 2010</a></span>). With proportional abundance data, it quantifies regional variance excess 
<img alt="inline image" class=section_image src=images/39.gif> (as in <span><a href=#b108 id=#b108R class="bibLink tab-link" data-tab=pane-pcw-references>ter Braak 1983</a></span>; see also Section 7 in <span><a href=#b110 id=#b110R class="bibLink tab-link" data-tab=pane-pcw-references>Tuomisto 2010</a></span>). With absolute abundance data, the squared Euclidean distance is not a function of alpha and gamma diversity at all (as in <span><a href=#b62 id=#b62R class="bibLink tab-link" data-tab=pane-pcw-references>Legendre et al. 2005, 2009</a></span>, <span><a href=#b2 id=#b2R class="bibLink tab-link" data-tab=pane-pcw-references>Arias‐González et al. 2008</a></span>).
 </p>
 
 <p> <span><a href=#b5 id=#b5R class="bibLink tab-link" data-tab=pane-pcw-references>Bacaro and Ricotta (2007)</a></span> discussed using the semivariogram to examine how “beta diversity” changes with distance. The semivariogram is obtained by plotting the semivariance against geographical distance. Because the semivariance equals the average squared Euclidean distance between sampling units in a given distance class, it can only be used to model changes in mean pairwise <sup>0</sup>β<sub>At</sub> or 
<img alt="inline image" class=section_image src=images/40.gif>. True beta diversity (β<sub>Md</sub>), regional‐to‐local diversity ratio (β<sub>Mt</sub>) and the relative effective species turnover measures (β<sub>Mt−1</sub> and β<sub>Pt</sub>) cannot be expressed in terms of squared Euclidean distances, so their behavior cannot be studied with the semivariogram. Instead, an analogous graph could be constructed using the average values per distance class from a dissimilarity matrix based on β<sub>Mt−1</sub> or (ranged) β<sub>Pt</sub>.
 </p>
 
 </section>
 
 <section class=article-section__sub-content id=ss7>
 
 <h3 class="article-section__sub-title section2" id=ss7-title> 2.2 Average of pairwise values between sampling units and a regional compositional centroid <span><img alt="inline image" class=section_image src=images/41.gif></span></h3>
 
 <p> <span><a href=#b1 id=#b1R class="bibLink tab-link" data-tab=pane-pcw-references>Anderson et al. (2006)</a></span> proposed that multivariate dispersion provides a better measure of “beta diversity” than average pairwise species turnover as calculated using the actual sampling units (
<img alt="inline image" class=section_image src=images/42.gif>). Multivariate dispersion is quantified as the mean distance in ordination space between a sampling unit and the regional compositional centroid. The centroid is a virtual sampling unit whose average compositional dissimilarity with the real sampling units is as small as possible, so it is found at the origin of the ordination axes obtained by subjecting a compositional dissimilarity matrix to principal coordinates analysis. If the chosen compositional dissimilarity measure is compatible with effective species turnover (Δ<i>c</i>), then the mean distance can be indicated by 
<img alt="inline image" class=section_image src=images/43.gif>. This quantifies <i>how much effective species turnover of the chosen kind is expected between the regional compositional centroid and a sampling unit drawn at random from the dataset</i>.
 </p>
 
 <p>Since being proposed by <span><a href=#b1 id=#b1R class="bibLink tab-link" data-tab=pane-pcw-references>Anderson et al. (2006)</a></span>, the 
<img alt="inline image" class=section_image src=images/44.gif> measure has been used at least once (<span><a href=#b109 id=#b109R class="bibLink tab-link" data-tab=pane-pcw-references>Terlizzi et al. 2009</a></span>) to quantify “beta diversity”. However, 
<img alt="inline image" class=section_image src=images/45.gif> is a measure of effective species turnover Δ<i>c</i> rather than of true beta diversity β<sub>Md</sub>. Since the centroid is by definition in the middle of the real sampling units, mean turnover between the centroid and a real sampling units is smaller than that between two real sampling units 
<img alt="inline image" class=section_image src=images/46.gif>. The latter is already smaller than the overall effective species turnover in the dataset Δ<i>c</i><sub>tot</sub>, because its maximum value is constrained by <i>N</i>=2. The value of 
<img alt="inline image" class=section_image src=images/47.gif> will therefore always be smaller than the values of
<img alt="inline image" class=section_image src=images/48.gif> and Δ<i>c</i><sub>tot</sub> for the same dataset. The three measures can be expected to be positively correlated across datasets, but the relationships are not linear (<a href=#f2 class=scrollableLink>Fig. 2</a>).
 </p>
 
 </section>
 
 <section class=article-section__sub-content id=ss8>
 
 <h3 class="article-section__sub-title section2" id=ss8-title> 2.3 Compositional gradient length Δ<i>c</i><sub><i>j,k</i>max</sub> and Δ<i>c'</i><sub>max</sub></h3>
 
 <p>The first axis of an ordination represents, by definition, the compositional dimension with most variation in a dataset (<span><a href=#b61 id=#b61R class="bibLink tab-link" data-tab=pane-pcw-references>Legendre and Legendre 1998</a></span>). Its length can hence be used as a measure of compositional gradient length. <a href=#f1 class=scrollableLink>Fig. 1</a> shows ordination scores, or the positions of the sampling units along the first ordination axis, in the level 1 table B. Compositional gradient length is approximated by the difference between the largest and smallest cell values in this table (Δ<i>c′</i><sub>max</sub>). Many studies have used the first axis of detrended correspondence analysis (DCA) to quantify “beta diversity” (<span><a href=#b47 id=#b47R class="bibLink tab-link" data-tab=pane-pcw-references>Hill and Gauch 1980</a></span>, <span><a href=#b78 id=#b78R class="bibLink tab-link" data-tab=pane-pcw-references>Økland 1986</a></span>, <span><a href=#b33 id=#b33R class="bibLink tab-link" data-tab=pane-pcw-references>Eilertsen et al. 1990</a></span>, <span><a href=#b79 id=#b79R class="bibLink tab-link" data-tab=pane-pcw-references>Økland et al. 1990</a></span>, <span><a href=#b75 id=#b75R class="bibLink tab-link" data-tab=pane-pcw-references>Naranjo et al. 1998</a></span>). The advantage of DCA axis length is that it is expressed in standard deviation units, which are considered comparable across datasets. However, the implicit dissimilarity measure in DCA is the chi‐square metric, which is not a function of gamma and alpha diversity. Therefore, DCA axis length quantifies compositional turnover in a way that is not compatible with those effective species turnover measures Δ<i>c</i> that can be considered beta components of diversity.
 </p>
 
 <p>A suitable effective species turnover measure Δ<i>c</i> can be used by subjecting a dissimilarity matrix (level 2 table D in <a href=#f1 class=scrollableLink>Fig. 1</a>) to principal coordinates analysis (PCoA). The length of the first PCoA axis (Δ<i>c′</i><sub>max</sub>) can be interpreted as <i>the maximum amount of effective species turnover of the chosen kind that can be observed in one compositional dimension in the dataset</i>. Compositional gradient length is then related to the highest values in the compositional dissimilarity matrix (Δ<i>c</i><sub><i>j,k</i>max</sub>). However, the interpretation of compositional axis length based on PCoA is complicated by the fact that the relative effective species turnover measures (β<sub>Mt−1</sub> and β<sub>Pt</sub>) saturate at a fixed maximum value when sampling units share no species. This leads to spurious structures in the ordination space and underestimation of compositional gradient length. The simplest solution to this problem is the “step‐across” or “extended dissimilarity” approach, in which intermediate sampling units are used to estimate the total compositional distance between sampling units that share no species (<span><a href=#b120 id=#b120R class="bibLink tab-link" data-tab=pane-pcw-references>Williamson 1978</a></span>, <span><a href=#b32 id=#b32R class="bibLink tab-link" data-tab=pane-pcw-references>De'ath 1999</a></span>). Hybrid multidimensional scaling (HMDS), in turn, uses small compositional dissimilarity values in a metric setting and large values in a non‐metric setting (<span><a href=#b35 id=#b35R class="bibLink tab-link" data-tab=pane-pcw-references>Faith et al. 1987</a></span>). This preserves the rank order of the dissimilarities rather than their actual values, which solves the problem of dissimilarity saturation but makes it difficult to compare gradient lengths across datasets. Although both approaches have potential, I am aware of no studies that have used PCoA or HMDS axis length as an explicit measure of compositional gradient length.
 </p>
 
 <p>Compositional gradient length is measured along a single dimension of the multidimensional compositional space, so only that part of total effective species turnover that takes place in this particular dimension is quantified. Datasets can differ substantially in how large a proportion of the total compositional variation takes place in the first dimension, so Δ<i>c</i><sub><i>j,k</i>max</sub> and Δ<i>c′</i><sub>max</sub> can rank datasets in a different way than Δ<i>c</i><sub>tot</sub> does.
 </p>
 
 </section>
 
 <section class=article-section__sub-content id=ss9>
 
 <h3 class="article-section__sub-title section2" id=ss9-title> 2.4 Compositional gradient length along an external gradient Δ<i>c</i><sub>(Δ<i>g</i>)</sub></h3>
 
 <p>So far, we have been operating with compositional data only. However, many ecological studies also record the values of some external factor <i>g</i> (such as spatial position or an environmental variable) that is hypothesised to contribute to change in community composition. The <i>g</i><sub><i>j</i></sub> values recorded in each sampling unit <i>j</i> then form a second raw data matrix (level 1 table C in <a href=#f1 class=scrollableLink>Fig. 1</a>). The compositional gradient length along a specified interval of the external gradient Δ<i>g</i> is then Δ<i>c</i><sub>(Δ<i>g</i>)</sub>, which quantifies <i>the amount of effective species turnover of the specified kind that is related to the explanatory gradient</i>. In <a href=#f1 class=scrollableLink>Fig. 1</a>, this is approximated by the difference between the <i>y</i> values that the regression line in the level 1 scatterplot obtains at <i>g</i><sub>max</sub> and <i>g</i><sub>min</sub>, i.e. ∣<i>c′</i><sub>(<i>g</i>max)</sub>−<i>c′</i><sub>(<i>g</i>min)</sub>∣.
 </p>
 
 <p>Another approach to quantifying Δ<i>c</i><sub>(Δ<i>g</i>)</sub> is to calculate the <i>beta‐turnover</i> measure of <span><a href=#b123 id=#b123R class="bibLink tab-link" data-tab=pane-pcw-references>Wilson and Shmida (1984)</a></span>, see also <span><a href=#b101 id=#b101R class="bibLink tab-link" data-tab=pane-pcw-references>Shmida and Wilson 1985</a></span>). To do so, one first arranges the sampling units along the external gradient of interest and defines for each species its total range along the gradient. Beta‐turnover then equals ∣<i>G</i><sub>(Δ<i>g</i>)</sub>+<i>L</i><sub>(Δ<i>g</i>)</sub>∣/2α<sub>int</sub>. Here α<sub>int</sub> is the arithmetic mean number of species at the points along the gradient, after the species ranges have been interpolated such that they are continuous. <i>G</i> is the cumulative number of species gained and <i>L</i> is the cumulative number of species lost between the beginning and the end of the gradient section of interest.
 </p>
 
 <div class=paragraph-displayedItem-text-type>All species that are observed along the gradient have to be gained once, unless they are already present in the beginning of the gradient section. Similarly, all species have to be lost once, unless they are still present in the end. Consequently, 
 <div class=inline-equation id=di-fml-18><span class="inline-equation__construct figure"><img class=figure__image src=images/49.gif alt=image title=image></span><span class=inline-equation__label></span></div> 
 </div>
 
 <div class=paragraph-displayedItem-text-type>where γ is the total species richness of the dataset, <i>a</i> is the number of species shared between the beginning and the end of the gradient section, <i>b</i> is the number of species present at the beginning but not the end of the gradient section, and <i>c</i> is the number of species present at the end but not the beginning of the gradient section. From this follows that
 <div class=inline-equation id=di-fml-19><span class="inline-equation__construct figure"><img class=figure__image src=images/50.gif alt=image title=image></span><span class=inline-equation__label></span></div> 
 </div>
 
 <p>where α<sub>b,e</sub> is mean species richness of the beginning and end of the gradient. If the endpoints have the same mean richness as any other points along the gradient, the term α<sub>b,e</sub>/α<sub>int</sub> equals unity. Beta‐turnover then appears similar to <sup>0</sup>β<sub>Mt−1</sub>=γ/α<sub>t</sub>−1 (see also <span><a href=#b116 id=#b116R class="bibLink tab-link" data-tab=pane-pcw-references>Vellend 2001</a></span>). However, there is one important difference: α<sub>int</sub> is based on interpolated ranges of species, which means that all species are assumed to be present at each point between their first and last occurrence along the habitat gradient, whether they were observed in intermediate sampling units or not. The filling of gaps in species ranges increases the value of α<sub>int</sub> above that of α<sub>t</sub>. The smaller the amount of interpolation that is needed to make species ranges continuous along the habitat gradient, the smaller the amount by which the value of α<sub>int</sub> exceeds the value of α<sub>t</sub>, and the smaller the amount by which <sup>0</sup>β<sub>Mt−1</sub> (i.e. Δ<i>c</i><sub>tot</sub>) exceeds the value of beta‐turnover (i.e. Δ<i>c</i><sub>(Δ<i>g</i>)</sub>).
 </p>
 
 <p>Most studies that have used beta‐turnover as a measure of “beta diversity” have applied it to sampling unit pairs (<span><a href=#b103 id=#b103R class="bibLink tab-link" data-tab=pane-pcw-references>Simmons and Cowling 1996</a></span>, <span><a href=#b75 id=#b75R class="bibLink tab-link" data-tab=pane-pcw-references>Naranjo et al. 1998</a></span>, <span><a href=#b31 id=#b31R class="bibLink tab-link" data-tab=pane-pcw-references>Davis et al. 1999</a></span>, <span><a href=#b56%20#b57 id="#b56 #b57R" class="bibLink tab-link" data-tab=pane-pcw-references>Koleff et al. 2003a,b</a></span>, <span><a href=#b6 id=#b6R class="bibLink tab-link" data-tab=pane-pcw-references>Bach et al. 2007</a></span>). In this case, no interpolation of species ranges is possible, so beta‐turnover takes the same value as <sup>0</sup>β<sub>Mt−1</sub> (= the one‐complement of the Sørensen index). Some studies used <i>N</i> &gt;2 sampling units arranged along a geographical rather than a habitat gradient, and counted the cumulative number of gained and lost species from one end of the gradient to the other without first interpolating species ranges (<span><a href=#b12 id=#b12R class="bibLink tab-link" data-tab=pane-pcw-references>Blackburn and Gaston 1996</a></span>, <span><a href=#b121 id=#b121R class="bibLink tab-link" data-tab=pane-pcw-references>Willig and Gannon 1997</a></span>). Disjunct species ranges cause the same species to be counted as lost and/or gained more than once, which leads to values that overestimate Δ<i>c</i><sub>(Δ<i>g</i>)</sub> and can even be larger than Δ<i>c</i><sub>tot</sub>.
 </p>
 
 <p>Another approach to estimating compositional gradient length along an external gradient is to first quantify the instantaneous rates of compositional turnover (to be defined in Section 4.6) for all points along the gradient, and then take the sum of these (which corresponds to taking the integral of the instantaneous turnover rate function; <span><a href=#b122 id=#b122R class="bibLink tab-link" data-tab=pane-pcw-references>Wilson and Mohler 1983</a></span>, <span><a href=#b81 id=#b81R class="bibLink tab-link" data-tab=pane-pcw-references>Oksanen and Tonteri 1995</a></span>). Depending on how the instantaneous rate of compositional turnover is quantified, this approach may or may not measure turnover in a way that is consistent with a beta component of diversity.
 </p>
 
 <p>Compositional gradient length along an external gradient Δ<i>c</i><sub>(Δ<i>g</i>)</sub> is dependent not only on compositional data, but also on which external reference gradient is chosen and which part of that gradient is observed. This is because only that part of total compositional gradient length Δ<i>c</i><sub><i>j,k</i>max</sub> or Δ<i>c′</i><sub>max</sub> is quantified that is related to the relevant interval of the external gradient of interest Δ<i>g</i>. A given compositional dataset can easily show a substantial amount of effective species turnover along one environmental gradient but very little along another. Consequently, Δ<i>c</i><sub>(Δ<i>g</i>)</sub> measures a different phenomenon than the corresponding total effective species turnover Δ<i>c</i><sub>tot</sub> does. The two may or may not be correlated across datasets, and this can depend on, among other things, whether the environmental gradients themselves are comparable.
 </p>
 
 </section>
 
 <section class=article-section__sub-content id=ss10>
 
 <h3 class="article-section__sub-title section2" id=ss10-title> 2.5 Number of half‐change units ΔΔ<i>g</i><sub>(Δlog(1–Δ<i>c</i>))</sub></h3>
 
 <div class=paragraph-displayedItem-text-type>In one of his examples on how “beta diversity” can be quantified, <span><a href=#b117 id=#b117R class="bibLink tab-link" data-tab=pane-pcw-references>Whittaker (1960)</a></span> introduced the number of half‐change units. The half‐change unit is derived from a scatterplot in which the <i>x</i> axis shows Δ<i>g</i><sub><i>j,k</i></sub> values and the <i>y</i> axis shows log(<i>C</i><sub>S</sub>) values, <i>C</i><sub>S</sub> being the Sørensen index and representing 1−Δ<i>c</i><sub><i>j,k</i></sub> (<a href=#f3 class=scrollableLink>Fig. 3</a>; compare with the level 2 scatterplot of <a href=#f1 class=scrollableLink>Fig. 1</a>). The number of half‐change units (<i>HC</i>) equals:
 <div class=inline-equation id=di-fml-20><span class="inline-equation__construct figure"><img class=figure__image src=images/51.gif alt=image title=image></span><span class=inline-equation__label></span></div> 
 </div>
 
 <section class=article-section__inline-figure>
 <figure class=figure id=f3><a target=_blank href=https://onlinelibrary.wiley.com/cms/asset/e81a2248-0820-49a5-b253-fa55395ad6f8/ecog_6148_f3.gif>
 <picture>
 <img class=figure__image src=images/52.gif data-lg-src=/cms/asset/e81a2248-0820-49a5-b253-fa55395ad6f8/ecog_6148_f3.gif alt=image title=image srcset sizes></picture></a><figcaption class=figure__caption>
 <div class=figure__caption__header><strong class=figure__title>Figure 3</strong><div class=figure-extra><a href=# class=open-figure-link>Open in figure viewer</a><a href="https://onlinelibrary.wiley.com/action/downloadFigures?id=f3&amp;doi=10.1111%2Fj.1600-0587.2009.06148.x" class=ppt-figure-link><i aria-hidden=true class=icon-Icon_Download></i><span>PowerPoint</span></a></div>
 </div>
 <div class="figure__caption figure__caption-text">
 <p>The derivation of the half‐change unit and the number of half‐change units in a dataset. The data from the level 2 scatterplot in <a href=#f1 class=scrollableLink>Fig. 1</a> is here displayed such that the <i>y</i> axis has been converted to similarities by subtracting from unity and then log‐transformed. Compositional similarity is quantified using the Sørensen index (<i>C</i><sub>S</sub>). The regression line shows the expected decrease in log(<i>C</i><sub>S</sub>) as a function of increasing intersample distance along gradient <i>g</i>. The half‐change unit is the distance Δ<i>g</i><sub>(0.5<i>C</i>)</sub> at which the expected compositional similarity equals half of the estimated value at Δ<i>g</i>=0, and the number of half‐change units is ΔΔ<i>g</i><sub>max</sub>/ΔΔ<i>g</i><sub>(Δ0.5<i>C</i>)</sub>.</p>
 
 </div>
 </figcaption>
 </figure>
 </section>
 
 <div class=paragraph-displayedItem-text-type>Simple geometry dictates that <i>HC</i> can also be calculated as (<a href=#f3 class=scrollableLink>Fig. 3</a>):
 <div class=inline-equation id=di-fml-21><span class="inline-equation__construct figure"><img class=figure__image src=images/53.gif alt=image title=image></span><span class=inline-equation__label></span></div> 
 </div>
 
 <p>Here <i>C</i><sub>S(Δ<i>g</i>max)</sub> is the expected Sørensen index value between those sampling units that are furthest away from each other along the environmental gradient of interest (see also <span><a href=#b118 id=#b118R class="bibLink tab-link" data-tab=pane-pcw-references>Whittaker 1972</a></span>, <span><a href=#b116 id=#b116R class="bibLink tab-link" data-tab=pane-pcw-references>Vellend 2001</a></span>). The half‐change unit itself is ΔΔ<i>g</i><sub>(Δ0.5<i>C</i>)</sub>, which quantifies <i>the amount of increase in environmental distance</i><i>needed to reduce compositional similarity between sampling units by one‐half</i>. The number of half‐change units then expresses <i>the length of the environmental distance gradient in multiples of the half‐change unit</i>. In terms of community composition, the number of half‐change units quantifies how many times the compositional similarity is halved along the Δ<i>g</i> axis, when comparison is consistently made with the value at the end of the previous half‐change unit. This measure assumes that compositional similarity <i>C</i><sub>S</sub> behaves like the number of atoms of a radioactive element: adding one half‐change unit to the <i>x</i> axis value always halves <i>C</i><sub>S</sub>, no matter what the initial Δ<i>g</i> value. For example, the regression model may forecast that <i>C</i><sub>S</sub> between sampling units in the same geographical location is 1.0 and <i>C</i><sub>S</sub> between sampling units 50 km apart is 0.5. Then <i>C</i><sub>S</sub> between sampling units 100 km apart is forecasted to be 0.25, between sampling units 150 km apart 0.125 and so on. A study extent of 150 km would then correspond to 150 km/50 km=3 half‐change units. As the number of half‐change units increases, the estimate that <i>HC</i> gives of the length of the explanatory gradient becomes less accurate, just as age determination based on radioactive decay becomes less accurate when the number of half‐times increases.
 </p>
 
 <p>The number of half‐change units represents a higher level of abstraction than the previous beta diversity measures, namely level 3 (<span><a href=#b113 id=#b113R class="bibLink tab-link" data-tab=pane-pcw-references>Tuomisto and Ruokolainen 2006, 2008</a></span>). This level is a result of taking Δ<i>c</i> values from level 2 and calculating ΔΔ<i>c</i> values for them. All pairwise ΔΔ<i>c</i><sub><i>j,k</i></sub> values can be arranged into a level 3 dissimilarity matrix with <i>N</i><sup>2</sup> rows and <i>N</i><sup>2</sup> columns, where each cell value is the difference between two sampling unit pairs in their within‐pair compositional dissimilarity (table F in <a href=#f1 class=scrollableLink>Fig. 1</a>). Level 1 data correspond to community composition <i>c</i>, level 2 data to difference in community composition Δ<i>c</i> and level 3 data to difference in difference in community composition ΔΔ<i>c</i>. In a similar way, level 3 data are derived for the environmental data (table G in <a href=#f1 class=scrollableLink>Fig. 1</a>). The number of half‐change units can be referred to by ΔΔ<i>g</i><sub>(Δlog(1–Δ<i>c</i>))</sub> which indicates that it quantifies level‐3 explanatory data in terms of differences in log‐transformed one‐complements of level 2 compositional dissimilarities. Obviously, values of ΔΔ<i>g</i><sub>(Δlog(1–Δ<i>c</i>))</sub> obtained for the same compositional data can vary widely depending on the choice of reference gradient <i>g</i>. The beta components of diversity correspond to Δ<i>c</i>, and therefore differ from <i>HC</i> in both the focal kind of data (compositional vs explanatory) and in the level of abstraction addressed (level 2 vs level 3). The number of half‐change units has nevertheless been used as a measure of “beta diversity” in several studies (<span><a href=#b117%20#b118 id="#b117 #b118R" class="bibLink tab-link" data-tab=pane-pcw-references>Whittaker 1960, 1972</a></span>, <span><a href=#b60 id=#b60R class="bibLink tab-link" data-tab=pane-pcw-references>Lee and La Roi 1979</a></span>, <span><a href=#b80 id=#b80R class="bibLink tab-link" data-tab=pane-pcw-references>Oksanen 1983</a></span>, <span><a href=#b116 id=#b116R class="bibLink tab-link" data-tab=pane-pcw-references>Vellend 2001</a></span>).
 </p>
 
 </section>
 
 </section>
 <section class=article-section__content id=ss11>
 
 <h2 class="article-section__title section__title section1" id=ss11-title> 3. Approaches that distort β<sub>M</sub> or Δ<i>c</i></h2>
 
 <section class=article-section__sub-content id=ss12>
 
 <h3 class="article-section__sub-title section2" id=ss12-title> 3.1 Compositional distinctness of a focal sampling unit <span><img alt="inline image" class=section_image src=images/54.gif></span></h3>
 
 <p>Compositional distinctness of a focal sampling unit <img alt="inline image" class=section_image src=images/55.gif> quantifies <i>how much species turnover is expected to take place between the focal sampling unit F and another sampling unit drawn at random from the dataset</i>. In <a href=#f2 class=scrollableLink>Fig. 2</a>, 
<img alt="inline image" class=section_image src=images/56.gif> is the mean pairwise species turnover between the focal sampling unit F and the other sampling units j within the specified geographical window.
 </p>
 
 <p>In practice, compositional distinctness of a focal sampling unit has mostly been used in macroecological studies where a large area is divided into a grid of equally‐sized cells. A moving window of a fixed number of grid cells is used, for example 3 by 3 cells. The grid cell at the center is the focal sampling unit, and the other grid cells in the window are the sampling units whose pairwise species turnover values with the focal sampling unit are averaged. Alternatively, the average of pairwise values obtained between the focal sampling unit and a specified number of its neighbours to any of the cardinal directions can be used.</p>
 
 <p>Although compositional distinctness of a focal sampling unit has been used to quantify “beta diversity” (<span><a href=#b64 id=#b64R class="bibLink tab-link" data-tab=pane-pcw-references>Lennon et al. 2001</a></span>, <span><a href=#b46 id=#b46R class="bibLink tab-link" data-tab=pane-pcw-references>Hewitt et al. 2005</a></span>, <span><a href=#b38 id=#b38R class="bibLink tab-link" data-tab=pane-pcw-references>Gaston et al. 2007</a></span>, <span><a href=#b45 id=#b45R class="bibLink tab-link" data-tab=pane-pcw-references>Hernández et al. 2008</a></span>), it is in fact insensitive to the amount of compositional heterogeneity within the window. Instead, the value of 
<img alt="inline image" class=section_image src=images/57.gif> depends on which sampling unit is chosen as the focal one; consider using sampling unit G instead of F in Setup 1 of <a href=#f2 class=scrollableLink>Fig. 2</a>. Obviously, what 
<img alt="inline image" class=section_image src=images/58.gif> quantifies is not correlated with either Δ<i>c</i><sub>tot</sub> or
<img alt="inline image" class=section_image src=images/59.gif>, so using 
<img alt="inline image" class=section_image src=images/60.gif> as a measure of “beta diversity” can give very misleading results.
 </p>
 
 </section>
 
 <section class=article-section__sub-content id=ss13>
 
 <h3 class="article-section__sub-title section2" id=ss13-title> 3.2 Compositional non‐nestedness of a species‐poor sampling unit in a species‐richer one</h3>
 
 <p> <span><a href=#b104 id=#b104R class="bibLink tab-link" data-tab=pane-pcw-references>Simpson (1943)</a></span> was interested in the degree to which species in a species‐poor sampling unit are a subset of species in a more species‐rich sampling unit, and quantified this with <i>S</i><sub>sim</sub>=<i>a</i>/[<i>a</i>+min(<i>b</i>, <i>c</i>)]. Here <i>a</i> is the number of species shared by both sampling units, <i>b</i> is the number of species unique to the first sampling unit and <i>c</i> is the number of species unique to the second sampling unit. This index measures the degree to which the species‐poorer sampling unit is compositionally nested within the species‐richer sampling unit. <i>S</i><sub>sim</sub> resembles the Sørensen index <i>C</i><sub>S</sub>=1−<sup>0</sup>β<sub>Mt−1</sub> but ignores all species that are unique to the species‐richer sampling unit, and hence obtains a larger value than <i>C</i><sub>S</sub> whenever <i>b</i> and <i>c</i> differ. Whereas <sup>0</sup>β<sub>Mt−1</sub> quantifies relative species turnover (which takes into account both species gains and losses between two sampling units), 1−<i>S</i><sub>sim</sub> quantifies <i>relative species loss when moving from the less species‐rich sampling unit to the more‐species rich one</i>, i.e. <i>the proportion of the species‐poorer sampling unit that is not nested in the species‐richer one</i>.
 </p>
 
 <p> <span><a href=#b9 id=#b9R class="bibLink tab-link" data-tab=pane-pcw-references>Baselga (2010)</a></span> partitioned “beta diversity”, which he equated with <sup>0</sup>β<sub>Mt−1</sub>, into two additive components, one of which was 1−<i>S</i><sub>sim</sub>. Several other studies have used 1−<i>S</i><sub>sim</sub> itself as a measure of “beta diversity” (<span><a href=#b57 id=#b57R class="bibLink tab-link" data-tab=pane-pcw-references>Koleff et al. 2003b</a></span>, <span><a href=#b72 id=#b72R class="bibLink tab-link" data-tab=pane-pcw-references>Mena and Vázquez‐Domínguez 2005</a></span>, <span><a href=#b10 id=#b10R class="bibLink tab-link" data-tab=pane-pcw-references>Baselga and Jiménez‐Valverde 2007</a></span>, <span><a href=#b8 id=#b8R class="bibLink tab-link" data-tab=pane-pcw-references>Baselga 2008</a></span>, <span><a href=#b58 id=#b58R class="bibLink tab-link" data-tab=pane-pcw-references>La Sorte et al. 2008</a></span>, <span><a href=#b89%20#b90 id="#b89 #b90R" class="bibLink tab-link" data-tab=pane-pcw-references>Qian 2008, 2009</a></span>, <span><a href=#b65 id=#b65R class="bibLink tab-link" data-tab=pane-pcw-references>Leprieur et al. 2009</a></span>). Most of these studies referred to the beta‐sim measure of <span><a href=#b64 id=#b64R class="bibLink tab-link" data-tab=pane-pcw-references>Lennon et al. (2001)</a></span>, but this is actually the mean 1−<i>S</i><sub>sim</sub> value between a focal sampling unit and each of its neighbours in a specified geographical window. Beta‐sim as originally described has therefore been used as a measure of “beta‐diversity” less often than appears from the number of times it has been cited (<span><a href=#b55 id=#b55R class="bibLink tab-link" data-tab=pane-pcw-references>Koleff and Gaston 2002</a></span>, <span><a href=#b38 id=#b38R class="bibLink tab-link" data-tab=pane-pcw-references>Gaston et al. 2007</a></span>, <span><a href=#b53 id=#b53R class="bibLink tab-link" data-tab=pane-pcw-references>Kallimanis et al. 2008</a></span>, <span><a href=#b71 id=#b71R class="bibLink tab-link" data-tab=pane-pcw-references>Melo et al. 2009</a></span>).
 </p>
 
 <p>Beta‐sim is analogous to compositional distinctness of a focal sampling unit (Section 3.1, above), but the phenomenon it quantifies is <i>the expected degree to which the less species‐rich sampling unit is compositionally not nested within the more species‐rich one when the focal sampling unit is compared to another sampling unit drawn at random from a geographical window</i>. Just like compositional distinctness of a focal sampling unit, beta‐sim is very sensitive to the choice of the focal sampling unit. Consequently, the correlation between beta‐sim and Δ<i>c</i><sub>tot</sub> or 
<img alt="inline image" class=section_image src=images/61.gif> may be weak, and interpreting beta‐sim as “beta diversity” can be very misleading.
 </p>
 
 </section>
 
 <section class=article-section__sub-content id=ss14>
 
 <h3 class="article-section__sub-title section2" id=ss14-title> 3.3 Regional approach with α and γ based on different datasets</h3>
 
 <p>It has been rather popular to calculate “beta diversity” such that the alpha and gamma diversities are derived from entirely or partly different datasets. In this case, however, the idea of partitioning γ into α and β components is lost, because the sampling units used to quantify α are not the same as those used to quantify γ. Sometimes such a discrepancy has arisen inadvertently when the aim was to harmonise sampling effort among sites: sites with very small sampling effort were excluded when quantifying α but included when quantifying γ (<span><a href=#b19 id=#b19R class="bibLink tab-link" data-tab=pane-pcw-references>Clarke and Lidgard 2000</a></span>).
 </p>
 
 <p>Some studies have divided the total species richness of a regional dataset by the species richness of a focal sampling unit to quantify β<sub>Mt</sub> (<span><a href=#b64 id=#b64R class="bibLink tab-link" data-tab=pane-pcw-references>Lennon et al. 2001</a></span>, <span><a href=#b38 id=#b38R class="bibLink tab-link" data-tab=pane-pcw-references>Gaston et al. 2007</a></span>), β<sub>Mt−1</sub> (<span><a href=#b53 id=#b53R class="bibLink tab-link" data-tab=pane-pcw-references>Kallimanis et al. 2008</a></span>) or log(β<sub>Mt</sub>) (<span><a href=#b106 id=#b106R class="bibLink tab-link" data-tab=pane-pcw-references>Soininen et al. 2007</a></span>). However, β<sub>Mt</sub>=γ/α<sub>t</sub> where 
<img alt="inline image" class=section_image src=images/62.gif> is the mean species richness in all the sampling units <i>j</i> that contributed to γ (<i>Diversity in relation to two classifications</i> in <span><a href=#b110 id=#b110R class="bibLink tab-link" data-tab=pane-pcw-references>Tuomisto 2010</a></span>). Replacing α<sub>t</sub> by the species richness of a focal sampling unit gives the ratio γ/γ<sub>F</sub> (where γ<sub>F</sub> is gamma diversity at a more local scale, within sampling unit F). This quantifies how many times as rich in species the regional dataset is than the focal sampling unit. Selecting a different focal sampling unit from the same regional dataset could give a very different value of γ/γ<sub>F</sub>. The degree of error made if this measure is interpreted as if it equaled <sup>0</sup>β<sub>Mt</sub> depends on the degree to which γ<sub>F</sub> deviates from the mean species diversity of all sampling units 
<img alt="inline image" class=section_image src=images/63.gif>.
 </p>
 
 <p>Following the beta‐2 measure of <span><a href=#b43 id=#b43R class="bibLink tab-link" data-tab=pane-pcw-references>Harrison et al. (1992)</a></span> and the beta‐3 measure of <span><a href=#b119 id=#b119R class="bibLink tab-link" data-tab=pane-pcw-references>Williams (1996)</a></span>, several studies have used the sampling unit with the highest species richness γ<sub><i>j</i>max</sub> as the focal sampling unit (<span><a href=#b12 id=#b12R class="bibLink tab-link" data-tab=pane-pcw-references>Blackburn and Gaston 1996</a></span>, <span><a href=#b82 id=#b82R class="bibLink tab-link" data-tab=pane-pcw-references>Oliver et al. 1998</a></span>, <span><a href=#b31 id=#b31R class="bibLink tab-link" data-tab=pane-pcw-references>Davis et al. 1999</a></span>, <span><a href=#b19 id=#b19R class="bibLink tab-link" data-tab=pane-pcw-references>Clarke and Lidgard 2000</a></span>, <span><a href=#b40 id=#b40R class="bibLink tab-link" data-tab=pane-pcw-references>Gray 2000</a></span>, <span><a href=#b54 id=#b54R class="bibLink tab-link" data-tab=pane-pcw-references>Koleff and Gaston 2001</a></span>, <span><a href=#b87 id=#b87R class="bibLink tab-link" data-tab=pane-pcw-references>Plaza Pinto et al. 2008</a></span>). This approach quantifies the smallest possible value of ranged <sup>0</sup>β<sub>Mt</sub> (in the case of beta‐2) or <sup>0</sup>β<sub>Pt</sub> (in the case of beta‐3), i.e. the value that would be obtained if all sampling units had as many species as the most species‐rich one that has actually been observed, but total species richness remained unchanged. The approach has been justified by arguing that using γ<sub><i>j</i>max</sub> makes the measure less sensitive to trends in within‐sampling‐unit species diversity (<span><a href=#b43 id=#b43R class="bibLink tab-link" data-tab=pane-pcw-references>Harrison et al. 1992</a></span>). However, such a trend is an indication of compositional heterogeneity, and therefore relevant when the focus is on beta diversity. A consequence of using γ<sub><i>j</i>max</sub> instead of 
<img alt="inline image" class=section_image src=images/64.gif> is that it makes the estimate of “beta diversity” dependent on the variability among sampling units in γ<sub><i>j</i></sub>, with the amount of underestimation being related to the amount by which the highest within‐sampling‐unit species richness γ<sub><i>j</i>max</sub> happens to exceed 
<img alt="inline image" class=section_image src=images/65.gif> in the data. An even more extreme approach was taken by <span><a href=#b2 id=#b2R class="bibLink tab-link" data-tab=pane-pcw-references>Arias‐González et al. (2008)</a></span>, who used the variability in γ<sub><i>j</i></sub> as a measure of “beta diversity”, and thereby entirely detached the concept of “beta diversity” from alpha and gamma diversity.
 </p>
 
 <p> <span><a href=#b42 id=#b42R class="bibLink tab-link" data-tab=pane-pcw-references>Harrison (1997)</a></span> used each sampling unit as the focal one in turn to obtain as many estimates of γ/γ<sub>F</sub> for each region as there were sampling units, and reported their arithmetic mean. However, mean 
<img alt="inline image" class=section_image src=images/66.gif> equals
<img alt="inline image" class=section_image src=images/67.gif> only when all sampling units have the same species richness γ<sub><i>j</i></sub>. As the variability in γ<sub><i>j</i></sub> among the sampling units increases, mean γ/γ<sub>F</sub> increasingly exceeds <sup>0</sup>β<sub>Mt</sub>.
 </p>
 
 <p> <span><a href=#b107 id=#b107R class="bibLink tab-link" data-tab=pane-pcw-references>Stevens and Willig (2002)</a></span> quantified “beta diversity” as 1−γ<sub>F</sub>/γ for different regions in the Americas such that γ<sub>F</sub> values were taken from publications reporting numbers of species observed in field sites, and γ was inferred from a compilation of species distribution maps. Rather than quantifying proportional species turnover (
<img alt="inline image" class=section_image src=images/68.gif>) in a defined dataset, this measure quantifies how large a proportion of the species that are thought to occur in a region were actually found in a given field sampling unit. Since the sampling effort was not uniform for either γ<sub>F</sub> or γ, the continental variation in 1−γ<sub>F</sub>/γ reflects variation in the quality of the distribution maps and the thoroughness of the field sampling to an unknown degree (<i>Biases and constraints</i>, below).
 </p>
 
 </section>
 
 <section class=article-section__sub-content id=ss15>
 
 <h3 class="article-section__sub-title section2" id=ss15-title> 3.4 Average of all pairwise values with compositional data taken from outside the sampling units of interest</h3>
 
 <p> <span><a href=#b73 id=#b73R class="bibLink tab-link" data-tab=pane-pcw-references>Mumby (2001)</a></span> suggested that “beta diversity” can be quantified with the help of a habitat map. First, field sampling units are classified into habitat types using compositional data. Then the habitat classification is extrapolated to the entire area of interest using a satellite image. Finally, a moving window of a fixed number of pixels is passed over the satellite image. For each window position, the <i>average of all pairwise compositional dissimilarity values between those field sampling units that represent the spectral habitat types found in the pixel window</i> is calculated.
 </p>
 
 <p>The main problem with the approach is that none of the field sampling units that are used to calculate average pairwise compositional dissimilarity between pixels in a given window need to come from that window. Therefore, the obtained compositional dissimilarity values represent the compositional heterogeneity of the habitat classes over the entire extent of the study, which may be considerably larger than the extent of the pixel window being considered. As a consequence, average compositional dissimilarity between pixels within the window is probably overestimated.</p>
 
 <p> <span><a href=#b41 id=#b41R class="bibLink tab-link" data-tab=pane-pcw-references>Harborne et al. (2006)</a></span> modified the approach by taking into account the proportional abundances of the different habitats in each window. They calculated the sum of the squared Bray‐Curtis index values between habitat type pairs, log‐transformed the sum, and multiplied the result by a diversity index based on habitat type abundances within the pixel window. It is difficult to see what this index quantifies, but clearly it is not related to any of the beta components that can be derived from gamma and alpha diversity.
 </p>
 
 </section>
 
 </section>
 <section class=article-section__content id=ss16>
 
 <h2 class="article-section__title section__title section1" id=ss16-title> 4. Approaches that quantify a rate of change</h2>
 
 <section class=article-section__sub-content id=ss17>
 
 <h3 class="article-section__sub-title section2" id=ss17-title> 4.1 Species diversity accumulation rate Δ<sup><i>q</i></sup><i>D</i>/Δ<i>x</i></h3>
 
 <div class=paragraph-displayedItem-text-type>Species diversity accumulation can be defined as the increase in the effective number of observed species <sup><i>q</i></sup><i>D</i> with increasing sampling effort (traditional species accumulation is obtained as a special case when <i>q</i>=0). Sampling effort can here be quantified with the number of sampling units, provided that each sampling unit (SU) represents the same sampling effort (in terms of the number of individuals observed, surface area inventoried, or other appropriate criterion). Species diversity accumulation rate is a measure of <i>how rapidly the effective number of species increases as sampling effort increases</i>. This can be calculated as Δ<i>y</i>/Δ<i>x</i>, which is the mean slope of a species diversity accumulation curve over a defined interval of sampling effort Δ<i>x</i>. Here Δ<i>y</i>=Δ<sup><i>q</i></sup><i>D</i> is the difference in mean observed species diversity corresponding to Δ<i>x</i>. If the <i>x</i> axis is log(sampling effort), then
 <div class=inline-equation id=di-fml-37><span class="inline-equation__construct figure"><img class=figure__image src=images/69.gif alt=image title=image></span><span class=inline-equation__label></span></div> 
 </div>
 
 <div class=paragraph-displayedItem-text-type>where <i>n</i> is the number of sampling units at the subset size of interest. Equally well, the <i>x</i> axis can be linear, in which case
 <div class=inline-equation id=di-fml-38><span class="inline-equation__construct figure"><img class=figure__image src=images/70.gif alt=image title=image></span><span class=inline-equation__label></span></div> 
 </div>
 
 <p>Both the interpretation of the result and its numerical value will be very different depending on whether the <i>x</i> axis is log‐transformed or not. In both cases, the value of Δ<i>x</i> is dependent on how the sampling units have been defined and how many of them there are.
 </p>
 
 <p> <span><a href=#b98 id=#b98R class="bibLink tab-link" data-tab=pane-pcw-references>Scheiner (2003)</a></span> classified species‐area curves (which are species accumulation curves where sampling effort is defined by surface area) into four different types according to the geographical distribution of sampling units. Type II curves are obtained when the study region is divided into a grid, and each grid cell is a sampling unit (lag=0). Type III curves are obtained when sampling units are spread over the entire extent of the study region but do not cover it all, so that adjacent sampling units are not contiguous (lag&gt;0). In practice, Type III curves are prevalent among studies that obtain their data by doing field work, whereas Type II curves are mostly used in macroecological studies that obtain their data from museum records or distribution maps. Scheiner's concepts can be made more general by allowing other definitions of sampling effort than surface area; let us here use the number of individuals. Then lag=0 means that all individuals of the target group that actually exist within the extent of the study are included in one of the sampling units, and lag &gt;0 means that a part of the individuals remain unobserved.
 </p>
 
 <p>Both Type II and Type III curves can be either spatially explicit (Type A) or spatially non‐explicit (Type B; <span><a href=#b98 id=#b98R class="bibLink tab-link" data-tab=pane-pcw-references>Scheiner 2003</a></span>). This distinction is what concerns us now, because it determines whether the <i>y</i> axis quantifies total species diversity in the focal set of <i>n</i> sampling units (γ) or mean species diversity within sampling units (α<sub>t</sub>). This leads to two kinds of effective species accumulation rate.
 </p>
 
 <section class=article-section__sub-content id=ss18>
 
 <h4 class="article-section__sub-title section3" id=ss18-title> 4.1.A Gamma diversity accumulation rate Δγ/(Δn SU) or Δγ/Δlog(n SU)</h4>
 
 <p>Spatially non‐explicit (Type B) species accumulation curves trace <i>the increase in mean observed gamma diversity as an increasing number of sampling units is taken into account</i>. Such curves can be produced by drawing sampling units at random from the entire pool of available sampling units, as is done in software such as EstimateS (<span><a href=#b23 id=#b23R class="bibLink tab-link" data-tab=pane-pcw-references>Colwell 2009</a></span>). The mean effective number of species for subset size <i>n</i> SU is then determined (
<img alt="inline image" class=section_image src=images/71.gif>) and an average species accumulation curve drawn. These curves have also been called collector's curves, rarefaction curves and randomised or smoothed species accumulation curves (<span><a href=#b24 id=#b24R class="bibLink tab-link" data-tab=pane-pcw-references>Colwell and Coddington 1994</a></span>, <span><a href=#b39 id=#b39R class="bibLink tab-link" data-tab=pane-pcw-references>Gotelli and Colwell 2001</a></span>, <span><a href=#b26 id=#b26R class="bibLink tab-link" data-tab=pane-pcw-references>Colwell et al. 2004</a></span>). Figure 4A shows an example based on species richness (<i>q</i>=0); examples of (unsmoothed) gamma diversity accumulation curves for different values of <i>q</i> can be found in <span><a href=#b69 id=#b69R class="bibLink tab-link" data-tab=pane-pcw-references>Magurran (1988</a></span>, p. 53).
 </p>
 
 <p>Average gamma diversity accumulation rate can be determined between any two subset sizes as the slope of the line connecting the corresponding points along the smoothed curve (<a href=#f4 class=scrollableLink>Fig. 4</a>A). It is noteworthy that lag decreases with increasing <i>n</i>, because larger <i>n</i> means that more sampling units are drawn from the same region. In contrast, grain and extent are constant, because the definition of a sampling unit and the total pool of sampling units remain unchanged.
 </p>
 
 <section class=article-section__inline-figure>
 <figure class=figure id=f4><a target=_blank href=https://onlinelibrary.wiley.com/cms/asset/d7dc5ff2-151b-484a-b986-eef2fea6638a/ecog_6148_f4.gif>
 <picture>
 <img class=figure__image src=images/72.gif data-lg-src=/cms/asset/d7dc5ff2-151b-484a-b986-eef2fea6638a/ecog_6148_f4.gif alt=image title=image srcset sizes></picture></a><figcaption class=figure__caption>
 <div class=figure__caption__header><strong class=figure__title>Figure 4</strong><div class=figure-extra><a href=# class=open-figure-link>Open in figure viewer</a><a href="https://onlinelibrary.wiley.com/action/downloadFigures?id=f4&amp;doi=10.1111%2Fj.1600-0587.2009.06148.x" class=ppt-figure-link><i aria-hidden=true class=icon-Icon_Download></i><span>PowerPoint</span></a></div>
 </div>
 <div class="figure__caption figure__caption-text">
 <p>Species diversity accumulation curves. (A) In the smoothed spatially non‐explicit curves, <i>n</i> sampling units are drawn at random from the full set of sampling units. For each subset size <i>n</i> SU, the curve shows mean gamma diversity 
<span><img alt="inline image" class=section_image src=images/73.gif></span>, which equals the gamma diversity of the entire dataset when <i>n</i>=<i>N</i>, and the expected species diversity in a single sampling unit when <i>n</i>=1. Mean Δγ<sub>(1;<i>n</i>)</sub> equals mean absolute species turnover in subsets of <i>n</i> sampling units β<sub>At(<i>n</i>,1)</sub> (scale at left). When <i>n</i>=<i>N</i>, Whittaker's species turnover can be obtained as Δγ<sub>(1;<i>N</i>)</sub>/α<sub>t(<i>N</i>,1)</sub> (scale at right). (B) In the spatially explicit curves, <i>m</i> spatially adjacent sampling units are pooled to form one new sampling unit. For each sampling unit size <i>m</i>, the curve shows the expected species diversity of a single sampling unit α<sub>t(<i>N</i>,<i>m</i>)</sub> in the entire dataset. The vertical difference Δα<sub>t(<i>N</i>,<i>m</i>;<i>N</i>,<i>N</i>)</sub> equals absolute species turnover β<sub>At(<i>N</i>,<i>m</i>)</sub> (scale at left). Proportional species turnover β<sub>Pt(<i>N</i>,<i>m</i>)</sub> can be obtained for any grain <i>m</i> as Δα<sub>t(<i>N</i>,<i>m</i>;<i>N</i>,<i>N</i>)</sub>/γ<sub>(<i>N</i>)</sub> (scale at right).</p>
 
 </div>
 </figcaption>
 </figure>
 </section>
 
 </section>
 
 <section class=article-section__sub-content id=ss19>
 
 <h4 class="article-section__sub-title section3" id=ss19-title> 4.1.B Alpha diversity accumulation rate Δα<sub>t</sub>/(Δm SU) or Δα<sub>t</sub>/Δlog(m SU)
 </h4>
 
 <p>In the spatially explicit curves (Type A), lag and extent remain constant but the grain of the sampling is changed. This is achieved by pooling <i>m</i> spatially adjacent original sampling units to form one larger sampling unit before quantifying alpha diversity α<sub>t(<i>N,m</i>)</sub> where <i>N</i> is the total number of sampling units in the dataset. Alpha diversity equals the generalised mean with exponent 1−<i>q</i> of the within‐sampling unit species diversities <sup><i>q</i></sup><i>D</i><sub>γ<i>j</i></sub>, which corresponds to the arithmetic mean when <i>q</i>=0 (<i>Diversity in relation to two classifications</i> in <span><a href=#b110 id=#b110R class="bibLink tab-link" data-tab=pane-pcw-references>Tuomisto 2010</a></span>). All <i>N</i> available original sampling units are used, but they are allocated to <i>N/m</i> new sampling units of <i>m</i> times the original size, such that each different value of <i>m</i> corresponds to a different grain.
 </p>
 
 <p>The alpha diversity accumulation curve traces <i>the increase in the expected species diversity within a sampling unit as the sampling unit grows larger</i> (<a href=#f4 class=scrollableLink>Fig. 4</a>B; see also <span><a href=#b66 id=#b66R class="bibLink tab-link" data-tab=pane-pcw-references>Lira‐Noriega et al. 2007</a></span>). Average alpha diversity accumulation rate can be determined between any two sampling unit sizes as the slope of the line connecting the corresponding points along the curve.
 </p>
 
 </section>
 
 <section class=article-section__sub-content id=ss20>
 
 <h4 class="article-section__sub-title section3" id=ss20-title> Interpretation of diversity accumulation rates</h4>
 
 <p>Gamma diversity accumulation curves (spatially non‐explicit or Type B curves) visualise the effect of across‐region sampling intensity, i.e. how inventorying more sampling units of a fixed sampling effort increases observed total species diversity. In contrast, alpha diversity accumulation curves (spatially explicit or Type A curves) visualise the effect of within‐site sampling intensity, i.e. how changing sampling unit size by allocating a larger part of the total sampling effort to a single sampling unit increases observed species diversity within sampling units. Note that this formulation of alpha diversity corresponds to α<sub>t</sub> rather than true alpha diversity α<sub>d</sub>; this is necessary if a beta component of diversity is to be read off the diversity accumulation curves.
 </p>
 
 <p>The degree to which the shapes of the spatially explicit and non‐explicit curves differ depends on the spatial structure of compositional heterogeneity within the study region. If species composition changes gradually from one sampling unit to the next, alpha diversity accumulation rate is small between small values of <i>m</i> and increases towards larger <i>m</i>, because all neighbouring sampling units are compositionally similar. As the degree of random variation and/or local‐scale patchiness increases, nearest neighbours can also be more different. Then the alpha diversity accumulation curve rises steeply at small values of <i>m</i> and levels off as <i>m</i> increases. If species are distributed at random (at a given grain), contiguous sampling units are no more similar than non‐contiguous sampling units, and the alpha diversity accumulation curve converges on the gamma diversity accumulation curve (at that grain). The degree of difference between the two types of curve is therefore an indicator of the degree of spatial aggregation in the data (see also <span><a href=#b44 id=#b44R class="bibLink tab-link" data-tab=pane-pcw-references>He and Legendre 2002</a></span>, <span><a href=#b83 id=#b83R class="bibLink tab-link" data-tab=pane-pcw-references>Olszewski 2004</a></span>). This interpretation assumes that mean diversity is calculated in the same way in both curves, i.e. that the generalised mean with exponent 1−<i>q</i> is used.
 </p>
 
 <p>These relationships are paralleled in the contrast between individual‐based and sample‐based rarefaction curves (<span><a href=#b24 id=#b24R class="bibLink tab-link" data-tab=pane-pcw-references>Colwell and Coddington 1994</a></span>, <span><a href=#b39 id=#b39R class="bibLink tab-link" data-tab=pane-pcw-references>Gotelli and Colwell 2001</a></span>, <span><a href=#b26 id=#b26R class="bibLink tab-link" data-tab=pane-pcw-references>Colwell et al. 2004</a></span>). An individual‐based rarefaction curve can be thought of as a gamma diversity accumulation curve based on sampling units that contain exactly one individual each. A sample‐based rarefaction curve is obtained when adjacent original sampling units (individuals) are pooled to form new, larger sampling units (as in the alpha diversity accumulation curve), and randomly selected subsets of these, at the desired grain, are used to construct a gamma diversity accumulation curve.
 </p>
 
 <p>Species diversity accumulation rate has been used or promoted as a measure of “beta diversity” (<span><a href=#b95 id=#b95R class="bibLink tab-link" data-tab=pane-pcw-references>Ricotta et al. 2002</a></span>, <span><a href=#b98 id=#b98R class="bibLink tab-link" data-tab=pane-pcw-references>Scheiner 2003, 2004</a></span>, <span><a href=#b84 id=#b84R class="bibLink tab-link" data-tab=pane-pcw-references>Passy and Blanchet 2007</a></span>). However, the slope of a species accumulation curve does not quantify compositional heterogeneity but the rate of change in gamma diversity with increasing number of sampling units (Δγ/Δ<i>x</i>) or the rate of change in alpha diversity with increasing sampling unit size (Δα<sub>t</sub>/Δ<i>x</i>). Different kinds of species turnover (Δ<i>c</i>) can be deduced from a species diversity accumulation curve, but all of these are represented by a difference in the <i>y</i> axis value between two points along the curve, not the slope of the curve.
 </p>
 
 <div class=paragraph-displayedItem-text-type>The smoothed gamma diversity accumulation curve is constructed by varying subset size, so the difference in <i>y</i> axis value between points that correspond to subset sizes <i>n</i> SU and <i>m</i> SU (with <i>n</i>&gt;<i>m</i>) equals 
<img alt="inline image" class=section_image src=images/74.gif>. Provided the generalised mean with exponent 1−<i>q</i> is used,
<img alt="inline image" class=section_image src=images/75.gif> equals 
<img alt="inline image" class=section_image src=images/76.gif>, so the vertical distance between the point at <i>n</i> and the point at <i>m</i>=1 becomes
 <div class=inline-equation id=di-fml-43><span class="inline-equation__construct figure"><img class=figure__image src=images/77.gif alt=image title=image></span><span class=inline-equation__label></span></div> 
 </div>
 
 <div class=paragraph-displayedItem-text-type>i.e. mean absolute effective species turnover in all subsets of <i>n</i> sampling units (<a href=#f4 class=scrollableLink>Fig. 4</a>A; see also <span><a href=#b29 id=#b29R class="bibLink tab-link" data-tab=pane-pcw-references>Crist and Veech 2006</a></span>, <span><a href=#b37 id=#b37R class="bibLink tab-link" data-tab=pane-pcw-references>Gardezi and Gonzalez 2008</a></span>). If the <i>y</i> axis is divided by α<sub>t(<i>N</i>,1)</sub>, the difference between the <i>y</i> values of the lowest and highest points of the curve becomes
 <div class=inline-equation id=di-fml-44><span class="inline-equation__construct figure"><img class=figure__image src=images/78.gif alt=image title=image></span><span class=inline-equation__label></span></div> 
 </div>
 
 <p>i.e. Whittaker's effective species turnover in the entire dataset. Unlike mean absolute effective species turnover, mean Whittaker's effective species turnover cannot always be accurately read off the gamma diversity accumulation curve when <i>n</i>&lt;<i>N</i>. This is because the mean of ratios, such as 
<img alt="inline image" class=section_image src=images/79.gif>, does not equal the ratio of the corresponding mean numerators (
<img alt="inline image" class=section_image src=images/80.gif>) and mean denominators (
<img alt="inline image" class=section_image src=images/81.gif>) unless either the geometric mean is used (i.e. <i>q</i>=1) or all denominators are equal. In real datasets, α<sub>t(<i>n</i>,1)</sub> values may vary among the different subsets of <i>n</i> sampling units, especially when <i>n</i> is small.
 </p>
 
 <div class=paragraph-displayedItem-text-type>The alpha diversity accumulation curve shows the actual increase in the mean within‐sampling unit species diversity as grain increases. The difference in <i>y</i> axis value between points that correspond to sampling unit sizes <i>n</i> SU and <i>m</i> SU (with <i>n</i>&gt;<i>m</i>) equals α<sub>t(<i>N</i>,<i>n</i>)</sub>−α<sub>t(<i>N</i>,<i>m</i>)</sub> (<a href=#f4 class=scrollableLink>Fig. 4</a>B). When the entire dataset forms a single sampling unit, α<sub>t(<i>N</i>,<i>N</i>)</sub> equals γ<sub>(<i>N</i>)</sub>. The difference in <i>y</i> axis value between this point and another point along the curve α<sub>t(<i>N</i>,<i>m</i>)</sub> equals
 <div class=inline-equation id=di-fml-48><span class="inline-equation__construct figure"><img class=figure__image src=images/82.gif alt=image title=image></span><span class=inline-equation__label></span></div> 
 </div>
 
 <div class=paragraph-displayedItem-text-type>i.e. absolute effective species turnover in the entire dataset when sampling units are <i>m</i> times as large as originally. Interpreting α<sub>t(<i>N</i>,<i>n</i>)</sub> as 
<img alt="inline image" class=section_image src=images/83.gif> allows quantifying mean absolute effective species turnover for any combination of extent and grain by
 <div class=inline-equation id=di-fml-50><span class="inline-equation__construct figure"><img class=figure__image src=images/84.gif alt=image title=image></span><span class=inline-equation__label></span></div> 
 </div>
 
 <div class=paragraph-displayedItem-text-type>If the <i>y</i> axis is divided by α<sub>t(<i>N</i>,<i>N</i>)</sub>=γ<sub>(<i>N</i>)</sub>, the vertical difference between the point <i>n</i>=<i>N</i> and another point <i>m</i> becomes
 <div class=inline-equation id=di-fml-51><span class="inline-equation__construct figure"><img class=figure__image src=images/85.gif alt=image title=image></span><span class=inline-equation__label></span></div> 
 </div>
 
 <p>i.e. proportional effective species turnover in the entire dataset at grain <i>m</i>. Unlike mean β<sub>At(<i>n,m</i>)</sub>, mean β<sub>Pt(<i>n,m</i>)</sub> cannot be read off the curve when <i>n</i>&lt;<i>N</i> because a different denominator (
<img alt="inline image" class=section_image src=images/86.gif>) needs to be used for each different value of <i>n</i>.
 </p>
 
 </section>
 
 </section>
 
 <section class=article-section__sub-content id=ss21>
 
 <h3 class="article-section__sub-title section2" id=ss21-title> 4.2 Species entropy accumulation rate Δlog(<sup><i>q</i></sup><i>D</i>)/Δ<i>x</i></h3>
 
 <p>If the species diversity accumulation curves in <a href=#f4 class=scrollableLink>Fig. 4</a> are modified such that the <i>y</i> axis shows species entropy instead of species diversity, the species entropy accumulation curve is obtained. The most commonly used entropy in this context has been the Rényi entropy <sup><i>q</i></sup><i>H</i>=log(<sup><i>q</i></sup><i>D</i>), because species accumulation curves are often displayed in a log‐log plot. The slope of this curve is the species entropy accumulation rate, which quantifies <i>how rapidly species entropy increases as log(sampling effort) increases</i>. Two kinds of species entropy accumulation rate can be derived analogously to the two kinds of species diversity accumulation rate.
 </p>
 
 <section class=article-section__sub-content id=ss22>
 
 <h4 class="article-section__sub-title section3" id=ss22-title> 4.2.A Gamma entropy accumulation rate Δlog(γ)/Δlog(n SU)</h4>
 
 <p>A smoothed spatially non‐explicit entropy accumulation curve can be obtained by log‐transforming the <i>y</i> values of the smoothed gamma diversity accumulation curve of <a href=#f4 class=scrollableLink>Fig. 4</a>A. The curve traces the increase in gamma entropy as the number of sampling units that are taken into account increases. The average gamma entropy accumulation rate at a specified interval of sampling effort equals <i>the average slope of the curve at that interval</i>.
 </p>
 
 </section>
 
 <section class=article-section__sub-content id=ss23>
 
 <h4 class="article-section__sub-title section3" id=ss23-title> 4.2.B Alpha entropy accumulation rate Δlog(α<sub>t</sub>)/Δlog(m SU)
 </h4>
 
 <p>A spatially explicit entropy accumulation curve is derived by log‐transforming the <i>y</i> axis of <a href=#f4 class=scrollableLink>Fig. 4</a>B. It traces the increase in alpha entropy as the sampling units grow larger (with total sampling effort remaining constant). The average alpha entropy accumulation rate at a specified interval of sampling unit size equals <i>the slope of the curve at that interval</i>.
 </p>
 
 </section>
 
 <section class=article-section__sub-content id=ss24>
 
 <h4 class="article-section__sub-title section3" id=ss24-title> Interpretation of entropy accumulation rates</h4>
 
 <p>Species entropy accumulation rate has been used or promoted as a measure of “beta diversity” (<span><a href=#b15 id=#b15R class="bibLink tab-link" data-tab=pane-pcw-references>Caswell and Cohen 1993</a></span>, <span><a href=#b96 id=#b96R class="bibLink tab-link" data-tab=pane-pcw-references>Rosenzweig 1995</a></span>, <span><a href=#b115 id=#b115R class="bibLink tab-link" data-tab=pane-pcw-references>Vazquez and Givnish 1998</a></span>, <span><a href=#b98 id=#b98R class="bibLink tab-link" data-tab=pane-pcw-references>Scheiner 2003, 2004</a></span>, <span><a href=#b105 id=#b105R class="bibLink tab-link" data-tab=pane-pcw-references>Smith 2008</a></span>), but the approach has also been criticised (<span><a href=#b28 id=#b28R class="bibLink tab-link" data-tab=pane-pcw-references>Connor and McCoy 1979</a></span>, <span><a href=#b123 id=#b123R class="bibLink tab-link" data-tab=pane-pcw-references>Wilson and Shmida 1984</a></span>). Indeed, the slope of an entropy accumulation curve does not quantify the amount of compositional heterogeneity in the data, and the two need not even be correlated. However, the difference in the <i>y</i> axis value between specific points along the curve can be interpreted in terms of the regional‐to‐local diversity ratio β<sub>Mt</sub>, provided that the generalised mean with exponent 1−<i>q</i> was used to calculate mean species diversity before log‐transformation. It seems that <span><a href=#b68 id=#b68R class="bibLink tab-link" data-tab=pane-pcw-references>MacArthur (1965</a></span>, p. 528) interpreted entropy accumulation curves in this way, rather than (as suggested by several authors) equating the slope of the curve with “beta diversity”.
 </p>
 
 <div class=paragraph-displayedItem-text-type>In the gamma entropy accumulation curve, the difference in <i>y</i> axis value between points that correspond to subset sizes <i>n</i> SU and <i>m</i> SU (with <i>n</i>&gt;<i>m</i>) equals
 <div class=inline-equation id=di-fml-53><span class="inline-equation__construct figure"><img class=figure__image src=images/87.gif alt=image title=image></span><span class=inline-equation__label></span></div> 
 </div>
 
 <div class=paragraph-displayedItem-text-type>When <i>n</i>=<i>N</i> and <i>m</i>=1, this equals
 <div class=inline-equation id=di-fml-54><span class="inline-equation__construct figure"><img class=figure__image src=images/88.gif alt=image title=image></span><span class=inline-equation__label></span></div> 
 </div>
 
 <p>i.e. the logarithm of regional‐to‐local diversity ratio in the entire dataset. As with gamma diversity accumulation curves (Section 4.1), this interpretation is not accurate with other values of <i>m</i> unless <i>q</i>=1. If log(β<sub>Mt(<i>N</i>,1)</sub>) is used as a measure of “beta diversity” instead of using β<sub>Mt(<i>N</i>,1)</sub> itself, then “beta diversity” is equated with regional Rényi entropy excess, which gives regional Shannon entropy excess 
<img alt="inline image" class=section_image src=images/89.gif> as a special case when <i>q</i>=1. This is not a beta component of true diversity but of a raw diversity index value (Sections 6 and ∞ in <span><a href=#b110 id=#b110R class="bibLink tab-link" data-tab=pane-pcw-references>Tuomisto 2010</a></span>).
 </p>
 
 <div class=paragraph-displayedItem-text-type>In the alpha entropy accumulation curve, the difference in <i>y</i> value between points that correspond to sampling unit sizes <i>n</i> SU and <i>m</i> SU (with <i>n</i>&gt;<i>m</i>) equals
 <div class=inline-equation id=di-fml-56><span class="inline-equation__construct figure"><img class=figure__image src=images/90.gif alt=image title=image></span><span class=inline-equation__label></span></div> 
 </div>
 
 <div class=paragraph-displayedItem-text-type>When the larger sampling unit contains the entire dataset (<i>n</i>=<i>N</i>), this can be interpreted as
 <div class=inline-equation id=di-fml-57><span class="inline-equation__construct figure"><img class=figure__image src=images/91.gif alt=image title=image></span><span class=inline-equation__label></span></div> 
 </div>
 
 <div class=paragraph-displayedItem-text-type>or the logarithm of regional‐to‐local diversity ratio in the entire dataset at grain <i>m</i>. Although
 <div class=inline-equation id=di-fml-58><span class="inline-equation__construct figure"><img class=figure__image src=images/92.gif alt=image title=image></span><span class=inline-equation__label></span></div> 
 </div>
 
 <p>has been used as a measure of “beta diversity” (<span><a href=#b3 id=#b3R class="bibLink tab-link" data-tab=pane-pcw-references>Arita and Rodriguez 2002</a></span>), it does not equal mean γ<sub>(<i>n</i>)</sub>/α<sub>t(<i>n,m</i>)</sub>=mean β<sub>Mt(<i>n</i>,<i>m</i>)</sub> when <i>n</i>&lt;<i>N</i> unless <i>q</i>=1 or the alpha diversities of all subsets with <i>n</i> sampling units are identical.
 </p>
 
 </section>
 
 </section>
 
 <section class=article-section__sub-content id=ss25>
 
 <h3 class="article-section__sub-title section2" id=ss25-title> 4.3 Rate of change in a beta component of diversity with increasing sampling effort Δβ<sub>M</sub>/Δ<i>x</i> or ΔΔ<i>c</i>/Δ<i>x</i></h3>
 
 <section class=article-section__sub-content id=ss26>
 
 <h4 class="article-section__sub-title section3" id=ss26-title> 4.3.A Accumulation rate of a beta component of diversity</h4>
 
 <div class=paragraph-displayedItem-text-type> <a href=#f5 class=scrollableLink>Figure 5</a>A shows the accumulation of arithmetic mean true beta diversity β<sub>Md</sub> with increasing subset size (<i>n</i> SU) for which α<sub>d(<i>n</i>,1)</sub> and γ<sub>(<i>n</i>)</sub> are quantified in a spatially non‐explicit setting. Mean true beta diversity accumulation rate between any two points along the curve equals the slope of the line connecting them. The slope between points corresponding to a single sampling unit and a set of <i>n</i> sampling units equals
 <div class=inline-equation id=di-fml-59><span class="inline-equation__construct figure"><img class=figure__image src=images/93.gif alt=image title=image></span><span class=inline-equation__label></span></div> 
 </div>
 
 <section class=article-section__inline-figure>
 <figure class=figure id=f5><a target=_blank href=https://onlinelibrary.wiley.com/cms/asset/e36097df-b3c8-49cb-8774-fa155f7d9862/ecog_6148_f5.gif>
 <picture>
 <img class=figure__image src=images/94.gif data-lg-src=/cms/asset/e36097df-b3c8-49cb-8774-fa155f7d9862/ecog_6148_f5.gif alt=image title=image srcset sizes></picture></a><figcaption class=figure__caption>
 <div class=figure__caption__header><strong class=figure__title>Figure 5</strong><div class=figure-extra><a href=# class=open-figure-link>Open in figure viewer</a><a href="https://onlinelibrary.wiley.com/action/downloadFigures?id=f5&amp;doi=10.1111%2Fj.1600-0587.2009.06148.x" class=ppt-figure-link><i aria-hidden=true class=icon-Icon_Download></i><span>PowerPoint</span></a></div>
 </div>
 <div class="figure__caption figure__caption-text">
 <p>(A) The accumulation of mean true beta diversity with increasing <i>n</i> in a spatially non‐explicit setting, in which a subset of <i>n</i> sampling units is randomly drawn from the entire dataset. The slopes of the lines quantify the average rate of change in mean true beta diversity as number of sampling units inventoried increases over a specified interval. (B) The decrease in true beta diversity with increasing <i>m</i> in a spatially explicit setting, in which <i>m</i> sampling units are pooled to form a single new sampling unit before calculating true alpha diversity. The squares represent a dataset with a strong spatial gradient in species composition, and the diamonds a dataset in which the species are distributed at random. The slopes of the lines quantify the average rate of change in true beta diversity as size of the sampling units increases over a specified interval. (C) The accumulation of mean proportional effective species turnover with increasing <i>n</i> in a spatially non‐explicit setting, in which a subset of <i>n</i> sampling units is randomly drawn from the entire dataset. The slopes of the lines quantify the average rate of change in proportional effective species turnover as an increasing proportion of the sampling units is taken into account over a specified interval.</p>
 
 </div>
 </figcaption>
 </figure>
 </section>
 
 <p>A numerically identical result (but with measurement unit 1/SU instead of CU/SU; <a href=#t1 class=scrollableLink>Table 1</a>) is obtained if the <i>y</i> axis shows regional‐to‐local diversity β<sub>Mt</sub> or Whittaker's effective species turnover β<sub>Mt−1</sub>. When <i>n</i>=<i>N</i>, the unitless value of the slope equals the one‐complement of the <sup><i>q</i></sup><i>C</i><sub>S<i>n</i></sub> similarity measure for the entire dataset (generalisation of the Sørensen index; Section 4 in <span><a href=#b110 id=#b110R class="bibLink tab-link" data-tab=pane-pcw-references>Tuomisto 2010</a></span>). When <i>n</i> &lt;<i>N</i>, mean 1−<sup><i>q</i></sup><i>C</i><sub>S<i>n</i></sub> for all subsets with <i>n</i> sampling units is obtained. For presence‐absence data, 1−<sup>0</sup><i>C</i><sub>S<i>n</i></sub> equals the beta‐1 measure of <span><a href=#b43 id=#b43R class="bibLink tab-link" data-tab=pane-pcw-references>Harrison et al. (1992)</a></span>. Beta‐1 and 1−<sup><i>q</i></sup><i>C</i><sub>S<i>n</i></sub> can therefore be used to quantify <i>the average rate at which compositional heterogeneity and Whittaker's (effective) species turnover increase as sampling effort increases from 1 to</i> N <i>sampling units</i>. Obviously, Δβ<sub>Md</sub>/(Δ<i>n</i> SU) does not quantify the same phenomenon as β<sub>Md</sub> itself, and the same is true of Δβ<sub>Mt</sub>/(Δ<i>n</i> SU) and Δβ<sub>Mt−1</sub>/(Δ<i>n</i> SU) (the latter can be referred to, less specifically, by ΔΔ<i>c</i>/Δ<i>x</i>).
 </p>
 
 <p>Beta‐1 has been justified by arguing that when beta diversity is constrained to a fixed range, its values can be compared across regions that differ in <i>N</i> (<span><a href=#b43 id=#b43R class="bibLink tab-link" data-tab=pane-pcw-references>Harrison et al. 1992</a></span>, <span><a href=#b12 id=#b12R class="bibLink tab-link" data-tab=pane-pcw-references>Blackburn and Gaston 1996</a></span>). However, beta‐1 is no less affected by <i>N</i> than β<sub>Md</sub> is. As the number of sampling units in a sample increases, each additional sampling unit contributes progressively fewer new species to the sample (unless all sampling units were identical to start with, or the region of interest has an effectively infinite species diversity). The amount of increase in mean β<sub>Md(<i>n</i>,1)</sub> per unit increase in sampling effort therefore necessarily decreases with increasing <i>n</i> within any one dataset (Section 4 in <span><a href=#b110 id=#b110R class="bibLink tab-link" data-tab=pane-pcw-references>Tuomisto 2010</a></span>). In general, beta‐1 values can therefore be expected to be smaller in datasets with large <i>N</i> than in datasets with small <i>N</i>.
 </p>
 
 </section>
 
 <section class=article-section__sub-content id=ss27>
 
 <h4 class="article-section__sub-title section3" id=ss27-title> 4.3.B Decay rate of a beta component of diversity</h4>
 
 <div class=paragraph-displayedItem-text-type> <a href=#f5 class=scrollableLink>Figure 5</a>B shows the decrease in true beta diversity β<sub>Md(<i>N</i>,<i>m</i>)</sub>=γ<sub>(<i>N</i>)</sub>/α<sub>d(<i>N</i>,<i>m</i>)</sub> with increasing <i>m</i>, where <i>m</i> is the number of sampling units fused to form one new sampling unit in a spatially explicit setting (see also <span><a href=#b66 id=#b66R class="bibLink tab-link" data-tab=pane-pcw-references>Lira‐Noriega et al. 2007</a></span>). Each point along the curve is a single value rather than a mean, because both γ<sub>(<i>N</i>)</sub> and α<sub>d(<i>N</i>,<i>m</i>)</sub> are based on the entire dataset. Increasing <i>m</i> increases α<sub>d(<i>N</i>,<i>m</i>)</sub> but has no effect on γ<sub>(<i>N</i>)</sub>, so the curve is monotonically decreasing. This is in contrast with the spatially non‐explicit curve (<a href=#f5 class=scrollableLink>Fig. 5</a>A), which is monotonically increasing because γ<sub>(<i>n</i>)</sub> increases with <i>n</i> but α<sub>d(<i>n</i>,1)</sub> remains approximately constant. The slope between the point corresponding to the original sampling units and sampling unit size <i>m</i> SU equals
 <div class=inline-equation id=di-fml-60><span class="inline-equation__construct figure"><img class=figure__image src=images/95.gif alt=image title=image></span><span class=inline-equation__label></span></div> 
 </div>
 
 <p>A numerically identical result (although with measurement unit 1/SU) is obtained if the <i>y</i> axis shows regional‐to‐local diversity β<sub>Mt</sub> or Whittaker's effective species turnover β<sub>Mt−1</sub>. This slope is ecologically interpretated as <i>the rate at which compositional heterogeneity and Whittaker's effective species turnover decrease as sampling unit size increases from 1 to</i> m <i>times the size of the original sampling units</i>. When <i>m</i>=<i>N</i>, β<sub>Mt(<i>N</i>,<i>m</i>)</sub> equals unity, and the above equation gives a result with the same absolute numeric value as beta‐1 and 1−<sup><i>q</i></sup><i>C</i><sub>S<i>N</i></sub> for the entire dataset (Section 4.3.A, above), but opposite sign. Obviously, beta diversity decay rate Δβ<sub>Md</sub>/(Δ<i>m</i> SU) does not quantify the same phenomenon as β<sub>Md</sub> itself, and the same is true of Δβ<sub>Mt</sub>/(Δ<i>m</i> SU) and Δβ<sub>Mt−1</sub>/(Δ<i>m</i> SU) (the latter can be referred to, less specifically, by ΔΔ<i>c</i>/Δ<i>x</i>).
 </p>
 
 <p>The spatially non‐explicit beta component accumulation curve is not affected by spatial structure in the compositional data, but the spatially explicit beta component decay curve is. The initial decrease in the beta component (when sampling units are still small) is more rapid when species are randomly rather than patchily distributed (<a href=#f5 class=scrollableLink>Fig. 5</a>B), although the mean rate of change between the extremes is the same as long as the same sampling unit definition is used at <i>m</i>=1.
 </p>
 
 </section>
 
 <section class=article-section__sub-content id=ss28>
 
 <h4 class="article-section__sub-title section3" id=ss28-title> 4.3.C Proportional species turnover accumulation rate</h4>
 
 <div class=paragraph-displayedItem-text-type> <a href=#f5 class=scrollableLink>Figure 5</a>C shows the increase in the arithmetic mean of proportional species turnover β<sub>Pt(<i>n</i>,1)</sub> when an increasing proportion of the available sampling units is taken into account in a spatially non‐explicit setting. The rate of change in mean proportional species turnover between subsets of a single sampling unit and <i>n</i> sampling units is the slope of the line connecting the corresponding points along the curve:
 <div class=inline-equation id=di-fml-61><span class="inline-equation__construct figure"><img class=figure__image src=images/96.gif alt=image title=image></span><span class=inline-equation__label></span></div> 
 </div>
 
 <p>The slope between the lowest and highest points along the curve is obtained when <i>n</i>=<i>N</i>, and it equals the one‐complement of the <sup><i>q</i></sup><i>C</i><sub>J<i>n</i></sub> measure discussed in Section 5 of the first part of the present review (generalisation of the Jaccard index; <span><a href=#b51 id=#b51R class="bibLink tab-link" data-tab=pane-pcw-references>Jost 2006</a></span> andTuomisto 2010). From <a href=#f5 class=scrollableLink>Fig. 5</a>C it can be seen that 1−<sup><i>q</i></sup><i>C</i><sub>J<i>n</i></sub> quantifies <i>the average rate at which proportional species turnover increases as proportional sampling effort increases from one</i> n<i>th part to all of the sampling units in the dataset</i>. This can also be referred to, less specifically, as ΔΔ<i>c</i>/Δ<i>x</i>, which obviously quantifies something different than Δ<i>c</i> (= β<sub>Pt</sub> in this case) itself does.
 </p>
 
 </section>
 
 </section>
 
 <section class=article-section__sub-content id=ss29>
 
 <h3 class="article-section__sub-title section2" id=ss29-title> 4.4 Rate of change in beta entropy Δlog(<sup><i>q</i></sup>β<sub>M</sub>)/Δ<i>x</i></h3>
 
 <div class=paragraph-displayedItem-text-type>If <a href=#f5 class=scrollableLink>Fig. 5</a>A were replotted such that both the <i>x</i> and the <i>y</i> axis were log‐transformed, the <i>y</i> axis would show the beta component of Rényi entropy [=log(<sup><i>q</i></sup>β<sub>Md</sub>)] rather than true beta diversity. The accumulation of arithmetic mean beta Rényi entropy with increasing log(sample size) would then be traced by a curve drawn through the points representing the arithmetic means of the log‐transformed <sup><i>q</i></sup>β<sub>Md(<i>n</i>,1)</sub> values corresponding to each log(<i>n</i>). The slope of a line connecting two points on this curve quantifies <i>the average rate at which mean beta Rényi entropy increases between the two sampling efforts</i>. When sample size increases from 1 SU to <i>n</i> SU, this equals:
 <div class=inline-equation id=di-fml-62><span class="inline-equation__construct figure"><img class=figure__image src=images/97.gif alt=image title=image></span><span class=inline-equation__label></span></div> 
 </div>
 
 <p>Exactly the same slope is obtained irrespective of whether the <i>y</i> axis shows log(<sup><i>q</i></sup>β<sub>Md</sub>) or log(<sup><i>q</i></sup>β<sub>Mt</sub>) values, even in terms of measurement units since these cancel out.
 </p>
 
 <p>When <i>q</i>=1, the slope between the lowest and highest point of the curve equals log(<sup>1</sup>β<sub>M(<i>N</i>,1)</sub>)/log(<i>N</i>). This equals <i>R</i><sub><i>hN</i></sub>, i.e. the Horn index of heterogeneity as generalised to <i>N</i> sampling units (Section 6 of <span><a href=#b110 id=#b110R class="bibLink tab-link" data-tab=pane-pcw-references>Tuomisto 2010</a></span>). When <i>q</i>=1 and <i>n</i>&lt;<i>N</i>, the equation gives the expected <i>R</i><sub><i>hN</i></sub> for a subset of <i>n</i> sampling units. Obviously, the Horn index and more generally Δlog(<sup><i>q</i></sup>β<sub>M</sub>)/Δ<i>x</i> quantifies a different phenomenon than <sup><i>q</i></sup>β<sub>M</sub> itself does.
 </p>
 
 </section>
 
 <section class=article-section__sub-content id=ss30>
 
 <h3 class="article-section__sub-title section2" id=ss30-title> 4.5 Species diversity or entropy accumulation rate with alpha and gamma diversities based on different data</h3>
 
 <p>In Sections 4.1 and 4.2, vertical differences between specific points in Scheiner's (2003) Type II and Type III species diversity or entropy accumulation curves were interpreted in terms of effective species turnover or regional‐to‐local diversity ratio. Such interpretations are possible when the same set of sampling units is used in all cases, because then <img alt="inline image" class=section_image src=images/98.gif> (for the spatially non‐explicit curves) and α<sub>t(<i>N</i>,<i>N</i>)</sub>=γ<sub>(<i>N</i>)</sub> (for the spatially explicit curves). This is not the case with Scheiner's (2003) Type I curves. These quantify species diversity or entropy accumulation in a set of nested sampling units, where each grain is represented by a single sampling unit, whose size also defines the extent of the study region at that grain. Type I curves therefore quantify simultaneously the increase in total species richness γ<sub>(<i>N</i>)</sub> as extent becomes larger, and the increase in the species richness of a focal sampling unit γ<sub>F</sub> as its size increases (these curves are typically based on presence‐absence data obtained from published species lists). If the grain and extent are decoupled, it becomes possible to quantify γ<sub>(<i>N</i>)</sub>−γ<sub>F</sub> or log(γ<sub>(<i>N</i>)</sub>/γ<sub>F</sub>) such that γ<sub>F</sub> is based on a smaller grain and extent than γ<sub>(<i>N</i>)</sub>. However, since these measures are based on the species richness of a focal sampling unit γ<sub>F</sub> rather than on the mean species richness of all sampling units 
<img alt="inline image" class=section_image src=images/99.gif>, they suffer from the same problems as the measures discussed in Section 3.3, above. Although the slopes of neither linear nor log‐transformed Type I curves quantify a beta component of diversity, they have been equated with “beta diversity” (<span><a href=#b64 id=#b64R class="bibLink tab-link" data-tab=pane-pcw-references>Lennon et al. 2001</a></span>, <span><a href=#b106 id=#b106R class="bibLink tab-link" data-tab=pane-pcw-references>Soininen et al. 2007</a></span>).
 </p>
 
 </section>
 
 <section class=article-section__sub-content id=ss31>
 
 <h3 class="article-section__sub-title section2" id=ss31-title> 4.6 Compositional turnover rate or effective species turnover rate Δ<i>c</i><sub>(Δ<i>g</i>)</sub>/Δ<i>g</i></h3>
 
 <p>Compositional turnover rate is <i>the rate at which community composition changes along an explanatory gradient</i>. This can be quantified between any two sampling units in a way that is compatible with the concept of effective species turnover by dividing a pairwise effective species turnover value Δ<i>c</i><sub><i>j,k</i></sub> with the corresponding Δ<i>g</i><sub><i>j,k</i></sub> value (see the level 2 tables D and E in <a href=#f1 class=scrollableLink>Fig. 1</a>). Effective species turnover rate is also approximated by the slope of the regression line in the level 1 scatterplot in <a href=#f1 class=scrollableLink>Fig. 1</a>; average turnover rate can be quantified for the entire observed range of the explanatory gradient, or smaller sections of the gradient can be used (<span><a href=#b78 id=#b78R class="bibLink tab-link" data-tab=pane-pcw-references>Økland 1986</a></span>). The instantaneous rate of compositional turnover can also be estimated using the derivatives of species response functions or of a compositional change function along the gradient of interest (<span><a href=#b13 id=#b13R class="bibLink tab-link" data-tab=pane-pcw-references>Bratton 1975</a></span>, <span><a href=#b122 id=#b122R class="bibLink tab-link" data-tab=pane-pcw-references>Wilson and Mohler 1983</a></span>, <span><a href=#b81 id=#b81R class="bibLink tab-link" data-tab=pane-pcw-references>Oksanen and Tonteri 1995</a></span>). However, if these methods are used, the results may not be compatible with effective species turnover as defined on the basis of alpha and gamma diversity. Compositional turnover rate has been called “beta diversity” (<span><a href=#b13 id=#b13R class="bibLink tab-link" data-tab=pane-pcw-references>Bratton 1975</a></span>, Cody 1975, 1986), but it is actually Δ<i>c</i><sub>(Δ<i>g</i>)</sub>/Δ<i>g</i> and therefore quantifies a different phenomenon than true beta diversity <sup><i>q</i></sup>β<sub>Md</sub> or even the effective species turnover measures Δ<i>c</i>.
 </p>
 
 <p>Before compositional turnover rate can be calculated, the external factor for the <i>x</i> axis has to be defined, which leads to two main variants of compositional turnover rate.
 </p>
 
 <p>a) The explanatory gradient is geographical location, i.e. the rate of compositional turnover is measured at a specific geographical location and in a specific compass direction.</p>
 
 <p>b)The explanatory gradient is an environmental variable (or a surrogate thereof, such as elevation), i.e. the rate of compositional turnover along the gradient is measured at a specific value (or interval) of the environmental variable of interest.</p>
 
 <p>Very different compositional turnover rates can be obtained with the same compositional data depending on which reference gradient is chosen. Compositional turnover rate has been used to identify zones of rapid turnover along an explanatory gradient of interest (such as elevation or ground water depth; <span><a href=#b78 id=#b78R class="bibLink tab-link" data-tab=pane-pcw-references>Økland 1986</a></span>). Pairwise turnover values between sampling units separated by a fixed distance along the gradient can also be plotted against the explanatory gradient itself. The points in such graphs actually represent (Δ<i>c</i><sub><i>j,k</i></sub>, <i>g</i><sub><i>j</i></sub>), but they can (with caution) be interpreted in terms of (Δ<i>c</i><sub><i>j,k</i></sub>/Δ<i>g</i><sub><i>j,k</i></sub>, <i>g</i><sub><i>j</i></sub>) since each Δ<i>c</i><sub><i>j,k</i></sub> corresponds to the same Δ<i>g</i><sub><i>j,k</i></sub> (<span><a href=#b11 id=#b11R class="bibLink tab-link" data-tab=pane-pcw-references>Beals 1969</a></span>, <span><a href=#b13 id=#b13R class="bibLink tab-link" data-tab=pane-pcw-references>Bratton 1975</a></span>, <span><a href=#b123 id=#b123R class="bibLink tab-link" data-tab=pane-pcw-references>Wilson and Shmida 1984</a></span>, <span><a href=#b6 id=#b6R class="bibLink tab-link" data-tab=pane-pcw-references>Bach et al. 2007</a></span>, <span><a href=#b48 id=#b48R class="bibLink tab-link" data-tab=pane-pcw-references>Jankowski et al. 2009</a></span>).
 </p>
 
 </section>
 
 <section class=article-section__sub-content id=ss32>
 
 <h3 class="article-section__sub-title section2" id=ss32-title> 4.7 Rate of change in (the logarithm of the one‐complement of) pairwise compositional turnover with increasing distance along an explanatory gradient (slope of a distance decay regression) ΔΔ<i>c</i><sub>(ΔΔ<i>g</i>)</sub>/ΔΔ<i>g</i> or Δlog(1–Δ<i>c</i>)<sub>(ΔΔ<i>g</i>)</sub>/ΔΔ<i>g</i></h3>
 
 <p>Rate of change in pairwise compositional turnover is the average rate at which sampling units become compositionally more dissimilar as distance between them along the external gradient of interest increases. It can be quantified as the slope of the regression line drawn through the level 2 scatterplot in <a href=#f1 class=scrollableLink>Fig. 1</a>, ΔΔc<sub>(ΔΔg)</sub>/ΔΔg. If the y axis values (Δc<sub>j,k</sub>) in this plot are subtracted from unity to convert them to similarities, a distance decay plot is obtained (<span><a href=#b76 id=#b76R class="bibLink tab-link" data-tab=pane-pcw-references>Nekola and White 1999</a></span>). The slopes of the two plots have the same absolute value but opposite signs. The distance decay plot can also be displayed with a logarithmic y axis, in which case the slope of the regression line equals Δlog(1–Δc)<sub>(ΔΔg)</sub>/ΔΔg (<a href=#f3 class=scrollableLink>Fig. 3</a>). The slopes obtained with linear vs logarithmic <i>y</i> axes quantify different things and their values are therefore not commensurate.
 </p>
 
 <p>Before the slope of a distance decay regression can be calculated, the external factor for the <i>x</i> axis has to be defined. Very different rates of change can be obtained with the same compositional data depending on which reference gradient is chosen. Two main variants can be distinguished.
 </p>
 
 <p>a) The factor of interest is geographical distance, i.e. the rate of change in (the logarithm of the one‐complement of) pairwise compositional turnover is measured as the geographical distance between sampling units increases. An average value can be obtained for all directions if space is assumed isotropic, but it is also possible to differentiate between rate of change in pairwise compositional turnover in different directions, for example north–south vs east–west.</p>
 
 <p>b)The factor of interest is environmental difference (or a surrogate thereof, such as difference in elevation), i.e. the rate of change in (the logarithm of the one‐complement of) pairwise compositional turnover is measured as the environmental dissimilarity between sampling units increases. Environmental difference can be defined using just one environmental variable or several at a time.</p>
 
 <p>In one of his examples, <span><a href=#b117 id=#b117R class="bibLink tab-link" data-tab=pane-pcw-references>Whittaker (1960)</a></span> introduced both variants, since he used geographical distance as a surrogate for environmental difference. However, he did not designate the rate of change as a measure of “beta diversity”, but rather used it as an intermediate step in estimating the number of half‐change units (Section 2.5, above). <span><a href=#b86 id=#b86R class="bibLink tab-link" data-tab=pane-pcw-references>Pielou (1975</a></span>, p. 101) reinterpreted “beta diversity” such that it was no longer the number of half‐change units ΔΔ<i>g</i><sub>(Δlog(1–Δ<i>c</i>))</sub>, but the inverse of the length of the half‐change unit 1/ΔΔ<i>g</i><sub>(Δ0.5<i>C</i>)</sub>. This equals the absolute value of the slope of the distance decay regression divided by a constant (log2; <a href=#f3 class=scrollableLink>Fig. 3</a>). <span><a href=#b22 id=#b22R class="bibLink tab-link" data-tab=pane-pcw-references>Cody (1993)</a></span> referred to the slope itself as “beta diversity”, and recently doing so has become quite popular. Some studies have used a log‐transformed <i>y</i> axis (<span><a href=#b117 id=#b117R class="bibLink tab-link" data-tab=pane-pcw-references>Whittaker 1960</a></span>, <span><a href=#b86 id=#b86R class="bibLink tab-link" data-tab=pane-pcw-references>Pielou 1975</a></span>, <span><a href=#b115 id=#b115R class="bibLink tab-link" data-tab=pane-pcw-references>Vazquez and Givnish 1998</a></span>, <span><a href=#b59 id=#b59R class="bibLink tab-link" data-tab=pane-pcw-references>Leach and Givnish 1999</a></span>, Qian et al. 2005, 2009, <span><a href=#b91 id=#b91R class="bibLink tab-link" data-tab=pane-pcw-references>Qian and Ricklefs 2007</a></span>, <span><a href=#b89 id=#b89R class="bibLink tab-link" data-tab=pane-pcw-references>Qian 2008</a></span>, <span><a href=#b105 id=#b105R class="bibLink tab-link" data-tab=pane-pcw-references>Smith 2008</a></span>), others a linear one (<span><a href=#b22 id=#b22R class="bibLink tab-link" data-tab=pane-pcw-references>Cody 1993</a></span>, <span><a href=#b27 id=#b27R class="bibLink tab-link" data-tab=pane-pcw-references>Condit et al. 2002</a></span>, <span><a href=#b30 id=#b30R class="bibLink tab-link" data-tab=pane-pcw-references>Davidar et al. 2007</a></span>, <span><a href=#b77 id=#b77R class="bibLink tab-link" data-tab=pane-pcw-references>Novotny et al. 2007</a></span>, <span><a href=#b74 id=#b74R class="bibLink tab-link" data-tab=pane-pcw-references>Muneepeerakul et al. 2008</a></span>, <span><a href=#b90 id=#b90R class="bibLink tab-link" data-tab=pane-pcw-references>Qian 2009</a></span>). Both approaches can be justified, but each quantifies a different rate of change (compare with Sections 4.1 and 4.2, above). Recently also the halving distance, i.e. the length of the half‐change unit Δ<i>g</i><sub>(0.5<i>C</i>)</sub>, has been used as a measure of “beta diversity” (<span><a href=#b90 id=#b90R class="bibLink tab-link" data-tab=pane-pcw-references>Qian 2009</a></span>). This equals the inverse of Pielou's measure if the same similarity index (<i>C</i><sub>S</sub>) is used.
 </p>
 
 <p>These rate‐related measures have led to much confusion, because the term “beta diversity” can now refer to concepts as different as the amount of effective species turnover (Δ<i>c</i>), the rate of effective species turnover (Δ<i>c</i><sub>(Δ<i>g</i>)</sub>/Δ<i>g</i>), the rate of change in effective species turnover (ΔΔ<i>c</i><sub>(ΔΔ<i>g</i>)</sub>/ΔΔ<i>g</i>), the amount of change in differences along an explanatory gradient (ΔΔ<i>g</i><sub>(Δlog(1–Δ<i>c</i>))</sub>), the halving distance (Δ<i>g</i><sub>(0.5<i>C</i>)</sub>), or the inverse of the halving distance (1/Δ<i>g</i><sub>(0.5<i>C</i>)</sub>). Sometimes two or three of these meanings have been used in the same paper (<span><a href=#b22 id=#b22R class="bibLink tab-link" data-tab=pane-pcw-references>Cody 1993</a></span>, <span><a href=#b115 id=#b115R class="bibLink tab-link" data-tab=pane-pcw-references>Vazquez and Givnish 1998</a></span>, <span><a href=#b59 id=#b59R class="bibLink tab-link" data-tab=pane-pcw-references>Leach and Givnish 1999</a></span>, <span><a href=#b77 id=#b77R class="bibLink tab-link" data-tab=pane-pcw-references>Novotny et al. 2007</a></span>, <span><a href=#b90 id=#b90R class="bibLink tab-link" data-tab=pane-pcw-references>Qian 2009</a></span>), which makes accurate communication of ideas and results very difficult. This is especially problematic because the slope of a regression line (such as ΔΔ<i>c</i><sub>(ΔΔ<i>g</i>)</sub>/ΔΔ<i>g</i>) is independent of the average <i>y</i> value of the observations (such as 
<img alt="inline image" class=section_image src=images/100.gif>). Thus, one cannot be used as predictor for the other. The only logical link between the two is that the steepness of the slope is constrained by the variability of the <i>y</i> values; if all <i>y</i> values are equal, the regression slope is necessarily zero.
 </p>
 
 </section>
 
 </section>
 <section class=article-section__content id=ss33>
 
 <h2 class="article-section__title section__title section1" id=ss33-title> Biases and constraints</h2>
 
 <section class=article-section__sub-content id=ss34>
 
 <h3 class="article-section__sub-title section2" id=ss34-title> The bottom line</h3>
 
 <p>Quantifying diversity necessitates that the target organism group be explicitly defined. The target group may be, for example, trees with diameter at breast height exceeding 10 cm, herbaceous vascular plants, breeding birds, ungulates, any aquatic animals caught in a net of a specified mesh size, any arthropods caught in a pitfall trap, a specific family of beetles caught in a pitfall trap, and so forth. Possible target groups differ greatly in the size of individuals, population density per unit area, detectability of the individuals and existing number of species, and all of these variables have a great impact on diversity estimates.</p>
 
 <p>Diversity can be calculated for any dataset (sites by species table), but meaningful ecological conclusions necessitate that sampling efforts are standardised among sampling units and datasets (<span><a href=#b39 id=#b39R class="bibLink tab-link" data-tab=pane-pcw-references>Gotelli and Colwell 2001</a></span>; see also Sections 4.1–4.4, above). There are many ways of quantifying sampling effort, and these are not interchangeable. One possibility is to define each sampling unit such that it contains a fixed total abundance of the organisms of interest. If the number of individuals observed is used as the abundance criterion, the results are relatively easy to compare, because each individual has an equal probability of contributing a new species to the dataset. However, sometimes a more appropriate measure of observed abundance may be biomass, basal area or cover. Another way of standardising sampling effort is to use a plot of a fixed surface area or a fixed period during which observations are made. When such approaches are used, it needs to be taken into account that differences in diversity among areas or target groups may simply be due to differences in the density of individuals. This, in turn, may depend on the size of the individuals or a variety of other factors that are specific to the target organism. These can be, for example, spatial (size and shape of the sampling unit in relation to the size of the organisms), temporal (observation season and time of day) or behavioral (territoriality and tendency to avoid being seen or caught). Great caution is therefore needed if diversity values obtained with different kinds of organisms are compared.
 </p>
 
 </section>
 
 <section class=article-section__sub-content id=ss35>
 
 <h3 class="article-section__sub-title section2" id=ss35-title> Extrapolating alpha and gamma diversity</h3>
 
 <p>All diversity components (α, β and γ) are exactly quantifiable in any dataset where the observed entities have been classified into both sampling units and species. However, often researchers are not interested in the diversity components of the dataset itself, but in using the dataset to infer diversity components for larger areas. Typically, the region of interest contains orders of magnitude more individuals than the sample dataset, such that the species identity of only a small proportion of the relevant individuals is known. Similarly, each site or habitat of interest may contain many more individuals than the sampling unit representing it. Using the species diversities of a single sampling unit <sup><i>q</i></sup><i>D</i><sub><i>γj</i></sub> and the sampled dataset <sup><i>q</i></sup><i>D</i><sub>γ</sub> (collectively referred to as diversity of the observed unit <sup><i>q</i></sup><i>D</i><sub>O</sub>) as estimates of the species diversity of the corresponding target units <sup><i>q</i></sup><i>D</i><sub>T</sub> (site, habitat or region) is therefore faced with extrapolation problems. It is also possible that the dataset includes individuals that do not belong to the target set. For example, the interest may be in breeding birds but the available dataset also includes non‐breeding individuals. This may cause <sup><i>q</i></sup><i>D</i><sub>T</sub> to be overestimated.
 </p>
 
 <p>More commonly, the target unit contains all the individuals of the observed unit and additional individuals as well. Then observed species richness (<sup>0</sup><i>D</i><sub>O</sub>) can at most equal the actual species richness of the target unit (<sup>0</sup><i>D</i><sub>T</sub>). The rarer a species, the more individuals need to be observed (on average) before it is found, so the more rare species there are, the larger the downward bias when <sup>0</sup><i>D</i><sub>O</sub> is used as an estimate of <sup>0</sup><i>D</i><sub>T</sub>. As <i>q</i> increases, the effect of the rare species on the value of <sup><i>q</i></sup><i>D</i><sub>O</sub> decreases, so the downward bias in <sup><i>q</i></sup><i>D</i><sub>T</sub> diminishes. However, usually both sampling and species distributions are spatially autocorrelated, which biases the sampling towards observing some species over others and overestimating the abundances of the observed species. Hence, <sup><i>q</i></sup><i>D</i><sub>T</sub> is probably underestimated at all values of <i>q</i>.
 </p>
 
 <p>In general, the larger the size discrepancy (difference in the number of individuals) between the observed unit and the target unit, the larger the underestimation error when <sup><i>q</i></sup><i>D</i><sub>O</sub> is used as an estimate of <sup><i>q</i></sup><i>D</i><sub>T</sub>. If all individuals of the target group that exist in the target unit are actually sampled, then <sup><i>q</i></sup><i>D</i><sub>O</sub> necessarily equals <sup><i>q</i></sup><i>D</i><sub>T</sub>. How large a proportion of the individuals in the target unit need to be observed for <sup><i>q</i></sup><i>D</i><sub>O</sub> to converge on <sup><i>q</i></sup><i>D</i><sub>T</sub> depends on the species richness, species abundance distribution and spatial structure of the target unit, as well as on sampling strategy (<span><a href=#b97 id=#b97R class="bibLink tab-link" data-tab=pane-pcw-references>Scheiner 1990</a></span>, <span><a href=#b24 id=#b24R class="bibLink tab-link" data-tab=pane-pcw-references>Colwell and Coddington 1994</a></span>, <span><a href=#b40 id=#b40R class="bibLink tab-link" data-tab=pane-pcw-references>Gray 2000</a></span>, <span><a href=#b39 id=#b39R class="bibLink tab-link" data-tab=pane-pcw-references>Gotelli and Colwell 2001</a></span>, <span><a href=#b17 id=#b17R class="bibLink tab-link" data-tab=pane-pcw-references>Chao et al. 2009</a></span>).
 </p>
 
 <p>Imagine two target units, one with 10 species and the other with 1000 species, each represented by an observed unit of 100 individuals. For the species‐rich target unit, <sup>0</sup><i>D</i><sub>O</sub> underestimates <sup>0</sup><i>D</i><sub>T</sub> by at least 90%, but for the species‐poor target unit, <sup>0</sup><i>D</i><sub>O</sub> may equal <sup>0</sup><i>D</i><sub>T</sub>. A much higher sampling effort is obviously needed in a species‐rich target unit than in a species‐poor one for it to be possible to observe all species. Furthermore, if a target unit is dominated by a single species, this will also dominate the sample and fewer species will be observed. If the degree of dominance is much higher in a species‐rich target unit than in a species‐poor one, it is even possible for <sup>0</sup><i>D</i><sub>O</sub> of the former to be smaller than <sup>0</sup><i>D</i><sub>O</sub> of the latter.
 </p>
 
 <p>The spatial and environmental setup of sampling within the target unit also affect bias. If species distributions within the target unit are spatially structured, a geographically biased sample can generally be expected to yield smaller <sup><i>q</i></sup><i>D</i><sub>O</sub> than a random sample with the same number of individuals. In practice, geographically biased sampling caused by limited accessibility in some parts of the target unit is extremely common. To counteract undersampling bias, known environmental gradients within the target unit can be used to maximise compositional coverage of the observations (e.g. by the “gradsect” method; <span><a href=#b4 id=#b4R class="bibLink tab-link" data-tab=pane-pcw-references>Austin and Heyligers 1989</a></span>).
 </p>
 
 <p>If each sampling unit is its own target unit, then <sup><i>q</i></sup><i>D</i><sub>O</sub>=<sup><i>q</i></sup><i>D</i><sub>T</sub> and the <sup><i>q</i></sup><i>D</i><sub>γ<i>j</i></sub> values have no undersampling bias. Nevertheless, the alpha diversity estimate for the target region (mean <sup><i>q</i></sup><i>D</i><sub>γ<i>j</i></sub>) will have sampling error if all sampling units together do not contain all individuals of the target region. If the sampling units are biased towards sites with higher or lower species diversity than the actual regional mean, then the alpha diversity estimate will be similarly biased.
 </p>
 
 <p>Often both sampling units and the corresponding target units are circumscribed by spatial coordinates. Then both <sup><i>q</i></sup><i>D</i><sub>γ<i>j</i></sub> and the corresponding <sup><i>q</i></sup><i>D</i><sub>T</sub> values may refer to unknown numbers of individuals, which makes estimating the degree of undersampling bias difficult. Especially macroecological studies suffer from this problem, because actual sampling efforts may differ by orders of magnitude among sampling units of the same surface area, but the ancillary information needed to estimate sampling efficiency is often not available. Comparing diversity patterns among organisms that differ in individual density, detectability and/or species richness is also risky even if sampling effort per surface area is uniform. Results of such studies need to be treated with extreme caution to avoid interpreting patterns in the vagaries of sampling as patterns in species diversity.
 </p>
 
 </section>
 
 <section class=article-section__sub-content id=ss36>
 
 <h3 class="article-section__sub-title section2" id=ss36-title> Extrapolating the beta component of diversity</h3>
 
 <p>The beta components of diversity (β<sub>M</sub> and Δ<i>c</i>) are quantified using available alpha and gamma diversity values, which are subject to the sampling biases discussed above if extrapolated beyond the actual dataset sampled. If the observed <sup><i>q</i></sup><i>D</i><sub>γ</sub> is an underestimate of the gamma diversity of the target region, the beta component will be underestimated. Conversely, if the observed mean <sup><i>q</i></sup><i>D</i><sub>γ<i>j</i></sub> is an underestimate of the alpha diversity of the target region, the beta component will be overestimated. Thereby, the beta component can be either underestimated or overestimated, depending on the relative magnitudes of the biases in the alpha and gamma diversity estimates.
 </p>
 
 <p>The implications of undersampling are somewhat different for the regional and pairwise approaches to quantifying the beta component. The following is written in terms of effective species turnover Δ<i>c</i>, but similar reasoning applies to the heterogeneity measures β<sub>M</sub> as well. Imagine that each sampling unit is representative of the corresponding target unit, but the total set of sampling units is not representative of the target region. Then the Δ<i>c</i><sub>tot</sub> value of the dataset is clearly an underestimate of Δ<i>c</i><sub>tot</sub> of the target region, but the pairwise Δ<i>c</i><sub><i>j,k</i></sub> values of the sampling units are accurate estimates of the pairwise Δ<i>c</i><sub><i>j,k</i></sub> values of the corresponding target units. The average of the pairwise values in the dataset is an unbiased estimate of the target average if sampling unit placement is unbiased, but can become either an underestimate or an overestimate with biased sampling. The situation is different if the sampling units are very small in relation to the target units, but their number is so large that the target region is well represented. In this case, both the regional and pairwise Δ<i>c</i> values are overestimates of their target values.
 </p>
 
 <p>Imagine further that a region of interest contains two habitats that have 10 species each and share 5 species, and two habitats that have 1000 species each and share 500 species. Whittaker's species turnover (<sup>0</sup>β<sub>Mt−1</sub>) between the two low‐diversity habitats is exactly the same as that between the two high‐diversity habitats, namely 0.5. This is not the case for sampling units containing 100 individuals each, however. If some species in a habitat have a higher probability of being included in the corresponding sampling unit than others (e.g. because overall abundances differ or species distributions are aggregated), then the values of <sup>0</sup>β<sub>Mt−1</sub> observed between sampling units can theoretically vary over the entire interval [0, 1] depending on vagaries of sampling, no matter what the actual <sup>0</sup>β<sub>Mt−1</sub> value between the corresponding habitats. If every species does have the same probability of being included in a sampling unit, then <sup>0</sup>β<sub>Mt−1</sub>≅ 0.5 between the sampling units representing the species‐poor habitats, because all species that are actually present in each habitat are probably included in the corresponding sampling unit. In contrast, <sup>0</sup>β<sub>Mt−1</sub>≅ 1 between the sampling units representing the species‐rich habitats, because the number of individuals sampled is only 10% of the number of species present in each habitat, and it is unlikely that the same shared species happen to be sampled in both habitats (<span><a href=#b124 id=#b124R class="bibLink tab-link" data-tab=pane-pcw-references>Wolda 1981</a></span>, <span><a href=#b97 id=#b97R class="bibLink tab-link" data-tab=pane-pcw-references>Scheiner 1990</a></span>, <span><a href=#b24 id=#b24R class="bibLink tab-link" data-tab=pane-pcw-references>Colwell and Coddington 1994</a></span>, <span><a href=#b88 id=#b88R class="bibLink tab-link" data-tab=pane-pcw-references>Plotkin and Muller‐Landau 2002</a></span>, <span><a href=#b16 id=#b16R class="bibLink tab-link" data-tab=pane-pcw-references>Chao et al. 2006</a></span>). It is obviously incorrect to conclude from such <sup>0</sup>β<sub>Mt−1</sub> values that Whittaker's species turnover is higher between the two species‐rich <i>habitats</i> than between the two species‐poor <i>habitats</i>. However, a justified interpretation can be made in terms of the sampling units themselves: Whittaker's species turnover really is higher between the two species‐rich <i>sampling units</i> than between the two species‐poor <i>sampling units</i>. This is, after all, what was actually measured.
 </p>
 
 <p>Diversity differences comparable to this example are probably quite common in nature, for example between tree inventories in tropical vs boreal forests. Consequently, great care is needed if patterns in observed β<sub>M</sub> or Δ<i>c</i> along external gradients, such as latitude, are interpreted in terms of habitats or other entities larger than the sampling unit. If there is a trend in species diversity along the gradient of interest, a spurious trend in compositional heterogeneity and effective species turnover may result simply because the degree of undersampling and hence the amount of bias depends on species diversity (<span><a href=#b25 id=#b25R class="bibLink tab-link" data-tab=pane-pcw-references>Colwell and Hurtt 1994</a></span>).
 </p>
 
 <p>When two target units are compositionally very similar, actual Δ<i>c</i><sub><i>j,k</i></sub> and β<sub>M<i>j,k</i></sub> values are small and can hence be overestimated by a much wider margin than when the actual Δ<i>c</i><sub><i>j,k</i></sub> and β<sub>M<i>j,k</i></sub> values are large. Severe undersampling therefore causes pairwise effective species turnover and compositional heterogeneity to converge towards high values for all sampling unit pairs (<span><a href=#b49 id=#b49R class="bibLink tab-link" data-tab=pane-pcw-references>Jobe 2008</a></span>, <span><a href=#b14 id=#b14R class="bibLink tab-link" data-tab=pane-pcw-references>Cardoso et al. 2009</a></span>). This makes unraveling ecological or spatial trends in Δ<i>c</i> and β<sub>M</sub> more difficult for species‐rich than for species‐poor habitats or target organisms (<span><a href=#b50 id=#b50R class="bibLink tab-link" data-tab=pane-pcw-references>Jones et al. 2008</a></span>). When abundance data are available, the undersampling bias can be corrected to some degree (<span><a href=#b16 id=#b16R class="bibLink tab-link" data-tab=pane-pcw-references>Chao et al. 2006</a></span>), but extrapolation of Δ<i>c</i> and β<sub>M</sub> should still be done with caution.
 </p>
 
 <p> <span><a href=#b14 id=#b14R class="bibLink tab-link" data-tab=pane-pcw-references>Cardoso et al. (2009)</a></span> tested how sensitive various “beta diversity” indices are to undersampling, and used their robustness to make recommendations on which to use. However, each one of the indices they tested quantifies a different phenomenon. These included Whittaker's species turnover β<sub>Mt−1</sub>, ranged proportional species turnover β<sub>Pt</sub>, non‐nestedness, measures based on the species richness of a focal sampling unit, and rates of change. Knowledge about the sampling behaviour of an index is important when interpreting the results, but the primary criterion for choosing an index should be whether or not it quantifies the phenomenon of interest.
 </p>
 
 </section>
 
 </section>
 <section class=article-section__content id=ss37>
 
 <h2 class="article-section__title section__title section1" id=ss37-title> Discussion</h2>
 
 <p>It follows from the diversity of beta diversities that researchers should be explicit about what kind of “beta diversity” they refer to. Some recent papers on “beta diversity” are not cited in the present review, because they did not define “beta diversity” explicitly, and could therefore not be placed in the general framework. The framework itself is not exhaustive, because some papers used such complicated calculations to derive “beta diversity” that I was unable to figure out the ecological meaning of the final variable. Many studies have failed to address their stated research question because the selected method of quantifying “beta diversity” was not compatible with it, or have compared their results with those of earlier studies in which a different kind of “beta diversity” had been used. Most of the indices that have been used as explicit measures of “beta diversity” actually do not measure a (beta) component of diversity at all (see, for example, <a href=#t1 class=scrollableLink>Table 1</a> in <span><a href=#b56 id=#b56R class="bibLink tab-link" data-tab=pane-pcw-references>Koleff et al. 2003a</a></span>). In the absence of clear criteria by which to select among the available indices, many studies have opted for using two or more indices in parallel. Sometimes one of these did measure a beta component of diversity, other times not.
 </p>
 
 <p>Species turnover and beta diversity are often considered synonymous, but <span><a href=#b116 id=#b116R class="bibLink tab-link" data-tab=pane-pcw-references>Vellend (2001)</a></span> argued that they should be maintained separate. Vellend considered beta diversity “a value that can be related mathematically to α‐ and γ‐diversity”, and species turnover “the rate or magnitude of change in species composition along predefined spatial or environmental gradients”. I find even these definitions too broad. Not only true beta diversity (β<sub>Md</sub>) and regional‐to‐local diversity ratio (β<sub>Mt</sub>) but also several other variables are simple functions of γ and α (<span><a href=#b110 id=#b110R class="bibLink tab-link" data-tab=pane-pcw-references>Tuomisto 2010</a></span>). Three of these (β<sub>At</sub>, β<sub>Mt−1</sub> and β<sub>Pt</sub>) quantify the total number or proportion of effective species that change among sampling units in a dataset, i.e. absolute or relative effective species turnover Δ<i>c</i>. This equals actual species turnover when based on presence‐absence data (<i>q</i>=0). Many other definitions of compositional turnover have also been used, but since these are not derivable from gamma and alpha diversity, they do not quantify (effective) species turnover in this strict sense. In addition, magnitude of total species turnover (Δ<i>c</i><sub>tot</sub> of Section 1.1, above) and species turnover in a single compositional dimension (Δ<i>c′</i><sub>max</sub> of Section 2.3) are not affected by the choice of a reference gradient, but species turnover along an external gradient is (Δ<i>c</i><sub>(Δ<i>g</i>)</sub> of Section 2.4). Furthermore, magnitude of change along a gradient does not equal rate of change along that same gradient (Δ<i>c</i><sub>(Δ<i>g</i>)</sub>/Δ<i>g</i> of Section 4.6). The magnitude corresponds to the difference in <i>y</i> axis value between two points on the regression line in the level 1 scatterplot in <a href=#f1 class=scrollableLink>Fig. 1</a>, whereas the rate corresponds to the slope of the regression line. The “species turnover” measure preferred by <span><a href=#b116 id=#b116R class="bibLink tab-link" data-tab=pane-pcw-references>Vellend (2001)</a></span> actually represents the next level of abstraction, and corresponds to the maximum difference in <i>x</i> axis values in the level 2 scatterplot of <a href=#f1 class=scrollableLink>Fig. 1</a>‘ΔΔ<i>g</i><sub>(Δlog(1–Δ<i>c</i>))</sub> of Section 2.5’. Rather than quantifying change in species composition, this measure expresses change in environmental difference in terms of change in similarity in species composition. <span><a href=#b117 id=#b117R class="bibLink tab-link" data-tab=pane-pcw-references>Whittaker (1960)</a></span> himself included in the original concept of “beta diversity” as well β<sub>M</sub> as Δ<i>c</i> and ΔΔ<i>g</i><sub>(Δlog(1–Δ<i>c</i>))</sub>; the recent literature on “beta diversity” has been so confused that a clarification of terminology is urgently needed.
 </p>
 
 <p> <span><a href=#b52 id=#b52R class="bibLink tab-link" data-tab=pane-pcw-references>Jurasinski et al. (2009)</a></span> reviewed different “beta diversity” concepts and divided them into two groups. The first group included multiplicative and additive partitioning of total species diversity. In the present paper, these correspond to regional‐to‐local diversity ratio (β<sub>Mt</sub>) and absolute effective species turnover (β<sub>At</sub>), respectively. The second group included a number of different phenomena subdivided into four categories. The first category included all approaches where sampling units are compared in a pairwise manner using dissimilarity coefficients. As explicit examples, they mentioned a number of indices that do not correspond to any beta component of diversity, and the Sørensen and Jaccard indices, which in the present review correspond to different basic definitions of a beta component. Their second category included the sum of squares of a sites by species matrix, which corresponds to the average of all pairwise squared Euclidean distances between sampling unit pairs (Section 2.1, above). Their third category included the average distance to a compositional centroid (Section 2.2, above), compositional gradient length (Section 2.3, above) and the number of half‐change units (Section 2.5, above). The present paper aims to clarify the differences among variants even further.
 </p>
 
 <p>Terminological confusion is apparent, for example, in the study by <span><a href=#b77 id=#b77R class="bibLink tab-link" data-tab=pane-pcw-references>Novotny et al. (2007)</a></span>. They used three different similarity measures to derive “beta diversity”. The Sørensen index corresponds to the one‐complement of β<sub>Mt−1</sub> (Section 1.2, above). The codominance index <i>C</i>(<i>d</i>) (used also by <span><a href=#b18 id=#b18R class="bibLink tab-link" data-tab=pane-pcw-references>Chave and Leigh 2002</a></span> and <span><a href=#b27 id=#b27R class="bibLink tab-link" data-tab=pane-pcw-references>Condit et al. 2002</a></span>) quantifies the probability that two individuals drawn at random from sampling units <i>d</i> kilometres apart belong to the same species. In high‐diversity systems, the “beta diversity” estimate obtained with <i>C</i>(<i>d</i>) approaches zero whatever the compositional similarity between the sampling units (<span><a href=#b51 id=#b51R class="bibLink tab-link" data-tab=pane-pcw-references>Jost 2006</a></span> and Section 7 of <span><a href=#b110 id=#b110R class="bibLink tab-link" data-tab=pane-pcw-references>Tuomisto 2010</a></span>). The third index was based on <i>γ</i><sub>F</sub>/<i>γ</i> where <i>γ</i><sub>F</sub> is the species richness of a focal site and <i>γ</i> the known species richness of the entire country. <i>γ</i><sub>F</sub>/<i>γ</i> would equal 1/β<sub>Mt</sub> if <i>γ</i><sub>F</sub> equaled average sampling unit diversity (Section 3.3, above), but it is more likely that <i>γ</i><sub><i>j</i>max</sub> was used. In tropical rain forest datasets, <i>γ</i><sub><i>j</i>max</sub> is easily orders of magnitude higher than mean sampling unit diversity, and Δ<i>c</i><sub>tot</sub> is hence considerably underestimated if <i>γ</i><sub><i>j</i>max</sub> is used instead of 
<img alt="inline image" class=section_image src=images/101.gif>. Furthermore, the authors' claim that <i>γ</i><sub>F</sub>/<i>γ</i> indicated “low beta diversity” cannot be evaluated, because no information on sampling effort was provided. The possible range of values that <i>γ</i><sub>F</sub>/<i>γ</i> can take given the data at hand is hence unknown (Section 5 of <span><a href=#b110 id=#b110R class="bibLink tab-link" data-tab=pane-pcw-references>Tuomisto 2010</a></span>; <i>Biases and constraints</i>, above). Their fourth “beta diversity” measure was distance decay rate ΔΔ<i>c</i><sub>(ΔΔ<i>g</i>)</sub>/ΔΔ<i>g</i> which is not correlated with any measure of Δ<i>c</i> (Section 4.7). Although their target region was over 500 km long, <span><a href=#b77 id=#b77R class="bibLink tab-link" data-tab=pane-pcw-references>Novotny et al. (2007)</a></span> used data from only 4–8 sampling units, which were purposefully placed in similar environments. This leads to <sup>0</sup><i>D</i><sub>O</sub> being a strongly downward biased estimate of <sup>0</sup><i>D</i><sub>T</sub> for gamma diversity. At the same time, very high within‐site sampling effort led to <sup>0</sup><i>D</i><sub>O</sub> being a rather good estimate of <sup>0</sup><i>D</i><sub>T</sub> for alpha diversity. The implications of this sampling strategy were not discussed when the observation of “low beta diversity” was extrapolated to tropical forests in other parts of the world.
 </p>
 
 <p>A more analytical approach was taken by <span><a href=#b57 id=#b57R class="bibLink tab-link" data-tab=pane-pcw-references>Koleff et al. (2003b)</a></span>, who made a laudable effort to evaluate how much the correlation between “beta diversity” and latitude depends on how the computations are done. They evaluated ten “beta diversity” indices, which quantify the following (in same order as in their Table 2):
 </p>
 
 <div class=paragraph-element>
 
 <ul class="rlist hanging">
 
 <li><span class=number>1 </span>
 
 
 <p>regional‐to‐local diversity ratio β<sub>Mt</sub> (Section 1.1, above)
 </p>
 
 </li>
 
 <li><span class=number>2 </span>
 
 
 <p>rate of change in regional‐to‐local diversity ratio with increasing number of sampling units Δβ<sub>Mt</sub>/Δ<i>x</i> (Section 4.3.A)
 </p>
 
 </li>
 
 <li><span class=number>3 </span>
 
 
 <p>amount of absolute species turnover along an external gradient β<sub>At(Δ<i>g</i>)</sub> (Section 2.4)
 </p>
 
 </li>
 
 <li><span class=number>4 </span>
 
 
 <p>amount of Whittaker's species turnover along an external gradient β<sub>Mt−1(Δ<i>g</i>)</sub> (Section 2.4)
 </p>
 
 </li>
 
 <li><span class=number>5 </span>
 
 
 <p>the rate at which the sum of pairwise β<sub>Mt−1</sub> values increases as the number of sampling units in a transect increases
 </p>
 
 </li>
 
 <li><span class=number>6 </span>
 
 
 <p>rate of change in 1/β<sub>Mt</sub> with increasing proportion of sampling units inventoried (Section 4.3.C)
 </p>
 
 </li>
 
 <li><span class=number>7 </span>
 
 
 <p>rate of change (with increasing number of sampling units) in how many times as many species all sampling units together contain than the most species‐rich one of them (Section 3.3)</p>
 
 </li>
 
 <li><span class=number>8 </span>
 
 
 <p>the proportion of species pairs that do not co‐occur in any sampling unit</p>
 
 </li>
 
 <li><span class=number>9 </span>
 
 
 <p>compositional non‐nestedness of a focal sampling unit (Section 3.2)</p>
 
 </li>
 
 <li><span class=number>10 </span>
 
 
 <p>distinctness in species richness of a focal sampling unit.</p>
 
 </li>
 
 </ul>
 
 </div>
 
 <p>In addition, <span><a href=#b57 id=#b57R class="bibLink tab-link" data-tab=pane-pcw-references>Koleff et al. (2003b)</a></span> tested how the results are affected by different methods of defining the sampling units and the study region. Method 1 computes pairwise “beta” index values for adjacent grid cells along a latitudinal transect, and hence quantifies if the amount of “beta” between adjacent latitudes is correlated with latitude. Method 2 computes the “beta” index values between entire latitudinal bands, so any latitudinal trend that may actually exist in the “beta” index between adjacent latitudes will be confounded by the effect of varying sampling unit size. This problem was pointed out and extensively discussed by <span><a href=#b57 id=#b57R class="bibLink tab-link" data-tab=pane-pcw-references>Koleff et al. (2003b)</a></span>. In method 3a, the entire latitudinal belt is used as the study region, and each grid cell within it as a sampling unit. These values reflect “beta” within each latitudinal belt, but latitudinal comparisons are problematic because both the extent of the study region and the number of sampling units vary among latitudes. In method 3b, “beta” is calculated pairwise for adjacent grid cells within each latitudinal belt and the average of these pairwise values is used. Such an average is less affected by the number of sampling units than a regional “beta” value, especially when pairwise comparisons are only done among adjacent neighbours. Consequently, the “beta” values obtained using method 3b are comparable across latitudes even when latitudinal belts differ in the number of sampling units, and this method is suitable when exploring if within‐latitude “beta” is correlated with latitude.
 </p>
 
 <p> <span><a href=#b57 id=#b57R class="bibLink tab-link" data-tab=pane-pcw-references>Koleff et al. (2003b)</a></span> obtained rather disparate results from the different methods, and concluded that we still do not know if there are latitudinal gradients in “beta diversity”. As discussed above, the different analytical approaches in effect quantified different phenomena, many of which are logically uncorrelated with each other. In hindsight, the outcome of the analyses was perhaps not very surprising; if you ask a different question, you may get a different answer. The logical next step is to explore if a more consistent pattern is found when several datasets are used to address the same question.
 </p>
 
 </section>
 <section class=article-section__content id=ss38>
 
 <h2 class="article-section__title section__title section1" id=ss38-title> Conclusions</h2>
 
 <p>The original definition of “beta diversity” by <span><a href=#b117 id=#b117R class="bibLink tab-link" data-tab=pane-pcw-references>Whittaker (1960)</a></span> was already very broad, and since the coining of the concept, it has been tremendously stretched to cover the most varied phenomena. Beta diversity and different kinds of species turnover can be linked to external factors in different ways, and related phenomena can be quantified at different levels of abstraction. Most of the derived phenomena are interesting and ecologically meaningful in their own right, but the accuracy of scientific communication necessitates that conceptually different things be referred to by different terms. Therefore, care is needed not to confuse beta diversity itself with the various other phenomena that are derived from alpha and gamma diversity or otherwise related to beta diversity. Documenting and understanding patterns in beta diversity and species turnover at different spatial scales, in different areas and for different target organisms is in many ways a huge challenge. Let us avoid the additional difficulties that emerge from the inconsistent use of terminology.
 </p>
 
 </section>
 <div class=article-section__content>
 
 <h2 class="article-section__title section__title section1" id=ss39-title> Acknowledgements</h2>
 
 <p>I thank Robert K. Colwell for inviting me to write this review, for many useful comments on several versions of the manuscript, and for patience and support through a process that took much more time and resulted in a considerably heavier paper than either one of us had bargained for. The review has greatly benefited from discussions with Kalle Ruokolainen on beta diversity and related concepts. Helpful suggestions were also made by Mirkka Jones and four anonymous reviewers. The Academy of Finland is acknowledged for funding.</p>
 
 <ol></ol>
 </div>
 <section class="article-section article-section__references" id=references-section>
 <div class="accordion article-accordion">
 <h2>
 <div tabindex=0 role=button class=accordion__control aria-controls=idss40 aria-expanded=false data-references="/action/ajaxShowEnhancedAbstract?widgetId=5cf4c79f-0ae9-4dc5-96ce-77f62de7ada9&amp;ajax=true&amp;doi=10.1111%2Fj.1600-0587.2009.06148.x&amp;pbContext=%3Bpage%3Astring%3AArticle%2FChapter+View%3Bctype%3Astring%3AJournal+Content%3Bwebsite%3Awebsite%3Apericles%3Barticle%3Aarticle%3Adoi%5C%3A10.1111%2Fj.1600-0587.2009.06148.x%3Bjournal%3Ajournal%3A16000587%3Bissue%3Aissue%3Adoi%5C%3A10.1111%2Feco.2010.33.issue-1%3Bwgroup%3Astring%3APublication+Websites%3BrequestedJournal%3Ajournal%3A16000587%3BpageGroup%3Astring%3APublication+Pages%3BsubPage%3Astring%3AFull+Text&amp;accordionHeadingWrapper=h2&amp;onlyLog=true"><span class=section__title id=references-section-1>References</span><i aria-hidden=true class=icon-section_arrow_d></i></div>
 </h2>
 <div class=accordion__content style=display:none role=region aria-labelledby=references-section-1 id=ss40>
 
 </div>
 </div>
 </section>
 </section>
 <div style=display:none>
 
 </div>
 <div class=pb-dropzone data-pb-dropzone=citedBy-dropzone>
 <section id=cited-by class="article-section article-section__citedBy cited-by"><div class="accordion article-accordion"><h2><div tabindex=0 role=button aria-controls=cited-by__content aria-expanded=false class=accordion__control><span id=citedby-section class=section__title>Citing Literature</span><i aria-hidden=true class=icon-section_arrow_d></i></div></h2><div id=cited-by__content class="accordion__content cited-by__content" style=display:none></div></div></section>
 </div>
 
</article></div></div><div class="col-lg-4 col-md-4 col-sm-4 hidden-xs hidden-sm hidden-md article-row-right fixed-element" style=top:-444px><div class="publication-cover-info clearfix">
 
 <div class="cover-image-wrapper cover-image__image hasDetails"><a href=https://onlinelibrary.wiley.com/toc/16000587/2010/33/1 title="View Volume 33, Issue 1"><img src=images/102.gif alt></a></div><div class="extra-info-wrapper cover-image__details"><p class=volume-issue><a href=https://onlinelibrary.wiley.com/toc/16000587/2010/33/1 title="View Volume 33, Issue 1"><span>Volume</span><span class=val>33</span>, <span>Issue</span><span class=val>1</span></a><p>February 2010<p class=page-range><span>Pages</span><span> 23-45</span></p>
 
 
</div>
</div>
 
 
 <div class=pb-ad>
 <div id=advert-rail-1 class="advert advert-rail" style=min-width:300px;min-height:250px data-google-query-id=CP2V3O3Ow-wCFdIw4AodNQQGwQ>
 
 
 
 <div id=google_ads_iframe_/2215/wly.ecology_000046/ecography_ecog_1__container__ style="border:0pt none"></div></div>
 </div>
 
<div class="sticko__parent scrollThenFix" style="height:calc(0px + 100vh)"><div class="tab tab--slide tab--flex sticko__md tabs--xs dynamic-sticko"><ul data-mobile-toggle=slide role=tablist class="rlist tab__nav w-slide--list cloned"><li role=presentation><a data-slide-clone=self href=#pane-pcw-figures aria-controls=#pane-pcw-figures role=tab data-toggle=tab id=pane-pcw-figurescon data-slide-target=#pane-pcw-figures class="article-row-right__tabLinks w-slide__btn figures-tab empty" tabindex=-1 aria-selected=false><i aria-hidden=true class=icon-multi-figure></i><h2>Figures</h2></a><li role=presentation><a data-slide-clone=self href=#pane-pcw-references aria-controls=#pane-pcw-references role=tab data-toggle=tab id=pane-pcw-referencescon data-slide-target=#pane-pcw-references class="article-row-right__tabLinks w-slide__btn references-tab empty" tabindex=-1 aria-selected=false><i aria-hidden=true class=icon-chain></i><h2>References</h2></a><li role=presentation><a data-slide-clone=self href=#pane-pcw-related aria-controls=#pane-pcw-related role=tab data-toggle=tab id=pane-pcw-relatedcon data-slide-target=#pane-pcw-related class="article-row-right__tabLinks w-slide__btn related-tab empty" tabindex=-1 aria-selected=false><i aria-hidden=true class=icon-related-circles-extend-button></i><h2>Related</h2></a><li role=presentation class=active><a data-slide-clone=self href=#pane-pcw-details aria-controls=#pane-pcw-details role=tab data-toggle=tab id=pane-pcw-detailscon data-slide-target=#pane-pcw-details class="article-row-right__tabLinks w-slide__btn details-tab" tabindex=-1 aria-selected=false><i aria-hidden=true class=icon-Icon_Information></i><h2>Information</h2></a></ul><ul role=presentation class="rlist tab__content sticko__child"><li id=pane-pcw-figures aria-labelledby=pane-pcw-figurescon role=tabpanel class="tab__pane article-row-right__panes empty sf-hidden" aria-hidden=true><li id=pane-pcw-references aria-labelledby=pane-pcw-referencescon role=tabpanel class="tab__pane article-row-right__panes empty sf-hidden" aria-hidden=true><li id=pane-pcw-related aria-labelledby=pane-pcw-relatedcon role=tabpanel class="tab__pane article-row-right__panes empty sf-hidden" aria-hidden=true><li id=pane-pcw-details aria-labelledby=pane-pcw-detailscon role=tabpanel class="tab__pane article-row-right__panes active" aria-hidden=true>
<section class=info-tab-wrapper>
<section class=metrics>
<h3 class=info-header>Metrics</h3>
<div class=section__body><div class=cited-by-count><label>Citations: </label><span>218</span></div>
<div class=pb-dropzone data-pb-dropzone=publicaitonContent-info-tab-metrics>
<div data-badge-popover=left data-badge-type=1 data-doi=10.1111/j.1600-0587.2009.06148.x data-hide-no-mentions=false class=altmetric-embed data-uuid=fb1fb052-e11d-187c-24be-f4dd7b01a74e><a target=_self href="https://www.altmetric.com/details.php?domain=onlinelibrary.wiley.com&amp;citation_id=4103085" rel=popover data-content="<div>    <div style='padding-left: 10px; line-height:18px; border-left: 16px solid #958899;'>
      <a class='link-to-altmetric-details-tab' target='_self' href='https://www.altmetric.com/details.php?domain=onlinelibrary.wiley.com&amp;citation_id=4103085&amp;tab=wikipedia'>
          Referenced in <b>4</b> Wikipedia pages
      </a>
    </div>

    <div class='altmetric-embed readers' style='margin-top: 10px;'>
          <div class='altmetric-embed tip_mendeley'
               style='padding-left: 10px; line-height:18px; border-left: 16px solid #A60000;'>
              <b>1075</b> readers on Mendeley
          </div>
          <div class='altmetric-embed tip_citeulike'
               style='padding-left: 10px; line-height:18px; border-left: 16px solid #BCD2EF;'>
              <b>3</b> readers on CiteULike
          </div>
    </div>
    <div style='margin-top: 10px; text-align: center;'>
        <a class='altmetric_details' target='_self' href='https://www.altmetric.com/details.php?domain=onlinelibrary.wiley.com&amp;citation_id=4103085'>
            See more details
        </a>
    </div>
</div>" style=display:inline-block data-badge-popover=left>
 <img alt="Article has an altmetric score of 3" src=images/103.png style=border:0;margin:0;max-width:none width=110 height=20>
</a></div>
</div>
</div>
</section>
<section class=details>
<h3 class=info-header>Details</h3>
<p class=copyright>© 2010 The Authors</p>
<p></p>
<ul class="cross-mark rlist">
</ul>
</section>
<section class=publication-history>
<h3 class=info-header>Publication History</h3>
<ul class=rlist>
<li><label>Issue Online: </label>04 March 2010
</li>
<li><label>Version of Record online: </label>11 February 2010
</li>
<li><label>Editorial history: </label>Manuscript Accepted 18 November 2009
</li>
</ul>
</section>
</section>
<li class=tab__spinner style=display:none></ul></div></div></div></div></article><div id=figure-viewer data-ux3-wrapper=figure-viewer data-ux3-transformed-by=figureInit data-ux3-role=parent role=dialog class="figure-viewer sf-hidden"></div>
 
 <div style=text-align:right!important><a style=font-weight:300!important;font-size:90% href=https://onlinelibrary.wiley.com/doi/pdf/10.1111/j.1600-0587.2009.06148.x>Download PDF</a></div>
</div>
</div></div></div>
 </section>
 
 </div>
 </div>
<div class="w-slide padding-top"><div class=w-slide_head><a href=# class=w-slide__back><i class=icon-arrow_l aria-hidden=true></i>back</a><span class=w-slide__title></span></div><div class=w-slide__content></div></div></main>
 </div>
 
 <div data-widget-def=pageFooter data-widget-id=053994f3-7d80-4bb1-97a4-c67fbc4768cd class=pageFooter>
 
 
 <footer class=page-footer>
 <div data-pb-dropzone=main>
 
 
 <div class=main-footer>
 
 <style class=sf-hidden>.footer-links{text-align:center}.footer-links h3{font-weight:300}.footer-links a{font-weight:600}@media (min-width:992px){.footer-links{text-align:left}}</style>
 <div data-widget-def=ux3-layout-widget data-widget-id=2753bec1-a79d-4403-827b-6c788ddd6db3 class=footer-sections>
 
 
 <h2 class=sr-only>
 Additional links
 </h2>
 
 <div class=container><div class=row><div class=col-md-3>
 
 <div class="footer-links mb-3">
 <h3>About Wiley Online Library</h3>
 <ul class=rlist>
 <li><a href=https://www.wiley.com/privacy>Privacy Policy</a></li>
 <li><a href=https://onlinelibrary.wiley.com/terms-and-conditions>Terms of Use</a></li>
 <li><a href=https://onlinelibrary.wiley.com/cookies>Cookies</a></li>
 <li><a href=https://onlinelibrary.wiley.com/accessibility>Accessibility</a></li>
 </ul>
</div>
</div><div class=col-md-3>
 
 <div class="footer-links mb-3">
 <h3>Help &amp; Support</h3>
 <ul class=rlist>
 <li><a href=https://hub.wiley.com/community/support/onlinelibrary>Contact Us</a></li>
 </ul>
 </div>
</div><div class=col-md-3>
 
 <div class="footer-links mb-3">
 <h3>Opportunities</h3>
 <ul class=rlist>
 <li><a href=https://onlinelibrary.wiley.com/agents>Subscription Agents</a></li>
 <li><a href=https://onlinelibrary.wiley.com/advertisers>Advertisers &amp; Corporate Partners</a></li>
 </ul>
 </div>
</div><div class=col-md-3>
 
 <div class="footer-links mb-3">
 <h3>Connect with Wiley</h3>
 <ul class=rlist>
 <li><a href=https://www.wiley.com/network>The Wiley Network</a></li>
 <li><a href=https://newsroom.wiley.com/>Wiley Press Room</a></li>
 </ul>
 </div>
</div></div></div>
 </div>
 
 <div data-widget-def=ux3-layout-widget data-widget-id=86584e25-e4db-4118-9550-9b4f5341a4ac class=copyright-container>
 
 
 <div class=container><div class=row><div class=col-md-12>
 <div data-widget-def=UX3HTMLWidget data-widget-id=bfbf6c5d-a650-4662-828f-5275fada9120 class=copyright>
 
 
 <div class="copyright pull-left">Copyright © 1999-2020 <a href=https://www.wiley.com/>John Wiley &amp; Sons, Inc</a>. All rights reserved</div>
 </div>
 
 
 <div class="footer-logo pull-right">
 
<a href=https://www.wiley.com/ title="Wiley Home Page" aria-label=Wiley><img alt="Wiley Home Page" src=images/104.png></a>
</div>
</div></div></div>
 </div>
 
</div>
 </div>
</footer>
 </div>
 
 
 <div id=iucr-failure class=sf-hidden> </div> 
 
 
<div class="popup login-popup hidden sf-hidden" role=dialog tabindex=-1 aria-labelledby=loginPopupHead>
 
</div>
 
 
<div class="popup change-password-drawer hidden sf-hidden" role=dialog tabindex=-1 aria-labelledby=changePasswordPopupHead>
 
</div>
 
 
<div class="popup registration-popup hidden sf-hidden" role=dialog tabindex=-1 aria-labelledby=createAccountPopupHead>
 
</div>
 
 
<div class="popup top-drawer request-reset-password-drawer hidden sf-hidden" role=dialog tabindex=-1 aria-labelledby=resetPasswordPopupHead>
 
</div>
 
 <div><div class=pb-dropzone data-pb-dropzone=col-0 title=col-0></div></div>
 
 
<div class="popup top-drawer request-username-drawer username-popup hidden sf-hidden" role=dialog tabindex=-1 aria-labelledby=requestUsernamePopupHead>
 
 
</div>
 
 
 </div>
 </div><img src=images/105.x>
 
<div style=visibility:hidden;height:1px;width:1px;position:absolute;top:-9999px;z-index:100000 id=_atssh class=sf-hidden></div><div id=crossmark-widget style=display:none></div><ul class="ui-autocomplete ui-front ui-menu ui-widget ui-widget-content ui-corner-all quickSearchAutocomplete" id=ui-id-1 tabindex=0 style=display:none></ul><ul class="ui-autocomplete ui-front ui-menu ui-widget ui-widget-content ui-corner-all quickSearchAutocomplete" id=ui-id-2 tabindex=0 style=display:none></ul><div class="altmetric-embed altmetric-popover altmetric-left" data-uuid=fb1fb052-e11d-187c-24be-f4dd7b01a74e id=_altmetric_popover_el_fb1fb052-e11d-187c-24be-f4dd7b01a74e style=margin:0px;display:none></div>