<!DOCTYPE html>
<html class="pb-page readcubeEpdfEnabled" data-request-id="10611e42-7790-48d6-b121-0f3ac28426a7" lang="en"><head data-pb-dropzone="head">
<meta http-equiv="content-type" content="text/html; charset=UTF-8"><script type="text/javascript" src="300lo.js"></script><script type="text/javascript" src="_ate.js"></script><meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="pbContext" content=";page:string:Article/Chapter View;article:article:doi\:10.1002/ecy.2000;website:website:esajournals;journal:journal:19399170;issue:issue:doi\:10.1002/ecy.2017.98.issue-11;wgroup:string:Publication Websites;pageGroup:string:Publication Pages;subPage:string:Full Text"><meta data-pb-head="head-widgets-start">
<iframe style="position: absolute; top: -10000px; left: -1000px;"></iframe><script type="text/javascript" id="www-widgetapi-script" src="www-widgetapi.js" async=""></script><script async="" src="prum.js"></script><script src="iframe_api.js"></script><script type="text/javascript" src="get"></script><script type="text/javascript" src="optout_check"></script><script async="" src="controltag.js"></script><script async="" src="analytics_002.js"></script><script async="" src="gpt.js"></script><script type="text/javascript" async="" src="controltag"></script><script type="text/javascript">var $DoubleClickZone = "ecology_ecy";var $DoubleClickSite =  "wly.ecology_000046";</script><script>digitalData = {"site":{"ip":"147.231.250.202","environment":"LIVE","website":"esajournals.onlinelibrary.wiley.com","serverDate":"2019-02-19"},"page":{"pageGroup":"Publication Pages","page":"Article/Chapter View","subPage":"Full Text","title":"title","errorCode":200},"identities":[{"type":"InstitutionUser","uuid":"c06a2c01-b1e9-43e9-b189-34aee8bccded","customerRecords":[{"customerDomain":"ALM-CU","customerNumber":"CZECHPAREN"}]},{"type":"InstitutionUser","uuid":"dc317c6d-4e79-4edd-a7aa-fe6f96d47659","customerRecords":[{"customerDomain":"ALM-CU","customerNumber":"EAL0002360"}]},{"type":"InstitutionUser","uuid":"f87f3477-0cb1-46d9-88b0-66f38b6bd635","customerRecords":[{"customerDomain":"ALM-CU","customerNumber":"EAL00000135137"}]},{"type":"InstitutionUser","uuid":"1f76b38f-f6e1-4fa7-8f83-b2857df23484","customerRecords":[{"customerDomain":"ALM-CU","customerNumber":"CASACCESS"}]},{"type":"RegionUser","uuid":"0c75c26d-a68d-4d19-a724-218cec4841a6"},{"type":"RegionUser","uuid":"20d013f5-e22b-4df0-ba93-d6d925db9e32"},{"type":"BasicGroup","uuid":"197fe835-5b5f-4f4e-b8e9-a74d68491a84"},{"type":"InstitutionUser","uuid":"43c50b66-e754-4676-b7b8-9ba1fc83da98","customerRecords":[{"customerDomain":"ALM-CU","customerNumber":"P#CASCOCHRANE"}]},{"type":"InstitutionUser","uuid":"c8187016-3274-4a0b-9bbd-22ff593bc21b","customerRecords":[{"customerDomain":"ALM-CU","customerNumber":"GP#SLAV"}]}],"publication":{"item":{"title":"Deciphering the enigma of undetected species, phylogenetic, and functional diversity based on Good‐Turing theory","doi":"10.1002/ecy.2000","access":"full","earliestDate":"2017-09-04","formats":["Abstract","First page image","Full text","PDF","References","Supplementary Material","AM pdf"],"topics":[{"taxonomyUri":"global-subject-codes","topicLabel":"Ecology \u0026 Organismal Biology","topicUri":"ls20"},{"taxonomyUri":"global-subject-codes","topicLabel":"LIFE SCIENCES","topicUri":"ls"},{"taxonomyUri":"fundref-funders","topicLabel":"Ministry of Science and Technology, Taiwan","topicUri":"501100004663"},{"taxonomyUri":"fundref-funders","topicLabel":"National Science Foundation","topicUri":"100000001"}]},"group":{"doi":"10.1002/ecy.2017.98.issue-11","access":"full","volume":"98","issue":"11","coverDate":"2017-11-01","topics":[{"taxonomyUri":"global-subject-codes","topicLabel":"Ecology \u0026 Organismal Biology","topicUri":"ls20"},{"taxonomyUri":"global-subject-codes","topicLabel":"LIFE SCIENCES","topicUri":"ls"}]},"series":{"title":"Ecology","doi":"10.1002/(ISSN)1939-9170","access":"no","code":"19399170","issn":"0012-9658","eissn":"1939-9170","topics":[{"taxonomyUri":"global-subject-codes","topicLabel":"Ecology \u0026 Organismal Biology","topicUri":"ls20"},{"taxonomyUri":"global-subject-codes","topicLabel":"LIFE SCIENCES","topicUri":"ls"},{"taxonomyUri":"gpt-zonename","topicLabel":"ecology_ECY","topicUri":"ecology_ecy"},{"taxonomyUri":"gpt-sitename","topicLabel":"wly.ecology_000046","topicUri":"wly.ecology_000046"}]}}}</script>






<meta charset="UTF-8">




<meta name="robots" content="noarchive">

















    <meta name="viewport" content="width=device-width,initial-scale=1,maximum-scale=10, user-scalable=1">




    <meta name="apple-itunes-app" content="app-id=1207192610">




    









    

    <meta name="citation_doi" content="10.1002/ecy.2000">

    <meta name="dc.identifier" content="10.1002/ecy.2000">




<meta name="citation_journal_title" content="Ecology">
<meta name="citation_title" content="Deciphering the enigma of undetected species, phylogenetic, and functional diversity based on Good‐Turing theory">
<meta name="citation_keywords" content="functional attribute diversity">
<meta name="citation_keywords" content="functional diversity">
<meta name="citation_keywords" content="phylogenetic diversity">
<meta name="citation_keywords" content="shared diversity">
<meta name="citation_keywords" content="species diversity">
<meta name="citation_keywords" content="taxonomic diversity">
<meta name="citation_volume" content="98">
<meta name="citation_issue" content="11">
<meta name="citation_firstpage" content="2914">
<meta name="citation_lastpage" content="2929">
<meta name="citation_publication_date" content="2017/11/01">
<meta name="citation_online_date" content="2017/11/01">
<meta name="citation_issn" content="1939-9170">
<meta name="citation_language" content="en">
<meta name="citation_author" content="Anne Chao">
<meta name="citation_author_institution" content="Institute of Statistics National Tsing Hua University Hsin‐Chu 30043 Taiwan">
<meta name="citation_author" content="Chun‐Huo Chiu">
<meta name="citation_author_institution" content="Institute of Statistics National Tsing Hua University Hsin‐Chu 30043 Taiwan">
<meta name="citation_author" content="Robert K. Colwell">
<meta name="citation_author_institution" content="Department of Ecology and Evolutionary Biology University of Connecticut Storrs Connecticut 06269 USA">
<meta name="citation_author_institution" content="University of Colorado Museum of Natural History Boulder Colorado 80309 USA">
<meta name="citation_author_institution" content="Departmento de Ecologia Universidade Federal de Goiás CP 131, 74.001‐970 Goiânia GO Brasil">
<meta name="citation_author" content="Luiz Fernando S. Magnago">
<meta name="citation_author_institution" content="Departamento de Biologia, Setor de Ecologia e Conservação Universidade Federal de Lavras Lavras 37200‐000 Brasil">
<meta name="citation_author" content="Robin L. Chazdon">
<meta name="citation_author_institution" content="Department of Ecology and Evolutionary Biology University of Connecticut Storrs Connecticut 06269 USA">
<meta name="citation_author_institution" content="Department of Ecology and Evolutionary Biology University of Colorado Boulder Colorado 80309 USA">
<meta name="citation_author" content="Nicholas J. Gotelli">
<meta name="citation_author_institution" content="Department of Biology University of Vermont Burlington Vermont 05405 USA">
<meta name="citation_abstract_html_url" content="https://esajournals.onlinelibrary.wiley.com/doi/abs/10.1002/ecy.2000">
<meta name="citation_fulltext_html_url" content="https://esajournals.onlinelibrary.wiley.com/doi/full/10.1002/ecy.2000">
<meta name="citation_pdf_url" content="https://esajournals.onlinelibrary.wiley.com/doi/pdf/10.1002/ecy.2000">
<meta name="citation_publisher" content="John Wiley &amp; Sons, Ltd">
<meta name="article_references" content="Chao, A. , Chiu, C. , Colwell, R. K., Magnago, L. F., Chazdon, R. L. and Gotelli, N. J. (2017), Deciphering the enigma of undetected species, phylogenetic, and functional diversity based on Good‐Turing theory. Ecology, 98: 2914-2929. doi:10.1002/ecy.2000">
<meta name="epdf_available" content="true">


<title>Deciphering the enigma of undetected species, phylogenetic, and 
functional diversity based on Good‐Turing theory - Chao - 2017 - Ecology
 - Wiley Online Library</title>




<meta data-pb-head="head-widgets-end">

<link rel="stylesheet" type="text/css" href="ga7iproduct.css"><link rel="stylesheet" type="text/css" href="build.css">
<link rel="stylesheet" type="text/css" href="print.css" media="print">
<link rel="stylesheet" type="text/css" href="default.css" id="pb-css" data-pb-css-id="t-1-v0/default.css">




<script async="" src="catchpoint-1549680351760.js"></script>
<meta name="referrer" content="origin-when-cross-origin">

<link rel="stylesheet" type="text/css" href="hub-1541755742850.css">
<link rel="stylesheet" type="text/css" href="iucr-1510734465460.css">
<link rel="stylesheet" type="text/css" href="readcube-1510565373110.css">
<link rel="stylesheet" type="text/css" href="dr-overrides-1542408466660.css">
<!-- link rel="stylesheet" type="text/css" href="/pb-assets/css/temporary-1525702334993.css" -->

<style>
/* PERICLES-7186: distorted cover images */
.journal-info-container .cover-image__image img {
    min-height: unset;
}

/* TEMPORARY CSS HACK TO FIX LIT-220590 */
.icon-icon-Rss::before {
    content: "\e629";
}

/* TEMPORARY CSS TO FIX ALIGNMENT OF TOP AD IN RIGHT RAIL */
.publication-cover-info {
    margin-bottom: 20px;
}

.article-row-right > .pb-ad {
    margin-left: 0;
    margin-bottom: 10px;
}

/* TEMPORARY HACK FOR PERICLES-6763 */
.table-of-content .issue-item h2 {
    font-size: 16px;
}

.table-of-content .issue-item h2 a.issue-item__title {
    display: inline;
}

/* TEMPORARY CSS FOR PERICLES-7445 */

input.saml__consent__yes {
    background: #005274;
    font-size: 12px;
    font-size: .75rem;
    padding: 12px 20px;
    border-radius: 0;
    margin-top: 20px;
    margin-right: 10px;
    width: 100px;
}

input.saml__consent__yes:hover {
    background: #003951;
}

/* HACK TO CHANGE LABEL IN LOGIN FORM */
.login-popup .login-form label[for="login"]:before {
    content: "Email or Customer ID";
    color: black;
}

.login-popup .login-form label[for="login"] {
    color: white;
}

/* PERICLES-2166: Site-wide maintenance message */

.admin-message {
    background-color: #005274;
    color: white;
}

/* TEMPORARY CSS FOR STAY-CONNECTED SOCIAL MEDIA ICONS */
.stay-connected ul  {
    list-style: none;
    margin: 0;
    padding: 0;
}

.stay-connected i {
    margin-right: 10px;
    font-size: 25px;
    line-height: 55px;
}

.stay-connected i.icon-hubs-rg-icon,
.stay-connected i.icon-hubs-soundcloud-icon {
    font-size: 41px;
}

.stay-connected .icon-hubs-pinterest-icon:before {
    color: white;
}

.stay-connected .icon-hubs-pinterest-icon {
    background-color: #BD081C;
    border: solid #BD081C;
    border-width: 8px;
}

/* PERICLES-5833: TEMPORARY CSS TO ADJUST POSITION OF ADS */
.article-mpu-bottom .pb-ad {
    margin: 0 0 0 195px;
}


.article-mpu-top .pb-ad {
    padding-top: 20px;
    margin: 0 0 -35px 0;
}

.banner-wrapper .pb-ad {
    padding: 0;
}

/* TEMPORARY CSS FOR THE SOCIETY MEMBER LOGIN PAGE */
.society-member-login .widget__header,
.society-member-login .show-registration,
.society-member-login .show-request-reset-password {
    display: none;
}

.society-member-login .login-form {
    margin-top: 20px;
}

.society-member-login .widget__body .input-group+.input-group {
    margin-top: 15px;
}


/* TEMPORARY CSS TO HIDE FPI IN TOP CONTENT ACTION WIDGET */
.top-content .firstPageImage {
	display: none;
}

/* PERICLES-5518: TEMPORARY CSS TO FIX ANCHOR TAG OFFSET ON CUSTOM PAGES */
.custom-page-hub .main-content :target:before {
	content: "";
	display: block;
	height: 205px;
	margin: -205px 0 0;
}

.custom-page-pericles .main-content :target:before {
	content: "";
	display: block;
	height: 140px;
	margin: -140px 0 0;
}

/* PERICLES-4777: TEMPORARY CSS TO HIDE OPTIONS IN THE SIDEBAR ON MRW PAGES */
.journal-resources li#TableofContents,
.journal-resources li#Topics,
.journal-resources li#ArticleTitlesAZ {
	display: none;
}

/* TEMPORARY FIX FOR UNWANTED MARGIN BELOW COVER IMAGES ON BOOK SERIES PAGES */
.remove-margin .cover-image {
	margin: 0;
}

/* TEMPORARY STYLE TO HIDE NUMBER OF DOWNLOADS ON ARTICLE PAGES */
.info-tab-wrapper .metrics .section__body div:first-child {
	display: none;
}

/* TEMPORARY FIX TO MAKE IT POSSIBLE TO EDIT RICH TEXT WIDGETS IN SLIDESHOWS */
.bs .modal-backdrop {
	display: none;
}

/* TEMPORARY FIX FOR MENU POSITION */
.hubpage-menu nav {
	padding-bottom: 0;
	position: relative;
}

/* TEMPORARY FOR MARGIN AROUND ADS */
.advert.advert-leaderboard {
	margin: 15px 0;
}

.banner-wrapper {
	box-shadow: none;
}

/* TEMPORARY TO FIX OVERFLOW FOR P IN JOURNAL-NEWS SLIDESHOWS */
.journal-news .slide-item p {
	overflow: unset;
}

/* EDITOR HIGHLIGHTS CSS */
div.ancillary-content.relateditem div.content {
	height: 10em;
}
div.ancillary-content.listitem div.content {
	height: 4em;
}

/* TEMPORARY Fix heading on Topical Index widget for MRWs */
.topical-index h2 {
    margin: 15px 0;
    font-size: 1.75rem;
    line-height: 28px;
    color: #1c1d1e;
    font-family: Source Serif Pro,serif;
    font-weight: normal;
}

/* PERICLES-7288 */
.issue-supplementary-files, .cover-image__virtual-issue {
    margin-bottom: 15px;
}

/* PERICLES-8875 */
.journal-news .slide-item img {
    float: left;
    margin-right: 10px;
}

</style>

<!-- The Adobe DTM script. Must come after the digitalData object -->
<script src="satelliteLib-c1ac050bbdc647f13a62578e71f0d817f8a6cbb5.js"></script><script src="satellite-5c473d3564746d6cdb002b20.js"></script><script src="satellite-5aac819d64746d762f005d2f.js"></script><script src="satellite-59cd501964746d5527006272.js"></script>

<!-- Google Search Console verification -->
<meta name="google-site-verification" content="ffz4i4DxIJCdGpRXe0XN6KG6o4FYI-nnFW9I_dg8piM">

<!-- KRUX Implementation-->
<script class="kxct" data-id="JMOONjNh" data-timing="async" data-version="1.9" type="text/javascript" xml:space="preserve">

	window.Krux || (( Krux = function () {
		Krux.q.push ( arguments )
	}).q = []);

	( function () {
		var k , m , s , src;
		k = document.createElement ( 'script' );
		k.type 	= 'text/javascript';
		k.async = true;
		src = ( m = location.href.match ( /\bkxsrc=([^&]+)/ )) && decodeURIComponent ( m [ 1 ]);
		k.src = /^https?:\/\/([a-z0-9_\-\.]+\.)?krxd\.net(:\d{1,5})?\//i.test ( src ) ? src : src === 'disable' ? '' : ( location.protocol === 'https:'?'https:':'http:' ) + '//cdn.krxd.net/controltag?confid=JMOONjNh';
		s = document.getElementsByTagName ( 'script' ) [ 0 ];
		s.parentNode.insertBefore ( k , s );
	} ());
</script>

<!-- KRUX Implementation cont'd -->
<script type="text/javascript">
	window.Krux || (( Krux = function () {
		Krux.q.push ( arguments );
	}).q = []);

	( function () {
		function retrieve ( n ) {
			var m , k = 'kx' + n;
			if ( window.localStorage ) {
				return window.localStorage [ k ] || '';
			} else if ( navigator.cookieEnabled ) {
				m = document.cookie.match ( k + '=([^;]*)' );
				return ( m && unescape ( m [ 1 ])) || '';
			} else {
				return '';
			}
		}

		Krux.user = retrieve ( 'user' );
		Krux.segments = retrieve ( 'segs' ) && retrieve ( 'segs' ).split ( ',' ) || [];
	}) ();
</script>

<!-- Grapeshot Implementation -->
<script>
	var gs_channels = 'default';
</script>

<script src="channels.cgi" async=""></script>

<!-- DoubleClick Implementation -->
<script type="text/javascript">
	var $WileyGooglePublisherID = 2215;

	switch("journal_19399170") {
		case "journal_23798068":
			var $DoubleClickSite = "wly.gn.life.sci_000061";
			var $DoubleClickZone = "cp-plant-bio_CPPB";
			break;
		case "journal_19343663":
			var $DoubleClickSite = "wly.gn.life.sci_000061";
			var $DoubleClickZone = "cp-protein-sci_CPPS";
			break;
		case "journal_19388969":
			var $DoubleClickSite = "wly.gn.life.sci_000061";
			var $DoubleClickZone = "cp-stem-cell-bio_CPSC";
			break;
		case "journal_19349262":
			var $DoubleClickSite = "wly.gn.life.sci_000061";
			var $DoubleClickZone = "cp-toxicology_CPTX";
			break;
		case "journal_25725637":
			var $DoubleClickSite = "wly.gn.life.sci_000061";
			var $DoubleClickZone = "cp-mri_CPMR";
			break;
		case "journal_19343647":
			var $DoubleClickSite = "wly.gn.life.sci_000061";
			var $DoubleClickZone = "cp-molecular-bio_CPMB";
			break;
		case "journal_19348533":
			var $DoubleClickSite = "wly.gn.life.sci_000061";
			var $DoubleClickZone = "cp-microbio_CPMI";
			break;
		case "journal_25725602":
			var $DoubleClickSite = "wly.gn.life.sci_000061";
			var $DoubleClickZone = "cp-food-a-chem_CPFA";
			break;
		case "journal_19342616":
			var $DoubleClickSite = "wly.gn.life.sci_000061";
			var $DoubleClickZone = "cp-cell-bio_CPCL";
			break;
		case "journal_1934340x":
			var $DoubleClickSite = "wly.gn.life.sci_000061";
			var $DoubleClickZone = "cp-bioinfo_CPBI";
			break;
		case "journal_21604762":
			var $DoubleClickSite = "wly.gn.life.sci_000061";
			var $DoubleClickZone = "cp-chem-bio_CPCB";
			break;
		case "journal_19483430":
			var $DoubleClickSite = "wly.gn.life.sci_000061";
			var $DoubleClickZone = "cp-ess-lab-tech_CPLT";
			break;
		case "journal_19349300":
			var $DoubleClickSite = "wly.gn.life.sci_000061";
			var $DoubleClickZone = "cp-cytometry_CPCY";
			break;
		case "journal_19348258":
			var $DoubleClickSite = "wly.gn.life.sci_000061";
			var $DoubleClickZone = "cp-human-gen_CPHG";
			break;
		case "journal_1934368x":
			var $DoubleClickSite = "wly.gn.life.sci_000061";
			var $DoubleClickZone = "cp-immun_CPIM";
			break;
		case "journal_21612617":
			var $DoubleClickSite = "wly.gn.life.sci_000061";
			var $DoubleClickZone = "cp-mouse-b_CPMO";
			break;
		case "journal_19348576":
			var $DoubleClickSite = "wly.gn.life.sci_000061";
			var $DoubleClickZone = "cp-neurosci_CPNE";
			break;
		case "journal_19349289":
			var $DoubleClickSite = "wly.gn.life.sci_000061";
			var $DoubleClickZone = "cp-n-acid-chem_CPNA";
			break;
		case "journal_19348290":
			var $DoubleClickSite = "wly.gn.life.sci_000061";
			var $DoubleClickZone = "cp-pharma_CPPH";
			break;
		case "journal_15518833":
			var $DoubleClickSite = "wly.civ.eng.con_000034";
			var $DoubleClickZone = "jrnl-american-water-works-assc_AWWA";
			break;
		case "journal_25738461":
			var $DoubleClickSite = "wly.cell.m.bio_000030";
			var $DoubleClickZone = "adv-in-cell-and-gene_ACG2";
			break;
		case "journal_15372197":
			var $DoubleClickSite = "wly.plant.sci_000100";
			var $DoubleClickZone = "am-j-o-botany_AJB2";
			break;
		case "journal_18832148":
			var $DoubleClickSite = "wly.cardio.dis_000029";
			var $DoubleClickZone = "j-o-arrhythmia_JOA3";
			break;
		case "journal_21576580":
			var $DoubleClickSite = "wly.cell.m.bio_000030";
			var $DoubleClickZone = "stem-cells-transl-med_SCT3";
			break;
		case "journal_15494918":
			var $DoubleClickSite = "wly.cell.m.bio_000030";
			var $DoubleClickZone = "stem-cells_STEM";
			break;
		case "journal_24750360":
			var $DoubleClickSite = "wly.geriat.ed_000067";
			var $DoubleClickZone = "aging-medicine_AGM2";
			break;
		case "journal_18780369":
			var $DoubleClickSite = "wly.nursing_000088";
			var $DoubleClickZone = "aorn_AORN";
			break;
		case "journal_19412444":
			var $DoubleClickSite = "wly.nutrition_000089";
			var $DoubleClickZone = "j-o-parent-and-enteral-nutrition_JPEN";
			break;
		case "journal_17582652":
			var $DoubleClickSite = "wly.inf.dis.mi_000072";
			var $DoubleClickZone = "j-international-aids-s_JIA2";
			break;
		case "journal_25721143":
			var $DoubleClickSite = "wly.sport.med_000112";
			var $DoubleClickZone = "j-orthoped-res_JOR";
			break;
		case "journal_15518701":
			var $DoubleClickSite = "wly.civ.eng.con_000034";
			var $DoubleClickZone = "opflow_OPFL";
			break;
		case "journal_14781239":
			var $DoubleClickSite = "wly.gastro.hep_000059";
			var $DoubleClickZone = "j-gastro-hepatology_YGH2";
			break;
		case "journal_19383673":
			var $DoubleClickSite = "wly.basic.med_000026";
			var $DoubleClickZone = "jrnl-leukocyte-biology_JLB";
			break;
		case "journal_24758817":
			var $DoubleClickSite = "wly.cell.m.bio_000030";
			var $DoubleClickZone = "peptide-science_PEP2";
			break;
		case "journal_24725390":
			var $DoubleClickSite = "wly.emerge.med_000050";
			var $DoubleClickZone = "academic-emer-med_acem_ET";
			break;
		case "journal_15589293":
			var $DoubleClickSite = "wly.indus.chem_000124";
			var $DoubleClickZone = "jrnl-surfact-detergents_JSDE";
			break;
		case "journal_15589307":
			var $DoubleClickSite = "wly.food.sci_000058";
			var $DoubleClickZone = "lipids_LIPD";
			break;
		case "journal_15589331":
			var $DoubleClickSite = "wly.chemistry_000033";
			var $DoubleClickZone = "j-amer-oil_chem";
			break;
		case "journal_18735924":
			var $DoubleClickSite = "wly.finan.inves_000057";
			var $DoubleClickZone = "rev-finan-econ_RFE";
			break;
		case "journal_25770470":
			var $DoubleClickSite = "wly.elec.e.eng_000049";
			var $DoubleClickZone = "j-quant-eng_QUE2";
			break;
		case "journal_25738615":
			var $DoubleClickSite = "wly.finan.inves_000057";
			var $DoubleClickZone = "Fin-plan-review";
			break;
		case "journal_19412452":
			var $DoubleClickSite = "wly.nutrition_000089";
			var $DoubleClickZone = "nutrition-in-clin-prac_NCP2";
			break;
		case "journal_19419635":
			var $DoubleClickSite = "wly.chem.engin_000032";
			var $DoubleClickZone = "plastics-eng_pe";
			break;
		case "journal_19433638":
			var $DoubleClickSite = "wly.food.sci_000058";
			var $DoubleClickZone = "cereal-chemistry_CCHE";
			break;
		case "journal_19506945":
			var $DoubleClickSite = "wly.neurology_000086";
			var $DoubleClickZone = "epileptic-disorders_EPD2";
			break;
		case "journal_24761281":
			var $DoubleClickSite = "wly.busi.manage_000028";
			var $DoubleClickZone = "consumer-psych-review_ARCP";
			break;
		case "journal_25770470":
			var $DoubleClickSite = "wly.mech.eng_000080";
			var $DoubleClickZone = "quant-engin_QUE";
			break;
		case "journal_25777408":
			var $DoubleClickSite = "wly.math_000079";
			var $DoubleClickZone = "concept-math-meth_CMM";
			break;
		case "journal_19433670":
			var $DoubleClickSite = "wly.dentistry_000041";
			var $DoubleClickZone = "j-of-periodontology_JPER";
			break;
		case "journal_21630097":
			var $DoubleClickSite = "wly.dentistry_000041";
			var $DoubleClickZone = "clin-advances-in-periodon_CAP";
			break;
		case "journal_26374943":
			var $DoubleClickSite = "wly.ecology_000046";
			var $DoubleClickZone = "environmental_DNA";
			break;
		case "journal_15488446":
			var $DoubleClickSite = "wly.aqua.fish_000017";
			var $DoubleClickZone = "fisheries_FSH";
			break;
		case "journal_25722611":
			var $DoubleClickSite = "wly.plant.sci_000100";
			var $DoubleClickZone = "plants-people-planet_PPP";
			break;
		case "journal_19341563":
			var $DoubleClickSite = "wly.anesth.pain_000020";
			var $DoubleClickZone = "j-injury-function-rehab_PMRJ";
			break;
		case "journal_15733599":
			var $DoubleClickSite = "wly.genetics_000065";
			var $DoubleClickZone = "j-gen-conseling_JGC4";
			break;
		case "journal_25778196":
			var $DoubleClickSite = "wly.indust.eng_000071";
			var $DoubleClickZone = "engineer_REPORTS";
			break;
		case "journal_25739832":
			var $DoubleClickSite = "wly.gn.life.sci_000061";
			var $DoubleClickZone = "faseb-bioadvances_FBA2";
			break;
		case "journal_18780261":
			var $DoubleClickSite = "wly.biochem_000130";
			var $DoubleClickZone = "molecular-oncology_MOL2";
			break;
		case "journal_13265377":
			var $DoubleClickSite = "wly.gn.int.med_000062";
			var $DoubleClickZone = "med_jrnl_of_AUS";
			break;
		case "journal_10969896":
			var $DoubleClickSite = "wly.pathology_000093";
			var $DoubleClickZone = "jrnl_path_PATH";
			break;
		case "journal_20564538":
			var $DoubleClickSite = "wly.pathology_000093";
			var $DoubleClickZone = "jrnl_path_CJP2";
			break;
		case "journal_15222454":
			var $DoubleClickSite = "wly.mat.sci_000078";
			var $DoubleClickZone = "vak_res_pract_2035";
			break;
		case "journal_15264610":
			var $DoubleClickSite = "wly.neurology_000086";
			var $DoubleClickZone = "headache_HEAD";
			break;
		case "journal_25749870":
			var $DoubleClickSite = "wly.pharm.med_000096";
			var $DoubleClickZone = "jrnl-amer-college-clin-pharm_JAC5";
			break;
		case "journal_2637496x":
			var $DoubleClickSite = "wly.elec.e.eng_000049";
			var $DoubleClickZone = "information-display_MSID";
			break;
		case "journal_26396181":
			var $DoubleClickSite = "wly.food.sci_000058";
			var $DoubleClickZone = "legume-science_LEG3";
			break;
		case "journal_15732665":
			var $DoubleClickSite = "wly.genetics_000065";
			var $DoubleClickZone = "Jrnl_Inh_Met_Dis_JIMD";
			break;
		case "journal_21928312":
			var $DoubleClickSite = "wly.genetics_000065";
			var $DoubleClickZone = "Jrnl_Inh_Met_Dis_Rep_JIMDreports";
	};

  if (!$DoubleClickSite) {
		var $DoubleClickSite = "undefined-site";
	};

	if (!$DoubleClickZone) {
		var $DoubleClickZone = "undefined-zone";
	};

	var $GooglePublisherCategory = "/" + $WileyGooglePublisherID + "/" + $DoubleClickSite + "/" + $DoubleClickZone;
</script>

<script>
	var googletag = googletag || {};

	googletag.cmd = googletag.cmd || [];

	( function () {

		var gads = document.createElement ( 'script' ) ,
		node = document.getElementsByTagName ( 'script' ) [ 0 ] ;

		gads.async = true;
		gads.src = document.location.protocol + '//www.googletagservices.com/tag/js/gpt.js';

		node.parentNode.insertBefore ( gads , node );

	}) ();
</script>

<script>
	googletag.cmd.push ( function () {

		var sizes = {
			leaderboard : googletag.sizeMapping ()
				.addSize ([ 300 , 100 ] , [ 300 , 50 ])
				.addSize ([ 740 , 100 ] , [ 728 , 90 ])
				.addSize ([ 0 , 0 ] , 'fluid')
				.build () ,
			rail : googletag.sizeMapping ()
				.addSize ([ 0 , 0 ] , [ 300 , 250 ])
				.build ()
		};

		googletag.pubads ()
			.setTargeting ( 'property' 	, 'value-of-property' 	) 	// Change and duplicate this line for as many additional keywords to send
			.setTargeting ( 'doi' , '10.1002/ecy.2000') // Publication DOI
			.setTargeting ( 'gs_cat' 	, gs_channels 		) 	// Grapeshot
			.setTargeting ( 'ksg' 		, Krux.segments 	) 	// Krux
			.setTargeting ( 'kuid' 		, Krux.user 		); 	// Krux

		googletag.defineSlot ( $GooglePublisherCategory, 'fluid', 'advert-leaderboard' )
			.defineSizeMapping ( sizes.leaderboard )
			.addService ( googletag.pubads ())
			.setCollapseEmptyDiv ( true );

		googletag.defineSlot ( $GooglePublisherCategory, 'fluid', 'advert-rail-1' )
			.defineSizeMapping ( sizes.rail )
			.setTargeting ( 'Position', 'advert-rail-1')
			.addService ( googletag.pubads ())
			.setCollapseEmptyDiv ( true );

		googletag.defineSlot ( $GooglePublisherCategory, 'fluid', 'advert-rail-2' )
			.defineSizeMapping ( sizes.rail )
			.setTargeting ( 'Position', 'advert-rail-2')
			.addService ( googletag.pubads ())
			.setCollapseEmptyDiv ( true );

		googletag.pubads ().enableSingleRequest ();
		googletag.enableServices ();
	});
	//]]>
</script>








    
        
    








    


<script>
    var digitalData = window.digitalData ? window.digitalData : {};
    digitalData.page.title = "Deciphering the enigma of undetected species, phylogenetic, and functional diversity based on Good‐Turing theory - Chao - 2017 - Ecology - Wiley Online Library";
</script>

<script>
    tracking = {};

    // events which don't generate server side events!
    tracking.adtmOnlyEvents = ["readcube.share.createlink", "readcube.ecommerce.load", "readcube.ecommerce.new",
        "readcube.ecommerce.userinfo", "readcube.ecommerce.register", "readcube.ecommerce.billing", "readcube.ecommerce.final",
        "readcube.ecommerce.purchasefailed"];

    tracking.ajaxData = function (eventName) {
        return {
            url: "/readcube/log",
            data: {
                "doi": digitalData.publication.item.doi,
                event: eventName
            }
        }
    };

    tracking.trackEvent = function (eventName, eventData, callback) {

        if (!tracking.adtmOnlyEvents.indexOf(eventName) >= 0) {
            tracking.trackUsage(eventName, callback);
        }
        tracking.trackAnalytics(eventName, eventData);
    };

    tracking.trackUsage = function (eventName, callback) {
        pr = $.ajax(tracking.ajaxData(eventName));

        if (callback) {
            pr.always(callback);
        }

    };

    tracking.trackAnalytics = function (eventName, eventData) {
        digitalData = window.digitalData ? digitalData : {};
        digitalData.event = digitalData.event || [];
        digitalData.event.push({
            "eventName": eventName,
            "eventData": eventData
        });
    }
</script>


    <script src="epdf_linker_002.js" async=""></script>
<style type="text/css">.at-icon{fill:#fff;border:0}.at-icon-wrapper{display:inline-block;overflow:hidden}a .at-icon-wrapper{cursor:pointer}.at-rounded,.at-rounded-element .at-icon-wrapper{border-radius:12%}.at-circular,.at-circular-element .at-icon-wrapper{border-radius:50%}.addthis_32x32_style .at-icon{width:2pc;height:2pc}.addthis_24x24_style .at-icon{width:24px;height:24px}.addthis_20x20_style .at-icon{width:20px;height:20px}.addthis_16x16_style .at-icon{width:1pc;height:1pc}#at16lb{display:none;position:absolute;top:0;left:0;width:100%;height:100%;z-index:1001;background-color:#000;opacity:.001}#at_complete,#at_error,#at_share,#at_success{position:static!important}.at15dn{display:none}#at15s,#at16p,#at16p form input,#at16p label,#at16p textarea,#at_share .at_item{font-family:arial,helvetica,tahoma,verdana,sans-serif!important;font-size:9pt!important;outline-style:none;outline-width:0;line-height:1em}* html #at15s.mmborder{position:absolute!important}#at15s.mmborder{position:fixed!important;width:250px!important}#at15s{background:url(data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAoAAAAKCAYAAACNMs+9AAAAGXRFWHRTb2Z0d2FyZQBBZG9iZSBJbWFnZVJlYWR5ccllPAAAABtJREFUeNpiZGBgaGAgAjAxEAlGFVJHIUCAAQDcngCUgqGMqwAAAABJRU5ErkJggg==);float:none;line-height:1em;margin:0;overflow:visible;padding:5px;text-align:left;position:absolute}#at15s a,#at15s span{outline:0;direction:ltr;text-transform:none}#at15s .at-label{margin-left:5px}#at15s .at-icon-wrapper{width:1pc;height:1pc;vertical-align:middle}#at15s .at-icon{width:1pc;height:1pc}.at4-icon{display:inline-block;background-repeat:no-repeat;background-position:top left;margin:0;overflow:hidden;cursor:pointer}.addthis_16x16_style .at4-icon,.addthis_default_style .at4-icon,.at4-icon,.at-16x16{width:1pc;height:1pc;line-height:1pc;background-size:1pc!important}.addthis_32x32_style .at4-icon,.at-32x32{width:2pc;height:2pc;line-height:2pc;background-size:2pc!important}.addthis_24x24_style .at4-icon,.at-24x24{width:24px;height:24px;line-height:24px;background-size:24px!important}.addthis_20x20_style .at4-icon,.at-20x20{width:20px;height:20px;line-height:20px;background-size:20px!important}.at4-icon.circular,.circular .at4-icon,.circular.aticon{border-radius:50%}.at4-icon.rounded,.rounded .at4-icon{border-radius:4px}.at4-icon-left{float:left}#at15s .at4-icon{text-indent:20px;padding:0;overflow:visible;white-space:nowrap;background-size:1pc;width:1pc;height:1pc;background-position:top left;display:inline-block;line-height:1pc}.addthis_vertical_style .at4-icon,.at4-follow-container .at4-icon{margin-right:5px}html>body #at15s{width:250px!important}#at15s.atm{background:none!important;padding:0!important;width:10pc!important}#at15s_inner{background:#fff;border:1px solid #fff;margin:0}#at15s_head{position:relative;background:#f2f2f2;padding:4px;cursor:default;border-bottom:1px solid #e5e5e5}.at15s_head_success{background:#cafd99!important;border-bottom:1px solid #a9d582!important}.at15s_head_success a,.at15s_head_success span{color:#000!important;text-decoration:none}#at15s_brand,#at15sptx,#at16_brand{position:absolute}#at15s_brand{top:4px;right:4px}.at15s_brandx{right:20px!important}a#at15sptx{top:4px;right:4px;text-decoration:none;color:#4c4c4c;font-weight:700}#at15sptx:hover{text-decoration:underline}#at16_brand{top:5px;right:30px;cursor:default}#at_hover{padding:4px}#at_hover .at_item,#at_share .at_item{background:#fff!important;float:left!important;color:#4c4c4c!important}#at_share .at_item .at-icon-wrapper{margin-right:5px}#at_hover .at_bold{font-weight:700;color:#000!important}#at_hover .at_item{width:7pc!important;padding:2px 3px!important;margin:1px;text-decoration:none!important}#at_hover .at_item.athov,#at_hover .at_item:focus,#at_hover .at_item:hover{margin:0!important}#at_hover .at_item.athov,#at_hover .at_item:focus,#at_hover .at_item:hover,#at_share .at_item.athov,#at_share .at_item:hover{background:#f2f2f2!important;border:1px solid #e5e5e5;color:#000!important;text-decoration:none}.ipad #at_hover .at_item:focus{background:#fff!important;border:1px solid #fff}.at15t{display:block!important;height:1pc!important;line-height:1pc!important;padding-left:20px!important;background-position:0 0;text-align:left}.addthis_button,.at15t{cursor:pointer}.addthis_toolbox a.at300b,.addthis_toolbox a.at300m{width:auto}.addthis_toolbox a{margin-bottom:5px;line-height:initial}.addthis_toolbox.addthis_vertical_style{width:200px}.addthis_button_facebook_like .fb_iframe_widget{line-height:100%}.addthis_button_facebook_like iframe.fb_iframe_widget_lift{max-width:none}.addthis_toolbox a.addthis_button_counter,.addthis_toolbox a.addthis_button_facebook_like,.addthis_toolbox a.addthis_button_facebook_send,.addthis_toolbox a.addthis_button_facebook_share,.addthis_toolbox a.addthis_button_foursquare,.addthis_toolbox a.addthis_button_google_plusone,.addthis_toolbox a.addthis_button_linkedin_counter,.addthis_toolbox a.addthis_button_pinterest_pinit,.addthis_toolbox a.addthis_button_tweet{display:inline-block}.at-share-tbx-element .google_plusone_iframe_widget>span>div{vertical-align:top!important}.addthis_toolbox span.addthis_follow_label{display:none}.addthis_toolbox.addthis_vertical_style span.addthis_follow_label{display:block;white-space:nowrap}.addthis_toolbox.addthis_vertical_style a{display:block}.addthis_toolbox.addthis_vertical_style.addthis_32x32_style a{line-height:2pc;height:2pc}.addthis_toolbox.addthis_vertical_style .at300bs{margin-right:4px;float:left}.addthis_toolbox.addthis_20x20_style span{line-height:20px}.addthis_toolbox.addthis_32x32_style span{line-height:2pc}.addthis_toolbox.addthis_pill_combo_style .addthis_button_compact .at15t_compact,.addthis_toolbox.addthis_pill_combo_style a{float:left}.addthis_toolbox.addthis_pill_combo_style a.addthis_button_tweet{margin-top:-2px}.addthis_toolbox.addthis_pill_combo_style .addthis_button_compact .at15t_compact{margin-right:4px}.addthis_default_style .addthis_separator{margin:0 5px;display:inline}div.atclear{clear:both}.addthis_default_style .addthis_separator,.addthis_default_style .at4-icon,.addthis_default_style .at300b,.addthis_default_style .at300bo,.addthis_default_style .at300bs,.addthis_default_style .at300m{float:left}.at300b img,.at300bo img{border:0}a.at300b .at4-icon,a.at300m .at4-icon{display:block}.addthis_default_style .at300b,.addthis_default_style .at300bo,.addthis_default_style .at300m{padding:0 2px}.at300b,.at300bo,.at300bs,.at300m{cursor:pointer}.addthis_button_facebook_like.at300b:hover,.addthis_button_facebook_like.at300bs:hover,.addthis_button_facebook_send.at300b:hover,.addthis_button_facebook_send.at300bs:hover{opacity:1}.addthis_20x20_style .at15t,.addthis_20x20_style .at300bs{overflow:hidden;display:block;height:20px!important;width:20px!important;line-height:20px!important}.addthis_32x32_style .at15t,.addthis_32x32_style .at300bs{overflow:hidden;display:block;height:2pc!important;width:2pc!important;line-height:2pc!important}.at300bs{overflow:hidden;display:block;background-position:0 0;height:1pc;width:1pc;line-height:1pc!important}.addthis_default_style .at15t_compact,.addthis_default_style .at15t_expanded{margin-right:4px}#at_share .at_item{width:123px!important;padding:4px;margin-right:2px;border:1px solid #fff}#at16p{background:url(data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAoAAAAKCAYAAACNMs+9AAAAGXRFWHRTb2Z0d2FyZQBBZG9iZSBJbWFnZVJlYWR5ccllPAAAABtJREFUeNpiZGBgaGAgAjAxEAlGFVJHIUCAAQDcngCUgqGMqwAAAABJRU5ErkJggg==);z-index:10000001;position:absolute;top:50%;left:50%;width:300px;padding:10px;margin:0 auto;margin-top:-185px;margin-left:-155px;font-family:arial,helvetica,tahoma,verdana,sans-serif;font-size:9pt;color:#5e5e5e}#at_share{margin:0;padding:0}#at16pt{position:relative;background:#f2f2f2;height:13px;padding:5px 10px}#at16pt a,#at16pt h4{font-weight:700}#at16pt h4{display:inline;margin:0;padding:0;font-size:9pt;color:#4c4c4c;cursor:default}#at16pt a{position:absolute;top:5px;right:10px;color:#4c4c4c;text-decoration:none;padding:2px}#at15sptx:focus,#at16pt a:focus{outline:thin dotted}#at15s #at16pf a{top:1px}#_atssh{width:1px!important;height:1px!important;border:0!important}.atm{width:10pc!important;padding:0;margin:0;line-height:9pt;letter-spacing:normal;font-family:arial,helvetica,tahoma,verdana,sans-serif;font-size:9pt;color:#444;background:url(data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAoAAAAKCAYAAACNMs+9AAAAGXRFWHRTb2Z0d2FyZQBBZG9iZSBJbWFnZVJlYWR5ccllPAAAABtJREFUeNpiZGBgaGAgAjAxEAlGFVJHIUCAAQDcngCUgqGMqwAAAABJRU5ErkJggg==);padding:4px}.atm-f{text-align:right;border-top:1px solid #ddd;padding:5px 8px}.atm-i{background:#fff;border:1px solid #d5d6d6;padding:0;margin:0;box-shadow:1px 1px 5px rgba(0,0,0,.15)}.atm-s{margin:0!important;padding:0!important}.atm-s a:focus{border:transparent;outline:0;transition:none}#at_hover.atm-s a,.atm-s a{display:block;text-decoration:none;padding:4px 10px;color:#235dab!important;font-weight:400;font-style:normal;transition:none}#at_hover.atm-s .at_bold{color:#235dab!important}#at_hover.atm-s a:hover,.atm-s a:hover{background:#2095f0;text-decoration:none;color:#fff!important}#at_hover.atm-s .at_bold{font-weight:700}#at_hover.atm-s a:hover .at_bold{color:#fff!important}.atm-s a .at-label{vertical-align:middle;margin-left:5px;direction:ltr}.at_PinItButton{display:block;width:40px;height:20px;padding:0;margin:0;background-image:url(//s7.addthis.com/static/t00/pinit00.png);background-repeat:no-repeat}.at_PinItButton:hover{background-position:0 -20px}.addthis_toolbox .addthis_button_pinterest_pinit{position:relative}.at-share-tbx-element .fb_iframe_widget span{vertical-align:baseline!important}#at16pf{height:auto;text-align:right;padding:4px 8px}.at-privacy-info{position:absolute;left:7px;bottom:7px;cursor:pointer;text-decoration:none;font-family:helvetica,arial,sans-serif;font-size:10px;line-height:9pt;letter-spacing:.2px;color:#666}.at-privacy-info:hover{color:#000}.body .wsb-social-share .wsb-social-share-button-vert{padding-top:0;padding-bottom:0}.body .wsb-social-share.addthis_counter_style .addthis_button_tweet.wsb-social-share-button{padding-top:40px}.body .wsb-social-share.addthis_counter_style .addthis_button_google_plusone.wsb-social-share-button{padding-top:0}.body .wsb-social-share.addthis_counter_style .addthis_button_facebook_like.wsb-social-share-button{padding-top:21px}@media print{#at4-follow,#at4-share,#at4-thankyou,#at4-whatsnext,#at4m-mobile,#at15s,.at4,.at4-recommended{display:none!important}}@media screen and (max-width:400px){.at4win{width:100%}}@media screen and (max-height:700px) and (max-width:400px){.at4-thankyou-inner .at4-recommended-container{height:122px;overflow:hidden}.at4-thankyou-inner .at4-recommended .at4-recommended-item:first-child{border-bottom:1px solid #c5c5c5}}</style><style type="text/css">.at-branding-logo{font-family:helvetica,arial,sans-serif;text-decoration:none;font-size:10px;display:inline-block;margin:2px 0;letter-spacing:.2px}.at-branding-logo .at-branding-icon{background-image:url("data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAoAAAAKCAMAAAC67D+PAAAAGXRFWHRTb2Z0d2FyZQBBZG9iZSBJbWFnZVJlYWR5ccllPAAAAAZQTFRF////+GlNUkcc1QAAAB1JREFUeNpiYIQDBjQmAwMmkwEM0JnY1WIxFyDAABGeAFEudiZsAAAAAElFTkSuQmCC")}.at-branding-logo .at-branding-icon,.at-branding-logo .at-privacy-icon{display:inline-block;height:10px;width:10px;margin-left:4px;margin-right:3px;margin-bottom:-1px;background-repeat:no-repeat}.at-branding-logo .at-privacy-icon{background-image:url("data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAkAAAAKCAMAAABR24SMAAAAGXRFWHRTb2Z0d2FyZQBBZG9iZSBJbWFnZVJlYWR5ccllPAAAABhQTFRF8fr9ot/xXcfn2/P5AKva////////AKTWodjhjAAAAAd0Uk5T////////ABpLA0YAAAA6SURBVHjaJMzBDQAwCAJAQaj7b9xifV0kUKJ9ciWxlzWEWI5gMF65KUTv0VKkjVeTerqE/x7+9BVgAEXbAWI8QDcfAAAAAElFTkSuQmCC")}.at-branding-logo span{text-decoration:none}.at-branding-logo .at-branding-addthis,.at-branding-logo .at-branding-powered-by{color:#666}.at-branding-logo .at-branding-addthis:hover{color:#333}.at-cv-with-image .at-branding-addthis,.at-cv-with-image .at-branding-addthis:hover{color:#fff}a.at-branding-logo:visited{color:initial}.at-branding-info{display:inline-block;padding:0 5px;color:#666;border:1px solid #666;border-radius:50%;font-size:10px;line-height:9pt;opacity:.7;transition:all .3s ease;text-decoration:none}.at-branding-info span{border:0;clip:rect(0 0 0 0);height:1px;margin:-1px;overflow:hidden;padding:0;position:absolute;width:1px}.at-branding-info:before{content:'i';font-family:Times New Roman}.at-branding-info:hover{color:#0780df;border-color:#0780df}</style><script src="satellite-5b4e5b3364746d482f001123.js"></script><script src="satellite-5c58af5e64746d5111000925.js"></script><script src="analytics.js"></script><script src="s-code-contents-7d04accfbc0deee918eb104cc0a954ffa836b5dd.js"></script><script async="" type="text/javascript" src="epdf_linker.js"></script><link rel="preload" href="integrator_002.js"><script type="text/javascript" src="integrator_002.js"></script><link rel="preload" href="integrator.js"><script type="text/javascript" src="integrator.js"></script><script src="pubads_impl_2019021401.js" async=""></script><script src="satellite-5ad74ae364746d5a68016a67.js"></script><link href="style.css" type="text/css" rel="stylesheet"><style type="text/css">.fancybox-margin{margin-right:0px;}</style><script type="text/javascript" src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.0/MathJax.js?config=TeX-AMS-MML_HTMLorMML">MathJax.Hub.Config({displayAlign: "left", "HTML-CSS": { linebreaks: {automatic: true,width: "690px"}, fonts: ["STIX-Web"]},"SVG":{ linebreaks: {automatic: true},font: "STIX-Web"},"fast-preview": { disabled: true },tex2jax: {inlineMath: [["\\(", "\\)"]],displayMath: [['\\[', '\\]']],processEscapes: true},styles: {"span.MathJax": {"padding-right": "2px"}}});MathJax.Hub.Startup.onload();</script><style type="text/css">.MathJax_Hover_Frame {border-radius: .25em; -webkit-border-radius: .25em; -moz-border-radius: .25em; -khtml-border-radius: .25em; box-shadow: 0px 0px 15px #83A; -webkit-box-shadow: 0px 0px 15px #83A; -moz-box-shadow: 0px 0px 15px #83A; -khtml-box-shadow: 0px 0px 15px #83A; border: 1px solid #A6D ! important; display: inline-block; position: absolute}
.MathJax_Menu_Button .MathJax_Hover_Arrow {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 4px; -webkit-border-radius: 4px; -moz-border-radius: 4px; -khtml-border-radius: 4px; font-family: 'Courier New',Courier; font-size: 9px; color: #F0F0F0}
.MathJax_Menu_Button .MathJax_Hover_Arrow span {display: block; background-color: #AAA; border: 1px solid; border-radius: 3px; line-height: 0; padding: 4px}
.MathJax_Hover_Arrow:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_Hover_Arrow:hover span {background-color: #CCC!important}
</style><style type="text/css">#MathJax_About {position: fixed; left: 50%; width: auto; text-align: center; border: 3px outset; padding: 1em 2em; background-color: #DDDDDD; color: black; cursor: default; font-family: message-box; font-size: 120%; font-style: normal; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 15px; -webkit-border-radius: 15px; -moz-border-radius: 15px; -khtml-border-radius: 15px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_About.MathJax_MousePost {outline: none}
.MathJax_Menu {position: absolute; background-color: white; color: black; width: auto; padding: 5px 0px; border: 1px solid #CCCCCC; margin: 0; cursor: default; font: menu; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 5px; -webkit-border-radius: 5px; -moz-border-radius: 5px; -khtml-border-radius: 5px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
.MathJax_MenuItem {padding: 1px 2em; background: transparent}
.MathJax_MenuArrow {position: absolute; right: .5em; padding-top: .25em; color: #666666; font-size: .75em}
.MathJax_MenuActive .MathJax_MenuArrow {color: white}
.MathJax_MenuArrow.RTL {left: .5em; right: auto}
.MathJax_MenuCheck {position: absolute; left: .7em}
.MathJax_MenuCheck.RTL {right: .7em; left: auto}
.MathJax_MenuRadioCheck {position: absolute; left: .7em}
.MathJax_MenuRadioCheck.RTL {right: .7em; left: auto}
.MathJax_MenuLabel {padding: 1px 2em 3px 1.33em; font-style: italic}
.MathJax_MenuRule {border-top: 1px solid #DDDDDD; margin: 4px 3px}
.MathJax_MenuDisabled {color: GrayText}
.MathJax_MenuActive {background-color: #606872; color: white}
.MathJax_MenuDisabled:focus, .MathJax_MenuLabel:focus {background-color: #E8E8E8}
.MathJax_ContextMenu:focus {outline: none}
.MathJax_ContextMenu .MathJax_MenuItem:focus {outline: none}
#MathJax_AboutClose {top: .2em; right: .2em}
.MathJax_Menu .MathJax_MenuClose {top: -10px; left: -10px}
.MathJax_MenuClose {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; font-family: 'Courier New',Courier; font-size: 24px; color: #F0F0F0}
.MathJax_MenuClose span {display: block; background-color: #AAA; border: 1.5px solid; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; line-height: 0; padding: 8px 0 6px}
.MathJax_MenuClose:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_MenuClose:hover span {background-color: #CCC!important}
.MathJax_MenuClose:hover:focus {outline: none}
</style><style type="text/css">.MJX_Assistive_MathML {position: absolute!important; top: 0; left: 0; clip: rect(1px, 1px, 1px, 1px); padding: 1px 0 0 0!important; border: 0!important; height: 1px!important; width: 1px!important; overflow: hidden!important; display: block!important; -webkit-touch-callout: none; -webkit-user-select: none; -khtml-user-select: none; -moz-user-select: none; -ms-user-select: none; user-select: none}
.MJX_Assistive_MathML.MJX_Assistive_MathML_Block {width: 100%!important}
</style><style type="text/css">#MathJax_Zoom {position: absolute; background-color: #F0F0F0; overflow: auto; display: block; z-index: 301; padding: .5em; border: 1px solid black; margin: 0; font-weight: normal; font-style: normal; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; -webkit-box-sizing: content-box; -moz-box-sizing: content-box; box-sizing: content-box; box-shadow: 5px 5px 15px #AAAAAA; -webkit-box-shadow: 5px 5px 15px #AAAAAA; -moz-box-shadow: 5px 5px 15px #AAAAAA; -khtml-box-shadow: 5px 5px 15px #AAAAAA; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_ZoomOverlay {position: absolute; left: 0; top: 0; z-index: 300; display: inline-block; width: 100%; height: 100%; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
#MathJax_ZoomFrame {position: relative; display: inline-block; height: 0; width: 0}
#MathJax_ZoomEventTrap {position: absolute; left: 0; top: 0; z-index: 302; display: inline-block; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
</style><style type="text/css">.MathJax_Preview {color: #888}
#MathJax_Message {position: fixed; left: 1px; bottom: 2px; background-color: #E6E6E6; border: 1px solid #959595; margin: 0px; padding: 2px 8px; z-index: 102; color: black; font-size: 80%; width: auto; white-space: nowrap}
#MathJax_MSIE_Frame {position: absolute; top: 0; left: 0; width: 0px; z-index: 101; border: 0px; margin: 0px; padding: 0px}
.MathJax_Error {color: #CC0000; font-style: italic}
span.MathJax {padding-right: 2px}
</style></head>
<body class="pb-ui">

    <a href="#article__content" id="skipNavigationLink" class="skip" tabindex="0">Skip to Main Content</a>




<script type="text/javascript">

    if(true) {
        document.getElementById("skipNavigationLink").onclick =function skipElement () {
            var element = document.getElementById('article__content');
            if(element == null || element == undefined) {
                element = document.getElementsByClassName('article__content').item(0);
            }
            element.setAttribute('tabindex','0');
            element.focus();

        }

    }

</script>    <div id="pb-page-content" data-ng-non-bindable="">
        <div data-pb-dropzone="main" data-pb-dropzone-name="Main">










        <div data-widget-def="pageHeader" data-widget-id="62175478-b98a-43d5-90f2-1f2045ac1a7e" class="pageHeader fixed-element" style="position: fixed;">
        



        
        <header class="page-header">
    <div data-pb-dropzone="main">
    









        <div data-widget-def="ux3-layout-widget" data-widget-id="bb5b5e2c-d3d6-4015-b586-eeade38f59d4" class="admin-message">
        



        
        <div class="container"><div class="row"><div>



        
        


        
    

</div></div></div>

        </div>
    




        
        




        
        <div class="container"><div class="row"><div class="contents col-md-12">









        <div data-widget-def="UX3InstitutionBanner" data-widget-id="4d1f2e23-571a-4b27-8ab7-e7cd8ca15964" class="institution-wrapper-mobile hidden-lg padded-content">
        



        
        <div class="institution"><div class="institution-info-wrapper"><span class="institution__intro">Access by</span><span class="institution__name">Czechs- Biological Institution Of Cas</span></div></div>

        </div>
    




        
        <div class="pull-left">









        <div data-widget-def="ux3-general-image" data-widget-id="ff262197-0fb8-4973-aaa2-2fceba4b5a61" class="logo-container hidden-xs hidden-sm hidden-md">
        



        
        <a href="https://esajournals.onlinelibrary.wiley.com/" title="The Ecological Society of America" tabindex="1"><img alt="The Ecological Society of America" src="logo-header-1508231723743.png"></a>

        </div>
    










        <div data-widget-def="ux3-general-image" data-widget-id="6b601b23-7301-47c8-8889-242d041241c4" class="logo-container-mobile hidden-lg hidden-md hidden-sm">
        



        
        <a href="https://esajournals.onlinelibrary.wiley.com/" title="The Ecological Society of America"><img alt="The Ecological Society of America" src="mobilehublogo-1520994005740.png"></a>

        </div>
    










        <div data-widget-def="UX3InstitutionBanner" data-widget-id="14d0e2f8-27d3-42d0-a477-72015e14e818" class="institution-wrapper hidden-xs hidden-sm hidden-md institution--block">
        



        
        <div class="institution"><div class="institution-info-wrapper"><span class="institution__intro">Access by</span><span class="institution__name">Czechs- Biological Institution Of Cas</span></div></div>

        </div>
    
</div><div class="pull-right">









        <div data-widget-def="UX3QuickSearchWidget" data-widget-id="48175bb0-0f70-46e6-bb89-7e2781fdb8e4" class="header-quicksearch">
        



        
        <div class="quick-search-wrapper"><a href="#" title="search" class="quick-search__item"><i aria-hidden="true" class="icon-search"></i><span class="quick-search__item__arrow"></span></a><div data-linkto-page="true" class="quick-search quickSearchFormContainer"><div class="main-search-field-wrapper" style="overflow: hidden;"><div class="tab clearfix"><input placeholder="Search" tabindex="13" class="fakeQSInput" type="search"><label for="search-within" class="sr-only">Search within</label><select name="search-within" id="search-within" role="tablist" tabindex="10"><option value="#pane-67065c09-4a88-49cd-934c-ac707951d35c0" selected="selected">This Journal</option><option value="#pane-67065c09-4a88-49cd-934c-ac707951d35c1">Anywhere</option></select><ul role="tablist" class="rlist tab__content"><li id="pane-67065c09-4a88-49cd-934c-ac707951d35c0" aria-labelledby="pane-67065c09-4a88-49cd-934c-ac707951d35c0-con" role="tab" class="tab__pane quick-search-pane__anywhere active"><form action="/action/doSearch" name="thisJournalQuickSearch" method="get" title="Quick Search"><div class="input-group option-0"><label for="searchField0" class="hiddenLabel">Search term</label><input name="AllField" id="searchField0" placeholder="Search" onfocus="this.value = this.value;" data-auto-complete-max-words="7" data-auto-complete-max-chars="32" data-contributors-conf="3" data-topics-conf="3" data-publication-titles-conf="3" data-history-items-conf="3" tabindex="9" class="autocomplete actualQSInput ui-autocomplete-input" autocomplete="off" type="search"><span role="status" aria-live="polite" class="ui-helper-hidden-accessible"></span><input name="SeriesKey" value="19399170" type="hidden"><div class="search-options-wrapper"><a href="https://esajournals.onlinelibrary.wiley.com/search/advanced?publication=19399170" title="" tabindex="12" class="advanced">  Advanced Search</a><a href="https://esajournals.onlinelibrary.wiley.com/search/advanced?publication=19399170#citation" title="" tabindex="12" class="citation">  Citation Search</a></div></div><button id="quick-search-btn" type="submit" title="Search" tabindex="11" class="btn quick-search__button icon-search"><span>Search</span></button></form></li><li id="pane-67065c09-4a88-49cd-934c-ac707951d35c1" aria-labelledby="pane-67065c09-4a88-49cd-934c-ac707951d35c1-con" role="tab" class="tab__pane quick-search-pane__anywhere"><form action="/action/doSearch" name="defaultQuickSearch" method="get" title="Quick Search"><div class="input-group option-1"><label for="searchField1" class="hiddenLabel">Search term</label><input name="AllField" id="searchField1" placeholder="Search" onfocus="this.value = this.value;" data-auto-complete-max-words="7" data-auto-complete-max-chars="32" data-contributors-conf="3" data-topics-conf="3" data-publication-titles-conf="3" data-history-items-conf="3" tabindex="9" class="autocomplete actualQSInput ui-autocomplete-input" autocomplete="off" type="search"><span role="status" aria-live="polite" class="ui-helper-hidden-accessible"></span><div class="search-options-wrapper"><a href="https://esajournals.onlinelibrary.wiley.com/search/advanced?publication=19399170" title="" tabindex="12" class="advanced">  Advanced Search</a><a href="https://esajournals.onlinelibrary.wiley.com/search/advanced?publication=19399170#citation" title="" tabindex="12" class="citation">  Citation Search</a></div></div><button id="quick-search-btn" type="submit" title="Search" tabindex="11" class="btn quick-search__button icon-search"><span>Search</span></button></form></li></ul></div></div></div></div>

        </div>
    










        <div data-widget-def="literatumNavigationLoginBar" data-widget-id="d384ca92-7137-45d6-b3c2-78c9c0a98c2b" class="login-bar-container">
        



        
        




    









    <div class="loginBar login-open">
        <a href="https://esajournals.onlinelibrary.wiley.com/action/showLogin?uri=%2Fdoi%2Ffull%2F10.1002%2Fecy.2000" class="show-login" role="button" tabindex="30">
            <span class="icon-profile_empty"></span>
            <span class="sign-in-label">Login / Register</span>
        </a>
            
    </div>



        </div>
    




        
        <a href="#main1" title="menu drawer" data-db-switch="icon-close_thin" data-db-target-for="hub-menu" class="hubpage-menu__burger hidden-lg"><i aria-hidden="true" class="icon-menu2"></i><span></span></a>
</div>
</div></div></div>
    </div>
</header>


        </div>
    




        
        




        
        










        <div data-widget-def="pageBody" data-widget-id="72100436-7a82-49fc-933b-c6c9d8c42914" class="pageBody hub-page-body body-text" style="padding-top: 71px;">
        



        
        <!--begin pagefulltext-->
<div class="page-body pagefulltext">
    <div data-pb-dropzone="main">
    



        
        <div>



        
        <div class="hubpage-menu">
  <nav role="navigation" data-db-target-of="hub-menu" class="container hubpage-menu__nav visible-lg dropBlock__holder">
    <ul class="pull-left rlist--inline">
      <li role="menuitem" tabindex="-1" aria-label="Journals" class="dropdown menu-parent"><a id="main1-1" href="#" title="Journals" data-toggle=" dropdown" data-slide-target="#hubMain-sub-1" class="dropdown__toggle w-slide__btn"><span>Journals</span><i aria-hidden="true" class="icon-section_arrow_d pull-right visible-lg"></i></a>
        <div id="hubMain-sub-1" aria-labelledby="main1-1" aria-hidden="true" role="menu" class="dropdown__menu">
          <div class="menu-items-container">
            <ul class="rlist">
              <li role="menuitem" tabindex="-1" aria-label="Ecology" class="menu-item"><a href="https://esajournals.onlinelibrary.wiley.com/journal/19399170/" title="Ecology"><span>Ecology</span></a></li>
              <li role="menuitem" tabindex="-1" aria-label="Ecological Applications" class="menu-item"><a href="https://esajournals.onlinelibrary.wiley.com/journal/19395582/" title="Ecological Applications"><span>Ecological Applications</span></a></li>
              <li role="menuitem" tabindex="-1" aria-label="Ecological Monographs" class="menu-item"><a href="https://esajournals.onlinelibrary.wiley.com/journal/15577015/" title="Ecological Monographs"><span>Ecological Monographs</span></a></li>
              <li role="menuitem" tabindex="-1" aria-label="Frontiers in Ecology and the Environment" class="menu-item"><a href="https://esajournals.onlinelibrary.wiley.com/journal/15409309/" title="Frontiers in Ecology and the Environment"><span>Frontiers in Ecology and the Environment</span></a></li>
              <li role="menuitem" tabindex="-1" aria-label="Ecosphere" class="menu-item"><a href="https://esajournals.onlinelibrary.wiley.com/journal/21508925/" title="Ecosphere"><span>Ecosphere</span>
                <div class="doi-access-wrapper">
                    <div class="access-type open-access"><i aria-hidden="true" class="icon-lock_open"></i>
                      <div class="doi-access">Open access</div>
                    </div>
                  </div></a></li>
              <li role="menuitem" tabindex="-1" aria-label="The Bulletin of the Ecological Society of America" class="menu-item"><a href="https://esajournals.onlinelibrary.wiley.com/journal/23276096/" title="The Bulletin of the Ecological Society of America"><span>The Bulletin of the Ecological Society of America</span></a></li>
            </ul>
          </div>
        </div>
      </li>
    </ul>
    <div class="pull-right">
      <ul class="rlist--inline pull-right">
        <li><a href="http://www.esa.org/esa/membership-services/joinrenew/" target="_blank">Become a Member</a></li>
        <li><a href="http://www.esa.org/" target="_blank">ESA.org</a></li>

      </ul>
    </div>
  </nav>
</div>

</div>




        
        <section class="hub-journal-bar">



        
        


        <div class="pb-ad">
            <aside role="complementary">
		<div id="advert-leaderboard" class="advert advert-leaderboard">
			<script>
				googletag.cmd.push 		( function () {
					googletag.display 	( 'advert-leaderboard' );
				});
			</script>

			<!-- No JavaScript support is a call to an image -->
			<noscript>
				<a href="/action/clickThrough?id=1013&url=http%3A%2F%2Fpubads.g.doubleclick.net%2Fgampad%2Fjump%3Fiu%3D%24googlePublisherCategory%26amp%3Bsz%3D728x90%26amp%3Btile%3D1%26amp%3Bc%3D%24random&loc=%2Fdoi%2Ffull%2F10.1002%2Fecy.2000&pubId=60241333&placeholderId=1015&productId=101018" target="_blank">
					<img src="http://pubads.g.doubleclick.net/gampad/ad?iu=$googlePublisherCategory&amp;sz=728x90&amp;tile=1&amp;c=$random">
				</a>
			</noscript>
		<div id="google_ads_iframe_/2215/wly.ecology_000046/ecology_ecy_0__container__" style="border: 0pt none; width: 100%; height: 0%;"></div></div>
	</aside>
        </div>
    





        
        
</section>










        <section data-widget-def="ux3-layout-widget" data-widget-id="0f7bccba-8328-4dbd-a1fa-e84c526dd9ab">
        



        
        <div class="container"><div class="row"><div class=" col-md-12">



        
        <div>



        
        <article data-figures="/action/ajaxShowFigures?widgetId=5cf4c79f-0ae9-4dc5-96ce-77f62de7ada9&amp;ajax=true&amp;doi=10.1002%2Fecy.2000&amp;pbContext=%3Bpage%3Astring%3AArticle%2FChapter+View%3Barticle%3Aarticle%3Adoi%5C%3A10.1002%2Fecy.2000%3Bwebsite%3Awebsite%3Aesajournals%3Bjournal%3Ajournal%3A19399170%3Bissue%3Aissue%3Adoi%5C%3A10.1002%2Fecy.2017.98.issue-11%3Bwgroup%3Astring%3APublication+Websites%3BpageGroup%3Astring%3APublication+Pages%3BsubPage%3Astring%3AFull+Text" data-references="/action/ajaxShowEnhancedAbstract?widgetId=5cf4c79f-0ae9-4dc5-96ce-77f62de7ada9&amp;ajax=true&amp;doi=10.1002%2Fecy.2000&amp;pbContext=%3Bpage%3Astring%3AArticle%2FChapter+View%3Barticle%3Aarticle%3Adoi%5C%3A10.1002%2Fecy.2000%3Bwebsite%3Awebsite%3Aesajournals%3Bjournal%3Ajournal%3A19399170%3Bissue%3Aissue%3Adoi%5C%3A10.1002%2Fecy.2017.98.issue-11%3Bwgroup%3Astring%3APublication+Websites%3BpageGroup%3Astring%3APublication+Pages%3BsubPage%3Astring%3AFull+Text" data-related="/action/ajaxShowRecommended?widgetId=5cf4c79f-0ae9-4dc5-96ce-77f62de7ada9&amp;ajax=true&amp;doi=10.1002%2Fecy.2000&amp;pbContext=%3Bpage%3Astring%3AArticle%2FChapter+View%3Barticle%3Aarticle%3Adoi%5C%3A10.1002%2Fecy.2000%3Bwebsite%3Awebsite%3Aesajournals%3Bjournal%3Ajournal%3A19399170%3Bissue%3Aissue%3Adoi%5C%3A10.1002%2Fecy.2017.98.issue-11%3Bwgroup%3Astring%3APublication+Websites%3BpageGroup%3Astring%3APublication+Pages%3BsubPage%3Astring%3AFull+Text&amp;showSubjects=true&amp;taxonomiesCodes=global-subject-codes" data-details="/action/ajaxShowPubInfo?widgetId=5cf4c79f-0ae9-4dc5-96ce-77f62de7ada9&amp;ajax=true&amp;doi=10.1002%2Fecy.2000&amp;pbContext=%3Bpage%3Astring%3AArticle%2FChapter+View%3Barticle%3Aarticle%3Adoi%5C%3A10.1002%2Fecy.2000%3Bwebsite%3Awebsite%3Aesajournals%3Bjournal%3Ajournal%3A19399170%3Bissue%3Aissue%3Adoi%5C%3A10.1002%2Fecy.2017.98.issue-11%3Bwgroup%3Astring%3APublication+Websites%3BpageGroup%3Astring%3APublication+Pages%3BsubPage%3Astring%3AFull+Text&amp;showSubjects=true&amp;taxonomiesCodes=global-subject-codes" data-mathjax="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.0/MathJax.js?config=TeX-AMS-MML_HTMLorMML"><div class="row article-row"><div class="col-sm-12 col-md-8 col-lg-8 article__content article-row-left"><div data-pb-dropzone="mrwBannerDropZone" class="mrw-banner-dropzone">



        
        
</div><div class="article-citation"><div class="citation">
   <div class="pb-dropzone" data-pb-dropzone="publicaitonContent-series-title">
      <div data-pb-dropzone="publicaitonContent-series-title" class="pb-dropzone">
  <h1><a href="https://esajournals.onlinelibrary.wiley.com/journal/19399170" title="Ecology homepage">Ecology</a></h1>
  <a href="https://esajournals.onlinelibrary.wiley.com/toc/19399170/98/11" class="volume-issue">Volume 98, Issue 11</a>
  <a href="https://esajournals.onlinelibrary.wiley.com/journal/19399170" title="Ecology homepage" class="citation--logo"><img class="journal-banner-image" test="test-value" alt="Ecology banner" src="19399170-1501384789330.jpg"></a>
</div>
   </div>
   <div class="doi-access-container clearfix"><span class="primary-heading">Article</span>
      <div class="doi-access-wrapper"><div class="full-access access-type"> <i aria-hidden="true" class="icon-icon-lock_open"></i> <div class="doi-access"> <div class="doi-access" tabindex="0">Full Access</div></div></div></div></div>
   <h2 class="citation__title">Deciphering the enigma of undetected species, phylogenetic, and functional diversity based on Good‐Turing theory</h2>
   <div class="rlist--inline loa comma visible-xs mobile-authors loa-authors-trunc">
      <div class="accordion-tabbed__tab-mobile  accordion__closed"><a href="https://esajournals.onlinelibrary.wiley.com/action/doSearch?ContribAuthorStored=Chao%2C+Anne" class="author-name accordion-tabbed__control" data-id="am1" data-db-target-for="am1" aria-controls="am1" aria-haspopup="true" id="am1_Ctrl"><span>Anne Chao<i aria-hidden="true" class="icon-mail_outline"></i></span><i aria-hidden="true" class="icon-section_arrow_d"></i></a><div class="author-info accordion-tabbed__content" data-db-target-of="am1" aria-labelledby="am1_Ctrl" id="am1">
            <p class="author-type">Corresponding Author</p>
            <p>
               </p><p><span>E-mail address: </span><a title="
                            Link to email address
                        " href="mailto:chao@stat.nthu.edu.tw">chao@stat.nthu.edu.tw</a></p>
            <p></p>
            <p>Institute of Statistics, National Tsing Hua University, Hsin‐Chu, 30043 Taiwan</p>
            <div class="bottom-info">E‐mail: 
               <p><a title="
                            Link to email address
                        " href="mailto:chao@stat.nthu.edu.tw" class="corr-email">chao@stat.nthu.edu.tw</a></p>
            </div><a class="moreInfoLink" href="https://esajournals.onlinelibrary.wiley.com/action/doSearch?ContribAuthorStored=Chao%2C+Anne">Search for more papers by this author</a></div>
      </div>
      <div class="accordion-tabbed__tab-mobile  accordion__closed"><a href="https://esajournals.onlinelibrary.wiley.com/action/doSearch?ContribAuthorStored=Chiu%2C+Chun-Huo" class="author-name accordion-tabbed__control" data-id="am2" data-db-target-for="am2" aria-controls="am2" aria-haspopup="true" id="am2_Ctrl"><span>Chun‐Huo Chiu</span><i aria-hidden="true" class="icon-section_arrow_d"></i></a><div class="author-info accordion-tabbed__content" data-db-target-of="am2" aria-labelledby="am2_Ctrl" id="am2">
            <p>Institute of Statistics, National Tsing Hua University, Hsin‐Chu, 30043 Taiwan</p><a class="moreInfoLink" href="https://esajournals.onlinelibrary.wiley.com/action/doSearch?ContribAuthorStored=Chiu%2C+Chun-Huo">Search for more papers by this author</a></div>
      </div>
      <div class="accordion-tabbed__tab-mobile  accordion__closed"><a href="https://esajournals.onlinelibrary.wiley.com/action/doSearch?ContribAuthorStored=Colwell%2C+Robert+K" class="author-name accordion-tabbed__control" data-id="am3" data-db-target-for="am3" aria-controls="am3" aria-haspopup="true" id="am3_Ctrl"><span>Robert K. Colwell</span><i aria-hidden="true" class="icon-section_arrow_d"></i></a><div class="author-info accordion-tabbed__content" data-db-target-of="am3" aria-labelledby="am3_Ctrl" id="am3">
            <p>Department of Ecology and Evolutionary Biology, University of Connecticut, Storrs, Connecticut, 06269 USA</p>
            <p>University of Colorado Museum of Natural History, Boulder, Colorado, 80309 USA</p>
            <p>Departmento de Ecologia, Universidade Federal de Goiás, , Goiânia, GO, Brasil</p><a class="moreInfoLink" href="https://esajournals.onlinelibrary.wiley.com/action/doSearch?ContribAuthorStored=Colwell%2C+Robert+K">Search for more papers by this author</a></div>
      </div>
      <div class="accordion-tabbed__tab-mobile  accordion__closed"><a href="https://esajournals.onlinelibrary.wiley.com/action/doSearch?ContribAuthorStored=Magnago%2C+Luiz+Fernando+S" class="author-name accordion-tabbed__control" data-id="am4" data-db-target-for="am4" aria-controls="am4" aria-haspopup="true" id="am4_Ctrl"><span>Luiz Fernando S. Magnago</span><i aria-hidden="true" class="icon-section_arrow_d"></i></a><div class="author-info accordion-tabbed__content" data-db-target-of="am4" aria-labelledby="am4_Ctrl" id="am4">
            <p>Departamento de Biologia, Setor de Ecologia e Conservação, Universidade Federal de Lavras, Lavras, 37200‐000 Brasil</p><a class="moreInfoLink" href="https://esajournals.onlinelibrary.wiley.com/action/doSearch?ContribAuthorStored=Magnago%2C+Luiz+Fernando+S">Search for more papers by this author</a></div>
      </div>
      <div class="accordion-tabbed__tab-mobile  accordion__closed"><a href="https://esajournals.onlinelibrary.wiley.com/action/doSearch?ContribAuthorStored=Chazdon%2C+Robin+L" class="author-name accordion-tabbed__control" data-id="am5" data-db-target-for="am5" aria-controls="am5" aria-haspopup="true" id="am5_Ctrl"><span>Robin L. Chazdon</span><i aria-hidden="true" class="icon-section_arrow_d"></i></a><div class="author-info accordion-tabbed__content" data-db-target-of="am5" aria-labelledby="am5_Ctrl" id="am5">
            <p>Department of Ecology and Evolutionary Biology, University of Connecticut, Storrs, Connecticut, 06269 USA</p>
            <p>Department of Ecology and Evolutionary Biology, University of Colorado, Boulder, Colorado, 80309 USA</p><a class="moreInfoLink" href="https://esajournals.onlinelibrary.wiley.com/action/doSearch?ContribAuthorStored=Chazdon%2C+Robin+L">Search for more papers by this author</a></div>
      </div>
      <div class="accordion-tabbed__tab-mobile  accordion__closed"><a href="https://esajournals.onlinelibrary.wiley.com/action/doSearch?ContribAuthorStored=Gotelli%2C+Nicholas+J" class="author-name accordion-tabbed__control" data-id="am6" data-db-target-for="am6" aria-controls="am6" aria-haspopup="true" id="am6_Ctrl"><span>Nicholas J. Gotelli</span><i aria-hidden="true" class="icon-section_arrow_d"></i></a><div class="author-info accordion-tabbed__content" data-db-target-of="am6" aria-labelledby="am6_Ctrl" id="am6">
            <p>Department of Biology, University of Vermont, Burlington, Vermont, 05405 USA</p><a class="moreInfoLink" href="https://esajournals.onlinelibrary.wiley.com/action/doSearch?ContribAuthorStored=Gotelli%2C+Nicholas+J">Search for more papers by this author</a></div>
      </div>
   </div>
   <div class="loa-wrapper loa-authors hidden-xs">
      <div id="sb-1" class="accordion">
         <div class="accordion-tabbed">
            <div class="accordion-tabbed__tab-mobile  accordion__closed"><a href="https://esajournals.onlinelibrary.wiley.com/action/doSearch?ContribAuthorStored=Chao%2C+Anne" class="author-name accordion-tabbed__control" data-id="a1" data-db-target-for="a1" aria-controls="a1" aria-haspopup="true" id="a1_Ctrl"><span>Anne Chao<i aria-hidden="true" class="icon-mail_outline"></i></span><i aria-hidden="true" class="icon-section_arrow_d"></i></a><div class="author-info accordion-tabbed__content" data-db-target-of="a1" aria-labelledby="a1_Ctrl" id="a1">
                  <p class="author-type">Corresponding Author</p>
                  <p>
                     </p><p><span>E-mail address: </span><a title="
                            Link to email address
                        " href="mailto:chao@stat.nthu.edu.tw">chao@stat.nthu.edu.tw</a></p>
                  <p></p>
                  <p>Institute of Statistics, National Tsing Hua University, Hsin‐Chu, 30043 Taiwan</p>
                  <div class="bottom-info">E‐mail: 
                     <p><a title="
                            Link to email address
                        " href="mailto:chao@stat.nthu.edu.tw" class="corr-email">chao@stat.nthu.edu.tw</a></p>
                  </div><a class="moreInfoLink" href="https://esajournals.onlinelibrary.wiley.com/action/doSearch?ContribAuthorStored=Chao%2C+Anne">Search for more papers by this author</a></div>
            </div>
            <div class="accordion-tabbed__tab-mobile  accordion__closed"><a href="https://esajournals.onlinelibrary.wiley.com/action/doSearch?ContribAuthorStored=Chiu%2C+Chun-Huo" class="author-name accordion-tabbed__control" data-id="a2" data-db-target-for="a2" aria-controls="a2" aria-haspopup="true" id="a2_Ctrl"><span>Chun‐Huo Chiu</span><i aria-hidden="true" class="icon-section_arrow_d"></i></a><div class="author-info accordion-tabbed__content" data-db-target-of="a2" aria-labelledby="a2_Ctrl" id="a2">
                  <p>Institute of Statistics, National Tsing Hua University, Hsin‐Chu, 30043 Taiwan</p><a class="moreInfoLink" href="https://esajournals.onlinelibrary.wiley.com/action/doSearch?ContribAuthorStored=Chiu%2C+Chun-Huo">Search for more papers by this author</a></div>
            </div>
            <div class="accordion-tabbed__tab-mobile  accordion__closed"><a href="https://esajournals.onlinelibrary.wiley.com/action/doSearch?ContribAuthorStored=Colwell%2C+Robert+K" class="author-name accordion-tabbed__control" data-id="a3" data-db-target-for="a3" aria-controls="a3" aria-haspopup="true" id="a3_Ctrl"><span>Robert K. Colwell</span><i aria-hidden="true" class="icon-section_arrow_d"></i></a><div class="author-info accordion-tabbed__content" data-db-target-of="a3" aria-labelledby="a3_Ctrl" id="a3">
                  <p>Department of Ecology and Evolutionary Biology, University of Connecticut, Storrs, Connecticut, 06269 USA</p>
                  <p>University of Colorado Museum of Natural History, Boulder, Colorado, 80309 USA</p>
                  <p>Departmento de Ecologia, Universidade Federal de Goiás, , Goiânia, GO, Brasil</p><a class="moreInfoLink" href="https://esajournals.onlinelibrary.wiley.com/action/doSearch?ContribAuthorStored=Colwell%2C+Robert+K">Search for more papers by this author</a></div>
            </div>
            <div class="accordion-tabbed__tab-mobile  accordion__closed"><a href="https://esajournals.onlinelibrary.wiley.com/action/doSearch?ContribAuthorStored=Magnago%2C+Luiz+Fernando+S" class="author-name accordion-tabbed__control" data-id="a4" data-db-target-for="a4" aria-controls="a4" aria-haspopup="true" id="a4_Ctrl"><span>Luiz Fernando S. Magnago</span><i aria-hidden="true" class="icon-section_arrow_d"></i></a><div class="author-info accordion-tabbed__content" data-db-target-of="a4" aria-labelledby="a4_Ctrl" id="a4">
                  <p>Departamento de Biologia, Setor de Ecologia e Conservação, Universidade Federal de Lavras, Lavras, 37200‐000 Brasil</p><a class="moreInfoLink" href="https://esajournals.onlinelibrary.wiley.com/action/doSearch?ContribAuthorStored=Magnago%2C+Luiz+Fernando+S">Search for more papers by this author</a></div>
            </div>
            <div class="accordion-tabbed__tab-mobile  accordion__closed"><a href="https://esajournals.onlinelibrary.wiley.com/action/doSearch?ContribAuthorStored=Chazdon%2C+Robin+L" class="author-name accordion-tabbed__control" data-id="a5" data-db-target-for="a5" aria-controls="a5" aria-haspopup="true" id="a5_Ctrl"><span>Robin L. Chazdon</span><i aria-hidden="true" class="icon-section_arrow_d"></i></a><div class="author-info accordion-tabbed__content" data-db-target-of="a5" aria-labelledby="a5_Ctrl" id="a5">
                  <p>Department of Ecology and Evolutionary Biology, University of Connecticut, Storrs, Connecticut, 06269 USA</p>
                  <p>Department of Ecology and Evolutionary Biology, University of Colorado, Boulder, Colorado, 80309 USA</p><a class="moreInfoLink" href="https://esajournals.onlinelibrary.wiley.com/action/doSearch?ContribAuthorStored=Chazdon%2C+Robin+L">Search for more papers by this author</a></div>
            </div>
            <div class="accordion-tabbed__tab-mobile  accordion__closed"><a href="https://esajournals.onlinelibrary.wiley.com/action/doSearch?ContribAuthorStored=Gotelli%2C+Nicholas+J" class="author-name accordion-tabbed__control" data-id="a6" data-db-target-for="a6" aria-controls="a6" aria-haspopup="true" id="a6_Ctrl"><span>Nicholas J. Gotelli</span><i aria-hidden="true" class="icon-section_arrow_d"></i></a><div class="author-info accordion-tabbed__content" data-db-target-of="a6" aria-labelledby="a6_Ctrl" id="a6">
                  <p>Department of Biology, University of Vermont, Burlington, Vermont, 05405 USA</p><a class="moreInfoLink" href="https://esajournals.onlinelibrary.wiley.com/action/doSearch?ContribAuthorStored=Gotelli%2C+Nicholas+J">Search for more papers by this author</a></div>
            </div>
         </div>
      </div>
   </div>
   <div class="epub-sections">
      <div class="epub-section"><span class="epub-state">First published:&nbsp;</span><span class="epub-date">04 September 2017</span></div>
      <div class="epub-section"><a class="epub-doi" href="https://doi.org/10.1002/ecy.2000">https://doi.org/10.1002/ecy.2000</a></div>
   </div>
   <div class="epub-sections">
      <div class="article-header__references-container no-truncate" id="ecy2000-note-1001">
         <p>Corresponding Editor: Tom E. X. Miller.</p>
      </div>
   </div>
</div></div><nav class="stickybar coolBar trans"><div class="stickybar__wrapper coolBar__wrapper clearfix" style="width: 690px; top: 71px;"><div class="rlist coolBar__zone"><div class="coolBar__section"><a href="#" data-db-target-for="article" data-db-switch="icon-close_thin" aria-haspopup="true" aria-controls="article_Pop" role="button" id="article_Ctrl" class="coolBar__ctrl hidden-lg"><i class="icon-abstract"></i><span>About</span></a><div data-db-target-of="article" aria-labelledby="article_Ctrl" role="menu" id="article_Pop" class="coolBar__drop fixed rlist dropBlock__holder"><div data-target=".article-row-right .tab__nav, .coolBar--download" data-remove="false" data-targetclass="hidden-xs hidden-sm" data-toggle="transplant" data-direction="from" data-transplant="self" class="transplant showit"><div class="transplanted-clone"><ul data-mobile-toggle="slide" class="rlist tab__nav w-slide--list"><li role="presentation" class=""><a data-slide-clone="self" href="#pane-pcw-figures" aria-controls="#pane-pcw-figures" role="tab" data-toggle="tab" id="pane-pcw-figurescon" data-slide-target="#pane-pcw-figures" class="article-row-right__tabLinks w-slide__btn figures-tab empty" aria-expanded="false"><i aria-hidden="true" class="icon-multi-figure"></i><span>Figures</span></a></li><li role="presentation"><a data-slide-clone="self" href="#pane-pcw-references" aria-controls="#pane-pcw-references" role="tab" data-toggle="tab" id="pane-pcw-referencescon" data-slide-target="#pane-pcw-references" class="article-row-right__tabLinks w-slide__btn references-tab empty"><i aria-hidden="true" class="icon-chain"></i><span>References</span></a></li><li role="presentation"><a data-slide-clone="self" href="#pane-pcw-related" aria-controls="#pane-pcw-related" role="tab" data-toggle="tab" id="pane-pcw-relatedcon" data-slide-target="#pane-pcw-related" class="article-row-right__tabLinks w-slide__btn related-tab empty"><i aria-hidden="true" class="icon-related-circles-extend-button"></i><span>Related</span></a></li><li role="presentation" class="active"><a data-slide-clone="self" href="#pane-pcw-details" aria-controls="#pane-pcw-details" role="tab" data-toggle="tab" id="pane-pcw-detailscon" data-slide-target="#pane-pcw-details" class="article-row-right__tabLinks w-slide__btn details-tab" aria-expanded="true"><i aria-hidden="true" class="icon-Icon_Information"></i><span>Information</span></a></li></ul><div class="coolBar__section coolBar--download readcubeEPdfLink"><a href="https://esajournals.onlinelibrary.wiley.com/doi/epdf/10.1002/ecy.2000" title="Article ePDF" class="coolBar__ctrl pdf-download"><img src="pdf-icon.png" alt="ePDF">PDF</a></div><div class="coolBar__section coolBar--download PdfLink pdfLink--is-hidden"><a href="https://esajournals.onlinelibrary.wiley.com/doi/pdf/10.1002/ecy.2000" title="Article PDF" class="coolBar__ctrl pdf-download"><img src="pdf-icon.png" alt="PDF">PDF</a></div></div></div></div></div><div class="coolBar__section coolBar--sections"><a href="#" data-db-target-for="sections" data-db-switch="icon-close_thin" aria-haspopup="true" aria-controls="sections_Pop" role="button" id="sections_Ctrl" class="coolBar__ctrl"><i class="icon-list-unordered"></i><span>Sections</span></a><div data-db-target-of="sections" aria-labelledby="sections_Ctrl" role="menu" id="sections_Pop" class="coolBar__drop dropBlock__holder rlist fixed"><ul class="rlist w-slide--list"><li role="menuitem"><a class="w-slide__hide" href="#d16875178"><span>Abstract</span></a></li><li role="menuitem"><a class="w-slide__hide" href="#ecy2000-sec-0001-title"><span> Introduction</span></a></li><li role="menuitem"><a class="w-slide__hide" href="#ecy2000-sec-0002-title"><span> Species Diversity: One‐Assemblage Good‐Turing Formula</span></a></li><li role="menuitem"><a class="w-slide__hide" href="#ecy2000-sec-0005-title"><span> Species Diversity: Two‐Assemblage Good‐Turing Formula</span></a></li><li role="menuitem"><a class="w-slide__hide" href="#ecy2000-sec-0008-title"><span> Phylogenetic Diversity</span></a></li><li role="menuitem"><a class="w-slide__hide" href="#ecy2000-sec-0011-title"><span> Functional Diversity</span></a></li><li role="menuitem"><a class="w-slide__hide" href="#ecy2000-sec-0014-title"><span> Example</span></a></li><li role="menuitem"><a class="w-slide__hide" href="#ecy2000-sec-0015-title"><span> Conclusion and Discussion</span></a></li><li role="menuitem"><a class="w-slide__hide" href="#ecy2000-sec-0016-title"><span> Acknowledgments</span></a></li><li role="menuitem"><a class="w-slide__hide" href="#support-information-section"><span>
                            Supporting Information
                        </span></a></li><li role="menuitem"><a class="w-slide__hide" href="#references-section-1"><span>Literature Cited</span></a></li></ul></div></div></div><ul data-cb-group="Article" data-cb-group-icon="icon-toc" class="rlist coolBar__first"></ul><div class="coolBar__second rlist"><div class="coolBar__section coolBar--download readcubeEPdfLink cloned"><a href="https://esajournals.onlinelibrary.wiley.com/doi/epdf/10.1002/ecy.2000" title="Article ePDF" class="coolBar__ctrl pdf-download"><img src="pdf-icon.png" alt="ePDF">PDF</a></div><div class="coolBar__section coolBar--download PdfLink pdfLink--is-hidden cloned"><a href="https://esajournals.onlinelibrary.wiley.com/doi/pdf/10.1002/ecy.2000" title="Article PDF" class="coolBar__ctrl pdf-download"><img src="pdf-icon.png" alt="PDF">PDF</a></div>



        
        <div class="relative"><a href="#" data-db-target-for="0e406ce9-5c43-4e2d-b741-e0b43d1f1865" data-db-switch="icon-close_thin" aria-haspopup="true" aria-controls="0e406ce9-5c43-4e2d-b741-e0b43d1f1865_Pop" role="button" id="0e406ce9-5c43-4e2d-b741-e0b43d1f1865_Ctrl" class="article-tools__ctrl"><i aria-hidden="true" class="icon-Icon_Tools"></i><span>Tools</span></a><div data-db-target-of="0e406ce9-5c43-4e2d-b741-e0b43d1f1865" aria-labelledby="0e406ce9-5c43-4e2d-b741-e0b43d1f1865_Ctrl" role="menu" id="0e406ce9-5c43-4e2d-b741-e0b43d1f1865_Pop" class="article-tools__block fixed dropBlock__holder"><ul class="rlist w-slide--list"><li role="presentation" class="article-tool"><a href="https://esajournals.onlinelibrary.wiley.com/action/rightsLink?doi=10.1002%2Fecy.2000&amp;mode=" role="menuitem" target="_blank"><i aria-hidden="true" class="icon-permissions-key"></i><span>Request permission</span></a></li><li role="presentation" class="article-tool"><a href="https://esajournals.onlinelibrary.wiley.com/action/showCitFormats?doi=10.1002%2Fecy.2000" role="menuitem" target="_self"><i aria-hidden="true" class="icon-export-icon"></i><span>Export citation</span></a></li><li role="presentation" class="article-tool"><a href="https://esajournals.onlinelibrary.wiley.com/personalize/addFavoritePublication?doi=10.1002%2Fecy.2000" role="menuitem" target="_self"><i aria-hidden="true" class="icon-favorites"></i><span>Add to favorites</span></a></li><li role="presentation" class="article-tool"><a href="https://esajournals.onlinelibrary.wiley.com/action/addCitationAlert?doi=10.1002%2Fecy.2000" role="menuitem" target="_self"><i aria-hidden="true" class="icon-Icon_Track-citations"></i><span>Track citation</span></a></li></ul></div></div>




        
        <!-- Go to www.addthis.com/dashboard to customize your tools --><script type="text/javascript" src="addthis_widget.js"></script><div class="relative"><a id="share__ctrl" href="#" data-db-target-for="dff42abe-4b17-4fe9-a9c5-625405cd1007" data-db-switch="icon-close_thin" aria-haspopup="true" aria-controls="share_Pop" role="button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i><span>Share</span></a><div data-db-target-of="dff42abe-4b17-4fe9-a9c5-625405cd1007" aria-labelledby="share__ctrl" role="menu" id="share_Pop" class="share__block dropBlock__holder fixed">



        
        <div data-readcube="/pb/widgets/readCubeP2PUrl/10.1002/ecy.2000/" class="readCube-sharing hidden"><p>Give access</p><a href="#" data-toggle="modal" data-target="#readCube" class="readCube-sharing__link"><i class="icon-users"></i>Share full text access</a></div><div class="ux-modal-container readCube-sharing__modal"><div id="readCube" class="modal"><div class="modal__dialog"><div class="modal__header"><button type="button" data-dismiss="modal" class="close"><i aria-hidden="true" class="icon-close_thin"></i></button><p>Share full text access</p></div><div class="modal__body"><form method="post"><fieldset><legend>Please review our <a href="https://esajournals.onlinelibrary.wiley.com/termsAndConditions" target="_blank">Terms and Conditions of Use</a> and check box below to share full-text version of article.</legend><div class="input-group"><label for="terms-and-conditions" class="checkbox--primary"><input id="terms-and-conditions" value="yes" required="" name="terms-and-conditions" type="checkbox"><span class="label-txt">I have read and accept the Wiley Online Library Terms and Conditions of Use.</span></label></div></fieldset><hr class="separator"><fieldset><label>Shareable Link</label><p>Use the link below to share a full-text version of this article with your friends and colleagues. <a href="https://esajournals.onlinelibrary.wiley.com/researchers/tools-resources/sharing" target="_blank" class="emphasis">Learn more.</a></p><div class="shareable-link"><div class="shareable-link__field"><span id="shareable-link__text"></span></div></div><button type="submit" disabled="disabled" class="btn shareable-link__btn">Copy URL</button></fieldset></form></div></div></div></div>
<p>Share a link</p><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a class="addthis_button_email" target="_blank" title="Email" href="#"><i aria-hidden="true" class="at-icon-wrapper icon-Email"></i>Email to a friend</a></li><li><a class="addthis_button_facebook" title="Facebook" href="#"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a class="addthis_button_twitter" title="Twitter" href="#"><i aria-hidden="true" class="at-icon-wrapper icon-twitter-t"></i>Twitter</a></li><li><a class="addthis_button_linkedin" target="_blank" title="LinkedIn" href="#"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linkedin</a></li><li><a class="addthis_button_google_plusone_share"><i aria-hidden="true" class="at-icon-wrapper icon-google-plus"></i>Google+</a></li><li><a class="addthis_button_reddit" target="_blank" title="Reddit" href="#"><i aria-hidden="true" class="at-icon-wrapper icon-reddit-alien"></i>Reddit</a></li><li><a class="addthis_button_citeulike" target="_blank" title="CiteULike" href="#"><i aria-hidden="true" class="at-icon-wrapper icon-citeulike"></i>CiteULike</a></li><div class="atclear"></div></ul></div></div>
</div></div></nav><div id="accessDenialslot" class="accessDenialslot"></div><div class="article__body"><style>
            .inline-equation {
            overflow-x:auto;
            }
        </style><article>
   
   <section class="article-section article-section__abstract" data-lang="en" id="section-1-en" style="" lang="en">
      <h2 id="d16875178" class="article-section__header section__title main main">Abstract</h2>
      <div class="article-section__content en main">
         
         <p>Estimating the species, phylogenetic, and functional 
diversity of a community is challenging because rare species are often 
undetected, even with intensive sampling. The Good‐Turing frequency 
formula, originally developed for cryptography, estimates in an 
ecological context the true frequencies of rare species in a single 
assemblage based on an incomplete sample of individuals. Until now, this
 formula has never been used to estimate undetected species, 
phylogenetic, and functional diversity. Here, we first generalize the 
Good‐Turing formula to incomplete sampling of two assemblages. The 
original formula and its two‐assemblage generalization provide a novel 
and unified approach to notation, terminology, and estimation of 
undetected biological diversity. For species richness, the Good‐Turing 
framework offers an intuitive way to derive the non‐parametric 
estimators of the undetected species richness in a single assemblage, 
and of the undetected species shared between two assemblages. For 
phylogenetic diversity, the unified approach leads to an estimator of 
the undetected Faith's phylogenetic diversity (<i>PD</i>, the total 
length of undetected branches of a phylogenetic tree connecting all 
species), as well as a new estimator of undetected <i>PD</i> shared 
between two phylogenetic trees. For functional diversity based on 
species traits, the unified approach yields a new estimator of 
undetected Walker et&nbsp;al.'s functional attribute diversity (<i>FAD</i>, the total species‐pairwise functional distance) in a single assemblage, as well as a new estimator of undetected <i>FAD</i>
 shared between two assemblages. Although some of the resulting 
estimators have been previously published (but derived with traditional 
mathematical inequalities), all taxonomic, phylogenetic, and functional 
diversity estimators are now derived under the same framework. All the 
derived estimators are theoretically lower bounds of the corresponding 
undetected diversities; our approach reveals the sufficient conditions 
under which the estimators are nearly unbiased, thus offering new 
insights. Simulation results are reported to numerically verify the 
performance of the derived estimators. We illustrate all estimators and 
assess their sampling uncertainty with an empirical dataset for 
Brazilian rain forest trees. These estimators should be widely 
applicable to many current problems in ecology, such as the effects of 
climate change on spatial and temporal beta diversity and the 
contribution of trait diversity to ecosystem multi‐functionality.</p>
         
      </div>
   </section>
   
   <section class="article-section article-section__full">
      
      <div class="article-section__content" id="ecy2000-sec-0001">
         
         <h2 class="article-section__title section__title section1" id="ecy2000-sec-0001-title"> Introduction</h2>
         
         <p>Nearly all biodiversity studies and analyses are based on 
sampling data taken from focal assemblages. However, due to practical 
limitations, it is virtually impossible to detect all species, 
especially in hyper‐diverse assemblages with many rare species. In 
almost every biodiversity survey and monitoring project, some proportion
 of the species that are present fail to be detected. Not only the 
presence, but also the functional traits of these species remain 
undetected. Moreover, the placement of undetected species on the 
phylogenetic tree of the observed species is unknown. Consequently, 
traditional measures of species, functional, and phylogenetic diversity 
from sample data typically underestimate the true diversities (observed 
plus undetected). The magnitude of this negative bias can be 
substantial.</p>
         
         <p>For species diversity, the estimation of undetected richness
 based on incomplete samples from a single assemblage has been widely 
applied, not only in ecology and conservation biology, but also in many 
other disciplines; see Colwell and Coddington (<span><a href="#ecy2000-bib-0020" class="bibLink tab-link" data-tab="pane-pcw-references">1994</a></span>), Chazdon et&nbsp;al. (<span><a href="#ecy2000-bib-0017" class="bibLink tab-link" data-tab="pane-pcw-references">1998</a></span>), Magurran (<span><a href="#ecy2000-bib-0039" class="bibLink tab-link" data-tab="pane-pcw-references">2004</a></span>), Chao (<span><a href="#ecy2000-bib-0007" class="bibLink tab-link" data-tab="pane-pcw-references">2005</a></span>), Hortal et&nbsp;al. (<span><a href="#ecy2000-bib-0033" class="bibLink tab-link" data-tab="pane-pcw-references">2006</a></span>), Gotelli and Colwell (<span><a href="#ecy2000-bib-0030" class="bibLink tab-link" data-tab="pane-pcw-references">2011</a></span>), Gotelli and Chao (<span><a href="#ecy2000-bib-0029" class="bibLink tab-link" data-tab="pane-pcw-references">2013</a></span>) and Chao and Chiu (<span><a href="#ecy2000-bib-0009" class="bibLink tab-link" data-tab="pane-pcw-references">2016</a></span>)
 for various applications. For two assemblages, shared species richness 
plays an important role in assessing assemblage overlap and forms a 
basis for constructing various types of beta diversity and 
(dis)similarity measures, such as the classic Sørensen and Jaccard 
indices (Colwell and Coddington <span><a href="#ecy2000-bib-0020" class="bibLink tab-link" data-tab="pane-pcw-references">1994</a></span>, Magurran <span><a href="#ecy2000-bib-0039" class="bibLink tab-link" data-tab="pane-pcw-references">2004</a></span>, Jost et&nbsp;al. <span><a href="#ecy2000-bib-0036" class="bibLink tab-link" data-tab="pane-pcw-references">2011</a></span>, Gotelli and Chao <span><a href="#ecy2000-bib-0029" class="bibLink tab-link" data-tab="pane-pcw-references">2013</a></span>).
 Compared with estimating species richness in a single assemblage, the 
estimation of shared species richness, taking undetected species into 
account, has received relatively little attention; see Chao and Chiu (<span><a href="#ecy2000-bib-0008" class="bibLink tab-link" data-tab="pane-pcw-references">2012</a></span>) for a review.
         </p>
         
         <p>In traditional measures of species diversity, all species 
(or taxa at some other rank) are considered to be equally distinct from 
one another. Species differences can be based directly on their 
evolutionary histories, either in the form of taxonomic classification 
or well‐supported phylogenetic trees. A rapidly growing literature 
addresses phylogenetic diversity metrics and related (dis)similarity 
measures; see Cavender‐Bares et&nbsp;al. (<span><a href="#ecy2000-bib-0004" class="bibLink tab-link" data-tab="pane-pcw-references">2012</a></span>) for a review. A widely used phylogenetic metric is Faith's (<span><a href="#ecy2000-bib-0023" class="bibLink tab-link" data-tab="pane-pcw-references">1992</a></span>) <i>PD</i>
 (phylogenetic diversity), which is defined as the sum of the branch 
lengths of a phylogenetic tree connecting all species in the target 
assemblage. Throughout this paper, <i>PD</i> refers to Faith's (<span><a href="#ecy2000-bib-0023" class="bibLink tab-link" data-tab="pane-pcw-references">1992</a></span>) <i>PD</i>. For most data sets, <i>PD</i> is highly correlated with species richness (e.g., Matos et&nbsp;al. <span><a href="#ecy2000-bib-0040" class="bibLink tab-link" data-tab="pane-pcw-references">2017</a></span>).
 When a sample fails to detect all species present, the 
lineages/branches associated with these undetected species are also 
missing from the phylogenetic tree of the observed species. The 
undetected <i>PD</i> in an incomplete sample was not discussed until recent years (Cardoso et&nbsp;al. <span><a href="#ecy2000-bib-0003" class="bibLink tab-link" data-tab="pane-pcw-references">2014</a></span>, Chao et&nbsp;al. <span><a href="#ecy2000-bib-0014" class="bibLink tab-link" data-tab="pane-pcw-references">2015a</a></span>).
         </p>
         
         <p>The phylogenetic version of the Jaccard dissimilarity index is referred to as the <i>UniFrac</i> measure, developed by Lozupone and Knight (<span><a href="#ecy2000-bib-0037" class="bibLink tab-link" data-tab="pane-pcw-references">2005</a></span>). The phylogenetic version of the Sørensen similarity index is referred to as the <i>PhyloSør</i> (phylo‐Sørensen) index, developed by Bryant et&nbsp;al. (<span><a href="#ecy2000-bib-0001" class="bibLink tab-link" data-tab="pane-pcw-references">2008</a></span>) and Ferrier et&nbsp;al. (<span><a href="#ecy2000-bib-0024" class="bibLink tab-link" data-tab="pane-pcw-references">2007</a></span>).
 All these phylogenetic (dis)similarity measures are based on the shared
 branch lengths between two phylogenetic trees. However, to our 
knowledge, the estimation of the undetected shared <i>PD</i> (i.e., the 
total length of undetected branches shared by two phylogenetic trees) 
has not previously been discussed in the literature.
         </p>
         
         <p>When species are described by a set of traits that affect 
organismal and/or ecosystem functioning, pairwise species differences 
within an assemblage can also be measured by the dissimilarity or 
distances between their trait profiles, which can be weighted or 
unweighted by their abundances. Functional diversity or trait diversity 
quantifies the diversity of species’ traits among coexisting species in 
an assemblage (Tilman et&nbsp;al. <span><a href="#ecy2000-bib-0049" class="bibLink tab-link" data-tab="pane-pcw-references">1997</a></span>, Diaz and Cabido <span><a href="#ecy2000-bib-0022" class="bibLink tab-link" data-tab="pane-pcw-references">2001</a></span>, Swenson et&nbsp;al. <span><a href="#ecy2000-bib-0048" class="bibLink tab-link" data-tab="pane-pcw-references">2012</a></span>).
 Functional diversity is regarded as key to understanding ecosystem 
processes and their response to environmental stress or disturbance 
(Cadotte et&nbsp;al. <span><a href="#ecy2000-bib-0002" class="bibLink tab-link" data-tab="pane-pcw-references">2009</a></span>). A distance‐based measure at the assemblage level for quantifying functional diversity is <i>FAD</i> (functional attribute diversity, as defined by Walker et&nbsp;al. <span><a href="#ecy2000-bib-0050" class="bibLink tab-link" data-tab="pane-pcw-references">1999</a></span>), which is the sum of the species‐pairwise functional distances. A modified version called <i>MFAD</i> (modified <i>FAD</i>) was proposed by Schmera et&nbsp;al. (<span><a href="#ecy2000-bib-0047" class="bibLink tab-link" data-tab="pane-pcw-references">2009</a></span>),
 who replaced species with “functional units” (here the collection of 
all species with identical traits is regarded as a single functional 
unit). For simplicity, we focus on the estimation of <i>FAD</i>, but a 
similar approach can be applied if species are replaced by functional 
units or other clusters of species. For incomplete samples, the traits 
of undetected species are missing, and thus their pairwise distances are
 not recorded and cannot be considered in the observed <i>FAD</i>. As far as we are aware, there have been no estimators previously developed for undetected <i>FAD</i> in a single assemblage or undetected <i>FAD</i> shared by two assemblages.
         </p>
         
         <p>In his famous cryptanalysis to crack German ciphers during 
World War II, Alan Turing, regarded as the founder of modern computer 
sciences, developed novel statistical methods to estimate the true 
frequencies of rare code elements (including still‐undetected code 
elements), based on the observed frequencies in “samples” of intercepted
 Nazi code. According to Good (<span><a href="#ecy2000-bib-0025" class="bibLink tab-link" data-tab="pane-pcw-references">1953</a></span>, <span><a href="#ecy2000-bib-0027" class="bibLink tab-link" data-tab="pane-pcw-references">2000</a></span>),
 Turing never published his wartime statistical work, but permitted Good
 to publish it after the war. The two influential papers by Good (<span><a href="#ecy2000-bib-0025" class="bibLink tab-link" data-tab="pane-pcw-references">1953</a></span>) and Good and Toulmin (<span><a href="#ecy2000-bib-0028" class="bibLink tab-link" data-tab="pane-pcw-references">1956</a></span>)
 presented Turing's wartime statistical work on the frequency formula 
and related topics. The frequency formula is now referred to as the 
Good‐Turing frequency formula, which has a wide range of applications in
 biological sciences, statistics, computer sciences, information 
sciences, and linguistics, among others (McGrayne <span><a href="#ecy2000-bib-0041" class="bibLink tab-link" data-tab="pane-pcw-references">2011</a></span>, p. 100).
         </p>
         
         <p>In an ecological context, Turing's statistical problem can 
be formulated as an estimation of the true frequencies of rare species 
when a random sample of individuals is drawn from an assemblage. In 
Turing's case, the species abundances are highly heterogeneous, with 
many rare species, so that all samples have undetected species. The 
Good‐Turing formula answers the following question: given a species that
 appears <i>r</i> times (<i>r&nbsp;</i>=<i>&nbsp;</i>0, 1, 2, …) in an incomplete sample of <i>n</i>
 individuals, what is its true relative frequency in the entire 
assemblage? As will be described below, Turing gave a surprising answer 
that is contrary to most people's intuition.
         </p>
         
         <p>Until now, the Good‐Turing formula has never been applied to
 estimate undetected species, phylogenetic, and functional diversity. In
 this paper, we generalize the Good‐Turing formula, originally developed
 for a single assemblage, to two assemblages. We also extend the 
Good‐Turing formula to a phylogenetic version that incorporates 
evolutionary history among species as well as a functional version that 
takes into account functional traits associated with each species. We 
show in this paper that the Good‐Turing formula and its generalizations 
can be used in a unified way to derive estimators of undetected species,
 phylogenetic, and functional diversities based on incomplete samples.</p>
         
         <p>For species diversity, we apply the Good‐Turing formula to 
intuitively derive an estimator of the number of undetected species in 
an assemblage. The resulting estimator turns out to be the Chao (<span><a href="#ecy2000-bib-0005" class="bibLink tab-link" data-tab="pane-pcw-references">1984</a></span>) non‐parametric lower bound. The two‐assemblage generalized formula yields Pan et&nbsp;al.'s (<span><a href="#ecy2000-bib-0042" class="bibLink tab-link" data-tab="pane-pcw-references">2009</a></span>)
 lower bound of the number of undetected shared species when a sample of
 individuals is taken from each of two assemblages. For phylogenetic 
diversity, the unified approach yields a recently published estimator of
 undetected <i>PD</i> in a single assemblage (Cardoso et&nbsp;al. <span><a href="#ecy2000-bib-0003" class="bibLink tab-link" data-tab="pane-pcw-references">2014</a></span>, Chao et&nbsp;al. <span><a href="#ecy2000-bib-0014" class="bibLink tab-link" data-tab="pane-pcw-references">2015a</a></span>). The two‐assemblage generalized formula yields a new estimator of the undetected <i>PD</i>
 shared between two assemblages. For functional diversity based on 
species traits, the formulas yield a new estimator of undetected <i>FAD</i> in a single assemblage as well as a new estimator of undetected <i>FAD</i> shared between two assemblages.
         </p>
         
         <p>All the resulting Good‐Turing estimators of undetected 
biodiversity measures (including three previously published, derived 
from traditional mathematical inequalities, and three new ones) are 
theoretically lower bounds of the corresponding undetected diversities. 
Good‐Turing's perspectives also reveal the sufficient conditions under 
which the derived estimators are nearly unbiased. It would be impossible
 to reveal these conditions with the traditional derivations of the 
corresponding estimators from inequalities. Thus, Good‐Turing 
perspectives further justify and provide new insights for the three 
established estimators.</p>
         
         <p>An important advance beyond previous studies is that 
Good‐Turing's perspective here provides a novel and unified approach to 
the notation, terminology, and estimation of undetected diversity. All 
taxonomic, phylogenetic and functional diversity estimators are now 
derived and linked under the same framework. Simulation results are 
reported here to numerically validate Good‐Turing theory and examine the
 performance of the derived estimators. An empirical dataset for tree 
species, collected in Brazilian rain forest, is used to illustrate all 
estimators.</p>
         
      </div>
      
      <div class="article-section__content" id="ecy2000-sec-0002">
         
         <h2 class="article-section__title section__title section1" id="ecy2000-sec-0002-title"> Species Diversity: One‐Assemblage Good‐Turing Formula</h2>
         
         <div class="article-section__sub-content" id="ecy2000-sec-0003">
            
            <h3 class="article-section__sub-title section2"> The original Good‐Turing formula</h3>
            
            <p>In the one‐assemblage model formulation, we assume that there are <i>S</i> species and their true species relative abundances (or generally, species detection probabilities) are denoted by (<i>p</i><sub>1</sub>, <i>p</i><sub>2</sub>, …, <i>p</i><sub><i>S</i></sub>), <img alt="urn:x-wiley:00129658:media:ecy2000:ecy2000-math-0001" class="section_image" src="ecy2000-math-0001.png">. Assume a sample of <i>n</i> individuals is selected with replacement. Here we follow Good‐Turing's original model (Good <span><a href="#ecy2000-bib-0025" class="bibLink tab-link" data-tab="pane-pcw-references">1953</a></span>)
 in which the detection probability of each species is simply its 
relative abundance; all our derivations can be directly extended to a 
general model in which species detection probability is proportional to 
the product of its abundance and individual detectability; see <i>Discussion</i>. Let <i>X</i><sub><i>i</i></sub> denote the species frequency (abundance) of the <i>i‐</i>th species in the sample, <i>i&nbsp;</i>=<i>&nbsp;</i>1, 2, …, <i>S</i>, <img alt="urn:x-wiley:00129658:media:ecy2000:ecy2000-math-0002" class="section_image" src="ecy2000-math-0002.png">. Only species with frequency <i>X&nbsp;</i>&gt;<i>&nbsp;</i>0 in the sample are detected. Those species with frequency <i>X&nbsp;</i>=<i>&nbsp;</i>0
 in the sample are not detected (although they are present in the 
assemblage) and are therefore not included in the sample data.
            </p>
            
            <p>We define the <i>abundance frequency count f</i><sub><i>r</i></sub> as the number of species each represented by exactly <i>r</i> individuals in the sample. Good (<span><a href="#ecy2000-bib-0025" class="bibLink tab-link" data-tab="pane-pcw-references">1953</a></span>) referred to it as “the frequency of frequency <i>r</i>.” Thus <i>f</i><sub>1</sub> is the number of “singletons” (those species that are represented by exactly 1 individual in the sample), and <i>f</i><sub>2</sub> is the number of “doubletons” (those that are represented by exactly 2 individuals in the sample). Let <i>S</i><sub><i>obs</i></sub> denote the total number of those species observed in the sample; <img alt="urn:x-wiley:00129658:media:ecy2000:ecy2000-math-0003" class="section_image" src="ecy2000-math-0003.png">. Also, <i>f</i><sub>0</sub> is the number of undetected species: species that are present in the assemblage of <i>S</i> species, but were not detected in the sample of <i>n</i> individuals and <i>S</i><sub><i>obs</i></sub> species. Therefore, we have <i>S</i><sub><i>obs</i></sub>&nbsp;+&nbsp;<i>f</i><sub>0</sub>&nbsp;=&nbsp;<i>S</i>.
            </p>
            
            <p>Turing's statistical problem can be formulated as follows. Given data, for those species that each appeared exactly <i>r</i> times (<i>r&nbsp;</i>=<i>&nbsp;</i>0, 1, 2, …) in an incomplete sample of <i>n</i> individuals, the mean of their true relative abundances/frequencies in the assemblage, α<sub><i>r</i></sub>, can be mathematically expressed as
               </p><div class="inline-equation" id="ecy2000-disp-0001"><span class="inline-equation__construct"><img class="" src="ecy2000-math-0004.png" alt="urn:x-wiley:00129658:media:ecy2000:ecy2000-math-0004" title="urn:x-wiley:00129658:media:ecy2000:ecy2000-math-0004"></span><span class="inline-equation__label">(1a)</span></div>where <i>I</i>(<i>A</i>) is the indicator function, i.e., <i>I</i>(<i>A</i>) = 1 if the event <i>A</i> occurs, and 0 otherwise. The numerator in Eq.&nbsp;(<a class="equationLink scrollableLink" href="#ecy2000-disp-0001" title="Link to equation">1a</a>) represents the total true relative frequencies of those species that each appeared exactly <i>r</i> times in the sample. Dividing the total by <i>f</i><sub><i>r</i></sub>, we obtain the mean (per species) of their relative frequencies. Turing and Good focused on the case of small <i>r</i>, i.e., rare species (or rare code elements, in Turing's case). Note that for the special case of <i>r&nbsp;</i>=<i>&nbsp;</i>0, Eq.&nbsp;(<a class="equationLink scrollableLink" href="#ecy2000-disp-0001" title="Link to equation">1a</a>) implies
               <div class="inline-equation" id="ecy2000-disp-0002"><span class="inline-equation__construct"><img class="" src="ecy2000-math-0005.png" alt="urn:x-wiley:00129658:media:ecy2000:ecy2000-math-0005" title="urn:x-wiley:00129658:media:ecy2000:ecy2000-math-0005"></span><span class="inline-equation__label">(1b)</span></div>which is the “coverage deficit” (Chao and Jost <span><a href="#ecy2000-bib-0011" class="bibLink tab-link" data-tab="pane-pcw-references">2012</a></span>) or the complement of the “sample coverage” defined in Good (<span><a href="#ecy2000-bib-0025" class="bibLink tab-link" data-tab="pane-pcw-references">1953</a></span>).
 The coverage deficit of the sample quantifies the proportion of the 
total individuals in the assemblage that belong to undetected species; 
it is also the probability that a new, previously‐undetected species 
would be found if the sample were enlarged by one individual. This is a 
very important measure in diversity estimation (Chao and Jost <span><a href="#ecy2000-bib-0011" class="bibLink tab-link" data-tab="pane-pcw-references">2012</a></span>).
            <p></p>
            
            <p>Turing and Good discovered a surprisingly simple and remarkably effective, although non‐intuitive, estimator for α<sub><i>r</i></sub>. The Good‐Turing frequency formula states that α<sub><i>r</i></sub>, <i>r&nbsp;</i>=<i>&nbsp;</i>0, 1, 2, …, is not estimated by its sample frequency <i>r</i>/<i>n</i>, but rather by
               </p><div class="inline-equation" id="ecy2000-disp-0003"><span class="inline-equation__construct"><img class="" src="ecy2000-math-0006.png" alt="urn:x-wiley:00129658:media:ecy2000:ecy2000-math-0006" title="urn:x-wiley:00129658:media:ecy2000:ecy2000-math-0006"></span><span class="inline-equation__label">(1c)</span></div>
            <p></p>
            
            <p>In other words, α<sub><i>r</i></sub> should be estimated by <i>r*</i>/<i>n</i>, where <i>r</i>*&nbsp;=&nbsp;(<i>r</i>&nbsp;+&nbsp;1)<i>f</i><sub><i>r</i>+1</sub>/<i>f</i><sub><i>r</i></sub>.
 The Good‐Turing frequency formula is thus contrary to most people's 
intuition because the estimator in (1c) depends not only on the sample 
frequency <i>r</i> of the focal species, but also on the frequency information derived from species in the next frequency class, <i>r&nbsp;</i>+<i>&nbsp;</i>1.
            </p>
            
            <p>Good (<span><a href="#ecy2000-bib-0025" class="bibLink tab-link" data-tab="pane-pcw-references">1953</a></span>) used a fully Bayesian approach to theoretically justify the formula (1c), whereas Robbins (<span><a href="#ecy2000-bib-0045" class="bibLink tab-link" data-tab="pane-pcw-references">1968</a></span>) derived it as an empirical Bayes estimator. Good (<span><a href="#ecy2000-bib-0027" class="bibLink tab-link" data-tab="pane-pcw-references">2000</a></span>)
 wrote “when preparing my 1953 article, I had forgotten Turing's 
somewhat informal proof in 1940 or 1941, which involved cards or urn 
models in some way, and I worked out a separate proof [Bayes estimator].
 I still don't recall Turing's proof.” Nevertheless, Good (<span><a href="#ecy2000-bib-0026" class="bibLink tab-link" data-tab="pane-pcw-references">1983</a></span>,
 p. 28) provided a very intuitive non‐Bayesian justification of the 
Good‐Turing frequency formula as follows: Given an original sample of 
size <i>n</i>, consider the probability of the event that the next individual will be a species that had appeared <i>r</i> times in the original sample. (Mathematically, this probability is simply <img alt="urn:x-wiley:00129658:media:ecy2000:ecy2000-math-0007" class="section_image" src="ecy2000-math-0007.png">, as defined in Eq. <a class="equationLink scrollableLink" href="#ecy2000-disp-0001" title="Link to equation">1a</a>.) If this event occurs, then the species to which the additional individual belongs must appear <i>r&nbsp;</i>+<i>&nbsp;</i>1 times in the enlarged sample of size <i>n&nbsp;</i>+<i>&nbsp;</i>1.
 Because the order in which individuals were sampled is assumed to be 
irrelevant, the total number of individuals in the enlarged sample of 
size <i>n&nbsp;</i>+<i>&nbsp;</i>1 for those species (that appeared in the additional individual and had appeared <i>r</i> times in the original sample) is (<i>r</i>&nbsp;+&nbsp;1)<i>f</i><sub><i>r</i>+1</sub>. Thus, the probability of the aforementioned event in the enlarged sample of size <i>n&nbsp;</i>+<i>&nbsp;</i>1 is <img alt="urn:x-wiley:00129658:media:ecy2000:ecy2000-math-0008" class="section_image" src="ecy2000-math-0008.png">, which can be approximated by <img alt="urn:x-wiley:00129658:media:ecy2000:ecy2000-math-0009" class="section_image" src="ecy2000-math-0009.png"> if <i>n</i> is large enough. Dividing this by the number of such species, <i>f</i><sub><i>r</i></sub>,
 we obtain the mean relative abundance of those species, which is the 
classic Good‐Turing frequency formula as given in Eq.&nbsp;(<a class="equationLink scrollableLink" href="#ecy2000-disp-0003" title="Link to equation">1c</a>). Chiu et&nbsp;al. (<span><a href="#ecy2000-bib-0019" class="bibLink tab-link" data-tab="pane-pcw-references">2014b</a></span>) proposed an improved formula <img alt="urn:x-wiley:00129658:media:ecy2000:ecy2000-math-0010" class="section_image" src="ecy2000-math-0010.png"> shown below for <i>r&nbsp;</i>=<i>&nbsp;</i>0, 1, 2, …,
               </p><div class="inline-equation" id="ecy2000-disp-0004"><span class="inline-equation__construct"><img class="" src="ecy2000-math-0011.png" alt="urn:x-wiley:00129658:media:ecy2000:ecy2000-math-0011" title="urn:x-wiley:00129658:media:ecy2000:ecy2000-math-0011"></span><span class="inline-equation__label">(1d)</span></div>This
 improved estimator generally has smaller mean squared error than the 
original Good‐Turing estimator. In our subsequent derivation, we adopt 
the rightmost term in Eq.&nbsp;(<a class="equationLink scrollableLink" href="#ecy2000-disp-0004" title="Link to equation">1d</a>); a simple non‐Bayesian proof is provided (in Appendix&nbsp;<a class="suppLink scrollableLink" href="#support-information-section">S1</a>) to facilitate the generalization to the two‐assemblage case.
            <p></p>
            
         </div>
         
         <div class="article-section__sub-content" id="ecy2000-sec-0004">
            
            <h3 class="article-section__sub-title section2"> Undetected species richness</h3>
            
            <p>Statistically, species richness (observed species plus 
the number of undetected species) is difficult to estimate accurately if
 there are many almost undetectable species in a hyper‐diverse 
community. Practically, an accurate lower bound for species richness is 
preferable to an inaccurate point estimator. We now demonstrate that the
 improved Good‐Turing formula can be intuitively used to provide a lower
 bound for the number of undetected species and to clearly reveal, for 
the first time, the conditions under which the lower bound is a nearly 
unbiased point estimator.</p>
            
            <p>For <i>r&nbsp;</i>=<i>&nbsp;</i>0, both Eq.&nbsp;(<a class="equationLink scrollableLink" href="#ecy2000-disp-0003" title="Link to equation">1c</a>) and Eq.&nbsp;(<a class="equationLink scrollableLink" href="#ecy2000-disp-0004" title="Link to equation">1d</a>) imply that the mean population relative frequency for those undetected species is approximately <img alt="urn:x-wiley:00129658:media:ecy2000:ecy2000-math-0012" class="section_image" src="ecy2000-math-0012.png">, which is not obtainable from observed data because <i>f</i><sub>0</sub> is unknown. However, this relation implies that the product of <i>α</i><sub>0</sub> and <i>f</i><sub>0</sub>,
 the estimated proportion of the total number of individuals that is due
 to undetected species, can be well estimated by the proportion of 
singletons, <i>f</i><sub>1</sub>/<i>n</i>. For notational simplicity, let <img alt="urn:x-wiley:00129658:media:ecy2000:ecy2000-math-0013" class="section_image" src="ecy2000-math-0013.png"> denote the estimator of the product of α<sub>0</sub> and <i>f</i><sub>0</sub>. Then we have
               </p><div class="inline-equation" id="ecy2000-disp-0005"><span class="inline-equation__construct"><img class="" src="ecy2000-math-0014.png" alt="urn:x-wiley:00129658:media:ecy2000:ecy2000-math-0014" title="urn:x-wiley:00129658:media:ecy2000:ecy2000-math-0014"></span><span class="inline-equation__label">(2a)</span></div>Equation&nbsp;(<a class="equationLink scrollableLink" href="#ecy2000-disp-0004" title="Link to equation">1d</a>) also implies that, for those species that appeared as singletons (<i>r&nbsp;</i>=<i>&nbsp;</i>1) in a sample, their mean relative frequency is estimated by
               <div class="inline-equation" id="ecy2000-disp-0006"><span class="inline-equation__construct"><img class="" src="ecy2000-math-0015.png" alt="urn:x-wiley:00129658:media:ecy2000:ecy2000-math-0015" title="urn:x-wiley:00129658:media:ecy2000:ecy2000-math-0015"></span><span class="inline-equation__label">(2b)</span></div>
            <p></p>
            
            <p>Intuitively, we expect that the mean relative frequency 
of all undetected species should be less than the mean relative 
frequency of all singletons, i.e., <i>α</i><sub>0</sub>&nbsp;≤&nbsp;<i>α</i><sub>1</sub>,
 and this ordering should be preserved by the corresponding estimates. 
Combining (2a) and (2b), we readily obtain a lower bound for the number 
of undetected species:
               </p><div class="inline-equation" id="ecy2000-disp-0007"><span class="inline-equation__construct"><img class="" src="ecy2000-math-0016.png" alt="urn:x-wiley:00129658:media:ecy2000:ecy2000-math-0016" title="urn:x-wiley:00129658:media:ecy2000:ecy2000-math-0016"></span><span class="inline-equation__label">(2c)</span></div>
            <p></p>
            
            <p>This lower bound for <i>f</i><sub>0</sub> is identical to that proved rigorously by Chao (<span><a href="#ecy2000-bib-0005" class="bibLink tab-link" data-tab="pane-pcw-references">1984</a></span>, <span><a href="#ecy2000-bib-0006" class="bibLink tab-link" data-tab="pane-pcw-references">1987</a></span>) by means of a Cauchy‐Schwarz inequality: <i>E</i>(<i>f</i><sub>0</sub>)&nbsp;×&nbsp;2<i>E</i>(<i>f</i><sub>2</sub>)&nbsp;≥&nbsp;(1&nbsp;−&nbsp;1/<i>n</i>)[<i>E</i>(<i>f</i><sub>1</sub>)]<sup>2</sup>,
 which may not be intuitively understood by most ecologists. Here 
Good‐Turing's approach is intuitive and provides a sufficient condition 
for the resulting estimator being unbiased, as elaborated later. Based 
on Eq.&nbsp;(<a class="equationLink scrollableLink" href="#ecy2000-disp-0007" title="Link to equation">2c</a>),
 the estimated number of undetected species is based exclusively on the 
information on the rarest observed species (the number of singletons and
 doubletons). The idea behind this lower bound is that detected abundant
 species carry negligible information about the undetected species; 
detected rare species carry nearly all such information. From Eq.&nbsp;(<a class="equationLink scrollableLink" href="#ecy2000-disp-0007" title="Link to equation">2c</a>), the Good‐Turing formula leads to the following Chao1 species richness estimator, with a slight modification when <i>f</i><sub>2</sub> = 0. (Colwell and Coddington <span><a href="#ecy2000-bib-0020" class="bibLink tab-link" data-tab="pane-pcw-references">1994</a></span> gave the name <i>Chao1</i> to this estimator):
               </p><div class="inline-equation" id="ecy2000-disp-0008"><span class="inline-equation__construct"><img class="" src="ecy2000-math-0017.png" alt="urn:x-wiley:00129658:media:ecy2000:ecy2000-math-0017" title="urn:x-wiley:00129658:media:ecy2000:ecy2000-math-0017"></span><span class="inline-equation__label">(3a)</span></div>
            <p></p>
            
            <p>Notice that, in the above derivation, if <img alt="urn:x-wiley:00129658:media:ecy2000:ecy2000-math-0018" class="section_image" src="ecy2000-math-0018.png">≈<img alt="urn:x-wiley:00129658:media:ecy2000:ecy2000-math-0019" class="section_image" src="ecy2000-math-0019.png"> (i.e., undetected species and singletons have identical mean relative abundances), then the inequality sign in Eq.&nbsp;(<a class="equationLink scrollableLink" href="#ecy2000-disp-0007" title="Link to equation">2c</a>)
 becomes an equality sign, implying that the lower bound becomes an 
unbiased point estimator. Only through the Good‐Turing perspectives can 
this condition be revealed.
            </p>
            
            <p>A simple sufficient condition for the Chao1 lower bound being nearly unbiased is that <i>rare</i> species (specifically, singletons and undetected species) have approximately homogenous abundances (because this implies <img alt="urn:x-wiley:00129658:media:ecy2000:ecy2000-math-0020" class="section_image" src="ecy2000-math-0020.png">≈<img alt="urn:x-wiley:00129658:media:ecy2000:ecy2000-math-0021" class="section_image" src="ecy2000-math-0021.png">); in this case, the abundant species could be highly heterogeneous without affecting the estimator. However, if <i>rare</i>
 species are heterogeneous and sample size is not sufficiently large, 
then at best we can provide only a lower bound, because the data provide
 insufficient information to accurately estimate diversity when rare 
species abundances are substantially heterogeneous. A simulation study 
is described to examine the performance of the Chao1 estimator in <i>Discussion</i>.
            </p>
            
            <p>Chao (<span><a href="#ecy2000-bib-0006" class="bibLink tab-link" data-tab="pane-pcw-references">1987</a></span>) applied a standard approximation method and obtained the following estimated variance estimators:
               </p><div class="inline-equation" id="ecy2000-disp-0009"><span class="inline-equation__construct"><img class="" src="ecy2000-math-0022.png" alt="urn:x-wiley:00129658:media:ecy2000:ecy2000-math-0022" title="urn:x-wiley:00129658:media:ecy2000:ecy2000-math-0022"></span><span class="inline-equation__label">(3b)</span></div>A
 confidence interval for species richness can then be obtained using a 
log‐transformation, so that the lower confidence bound is always greater
 than the observed species richness if sampling is incomplete (i.e., 
there are singletons in the sample). When <i>f</i><sub>2</sub> = 0, a slight modification of the variance formula is needed; see Chao and Chiu (<span><a href="#ecy2000-bib-0008" class="bibLink tab-link" data-tab="pane-pcw-references">2012</a></span>).
            <p></p>
            
         </div>
         
      </div>
      
      <div class="article-section__content" id="ecy2000-sec-0005">
         
         <h2 class="article-section__title section__title section1" id="ecy2000-sec-0005-title"> Species Diversity: Two‐Assemblage Good‐Turing Formula</h2>
         
         <div class="article-section__sub-content" id="ecy2000-sec-0006">
            
            <h3 class="article-section__sub-title section2"> Two‐assemblage Good‐Turing formulas</h3>
            
            <p>We now extend the one‐assemblage model formulation and 
data framework to two assemblages (I and II), which can differ not only 
in their species richness, but also in their species composition. Assume
 that there are <i>S</i> species in the <i>pooled</i> assemblage. The true relative species abundances or frequencies in Assemblages I and II are denoted by (<i>p</i><sub>11</sub>,&nbsp;<i>p</i><sub>21</sub>,&nbsp;…,&nbsp;<i>p</i><sub><i>S</i>1</sub>) and, (<i>p</i><sub>12</sub>,&nbsp;<i>p</i><sub>22</sub>,&nbsp;…,&nbsp;<i>p</i><sub><i>S</i>2</sub>) respectively, <img alt="urn:x-wiley:00129658:media:ecy2000:ecy2000-math-0023" class="section_image" src="ecy2000-math-0023.png"> Let the number of shared species between the two assemblages be <i>S</i><sub><i>shared</i></sub> (or <i>S</i><sub>12</sub> for notational simplicity). Without loss of generality, we assume that the first <i>S</i><sub><i>shared</i></sub> species in the pooled assemblage are the shared species.
            </p>
            
            <p>A random sample is taken from each of the two assemblages (Sample I with size <i>n</i><sub>1</sub> from Assemblage I and Sample II with size <i>n</i><sub>2</sub> from Assemblage II). Denote the number of observed shared species by <i>S</i><sub><i>shared</i>,<i>obs</i></sub>, and the observed species abundances in the two assemblages, respectively, by (<i>X</i><sub>11</sub>,&nbsp;<i>X</i><sub>21</sub>,&nbsp;…,&nbsp;<i>X</i><sub><i>S</i>1</sub>) and (<i>X</i><sub>12</sub>,&nbsp;<i>X</i><sub>22</sub>,&nbsp;…,&nbsp;<i>X</i><sub><i>S</i>2</sub>). For any two non‐negative integers <i>r</i> and <i>v,</i> define
               </p><div class="inline-equation" id="ecy2000-disp-0010"><span class="inline-equation__construct"><img class="" src="ecy2000-math-0024.png" alt="urn:x-wiley:00129658:media:ecy2000:ecy2000-math-0024" title="urn:x-wiley:00129658:media:ecy2000:ecy2000-math-0024"></span><span class="inline-equation__label">(4a)</span></div>That is, <i>f</i><sub><i>rv</i></sub> denotes the number of <i>shared</i> species that are observed <i>r</i> times in Sample I and <i>v</i> times in Sample II. In particular, <i>f</i><sub>11</sub> denotes the number of shared species that are singletons in both samples, and <i>f</i><sub>00</sub> denotes the number of shared species that are undetected in both samples. Also, let <i>f</i><sub><i>r</i>+</sub> denote the number of shared species that are observed <i>r</i> times in Sample I and are observed at least once (using a “+” sign to replace the index <i>v</i>) in Sample II, with a similar symmetric definition for <i>f</i><sub>+<i>v</i></sub>. Thus, <i>f</i><sub><i>+</i>+</sub> becomes the number of observed species shared between the two samples. Mathematically, we have the following expressions:
               <div class="inline-equation" id="ecy2000-disp-0011"><span class="inline-equation__construct"><img class="" src="ecy2000-math-0025.png" alt="urn:x-wiley:00129658:media:ecy2000:ecy2000-math-0025" title="urn:x-wiley:00129658:media:ecy2000:ecy2000-math-0025"></span><span class="inline-equation__label">(4b)</span></div>
               <div class="inline-equation" id="ecy2000-disp-0012"><span class="inline-equation__construct"><img class="" src="ecy2000-math-0026.png" alt="urn:x-wiley:00129658:media:ecy2000:ecy2000-math-0026" title="urn:x-wiley:00129658:media:ecy2000:ecy2000-math-0026"></span><span class="inline-equation__label">(4c)</span></div>
               <div class="inline-equation" id="ecy2000-disp-0013"><span class="inline-equation__construct"><img class="" src="ecy2000-math-0027.png" alt="urn:x-wiley:00129658:media:ecy2000:ecy2000-math-0027" title="urn:x-wiley:00129658:media:ecy2000:ecy2000-math-0027"></span><span class="inline-equation__label">(4d)</span></div>
            <p></p>
            
            <p>Here we generalize the original Good‐Turing formula to 
two assemblages. There are two parts to the generalization. The first 
part, below, is a direct generalization of the original formula; the 
second part is proved in Appendix&nbsp;<a class="suppLink scrollableLink" href="#support-information-section">S1</a> by an argument parallel to that applied in the developing the original formula.
            </p>
            
            <p>(1) Given two‐sample data, let <img alt="urn:x-wiley:00129658:media:ecy2000:ecy2000-math-0028" class="section_image" src="ecy2000-math-0028.png"> be the mean of the true relative frequencies in Assemblage I for those shared species that each appeared exactly <i>r</i>
 times in Sample I and appeared at least once in Sample II. A direct 
generalization of the original Good‐Turing formula in Eq.&nbsp;(<a class="equationLink scrollableLink" href="#ecy2000-disp-0004" title="Link to equation">1d</a>) leads to
               </p><div class="inline-equation" id="ecy2000-disp-0014"><span class="inline-equation__construct"><img class="" src="ecy2000-math-0029.png" alt="urn:x-wiley:00129658:media:ecy2000:ecy2000-math-0029" title="urn:x-wiley:00129658:media:ecy2000:ecy2000-math-0029"></span><span class="inline-equation__label">(5a)</span></div>
            <p></p>
            
            <p>Similarly, we have a symmetric formula for the mean of 
the true relative frequencies in Assemblage II for those shared species 
that appeared at least once in Sample I and appeared <i>v</i> times in Sample II.
               </p><div class="inline-equation" id="ecy2000-disp-0015"><span class="inline-equation__construct"><img class="" src="ecy2000-math-0030.png" alt="urn:x-wiley:00129658:media:ecy2000:ecy2000-math-0030" title="urn:x-wiley:00129658:media:ecy2000:ecy2000-math-0030"></span><span class="inline-equation__label">(5b)</span></div>
            <p></p>
            
            <p>(2) For any shared species that appeared exactly <i>r</i>
 times in each sample, consider calculating the product of its true 
relative abundances in the two assemblages; the mean of the products 
among all such shared species (there are <i>f</i><sub><i>rr</i></sub> such shared species) can be expressed as <img alt="urn:x-wiley:00129658:media:ecy2000:ecy2000-math-0031" class="section_image" src="ecy2000-math-0031.png">, <i>r&nbsp;</i>=<i>&nbsp;</i>0, 1, 2… The following generalized two‐assemblage Good‐Turing formula provides an estimator for <i>α</i><sub><i>rr</i></sub> (see Appendix&nbsp;<a class="suppLink scrollableLink" href="#support-information-section">S1</a> for a proof):
               </p><div class="inline-equation" id="ecy2000-disp-0016"><span class="inline-equation__construct"><img class="" src="ecy2000-math-0032.png" alt="urn:x-wiley:00129658:media:ecy2000:ecy2000-math-0032" title="urn:x-wiley:00129658:media:ecy2000:ecy2000-math-0032"></span><span class="inline-equation__label">(5c)</span></div>The generalized formulas in Eqs.&nbsp;(<a class="equationLink scrollableLink" href="#ecy2000-disp-0014" title="Link to equation">5a</a>)–(<a class="equationLink scrollableLink" href="#ecy2000-disp-0016" title="Link to equation">5c</a>) lead elegantly to an estimator of undetected shared species richness between two assemblages, as shown below.
            <p></p>
            
         </div>
         
         <div class="article-section__sub-content" id="ecy2000-sec-0007">
            
            <h3 class="article-section__sub-title section2"> Undetected shared species richness between two assemblages</h3>
            
            <p>Under the two‐assemblage model formulation and data 
framework, the true number of shared species can be expressed as the sum
 of four terms:
               </p><div class="inline-equation" id="ecy2000-disp-0017"><span class="inline-equation__construct"><img class="" src="ecy2000-math-0033.png" alt="urn:x-wiley:00129658:media:ecy2000:ecy2000-math-0033" title="urn:x-wiley:00129658:media:ecy2000:ecy2000-math-0033"></span><span class="inline-equation__label">(6)</span></div>The four terms in the right hand side of Eq.&nbsp;<a class="equationLink scrollableLink" href="#ecy2000-disp-0017" title="Link to equation">6</a>, as defined earlier, represent, respectively, the number of shared species observed in both samples (<i>f</i><sub>++</sub>), the number of shared species observed only in Sample II (<i>f</i><sub>0+</sub>), the number of shared species observed only in Sample I (<i>f</i><sub>+0</sub>), and the number of shared species undetected in both samples (<i>f</i><sub>00</sub>),
 but present in both assemblages. Only the first term is observable. The
 sum of the last three terms represents the total undetected shared 
species richness. Applying Eqs.&nbsp;(<a class="equationLink scrollableLink" href="#ecy2000-disp-0014" title="Link to equation">5a</a>) and (<a class="equationLink scrollableLink" href="#ecy2000-disp-0015" title="Link to equation">5b</a>) and using notation and derivation similar to Eq.&nbsp;(<a class="equationLink scrollableLink" href="#ecy2000-disp-0007" title="Link to equation">2c</a>), we have
               <div class="inline-equation" id="ecy2000-disp-0018"><span class="inline-equation__construct"><img class="" src="ecy2000-math-0034.png" alt="urn:x-wiley:00129658:media:ecy2000:ecy2000-math-0034" title="urn:x-wiley:00129658:media:ecy2000:ecy2000-math-0034"></span><span class="inline-equation__label">(7a)</span></div>and a symmetric expression
               <div class="inline-equation" id="ecy2000-disp-0019"><span class="inline-equation__construct"><img class="" src="ecy2000-math-0035.png" alt="urn:x-wiley:00129658:media:ecy2000:ecy2000-math-0035" title="urn:x-wiley:00129658:media:ecy2000:ecy2000-math-0035"></span><span class="inline-equation__label">(7b)</span></div>
            <p></p>
            
            <p>Furthermore, we expect that <i>α</i><sub>00</sub>&nbsp;≤&nbsp;<i>α</i><sub>11</sub>, and this ordering is preserved by the corresponding estimates, implying the following inequality, from Eq.&nbsp;(<a class="equationLink scrollableLink" href="#ecy2000-disp-0016" title="Link to equation">5c</a>):
               </p><div class="inline-equation" id="ecy2000-disp-0020"><span class="inline-equation__construct"><img class="" src="ecy2000-math-0036.png" alt="urn:x-wiley:00129658:media:ecy2000:ecy2000-math-0036" title="urn:x-wiley:00129658:media:ecy2000:ecy2000-math-0036"></span><span class="inline-equation__label">(7c)</span></div>Combining (6) and (7a) – (7c), we obtain the following estimator of shared species richness:
               <div class="inline-equation" id="ecy2000-disp-0021"><span class="inline-equation__construct"><img class="" src="ecy2000-math-0037.png" alt="urn:x-wiley:00129658:media:ecy2000:ecy2000-math-0037" title="urn:x-wiley:00129658:media:ecy2000:ecy2000-math-0037"></span><span class="inline-equation__label">(7d)</span></div>where <i>k</i><sub><i>i</i></sub>&nbsp;=&nbsp;(<i>n</i><sub><i>i</i></sub>&nbsp;−&nbsp;1)/<i>n</i><sub><i>i</i></sub>, <i>i&nbsp;</i>=<i>&nbsp;</i>1, 2. A modification similar to that in Eq.&nbsp;(<a class="equationLink scrollableLink" href="#ecy2000-disp-0008" title="Link to equation">3a</a>)
 can be applied to each term to avoid a zero divisor. The sum of the 
last three terms estimates the undetected shared species richness. The 
resulting estimator in Eq.&nbsp;(<a class="equationLink scrollableLink" href="#ecy2000-disp-0021" title="Link to equation">7d</a>) is identical to that derived by Pan et&nbsp;al. (<span><a href="#ecy2000-bib-0042" class="bibLink tab-link" data-tab="pane-pcw-references">2009</a></span>),
 who derived it by means of complicated mathematical inequalities. Here 
we show that Good‐Turing's framework provides a simple, unified approach
 to inferring undetected diversity, not only for one assemblage but also
 for two assemblages. The estimator in Eq.&nbsp;(<a class="equationLink scrollableLink" href="#ecy2000-disp-0021" title="Link to equation">7d</a>) is referred to as the <i>Chao1‐shared</i>
 estimator because it can be regarded as an extension of the 
single‐assemblage Chao1 estimator (Eq. 3a) to the case of two 
assemblages. Pan et&nbsp;al. (<span><a href="#ecy2000-bib-0042" class="bibLink tab-link" data-tab="pane-pcw-references">2009</a></span>)
 also derived a variance estimator by using a standard approximation 
theory, allowing construction of a confidence interval for true shared 
species richness. From the above derivation, the Chao1‐shared lower 
bound becomes a nearly unbiased point estimator provided <img alt="urn:x-wiley:00129658:media:ecy2000:ecy2000-math-0038" class="section_image" src="ecy2000-math-0038.png"> and <img alt="urn:x-wiley:00129658:media:ecy2000:ecy2000-math-0039" class="section_image" src="ecy2000-math-0039.png">
 in the derivations. Considering only shared species, we see that a 
simple sufficient condition is that the undetected species and the 
detected singletons have approximately the same abundances in each of 
the two assemblages.
            <p></p>
            
         </div>
         
      </div>
      
      <div class="article-section__content" id="ecy2000-sec-0008">
         
         <h2 class="article-section__title section__title section1" id="ecy2000-sec-0008-title"> Phylogenetic Diversity</h2>
         
         <div class="article-section__sub-content" id="ecy2000-sec-0009">
            
            <h3 class="article-section__sub-title section2"> Undetected Faith's PD in a single assemblage</h3>
            
            <p>To formulate phylogenetic diversity, we assume that all <i>S</i>
 species in an assemblage are connected by a rooted ultrametric or 
non‐ultrametric phylogenetic tree, with all species, observed and 
unobserved, as tip nodes. In this paper, all phylogenetic diversity 
measures and estimators are computed from a given fixed reference point 
that is ancestral to all taxa considered in the study. The choice of the
 reference point is thus independent of the sampling data. Assume that 
there are <i>B</i> branch segments and <i>B</i> corresponding nodes, <i>B&nbsp;</i> ≥ &nbsp;<i>S</i>. Let <i>a</i><sub><i>i</i></sub> denote the total relative abundance of the species descended from the <i>i</i>th node/branch, <i>i&nbsp;=&nbsp;</i>1, 2, …, <i>B</i>, and <i>L</i><sub><i>i</i></sub> denote the length of branch <i>i</i>. Therefore, the set of species relative abundances (<i>p</i><sub>1</sub>, <i>p</i><sub>2</sub>, …, <i>p</i><sub><i>S</i></sub>) is expanded to a larger relative abundance set <img alt="urn:x-wiley:00129658:media:ecy2000:ecy2000-math-0040" class="section_image" src="ecy2000-math-0040.png"> with (<i>p</i><sub>1</sub>, <i>p</i><sub>2</sub>, …, <i>p</i><sub><i>S</i></sub>) as its first <i>S</i> elements. For simplicity, we refer to <i>a</i><sub><i>i</i></sub> as the <i>node/branch relative abundance</i> of the <i>i</i>th node/branch, although <img alt="urn:x-wiley:00129658:media:ecy2000:ecy2000-math-0041" class="section_image" src="ecy2000-math-0041.png"> is not necessarily equal to unity; see Fig.&nbsp;1 of Chao et&nbsp;al. (<span><a href="#ecy2000-bib-0014" class="bibLink tab-link" data-tab="pane-pcw-references">2015a</a></span>) for an illustrative example. Faith's (<span><a href="#ecy2000-bib-0023" class="bibLink tab-link" data-tab="pane-pcw-references">1992</a></span>) <i>PD</i> is expressed as <img alt="urn:x-wiley:00129658:media:ecy2000:ecy2000-math-0042" class="section_image" src="ecy2000-math-0042.png">.
            </p>
            
            <p>We assume an empirical sample of <i>n</i> individuals with sample species abundances <img alt="urn:x-wiley:00129658:media:ecy2000:ecy2000-math-0043" class="section_image" src="ecy2000-math-0043.png"> is taken from the assemblage. Define <img alt="urn:x-wiley:00129658:media:ecy2000:ecy2000-math-0044" class="section_image" src="ecy2000-math-0044.png"> as the sum of the observed species abundances for those species in the sample that are descended from branch <i>i</i>. Then we can expand the set of observed species abundances to a larger branch abundance set <img alt="urn:x-wiley:00129658:media:ecy2000:ecy2000-math-0045" class="section_image" src="ecy2000-math-0045.png"> with <img alt="urn:x-wiley:00129658:media:ecy2000:ecy2000-math-0046" class="section_image" src="ecy2000-math-0046.png"> as its first <i>S</i> elements. We refer to <img alt="urn:x-wiley:00129658:media:ecy2000:ecy2000-math-0047" class="section_image" src="ecy2000-math-0047.png"> <img alt="urn:x-wiley:00129658:media:ecy2000:ecy2000-math-0048" class="section_image" src="ecy2000-math-0048.png">, as the <i>sample node/branch abundance</i> of node/branch <i>i</i>. Let
               </p><div class="inline-equation" id="ecy2000-disp-0022"><span class="inline-equation__construct"><img class="" src="ecy2000-math-0049.png" alt="urn:x-wiley:00129658:media:ecy2000:ecy2000-math-0049" title="urn:x-wiley:00129658:media:ecy2000:ecy2000-math-0049"></span><span class="inline-equation__label">(8a)</span></div>be the total length of those branches with sample abundance <i>r</i> in the set <img alt="urn:x-wiley:00129658:media:ecy2000:ecy2000-math-0050" class="section_image" src="ecy2000-math-0050.png">, where the indicator function <i>I</i> (∙) is defined in Eq.&nbsp;(<a class="equationLink scrollableLink" href="#ecy2000-disp-0001" title="Link to equation">1a</a>). The undetected <i>PD</i> in the sample is <i>g</i><sub>0</sub>, which is the total length of undetected branches; <i>g</i><sub>0</sub> is unknown but <img alt="urn:x-wiley:00129658:media:ecy2000:ecy2000-math-0051" class="section_image" src="ecy2000-math-0051.png"> can be computed from the sample and the observed tree (the tree spanned by the observed species). For <i>r&nbsp;</i>&gt;<i>&nbsp;</i>0, <i>g</i><sub><i>r</i></sub> is identical to the total length of the branches with sample abundance <i>r</i> in the observed tree. For example, <i>g</i><sub>1</sub> denotes the total length of those nodes/branches with sample abundance = 1 in the observed tree; <i>g</i><sub>2</sub> denotes the total length of those branches with sample abundance = 2 in the observed tree. Let <i>PD</i><sub><i>obs</i></sub> denote the observed <i>PD</i>. Then we have <img alt="urn:x-wiley:00129658:media:ecy2000:ecy2000-math-0052" class="section_image" src="ecy2000-math-0052.png"> and <i>PD</i>&nbsp;=&nbsp;<i>PD</i><sub><i>obs</i></sub>&nbsp;+&nbsp;<i>g</i><sub>0</sub>. A simple example is provided in Appendix&nbsp;<a class="suppLink scrollableLink" href="#support-information-section">S2</a>: Fig.&nbsp;S1 to illustrate these measures.
            <p></p>
            
            <p>We now extend the Good‐Turing frequency formula to its 
phylogenetic version. For those nodes/branches that each is with a 
sample abundance/frequency of <i>r</i>, the branch‐length‐weighted mean (per unit length) of their true abundances in the entire assemblage, λ<sub><i>r</i></sub>, can be mathematically expressed as
               </p><div class="inline-equation" id="ecy2000-disp-0023"><span class="inline-equation__construct"><img class="" src="ecy2000-math-0053.png" alt="urn:x-wiley:00129658:media:ecy2000:ecy2000-math-0053" title="urn:x-wiley:00129658:media:ecy2000:ecy2000-math-0053"></span><span class="inline-equation__label"></span></div>Derivation
 steps parallel to those used for the species diversity lead to the 
following phylogenetic version of the single‐assemblage Good‐Turing 
frequency formula (Appendix&nbsp;<a class="suppLink scrollableLink" href="#support-information-section">S2</a>):
               <div class="inline-equation" id="ecy2000-disp-0024"><span class="inline-equation__construct"><img class="" src="ecy2000-math-0054.png" alt="urn:x-wiley:00129658:media:ecy2000:ecy2000-math-0054" title="urn:x-wiley:00129658:media:ecy2000:ecy2000-math-0054"></span><span class="inline-equation__label">(8b)</span></div>
            <p></p>
            
            <p>Intuitively, we expect that <i>λ</i><sub>0</sub>&nbsp;≤&nbsp;<i>λ</i><sub>1</sub>, and this ordering is preserved by the corresponding estimates. From the above formula, a lower bound for the undetected <i>PD</i> is obtained:
               </p><div class="inline-equation" id="ecy2000-disp-0025"><span class="inline-equation__construct"><img class="" src="ecy2000-math-0055.png" alt="urn:x-wiley:00129658:media:ecy2000:ecy2000-math-0055" title="urn:x-wiley:00129658:media:ecy2000:ecy2000-math-0055"></span><span class="inline-equation__label">(8c)</span></div>The lower bound in Eq.&nbsp;(<a class="equationLink scrollableLink" href="#ecy2000-disp-0025" title="Link to equation">8c</a>) is identical to that proposed in Chao et&nbsp;al. (<span><a href="#ecy2000-bib-0014" class="bibLink tab-link" data-tab="pane-pcw-references">2015a</a></span>)
 via the Cauchy‐Schwarz inequality. Here, instead, we prove it under a 
unified framework by means of a phylogenetic version of the original 
Good‐Turing formula. Cardoso et&nbsp;al. (<span><a href="#ecy2000-bib-0003" class="bibLink tab-link" data-tab="pane-pcw-references">2014</a></span>) adapted the Chao1 estimator (Eq. 3a) to obtain the same estimator.
            <p></p>
            
            <p>When <i>g</i><sub>2</sub> is relatively small, including the case of <i>g</i><sub>2</sub>
 = 0, the above estimator may yield an extremely large value and thus 
exhibit a large variance. To cope with such cases, Chao et&nbsp;al. (<span><a href="#ecy2000-bib-0014" class="bibLink tab-link" data-tab="pane-pcw-references">2015a</a></span>) and Hsieh and Chao (<span><a href="#ecy2000-bib-0034" class="bibLink tab-link" data-tab="pane-pcw-references">2017</a></span>) proposed the following modified Chao1‐<i>PD</i> estimator:
               </p><div class="inline-equation" id="ecy2000-disp-0026"><span class="inline-equation__construct"><img class="" src="ecy2000-math-0056.png" alt="urn:x-wiley:00129658:media:ecy2000:ecy2000-math-0056" title="urn:x-wiley:00129658:media:ecy2000:ecy2000-math-0056"></span><span class="inline-equation__label">(8d)</span></div>where <img alt="urn:x-wiley:00129658:media:ecy2000:ecy2000-math-0057" class="section_image" src="ecy2000-math-0057.png"> and <img alt="urn:x-wiley:00129658:media:ecy2000:ecy2000-math-0058" class="section_image" src="ecy2000-math-0058.png">
 denote, respectively, the number of nodes/branches with abundance = 1 
and abundance = 2 in the observed tree. Since any node/branch with 
abundance = 1 can occur only at the tip nodes, we have <img alt="urn:x-wiley:00129658:media:ecy2000:ecy2000-math-0059" class="section_image" src="ecy2000-math-0059.png"> i.e., <img alt="urn:x-wiley:00129658:media:ecy2000:ecy2000-math-0060" class="section_image" src="ecy2000-math-0060.png"> is identical to the number of singletons at the tip nodes. However, <img alt="urn:x-wiley:00129658:media:ecy2000:ecy2000-math-0061" class="section_image" src="ecy2000-math-0061.png"> is not necessarily equal to <i>f</i><sub>2</sub>; see Appendix&nbsp;<a class="suppLink scrollableLink" href="#support-information-section">S2</a>: Fig.&nbsp;S1 for an example.
            <p></p>
            
            <p>Based on Eq.&nbsp;(<a class="equationLink scrollableLink" href="#ecy2000-disp-0025" title="Link to equation">8c</a>), the Chao1‐<i>PD</i> estimator is nearly unbiased when <img alt="urn:x-wiley:00129658:media:ecy2000:ecy2000-math-0062" class="section_image" src="ecy2000-math-0062.png">,
 which means that the singletons (which occur only at the tip nodes) and
 the undetected nodes (which can be tip or interior nodes) have the same
 length‐weighted mean abundances. A sufficient simple condition is that 
all rare nodes (including the singletons and the undetected) have 
approximately the same abundances. The variance of the Chao1‐<i>PD</i> estimator can be obtained using Eq.&nbsp;(<a class="equationLink scrollableLink" href="#ecy2000-disp-0009" title="Link to equation">3b</a>) with {<i>f</i><sub>1</sub>, <i>f</i><sub>2</sub>} being replaced by {<i>g</i><sub>1</sub>, <i>g</i><sub>2</sub>}. The construction of the confidence interval for Faith's <i>PD</i> based on the Chao1‐<i>PD</i> estimator can be similarly obtained.
            </p>
            
            <p>From the derivations above, we see that all estimation 
procedures and the derivation steps for developing the phylogenetic 
version of the Good‐Turing formula and <i>PD</i> estimators are parallel
 to those for the original formula and species richness estimators. A 
summary of formulas and descriptions for estimating species richness and
 Faith's <i>PD</i> is provided in Appendix&nbsp;<a class="suppLink scrollableLink" href="#support-information-section">S2</a>:
 Table&nbsp;S1, where the analogy between the two estimation frameworks 
is transparently displayed. The analogy was first proposed by Faith (<span><a href="#ecy2000-bib-0023" class="bibLink tab-link" data-tab="pane-pcw-references">1992</a></span>).
 From Faith's perspective, each unit‐length branch is regarded as a 
“feature” in phylogenetic diversity (like a “species” in species 
diversity). Chao et&nbsp;al. (<span><a href="#ecy2000-bib-0012" class="bibLink tab-link" data-tab="pane-pcw-references">2014a</a></span>) subsequently referred to each unit‐length branch segment as a <i>phylogenetic entity</i>.
 For example, a branch of 8‐unit length is counted as 8 phylogenetic 
entities. All entities are phylogenetically equally distinct, just as 
all species are assumed taxonomically equally distinct in computing 
species richness. Instead of species, for <i>PD</i> we are measuring the
 total number of phylogenetic entities, or equivalently, the total 
branch length (because each entity has length of unity). Using this 
perspective, the measures of branch lengths <img alt="urn:x-wiley:00129658:media:ecy2000:ecy2000-math-0063" class="section_image" src="ecy2000-math-0063.png"> in estimating <i>PD</i> play the same role as the frequency counts <img alt="urn:x-wiley:00129658:media:ecy2000:ecy2000-math-0064" class="section_image" src="ecy2000-math-0064.png">
 in estimating species richness. This analogy to counting‐up species 
means that most ecological indices defined at the species level can be 
converted to <i>PD</i> equivalents (by counting phylogenetic entities rather than species).
            </p>
            
         </div>
         
         <div class="article-section__sub-content" id="ecy2000-sec-0010">
            
            <h3 class="article-section__sub-title section2"> Undetected shared PD between two assemblages</h3>
            
            <p>Following the approach to the two‐assemblage model formulation and the data framework described in the section <i>Two‐assemblage Good‐Turing Formulas</i>, we assume that all <i>S</i> species of the <i>pooled</i> assemblage are indexed by 1, 2, …, <i>S</i>. Assume these <i>S</i> species are connected by a rooted ultrametric or non‐ultrametric phylogenetic tree, with the <i>S</i>
 species as tip nodes. Given a fixed reference point that is ancestral 
to all taxa considered in the study, we assume that there are <i>B</i> branch segments and <i>B</i> corresponding nodes in the pooled tree, and let <i>L</i><sub><i>i</i></sub> denote the length of branch <i>i</i>.
 Because the phylogenetic tree of each individual assemblage is a 
sub‐tree of the pooled tree, the diversity for each individual 
assemblage can be computed from the pooled tree structure with only the 
node (or branch) abundances varying between assemblages, as illustrated 
by Chiu et&nbsp;al. (<span><a href="#ecy2000-bib-0018" class="bibLink tab-link" data-tab="pane-pcw-references">2014a</a></span>, Fig.&nbsp;2). Assume that there are <i>B</i><sub>12</sub>
 branches shared by the two assemblages for the given reference point, 
and without losing generality, these shared branches are indexed from 1,
 2, …, <i>B</i><sub>12</sub> for notational simplicity. Denote the true shared <i>PD</i> (the total length of branch segments shared by the two individual sub‐trees) by <i>PD</i><sub><i>shared</i></sub> and the shared <i>PD</i> between two observed trees by <i>PD</i><sub><i>shared,obs</i></sub>.
            </p>
            
            <p>As in the preceding section, the two sets of relative 
species abundances in Assemblages I and II are, respectively, expanded 
to the <i>node</i>/<i>branch relative abundance (or frequency)</i> sets <img alt="urn:x-wiley:00129658:media:ecy2000:ecy2000-math-0065" class="section_image" src="ecy2000-math-0065.png"> and <img alt="urn:x-wiley:00129658:media:ecy2000:ecy2000-math-0066" class="section_image" src="ecy2000-math-0066.png">. We also expand the set of observed species frequencies <img alt="urn:x-wiley:00129658:media:ecy2000:ecy2000-math-0067" class="section_image" src="ecy2000-math-0067.png"> in Assemblage I to a larger sample node/branch frequency set <img alt="urn:x-wiley:00129658:media:ecy2000:ecy2000-math-0068" class="section_image" src="ecy2000-math-0068.png">. Similarly, we extend the set of observed species frequencies <img alt="urn:x-wiley:00129658:media:ecy2000:ecy2000-math-0069" class="section_image" src="ecy2000-math-0069.png"> in Assemblage II to a larger set <img alt="urn:x-wiley:00129658:media:ecy2000:ecy2000-math-0070" class="section_image" src="ecy2000-math-0070.png">. The frequency counts of shared species, <i>f</i><sub><i>rv</i></sub>, (Eq. 4a), are also extended to their phylogenetic versions:
               </p><div class="inline-equation" id="ecy2000-disp-0027"><span class="inline-equation__construct"><img class="" src="ecy2000-math-0071.png" alt="urn:x-wiley:00129658:media:ecy2000:ecy2000-math-0071" title="urn:x-wiley:00129658:media:ecy2000:ecy2000-math-0071"></span><span class="inline-equation__label">(9)</span></div>Here <i>g</i><sub><i>rv</i></sub> measures the total length of those shared branches with node/branch abundance <i>r</i> in Sample I and node/branch abundance <i>v</i> in Sample II. We can similarly define <i>g</i><sub><i>r</i>+</sub>, <i>g</i><sub>+<i>v</i></sub> and <i>g</i><sub>++</sub>, as we did in Eqs.&nbsp;(<a class="equationLink scrollableLink" href="#ecy2000-disp-0011" title="Link to equation">4b</a>)–(<a class="equationLink scrollableLink" href="#ecy2000-disp-0013" title="Link to equation">4d</a>). Here <i>g</i><sub><i>r</i>+</sub> denotes the total length of those shared branches that have abundance <i>r</i> in Sample I and non‐zero node/branch abundance in Sample II, with a similar interpretation for <i>g</i><sub>+<i>v</i></sub>. Now <i>g</i><sub>++</sub> becomes the observed <i>PD</i> shared by the two samples, i.e., <i>g</i><sub>++</sub>&nbsp;=&nbsp;<i>PD</i><sub><i>shared</i>,<i>obs</i></sub>. Thus {<i>g</i><sub><i>rv</i></sub>, <i>g</i><sub><i>r</i>+</sub>, <i>g</i><sub>+<i>v</i></sub>, <i>g</i><sub>++</sub>; <i>r</i>,<i> v&nbsp;</i>=<i>&nbsp;</i>0, 1, 2…} are analogous to {<i>f</i><sub><i>rv</i></sub>, <i>f</i><sub><i>r</i>+</sub>, <i>f</i><sub>+<i>v</i></sub>, <i>f</i><sub>++</sub>; <i>r</i>,<i> v&nbsp;</i>=<i>&nbsp;</i>0, 1, 2…} as defined in Eqs.&nbsp;(<a class="equationLink scrollableLink" href="#ecy2000-disp-0010" title="Link to equation">4a</a>) – (<a class="equationLink scrollableLink" href="#ecy2000-disp-0013" title="Link to equation">4d</a>).
 The only difference is that each of the former set measures the lengths
 of shared branches, whereas each of the later set counts shared 
species.
            <p></p>
            
            <p>The true shared <i>PD</i> can be expressed as the sum of four terms:
               </p><div class="inline-equation" id="ecy2000-disp-0028"><span class="inline-equation__construct"><img class="" src="ecy2000-math-0072.png" alt="urn:x-wiley:00129658:media:ecy2000:ecy2000-math-0072" title="urn:x-wiley:00129658:media:ecy2000:ecy2000-math-0072"></span><span class="inline-equation__label">(10a)</span></div>The four terms in the right hand side of the above equation&nbsp;represent, respectively, the observed shared <i>PD</i>, the shared <i>PD</i> that is missed only in Sample I, the shared <i>PD</i> that is missed only in Sample II, and the shared <i>PD</i> that is missed by both samples. For any shared branch segment with node/branch abundance <i>r</i>
 in each observed branch set, consider calculating the product of its 
true branch relative abundances in the two assemblages. The 
branch‐length‐weighted mean (per unit‐length) of the products among all 
such shared branches (their total length is <i>g</i><sub><i>rr</i></sub>) can be expressed as
               <div class="inline-equation" id="ecy2000-disp-0029"><span class="inline-equation__construct"><img class="" src="ecy2000-math-0073.png" alt="urn:x-wiley:00129658:media:ecy2000:ecy2000-math-0073" title="urn:x-wiley:00129658:media:ecy2000:ecy2000-math-0073"></span><span class="inline-equation__label"></span></div>
            <p></p>
            
            <p>The two‐assemblage phylogenetic version of the Good‐Turing formula (see Appendix&nbsp;<a class="suppLink scrollableLink" href="#support-information-section">S2</a> for a proof) provides the following estimator of λ<sub><i>rr</i></sub> which is similar to that for shared species richness in Eq.&nbsp;(<a class="equationLink scrollableLink" href="#ecy2000-disp-0016" title="Link to equation">5c</a>):
               </p><div class="inline-equation" id="ecy2000-disp-0030"><span class="inline-equation__construct"><img class="" src="ecy2000-math-0074.png" alt="urn:x-wiley:00129658:media:ecy2000:ecy2000-math-0074" title="urn:x-wiley:00129658:media:ecy2000:ecy2000-math-0074"></span><span class="inline-equation__label">(10b)</span></div>Under the intuitive expectation that <i>λ</i><sub>00</sub>&nbsp;≤&nbsp;<i>λ</i><sub>11</sub> (and this ordering is preserved by the corresponding estimates), we obtain the following lower bound of <i>g</i><sub>00</sub>:
               <div class="inline-equation" id="ecy2000-disp-0031"><span class="inline-equation__construct"><img class="" src="ecy2000-math-0075.png" alt="urn:x-wiley:00129658:media:ecy2000:ecy2000-math-0075" title="urn:x-wiley:00129658:media:ecy2000:ecy2000-math-0075"></span><span class="inline-equation__label">(10c)</span></div>
            <p></p>
            
            <p>Then the same estimation procedures we used to develop the Chao1‐shared estimator in Eq.&nbsp;(<a class="equationLink scrollableLink" href="#ecy2000-disp-0021" title="Link to equation">7d</a>) lead to the following Chao1‐<i>PD</i>‐shared estimator (Appendix&nbsp;<a class="suppLink scrollableLink" href="#support-information-section">S2</a>):
               </p><div class="inline-equation" id="ecy2000-disp-0032"><span class="inline-equation__construct"><img class="" src="ecy2000-math-0076.png" alt="urn:x-wiley:00129658:media:ecy2000:ecy2000-math-0076" title="urn:x-wiley:00129658:media:ecy2000:ecy2000-math-0076"></span><span class="inline-equation__label">(10d)</span></div>where <i>k</i><sub><i>i</i></sub>&nbsp;=&nbsp;(<i>n</i><sub><i>i</i></sub>&nbsp;−&nbsp;1)/<i>n</i><sub><i>i</i></sub>, <i>i&nbsp;</i>=<i>&nbsp;</i>1,
 2. The sum of the last three terms estimates the total length of 
undetected branches shared by two individual sub‐trees. A modification 
similar to that proposed in Eq.&nbsp;(<a class="equationLink scrollableLink" href="#ecy2000-disp-0026" title="Link to equation">8d</a>) can be applied to each of the three terms. The sufficient conditions for the Chao1‐<i>PD</i>‐shared
 estimator being nearly unbiased can be similarly formulated as those 
for the Chao1‐shared estimator, simply by replacing “shared species” 
with “shared nodes/branches.”
            <p></p>
            
            <p>The variance and confidence interval associated with this
 estimator follow directly from those for the Chao1‐shared estimator, by
 replacing the counts {<i>f</i><sub><i>rv</i></sub>, <i>f</i><sub><i>r</i>+</sub>, <i>f</i><sub>+<i>v</i></sub>, <i>f</i><sub>++</sub>; <i>r</i>,<i> v&nbsp;</i>=<i>&nbsp;</i>0, 1, 2, …} with {<i>g</i><sub><i>rv</i></sub>, <i>g</i><sub><i>r</i>+</sub>, <i>g</i><sub>+<i>v</i></sub>, <i>g</i><sub><i>++</i></sub>; <i>r</i>,<i> v&nbsp;</i>=<i>&nbsp;</i>0, 1, 2, …} in the formulas. This correspondence also reflects the analogy between the estimation of shared species and shared <i>PD</i>
 between two assemblages. A comparison of estimation formulas used in 
estimating shared species richness and in estimating shared <i>PD</i> is provided in Appendix&nbsp;<a class="suppLink scrollableLink" href="#support-information-section">S2</a>:
 Table&nbsp;S2, where any unit‐length branch shared by two assemblages 
plays the same role as a “shared species” in the estimation of shared 
species richness. The measures of shared branch lengths <img alt="urn:x-wiley:00129658:media:ecy2000:ecy2000-math-0077" class="section_image" src="ecy2000-math-0077.png"> used in estimating shared <i>PD</i> play a role corresponding to the frequency counts <img alt="urn:x-wiley:00129658:media:ecy2000:ecy2000-math-0078" class="section_image" src="ecy2000-math-0078.png">
 in estimating shared species diversity. Hence, not only 
within‐assemblage but also between‐assemblage measures defined at the 
species level can be converted to <i>PD</i> equivalents.
            </p>
            
         </div>
         
      </div>
      
      <div class="article-section__content" id="ecy2000-sec-0011">
         
         <h2 class="article-section__title section__title section1" id="ecy2000-sec-0011-title"> Functional Diversity</h2>
         
         <div class="article-section__sub-content" id="ecy2000-sec-0012">
            
            <h3 class="article-section__sub-title section2"> Undetected FAD in a single assemblage</h3>
            
            <p>Consider an assemblage in which all species are 
characterized by a set of functional traits, which can be categorical or
 continuous variables. The pairwise distances are calculated by some 
distance metric (e.g., Euclidean distance or Gower distance) based on 
the values of species traits. Let <i>d</i><sub><i>ij</i></sub> be the functional distance between the <i>i</i>th and <i>j</i>th species, with <i>d</i><sub><i>ij</i></sub> <i>= d</i><sub><i>ji</i></sub> &gt; 0. Walker et&nbsp;al.'s (<span><a href="#ecy2000-bib-0050" class="bibLink tab-link" data-tab="pane-pcw-references">1999</a></span>) <i>FAD</i> in the entire assemblage is expressed as <img alt="urn:x-wiley:00129658:media:ecy2000:ecy2000-math-0079" class="section_image" src="ecy2000-math-0079.png">.
 In our estimation procedures, we do not require zero distance between 
any two species. That is, intraspecific variability in traits is 
allowed, so that <i>d</i><sub><i>ii</i></sub> may be greater than zero. 
For example, when trait values are available at the individual level, we
 can simply define the distance between two species as the mean of all 
distances between any pair of individuals, one chosen respectively from 
each of the two species.
            </p>
            
            <p>In our approach, one unit length of distance is analogous
 to a pair of species shared between two identical assemblages. We first
 extend all frequency counts of shared species, <i>f</i><sub><i>rv</i></sub>, defined in Eqs.&nbsp;(<a class="equationLink scrollableLink" href="#ecy2000-disp-0010" title="Link to equation">4a</a>)–(<a class="equationLink scrollableLink" href="#ecy2000-disp-0013" title="Link to equation">4d</a>) to the corresponding measures of pairwise distance based on a sample of <i>n</i> individuals, with a species sample abundance set <img alt="urn:x-wiley:00129658:media:ecy2000:ecy2000-math-0080" class="section_image" src="ecy2000-math-0080.png">. Let <i>F</i><sub><i>rv</i></sub> denote the total distance of those species pairs with sample abundances <i>r</i> and <i>v</i> respectively for the two species in each pair. That is,
               </p><div class="inline-equation" id="ecy2000-disp-0033"><span class="inline-equation__construct"><img class="" src="ecy2000-math-0081.png" alt="urn:x-wiley:00129658:media:ecy2000:ecy2000-math-0081" title="urn:x-wiley:00129658:media:ecy2000:ecy2000-math-0081"></span><span class="inline-equation__label">(11)</span></div>We can similarly define <i>F</i><sub><i>r</i>+</sub>, <i>F</i><sub>+<i>v</i></sub> and <i>F</i><sub>++</sub>, as we did in Eqs.&nbsp;(<a class="equationLink scrollableLink" href="#ecy2000-disp-0011" title="Link to equation">4b</a>) – (<a class="equationLink scrollableLink" href="#ecy2000-disp-0013" title="Link to equation">4d</a>). Here <i>F</i><sub><i>r</i>+</sub> denotes the total distance of those species pairs with sample abundance <i>r</i> and <i>k</i>, where <i>k&nbsp;</i>&gt;<i>&nbsp;</i>0 can be any positive integer, with a similar definition for <i>F</i><sub>+<i>v</i></sub>, while <i>F</i><sub>++</sub> becomes the observed <i>FAD</i> in the sample, i.e., <i>F</i><sub>++</sub>&nbsp;=&nbsp;<i>FAD</i><sub><i>obs</i></sub>.
            <p></p>
            
            <p>Given one‐sample data (data for a single assemblage) and the species‐pairwise distance matrix for all species, the true <i>FAD</i> can be expressed as the sum of four terms:
               </p><div class="inline-equation" id="ecy2000-disp-0034"><span class="inline-equation__construct"><img class="" src="ecy2000-math-0082.png" alt="urn:x-wiley:00129658:media:ecy2000:ecy2000-math-0082" title="urn:x-wiley:00129658:media:ecy2000:ecy2000-math-0082"></span><span class="inline-equation__label">(12a)</span></div>The four terms in the right hand side of the above equation&nbsp;denote, respectively, the observed <i>FAD</i>, the undetected <i>FAD</i> for species pairs in which one species is missing from the sample (<i>F</i><sub>0+</sub> = <i>F</i><sub>+0</sub>), and the undetected <i>FAD</i> for species pairs in which both species are missing from the sample (<i>F</i><sub>00</sub>). This expression is analogous to Eq.&nbsp;<a class="equationLink scrollableLink" href="#ecy2000-disp-0017" title="Link to equation">6</a> for shared species and to Eq.&nbsp;(<a class="equationLink scrollableLink" href="#ecy2000-disp-0028" title="Link to equation">10a</a>) for shared <i>PD</i>. Out of all the species which appeared exactly <i>r</i>&nbsp;times
 in the sample, consider choosing any two to form a pair and calculating
 the product of their true relative abundances in the assemblage. The 
distance‐weighted mean (per unit‐distance) of the products among all 
such pairs (their total pairwise distance is&nbsp;<i>F</i><sub><i>rr</i></sub>) can be expressed as
               <div class="inline-equation" id="ecy2000-disp-0035"><span class="inline-equation__construct"><img class="" src="ecy2000-math-0083.png" alt="urn:x-wiley:00129658:media:ecy2000:ecy2000-math-0083" title="urn:x-wiley:00129658:media:ecy2000:ecy2000-math-0083"></span><span class="inline-equation__label"></span></div>
            <p></p>
            
            <p>The functional version of the Good‐Turing frequency formula provides the following estimator of θ<sub><i>rr</i></sub> (see Appendix&nbsp;<a class="suppLink scrollableLink" href="#support-information-section">S3</a> for a proof):
               </p><div class="inline-equation" id="ecy2000-disp-0036"><span class="inline-equation__construct"><img class="" src="ecy2000-math-0084.png" alt="urn:x-wiley:00129658:media:ecy2000:ecy2000-math-0084" title="urn:x-wiley:00129658:media:ecy2000:ecy2000-math-0084"></span><span class="inline-equation__label">(12b)</span></div>The
 above formula is slightly different from the two‐sample Good‐Turing 
formulas for estimating shared species (Eq. 5c) and shared <i>PD</i> 
(Eq. 10b), because, here, we only have one‐sample data taken from a 
single assemblage, whereas the data for the previous formulas are taken 
from two different assemblages. Under the intuitive expectation that <i>θ</i><sub>00</sub>&nbsp;≤&nbsp;<i>θ</i><sub>11</sub> (and this ordering is preserved by the corresponding estimates), we obtain the following lower bound for <i>F</i><sub>00</sub>:
               <div class="inline-equation" id="ecy2000-disp-0037"><span class="inline-equation__construct"><img class="" src="ecy2000-math-0085.png" alt="urn:x-wiley:00129658:media:ecy2000:ecy2000-math-0085" title="urn:x-wiley:00129658:media:ecy2000:ecy2000-math-0085"></span><span class="inline-equation__label">(12c)</span></div>
            <p></p>
            
            <p>Then the same estimation procedures as we obtained for Chao1‐shared in Eq.&nbsp;(<a class="equationLink scrollableLink" href="#ecy2000-disp-0021" title="Link to equation">7d</a>) leads to the following Chao1‐<i>FAD</i> estimator:
               </p><div class="inline-equation" id="ecy2000-disp-0038"><span class="inline-equation__construct"><img class="" src="ecy2000-math-0086.png" alt="urn:x-wiley:00129658:media:ecy2000:ecy2000-math-0086" title="urn:x-wiley:00129658:media:ecy2000:ecy2000-math-0086"></span><span class="inline-equation__label">(12d)</span></div>where <i>k</i>&nbsp;=&nbsp;(<i>n</i>&nbsp;−&nbsp;1)/<i>n</i>. The sum of the last three terms estimates the undetected <i>FAD</i>. A modification similar to that proposed in Eq.&nbsp;(<a class="equationLink scrollableLink" href="#ecy2000-disp-0026" title="Link to equation">8d</a>)
 can be applied to each of the three terms. The variance and confidence 
interval associated with this estimator follow directly from those for 
the Chao1‐shared estimator. Under the condition that undetected species 
and singletons have approximately homogenous abundances, the Chao1‐<i>FAD</i> estimator is nearly unbiased for any given species‐pairwise distance matrix.
            <p></p>
            
            <p>A summary of formulas and descriptions for estimating shared species richness and <i>FAD</i> is given in Appendix&nbsp;<a class="suppLink scrollableLink" href="#support-information-section">S3</a>: Table&nbsp;S1, where the analogy between the estimation procedures of the two measures can be seen. Chao et&nbsp;al. (<span><a href="#ecy2000-bib-0012" class="bibLink tab-link" data-tab="pane-pcw-references">2014a</a></span>) define a “functional entity” as a species pair with one unit of distance between the two species. In <i>FAD</i>,
 a functional entity plays the same role as a “shared species” between 
two assemblages. For example, a species‐pair with distance <i>d</i><sub><i>ij</i></sub> = 5 is counted as 5 “shared species” (i.e., 5 functional entities). Thus the measures of total distances of species pairs, {<i>F</i><sub><i>rv</i></sub>, <i>F</i><sub><i>r</i>+</sub>, <i>F</i><sub>+<i>v</i></sub>, <i>F</i><sub>++</sub>; <i>r</i>,<i> v&nbsp;</i>=<i>&nbsp;</i>0, 1, 2, …}, play the same roles as the counts of shared species richness {<i>f</i><sub><i>rv</i></sub>, <i>f</i><sub><i>r</i>+</sub>, <i>f</i><sub>+<i>v</i></sub>, <i>f</i><sub>++</sub>; <i>r</i>,<i> v&nbsp;</i>=<i>&nbsp;</i>0, 1, 2, …} (defined in Eqs. 4a – 4d).
            </p>
            
         </div>
         
         <div class="article-section__sub-content" id="ecy2000-sec-0013">
            
            <h3 class="article-section__sub-title section2"> Undetected shared FAD between two assemblages</h3>
            
            <p>Under the two‐assemblage model formulation and data framework described in the section <i>Two‐assemblage Good‐Turing Formulas</i>, we further assume that the functional distance between the <i>i</i>th and <i>j</i>th species for all <i>S</i> species of the <i>pooled</i> assemblage is denoted as <i>d</i><sub><i>ij</i></sub>, with <i>d</i><sub><i>ij</i></sub> <i>= d</i><sub><i>ji</i></sub> &gt; 0. The <i>FAD</i> of each individual assemblage can be computed from a sub‐matrix of the distance matrix of all <i>S</i> species in the pooled assemblage. Denote the number of shared species by <i>S</i><sub>12</sub>, shared <i>FAD</i> (i.e., the total species‐pairwise distance of two species each is shared by two assemblages) by <i>FAD</i><sub><i>shared</i></sub>, and the <i>FAD</i> based on the observed shared species by <i>FAD</i><sub><i>shared,</i></sub><sub><i>obs</i></sub>.
            </p>
            
            <p>Given two‐sample data with species sample abundances (<i>X</i><sub>11</sub>,&nbsp;<i>X</i><sub>21</sub>,&nbsp;…,&nbsp;<i>X</i><sub><i>S</i>1</sub>) and (<i>X</i><sub>12</sub>,&nbsp;<i>X</i><sub>22</sub>,&nbsp;…,&nbsp;<i>X</i><sub><i>S</i>2</sub>), as defined earlier, we extend all frequency counts <i>F</i><sub><i>rv</i></sub> defined in Eq.&nbsp;<a class="equationLink scrollableLink" href="#ecy2000-disp-0033" title="Link to equation">11</a> to the two‐assemblage case. That is, we define for <i>r</i>,<i> v</i>,<i> k</i>,<i> m&nbsp;</i>=<i>&nbsp;</i>0, 1, 2…
               </p><div class="inline-equation" id="ecy2000-disp-0039"><span class="inline-equation__construct"><img class="" src="ecy2000-math-0087.png" alt="urn:x-wiley:00129658:media:ecy2000:ecy2000-math-0087" title="urn:x-wiley:00129658:media:ecy2000:ecy2000-math-0087"></span><span class="inline-equation__label">(13)</span></div>Here <i>F</i><sub>(<i>rv</i>)(<i>km</i>)</sub> measures the total functional distance of those pairs of shared species with abundances (<i>r</i>,<i> v</i>) in Sample I and abundances (<i>k</i>,<i> m</i>)
 in Sample II for the two species in each pair. Where any index is 
replaced by a “+” sign, it means that those species occur at least 
once&nbsp;in the corresponding sample. Thus, we can define <i>F</i><sub>(+<i>v</i>)(<i>km</i>)</sub>, <i>F</i><sub>(+<i>v</i>)(+<i>m</i>)</sub>,&nbsp; <i>F</i><sub>(++)(<i>km</i>)</sub>, and <i>F</i><sub>(<i>rv</i>)(+<i>m</i>)</sub>, etc. Then shared <i>FAD</i> can be expressed as the sum of 16 terms (see Appendix&nbsp;<a class="suppLink scrollableLink" href="#support-information-section">S3</a>: Table&nbsp;S2 for illustrative details)
               <div class="inline-equation" id="ecy2000-disp-0040"><span class="inline-equation__construct"><img class="" src="ecy2000-math-0088.png" alt="urn:x-wiley:00129658:media:ecy2000:ecy2000-math-0088" title="urn:x-wiley:00129658:media:ecy2000:ecy2000-math-0088"></span><span class="inline-equation__label">(14a)</span></div>Here <i>F</i><sub>(++)(++)</sub> = <i>FAD</i><sub><i>shared,obs</i></sub> denotes the observed shared <i>FAD</i>;
 the other 15 terms are unknown. For each term, we can apply 
similar‐type Good‐Turing formulas to obtain an estimator as given in 
Appendix&nbsp;<a class="suppLink scrollableLink" href="#support-information-section">S3</a>: Table&nbsp;S3. Then we have the following Chao1‐<i>FAD</i>‐shared estimator:
               <div class="inline-equation" id="ecy2000-disp-0041"><span class="inline-equation__construct"><img class="" src="ecy2000-math-0089.png" alt="urn:x-wiley:00129658:media:ecy2000:ecy2000-math-0089" title="urn:x-wiley:00129658:media:ecy2000:ecy2000-math-0089"></span><span class="inline-equation__label">(14b)</span></div>A
 bootstrap method can be used to assess the sampling variance of the 
above estimator and to obtain the associated confidence interval.
            <p></p>
            
         </div>
         
      </div>
      
      <div class="article-section__content" id="ecy2000-sec-0014">
         
         <h2 class="article-section__title section__title section1" id="ecy2000-sec-0014-title"> Example</h2>
         
         <p>We apply all the estimators derived from the Good‐Turing 
frequency formula and its generalizations to the rain‐forest tree data 
described and discussed in Magnago et&nbsp;al. (<span><a href="#ecy2000-bib-0038" class="bibLink tab-link" data-tab="pane-pcw-references">2014</a></span>).
 The tree species abundance data were collected between January 2011 and
 January 2012 from 11 forest fragments in Espírito Santo State, in 
south‐eastern Brazil. Sampling data for 10 fragments included one Edge 
and one Interior transect, whereas in one fragment sampling data 
included two Edge and two Interior transects (Magnago et&nbsp;al. <span><a href="#ecy2000-bib-0038" class="bibLink tab-link" data-tab="pane-pcw-references">2014</a></span>),
 with a distance of 5.7&nbsp;±&nbsp;2.4&nbsp;km between transects. In 
total, the study comprised 11 fragments and 24 transects. Each Edge 
transect was placed about 5&nbsp;m inside the fragment and parallel to 
the forest edge, and each Interior transect was located at least 
300&nbsp;m from the nearest edge. One of the goals of the original study
 was to compare functional diversity between Edge habitat and Interior 
habitat. Within each transect, every living tree with a diameter at 
breast height (DBH) &gt; 4.8&nbsp;cm and 1.3&nbsp;m height was recorded.
         </p>
         
         <p>In the original data (Table&nbsp;S2 of Magnago et&nbsp;al. <span><a href="#ecy2000-bib-0038" class="bibLink tab-link" data-tab="pane-pcw-references">2014</a></span>),
 pooled from all 24 transect samples in 11 fragments, there were 443 
species among 4140 individual trees. However, in order to construct the 
phylogenetic tree for the observed species, using Phylomatic (<a href="http://www.phylodiversity.net/phylomatic" class="linkBehavior">http://www.phylodiversity.net/phylomatic</a>; Webb and Donoghue <span><a href="#ecy2000-bib-0051" class="bibLink tab-link" data-tab="pane-pcw-references">2005</a></span>), we had to exclude 18 species, including 5 species not identified in the species list of Magnago et&nbsp;al. (<span><a href="#ecy2000-bib-0038" class="bibLink tab-link" data-tab="pane-pcw-references">2014</a></span>)
 and 13 species not included in Phylomatic. Among the 18 species 
excluded, 12 species were found in both habitats, 3 species were unique 
to the Edge habitat and 3 species were unique to the Interior habitat. 
The data considered in our analysis thus include 425 species from a 
total of 3868 individuals; species abundance data are available in 
Github (<a href="https://github.com/AnneChao" class="linkBehavior">https://github.com/AnneChao</a>). The species abundance frequency counts of these 425 species are summarized in Table&nbsp;<a class="tableLink scrollableLink" title="Link to table" href="#ecy2000-tbl-0001">1</a>.
 There were 319 species (including 110 singletons and 48 doubletons) 
among 1794 individuals in the data from the Edge habitat, and 356 
species (including 123 singletons and 48 doubletons) among 2074 
individuals in the data from the Interior habitat. There were 250 
species in common between the two habitats in the data. The sample 
coverage estimates for these two habitats are nearly equal (93.9% for 
the Edge habitat and 94.1% for the Interior habitat), in spite of 
different sample sizes.
         </p>
         
         <div class="article-table-content" id="ecy2000-tbl-0001">
            <header class="article-table-caption"><span class="table-caption__label">Table 1.
            </span>The species frequency counts of the tree abundance 
data from two habitats (Edge and Interior) in south‐eastern Brazil 
(Magnago et&nbsp;al. <span><a href="#ecy2000-bib-0038" class="bibLink tab-link" data-tab="pane-pcw-references">2014</a></span>), where <i>f</i><sub><i>i</i></sub> denotes the number of species represented by exactly <i>i</i> individuals in the sample
            </header>
            
            
            
            <div class="article-table-content-wrapper">
               <table class="table article-section__table">
                  
                  
                  
                  
                  <thead>
                     
                     <tr>
                        
                        <th rowspan="2" class="bottom-bordered-cell right-bordered-cell left-aligned">Number of individuals in sample, <i>i</i></th>
                        
                        <th colspan="2" class="bottom-bordered-cell center-aligned">Number of species, <i>f</i><sub><i>i</i></sub></th>
                        
                     </tr>
                     
                     <tr>
                        
                        <th class="bottom-bordered-cell center-aligned">Edge</th>
                        
                        <th class="bottom-bordered-cell center-aligned">Interior</th>
                        
                     </tr>
                     
                  </thead>
                  
                  <tbody>
                     
                     <tr>
                        
                        <td class="right-bordered-cell left-aligned">1</td>
                        
                        <td class="center-aligned">110</td>
                        
                        <td class="center-aligned">123</td>
                        
                     </tr>
                     
                     <tr>
                        
                        <td class="right-bordered-cell left-aligned">2</td>
                        
                        <td class="center-aligned">48</td>
                        
                        <td class="center-aligned">48</td>
                        
                     </tr>
                     
                     <tr>
                        
                        <td class="right-bordered-cell left-aligned">3</td>
                        
                        <td class="center-aligned">38</td>
                        
                        <td class="center-aligned">41</td>
                        
                     </tr>
                     
                     <tr>
                        
                        <td class="right-bordered-cell left-aligned">4</td>
                        
                        <td class="center-aligned">28</td>
                        
                        <td class="center-aligned">32</td>
                        
                     </tr>
                     
                     <tr>
                        
                        <td class="right-bordered-cell left-aligned">5</td>
                        
                        <td class="center-aligned">13</td>
                        
                        <td class="center-aligned">19</td>
                        
                     </tr>
                     
                     <tr>
                        
                        <td class="right-bordered-cell left-aligned">6</td>
                        
                        <td class="center-aligned">11</td>
                        
                        <td class="center-aligned">17</td>
                        
                     </tr>
                     
                     <tr>
                        
                        <td class="right-bordered-cell left-aligned">7</td>
                        
                        <td class="center-aligned">12</td>
                        
                        <td class="center-aligned">6</td>
                        
                     </tr>
                     
                     <tr>
                        
                        <td class="right-bordered-cell left-aligned">8</td>
                        
                        <td class="center-aligned">5</td>
                        
                        <td class="center-aligned">7</td>
                        
                     </tr>
                     
                     <tr>
                        
                        <td class="right-bordered-cell left-aligned">9</td>
                        
                        <td class="center-aligned">4</td>
                        
                        <td class="center-aligned">6</td>
                        
                     </tr>
                     
                     <tr>
                        
                        <td class="right-bordered-cell left-aligned">10</td>
                        
                        <td class="center-aligned">6</td>
                        
                        <td class="center-aligned">7</td>
                        
                     </tr>
                     
                     <tr>
                        
                        <td class="right-bordered-cell left-aligned">11</td>
                        
                        <td class="center-aligned">3</td>
                        
                        <td class="center-aligned">6</td>
                        
                     </tr>
                     
                     <tr>
                        
                        <td class="right-bordered-cell left-aligned">12</td>
                        
                        <td class="center-aligned">4</td>
                        
                        <td class="center-aligned">3</td>
                        
                     </tr>
                     
                     <tr>
                        
                        <td class="right-bordered-cell left-aligned">13</td>
                        
                        <td class="center-aligned">3</td>
                        
                        <td class="center-aligned">3</td>
                        
                     </tr>
                     
                     <tr>
                        
                        <td class="right-bordered-cell left-aligned">14</td>
                        
                        <td class="center-aligned">5</td>
                        
                        <td class="center-aligned">3</td>
                        
                     </tr>
                     
                     <tr>
                        
                        <td class="right-bordered-cell left-aligned">15</td>
                        
                        <td class="center-aligned">2</td>
                        
                        <td class="center-aligned">3</td>
                        
                     </tr>
                     
                     <tr>
                        
                        <td class="right-bordered-cell left-aligned">16</td>
                        
                        <td class="center-aligned">4</td>
                        
                        <td class="center-aligned">4</td>
                        
                     </tr>
                     
                     <tr>
                        
                        <td class="right-bordered-cell left-aligned">17</td>
                        
                        <td class="center-aligned">2</td>
                        
                        <td class="center-aligned">2</td>
                        
                     </tr>
                     
                     <tr>
                        
                        <td class="right-bordered-cell left-aligned">18</td>
                        
                        <td class="center-aligned">2</td>
                        
                        <td class="center-aligned">1</td>
                        
                     </tr>
                     
                     <tr>
                        
                        <td class="right-bordered-cell left-aligned">19</td>
                        
                        <td class="center-aligned">2</td>
                        
                        <td class="center-aligned">3</td>
                        
                     </tr>
                     
                     <tr>
                        
                        <td class="right-bordered-cell left-aligned">20</td>
                        
                        <td class="center-aligned">2</td>
                        
                        <td class="center-aligned">4</td>
                        
                     </tr>
                     
                     <tr>
                        
                        <td class="right-bordered-cell left-aligned">21</td>
                        
                        <td class="center-aligned">1</td>
                        
                        <td class="center-aligned">5</td>
                        
                     </tr>
                     
                     <tr>
                        
                        <td class="right-bordered-cell left-aligned">23</td>
                        
                        <td class="center-aligned">2</td>
                        
                        <td class="center-aligned">2</td>
                        
                     </tr>
                     
                     <tr>
                        
                        <td class="right-bordered-cell left-aligned">25</td>
                        
                        <td class="center-aligned">1</td>
                        
                        <td class="center-aligned">1</td>
                        
                     </tr>
                     
                     <tr>
                        
                        <td class="right-bordered-cell left-aligned">27</td>
                        
                        <td class="center-aligned">1</td>
                        
                        <td class="center-aligned">2</td>
                        
                     </tr>
                     
                     <tr>
                        
                        <td class="right-bordered-cell left-aligned">28</td>
                        
                        <td class="center-aligned">1</td>
                        
                        <td class="center-aligned">1</td>
                        
                     </tr>
                     
                     <tr>
                        
                        <td class="right-bordered-cell left-aligned">30</td>
                        
                        <td class="center-aligned"></td>
                        
                        <td class="center-aligned">1</td>
                        
                     </tr>
                     
                     <tr>
                        
                        <td class="right-bordered-cell left-aligned">32</td>
                        
                        <td class="center-aligned">1</td>
                        
                        <td class="center-aligned">2</td>
                        
                     </tr>
                     
                     <tr>
                        
                        <td class="right-bordered-cell left-aligned">34</td>
                        
                        <td class="center-aligned"></td>
                        
                        <td class="center-aligned">1</td>
                        
                     </tr>
                     
                     <tr>
                        
                        <td class="right-bordered-cell left-aligned">35</td>
                        
                        <td class="center-aligned"></td>
                        
                        <td class="center-aligned">1</td>
                        
                     </tr>
                     
                     <tr>
                        
                        <td class="right-bordered-cell left-aligned">36</td>
                        
                        <td class="center-aligned">2</td>
                        
                        <td class="center-aligned"></td>
                        
                     </tr>
                     
                     <tr>
                        
                        <td class="right-bordered-cell left-aligned">37</td>
                        
                        <td class="center-aligned">1</td>
                        
                        <td class="center-aligned"></td>
                        
                     </tr>
                     
                     <tr>
                        
                        <td class="right-bordered-cell left-aligned">41</td>
                        
                        <td class="center-aligned">1</td>
                        
                        <td class="center-aligned"></td>
                        
                     </tr>
                     
                     <tr>
                        
                        <td class="right-bordered-cell left-aligned">45</td>
                        
                        <td class="center-aligned">1</td>
                        
                        <td class="center-aligned"></td>
                        
                     </tr>
                     
                     <tr>
                        
                        <td class="right-bordered-cell left-aligned">46</td>
                        
                        <td class="center-aligned">1</td>
                        
                        <td class="center-aligned"></td>
                        
                     </tr>
                     
                     <tr>
                        
                        <td class="right-bordered-cell left-aligned">49</td>
                        
                        <td class="center-aligned">1</td>
                        
                        <td class="center-aligned"></td>
                        
                     </tr>
                     
                     <tr>
                        
                        <td class="right-bordered-cell left-aligned">52</td>
                        
                        <td class="center-aligned"></td>
                        
                        <td class="center-aligned">1</td>
                        
                     </tr>
                     
                     <tr>
                        
                        <td class="right-bordered-cell left-aligned">110</td>
                        
                        <td class="center-aligned">1</td>
                        
                        <td class="center-aligned"></td>
                        
                     </tr>
                     
                     <tr>
                        
                        <td class="right-bordered-cell left-aligned">123</td>
                        
                        <td class="center-aligned"></td>
                        
                        <td class="center-aligned">1</td>
                        
                     </tr>
                     
                     <tr>
                        
                        <td class="right-bordered-cell left-aligned">140</td>
                        
                        <td class="center-aligned"></td>
                        
                        <td class="center-aligned">1</td>
                        
                     </tr>
                     
                  </tbody>
                  
               </table>
            </div>
            
            
            
            <div class="article-section__table-footnotes">
               <h3 class="table-notes-label">Notes</h3>
               <ul>
                  <li id="ecy2000-note-0002">
                     Detailed species abundances are taken from Table&nbsp;S2 of Magnago et&nbsp;al. (<span><a href="#ecy2000-bib-0038" class="bibLink tab-link" data-tab="pane-pcw-references">2014</a></span>). For the Edge habitat, <i>S</i><sub><i>obs</i></sub><i>&nbsp;</i>=&nbsp;319, <i>n&nbsp;</i>=&nbsp;1794, sample coverage estimate&nbsp;=&nbsp;93.9%; for the Interior habitat, <i>S</i><sub><i>obs</i></sub>&nbsp;=&nbsp;356, <i>n&nbsp;</i>=&nbsp;2074, sample coverage estimate&nbsp;=&nbsp;94.1%.
                     
                  </li>
               </ul>
            </div>
         </div>
         
         <p>To illustrate our estimators, we focus only on the diversity
 analysis for the pooled samples over all fragments. That is, we regard 
the pooled tree records as sampling data from the entire study area 
represented by the 11 fragments and aim to infer and compare the 
estimated true or asymptotic diversity (observed plus undetected) of the
 two habitats for the whole study area. We could also apply our 
estimation to the Edge and Interior transect data within each fragment 
to compare the diversity of the two habitats; in this case, the target 
becomes the estimated true diversity of each fragment. Comparison of 
compositional, phylogenetic, and functional differentiation among the 11
 fragments, i.e., beta diversity with adjustment for under‐sampling bias
 will be reported elsewhere.</p>
         
         <p>Estimates for species richness and shared species richness between the two habitats appear in Table&nbsp;<a class="tableLink scrollableLink" title="Link to table" href="#ecy2000-tbl-0002">2</a>.
 The undetected species richness estimates (Eq. 2c) for the Edge and 
Interior habitats are, respectively, 126 and 158 species, implying that 
the total richness estimate (Eq. 3a) for the Edge habitat is 445 
species, with a 95% confidence interval of (396, 525), and for the 
Interior habitat 514 species, with a 95% confidence interval of (455, 
609). These results show that the Interior habitat has higher estimated 
species richness; however, the difference is not statistically 
significant at a level of 5%. The undetected shared species richness 
between the two habitats is estimated to be 139, leading to an estimate 
of shared richness (Eq. 7d) of 389 species, with a 95% confidence 
interval of (347, 450). Although the true species richness and shared 
species richness between the two habitats are unknown and our estimates 
theoretically represent lower bounds, the data collector among us (L. F.
 S. Magnago) believes these estimates provide reasonable adjustments 
based on his experiences in the fields, CVRD (Herbarium of Natural Vale 
Reserve) herbarium collection, and other floristic studies for the 
region (Rolim and Nascimento <span><a href="#ecy2000-bib-0046" class="bibLink tab-link" data-tab="pane-pcw-references">1997</a></span>, Jesus and Rolim <span><a href="#ecy2000-bib-0035" class="bibLink tab-link" data-tab="pane-pcw-references">2005</a></span>, Paula and Soares <span><a href="#ecy2000-bib-0043" class="bibLink tab-link" data-tab="pane-pcw-references">2011</a></span>).
 We also assume that this conclusion can be extended to the following 
estimates of phylogenetic diversity and functional diversity. The 
biological processes that may give rise detection failure are elaborated
 in the <i>Discussion</i> section.
         </p>
         
         <div class="article-table-content" id="ecy2000-tbl-0002">
            <header class="article-table-caption"><span class="table-caption__label">Table 2.
            </span>Summary of species data and richness estimates based 
on tree species abundance data for the Edge and Interior habitats in 
forest fragments of south‐eastern Brazil (Magnago et&nbsp;al. <span><a href="#ecy2000-bib-0038" class="bibLink tab-link" data-tab="pane-pcw-references">2014</a></span>),
 showing (a) undetected species richness and Chao1 richness point and 
interval estimates for each habitat (see Eq. 3a) and (b) undetected 
shared species richness between the two habitats and the corresponding 
Chao1‐shared richness point and interval estimates (see Eq. 7d)
            </header>
            
            
            
            <div class="article-table-content-wrapper">
               <table class="table article-section__table">
                  
                  
                  
                  
                  
                  
                  
                  
                  
                  <thead>
                     
                     <tr>
                        
                        <th class="bottom-bordered-cell right-bordered-cell left-aligned">Habitat</th>
                        
                        <th class="bottom-bordered-cell center-aligned">Sample size</th>
                        
                        <th class="bottom-bordered-cell center-aligned">
                           <i>f</i>
                           <sub>1</sub>
                           
                        </th>
                        
                        <th class="bottom-bordered-cell center-aligned">
                           <i>f</i>
                           <sub>2</sub>
                           
                        </th>
                        
                        <th class="bottom-bordered-cell center-aligned">Observed richness</th>
                        
                        <th class="bottom-bordered-cell center-aligned">Undetected richness</th>
                        
                        <th class="bottom-bordered-cell center-aligned">Chao1 richness</th>
                        
                        <th class="bottom-bordered-cell center-aligned">95% conf. interval</th>
                        
                     </tr>
                     
                  </thead>
                  
                  <tbody>
                     
                     <tr>
                        
                        <td colspan="8" class="left-aligned">(a)</td>
                        
                     </tr>
                     
                     <tr>
                        
                        <td class="right-bordered-cell left-aligned">Edge</td>
                        
                        <td class="center-aligned">1794</td>
                        
                        <td class="center-aligned">110</td>
                        
                        <td class="center-aligned">48</td>
                        
                        <td class="center-aligned">319</td>
                        
                        <td class="center-aligned">126</td>
                        
                        <td class="center-aligned">445</td>
                        
                        <td class="center-aligned">396, 525</td>
                        
                     </tr>
                     
                     <tr>
                        
                        <td class="right-bordered-cell left-aligned">Interior</td>
                        
                        <td class="center-aligned">2074</td>
                        
                        <td class="center-aligned">123</td>
                        
                        <td class="center-aligned">48</td>
                        
                        <td class="center-aligned">356</td>
                        
                        <td class="center-aligned">158</td>
                        
                        <td class="center-aligned">514</td>
                        
                        <td class="center-aligned">455, 609</td>
                        
                     </tr>
                     
                  </tbody>
                  
               </table>
            </div>
            
            <div class="article-table-content-wrapper">
               <table class="table article-section__table">
                  
                  
                  
                  
                  
                  
                  
                  
                  
                  
                  
                  
                  
                  
                  <thead>
                     
                     <tr>
                        
                        <th class="bottom-bordered-cell right-bordered-cell left-aligned">Observed shared richness</th>
                        
                        <th class="bottom-bordered-cell center-aligned">
                           <i>f</i>
                           <sub><i>+</i>1</sub>
                           
                        </th>
                        
                        <th class="bottom-bordered-cell center-aligned">
                           <i>f</i>
                           <sub><i>+</i>2</sub>
                           
                        </th>
                        
                        <th class="bottom-bordered-cell center-aligned">
                           <i>f</i>
                           <sub>1<i>+</i></sub>
                           
                        </th>
                        
                        <th class="bottom-bordered-cell center-aligned">
                           <i>f</i>
                           <sub>2<i>+</i></sub>
                           
                        </th>
                        
                        <th class="bottom-bordered-cell center-aligned">
                           <i>f</i>
                           <sub>11</sub>
                           
                        </th>
                        
                        <th class="bottom-bordered-cell center-aligned">
                           <i>f</i>
                           <sub>22</sub>
                           
                        </th>
                        
                        <th class="bottom-bordered-cell center-aligned">
                           <img alt="urn:x-wiley:00129658:media:ecy2000:ecy2000-math-0090" class="section_image" src="ecy2000-math-0090.png">
                           
                        </th>
                        
                        <th class="bottom-bordered-cell center-aligned">
                           <img alt="urn:x-wiley:00129658:media:ecy2000:ecy2000-math-0091" class="section_image" src="ecy2000-math-0091.png">
                           
                        </th>
                        
                        <th class="bottom-bordered-cell center-aligned">
                           <img alt="urn:x-wiley:00129658:media:ecy2000:ecy2000-math-0092" class="section_image" src="ecy2000-math-0092.png">
                           
                        </th>
                        
                        <th class="bottom-bordered-cell center-aligned">Undetected shared richness</th>
                        
                        <th class="bottom-bordered-cell center-aligned">Chao1‐shared richness</th>
                        
                        <th class="bottom-bordered-cell center-aligned">95% conf. interval</th>
                        
                     </tr>
                     
                  </thead>
                  
                  <tbody>
                     
                     <tr>
                        
                        <td colspan="13" class="left-aligned">(b)</td>
                        
                     </tr>
                     
                     <tr>
                        
                        <td class="right-bordered-cell left-aligned">250</td>
                        
                        <td class="center-aligned">64</td>
                        
                        <td class="center-aligned">30</td>
                        
                        <td class="center-aligned">60</td>
                        
                        <td class="center-aligned">37</td>
                        
                        <td class="center-aligned">25</td>
                        
                        <td class="center-aligned">7</td>
                        
                        <td class="center-aligned">68</td>
                        
                        <td class="center-aligned">49</td>
                        
                        <td class="center-aligned">22</td>
                        
                        <td class="center-aligned">139</td>
                        
                        <td class="center-aligned">389</td>
                        
                        <td class="center-aligned">347, 450</td>
                        
                     </tr>
                     
                  </tbody>
                  
               </table>
            </div>
            
            
         </div>
         
         <p>The phylogenetic tree (in Newick format) of the 425 observed
 species which was constructed using the software Phylomatic (Webb and 
Donoghue <span><a href="#ecy2000-bib-0051" class="bibLink tab-link" data-tab="pane-pcw-references">2005</a></span>) is given in Github (<a href="https://github.com/AnneChao" class="linkBehavior">https://github.com/AnneChao</a>). The observed Faith's <i>PD</i> values are, respectively, 24516 and 27727&nbsp;Myr in the Edge and Interior habitat, with a shared <i>PD</i> 20680&nbsp;Myr. Table&nbsp;<a class="tableLink scrollableLink" title="Link to table" href="#ecy2000-tbl-0003">3</a> shows the estimates for Faith's <i>PD</i> in each habitat and shared <i>PD</i> between the two habitats. The <i>PD</i>
 estimates (Eq. 8d) for the Edge and Interior habitats are, 
respectively, 32011 with a 95% confidence interval of (31542, 32511), 
and 34550 with a 95% confidence interval of (34143, 34983). The two 
intervals do not overlap, implying that the Interior habitat has 
significantly higher <i>PD</i> than the Edge habitat. The shared <i>PD</i> between the two habitats is estimated (Eq. 10d) to be 29360&nbsp;Myr, with a 95% confidence interval of (28976, 29761).
         </p>
         
         <div class="article-table-content" id="ecy2000-tbl-0003">
            <header class="article-table-caption"><span class="table-caption__label">Table 3.
            </span>Summary of phylogenetic data and diversity estimates 
for the Edge and Interior habitats in forest fragments of south‐eastern 
Brazil (Magnago et&nbsp;al. <span><a href="#ecy2000-bib-0038" class="bibLink tab-link" data-tab="pane-pcw-references">2014</a></span>), showing (a) undetected <i>PD</i> and Chao1‐<i>PD</i> point and interval estimates for each habitat (see Eq. 8d), and (b) undetected shared <i>PD</i> between the two habitats and the corresponding Chao1‐<i>PD</i>‐shared point and interval estimates (see Eq. 10d)
            </header>
            
            
            
            <div class="article-table-content-wrapper">
               <table class="table article-section__table">
                  
                  
                  
                  
                  
                  
                  
                  
                  
                  <thead>
                     
                     <tr>
                        
                        <th class="bottom-bordered-cell right-bordered-cell left-aligned">Habitat</th>
                        
                        <th class="bottom-bordered-cell center-aligned">Sample size</th>
                        
                        <th class="bottom-bordered-cell center-aligned">
                           <i>g</i>
                           <sub>1</sub>
                           
                        </th>
                        
                        <th class="bottom-bordered-cell center-aligned">
                           <i>g</i>
                           <sub>2</sub>
                           
                        </th>
                        
                        <th class="bottom-bordered-cell center-aligned">Observed <i>PD</i></th>
                        
                        <th class="bottom-bordered-cell center-aligned">Undetected <i>PD</i></th>
                        
                        <th class="bottom-bordered-cell center-aligned">Chao1‐<i>PD</i></th>
                        
                        <th class="bottom-bordered-cell center-aligned">95% conf. interval</th>
                        
                     </tr>
                     
                  </thead>
                  
                  <tbody>
                     
                     <tr>
                        
                        <td colspan="8" class="left-aligned">(a)</td>
                        
                     </tr>
                     
                     <tr>
                        
                        <td class="right-bordered-cell left-aligned">Edge</td>
                        
                        <td class="center-aligned">1794</td>
                        
                        <td class="center-aligned">6578</td>
                        
                        <td class="center-aligned">2885</td>
                        
                        <td class="center-aligned">24516</td>
                        
                        <td class="center-aligned">7495</td>
                        
                        <td class="center-aligned">32011</td>
                        
                        <td class="center-aligned">31542, 32511</td>
                        
                     </tr>
                     
                     <tr>
                        
                        <td class="right-bordered-cell left-aligned">Interior</td>
                        
                        <td class="center-aligned">2074</td>
                        
                        <td class="center-aligned">7065</td>
                        
                        <td class="center-aligned">3656</td>
                        
                        <td class="center-aligned">27727</td>
                        
                        <td class="center-aligned">6823</td>
                        
                        <td class="center-aligned">34550</td>
                        
                        <td class="center-aligned">34143, 34983</td>
                        
                     </tr>
                     
                  </tbody>
                  
               </table>
            </div>
            
            <div class="article-table-content-wrapper">
               <table class="table article-section__table">
                  
                  
                  
                  
                  
                  
                  
                  
                  
                  
                  
                  
                  
                  
                  <thead>
                     
                     <tr>
                        
                        <th class="bottom-bordered-cell right-bordered-cell left-aligned">Observed shared <i>PD</i></th>
                        
                        <th class="bottom-bordered-cell center-aligned">
                           <i>g</i>
                           <sub><i>+</i>1</sub>
                           
                        </th>
                        
                        <th class="bottom-bordered-cell center-aligned">
                           <i>g</i>
                           <sub><i>+</i>2</sub>
                           
                        </th>
                        
                        <th class="bottom-bordered-cell center-aligned">
                           <i>g</i>
                           <sub>1<i>+</i></sub>
                           
                        </th>
                        
                        <th class="bottom-bordered-cell center-aligned">
                           <i>g</i>
                           <sub>2<i>+</i></sub>
                           
                        </th>
                        
                        <th class="bottom-bordered-cell center-aligned">
                           <i>g</i>
                           <sub>11</sub>
                           
                        </th>
                        
                        <th class="bottom-bordered-cell center-aligned">
                           <i>g</i>
                           <sub>22</sub>
                           
                        </th>
                        
                        <th class="bottom-bordered-cell center-aligned">
                           <img alt="urn:x-wiley:00129658:media:ecy2000:ecy2000-math-0093" class="section_image" src="ecy2000-math-0093.png">
                           
                        </th>
                        
                        <th class="bottom-bordered-cell center-aligned">
                           <img alt="urn:x-wiley:00129658:media:ecy2000:ecy2000-math-0094" class="section_image" src="ecy2000-math-0094.png">
                           
                        </th>
                        
                        <th class="bottom-bordered-cell center-aligned">
                           <img alt="urn:x-wiley:00129658:media:ecy2000:ecy2000-math-0095" class="section_image" src="ecy2000-math-0095.png">
                           
                        </th>
                        
                        <th class="bottom-bordered-cell center-aligned">Undetected shared <i>PD</i></th>
                        
                        <th class="bottom-bordered-cell center-aligned">Chao1‐<i>PD</i> shared
                        </th>
                        
                        <th class="bottom-bordered-cell center-aligned">95% conf. interval</th>
                        
                     </tr>
                     
                  </thead>
                  
                  <tbody>
                     
                     <tr>
                        
                        <td colspan="13" class="left-aligned">(b)</td>
                        
                     </tr>
                     
                     <tr>
                        
                        <td class="right-bordered-cell left-aligned">20680</td>
                        
                        <td class="center-aligned">3888</td>
                        
                        <td class="center-aligned">2177</td>
                        
                        <td class="center-aligned">3929</td>
                        
                        <td class="center-aligned">2125</td>
                        
                        <td class="center-aligned">1711</td>
                        
                        <td class="center-aligned">463</td>
                        
                        <td class="center-aligned">3470</td>
                        
                        <td class="center-aligned">3630</td>
                        
                        <td class="center-aligned">1579</td>
                        
                        <td class="center-aligned">8680</td>
                        
                        <td class="center-aligned">29360</td>
                        
                        <td class="center-aligned">28976, 29761</td>
                        
                     </tr>
                     
                  </tbody>
                  
               </table>
            </div>
            
            
            
            <div class="article-section__table-footnotes">
               <h3 class="table-notes-label">Note</h3>
               <ul>
                  <li id="ecy2000-note-0003">
                     The phylogenetic tree for all observed species is based on Phylomatic (Webb and Donoghue <span><a href="#ecy2000-bib-0051" class="bibLink tab-link" data-tab="pane-pcw-references">2005</a></span>).
                     
                  </li>
               </ul>
            </div>
         </div>
         
         <p>The functional traits for these tree data were based 
primarily on morphological and physical characteristics of trees, their 
roles as trophic resources, their dispersal modes, and their roles in 
carbon storage and forest structure. All observed species were described
 by a set of six functional traits, including five categorical 
variables: fruit size (size categories), seed size (size categories), 
fruit type, fruit dispersal syndrome, and successional group, together 
with one quantitative variable: wood density. Based on these six traits,
 the species pairwise distance matrix for the data (pooled across 
replicates) was calculated by a Gower mixed‐variables coefficient of 
distance with equal weights for all traits; the Gower distance matrix of
 the 425 observed species is provided in Github (<a href="https://github.com/AnneChao" class="linkBehavior">https://github.com/AnneChao</a>). Table&nbsp;<a class="tableLink scrollableLink" title="Link to table" href="#ecy2000-tbl-0004">4</a> shows the estimates for <i>FAD</i> in each habitat and shared <i>FAD</i> between the two habitats. In the Edge and Interior habitats, the observed <i>FAD</i> (total pairwise Gower distances) are 36603 and 43438, respectively. Based on Eq.&nbsp;(<a class="equationLink scrollableLink" href="#ecy2000-disp-0038" title="Link to equation">12d</a>), the <i>FAD</i>
 estimate is 69670 in the Edge habitat, with a 95% confidence interval 
of (68764, 70602), and 86802 in the Interior habitat, with a 95% 
confidence interval of (85659, 87975). Here each estimate of true <i>FAD</i> is almost double the observed <i>FAD</i>. We can conclude that the <i>FAD</i> in the Interior habitat is significantly higher than the <i>FAD</i> in the Edge habitat because the two intervals do not overlap. The shared <i>FAD</i>
 between the two habitats is estimated (Eq. 14b) to be 49061 with a 95% 
bootstrap confidence interval of (42034, 58562) based on 200 bootstrap 
replications. This wide confidence interval reflects large sampling 
variance due to the estimation of 15 parameters and each estimate is 
subject to some degree of sampling uncertainty.
         </p>
         
         <div class="article-table-content" id="ecy2000-tbl-0004">
            <header class="article-table-caption"><span class="table-caption__label">Table 4.
            </span>Summary of functional data and diversity estimates 
based on tree species abundance data, as well as species traits 
collected from the Edge and Interior habitats of forest fragments in 
south‐eastern Brazil (Magnago et&nbsp;al. <span><a href="#ecy2000-bib-0038" class="bibLink tab-link" data-tab="pane-pcw-references">2014</a></span>), showing (a) undetected <i>FAD</i> and Chao1‐<i>FAD</i> point and interval estimates for each habitat (see Eq. 12d), and (b) undetected shared <i>FAD</i> between the two habitats and the corresponding Chao1‐<i>FAD</i>‐shared point and interval estimates (see Eq. 14b and Appendix&nbsp;<a class="suppLink scrollableLink" href="#support-information-section">S3</a>)
            </header>
            
            
            
            <div class="article-table-content-wrapper">
               <table class="table article-section__table">
                  
                  
                  
                  
                  
                  
                  
                  
                  
                  
                  
                  
                  <thead>
                     
                     <tr>
                        
                        <th class="bottom-bordered-cell right-bordered-cell left-aligned">Habitat</th>
                        
                        <th class="bottom-bordered-cell center-aligned">Observed <i>FAD</i></th>
                        
                        <th class="bottom-bordered-cell center-aligned"><i>F</i><sub><i>+</i>1</sub> (<i>= F</i><sub>1<i>+</i></sub>)
                        </th>
                        
                        <th class="bottom-bordered-cell center-aligned"><i>F</i><sub><i>+</i>2</sub> (<i>=F</i><sub>2<i>+</i></sub>)
                        </th>
                        
                        <th class="bottom-bordered-cell center-aligned">
                           <i>F</i>
                           <sub>11</sub>
                           
                        </th>
                        
                        <th class="bottom-bordered-cell center-aligned">
                           <i>F</i>
                           <sub>22</sub>
                           
                        </th>
                        
                        <th class="bottom-bordered-cell center-aligned">
                           <img alt="urn:x-wiley:00129658:media:ecy2000:ecy2000-math-0096" class="section_image" src="ecy2000-math-0096.png">
                           
                        </th>
                        
                        <th class="bottom-bordered-cell center-aligned">
                           <img alt="urn:x-wiley:00129658:media:ecy2000:ecy2000-math-0097" class="section_image" src="ecy2000-math-0097.png">
                           
                        </th>
                        
                        <th class="bottom-bordered-cell center-aligned">Undetected <i>FAD</i></th>
                        
                        <th class="bottom-bordered-cell center-aligned">Chao1‐<i>FAD</i></th>
                        
                        <th class="bottom-bordered-cell center-aligned">95% conf. interval</th>
                        
                     </tr>
                     
                  </thead>
                  
                  <tbody>
                     
                     <tr>
                        
                        <td colspan="11" class="left-aligned">(a)</td>
                        
                     </tr>
                     
                     <tr>
                        
                        <td class="right-bordered-cell left-aligned">Edge</td>
                        
                        <td class="center-aligned">36603</td>
                        
                        <td class="center-aligned">12452</td>
                        
                        <td class="center-aligned">5572</td>
                        
                        <td class="center-aligned">4200</td>
                        
                        <td class="center-aligned">837</td>
                        
                        <td class="center-aligned">13906</td>
                        
                        <td class="center-aligned">5256</td>
                        
                        <td class="center-aligned">33068</td>
                        
                        <td class="center-aligned">69670</td>
                        
                        <td class="center-aligned">68764, 70602</td>
                        
                     </tr>
                     
                     <tr>
                        
                        <td class="right-bordered-cell left-aligned">Interior</td>
                        
                        <td class="center-aligned">43438</td>
                        
                        <td class="center-aligned">14769</td>
                        
                        <td class="center-aligned">6059</td>
                        
                        <td class="center-aligned">4940</td>
                        
                        <td class="center-aligned">825</td>
                        
                        <td class="center-aligned">17992</td>
                        
                        <td class="center-aligned">7380</td>
                        
                        <td class="center-aligned">43364</td>
                        
                        <td class="center-aligned">86802</td>
                        
                        <td class="center-aligned">85659, 87975</td>
                        
                     </tr>
                     
                  </tbody>
                  
               </table>
            </div>
            
            <div class="article-table-content-wrapper">
               <table class="table article-section__table">
                  
                  
                  
                  
                  
                  
                  
                  
                  
                  
                  
                  
                  
                  
                  <thead>
                     
                     <tr>
                        
                        <th class="bottom-bordered-cell right-bordered-cell left-aligned">Observed shared <i>FAD</i></th>
                        
                        <th class="bottom-bordered-cell center-aligned">
                           <img alt="urn:x-wiley:00129658:media:ecy2000:ecy2000-math-0098" class="section_image" src="ecy2000-math-0098.png">
                           
                        </th>
                        
                        <th class="bottom-bordered-cell center-aligned">
                           <img alt="urn:x-wiley:00129658:media:ecy2000:ecy2000-math-0099" class="section_image" src="ecy2000-math-0099.png">
                           
                        </th>
                        
                        <th class="bottom-bordered-cell center-aligned">
                           <img alt="urn:x-wiley:00129658:media:ecy2000:ecy2000-math-0100" class="section_image" src="ecy2000-math-0100.png">
                           
                        </th>
                        
                        <th class="bottom-bordered-cell center-aligned">
                           <img alt="urn:x-wiley:00129658:media:ecy2000:ecy2000-math-0101" class="section_image" src="ecy2000-math-0101.png">
                           
                        </th>
                        
                        <th class="bottom-bordered-cell center-aligned">
                           <img alt="urn:x-wiley:00129658:media:ecy2000:ecy2000-math-0102" class="section_image" src="ecy2000-math-0102.png">
                           
                        </th>
                        
                        <th class="bottom-bordered-cell center-aligned">
                           <img alt="urn:x-wiley:00129658:media:ecy2000:ecy2000-math-0103" class="section_image" src="ecy2000-math-0103.png">
                           
                        </th>
                        
                        <th class="bottom-bordered-cell center-aligned">
                           <img alt="urn:x-wiley:00129658:media:ecy2000:ecy2000-math-0104" class="section_image" src="ecy2000-math-0104.png">
                           
                        </th>
                        
                        <th class="bottom-bordered-cell center-aligned">
                           <img alt="urn:x-wiley:00129658:media:ecy2000:ecy2000-math-0105" class="section_image" src="ecy2000-math-0105.png">
                           
                        </th>
                        
                        <th class="bottom-bordered-cell center-aligned">
                           <img alt="urn:x-wiley:00129658:media:ecy2000:ecy2000-math-0106" class="section_image" src="ecy2000-math-0106.png">
                           
                        </th>
                        
                        <th class="bottom-bordered-cell center-aligned">Undetected shared <i>FAD</i></th>
                        
                        <th class="bottom-bordered-cell center-aligned">Chao1‐<i>FAD</i> shared
                        </th>
                        
                        <th class="bottom-bordered-cell center-aligned">95% conf. interval</th>
                        
                     </tr>
                     
                  </thead>
                  
                  <tbody>
                     
                     <tr>
                        
                        <td colspan="13" class="left-aligned">(b)</td>
                        
                     </tr>
                     
                     <tr>
                        
                        <td class="right-bordered-cell left-aligned">22079</td>
                        
                        <td class="center-aligned">727</td>
                        
                        <td class="center-aligned">5793</td>
                        
                        <td class="center-aligned">671</td>
                        
                        <td class="center-aligned">3906</td>
                        
                        <td class="center-aligned">1028</td>
                        
                        <td class="center-aligned">166</td>
                        
                        <td class="center-aligned">1753</td>
                        
                        <td class="center-aligned">128</td>
                        
                        <td class="center-aligned">34</td>
                        
                        <td class="center-aligned">26981</td>
                        
                        <td class="center-aligned">49061</td>
                        
                        <td class="center-aligned">42034, 58562</td>
                        
                     </tr>
                     
                  </tbody>
                  
               </table>
            </div>
            
            
            
            <div class="article-section__table-footnotes">
               <h3 class="table-notes-label">Note</h3>
               <ul>
                  <li id="ecy2000-note-0004">
                     The functional distance matrix between any two 
observed species in the pooled assemblage is obtained by Gower distance.
                     
                  </li>
               </ul>
            </div>
         </div>
         
         <p>Our analysis suggests that the pooled Interior habitat has 
significantly higher phylogenetic diversity and significantly higher 
functional diversity than the pooled Edge habitat, but no such 
significance can be concluded for species diversity. These results are 
generally consistent with previous findings (Magnago et&nbsp;al. <span><a href="#ecy2000-bib-0038" class="bibLink tab-link" data-tab="pane-pcw-references">2014</a></span>, Matos et&nbsp;al. <span><a href="#ecy2000-bib-0040" class="bibLink tab-link" data-tab="pane-pcw-references">2017</a></span>)
 based on empirical data and model expectations. The previous authors 
found that the Edge habitat characteristics (microclimate, light, etc.) 
do not change with increasing fragment size whereas the Interior habitat
 characteristics change with fragment size. Consequently, fragment size 
usually does not produce effects on species richness and composition at 
fragment edges, but strongly influences the diversity in the interior of
 fragments. Because all fragment sizes are considered together in this 
paper, we thus expect greater diversity values for Interior than for 
Edge habitats.
         </p>
         
      </div>
      
      <div class="article-section__content" id="ecy2000-sec-0015">
         
         <h2 class="article-section__title section__title section1" id="ecy2000-sec-0015-title"> Conclusion and Discussion</h2>
         
         <p>We have generalized the original one‐assemblage Good‐Turing 
frequency formula (Eqs. 1c and 1d) to the case of two assemblages (Eq. 
5c), and also extended it to a phylogenetic version (Eqs. 8b and 10b) as
 well as a functional trait version (Eq. 12b and Appendix&nbsp;<a class="suppLink scrollableLink" href="#support-information-section">S3</a>:
 Table&nbsp;S3). We have also applied the original and generalized 
formulas to obtain various estimators of undetected species, 
phylogenetic, and functional diversity, as summarized below.
         </p>
         
         <p>
            
            </p><ol start="1" class="">
               
               
               <li>For species diversity, the estimator of species 
richness derived from the Good‐Turing frequency formula is identical to 
the Chao1 estimator (Chao <span><a href="#ecy2000-bib-0005" class="bibLink tab-link" data-tab="pane-pcw-references">1984</a></span>, <span><a href="#ecy2000-bib-0006" class="bibLink tab-link" data-tab="pane-pcw-references">1987</a></span>) in Eq.&nbsp;(<a class="equationLink scrollableLink" href="#ecy2000-disp-0008" title="Link to equation">3a</a>). The estimator of shared species between two assemblages in Eq.&nbsp;(<a class="equationLink scrollableLink" href="#ecy2000-disp-0021" title="Link to equation">7d</a>) is identical to the Chao1‐shared species estimator proposed in Pan et&nbsp;al. (<span><a href="#ecy2000-bib-0042" class="bibLink tab-link" data-tab="pane-pcw-references">2009</a></span>).
               </li>
               
               
               <li>For phylogenetic diversity, the resulting estimator (Eq. 8d) of Faith's <i>PD</i> is identical to the Chao1‐<i>PD</i> estimator proposed recently by Chao et&nbsp;al. (<span><a href="#ecy2000-bib-0014" class="bibLink tab-link" data-tab="pane-pcw-references">2015a</a></span>), but the estimator of the shared Faith's <i>PD</i> (Eq. 10d) is new.
               </li>
               
               
               <li>For functional trait diversity, the estimator of <i>FAD</i> in a single assemblage (Eq. 12d) and shared <i>FAD</i> between two assemblages (Eq. 14b) are both new; see Appendix&nbsp;<a class="suppLink scrollableLink" href="#support-information-section">S3</a>: Tables S1–S3 for a summary.
               </li>
               
            </ol>
            
         <p></p>
         
         <p>The R code “Good‐Turing” for computing all the estimators discussed in this paper is available in Github (<a href="https://github.com/AnneChao" class="linkBehavior">https://github.com/AnneChao</a>)
 along with a description of the running procedures. As an alternative, 
readers without a background in R, can utilize the online software 
“GoodTuring”, made available from <a href="https://chao.shinyapps.io/GoodTuring/" class="linkBehavior">https://chao.shinyapps.io/GoodTuring/</a> to facilitate all computations.
         </p>
         
         <p>We have proved that each of the derived estimators is 
theoretically a lower bound of the corresponding diversity. 
Good‐Turing's perspectives reveal the sufficient conditions under which 
the resulting estimator is nearly unbiased. For example, a simple 
sufficient condition for the Chao1 species richness estimator being 
nearly unbiased is that <i>rare</i> species (specifically, undetected 
species and singletons in sample) have approximately homogenous 
abundances. Similar conditions for other estimators are also clearly 
specified in each subsection. See the next paragraph for more relaxed 
conditions.
         </p>
         
         <p>Although, in our derivations, we follow the Good‐Turing 
original model by assuming that the detection probability of each 
species is simply its relative abundance, all our derivations can be 
directly extended to a general model as discussed in Chao and Chiu (<span><a href="#ecy2000-bib-0009" class="bibLink tab-link" data-tab="pane-pcw-references">2016</a></span>).
 The general model assumes that detection probability is proportional to
 the product of abundance and individual detectability, which may vary 
among species. Based on samples of individuals, individual detectability
 for mobile organisms is determined by many possible factors such as 
individual movement patterns, color, size, habitat, life cycle and 
vocalizations; for assemblages of sessile organisms, such&nbsp;as trees,
 that are surveyed from selected sampling units (e.g., transects, plots 
or quadrats), individual detectability may depend on the area and 
topography of selected sampling units, species spatial/temporal 
aggregation or clustering, life history stage, as well as other factors.
 Consequently, our estimators are actually valid in more relaxed 
conditions. For example, the Chao1 species richness estimator is nearly 
unbiased when <i>hard‐to‐detect</i> species (specifically, undetected 
species and species detected by one individual) have approximately 
homogenous detection probabilities. Similar relaxed conditions can be 
formulated for other estimators. Under the special case that all 
individuals have the same detectability, the detection probability of 
each species reduces to its relative abundance. In other applications in
 which species detection probability may vary with time, our method can 
also be applied when the probability of detecting a species is modeled 
as the average detection probability over the sampling‐time interval.
         </p>
         
         <p>When rare or hard‐to‐detect species are highly heterogeneous
 in detection probabilities, such as in microbial assemblages or DNA 
sequencing data, all estimators derived in this paper provide 
non‐parametric lower bounds which are valid for species, phylogenetic 
and functional diversities. In such assemblages, sample data do not 
provide sufficient information to accurately estimate asymptotic 
diversities due to heterogeneity of rare species; no statistical methods
 can produce reliable estimates unless strong assumptions are made. In 
such cases, from our perspective, an accurate lower bound is more 
practically useful than an imprecise point estimate. In Appendix&nbsp;<a class="suppLink scrollableLink" href="#support-information-section">S4</a>,
 some representative simulation results are reported to validate the 
estimators derived from Good‐Turing theory and also to demonstrate that 
our estimators provide useful and informative lower bounds when accurate
 point estimators are not attainable.
         </p>
         
         <p>For cases that fail to meet or may not be assured to meet 
the criteria for nearly‐unbiased point estimation, how can we make fair 
comparison of diversities across studies? We suggest using a 
non‐asymptotic approach via sample‐size‐ and coverage‐based rarefaction 
and extrapolation on the basis of standardized sample size or sample 
completeness (as measured by sample coverage). This non‐asymptotic 
approach facilitates fair comparison of diversities for equally‐large or
 equally‐complete samples across multiple assemblages. For species 
richness, sample‐size‐based rarefaction and extrapolation methods were 
developed by Colwell et&nbsp;al. (<span><a href="#ecy2000-bib-0021" class="bibLink tab-link" data-tab="pane-pcw-references">2012</a></span>), and corresponding coverage‐based methods were proposed by Chao and Jost (<span><a href="#ecy2000-bib-0011" class="bibLink tab-link" data-tab="pane-pcw-references">2012</a></span>). For Faith's <i>PD</i>, a similar non‐asymptotic method was recently presented by Chao et&nbsp;al. (<span><a href="#ecy2000-bib-0014" class="bibLink tab-link" data-tab="pane-pcw-references">2015a</a></span>) and Hsieh and Chao (<span><a href="#ecy2000-bib-0034" class="bibLink tab-link" data-tab="pane-pcw-references">2017</a></span>). Rarefaction and extrapolation methods for functional diversity are still under development by the authors.
         </p>
         
         <p>For the analysis of multiple‐assemblage phylogenetic structures, Webb et&nbsp;al. (<span><a href="#ecy2000-bib-0052" class="bibLink tab-link" data-tab="pane-pcw-references">2002</a></span>) and Webb et&nbsp;al. (<span><a href="#ecy2000-bib-0053" class="bibLink tab-link" data-tab="pane-pcw-references">2008</a></span>) proposed using a null model randomization test based on <i>PD</i> (or other metrics) to assess whether a specific assemblage has a higher <i>PD</i> (more even or uniform) or a lower <i>PD</i> (phylogenetic aggregation or clustering) than expected from an assembly randomly sampled from the <i>observed</i> species pool. That is, the observed species pool is the composite list of species from all assemblages; <i>PD</i>
 is calculated from the phylogenetic tree spanned by these detected 
species. In Webb et&nbsp;al.'s approach and in most null model or 
randomization tests (Gotelli et&nbsp;al. <span><a href="#ecy2000-bib-0032" class="bibLink tab-link" data-tab="pane-pcw-references">2010</a></span>
 is an exception), it is assumed that (1) sampling is complete for all 
assemblages, and (2) the aggregation of observed species for all 
assemblages constitutes the “complete” species pool. However, for the 
Brazilian rain forest data analyzed in our analysis, if we treat each 
fragment as an assemblage, then neither of these assumptions is 
satisfied. Within each fragment, some species were undetected by the 
transect data, and some species remained undetected even in the pooled 
transect data for the entire study area represented by the 11 fragments.
 In the following, we use the Brazilian data to illustrate our suggested
 modifications.
         </p>
         
         <p>
            
            </p><ol start="1" class="">
               
               
               <li>Based on the pooled transect data from all 11 
fragments, the Chao1 richness estimator yields the estimated number of 
species for the complete pool, which comprises all species recorded in 
the fragment transects plus undetected species in the study area 
represented by these fragments. Following the approach of Chao 
et&nbsp;al. (<span><a href="#ecy2000-bib-0015" class="bibLink tab-link" data-tab="pane-pcw-references">2015b</a></span>),
 we can construct a complete species‐rank abundance distribution (RAD) 
by separately adjusting the sample relative abundances for the set of 
species detected in the pooled, observed data and estimating the 
relative abundances for the set of species undetected in the pooled 
data, but inferred to be present in the area represented by all 
fragments. The combined RAD then fully characterizes the taxonomic 
assemblage structure. Thus, not only species richness but also species 
relative abundances can be estimated for all species in the estimated 
complete pool. For this analysis, it is not necessary to know the 
identities of the undetected species, but we do need to estimate their 
number and relative abundances in the complete pool.
               </li>
               
               
               <li>Next, a random assemblage of the same size as each 
specific assemblage is sampled from the complete pool. If the sampled 
assemblage includes species that belong to the group of undetected 
species in the pooled data, then we treat those species as unresolved in
 their phylogenetic placement and locate those undetected species on the
 observed tree in some random manner (e.g., Rangel et&nbsp;al. <span><a href="#ecy2000-bib-0044" class="bibLink tab-link" data-tab="pane-pcw-references">2015</a></span>). How to optimally locate undetected species is still under investigation. The Chao1‐<i>PD</i>
 estimate (Eq. 8d) can thus be calculated not only for the data of the 
specific assemblage but also for each random assemblage sampled from the
 complete pool.
               </li>
               
               
               <li>After many assemblages of the same size have been 
randomly selected from the complete species pool as in Step (2), the 
mean and standard error of the resulting <i>PD</i> values can be computed to obtain the standardized effect size (Gotelli and McCabe <span><a href="#ecy2000-bib-0031" class="bibLink tab-link" data-tab="pane-pcw-references">2002</a></span>), adjusted for the under‐sampling biases for both the specific assemblage and the pooled assemblage.
               </li>
               
            </ol>
            
         <p></p>
         
         <p>A similar procedure can be applied to <i>FAD</i> and other 
metrics, although this procedure should first be benchmarked with 
simulated and empirical data sets to assess its performance and to see 
if this procedure is an effective remedy for the under‐sampling problem 
that is pervasive in the analysis of community structure based on 
standardized biodiversity sampling of multiple samples.
         </p>
         
         <p>This paper is restricted to the estimation of diversity in 
one assemblage and the estimation of shared diversity between two 
assemblages. When there are more than two assemblages, our approach can 
be further extended to such cases. All the derivations are nearly 
parallel. For example, we can directly obtain an estimator of the 
species shared by multiple assemblages based on sampling data from each 
assemblage, and the resulting estimator is identical to the one proposed
 by Pan et&nbsp;al. (<span><a href="#ecy2000-bib-0042" class="bibLink tab-link" data-tab="pane-pcw-references">2009</a></span>).
         </p>
         
         <p>Our derivation in this paper is limited to individual‐based 
abundance data for individuals sampled randomly from assemblages. In 
many ecological field studies, the sampling unit is not an individual, 
but a trap, net, quadrat, plot, or timed survey. For such studies, the 
sampling units, not the individuals, are sampled randomly and 
independently. In these cases, estimation is usually based on a set of 
sampling units in which only the incidence (detection or non‐detection) 
of each species is recorded. This type of data is referred to as 
(multiple) incidence data. The sampling model and estimation were 
developed in Colwell et&nbsp;al. (<span><a href="#ecy2000-bib-0021" class="bibLink tab-link" data-tab="pane-pcw-references">2012</a></span>) and Chao et&nbsp;al. (<span><a href="#ecy2000-bib-0013" class="bibLink tab-link" data-tab="pane-pcw-references">2014b</a></span>). Chao and Colwell (<span><a href="#ecy2000-bib-0010" class="bibLink tab-link" data-tab="pane-pcw-references">2017</a></span>)
 recently extended the Good‐Turing frequency formula and its 
generalizations to incidence data. All estimators can be computed from 
the&nbsp;online software “GoodTuring”. We expect that Good‐Turing's 
theory will find wide applications in biodiversity studies; see Chao 
et&nbsp;al. (<span><a href="#ecy2000-bib-0016" class="bibLink tab-link" data-tab="pane-pcw-references">2017</a></span>) for a recent application.
         </p>
         
      </div>
      
      <div class="article-section__content">
         
         <h2 class="article-section__title section__title section1" id="ecy2000-sec-0016-title"> Acknowledgments</h2>
         
         <p>The authors thank a Subject Matter Editor (Tom Miller), 
Joaquín Hortal, and an anonymous reviewer for very thoughtful and 
helpful comments and suggestions. This work was supported by the Taiwan 
Ministry of Science and Technology under Contracts 104‐2628‐M007‐003 and
 105‐2628‐M007‐001 (for AC) and 104‐2118‐M‐002‐008‐MY3 (for CHC). RKC 
and RLC were supported by CAPES Ciência sem Fronteiras (Brazil). LFSM 
was supported by CAPES/PNPD. NJG was supported by U. S. NSF DEB 1257625,
 NSF DEB 1144055, and NSF DEB 1136644. The fieldwork survey was 
supported by Reserva Natural Vale, Fibria Celulose S.A., Marcos Daniel 
Institute, Pro‐Tapir project and Reserva Biológica de Sooretama.</p>
         
         <ol></ol>
      </div>
      
      <section class="article-section article-section__supporting">
         <div class="accordion article-accordion"><a class="accordion__control" aria-controls="d16875177e396" aria-expanded="false">
               <h2 class="section__title" id="support-information-section">
                            Supporting Information
                        </h2><i aria-hidden="true" class="icon-section_arrow_d"></i></a><div class="accordion__content" style="display: none;" id="d16875177e396" aria-hidden="false">
               <div class="support-info__table-wrapper article-table-content-wrapper">
                  
                  
                  
                  
                  
                  <table class="support-info__table table article-section__table">
                     <thead>
                        <tr><th id="article-filename" scope="col">Filename</th>
                        <th id="article-description" scope="col">Description</th>
                     </tr></thead>
                     <tbody>
                        <tr id="ecy2000-sup-0001">
                           <td headers="article-filename"><a href="https://esajournals.onlinelibrary.wiley.com/action/downloadSupplement?doi=10.1002%2Fecy.2000&amp;file=ecy2000-sup-0001-AppendixS1.pdf">ecy2000-sup-0001-AppendixS1.pdf</a>PDF document, 364.6 KB
                           </td>
                           <td headers="article-description">&nbsp;</td>
                        </tr>
                        <tr id="ecy2000-sup-0002">
                           <td headers="article-filename"><a href="https://esajournals.onlinelibrary.wiley.com/action/downloadSupplement?doi=10.1002%2Fecy.2000&amp;file=ecy2000-sup-0002-AppendixS2.pdf">ecy2000-sup-0002-AppendixS2.pdf</a>PDF document, 760.2 KB
                           </td>
                           <td headers="article-description">&nbsp;</td>
                        </tr>
                        <tr id="ecy2000-sup-0003">
                           <td headers="article-filename"><a href="https://esajournals.onlinelibrary.wiley.com/action/downloadSupplement?doi=10.1002%2Fecy.2000&amp;file=ecy2000-sup-0003-AppendixS3.pdf">ecy2000-sup-0003-AppendixS3.pdf</a>PDF document, 442.2 KB
                           </td>
                           <td headers="article-description">&nbsp;</td>
                        </tr>
                        <tr id="ecy2000-sup-0004">
                           <td headers="article-filename"><a href="https://esajournals.onlinelibrary.wiley.com/action/downloadSupplement?doi=10.1002%2Fecy.2000&amp;file=ecy2000-sup-0004-AppendixS4.pdf">ecy2000-sup-0004-AppendixS4.pdf</a>PDF document, 587.3 KB
                           </td>
                           <td headers="article-description">&nbsp;</td>
                        </tr>
                     </tbody>
                  </table>
                  <p class="disclaimer_text">Please note: The publisher 
is not responsible for the content or functionality of any supporting 
information supplied by the authors. Any queries (other than missing 
content) should be directed to the corresponding author for the article.</p>
               </div>
            </div>
         </div>
      </section>
      <section class="article-section article-section__references" id="references-section">
         <div class="accordion article-accordion"><a href="#" class="accordion__control" aria-controls="idecy2000-bibl-0001" aria-expanded="false" data-references="/action/ajaxShowEnhancedAbstract?widgetId=5cf4c79f-0ae9-4dc5-96ce-77f62de7ada9&amp;ajax=true&amp;doi=10.1002%2Fecy.2000&amp;pbContext=%3Bpage%3Astring%3AArticle%2FChapter+View%3Barticle%3Aarticle%3Adoi%5C%3A10.1002%2Fecy.2000%3Bwebsite%3Awebsite%3Aesajournals%3Bjournal%3Ajournal%3A19399170%3Bissue%3Aissue%3Adoi%5C%3A10.1002%2Fecy.2017.98.issue-11%3Bwgroup%3Astring%3APublication+Websites%3BpageGroup%3Astring%3APublication+Pages%3BsubPage%3Astring%3AFull+Text&amp;onlyLog=true">
               <h2 class="section__title" id="references-section-1">Literature Cited</h2><i aria-hidden="true" class="icon-section_arrow_d"></i></a><div class="accordion__content" style="display: none;" id="ecy2000-bibl-0001">
               <ul class="rlist separator">
                  
                  
                  <li data-bib-id="ecy2000-bib-0001">Bryant, J. A., C. Lamanna, H. Morlon, A. J. Kerkhoff, B. J. Enquist, and J. L. Green. <span class="pubYear">2008</span>. <span class="articleTitle">Microbes on mountainsides: contrasting elevational patterns of bacterial and plant diversity</span>. <i>Proceedings of the National Academy of Sciences of the United States of America</i> <span class="vol">105</span>:11505–11511.
                     <div class="extra-links" id="e_1_2_9_2_1"><span>
                           <a href="https://esajournals.onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=16&amp;doi=10.1002%2Fecy.2000&amp;key=10.1073%2Fpnas.0801920105" onclick="newWindow(this.href);return false" target="_blank">Crossref</a></span><span>
                           <a href="https://esajournals.onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=32&amp;doi=10.1002%2Fecy.2000&amp;key=1%3ACAS%3A528%3ADC%252BD1cXhtVSjtr%252FF" onclick="newWindow(this.href);return false" target="_blank">CAS</a></span><span>
                           <a href="https://esajournals.onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=8&amp;doi=10.1002%2Fecy.2000&amp;key=18695215" onclick="newWindow(this.href);return false" target="_blank">PubMed</a></span><span>
                           <a href="https://esajournals.onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1002%2Fecy.2000&amp;key=000258561200008" onclick="newWindow(this.href);return false" target="_blank">Web of Science®</a></span><span><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=105&amp;publication_year=2008&amp;pages=11505-11511&amp;journal=Proceedings+of+the+National+Academy+of+Sciences+of+the+United+States+of+America&amp;author=J.+A.+Bryant&amp;author=C.+Lamanna&amp;author=H.+Morlon&amp;author=A.+J.+Kerkhoff&amp;author=B.+J.+Enquist&amp;author=J.+L.+Green&amp;title=Microbes+on+mountainsides%3A+contrasting+elevational+patterns+of+bacterial+and+plant+diversity" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="ecy2000-bib-0002">Cadotte, M. W., J. Cavender‐Bares, D. Tilman, and T. H. Oakley. <span class="pubYear">2009</span>. <span class="articleTitle">Using phylogenetic, functional and trait diversity to understand patterns of plant community productivity</span>. <i>PLoS ONE</i> <span class="vol">4</span>:e5695.
                     <div class="extra-links" id="e_1_2_9_3_1"><span>
                           <a href="https://esajournals.onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=16&amp;doi=10.1002%2Fecy.2000&amp;key=10.1371%2Fjournal.pone.0005695" onclick="newWindow(this.href);return false" target="_blank">Crossref</a></span><span>
                           <a href="https://esajournals.onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=32&amp;doi=10.1002%2Fecy.2000&amp;key=1%3ACAS%3A528%3ADC%252BD1MXhtlOhtbfE" onclick="newWindow(this.href);return false" target="_blank">CAS</a></span><span>
                           <a href="https://esajournals.onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=8&amp;doi=10.1002%2Fecy.2000&amp;key=19479086" onclick="newWindow(this.href);return false" target="_blank">PubMed</a></span><span>
                           <a href="https://esajournals.onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1002%2Fecy.2000&amp;key=000266415000008" onclick="newWindow(this.href);return false" target="_blank">Web of Science®</a></span><span><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=4&amp;publication_year=2009&amp;pages=e5695&amp;journal=PLoS+ONE&amp;author=M.+W.+Cadotte&amp;author=J.+Cavender%E2%80%90Bares&amp;author=D.+Tilman&amp;author=T.+H.+Oakley&amp;title=Using+phylogenetic%2C+functional+and+trait+diversity+to+understand+patterns+of+plant+community+productivity" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="ecy2000-bib-0003">Cardoso, P., F. Rigal, P. A. Borges, and J. C. Carvalho. <span class="pubYear">2014</span>. <span class="articleTitle">A new frontier in biodiversity inventory: a proposal for estimators of phylogenetic and functional diversity</span>. <i>Methods in Ecology and Evolution</i> <span class="vol">5</span>:452–461.
                     <div class="extra-links" id="e_1_2_9_4_1"><span>
                           <a href="https://besjournals.onlinelibrary.wiley.com/doi/10.1111/2041-210X.12173" onclick="newWindow(this.href);return false" target="_blank">Wiley Online Library</a></span><span>
                           <a href="https://esajournals.onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1002%2Fecy.2000&amp;key=000336500500006" onclick="newWindow(this.href);return false" target="_blank">Web of Science®</a></span><span><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=5&amp;publication_year=2014&amp;pages=452-461&amp;journal=Methods+in+Ecology+and+Evolution&amp;author=P.+Cardoso&amp;author=F.+Rigal&amp;author=P.+A.+Borges&amp;author=J.+C.+Carvalho&amp;title=A+new+frontier+in+biodiversity+inventory%3A+a+proposal+for+estimators+of+phylogenetic+and+functional+diversity" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="ecy2000-bib-0004">Cavender‐Bares, J., D. D. Ackerly, and K. H. Kozak. <span class="pubYear">2012</span>. <span class="articleTitle">Integrating ecology and phylogenetics: the footprint of history in modern‐day communities</span>. <i>Ecology</i> <span class="vol">93</span>:S1–S3.
                     <div class="extra-links" id="e_1_2_9_5_1"><span>
                           <a href="https://esajournals.onlinelibrary.wiley.com/doi/10.1890/12-0092.1" onclick="newWindow(this.href);return false" target="_blank">Wiley Online Library</a></span><span>
                           <a href="https://esajournals.onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1002%2Fecy.2000&amp;key=000307302400001" onclick="newWindow(this.href);return false" target="_blank">Web of Science®</a></span><span><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=93&amp;publication_year=2012&amp;pages=S1-S3&amp;journal=Ecology&amp;author=J.+Cavender%E2%80%90Bares&amp;author=D.+D.+Ackerly&amp;author=K.+H.+Kozak&amp;title=Integrating+ecology+and+phylogenetics%3A+the+footprint+of+history+in+modern%E2%80%90day+communities" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="ecy2000-bib-0005">Chao, A. <span class="pubYear">1984</span>. <span class="articleTitle">Nonparametric estimation of the number of classes in a population</span>. <i>Scandinavian Journal of Statistics</i> <span class="vol">11</span>:265–270.
                     <div class="extra-links" id="e_1_2_9_6_1"><span>
                           <a href="https://onlinelibrary.wiley.com/doi/10.1111/j.1462-2920.2006.01049.x" onclick="newWindow(this.href);return false" target="_blank">Wiley Online Library</a></span><span>
                           <a href="https://esajournals.onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=32&amp;doi=10.1002%2Fecy.2000&amp;key=1%3ACAS%3A528%3ADC%252BD2sXkslequr8%253D" onclick="newWindow(this.href);return false" target="_blank">CAS</a></span><span>
                           <a href="https://esajournals.onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=8&amp;doi=10.1002%2Fecy.2000&amp;key=16913925" onclick="newWindow(this.href);return false" target="_blank">PubMed</a></span><span>
                           <a href="https://esajournals.onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1002%2Fecy.2000&amp;key=A1984AJZ6300006" onclick="newWindow(this.href);return false" target="_blank">Web of Science®</a></span><span><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=11&amp;publication_year=1984&amp;pages=265-270&amp;journal=Scandinavian+Journal+of+Statistics&amp;author=A.+Chao&amp;title=Nonparametric+estimation+of+the+number+of+classes+in+a+population" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="ecy2000-bib-0006">Chao, A. <span class="pubYear">1987</span>. <span class="articleTitle">Estimating the population size for capture‐recapture data with unequal catchability</span>. <i>Biometrics</i> <span class="vol">43</span>:783–791.
                     <div class="extra-links" id="e_1_2_9_7_1"><span>
                           <a href="https://esajournals.onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=16&amp;doi=10.1002%2Fecy.2000&amp;key=10.2307%2F2531532" onclick="newWindow(this.href);return false" target="_blank">Crossref</a></span><span>
                           <a href="https://esajournals.onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=32&amp;doi=10.1002%2Fecy.2000&amp;key=1%3ASTN%3A280%3ADyaL1c7gsl2qsw%253D%253D" onclick="newWindow(this.href);return false" target="_blank">CAS</a></span><span>
                           <a href="https://esajournals.onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=8&amp;doi=10.1002%2Fecy.2000&amp;key=3427163" onclick="newWindow(this.href);return false" target="_blank">PubMed</a></span><span>
                           <a href="https://esajournals.onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1002%2Fecy.2000&amp;key=A1987L357600004" onclick="newWindow(this.href);return false" target="_blank">Web of Science®</a></span><span><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=43&amp;publication_year=1987&amp;pages=783-791&amp;journal=Biometrics&amp;author=A.+Chao&amp;title=Estimating+the+population+size+for+capture%E2%80%90recapture+data+with+unequal+catchability" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="ecy2000-bib-0007">Chao, A. <span class="pubYear">2005</span>. <span class="chapterTitle">Species estimation and applications</span>. Pages 7907–7916 <i>in</i> N. Balakrishnan, C. Read, and B. Vidakovic, editors. <span class="bookTitle">Encyclopedia of statistical sciences</span>. Wiley, New York.
                     <div class="extra-links" id="e_1_2_9_8_1"><span><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;publication_year=2005&amp;pages=7907-7916&amp;author=A.+Chao&amp;title=Encyclopedia+of+statistical+sciences" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="ecy2000-bib-0008">Chao, A., and C.‐H. Chiu. <span class="pubYear">2012</span>. <span class="chapterTitle">Estimation of species richness and shared species richness</span>. Pages 76–111 <i>in</i> N. Balakrishnan, editor. <span class="bookTitle">Methods and applications of statistics in the atmospheric and earth sciences</span>. Wiley, New York, USA.
                     <div class="extra-links" id="e_1_2_9_9_1"><span><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;publication_year=2012&amp;pages=76-111&amp;author=A.+Chao&amp;author=C.%E2%80%90H.+Chiu&amp;title=Methods+and+applications+of+statistics+in+the+atmospheric+and+earth+sciences" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="ecy2000-bib-0009">Chao, A., and C.‐H. Chiu. <span class="pubYear">2016</span>. Species richness: estimation and comparison. Wiley StatsRef: Statistics Reference Online. 1–26.
                     <div class="extra-links" id="e_1_2_9_10_1"><span><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;publication_year=2016&amp;author=A.+Chao&amp;author=C.%E2%80%90H.+Chiu" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="ecy2000-bib-0010">Chao, A., and R. K. Colwell. <span class="pubYear">2017</span>. <span class="articleTitle">Thirty
 years of progeny from Chao's inequality: estimating and comparing 
richness with incidence data and incomplete sampling (invited article)</span>. <i>Statistics and Operation Research Transactions</i> <span class="vol">41</span>:3–54.
                     <div class="extra-links" id="e_1_2_9_11_1"><span>
                           <a href="https://esajournals.onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1002%2Fecy.2000&amp;key=000407307600001" onclick="newWindow(this.href);return false" target="_blank">Web of Science®</a></span><span><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=41&amp;publication_year=2017&amp;pages=3-54&amp;journal=Statistics+and+Operation+Research+Transactions&amp;author=A.+Chao&amp;author=R.+K.+Colwell&amp;title=Thirty+years+of+progeny+from+Chao%27s+inequality%3A+estimating+and+comparing+richness+with+incidence+data+and+incomplete+sampling+%28invited+article%29" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="ecy2000-bib-0011">Chao, A., and L. Jost. <span class="pubYear">2012</span>. <span class="articleTitle">Coverage‐based rarefaction and extrapolation: standardizing samples by completeness rather than size</span>. <i>Ecology</i> <span class="vol">93</span>:2533–2547.
                     <div class="extra-links" id="e_1_2_9_12_1"><span>
                           <a href="https://esajournals.onlinelibrary.wiley.com/doi/10.1890/11-1952.1" onclick="newWindow(this.href);return false" target="_blank">Wiley Online Library</a></span><span>
                           <a href="https://esajournals.onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=32&amp;doi=10.1002%2Fecy.2000&amp;key=1%3ASTN%3A280%3ADC%252BC3svgvVKrtw%253D%253D" onclick="newWindow(this.href);return false" target="_blank">CAS</a></span><span>
                           <a href="https://esajournals.onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=8&amp;doi=10.1002%2Fecy.2000&amp;key=23431585" onclick="newWindow(this.href);return false" target="_blank">PubMed</a></span><span>
                           <a href="https://esajournals.onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1002%2Fecy.2000&amp;key=000312892500005" onclick="newWindow(this.href);return false" target="_blank">Web of Science®</a></span><span><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=93&amp;publication_year=2012&amp;pages=2533-2547&amp;journal=Ecology&amp;author=A.+Chao&amp;author=L.+Jost&amp;title=Coverage%E2%80%90based+rarefaction+and+extrapolation%3A+standardizing+samples+by+completeness+rather+than+size" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="ecy2000-bib-0012">Chao, A., C.‐H. Chiu, and L. Jost. <span class="pubYear">2014a</span>. <span class="articleTitle">Unifying
 species diversity, phylogenetic diversity, functional diversity, and 
related similarity and differentiation measures through Hill numbers</span>. <i>Annual Review of Ecology, Evolution, and Systematics</i> <span class="vol">45</span>:297–324.
                     <div class="extra-links" id="e_1_2_9_13_1"><span>
                           <a href="https://esajournals.onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=16&amp;doi=10.1002%2Fecy.2000&amp;key=10.1146%2Fannurev-ecolsys-120213-091540" onclick="newWindow(this.href);return false" target="_blank">Crossref</a></span><span>
                           <a href="https://esajournals.onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1002%2Fecy.2000&amp;key=000348461700014" onclick="newWindow(this.href);return false" target="_blank">Web of Science®</a></span><span><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=45&amp;publication_year=2014a&amp;pages=297-324&amp;journal=Annual+Review+of+Ecology%2C+Evolution%2C+and+Systematics&amp;author=A.+Chao&amp;author=C.%E2%80%90H.+Chiu&amp;author=L.+Jost&amp;title=Unifying+species+diversity%2C+phylogenetic+diversity%2C+functional+diversity%2C+and+related+similarity+and+differentiation+measures+through+Hill+numbers" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="ecy2000-bib-0013">Chao, A., N. J. Gotelli, T. C. Hsieh, E. L. Sander, K. Ma, R. K. Colwell, and A. M. Ellison. <span class="pubYear">2014b</span>. <span class="articleTitle">Rarefaction and extrapolation with Hill numbers: a framework for sampling and estimation in species diversity studies</span>. <i>Ecological Monographs</i> <span class="vol">84</span>:45–67.
                     <div class="extra-links" id="e_1_2_9_14_1"><span>
                           <a href="https://esajournals.onlinelibrary.wiley.com/doi/10.1890/13-0133.1" onclick="newWindow(this.href);return false" target="_blank">Wiley Online Library</a></span><span>
                           <a href="https://esajournals.onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1002%2Fecy.2000&amp;key=000331215700004" onclick="newWindow(this.href);return false" target="_blank">Web of Science®</a></span><span><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=84&amp;publication_year=2014b&amp;pages=45-67&amp;journal=Ecological+Monographs&amp;author=A.+Chao&amp;author=N.+J.+Gotelli&amp;author=T.+C.+Hsieh&amp;author=E.+L.+Sander&amp;author=K.+Ma&amp;author=R.+K.+Colwell&amp;author=A.+M.+Ellison&amp;title=Rarefaction+and+extrapolation+with+Hill+numbers%3A+a+framework+for+sampling+and+estimation+in+species+diversity+studies" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="ecy2000-bib-0014">Chao, A., C.‐H. Chiu, T. C. Hsieh, T. Davis, D. A. Nipperess, and D. P. Faith. <span class="pubYear">2015a</span>. <span class="articleTitle">Rarefaction and extrapolation of phylogenetic diversity</span>. <i>Methods in Ecology and Evolution</i> <span class="vol">6</span>:380–388.
                     <div class="extra-links" id="e_1_2_9_15_1"><span>
                           <a href="https://besjournals.onlinelibrary.wiley.com/doi/10.1111/2041-210X.12247" onclick="newWindow(this.href);return false" target="_blank">Wiley Online Library</a></span><span>
                           <a href="https://esajournals.onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1002%2Fecy.2000&amp;key=000352794100003" onclick="newWindow(this.href);return false" target="_blank">Web of Science®</a></span><span><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=6&amp;publication_year=2015a&amp;pages=380-388&amp;journal=Methods+in+Ecology+and+Evolution&amp;author=A.+Chao&amp;author=C.%E2%80%90H.+Chiu&amp;author=T.+C.+Hsieh&amp;author=T.+Davis&amp;author=D.+A.+Nipperess&amp;author=D.+P.+Faith&amp;title=Rarefaction+and+extrapolation+of+phylogenetic+diversity" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="ecy2000-bib-0015">Chao, A., T. C. Hsieh, R. L. Chazdon, R. K. Colwell, and N. J. Gotelli. <span class="pubYear">2015b</span>. <span class="articleTitle">Unveiling the species‐rank abundance distribution by generalizing the Good‐Turing sample coverage theory</span>. <i>Ecology</i> <span class="vol">96</span>:1189–1201.
                     <div class="extra-links" id="e_1_2_9_16_1"><span>
                           <a href="https://esajournals.onlinelibrary.wiley.com/doi/10.1890/14-0550.1" onclick="newWindow(this.href);return false" target="_blank">Wiley Online Library</a></span><span>
                           <a href="https://esajournals.onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=32&amp;doi=10.1002%2Fecy.2000&amp;key=1%3ASTN%3A280%3ADC%252BC28%252Fpt1amsQ%253D%253D" onclick="newWindow(this.href);return false" target="_blank">CAS</a></span><span>
                           <a href="https://esajournals.onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=8&amp;doi=10.1002%2Fecy.2000&amp;key=26236834" onclick="newWindow(this.href);return false" target="_blank">PubMed</a></span><span>
                           <a href="https://esajournals.onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1002%2Fecy.2000&amp;key=000354119300005" onclick="newWindow(this.href);return false" target="_blank">Web of Science®</a></span><span><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=96&amp;publication_year=2015b&amp;pages=1189-1201&amp;journal=Ecology&amp;author=A.+Chao&amp;author=T.+C.+Hsieh&amp;author=R.+L.+Chazdon&amp;author=R.+K.+Colwell&amp;author=N.+J.+Gotelli&amp;title=Unveiling+the+species%E2%80%90rank+abundance+distribution+by+generalizing+the+Good%E2%80%90Turing+sample+coverage+theory" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="ecy2000-bib-0016">Chao, A., R. K. Colwell, C.‐H. Chiu, and D. Townsend. <span class="pubYear">2017</span>. <span class="articleTitle">Seen
 once or more than once: using Good‐Turing theory to estimate species 
richness using only unique observations and&nbsp;a species list</span>. <i>Methods in Ecology and Evolution</i> DOI: <a target="_blank" class="accessionId" href="https://doi.org/10.1111/2041-210X.12768" title="Link to external resource: 10.1111/2041‐210X.12768">10.1111/2041‐210X.12768</a>.
                     <div class="extra-links" id="e_1_2_9_17_1"><span>
                           <a href="https://besjournals.onlinelibrary.wiley.com/doi/10.1111/2041-210X.12768" onclick="newWindow(this.href);return false" target="_blank">Wiley Online Library</a></span><span>
                           <a href="https://esajournals.onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=8&amp;doi=10.1002%2Fecy.2000&amp;key=19262458" onclick="newWindow(this.href);return false" target="_blank">PubMed</a></span><span>
                           <a href="https://esajournals.onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1002%2Fecy.2000&amp;key=000412858600006" onclick="newWindow(this.href);return false" target="_blank">Web of Science®</a></span><span><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;publication_year=2017&amp;journal=Methods+in+Ecology+and+Evolution&amp;author=A.+Chao&amp;author=R.+K.+Colwell&amp;author=C.%E2%80%90H.+Chiu&amp;author=D.+Townsend&amp;title=Seen+once+or+more+than+once%3A+using+Good%E2%80%90Turing+theory+to+estimate+species+richness+using+only+unique+observations+and%C2%A0a+species+list" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="ecy2000-bib-0017">Chazdon, R. L., R. K. Colwell, J. S. Denslow, and M. R. Guariguata. <span class="pubYear">1998</span>. <span class="chapterTitle">Statistical
 methods for estimating species richness of woody regeneration in 
primary and secondary rain forests of Northeastern Costa Rica</span>. Pages 285–309 <i>in</i> F. Dallmeier, and J. A. Comiskey, editors. <span class="bookTitle">Forest biodiversity research, monitoring and modeling: conceptual background and old world case studies</span>. Parthenon Publishing, Paris.
                     <div class="extra-links" id="e_1_2_9_18_1"><span>
                           <a href="https://nph.onlinelibrary.wiley.com/doi/10.1111/j.1469-8137.2005.01376.x" onclick="newWindow(this.href);return false" target="_blank">Wiley Online Library</a></span><span>
                           <a href="https://esajournals.onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=32&amp;doi=10.1002%2Fecy.2000&amp;key=1%3ACAS%3A528%3ADC%252BD2MXlsFSltLc%253D" onclick="newWindow(this.href);return false" target="_blank">CAS</a></span><span>
                           <a href="https://esajournals.onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=8&amp;doi=10.1002%2Fecy.2000&amp;key=15869663" onclick="newWindow(this.href);return false" target="_blank">PubMed</a></span><span>
                           <a href="https://esajournals.onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1002%2Fecy.2000&amp;key=000074779600016" onclick="newWindow(this.href);return false" target="_blank">Web of Science®</a></span><span><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;publication_year=1998&amp;pages=285-309&amp;author=R.+L.+Chazdon&amp;author=R.+K.+Colwell&amp;author=J.+S.+Denslow&amp;author=M.+R.+Guariguata&amp;title=Forest+biodiversity+research%2C+monitoring+and+modeling%3A+conceptual+background+and+old+world+case+studies" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="ecy2000-bib-0018">Chiu, C.‐H., L. Jost, and A. Chao. <span class="pubYear">2014a</span>. <span class="articleTitle">Phylogenetic beta diversity, similarity, and differentiation measures based on Hill numbers</span>. <i>Ecological Monographs</i> <span class="vol">84</span>:21–44.
                     <div class="extra-links" id="e_1_2_9_19_1"><span>
                           <a href="https://esajournals.onlinelibrary.wiley.com/doi/10.1890/12-0960.1" onclick="newWindow(this.href);return false" target="_blank">Wiley Online Library</a></span><span>
                           <a href="https://esajournals.onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1002%2Fecy.2000&amp;key=000331215700003" onclick="newWindow(this.href);return false" target="_blank">Web of Science®</a></span><span><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=84&amp;publication_year=2014a&amp;pages=21-44&amp;journal=Ecological+Monographs&amp;author=C.%E2%80%90H.+Chiu&amp;author=L.+Jost&amp;author=A.+Chao&amp;title=Phylogenetic+beta+diversity%2C+similarity%2C+and+differentiation+measures+based+on+Hill+numbers" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="ecy2000-bib-0019">Chiu, C.‐H., Y. T. Wang, B. A. Walther, and A. Chao. <span class="pubYear">2014b</span>. <span class="articleTitle">An improved nonparametric lower bound of species richness via a modified Good‐Turing frequency formula</span>. <i>Biometrics</i> <span class="vol">70</span>:671–682.
                     <div class="extra-links" id="e_1_2_9_20_1"><span>
                           <a href="https://onlinelibrary.wiley.com/doi/10.1111/biom.12200" onclick="newWindow(this.href);return false" target="_blank">Wiley Online Library</a></span><span>
                           <a href="https://esajournals.onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=8&amp;doi=10.1002%2Fecy.2000&amp;key=24945937" onclick="newWindow(this.href);return false" target="_blank">PubMed</a></span><span>
                           <a href="https://esajournals.onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1002%2Fecy.2000&amp;key=000342660700021" onclick="newWindow(this.href);return false" target="_blank">Web of Science®</a></span><span><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=70&amp;publication_year=2014b&amp;pages=671-682&amp;journal=Biometrics&amp;author=C.%E2%80%90H.+Chiu&amp;author=Y.+T.+Wang&amp;author=B.+A.+Walther&amp;author=A.+Chao&amp;title=An+improved+nonparametric+lower+bound+of+species+richness+via+a+modified+Good%E2%80%90Turing+frequency+formula" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="ecy2000-bib-0020">Colwell, R. K., and J. A. Coddington. <span class="pubYear">1994</span>. <span class="articleTitle">Estimating terrestrial biodiversity through extrapolation</span>. <i>Philosophical Transactions of the Royal Society of London B ‐ Biological Sciences</i> <span class="vol">345</span>:101–118.
                     <div class="extra-links" id="e_1_2_9_21_1"><span>
                           <a href="https://esajournals.onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=16&amp;doi=10.1002%2Fecy.2000&amp;key=10.1098%2Frstb.1994.0091" onclick="newWindow(this.href);return false" target="_blank">Crossref</a></span><span>
                           <a href="https://esajournals.onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=32&amp;doi=10.1002%2Fecy.2000&amp;key=1%3ASTN%3A280%3ADyaK2M%252Fmt1GltA%253D%253D" onclick="newWindow(this.href);return false" target="_blank">CAS</a></span><span>
                           <a href="https://esajournals.onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=8&amp;doi=10.1002%2Fecy.2000&amp;key=7972351" onclick="newWindow(this.href);return false" target="_blank">PubMed</a></span><span>
                           <a href="https://esajournals.onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1002%2Fecy.2000&amp;key=A1994PA85500011" onclick="newWindow(this.href);return false" target="_blank">Web of Science®</a></span><span><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=345&amp;publication_year=1994&amp;pages=101-118&amp;journal=Philosophical+Transactions+of+the+Royal+Society+of+London+B+%E2%80%90+Biological+Sciences&amp;author=R.+K.+Colwell&amp;author=J.+A.+Coddington&amp;title=Estimating+terrestrial+biodiversity+through+extrapolation" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="ecy2000-bib-0021">Colwell, R. K., A. Chao, N. J. Gotelli, S.‐Y. Lin, C. X. Mao, R.&nbsp;L. Chazdon, and J. T. Longino. <span class="pubYear">2012</span>. <span class="articleTitle">Models and estimators linking individual‐based and sample‐based rarefaction, extrapolation and comparison of assemblages</span>. <i>Journal of Plant Ecology</i> <span class="vol">5</span>:3–21.
                     <div class="extra-links" id="e_1_2_9_22_1"><span>
                           <a href="https://esajournals.onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=16&amp;doi=10.1002%2Fecy.2000&amp;key=10.1093%2Fjpe%2Frtr044" onclick="newWindow(this.href);return false" target="_blank">Crossref</a></span><span>
                           <a href="https://esajournals.onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1002%2Fecy.2000&amp;key=000299103600002" onclick="newWindow(this.href);return false" target="_blank">Web of Science®</a></span><span><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=5&amp;publication_year=2012&amp;pages=3-21&amp;journal=Journal+of+Plant+Ecology&amp;author=R.+K.+Colwell&amp;author=A.+Chao&amp;author=N.+J.+Gotelli&amp;author=S.%E2%80%90Y.+Lin&amp;author=C.+X.+Mao&amp;author=R.%C2%A0L.+Chazdon&amp;author=J.+T.+Longino&amp;title=Models+and+estimators+linking+individual%E2%80%90based+and+sample%E2%80%90based+rarefaction%2C+extrapolation+and+comparison+of+assemblages" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="ecy2000-bib-0022">Diaz, S., and M. Cabido. <span class="pubYear">2001</span>. <span class="articleTitle">Vive la différence: plant functional diversity matters to ecosystem processes</span>. <i>Trends in Ecology and Evolution</i> <span class="vol">16</span>:646–655.
                     <div class="extra-links" id="e_1_2_9_23_1"><span>
                           <a href="https://esajournals.onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=16&amp;doi=10.1002%2Fecy.2000&amp;key=10.1016%2FS0169-5347%2801%2902283-2" onclick="newWindow(this.href);return false" target="_blank">Crossref</a></span><span>
                           <a href="https://esajournals.onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1002%2Fecy.2000&amp;key=000171714800016" onclick="newWindow(this.href);return false" target="_blank">Web of Science®</a></span><span><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=16&amp;publication_year=2001&amp;pages=646-655&amp;journal=Trends+in+Ecology+and+Evolution&amp;author=S.+Diaz&amp;author=M.+Cabido&amp;title=Vive+la+diff%C3%A9rence%3A+plant+functional+diversity+matters+to+ecosystem+processes" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="ecy2000-bib-0023">Faith, D. P. <span class="pubYear">1992</span>. <span class="articleTitle">Conservation evaluation and phylogenetic diversity</span>. <i>Biological Conservation</i> <span class="vol">61</span>:1–10.
                     <div class="extra-links" id="e_1_2_9_24_1"><span>
                           <a href="https://esajournals.onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=16&amp;doi=10.1002%2Fecy.2000&amp;key=10.1016%2F0006-3207%2892%2991201-3" onclick="newWindow(this.href);return false" target="_blank">Crossref</a></span><span>
                           <a href="https://esajournals.onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=32&amp;doi=10.1002%2Fecy.2000&amp;key=1%3ACAS%3A528%3ADC%252BD28Xpt1Ciu74%253D" onclick="newWindow(this.href);return false" target="_blank">CAS</a></span><span>
                           <a href="https://esajournals.onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=8&amp;doi=10.1002%2Fecy.2000&amp;key=16893466" onclick="newWindow(this.href);return false" target="_blank">PubMed</a></span><span>
                           <a href="https://esajournals.onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1002%2Fecy.2000&amp;key=A1992HX12300001" onclick="newWindow(this.href);return false" target="_blank">Web of Science®</a></span><span><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=61&amp;publication_year=1992&amp;pages=1-10&amp;journal=Biological+Conservation&amp;author=D.+P.+Faith&amp;title=Conservation+evaluation+and+phylogenetic+diversity" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="ecy2000-bib-0024">Ferrier, S., G. Manion, J. Elith, and K. Richardson. <span class="pubYear">2007</span>. <span class="articleTitle">Using generalized dissimilarity modelling to analyse and predict patterns of beta diversity in regional biodiversity assessment</span>. <i>Diversity and Distributions</i> <span class="vol">13</span>:252–264.
                     <div class="extra-links" id="e_1_2_9_25_1"><span>
                           <a href="https://onlinelibrary.wiley.com/doi/10.1111/j.1472-4642.2007.00341.x" onclick="newWindow(this.href);return false" target="_blank">Wiley Online Library</a></span><span>
                           <a href="https://esajournals.onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=8&amp;doi=10.1002%2Fecy.2000&amp;key=Can%27t+run+EXECUTE.+Wrong+input+line" onclick="newWindow(this.href);return false" target="_blank">PubMed</a></span><span>
                           <a href="https://esajournals.onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1002%2Fecy.2000&amp;key=000245812100002" onclick="newWindow(this.href);return false" target="_blank">Web of Science®</a></span><span><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=13&amp;publication_year=2007&amp;pages=252-264&amp;journal=Diversity+and+Distributions&amp;author=S.+Ferrier&amp;author=G.+Manion&amp;author=J.+Elith&amp;author=K.+Richardson&amp;title=Using+generalized+dissimilarity+modelling+to+analyse+and+predict+patterns+of+beta+diversity+in+regional+biodiversity+assessment" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="ecy2000-bib-0025">Good, I. J. <span class="pubYear">1953</span>. <span class="articleTitle">The population frequencies of species and the estimation of population parameters</span>. <i>Biometrika</i> <span class="vol">40</span>:237–264.
                     <div class="extra-links" id="e_1_2_9_26_1"><span>
                           <a href="https://onlinelibrary.wiley.com/doi/10.1111/j.1574-6941.2006.00105.x" onclick="newWindow(this.href);return false" target="_blank">Wiley Online Library</a></span><span>
                           <a href="https://esajournals.onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=32&amp;doi=10.1002%2Fecy.2000&amp;key=1%3ACAS%3A528%3ADC%252BD28XntFaqtLc%253D" onclick="newWindow(this.href);return false" target="_blank">CAS</a></span><span>
                           <a href="https://esajournals.onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=8&amp;doi=10.1002%2Fecy.2000&amp;key=13096665" onclick="newWindow(this.href);return false" target="_blank">PubMed</a></span><span>
                           <a href="https://esajournals.onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1002%2Fecy.2000&amp;key=A1953UZ96400001" onclick="newWindow(this.href);return false" target="_blank">Web of Science®</a></span><span><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=40&amp;publication_year=1953&amp;pages=237-264&amp;journal=Biometrika&amp;author=I.+J.+Good&amp;title=The+population+frequencies+of+species+and+the+estimation+of+population+parameters" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="ecy2000-bib-0026">Good, I. J. <span class="pubYear">1983</span>. <span class="bookTitle">Good thinking: the foundations of probability&nbsp;and its applications</span>. University of Minnesota Press, Minneapolis, USA.
                     <div class="extra-links" id="e_1_2_9_27_1"><span><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;publication_year=1983&amp;author=I.+J.+Good&amp;title=Good+thinking%3A+the+foundations+of+probability%C2%A0and+its+applications" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="ecy2000-bib-0027">Good, I. J. <span class="pubYear">2000</span>. <span class="articleTitle">Turing's anticipation of empirical Bayes in connection with the cryptanalysis of the naval enigma</span>. <i>Journal of Statistical Computation and Simulation</i> <span class="vol">66</span>:101–111.
                     <div class="extra-links" id="e_1_2_9_28_1"><span>
                           <a href="https://esajournals.onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=16&amp;doi=10.1002%2Fecy.2000&amp;key=10.1080%2F00949650008812016" onclick="newWindow(this.href);return false" target="_blank">Crossref</a></span><span>
                           <a href="https://esajournals.onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1002%2Fecy.2000&amp;key=000088795400003" onclick="newWindow(this.href);return false" target="_blank">Web of Science®</a></span><span><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=66&amp;publication_year=2000&amp;pages=101-111&amp;journal=Journal+of+Statistical+Computation+and+Simulation&amp;author=I.+J.+Good&amp;title=Turing%27s+anticipation+of+empirical+Bayes+in+connection+with+the+cryptanalysis+of+the+naval+enigma" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="ecy2000-bib-0028">Good, I. J., and G. Toulmin. <span class="pubYear">1956</span>. <span class="articleTitle">The number of new species and the increase of population coverage when a sample is increased</span>. <i>Biometrika</i> <span class="vol">43</span>:45–63.
                     <div class="extra-links" id="e_1_2_9_29_1"><span>
                           <a href="https://esajournals.onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=16&amp;doi=10.1002%2Fecy.2000&amp;key=10.1093%2Fbiomet%2F43.1-2.45" onclick="newWindow(this.href);return false" target="_blank">Crossref</a></span><span>
                           <a href="https://esajournals.onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1002%2Fecy.2000&amp;key=A1956XF69200005" onclick="newWindow(this.href);return false" target="_blank">Web of Science®</a></span><span><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=43&amp;publication_year=1956&amp;pages=45-63&amp;journal=Biometrika&amp;author=I.+J.+Good&amp;author=G.+Toulmin&amp;title=The+number+of+new+species+and+the+increase+of+population+coverage+when+a+sample+is+increased" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="ecy2000-bib-0029">Gotelli, N. J., and A. Chao. <span class="pubYear">2013</span>. <span class="chapterTitle">Measuring and estimating species richness, species diversity, and biotic similarity from sampling data</span>. Pages 195–211 <i>in</i> S. A. Levin, editor. <span class="bookTitle">Encyclopedia of biodiversity</span>. <span class="edition">Second edition</span>. Volume <span class="vol">5</span>. Academic Press, Waltham, MA, USA.
                     <div class="extra-links" id="e_1_2_9_30_1"><span>
                           <a href="https://esajournals.onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=16&amp;doi=10.1002%2Fecy.2000&amp;key=10.1016%2FB978-0-12-384719-5.00424-X" onclick="newWindow(this.href);return false" target="_blank">Crossref</a></span><span><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;publication_year=2013&amp;pages=195-211&amp;author=N.+J.+Gotelli&amp;author=A.+Chao&amp;title=Encyclopedia+of+biodiversity" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="ecy2000-bib-0030">Gotelli, N. J., and R. K. Colwell. <span class="pubYear">2011</span>. <span class="chapterTitle">Estimating species richness</span>. Pages 39–54 <i>in</i> A. Magurran, and B. McGill, editors. <span class="bookTitle">Biological diversity: frontiers in measurement and assessment</span>. Oxford University Press, Oxford.
                     <div class="extra-links" id="e_1_2_9_31_1"><span><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;publication_year=2011&amp;pages=39-54&amp;author=N.+J.+Gotelli&amp;author=R.+K.+Colwell&amp;title=Biological+diversity%3A+frontiers+in+measurement+and+assessment" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="ecy2000-bib-0031">Gotelli, N. J., and D. J. McCabe. <span class="pubYear">2002</span>. <span class="articleTitle">Species co‐occurrence: a meta‐analysis of J.M. Diamond's assembly rules model</span>. <i>Ecology</i> <span class="vol">83</span>:2091–2096.
                     <div class="extra-links" id="e_1_2_9_32_1"><span>
                           <a href="https://esajournals.onlinelibrary.wiley.com/doi/10.1890/0012-9658%282002%29083%5B2091%3ASCOAMA%5D2.0.CO%3B2" onclick="newWindow(this.href);return false" target="_blank">Wiley Online Library</a></span><span>
                           <a href="https://esajournals.onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1002%2Fecy.2000&amp;key=000177434200003" onclick="newWindow(this.href);return false" target="_blank">Web of Science®</a></span><span><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=83&amp;publication_year=2002&amp;pages=2091-2096&amp;journal=Ecology&amp;author=N.+J.+Gotelli&amp;author=D.+J.+McCabe&amp;title=Species+co%E2%80%90occurrence%3A+a+meta%E2%80%90analysis+of+J.M.+Diamond%27s+assembly+rules+model" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="ecy2000-bib-0032">Gotelli, N. J., R. M. Dorazio, A. M. Ellison, and G. D. Grossman. <span class="pubYear">2010</span>. <span class="articleTitle">Detecting temporal trends in species assemblages with bootstrapping procedures and hierarchical models</span>. <i>Philosophical Transactions of the Royal Society B</i> <span class="vol">365</span>:3621–3631.
                     <div class="extra-links" id="e_1_2_9_33_1"><span>
                           <a href="https://esajournals.onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=16&amp;doi=10.1002%2Fecy.2000&amp;key=10.1098%2Frstb.2010.0262" onclick="newWindow(this.href);return false" target="_blank">Crossref</a></span><span>
                           <a href="https://esajournals.onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=8&amp;doi=10.1002%2Fecy.2000&amp;key=20980311" onclick="newWindow(this.href);return false" target="_blank">PubMed</a></span><span>
                           <a href="https://esajournals.onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1002%2Fecy.2000&amp;key=000283665200004" onclick="newWindow(this.href);return false" target="_blank">Web of Science®</a></span><span><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=365&amp;publication_year=2010&amp;pages=3621-3631&amp;journal=Philosophical+Transactions+of+the+Royal+Society+B&amp;author=N.+J.+Gotelli&amp;author=R.+M.+Dorazio&amp;author=A.+M.+Ellison&amp;author=G.+D.+Grossman&amp;title=Detecting+temporal+trends+in+species+assemblages+with+bootstrapping+procedures+and+hierarchical+models" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="ecy2000-bib-0033">Hortal, J., P. A. V. Borges, and C. Gaspar. <span class="pubYear">2006</span>. <span class="articleTitle">Evaluating the performance of species richness estimators: sensitivity to sample grain size</span>. <i>Journal of Animal Ecology</i> <span class="vol">75</span>:274–287.
                     <div class="extra-links" id="e_1_2_9_34_1"><span>
                           <a href="https://besjournals.onlinelibrary.wiley.com/doi/10.1111/j.1365-2656.2006.01048.x" onclick="newWindow(this.href);return false" target="_blank">Wiley Online Library</a></span><span>
                           <a href="https://esajournals.onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=8&amp;doi=10.1002%2Fecy.2000&amp;key=16903065" onclick="newWindow(this.href);return false" target="_blank">PubMed</a></span><span>
                           <a href="https://esajournals.onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1002%2Fecy.2000&amp;key=000235043700028" onclick="newWindow(this.href);return false" target="_blank">Web of Science®</a></span><span><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=75&amp;publication_year=2006&amp;pages=274-287&amp;journal=Journal+of+Animal+Ecology&amp;author=J.+Hortal&amp;author=P.+A.+V.+Borges&amp;author=C.+Gaspar&amp;title=Evaluating+the+performance+of+species+richness+estimators%3A+sensitivity+to+sample+grain+size" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="ecy2000-bib-0034">Hsieh, T. C., and A. Chao. <span class="pubYear">2017</span>. <span class="articleTitle">Rarefaction and extrapolation: making fair comparison of abundance‐sensitive phylogenetic diversity among multiple assemblages</span>. <i>Systematic Biology</i> <span class="vol">66</span>:100–111.
                     <div class="extra-links" id="e_1_2_9_35_1"><span>
                           <a href="https://esajournals.onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=32&amp;doi=10.1002%2Fecy.2000&amp;key=1%3ASTN%3A280%3ADC%252BC1cnltVWmtw%253D%253D" onclick="newWindow(this.href);return false" target="_blank">CAS</a></span><span>
                           <a href="https://esajournals.onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=8&amp;doi=10.1002%2Fecy.2000&amp;key=28173592" onclick="newWindow(this.href);return false" target="_blank">PubMed</a></span><span>
                           <a href="https://esajournals.onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1002%2Fecy.2000&amp;key=000397124100012" onclick="newWindow(this.href);return false" target="_blank">Web of Science®</a></span><span><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=66&amp;publication_year=2017&amp;pages=100-111&amp;journal=Systematic+Biology&amp;author=T.+C.+Hsieh&amp;author=A.+Chao&amp;title=Rarefaction+and+extrapolation%3A+making+fair+comparison+of+abundance%E2%80%90sensitive+phylogenetic+diversity+among+multiple+assemblages" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="ecy2000-bib-0035">Jesus, R. M., and S. G. Rolim. <span class="pubYear">2005</span>. <span class="articleTitle">Fitossociologia da floresta atlântica de tabuleiro em Linhares (ES)</span>. <i>Boletim Técnico SIF</i> <span class="vol">19</span>:1–149.
                     <div class="extra-links" id="e_1_2_9_36_1"><span><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=19&amp;publication_year=2005&amp;pages=1-149&amp;journal=Boletim+T%C3%A9cnico+SIF&amp;author=R.+M.+Jesus&amp;author=S.+G.+Rolim&amp;title=Fitossociologia+da+floresta+atl%C3%A2ntica+de+tabuleiro+em+Linhares+%28ES%29" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="ecy2000-bib-0036">Jost, L., A. Chao, and R. Chazdon. <span class="pubYear">2011</span>. <span class="chapterTitle">Compositional similarity and beta diversity</span>. Pages 66–84 <i>in</i> A. Magurran, and B.&nbsp;McGill, editors. <span class="bookTitle">Biological diversity: frontiers in measurement and assessment</span>. Oxford University Press, Oxford, UK.
                     <div class="extra-links" id="e_1_2_9_37_1"><span><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;publication_year=2011&amp;pages=66-84&amp;author=L.+Jost&amp;author=A.+Chao&amp;author=R.+Chazdon&amp;title=Biological+diversity%3A+frontiers+in+measurement+and+assessment" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="ecy2000-bib-0037">Lozupone, C., and R. Knight. <span class="pubYear">2005</span>. <span class="articleTitle">UniFrac: a new phylogenetic method for comparing microbial communities</span>. <i>Applied and Environmental Microbiology</i> <span class="vol">71</span>:8228–8235.
                     <div class="extra-links" id="e_1_2_9_38_1"><span>
                           <a href="https://esajournals.onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=16&amp;doi=10.1002%2Fecy.2000&amp;key=10.1128%2FAEM.71.12.8228-8235.2005" onclick="newWindow(this.href);return false" target="_blank">Crossref</a></span><span>
                           <a href="https://esajournals.onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=32&amp;doi=10.1002%2Fecy.2000&amp;key=1%3ACAS%3A528%3ADC%252BD2MXhtlehtb7K" onclick="newWindow(this.href);return false" target="_blank">CAS</a></span><span>
                           <a href="https://esajournals.onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=8&amp;doi=10.1002%2Fecy.2000&amp;key=16332807" onclick="newWindow(this.href);return false" target="_blank">PubMed</a></span><span>
                           <a href="https://esajournals.onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1002%2Fecy.2000&amp;key=000234417600073" onclick="newWindow(this.href);return false" target="_blank">Web of Science®</a></span><span><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=71&amp;publication_year=2005&amp;pages=8228-8235&amp;journal=Applied+and+Environmental+Microbiology&amp;author=C.+Lozupone&amp;author=R.+Knight&amp;title=UniFrac%3A+a+new+phylogenetic+method+for+comparing+microbial+communities" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="ecy2000-bib-0038">Magnago, L. F. S., D. P. Edwards, F. A. Edwards, A. Magrach, S. V. Martins, and W. F. Laurance. <span class="pubYear">2014</span>. <span class="articleTitle">Functional attributes change but functional richness is unchanged after fragmentation of Brazilian Atlantic forests</span>. <i>Journal of Ecology</i> <span class="vol">102</span>:475–485.
                     <div class="extra-links" id="e_1_2_9_39_1"><span>
                           <a href="https://besjournals.onlinelibrary.wiley.com/doi/10.1111/1365-2745.12206" onclick="newWindow(this.href);return false" target="_blank">Wiley Online Library</a></span><span>
                           <a href="https://esajournals.onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1002%2Fecy.2000&amp;key=000331467300018" onclick="newWindow(this.href);return false" target="_blank">Web of Science®</a></span><span><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=102&amp;publication_year=2014&amp;pages=475-485&amp;journal=Journal+of+Ecology&amp;author=L.+F.+S.+Magnago&amp;author=D.+P.+Edwards&amp;author=F.+A.+Edwards&amp;author=A.+Magrach&amp;author=S.+V.+Martins&amp;author=W.+F.+Laurance&amp;title=Functional+attributes+change+but+functional+richness+is+unchanged+after+fragmentation+of+Brazilian+Atlantic+forests" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="ecy2000-bib-0039">Magurran, A. E. <span class="pubYear">2004</span>. <span class="bookTitle">Measuring biological diversity</span>. Blackwell, Oxford, UK.
                     <div class="extra-links" id="e_1_2_9_40_1"><span>
                           <a href="https://esajournals.onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=16&amp;doi=10.1002%2Fecy.2000&amp;key=10.2989%2F16085910409503825" onclick="newWindow(this.href);return false" target="_blank">Crossref</a></span><span>
                           <a href="https://esajournals.onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=32&amp;doi=10.1002%2Fecy.2000&amp;key=1%3ACAS%3A528%3ADC%252BD28XhtVCntLs%253D" onclick="newWindow(this.href);return false" target="_blank">CAS</a></span><span>
                           <a href="https://esajournals.onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=8&amp;doi=10.1002%2Fecy.2000&amp;key=16529380" onclick="newWindow(this.href);return false" target="_blank">PubMed</a></span><span>
                           <a href="https://esajournals.onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1002%2Fecy.2000&amp;key=000226601800004" onclick="newWindow(this.href);return false" target="_blank">Web of Science®</a></span><span><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;publication_year=2004&amp;author=A.+E.+Magurran&amp;title=Measuring+biological+diversity" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="ecy2000-bib-0040">Matos, F. A. R., L. F. S. Magnago, M. Gastauer, J. M. B. Carreiras, M. Simonelli, J. A. A. Meira‐Neto, and D. P. Edwards. <span class="pubYear">2017</span>. <span class="articleTitle">Effects of landscape configuration and composition on phylogenetic diversity of trees in a highly fragmented tropical forest</span>. <i>Journal of Ecology</i> <span class="vol">105</span>:265–276.
                     <div class="extra-links" id="e_1_2_9_41_1"><span>
                           <a href="https://besjournals.onlinelibrary.wiley.com/doi/10.1111/1365-2745.12661" onclick="newWindow(this.href);return false" target="_blank">Wiley Online Library</a></span><span>
                           <a href="https://esajournals.onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1002%2Fecy.2000&amp;key=000390331000026" onclick="newWindow(this.href);return false" target="_blank">Web of Science®</a></span><span><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=105&amp;publication_year=2017&amp;pages=265-276&amp;journal=Journal+of+Ecology&amp;author=F.+A.+R.+Matos&amp;author=L.+F.+S.+Magnago&amp;author=M.+Gastauer&amp;author=J.+M.+B.+Carreiras&amp;author=M.+Simonelli&amp;author=J.+A.+A.+Meira%E2%80%90Neto&amp;author=D.+P.+Edwards&amp;title=Effects+of+landscape+configuration+and+composition+on+phylogenetic+diversity+of+trees+in+a+highly+fragmented+tropical+forest" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="ecy2000-bib-0041">McGrayne, S. B. <span class="pubYear">2011</span>. <span class="bookTitle">The
 theory that would not die: how Bayes’ rule cracked the enigma code, 
hunted down Russian submarines, and emerged triumphant from two 
centuries of controversy</span>. Yale University Press, New Haven, USA.
                     <div class="extra-links" id="e_1_2_9_42_1"><span>
                           <a href="https://esajournals.onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1002%2Fecy.2000&amp;key=000289726700022" onclick="newWindow(this.href);return false" target="_blank">Web of Science®</a></span><span><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;publication_year=2011&amp;author=S.+B.+McGrayne&amp;title=The+theory+that+would+not+die%3A+how+Bayes%E2%80%99+rule+cracked+the+enigma+code%2C+hunted+down+Russian+submarines%2C+and+emerged+triumphant+from+two+centuries+of+controversy" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="ecy2000-bib-0042">Pan, H. Y., A. Chao, and W. Foissner. <span class="pubYear">2009</span>. <span class="articleTitle">A non‐parametric lower bound for the number of species shared by multiple communities</span>. <i>Journal of Agricultural, Biological and Environmental Statistics</i> <span class="vol">14</span>:452–468.
                     <div class="extra-links" id="e_1_2_9_43_1"><span>
                           <a href="https://esajournals.onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=16&amp;doi=10.1002%2Fecy.2000&amp;key=10.1198%2Fjabes.2009.07113" onclick="newWindow(this.href);return false" target="_blank">Crossref</a></span><span>
                           <a href="https://esajournals.onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=8&amp;doi=10.1002%2Fecy.2000&amp;key=20622922" onclick="newWindow(this.href);return false" target="_blank">PubMed</a></span><span>
                           <a href="https://esajournals.onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1002%2Fecy.2000&amp;key=000272634200006" onclick="newWindow(this.href);return false" target="_blank">Web of Science®</a></span><span><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=14&amp;publication_year=2009&amp;pages=452-468&amp;journal=Journal+of+Agricultural%2C+Biological+and+Environmental+Statistics&amp;author=H.+Y.+Pan&amp;author=A.+Chao&amp;author=W.+Foissner&amp;title=A+non%E2%80%90parametric+lower+bound+for+the+number+of+species+shared+by+multiple+communities" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="ecy2000-bib-0043">Paula, A., and J. J. Soares. <span class="pubYear">2011</span>. <span class="articleTitle">Estrutura horizontal de um trecho de Floresta Ombrófila Densa das Terras Baixas na Reserva Biológica de Sooretama, Linhares, ES</span>. <i>Revista Floresta</i> <span class="vol">41</span>:321–334.
                     <div class="extra-links" id="e_1_2_9_44_1"><span><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=41&amp;publication_year=2011&amp;pages=321-334&amp;journal=Revista+Floresta&amp;author=A.+Paula&amp;author=J.+J.+Soares&amp;title=Estrutura+horizontal+de+um+trecho+de+Floresta+Ombr%C3%B3fila+Densa+das+Terras+Baixas+na+Reserva+Biol%C3%B3gica+de+Sooretama%2C+Linhares%2C+ES" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="ecy2000-bib-0044">Rangel, T. F., R. K. Colwell, G. R. Graves, K. Fučíková, C. Rahbek, and J. A. F. Diniz‐Filho. <span class="pubYear">2015</span>. <span class="articleTitle">Phylogenetic uncertainty revisited: Implications for ecological analyses</span>. <i>Evolution</i> <span class="vol">69</span>:1301–1312.
                     <div class="extra-links" id="e_1_2_9_45_1"><span>
                           <a href="https://onlinelibrary.wiley.com/doi/10.1111/evo.12644" onclick="newWindow(this.href);return false" target="_blank">Wiley Online Library</a></span><span>
                           <a href="https://esajournals.onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=8&amp;doi=10.1002%2Fecy.2000&amp;key=25800868" onclick="newWindow(this.href);return false" target="_blank">PubMed</a></span><span>
                           <a href="https://esajournals.onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1002%2Fecy.2000&amp;key=000354561600015" onclick="newWindow(this.href);return false" target="_blank">Web of Science®</a></span><span><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=69&amp;publication_year=2015&amp;pages=1301-1312&amp;journal=Evolution&amp;author=T.+F.+Rangel&amp;author=R.+K.+Colwell&amp;author=G.+R.+Graves&amp;author=K.+Fu%C4%8D%C3%ADkov%C3%A1&amp;author=C.+Rahbek&amp;author=J.+A.+F.+Diniz%E2%80%90Filho&amp;title=Phylogenetic+uncertainty+revisited%3A+Implications+for+ecological+analyses" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="ecy2000-bib-0045">Robbins, H. E. <span class="pubYear">1968</span>. <span class="articleTitle">Estimating the total probability of the unobserved outcomes of an experiment</span>. <i>The Annals of Mathematical Statistics</i> <span class="vol">39</span>:256–257.
                     <div class="extra-links" id="e_1_2_9_46_1"><span>
                           <a href="https://esajournals.onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=16&amp;doi=10.1002%2Fecy.2000&amp;key=10.1214%2Faoms%2F1177698526" onclick="newWindow(this.href);return false" target="_blank">Crossref</a></span><span>
                           <a href="https://esajournals.onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1002%2Fecy.2000&amp;key=A1968A681400026" onclick="newWindow(this.href);return false" target="_blank">Web of Science®</a></span><span><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=39&amp;publication_year=1968&amp;pages=256-257&amp;journal=The+Annals+of+Mathematical+Statistics&amp;author=H.+E.+Robbins&amp;title=Estimating+the+total+probability+of+the+unobserved+outcomes+of+an+experiment" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="ecy2000-bib-0046">Rolim, S. G., and H. E. M. Nascimento. <span class="pubYear">1997</span>. <span class="articleTitle">Análise
 da riqueza, diversidade e relação espécie‐abundância de uma comunidade 
arbórea tropical em diferentes intensidades amostrais</span>. <i>Scientia Forestalis</i> <span class="vol">52</span>:7–16.
                     <div class="extra-links" id="e_1_2_9_47_1"><span><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=52&amp;publication_year=1997&amp;pages=7-16&amp;journal=Scientia+Forestalis&amp;author=S.+G.+Rolim&amp;author=H.+E.+M.+Nascimento&amp;title=An%C3%A1lise+da+riqueza%2C+diversidade+e+rela%C3%A7%C3%A3o+esp%C3%A9cie%E2%80%90abund%C3%A2ncia+de+uma+comunidade+arb%C3%B3rea+tropical+em+diferentes+intensidades+amostrais" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="ecy2000-bib-0047">Schmera, D., T. Erös, and J. Podani. <span class="pubYear">2009</span>. <span class="articleTitle">A measure for assessing functional diversity in ecological communities</span>. <i>Aquatic Ecology</i> <span class="vol">43</span>:157–167.
                     <div class="extra-links" id="e_1_2_9_48_1"><span>
                           <a href="https://esajournals.onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=16&amp;doi=10.1002%2Fecy.2000&amp;key=10.1007%2Fs10452-007-9152-9" onclick="newWindow(this.href);return false" target="_blank">Crossref</a></span><span>
                           <a href="https://esajournals.onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1002%2Fecy.2000&amp;key=000263159200014" onclick="newWindow(this.href);return false" target="_blank">Web of Science®</a></span><span><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=43&amp;publication_year=2009&amp;pages=157-167&amp;journal=Aquatic+Ecology&amp;author=D.+Schmera&amp;author=T.+Er%C3%B6s&amp;author=J.+Podani&amp;title=A+measure+for+assessing+functional+diversity+in+ecological+communities" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="ecy2000-bib-0048">Swenson, N. G., D. L. Erickson, X. Mi, N. A. Bourg, J. Forero‐Montaña, X. Ge, R. Howe, J. K. Lake, X. Liu, and K. Ma. <span class="pubYear">2012</span>. <span class="articleTitle">Phylogenetic and functional alpha and beta diversity in&nbsp;temperate and tropical tree communities</span>. <i>Ecology</i> <span class="vol">93</span>:S112–S125.
                     <div class="extra-links" id="e_1_2_9_49_1"><span>
                           <a href="https://esajournals.onlinelibrary.wiley.com/doi/10.1890/11-0402.1" onclick="newWindow(this.href);return false" target="_blank">Wiley Online Library</a></span><span>
                           <a href="https://esajournals.onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1002%2Fecy.2000&amp;key=000307302400010" onclick="newWindow(this.href);return false" target="_blank">Web of Science®</a></span><span><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=93&amp;publication_year=2012&amp;pages=S112-S125&amp;journal=Ecology&amp;author=N.+G.+Swenson&amp;author=D.+L.+Erickson&amp;author=X.+Mi&amp;author=N.+A.+Bourg&amp;author=J.+Forero%E2%80%90Monta%C3%B1a&amp;author=X.+Ge&amp;author=R.+Howe&amp;author=J.+K.+Lake&amp;author=X.+Liu&amp;author=K.+Ma&amp;title=Phylogenetic+and+functional+alpha+and+beta+diversity+in%C2%A0temperate+and+tropical+tree+communities" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="ecy2000-bib-0049">Tilman, D., J. Knops, D. Wedin, P. Reich, M. Ritchie, and E.&nbsp;Siemann. <span class="pubYear">1997</span>. <span class="articleTitle">The influence of functional diversity and&nbsp;composition on ecosystem processes</span>. <i>Science</i> <span class="vol">277</span>:1300–1302.
                     <div class="extra-links" id="e_1_2_9_50_1"><span>
                           <a href="https://esajournals.onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=16&amp;doi=10.1002%2Fecy.2000&amp;key=10.1126%2Fscience.277.5330.1300" onclick="newWindow(this.href);return false" target="_blank">Crossref</a></span><span>
                           <a href="https://esajournals.onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=32&amp;doi=10.1002%2Fecy.2000&amp;key=1%3ACAS%3A528%3ADyaK2sXlslans7s%253D" onclick="newWindow(this.href);return false" target="_blank">CAS</a></span><span>
                           <a href="https://esajournals.onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=8&amp;doi=10.1002%2Fecy.2000&amp;key=Can%27t+run+EXECUTE.+Wrong+input+line" onclick="newWindow(this.href);return false" target="_blank">PubMed</a></span><span>
                           <a href="https://esajournals.onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1002%2Fecy.2000&amp;key=A1997XT82700053" onclick="newWindow(this.href);return false" target="_blank">Web of Science®</a></span><span><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=277&amp;publication_year=1997&amp;pages=1300-1302&amp;journal=Science&amp;author=D.+Tilman&amp;author=J.+Knops&amp;author=D.+Wedin&amp;author=P.+Reich&amp;author=M.+Ritchie&amp;author=E.+Siemann&amp;title=The+influence+of+functional+diversity+and%C2%A0composition+on+ecosystem+processes" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="ecy2000-bib-0050">Walker, B., A. Kinzig, and J. Langridge. <span class="pubYear">1999</span>. <span class="articleTitle">Plant attribute diversity, resilience, and ecosystem function: the nature and significance of dominant and minor species</span>. <i>Ecosystems</i> <span class="vol">2</span>:95–113.
                     <div class="extra-links" id="e_1_2_9_51_1"><span>
                           <a href="https://esajournals.onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=16&amp;doi=10.1002%2Fecy.2000&amp;key=10.1007%2Fs100219900062" onclick="newWindow(this.href);return false" target="_blank">Crossref</a></span><span>
                           <a href="https://esajournals.onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1002%2Fecy.2000&amp;key=000080173500001" onclick="newWindow(this.href);return false" target="_blank">Web of Science®</a></span><span><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=2&amp;publication_year=1999&amp;pages=95-113&amp;journal=Ecosystems&amp;author=B.+Walker&amp;author=A.+Kinzig&amp;author=J.+Langridge&amp;title=Plant+attribute+diversity%2C+resilience%2C+and+ecosystem+function%3A+the+nature+and+significance+of+dominant+and+minor+species" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="ecy2000-bib-0051">Webb, C. O., and M. J. Donoghue. <span class="pubYear">2005</span>. <span class="articleTitle">Phylomatic: tree assembly for applied phylogenetics</span>. <i>Molecular Ecology Notes</i> <span class="vol">5</span>:181–183.
                     <div class="extra-links" id="e_1_2_9_52_1"><span>
                           <a href="https://onlinelibrary.wiley.com/doi/10.1111/j.1471-8286.2004.00829.x" onclick="newWindow(this.href);return false" target="_blank">Wiley Online Library</a></span><span>
                           <a href="https://esajournals.onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=8&amp;doi=10.1002%2Fecy.2000&amp;key=Can%27t+run+EXECUTE.+Wrong+input+line" onclick="newWindow(this.href);return false" target="_blank">PubMed</a></span><span>
                           <a href="https://esajournals.onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1002%2Fecy.2000&amp;key=000227598500058" onclick="newWindow(this.href);return false" target="_blank">Web of Science®</a></span><span><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=5&amp;publication_year=2005&amp;pages=181-183&amp;journal=Molecular+Ecology+Notes&amp;author=C.+O.+Webb&amp;author=M.+J.+Donoghue&amp;title=Phylomatic%3A+tree+assembly+for+applied+phylogenetics" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="ecy2000-bib-0052">Webb, C. O., D. D. Ackerly, M. A. McPeek, and M. J. Donoghue. <span class="pubYear">2002</span>. <span class="articleTitle">Phylogenies and community ecology</span>. <i>Annual Review of Ecology and Systematics</i> <span class="vol">33</span>:475–505.
                     <div class="extra-links" id="e_1_2_9_53_1"><span>
                           <a href="https://esajournals.onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=16&amp;doi=10.1002%2Fecy.2000&amp;key=10.1146%2Fannurev.ecolsys.33.010802.150448" onclick="newWindow(this.href);return false" target="_blank">Crossref</a></span><span>
                           <a href="https://esajournals.onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=8&amp;doi=10.1002%2Fecy.2000&amp;key=10856198" onclick="newWindow(this.href);return false" target="_blank">PubMed</a></span><span>
                           <a href="https://esajournals.onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1002%2Fecy.2000&amp;key=000180007000018" onclick="newWindow(this.href);return false" target="_blank">Web of Science®</a></span><span><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=33&amp;publication_year=2002&amp;pages=475-505&amp;journal=Annual+Review+of+Ecology+and+Systematics&amp;author=C.+O.+Webb&amp;author=D.+D.+Ackerly&amp;author=M.+A.+McPeek&amp;author=M.+J.+Donoghue&amp;title=Phylogenies+and+community+ecology" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="ecy2000-bib-0053">Webb, C. O., D. D. Ackerly, and S. W. Kembel. <span class="pubYear">2008</span>. <span class="articleTitle">Phylocom: software for the analysis of phylogenetic community structure and trait evolution</span>. <i>Bioinformatics</i> <span class="vol">24</span>:2098–2100.
                     <div class="extra-links" id="e_1_2_9_54_1"><span>
                           <a href="https://esajournals.onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=16&amp;doi=10.1002%2Fecy.2000&amp;key=10.1093%2Fbioinformatics%2Fbtn358" onclick="newWindow(this.href);return false" target="_blank">Crossref</a></span><span>
                           <a href="https://esajournals.onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=32&amp;doi=10.1002%2Fecy.2000&amp;key=1%3ACAS%3A528%3ADC%252BD1cXhtFWhurjM" onclick="newWindow(this.href);return false" target="_blank">CAS</a></span><span>
                           <a href="https://esajournals.onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=8&amp;doi=10.1002%2Fecy.2000&amp;key=18678590" onclick="newWindow(this.href);return false" target="_blank">PubMed</a></span><span>
                           <a href="https://esajournals.onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1002%2Fecy.2000&amp;key=000258959600020" onclick="newWindow(this.href);return false" target="_blank">Web of Science®</a></span><span><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=24&amp;publication_year=2008&amp;pages=2098-2100&amp;journal=Bioinformatics&amp;author=C.+O.+Webb&amp;author=D.+D.+Ackerly&amp;author=S.+W.+Kembel&amp;title=Phylocom%3A+software+for+the+analysis+of+phylogenetic+community+structure+and+trait+evolution" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
               </ul>
            </div>
         </div>
      </section>
      
   </section>
   <div style="display:none">
      <ul class="custom-style-list noteList"></ul>
   </div>
   
</article></div></div><div class="col-lg-4 col-md-4 col-sm-4 hidden-xs hidden-sm hidden-md sticko__parent article-row-right"><div class="publication-cover-info clearfix">



        
        <div class="cover-image-wrapper cover-image__image hasDetails"><a href="https://esajournals.onlinelibrary.wiley.com/toc/19399170/2017/98/11"><img src="ecy.jpg" alt="Publication cover image"></a></div><div class="extra-info-wrapper cover-image__details"><p class="volume-issue"><a href="https://esajournals.onlinelibrary.wiley.com/toc/19399170/2017/98/11"><span>Volume</span><span class="val">98</span>, <span>Issue</span><span class="val">11</span></a></p><p>November 2017</p><p class="page-range"><span>Pages</span><span> 2914-2929</span></p>



        
        
</div>
</div>



        
        


        <div class="pb-ad">
            <div id="advert-rail-1" class="advert advert-rail">
			<script>

				googletag.cmd.push 		( function () {
					googletag.display 	( 'advert-rail-1' );
				});

			</script>

			<!-- No JavaScript support is a call to an image -->
			<noscript>
				<a href="/action/clickThrough?id=1011&url=https%3A%2F%2Fpubads.g.doubleclick.net%2Fgampad%2Fjump%3Fiu%3D%24googlePublisherCategory%26amp%3Bsz%3D160x600%7C160x320%7C160x160%7C120x600%26amp%3Btile%3D2%26amp%3Bc%3D%24random&loc=%2Fdoi%2Ffull%2F10.1002%2Fecy.2000&pubId=60241333&placeholderId=1014&productId=101018" target="_blank">
					<img src="https://pubads.g.doubleclick.net/gampad/ad?iu=$googlePublisherCategory&amp;sz=160x600|160x320|160x160|120x600&amp;tile=2&amp;c=$random">
				</a>
			</noscript>
		<div id="google_ads_iframe_/2215/wly.ecology_000046/ecology_ecy_1__container__" style="border: 0pt none; width: 300px; height: 250px;"></div></div>
        </div>
    

<div class="tab tab--slide tab--flex sticko__md tabs--xs dynamic-sticko tab--flex tabs--xs js--sticko" style="width: 329.983px; top: 81px; position: static; height: 0px;"><ul data-mobile-toggle="slide" class="rlist tab__nav w-slide--list cloned"><li role="presentation" class=""><a data-slide-clone="self" href="#pane-pcw-figures" aria-controls="#pane-pcw-figures" role="tab" data-toggle="tab" id="pane-pcw-figurescon" data-slide-target="#pane-pcw-figures" class="article-row-right__tabLinks w-slide__btn figures-tab empty" aria-expanded="false"><i aria-hidden="true" class="icon-multi-figure"></i><span>Figures</span></a></li><li role="presentation"><a data-slide-clone="self" href="#pane-pcw-references" aria-controls="#pane-pcw-references" role="tab" data-toggle="tab" id="pane-pcw-referencescon" data-slide-target="#pane-pcw-references" class="article-row-right__tabLinks w-slide__btn references-tab empty"><i aria-hidden="true" class="icon-chain"></i><span>References</span></a></li><li role="presentation"><a data-slide-clone="self" href="#pane-pcw-related" aria-controls="#pane-pcw-related" role="tab" data-toggle="tab" id="pane-pcw-relatedcon" data-slide-target="#pane-pcw-related" class="article-row-right__tabLinks w-slide__btn related-tab empty"><i aria-hidden="true" class="icon-related-circles-extend-button"></i><span>Related</span></a></li><li role="presentation" class="active"><a data-slide-clone="self" href="#pane-pcw-details" aria-controls="#pane-pcw-details" role="tab" data-toggle="tab" id="pane-pcw-detailscon" data-slide-target="#pane-pcw-details" class="article-row-right__tabLinks w-slide__btn details-tab" aria-expanded="true"><i aria-hidden="true" class="icon-Icon_Information"></i><span>Information</span></a></li></ul><ul role="tablist" class="rlist tab__content sticko__child" style="height: 0px; overflow-y: auto;"><li id="pane-pcw-figures" aria-labelledby="pane-pcw-figurescon" role="tab" class="tab__pane article-row-right__panes empty"></li><li id="pane-pcw-references" aria-labelledby="pane-pcw-referencescon" role="tab" class="tab__pane article-row-right__panes empty"></li><li id="pane-pcw-related" aria-labelledby="pane-pcw-relatedcon" role="tab" class="tab__pane article-row-right__panes empty"></li><li id="pane-pcw-details" aria-labelledby="pane-pcw-detailscon" role="tab" class="tab__pane article-row-right__panes active"><div class="contentTab"></div><div class="emptyTab">



        
        


        
    





        
        


        <div class="pb-ad">
            <aside>
	<div id="advert-rail-2" class="advert advert-rail">
		<script>

			googletag.cmd.push 		( function () {
				googletag.display 	( 'advert-rail-2' );
			});

		</script>

		<!-- No JavaScript support is a call to an image -->
		<noscript>
			<a href="/action/clickThrough?id=1012&url=http%3A%2F%2Fpubads.g.doubleclick.net%2Fgampad%2Fjump%3Fiu%3D%24googlePublisherCategory%26amp%3Bsz%3D160x600%7C160x320%7C160x160%7C120x600%26amp%3Btile%3D2%26amp%3Bc%3D%24random&loc=%2Fdoi%2Ffull%2F10.1002%2Fecy.2000&pubId=60241333&placeholderId=1013&productId=101018" target="_blank">
				<img src="http://pubads.g.doubleclick.net/gampad/ad?iu=$googlePublisherCategory&amp;sz=160x600|160x320|160x160|120x600&amp;tile=2&amp;c=$random">
			</a>
		</noscript>
	<div id="google_ads_iframe_/2215/wly.ecology_000046/ecology_ecy_2__container__" style="border: 0pt none; width: 300px; height: 250px;"></div></div>
</aside>
        </div>
    

</div></li><li class="tab__spinner"><img src="spinner.gif" id="spinner" style="width: 100%" alt=""></li></ul></div></div></div></article><script>var articleRef = document.querySelector('.article__body:not(.show-references) .article__references');
if (articleRef) { articleRef.style.display = "none"; }

</script><div id="figure-viewer" data-ux3-wrapper="figure-viewer" data-ux3-transformed-by="figureInit" data-ux3-role="parent" role="dialog" class="figure-viewer"><div class="figure-viewer__reg__top clearfix"><div class="figure-viewer__top__right"><a title="Close Figure Viewer" href="#" data-ux3-role="controller" class="figure-viewer__ctrl__close"><span class="icon-close_thin"></span></a></div><div class="figure-viewer__top__left"><a title="Browse All Figures" href="#" class="figure-viewer__ctrl__browse"><span class="icon-allfigures"></span></a><a title="Return to Figure" href="#" class="figure-viewer__ctrl__return is-hidden"><span class="icon-arrow-left"></span></a><button title="zoom out" class="figure-viewer__label__zoom icon-zoom-out zoom-out"></button><span tabindex="1" class="zoomSlider js__zoom-slider ui-slider"><input class="zoom-range" value="50" type="range"></span><button title="zoom in" class="figure-viewer__label__zoom icon-zoom-in zoom-in"></button></div></div><div class="figure-viewer__reg__center" style="height: 100%;"><div class="figure-viewer__cent__left"><a title="Previous Figure" href="#" class="figure-viewer__ctrl__prev"><span class="icon-arrow_l"></span></a><a title="Next Figure" href="#" class="figure-viewer__ctrl__next"><span class="icon-arrow_r"></span></a><div class="figure-viewer__hold__fig"><figure class="holder"></figure></div><div class="figure-viewer__hold__list clearfix container"></div></div><div class="figure-viewer__cent__right"><div class="figure-viewer__title"><a title="Open/Close Caption" href="#" class="figure-viewer__ctrl__caption"><span class="icon-doublearrow"></span><span class="figure-viewer__caption__label">Caption</span></a><span class="figure-viewer__title__text"></span></div><div class="figure-viewer__hold__figcap"></div></div></div></div>
</div>
</div></div></div>

        </section>
    
    </div>
</div><!--end pagefulltext-->

        </div>
    










        <div data-widget-def="pageFooter" data-widget-id="053994f3-7d80-4bb1-97a4-c67fbc4768cd" class="pageFooter">
        



        
        <footer class="page-footer">
    <div data-pb-dropzone="main">
    









        <section data-widget-def="ux3-layout-widget" data-widget-id="efb69c87-9ba0-4904-93cf-43e770aee16e" class="footer-sections padded-content hub-footer hub-first-footer">
        



        
        <div class="container"><div class="row"><div>









        <div data-widget-def="ux3-layout-widget" data-widget-id="cae47753-902f-4371-98d0-422ea9741270">
        



        
        <div class="container"><div class="row"><div class=" col-sm-6">



        
        <a href="https://esajournals.onlinelibrary.wiley.com/" title="The Ecological Society of America Logo"><img alt="The Ecological Society of America Logo" src="logo-footer-1508231358113.png"></a>




        
        <div class="hub-copyright hiddenLabel--xs hiddenLabel--sm hiddenLabel--md">
    © 2019 Ecological Society of America. All rights reserved.
</div>


</div><div class=" col-sm-6">



        
        <div id="rich-text-14cafa34-f103-4af7-8e96-c8b8dab06e68" class="pb-rich-text" style="overflow: auto;">
        <p style="text-align: right;"><span class="footer__list--item" style="margin: 0px; padding: 0px; border: 0px; outline: 0px; font-size: 13px; vertical-align: baseline; background: 0px 0px transparent; display: inline-block; color: #424242; font-family: 'Open Sans', Arial, Helvetica, 'Lucida Sans Unicode', sans-serif; text-align: right;">ESA Headquarters</span><span style="color: #424242; font-family: 'Open Sans', Arial, Helvetica, 'Lucida Sans Unicode', sans-serif; font-size: 13px; text-align: right; background-color: transparent;">&nbsp;|&nbsp;</span><span class="footer__list--item" style="margin: 0px; padding: 0px; border: 0px; outline: 0px; font-size: 13px; vertical-align: baseline; background: 0px 0px transparent; display: inline-block; color: #424242; font-family: 'Open Sans', Arial, Helvetica, 'Lucida Sans Unicode', sans-serif; text-align: right;">1990 M Street, NW</span><span style="color: #424242; font-family: 'Open Sans', Arial, Helvetica, 'Lucida Sans Unicode', sans-serif; font-size: 13px; text-align: right; background-color: transparent;">&nbsp;|&nbsp;</span><span class="footer__list--item" style="margin: 0px; padding: 0px; border: 0px; outline: 0px; font-size: 13px; vertical-align: baseline; background: 0px 0px transparent; display: inline-block; color: #424242; font-family: 'Open Sans', Arial, Helvetica, 'Lucida Sans Unicode', sans-serif; text-align: right;">Suite 700</span><br style="color: #424242; font-family: 'Open Sans', Arial, Helvetica, 'Lucida Sans Unicode', sans-serif; font-size: 13px; text-align: right; background-color: transparent;"><span class="footer__list--item" style="margin: 0px; padding: 0px; border: 0px; outline: 0px; font-size: 13px; vertical-align: baseline; background: 0px 0px transparent; display: inline-block; color: #424242; font-family: 'Open Sans', Arial, Helvetica, 'Lucida Sans Unicode', sans-serif; text-align: right;">Washington, DC 20036</span><span style="color: #424242; font-family: 'Open Sans', Arial, Helvetica, 'Lucida Sans Unicode', sans-serif; font-size: 13px; text-align: right; background-color: transparent;">&nbsp;|&nbsp;</span><span class="footer__list--item" style="margin: 0px; padding: 0px; border: 0px; outline: 0px; font-size: 13px; vertical-align: baseline; background: 0px 0px transparent; display: inline-block; color: #424242; font-family: 'Open Sans', Arial, Helvetica, 'Lucida Sans Unicode', sans-serif; text-align: right;">phone 202-833-8773</span><span style="color: #424242; font-family: 'Open Sans', Arial, Helvetica, 'Lucida Sans Unicode', sans-serif; font-size: 13px; text-align: right; background-color: transparent;">&nbsp;|&nbsp;</span><span class="footer__list--item" style="margin: 0px; padding: 0px; border: 0px; outline: 0px; font-size: 13px; vertical-align: baseline; background: 0px 0px transparent; display: inline-block; color: #424242; font-family: 'Open Sans', Arial, Helvetica, 'Lucida Sans Unicode', sans-serif; text-align: right;">email: <a href="mailto:esajournals@esa.org">esajournals@esa.org</a></span></p>
</div>

</div></div></div>

        </div>
    




        
        <div class="col-md-12">
    <div class="hub-copyright visible-xs visible-sm visible-md">
        © 2019 Ecological Society of America. All rights reserved.
    </div>
</div>
</div></div></div>

        </section>
    




        
        <div class="main-footer">



        
        <style>
    .footer-links {
        text-align: center;
    }
    .footer-links h3 {
        font-weight: 300;
    }
    .footer-links a {
        font-weight: 600;
    }
    @media (min-width: 992px) {
        .footer-links {
            text-align: left;
        }
    }
</style>












        <div data-widget-def="ux3-layout-widget" data-widget-id="2753bec1-a79d-4403-827b-6c788ddd6db3" class="footer-sections">
        



        
            <h3 class="hidden accessibility-enabled">
                Additional links
            </h3>
        
        <div class="container"><div class="row"><div class=" col-md-3">



        
        <div class="footer-links mb-3">
  <h3>About Wiley Online Library</h3>
  <ul class="rlist">
    <li><a href="https://www.wiley.com/privacy">Privacy Policy</a></li>
    <li><a href="https://onlinelibrary.wiley.com/terms-and-conditions">Terms of Use</a></li>
    <li><a href="https://onlinelibrary.wiley.com/cookies">Cookies</a></li>
    <li><a href="https://onlinelibrary.wiley.com/accessibility">Accessibility</a></li>
  </ul>
</div>
</div><div class=" col-md-3">



        
         <div class="footer-links mb-3">
    <h3>Help &amp; Support</h3>
    <ul class="rlist">
      <li><a href="https://hub.wiley.com/community/support/onlinelibrary">Contact Us</a></li>
    </ul>
  </div>
</div><div class=" col-md-3">



        
         <div class="footer-links mb-3">
    <h3>Opportunities</h3>
    <ul class="rlist">
      <li><a href="https://onlinelibrary.wiley.com/agents">Subscription Agents</a></li>
      <li><a href="https://onlinelibrary.wiley.com/advertisers">Advertisers &amp; Corporate Partners</a></li>
    </ul>
  </div>
</div><div class=" col-md-3">



        
        <div class="footer-links mb-3">
    <h3>Connect with Wiley</h3>
    <ul class="rlist">
      <li><a href="https://hub.wiley.com/community/exchanges/">The Wiley Network</a></li>
      <li><a href="http://newsroom.wiley.com/">Wiley Press Room</a></li>
    </ul>
  </div>
</div></div></div>

        </div>
    










        <div data-widget-def="ux3-layout-widget" data-widget-id="86584e25-e4db-4118-9550-9b4f5341a4ac" class="copyright-container">
        



        
        <div class="container"><div class="row"><div class=" col-md-12">









        <div data-widget-def="UX3HTMLWidget" data-widget-id="bfbf6c5d-a650-4662-828f-5275fada9120" class="copyright">
        



        
        <div class="copyright pull-left">Copyright © 1999-2019 <a href="http://www.wiley.com/">John Wiley &amp; Sons, Inc</a>. All rights reserved</div>

        </div>
    










        <div data-widget-def="ux3-general-image" data-widget-id="063fa3e6-c0a3-4672-a2e6-62aa1b09ed11" class="footer-logo pull-right">
        



        
        <a href="http://www.wiley.com/" title="Wiley Home Page"><img alt="Wiley Home Page" src="footer-logo-wiley-1510029248417.png"></a>

        </div>
    
</div></div></div>

        </div>
    
</div>
    </div>
</footer>

        </div>
    




        
        
















<div class="popup login-popup hidden" role="dialog" tabindex="-1" aria-labelledby="modal__header" aria-hidden="true">



    <div class="content">
        <a href="#" class="close" role="button" aria-label="Cancel Login">
            <i class="icon-tools_close" aria-hidden="true"></i>
        </a>


        <div class="modal__header">
            <div class="header_logo">
                <img src="logo-header-1526603583437.png" alt="">
            </div>
        </div>

        <div class="modal__body">
            <div class="ssoLogin">
                
                    <h2>Log in with your society membership</h2>
                    <a href="https://esajournals.onlinelibrary.wiley.com/action/sso?redirectUri=%2Fdoi%2Ffull%2F10.1002%2Fecy.2000" class="link">
                        <span class="sign-in-label">Log in with ESA</span>
                    </a>
                
            </div>

            <h2>Log in to Wiley Online Library</h2>

            





























<div class="login-form">
    

    

    
    <form action="/action/doLogin?societyURLCode=" class="bordered" method="post"><input name="id" value="67065c09-4a88-49cd-934c-ac707951d35c" type="hidden">
        
            <input name="popup" value="true" type="hidden">
        
        

        
        <input name="loginUri" value="/doi/full/10.1002/ecy.2000" type="hidden">
        
            




<input name="redirectUri" value="/doi/full/10.1002/ecy.2000" type="hidden">
<input name="loginUri" value="/doi/full/10.1002/ecy.2000" type="hidden">
            
        
        
        
                



















    
    
    <div class="input-group">
        <div class="label">

            
                    <label for="login">Email or Customer ID</label>
                
        </div>
        <input id="username" class="login" name="login" size="15" placeholder="Enter your email" autocorrect="off" spellcheck="false" autocapitalize="off" type="text">
        
                    
                        <div class="actions">
                            
                        </div>
                    
                
    </div>


                
                
















    
    <div class="input-group">
        <div class="label">
            <label for="password">Password</label>
        </div>
        <input id="password" class="password" name="password" value="" autocomplete="off" placeholder="Enter your password" autocorrect="off" spellcheck="false" autocapitalize="off" type="password">
        <span class="password-eye-icon icon-eye hidden"></span>
        
                
    </div>

                
                    
                        <div class="actions">
                            
                            <a href="https://esajournals.onlinelibrary.wiley.com/action/requestResetPassword" class="link show-request-reset-password">
                                Forgot password?
                            </a>
                        </div>
                    
                
            
        
        <div class="loginExtraBeans-dropZone" data-pb-dropzone="loginExtraBeans">
            
        </div>

        
                <div class="align-end">
                    
                    
                            
                            <span class="submit " disabled="disabled">
                                <input class="button btn submit primary no-margin-bottom" name="submitButton" value="Log In" disabled="disabled" type="submit">
                            </span>
                        
                </div></form>
    <div class="societySpecificInstructions">
        
    </div>
</div>

            
                <div class="login-container clearfix form-btn">
                    <a href="https://esajournals.onlinelibrary.wiley.com/action/registration" class="linkBtn float-left">
                        
                        NEW USER &gt;
                    </a>
                    
                        <a href="https://esajournals.onlinelibrary.wiley.com/action/ssostart?redirectUri=%2Fdoi%2Ffull%2F10.1002%2Fecy.2000" class="linkBtn float-right instituation-login">
                            
                            INSTITUTIONAL LOGIN &gt;
                        </a>
                    
                </div>
            

        </div>
    </div>
</div>




        
        






<div class="popup change-password-drawer hidden">
    <div class="content">
        <a href="#" class="close cancel"><i class="icon-close_thin"></i></a>
        <div class="modal__header">


            <div class="header_logo">
                <img src="logo-header-1526603583437.png" alt="">
            </div>
            <h2>Change Password</h2>
        </div>
        <div class="modal__body">

            <div class="form">

                <form action="/action/changePassword" method="post"><div class="message error"></div>
                    <input name="submit" value="submit" type="hidden">
                    <div class="input-group">
                        <div class="label">
                            <label for="a589574e-bb98-4c6e-8fed-67365ff05357-old">Old Password</label>
                        </div>
                        <input id="a589574e-bb98-4c6e-8fed-67365ff05357-old" class="old" name="old" value="" autocomplete="off" type="password">
                        <span class="password-eye-icon icon-eye hidden"></span>
                    </div>
                    <div class="input-group">
                        <div class="label">
                            <label for="a589574e-bb98-4c6e-8fed-67365ff05357-new">New Password</label>
                        </div>
                        <input id="a589574e-bb98-4c6e-8fed-67365ff05357-new" class="pass-hint new" name="new" value="" autocomplete="off" type="password">
                        <span class="password-eye-icon icon-eye hidden"></span>
                        
                            







    


    


    


<div class="password-strength-indicator" data-min="10" data-max="32" data-strength="3">
    <span class="text too-short">Too Short</span>
    <span class="text weak">Weak</span>
    <span class="text medium">Medium</span>
    <span class="text strong">Strong</span>
    <span class="text very-strong">Very Strong</span>
    <span class="text too-long">Too Long</span>
</div>
                        
                    </div>
                    <input class="button primary submit" value="Submit" disabled="disabled" type="submit"></form>
            </div>
            <div class="success-template hidden">
                <h2>Congrats!</h2>
                <p>Your password has been changed</p>
            </div>
        </div>
    </div>
</div>





        
        





<div class="popup registration-popup hidden">



    <div class="content">
        <a href="#" class="close"><i class="icon-tools_close"></i></a>
        <div class="modal__header">
            <div class="logoDropZone" data-pb-dropzone="logoDropZone">
                
            </div>

            <h2>Create a new account</h2>
        </div>

        <div class="modal__body">
            <form action="/action/registration" class="registration-form" method="post"><input name="redirectUri" value="/doi/full/10.1002/ecy.2000" type="hidden">
                <div class="input-group">
                    <div class="label">
                        <label for="4e647394-f751-4441-baa4-df426bca4b6e.email">Email or Customer ID</label>
                    </div>
                    <input id="4e647394-f751-4441-baa4-df426bca4b6e.email" class="email" name="email" size="15" type="text">
                </div>
                <div class="submit">
                    <input class="button submit primary" value="Register" disabled="disabled" type="submit">
                </div></form>
            <div class="center">
                <a href="https://esajournals.onlinelibrary.wiley.com/action/showLogin" class="show-login">
                    
                    Returning user
                </a>
            </div>
        </div>
    </div>
</div>




        
        



<div class="popup top-drawer request-reset-password-drawer hidden">
    <div class="content">
        <div class="modal__header">
            <a href="#" class="cancel close"><i class="icon-close_thin"></i></a>

            <div class="header_logo">
                <img src="logo-header-1526603583437.png" alt="">
            </div>

            <h2>Forgot your password?</h2>
            <p class="sub">Enter your email address below. </p>
        </div>

        <div class="modal__body">

            









<div class="error"></div>


<form action="/action/requestResetPassword" class="request-reset-password-form" method="post"><input name="requestResetPassword" value="true" type="hidden">
    
            <div class="input-group">
                



















    
    
    <div class="input-group">
        <div class="label">

            
                    <label for="email">Email or Customer ID</label>
                
        </div>
        <input id="email" class="email" name="email" placeholder="Enter your email" autocorrect="off" spellcheck="false" autocapitalize="off" type="email">
        
    </div>

            </div>
        


    
            <div class="password-recaptcha-ajax"></div>
        

    <div class="form-btn">
        <input class="button btn primary submit" name="submit" value="RESET PASSWORD" disabled="disabled" type="submit">
    </div></form>
            

            <div class="message error"></div>

            <div class="success-template hidden">
                <p>"Please check your email for instructions on resetting your password.
                    If you do not receive an email within 10 minutes, your email address may not be registered,
                    and you may need to create a new Wiley Online Library account.</p>
            </div>
        </div>
    </div>
</div>





        
        




<div class="popup top-drawer request-username-drawer username-popup hidden">
    <a href="#" class="close"><i class="icon-close_thin"></i></a>
    <div class="content">
        <h2>Request Username</h2>
        <div class="form">
            <p>Can't sign in? Forgot your username?</p>
            <p class="sub">Enter your email address below and we will send you your username</p>
            <div class="message error"></div>
            <form action="/action/requestUsername" method="post"><input name="requestUsername" value="requestUsername" type="hidden">
                <div class="input-group">
                    <div class="label">
                        <label for="ac834f24-aa07-4ad2-9d13-f77c843f21cb.email">Email or Customer ID</label>
                    </div>
                    <input id="ac834f24-aa07-4ad2-9d13-f77c843f21cb.email" class="email" name="email" size="15" type="text">
                </div>

                <div class="username-recaptcha-ajax">
                </div>

                <input class="button primary submit" name="submit" value="Submit" disabled="disabled" type="submit">
                <div class="center">
                    <a href="#" class="cancel">Close</a>
                </div></form>
        </div>
        <div class="success-template hidden note">
            <p>If the address matches an existing account you will receive an email with instructions to retrieve your username</p>
        </div>
    </div>
</div>




        
        <!-- Belongs at the bottom of the page -->

<script type="text/javascript">
if(window._satellite) { 
_satellite.pageBottom(); 
}
</script><script src="satellite-5aaec36164746d79c1001d14.js"></script>
        </div>
    </div><div style="visibility: hidden; height: 1px; width: 1px; position: absolute; top: -9999px; z-index: 100000;" id="_atssh"><iframe id="_atssh854" title="AddThis utility frame" style="height: 1px; width: 1px; position: absolute; top: 0px; z-index: 100000; border: 0px none; left: 0px;" src="https://s7.addthis.com/static/sh.e4e8af4de595fdb10ec1459d.html#rand=0.5104335150951709&amp;iit=1550577919326&amp;tmr=load%3D1550577911858%26core%3D1550577911894%26main%3D1550577919318%26ifr%3D1550577919332&amp;cb=0&amp;cdn=0&amp;md=0&amp;kw=&amp;ab=-&amp;dh=esajournals.onlinelibrary.wiley.com&amp;dr=&amp;du=https%3A%2F%2Fesajournals.onlinelibrary.wiley.com%2Fdoi%2Ffull%2F10.1002%2Fecy.2000&amp;href=https%3A%2F%2Fesajournals.onlinelibrary.wiley.com%2Fdoi%2Ffull%2F10.1002%2Fecy.2000&amp;dt=Deciphering%20the%20enigma%20of%20undetected%20species%2C%20phylogenetic%2C%20and%20functional%20diversity%20based%20on%20Good%E2%80%90Turing%20theory%20-%20Chao%20-%202017%20-%20Ecology%20-%20Wiley%20Online%20Library&amp;dbg=0&amp;cap=tc%3D0%26ab%3D0&amp;inst=1&amp;jsl=1&amp;prod=undefined&amp;lng=en&amp;ogt=&amp;pc=men&amp;pub=xa-4faab26f2cff13a7&amp;ssl=1&amp;sid=5c6bf0f7958bc629&amp;srf=0.01&amp;ver=300&amp;xck=0&amp;xtr=0&amp;og=&amp;csi=undefined&amp;rev=v8.3.38-wp&amp;ct=1&amp;xld=1&amp;xd=1"></iframe></div><style id="service-icons-0"></style>


<script type="text/javascript" src="main.js"></script><div class="kxhead" data-id="JMOONjNh" style="display:none !important;"><span class="kxtag kxinvisible" data-id="20226"><script>
// this tag is intentionally blank
</script></span><span class="kxtag kxinvisible" data-id="20227"></span><span class="kxtag kxinvisible" data-id="20229"><script>
(function() {
  // adadvisor redirects to http://logger... so it's not https safe
  if (location.protocol == "http:") {
     var u = "https://aa.agkn.com/adscores/g.js?sid=9212244187&_kdpid=2111c0af-fc3a-446f-ab07-63aa74fbde8e";
     (new Image()).src = u;
   }
})();
</script></span><span class="kxtag kxinvisible" data-id="20234"><script type="text/javascript">
var findPh = document.getElementsByTagName("h1")[0]
if(findPh != null) { var pageheader = document.getElementsByTagName("h1")[0].innerHTML }

// Using Omniture prop1 to produce page attribute Page Type
Krux('set', 'page_attr_omniture_prop1', typeof window.s =='object' && window.s['prop1']);

// Using Omniture prop2 to produce page attribute Page Subtype
Krux('set', 'page_attr_omniture_prop2', typeof window.s =='object' && window.s['prop2']);

// Using Omniture prop3 to produce page attribute Title
Krux('set', 'page_attr_omniture_prop3', typeof window.s =='object' && window.s['prop3']);

// Using Omniture prop4 to produce page attribute Subject Codes
Krux('set', 'page_attr_omniture_prop4', typeof window.s =='object' && window.s['prop4']);

// Using Omniture prop7 to produce page attribute Article Title
Krux('set', 'page_attr_omniture_prop7', typeof window.s =='object' && window.s['prop7']);

// Using Omniture prop8 to produce page attribute Journal Volume and Issue
Krux('set', 'page_attr_omniture_prop8', typeof window.s =='object' && window.s['prop8']);

// Using Omniture prop9 to produce page attribute Article Access State
Krux('set', 'page_attr_omniture_prop9', typeof window.s =='object' && window.s['prop9']);

// Using Omniture prop10 to produce page attribute Publication Access Status
Krux('set', 'page_attr_omniture_prop10', typeof window.s =='object' && window.s['prop10']);

// Using Omniture prop13 to produce page attribute Journal DOI
Krux('set', 'page_attr_omniture_prop13', typeof window.s =='object' && window.s['prop13']);

// Using Omniture prop14 to produce page attribute Article / Chapter DOI
Krux('set', 'page_attr_omniture_prop14', typeof window.s =='object' && window.s['prop14']);

// Using Omniture prop15 to produce page attribute Issue DOI
Krux('set', 'page_attr_omniture_prop15', typeof window.s =='object' && window.s['prop15']);

// Using Omniture prop16 to produce page attribute MRW Chapter
Krux('set', 'page_attr_omniture_prop16', typeof window.s =='object' && window.s['prop16']);

// Using Omniture prop20 to produce page attribute Site Search Query
Krux('set', 'page_attr_omniture_prop20', typeof window.s =='object' && window.s['prop20']);

// Using Omniture prop26 to produce page attribute Logged In or Out
Krux('set', 'user_attr_omniture_prop26', typeof window.s =='object' && window.s['prop26']);

// Using Omniture channel to produce page attribute Channel
Krux('set', 'page_attr_omniture_channel', typeof window.s =='object' && window.s['channel']);

// Using Omniture prop33 to produce page attribute Primary Subject Code
Krux('set', 'page_attr_omniture_prop33', typeof window.s =='object' && window.s['prop33']);

// Using pageheader to produce page attribute Page Header
Krux('scrape', { 'page_attr_wileyonlinelib_pageheader': {javascript: 'pageheader'}});

</script></span><span class="kxtag kxinvisible" data-id="20238"></span><span class="kxtag kxinvisible" data-id="20241"><script>
    (function() {
        var kuid = Krux('get', 'user');
        if (kuid) {
            var prefix = location.protocol == 'https:' ? "https:" : "http:";
            var kurl_params = encodeURIComponent("_kuid=" + kuid + "&_kdpid=d7158cb7-a851-4e3c-b7ab-cc9e815b2399&dlxid=<na_id>&dlxdata=<na_da>");
            var kurl = prefix + "//beacon.krxd.net/data.gif?" + kurl_params;
            var dlx_url = '//r.nexac.com/e/getdata.xgi?dt=br&pkey=gwjn34gwjnh86&ru=' + kurl;
            var i = new Image();
            i.src = dlx_url;
        }
    })();
</script></span><span class="kxtag kxinvisible" data-id="20243"></span><span class="kxtag kxinvisible" data-id="20247"><script type="text/javascript">
(function() {
    (new Image()).src = '//aa.agkn.com/adscores/g.js?sid=9212244187&_kdpid=2111c0af-fc3a-446f-ab07-63aa74fbde8e';
})();
</script></span><span class="kxtag kxinvisible" data-id="20248"><script>
(function(){
        var kuid = Krux('get', 'user');
        var prefix = window.location.protocol == 'https:' ? 'https:' : 'http:';
        if (kuid) {
           var url = prefix + '//pix.btrll.com/partner/868092.png';
           (new Image()).src = url;
        }
})();
</script></span><span class="kxtag kxinvisible" data-id="20251"><script>
    (function() {
        var kuid = Krux('get', 'user');
        if (kuid) {
            var prefix = location.protocol == 'https:' ? "https:" : "http:";
            var kurl_params = encodeURIComponent("_kuid=" + kuid + "&_kdpid=8da8b14d-5569-4bec-bcea-722864ee8d06&dlxid=<na_id>&dlxdata=<na_da>");
            var kurl = prefix + "//beacon.krxd.net/data.gif?" + kurl_params;
            var dlx_url = '//r.nexac.com/e/getdata.xgi?dt=br&pkey=iefs40iefsj26&ru=' + kurl;
            var i = new Image();
            i.src = dlx_url;
        }
    })();
</script></span><span class="kxtag kxinvisible" data-id="20252"><script>
    (function() {
        var kuid = Krux('get', 'user');
        if (kuid) {
            var prefix = location.protocol == 'https:' ? "https:" : "http:";
            var kurl_params = encodeURIComponent("_kuid=" + kuid + "&_kdpid=bef9f122-393d-4c45-8d8d-32d8be7ac433&dlxid=<na_id>&dlxdata=<na_da>");
            var kurl = prefix + "//beacon.krxd.net/data.gif?" + kurl_params;
            var dlx_url = '//r.nexac.com/e/getdata.xgi?dt=br&pkey=iyzu39iyzud95&ru=' + kurl;
            var i = new Image();
            i.src = dlx_url;
        }
    })();
</script></span><span class="kxtag kxinvisible" data-id="20253"><script>
    (function() {
        var allowUserMatch = function() {
            var GDN_SITE_ID = '1282650';
 
            if (Krux('get', 'config').params.client_type === 'marketer') {
                try {
                    var params = Krux('require:marketer').getParams(
                            Krux('require:sizzle').find('script[src*="' + Krux('get', 'confid') + '"]')
                    );
                    return GDN_SITE_ID !== (params.siteid || params.kxsiteid);
                }
                catch (e) {
                    // In case we don't find the script tag or the URL parser fails, just allow matching.
                }
            }
            return true;
        };
         
        var kuid = Krux('get', 'user');
        if (allowUserMatch() && kuid) {
            var prefix = location.protocol == 'https:' ? "https:" : "http:";
            var kurl_params = encodeURIComponent("_kuid=" + kuid + "&_kdpid=8bf57916-aac8-4f01-a386-4baf103b3e1f&dlxid=<na_id>&dlxdata=<na_da>");
            var kurl = prefix + "//beacon.krxd.net/data.gif?" + kurl_params;
            var dlx_url = '//r.nexac.com/e/getdata.xgi?dt=br&pkey=bckw15bckwu20&ru=' + kurl;
            var i = new Image();
            i.src = dlx_url;
        }
    })();
</script></span><span class="kxtag kxinvisible" data-id="20254"><script>
    (function() {
        var allowUserMatch = function() {
            var GDN_SITE_ID = '1282650';

            if (Krux('get', 'config').params.client_type === 'marketer') {
                try {
                    var params = Krux('require:marketer').getParams(
                            Krux('require:sizzle').find('script[src*="' + Krux('get', 'confid') + '"]')
                    );
                    return GDN_SITE_ID !== (params.siteid || params.kxsiteid);
                }
                catch (e) {
                    // In case we don't find the script tag or the URL parser fails, just allow matching.
                }
            }
            return true;
        };

        var kuid = Krux('get', 'user');
        if (allowUserMatch() && kuid) {
            var prefix = location.protocol == 'https:' ? "https:" : "http:";
            var kurl_params = encodeURIComponent("_kuid=" + kuid + "&_kdpid=d7158cb7-a851-4e3c-b7ab-cc9e815b2399&dlxid=<na_id>&dlxdata=<na_da>");
            var kurl = prefix + "//beacon.krxd.net/data.gif?" + kurl_params;
            var dlx_url = '//r.nexac.com/e/getdata.xgi?dt=br&pkey=gwjn34gwjnh86&ru=' + kurl;
            var i = new Image();
            i.src = dlx_url;
        }
    })();
</script></span><span class="kxtag kxinvisible" data-id="20255"><script>
    (function() {
        var kuid = Krux('get', 'user');
        if (kuid) {
            var prefix = location.protocol == 'https:' ? "https:" : "http:";
            var kurl_params = encodeURIComponent("_kuid=" + kuid + "&_kdpid=536f0daa-aaaa-4d9e-9a25-dde40646786a&dlxid=<na_id>&dlxdata=<na_da>");
            var kurl = prefix + "//beacon.krxd.net/data.gif?" + kurl_params;
            var dlx_url = '//r.nexac.com/e/getdata.xgi?dt=br&pkey=iefs40iefsj26&ru=' + kurl;
            var i = new Image();
            i.src = dlx_url;
        }
    })();
</script></span><span class="kxtag kxinvisible" data-id="20256"><script>
    (function() {
        var kuid = Krux('get', 'user');
        if (kuid) {
            var prefix = location.protocol == 'https:' ? "https:" : "http:";
            var kurl_params = encodeURIComponent("_kuid=" + kuid + "&_kdpid=7c6392c9-e878-492c-8b14-bf06e3828ebd&dlxid=<na_id>&dlxdata=<na_da>");
            var kurl = prefix + "//beacon.krxd.net/data.gif?" + kurl_params;
            var dlx_url = '//r.nexac.com/e/getdata.xgi?dt=br&pkey=rowp70rowpu60&ru=' + kurl;
            var i = new Image();
            i.src = dlx_url;
        }
    })();
</script></span><span class="kxtag kxinvisible" data-id="20257"><script>
    (function() {
        var allowUserMatch = function() {
            var GDN_SITE_ID = '1282650';

            if (Krux('get', 'config').params.client_type === 'marketer') {
                try {
                    var params = Krux('require:marketer').getParams(
                            Krux('require:sizzle').find('script[src*="' + Krux('get', 'confid') + '"]')
                    );
                    return GDN_SITE_ID !== (params.siteid || params.kxsiteid);
                }
                catch (e) {
                    // In case we don't find the script tag or the URL parser fails, just allow matching.
                }
            }
            return true;
        };
        
        var kuid = Krux('get', 'user');
        if (allowUserMatch() && kuid) {
            var prefix = location.protocol == 'https:' ? "https:" : "http:";
            var kurl_params = encodeURIComponent("_kuid=" + kuid + "&_kdpid=bb8ae0e2-9cd7-45b2-ad37-7737269627d8&dlxid=<na_id>&dlxdata=<na_da>");
            var kurl = prefix + "//beacon.krxd.net/data.gif?" + kurl_params;
            var dlx_url = '//r.nexac.com/e/getdata.xgi?dt=br&pkey=rsxs71rsxsk73&ru=' + kurl;
            var i = new Image();
            i.src = dlx_url;
        }
    })();
</script></span></div>
<script type="text/javascript" src="ga7iproduct.js"></script>
<script>var _prum=[['id','59e8fecb3847311aab7b23c6'],['mark','firstbyte',(new Date()).getTime()]];(function(){var s=document.getElementsByTagName('script')[0],p=document.createElement('script');p.async='async';p.src='//rum-static.pingdom.net/prum.min.js';s.parentNode.insertBefore(p,s);})();</script>

<div id="crossmark-widget" style="display: none;"><div class="crossmark-reset crossmark-overlay"></div><div class="crossmark-reset crossmark-popup"><div class="crossmark-reset crossmark-popup__offset"><div class="crossmark-reset crossmark-popup__inner"><div class="crossmark-reset crossmark-popup__header"><a target="_blank" href="http://www.crossref.org/crossmark"><img class="crossmark-reset crossmark-popup__logo"></a><button class="crossmark-reset crossmark-popup__btn-close"></button></div><div class="crossmark-reset crossmark-popup__content-wrapper"><iframe class="crossmark-reset crossmark-popup__content"></iframe></div></div></div></div></div><div class="w-slide"><div class="w-slide_head"><a href="#" class="w-slide__back"><i class=" icon-arrow_l" aria-hidden="true"></i>back</a><span class="w-slide__title"></span></div><div class="w-slide__content"></div></div><ul class="ui-autocomplete ui-front ui-menu ui-widget ui-widget-content ui-corner-all quickSearchAutocomplete" id="ui-id-1" tabindex="0" style="display: none;"></ul><ul class="ui-autocomplete ui-front ui-menu ui-widget ui-widget-content ui-corner-all quickSearchAutocomplete" id="ui-id-2" tabindex="0" style="display: none;"></ul></body><iframe src="https://injections.readcube.com/html/track_epdf.3307bb36.html?action=Link&amp;publisher=wiley&amp;status=OK" style="position: absolute; visibility: hidden; width: 0px; height: 0px;"></iframe><iframe id="kx-proxy-JMOONjNh" src="https://cdn.krxd.net/partnerjs/xdi/proxy.3d2100fd7107262ecb55ce6847f01fa5.html#%21kxcid=JMOONjNh&amp;kxt=https%3A%2F%2Fesajournals.onlinelibrary.wiley.com&amp;kxcl=cdn&amp;kxp=" style="display: none; visibility: hidden; height: 0; width: 0;"></iframe></html>