<!DOCTYPE html>
<html xmlns:og="http://ogp.me/ns#" xmlns:fb="https://www.facebook.com/2008/fbml" class=" js flexbox flexboxlegacy canvas canvastext webgl no-touch geolocation postmessage no-websqldatabase indexeddb hashchange history draganddrop websockets rgba hsla multiplebgs backgroundsize borderimage borderradius boxshadow textshadow opacity cssanimations csscolumns cssgradients no-cssreflections csstransforms csstransforms3d csstransitions fontface generatedcontent video audio localstorage sessionstorage webworkers applicationcache svg inlinesvg smil svgclippaths" lang="en"><head>
<meta http-equiv="content-type" content="text/html; charset=UTF-8"><meta class="foundation-mq-small"><meta class="foundation-mq-small-only"><meta class="foundation-mq-medium"><meta class="foundation-mq-medium-only"><meta class="foundation-mq-large"><meta class="foundation-mq-large-only"><meta class="foundation-mq-xlarge"><meta class="foundation-mq-xlarge-only"><meta class="foundation-mq-xxlarge"><meta class="foundation-data-attribute-namespace">
<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge,chrome=1">
<meta content="width=device-width, initial-scale=1.0" name="viewport">
<title>Remote Sensing  | Free Full-Text | Delineating Individual Trees 
from Lidar Data: A Comparison of Vector- and Raster-based Segmentation 
Approaches | HTML</title><link rel="stylesheet" href="font-awesome.css">
<link rel="stylesheet" href="jquery.css">
<link rel="stylesheet" href="chosen.css">
<link rel="stylesheet" href="main2.css">
<link rel="mask-icon" href="https://www.mdpi.com/img/mask-icon-128.svg?c1c7eca266cd7013" color="#4f5671">
<link rel="apple-touch-icon" sizes="57x57" href="https://www.mdpi.com/icon/apple-touch-icon-57x57.png">
<link rel="apple-touch-icon" sizes="60x60" href="https://www.mdpi.com/icon/apple_touch-icon-60x60.png">
<link rel="apple-touch-icon" sizes="72x72" href="https://www.mdpi.com/icon/apple-touch-icon-72x72.png">
<link rel="apple-touch-icon" sizes="76x76" href="https://www.mdpi.com/icon/apple-touch-icon-76x76.png">
<link rel="manifest" href="https://www.mdpi.com/manifest.json">
<meta name="theme-color" content="#ffffff">
<meta name="application-name" content="&nbsp;">
<link rel="apple-touch-startup-image" href="https://www.mdpi.com/img/journals/remotesensing-logo-sq.png">
<link rel="apple-touch-icon" href="https://www.mdpi.com/img/journals/remotesensing-logo-sq.png">
<meta name="msapplication-TileImage" content="/img/journals/remotesensing-logo-sq.png">
<style>
h1, h2 {
    color: #37702E;
}

.batch_articles a {
    color: #000;
}

    a, .batch_articles .authors a, a:focus, a:hover, .batch_articles a:focus, .batch_articles a:hover {
        color: #37702E;
    }

    span.label a {
        color: #fff;
    }

    #content div.generic-item a.title-link:hover {
        color: #37702E;
    }

    #content #middle-column .generic-item.article-item a.title-link:hover {
        color: #37702E;
    }

    .art-authors a {
        color: #000;
        font-weight: 700;
    }

#content .inverted h2 {
    background-color: #37702E;
}

header {
    background-color: #37702E;
}

#full-size-menu .top-bar, #full-size-menu li.menu-item span.user-email, nav.tab-bar {
    background-color:   #37702E
}

.top-bar-section li:not(.has-form) a:not(.button) {
    background-color:   #37702E
}

#full-size-menu li.menu-item .dropdown-wrapper li a:hover {
    color:   #37702E
}

#full-size-menu li.menu-item a:hover, #full-size-menu li.menu.item a:focus, nav.tab-bar a:hover {
    }
#full-size-menu li.menu.item a:active, #full-size-menu li.menu.item a.active {
    background-color: #96BF90;
}

#full-size-menu li.menu-item a.open-mega-menu.active, #full-size-menu li.menu-item div.mega-menu, a.open-mega-menu.active {
    background-color: #96BF90;
    color: #000;
}

#full-size-menu li.menu-item div.mega-menu li, #full-size-menu li.menu-item div.mega-menu a {
    background-color: #96BF90;
    color: #000;
    border-color: #9a9a9a;
}

div.type-section h1 {
    padding-left: 15px !important;
    background-color: #37702E;
    color: #fff;
    font-size: 20px;
    line-height: 26px;
    font-weight: 300;
}

div.type-section h3 {
    margin-left: 15px;
    margin-bottom: 0px;
    font-weight: 300;
}

.journal-tabs .tab-title.active a {
    background-color: #37702E;
    border-color: #37702E;
    color: #fff;
}

</style>
<link rel="stylesheet" href="jquery-ui-1.css">
<link rel="stylesheet" href="article-html.css">
<style>
        .html-profile-nav .top-bar {
            background-color: #37702E;
        }
    </style>
<meta name="title" content="Delineating Individual Trees from Lidar Data: A Comparison of Vector- and Raster-based Segmentation Approaches">
<meta name="description" content="Light detection and ranging (lidar) data is increasingly being used for ecosystem monitoring across geographic scales. This work concentrates on delineating individual trees in topographically-complex, mixed conifer forest across the California’s Sierra Nevada. We delineated individual trees using vector data and a 3D lidar point cloud segmentation algorithm, and using raster data with an object-based image analysis (OBIA) of a canopy height model (CHM). The two approaches are compared to each other and to ground reference data. We used high density (9 pulses/m2), discreet lidar data and WorldView-2 imagery to delineate individual trees, and to classify them by species or species types. We also identified a new method to correct artifacts in a high-resolution CHM. Our main focus was to determine the difference between the two types of approaches and to identify the one that produces more realistic results. We compared the delineations via tree detection, tree heights, and the shape of the generated polygons. The tree height agreement was high between the two approaches and the ground data (r2: 0.93–0.96). Tree detection rates increased for more dominant trees (8–100 percent). The two approaches delineated tree boundaries that differed in shape: the lidar-approach produced fewer, more complex, and larger polygons that more closely resembled real forest structure.">
<link rel="image_src" href="remotesensing-logo.png">
<meta name="dc.title" content="Delineating Individual Trees from Lidar Data: A Comparison of Vector- and Raster-based Segmentation Approaches">
<meta name="dc.creator" content="Marek K. Jakubowski">
<meta name="dc.creator" content="Wenkai Li">
<meta name="dc.creator" content="Qinghua Guo">
<meta name="dc.creator" content="Maggi Kelly">
<meta name="dc.type" content="Article">
<meta name="dc.source" content="Remote Sensing 2013, Vol. 5, Pages 4163-4186">
<meta name="dc.date" content="2013-08-26">
<meta name="dc.identifier" content="10.3390/rs5094163">
<meta name="dc.publisher" content="Multidisciplinary Digital Publishing Institute">
<meta name="dc.rights" content="http://creativecommons.org/licenses/by/3.0/">
<meta name="dc.format" content="application/pdf">
<meta name="dc.language" content="en">
<meta name="dc.description" content="Light detection and ranging (lidar) data is increasingly being used for ecosystem monitoring across geographic scales. This work concentrates on delineating individual trees in topographically-complex, mixed conifer forest across the California’s Sierra Nevada. We delineated individual trees using vector data and a 3D lidar point cloud segmentation algorithm, and using raster data with an object-based image analysis (OBIA) of a canopy height model (CHM). The two approaches are compared to each other and to ground reference data. We used high density (9 pulses/m2), discreet lidar data and WorldView-2 imagery to delineate individual trees, and to classify them by species or species types. We also identified a new method to correct artifacts in a high-resolution CHM. Our main focus was to determine the difference between the two types of approaches and to identify the one that produces more realistic results. We compared the delineations via tree detection, tree heights, and the shape of the generated polygons. The tree height agreement was high between the two approaches and the ground data (r2: 0.93–0.96). Tree detection rates increased for more dominant trees (8–100 percent). The two approaches delineated tree boundaries that differed in shape: the lidar-approach produced fewer, more complex, and larger polygons that more closely resembled real forest structure.">
<meta name="dc.subject" content="lidar">
<meta name="dc.subject" content="individual tree">
<meta name="dc.subject" content="segmentation">
<meta name="dc.subject" content="object based">
<meta name="dc.subject" content="DEM">
<meta name="dc.subject" content="raster">
<meta name="dc.subject" content="vector">
<meta name="prism.publicationName" content="Remote Sensing">
<meta name="prism.publicationDate" content="2013-08-26">
<meta name="prism.volume" content="5">
<meta name="prism.number" content="9">
<meta name="prism.section" content="Article">
<meta name="prism.startingPage" content="4163">
<meta name="prism.endingPage" content="4186">
<meta name="citation_journal_title" content="Remote Sensing">
<meta name="citation_publisher" content="Multidisciplinary Digital Publishing Institute">
<meta name="citation_authors" content="Jakubowski, Marek K.; Li, Wenkai ; Guo, Qinghua ; Kelly, Maggi ">
<meta name="citation_title" content="Delineating Individual Trees from Lidar Data: A Comparison of Vector- and Raster-based Segmentation Approaches">
<meta name="citation_publication_date" content="2013/9">
<meta name="citation_online_date" content="2013/08/26">
<meta name="citation_volume" content="5">
<meta name="citation_issue" content="9">
<meta name="citation_firstpage" content="4163">
<meta name="citation_author" content="Jakubowski, Marek K.">
<meta name="citation_author" content="Li, Wenkai">
<meta name="citation_author" content="Guo, Qinghua">
<meta name="citation_author" content="Kelly, Maggi">
<meta name="citation_lastpage" content="4186">
<meta name="citation_doi" content="10.3390/rs5094163">
<meta name="citation_id" content="mdpi-rs5094163">
<meta name="citation_abstract_html_url" content="https://www.mdpi.com/2072-4292/5/9/4163">
<meta name="citation_pdf_url" content="https://www.mdpi.com/2072-4292/5/9/4163/pdf">
<link rel="alternate" type="application/pdf" title="PDF Full-Text" href="https://www.mdpi.com/2072-4292/5/9/4163/pdf">
<meta name="fulltext_pdf" content="https://www.mdpi.com/2072-4292/5/9/4163/pdf">
<meta name="citation_fulltext_html_url" content="https://www.mdpi.com/2072-4292/5/9/4163/htm">
<link rel="alternate" type="text/html" title="HTML Full-Text" href="https://www.mdpi.com/2072-4292/5/9/4163/htm">
<meta name="fulltext_html" content="https://www.mdpi.com/2072-4292/5/9/4163/htm">
<link rel="alternate" type="text/xml" title="XML Full-Text" href="https://www.mdpi.com/2072-4292/5/9/4163/xml">
<meta name="fulltext_xml" content="https://www.mdpi.com/2072-4292/5/9/4163/xml">
<meta name="twitter:card" content="summary">
<meta name="twitter:site" content="@MDPIOpenAccess">
<meta name="twitter:image" content="https://www.mdpi.com/img/journals/remotesensing-logo-sq.png">
<meta property="fb:app_id" content="131189377574">
<meta property="og:site_name" content="MDPI">
<meta property="og:type" content="article">
<meta property="og:url" content="https://www.mdpi.com/2072-4292/5/9/4163">
<meta property="og:title" content="Delineating Individual Trees from Lidar Data: A Comparison of Vector- and Raster-based Segmentation Approaches">
<meta property="og:description" content="Light detection and ranging (lidar) data is increasingly being used for ecosystem monitoring across geographic scales. This work concentrates on delineating individual trees in topographically-complex, mixed conifer forest across the California’s Sierra Nevada. We delineated individual trees using vector data and a 3D lidar point cloud segmentation algorithm, and using raster data with an object-based image analysis (OBIA) of a canopy height model (CHM). The two approaches are compared to each other and to ground reference data. We used high density (9 pulses/m2), discreet lidar data and WorldView-2 imagery to delineate individual trees, and to classify them by species or species types. We also identified a new method to correct artifacts in a high-resolution CHM. Our main focus was to determine the difference between the two types of approaches and to identify the one that produces more realistic results. We compared the delineations via tree detection, tree heights, and the shape of the generated polygons. The tree height agreement was high between the two approaches and the ground data (r2: 0.93–0.96). Tree detection rates increased for more dominant trees (8–100 percent). The two approaches delineated tree boundaries that differed in shape: the lidar-approach produced fewer, more complex, and larger polygons that more closely resembled real forest structure.">
<meta property="og:image" content="https://www.mdpi.com/img/journals/remotesensing-logo-print.png">
<link rel="alternate" type="application/rss+xml" title="MDPI Publishing - Latest articles" href="https://www.mdpi.com/rss">
<script async="" src="analytics.js"></script><script type="text/x-mathjax-config;executed=true">
    MathJax.Hub.processSectionDelay = 0;
      MathJax.Hub.Config({
        menuSettings: {
          CHTMLpreview: false
        },
        "CHTML-preview":{
          disabled: true
        },
        "HTML-CSS": {
          availableFonts: ["TeX"],
          preferredFonts: "TeX",
          webFont:"TeX",
          imageFont:"TeX",
          undefinedFamily:"'Arial Unicode MS',serif"
        },
        "TeX": {
          extensions: ["noErrors.js"],
          noErrors: {
            inlineDelimiters: ["",""],
            multiLine: true,
            style: {
              "font-size":   "90%",
              "text-align":  "left",
              "color":       "black",
              "padding":     "1px 3px",
              "border":      "1px solid"
            }
          }
        }
      });
    </script>
<script type="text/javascript" src="MathJax.js"></script>
<meta name="google-site-verification" content="PxTlsg7z2S00aHroktQd57fxygEjMiNHydKn3txhvwY">
<!--[if lt IE 9]>
            <script>var browserIe8 = true;</script>
            <link rel="stylesheet" href="https://www.mdpi.com/assets/css/ie8foundationfix.css?bb4865e3906584db">
            <script src="//html5shiv.googlecode.com/svn/trunk/html5.js"></script>
            <script src="//cdnjs.cloudflare.com/ajax/libs/html5shiv/3.6.2/html5shiv.js"></script>
            <script src="//s3.amazonaws.com/nwapi/nwmatcher/nwmatcher-1.2.5-min.js"></script>
            <script src="//html5base.googlecode.com/svn-history/r38/trunk/js/selectivizr-1.0.3b.js"></script>
            <script src="//cdnjs.cloudflare.com/ajax/libs/respond.js/1.1.0/respond.min.js"></script>
            <script src="https://www.mdpi.com/assets/js/ie8/ie8patch.js?9e1d3c689a0471df"></script>
            <script src="https://www.mdpi.com/assets/js/ie8/rem.min.js?94b62787dcd6d2f2"></script>            
                                                        <![endif]-->
<script>
            (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
            (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
            m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
            })(window,document,'script','https://www.google-analytics.com/analytics.js','ga');

            ga('create', 'UA-5824718-2', 'auto');
            ga('require', 'displayfeatures');
            ga('send', 'pageview');
        </script>
<style></style><meta class="foundation-mq-topbar"><style type="text/css">.MathJax_Hover_Frame {border-radius: .25em; -webkit-border-radius: .25em; -moz-border-radius: .25em; -khtml-border-radius: .25em; box-shadow: 0px 0px 15px #83A; -webkit-box-shadow: 0px 0px 15px #83A; -moz-box-shadow: 0px 0px 15px #83A; -khtml-box-shadow: 0px 0px 15px #83A; border: 1px solid #A6D ! important; display: inline-block; position: absolute}
.MathJax_Menu_Button .MathJax_Hover_Arrow {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 4px; -webkit-border-radius: 4px; -moz-border-radius: 4px; -khtml-border-radius: 4px; font-family: 'Courier New',Courier; font-size: 9px; color: #F0F0F0}
.MathJax_Menu_Button .MathJax_Hover_Arrow span {display: block; background-color: #AAA; border: 1px solid; border-radius: 3px; line-height: 0; padding: 4px}
.MathJax_Hover_Arrow:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_Hover_Arrow:hover span {background-color: #CCC!important}
</style><style type="text/css">#MathJax_About {position: fixed; left: 50%; width: auto; text-align: center; border: 3px outset; padding: 1em 2em; background-color: #DDDDDD; color: black; cursor: default; font-family: message-box; font-size: 120%; font-style: normal; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 15px; -webkit-border-radius: 15px; -moz-border-radius: 15px; -khtml-border-radius: 15px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_About.MathJax_MousePost {outline: none}
.MathJax_Menu {position: absolute; background-color: white; color: black; width: auto; padding: 5px 0px; border: 1px solid #CCCCCC; margin: 0; cursor: default; font: menu; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 5px; -webkit-border-radius: 5px; -moz-border-radius: 5px; -khtml-border-radius: 5px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
.MathJax_MenuItem {padding: 1px 2em; background: transparent}
.MathJax_MenuArrow {position: absolute; right: .5em; padding-top: .25em; color: #666666; font-size: .75em}
.MathJax_MenuActive .MathJax_MenuArrow {color: white}
.MathJax_MenuArrow.RTL {left: .5em; right: auto}
.MathJax_MenuCheck {position: absolute; left: .7em}
.MathJax_MenuCheck.RTL {right: .7em; left: auto}
.MathJax_MenuRadioCheck {position: absolute; left: .7em}
.MathJax_MenuRadioCheck.RTL {right: .7em; left: auto}
.MathJax_MenuLabel {padding: 1px 2em 3px 1.33em; font-style: italic}
.MathJax_MenuRule {border-top: 1px solid #DDDDDD; margin: 4px 3px}
.MathJax_MenuDisabled {color: GrayText}
.MathJax_MenuActive {background-color: #606872; color: white}
.MathJax_MenuDisabled:focus, .MathJax_MenuLabel:focus {background-color: #E8E8E8}
.MathJax_ContextMenu:focus {outline: none}
.MathJax_ContextMenu .MathJax_MenuItem:focus {outline: none}
#MathJax_AboutClose {top: .2em; right: .2em}
.MathJax_Menu .MathJax_MenuClose {top: -10px; left: -10px}
.MathJax_MenuClose {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; font-family: 'Courier New',Courier; font-size: 24px; color: #F0F0F0}
.MathJax_MenuClose span {display: block; background-color: #AAA; border: 1.5px solid; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; line-height: 0; padding: 8px 0 6px}
.MathJax_MenuClose:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_MenuClose:hover span {background-color: #CCC!important}
.MathJax_MenuClose:hover:focus {outline: none}
</style><style type="text/css">.MJX_Assistive_MathML {position: absolute!important; top: 0; left: 0; clip: rect(1px, 1px, 1px, 1px); padding: 1px 0 0 0!important; border: 0!important; height: 1px!important; width: 1px!important; overflow: hidden!important; display: block!important; -webkit-touch-callout: none; -webkit-user-select: none; -khtml-user-select: none; -moz-user-select: none; -ms-user-select: none; user-select: none}
.MJX_Assistive_MathML.MJX_Assistive_MathML_Block {width: 100%!important}
</style><style type="text/css">#MathJax_Zoom {position: absolute; background-color: #F0F0F0; overflow: auto; display: block; z-index: 301; padding: .5em; border: 1px solid black; margin: 0; font-weight: normal; font-style: normal; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; -webkit-box-sizing: content-box; -moz-box-sizing: content-box; box-sizing: content-box; box-shadow: 5px 5px 15px #AAAAAA; -webkit-box-shadow: 5px 5px 15px #AAAAAA; -moz-box-shadow: 5px 5px 15px #AAAAAA; -khtml-box-shadow: 5px 5px 15px #AAAAAA; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_ZoomOverlay {position: absolute; left: 0; top: 0; z-index: 300; display: inline-block; width: 100%; height: 100%; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
#MathJax_ZoomFrame {position: relative; display: inline-block; height: 0; width: 0}
#MathJax_ZoomEventTrap {position: absolute; left: 0; top: 0; z-index: 302; display: inline-block; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
</style><style type="text/css">.MathJax_Preview {color: #888}
#MathJax_Message {position: fixed; left: 1px; bottom: 2px; background-color: #E6E6E6; border: 1px solid #959595; margin: 0px; padding: 2px 8px; z-index: 102; color: black; font-size: 80%; width: auto; white-space: nowrap}
#MathJax_MSIE_Frame {position: absolute; top: 0; left: 0; width: 0px; z-index: 101; border: 0px; margin: 0px; padding: 0px}
.MathJax_Error {color: #CC0000; font-style: italic}
</style><style type="text/css">.MJXp-script {font-size: .8em}
.MJXp-right {-webkit-transform-origin: right; -moz-transform-origin: right; -ms-transform-origin: right; -o-transform-origin: right; transform-origin: right}
.MJXp-bold {font-weight: bold}
.MJXp-italic {font-style: italic}
.MJXp-scr {font-family: MathJax_Script,'Times New Roman',Times,STIXGeneral,serif}
.MJXp-frak {font-family: MathJax_Fraktur,'Times New Roman',Times,STIXGeneral,serif}
.MJXp-sf {font-family: MathJax_SansSerif,'Times New Roman',Times,STIXGeneral,serif}
.MJXp-cal {font-family: MathJax_Caligraphic,'Times New Roman',Times,STIXGeneral,serif}
.MJXp-mono {font-family: MathJax_Typewriter,'Times New Roman',Times,STIXGeneral,serif}
.MJXp-largeop {font-size: 150%}
.MJXp-largeop.MJXp-int {vertical-align: -.2em}
.MJXp-math {display: inline-block; line-height: 1.2; text-indent: 0; font-family: 'Times New Roman',Times,STIXGeneral,serif; white-space: nowrap; border-collapse: collapse}
.MJXp-display {display: block; text-align: center; margin: 1em 0}
.MJXp-math span {display: inline-block}
.MJXp-box {display: block!important; text-align: center}
.MJXp-box:after {content: " "}
.MJXp-rule {display: block!important; margin-top: .1em}
.MJXp-char {display: block!important}
.MJXp-mo {margin: 0 .15em}
.MJXp-mfrac {margin: 0 .125em; vertical-align: .25em}
.MJXp-denom {display: inline-table!important; width: 100%}
.MJXp-denom > * {display: table-row!important}
.MJXp-surd {vertical-align: top}
.MJXp-surd > * {display: block!important}
.MJXp-script-box > *  {display: table!important; height: 50%}
.MJXp-script-box > * > * {display: table-cell!important; vertical-align: top}
.MJXp-script-box > *:last-child > * {vertical-align: bottom}
.MJXp-script-box > * > * > * {display: block!important}
.MJXp-mphantom {visibility: hidden}
.MJXp-munderover {display: inline-table!important}
.MJXp-over {display: inline-block!important; text-align: center}
.MJXp-over > * {display: block!important}
.MJXp-munderover > * {display: table-row!important}
.MJXp-mtable {vertical-align: .25em; margin: 0 .125em}
.MJXp-mtable > * {display: inline-table!important; vertical-align: middle}
.MJXp-mtr {display: table-row!important}
.MJXp-mtd {display: table-cell!important; text-align: center; padding: .5em 0 0 .5em}
.MJXp-mtr > .MJXp-mtd:first-child {padding-left: 0}
.MJXp-mtr:first-child > .MJXp-mtd {padding-top: 0}
.MJXp-mlabeledtr {display: table-row!important}
.MJXp-mlabeledtr > .MJXp-mtd:first-child {padding-left: 0}
.MJXp-mlabeledtr:first-child > .MJXp-mtd {padding-top: 0}
.MJXp-merror {background-color: #FFFF88; color: #CC0000; border: 1px solid #CC0000; padding: 1px 3px; font-style: normal; font-size: 90%}
.MJXp-scale0 {-webkit-transform: scaleX(.0); -moz-transform: scaleX(.0); -ms-transform: scaleX(.0); -o-transform: scaleX(.0); transform: scaleX(.0)}
.MJXp-scale1 {-webkit-transform: scaleX(.1); -moz-transform: scaleX(.1); -ms-transform: scaleX(.1); -o-transform: scaleX(.1); transform: scaleX(.1)}
.MJXp-scale2 {-webkit-transform: scaleX(.2); -moz-transform: scaleX(.2); -ms-transform: scaleX(.2); -o-transform: scaleX(.2); transform: scaleX(.2)}
.MJXp-scale3 {-webkit-transform: scaleX(.3); -moz-transform: scaleX(.3); -ms-transform: scaleX(.3); -o-transform: scaleX(.3); transform: scaleX(.3)}
.MJXp-scale4 {-webkit-transform: scaleX(.4); -moz-transform: scaleX(.4); -ms-transform: scaleX(.4); -o-transform: scaleX(.4); transform: scaleX(.4)}
.MJXp-scale5 {-webkit-transform: scaleX(.5); -moz-transform: scaleX(.5); -ms-transform: scaleX(.5); -o-transform: scaleX(.5); transform: scaleX(.5)}
.MJXp-scale6 {-webkit-transform: scaleX(.6); -moz-transform: scaleX(.6); -ms-transform: scaleX(.6); -o-transform: scaleX(.6); transform: scaleX(.6)}
.MJXp-scale7 {-webkit-transform: scaleX(.7); -moz-transform: scaleX(.7); -ms-transform: scaleX(.7); -o-transform: scaleX(.7); transform: scaleX(.7)}
.MJXp-scale8 {-webkit-transform: scaleX(.8); -moz-transform: scaleX(.8); -ms-transform: scaleX(.8); -o-transform: scaleX(.8); transform: scaleX(.8)}
.MJXp-scale9 {-webkit-transform: scaleX(.9); -moz-transform: scaleX(.9); -ms-transform: scaleX(.9); -o-transform: scaleX(.9); transform: scaleX(.9)}
.MathJax_PHTML .noError {vertical-align: ; font-size: 90%; text-align: left; color: black; padding: 1px 3px; border: 1px solid}
</style><script type="text/javascript" src="https://www.mdpi.com/bundles/mathjax/jax/output/HTML-CSS/jax.js?V=2.7.1"></script></head>
<body><div id="MathJax_Message" style="">Loading [MathJax]/jax/output/HTML-CSS/jax.js</div>
<div class="direction direction_right" id="small_right" style="border-right-width: 0px; padding: 0px;"><i class="fa fa-caret-right fa-2x" style="float: right; margin: 26px 5px 0px 0px; font-size: 24px;"></i></div>
<div class="big_direction direction_right" id="big_right" style="border-right-width: 0px;  padding: 0 0 0 1em;">
<i class="fa fa-caret-right fa-2x" style="margin: 26px 5px 0px 0px; float:right; font-size: 24px;"></i>
<div style="text-align: right; position:relative; margin: 0 1.7em 0 0">
Next Article in Journal<br>
<div><a href="https://www.mdpi.com/2072-4292/5/9/4187">Assessing the Sensitivity of the OMI-NO<sub>2</sub> Product to Emission Changes across Europe</a></div>
</div>
</div>
<div class="direction" id="small_left" style="border-left-width: 0px;"><i class="fa fa-caret-left fa-2x" style="margin: 26px 4px; font-size: 24px;"></i></div>
<div class="big_direction" id="big_left" style="border-left-width: 0px;">
<i class="fa fa-caret-left fa-2x" style="float: left; margin: 26px 4px; font-size: 24px;"></i>
<div>
Previous Article in Journal<br>
<div><a href="https://www.mdpi.com/2072-4292/5/9/4145">River Discharge Estimation by Using Altimetry Data and Simplified Flood Routing Modeling</a></div>
</div>
</div>
<div style="clear: both;">
<div id="captchaModal" class="reveal-modal" data-reveal="" aria-label="Captcha" aria-hidden="true" role="dialog"></div>
<div id="layoutTypeModal" class="reveal-modal" data-reveal="" aria-labelledby="modalTitle" aria-hidden="true" role="dialog">
<h2 id="modalTitle">Choose your preferred view mode</h2>
<div class="row">
<div class="large-12 medium-12 small-12 columns">
Please select whether you prefer to view the MDPI pages with a view tailored for <strong>mobile</strong> displays or to view the MDPI
pages in the normal scrollable <strong>desktop</strong> version. This selection will be stored into your cookies and used automatically
in next visits. You can also change the view style at any point from the main header when using the pages with your
mobile device.
</div>
<div class="large-12 medium-12 small-12 columns">
<a class="button-new display-type" href="https://www.mdpi.com/toggle_desktop_layout_cookie/mobile">
Use <strong>mobile</strong> version
</a>
<a class="button-new display-type" href="https://www.mdpi.com/toggle_desktop_layout_cookie/desktop">
Use <strong>desktop</strong> version
</a>
</div>
</div>
</div>
<div id="container">
<noscript>
                    <div id="no-javascript">
                        You seem to have javascript disabled. Please note that many of the page functionalities won't work as expected without javascript enabled.
                    </div>
                </noscript>
<div class="off-canvas-wrap" data-offcanvas="">
<div class="inner-wrap">
<div class="show-for-medium-down" style="background-color: #007F7F; margin: 0 -10px; text-align: center; padding: 8px;">
<a class="hide-show-desktop-option" style="color: #fff; position: absolute; right: 0; font-size: 16px; padding: 0 10px;" href="https://www.mdpi.com/toggle_hide_desktop_layout_option_cookie">X</a>
<a style="color: #fff; display: inline-block; text-align: center;  font-size: 16px;" href="https://www.mdpi.com/toggle_desktop_layout_cookie">switch to <strong>desktop version</strong></a>
</div>
<nav class="tab-bar show-for-medium-down">
<div class="row full-width collapse">
<div class="medium-3 small-4 columns">
<a title="MDPI main page" class="left-off-canvas-toggle" href="#">
<div class="show-for-medium">Menu</div>
<div class="show-for-small"><i class="fa fa-bars"></i></div>
</a>
<a title="Toggle search" class="search-toggle" onclick='$("#search-inputs").toggleClass("active"); return false;' href="#">
<div class="show-for-medium">Search</div>
<div class="show-for-small"><i class="fa fa-search"></i></div>
</a>
<a href="https://www.mdpi.com/toggle_desktop_layout_cookie" title="Switch between mobile and desktop views">
<div class="show-for-small"><i class="fa fa-expand"></i></div>
</a>
</div>
<div class="medium-6 small-4 columns centered">
<div class="show-for-medium">
MDPI — Remote Sensing
</div>
<div class="show-for-small">
<a href="https://www.mdpi.com/journal/remotesensing">
Remote Sensing
</a>
</div>
</div>
<div class="medium-3 small-4 columns right-aligned">
<a href="https://susy.mdpi.com/user/login">Log in</a>
</div>
</div>
</nav>
<aside class="left-off-canvas-menu">
<ul class="off-canvas-list">
<li>
<a href="https://www.mdpi.com/">MDPI</a>
</li>
<li>
<a href="https://www.mdpi.com/about/journals">Journals A–Z</a>
</li>
<li>
<a href="#">Information &amp; Guidelines</a>
<ul>
<li>
<a href="https://www.mdpi.com/authors">For Authors</a>
</li>
<li>
<a href="https://www.mdpi.com/reviewers">For Reviewers</a>
</li>
<li>
<a href="https://www.mdpi.com/editors">For Editors</a>
</li>
<li>
<a href="https://www.mdpi.com/librarians">For Librarians</a>
</li>
<li>
<a href="https://www.mdpi.com/publishing_services">For Publishers</a>
</li>
<li>
<a href="https://www.mdpi.com/societies">For Societies</a>
</li>
<li>
<a href="https://www.mdpi.com/about/apc">Article Processing Charges</a>
</li>
<li>
<a href="https://www.mdpi.com/about/openaccess">Open Access Policy</a>
</li>
<li>
<a href="https://www.mdpi.com/about/ioap">Institutional Open Access Program</a>
</li>
<li>
<a href="https://www.mdpi.com/editorial_process">Editorial Process</a>
</li>
<li>
<a href="https://www.mdpi.com/authors/english">English Editing Service</a>
</li>
<li>
<a href="http://blog.mdpi.com/" onclick="ga('send', 'pageview', 'homeBlog');" target="_blank">MDPI Blog</a>
</li>
</ul>
</li>
<li>
<a href="#">Initiatives</a>
<ul>
<li>
<a href="https://sciforum.net/" target="_blank">Sciforum</a>
</li>
<li>
<a href="https://www.preprints.org/" target="_blank">Preprints</a>
</li>
<li>
<a href="https://www.scilit.net/" target="_blank">Scilit</a>
</li>
<li>
<a href="https://www.mdpi.com/books" target="_blank">MDPI Books</a>
</li>
<li>
<a href="https://encyclopedia.pub/" target="_blank">Encyclopedia</a>
</li>
</ul>
</li>
<li>
<a href="https://www.mdpi.com/about">About</a>
</li>
<li>
<a href="https://susy.mdpi.com/user/login">Login</a>
</li>
<li>
<a href="https://www.mdpi.com/user/register">Register</a>
</li>
<li>
<a class="js-journal-active-only-link js-journal-active-only-submit-link" href="https://susy.mdpi.com/user/manuscripts/upload?journal=remotesensing" onclick="ga('send', 'event', 'Submit Paper', 'Send', 'Remote Sensing');" data-disabledmessage="new submissions are not possible.">Submit</a>
</li>
<li>
<a href="https://www.mdpi.com/toggle_desktop_layout_cookie">Switch to Desktop Version</a>
</li>
<li>
<a href="#" onclick="$('.off-canvas-wrap').foundation('offcanvas', 'toggle', 'move-right'); return false;">Close</a>
</li>
</ul>
</aside>
<section class="main-section">
<header>
<div id="full-size-menu" class="show-for-large-up">
<nav class="top-bar" data-topbar="">
<ul class="title-area">
<li class="name"></li>
<li class="toggle-topbar menu-icon"><a href="#">Menu</a></li>
</ul>


<section class="top-bar-section">
<ul class="full-width">
<li class="menu-item">
<a href="https://www.mdpi.com/">MDPI</a>
</li>
<li class="menu-item">
<a href="https://www.mdpi.com/about/journals">Journals A–Z</a>
</li>
<li class="menu-item has-dropdown not-click">
<a href="#">Information &amp; Guidelines</a>
<ul class="dropdown dropdown-wrapper"><li class="title back js-generated"><h5><a href="javascript:void(0)">Back</a></h5></li><li class="parent-link hide-for-medium-up"><a class="parent-link js-generated" href="#">Information &amp; Guidelines</a></li>
<li>
<div class="row">
<div class="small-4 columns right-border">
<ul>
<li>
<a href="https://www.mdpi.com/authors">For Authors</a>
</li>
<li>
<a href="https://www.mdpi.com/reviewers">For Reviewers</a>
</li>
<li>
<a href="https://www.mdpi.com/editors">For Editors</a>
</li>
<li>
<a href="https://www.mdpi.com/librarians">For Librarians</a>
</li>
<li>
<a href="https://www.mdpi.com/publishing_services">For Publishers</a>
</li>
<li>
 <a href="https://www.mdpi.com/societies">For Societies</a>
</li>
</ul>
</div>
<div class="small-8 columns">
<ul>
<li>
<a href="https://www.mdpi.com/about/apc">Article Processing Charges</a>
</li>
<li>
<a href="https://www.mdpi.com/about/openaccess">Open Access Policy</a>
</li>
<li>
<a href="https://www.mdpi.com/about/ioap">Institutional Open Access Program</a>
</li>
<li>
<a href="https://www.mdpi.com/editorial_process">Editorial Process</a>
</li>
<li>
<a href="https://www.mdpi.com/authors/english">English Editing Service</a>
</li>
<li>
<a href="http://blog.mdpi.com/" onclick="ga('send', 'pageview', 'homeBlog');" target="_blank">MDPI Blog</a>
</li>
</ul>
</div>
</div>
</li>
</ul>
</li>
<li class="menu-item has-dropdown not-click">
<a href="#">Initiatives</a>
<ul class="dropdown dropdown-wrapper dropdown-icons"><li class="title back js-generated"><h5><a href="javascript:void(0)">Back</a></h5></li><li class="parent-link hide-for-medium-up"><a class="parent-link js-generated" href="#">Initiatives</a></li>
<li>
<div class="row">
<div class="small-12 columns">
<ul>
<li>
<a href="https://sciforum.net/" target="_blank">
<img src="logo-sciforum.png" alt="">
Sciforum
</a>
</li>
<li>
<a href="https://www.preprints.org/" target="_blank">
<img src="logo-preprints.png" alt="">
Preprints
</a>
</li>
<li>
<a href="https://www.scilit.net/" target="_blank">
<img src="logo-scilit.png" alt="">
Scilit
</a>
</li>
<li>
<a href="https://www.mdpi.com/books" target="_blank">
<img src="logo-books.png" alt="">
MDPI Books
</a>
</li>
<li>
<a href="https://encyclopedia.pub/" target="_blank">
<img src="logo-encyclopedia.png" alt="">
Encyclopedia
</a>
</li>
</ul>
</div>
</div>
</li>
</ul>
</li>
<li class="menu-item">
<a href="https://www.mdpi.com/about">About</a>
</li>
</ul>
</section><section class="top-bar-section">
<ul class="right">
<li class="menu-item">
<a href="https://susy.mdpi.com/user/login">Login</a>
</li>
<li class="menu-item">
<a href="https://www.mdpi.com/user/register">Register</a>
</li>
<li class="menu-item">
<a class="js-journal-active-only-link js-journal-active-only-submit-link" href="https://susy.mdpi.com/user/manuscripts/upload?journal=remotesensing" onclick="ga('send', 'event', 'Submit Paper', 'Send', 'Remote Sensing');" data-disabledmessage="new submissions are not possible.">Submit</a>
</li>
</ul>
</section></nav>
</div>
<div class="row">
</div>
</header>
<div id="search-container">
<div class="row full-width" id="searchBar">
<div class="large-20 columns show-for-large-up" style="padding: 0px; padding-left: 10px; line-height: 70px;">
<a href="https://www.mdpi.com/journal/remotesensing">
<img src="remotesensing-logo.png" alt="remotesensing-logo" title="remotesensing Open Access Journals" style="padding: 1px 0 12px 5px;">
</a>
</div>
<form style="padding: 0 10px; background-color: inherit;" class="large-60 medium-12 columns " action="/search" method="get">
<div id="search-inputs" class="row full-width hide-medium-down-initially">
<div class="row" style="margin: 0 5px;">
<div class="large-9 medium-9 small-12 columns">
<div class="row">
<div class="large-6 medium-6 small-12 columns" style="margin-top: 0px; padding-bottom: 0px;">
<div style="height: 14px;" class="show-for-medium-up"></div>
<div class="search-row">
<span class="lb">
Title / Keyword
</span>
<span class="field">
<input id="q" tabindex="1" name="q" type="text">
</span>
</div>
<div class="search-row">
<span class="lb">
Author / Affiliation
</span>
<span class="field">
<input id="authors" tabindex="2" name="authors" type="text">
</span>
</div>
<div style="height: 12px;" class="show-for-medium-up"></div>
</div>
<div class="large-6 medium-6 small-12 columns" style="margin-top: 0px; padding-bottom: 0px;">
<div style="height: 14px;" class="show-for-medium-up"></div>
<div class="search-row">
<span class="lb">
Journal
</span>
<span class="field">
<select id="journal" tabindex="-1" name="journal" class="chosen-select" style="display: none;">
<option value="">all</option>
<option value="acoustics">Acoustics</option>
<option value="actuators">Actuators</option>
<option value="admsci">Administrative Sciences</option>
<option value="aerospace">Aerospace</option>
<option value="agriculture">Agriculture</option>
<option value="agriengineering">AgriEngineering</option>
<option value="agronomy">Agronomy</option>
<option value="algorithms">Algorithms</option>
<option value="animals">Animals</option>
<option value="antibiotics">Antibiotics</option>
<option value="antibodies">Antibodies</option>
<option value="antioxidants">Antioxidants</option>
<option value="applsci">Applied Sciences</option>
<option value="asi">ASI</option>
<option value="arts">Arts</option>
<option value="atmosphere">Atmosphere</option>
<option value="atoms">Atoms</option>
<option value="axioms">Axioms</option>
<option value="batteries">Batteries</option>
<option value="behavsci">Behavioral Sciences</option>
<option value="beverages">Beverages</option>
<option value="BDCC">BDCC</option>
<option value="bioengineering">Bioengineering</option>
<option value="biology">Biology</option>
<option value="biomedicines">Biomedicines</option>
<option value="biomimetics">Biomimetics</option>
<option value="biomolecules">Biomolecules</option>
<option value="biosensors">Biosensors</option>
<option value="brainsci">Brain Sciences</option>
<option value="buildings">Buildings</option>
<option value="carbon">C</option>
<option value="cancers">Cancers</option>
<option value="catalysts">Catalysts</option>
<option value="cells">Cells</option>
<option value="ceramics">Ceramics</option>
<option value="challenges">Challenges</option>
<option value="ChemEngineering">ChemEngineering</option>
<option value="chemistry">Chemistry</option>
<option value="chemosensors">Chemosensors</option>
<option value="children">Children</option>
<option value="climate">Climate</option>
<option value="clockssleep">Clocks &amp; Sleep</option>
<option value="coatings">Coatings</option>
<option value="colloids">Colloids and Interfaces</option>
<option value="computation">Computation</option>
<option value="computers">Computers</option>
<option value="condensedmatter">Condensed Matter</option>
<option value="cosmetics">Cosmetics</option>
<option value="cryptography">Cryptography</option>
<option value="crystals">Crystals</option>
<option value="data">Data</option>
<option value="dentistry">Dentistry Journal</option>
<option value="designs">Designs</option>
<option value="diagnostics">Diagnostics</option>
<option value="diseases">Diseases</option>
<option value="diversity">Diversity</option>
<option value="drones">Drones</option>
<option value="econometrics">Econometrics</option>
<option value="economies">Economies</option>
<option value="education">Education Sciences</option>
<option value="electronics">Electronics</option>
<option value="energies">Energies</option>
<option value="entropy">Entropy</option>
<option value="environments">Environments</option>
<option value="epigenomes">Epigenomes</option>
<option value="fermentation">Fermentation</option>
<option value="fibers">Fibers</option>
<option value="fire">Fire</option>
<option value="fishes">Fishes</option>
<option value="fluids">Fluids</option>
<option value="foods">Foods</option>
<option value="forecasting">Forecasting</option>
<option value="forests">Forests</option>
<option value="fractalfract">Fractal Fract</option>
<option value="futureinternet">Future Internet</option>
<option value="galaxies">Galaxies</option>
<option value="games">Games</option>
<option value="gastrointestdisord">GastrointestDisord</option>
<option value="gels">Gels</option>
<option value="genealogy">Genealogy</option>
<option value="genes">Genes</option>
<option value="geosciences">Geosciences</option>
<option value="geriatrics">Geriatrics</option>
<option value="healthcare">Healthcare</option>
<option value="heritage">Heritage</option>
<option value="high-throughput">High-Throughput</option>
<option value="horticulturae">Horticulturae</option>
<option value="humanities">Humanities</option>
<option value="hydrology">Hydrology</option>
<option value="informatics">Informatics</option>
<option value="information">Information</option>
<option value="infrastructures">Infrastructures</option>
<option value="inorganics">Inorganics</option>
<option value="insects">Insects</option>
<option value="instruments">Instruments</option>
<option value="ijerph">IJERPH</option>
<option value="ijfs">IJFS</option>
<option value="ijms">IJMS</option>
<option value="IJNS">IJNS</option>
<option value="ijtpp">IJTPP</option>
<option value="inventions">Inventions</option>
<option value="ijgi">IJGI</option>
<option value="J">J</option>
<option value="jcdd">JCDD</option>
<option value="jcm">JCM</option>
<option value="jcs">J. Compos. Sci.</option>
<option value="jdb">JDB</option>
<option value="jfb">JFB</option>
<option value="jfmk">JFMK</option>
<option value="jof">JoF</option>
<option value="jimaging">J. Imaging</option>
<option value="jintelligence">J. Intell.</option>
<option value="jlpea">JLPEA</option>
<option value="jmmp">JMMP</option>
<option value="jmse">JMSE</option>
<option value="JOItmC">JOItmC</option>
<option value="jpm">JPM</option>
<option value="jrfm">JRFM</option>
<option value="jsan">JSAN</option>
<option value="land">Land</option>
<option value="languages">Languages</option>
<option value="laws">Laws</option>
<option value="life">Life</option>
<option value="logistics">Logistics</option>
<option value="lubricants">Lubricants</option>
<option value="make">MAKE</option>
<option value="machines">Machines</option>
<option value="magnetochemistry">Magnetochemistry</option>
<option value="marinedrugs">Marine Drugs</option>
<option value="materials">Materials</option>
<option value="mca">MCA</option>
<option value="mathematics">Mathematics</option>
<option value="medsci">Medical Sciences</option>
<option value="medicina">Medicina</option>
<option value="medicines">Medicines</option>
<option value="membranes">Membranes</option>
<option value="metabolites">Metabolites</option>
<option value="metals">Metals</option>
<option value="mps">MPs</option>
<option value="micromachines">Micromachines</option>
<option value="microorganisms">Microorganisms</option>
<option value="minerals">Minerals</option>
<option value="molbank">Molbank</option>
<option value="molecules">Molecules</option>
<option value="mti">MTI</option>
<option value="nanomaterials">Nanomaterials</option>
<option value="neuroglia">Neuroglia</option>
<option value="ncrna">ncRNA</option>
<option value="nutrients">Nutrients</option>
<option value="particles">Particles</option>
<option value="pathogens">Pathogens</option>
<option value="pharmaceuticals">Pharmaceuticals</option>
<option value="pharmaceutics">Pharmaceutics</option>
<option value="pharmacy">Pharmacy</option>
<option value="philosophies">Philosophies</option>
<option value="photonics">Photonics</option>
<option value="physics">Physics</option>
<option value="plants">Plants</option>
<option value="plasma">Plasma</option>
<option value="polymers">Polymers</option>
<option value="proceedings">Proceedings</option>
<option value="processes">Processes</option>
<option value="proteomes">Proteomes</option>
<option value="psych">Psych</option>
<option value="publications">Publications</option>
<option value="qubs">QuBS</option>
<option value="quantumrep">Quantum Reports</option>
<option value="quaternary">Quaternary</option>
<option value="recycling">Recycling</option>
<option value="religions">Religions</option>
<option value="remotesensing" selected="selected">Remote Sensing</option>
<option value="reports">Reports</option>
<option value="resources">Resources</option>
<option value="risks">Risks</option>
<option value="robotics">Robotics</option>
<option value="safety">Safety</option>
<option value="sci">Sci</option>
<option value="scipharm">Sci. Pharm.</option>
<option value="sensors">Sensors</option>
<option value="separations">Separations</option>
<option value="smartcities">Smart Cities</option>
<option value="socsci">Social Sciences</option>
<option value="societies">Societies</option>
<option value="soilsystems">Soil Syst.</option>
<option value="sports">Sports</option>
<option value="stats">Stats</option>
<option value="surfaces">Surfaces</option>
<option value="sustainability">Sustainability</option>
<option value="symmetry">Symmetry</option>
<option value="systems">Systems</option>
<option value="technologies">Technologies</option>
<option value="toxics">Toxics</option>
<option value="toxins">Toxins</option>
<option value="tropicalmed">TropicalMed</option>
<option value="universe">Universe</option>
<option value="urbansci">Urban Science</option>
<option value="vaccines">Vaccines</option>
<option value="vehicles">Vehicles</option>
<option value="vetsci">Veterinary Sciences</option>
<option value="vibration">Vibration</option>
<option value="viruses">Viruses</option>
<option value="vision">Vision</option>
<option value="water">Water</option>
<option value="wevj">WEVJ</option>
</select><div class="chosen-container chosen-container-single" title="" id="journal_chosen" style="width: 100%;"><a class="chosen-single">
  <span>Remote Sensing</span>
  <div><b></b></div>
</a>
<div class="chosen-drop">
  <div class="chosen-search">
    <input class="chosen-search-input" autocomplete="off" tabindex="4" type="text">
  </div>
  <ul class="chosen-results"></ul>
</div></div>
</span>
</div>
<div class="search-row">
<span class="lb">
Article Type
</span>
<span class="field">
<select id="article_type" tabindex="-1" name="article_type" class="chosen-select" style="display: none;">
<option value="" selected="selected">all</option>
<option value="addendum">Addendum</option>
<option value="research-article">Article</option>
<option value="book-review">Book Review</option>
<option value="case-report">Case Report</option>
<option value="article-commentary">Comment</option>
<option value="commentary">Commentary</option>
<option value="rapid-communication">Communication</option>
<option value="concept-paper">Concept Paper</option>
<option value="conference-report">Conference Report</option>
<option value="correction">Correction</option>
<option value="creative">Creative</option>
<option value="data-descriptor">Data Descriptor</option>
<option value="discussion">Discussion</option>
<option value="editorial">Editorial</option>
<option value="erratum">Erratum</option>
<option value="essay">Essay</option>
<option value="expression-of-concern">Expression of Concern</option>
<option value="interesting-image">Interesting Images</option>
<option value="letter">Letter</option>
<option value="meeting-report">Meeting Report</option>
<option value="books-received">New Book Received</option>
<option value="obituary">Obituary</option>
<option value="opinion">Opinion</option>
<option value="project-report">Project Report</option>
<option value="reply">Reply</option>
<option value="retraction">Retraction</option>
<option value="review-article">Review</option>
<option value="note">Short Note</option>
<option value="technical-note">Technical Note</option>
<option value="tutorial">Tutorial</option>
</select><div class="chosen-container chosen-container-single" title="" id="article_type_chosen" style="width: 100%;"><a class="chosen-single">
  <span>all</span>
  <div><b></b></div>
</a>
<div class="chosen-drop">
  <div class="chosen-search">
    <input class="chosen-search-input" autocomplete="off" tabindex="3" type="text">
  </div>
  <ul class="chosen-results"></ul>
</div></div>
</span>
</div>
</div>
</div>
</div>
<div class="large-3 medium-3 small-12 columns">
<div class="row">
<div class="large-6 medium-6 small-12 columns" style="margin-top: 0px; padding-bottom: 0px;">
<div style="height: 14px;" class="show-for-medium-up"></div>
<div class="search-row">
<input class="show-for-medium-up" style="visibility: hidden;" type="text">
</div>
<div class="search-row show-for-medium-up">
<a href="https://www.mdpi.com/search?advanced&amp;journal=remotesensing" style="display: block; text-align: center; font-size: 12px; color: #555; height: 24px; line-height: 24px;">Advanced</a>
</div>
</div>
<div class="large-6 medium-6 small-12 columns" style="margin-top: 0px; padding-bottom: 0px;">
<div style="height: 14px;" class="show-for-medium-up"></div>
<div class="search-row show-for-medium-up">
<input class="show-for-medium-up" style="visibility: hidden;" type="text">
</div>
<div class="search-row">
<span class="lb">
<a class="main-search-advanced show-for-small-down" href="https://www.mdpi.com/search?advanced&amp;journal=remotesensing">Advanced</a>
</span>
<span class="field small">
<input id="search" value="Search" class="searchButton full-width" tabindex="12" name="search" type="submit">
</span>
</div>
</div>
</div>
</div>
</div>
</div>
</form>
<div class="large-20 columns show-for-large-up" style="margin-top: 0px; padding-bottom: 0px; text-align: right; position: relative;">
<style>
        .iflogo1 {
            float: right;
            /*width: 33%;*/
            margin-left: -2%;
            position: relative;
            margin-top: 4px;
        }
        .iflogo1-if{
            border-radius: 50%;background-color: #FAC902;
        }
        .iflogo2 {
            position: relative;
            width: 75px;
            /*height: auto !important;*/
            height: 7em; !important;
            max-height: 77px !important;
        }
        .iflogo2-if {
            width: 100%;
            height: 5.8vmax;
            max-height: 75px;
            border-radius: 50%;
            text-align: center;
            padding: 19% 0 0;
            font-weight: 700;
            /*line-height: 1.1;*/ line-height: 0.8vmax;
            font-family: Georgia,"Times New Roman",serif;
            letter-spacing: 0.05vmax;
            color: #404040;
        }
        .iflogo1-text{
            font-size: 0.6vmax;
            font-size: 79%;
            line-height: 0.8vmax;
            line-height: 90%;
            color: #404040;
        }
        .iflogo1-number{
            line-height: 0.8vmax;
            font-size: 130%;
            line-height: 13pt;
            font-size: 100%;
            color: #404040;
        }

        .header-impact-factor-sci{
            margin-left: 2px !important;
        }
        .text1 {
            /*float: left;
            padding-left: 11px;*/
            font-family: "Helvetica Neue", Helvetica, Arial, sans-serif;
            /* font-style: italic;*/
            font-size: 7pt;
            line-height: 14pt;
            letter-spacing: 0px;
        }
        .text2 {
            font-family: "Helvetica Neue", Helvetica, Arial, sans-serif;
            font-weight: bold;
            line-height: 8pt;
            font-size: 12pt;
            letter-spacing: 0px;
        }
        .text2-compendex {
            font-family: "Helvetica Neue", Helvetica, Arial, sans-serif;
            font-weight: bold;
            line-height: 8pt;
            font-size: 7.5pt;
            letter-spacing: -0.5px;
        }
    </style>
<div class="header-impact-factor">
<img id="ie8_if_background" src="if_background.png" alt="impact-factor" title="impact factor" style="display: none; position: absolute; right: 10px; z-index: 0; height: 75px; max-height: 75px;">
<div class="header-impact-factor-sci">
<a href="https://www.mdpi.com/journal/remotesensing/stats" style="color: inherit; text-decoration: none;">
<span class="text">IMPACT<br>FACTOR</span><br>
<span class="number">3.406</span>
</a>
</div></div>
</div>
</div>
</div>
<div id="content">
<div class="row full-width">
<div id="left-column" class="dense large-20 medium-3 small-12 columns show-for-large-up">
<div class="top-border">
<div class="inverted" style="margin-bottom: 10px; margin-top: -10px; text-align: center;">
<h2>
<a style="color: #fff; font-size: 15px;" href="https://www.mdpi.com/2072-4292/5/9">
Volume 5, Issue 9 </a>
</h2>
</div>
<a href="#" class="bolded-text link-article-menu show-for-small" style="margin-top: 20px;">
<span class="closed">►</span>
<span class="open" style="display: none;">▼</span>
Article Menu
</a>
<div class="hide-small-down-initially">
<div id="counts-wrapper" class="row generic-item no-border">
<div id="js-counts-wrapper__views" class="small-6 hide columns count-div-container">
<a href="#stats_id"><div class="count-div margin-right">
<span class="name">Views</span>
<span class="count view-number"></span>
</div></a>
</div>
<div id="js-counts-wrapper__downloads" class="small-6 hide columns count-div-container">
<a href="#stats_id"><div class="count-div margin-left">
<span class="name">Downloads</span>
<span class="count download-number"></span>
</div></a>
</div>
<div id="js-counts-wrapper__citations" class="small-6 columns hide count-div-container">
<a href="#framed_div_cited_count"><div style="height: 67px;" class="count-div margin-right">
<span class="name citations-text">No citations found yet</span>
<span class="count citations-number" style="display:none;">0</span>
</div></a>
</div>
<div id="js-counts-wrapper__altmetrics" class="small-6 columns hide count-div-container">
<div class="count-div margin-left">
<span class="name default-alt">Metrics</span>
<span class="count default-alt">
<span>0</span>
</span>
<div data-badge-type="donut" class="altmetric-embed" data-doi="10.3390/rs5094163"></div>
</div>
</div>
</div>
<div class="generic-item no-border">
<h2>Article Versions</h2>
<ul class="side-menu-ul">
<li class="side-menu-li">
<a href="https://www.mdpi.com/2072-4292/5/9/4163">Abstract</a>
</li>
<li class="side-menu-li">
<a href="https://www.mdpi.com/2072-4292/5/9/4163/pdf" onclick="ga('send', 'pageview', $(this).attr('href'));" id="pdf_link">Full-Text PDF</a><span style="font-size: 8pt"> [3993 KB]</span>
</li>
<li class="side-menu-li">
<a href="https://www.mdpi.com/2072-4292/5/9/4163/htm" onclick="ga('send', 'pageview', $(this).attr('href'));" id="html_link">Full-Text HTML</a>
</li>
<li class="side-menu-li">
<a id="js-xml-access-captcha" href="#" data-target="/2072-4292/5/9/4163/xml" class="accessCaptcha" onclick="ga('send', 'pageview', '/2072-4292/5/9/4163/xml');">Full-Text XML</a>
</li>
<li class="side-menu-li">
<a href="https://www.mdpi.com/2072-4292/5/9/4163/epub" onclick="ga('send', 'pageview', '/2072-4292/5/9/4163/epub');" id="epub_link">Full-Text Epub</a>
</li>
<li class="side-menu-li">
<a href="https://www.mdpi.com/2072-4292/5/9/4163/notes" onclick="ga('send', 'pageview', '/2072-4292/5/9/4163/notes');">Article Versions Notes</a>
</li>
</ul>
</div>
<div class="generic-item no-border">
<h2>Related Info</h2>
<ul class="side-menu-ul">
<li class="side-menu-li">
<a href="https://scholar.google.com/scholar?q=Delineating%20Individual%20Trees%20from%20Lidar%20Data:%20A%20Comparison%20of%20Vector-%20and%20Raster-based%20Segmentation%20Approaches" target="_blank">Google Scholar</a>
</li>
<li class="side-menu-li">
<a href="https://www.mdpi.com/2072-4292/5/9/4163/reprints">Order Reprints</a>
</li>
</ul>
</div>
<div class="generic-item no-border">
<h2>More by Authors</h2>
<ul class="side-menu-ul">
<li class="side-menu-li">
<a class="expand" onclick='$(this).closest("li").next("div").toggle(); return false;'>on DOAJ</a>
</li>
<div id="AuthorDOAJExpand" style="display:none;">
<ul class="submenu">
<li>
<a href="http://doaj.org/search?source=%7B%22query%22:%7B%22query_string%22:%7B%22query%22:%22Marek%20K.%20Jakubowski%22,%22default_operator%22:%22AND%22%7D%7D%7D" target="_blank">Jakubowski, MK.</a>
</li>
<li>
<a href="http://doaj.org/search?source=%7B%22query%22:%7B%22query_string%22:%7B%22query%22:%22Wenkai%20Li%22,%22default_operator%22:%22AND%22%7D%7D%7D" target="_blank">Li, W</a>
</li>
<li>
<a href="http://doaj.org/search?source=%7B%22query%22:%7B%22query_string%22:%7B%22query%22:%22Qinghua%20Guo%22,%22default_operator%22:%22AND%22%7D%7D%7D" target="_blank">Guo, Q</a>
</li>
<li>
<a href="http://doaj.org/search?source=%7B%22query%22:%7B%22query_string%22:%7B%22query%22:%22Maggi%20Kelly%22,%22default_operator%22:%22AND%22%7D%7D%7D" target="_blank">Kelly, M</a>
</li>
</ul>
</div>
<li class="side-menu-li">
<a class="expand" onclick='$(this).closest("li").next("div").toggle(); return false;'>on Google Scholar</a>
</li>
<div id="AuthorGoogleExpand" style="display:none;">
<ul class="submenu">
<li>
<a href="https://scholar.google.com/scholar?q=Marek%20K.%20Jakubowski" target="_blank">Jakubowski, MK.</a>
</li>
<li>
<a href="https://scholar.google.com/scholar?q=Wenkai%20Li" target="_blank">Li, W</a>
</li>
<li>
<a href="https://scholar.google.com/scholar?q=Qinghua%20Guo" target="_blank">Guo, Q</a>
</li>
<li>
<a href="https://scholar.google.com/scholar?q=Maggi%20Kelly" target="_blank">Kelly, M</a>
</li>
</ul>
</div>
<li class="side-menu-li">
<a class="expand" onclick='$(this).closest("li").next("div").toggle(); return false;'>on PubMed</a>
</li>
<div id="AuthorPubMedExpand" style="display:none;">
<ul class="submenu">
<li>
<a href="http://www.pubmed.com/?cmd=Search&amp;term=Marek%20K.%20Jakubowski" target="_blank">Jakubowski, MK.</a>
</li>
<li>
<a href="http://www.pubmed.com/?cmd=Search&amp;term=Wenkai%20Li" target="_blank">Li, W</a>
</li>
<li>
<a href="http://www.pubmed.com/?cmd=Search&amp;term=Qinghua%20Guo" target="_blank">Guo, Q</a>
</li>
<li>
<a href="http://www.pubmed.com/?cmd=Search&amp;term=Maggi%20Kelly" target="_blank">Kelly, M</a>
</li>
</ul>
</div>
</ul>
</div>
<div class="generic-item no-border">
<h2>Export Article</h2>

<form style="margin:0; padding:0; display:inline;" name="export-bibtex" method="POST" action="/export">
<input name="articles_ids[]" value="28270" type="hidden">
<input name="export_format_top" value="bibtex" type="hidden">
<input name="export_submit_top" value="" type="hidden">
</form>

<form style="margin:0; padding:0; display:inline;" name="export-endnote" method="POST" action="/export">
<input name="articles_ids[]" value="28270" type="hidden">
<input name="export_format_top" value="endnote_no_abstract" type="hidden">
<input name="export_submit_top" value="" type="hidden">
</form>

<form style="margin:0; padding:0; display:inline;" name="export-ris" method="POST" action="/export">
<input name="articles_ids[]" value="28270" type="hidden">
<input name="export_format_top" value="ris" type="hidden">
<input name="export_submit_top" value="" type="hidden">
</form>
<ul class="side-menu-ul">
<li class="side-menu-li">
<a href="javascript:window.document.forms['export-bibtex'].submit()">BibTeX</a>
</li>
<li class="side-menu-li">
<a href="javascript:window.document.forms['export-endnote'].submit()">EndNote</a>
</li>
<li class="side-menu-li"><a href="javascript:window.document.forms['export-ris'].submit()">RIS</a></li>
</ul>
</div>
<div class="generic-item no-border social-media-links" style="margin-top: 10px; text-align: left;"><a href="mailto:?&amp;subject=From%20MDPI%3A%20%22Delineating%20Individual%20Trees%20from%20Lidar%20Data%3A%20A%20Comparison%20of%20Vector-%20and%20Raster-based%20Segmentation%20Approaches%22&amp;body=http%3A%2F%2Fwww.mdpi.com%2F2072-4292%2F5%2F9%2F4163%3A%0A%0ADelineating%20Individual%20Trees%20from%20Lidar%20Data%3A%20A%20Comparison%20of%20Vector-%20and%20Raster-based%20Segmentation%20Approaches%0A%0AAbstract%3A%20Light%20detection%20and%20ranging%20%28lidar%29%20data%20is%20increasingly%20being%20used%20for%20ecosystem%20monitoring%20across%20geographic%20scales.%20This%20work%20concentrates%20on%20delineating%20individual%20trees%20in%20topographically-complex%2C%20mixed%20conifer%20forest%20across%20the%20California%E2%80%99s%20Sierra%20Nevada.%20We%20delineated%20individual%20trees%20using%20vector%20data%20and%20a%203D%20lidar%20point%20cloud%20segmentation%20algorithm%2C%20and%20using%20raster%20data%20with%20an%20object-based%20image%20analysis%20%28OBIA%29%20of%20a%20canopy%20height%20model%20%28CHM%29.%20The%20two%20approaches%20are%20compared%20to%20each%20other%20and%20to%20ground%20reference%20data.%20We%20used%20high%20density%20%289%20pulses%2Fm2%29%2C%20discreet%20lidar%20data%20and%20WorldView-2%20imagery%20to%20delineate%20individual%20trees%2C%20and%20to%20classify%20them%20by%20species%20or%20species%20types.%20We%20also%20identified%20a%20new%20method%20to%20correct%20artifacts%20in%20a%20high-resolution%20CHM.%20Our%20main%20focus%20was%20to%20determine%20the%20difference%20between%20the%20two%20types%20of%20approaches%20and%20to%20identify%20the%20one%20that%20produces%20more%20realistic%20results.%20We%20compared%20the%20delineations%20via%20tree%20detection%2C%20tree%20heights%2C%20and%20the%20shape%20of%20the%20generated%20polygons.%20The%20tree%20height%20agreement%20was%20high%20between%20the%20two%20approaches%20and%20the%20ground%20data%20%28r2%3A%200.93%E2%80%930.96%29.%20Tree%20detection%20rates[...]">
<i class="fa fa-envelope-square" style="font-size: 30px;"></i>
</a>
<a href="https://twitter.com/home?status=%23mdpiremotesensing+Delineating+Individual+Trees+from+Lidar+Data%3A+A+Comparison+of+Vector-+and+Raster-based+Segmentation+Approaches+http%3A%2F%2Fwww.mdpi.com%2F2072-4292%2F5%2F9%2F4163++%40RemoteSens_MDPI" onclick="windowOpen(this.href,600,800); return false" target="_blank">
<i class="fa fa-twitter-square" style="font-size: 30px;"></i>
</a>
<a href="http://www.linkedin.com/shareArticle?mini=true&amp;url=http%3A%2F%2Fwww.mdpi.com%2F2072-4292%2F5%2F9%2F4163&amp;title=Delineating%20Individual%20Trees%20from%20Lidar%20Data%3A%20A%20Comparison%20of%20Vector-%20and%20Raster-based%20Segmentation%20Approaches%26source%3Dhttps%3A%2F%2Fwww.mdpi.com%26summary%3DLight%20detection%20and%20ranging%20%28lidar%29%20data%20is%20increasingly%20being%20used%20for%20ecosystem%20monitoring%20across%20geographic%20scales.%20This%20work%20concentrates%20on%20delineating%20individual%20trees%20in%20topographically-complex%2C%20mixed%20conifer%20forest%20across%20the%20California%E2%80%99s%20%5B...%5D" onclick="windowOpen(this.href,600,800); return false" target="_blank">
<i class="fa fa-linkedin-square" style="font-size: 30px;"></i>
</a>
<a href="https://www.facebook.com/sharer.php?u=https://www.mdpi.com/2072-4292/5/9/4163" target="_blank">
<i class="fa fa-facebook-square" style="font-size: 30px;"></i>
</a>
</div>
<div class="generic-item no-border" style="margin-top: 10px; padding-right: 5px;">
<div id="scifeed-modal" style="max-width: 500px;" class="reveal-modal inverted" data-reveal="" aria-labelledby="modalTitle" aria-hidden="true" role="dialog">
<h2 id="modalTitle">Create a <em>SciFeed</em> alert for new publications</h2>
<form id="scifeed_form" style="background: inherit;" action="">
<div class="row">
<input id="id_doi" name="doi" value="10.3390/rs5094163" type="hidden">
<div class="large-6 medium-6 small-12 columns">
<h3>With following keywords</h3>
<div>
<input name="kwd[]" value="lidar" type="checkbox">
<label>lidar</label>
</div>
<div>
<input name="kwd[]" value="individual tree" type="checkbox">
<label>individual tree</label>
</div>
<div>
<input name="kwd[]" value="segmentation" type="checkbox">
<label>segmentation</label>
</div>
<div>
<input name="kwd[]" value="object based" type="checkbox">
<label>object based</label>
</div>
<div>
<input name="kwd[]" value="DEM" type="checkbox">
<label>DEM</label>
</div>
<div>
<input name="kwd[]" value="raster" type="checkbox">
<label>raster</label>
</div>
<div>
<input name="kwd[]" value="vector" type="checkbox">
<label>vector</label>
</div>
</div>
<div class="large-6 medium-6 small-12 columns">
<h3>By following authors</h3>
<div>
<input name="authors[]" value="Marek K. Jakubowski" type="checkbox">
<label>Marek K. Jakubowski</label>
</div>
<div>
<input name="authors[]" value="Wenkai Li" type="checkbox">
<label>Wenkai Li</label>
</div>
<div>
<input name="authors[]" value="Qinghua Guo" type="checkbox">
<label>Qinghua Guo</label>
</div>
<div>
<input name="authors[]" value="Maggi Kelly" type="checkbox">
<label>Maggi Kelly</label>
</div>
</div>
</div>
<div class="row" style="border-top: 1px solid #ccc;">
<div class="large-12 medium-12 small-12 columns">
<a style="display: block; height: 20px; float: right;" onclick="sendScifeedFrom($('#scifeed_form'), true);return false;">Advanced options</a>
<div style="float: left;">
<div style="float: left;">
<div style="padding: 2px 0;">
<label style="display: inline-block; width: 40px; margin-left: 0px;">Email:</label>
<input name="scifeed_email" style="max-width: 170px;" placeholder="Your email" id="scifeed_email" required="true" type="text">
</div>
<div style="min-width: 215px; padding: 2px 0;">
<label style="display: inline-block; width: 40px; margin-left: 0px;">Freq:</label>
<select style="max-width: 170px; display: inline-block;" name="frequency" id="frequency">
<option value="daily" selected="selected"> Daily </option>
<option value="weekly"> Weekly </option>
<option value="monthly"> Monthly </option>
</select>
</div>
</div>
<div style="float: left; margin-right: 10px; border-left: 1px solid #ccc; margin-left: 10px; padding-left: 10px;">
<div style="height: 20px; margin-top: 3px;">
<input class="split_feeds" name="split_feeds" id="split_feeds" checked="checked" type="checkbox">
<label style="margin-right: 0px;"><em>One email with all search results</em></label>
</div>
<div style="height: 20px; margin-top: 6px;">
<input class="split_feeds" name="split_feeds_" id="split_feeds2" type="checkbox">
<label style="margin-right: 0px;"><em>One email for each search</em></label>
</div>
</div>
</div>
</div>
</div>
<div class="row" style="border-top: 1px solid #ccc;">
<div class="large-12 medium-12 small-12 columns">
<div class="scifeed_subscription">
<div class="ui-helper-hidden element" id="captchaSection"><img id="gregwar_captcha_scifeed" src="gcb_captcha.jpeg" alt="" title="captcha" class="img-fixed" height="50" width="130"><script type="text/javascript">
                            function reload_gregwar_captcha_scifeed() {
                                var img = document.getElementById('gregwar_captcha_scifeed');
                                img.src = '/generate-captcha/gcb_captcha?n=' + (new Date()).getTime();
                            }
                        </script><a class="captcha_reload" href="javascript:reload_gregwar_captcha_scifeed();">Renew</a><input id="form_captcha" name="form[captcha]" required="required" type="text"></div>
<br>
<input id="scifeed_submit1" value="Create SciFeed" class="submit scifeed_subscription subscription_toggle" onclick="sendScifeedFrom($('#scifeed_form'));return false;" type="button">
</div>
</div>
</div>
</form>
<div class="row scifeed_msg_error" id="scifeed_error_msg" style="display: none; border-top: 1px solid #ccc; background-color: #FFC1C1;"></div>
<div class="row scifeed_msg warning" id="scifeed_notice_msg" style="display: none; border-top: 1px solid #ccc; background-color: #fff8ce;"></div>
<a class="close-reveal-modal" aria-label="Close">×</a>
</div>
<span style="display:none" id="scifeed_hidden_flag"></span>
<span style="display:none" id="scifeed_subscribe_url">/ajax/scifeed/subscribe</span>
</div>
<div class="generic-item no-border">
<div class="menu-caption" id="html-quick-links-title">
<h2>Quick Links</h2>
</div><ul class="menu html-nav side-menu-ul affix-top" data-prev-node="#html-quick-links-title"><li class="side-menu-li"><a href="#html-abstract">Abstract</a></li><li class="side-menu-li"><a href="#Introduction"> Introduction</a></li><li class="side-menu-li"><a href="#Methods"> Methods</a></li><li class="side-menu-li"><a href="#Results"> Results</a></li><li class="side-menu-li"><a href="#Discussion"> Discussion</a></li><li class="side-menu-li"><a href="#Conclusions"> Conclusions</a></li><li class="side-menu-li"><a href="#html-ack">Acknowledgments</a></li><li class="side-menu-li"><a href="#html-notes">Conflict of Interest</a></li><li class="side-menu-li"><a href="#html-references_list">References and Notes</a></li></ul>
</div>
</div>
</div>
</div>
<div id="middle-column" class="large-80 small-12 columns medium-12">
<div class="top-border">
<div class="hypothesis_container">

<div class="html-profile-nav affix-top" style="width: 940px;"> <div class="top-bar"> <div class="nav-sidebar-btn show-for-large-up" data-status="opened"> <span class="nav-sidebar-action" style="display:none;">Close Sidebar</span> <i class="fa fa-caret-left fa-2x"></i> </div> <div class="nav-close-btn">Close Menu</div> <div class="nav-loading" style="display: block;">loading...</div> <div class="nav-btn"> <i class="fa fa-cog fa-2x"></i> </div> </div> <div class="html-article-menu"> <div class="html-first-step"> <ul class="html-font-family"> <li style="width:100%" class="html-font-label">Font Type</li> <li> <span><i style="font-family:Arial, Arial, Helvetica, sans-serif;" data-fontfamily="Arial, Arial, Helvetica, sans-serif">Arial</i></span> <span><i style="font-family:Georgia1, Georgia, serif;" data-fontfamily="Georgia1, Georgia, serif">Georgia</i></span> <span><i style="font-family:Palatino Linotype, Book Antiqua, Palatino, serif;" data-fontfamily="Palatino Linotype, Book Antiqua, Palatino, serif">Palatino Linotype</i></span> <span><i style="font-family:Verdana, Verdana, Geneva, sans-serif;" data-fontfamily="Verdana, Verdana, Geneva, sans-serif">Verdana</i></span> </li> </ul> <ul class="html-recomment-ul"> <li> <div class="html-recomment-button">Default Settings</div> </li> </ul> <ul class="html-font-resize"> <li style="width:100%" class="html-font-label">Font Size</li> <li> <span class="a1" data-percent="100">Aa</span><span class="a2" data-percent="130">Aa</span><span class="a3" data-percent="160">Aa</span><span class="a4" data-percent="190">Aa</span><span class="a5" data-percent="220">Aa</span> </li> </ul> </div> <ul class="html-line-space"> <li class="html-font-label" style="width:55%;">Line Spacing</li> <li> <span class="a1" data-line-height="1.5em"><i class="fa"></i></span><span class="a2" data-line-height="1.8em"><i class="fa"></i></span><span class="a3" data-line-height="2.1em"><i class="fa"></i></span><span class="a4" data-line-height="2.4em"><i class="fa"></i></span><span class="a5" data-line-height="2.7em"><i class="fa"></i></span> </li> </ul> <ul class="html-font-bg"> <li style="width:100%" class="html-font-label">Background</li> <li> <div class="html-nav-bg html-nav-bright" data-bg="bright"> <i class="fa fa-file-text"></i></div><div class="html-nav-bg html-nav-dark" data-bg="dark"> <i class="fa fa-file-text-o"></i></div><div class="html-nav-bg html-nav-creme" data-bg="creme"> <i class="fa fa-file-text"></i></div> </li> </ul> <ul class="html-column-width"> <li class="html-font-label">Column Width</li> <li> <span class="a1" data-column-width="20%"><i class="fa"></i></span><span class="a2" data-column-width="15%"><i class="fa"></i></span><span class="a3" data-column-width="10%"><i class="fa"></i></span><span class="a4" data-column-width="5%"><i class="fa"></i></span><span class="a5" data-column-width="0%"><i class="fa"></i></span> </li> </ul> </div> </div><article>
<div class="html-front">
<span class="label openaccess" style="float: right;">Open Access</span>
<div class="html-art-header">
<p>
<i>Remote Sensing</i>
<b>2013</b>,
<i>5</i>(9),
4163-4186;
doi:<a href="https://dx.doi.org/10.3390/rs5094163" target="_blank">10.3390/rs5094163</a>
</p>
</div>
<div class="html-art-type">
Article
</div>
<div id="html-article-title">Delineating Individual Trees from Lidar Data: A Comparison of Vector- and Raster-based Segmentation Approaches</div>
<div class="html-author-group"><span class="inlineblock"><a itemprop="author" href="https://www.mdpi.com/search?authors=Marek%20K.%20Jakubowski&amp;orcid=">Marek K. Jakubowski</a><sup> 1</sup><div style="display: inline; margin-left: 5px;"></div><a class="toEncode emailCaptcha" data-author-id="193173" href="mailto:please_login"><sup><i class="fa fa-envelope-o"></i></sup></a>, </span><span class="inlineblock"><a itemprop="author" href="https://www.mdpi.com/search?authors=Wenkai%20Li&amp;orcid=">Wenkai Li</a><sup> 2</sup><div style="display: inline; margin-left: 5px;"></div><a class="toEncode emailCaptcha" data-author-id="193174" href="mailto:please_login"><sup><i class="fa fa-envelope-o"></i></sup></a>, </span><span class="inlineblock"><a itemprop="author" href="https://www.mdpi.com/search?authors=Qinghua%20Guo&amp;orcid=">Qinghua Guo</a><sup> 2</sup><div style="display: inline; margin-left: 5px;"></div><a class="toEncode emailCaptcha" data-author-id="193175" href="mailto:please_login"><sup><i class="fa fa-envelope-o"></i></sup></a> and </span><span class="inlineblock"><a itemprop="author" href="https://www.mdpi.com/search?authors=Maggi%20Kelly&amp;orcid=0000-0002-0198-2822">Maggi Kelly</a><sup> 1,</sup>*
<a class="toEncode emailCaptcha" data-author-id="193176" href="mailto:please_login"><sup><i class="fa fa-envelope-o"></i></sup></a><a href="https://orcid.org/0000-0002-0198-2822" target="_blank"><img src="orcid.png" title="OrcID" style="position: relative; width: 13px; margin-left: 3px; max-width: 13px !important; height: auto; top: -5px;"></a></span></div>
<div class="art-affiliations">
<div class="affiliation">
<div class="affiliation-item"><sup>1</sup></div>
<div class="affiliation-name">Department of Environmental Science, 
Policy, and Management, University of California, Berkeley, 130 Mulford 
Hall #3114, Berkeley, CA 94720, USA</div>
</div>
<div class="affiliation">
<div class="affiliation-item"><sup>2</sup></div>
<div class="affiliation-name">School of Engineering, University of California, Merced, 5200 North Lake Rd., Merced, CA 95343, USA</div>
</div>
</div>
<div style="height: 10px;"></div>
<div class="affiliation">
<div class="affiliation-item"><sup>*</sup></div>
<div class="affiliation-name">Author to whom correspondence should be addressed. </div>
</div>
<div class="html-history">
Received: 28 June 2013; in revised form: 22 August 2013 / Accepted: 22 August 2013 / Published: 26 August 2013
</div>
<section class="html-abstract" id="html-abstract">
<h2 id="html-abstract-title">Abstract</h2><b>:</b>
Light detection and ranging (lidar) data is increasingly being used for 
ecosystem monitoring across geographic scales. This work concentrates on
 delineating individual trees in topographically-complex, mixed conifer 
forest across the California’s Sierra Nevada. We delineated individual 
trees using vector data and a 3D lidar point cloud segmentation 
algorithm, and using raster data with an object-based image analysis 
(OBIA) of a canopy height model (CHM). The two approaches are compared 
to each other and to ground reference data. We used high density (9 
pulses/m<sup>2</sup>), discreet lidar data and WorldView-2 imagery to 
delineate individual trees, and to classify them by species or species 
types. We also identified a new method to correct artifacts in a 
high-resolution CHM. Our main focus was to determine the difference 
between the two types of approaches and to identify the one that 
produces more realistic results. We compared the delineations via tree 
detection, tree heights, and the shape of the generated polygons. The 
tree height agreement was high between the two approaches and the ground
 data (r<sup>2</sup>: 0.93–0.96). Tree detection rates increased for 
more dominant trees (8–100 percent). The two approaches delineated tree 
boundaries that differed in shape: the lidar-approach produced fewer, 
more complex, and larger polygons that more closely resembled real 
forest structure.
</section>
<div id="html-keywords">
<div class="html-gwd-group"><div id="html-keywords-title">Keywords:</div>
lidar; individual tree; segmentation; object based; DEM; raster; vector</div>
</div>
</div>
<div class="html-body">
<section id="Introduction" type="other"><h2 data-nested="1"> 1. Introduction</h2><p>Remote
 sensing has been established as one of the primary tools for 
broad-scale analysis of forest systems. Metrics relating to forest 
structure such as tree height, canopy cover and biomass can be estimated
 using remote sensing [<a href="#b1-remotesensing-05-04163" class="html-bibr" title="">1</a>–<a href="#b3-remotesensing-05-04163" class="html-bibr" title="">3</a>].
 More recently, as sensor technology has improved and expanded to yield 
higher resolution optical data as well as light detection and ranging 
(lidar) data, it has become increasingly possible to detect individual 
trees. The successful detection and delineation of individual trees is 
critical in forest science, allowing for studies of individual tree 
demography and growth modeling [<a href="#b4-remotesensing-05-04163" class="html-bibr" title="">4</a>–<a href="#b6-remotesensing-05-04163" class="html-bibr" title="">6</a>], understanding of wildlife habitat and behavior [<a href="#b7-remotesensing-05-04163" class="html-bibr" title="">7</a>,<a href="#b8-remotesensing-05-04163" class="html-bibr" title="">8</a>], and more precise measures of biomass in forests [<a href="#b9-remotesensing-05-04163" class="html-bibr" title="">9</a>,<a href="#b10-remotesensing-05-04163" class="html-bibr" title="">10</a>].</p><p>Many approaches have been developed to detect and delineate individual trees from remotely sensed data (<a href="#fig_body_display_f1-remotesensing-05-04163" class="html-fig html-figpopup">Figure 1</a>). Early on, studies focused on assessing individual trees based on optical imagery. Sheng, <span class="html-italic">et al</span>. [<a href="#b11-remotesensing-05-04163" class="html-bibr" title="">11</a>]
 used stereoscopic aerial photographs to identify, model, and measure a 
single coniferous tree using photogrammetric techniques. Later studies 
used a variation of local maxima [<a href="#b12-remotesensing-05-04163" class="html-bibr" title="">12</a>] and adaptive binarization [<a href="#b13-remotesensing-05-04163" class="html-bibr" title="">13</a>] for tree <span class="html-italic">detection</span>, as well as region-growing [<a href="#b14-remotesensing-05-04163" class="html-bibr" title="">14</a>], edge detection [<a href="#b15-remotesensing-05-04163" class="html-bibr" title="">15</a>], and valley-following [<a href="#b16-remotesensing-05-04163" class="html-bibr" title="">16</a>] for tree <span class="html-italic">delineation</span>.</p><div class="html-fig-wrap" id="f1-remotesensing-05-04163">
<div class="html-fig_img">
<div class="html-figpopup html-figpopup-link" href="#fig_body_display_f1-remotesensing-05-04163">
<img alt="Remotesensing 05 04163f1 1024" data-large="/remotesensing/remotesensing-05-04163/article_deploy/html/images/remotesensing-05-04163f1-1024.png" data-original="/remotesensing/remotesensing-05-04163/article_deploy/html/images/remotesensing-05-04163f1.png" src="remotesensing-05-04163f1-1024">
<a class="html-expand html-figpopup" href="#fig_body_display_f1-remotesensing-05-04163"></a>
</div>
</div>
<div class="html-fig_description">
<b>Figure 1.</b>
Past studies that detect or delineate individual trees from either 
imagery, lidar, or their combination. The published work is arranged by 
year, from top to bottom.

</div>
</div><p>With the wide introduction of lidar into remote sensing, an 
increasing number of studies have undertaken individual tree detection [<a href="#b17-remotesensing-05-04163" class="html-bibr" title="">17</a>,<a href="#b18-remotesensing-05-04163" class="html-bibr" title="">18</a>].
 Through time, these studies have shown increased complexity of 
analyses, increased accuracy of results, and a focus on the use of lidar
 data alone. A combination of variable window size filtering [<a href="#b19-remotesensing-05-04163" class="html-bibr" title="">19</a>,<a href="#b20-remotesensing-05-04163" class="html-bibr" title="">20</a>], spatial wavelet analysis [<a href="#b5-remotesensing-05-04163" class="html-bibr" title="">5</a>], <span class="html-italic">k</span>-means clustering [<a href="#b21-remotesensing-05-04163" class="html-bibr" title="">21</a>], and morphological analyses [<a href="#b22-remotesensing-05-04163" class="html-bibr" title="">22</a>] were used to identify trees and estimate their biophysical parameters. Brandtberg <span class="html-italic">et al</span>. [<a href="#b23-remotesensing-05-04163" class="html-bibr" title="">23</a>]
 Gaussian-smoothed a canopy height model (CHM) at numerous scales to 
create a scale-space structure and then detected trees which were 
represented as “blobs” in the scale-space structure, while Gleason and 
Im [<a href="#b24-remotesensing-05-04163" class="html-bibr" title="">24</a>]
 combined object recognition, local maxima filtering, and hill climbing 
algorithms to delineate them. Most commonly, a variation of 
region-growing algorithms was applied for tree delineation from CHM [<a href="#b20-remotesensing-05-04163" class="html-bibr" title="">20</a>,<a href="#b25-remotesensing-05-04163" class="html-bibr" title="">25</a>–<a href="#b28-remotesensing-05-04163" class="html-bibr" title="">28</a>]. The latter approach was later used to predict tree volume on a semi-individual tree level [<a href="#b29-remotesensing-05-04163" class="html-bibr" title="">29</a>]. The efficacies of a few of the above methods were compared in a coastal foothill region of Oregon [<a href="#b30-remotesensing-05-04163" class="html-bibr" title="">30</a>].
 All of the region-growing studies relied on first identifying the tree 
tops with a local maximum filter. In one case where tropical canopy was 
thick, [<a href="#b31-remotesensing-05-04163" class="html-bibr" title="">31</a>] manually delineated trees to classify them into tree species. Kaartinen <span class="html-italic">et al</span>. [<a href="#b17-remotesensing-05-04163" class="html-bibr" title="">17</a>]
 provide a comprehensive review of 33 tree detection methods, and found 
automated methods outperformed manual ones, especially with high trees.</p><p>More recent studies focus on full exploitation of the point cloud and do not transform the lidar data into a raster. Lee <span class="html-italic">et al</span>. [<a href="#b32-remotesensing-05-04163" class="html-bibr" title="">32</a>]
 implemented an adaptive clustering algorithm similar to watershed 
segmentation except that the method was applied to the 3D lidar points. 
Their algorithm relies on training data to perform segmentation based on
 supervised learning. Li <span class="html-italic">et al</span>. [<a href="#b33-remotesensing-05-04163" class="html-bibr" title="">33</a>]
 designed and tested an individual tree segmentation approach that does 
not require such training data. Their algorithm exploits the spacing 
between the tops of trees to identify and group points into a single 
tree based on simple rules of proximity and likely tree shape.</p><p>Simultaneous
 with developments in lidar technology has been the increase in spatial 
resolution of optical image sensors. Classification algorithms developed
 for courser imagery often perform poorly at fine scales due to high 
local heterogeneity [<a href="#b34-remotesensing-05-04163" class="html-bibr" title="">34</a>–<a href="#b36-remotesensing-05-04163" class="html-bibr" title="">36</a>],
 and thus finer resolution imagery has necessitated a change in the way 
imagery is processed: from pixel-based to object-based image analysis 
(OBIA). The OBIA approach first segments imagery into meaningful objects
 through the use of spectral and spatial context. This allows for image 
processing and Geographic Information System (GIS) functionality [<a href="#b37-remotesensing-05-04163" class="html-bibr" title="">37</a>].
 The OBIA approach is inherently multi-scalar, as image segments are 
captured and operate hierarchically. At the landscape scale, studies 
have successfully extracted biophysical forest inventory parameters from
 imagery [<a href="#b38-remotesensing-05-04163" class="html-bibr" title="">38</a>] and used lidar to classify land covers [<a href="#b39-remotesensing-05-04163" class="html-bibr" title="">39</a>,<a href="#b40-remotesensing-05-04163" class="html-bibr" title="">40</a>]. At stand-scale, Ke <span class="html-italic">et al</span>. [<a href="#b41-remotesensing-05-04163" class="html-bibr" title="">41</a>]
 used a combination of lidar and optical imagery to classify tree 
species. A number of geospatial image segmentation programs are 
available, including BerkeleyImageSeg [<a href="#b42-remotesensing-05-04163" class="html-bibr" title="">42</a>] and Trimble eCognition [<a href="#b43-remotesensing-05-04163" class="html-bibr" title="">43</a>].</p><p>Less research has been done using OBIA to extract individual trees [<a href="#b35-remotesensing-05-04163" class="html-bibr" title="">35</a>,<a href="#b44-remotesensing-05-04163" class="html-bibr" title="">44</a>],
 and there have been few comparisons of OBIA-derived approaches and 
lidar point cloud segmentation (one exception is Kaartinen <span class="html-italic">et al.</span> [<a href="#b17-remotesensing-05-04163" class="html-bibr" title="">17</a>]).
 In this work, we compare two paradigms of segmentation for individual 
tree delineation. The first approach uses 3D lidar point cloud 
segmentation algorithm developed by Li, Guo, Jakubowski and Kelly [<a href="#b33-remotesensing-05-04163" class="html-bibr" title="">33</a>]
 (hereafter called “lidar-derived”), and the second uses an OBIA 
approach with surfaces derived from lidar data (hereafter called 
“OBIA”). The latter analysis uses a novel method for correcting 
artifacts in a CHM by filling in erroneous height values. We evaluate 
both types of tree polygons in terms of their area, shape index, 
compactness, and ability to predict ground reference tree height, all 
across a gradient of tree densities. Finally, we classify the tree 
objects into species and species type, and validate the results against 
ground reference data using a 10-fold cross-validation.</p></section><section id="Methods" type="methods"><h2 data-nested="1"> 2. Methods</h2><section id="StudyAreaDescription" type=""><h4 class="html-italic" data-nested="2"> 2.1. Study Area Description</h4><p>The
 study area (9,950 ha) is comprised of topographically complex 
mixed-conifer forest within California’s Sierra Nevada mountain range 
(center: 39°07’N, 120°36’W). The elevation varies from 600 to 2,186 m 
above sea level and much of the land is characterized by steep slopes (<a href="#fig_body_display_f2-remotesensing-05-04163" class="html-fig html-figpopup">Figure 2</a>).
 The forest is mostly managed by the US Department of Agriculture, 
Forest Service, with a few small private inholdings. The forest 
composition is predominantly conifer, with the dominant species being 
red and white fir (<span class="html-italic">Abies magnifica</span> and <span class="html-italic">Abies concolor</span>, respectively), sugar pine (<span class="html-italic">Pinus lambertiana</span>), ponderosa pine (<span class="html-italic">Pinus ponderosa</span>), and incense-cedar (<span class="html-italic">Calocedrus decurrens</span>). Less than 10 percent of the forest is non-conifer, dominated by the California black oak (<span class="html-italic">Quercus kelloggii</span>).
 Overall, the area experiences typical Mediterranean climate conditions:
 an average precipitation of 1,182 mm/year and frequent (5 to 15 year 
intervals) low-severity fires [<a href="#b45-remotesensing-05-04163" class="html-bibr" title="">45</a>].</p><div class="html-fig-wrap" id="f2-remotesensing-05-04163">
<div class="html-fig_img">
<div class="html-figpopup html-figpopup-link" href="#fig_body_display_f2-remotesensing-05-04163">
<img alt="Remotesensing 05 04163f2 1024" data-large="/remotesensing/remotesensing-05-04163/article_deploy/html/images/remotesensing-05-04163f2-1024.png" data-original="/remotesensing/remotesensing-05-04163/article_deploy/html/images/remotesensing-05-04163f2.png" src="remotesensing-05-04163f2-1024">
<a class="html-expand html-figpopup" href="#fig_body_display_f2-remotesensing-05-04163"></a>
</div>
</div>
<div class="html-fig_description">
<b>Figure 2.</b>
The study area (9,950 ha), mostly within Tahoe National Forest, is 
covered by mixed conifer forest, and covers topographically complex 
terrain. The nine 90 × 90 m square areas are indicated by red lines.

</div>
</div></section><section id="GroundReferenceDataandSpatialProtocol" type=""><h4 class="html-italic" data-nested="2"> 2.2. Ground Reference Data and Spatial Protocol</h4><p>This experiment is part of a larger study—Sierra Nevada Adaptive Management Project—in which 411 circular plots (area = 500 m<sup>2</sup>)
 were established throughout the study area to characterize the fire and
 forest health ecology. The plots were centered at even coordinates 
(e.g., 700,000 m E, 4,333,000 m N; 700,500 m E, 4,333,500 m N, <span class="html-italic">etc</span>.) according to the Universal Transverse Mercator (UTM) system. In the work described here, we chose nine plots (<a href="#fig_body_display_f2-remotesensing-05-04163" class="html-fig html-figpopup">Figures 2</a> and <a href="#fig_body_display_f3-remotesensing-05-04163" class="html-fig html-figpopup">3</a>)
 representative of three overstory tree densities: low (&lt;200 
trees/ha), medium (≥200 to &lt;400 trees/ha), and high (≥400 trees/ha). 
Plots were classified according to tree density. Within each plot, we 
geolocated trees according to a spatial protocol (described below), and 
collected measurements relating to trees (height, species, dominance, 
height to live crown, and diameter at breast height or DBH), shrub 
(species, and height), and fuels (fuel loads and ladder fuels). More 
information can be found in [<a href="#b46-remotesensing-05-04163" class="html-bibr" title="">46</a>].</p><div class="html-fig-wrap" id="f3-remotesensing-05-04163">
<div class="html-fig_img">
<div class="html-figpopup html-figpopup-link" href="#fig_body_display_f3-remotesensing-05-04163">
<img alt="Remotesensing 05 04163f3 1024" data-large="/remotesensing/remotesensing-05-04163/article_deploy/html/images/remotesensing-05-04163f3-1024.png" data-original="/remotesensing/remotesensing-05-04163/article_deploy/html/images/remotesensing-05-04163f3.png" src="remotesensing-05-04163f3-1024">
<a class="html-expand html-figpopup" href="#fig_body_display_f3-remotesensing-05-04163"></a>
</div>
</div>
<div class="html-fig_description">
<b>Figure 3.</b>
Nine 90 × 90 m square focal areas were segmented using the lidar-derived
 and OBIA-derived methods. The areas varied across tree density, from 
low (<b>left</b> column), medium (<b>middle</b> column), to high (<b>right</b> column). The color above corresponds to corrected CHM height values.

</div>
</div><p>The trees were divided into three classes: overstory (DBH ≥ 
19.5 cm), mid-size (5 ≤ DBH &lt; 19.5) and small (DBH &lt; 5 cm). All 
overstory trees were tagged with a unique identification number and 
georeferenced according to the spatial protocol. Although all trees 
within the plots were measured and described, in this work we only 
consider the overstory trees (n = 141) since the position of the smaller
 trees were not recorded. In addition to the overstory trees, we 
selectively measured and georeferenced large and/or isolated “marker” 
trees (n = 57) outside of the plots, for the purpose of improving the 
match between lidar and ground reference data. Both, the overstory and 
the marker trees (n = 198) were used in the analyses.</p><p>The plots 
and the trees were georeferenced in the following manner. First, we 
stabilized a Trimble GeoXH differential Global Positioning System (dGPS)
 in the most open canopy area within or near the plot (≤30 m from the 
plot center), where it recorded at least 300 measurements below 
positional dilution of precision (PDOP) of 5. In most circumstances, we 
recorded between 1,000 and 7,000 measurements for each plot position. We
 used a Trimble Zephyr antenna elevated 3 m from the ground to reduce 
multipath issues. The dGPS data were post-processed using University 
NAVSTAR Consortium (UNAVCO) and Continuously Operating Reference 
Stations (CORS), all located ≤20 km from the field measurements. To 
georeference the trees, we used an Impulse Laser Rangefinder and Impulse
 Electronic Compass. The electronic compass was carefully stabilized and
 calibrated on each plot before recording the trees’ positions. We used 
the “filter” option in the range finder, so that it would only record 
distances to a designated target (positioned on the measured trees). 
Since the rangefinder requires a clear line of sight to its target, some
 measurements were collected from two and sometimes three positions in 
plots with very high tree or shrub density. They were later combined in 
GIS. Since the error of the laser rangefinder is ≤2 cm, and the compass 
degree error ≤0.5 degrees, we only considered marker trees &lt;50 m from
 the measuring unit. All tree positions were manually verified and in a 
few cases moved to increase the correspondence to the lidar data.</p></section><section id="MultispectralImagery" type=""><h4 class="html-italic" data-nested="2"> 2.3. Multispectral Imagery</h4><p>We
 used DigitalGlobe’s WorldView-2 data in the species prediction part of 
this work (described below). The imagery includes eight spectral and one
 panchromatic band (<a href="#table_body_display_t1-remotesensing-05-04163" class="html-table html-tablepopup">Table 1</a>).
 A single image, acquired on June 14th, 2010 at 12:14 local time (sun 
elevation = 71.3°), was used for the analysis. The data was delivered in
 LV1B product level: 16-bit and radiometrically corrected. The sky was 
clear (cloud cover = 0.077 %) and no snow was present in any of the 
analyzed plots.</p><div class="html-table-wrap" id="t1-remotesensing-05-04163"><div class="html-table_wrap_discription">
<b>Table 1.</b>
WorldView-2 specifications of the spectral bands.
</div>
<div class="html-table_wrap_td">
<div class="html-tablepopup html-tablepopup-link" href="#table_body_display_t1-remotesensing-05-04163">
<img alt="Table" src="table.png">
<a class="html-expand html-tablepopup" href="#table_body_display_t1-remotesensing-05-04163"></a>
</div>
</div>

</div><p>The data was orthorectified and then pan-sharpened before the 
analysis. The orthorectification was performed in EXELIS’s ENVI 5 [<a href="#b47-remotesensing-05-04163" class="html-bibr" title="">47</a>]
 software using a combination of a Rational Polynomial Coefficients 
(RPC) sensor model and a 0.5 m digital elevation model (DEM) generated 
from the lidar data. The image and the DEM both used bilinear 
resampling. We used −24.557 m as the geoid offset based on National 
Geodetic Survey’s GEOID99 model computed at the center of the image 
scene. The data was projected into the UTM (zone 10S) coordinate system 
with respect to 1983 North American Datum (NAD83) at 50 cm pixel ground 
sample distance (GSD) to match the remainder of the data. We 
pan-sharpened the spectral data cube using the Gram-Schmidt algorithm 
and bilinear resampling [<a href="#b48-remotesensing-05-04163" class="html-bibr" title="">48</a>].</p></section><section id="LiDARData" type=""><h4 class="html-italic" data-nested="2"> 2.4. LiDAR Data</h4><p>Lidar
 data was acquired between 18 and 21 September 2008 by the National 
Center for Airborne Laser Mapping (NCALM) using the Optech GEMINI 
Airborne Laser Terrain Mapper (ALTM) sensor. Flying at an average height
 of 1,000 m AGL, the sensor collected up to 4 returns per pulse at an 
average of 9 and a minimum of 6 pulses/m<sup>2</sup>. We confirmed the 
horizontal accuracy to be 5.5–10 cm, and the average vertical accuracy 
to be 7.5 cm. The instrument was operated at a 70 kHz pulse rate 
frequency (PRF), with a 40 Hz scan frequency, and a ±20 degree scan 
angle. All even-numbered flight lines were flown twice in order to 
obtain high point density. The final point cloud often exceeded 20 
points/m<sup>2</sup> because of dense vegetation. All data were 
delivered in the UTM coordinate system with respect to NAD83 and the 
1988 North American Vertical Datum using the NGS GEOID03 model. NCALM 
preprocessed the data to remove erroneous below-surface and air points. A
 5 m search radius was used to identify and remove isolated points. The 
points were classified as ground and above-ground using TerraScan’s 
iterative triangle-building surface model classification [<a href="#b49-remotesensing-05-04163" class="html-bibr" title="">49</a>]. From the entire point cloud, we extracted 100 m squares centered at plot centers.</p></section><section id="SegmentingthePointCloudintoIndividualTrees" type=""><h4 class="html-italic" data-nested="2"> 2.5. Segmenting the Point Cloud into Individual Trees</h4><p>Individual trees were segmented using a tree segmentation algorithm [<a href="#b33-remotesensing-05-04163" class="html-bibr" title="">33</a>].
 The algorithm exploits the spacing between the tops of trees to 
identify and group points into a single individual tree. An adaptive 
spacing threshold (<span class="html-italic">dt</span>) was used to improve the segmentation accuracy: <span class="html-italic">dt</span>
 = 2 and 1.5 m when tree heights were ≥15 and &lt;15 m, respectively. 
This spacing threshold was chosen based on our knowledge of the forest 
in our study areas. We assume that the tree spacing at the upper lever 
is greater than 1 m, and that the taller trees have larger crown sizes 
and hence larger spacing. We did not use a minimum spacing rule or the 
shape index threshold as it did not improve the accuracy of the 
classification. Prior to the segmentation, the point cloud was 
normalized by subtracting the DEM ground values [<a href="#b32-remotesensing-05-04163" class="html-bibr" title="">32</a>].
 Following the segmentation, we converted the segmented point cloud into
 two-dimensional tree polygons by calculating the concave hull around 
the lidar points identified as a single tree (<a href="#fig_body_display_f4-remotesensing-05-04163" class="html-fig html-figpopup">Figure 4</a>). Further, each tree’s maximum-height point was extracted for later analysis.</p><div class="html-fig-wrap" id="f4-remotesensing-05-04163">
<div class="html-fig_img">
<div class="html-figpopup html-figpopup-link" href="#fig_body_display_f4-remotesensing-05-04163">
<img alt="Remotesensing 05 04163f4 1024" data-large="/remotesensing/remotesensing-05-04163/article_deploy/html/images/remotesensing-05-04163f4-1024.png" data-original="/remotesensing/remotesensing-05-04163/article_deploy/html/images/remotesensing-05-04163f4.png" src="remotesensing-05-04163f4-1024">
<a class="html-expand html-figpopup" href="#fig_body_display_f4-remotesensing-05-04163"></a>
</div>
</div>
<div class="html-fig_description">
<b>Figure 4.</b>
The normalized lidar point cloud was segmented into individual trees using 3D point classification algorithm (<b>A</b>). The classified points were delineated using concave hull in the <span class="html-italic">x-y</span> space (<b>B</b>).
 The thicker tree boundaries exemplify that the lidar-segmentation 
algorithm allows for polygons to overlap. The lidar points are 
identically color-coded by individual trees in both sub-figures.

</div>
</div></section><section id="PreparingtheDataforImageSegmentation" type=""><h4 class="html-italic" data-nested="2"> 2.6. Preparing the Data for Image Segmentation</h4><p>We
 generated DSM (20 cm GSD) from the 100 m square point clouds using 
LAStools, which uses triangulated irregular network (TIN) interpolation 
to calculate the raster values. We determined that TIN interpolation was
 sufficient for generating these surfaces due to the very high spatial. 
LAStools used the maximum of all lidar points (ground and non-ground) 
within a moving kernel window to generate the DSM, whereas only the 
points classified as ground were used to generate the DEM. Preliminary 
analysis indicated that it was an appropriate resolution for accurate 
crown delineation—50 cm GSD surfaces were too course to create 
meaningful individual tree segments. We then generated a CHM by simple 
image subtraction (<a href="#FD1" class="html-disp-formula">Equation (1)</a>).
</p><div class="html-disp-formula-info" id="FD1">
<div class="f">
<span class="MathJax_Preview"><math display="block">
<mrow>
<mi mathvariant="italic">CHM</mi>
<mo>=</mo>
<mi mathvariant="italic">DSM</mi>
<mo>−</mo>
<mi mathvariant="italic">DEM</mi></mrow></math></span><script type="math/mml" id="MathJax-Element-1"><math display="block">
<mrow>
<mi mathvariant="italic">CHM</mi>
<mo>=</mo>
<mi mathvariant="italic">DSM</mi>
<mo>−</mo>
<mi mathvariant="italic">DEM</mi></mrow></math></script>
</div>
<div class="l">
<label>(1)</label>
</div>
</div><p></p></section><section id="CorrectingforCHMArtifacts" type=""><h4 class="html-italic" data-nested="2"> 2.7. Correcting for CHM Artifacts</h4><p>An
 accurate delineation of the tree crowns required generating a CHM at a 
high resolution (20 cm GSD), such that there was not always a lidar 
point within each pixel. As a result, the CHM contained a number of 
artifacts that prohibited successful segmentation using the OBIA 
framework. In particular, the CHM contained many severe elevation drops,
 especially in the middle of the canopy. The elevation drop artifacts 
are directly related to the scanning pattern of the lidar sensor, <span class="html-italic">i.e.</span>, the <span class="html-italic">z</span>-value thin troughs are located in between the lines of lidar scans (<a href="#fig_body_display_f5-remotesensing-05-04163" class="html-fig html-figpopup">Figure 5</a>). The challenge is further complicated since the artifact pixels are <span class="html-italic">not</span>
 no-data values; rather their value is well within a typical CHM. This 
problem has been noted before, especially when attempting to delineate 
individual trees [<a href="#b50-remotesensing-05-04163" class="html-bibr" title="">50</a>].
 Filtering the image using a blurring or a median filter is one 
solution, albeit with negative consequences: the effective sharpness and
 thus the ability to identify the trees’ edges diminishes substantially 
(typically at least a 5 × 5 or a 7 × 7 kernel must be used to minimize 
the artifacts). Another option is to interpolate the surfaces at a 
higher pixel size, although that approach also suffers from similar 
consequences. We identified a new method to mitigate this problem.</p><div class="html-fig-wrap" id="f5-remotesensing-05-04163">
<div class="html-fig_img">
<div class="html-figpopup html-figpopup-link" href="#fig_body_display_f5-remotesensing-05-04163">
<img alt="Remotesensing 05 04163f5 1024" data-large="/remotesensing/remotesensing-05-04163/article_deploy/html/images/remotesensing-05-04163f5-1024.png" data-original="/remotesensing/remotesensing-05-04163/article_deploy/html/images/remotesensing-05-04163f5.png" src="remotesensing-05-04163f5-1024">
<a class="html-expand html-figpopup" href="#fig_body_display_f5-remotesensing-05-04163"></a>
</div>
</div>
<div class="html-fig_description">
<b>Figure 5.</b>
A number of steps were taken to correct the original CHM with artifacts.
 The horizontal profiles across canopy and ground (right) depict how the
 mathematical steps altered the raster surface.

</div>
</div><p>First, a CHM was smoothed such that its continuity across tree 
crowns is restored and, simultaneously, the crowns’ distinct edges are 
preserved. This can be achieved with an adaptive enhanced Lee filter [<a href="#b51-remotesensing-05-04163" class="html-bibr" title="">51</a>,<a href="#b52-remotesensing-05-04163" class="html-bibr" title="">52</a>] or through morphological filtering [<a href="#b53-remotesensing-05-04163" class="html-bibr" title="">53</a>].
 The enhanced Lee filter reduces speckle while preserving texture. It 
uses the local coefficient of variation to classify pixels as either 
homogeneous, heterogeneous, or point targets, and then replaces the 
targeted pixels based on the classification. The second method uses two 
cycles of the morphological closing operator (dilation followed by 
erosion). Closing filtering fills the small holes and fuses thin 
troughs. In this work we used the morphology filter approach (<a href="#FD2" class="html-disp-formula">Equation (2)</a>) as it produced better results with our data.
</p><div class="html-disp-formula-info" id="FD2">
<div class="f">
<span class="MathJax_Preview"><math display="block">
<mrow>
<msub>
<mrow>
<mi mathvariant="italic">CHM</mi></mrow>
<mrow>
<mi mathvariant="italic">smooth</mi></mrow></msub>
<mo>=</mo>
<msub>
<mrow>
<mi>ϕ</mi></mrow>
<mi>k</mi></msub>
<mrow>
<mo>(</mo>
<mrow>
<msub>
<mrow>
<mi>ϕ</mi></mrow>
<mi>k</mi></msub>
<mo stretchy="false">(</mo>
<mi mathvariant="italic">CHM</mi>
<mo stretchy="false">)</mo></mrow>
<mo>)</mo></mrow>
<mo>,</mo>
<mtext>where</mtext>
<mo> </mo>
<mi>k</mi>
<mo>=</mo>
<mrow>
<mo>(</mo>
<mrow>
<mtable>
<mtr>
<mtd>
<mn>1</mn></mtd>
<mtd>
<mn>1</mn></mtd>
<mtd>
<mn>1</mn></mtd></mtr>
<mtr>
<mtd>
<mn>1</mn></mtd>
<mtd>
<mn>1</mn></mtd>
<mtd>
<mn>1</mn></mtd></mtr>
<mtr>
<mtd>
<mn>1</mn></mtd>
<mtd>
<mn>1</mn></mtd>
<mtd>
<mn>1</mn></mtd></mtr></mtable></mrow>
<mo>)</mo></mrow></mrow></math></span><script type="math/mml" id="MathJax-Element-2"><math display="block">
<mrow>
<msub>
<mrow>
<mi mathvariant="italic">CHM</mi></mrow>
<mrow>
<mi mathvariant="italic">smooth</mi></mrow></msub>
<mo>=</mo>
<msub>
<mrow>
<mi>ϕ</mi></mrow>
<mi>k</mi></msub>
<mrow>
<mo>(</mo>
<mrow>
<msub>
<mrow>
<mi>ϕ</mi></mrow>
<mi>k</mi></msub>
<mo stretchy="false">(</mo>
<mi mathvariant="italic">CHM</mi>
<mo stretchy="false">)</mo></mrow>
<mo>)</mo></mrow>
<mo>,</mo>
<mtext>where</mtext>
<mo> </mo>
<mi>k</mi>
<mo>=</mo>
<mrow>
<mo>(</mo>
<mrow>
<mtable>
<mtr>
<mtd>
<mn>1</mn></mtd>
<mtd>
<mn>1</mn></mtd>
<mtd>
<mn>1</mn></mtd></mtr>
<mtr>
<mtd>
<mn>1</mn></mtd>
<mtd>
<mn>1</mn></mtd>
<mtd>
<mn>1</mn></mtd></mtr>
<mtr>
<mtd>
<mn>1</mn></mtd>
<mtd>
<mn>1</mn></mtd>
<mtd>
<mn>1</mn></mtd></mtr></mtable></mrow>
<mo>)</mo></mrow></mrow></math></script>
</div>
<div class="l">
<label>(2)</label>
</div>
</div><p></p><p>Although the smoothed CHM was an improvement over the 
original, we strive to use the original CHM values as much as possible. 
The next few steps develop a technique that replaces the artifact pixels
 of the original CHM while preserving the surrounding original correct 
height values. Ultimately, we derived a continuous mask used to blend 
the two images: the correct pixels from <span class="html-italic">CHM</span> and the replaced artifact pixels from <span class="html-italic">CHM<sub>smooth</sub></span>.</p><p>First, a simple mask was generated by subtracting the artifact CHM from the smoothed CHM (<a href="#FD3" class="html-disp-formula">Equation (3)</a>). The simple mask was normalized (<a href="#FD4" class="html-disp-formula">Equation (4)</a>) to make the process reproducible for landscapes where the CHM values may vary to a smaller or larger degree.
</p><div class="html-disp-formula-info" id="FD3">
<div class="f">
<span class="MathJax_Preview"><math display="block">
<mrow>
<msub>
<mrow>
<mi mathvariant="italic">Mask</mi></mrow>
<mrow>
<mi mathvariant="italic">diff</mi></mrow></msub>
<mo>=</mo>
<msub>
<mrow>
<mi mathvariant="italic">CHM</mi></mrow>
<mrow>
<mi mathvariant="italic">smooth</mi></mrow></msub>
<mo>−</mo>
<mi mathvariant="italic">CHM</mi></mrow></math></span><script type="math/mml" id="MathJax-Element-3"><math display="block">
<mrow>
<msub>
<mrow>
<mi mathvariant="italic">Mask</mi></mrow>
<mrow>
<mi mathvariant="italic">diff</mi></mrow></msub>
<mo>=</mo>
<msub>
<mrow>
<mi mathvariant="italic">CHM</mi></mrow>
<mrow>
<mi mathvariant="italic">smooth</mi></mrow></msub>
<mo>−</mo>
<mi mathvariant="italic">CHM</mi></mrow></math></script>
</div>
<div class="l">
<label>(3)</label>
</div>
</div><div class="html-disp-formula-info" id="FD4">
<div class="f">
<span class="MathJax_Preview"><math display="block">
<mrow>
<msub>
<mrow>
<mi mathvariant="italic">Mask</mi></mrow>
<mrow>
<mi mathvariant="italic">norm</mi></mrow></msub>
<mo>=</mo>
<mfrac>
<mrow>
<msub>
<mrow>
<mi mathvariant="italic">Mask</mi></mrow>
<mrow>
<mi mathvariant="italic">diff</mi></mrow></msub></mrow>
<mrow>
<mtext>MAX</mtext>
<mrow>
<mo stretchy="false">(</mo>
<mrow>
<msub>
<mrow>
<mi mathvariant="italic">Mask</mi></mrow>
<mrow>
<mi mathvariant="italic">diff</mi></mrow></msub></mrow>
<mo stretchy="false">)</mo></mrow></mrow></mfrac></mrow></math></span><script type="math/mml" id="MathJax-Element-4"><math display="block">
<mrow>
<msub>
<mrow>
<mi mathvariant="italic">Mask</mi></mrow>
<mrow>
<mi mathvariant="italic">norm</mi></mrow></msub>
<mo>=</mo>
<mfrac>
<mrow>
<msub>
<mrow>
<mi mathvariant="italic">Mask</mi></mrow>
<mrow>
<mi mathvariant="italic">diff</mi></mrow></msub></mrow>
<mrow>
<mtext>MAX</mtext>
<mrow>
<mo stretchy="false">(</mo>
<mrow>
<msub>
<mrow>
<mi mathvariant="italic">Mask</mi></mrow>
<mrow>
<mi mathvariant="italic">diff</mi></mrow></msub></mrow>
<mo stretchy="false">)</mo></mrow></mrow></mfrac></mrow></math></script>
</div>
<div class="l">
<label>(4)</label>
</div>
</div><p></p><p>A binary mask was created by reducing near-zero values to zero and all other values to one (<a href="#FD5" class="html-disp-formula">Equation (5)</a>). The Boolean mask was convolved with a simple matrix and the normalized mask was added (<a href="#FD6" class="html-disp-formula">Equation (6)</a>). To increase the impact of the normalized mask, we multiplied it by a constant, <span class="html-italic">b</span>.
 The convolution part of this operation ensures that no hard edges are 
generated when the smoothed CHM pixels are added to the original CHM. 
Setting <span class="html-italic">b</span> to a very high number decreases the feathering effect when adding back the smoothed CHM pixels.
</p><div class="html-disp-formula-info" id="FD5">
<div class="f">
<span class="MathJax_Preview"><math display="block">
<mrow>
<msub>
<mrow>
<mi mathvariant="italic">Mask</mi></mrow>
<mrow>
<mi mathvariant="italic">Boolean</mi></mrow></msub>
<mo>=</mo>
<mrow>
<mo>{</mo>
<mrow>
<mtable columnalign="left">
<mtr columnalign="left">
<mtd columnalign="left">
<mrow>
<mn>0</mn>
<mo>,</mo>
<msub>
<mrow>
<mi mathvariant="italic">Mask</mi></mrow>
<mrow>
<mi mathvariant="italic">norm</mi></mrow></msub>
<mo stretchy="false">(</mo>
<mi>x</mi>
<mo>,</mo>
<mi>y</mi>
<mo stretchy="false">)</mo>
<mo>≤</mo>
<mi>a</mi></mrow></mtd></mtr>
<mtr columnalign="left">
<mtd columnalign="left">
<mrow>
<mn>1</mn>
<mo>,</mo>
<msub>
<mrow>
<mi mathvariant="italic">Mask</mi></mrow>
<mrow>
<mi mathvariant="italic">norm</mi></mrow></msub>
<mo stretchy="false">(</mo>
<mi>x</mi>
<mo>,</mo>
<mi>y</mi>
<mo stretchy="false">)</mo>
<mo>&gt;</mo>
<mi>a</mi>
<mo>'</mo></mrow></mtd></mtr></mtable>
<mtext>where</mtext>
<mo> </mo>
<mi>a</mi>
<mo>∈</mo>
<mo stretchy="false">[</mo>
<mn>0</mn>
<mo>,</mo>
<mn>0.1</mn>
<mo stretchy="false">]</mo></mrow></mrow></mrow></math></span><script type="math/mml" id="MathJax-Element-5"><math display="block">
<mrow>
<msub>
<mrow>
<mi mathvariant="italic">Mask</mi></mrow>
<mrow>
<mi mathvariant="italic">Boolean</mi></mrow></msub>
<mo>=</mo>
<mrow>
<mo>{</mo>
<mrow>
<mtable columnalign="left">
<mtr columnalign="left">
<mtd columnalign="left">
<mrow>
<mn>0</mn>
<mo>,</mo>
<msub>
<mrow>
<mi mathvariant="italic">Mask</mi></mrow>
<mrow>
<mi mathvariant="italic">norm</mi></mrow></msub>
<mo stretchy="false">(</mo>
<mi>x</mi>
<mo>,</mo>
<mi>y</mi>
<mo stretchy="false">)</mo>
<mo>≤</mo>
<mi>a</mi></mrow></mtd></mtr>
<mtr columnalign="left">
<mtd columnalign="left">
<mrow>
<mn>1</mn>
<mo>,</mo>
<msub>
<mrow>
<mi mathvariant="italic">Mask</mi></mrow>
<mrow>
<mi mathvariant="italic">norm</mi></mrow></msub>
<mo stretchy="false">(</mo>
<mi>x</mi>
<mo>,</mo>
<mi>y</mi>
<mo stretchy="false">)</mo>
<mo>&gt;</mo>
<mi>a</mi>
<mo>'</mo></mrow></mtd></mtr></mtable>
<mtext>where</mtext>
<mo> </mo>
<mi>a</mi>
<mo>∈</mo>
<mo stretchy="false">[</mo>
<mn>0</mn>
<mo>,</mo>
<mn>0.1</mn>
<mo stretchy="false">]</mo></mrow></mrow></mrow></math></script>
</div>
<div class="l">
<label>(5)</label>
</div>
</div><div class="html-disp-formula-info" id="FD6">
<div class="f">
<span class="MathJax_Preview"><math display="block">
<mrow>
<msub>
<mrow>
<mi mathvariant="italic">Mask</mi></mrow>
<mrow>
<mi mathvariant="italic">smooth</mi></mrow></msub>
<mo>=</mo>
<msub>
<mrow>
<mi mathvariant="italic">Mask</mi></mrow>
<mrow>
<mi mathvariant="italic">Boolean</mi></mrow></msub>
<mo>*</mo>
<mrow>
<mo>(</mo>
<mrow>
<mtable>
<mtr>
<mtd>
<mn>1</mn></mtd>
<mtd>
<mn>1</mn></mtd>
<mtd>
<mn>1</mn></mtd></mtr>
<mtr>
<mtd>
<mn>1</mn></mtd>
<mtd>
<mn>1</mn></mtd>
<mtd>
<mn>1</mn></mtd></mtr>
<mtr>
<mtd>
<mn>1</mn></mtd>
<mtd>
<mn>1</mn></mtd>
<mtd>
<mn>1</mn></mtd></mtr></mtable></mrow>
<mo>)</mo></mrow>
<mo>+</mo>
<msub>
<mrow>
<mi mathvariant="italic">Mask</mi></mrow>
<mrow>
<mi mathvariant="italic">norm</mi></mrow></msub>
<mo>⋅</mo>
<mi>b</mi>
<mo>,</mo>
<mtext>where</mtext>
<mo> </mo>
<mi>b</mi>
<mo>∈</mo>
<mo stretchy="false">[</mo>
<mn>0</mn>
<mo>,</mo>
<mn>10</mn>
<mo stretchy="false">]</mo></mrow></math></span><script type="math/mml" id="MathJax-Element-6"><math display="block">
<mrow>
<msub>
<mrow>
<mi mathvariant="italic">Mask</mi></mrow>
<mrow>
<mi mathvariant="italic">smooth</mi></mrow></msub>
<mo>=</mo>
<msub>
<mrow>
<mi mathvariant="italic">Mask</mi></mrow>
<mrow>
<mi mathvariant="italic">Boolean</mi></mrow></msub>
<mo>*</mo>
<mrow>
<mo>(</mo>
<mrow>
<mtable>
<mtr>
<mtd>
<mn>1</mn></mtd>
<mtd>
<mn>1</mn></mtd>
<mtd>
<mn>1</mn></mtd></mtr>
<mtr>
<mtd>
<mn>1</mn></mtd>
<mtd>
<mn>1</mn></mtd>
<mtd>
<mn>1</mn></mtd></mtr>
<mtr>
<mtd>
<mn>1</mn></mtd>
<mtd>
<mn>1</mn></mtd>
<mtd>
<mn>1</mn></mtd></mtr></mtable></mrow>
<mo>)</mo></mrow>
<mo>+</mo>
<msub>
<mrow>
<mi mathvariant="italic">Mask</mi></mrow>
<mrow>
<mi mathvariant="italic">norm</mi></mrow></msub>
<mo>⋅</mo>
<mi>b</mi>
<mo>,</mo>
<mtext>where</mtext>
<mo> </mo>
<mi>b</mi>
<mo>∈</mo>
<mo stretchy="false">[</mo>
<mn>0</mn>
<mo>,</mo>
<mn>10</mn>
<mo stretchy="false">]</mo></mrow></math></script>
</div>
<div class="l">
<label>(6)</label>
</div>
</div><p></p><p>The smoothed mask was normalized to create a standard mask range from zero to one (<a href="#FD7" class="html-disp-formula">Equation (7)</a>). The final CHM was compiled by removing the artifact pixels (via the inverse of <span class="html-italic">Mask</span>) and filling in smoothed CHM values where <span class="html-italic">Mask</span> permits (<a href="#FD8" class="html-disp-formula">Equation (8)</a>). Constant <span class="html-italic">c</span>
 was used to adjust the corrected values such that they match the 
original CHM. The process is further described and illustrated in <a href="#fig_body_display_f5-remotesensing-05-04163" class="html-fig html-figpopup">Figure 5</a>.
</p><div class="html-disp-formula-info" id="FD7">
<div class="f">
<span class="MathJax_Preview"><math display="block">
<mrow>
<mi mathvariant="italic">Mask</mi>
<mo>=</mo>
<mfrac>
<mrow>
<msub>
<mrow>
<mi mathvariant="italic">Mask</mi></mrow>
<mrow>
<mi mathvariant="italic">smooth</mi></mrow></msub></mrow>
<mrow>
<mtext>MAX</mtext>
<mrow>
<mo stretchy="false">(</mo>
<mrow>
<msub>
<mrow>
<mi mathvariant="italic">Mask</mi></mrow>
<mrow>
<mi mathvariant="italic">smooth</mi></mrow></msub></mrow>
<mo stretchy="false">)</mo></mrow></mrow></mfrac></mrow></math></span><script type="math/mml" id="MathJax-Element-7"><math display="block">
<mrow>
<mi mathvariant="italic">Mask</mi>
<mo>=</mo>
<mfrac>
<mrow>
<msub>
<mrow>
<mi mathvariant="italic">Mask</mi></mrow>
<mrow>
<mi mathvariant="italic">smooth</mi></mrow></msub></mrow>
<mrow>
<mtext>MAX</mtext>
<mrow>
<mo stretchy="false">(</mo>
<mrow>
<msub>
<mrow>
<mi mathvariant="italic">Mask</mi></mrow>
<mrow>
<mi mathvariant="italic">smooth</mi></mrow></msub></mrow>
<mo stretchy="false">)</mo></mrow></mrow></mfrac></mrow></math></script>
</div>
<div class="l">
<label>(7)</label>
</div>
</div><div class="html-disp-formula-info" id="FD8">
<div class="f">
<span class="MathJax_Preview"><math display="block">
<mrow>
<msub>
<mrow>
<mi mathvariant="italic">CHM</mi></mrow>
<mrow>
<mi mathvariant="italic">corrected</mi></mrow></msub>
<mo>=</mo>
<mi mathvariant="italic">CHM</mi>
<mo>⋅</mo>
<mo stretchy="false">(</mo>
<mo>−</mo>
<mn>1</mn>
<mo>⋅</mo>
<mi mathvariant="italic">Mask</mi>
<mo>+</mo>
<mn>1</mn>
<mo stretchy="false">)</mo>
<mo>+</mo>
<msub>
<mrow>
<mi mathvariant="italic">CHM</mi></mrow>
<mrow>
<mi mathvariant="italic">smooth</mi></mrow></msub>
<mo>⋅</mo>
<mi mathvariant="italic">Mask</mi>
<mo>⋅</mo>
<mi>c</mi>
<mo>,</mo>
<mtext>where</mtext>
<mo> </mo>
<mi>c</mi>
<mo>∈</mo>
<mo stretchy="false">[</mo>
<mn>1</mn>
<mo>,</mo>
<mn>2</mn>
<mo stretchy="false">]</mo></mrow></math></span><script type="math/mml" id="MathJax-Element-8"><math display="block">
<mrow>
<msub>
<mrow>
<mi mathvariant="italic">CHM</mi></mrow>
<mrow>
<mi mathvariant="italic">corrected</mi></mrow></msub>
<mo>=</mo>
<mi mathvariant="italic">CHM</mi>
<mo>⋅</mo>
<mo stretchy="false">(</mo>
<mo>−</mo>
<mn>1</mn>
<mo>⋅</mo>
<mi mathvariant="italic">Mask</mi>
<mo>+</mo>
<mn>1</mn>
<mo stretchy="false">)</mo>
<mo>+</mo>
<msub>
<mrow>
<mi mathvariant="italic">CHM</mi></mrow>
<mrow>
<mi mathvariant="italic">smooth</mi></mrow></msub>
<mo>⋅</mo>
<mi mathvariant="italic">Mask</mi>
<mo>⋅</mo>
<mi>c</mi>
<mo>,</mo>
<mtext>where</mtext>
<mo> </mo>
<mi>c</mi>
<mo>∈</mo>
<mo stretchy="false">[</mo>
<mn>1</mn>
<mo>,</mo>
<mn>2</mn>
<mo stretchy="false">]</mo></mrow></math></script>
</div>
<div class="l">
<label>(8)</label>
</div>
</div><p></p></section><section id="CHMSegmentation" type=""><h4 class="html-italic" data-nested="2"> 2.8. CHM Segmentation</h4><p>We used the multi-resolution segmentation approach in eCognition 8.8 [<a href="#b43-remotesensing-05-04163" class="html-bibr" title="">43</a>]
 to divide the CHM into individual trees. The segmentation process 
relies on user-specified parameters regarding the scale, shape and 
spectral criterion of homogeneity, and a compactness ratio. We used the 
estimation of scale parameter (ESP) tool to determine the optimal 
parameters [<a href="#b54-remotesensing-05-04163" class="html-bibr" title="">54</a>],
 but the shape and the compactness were determined visually. All three 
parameters were altered depending on the particular segmentation 
resolution or purpose (e.g., segmenting small isolated trees <span class="html-italic">vs.</span> large clumped tree stands).</p><p>The
 CHM was first segmented into large, coarse objects, which were then 
subdivided into individual tree crowns and ground. The subdivision into 
tree crowns was an iterative process of classification and segmentation.
 The classification in this case was a means to an end rather than an 
attempt to produce correctly classified objects. For example, objects 
that were likely short trees surrounded by ground pixels (low mean <span class="html-italic">z</span>-values)
 were identified and segmented further into very small objects to 
discriminate between actual ground objects and the trees. The small tree
 objects were then merged together to produce meaningful tree polygons. 
Next, medium-height tree objects were identified at the coarse-level and
 similarly, steps were taken to transform them into meaningful polygons.
 This processed continued until all possible objects of interest were 
segmented. Baatz, Hoffmann and Willhauck [<a href="#b44-remotesensing-05-04163" class="html-bibr" title="">44</a>]
 previously described this approach as “object-orient” OBIA, where 
meaningful objects are segmented, one object type at a time instead of 
trying to segment the entire image in one step. Various object metrics 
were used to isolate trees prior to each sub-segmentation (<a href="#table_body_display_t2-remotesensing-05-04163" class="html-table html-tablepopup">Table 2</a>: segmentation). A single complicated rule set (74 steps) was developed and applied to the entire study area.</p><div class="html-table-wrap" id="t2-remotesensing-05-04163"><div class="html-table_wrap_discription">
<b>Table 2.</b>
Polygon attributes were calculated and were imported at three different 
stages of data processing: during the segmentation, classification of 
species, and feature optimization.
</div>
<div class="html-table_wrap_td">
<div class="html-tablepopup html-tablepopup-link" href="#table_body_display_t2-remotesensing-05-04163">
<img alt="Table" src="table.png">
<a class="html-expand html-tablepopup" href="#table_body_display_t2-remotesensing-05-04163"></a>
</div>
</div>

</div><p>During preliminary analysis we determined that optical imagery 
did not improve the delineation of the tree crowns. We also did not use 
the lidar intensity as it was inconsistent across sensor acquisitions 
and varied depending on the gain control settings of the lidar sensor [<a href="#b55-remotesensing-05-04163" class="html-bibr" title="">55</a>,<a href="#b56-remotesensing-05-04163" class="html-bibr" title="">56</a>], the incidence angle, atmospheric dispersion and absorption, and bidirectional reflectance coefficients [<a href="#b57-remotesensing-05-04163" class="html-bibr" title="">57</a>]. The segmentation was based on nine 90 × 90 m squares (8,100 m<sup>2</sup>)
 extracted from the corrected CHMs at plot centers to avoid any edge 
effects due to the raster filtering. The segments were smoothed by 
eCognition to remove the staircase-boundary effect, so that the polygon 
boundary no longer followed the edges of individual pixels.</p></section><section id="PredictingSpecies" type=""><h4 class="html-italic" data-nested="2"> 2.9. Predicting Species</h4><p>We used Waikato Environment for Knowledge Analysis (WEKA) software [<a href="#b58-remotesensing-05-04163" class="html-bibr" title="">58</a>] to classify the segments into four individual tree species (<span class="html-italic">Abies concolor, Pinus lambertiana, Pinus ponderosa, Psuedotsuga menziesii</span>) and species types (e.g., pines <span class="html-italic">vs.</span>
 firs). WEKA is a collection of machine learning algorithms for data 
mining tasks developed by the Machine Learning Group at the University 
of Waikato. WEKA has proven to be an effective tool in classification, 
especially when many predictive attributes are available [<a href="#b59-remotesensing-05-04163" class="html-bibr" title="">59</a>]. Two species, <span class="html-italic">Calocedrus decurrens</span>, and <span class="html-italic">Quercus kelloggii</span>, were eliminated from the analysis because of low sample size (n = 4, each).</p><p>We
 extracted a number of object metrics to feed the classification 
routines, including object shape, lidar, and multispectral image metrics
 (<a href="#table_body_display_t2-remotesensing-05-04163" class="html-table html-tablepopup">Table 2</a>:
 classification). Additionally, we ran a feature optimization algorithm 
based on Pearson’s correlation coefficient to identify the most 
influential metrics (as measured by the correlation) and to avoid 
overfitting the classification models. All metrics were normalized prior
 to the classification. We identified two classifiers that best 
predicted the objects: LibLINEAR for individual species and the radial 
basis function (RBF) neural network for species type. LibLINEAR is a 
library for large-scale linear classification that supports logistic 
regression and linear support vector machines (SVM) [<a href="#b60-remotesensing-05-04163" class="html-bibr" title="">60</a>].
 We used the SVM classifier for predicting the tree species. An SVM 
classifier uses data close to the support vectors (class boundaries) to 
fit a hyperplane to the training data within the feature Space [<a href="#b61-remotesensing-05-04163" class="html-bibr" title="">61</a>]. The RBF network uses <span class="html-italic">k</span>-means
 clustering to determine basis functions and then calculates logistic 
regression based on these functions. Multivariate Gaussian functions are
 then fitted to all classes [<a href="#b62-remotesensing-05-04163" class="html-bibr" title="">62</a>].</p></section><section id="ComparisonofSegments" type=""><h4 class="html-italic" data-nested="2"> 2.10. Comparison of Segments</h4><p>We
 compared the two segmentation approaches to each other and against the 
ground reference data. In validating the results to georeferenced trees,
 we evaluated the estimation of tree heights, species, and how tree 
detection varies across tree crown classes (e.g., dominant, 
intermediate, <span class="html-italic">etc</span>.).</p><p>In order to 
compare the two types of polygons, we first matched their tree height 
predictions to ensure that correct pairs of polygons were used for the 
comparison. Next, we compared the polygon shape characteristics by 
considering their area, shape index, and compactness. Shape index is a 
widely applicable measure of shape complexity as compared to a square: a
 higher index indicates a more complex polygon [<a href="#b63-remotesensing-05-04163" class="html-bibr" title="">63</a>,<a href="#b64-remotesensing-05-04163" class="html-bibr" title="">64</a>]. Jiao and Liu [<a href="#b65-remotesensing-05-04163" class="html-bibr" title="">65</a>]
 found that, among other shape metrics, shape index is an effective 
descriptor of complexity. Similarly, compactness is a measure of 
complexity as compared to a circle [<a href="#b66-remotesensing-05-04163" class="html-bibr" title="">66</a>]. Both metrics are described in <a href="#FD9" class="html-disp-formula">Equations (9)</a> and <a href="#FD10" class="html-disp-formula">(10)</a>, where <span class="html-italic">SI</span> is shape index, <span class="html-italic">C</span> is compactness, <span class="html-italic">P</span> is perimeter, and <span class="html-italic">A</span> is area.
</p><div class="html-disp-formula-info" id="FD9">
<div class="f">
<span class="MathJax_Preview"><math display="block">
<mrow>
<mi mathvariant="italic">SI</mi>
<mo>=</mo>
<mfrac>
<mi>P</mi>
<mrow>
<mn>4</mn>
<msqrt>
<mi>A</mi></msqrt></mrow></mfrac>
<mo>,</mo>
<mtext>where</mtext>
<mo> </mo>
<mi mathvariant="italic">SI</mi>
<mo>∈</mo>
<mrow>
<mo stretchy="false">[</mo>
<mrow>
<mrow>
<mrow>
<mfrac>
<mrow>
<msqrt>
<mi>π</mi></msqrt></mrow>
<mn>2</mn></mfrac>
<mo>,</mo>
<mo>∞</mo></mrow>
<mo stretchy="false">)</mo></mrow>
<mo>,</mo></mrow></mrow></mrow></math></span><script type="math/mml" id="MathJax-Element-9"><math display="block">
<mrow>
<mi mathvariant="italic">SI</mi>
<mo>=</mo>
<mfrac>
<mi>P</mi>
<mrow>
<mn>4</mn>
<msqrt>
<mi>A</mi></msqrt></mrow></mfrac>
<mo>,</mo>
<mtext>where</mtext>
<mo> </mo>
<mi mathvariant="italic">SI</mi>
<mo>∈</mo>
<mrow>
<mo stretchy="false">[</mo>
<mrow>
<mrow>
<mrow>
<mfrac>
<mrow>
<msqrt>
<mi>π</mi></msqrt></mrow>
<mn>2</mn></mfrac>
<mo>,</mo>
<mo>∞</mo></mrow>
<mo stretchy="false">)</mo></mrow>
<mo>,</mo></mrow></mrow></mrow></math></script>
</div>
<div class="l">
<label>(9)</label>
</div>
</div><div class="html-disp-formula-info" id="FD10">
<div class="f">
<span class="MathJax_Preview"><math display="block">
<mrow>
<mi>C</mi>
<mo>=</mo>
<mfrac>
<mrow>
<mn>4</mn>
<mi>π</mi>
<mi>A</mi></mrow>
<mrow>
<msup>
<mrow>
<mi>P</mi></mrow>
<mn>2</mn></msup></mrow></mfrac>
<mo>,</mo>
<mtext>where</mtext>
<mo> </mo>
<mi>C</mi>
<mo>∈</mo>
<mo stretchy="false">(</mo>
<mn>0</mn>
<mo>,</mo>
<mn>1</mn>
<mo stretchy="false">]</mo>
<mo>,</mo></mrow></math></span><script type="math/mml" id="MathJax-Element-10"><math display="block">
<mrow>
<mi>C</mi>
<mo>=</mo>
<mfrac>
<mrow>
<mn>4</mn>
<mi>π</mi>
<mi>A</mi></mrow>
<mrow>
<msup>
<mrow>
<mi>P</mi></mrow>
<mn>2</mn></msup></mrow></mfrac>
<mo>,</mo>
<mtext>where</mtext>
<mo> </mo>
<mi>C</mi>
<mo>∈</mo>
<mo stretchy="false">(</mo>
<mn>0</mn>
<mo>,</mo>
<mn>1</mn>
<mo stretchy="false">]</mo>
<mo>,</mo></mrow></math></script>
</div>
<div class="l">
<label>(10)</label>
</div>
</div><p></p><p>Since the lidar segmentation generated fewer polygons 
than OBIA, we used the former as a basis for comparison in order to 
avoid double-counting of the polygons. In particular, the tree top 
points as detected by the lidar segmentation algorithms were used to 
intersect and collect information from both types of polygons. All 
polygons that contact the 90 m CHM edges were eliminated from the 
analysis resulting in n = 949 (<a href="#fig_body_display_f6-remotesensing-05-04163" class="html-fig html-figpopup">Figure 6</a>).</p><div class="html-fig-wrap" id="f6-remotesensing-05-04163">
<div class="html-fig_img">
<div class="html-figpopup html-figpopup-link" href="#fig_body_display_f6-remotesensing-05-04163">
<img alt="Remotesensing 05 04163f6 1024" data-large="/remotesensing/remotesensing-05-04163/article_deploy/html/images/remotesensing-05-04163f6-1024.png" data-original="/remotesensing/remotesensing-05-04163/article_deploy/html/images/remotesensing-05-04163f6.png" src="remotesensing-05-04163f6-1024">
<a class="html-expand html-figpopup" href="#fig_body_display_f6-remotesensing-05-04163"></a>
</div>
</div>
<div class="html-fig_description">
<b>Figure 6.</b>
Each 90 m focal area was centered at the plot center. The ground 
reference data included overstory trees (blue dots), overstory dominant 
and co-dominant trees (red dots), and marker trees (white). The blue 
(lidar-derived) and white (OBIA-derived) polygons show results of the 
data segmentation, while the crosses indicate the positions of tree tops
 as detected by the 3D segmentation algorithm. Tree heights (meters) are
 indicated next to ground-referenced trees.

</div>
</div></section></section><section id="Results" type="other"><h2 data-nested="1"> 3. Results</h2><p>We
 used an OBIA approach to segment the CHM and a point cloud segmentation
 algorithm to segment the lidar data into individual trees. Segments at 
the edges of the considered areas were excluded from any of the 
analyses. Across all nine 90 × 90 m areas, a total of 2,875 objects were
 delineated using OBIA and 949 based on the lidar point cloud analysis. 
The OBIA method over-segmented many large trees, although in some cases,
 it correctly delineated smaller trees that the lidar approach had 
missed. This was especially true when the smaller trees were surrounded 
by large, taller species. Overall, the lidar-derived method produced 
larger polygons, especially for the tall and large trees. The agreements
 of tree height estimation between the ground reference data and the 
generated polygons were high for both methods. All dominant ground 
referenced trees were detected by both of the approaches; the tree 
detection rates dropped considerably for intermediate and suppressed 
trees, as well as for dead trees. Tree height accuracy declined in areas
 of high tree density.</p><section id="PolygonsGroundReferenceData" type=""><h4 class="html-italic" data-nested="2"> 3.1. Polygons <span class="html-italic">vs.</span> Ground Reference Data</h4><p>We
 intersected the ground reference tree points with lidar and OBIA 
polygons to validate the results. We cross referenced the tree heights 
measured on the ground against the maximum lidar <span class="html-italic">z</span>-value
 points within the lidar polygons, and the maximum pixel value of the 
corrected CHM within the OBIA polygons. In cases where there were more 
than one ground reference points within a polygon, we kept only the 
point that best matched the CHM height (n = 106). Pearson’s squared 
correlation coefficients for the lidar polygons and for the OBIA 
polygons <span class="html-italic">vs.</span> ground measurements were r<sup>2</sup> = 0.9163 and r<sup>2</sup> = 0.9309, respectively (<a href="#fig_body_display_f7-remotesensing-05-04163" class="html-fig html-figpopup">Figure 7</a>).</p><div class="html-fig-wrap" id="f7-remotesensing-05-04163">
<div class="html-fig_img">
<div class="html-figpopup html-figpopup-link" href="#fig_body_display_f7-remotesensing-05-04163">
<img alt="Remotesensing 05 04163f7 1024" data-large="/remotesensing/remotesensing-05-04163/article_deploy/html/images/remotesensing-05-04163f7-1024.png" data-original="/remotesensing/remotesensing-05-04163/article_deploy/html/images/remotesensing-05-04163f7.png" src="remotesensing-05-04163f7-1024">
<a class="html-expand html-figpopup" href="#fig_body_display_f7-remotesensing-05-04163"></a>
</div>
</div>
<div class="html-fig_description">
<b>Figure 7.</b>
Ground reference data <span class="html-italic">vs</span>. the lidar-derived (<b>A</b>) and OBIA-derived (<b>B</b>) tree heights.

</div>
</div><p>We also analyzed the rate of tree detection by both methods in various crown classes (<a href="#fig_body_display_f8-remotesensing-05-04163" class="html-fig html-figpopup">Figure 8</a>).
 All dominant overstory trees (DBH ≥ 19.5 cm) were detected by both of 
the segmentation approaches (n = 16). However, the rate of detection 
dropped when trees were occluded by taller or bigger trees. The OBIA 
method detected a larger percentage of the non-dominant trees. Both 
methods detected less than 50 percent of dead trees.</p><div class="html-fig-wrap" id="f8-remotesensing-05-04163">
<div class="html-fig_img">
<div class="html-figpopup html-figpopup-link" href="#fig_body_display_f8-remotesensing-05-04163">
<img alt="Remotesensing 05 04163f8 1024" data-large="/remotesensing/remotesensing-05-04163/article_deploy/html/images/remotesensing-05-04163f8-1024.png" data-original="/remotesensing/remotesensing-05-04163/article_deploy/html/images/remotesensing-05-04163f8.png" src="remotesensing-05-04163f8-1024">
<a class="html-expand html-figpopup" href="#fig_body_display_f8-remotesensing-05-04163"></a>
</div>
</div>
<div class="html-fig_description">
<b>Figure 8.</b>
Overstory tree (DBH ≥ 19.5 cm) detection rates, classified by crown 
class. The detection rates of the dominant trees dropped considerably in
 the less-dominant tree crowns. Dead trees, including snags, were 
detected less than 50 percent of the time by both methods.

</div>
</div><p>We used the tree-top points, as detected by the lidar point 
cloud segmentation algorithm, to cross reference the lidar and OBIA 
polygons. We present the results as a function of (1) the method used to
 derive the polygons and (2) tree density.</p><p>The tree heights derived by both methods correlate very well (r<sup>2</sup> = 0.9545) indicating that the polygons detected the same trees (<a href="#fig_body_display_f9-remotesensing-05-04163" class="html-fig html-figpopup">Figure 9</a>).
 The OBIA approach estimated higher tree heights when predicting the 
tallest trees. The area, shape index, and compactness metrics differed 
significantly between the two methods. The area plot (<a href="#fig_body_display_f9-remotesensing-05-04163" class="html-fig html-figpopup">Figure 9B</a>)
 clearly reveals that the lidar-derived polygons tend to be larger than 
the OBIA polygons. The range of lidar polygons is twice that of the OBIA
 polygons. In general, the compactness and shape index plots indicate 
that the lidar polygons are more complex. Both approaches produced 
polygons that were much more complex than simple geometric polygons 
(circle or a square).</p><div class="html-fig-wrap" id="f9-remotesensing-05-04163">
<div class="html-fig_img">
<div class="html-figpopup html-figpopup-link" href="#fig_body_display_f9-remotesensing-05-04163">
<img alt="Remotesensing 05 04163f9a 1024" data-large="/remotesensing/remotesensing-05-04163/article_deploy/html/images/remotesensing-05-04163f9a-1024.png" data-original="/remotesensing/remotesensing-05-04163/article_deploy/html/images/remotesensing-05-04163f9a.png" src="remotesensing-05-04163f9a-1024"><img alt="Remotesensing 05 04163f9b 1024" data-large="/remotesensing/remotesensing-05-04163/article_deploy/html/images/remotesensing-05-04163f9b-1024.png" data-original="/remotesensing/remotesensing-05-04163/article_deploy/html/images/remotesensing-05-04163f9b.png" src="remotesensing-05-04163f9b-1024">
<a class="html-expand html-figpopup" href="#fig_body_display_f9-remotesensing-05-04163"></a>
</div>
</div>
<div class="html-fig_description">
<b>Figure 9.</b>
(<b>A</b>) The tree-heights derived by the two methods were more similar than (<b>B</b>) their shape in terms of the area or (<b>C</b>,<b>D</b>)shape complexity.

</div>
</div></section><section id="TreeDensityEffects" type=""><h4 class="html-italic" data-nested="2"> 3.2. Tree Density Effects</h4><p>As
 expected, the tree density also affected tree height correlations. We 
summarize these effects with squared correlation coefficients as a 
function of tree density (<a href="#fig_body_display_f10-remotesensing-05-04163" class="html-fig html-figpopup">Figure 10</a>).
 Both methods performed best in areas with medium tree density when 
predicting the tree height. However, the OBIA method worked slightly 
better in low density areas, and considerably worse in high density 
areas.</p><div class="html-fig-wrap" id="f10-remotesensing-05-04163">
<div class="html-fig_img">
<div class="html-figpopup html-figpopup-link" href="#fig_body_display_f10-remotesensing-05-04163">
<img alt="Remotesensing 05 04163f10 1024" data-large="/remotesensing/remotesensing-05-04163/article_deploy/html/images/remotesensing-05-04163f10-1024.png" data-original="/remotesensing/remotesensing-05-04163/article_deploy/html/images/remotesensing-05-04163f10.png" src="remotesensing-05-04163f10-1024">
<a class="html-expand html-figpopup" href="#fig_body_display_f10-remotesensing-05-04163"></a>
</div>
</div>
<div class="html-fig_description">
<b>Figure 10.</b>
The tree height agreement between ground reference data and derived polygons decreased in high tree density areas (<b>A</b>).
 The area, shape complexity, and predicted tree heights of the derived 
polygons were more different in the high tree-density areas (<b>B</b>).

</div>
</div><p>The agreement between tree height predictions slightly declined
 in high tree density areas, but for the most part remained constant (r<sup>2</sup> = 0.9596 <span class="html-italic">vs.</span> r<sup>2</sup>
 = 0.9401 in low and high tree densities, respectively). The shape 
characteristics of the two polygon types were less correlated than the 
tree heights. The area correlation declined at high tree densities, 
where the lidar polygons were consistently larger than the OBIA 
polygons. OBIA polygons increased in complexity at higher tree densities
 and were therefore less similar to the lidar polygons.</p></section><section id="PredictingTreeSpecies" type=""><h4 class="html-italic" data-nested="2"> 3.3. Predicting Tree Species</h4><p>We used all polygon features (<a href="#table_body_display_t2-remotesensing-05-04163" class="html-table html-tablepopup">Table 2</a>)
 to predict individual species using the SVM algorithm from the 
LibLINEAR approach. The individual species predictions were poor: only 
56 percent were correctly classified. However, the RBF neural network 
algorithm was able to correctly classify up to 75 percent of the species
 types (e.g., pines <span class="html-italic">vs.</span> firs). In the latter case, we used only six predictive metrics based on the suggestion from a feature optimization algorithm.</p></section></section><section id="Discussion" type="other"><h2 data-nested="1"> 4. Discussion</h2><p>The
 high Pearson’s squared correlation coefficients between tree polygons 
and the ground reference data indicate that both lidar and OBIA methods 
detected individual trees well: r<sup>2</sup> = 0.9163 for the lidar-derived and r<sup>2</sup>
 = 0.9309 for the OBIA-derived polygons. Further, the good fit with the 
OBIA approach also indicates that the CHM correction method worked well.
 While the lidar segmentation polygons were usually detected and 
delineated correctly, the locations of the tops of trees did not always 
match those of the ground reference data (<a href="#fig_body_display_f6-remotesensing-05-04163" class="html-fig html-figpopup">Figure 6</a>).
 This may be a result of trees that are leaning or not straight, as 
ground-reference positions were measured at the tree trunks, at 
breast-height level (1.3 m from the ground), as suggested by Brokaw and 
Thompson [<a href="#b67-remotesensing-05-04163" class="html-bibr" title="">67</a>].</p><p>All of the dominant and most of the co-dominant trees were detected by both methods (<a href="#fig_body_display_f8-remotesensing-05-04163" class="html-fig html-figpopup">Figure 8</a>).
 Although the OBIA approach had a higher detection rate for non-dominant
 trees than the lidar approach, this is likely due to the 
oversegmentation found with the OBIA approach. Consequently, these 
optimistic OBIA results should be explored further. This caution is 
supported by the tree height <span class="html-italic">vs.</span> ground data fits: the overall fit of the lidar-derived tree heights is tighter (<a href="#fig_body_display_f7-remotesensing-05-04163" class="html-fig html-figpopup">Figure 7</a>). The dominant tree detection rates of both methods were higher than those previously reported [<a href="#b28-remotesensing-05-04163" class="html-bibr" title="">28</a>] possibly due to the higher point density in our data.</p><p>In
 comparing the ability of each method to delineate trees, one conclusion
 clearly stands out: while the detection rate and evaluated tree heights
 were comparable, the shapes of the derived objects were very different.
 Besides a few outlier points, the main discrepancy between lidar- and 
OBIA-derived tree heights was due to the tallest trees (&gt;50 m), where
 the OBIA approach tended to overestimate height. This is likely due to 
the constants, <span class="html-italic">a</span>, <span class="html-italic">b</span> and <span class="html-italic">c</span>, used to correct the CHM surface. Previous studies [<a href="#b5-remotesensing-05-04163" class="html-bibr" title="">5</a>,<a href="#b19-remotesensing-05-04163" class="html-bibr" title="">19</a>]
 have shown that tree heights derived from traditional CHM tend to be 
underestimated and thus, future research should evaluate whether the 
tall-tree heights derived from the corrected CHM are true.</p><p>It is clear from visual inspection that the lidar and OBIA polygons differ in shape (<a href="#fig_body_display_f6-remotesensing-05-04163" class="html-fig html-figpopup">Figure 6</a>). This was further verified by the comparison of their areas and shape complexities (<a href="#fig_body_display_f9-remotesensing-05-04163" class="html-fig html-figpopup">Figure 9</a> and <a href="#fig_body_display_f10-remotesensing-05-04163" class="html-fig html-figpopup">10</a>).
 One explanation for the compactness of the OBIA polygons is that the 
segmentation algorithm does not allow polygon overlap. In contrast, the 
lidar-derived tree canopy polygons often overlapped because lidar points
 classified as one tree may be on top of, underneath, or within another 
tree in the <span class="html-italic">x-y</span> space (<a href="#fig_body_display_f4-remotesensing-05-04163" class="html-fig html-figpopup">Figure 4</a>).
 This is a more realistic representation of real forest canopy, 
especially in complex, dense forests such as those in our study area. 
Another reason for why the lidar polygons were more complex may be 
because of the way the concave hulls were constructed. Concave hulls 
that do not follow the lidar points as closely (by changing the <span class="html-italic">α</span> parameter when generating an alpha-shape) would lead to a simpler tree crown boundary and thus a simpler shape.</p><p>In general, the lidar polygons were larger than the OBIA polygons, as is evident in the area comparison plot (<a href="#fig_body_display_f9-remotesensing-05-04163" class="html-fig html-figpopup">Figure 9</a>).
 Effectively, this means that the lidar segmentation method tended to 
under-segment and under-detect trees, while the OBIA method 
over-segmented the trees. In particular, the lidar segmentation 
algorithm sometimes erroneously grouped trees into one segment when the 
trees were very close together (<span class="html-italic">i.e.</span>, their crowns were merged or intertwined) or when the tops of the trees were not clearly detectable [<a href="#b33-remotesensing-05-04163" class="html-bibr" title="">33</a>].
 Correct segmentation in such circumstances (especially when canopy 
cover &gt; 50 percent) is still problematic regardless of the used 
method; it is an active area of research [<a href="#b5-remotesensing-05-04163" class="html-bibr" title="">5</a>].
 This is further evident when taking the tree density into 
consideration. In high tree density areas where trees grow in clusters, 
the disagreement between the area of the lidar- and OBIA-derived 
polygons increased and the overall accuracy of tree heights (as compared
 to ground truth) decreased (<a href="#fig_body_display_f10-remotesensing-05-04163" class="html-fig html-figpopup">Figure 10</a>). This is consistent with previous research where individual tree detection accuracy decreased in densely vegetated areas [<a href="#b26-remotesensing-05-04163" class="html-bibr" title="">26</a>].
 The lidar polygons were created from concave hulls around the segmented
 lidar points, and this could be another reason for why the lidar 
polygons were larger.</p><p>The low accuracy of individual tree species 
classification is not surprising considering the study area. Even on the
 ground, the four classified species (Douglas fir, white fir, sugar 
pine, and ponderosa pine) can be easily misidentified by an untrained 
eye. On the other hand, the overall shape of firs <span class="html-italic">vs.</span>
 pines is more discernable, and more accurately classified: the firs are
 more conical and the pines are more oval, especially in mature species.
 This is evident in the improved accuracy of tree type classification 
for firs <span class="html-italic">vs.</span> pines. Including 
hyperspectral data as a predictor would likely improve the results even 
further, and possibly help with the individual tree species 
classification. It is also possible that the classification results 
would improve had the lidar and WorldView-2 data been collected in 
closer temporal proximity. Although we took great care to spatially 
match the two datasets, and no major ecological event occurred in 
between the two data collections, it is possible that lidar and optical 
data collected simultaneously would yield better classification of the 
species. We should note that many other algorithms were tested during 
the preliminary analysis, including rule-based, decision tree and other 
machine learning classifiers; only the best results are presented here.</p><p>One
 of key challenges during the OBIA CHM segmentation process was the 
assignment of the correct scale, shape, and compactness parameters to 
obtain meaningful objects. Using previously established methods [<a href="#b54-remotesensing-05-04163" class="html-bibr" title="">54</a>] to determine the scale factor helped in the choice of these parameters; however, as Suarez, <span class="html-italic">et al</span>. [<a href="#b68-remotesensing-05-04163" class="html-bibr" title="">68</a>]
 discusses, designing an effective rule-set is still a largely manual 
process with potentially unlimited trial-and-error iterations. Efforts 
in future OBIA research will likely concentrate on designing processes 
to make the segmentation process less reliant on analyst input, and thus
 more deterministic and repeatable.</p></section><section id="Conclusions" type="other"><h2 data-nested="1"> 5. Conclusions</h2><p>The
 successful detection and delineation of individual trees is critical in
 forest science, allowing for multi-scale analysis of the role of trees 
in forest functioning. There are many possible methods to delineate 
trees with remotely sensed data. In this work, we compared two 
segmentation algorithms for individual tree delineation: a lidar-derived
 method that uses 3D lidar points, and an OBIA approach that make use of
 the lidar canopy height model. First, we segmented a lidar point cloud 
into trees using a 3D segmentation algorithm. Next, we developed a new 
technique to produce a CHM without common lidar artifacts in order to 
segment it using an OBIA-derived method. The two methods were compared 
in <span class="html-italic">x-y</span> space in terms of their 
agreement to ground referenced tree heights, as well as tree detection 
across crown class and tree density. The overall tree height agreements 
of the OBIA- and the lidar-derived polygons were high (r<sup>2</sup> = 0.9309 and r<sup>2</sup>
 = 0.9163, respectively) and decreased in densely vegetated areas. The 
two types of objects were different in terms of polygon area and shape 
complexity. The OBIA objects were more likely to over-segment while the 
lidar objects were more likely to under-segment the trees, although the 
latter produced polygons more similar in shape to real tree crowns. 
Further research is necessary to automate the OBIA segmentation process 
and to improve 3D segmentation of the point cloud in dense forests.</p></section>
</div>
<div class="html-back">
<section id="html-ack" class="html-ack"><h2>Acknowledgments</h2><p>This 
is SNAMP Publication #24. The Sierra Nevada Adaptive Management Project 
is funded by USDA Forest Service Region 5, USDA Forest Service Pacific 
Southwest Research Station, US Fish and Wildlife Service, California 
Department of Water Resources, California Department of Fish and Game, 
California Department of Forestry and Fire Protection, and the Sierra 
Nevada Conservancy.</p></section><section id="html-notes" class="html-notes"><h2>Conflict of Interest</h2><p>The authors declare no conflict of interest.</p></section><section id="html-references_list"><h2>References and Notes</h2><ol class="html-xx"><li id="b1-remotesensing-05-04163" class="html-x" data-content="1.">Hyyppä,
 J.; Hyyppä, H.; Yu, X.; Kaartinen, H.; Kukko, A.; Holopainen, M. Forest
 Inventory Using Small-Footprint Airborne Lidar. In <span class="html-italic">Topographic Laser Ranging and Scanning: Principles and Processing</span>; Shan, J., Toth, C.K., Eds.; Taylor &amp; Francis Group: New York, NY, USA, 2009; pp. 335–370. [<a href="https://scholar.google.com/scholar_lookup?title=Forest%20Inventory%20Using%20Small-Footprint%20Airborne%20Lidar&amp;author=Hyypp%C3%A4,+J.&amp;author=Hyypp%C3%A4,+H.&amp;author=Yu,+X.&amp;author=Kaartinen,+H.&amp;author=Kukko,+A.&amp;author=Holopainen,+M&amp;publication_year=2009&amp;pages=335%E2%80%93370" class="google-scholar" target="_blank">Google Scholar</a>]</li><li id="b2-remotesensing-05-04163" class="html-x" data-content="2.">Jensen, J.R. <span class="html-italic">Remote Sensing of the Environment: An Earth Resource Perspective</span>, 2nd ed.; Prentice Hall: Upper Saddle River, NJ, USA, 2007; pp. 1–591. [<a href="https://scholar.google.com/scholar_lookup?title=Remote+Sensing+of+the+Environment:+An+Earth+Resource+Perspective&amp;author=Jensen,+J.R.&amp;publication_year=2007" class="google-scholar" target="_blank">Google Scholar</a>]</li><li id="b3-remotesensing-05-04163" class="html-x" data-content="3.">Lefsky,
 M.A.; Cohen, W.B.; Harding, D.J.; Parker, G.G.; Acker, S.A.; Gower, 
S.T. Lidar remote sensing of above-ground biomass in three biomes. <span class="html-italic">Glob. Ecol. Biogeogr</span> <b>2002</b>, <span class="html-italic">11</span>, 393–399. [<a href="https://scholar.google.com/scholar_lookup?title=Lidar%20remote%20sensing%20of%20above-ground%20biomass%20in%20three%20biomes&amp;author=Lefsky,+M.A.&amp;author=Cohen,+W.B.&amp;author=Harding,+D.J.&amp;author=Parker,+G.G.&amp;author=Acker,+S.A.&amp;author=Gower,+S.T.&amp;publication_year=2002&amp;journal=Glob.+Ecol.+Biogeogr&amp;volume=11&amp;pages=393%E2%80%93399" class="google-scholar" target="_blank">Google Scholar</a>]</li><li id="b4-remotesensing-05-04163" class="html-x" data-content="4.">Clark,
 M.L.; Clark, D.B.; Roberts, D.A. Small-footprint lidar estimation of 
sub-canopy elevation and tree height in a tropical rain forest 
landscape. <span class="html-italic">Remote Sens. Environ</span> <b>2004</b>, <span class="html-italic">91</span>, 68–89. [<a href="https://scholar.google.com/scholar_lookup?title=Small-footprint%20lidar%20estimation%20of%20sub-canopy%20elevation%20and%20tree%20height%20in%20a%20tropical%20rain%20forest%20landscape&amp;author=Clark,+M.L.&amp;author=Clark,+D.B.&amp;author=Roberts,+D.A.&amp;publication_year=2004&amp;journal=Remote+Sens.+Environ&amp;volume=91&amp;pages=68%E2%80%9389" class="google-scholar" target="_blank">Google Scholar</a>]</li><li id="b5-remotesensing-05-04163" class="html-x" data-content="5.">Falkowski,
 M.J.; Smith, A.M.S.; Gessler, P.E.; Hudak, A.T.; Vierling, L.A.; Evans,
 J.S. The influence of conifer forest canopy cover on the accuracy of 
two individual tree measurement algorithms using lidar data. <span class="html-italic">Can. J. Remote Sens</span> <b>2008</b>, <span class="html-italic">34</span>, 338–350. [<a href="https://scholar.google.com/scholar_lookup?title=The%20influence%20of%20conifer%20forest%20canopy%20cover%20on%20the%20accuracy%20of%20two%20individual%20tree%20measurement%20algorithms%20using%20lidar%20data&amp;author=Falkowski,+M.J.&amp;author=Smith,+A.M.S.&amp;author=Gessler,+P.E.&amp;author=Hudak,+A.T.&amp;author=Vierling,+L.A.&amp;author=Evans,+J.S.&amp;publication_year=2008&amp;journal=Can.+J.+Remote+Sens&amp;volume=34&amp;pages=338%E2%80%93350" class="google-scholar" target="_blank">Google Scholar</a>]</li><li id="b6-remotesensing-05-04163" class="html-x" data-content="6.">Vepakomma,
 U.; St-Onge, B.; Kneeshaw, D. Response of a boreal forest to canopy 
opening: assessing vertical and lateral tree growth with multi-temporal 
lidar data. <span class="html-italic">Ecol. Appl</span> <b>2011</b>, <span class="html-italic">21</span>, 99–121. [<a href="https://scholar.google.com/scholar_lookup?title=Response%20of%20a%20boreal%20forest%20to%20canopy%20opening:%20assessing%20vertical%20and%20lateral%20tree%20growth%20with%20multi-temporal%20lidar%20data&amp;author=Vepakomma,+U.&amp;author=St-Onge,+B.&amp;author=Kneeshaw,+D.&amp;publication_year=2011&amp;journal=Ecol.+Appl&amp;volume=21&amp;pages=99%E2%80%93121" class="google-scholar" target="_blank">Google Scholar</a>]</li><li id="b7-remotesensing-05-04163" class="html-x" data-content="7.">Garcia-Feced,
 C.; Temple, D.; Kelly, M. Characterizing California Spotted Owl nest 
sites and their associated forest stands using Lidar data. <span class="html-italic">J. For</span> <b>2011</b>, <span class="html-italic">108</span>, 436–443. [<a href="https://scholar.google.com/scholar_lookup?title=Characterizing%20California%20Spotted%20Owl%20nest%20sites%20and%20their%20associated%20forest%20stands%20using%20Lidar%20data&amp;author=Garcia-Feced,+C.&amp;author=Temple,+D.&amp;author=Kelly,+M.&amp;publication_year=2011&amp;journal=J.+For&amp;volume=108&amp;pages=436%E2%80%93443" class="google-scholar" target="_blank">Google Scholar</a>]</li><li id="b8-remotesensing-05-04163" class="html-x" data-content="8.">Zhao,
 F.; Sweitzer, R.A.; Guo, Q.; Kelly, M. Characterizing habitats 
associated with fisher den structures in the Southern Sierra Nevada, 
California using discrete return lidar. <span class="html-italic">Forest Ecol. Manag</span> <b>2012</b>, <span class="html-italic">280</span>, 112–119. [<a href="https://scholar.google.com/scholar_lookup?title=Characterizing%20habitats%20associated%20with%20fisher%20den%20structures%20in%20the%20Southern%20Sierra%20Nevada,%20California%20using%20discrete%20return%20lidar&amp;author=Zhao,+F.&amp;author=Sweitzer,+R.A.&amp;author=Guo,+Q.&amp;author=Kelly,+M.&amp;publication_year=2012&amp;journal=Forest+Ecol.+Manag&amp;volume=280&amp;pages=112%E2%80%93119" class="google-scholar" target="_blank">Google Scholar</a>]</li><li id="b9-remotesensing-05-04163" class="html-x" data-content="9.">Popescu,
 S.C.; Wynne, R.H.; Nelson, R.F. Measuring individual tree crown 
diameter with lidar and assessing its influence on estimating forest 
volume and biomass. <span class="html-italic">Can. J. Remote Sens</span> <b>2003</b>, <span class="html-italic">29</span>, 564–577. [<a href="https://scholar.google.com/scholar_lookup?title=Measuring%20individual%20tree%20crown%20diameter%20with%20lidar%20and%20assessing%20its%20influence%20on%20estimating%20forest%20volume%20and%20biomass&amp;author=Popescu,+S.C.&amp;author=Wynne,+R.H.&amp;author=Nelson,+R.F.&amp;publication_year=2003&amp;journal=Can.+J.+Remote+Sens&amp;volume=29&amp;pages=564%E2%80%93577" class="google-scholar" target="_blank">Google Scholar</a>]</li><li id="b10-remotesensing-05-04163" class="html-xx" data-content="10.">Zhao,
 F.; Guo, Q.; Kelly, M. Allometric equation choice impacts lidar-based 
forest biomass estimates: A case study from the Sierra National Forest, 
CA. <span class="html-italic">Agric. For. Meteorol</span> <b>2012</b>, <span class="html-italic">165</span>, 64–72. [<a href="https://scholar.google.com/scholar_lookup?title=Allometric%20equation%20choice%20impacts%20lidar-based%20forest%20biomass%20estimates:%20A%20case%20study%20from%20the%20Sierra%20National%20Forest,%20CA&amp;author=Zhao,+F.&amp;author=Guo,+Q.&amp;author=Kelly,+M.&amp;publication_year=2012&amp;journal=Agric.+For.+Meteorol&amp;volume=165&amp;pages=64%E2%80%9372" class="google-scholar" target="_blank">Google Scholar</a>]</li><li id="b11-remotesensing-05-04163" class="html-xx" data-content="11.">Sheng, Y.; Gong, P.; Biging, G.S. Model-based conifer-crown surface reconstruction from high-resolution aerial images. <span class="html-italic">Photogramm. Eng. Remote Sensing</span> <b>2001</b>, <span class="html-italic">67</span>, 957–965. [<a href="https://scholar.google.com/scholar_lookup?title=Model-based%20conifer-crown%20surface%20reconstruction%20from%20high-resolution%20aerial%20images&amp;author=Sheng,+Y.&amp;author=Gong,+P.&amp;author=Biging,+G.S.&amp;publication_year=2001&amp;journal=Photogramm.+Eng.+Remote+Sensing&amp;volume=67&amp;pages=957%E2%80%93965" class="google-scholar" target="_blank">Google Scholar</a>]</li><li id="b12-remotesensing-05-04163" class="html-xx" data-content="12.">Maltamo,
 M.; Mustonen, K.; Hyyppä, J.; Pitkänen, J.; Yu, X. The accuracy of 
estimating individual tree variables with airborne laser scanning in a 
boreal nature reserve. <span class="html-italic">Can. J. Forest Res</span> <b>2004</b>, <span class="html-italic">34</span>, 1791–1801. [<a href="https://scholar.google.com/scholar_lookup?title=The%20accuracy%20of%20estimating%20individual%20tree%20variables%20with%20airborne%20laser%20scanning%20in%20a%20boreal%20nature%20reserve&amp;author=Maltamo,+M.&amp;author=Mustonen,+K.&amp;author=Hyypp%C3%A4,+J.&amp;author=Pitk%C3%A4nen,+J.&amp;author=Yu,+X.&amp;publication_year=2004&amp;journal=Can.+J.+Forest+Res&amp;volume=34&amp;pages=1791%E2%80%931801" class="google-scholar" target="_blank">Google Scholar</a>]</li><li id="b13-remotesensing-05-04163" class="html-xx" data-content="13.">Pitkänen,
 J. Individual tree detection in digital aerial images by combining 
locally adaptive binarization and local maxima methods. <span class="html-italic">Can. J. Forest Res</span> <b>2001</b>, <span class="html-italic">31</span>, 832–844. [<a href="https://scholar.google.com/scholar_lookup?title=Individual%20tree%20detection%20in%20digital%20aerial%20images%20by%20combining%20locally%20adaptive%20binarization%20and%20local%20maxima%20methods&amp;author=Pitk%C3%A4nen,+J.&amp;publication_year=2001&amp;journal=Can.+J.+Forest+Res&amp;volume=31&amp;pages=832%E2%80%93844" class="google-scholar" target="_blank">Google Scholar</a>]</li><li id="b14-remotesensing-05-04163" class="html-xx" data-content="14.">Wang,
 L.; Gong, P.; Biging, G.S. Individual tree-crown delineation and 
treetop detection in high-spatial-resolution aerial imagery. <span class="html-italic">Photogramm. Eng. Remote Sensing</span> <b>2004</b>, <span class="html-italic">70</span>, 351–358. [<a href="https://scholar.google.com/scholar_lookup?title=Individual%20tree-crown%20delineation%20and%20treetop%20detection%20in%20high-spatial-resolution%20aerial%20imagery&amp;author=Wang,+L.&amp;author=Gong,+P.&amp;author=Biging,+G.S.&amp;publication_year=2004&amp;journal=Photogramm.+Eng.+Remote+Sensing&amp;volume=70&amp;pages=351%E2%80%93358" class="google-scholar" target="_blank">Google Scholar</a>]</li><li id="b15-remotesensing-05-04163" class="html-xx" data-content="15.">Brandtberg,
 T.; Walter, F. Automated delineation of individual tree crowns in high 
spatial resolution aerial images by multiple-scale analysis. <span class="html-italic">Mach. Vis. Appl</span> <b>1998</b>, <span class="html-italic">11</span>, 64–73. [<a href="https://scholar.google.com/scholar_lookup?title=Automated%20delineation%20of%20individual%20tree%20crowns%20in%20high%20spatial%20resolution%20aerial%20images%20by%20multiple-scale%20analysis&amp;author=Brandtberg,+T.&amp;author=Walter,+F.&amp;publication_year=1998&amp;journal=Mach.+Vis.+Appl&amp;volume=11&amp;pages=64%E2%80%9373" class="google-scholar" target="_blank">Google Scholar</a>]</li><li id="b16-remotesensing-05-04163" class="html-xx" data-content="16.">Gougeon, F.A.; Leckie, D.G. The individual tree crown approach applied to Ikonos images of a coniferous plantation area. <span class="html-italic">Photogramm. Eng. Remote Sensing</span> <b>2006</b>, <span class="html-italic">72</span>, 1287–1297. [<a href="https://scholar.google.com/scholar_lookup?title=The%20individual%20tree%20crown%20approach%20applied%20to%20Ikonos%20images%20of%20a%20coniferous%20plantation%20area&amp;author=Gougeon,+F.A.&amp;author=Leckie,+D.G.&amp;publication_year=2006&amp;journal=Photogramm.+Eng.+Remote+Sensing&amp;volume=72&amp;pages=1287%E2%80%931297" class="google-scholar" target="_blank">Google Scholar</a>]</li><li id="b17-remotesensing-05-04163" class="html-xx" data-content="17.">Kaartinen,
 H.; Hyyppä, J.; Yu, X.; Vastaranta, M.; Hyyppä, H.; Kukko, A.; 
Holopainen, M.; Heipke, C.; Hirschmugl, M.; Morsdorf, F.; Næsset, E.; 
Pitkänen, J.; Popescu, S.; Solberg, S.; Wolf, B.M.; Wu, J.-C. An 
International Comparison of Individual Tree Detection and Extraction 
Using Airborne Laser Scanning. <span class="html-italic">Remote Sens</span> <b>2012</b>, <span class="html-italic">4</span>, 950–974. [<a href="https://scholar.google.com/scholar_lookup?title=An%20International%20Comparison%20of%20Individual%20Tree%20Detection%20and%20Extraction%20Using%20Airborne%20Laser%20Scanning&amp;author=Kaartinen,+H.&amp;author=Hyypp%C3%A4,+J.&amp;author=Yu,+X.&amp;author=Vastaranta,+M.&amp;author=Hyypp%C3%A4,+H.&amp;author=Kukko,+A.&amp;author=Holopainen,+M.&amp;author=Heipke,+C.&amp;author=Hirschmugl,+M.&amp;author=Morsdorf,+F.&amp;publication_year=2012&amp;journal=Remote+Sens&amp;volume=4&amp;pages=950%E2%80%93974" class="google-scholar" target="_blank">Google Scholar</a>]</li><li id="b18-remotesensing-05-04163" class="html-xx" data-content="18.">Hyyppä, J.; Inkinen, M. Detecting and estimating attributes for single trees using laser scanner. <span class="html-italic">Photogramm. J. Finl</span> <b>1999</b>, <span class="html-italic">16</span>, 27–42. [<a href="https://scholar.google.com/scholar_lookup?title=Detecting%20and%20estimating%20attributes%20for%20single%20trees%20using%20laser%20scanner&amp;author=Hyypp%C3%A4,+J.&amp;author=Inkinen,+M.&amp;publication_year=1999&amp;journal=Photogramm.+J.+Finl&amp;volume=16&amp;pages=27%E2%80%9342" class="google-scholar" target="_blank">Google Scholar</a>]</li><li id="b19-remotesensing-05-04163" class="html-xx" data-content="19.">Popescu,
 S.C.; Wynne, R.H. Seeing the trees in the forest: using lidar and 
multispectral data fusion with local filtering and variable window size 
for estimating tree height. <span class="html-italic">Photogramm. Eng. Remote Sensing</span> <b>2004</b>, <span class="html-italic">70</span>, 589–604. [<a href="https://scholar.google.com/scholar_lookup?title=Seeing%20the%20trees%20in%20the%20forest:%20using%20lidar%20and%20multispectral%20data%20fusion%20with%20local%20filtering%20and%20variable%20window%20size%20for%20estimating%20tree%20height&amp;author=Popescu,+S.C.&amp;author=Wynne,+R.H.&amp;publication_year=2004&amp;journal=Photogramm.+Eng.+Remote+Sensing&amp;volume=70&amp;pages=589%E2%80%93604" class="google-scholar" target="_blank">Google Scholar</a>]</li><li id="b20-remotesensing-05-04163" class="html-xx" data-content="20.">Chen, Q.; Baldocchi, D.; Gong, P.; Kelly, M. Isolating individual trees in a savanna woodland using small footprint LIDAR data. <span class="html-italic">Photogramm. Eng. Remote Sensing</span> <b>2006</b>, <span class="html-italic">72</span>, 923–932. [<a href="https://scholar.google.com/scholar_lookup?title=Isolating%20individual%20trees%20in%20a%20savanna%20woodland%20using%20small%20footprint%20LIDAR%20data&amp;author=Chen,+Q.&amp;author=Baldocchi,+D.&amp;author=Gong,+P.&amp;author=Kelly,+M.&amp;publication_year=2006&amp;journal=Photogramm.+Eng.+Remote+Sensing&amp;volume=72&amp;pages=923%E2%80%93932" class="google-scholar" target="_blank">Google Scholar</a>]</li><li id="b21-remotesensing-05-04163" class="html-xx" data-content="21.">Morsdorf,
 F.; Meier, E.; Kötz, B.; Itten, K.I.; Dobbertin, M.; Allgöwer, B. 
LIDAR-based geometric reconstruction of boreal type forest stands at 
single tree level for forest and wildland fire management. <span class="html-italic">Remote Sens. Environ</span> <b>2004</b>, <span class="html-italic">92</span>, 353–362. [<a href="https://scholar.google.com/scholar_lookup?title=LIDAR-based%20geometric%20reconstruction%20of%20boreal%20type%20forest%20stands%20at%20single%20tree%20level%20for%20forest%20and%20wildland%20fire%20management&amp;author=Morsdorf,+F.&amp;author=Meier,+E.&amp;author=K%C3%B6tz,+B.&amp;author=Itten,+K.I.&amp;author=Dobbertin,+M.&amp;author=Allg%C3%B6wer,+B.&amp;publication_year=2004&amp;journal=Remote+Sens.+Environ&amp;volume=92&amp;pages=353%E2%80%93362" class="google-scholar" target="_blank">Google Scholar</a>]</li><li id="b22-remotesensing-05-04163" class="html-xx" data-content="22.">Kwak,
 D.A.; Lee, W.K.; Lee, J.H.; Biging, G.S.; Gong, P. Detection of 
individual trees and estimation of tree height using LiDAR data. <span class="html-italic">J. Forest Res</span> <b>2007</b>, <span class="html-italic">12</span>, 425–434. [<a href="https://scholar.google.com/scholar_lookup?title=Detection%20of%20individual%20trees%20and%20estimation%20of%20tree%20height%20using%20LiDAR%20data&amp;author=Kwak,+D.A.&amp;author=Lee,+W.K.&amp;author=Lee,+J.H.&amp;author=Biging,+G.S.&amp;author=Gong,+P.&amp;publication_year=2007&amp;journal=J.+Forest+Res&amp;volume=12&amp;pages=425%E2%80%93434" class="google-scholar" target="_blank">Google Scholar</a>]</li><li id="b23-remotesensing-05-04163" class="html-xx" data-content="23.">Brandtberg,
 T.; Warner, T.A.; Landenberger, R.E.; McGraw, J.B. Detection and 
analysis of individual leaf-off tree crowns in small footprint, high 
sampling density lidar data from the eastern deciduous forest in North 
America. <span class="html-italic">Remote Sens. Environ</span> <b>2003</b>, <span class="html-italic">85</span>, 290–303. [<a href="https://scholar.google.com/scholar_lookup?title=Detection%20and%20analysis%20of%20individual%20leaf-off%20tree%20crowns%20in%20small%20footprint,%20high%20sampling%20density%20lidar%20data%20from%20the%20eastern%20deciduous%20forest%20in%20North%20America&amp;author=Brandtberg,+T.&amp;author=Warner,+T.A.&amp;author=Landenberger,+R.E.&amp;author=McGraw,+J.B.&amp;publication_year=2003&amp;journal=Remote+Sens.+Environ&amp;volume=85&amp;pages=290%E2%80%93303" class="google-scholar" target="_blank">Google Scholar</a>]</li><li id="b24-remotesensing-05-04163" class="html-xx" data-content="24.">Gleason, C.J.; Im, J. A fusion approach for tree crown delineation from LiDAR data. <span class="html-italic">Photogramm. Eng. Remote Sensing</span> <b>2012</b>, <span class="html-italic">78</span>, 679–692. [<a href="https://scholar.google.com/scholar_lookup?title=A%20fusion%20approach%20for%20tree%20crown%20delineation%20from%20LiDAR%20data&amp;author=Gleason,+C.J.&amp;author=Im,+J.&amp;publication_year=2012&amp;journal=Photogramm.+Eng.+Remote+Sensing&amp;volume=78&amp;pages=679%E2%80%93692" class="google-scholar" target="_blank">Google Scholar</a>]</li><li id="b25-remotesensing-05-04163" class="html-xx" data-content="25.">Koch, B.; Heyder, U.; Weinacker, H. Detection of individual tree crowns in airborne lidar data. <span class="html-italic">Photogramm. Eng. Remote Sensing</span> <b>2006</b>, <span class="html-italic">72</span>, 357–363. [<a href="https://scholar.google.com/scholar_lookup?title=Detection%20of%20individual%20tree%20crowns%20in%20airborne%20lidar%20data&amp;author=Koch,+B.&amp;author=Heyder,+U.&amp;author=Weinacker,+H.&amp;publication_year=2006&amp;journal=Photogramm.+Eng.+Remote+Sensing&amp;volume=72&amp;pages=357%E2%80%93363" class="google-scholar" target="_blank">Google Scholar</a>]</li><li id="b26-remotesensing-05-04163" class="html-xx" data-content="26.">Yu,
 X.; Hyyppä, J.; Vastaranta, M.; Holopainen, M.; Viitala, R. Predicting 
individual tree attributes from airborne laser point clouds based on the
 random forests technique. <span class="html-italic">ISPRS J. Photogramm</span> <b>2011</b>, <span class="html-italic">66</span>, 28–37. [<a href="https://scholar.google.com/scholar_lookup?title=Predicting%20individual%20tree%20attributes%20from%20airborne%20laser%20point%20clouds%20based%20on%20the%20random%20forests%20technique&amp;author=Yu,+X.&amp;author=Hyypp%C3%A4,+J.&amp;author=Vastaranta,+M.&amp;author=Holopainen,+M.&amp;author=Viitala,+R.&amp;publication_year=2011&amp;journal=ISPRS+J.+Photogramm&amp;volume=66&amp;pages=28%E2%80%9337" class="google-scholar" target="_blank">Google Scholar</a>]</li><li id="b27-remotesensing-05-04163" class="html-xx" data-content="27.">Hyyppä,
 J.; Kelle, O.; Lehikoinen, M.; Inkinen, M. A segmentation-based method 
to retrieve stem volume estimates from 3-D tree height models produced 
by laser scanners. <span class="html-italic">ISPRS J. Photogramm</span> <b>2001</b>, <span class="html-italic">39</span>, 969–975. [<a href="https://scholar.google.com/scholar_lookup?title=A%20segmentation-based%20method%20to%20retrieve%20stem%20volume%20estimates%20from%203-D%20tree%20height%20models%20produced%20by%20laser%20scanners&amp;author=Hyypp%C3%A4,+J.&amp;author=Kelle,+O.&amp;author=Lehikoinen,+M.&amp;author=Inkinen,+M.&amp;publication_year=2001&amp;journal=ISPRS+J.+Photogramm&amp;volume=39&amp;pages=969%E2%80%93975" class="google-scholar" target="_blank">Google Scholar</a>]</li><li id="b28-remotesensing-05-04163" class="html-xx" data-content="28.">Solberg,
 S.; Naesset, E.; Hanssen, K.H.; Christiansen, E. Mapping defoliation 
during a severe insect attack on Scots pine using airborne laser 
scanning. <span class="html-italic">Remote Sens. Environ</span> <b>2006</b>, <span class="html-italic">102</span>, 364–376. [<a href="https://scholar.google.com/scholar_lookup?title=Mapping%20defoliation%20during%20a%20severe%20insect%20attack%20on%20Scots%20pine%20using%20airborne%20laser%20scanning&amp;author=Solberg,+S.&amp;author=Naesset,+E.&amp;author=Hanssen,+K.H.&amp;author=Christiansen,+E.&amp;publication_year=2006&amp;journal=Remote+Sens.+Environ&amp;volume=102&amp;pages=364%E2%80%93376" class="google-scholar" target="_blank">Google Scholar</a>]</li><li id="b29-remotesensing-05-04163" class="html-xx" data-content="29.">Breidenbach,
 J.; Næsset, E.; Lien, V.; Gobakken, T.; Solberg, S. Prediction of 
species specific forest inventory attributes using a nonparametric 
semi-individual tree crown approach based on fused airborne laser 
scanning and multispectral data. <span class="html-italic">Remote Sens. Environ</span> <b>2010</b>, <span class="html-italic">114</span>, 911–924. [<a href="https://scholar.google.com/scholar_lookup?title=Prediction%20of%20species%20specific%20forest%20inventory%20attributes%20using%20a%20nonparametric%20semi-individual%20tree%20crown%20approach%20based%20on%20fused%20airborne%20laser%20scanning%20and%20multispectral%20data&amp;author=Breidenbach,+J.&amp;author=N%C3%A6sset,+E.&amp;author=Lien,+V.&amp;author=Gobakken,+T.&amp;author=Solberg,+S&amp;publication_year=2010&amp;journal=Remote+Sens.+Environ&amp;volume=114&amp;pages=911%E2%80%93924" class="google-scholar" target="_blank">Google Scholar</a>]</li><li id="b30-remotesensing-05-04163" class="html-xx" data-content="30.">Edson,
 C.; Wing, M.G. Airborne light Detection and Ranging (LiDAR) for 
individual tree stem location, height, and biomass measurements. <span class="html-italic">Remote Sens</span> <b>2011</b>, <span class="html-italic">3</span>, 2494–2528. [<a href="https://scholar.google.com/scholar_lookup?title=Airborne%20light%20Detection%20and%20Ranging%20%28LiDAR%29%20for%20individual%20tree%20stem%20location,%20height,%20and%20biomass%20measurements&amp;author=Edson,+C.&amp;author=Wing,+M.G.&amp;publication_year=2011&amp;journal=Remote+Sens&amp;volume=3&amp;pages=2494%E2%80%932528" class="google-scholar" target="_blank">Google Scholar</a>]</li><li id="b31-remotesensing-05-04163" class="html-xx" data-content="31.">Féret,
 J.-B.; Asner, G.P. Semi-supervised methods to identify individual 
crowns of lowland tropical canopy species using imaging spectroscopy and
 LiDAR. <span class="html-italic">Remote Sens</span> <b>2012</b>, <span class="html-italic">4</span>, 2457–2476. [<a href="https://scholar.google.com/scholar_lookup?title=Semi-supervised%20methods%20to%20identify%20individual%20crowns%20of%20lowland%20tropical%20canopy%20species%20using%20imaging%20spectroscopy%20and%20LiDAR&amp;author=F%C3%A9ret,+J.-B.&amp;author=Asner,+G.P.&amp;publication_year=2012&amp;journal=Remote+Sens&amp;volume=4&amp;pages=2457%E2%80%932476" class="google-scholar" target="_blank">Google Scholar</a>]</li><li id="b32-remotesensing-05-04163" class="html-xx" data-content="32.">Lee,
 H.; Slatton, K.C.; Roth, B.E.; Cropper, W.P. Adaptive clustering of 
airborne LiDAR data to segment individual tree crowns in managed pine 
forests. <span class="html-italic">Int. J. Remote Sens</span> <b>2010</b>, <span class="html-italic">31</span>, 117–139. [<a href="https://scholar.google.com/scholar_lookup?title=Adaptive%20clustering%20of%20airborne%20LiDAR%20data%20to%20segment%20individual%20tree%20crowns%20in%20managed%20pine%20forests&amp;author=Lee,+H.&amp;author=Slatton,+K.C.&amp;author=Roth,+B.E.&amp;author=Cropper,+W.P.&amp;publication_year=2010&amp;journal=Int.+J.+Remote+Sens&amp;volume=31&amp;pages=117%E2%80%93139" class="google-scholar" target="_blank">Google Scholar</a>]</li><li id="b33-remotesensing-05-04163" class="html-xx" data-content="33.">Li, W.; Guo, Q.; Jakubowski, M.; Kelly, M. A new method for segmenting individual trees from the lidar point cloud. <span class="html-italic">Photogramm. Eng. Remote Sensing</span> <b>2012</b>, <span class="html-italic">78</span>, 75–84. [<a href="https://scholar.google.com/scholar_lookup?title=A%20new%20method%20for%20segmenting%20individual%20trees%20from%20the%20lidar%20point%20cloud&amp;author=Li,+W.&amp;author=Guo,+Q.&amp;author=Jakubowski,+M.&amp;author=Kelly,+M.&amp;publication_year=2012&amp;journal=Photogramm.+Eng.+Remote+Sensing&amp;volume=78&amp;pages=75%E2%80%9384" class="google-scholar" target="_blank">Google Scholar</a>]</li><li id="b34-remotesensing-05-04163" class="html-xx" data-content="34.">Cleve,
 C.; Kelly, M.; Kearns, F.R.; Moritz, M. Classification of the 
wildland–urban interface: A comparison of pixel-and object-based 
classifications using high-resolution aerial photography. <span class="html-italic">Compu. Environ. Urban Syst</span> <b>2008</b>, <span class="html-italic">32</span>, 317–326. [<a href="https://scholar.google.com/scholar_lookup?title=Classification%20of%20the%20wildland%E2%80%93urban%20interface:%20A%20comparison%20of%20pixel-and%20object-based%20classifications%20using%20high-resolution%20aerial%20photography&amp;author=Cleve,+C.&amp;author=Kelly,+M.&amp;author=Kearns,+F.R.&amp;author=Moritz,+M.&amp;publication_year=2008&amp;journal=Compu.+Environ.+Urban+Syst&amp;volume=32&amp;pages=317%E2%80%93326" class="google-scholar" target="_blank">Google Scholar</a>]</li><li id="b35-remotesensing-05-04163" class="html-xx" data-content="35.">Guo,
 Q.C.; Kelly, M.; Gong, P.; Liu, D. An object-based classification 
approach in mapping tree mortality using high spatial resolution 
imagery. <span class="html-italic">GIScience Remote Sens</span> <b>2007</b>, <span class="html-italic">44</span>, 24–47. [<a href="https://scholar.google.com/scholar_lookup?title=An%20object-based%20classification%20approach%20in%20mapping%20tree%20mortality%20using%20high%20spatial%20resolution%20imagery&amp;author=Guo,+Q.C.&amp;author=Kelly,+M.&amp;author=Gong,+P.&amp;author=Liu,+D.&amp;publication_year=2007&amp;journal=GIScience+Remote+Sens&amp;volume=44&amp;pages=24%E2%80%9347" class="google-scholar" target="_blank">Google Scholar</a>]</li><li id="b36-remotesensing-05-04163" class="html-xx" data-content="36.">Kelly,
 M.; Blanchard, S.; Kersten, E.; Koy, K. Object-based analysis of 
imagery in support of public health: new avenues of research. <span class="html-italic">Remote Sens</span> <b>2011</b>, <span class="html-italic">3</span>, 2321–2345. [<a href="https://scholar.google.com/scholar_lookup?title=Object-based%20analysis%20of%20imagery%20in%20support%20of%20public%20health:%20new%20avenues%20of%20research&amp;author=Kelly,+M.&amp;author=Blanchard,+S.&amp;author=Kersten,+E.&amp;author=Koy,+K.&amp;publication_year=2011&amp;journal=Remote+Sens&amp;volume=3&amp;pages=2321%E2%80%932345" class="google-scholar" target="_blank">Google Scholar</a>]</li><li id="b37-remotesensing-05-04163" class="html-xx" data-content="37.">Blaschke, T. Object-based image analysis for remote sensing. <span class="html-italic">ISPRS J. Photogramm</span> <b>2010</b>, <span class="html-italic">65</span>, 2–16. [<a href="https://scholar.google.com/scholar_lookup?title=Object-based%20image%20analysis%20for%20remote%20sensing&amp;author=Blaschke,+T.&amp;publication_year=2010&amp;journal=ISPRS+J.+Photogramm&amp;volume=65&amp;pages=2%E2%80%9316" class="google-scholar" target="_blank">Google Scholar</a>]</li><li id="b38-remotesensing-05-04163" class="html-xx" data-content="38.">Chubey,
 M.S.; Franklin, S.E.; Wulder, M.A. Object-based analysis of Ikonos-2 
imagery for extraction of forest inventory parameters. <span class="html-italic">Photogramm. Eng. Remote Sensing</span> <b>2006</b>, <span class="html-italic">72</span>, 383–394. [<a href="https://scholar.google.com/scholar_lookup?title=Object-based%20analysis%20of%20Ikonos-2%20imagery%20for%20extraction%20of%20forest%20inventory%20parameters&amp;author=Chubey,+M.S.&amp;author=Franklin,+S.E.&amp;author=Wulder,+M.A.&amp;publication_year=2006&amp;journal=Photogramm.+Eng.+Remote+Sensing&amp;volume=72&amp;pages=383%E2%80%93394" class="google-scholar" target="_blank">Google Scholar</a>]</li><li id="b39-remotesensing-05-04163" class="html-xx" data-content="39.">Brennan, R.; Webster, T. Object-oriented land cover classification of lidar-derived surfaces. <span class="html-italic">Can. J. Remote Sens</span> <b>2006</b>, <span class="html-italic">32</span>, 162–172. [<a href="https://scholar.google.com/scholar_lookup?title=Object-oriented%20land%20cover%20classification%20of%20lidar-derived%20surfaces&amp;author=Brennan,+R.&amp;author=Webster,+T.&amp;publication_year=2006&amp;journal=Can.+J.+Remote+Sens&amp;volume=32&amp;pages=162%E2%80%93172" class="google-scholar" target="_blank">Google Scholar</a>]</li><li id="b40-remotesensing-05-04163" class="html-xx" data-content="40.">Antonarakis, A.S.; Richards, K.S.; Brasington, J. Object-based land cover classification using airborne LiDAR. <span class="html-italic">Remote Sens. Environ</span> <b>2008</b>, <span class="html-italic">112</span>, 2988–2998. [<a href="https://scholar.google.com/scholar_lookup?title=Object-based%20land%20cover%20classification%20using%20airborne%20LiDAR&amp;author=Antonarakis,+A.S.&amp;author=Richards,+K.S.&amp;author=Brasington,+J.&amp;publication_year=2008&amp;journal=Remote+Sens.+Environ&amp;volume=112&amp;pages=2988%E2%80%932998" class="google-scholar" target="_blank">Google Scholar</a>]</li><li id="b41-remotesensing-05-04163" class="html-xx" data-content="41.">Ke,
 Y.; Quackenbush, L.J.; Im, J. Synergistic use of QuickBird 
multispectral imagery and LIDAR data for object-based forest species 
classification. <span class="html-italic">Remote Sens. Environ</span> <b>2010</b>, <span class="html-italic">114</span>, 1141–1154. [<a href="https://scholar.google.com/scholar_lookup?title=Synergistic%20use%20of%20QuickBird%20multispectral%20imagery%20and%20LIDAR%20data%20for%20object-based%20forest%20species%20classification&amp;author=Ke,+Y.&amp;author=Quackenbush,+L.J.&amp;author=Im,+J.&amp;publication_year=2010&amp;journal=Remote+Sens.+Environ&amp;volume=114&amp;pages=1141%E2%80%931154" class="google-scholar" target="_blank">Google Scholar</a>]</li><li id="b42-remotesensing-05-04163" class="html-xx" data-content="42.">Clinton,
 N.; Holt, A.; Scarborough, J.; Yan, L.; Gong, P. Accuracy assessment 
measures for object-based image segmentation goodness. <span class="html-italic">Photogramm. Eng. Remote Sensing</span> <b>2010</b>, <span class="html-italic">76</span>, 289–299. [<a href="https://scholar.google.com/scholar_lookup?title=Accuracy%20assessment%20measures%20for%20object-based%20image%20segmentation%20goodness&amp;author=Clinton,+N.&amp;author=Holt,+A.&amp;author=Scarborough,+J.&amp;author=Yan,+L.&amp;author=Gong,+P.&amp;publication_year=2010&amp;journal=Photogramm.+Eng.+Remote+Sensing&amp;volume=76&amp;pages=289%E2%80%93299" class="google-scholar" target="_blank">Google Scholar</a>]</li><li id="b43-remotesensing-05-04163" class="html-xx" data-content="43.">Trimble. <span class="html-italic">eCognition Developer</span>; Version 8.8; Trimble: Munich, Germany, 2012. [<a href="https://scholar.google.com/scholar_lookup?title=eCognition+Developer&amp;author=Trimble&amp;publication_year=2012" class="google-scholar" target="_blank">Google Scholar</a>]</li><li id="b44-remotesensing-05-04163" class="html-xx" data-content="44.">Baatz, M.; Hoffmann, C.; Willhauck, G. Progressing from Object-Based to Object-Oriented Image Analysis. In <span class="html-italic">Object-Based Image Analysis: Spatial Concepts for Knowledge-Driven Remote Sensing Applications</span>; Blaschke, T., Lang, S., Hay, G.J., Eds.; Springer: Berlin/Heidelberg, Germany, 2008; pp. 29–42. [<a href="https://scholar.google.com/scholar_lookup?title=Progressing%20from%20Object-Based%20to%20Object-Oriented%20Image%20Analysis&amp;author=Baatz,+M.&amp;author=Hoffmann,+C.&amp;author=Willhauck,+G&amp;publication_year=2008&amp;pages=29%E2%80%9342" class="google-scholar" target="_blank">Google Scholar</a>]</li><li id="b45-remotesensing-05-04163" class="html-xx" data-content="45.">Stephens,
 S.L.; Collins, B.M. Fire regimes of mixed conifer forests in the 
north-central Sierra Nevada at multiple spatial scales. <span class="html-italic">Northwest Sci</span> <b>2004</b>, <span class="html-italic">78</span>, 12. [<a href="https://scholar.google.com/scholar_lookup?title=Fire%20regimes%20of%20mixed%20conifer%20forests%20in%20the%20north-central%20Sierra%20Nevada%20at%20multiple%20spatial%20scales&amp;author=Stephens,+S.L.&amp;author=Collins,+B.M.&amp;publication_year=2004&amp;journal=Northwest+Sci&amp;volume=78&amp;pages=12" class="google-scholar" target="_blank">Google Scholar</a>]</li><li id="b46-remotesensing-05-04163" class="html-xx" data-content="46.">Jakubowski, M.K.; Guo, Q.; Kelly, M. Tradeoffs between lidar pulse density and forest measurement accuracy. <span class="html-italic">Remote Sens. Environ</span> <b>2013</b>, <span class="html-italic">130</span>, 245–253. [<a href="https://scholar.google.com/scholar_lookup?title=Tradeoffs%20between%20lidar%20pulse%20density%20and%20forest%20measurement%20accuracy&amp;author=Jakubowski,+M.K.&amp;author=Guo,+Q.&amp;author=Kelly,+M.&amp;publication_year=2013&amp;journal=Remote+Sens.+Environ&amp;volume=130&amp;pages=245%E2%80%93253" class="google-scholar" target="_blank">Google Scholar</a>]</li><li id="b47-remotesensing-05-04163" class="html-xx" data-content="47.">Exelis. <span class="html-italic">ENVI Software</span>; Exelis: McLean, VA, USA, 2012. [<a href="https://scholar.google.com/scholar_lookup?title=ENVI+Software&amp;author=Exelis&amp;publication_year=2012" class="google-scholar" target="_blank">Google Scholar</a>]</li><li id="b48-remotesensing-05-04163" class="html-xx" data-content="48.">Laben,
 C.A.; Brower, B.V. Process for Enhancing the Spatial Resolution of 
Multispectral Imagery Using Pan-Sharpening. US Patent No. 6,011,875,. 4 
January 2000. [<a href="https://scholar.google.com/scholar_lookup?title=Process%20for%20Enhancing%20the%20Spatial%20Resolution%20of%20Multispectral%20Imagery%20Using%20Pan-Sharpening&amp;author=Laben,+C.A.&amp;author=Brower,+B.V.&amp;publication_year=2000" class="google-scholar" target="_blank">Google Scholar</a>]</li><li id="b49-remotesensing-05-04163" class="html-xx" data-content="49.">Chang, Y.; Habib, A.; Lee, D.; Yom, J. Automatic classification of lidar data into ground and non-ground points. <span class="html-italic">Int. Arch. Photogramm. Remote Sens. Spat. Inf. Sci</span> <b>2008</b>, <span class="html-italic">37</span>, 463–468. [<a href="https://scholar.google.com/scholar_lookup?title=Automatic%20classification%20of%20lidar%20data%20into%20ground%20and%20non-ground%20points&amp;author=Chang,+Y.&amp;author=Habib,+A.&amp;author=Lee,+D.&amp;author=Yom,+J.&amp;publication_year=2008&amp;journal=Int.+Arch.+Photogramm.+Remote+Sens.+Spat.+Inf.+Sci&amp;volume=37&amp;pages=463%E2%80%93468" class="google-scholar" target="_blank">Google Scholar</a>]</li><li id="b50-remotesensing-05-04163" class="html-xx" data-content="50.">Leckie,
 D.; Gougeon, F.; Hill, D.; Quinn, R.; Armstrong, L.; Shreenan, R. 
Combined high-density lidar and multispectral imagery for individual 
tree crown analysis. <span class="html-italic">Can. J. Remote Sens</span> <b>2003</b>, <span class="html-italic">29</span>, 633–649. [<a href="https://scholar.google.com/scholar_lookup?title=Combined%20high-density%20lidar%20and%20multispectral%20imagery%20for%20individual%20tree%20crown%20analysis&amp;author=Leckie,+D.&amp;author=Gougeon,+F.&amp;author=Hill,+D.&amp;author=Quinn,+R.&amp;author=Armstrong,+L.&amp;author=Shreenan,+R.&amp;publication_year=2003&amp;journal=Can.+J.+Remote+Sens&amp;volume=29&amp;pages=633%E2%80%93649" class="google-scholar" target="_blank">Google Scholar</a>]</li><li id="b51-remotesensing-05-04163" class="html-xx" data-content="51.">Lee, J.-S. Digital Image Enhancement and Noise Filtering by Use of Local Statistics. <span class="html-italic">IEEE Trans. Pattern Anal. Mach. Intell.</span> <b>1980</b>, <span class="html-italic">PAMI–2</span>, 165–168. [<a href="https://scholar.google.com/scholar_lookup?title=Digital%20Image%20Enhancement%20and%20Noise%20Filtering%20by%20Use%20of%20Local%20Statistics&amp;author=Lee,+J.-S.&amp;publication_year=1980&amp;journal=IEEE+Trans.+Pattern+Anal.+Mach.+Intell.&amp;volume=PAMI%E2%80%932&amp;pages=165%E2%80%93168" class="google-scholar" target="_blank">Google Scholar</a>]</li><li id="b52-remotesensing-05-04163" class="html-xx" data-content="52.">Lopes, A.; Touzi, R.; Nezry, E. Adaptive speckle filters and scene heterogeneity. <span class="html-italic">IEEE Trans. Geosci. Remote Sens</span> <b>1990</b>, <span class="html-italic">28</span>, 992–1000. [<a href="https://scholar.google.com/scholar_lookup?title=Adaptive%20speckle%20filters%20and%20scene%20heterogeneity&amp;author=Lopes,+A.&amp;author=Touzi,+R.&amp;author=Nezry,+E.&amp;publication_year=1990&amp;journal=IEEE+Trans.+Geosci.+Remote+Sens&amp;volume=28&amp;pages=992%E2%80%931000" class="google-scholar" target="_blank">Google Scholar</a>]</li><li id="b53-remotesensing-05-04163" class="html-xx" data-content="53.">Haralick, R.M.; Sternberg, S.R.; Zhuang, X. Image analysis using mathematical morphology. <span class="html-italic">IEEE Trans. Pattern Anal. Mach. Intell.</span> <b>1987</b>, 532–550. [<a href="https://scholar.google.com/scholar_lookup?title=Image%20analysis%20using%20mathematical%20morphology&amp;author=Haralick,+R.M.&amp;author=Sternberg,+S.R.&amp;author=Zhuang,+X&amp;publication_year=1987&amp;journal=IEEE+Trans.+Pattern+Anal.+Mach.+Intell.&amp;pages=532%E2%80%93550" class="google-scholar" target="_blank">Google Scholar</a>]</li><li id="b54-remotesensing-05-04163" class="html-xx" data-content="54.">Drǎguţ,
 L.; Tiede, D.; Levick, S.R. ESP: a tool to estimate scale parameter for
 multiresolution image segmentation of remotely sensed data. <span class="html-italic">Int. J. Geogr. Inf. Sci</span> <b>2010</b>, <span class="html-italic">24</span>, 859–871. [<a href="https://scholar.google.com/scholar_lookup?title=ESP:%20a%20tool%20to%20estimate%20scale%20parameter%20for%20multiresolution%20image%20segmentation%20of%20remotely%20sensed%20data&amp;author=Dra%CC%8Cgu%C5%A3,+L.&amp;author=Tiede,+D.&amp;author=Levick,+S.R.&amp;publication_year=2010&amp;journal=Int.+J.+Geogr.+Inf.+Sci&amp;volume=24&amp;pages=859%E2%80%93871" class="google-scholar" target="_blank">Google Scholar</a>]</li><li id="b55-remotesensing-05-04163" class="html-xx" data-content="55.">Leonard, J. <span class="html-italic">Technical Approach for LIDAR Acquisition and Processing</span>; EarthData Inc.: Frederick, MD, USA, 2005; pp. 1–20. [<a href="https://scholar.google.com/scholar_lookup?title=Technical+Approach+for+LIDAR+Acquisition+and+Processing&amp;author=Leonard,+J&amp;publication_year=2005" class="google-scholar" target="_blank">Google Scholar</a>]</li><li id="b56-remotesensing-05-04163" class="html-xx" data-content="56.">Im, J.; Jensen, J.R.; Hodgson, M.E. Object-based land cover classification using high-posting-density LiDAR data. <span class="html-italic">GIScience Remote Sens</span> <b>2008</b>, <span class="html-italic">45</span>, 209–228. [<a href="https://scholar.google.com/scholar_lookup?title=Object-based%20land%20cover%20classification%20using%20high-posting-density%20LiDAR%20data&amp;author=Im,+J.&amp;author=Jensen,+J.R.&amp;author=Hodgson,+M.E.&amp;publication_year=2008&amp;journal=GIScience+Remote+Sens&amp;volume=45&amp;pages=209%E2%80%93228" class="google-scholar" target="_blank">Google Scholar</a>]</li><li id="b57-remotesensing-05-04163" class="html-xx" data-content="57.">Baltsavias, E.P. Airborne laser scanning: basic relations and formulas. <span class="html-italic">ISPRS J. Photogram</span> <b>1999</b>, <span class="html-italic">54</span>, 199–214. [<a href="https://scholar.google.com/scholar_lookup?title=Airborne%20laser%20scanning:%20basic%20relations%20and%20formulas&amp;author=Baltsavias,+E.P.&amp;publication_year=1999&amp;journal=ISPRS+J.+Photogram&amp;volume=54&amp;pages=199%E2%80%93214" class="google-scholar" target="_blank">Google Scholar</a>]</li><li id="b58-remotesensing-05-04163" class="html-xx" data-content="58.">Frank,
 E.; Hall, M.; Holmes, G.; Kirkby, R.; Pfahringer, B.; Witten, I.H.; 
Trigg, L. Weka—A Machine Learning Workbench for Data Mining. In <span class="html-italic">Data Mining and Knowledge Discovery Handbook</span>; Maimon, O., Rokach, L., Eds.; Springer: New York, NY, USA, 2010; pp. 1269–1277. [<a href="https://scholar.google.com/scholar_lookup?title=Weka%E2%80%94A%20Machine%20Learning%20Workbench%20for%20Data%20Mining&amp;author=Frank,+E.&amp;author=Hall,+M.&amp;author=Holmes,+G.&amp;author=Kirkby,+R.&amp;author=Pfahringer,+B.&amp;author=Witten,+I.H.&amp;author=Trigg,+L&amp;publication_year=2010&amp;pages=1269%E2%80%931277" class="google-scholar" target="_blank">Google Scholar</a>]</li><li id="b59-remotesensing-05-04163" class="html-xx" data-content="59.">Jakubowski,
 M.K.; Guo, Q.; Collins, B.; Stephens, S.; Kelly, M. Predicting surface 
fuel models and fuel metrics using lidar and imagery in dense, 
mountainous forest. <span class="html-italic">Photogramm. Eng. Remote Sensing</span> <b>2013</b>, <span class="html-italic">79</span>, 37–50. [<a href="https://scholar.google.com/scholar_lookup?title=Predicting%20surface%20fuel%20models%20and%20fuel%20metrics%20using%20lidar%20and%20imagery%20in%20dense,%20mountainous%20forest&amp;author=Jakubowski,+M.K.&amp;author=Guo,+Q.&amp;author=Collins,+B.&amp;author=Stephens,+S.&amp;author=Kelly,+M.&amp;publication_year=2013&amp;journal=Photogramm.+Eng.+Remote+Sensing&amp;volume=79&amp;pages=37%E2%80%9350" class="google-scholar" target="_blank">Google Scholar</a>]</li><li id="b60-remotesensing-05-04163" class="html-xx" data-content="60.">Fan, R.E.; Chang, K.W.; Hsieh, C.J.; Wang, X.R.; Lin, C.J. LIBLINEAR: A library for large linear classification. <span class="html-italic">J. Mach. Learn. Res</span> <b>2008</b>, <span class="html-italic">9</span>, 1871–1874. [<a href="https://scholar.google.com/scholar_lookup?title=LIBLINEAR:%20A%20library%20for%20large%20linear%20classification&amp;author=Fan,+R.E.&amp;author=Chang,+K.W.&amp;author=Hsieh,+C.J.&amp;author=Wang,+X.R.&amp;author=Lin,+C.J.&amp;publication_year=2008&amp;journal=J.+Mach.+Learn.+Res&amp;volume=9&amp;pages=1871%E2%80%931874" class="google-scholar" target="_blank">Google Scholar</a>]</li><li id="b61-remotesensing-05-04163" class="html-xx" data-content="61.">Keerthi,
 S.S.; Shevade, S.K.; Bhattacharyya, C.; Murthy, K.R.K. Improvements to 
Platt's SMO algorithm for SVM classifier design. <span class="html-italic">Neural Comput</span> <b>2001</b>, <span class="html-italic">13</span>, 637–649. [<a href="https://scholar.google.com/scholar_lookup?title=Improvements%20to%20Platt%27s%20SMO%20algorithm%20for%20SVM%20classifier%20design&amp;author=Keerthi,+S.S.&amp;author=Shevade,+S.K.&amp;author=Bhattacharyya,+C.&amp;author=Murthy,+K.R.K.&amp;publication_year=2001&amp;journal=Neural+Comput&amp;volume=13&amp;pages=637%E2%80%93649" class="google-scholar" target="_blank">Google Scholar</a>]</li><li id="b62-remotesensing-05-04163" class="html-xx" data-content="62.">Buhmann, M.D. Radial basis functions. <span class="html-italic">Acta Numer</span> <b>2000</b>, <span class="html-italic">9</span>, 1–38. [<a href="https://scholar.google.com/scholar_lookup?title=Radial%20basis%20functions&amp;author=Buhmann,+M.D.&amp;publication_year=2000&amp;journal=Acta+Numer&amp;volume=9&amp;pages=1%E2%80%9338" class="google-scholar" target="_blank">Google Scholar</a>]</li><li id="b63-remotesensing-05-04163" class="html-xx" data-content="63.">Forman, R.T.T.; Godron, M. <span class="html-italic">Landscape Ecology</span>; JohnWiley and Sons: New York, NY, USA, 1986; pp. 1–619. [<a href="https://scholar.google.com/scholar_lookup?title=Landscape+Ecology&amp;author=Forman,+R.T.T.&amp;author=Godron,+M&amp;publication_year=1986" class="google-scholar" target="_blank">Google Scholar</a>]</li><li id="b64-remotesensing-05-04163" class="html-xx" data-content="64.">McGarigal, K.; Cushman, S.A.; Neel, M.C.; Ene, E. <span class="html-italic">FRAGSTATS: Spatial Pattern Analysis Program for Categorical Maps</span>; Version 4; University of Massachusetts Amherst: MA, USA, 2002. [<a href="https://scholar.google.com/scholar_lookup?title=FRAGSTATS:+Spatial+Pattern+Analysis+Program+for+Categorical+Maps&amp;author=McGarigal,+K.&amp;author=Cushman,+S.A.&amp;author=Neel,+M.C.&amp;author=Ene,+E&amp;publication_year=2002" class="google-scholar" target="_blank">Google Scholar</a>]</li><li id="b65-remotesensing-05-04163" class="html-xx" data-content="65.">Jiao, L.; Liu, Y. Analyzing the shape characteristics of land use classes in remote sensing imagery. <span class="html-italic">ISPRS Ann. Photogramm. Remote Sens. Spat. Inf. Sci.</span> <b>2012</b>, <span class="html-italic">I-7</span>, 135–140. [<a href="https://scholar.google.com/scholar_lookup?title=Analyzing%20the%20shape%20characteristics%20of%20land%20use%20classes%20in%20remote%20sensing%20imagery&amp;author=Jiao,+L.&amp;author=Liu,+Y&amp;publication_year=2012&amp;journal=ISPRS+Ann.+Photogramm.+Remote+Sens.+Spat.+Inf.+Sci.&amp;volume=I-7&amp;pages=135%E2%80%93140" class="google-scholar" target="_blank">Google Scholar</a>]</li><li id="b66-remotesensing-05-04163" class="html-xx" data-content="66.">Trani, M.K.; Giles, R.H., Jr. An analysis of deforestation: Metrics used to describe pattern change. <span class="html-italic">Forest Ecol. Manag</span> <b>1999</b>, <span class="html-italic">114</span>, 459–470. [<a href="https://scholar.google.com/scholar_lookup?title=An%20analysis%20of%20deforestation:%20Metrics%20used%20to%20describe%20pattern%20change&amp;author=Trani,+M.K.&amp;author=Giles,+R.H.,+Jr&amp;publication_year=1999&amp;journal=Forest+Ecol.+Manag&amp;volume=114&amp;pages=459%E2%80%93470" class="google-scholar" target="_blank">Google Scholar</a>]</li><li id="b67-remotesensing-05-04163" class="html-xx" data-content="67.">Brokaw, N.; Thompson, J. The H for DBH. <span class="html-italic">Forest Ecol. Manag</span> <b>2000</b>, <span class="html-italic">129</span>, 89–91. [<a href="https://scholar.google.com/scholar_lookup?title=The%20H%20for%20DBH&amp;author=Brokaw,+N.&amp;author=Thompson,+J.&amp;publication_year=2000&amp;journal=Forest+Ecol.+Manag&amp;volume=129&amp;pages=89%E2%80%9391" class="google-scholar" target="_blank">Google Scholar</a>]</li><li id="b68-remotesensing-05-04163" class="html-xx" data-content="68.">Suarez,
 J.C.; Ontiveros, C.; Smith, S.; Snape, S. Use of airborne LiDAR and 
aerial photography in the estimation of individual tree heights in 
forestry. <span class="html-italic">Comput. Geosci</span> <b>2005</b>, <span class="html-italic">31</span>, 253–262. [<a href="https://scholar.google.com/scholar_lookup?title=Use%20of%20airborne%20LiDAR%20and%20aerial%20photography%20in%20the%20estimation%20of%20individual%20tree%20heights%20in%20forestry&amp;author=Suarez,+J.C.&amp;author=Ontiveros,+C.&amp;author=Smith,+S.&amp;author=Snape,+S.&amp;publication_year=2005&amp;journal=Comput.+Geosci&amp;volume=31&amp;pages=253%E2%80%93262" class="google-scholar" target="_blank">Google Scholar</a>]</li></ol></section><section id="FiguresandTables" type="display-objects">
<div class="html-fig_show  mfp-hide" id="fig_body_display_f1-remotesensing-05-04163">
<div class="html-caption"> <b>Figure 1.</b>
Past studies that detect or delineate individual trees from either 
imagery, lidar, or their combination. The published work is arranged by 
year, from top to bottom.</div>
<div class="html-img"><a href="https://www.mdpi.com/remotesensing/remotesensing-05-04163/article_deploy/html/images/remotesensing-05-04163f1.png" class="html-img-zoom" target="_blank"><img alt="Remotesensing 05 04163f1 1024" data-large="/remotesensing/remotesensing-05-04163/article_deploy/html/images/remotesensing-05-04163f1-1024.png" data-original="/remotesensing/remotesensing-05-04163/article_deploy/html/images/remotesensing-05-04163f1.png" src="remotesensing-05-04163f1-1024"></a></div>
</div>
<div class="html-fig_show  mfp-hide" id="fig_body_display_f2-remotesensing-05-04163">
<div class="html-caption"> <b>Figure 2.</b>
The study area (9,950 ha), mostly within Tahoe National Forest, is 
covered by mixed conifer forest, and covers topographically complex 
terrain. The nine 90 × 90 m square areas are indicated by red lines.</div>
<div class="html-img"><a href="https://www.mdpi.com/remotesensing/remotesensing-05-04163/article_deploy/html/images/remotesensing-05-04163f2.png" class="html-img-zoom" target="_blank"><img alt="Remotesensing 05 04163f2 1024" data-large="/remotesensing/remotesensing-05-04163/article_deploy/html/images/remotesensing-05-04163f2-1024.png" data-original="/remotesensing/remotesensing-05-04163/article_deploy/html/images/remotesensing-05-04163f2.png" src="remotesensing-05-04163f2-1024"></a></div>
</div>
<div class="html-fig_show  mfp-hide" id="fig_body_display_f3-remotesensing-05-04163">
<div class="html-caption"> <b>Figure 3.</b>
Nine 90 × 90 m square focal areas were segmented using the lidar-derived
 and OBIA-derived methods. The areas varied across tree density, from 
low (<b>left</b> column), medium (<b>middle</b> column), to high (<b>right</b> column). The color above corresponds to corrected CHM height values.</div>
<div class="html-img"><a href="https://www.mdpi.com/remotesensing/remotesensing-05-04163/article_deploy/html/images/remotesensing-05-04163f3.png" class="html-img-zoom" target="_blank"><img alt="Remotesensing 05 04163f3 1024" data-large="/remotesensing/remotesensing-05-04163/article_deploy/html/images/remotesensing-05-04163f3-1024.png" data-original="/remotesensing/remotesensing-05-04163/article_deploy/html/images/remotesensing-05-04163f3.png" src="remotesensing-05-04163f3-1024"></a></div>
</div>
<div class="html-fig_show  mfp-hide" id="fig_body_display_f4-remotesensing-05-04163">
<div class="html-caption"> <b>Figure 4.</b>
The normalized lidar point cloud was segmented into individual trees using 3D point classification algorithm (<b>A</b>). The classified points were delineated using concave hull in the <span class="html-italic">x-y</span> space (<b>B</b>).
 The thicker tree boundaries exemplify that the lidar-segmentation 
algorithm allows for polygons to overlap. The lidar points are 
identically color-coded by individual trees in both sub-figures.</div>
<div class="html-img"><a href="https://www.mdpi.com/remotesensing/remotesensing-05-04163/article_deploy/html/images/remotesensing-05-04163f4.png" class="html-img-zoom" target="_blank"><img alt="Remotesensing 05 04163f4 1024" data-large="/remotesensing/remotesensing-05-04163/article_deploy/html/images/remotesensing-05-04163f4-1024.png" data-original="/remotesensing/remotesensing-05-04163/article_deploy/html/images/remotesensing-05-04163f4.png" src="remotesensing-05-04163f4-1024"></a></div>
</div>
<div class="html-fig_show  mfp-hide" id="fig_body_display_f5-remotesensing-05-04163">
<div class="html-caption"> <b>Figure 5.</b>
A number of steps were taken to correct the original CHM with artifacts.
 The horizontal profiles across canopy and ground (right) depict how the
 mathematical steps altered the raster surface.</div>
<div class="html-img"><a href="https://www.mdpi.com/remotesensing/remotesensing-05-04163/article_deploy/html/images/remotesensing-05-04163f5.png" class="html-img-zoom" target="_blank"><img alt="Remotesensing 05 04163f5 1024" data-large="/remotesensing/remotesensing-05-04163/article_deploy/html/images/remotesensing-05-04163f5-1024.png" data-original="/remotesensing/remotesensing-05-04163/article_deploy/html/images/remotesensing-05-04163f5.png" src="remotesensing-05-04163f5-1024"></a></div>
</div>
<div class="html-fig_show  mfp-hide" id="fig_body_display_f6-remotesensing-05-04163">
<div class="html-caption"> <b>Figure 6.</b>
Each 90 m focal area was centered at the plot center. The ground 
reference data included overstory trees (blue dots), overstory dominant 
and co-dominant trees (red dots), and marker trees (white). The blue 
(lidar-derived) and white (OBIA-derived) polygons show results of the 
data segmentation, while the crosses indicate the positions of tree tops
 as detected by the 3D segmentation algorithm. Tree heights (meters) are
 indicated next to ground-referenced trees.</div>
<div class="html-img"><a href="https://www.mdpi.com/remotesensing/remotesensing-05-04163/article_deploy/html/images/remotesensing-05-04163f6.png" class="html-img-zoom" target="_blank"><img alt="Remotesensing 05 04163f6 1024" data-large="/remotesensing/remotesensing-05-04163/article_deploy/html/images/remotesensing-05-04163f6-1024.png" data-original="/remotesensing/remotesensing-05-04163/article_deploy/html/images/remotesensing-05-04163f6.png" src="remotesensing-05-04163f6-1024"></a></div>
</div>
<div class="html-fig_show  mfp-hide" id="fig_body_display_f7-remotesensing-05-04163">
<div class="html-caption"> <b>Figure 7.</b>
Ground reference data <span class="html-italic">vs</span>. the lidar-derived (<b>A</b>) and OBIA-derived (<b>B</b>) tree heights.</div>
<div class="html-img"><a href="https://www.mdpi.com/remotesensing/remotesensing-05-04163/article_deploy/html/images/remotesensing-05-04163f7.png" class="html-img-zoom" target="_blank"><img alt="Remotesensing 05 04163f7 1024" data-large="/remotesensing/remotesensing-05-04163/article_deploy/html/images/remotesensing-05-04163f7-1024.png" data-original="/remotesensing/remotesensing-05-04163/article_deploy/html/images/remotesensing-05-04163f7.png" src="remotesensing-05-04163f7-1024"></a></div>
</div>
<div class="html-fig_show  mfp-hide" id="fig_body_display_f8-remotesensing-05-04163">
<div class="html-caption"> <b>Figure 8.</b>
Overstory tree (DBH ≥ 19.5 cm) detection rates, classified by crown 
class. The detection rates of the dominant trees dropped considerably in
 the less-dominant tree crowns. Dead trees, including snags, were 
detected less than 50 percent of the time by both methods.</div>
<div class="html-img"><a href="https://www.mdpi.com/remotesensing/remotesensing-05-04163/article_deploy/html/images/remotesensing-05-04163f8.png" class="html-img-zoom" target="_blank"><img alt="Remotesensing 05 04163f8 1024" data-large="/remotesensing/remotesensing-05-04163/article_deploy/html/images/remotesensing-05-04163f8-1024.png" data-original="/remotesensing/remotesensing-05-04163/article_deploy/html/images/remotesensing-05-04163f8.png" src="remotesensing-05-04163f8-1024"></a></div>
</div>
<div class="html-fig_show  mfp-hide" id="fig_body_display_f9-remotesensing-05-04163">
<div class="html-caption"> <b>Figure 9.</b>
(<b>A</b>) The tree-heights derived by the two methods were more similar than (<b>B</b>) their shape in terms of the area or (<b>C</b>,<b>D</b>)shape complexity.</div>
<div class="html-img"><a href="https://www.mdpi.com/remotesensing/remotesensing-05-04163/article_deploy/html/images/remotesensing-05-04163f9a.png" class="html-img-zoom" target="_blank"><img alt="Remotesensing 05 04163f9a 1024" data-large="/remotesensing/remotesensing-05-04163/article_deploy/html/images/remotesensing-05-04163f9a-1024.png" data-original="/remotesensing/remotesensing-05-04163/article_deploy/html/images/remotesensing-05-04163f9a.png" src="remotesensing-05-04163f9a-1024"></a><a href="https://www.mdpi.com/remotesensing/remotesensing-05-04163/article_deploy/html/images/remotesensing-05-04163f9b.png" class="html-img-zoom" target="_blank"><img alt="Remotesensing 05 04163f9b 1024" data-large="/remotesensing/remotesensing-05-04163/article_deploy/html/images/remotesensing-05-04163f9b-1024.png" data-original="/remotesensing/remotesensing-05-04163/article_deploy/html/images/remotesensing-05-04163f9b.png" src="remotesensing-05-04163f9b-1024"></a></div>
</div>
<div class="html-fig_show  mfp-hide" id="fig_body_display_f10-remotesensing-05-04163">
<div class="html-caption"> <b>Figure 10.</b>
The tree height agreement between ground reference data and derived polygons decreased in high tree density areas (<b>A</b>).
 The area, shape complexity, and predicted tree heights of the derived 
polygons were more different in the high tree-density areas (<b>B</b>).</div>
<div class="html-img"><a href="https://www.mdpi.com/remotesensing/remotesensing-05-04163/article_deploy/html/images/remotesensing-05-04163f10.png" class="html-img-zoom" target="_blank"><img alt="Remotesensing 05 04163f10 1024" data-large="/remotesensing/remotesensing-05-04163/article_deploy/html/images/remotesensing-05-04163f10-1024.png" data-original="/remotesensing/remotesensing-05-04163/article_deploy/html/images/remotesensing-05-04163f10.png" src="remotesensing-05-04163f10-1024"></a></div>
</div>
<div class="html-table_show  mfp-hide " id="table_body_display_t1-remotesensing-05-04163">
<table frame="hsides" rules="groups">
<caption><b>Table 1.</b>
WorldView-2 specifications of the spectral bands.</caption>
<thead><tr><th valign="bottom" align="center"><b>Band</b></th><th valign="bottom" align="center"><b>Spectral Region</b></th><th valign="bottom" align="center"><b>GSD (nadir)</b></th><th valign="bottom" align="center"><b>GSD (20° off nadir)</b></th></tr></thead><tbody><tr><td valign="top" align="left">1 (Coastal)</td><td valign="top" align="center">400–450 nm</td><td valign="top" align="center">185 cm</td><td valign="top" align="center">207 cm</td></tr><tr><td valign="top" align="left">2 (Blue)</td><td valign="top" align="center">450–510 nm</td><td valign="top" align="center">185 cm</td><td valign="top" align="center">207 cm</td></tr><tr><td valign="top" align="left">3 (Green)</td><td valign="top" align="center">510–580 nm</td><td valign="top" align="center">185 cm</td><td valign="top" align="center">207 cm</td></tr><tr><td valign="top" align="left">4 (Yellow)</td><td valign="top" align="center">585–625 nm</td><td valign="top" align="center">185 cm</td><td valign="top" align="center">207 cm</td></tr><tr><td valign="top" align="left">5 (Red)</td><td valign="top" align="center">630–690 nm</td><td valign="top" align="center">185 cm</td><td valign="top" align="center">207 cm</td></tr><tr><td valign="top" align="left">6 (Red edge)</td><td valign="top" align="center">705–745 nm</td><td valign="top" align="center">185 cm</td><td valign="top" align="center">207 cm</td></tr><tr><td valign="top" align="left">7 (NIR1)</td><td valign="top" align="center">770–895 nm</td><td valign="top" align="center">185 cm</td><td valign="top" align="center">207 cm</td></tr><tr><td valign="top" align="left">8 (NIR2)</td><td valign="top" align="center">860–1040 nm</td><td valign="top" align="center">185 cm</td><td valign="top" align="center">207 cm</td></tr><tr><td valign="top" align="left">Panchromatic</td><td valign="top" align="center">450–800 nm</td><td valign="top" align="center">46 cm</td><td valign="top" align="center">52 cm</td></tr></tbody>
</table>
</div>
<div class="html-table_show  mfp-hide " id="table_body_display_t2-remotesensing-05-04163">
<table frame="hsides" rules="groups">
<caption><b>Table 2.</b>
Polygon attributes were calculated and were imported at three different 
stages of data processing: during the segmentation, classification of 
species, and feature optimization.</caption>
<thead><tr><th valign="bottom" align="center"><b>Object Metric</b></th><th valign="bottom" align="center"><b>Segmentation</b></th><th valign="bottom" align="center"><b>Classification</b></th><th valign="bottom" align="center"><b>Feature Optimized</b></th></tr></thead><tbody><tr><td valign="top" align="left">Border Shape</td><td valign="top" align="center"></td><td valign="top" align="center">×</td><td valign="top" align="center"></td></tr><tr><td valign="top" align="left">Compactness</td><td valign="top" align="center"></td><td valign="top" align="center">×</td><td valign="top" align="center"></td></tr><tr><td valign="top" align="left">Density</td><td valign="top" align="center"></td><td valign="top" align="center">×</td><td valign="top" align="center"></td></tr><tr><td valign="top" align="left">Length/width Ratio</td><td valign="top" align="center">×</td><td valign="top" align="center">×</td><td valign="top" align="center">×</td></tr><tr><td valign="top" align="left">Square Pixel</td><td valign="top" align="center">×</td><td valign="top" align="center">×</td><td valign="top" align="center"></td></tr><tr><td valign="top" align="left">Roundness</td><td valign="top" align="center"></td><td valign="top" align="center">×</td><td valign="top" align="center"></td></tr><tr><td valign="top" align="left">Shape Index</td><td valign="top" align="center"></td><td valign="top" align="center">×</td><td valign="top" align="center"></td></tr><tr><td valign="top" align="left">User Defined<a href="#tfn1-remotesensing-05-04163" class="html-table-fn"><sup>*</sup></a></td><td valign="top" align="center">×</td><td valign="top" align="center">×</td><td valign="top" align="center"></td></tr><tr><td valign="top" align="left">Area</td><td valign="top" align="center">×</td><td valign="top" align="center">×</td><td valign="top" align="center"></td></tr><tr><td valign="top" align="left">Neighbor Proximity (ground)</td><td valign="top" align="center">×</td><td valign="top" align="center"></td><td valign="top" align="center"></td></tr><tr><td valign="top" align="left">Maximum Lidar Height</td><td valign="top" align="center">×</td><td valign="top" align="center">×</td><td valign="top" align="center">×</td></tr><tr><td valign="top" align="left">Mean Lidar Height</td><td valign="top" align="center">×</td><td valign="top" align="center">×</td><td valign="top" align="center">×</td></tr><tr><td valign="top" align="left">Minimum Lidar Height</td><td valign="top" align="center">×</td><td valign="top" align="center"></td><td valign="top" align="center"></td></tr><tr><td valign="top" align="left">Mean Difference to Neighbor</td><td valign="top" align="center">×</td><td valign="top" align="center">×</td><td valign="top" align="center">×</td></tr><tr><td valign="top" align="left">8 Metrics: Mean(WV2 band i)</td><td valign="top" align="center"></td><td valign="top" align="center">×</td><td valign="top" align="center"></td></tr><tr><td valign="top" align="left">8 Metrics: Standard Deviation(WV2 band i)</td><td valign="top" align="center"></td><td valign="top" align="center">×</td><td valign="top" align="center"></td></tr><tr><td valign="top" align="left">NDVI (Maximum)</td><td valign="top" align="center"></td><td valign="top" align="center">×</td><td valign="top" align="center">×</td></tr><tr><td valign="top" align="left">NDVI (Mean)</td><td valign="top" align="center"></td><td valign="top" align="center">×</td><td valign="top" align="center">×</td></tr></tbody>
</table>
<p class="html-table_foot"><span class="html-fn-label"><sup id="tfn1-remotesensing-05-04163">*</sup></span><span class="html-fn-content">User Defined = (border index) × (shape index) × (compactness) × (roundness).</span></p>
</div></section>
<section><p id="html-copyright">© 2013 by the authors; licensee MDPI, 
Basel, Switzerland This article is an open access article distributed 
under the terms and conditions of the Creative Commons Attribution 
license ( <a href="https://creativecommons.org/licenses/by/3.0/" target="_blank">http://creativecommons.org/licenses/by/3.0/</a>).</p></section>
</div>
</article>
<style type="text/css">
  body{
    font-size: 11pt;
  }
</style>
</div>
</div>
</div>
</div>
<div class="row full-width">
<div class="small-12 columns journal-info">
<div class="clear"></div>
<div class="journal-info">
<span><em><a href="https://www.mdpi.com/journal/remotesensing">Remote Sens.</a></em></span>
<span>EISSN 2072-4292</span>
<span>Published by MDPI AG, Basel, Switzerland</span>
<span><a href="https://www.mdpi.com/rss/journal/remotesensing">RSS</a></span>
<span><a href="https://www.mdpi.com/journal/remotesensing/toc-alert">E-Mail Table of Contents Alert</a></span>
</div>
</div>
</div>
<div id="actionDisabledModal" class="reveal-modal" data-reveal="" aria-labelledby="actionDisableModalTitle" aria-hidden="true" role="dialog" style="width: 300px;">
<h2 id="actionDisableModalTitle">Notice</h2>
<form action="/email/captcha" method="post" id="emailCaptchaForm">
<div class="row">
<div id="js-action-disabled-modal-text" class="small-12 columns">
</div>
<div id="js-action-disabled-modal-submit" class="small-12 columns" style="margin-top: 10px; display: none;">
You can make submissions to other journals
<a href="https://susy.mdpi.com/user/manuscripts/upload" onclick="ga('send', 'event', 'Submit Paper', 'Send', 'Generic');">here</a>.
</div>
</div>
</form>
<a class="close-reveal-modal" aria-label="Close">×</a>
</div>
</div>
</section>
<footer>
<div class="row full-width footer-links" data-equalizer="footer" data-equalizer-mq="small">
<div class="large-2 medium-3 small-6 columns" data-equalizer-watch="footer" style="height: 234px;">
<h3>
Further Information
</h3>
<a href="https://www.mdpi.com/about/apc">
Article&nbsp;Processing&nbsp;Charges
</a>
<a href="https://payment.mdpi.com/" target="_blank">
Pay an Invoice
</a>
<a href="https://www.mdpi.com/about/openaccess">
Open Access Policy
</a>
<a href="https://www.mdpi.com/about/termsofuse">
Terms of Use
</a>
<a href="https://www.mdpi.com/about/terms-and-conditions">
Terms and Conditions
</a>
<a href="https://www.mdpi.com/about/privacy">
Privacy Policy
</a>
<a href="https://www.mdpi.com/about/contact">
Contact MDPI
</a>
<a href="https://www.mdpi.com/about/jobs">
Jobs at MDPI
</a>
</div>
<div class="large-2 medium-3 small-6 columns" data-equalizer-watch="footer" style="height: 234px;">
<h3>
Guidelines
</h3>
<a href="https://www.mdpi.com/authors">
For Authors
</a>
<a href="https://www.mdpi.com/reviewers">
 For Reviewers
</a>
<a href="https://www.mdpi.com/editors">
For Editors
</a>
<a href="https://www.mdpi.com/librarians">
For Librarians
</a>
<a href="https://www.mdpi.com/publishing_services">
For Publishers
</a>
<a href="https://www.mdpi.com/societies">
For Societies
</a>
</div>
<div class="large-2 medium-3 small-6 columns">
<h3>
MDPI Initiatives
</h3>
<a href="https://www.mdpi.com/about/ioap">
Institutional Open Access Program (IOAP)
</a>
<a href="https://sciforum.net/" target="_blank">
Sciforum
</a>
<a href="https://www.preprints.org/" target="_blank">
Preprints
</a>
<a href="https://www.scilit.net/" target="_blank">
Scilit
</a>
<a href="https://www.mdpi.com/books" target="_blank">
MDPI Books
</a>
<a href="https://encyclopedia.pub/" target="_blank">
Encyclopedia
</a>
<a href="http://blog.mdpi.com/" onclick="ga('send', 'pageview', 'homeBlog');" target="_blank">
MDPI Blog
</a>
</div>
<div class="large-2 medium-3 small-6 right-border-large-without columns">
<h3>
Follow MDPI
</h3>
<a href="https://www.linkedin.com/company/mdpi" target="_blank">
LinkedIn
</a>
<a href="https://www.facebook.com/MDPIOpenAccessPublishing" target="_blank">
Facebook
</a>
<a href="https://twitter.com/MDPIOpenAccess" target="_blank">
Twitter
</a>
</div>
<div id="footer-subscribe" class="large-4 medium-12 small-12 left-border-large columns">
<form id="newsletter" method="POST" action="/subscribe">
<h3>
Subscribe to receive issue release notifications and newsletters from MDPI journals
</h3>
Select Journal/Journals:
<select multiple="multiple" id="newsletter-journal" class="foundation-select jqmsLoaded" name="journals[]" style="display: none;">
<option value="acoustics">Acoustics</option>
<option value="actuators">Actuators</option>
<option value="admsci">Administrative Sciences</option>
<option value="aerospace">Aerospace</option>
<option value="agriculture">Agriculture</option>
<option value="agriengineering">AgriEngineering</option>
<option value="agronomy">Agronomy</option>
<option value="algorithms">Algorithms</option>
<option value="animals">Animals</option>
<option value="antibiotics">Antibiotics</option>
<option value="antibodies">Antibodies</option>
<option value="antioxidants">Antioxidants</option>
<option value="applsci">Applied Sciences</option>
<option value="asi">Applied System Innovation</option>
<option value="arts">Arts</option>
<option value="atmosphere">Atmosphere</option>
<option value="atoms">Atoms</option>
<option value="axioms">Axioms</option>
<option value="batteries">Batteries</option>
<option value="behavsci">Behavioral Sciences</option>
<option value="beverages">Beverages</option>
<option value="BDCC">Big Data and Cognitive Computing</option>
<option value="bioengineering">Bioengineering</option>
<option value="biology">Biology</option>
<option value="biomedicines">Biomedicines</option>
<option value="biomimetics">Biomimetics</option>
<option value="biomolecules">Biomolecules</option>
<option value="biosensors">Biosensors</option>
<option value="brainsci">Brain Sciences</option>
<option value="buildings">Buildings</option>
<option value="carbon">C</option>
<option value="cancers">Cancers</option>
<option value="catalysts">Catalysts</option>
<option value="cells">Cells</option>
<option value="ceramics">Ceramics</option>
<option value="challenges">Challenges</option>
<option value="ChemEngineering">ChemEngineering</option>
<option value="chemistry">Chemistry</option>
<option value="chemosensors">Chemosensors</option>
<option value="children">Children</option>
<option value="climate">Climate</option>
<option value="clockssleep">Clocks &amp; Sleep</option>
<option value="coatings">Coatings</option>
<option value="colloids">Colloids and Interfaces</option>
<option value="computation">Computation</option>
<option value="computers">Computers</option>
<option value="condensedmatter">Condensed Matter</option>
<option value="cosmetics">Cosmetics</option>
<option value="cryptography">Cryptography</option>
<option value="crystals">Crystals</option>
<option value="data">Data</option>
<option value="dentistry">Dentistry Journal</option>
<option value="designs">Designs</option>
<option value="diagnostics">Diagnostics</option>
<option value="diseases">Diseases</option>
<option value="diversity">Diversity</option>
<option value="drones">Drones</option>
<option value="econometrics">Econometrics</option>
<option value="economies">Economies</option>
<option value="education">Education Sciences</option>
<option value="electronics">Electronics</option>
<option value="energies">Energies</option>
<option value="entropy">Entropy</option>
<option value="environments">Environments</option>
<option value="epigenomes">Epigenomes</option>
<option value="fermentation">Fermentation</option>
<option value="fibers">Fibers</option>
<option value="fire">Fire</option>
<option value="fishes">Fishes</option>
<option value="fluids">Fluids</option>
<option value="foods">Foods</option>
<option value="forecasting">Forecasting</option>
<option value="forests">Forests</option>
<option value="fractalfract">Fractal and Fractional</option>
<option value="futureinternet">Future Internet</option>
<option value="galaxies">Galaxies</option>
<option value="games">Games</option>
<option value="gastrointestdisord">Gastrointestinal Disorders</option>
<option value="gels">Gels</option>
<option value="genealogy">Genealogy</option>
<option value="genes">Genes</option>
<option value="geosciences">Geosciences</option>
<option value="geriatrics">Geriatrics</option>
<option value="healthcare">Healthcare</option>
<option value="heritage">Heritage</option>
<option value="high-throughput">High-Throughput</option>
<option value="horticulturae">Horticulturae</option>
<option value="humanities">Humanities</option>
<option value="hydrology">Hydrology</option>
<option value="informatics">Informatics</option>
<option value="information">Information</option>
<option value="infrastructures">Infrastructures</option>
<option value="inorganics">Inorganics</option>
<option value="insects">Insects</option>
<option value="instruments">Instruments</option>
<option value="ijerph">International Journal of Environmental Research and Public Health</option>
<option value="ijfs">International Journal of Financial Studies</option>
<option value="ijms">International Journal of Molecular Sciences</option>
<option value="IJNS">International Journal of Neonatal Screening</option>
<option value="ijtpp">International Journal of Turbomachinery, Propulsion and Power</option>
<option value="inventions">Inventions</option>
<option value="ijgi">ISPRS International Journal of Geo-Information</option>
<option value="J">J</option>
<option value="jcdd">Journal of Cardiovascular Development and Disease</option>
<option value="jcm">Journal of Clinical Medicine</option>
<option value="jcs">Journal of Composites Science</option>
<option value="jdb">Journal of Developmental Biology</option>
<option value="jfb">Journal of Functional Biomaterials</option>
<option value="jfmk">Journal of Functional Morphology and Kinesiology</option>
<option value="jof">Journal of Fungi</option>
<option value="jimaging">Journal of Imaging</option>
<option value="jintelligence">Journal of Intelligence</option>
<option value="jlpea">Journal of Low Power Electronics and Applications</option>
<option value="jmmp">Journal of Manufacturing and Materials Processing</option>
<option value="jmse">Journal of Marine Science and Engineering</option>
<option value="JOItmC">Journal of Open Innovation: Technology, Market, and Complexity</option>
<option value="jpm">Journal of Personalized Medicine</option>
<option value="jrfm">Journal of Risk and Financial Management</option>
<option value="jsan">Journal of Sensor and Actuator Networks</option>
<option value="land">Land</option>
<option value="languages">Languages</option>
<option value="laws">Laws</option>
<option value="life">Life</option>
<option value="logistics">Logistics</option>
<option value="lubricants">Lubricants</option>
<option value="make">Machine Learning and Knowledge Extraction</option>
<option value="machines">Machines</option>
<option value="magnetochemistry">Magnetochemistry</option>
<option value="marinedrugs">Marine Drugs</option>
<option value="materials">Materials</option>
<option value="mca">Mathematical and Computational Applications</option>
<option value="mathematics">Mathematics</option>
<option value="medsci">Medical Sciences</option>
<option value="medicina">Medicina</option>
<option value="medicines">Medicines</option>
<option value="membranes">Membranes</option>
<option value="metabolites">Metabolites</option>
<option value="metals">Metals</option>
<option value="mps">Methods and Protocols</option>
<option value="micromachines">Micromachines</option>
<option value="microorganisms">Microorganisms</option>
<option value="minerals">Minerals</option>
<option value="molbank">Molbank</option>
<option value="molecules">Molecules</option>
<option value="mti">Multimodal Technologies and Interaction</option>
<option value="nanomaterials">Nanomaterials</option>
<option value="neuroglia">Neuroglia</option>
<option value="ncrna">Non-Coding RNA</option>
<option value="nutrients">Nutrients</option>
<option value="particles">Particles</option>
<option value="pathogens">Pathogens</option>
<option value="pharmaceuticals">Pharmaceuticals</option>
<option value="pharmaceutics">Pharmaceutics</option>
<option value="pharmacy">Pharmacy</option>
<option value="philosophies">Philosophies</option>
<option value="photonics">Photonics</option>
<option value="physics">Physics</option>
<option value="plants">Plants</option>
<option value="plasma">Plasma</option>
<option value="polymers">Polymers</option>
<option value="proceedings">Proceedings</option>
<option value="processes">Processes</option>
<option value="proteomes">Proteomes</option>
<option value="psych">Psych</option>
<option value="publications">Publications</option>
<option value="qubs">Quantum Beam Science</option>
<option value="quantumrep">Quantum Reports</option>
<option value="quaternary">Quaternary</option>
<option value="recycling">Recycling</option>
<option value="religions">Religions</option>
<option value="remotesensing">Remote Sensing</option>
<option value="reports">Reports</option>
<option value="resources">Resources</option>
<option value="risks">Risks</option>
<option value="robotics">Robotics</option>
<option value="safety">Safety</option>
<option value="sci">Sci</option>
<option value="scipharm">Scientia Pharmaceutica</option>
<option value="sensors">Sensors</option>
<option value="separations">Separations</option>
 <option value="smartcities">Smart Cities</option>
<option value="socsci">Social Sciences</option>
<option value="societies">Societies</option>
<option value="soilsystems">Soil Systems</option>
<option value="sports">Sports</option>
<option value="stats">Stats</option>
<option value="surfaces">Surfaces</option>
<option value="sustainability">Sustainability</option>
<option value="symmetry">Symmetry</option>
<option value="systems">Systems</option>
<option value="technologies">Technologies</option>
<option value="toxics">Toxics</option>
<option value="toxins">Toxins</option>
<option value="tropicalmed">Tropical Medicine and Infectious Disease</option>
<option value="universe">Universe</option>
<option value="urbansci">Urban Science</option>
<option value="vaccines">Vaccines</option>
<option value="vehicles">Vehicles</option>
<option value="vetsci">Veterinary Sciences</option>
<option value="vibration">Vibration</option>
<option value="viruses">Viruses</option>
<option value="vision">Vision</option>
<option value="water">Water</option>
<option value="wevj">World Electric Vehicle Journal</option>
</select><div class="ms-options-wrap" style="position: relative;"><button>Select options</button><div class="ms-options" style="min-height: 130px; max-height: 130px; overflow: auto; display: none;"><div class="ms-search"><input placeholder="Search" type="text"></div><ul style="column-count: 1; column-gap: 0px;"><li class=""><label for="ms-opt-1" style="padding-left: 21px;"><input value="acoustics" title="Acoustics" id="ms-opt-1" type="checkbox">Acoustics</label></li><li class=""><label for="ms-opt-2" style="padding-left: 21px;"><input value="actuators" title="Actuators" id="ms-opt-2" type="checkbox">Actuators</label></li><li class=""><label for="ms-opt-3" style="padding-left: 21px;"><input value="admsci" title="Administrative Sciences" id="ms-opt-3" type="checkbox">Administrative Sciences</label></li><li class=""><label for="ms-opt-4" style="padding-left: 21px;"><input value="aerospace" title="Aerospace" id="ms-opt-4" type="checkbox">Aerospace</label></li><li class=""><label for="ms-opt-5" style="padding-left: 21px;"><input value="agriculture" title="Agriculture" id="ms-opt-5" type="checkbox">Agriculture</label></li><li class=""><label for="ms-opt-6" style="padding-left: 21px;"><input value="agriengineering" title="AgriEngineering" id="ms-opt-6" type="checkbox">AgriEngineering</label></li><li class=""><label for="ms-opt-7" style="padding-left: 21px;"><input value="agronomy" title="Agronomy" id="ms-opt-7" type="checkbox">Agronomy</label></li><li class=""><label for="ms-opt-8" style="padding-left: 21px;"><input value="algorithms" title="Algorithms" id="ms-opt-8" type="checkbox">Algorithms</label></li><li class=""><label for="ms-opt-9" style="padding-left: 21px;"><input value="animals" title="Animals" id="ms-opt-9" type="checkbox">Animals</label></li><li class=""><label for="ms-opt-10" style="padding-left: 21px;"><input value="antibiotics" title="Antibiotics" id="ms-opt-10" type="checkbox">Antibiotics</label></li><li class=""><label for="ms-opt-11" style="padding-left: 21px;"><input value="antibodies" title="Antibodies" id="ms-opt-11" type="checkbox">Antibodies</label></li><li class=""><label for="ms-opt-12" style="padding-left: 21px;"><input value="antioxidants" title="Antioxidants" id="ms-opt-12" type="checkbox">Antioxidants</label></li><li class=""><label for="ms-opt-13" style="padding-left: 21px;"><input value="applsci" title="Applied Sciences" id="ms-opt-13" type="checkbox">Applied Sciences</label></li><li class=""><label for="ms-opt-14" style="padding-left: 21px;"><input value="asi" title="Applied System Innovation" id="ms-opt-14" type="checkbox">Applied System Innovation</label></li><li class=""><label for="ms-opt-15" style="padding-left: 21px;"><input value="arts" title="Arts" id="ms-opt-15" type="checkbox">Arts</label></li><li class=""><label for="ms-opt-16" style="padding-left: 21px;"><input value="atmosphere" title="Atmosphere" id="ms-opt-16" type="checkbox">Atmosphere</label></li><li class=""><label for="ms-opt-17" style="padding-left: 21px;"><input value="atoms" title="Atoms" id="ms-opt-17" type="checkbox">Atoms</label></li><li class=""><label for="ms-opt-18" style="padding-left: 21px;"><input value="axioms" title="Axioms" id="ms-opt-18" type="checkbox">Axioms</label></li><li class=""><label for="ms-opt-19" style="padding-left: 21px;"><input value="batteries" title="Batteries" id="ms-opt-19" type="checkbox">Batteries</label></li><li class=""><label for="ms-opt-20" style="padding-left: 21px;"><input value="behavsci" title="Behavioral Sciences" id="ms-opt-20" type="checkbox">Behavioral Sciences</label></li><li class=""><label for="ms-opt-21" style="padding-left: 21px;"><input value="beverages" title="Beverages" id="ms-opt-21" type="checkbox">Beverages</label></li><li class=""><label for="ms-opt-22" style="padding-left: 21px;"><input value="BDCC" title="Big Data and Cognitive Computing" id="ms-opt-22" type="checkbox">Big Data and Cognitive Computing</label></li><li class=""><label for="ms-opt-23" style="padding-left: 21px;"><input value="bioengineering" title="Bioengineering" id="ms-opt-23" type="checkbox">Bioengineering</label></li><li class=""><label for="ms-opt-24" style="padding-left: 21px;"><input value="biology" title="Biology" id="ms-opt-24" type="checkbox">Biology</label></li><li class=""><label for="ms-opt-25" style="padding-left: 21px;"><input value="biomedicines" title="Biomedicines" id="ms-opt-25" type="checkbox">Biomedicines</label></li><li class=""><label for="ms-opt-26" style="padding-left: 21px;"><input value="biomimetics" title="Biomimetics" id="ms-opt-26" type="checkbox">Biomimetics</label></li><li class=""><label for="ms-opt-27" style="padding-left: 21px;"><input value="biomolecules" title="Biomolecules" id="ms-opt-27" type="checkbox">Biomolecules</label></li><li class=""><label for="ms-opt-28" style="padding-left: 21px;"><input value="biosensors" title="Biosensors" id="ms-opt-28" type="checkbox">Biosensors</label></li><li class=""><label for="ms-opt-29" style="padding-left: 21px;"><input value="brainsci" title="Brain Sciences" id="ms-opt-29" type="checkbox">Brain Sciences</label></li><li class=""><label for="ms-opt-30" style="padding-left: 21px;"><input value="buildings" title="Buildings" id="ms-opt-30" type="checkbox">Buildings</label></li><li class=""><label for="ms-opt-31" style="padding-left: 21px;"><input value="carbon" title="C" id="ms-opt-31" type="checkbox">C</label></li><li class=""><label for="ms-opt-32" style="padding-left: 21px;"><input value="cancers" title="Cancers" id="ms-opt-32" type="checkbox">Cancers</label></li><li class=""><label for="ms-opt-33" style="padding-left: 21px;"><input value="catalysts" title="Catalysts" id="ms-opt-33" type="checkbox">Catalysts</label></li><li class=""><label for="ms-opt-34" style="padding-left: 21px;"><input value="cells" title="Cells" id="ms-opt-34" type="checkbox">Cells</label></li><li class=""><label for="ms-opt-35" style="padding-left: 21px;"><input value="ceramics" title="Ceramics" id="ms-opt-35" type="checkbox">Ceramics</label></li><li class=""><label for="ms-opt-36" style="padding-left: 21px;"><input value="challenges" title="Challenges" id="ms-opt-36" type="checkbox">Challenges</label></li><li class=""><label for="ms-opt-37" style="padding-left: 21px;"><input value="ChemEngineering" title="ChemEngineering" id="ms-opt-37" type="checkbox">ChemEngineering</label></li><li class=""><label for="ms-opt-38" style="padding-left: 21px;"><input value="chemistry" title="Chemistry" id="ms-opt-38" type="checkbox">Chemistry</label></li><li class=""><label for="ms-opt-39" style="padding-left: 21px;"><input value="chemosensors" title="Chemosensors" id="ms-opt-39" type="checkbox">Chemosensors</label></li><li class=""><label for="ms-opt-40" style="padding-left: 21px;"><input value="children" title="Children" id="ms-opt-40" type="checkbox">Children</label></li><li class=""><label for="ms-opt-41" style="padding-left: 21px;"><input value="climate" title="Climate" id="ms-opt-41" type="checkbox">Climate</label></li><li class=""><label for="ms-opt-42" style="padding-left: 21px;"><input value="clockssleep" title="Clocks &amp; Sleep" id="ms-opt-42" type="checkbox">Clocks &amp; Sleep</label></li><li class=""><label for="ms-opt-43" style="padding-left: 21px;"><input value="coatings" title="Coatings" id="ms-opt-43" type="checkbox">Coatings</label></li><li class=""><label for="ms-opt-44" style="padding-left: 21px;"><input value="colloids" title="Colloids and Interfaces" id="ms-opt-44" type="checkbox">Colloids and Interfaces</label></li><li class=""><label for="ms-opt-45" style="padding-left: 21px;"><input value="computation" title="Computation" id="ms-opt-45" type="checkbox">Computation</label></li><li class=""><label for="ms-opt-46" style="padding-left: 21px;"><input value="computers" title="Computers" id="ms-opt-46" type="checkbox">Computers</label></li><li class=""><label for="ms-opt-47" style="padding-left: 21px;"><input value="condensedmatter" title="Condensed Matter" id="ms-opt-47" type="checkbox">Condensed Matter</label></li><li class=""><label for="ms-opt-48" style="padding-left: 21px;"><input value="cosmetics" title="Cosmetics" id="ms-opt-48" type="checkbox">Cosmetics</label></li><li class=""><label for="ms-opt-49" style="padding-left: 21px;"><input value="cryptography" title="Cryptography" id="ms-opt-49" type="checkbox">Cryptography</label></li><li class=""><label for="ms-opt-50" style="padding-left: 21px;"><input value="crystals" title="Crystals" id="ms-opt-50" type="checkbox">Crystals</label></li><li class=""><label for="ms-opt-51" style="padding-left: 21px;"><input value="data" title="Data" id="ms-opt-51" type="checkbox">Data</label></li><li class=""><label for="ms-opt-52" style="padding-left: 21px;"><input value="dentistry" title="Dentistry Journal" id="ms-opt-52" type="checkbox">Dentistry Journal</label></li><li class=""><label for="ms-opt-53" style="padding-left: 21px;"><input value="designs" title="Designs" id="ms-opt-53" type="checkbox">Designs</label></li><li class=""><label for="ms-opt-54" style="padding-left: 21px;"><input value="diagnostics" title="Diagnostics" id="ms-opt-54" type="checkbox">Diagnostics</label></li><li class=""><label for="ms-opt-55" style="padding-left: 21px;"><input value="diseases" title="Diseases" id="ms-opt-55" type="checkbox">Diseases</label></li><li class=""><label for="ms-opt-56" style="padding-left: 21px;"><input value="diversity" title="Diversity" id="ms-opt-56" type="checkbox">Diversity</label></li><li class=""><label for="ms-opt-57" style="padding-left: 21px;"><input value="drones" title="Drones" id="ms-opt-57" type="checkbox">Drones</label></li><li class=""><label for="ms-opt-58" style="padding-left: 21px;"><input value="econometrics" title="Econometrics" id="ms-opt-58" type="checkbox">Econometrics</label></li><li class=""><label for="ms-opt-59" style="padding-left: 21px;"><input value="economies" title="Economies" id="ms-opt-59" type="checkbox">Economies</label></li><li class=""><label for="ms-opt-60" style="padding-left: 21px;"><input value="education" title="Education Sciences" id="ms-opt-60" type="checkbox">Education Sciences</label></li><li class=""><label for="ms-opt-61" style="padding-left: 21px;"><input value="electronics" title="Electronics" id="ms-opt-61" type="checkbox">Electronics</label></li><li class=""><label for="ms-opt-62" style="padding-left: 21px;"><input value="energies" title="Energies" id="ms-opt-62" type="checkbox">Energies</label></li><li class=""><label for="ms-opt-63" style="padding-left: 21px;"><input value="entropy" title="Entropy" id="ms-opt-63" type="checkbox">Entropy</label></li><li class=""><label for="ms-opt-64" style="padding-left: 21px;"><input value="environments" title="Environments" id="ms-opt-64" type="checkbox">Environments</label></li><li class=""><label for="ms-opt-65" style="padding-left: 21px;"><input value="epigenomes" title="Epigenomes" id="ms-opt-65" type="checkbox">Epigenomes</label></li><li class=""><label for="ms-opt-66" style="padding-left: 21px;"><input value="fermentation" title="Fermentation" id="ms-opt-66" type="checkbox">Fermentation</label></li><li class=""><label for="ms-opt-67" style="padding-left: 21px;"><input value="fibers" title="Fibers" id="ms-opt-67" type="checkbox">Fibers</label></li><li class=""><label for="ms-opt-68" style="padding-left: 21px;"><input value="fire" title="Fire" id="ms-opt-68" type="checkbox">Fire</label></li><li class=""><label for="ms-opt-69" style="padding-left: 21px;"><input value="fishes" title="Fishes" id="ms-opt-69" type="checkbox">Fishes</label></li><li class=""><label for="ms-opt-70" style="padding-left: 21px;"><input value="fluids" title="Fluids" id="ms-opt-70" type="checkbox">Fluids</label></li><li class=""><label for="ms-opt-71" style="padding-left: 21px;"><input value="foods" title="Foods" id="ms-opt-71" type="checkbox">Foods</label></li><li class=""><label for="ms-opt-72" style="padding-left: 21px;"><input value="forecasting" title="Forecasting" id="ms-opt-72" type="checkbox">Forecasting</label></li><li class=""><label for="ms-opt-73" style="padding-left: 21px;"><input value="forests" title="Forests" id="ms-opt-73" type="checkbox">Forests</label></li><li class=""><label for="ms-opt-74" style="padding-left: 21px;"><input value="fractalfract" title="Fractal and Fractional" id="ms-opt-74" type="checkbox">Fractal and Fractional</label></li><li class=""><label for="ms-opt-75" style="padding-left: 21px;"><input value="futureinternet" title="Future Internet" id="ms-opt-75" type="checkbox">Future Internet</label></li><li class=""><label for="ms-opt-76" style="padding-left: 21px;"><input value="galaxies" title="Galaxies" id="ms-opt-76" type="checkbox">Galaxies</label></li><li class=""><label for="ms-opt-77" style="padding-left: 21px;"><input value="games" title="Games" id="ms-opt-77" type="checkbox">Games</label></li><li class=""><label for="ms-opt-78" style="padding-left: 21px;"><input value="gastrointestdisord" title="Gastrointestinal Disorders" id="ms-opt-78" type="checkbox">Gastrointestinal Disorders</label></li><li class=""><label for="ms-opt-79" style="padding-left: 21px;"><input value="gels" title="Gels" id="ms-opt-79" type="checkbox">Gels</label></li><li class=""><label for="ms-opt-80" style="padding-left: 21px;"><input value="genealogy" title="Genealogy" id="ms-opt-80" type="checkbox">Genealogy</label></li><li class=""><label for="ms-opt-81" style="padding-left: 21px;"><input value="genes" title="Genes" id="ms-opt-81" type="checkbox">Genes</label></li><li class=""><label for="ms-opt-82" style="padding-left: 21px;"><input value="geosciences" title="Geosciences" id="ms-opt-82" type="checkbox">Geosciences</label></li><li class=""><label for="ms-opt-83" style="padding-left: 21px;"><input value="geriatrics" title="Geriatrics" id="ms-opt-83" type="checkbox">Geriatrics</label></li><li class=""><label for="ms-opt-84" style="padding-left: 21px;"><input value="healthcare" title="Healthcare" id="ms-opt-84" type="checkbox">Healthcare</label></li><li class=""><label for="ms-opt-85" style="padding-left: 21px;"><input value="heritage" title="Heritage" id="ms-opt-85" type="checkbox">Heritage</label></li><li class=""><label for="ms-opt-86" style="padding-left: 21px;"><input value="high-throughput" title="High-Throughput" id="ms-opt-86" type="checkbox">High-Throughput</label></li><li class=""><label for="ms-opt-87" style="padding-left: 21px;"><input value="horticulturae" title="Horticulturae" id="ms-opt-87" type="checkbox">Horticulturae</label></li><li class=""><label for="ms-opt-88" style="padding-left: 21px;"><input value="humanities" title="Humanities" id="ms-opt-88" type="checkbox">Humanities</label></li><li class=""><label for="ms-opt-89" style="padding-left: 21px;"><input value="hydrology" title="Hydrology" id="ms-opt-89" type="checkbox">Hydrology</label></li><li class=""><label for="ms-opt-90" style="padding-left: 21px;"><input value="informatics" title="Informatics" id="ms-opt-90" type="checkbox">Informatics</label></li><li class=""><label for="ms-opt-91" style="padding-left: 21px;"><input value="information" title="Information" id="ms-opt-91" type="checkbox">Information</label></li><li class=""><label for="ms-opt-92" style="padding-left: 21px;"><input value="infrastructures" title="Infrastructures" id="ms-opt-92" type="checkbox">Infrastructures</label></li><li class=""><label for="ms-opt-93" style="padding-left: 21px;"><input value="inorganics" title="Inorganics" id="ms-opt-93" type="checkbox">Inorganics</label></li><li class=""><label for="ms-opt-94" style="padding-left: 21px;"><input value="insects" title="Insects" id="ms-opt-94" type="checkbox">Insects</label></li><li class=""><label for="ms-opt-95" style="padding-left: 21px;"><input value="instruments" title="Instruments" id="ms-opt-95" type="checkbox">Instruments</label></li><li class=""><label for="ms-opt-96" style="padding-left: 21px;"><input value="ijerph" title="International Journal of Environmental Research and Public Health" id="ms-opt-96" type="checkbox">International Journal of Environmental Research and Public Health</label></li><li class=""><label for="ms-opt-97" style="padding-left: 21px;"><input value="ijfs" title="International Journal of Financial Studies" id="ms-opt-97" type="checkbox">International Journal of Financial Studies</label></li><li class=""><label for="ms-opt-98" style="padding-left: 21px;"><input value="ijms" title="International Journal of Molecular Sciences" id="ms-opt-98" type="checkbox">International Journal of Molecular Sciences</label></li><li class=""><label for="ms-opt-99" style="padding-left: 21px;"><input value="IJNS" title="International Journal of Neonatal Screening" id="ms-opt-99" type="checkbox">International Journal of Neonatal Screening</label></li><li class=""><label for="ms-opt-100" style="padding-left: 21px;"><input value="ijtpp" title="International Journal of Turbomachinery, Propulsion and Power" id="ms-opt-100" type="checkbox">International Journal of Turbomachinery, Propulsion and Power</label></li><li class=""><label for="ms-opt-101" style="padding-left: 21px;"><input value="inventions" title="Inventions" id="ms-opt-101" type="checkbox">Inventions</label></li><li class=""><label for="ms-opt-102" style="padding-left: 21px;"><input value="ijgi" title="ISPRS International Journal of Geo-Information" id="ms-opt-102" type="checkbox">ISPRS International Journal of Geo-Information</label></li><li class=""><label for="ms-opt-103" style="padding-left: 21px;"><input value="J" title="J" id="ms-opt-103" type="checkbox">J</label></li><li class=""><label for="ms-opt-104" style="padding-left: 21px;"><input value="jcdd" title="Journal of Cardiovascular Development and Disease" id="ms-opt-104" type="checkbox">Journal of Cardiovascular Development and Disease</label></li><li class=""><label for="ms-opt-105" style="padding-left: 21px;"><input value="jcm" title="Journal of Clinical Medicine" id="ms-opt-105" type="checkbox">Journal of Clinical Medicine</label></li><li class=""><label for="ms-opt-106" style="padding-left: 21px;"><input value="jcs" title="Journal of Composites Science" id="ms-opt-106" type="checkbox">Journal of Composites Science</label></li><li class=""><label for="ms-opt-107" style="padding-left: 21px;"><input value="jdb" title="Journal of Developmental Biology" id="ms-opt-107" type="checkbox">Journal of Developmental Biology</label></li><li class=""><label for="ms-opt-108" style="padding-left: 21px;"><input value="jfb" title="Journal of Functional Biomaterials" id="ms-opt-108" type="checkbox">Journal of Functional Biomaterials</label></li><li class=""><label for="ms-opt-109" style="padding-left: 21px;"><input value="jfmk" title="Journal of Functional Morphology and Kinesiology" id="ms-opt-109" type="checkbox">Journal of Functional Morphology and Kinesiology</label></li><li class=""><label for="ms-opt-110" style="padding-left: 21px;"><input value="jof" title="Journal of Fungi" id="ms-opt-110" type="checkbox">Journal of Fungi</label></li><li class=""><label for="ms-opt-111" style="padding-left: 21px;"><input value="jimaging" title="Journal of Imaging" id="ms-opt-111" type="checkbox">Journal of Imaging</label></li><li class=""><label for="ms-opt-112" style="padding-left: 21px;"><input value="jintelligence" title="Journal of Intelligence" id="ms-opt-112" type="checkbox">Journal of Intelligence</label></li><li class=""><label for="ms-opt-113" style="padding-left: 21px;"><input value="jlpea" title="Journal of Low Power Electronics and Applications" id="ms-opt-113" type="checkbox">Journal of Low Power Electronics and Applications</label></li><li class=""><label for="ms-opt-114" style="padding-left: 21px;"><input value="jmmp" title="Journal of Manufacturing and Materials Processing" id="ms-opt-114" type="checkbox">Journal of Manufacturing and Materials Processing</label></li><li class=""><label for="ms-opt-115" style="padding-left: 21px;"><input value="jmse" title="Journal of Marine Science and Engineering" id="ms-opt-115" type="checkbox">Journal of Marine Science and Engineering</label></li><li class=""><label for="ms-opt-116" style="padding-left: 21px;"><input value="JOItmC" title="Journal of Open Innovation: Technology, Market, and Complexity" id="ms-opt-116" type="checkbox">Journal of Open Innovation: Technology, Market, and Complexity</label></li><li class=""><label for="ms-opt-117" style="padding-left: 21px;"><input value="jpm" title="Journal of Personalized Medicine" id="ms-opt-117" type="checkbox">Journal of Personalized Medicine</label></li><li class=""><label for="ms-opt-118" style="padding-left: 21px;"><input value="jrfm" title="Journal of Risk and Financial Management" id="ms-opt-118" type="checkbox">Journal of Risk and Financial Management</label></li><li class=""><label for="ms-opt-119" style="padding-left: 21px;"><input value="jsan" title="Journal of Sensor and Actuator Networks" id="ms-opt-119" type="checkbox">Journal of Sensor and Actuator Networks</label></li><li class=""><label for="ms-opt-120" style="padding-left: 21px;"><input value="land" title="Land" id="ms-opt-120" type="checkbox">Land</label></li><li class=""><label for="ms-opt-121" style="padding-left: 21px;"><input value="languages" title="Languages" id="ms-opt-121" type="checkbox">Languages</label></li><li class=""><label for="ms-opt-122" style="padding-left: 21px;"><input value="laws" title="Laws" id="ms-opt-122" type="checkbox">Laws</label></li><li class=""><label for="ms-opt-123" style="padding-left: 21px;"><input value="life" title="Life" id="ms-opt-123" type="checkbox">Life</label></li><li class=""><label for="ms-opt-124" style="padding-left: 21px;"><input value="logistics" title="Logistics" id="ms-opt-124" type="checkbox">Logistics</label></li><li class=""><label for="ms-opt-125" style="padding-left: 21px;"><input value="lubricants" title="Lubricants" id="ms-opt-125" type="checkbox">Lubricants</label></li><li class=""><label for="ms-opt-126" style="padding-left: 21px;"><input value="make" title="Machine Learning and Knowledge Extraction" id="ms-opt-126" type="checkbox">Machine Learning and Knowledge Extraction</label></li><li class=""><label for="ms-opt-127" style="padding-left: 21px;"><input value="machines" title="Machines" id="ms-opt-127" type="checkbox">Machines</label></li><li class=""><label for="ms-opt-128" style="padding-left: 21px;"><input value="magnetochemistry" title="Magnetochemistry" id="ms-opt-128" type="checkbox">Magnetochemistry</label></li><li class=""><label for="ms-opt-129" style="padding-left: 21px;"><input value="marinedrugs" title="Marine Drugs" id="ms-opt-129" type="checkbox">Marine Drugs</label></li><li class=""><label for="ms-opt-130" style="padding-left: 21px;"><input value="materials" title="Materials" id="ms-opt-130" type="checkbox">Materials</label></li><li class=""><label for="ms-opt-131" style="padding-left: 21px;"><input value="mca" title="Mathematical and Computational Applications" id="ms-opt-131" type="checkbox">Mathematical and Computational Applications</label></li><li class=""><label for="ms-opt-132" style="padding-left: 21px;"><input value="mathematics" title="Mathematics" id="ms-opt-132" type="checkbox">Mathematics</label></li><li class=""><label for="ms-opt-133" style="padding-left: 21px;"><input value="medsci" title="Medical Sciences" id="ms-opt-133" type="checkbox">Medical Sciences</label></li><li class=""><label for="ms-opt-134" style="padding-left: 21px;"><input value="medicina" title="Medicina" id="ms-opt-134" type="checkbox">Medicina</label></li><li class=""><label for="ms-opt-135" style="padding-left: 21px;"><input value="medicines" title="Medicines" id="ms-opt-135" type="checkbox">Medicines</label></li><li class=""><label for="ms-opt-136" style="padding-left: 21px;"><input value="membranes" title="Membranes" id="ms-opt-136" type="checkbox">Membranes</label></li><li class=""><label for="ms-opt-137" style="padding-left: 21px;"><input value="metabolites" title="Metabolites" id="ms-opt-137" type="checkbox">Metabolites</label></li><li class=""><label for="ms-opt-138" style="padding-left: 21px;"><input value="metals" title="Metals" id="ms-opt-138" type="checkbox">Metals</label></li><li class=""><label for="ms-opt-139" style="padding-left: 21px;"><input value="mps" title="Methods and Protocols" id="ms-opt-139" type="checkbox">Methods and Protocols</label></li><li class=""><label for="ms-opt-140" style="padding-left: 21px;"><input value="micromachines" title="Micromachines" id="ms-opt-140" type="checkbox">Micromachines</label></li><li class=""><label for="ms-opt-141" style="padding-left: 21px;"><input value="microorganisms" title="Microorganisms" id="ms-opt-141" type="checkbox">Microorganisms</label></li><li class=""><label for="ms-opt-142" style="padding-left: 21px;"><input value="minerals" title="Minerals" id="ms-opt-142" type="checkbox">Minerals</label></li><li class=""><label for="ms-opt-143" style="padding-left: 21px;"><input value="molbank" title="Molbank" id="ms-opt-143" type="checkbox">Molbank</label></li><li class=""><label for="ms-opt-144" style="padding-left: 21px;"><input value="molecules" title="Molecules" id="ms-opt-144" type="checkbox">Molecules</label></li><li class=""><label for="ms-opt-145" style="padding-left: 21px;"><input value="mti" title="Multimodal Technologies and Interaction" id="ms-opt-145" type="checkbox">Multimodal Technologies and Interaction</label></li><li class=""><label for="ms-opt-146" style="padding-left: 21px;"><input value="nanomaterials" title="Nanomaterials" id="ms-opt-146" type="checkbox">Nanomaterials</label></li><li class=""><label for="ms-opt-147" style="padding-left: 21px;"><input value="neuroglia" title="Neuroglia" id="ms-opt-147" type="checkbox">Neuroglia</label></li><li class=""><label for="ms-opt-148" style="padding-left: 21px;"><input value="ncrna" title="Non-Coding RNA" id="ms-opt-148" type="checkbox">Non-Coding RNA</label></li><li class=""><label for="ms-opt-149" style="padding-left: 21px;"><input value="nutrients" title="Nutrients" id="ms-opt-149" type="checkbox">Nutrients</label></li><li class=""><label for="ms-opt-150" style="padding-left: 21px;"><input value="particles" title="Particles" id="ms-opt-150" type="checkbox">Particles</label></li><li class=""><label for="ms-opt-151" style="padding-left: 21px;"><input value="pathogens" title="Pathogens" id="ms-opt-151" type="checkbox">Pathogens</label></li><li class=""><label for="ms-opt-152" style="padding-left: 21px;"><input value="pharmaceuticals" title="Pharmaceuticals" id="ms-opt-152" type="checkbox">Pharmaceuticals</label></li><li class=""><label for="ms-opt-153" style="padding-left: 21px;"><input value="pharmaceutics" title="Pharmaceutics" id="ms-opt-153" type="checkbox">Pharmaceutics</label></li><li class=""><label for="ms-opt-154" style="padding-left: 21px;"><input value="pharmacy" title="Pharmacy" id="ms-opt-154" type="checkbox">Pharmacy</label></li><li class=""><label for="ms-opt-155" style="padding-left: 21px;"><input value="philosophies" title="Philosophies" id="ms-opt-155" type="checkbox">Philosophies</label></li><li class=""><label for="ms-opt-156" style="padding-left: 21px;"><input value="photonics" title="Photonics" id="ms-opt-156" type="checkbox">Photonics</label></li><li class=""><label for="ms-opt-157" style="padding-left: 21px;"><input value="physics" title="Physics" id="ms-opt-157" type="checkbox">Physics</label></li><li class=""><label for="ms-opt-158" style="padding-left: 21px;"><input value="plants" title="Plants" id="ms-opt-158" type="checkbox">Plants</label></li><li class=""><label for="ms-opt-159" style="padding-left: 21px;"><input value="plasma" title="Plasma" id="ms-opt-159" type="checkbox">Plasma</label></li><li class=""><label for="ms-opt-160" style="padding-left: 21px;"><input value="polymers" title="Polymers" id="ms-opt-160" type="checkbox">Polymers</label></li><li class=""><label for="ms-opt-161" style="padding-left: 21px;"><input value="proceedings" title="Proceedings" id="ms-opt-161" type="checkbox">Proceedings</label></li><li class=""><label for="ms-opt-162" style="padding-left: 21px;"><input value="processes" title="Processes" id="ms-opt-162" type="checkbox">Processes</label></li><li class=""><label for="ms-opt-163" style="padding-left: 21px;"><input value="proteomes" title="Proteomes" id="ms-opt-163" type="checkbox">Proteomes</label></li><li class=""><label for="ms-opt-164" style="padding-left: 21px;"><input value="psych" title="Psych" id="ms-opt-164" type="checkbox">Psych</label></li><li class=""><label for="ms-opt-165" style="padding-left: 21px;"><input value="publications" title="Publications" id="ms-opt-165" type="checkbox">Publications</label></li><li class=""><label for="ms-opt-166" style="padding-left: 21px;"><input value="qubs" title="Quantum Beam Science" id="ms-opt-166" type="checkbox">Quantum Beam Science</label></li><li class=""><label for="ms-opt-167" style="padding-left: 21px;"><input value="quantumrep" title="Quantum Reports" id="ms-opt-167" type="checkbox">Quantum Reports</label></li><li class=""><label for="ms-opt-168" style="padding-left: 21px;"><input value="quaternary" title="Quaternary" id="ms-opt-168" type="checkbox">Quaternary</label></li><li class=""><label for="ms-opt-169" style="padding-left: 21px;"><input value="recycling" title="Recycling" id="ms-opt-169" type="checkbox">Recycling</label></li><li class=""><label for="ms-opt-170" style="padding-left: 21px;"><input value="religions" title="Religions" id="ms-opt-170" type="checkbox">Religions</label></li><li class=""><label for="ms-opt-171" style="padding-left: 21px;"><input value="remotesensing" title="Remote Sensing" id="ms-opt-171" type="checkbox">Remote Sensing</label></li><li class=""><label for="ms-opt-172" style="padding-left: 21px;"><input value="reports" title="Reports" id="ms-opt-172" type="checkbox">Reports</label></li><li class=""><label for="ms-opt-173" style="padding-left: 21px;"><input value="resources" title="Resources" id="ms-opt-173" type="checkbox">Resources</label></li><li class=""><label for="ms-opt-174" style="padding-left: 21px;"><input value="risks" title="Risks" id="ms-opt-174" type="checkbox">Risks</label></li><li class=""><label for="ms-opt-175" style="padding-left: 21px;"><input value="robotics" title="Robotics" id="ms-opt-175" type="checkbox">Robotics</label></li><li class=""><label for="ms-opt-176" style="padding-left: 21px;"><input value="safety" title="Safety" id="ms-opt-176" type="checkbox">Safety</label></li><li class=""><label for="ms-opt-177" style="padding-left: 21px;"><input value="sci" title="Sci" id="ms-opt-177" type="checkbox">Sci</label></li><li class=""><label for="ms-opt-178" style="padding-left: 21px;"><input value="scipharm" title="Scientia Pharmaceutica" id="ms-opt-178" type="checkbox">Scientia Pharmaceutica</label></li><li class=""><label for="ms-opt-179" style="padding-left: 21px;"><input value="sensors" title="Sensors" id="ms-opt-179" type="checkbox">Sensors</label></li><li class=""><label for="ms-opt-180" style="padding-left: 21px;"><input value="separations" title="Separations" id="ms-opt-180" type="checkbox">Separations</label></li><li class=""><label for="ms-opt-181" style="padding-left: 21px;"><input value="smartcities" title="Smart Cities" id="ms-opt-181" type="checkbox">Smart Cities</label></li><li class=""><label for="ms-opt-182" style="padding-left: 21px;"><input value="socsci" title="Social Sciences" id="ms-opt-182" type="checkbox">Social Sciences</label></li><li class=""><label for="ms-opt-183" style="padding-left: 21px;"><input value="societies" title="Societies" id="ms-opt-183" type="checkbox">Societies</label></li><li class=""><label for="ms-opt-184" style="padding-left: 21px;"><input value="soilsystems" title="Soil Systems" id="ms-opt-184" type="checkbox">Soil Systems</label></li><li class=""><label for="ms-opt-185" style="padding-left: 21px;"><input value="sports" title="Sports" id="ms-opt-185" type="checkbox">Sports</label></li><li class=""><label for="ms-opt-186" style="padding-left: 21px;"><input value="stats" title="Stats" id="ms-opt-186" type="checkbox">Stats</label></li><li class=""><label for="ms-opt-187" style="padding-left: 21px;"><input value="surfaces" title="Surfaces" id="ms-opt-187" type="checkbox">Surfaces</label></li><li class=""><label for="ms-opt-188" style="padding-left: 21px;"><input value="sustainability" title="Sustainability" id="ms-opt-188" type="checkbox">Sustainability</label></li><li class=""><label for="ms-opt-189" style="padding-left: 21px;"><input value="symmetry" title="Symmetry" id="ms-opt-189" type="checkbox">Symmetry</label></li><li class=""><label for="ms-opt-190" style="padding-left: 21px;"><input value="systems" title="Systems" id="ms-opt-190" type="checkbox">Systems</label></li><li class=""><label for="ms-opt-191" style="padding-left: 21px;"><input value="technologies" title="Technologies" id="ms-opt-191" type="checkbox">Technologies</label></li><li class=""><label for="ms-opt-192" style="padding-left: 21px;"><input value="toxics" title="Toxics" id="ms-opt-192" type="checkbox">Toxics</label></li><li class=""><label for="ms-opt-193" style="padding-left: 21px;"><input value="toxins" title="Toxins" id="ms-opt-193" type="checkbox">Toxins</label></li><li class=""><label for="ms-opt-194" style="padding-left: 21px;"><input value="tropicalmed" title="Tropical Medicine and Infectious Disease" id="ms-opt-194" type="checkbox">Tropical Medicine and Infectious Disease</label></li><li class=""><label for="ms-opt-195" style="padding-left: 21px;"><input value="universe" title="Universe" id="ms-opt-195" type="checkbox">Universe</label></li><li class=""><label for="ms-opt-196" style="padding-left: 21px;"><input value="urbansci" title="Urban Science" id="ms-opt-196" type="checkbox">Urban Science</label></li><li class=""><label for="ms-opt-197" style="padding-left: 21px;"><input value="vaccines" title="Vaccines" id="ms-opt-197" type="checkbox">Vaccines</label></li><li class=""><label for="ms-opt-198" style="padding-left: 21px;"><input value="vehicles" title="Vehicles" id="ms-opt-198" type="checkbox">Vehicles</label></li><li class=""><label for="ms-opt-199" style="padding-left: 21px;"><input value="vetsci" title="Veterinary Sciences" id="ms-opt-199" type="checkbox">Veterinary Sciences</label></li><li class=""><label for="ms-opt-200" style="padding-left: 21px;"><input value="vibration" title="Vibration" id="ms-opt-200" type="checkbox">Vibration</label></li><li class=""><label for="ms-opt-201" style="padding-left: 21px;"><input value="viruses" title="Viruses" id="ms-opt-201" type="checkbox">Viruses</label></li><li class=""><label for="ms-opt-202" style="padding-left: 21px;"><input value="vision" title="Vision" id="ms-opt-202" type="checkbox">Vision</label></li><li class=""><label for="ms-opt-203" style="padding-left: 21px;"><input value="water" title="Water" id="ms-opt-203" type="checkbox">Water</label></li><li class=""><label for="ms-opt-204" style="padding-left: 21px;"><input value="wevj" title="World Electric Vehicle Journal" id="ms-opt-204" type="checkbox">World Electric Vehicle Journal</label></li></ul></div></div>
<input name="email" placeholder="Your email address here..." required="required" type="email">
<button class="genericCaptcha" type="submit">Subscribe</button>
</form>
</div>
</div>
<div id="footer-copyright">
© 1996-2019 MDPI (Basel, Switzerland) unless otherwise stated
</div>
</footer>
<a class="exit-off-canvas"></a>
</div>
</div>
</div>
<a href="#" class="back-to-top">Back to Top</a>
<script src="modernizr.js"></script>
<script src="jquery-1.js"></script>
<script src="foundation-5.js"></script>
<script src="foundation-5_002.js"></script>
<script src="jquery.js"></script>
<script src="jquery_002.js"></script>
<script>
            $('select.foundation-select').multiselect({
                search: true,
                minHeight: 130,
                maxHeight: 130,
            });

            $(document).foundation({
                orbit: {
                    timer_speed: 4000,
                }
            });

            // old browser fix - this way the console log rows won't throw (silent) errors in browsers not supporting console log
            if (!window.console) window.console = {};
            if (!window.console.log) window.console.log = function () { };

            $(document).ready(function()
            {
                $(".chosen-select").chosen({ disable_search_threshold: 1, width: "100%" });

                $(".toEncode").each(function(e) {
                    var oldHref = $(this).attr("href");
                    var newHref = oldHref.replace('.botdefense.please.enable.javaScript.','@');
                    $(this).attr("href", newHref);

                    if (!$(this).hasClass("emailCaptcha"))
                    {
                        $(this).html(newHref.replace('mailto:', ''));
                    }
                    
                                    });

                $(document).on('opened.fndtn.reveal', '[data-reveal]', function()
                {
                    $(document).foundation('equalizer', 'reflow');
                });

                // fix the images that have tag height / width defined
                // otherwise the default foundation styles overwrite the tag definitions
                $("img").each(function()
                {
                   if ($(this).attr('width') != undefined || $(this).attr('height') != undefined)
                   {
                       $(this).addClass("img-fixed");
                   }
                });

                $(".hide-show-desktop-option").click(function(e) 
                {
                    e.preventDefault();
                    var parentDiv = $(this).closest("div");

                    $.ajax({
                        url: $(this).attr('href'),
                        success: function(msg)
                        {
                            parentDiv.removeClass().hide();
                        }
                    });
                });

                
                /*
                * handle whole row as a link if the row contains only one visible link
                */
                $("table.new tr").hover(function()
                {
                   if ($(this).find("td:visible a").length == 1)
                   {
                       $(this).addClass("single-link");
                   }
                },
                function()
                {
                   $(this).removeClass("single-link");
                });

                $("table.new:not(.table-of-tables)").on("click", "tr.single-link", function(e)
                {
                    var target = $(e.target);
                    if (!e.ctrlKey && !target.is("a")) {
                        $(this).find("td:visible a")[0].click();
                    }
                });
            });
            
        </script>
<script src="lib.js"></script>
<script src="mdpi.js"></script>
<script>var banners_url = 'https://serve.mdpi.com';</script>
<script type="text/javascript" src="ifvisible.js"></script>
<script src="affix.js"></script>
<script src="storage.js"></script>
<script src="jquery-scrollspy.js"></script>
<script src="magnific-popup.js"></script>
<script src="underscore.js"></script>
<script type="text/javascript">
    $('document').ready(function(){
      if(1 == 0){
        var doi = $('.html-art-header').find('a');
        var text = doi.text();
        doi.remove();
        $('.html-art-header p').append(text);
        $('.html-art-header').html($('.html-art-header')[0].innerHTML.replace(/doi:\s+/,'doi:&nbsp;'))
      }

      $("#left-column").addClass("show-for-large-up");
      $("#middle-column").removeClass("medium-9").removeClass("left-bordered").addClass("medium-12");
    });
	$(document).on('DOMNodeInserted', function(e) {

        var element = $(e.target);

        if (element.hasClass('menu') && element.hasClass('html-nav') )
        {
            element.addClass("side-menu-ul");
            element.find("li").addClass("side-menu-li");
        }
    });
  </script>
<script src="jquery-ui-1.js"></script>
<script src="articles.js"></script>
<script>
        repositionOpenSideBar = function() {
            $('#left-column').addClass("show-for-large-up").show();
            $('#middle-column').removeClass('large-12').removeClass('medium-12');
            $('#middle-column').addClass('large-80');
        }

        repositionCloseSideBar = function() {
            $('#left-column').removeClass("show-for-large-up").hide();
            $('#middle-column').removeClass('large-80');
            $('#middle-column').addClass('large-12').addClass('medium-12');
        }
    </script>
<script>
        $(document).ready(function() {

            $(document).on('keyup', function (e) {
                if (e.keyCode == 27) {

                    var hElem = $(this).find(".annotator-adder");

                    if (hElem.length){
                        hElem.css({'visibility':'hidden'});
                    } else {
                        document.querySelector("hypothesis-adder").shadowRoot.querySelector(".annotator-adder").style.visibility = "hidden";
                    }

                }
            });
        });
    </script>
<script>
            window.hypothesisConfig = function () {
                return {
                    sidebarAppUrl: 'https://commenting.mdpi.com/app.html',
                    showHighlights:   'whenSidebarOpen'  ,
                    openSidebar:  false  ,
                    assetRoot: 'https://commentingres.mdpi.com/hypothesis',
                    services: [{
                        apiUrl: 'https://commenting.mdpi.com/api/',
                        authority: 'sciforum',
                        grantToken: '',
                        doi: '10.3390/rs5094163'
                    }],
                };
            };
            MathJax . Hub . Startup . signal . Interest ( function ( message ) {
                if ( message == 'End' ){
                var hypoLink = document . getElementById ( "hypothesis_frame" )
                hypoLink . setAttribute ( "src" , "https://commenting.mdpi.com/embed.js" )
                }
});
        </script>
<script async="" id="hypothesis_frame"></script>
<!--[if lt IE 9]>
                        <script src="https://www.mdpi.com/assets/js/ie8/ie8.js?6eef8fcbc831f5bd"></script>
            <script src="https://www.mdpi.com/assets/js/ie8/jquery.xdomainrequest.min.js?a945caca315782b0"></script>
        <![endif]-->


</div></body></html>